{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "cbbc7572",
   "metadata": {},
   "source": [
    "# This is a Modeller Batch Script\n",
    "## Script made under guidance of Rawal Genomics Team\n",
    "### Script made by Prateek Paul\n",
    "Source: https://salilab.org/modeller/tutorial/basic.html"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2e07ffe0",
   "metadata": {},
   "source": [
    "#### #pip install Bio\n",
    "##### Install Modeller from https://salilab.org/modeller/download_installation.html\n",
    "##### Also download suplementary files and put them in same folder https://salilab.org/modeller/supplemental.html\n",
    "##### licence key https://salilab.org/modeller/registration.html\n",
    "#### code references https://salilab.org/modeller/tutorial/basic.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "be5c7ce5",
   "metadata": {},
   "outputs": [],
   "source": [
    "#importing all modelues\n",
    "import os\n",
    "import glob\n",
    "import Bio\n",
    "import urllib\n",
    "import shutil, sys \n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "from Bio.PDB import *\n",
    "from Bio import SeqIO\n",
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "#sys.stdout = open(\"dump.txt\", \"w\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "0ac4df9a",
   "metadata": {},
   "outputs": [],
   "source": [
    "#creating folder so that main folder looks clean\n",
    "def folder():\n",
    "    try:\n",
    "        os.mkdir('Ready')\n",
    "        os.mkdir('ali')\n",
    "        os.mkdir('buildprofile')\n",
    "        os.mkdir('PDB')\n",
    "    except:\n",
    "        print('folder exists')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "875173f9",
   "metadata": {},
   "outputs": [],
   "source": [
    "#generating main sequence ali files\n",
    "def aligen(ID, Seq):\n",
    "    ids,v=ID.split('.')\n",
    "    file1 = open(\"ali/\"+ids+\".ali\", \"w\")\n",
    "    file1.write(\">P1;\"+ids+\"\\n\")\n",
    "    file1.write(\"sequence:\"+ids+\":::::::0.00: 0.00\"+\"\\n\")\n",
    "    file1.write(Seq+\"*\")\n",
    "    file1.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "3ac283f4",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Searching for structures related to the main sequence \n",
    "def build_profile(ID):\n",
    "    ids,v=ID.split('.')\n",
    "    log.verbose()\n",
    "    env = Environ()\n",
    "    sdb = SequenceDB(env)\n",
    "    sdb.read(seq_database_file='pdb_95.pir', seq_database_format='PIR',\n",
    "             chains_list='ALL', minmax_db_seq_len=(30, 4000), clean_sequences=True)\n",
    "    sdb.write(seq_database_file='pdb_95.bin', seq_database_format='BINARY',\n",
    "              chains_list='ALL')\n",
    "    sdb.read(seq_database_file='pdb_95.bin', seq_database_format='BINARY',\n",
    "             chains_list='ALL')\n",
    "    aln = Alignment(env)\n",
    "    aln.append(file=\"ali/\"+ids+\".ali\", alignment_format='PIR', align_codes='ALL')\n",
    "    prf = aln.to_profile()\n",
    "    prf.build(sdb, matrix_offset=-450, rr_file='${LIB}/blosum62.sim.mat',\n",
    "              gap_penalties_1d=(-500, -50), n_prof_iterations=1,\n",
    "              check_profile=False, max_aln_evalue=0.01)\n",
    "    prf.write(file=\"buildprofile/\"+ids+'build_profile.prf', profile_format='TEXT')\n",
    "    aln = prf.to_alignment()\n",
    "    aln.write(file=\"buildprofile/\"+ids+'build_profile.ali', alignment_format='PIR')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "ce61cb5d",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Downloading the selected pdb files required for the templetes\n",
    "def pdbDn(pdbid):\n",
    "    urllib.request.urlretrieve('http://files.rcsb.org/download/'+pdbid+'.pdb', pdbid+'.pdb')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "a0576529",
   "metadata": {},
   "outputs": [],
   "source": [
    "#comparing the downloaded pdb\n",
    "#presently this step is being skipped \n",
    "#and the code is taking the top hit for the pdb \n",
    "def compare(listpdb,ID):\n",
    "    ids,v=ID.split('.')\n",
    "    env = Environ()\n",
    "    aln = Alignment(env)\n",
    "    for (pdb, chain) in (listpdb):\n",
    "        m = Model(env, file=pdb, model_segment=('FIRST:'+chain, 'LAST:'+chain))\n",
    "        aln.append_model(m, atom_files=pdb, align_codes=pdb+chain)\n",
    "    aln.malign()\n",
    "    aln.malign3d()\n",
    "    aln.compare_structures()\n",
    "    aln.id_table(matrix_file='family.mat')\n",
    "    env.dendrogram(matrix_file='family.mat', cluster_cut=-1.0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "c49b1b76",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Aligning Main seq with the template\n",
    "def align2d(pdbid,chain,ID):\n",
    "    ids,v=ID.split('.')\n",
    "    env = Environ()\n",
    "    aln = Alignment(env)\n",
    "    mdl = Model(env, file=pdbid, model_segment=('FIRST:A','LAST:A'))\n",
    "    aln.append_model(mdl, align_codes=pdbid+chain, atom_files=pdbid+'.pdb')\n",
    "    aln.append(file=\"ali/\"+ids+'.ali', align_codes=ids)\n",
    "    aln.align2d()\n",
    "    aln.write(file=\"ali/\"+ids+'-'+pdbid+chain+'.ali', alignment_format='PIR')\n",
    "    aln.write(file=\"ali/\"+ids+'-'+pdbid+chain+'.pap', alignment_format='PAP')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "6327cfbf",
   "metadata": {},
   "outputs": [],
   "source": [
    "#building the profile for the selected pdb to main seq\n",
    "def prf(ids):\n",
    "        a_file = open(\"buildprofile/\"+ids+'build_profile.prf', \"r\")\n",
    "        lines = a_file.readlines()\n",
    "        a_file.close()\n",
    "        line=lines[6:]\n",
    "        f = open(\"buildprofile/\"+ids+'build_profile.prf', \"w\")\n",
    "        #f.write('sno \\t id \\t l1 \\t l2')\n",
    "        for x in line:\n",
    "            for z in x.split():\n",
    "                f.write(z+'\\t')\n",
    "            f.write('\\n')\n",
    "        f.close()\n",
    "        df = pd.read_csv(\"buildprofile/\"+ids+'build_profile.prf', sep='\\t', header=None)\n",
    "        #return (len(df))\n",
    "        df=df.sort_values(by=([10,11]), ascending=(False,False))\n",
    "        df=df.reset_index()\n",
    "        identity = df[10]\n",
    "        #y=identity[1]\n",
    "        if len(df) > 0: \n",
    "            pdbids=df[1]\n",
    "            if len(df) > 1:\n",
    "                x=pdbids[1]\n",
    "                pdbDn(x[:4])\n",
    "            else:\n",
    "                x = 'cannot'\n",
    "        else:\n",
    "            x = 'cannot'        \n",
    "        return (len(df)), identity,x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "d3055ad5",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Model building\n",
    "def model(ali,ID,pdbid):\n",
    "    ids,v=ID.split('.')\n",
    "    env = Environ()\n",
    "    a = AutoModel(env, alnfile=ali,\n",
    "                  knowns=pdbid, sequence=ids,\n",
    "                  assess_methods=(assess.DOPE,\n",
    "                                  assess.GA341))\n",
    "    a.starting_model = 1\n",
    "    a.ending_model = 5\n",
    "    a.make()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "732eee48",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Model evaluation and removing of the other files\n",
    "def eva(ids):\n",
    "    dope = 0.0\n",
    "    for file in glob.glob(ids+\".*.pdb\"):\n",
    "        fh = open(file)\n",
    "        for each in fh:\n",
    "            if \"REMARK   6 DOPE score:\" in each:\n",
    "                b = each.split(' ')\n",
    "                r = b[6].strip()\n",
    "                if float(r) < float(dope):\n",
    "                    dope=b[6]\n",
    "                    mainfile=file\n",
    "        fh.close()\n",
    "    pdbs=glob.glob(ids+\".*.pdb\")\n",
    "    shutil.move(mainfile, 'Ready')\n",
    "    for pdbmod in glob.glob(ids+\".*\"):\n",
    "        os.remove(pdbmod)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "e23f3636",
   "metadata": {},
   "outputs": [],
   "source": [
    "def main():\n",
    "    folder()\n",
    "    \n",
    "    with open(\"AAHK01.fasta\") as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            def model(ali,ID,pdbid):\n",
    "                ids,v=ID.split('.')\n",
    "                env = Environ()\n",
    "                a = AutoModel(env, alnfile=ali,\n",
    "                              knowns=pdbid, sequence=ids,\n",
    "                              assess_methods=(assess.DOPE,\n",
    "                                              assess.GA341))\n",
    "                a.starting_model = 1\n",
    "                a.ending_model = 5\n",
    "                a.make()\n",
    "            \n",
    "            \n",
    "            ID=record.id\n",
    "            Seq=record.seq\n",
    "            Seq=str(Seq)\n",
    "            ids,v=ID.split('.')\n",
    "            aligen(ID, Seq)\n",
    "\n",
    "            build_profile(ID)\n",
    "            \n",
    "            length,identity,x=prf(ids)\n",
    "            \n",
    "            if x == 'cannot':\n",
    "                print('next file')\n",
    "            else:\n",
    "                if length > 1:\n",
    "                    if int(identity[1]) > 34:\n",
    "                        align2d(x[:4],x[4:5],ID)\n",
    "                        path = 'ali/'+ids+'-'+x+'.ali'\n",
    "                        model(path,ID,x)\n",
    "                        eva(ids)\n",
    "                        \n",
    "    #pdbsall=glob.glob(\"*.pdb\")\n",
    "    for pdbsall in glob.glob(\"*.pdb\"):\n",
    "        shutil.move(pdbsall, 'PDB')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "9b745ab6",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "folder exists\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       191566     187.076     0.183\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       192094     187.592     0.183\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       240694     235.053     0.230\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       253994     248.041     0.242\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       523260     510.996     0.499\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      4084383    3988.655     3.895\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4086625    3990.845     3.897\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4088075    3992.261     3.899\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4090975    3995.093     3.901\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4096775    4000.757     3.907\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     17535174   17124.193    16.723\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17546774   17135.521    16.734\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17569974   17158.178    16.756\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17616374   17203.490    16.800\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17709174   17294.115    16.889\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     24428374   23855.834    23.297\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00183.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       525580     513.262     0.501\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       527030     514.678     0.503\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       529930     517.510     0.505\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       535730     523.174     0.511\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       547330     534.502     0.522\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       549938     537.049     0.524\n",
      "\n",
      "Read the alignment from file       : ali/EAO00183.ali\n",
      "\n",
      "Total number of alignment positions:   653\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00183     653      1    EAO00183\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:       552883     539.925     0.527\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       878058     857.479     0.837\n",
      "profile_iteration_> processing sequence:       1    653      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       879136     858.531     0.838\n",
      "profile_iteration_> processing sequence:    6720    653      1     6.3437500     0.0009440     0.0000036  1059\n",
      "profile_iteration_> processing sequence:   13439    653      1    12.4687500     0.0009278     0.0000036  1078\n",
      "profile_iteration_> processing sequence:   20158    653      1    18.5625000     0.0009209     0.0000036  1086\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       879511     858.897     0.839\n",
      "profile_iteration_> processing sequence:   26877    653      1    24.7031250     0.0009191     0.0000036  1088\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       879747     859.128     0.839\n",
      "profile_iteration_> processing sequence:   33596    653      1    30.9218750     0.0009204     0.0000036  1086\n",
      "profile_iteration_> processing sequence:   40315    653      1    37.0000000     0.0009178     0.0000036  1090\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       882697     862.009     0.842\n",
      "profile_iteration_> processing sequence:   47034    653      1    43.2343750     0.0009192     0.0000036  1088\n",
      "profile_iteration_> processing sequence:   53753    653      1    49.5625000     0.0009220     0.0000036  1085\n",
      "profile_iteration_> processing sequence:   60472    653      1    55.7031250     0.0009211     0.0000036  1086\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       886207     865.437     0.845\n",
      "profile_iteration_> processing sequence:   67191    653      1    62.0937500     0.0009241     0.0000036  1082\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    28.08501\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    270.7707\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0289,      121,        0.0106,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00015   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00074   0.00004\n",
      "Z:         1  -2.35000   0.00089   0.00016\n",
      "Z:         1  -2.25000   0.00149   0.00055\n",
      "Z:         1  -2.15000   0.00342   0.00163\n",
      "Z:         1  -2.05000   0.00551   0.00416\n",
      "Z:         1  -1.95000   0.01146   0.00933\n",
      "Z:         1  -1.85000   0.02456   0.01872\n",
      "Z:         1  -1.75000   0.04301   0.03400\n",
      "Z:         1  -1.65000   0.06742   0.05658\n",
      "Z:         1  -1.55000   0.09153   0.08721\n",
      "Z:         1  -1.45000   0.11847   0.12565\n",
      "Z:         1  -1.35000   0.19050   0.17059\n",
      "Z:         1  -1.25000   0.21744   0.21981\n",
      "Z:         1  -1.15000   0.27325   0.27052\n",
      "Z:         1  -1.05000   0.29736   0.31973\n",
      "Z:         1  -0.95000   0.33814   0.36469\n",
      "Z:         1  -0.85000   0.40451   0.40316\n",
      "Z:         1  -0.75000   0.39975   0.43360\n",
      "Z:         1  -0.65000   0.49470   0.45519\n",
      "Z:         1  -0.55000   0.43338   0.46779\n",
      "Z:         1  -0.45000   0.51271   0.47182\n",
      "Z:         1  -0.35000   0.43160   0.46811\n",
      "Z:         1  -0.25000   0.46062   0.45775\n",
      "Z:         1  -0.15000   0.39082   0.44196\n",
      "Z:         1  -0.05000   0.40496   0.42194\n",
      "Z:         1   0.05000   0.38234   0.39889\n",
      "Z:         1   0.15000   0.38427   0.37383\n",
      "Z:         1   0.25000   0.34528   0.34769\n",
      "Z:         1   0.35000   0.32474   0.32121\n",
      "Z:         1   0.45000   0.32296   0.29500\n",
      "Z:         1   0.55000   0.26953   0.26953\n",
      "Z:         1   0.65000   0.25494   0.24513\n",
      "Z:         1   0.75000   0.21535   0.22205\n",
      "Z:         1   0.85000   0.20985   0.20043\n",
      "Z:         1   0.95000   0.17993   0.18036\n",
      "Z:         1   1.05000   0.17190   0.16185\n",
      "Z:         1   1.15000   0.15195   0.14489\n",
      "Z:         1   1.25000   0.13335   0.12944\n",
      "Z:         1   1.35000   0.12129   0.11542\n",
      "Z:         1   1.45000   0.10210   0.10275\n",
      "Z:         1   1.55000   0.08855   0.09133\n",
      "Z:         1   1.65000   0.08290   0.08109\n",
      "Z:         1   1.75000   0.07635   0.07191\n",
      "Z:         1   1.85000   0.06846   0.06371\n",
      "Z:         1   1.95000   0.06266   0.05639\n",
      "Z:         1   2.05000   0.05507   0.04988\n",
      "Z:         1   2.15000   0.04361   0.04409\n",
      "Z:         1   2.25000   0.03557   0.03895\n",
      "Z:         1   2.35000   0.03393   0.03439\n",
      "Z:         1   2.45000   0.02902   0.03035\n",
      "Z:         1   2.55000   0.02872   0.02678\n",
      "Z:         1   2.65000   0.02441   0.02361\n",
      "Z:         1   2.75000   0.02128   0.02082\n",
      "Z:         1   2.85000   0.01533   0.01835\n",
      "Z:         1   2.95000   0.01682   0.01617\n",
      "Z:         1   3.05000   0.01488   0.01424\n",
      "Z:         1   3.15000   0.01235   0.01255\n",
      "Z:         1   3.25000   0.01339   0.01105\n",
      "Z:         1   3.35000   0.00863   0.00973\n",
      "Z:         1   3.45000   0.00923   0.00857\n",
      "Z:         1   3.55000   0.00685   0.00754\n",
      "Z:         1   3.65000   0.00580   0.00664\n",
      "Z:         1   3.75000   0.00685   0.00584\n",
      "Z:         1   3.85000   0.00387   0.00514\n",
      "Z:         1   3.95000   0.00655   0.00453\n",
      "Z:         1   4.05000   0.00446   0.00398\n",
      "Z:         1   4.15000   0.00417   0.00350\n",
      "Z:         1   4.25000   0.00283   0.00308\n",
      "Z:         1   4.35000   0.00298   0.00271\n",
      "Z:         1   4.45000   0.00208   0.00239\n",
      "Z:         1   4.55000   0.00134   0.00210\n",
      "Z:         1   4.65000   0.00223   0.00185\n",
      "Z:         1   4.75000   0.00238   0.00163\n",
      "Z:         1   4.85000   0.00149   0.00143\n",
      "Z:         1   4.95000   0.00149   0.00126\n",
      "Z:         1   5.05000   0.00104   0.00111\n",
      "Z:         1   5.15000   0.00149   0.00097\n",
      "Z:         1   5.25000   0.00193   0.00086\n",
      "Z:         1   5.35000   0.00134   0.00075\n",
      "Z:         1   5.45000   0.00060   0.00066\n",
      "Z:         1   5.55000   0.00045   0.00058\n",
      "Z:         1   5.65000   0.00045   0.00051\n",
      "Z:         1   5.75000   0.00074   0.00045\n",
      "Z:         1   5.85000   0.00015   0.00040\n",
      "Z:         1   5.95000   0.00045   0.00035\n",
      "Z:         1   6.05000   0.00030   0.00031\n",
      "Z:         1   6.15000   0.00000   0.00027\n",
      "Z:         1   6.25000   0.00000   0.00024\n",
      "Z:         1   6.35000   0.00045   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00000   0.00016\n",
      "Z:         1   6.65000   0.00015   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00045   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00015   0.00006\n",
      "Z:         1   7.45000   0.00030   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      1425607    1392.194     1.360\n",
      "> 6ql4A                      1    2060   20650     660     653   40.07     0.0           2   271    10   310    14   290\n",
      "> 6vjfD                      1    6389   42150     348     653   52.35     0.0           3   306     4   309    12   330\n",
      "> 3l43A                      1    9090   35050     242     653   61.41     0.0           4   241    28   283     1   241\n",
      "> 3snhA                      1   12012   46500     652     653   39.44     0.0           5   595     1   641     1   644\n",
      "> 3zvrA                      1   13410   52750     669     653   40.12     0.0           6   618     1   641     2   664\n",
      "> 6di7A                      1   15154   38950     279     653   55.91     0.0           7   267    25   291     1   279\n",
      "> 3t34A                      1   16250   35800     335     653   52.96     0.0           8   286    25   311    28   314\n",
      "> 4p4sA                      1   18812   25550     301     653   43.22     0.0           9   266    26   294    12   284\n",
      "> 4p4uA                      1   18813   24350     338     653   38.82     0.0          10   298    10   319    11   314\n",
      "> 6fgzA                      1   21305   45600     526     653   41.14     0.0          11   524     1   645     1   525\n",
      "> 6defA                      1   23259   43150     367     653   52.94     0.0          12   310     4   313    13   335\n",
      "> 6defD                      1   23260   42200     336     653   53.70     0.0          13   299     4   302    13   323\n",
      "> 1jx2A                      1   26896   39250    1039     653   58.36     0.0          14   277     1   302   758  1038\n",
      "> 5gtmA                      1   34494   21200     520     653   30.36     0.0          15   441    10   512     7   454\n",
      "> 4bejA                      1   34848   47800     540     653   44.05     0.0          16   538     1   645     1   538\n",
      "> 4bejC                      1   34849   49100     534     653   45.13     0.0          17   532     1   648     1   534\n",
      "> 5a3fA                      1   38342   53750     691     653   40.20     0.0          18   625     1   641     1   684\n",
      "> 5a3fB                      1   38343   50100     555     653   45.45     0.0          19   543     1   641     1   550\n",
      "> 4whjA                      1   38855   22900     565     653   30.49     0.0          20   538    10   645    10   560\n",
      "> 2akaB                      1   40363   42100     299     653   58.19     0.0          21   298     1   302     1   299\n",
      "> 6jsjA                      1   44242   18650     625     653   36.95     0.0          22   295     1   330     8   302\n",
      "> 4h1uA                      1   52952   41100     358     653   51.39     0.0          23   310     1   313     1   323\n",
      "> 2x2eA                      1   53693   42550     341     653   56.47     0.0          24   316     1   325     1   317\n",
      "> 3szrA                      1   54739   25350     559     653   30.43     0.0          25   478    10   512    10   492\n",
      "> 5d3qA                      1   61954   42550     328     653   56.96     0.0          26   308     1   312     2   310\n",
      "> 6jtgA                      1   63712   13250     330     653   32.44     0.0          27   247     4   259     2   263\n",
      "> 3w6pA                      1   64354   41450     333     653   53.21     0.0          28   308     1   315     3   314\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      1429725    1396.216     1.363\n",
      "> 6djqB                      1   65638   41200     330     653   56.57     0.0          29   296     4   318    12   308\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    62.093750\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000924\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1085.383179\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.140625\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000927\n",
      "profile_iteration__> Overall performance [seq/sec]:  1078.306885\n",
      "openf___224_> Open           buildprofile/EAO00183build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1292632    1262.336     1.233\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1295240    1264.883     1.235\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1296320    1265.938     1.236\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1297540    1267.129     1.237\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1298500    1268.066     1.238\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1300876    1270.387     1.241\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1303344    1272.797     1.243\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1304408    1273.836     1.244\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1305548    1274.949     1.245\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1306608    1275.984     1.246\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1307796    1277.145     1.247\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1309888    1279.188     1.249\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1311124    1280.395     1.250\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1312316    1281.559     1.252\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1313420    1282.637     1.253\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1315180    1284.355     1.254\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1317328    1286.453     1.256\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1319452    1288.527     1.258\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1321948    1290.965     1.261\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1324116    1293.082     1.263\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1326264    1295.180     1.265\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1327452    1296.340     1.266\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1328628    1297.488     1.267\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1329864    1298.695     1.268\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1331124    1299.926     1.269\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1333032    1301.789     1.271\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1334260    1302.988     1.272\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1335244    1303.949     1.273\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1336472    1305.148     1.275\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1337652    1306.301     1.276\n",
      "openf___224_> Open           buildprofile/EAO00183build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       191566     187.076     0.183\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       192094     187.592     0.183\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       240694     235.053     0.230\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       253994     248.041     0.242\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           1jx2.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       266204     259.965     0.254\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       266280     260.039     0.254\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       266451     260.206     0.254\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       267301     261.036     0.255\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       268559     262.265     0.256\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       270463     264.124     0.258\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       273319     266.913     0.261\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       273359     266.952     0.261\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       273449     267.040     0.261\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       277733     271.224     0.265\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       277793     271.282     0.265\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       277928     271.414     0.265\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       284354     277.689     0.271\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       284442     277.775     0.271\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       284640     277.969     0.271\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       284772     278.098     0.272\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       285069     278.388     0.272\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       294691     287.784     0.281\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       294891     287.979     0.281\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       295341     288.419     0.282\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       309791     302.530     0.295\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       310091     302.823     0.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       310766     303.482     0.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       332424     324.633     0.317\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       332872     325.070     0.317\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       333880     326.055     0.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       366384     357.797     0.349\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       367056     358.453     0.350\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       368568     359.930     0.351\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       417324     407.543     0.398\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       418332     408.527     0.399\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       420600     410.742     0.401\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       493734     482.162     0.471\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       495246     483.639     0.472\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       498648     486.961     0.476\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       608332     594.074     0.580\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1391020    1358.418     1.327\n",
      "read_mo_297_> Segments, residues, atoms:        1     1039     8297\n",
      "read_mo_298_> Segment:        1  13:A  1091:A     8297\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1421122    1387.814     1.355\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2028439    1980.897     1.934\n",
      "openf___224_> Open           ali/EAO00183.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2335928    2281.180     2.228\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2338536    2283.727     2.230\n",
      "\n",
      "Read the alignment from file       : ali/EAO00183.ali\n",
      "\n",
      "Total number of alignment positions:  1039\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      1jx2A    1039      1    1jx2.pdb MOL_ID  1; MOLECULE  MYOSIN-2 HEAVY CHAIN,DYNAMIN-A; CHAIN  A; FRAGMENT  CATALYT\n",
      "  2   EAO00183     653      1    EAO00183\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2339460    2284.629     2.231\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :         1042\n",
      "  Score                    :  339086.4062\n",
      "openf___224_> Open           ali/EAO00183-1jx2A.ali\n",
      "openf___224_> Open           ali/EAO00183-1jx2A.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       191566     187.076     0.183\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       192094     187.592     0.183\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       240694     235.053     0.230\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       253994     248.041     0.242\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00183-1jx2A.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       265959     259.726     0.254\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       267409     261.142     0.255\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       270309     263.974     0.258\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       276109     269.638     0.263\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       287709     280.966     0.274\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       310909     303.622     0.297\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       774917     756.755     0.739\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       779069     760.810     0.743\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       781677     763.356     0.745\n",
      "\n",
      "Read the alignment from file       : ali/EAO00183-1jx2A.ali\n",
      "\n",
      "Total number of alignment positions:  1042\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      1jx2A    1039      1    1jx2.pdb MOL_ID  1; MOLECULE  MYOSIN-2 HEAVY CHAIN,DYNAMIN-A; CHAIN  A; FRAGMENT  CATALYT\n",
      "  2   EAO00183     653      1    EAO00183\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           1jx2.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1095102    1069.436     1.044\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1095102    1069.436     1.044\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1095273    1069.603     1.045\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1096123    1070.433     1.045\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1097381    1071.661     1.047\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1099285    1073.521     1.048\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1102141    1076.310     1.051\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1102141    1076.310     1.051\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1102231    1076.397     1.051\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1106515    1080.581     1.055\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1106515    1080.581     1.055\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1106650    1080.713     1.055\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1113076    1086.988     1.062\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1113076    1086.988     1.062\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1113274    1087.182     1.062\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1113274    1087.182     1.062\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1113571    1087.472     1.062\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1123193    1096.868     1.071\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1123193    1096.868     1.071\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1123643    1097.308     1.072\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1138093    1111.419     1.085\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1138093    1111.419     1.085\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1138768    1112.078     1.086\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1160426    1133.229     1.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1160426    1133.229     1.107\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1161434    1134.213     1.108\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1193938    1165.955     1.139\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1193938    1165.955     1.139\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1195450    1167.432     1.140\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1244206    1215.045     1.187\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1244206    1215.045     1.187\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1246474    1217.260     1.189\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1319608    1288.680     1.258\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1319608    1288.680     1.258\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1323010    1292.002     1.262\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432694    1399.115     1.366\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1431830    1398.271     1.365\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    195     1  212   219      K     T    8.195\n",
      "    760     1  781   786      V     N   10.441\n",
      "    859     1  892   901      D     M    9.547\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1571082    1534.260     1.498\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1572682    1535.822     1.500\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1576662    1539.709     1.504\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1578662    1541.662     1.506\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1581662    1544.592     1.508\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1586162    1548.986     1.513\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1591734    1554.428     1.518\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1594534    1557.162     1.521\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1598734    1561.264     1.525\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1605034    1567.416     1.531\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1607806    1570.123     1.533\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1609206    1571.490     1.535\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1711570    1671.455     1.632\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2213722    2161.838     2.111\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2288082    2234.455     2.182\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      653   5105   5179      0         0      2058\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2502438    2443.787     2.387\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      653   5105   5179   6984      8161      2058\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2731454    2667.436     2.605\n",
      "patch_______> segment topology patched using RTF:  653 ;  LEU  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      653   5106   5180   6986      8163      2059\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      653   5106   5180   6986      8163      2059\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     3931     1175\n",
      "openf___224_> Open           EAO00183.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      653     5106     5106\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      2838903    2772.366     2.707\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      2847095    2780.366     2.715\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      2863479    2796.366     2.731\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      2896247    2828.366     2.762\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      2929015    2860.366     2.793\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      2994551    2924.366     2.856\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3027319    2956.366     2.887\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3092855    3020.366     2.950\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3125623    3052.366     2.981\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3256695    3180.366     3.106\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3322231    3244.366     3.168\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3387767    3308.366     3.231\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3518839    3436.366     3.356\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3780983    3692.366     3.606\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      3912055    3820.366     3.731\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      4043127    3948.366     3.856\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      4305271    4204.366     4.106\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      4829559    4716.366     4.606\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5091703    4972.366     4.856\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     5180    6986    7724    2035\n",
      "              Total number of restraints before, now         :        0    21925\n",
      "make_re_422_> Number of previous, current restraints         :        0    21925\n",
      "make_re_423_> Number of previous, current selected restraints:        0    21925\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5466119    5338.007     5.213\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       17  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       18  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       17  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       18  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       17  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       18  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       17  LYS\n",
      "getdata_643_> Protein accepted:  1jx2A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    21925    22576\n",
      "make_re_423_> Number of previous, current selected restraints:    21925    22576\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5466135    5338.022     5.213\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  1jx2A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      676\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    22576    23228\n",
      "make_re_423_> Number of previous, current selected restraints:    22576    22552\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5466135    5338.022     5.213\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  1jx2A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5728279    5594.022     5.463\n",
      "make_re_422_> Number of previous, current restraints         :    23228    23798\n",
      "make_re_423_> Number of previous, current selected restraints:    22552    23122\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5728279    5594.022     5.463\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  1jx2A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    23798    24235\n",
      "make_re_423_> Number of previous, current selected restraints:    23122    23559\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5728279    5594.022     5.463\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  1jx2A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    24235    24441\n",
      "make_re_423_> Number of previous, current selected restraints:    23559    23765\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5728279    5594.022     5.463\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  1jx2A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    24441    24523\n",
      "make_re_423_> Number of previous, current selected restraints:    23765    23847\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:      7198771    7030.050     6.865\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7723059    7542.050     7.365\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8247347    8054.050     7.865\n",
      "make_re_422_> Number of previous, current restraints         :    24523    34873\n",
      "make_re_423_> Number of previous, current selected restraints:    23847    34197\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9295923    9078.050     8.865\n",
      "make_re_422_> Number of previous, current restraints         :    34873    45174\n",
      "make_re_423_> Number of previous, current selected restraints:    34197    44498\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9820211    9590.050     9.365\n",
      "make_re_422_> Number of previous, current restraints         :    45174    50505\n",
      "make_re_423_> Number of previous, current selected restraints:    44498    49829\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    50505    51281\n",
      "make_re_423_> Number of previous, current selected restraints:    49829    50605\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    51281    51281\n",
      "make_re_423_> Number of previous, current selected restraints:    50605    50605\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    51281    51281\n",
      "make_re_423_> Number of previous, current selected restraints:    50605    50605\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    51281    51281\n",
      "make_re_423_> Number of previous, current selected restraints:    50605    50605\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    51281    51281\n",
      "make_re_423_> Number of previous, current selected restraints:    50605    50605\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    51281    51281\n",
      "make_re_423_> Number of previous, current selected restraints:    50605    50605\n",
      "rmdupl__427_>     4029 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    51281    46576\n",
      "openf___224_> Open           EAO00183.rsr\n",
      "openf___224_> Open           EAO00183.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9325101    9106.544     8.893\n",
      "rdcsr2__307_> Number of restraints read    :    46576\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    46576    46576\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00183.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      9325133    9106.575     8.893\n",
      "read_mo_297_> Segments, residues, atoms:        1      653     5106\n",
      "read_mo_298_> Segment:        1  1:A  653:A     5106\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5106     5106        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    27134\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    30588\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    32747\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    34959\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    37150\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    38832\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    40673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    42690\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    43535\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45692\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9740249    9511.962     9.289\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45976\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "iupac_m_485_> OD1/2 will be swapped:      138.5471       50  50\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9053       61  61\n",
      "iupac_m_486_> OE1/2 will be swapped:      -99.1411       78  78\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.9228       82  82\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.5744       93  93\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.3759       99  99\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.3687      115  115\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4887      116  116\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.3294      123  123\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      162.7173      151  151\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4918      160  160\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.0962      178  178\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.7969      192  192\n",
      "iupac_m_487_> NH1/2 swapped:      179.3804      197  197\n",
      "iupac_m_485_> OD1/2 will be swapped:      135.9160      209  209\n",
      "iupac_m_487_> NH1/2 swapped:      179.5223      226  226\n",
      "iupac_m_487_> NH1/2 swapped:      179.7643      235  235\n",
      "iupac_m_485_> OD1/2 will be swapped:       93.5733      242  242\n",
      "iupac_m_486_> OE1/2 will be swapped:       92.5214      251  251\n",
      "iupac_m_485_> OD1/2 will be swapped:      163.3716      252  252\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.4966      253  253\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -102.2491      256  256\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.2319      257  257\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.6584      269  269\n",
      "iupac_m_485_> OD1/2 will be swapped:      128.0209      289  289\n",
      "iupac_m_486_> OE1/2 will be swapped:      165.5213      293  293\n",
      "iupac_m_485_> OD1/2 will be swapped:     -174.4947      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      150.8073      310  310\n",
      "iupac_m_485_> OD1/2 will be swapped:      101.6091      316  316\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.8781      333  333\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -169.7054      347  347\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.4227      360  360\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      107.0452      362  362\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.1239      364  364\n",
      "iupac_m_487_> NH1/2 swapped:      179.8830      410  410\n",
      "iupac_m_487_> NH1/2 swapped:      178.7980      415  415\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.4162      417  417\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.5600      418  418\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.9908      426  426\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      162.0199      436  436\n",
      "iupac_m_485_> OD1/2 will be swapped:      129.2827      443  443\n",
      "iupac_m_487_> NH1/2 swapped:     -177.7322      457  457\n",
      "iupac_m_487_> NH1/2 swapped:     -177.2568      472  472\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.6389      478  478\n",
      "iupac_m_487_> NH1/2 swapped:     -178.2489      495  495\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.4591      497  497\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -113.3304      530  530\n",
      "iupac_m_485_> OD1/2 will be swapped:      129.8012      567  567\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.6534      570  570\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.1918      573  573\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.7479      582  582\n",
      "iupac_m_487_> NH1/2 swapped:      179.1644      597  597\n",
      "iupac_m_486_> OE1/2 will be swapped:     -159.4503      598  598\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.9621      599  599\n",
      "iupac_m_487_> NH1/2 swapped:      179.9093      603  603\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.9533      611  611\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.1701      620  620\n",
      "iupac_m_486_> OE1/2 will be swapped:     -118.1124      623  623\n",
      "iupac_m_486_> OE1/2 will be swapped:      -97.3916      643  643\n",
      "iupac_m_486_> OE1/2 will be swapped:     -116.8301      646  646\n",
      "openf___224_> Open           ${LIB}/atmcls-mf.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9740677    9512.380     9.289\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9740741    9512.442     9.289\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9740799    9512.499     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9740863    9512.562     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9740929    9512.626     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9740993    9512.688     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9741149    9512.841     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9741277    9512.966     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9741571    9513.253     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9741763    9513.440     9.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9742267    9513.933     9.291\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9742523    9514.183     9.291\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9743519    9515.155     9.292\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9743903    9515.530     9.293\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9744479    9516.093     9.293\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9746513    9518.079     9.295\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9747409    9518.954     9.296\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9751861    9523.302     9.300\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9753205    9524.614     9.301\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9762823    9534.007     9.311\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9764807    9535.944     9.312\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9785453    9556.106     9.332\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9788461    9559.044     9.335\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9834427    9603.933     9.379\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9838907    9608.308     9.383\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9940127    9707.155     9.480\n",
      "openf___224_> Open           ${LIB}/dist-mf.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9940523    9707.542     9.480\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9941319    9708.319     9.481\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9942115    9709.097     9.482\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9943303    9710.257     9.483\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9943703    9710.647     9.483\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9944303    9711.233     9.484\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9945203    9712.112     9.484\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9946551    9713.429     9.486\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9946751    9713.624     9.486\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9948775    9715.601     9.488\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9949075    9715.894     9.488\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9952111    9718.858     9.491\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9952559    9719.296     9.491\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9957111    9723.741     9.496\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9957783    9724.397     9.496\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9964611    9731.065     9.503\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9965619    9732.050     9.504\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9975863    9742.054     9.514\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9976463    9742.640     9.514\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9976863    9743.030     9.515\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9978375    9744.507     9.516\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9993739    9759.511     9.531\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9994639    9760.390     9.532\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9995239    9760.976     9.532\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      9997507    9763.190     9.534\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10020555    9785.698     9.556\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10021899    9787.011     9.558\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10022799    9787.890     9.558\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10026203    9791.214     9.562\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10060775    9824.976     9.595\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10062791    9826.944     9.597\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10064139    9828.261     9.598\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10069243    9833.245     9.603\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10121099    9883.886     9.652\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10124123    9886.839     9.655\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10126147    9888.815     9.657\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10133803    9896.292     9.664\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10211587    9972.253     9.739\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10216123    9976.683     9.743\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10219159    9979.647     9.746\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10230643    9990.862     9.757\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10347319   10104.804     9.868\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10354123   10111.448     9.874\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10358675   10115.894     9.879\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10375903   10132.718     9.895\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10550919   10303.632    10.062\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10561131   10313.604    10.072\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10567959   10320.272    10.078\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10593799   10345.507    10.103\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10856323   10601.878    10.353\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10871635   10616.831    10.368\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10881879   10626.835    10.378\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     10920639   10664.687    10.415\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     11314423   11049.241    10.790\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     11337391   11071.671    10.812\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     11352755   11086.675    10.827\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     11410895   11143.452    10.882\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     12001571   11720.284    11.446\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     12036023   11753.929    11.478\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     12059071   11776.437    11.500\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     12146283   11861.604    11.584\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13032299   12726.854    12.429\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13083983   12777.327    12.478\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13118555   12811.089    12.511\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13249371   12938.839    12.636\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14578395   14236.714    13.903\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                                0       0       0       0   12561\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14993711   14642.296    14.299\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15238799   14881.640    14.533\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15728975   15360.327    15.000\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16709327   16317.702    15.935\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     18670031   18232.453    17.805\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     22591439   22061.953    21.545\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   827367\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -47757.8711\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -47757.871094\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    21.746000\n",
      "Sequence length          :          653\n",
      "Compactness              :     0.053179\n",
      "Native energy (pair)     :    90.431399\n",
      "Native energy (surface)  :    22.331924\n",
      "Native energy (combined) :     4.688135\n",
      "Z score (pair)           :    -2.203604\n",
      "Z score (surface)        :    -3.250094\n",
      "Z score (combined)       :    -3.802759\n",
      "GA341 score              :     0.251083\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10284\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4332.3770\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5180       0      0   0.004   0.004      29.161       1.000\n",
      " 2 Bond angle potential               :    6986       0      8   1.826   1.826      443.93       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3019       0    111  48.870  48.870      1130.7       1.000\n",
      " 4 Stereochemical improper torsion pot:    2035       0      0   1.132   1.132      50.740       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10284       0      0   0.003   0.003      11.311       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10350       0     16   0.737   0.737      638.77       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10301       1     46   0.843   0.843      866.40       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     652       1      3   8.010   8.010      136.54       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     570       0      4  79.521  79.521      156.30       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     437       0      0  77.997  77.997      162.26       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     206       0      0  73.462  73.462      144.79       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      82       0      0  85.102  85.102      49.309       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5331       0      0   0.638   0.638      90.660       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     651     122     98  33.590  74.964      344.24       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     776       0      0   1.924   1.924      77.292       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00183.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00183.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   78105.2188\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  38449 537E 533S N   O    4184 4164    5.61    2.81    2.81    4.84    2.81    2.81    4.84\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  18411 540D 540D CA  C    4209 4214   -5.40 -179.18  173.79   40.39 -179.18  173.79   40.39\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17223   3Q   4L C   N      24   26   59.78  -70.70  138.50   13.72 -108.50  177.21    9.15\n",
      "    1          4L   4L N   CA     26   27 -171.95  141.60                  132.50\n",
      "    2  17239  19K  20M C   N     147  149  -81.91  -73.00   16.58    1.03  -63.40  163.57   25.89\n",
      "    2         20M  20M N   CA    149  150  156.98  143.00                  -40.50\n",
      "    3  17245  25N  26L C   N     196  198  -64.35  -70.70    6.47    0.62  -63.50  175.91   24.47\n",
      "    3         26L  26L N   CA    198  199  142.89  141.60                  -41.20\n",
      "    4  17247  27P  28Q C   N     211  213  -82.00  -73.00   10.90    0.85  -63.80  175.80   24.93\n",
      "    4         28Q  28Q N   CA    213  214  134.56  140.70                  -40.30\n",
      "    5  17248  28Q  29I C   N     220  222 -137.73 -120.60   30.00    1.23  -63.40  177.75   31.85\n",
      "    5         29I  29I N   CA    222  223  154.93  130.30                  -43.60\n",
      "    6  17252  32V  33G C   N     247  249  -60.86  -62.40    2.23    0.31   82.20  151.99   11.59\n",
      "    6         33G  33G N   CA    249  250  -42.81  -41.20                    8.50\n",
      "    7  17265  45A  46I C   N     332  334   61.76  -97.30  164.76    8.44 -120.60 -176.48   13.98\n",
      "    7         46I  46I N   CA    334  335   84.23  127.20                  130.30\n",
      "    8  17274  54R  55G C   N     405  407   86.13   78.70    7.95    0.31   82.20  177.47    8.42\n",
      "    8         55G  55G N   CA    407  408 -168.92 -166.10                    8.50\n",
      "    9  17275  55G  56S C   N     409  411  -86.43  -72.40   16.44    0.88  -64.10  165.53   13.45\n",
      "    9         56S  56S N   CA    411  412  160.98  152.40                  -35.00\n",
      "   10  17291  71L  72P C   N     529  531  -74.24  -64.50   47.67    3.09  -58.70  136.53   12.56\n",
      "   10         72P  72P N   CA    531  532 -166.14  147.20                  -30.50\n",
      "   11  17293  73K  74T C   N     545  547 -129.86 -124.80   43.45    1.72  -63.20  147.20   23.20\n",
      "   11         74T  74T N   CA    547  548 -173.35  143.50                  -42.10\n",
      "   12  17294  74T  75S C   N     552  554  122.54 -136.60  105.08    3.48  -64.10 -134.43   28.61\n",
      "   12         75S  75S N   CA    554  555 -179.32  151.20                  -35.00\n",
      "   13  17311  91F  92D C   N     706  708   82.34   54.50  125.69   11.49  -63.30 -146.19   23.83\n",
      "   13         92D  92D N   CA    708  709  163.46   40.90                  -40.00\n",
      "   14  17317  97N  98E C   N     758  760  -82.60  -69.30   27.00    1.60  -63.60  154.87   21.71\n",
      "   14         98E  98E N   CA    760  761  166.00  142.50                  -40.30\n",
      "   15  17348 128L 129N C   N    1004 1006  -67.27 -119.90   74.51    2.18  -63.20  125.41   15.46\n",
      "   15        129N 129N N   CA   1006 1007   84.25  137.00                  -41.10\n",
      "   16  17349 129N 130L C   N    1012 1014 -154.61 -108.50   85.82    4.05  -63.50  145.86   25.72\n",
      "   16        130L 130L N   CA   1014 1015 -155.11  132.50                  -41.20\n",
      "   17  17350 130L 131T C   N    1020 1022  -68.38  -78.10   10.16    0.67  -63.20  165.24   21.53\n",
      "   17        131T 131T N   CA   1022 1023  152.74  149.80                  -42.10\n",
      "   18  17351 131T 132L C   N    1027 1029  -70.16  -70.70   10.81    0.84  -63.50  166.54   23.51\n",
      "   18        132L 132L N   CA   1029 1030  152.40  141.60                  -41.20\n",
      "   19  17352 132L 133V C   N    1035 1037 -113.59 -125.40   17.76    0.52  -62.40  179.87   20.33\n",
      "   19        133V 133V N   CA   1037 1038  130.04  143.30                  -42.40\n",
      "   20  17361 141N 142A C   N    1100 1102  -52.45  -68.20   27.99    2.71  -62.50  150.99   24.27\n",
      "   20        142A 142A N   CA   1102 1103  168.44  145.30                  -40.90\n",
      "   21  17362 142A 143V C   N    1105 1107  -52.15  -73.50   31.29    2.69  -62.40  155.86   19.15\n",
      "   21        143V 143V N   CA   1107 1108  162.08  139.20                  -42.40\n",
      "   22  17363 143V 144G C   N    1112 1114  150.64   82.20   80.85    3.32  -62.40  147.11   26.00\n",
      "   22        144G 144G N   CA   1114 1115  -34.53    8.50                  -41.20\n",
      "   23  17371 151D 152R C   N    1173 1175  144.11 -125.20   91.19    3.42   57.30  141.80   15.76\n",
      "   23        152R 152R N   CA   1175 1176  150.13  140.60                   38.00\n",
      "   24  17373 153Q 154I C   N    1193 1195 -154.13 -120.60   77.52    3.31  -63.40  147.42   27.32\n",
      "   24        154I 154I N   CA   1195 1196 -159.80  130.30                  -43.60\n",
      "   25  17388 168I 169I C   N    1313 1315  -93.58 -120.60   46.79    1.92  -63.40  139.02   21.83\n",
      "   25        169I 169I N   CA   1315 1316   92.10  130.30                  -43.60\n",
      "   26  17390 170L 171A C   N    1329 1331  -52.44  -68.20   17.15    1.21  -62.50  179.72   29.96\n",
      "   26        171A 171A N   CA   1331 1332  138.54  145.30                  -40.90\n",
      "   27  17399 179L 180A C   N    1391 1393   70.65   55.40   16.01    1.45  -62.50  152.44   31.00\n",
      "   27        180A 180A N   CA   1393 1394   33.33   38.20                  -40.90\n",
      "   28  17411 191L 192D C   N    1479 1481 -126.65  -96.50   53.98    2.23  -63.30  126.44   13.43\n",
      "   28        192D 192D N   CA   1481 1482   69.43  114.20                  -40.00\n",
      "   29  17417 197R 198T C   N    1526 1528  -60.88  -63.20    4.03    0.43 -124.80 -177.35    9.66\n",
      "   29        198T 198T N   CA   1528 1529  -45.41  -42.10                  143.50\n",
      "   30  17425 205L 206D C   N    1583 1585 -105.19  -70.90   34.30    2.57  -63.30  173.98   24.16\n",
      "   30        206D 206D N   CA   1585 1586  151.14  150.30                  -40.00\n",
      "   31  17445 225L 226R C   N    1728 1730  -85.10 -125.20   59.15    3.26  -63.00  136.62   19.98\n",
      "   31        226R 226R N   CA   1730 1731 -175.92  140.60                  -41.10\n",
      "   32  17446 226R 227H C   N    1739 1741  -54.55  -67.60   15.14    1.33  -63.20  170.23   20.76\n",
      "   32        227H 227H N   CA   1741 1742  147.69  140.00                  -42.30\n",
      "   33  17448 228G 229F C   N    1753 1755 -138.59 -124.20   93.59    4.04  -63.20  109.87   19.58\n",
      "   33        229F 229F N   CA   1755 1756 -124.22  143.30                  -44.30\n",
      "   34  17459 239D 240I C   N    1840 1842 -106.35  -97.30   16.79    1.56  -63.40 -179.76   31.12\n",
      "   34        240I 240I N   CA   1842 1843  141.35  127.20                  -43.60\n",
      "   35  17464 244K 245G C   N    1879 1881  -66.27  -62.40    5.82    1.07   82.20  158.01   12.06\n",
      "   35        245G 245G N   CA   1881 1882  -45.55  -41.20                    8.50\n",
      "   36  17478 258H 259N C   N    1994 1996 -116.32 -119.90   15.04    0.59  -63.20  171.91   19.08\n",
      "   36        259N 259N N   CA   1996 1997  122.40  137.00                  -41.10\n",
      "   37  17481 261P 262S C   N    2019 2021 -131.72 -136.60   48.21    2.54  -64.10  142.85   15.21\n",
      "   37        262S 262S N   CA   2021 2022 -160.84  151.20                  -35.00\n",
      "   38  17499 279L 280N C   N    2160 2162  -76.54 -119.90   73.38    2.27  -63.20  119.64   14.26\n",
      "   38        280N 280N N   CA   2162 2163   77.80  137.00                  -41.10\n",
      "   39  17500 280N 281Q C   N    2168 2170 -100.38 -121.10   36.54    1.30  -63.80  154.30   20.95\n",
      "   39        281Q 281Q N   CA   2170 2171  109.60  139.70                  -40.30\n",
      "   40  17512 292P 293E C   N    2267 2269   57.20   54.60    2.66    0.33  -63.60  146.69   25.22\n",
      "   40        293E 293E N   CA   2269 2270   42.92   42.40                  -40.30\n",
      "   41  17522 302L 303G C   N    2352 2354   26.44   78.70   80.41    1.58  -62.40  109.37   16.15\n",
      "   41        303G 303G N   CA   2354 2355 -104.99 -166.10                  -41.20\n",
      "   42  17523 303G 304S C   N    2356 2358  -75.95  -72.40    5.97    0.27  -64.10  168.21   12.93\n",
      "   42        304S 304S N   CA   2358 2359  157.21  152.40                  -35.00\n",
      "   43  17524 304S 305T C   N    2362 2364  -88.24 -124.80   77.70    2.45  -63.20  119.69   13.84\n",
      "   43        305T 305T N   CA   2364 2365   74.94  143.50                  -42.10\n",
      "   44  17529 309M 310E C   N    2403 2405  -75.33  -63.60   83.03   10.47  -63.60   83.03   10.47\n",
      "   44        310E 310E N   CA   2405 2406   41.90  -40.30                  -40.30\n",
      "   45  17556 336T 337L C   N    2615 2617  -79.68  -70.70   23.22    1.48  -63.50  156.63   22.71\n",
      "   45        337L 337L N   CA   2617 2618  163.01  141.60                  -41.20\n",
      "   46  17562 342D 343G C   N    2664 2666  -60.11  -62.40    2.43    0.46   82.20  150.48   11.44\n",
      "   46        343G 343G N   CA   2666 2667  -40.39  -41.20                    8.50\n",
      "   47  17574 354G 355G C   N    2742 2744  -90.56  -80.20   13.98    0.48   82.20 -114.09    7.64\n",
      "   47        355G 355G N   CA   2744 2745 -176.50  174.10                    8.50\n",
      "   48  17575 355G 356A C   N    2746 2748 -133.78 -134.00    6.26    0.35  -62.50 -167.92   35.29\n",
      "   48        356A 356A N   CA   2748 2749  140.74  147.00                  -40.90\n",
      "   49  17576 356A 357R C   N    2751 2753 -150.71 -125.20   25.67    0.95  -63.00 -163.95   32.44\n",
      "   49        357R 357R N   CA   2753 2754  143.57  140.60                  -41.10\n",
      "   50  17577 357R 358L C   N    2762 2764 -125.33 -108.50   85.00    4.74  -63.50  109.51   13.51\n",
      "   50        358L 358L N   CA   2764 2765   49.18  132.50                  -41.20\n",
      "   51  17578 358L 359D C   N    2770 2772  -95.96  -70.90   36.41    1.43  -63.30  146.97   20.23\n",
      "   51        359D 359D N   CA   2772 2773  176.71  150.30                  -40.00\n",
      "   52  17581 361I 362F C   N    2798 2800  -71.90 -124.20   84.95    2.47  -63.20  120.97   16.22\n",
      "   52        362F 362F N   CA   2800 2801   76.36  143.30                  -44.30\n",
      "   53  17589 369Y 370V C   N    2865 2867  -33.36  -73.50   62.89    4.57  -62.40  136.31   19.02\n",
      "   53        370V 370V N   CA   2867 2868   90.78  139.20                  -42.40\n",
      "   54  17590 370V 371N C   N    2872 2874 -117.68  -63.20   71.45    8.20  -63.20   71.45    8.20\n",
      "   54        371N 371N N   CA   2874 2875    5.13  -41.10                  -41.10\n",
      "   55  17591 371N 372G C   N    2880 2882  -64.35  -62.40    3.57    0.49   82.20  153.81   11.64\n",
      "   55        372G 372G N   CA   2882 2883  -38.21  -41.20                    8.50\n",
      "   56  17592 372G 373L C   N    2884 2886  -59.37  -70.70   18.26    1.14  -63.50  168.53   23.64\n",
      "   56        373L 373L N   CA   2886 2887  127.28  141.60                  -41.20\n",
      "   57  17593 373L 374T C   N    2892 2894  -71.67  -78.10   57.46    2.25  -63.20  135.07   16.78\n",
      "   57        374T 374T N   CA   2894 2895   92.70  149.80                  -42.10\n",
      "   58  17594 374T 375A C   N    2899 2901  -81.91  -68.20   14.95    1.05  -62.50  168.95   28.68\n",
      "   58        375A 375A N   CA   2901 2902  151.27  145.30                  -40.90\n",
      "   59  17598 378E 379L C   N    2931 2933  -67.52  -63.50    4.80    0.88  -70.70  174.60   13.35\n",
      "   59        379L 379L N   CA   2933 2934  -43.83  -41.20                  141.60\n",
      "   60  17600 380T 381D C   N    2946 2948   46.91  -70.90  124.76   10.37  -70.90  124.76   10.37\n",
      "   60        381D 381D N   CA   2948 2949 -168.62  150.30                  150.30\n",
      "   61  17608 388A 389R C   N    3015 3017  -59.05 -125.20   66.23    2.74  -63.00  175.20   23.41\n",
      "   61        389R 389R N   CA   3017 3018  143.74  140.60                  -41.10\n",
      "   62  17609 389R 390N C   N    3026 3028 -110.58 -119.90   13.70    0.40  -63.20  174.61   19.60\n",
      "   62        390N 390N N   CA   3028 3029  126.96  137.00                  -41.10\n",
      "   63  17628 408L 409S C   N    3167 3169  -96.33  -72.40   67.59    3.25  -64.10  114.03   10.49\n",
      "   63        409S 409S N   CA   3169 3170 -144.39  152.40                  -35.00\n",
      "   64  17639 419P 420S C   N    3259 3261   96.18   56.90  100.02    9.73  -64.10 -131.12   28.06\n",
      "   64        420S 420S N   CA   3261 3262  128.39   36.40                  -35.00\n",
      "   65  17642 422K 423C C   N    3283 3285  -67.28  -63.00    7.98    1.29 -117.90  178.40    7.66\n",
      "   65        423C 423C N   CA   3285 3286  -47.83  -41.10                  141.10\n",
      "   66  17645 425Q 426F C   N    3305 3307   74.56   58.10   24.81    1.55  -63.20  149.72   26.66\n",
      "   66        426F 426F N   CA   3307 3308   14.34   32.90                  -44.30\n",
      "   67  17650 430E 431L C   N    3352 3354  -78.93 -108.50   61.10    2.93  -63.50  121.22   16.01\n",
      "   67        431L 431L N   CA   3354 3355   79.03  132.50                  -41.20\n",
      "   68  17652 432I 433K C   N    3368 3370 -121.37 -118.00   40.07    1.90  -62.90  151.88   23.22\n",
      "   68        433K 433K N   CA   3370 3371  179.03  139.10                  -40.80\n",
      "   69  17653 433K 434I C   N    3377 3379 -112.48 -120.60   14.92    1.13  -63.40 -179.62   31.37\n",
      "   69        434I 434I N   CA   3379 3380  142.82  130.30                  -43.60\n",
      "   70  17658 438C 439A C   N    3414 3416  176.82 -134.00   49.58    1.42  -62.50 -154.87   39.97\n",
      "   70        439A 439A N   CA   3416 3417  153.23  147.00                  -40.90\n",
      "   71  17659 439A 440S C   N    3419 3421 -136.34 -136.60   40.14    2.03  -64.10  162.95   10.36\n",
      "   71        440S 440S N   CA   3421 3422  111.06  151.20                  -35.00\n",
      "   72  17663 443D 444N C   N    3450 3452  -67.92  -63.20    7.51    0.82   55.90  144.64   17.81\n",
      "   72        444N 444N N   CA   3452 3453  -35.26  -41.10                   39.50\n",
      "   73  17665 445F 446P C   N    3469 3471  -52.23  -58.70   31.99    2.16  -64.50  151.47   11.96\n",
      "   73        446P 446P N   CA   3471 3472  -61.83  -30.50                  147.20\n",
      "   74  17666 446P 447R C   N    3476 3478 -153.40 -125.20   34.41    1.83  -63.00 -174.52   21.00\n",
      "   74        447R 447R N   CA   3478 3479  120.87  140.60                  -41.10\n",
      "   75  17668 448L 449K C   N    3495 3497 -114.04 -118.00   86.09    4.23  -62.90  107.10   16.96\n",
      "   75        449K 449K N   CA   3497 3498 -134.90  139.10                  -40.80\n",
      "   76  17676 456C 457R C   N    3554 3556 -122.51 -125.20   10.24    0.54  -63.00  178.63   28.23\n",
      "   76        457R 457R N   CA   3556 3557  150.48  140.60                  -41.10\n",
      "   77  17684 464R 465A C   N    3629 3631   74.61   55.40   99.77    4.53 -134.00 -144.55    6.54\n",
      "   77        465A 465A N   CA   3631 3632  -59.71   38.20                  147.00\n",
      "   78  17685 465A 466P C   N    3634 3636  -79.85  -58.70   29.88    2.11  -64.50  157.35   12.57\n",
      "   78        466P 466P N   CA   3636 3637   -9.40  -30.50                  147.20\n",
      "   79  17690 470H 471V C   N    3672 3674  -61.42  -73.50   13.67    1.10  -62.40  172.00   21.71\n",
      "   79        471V 471V N   CA   3674 3675  145.61  139.20                  -42.40\n",
      "   80  17691 471V 472R C   N    3679 3681   82.67 -125.20  152.20    6.23  -63.00 -130.78   40.34\n",
      "   80        472R 472R N   CA   3681 3682  135.89  140.60                  -41.10\n",
      "   81  17692 472R 473T C   N    3690 3692 -165.04 -124.80   78.72    2.45  -63.20  147.53   24.95\n",
      "   81        473T 473T N   CA   3692 3693 -148.84  143.50                  -42.10\n",
      "   82  17693 473T 474I C   N    3697 3699  -85.96  -63.40   75.31   11.67  -63.40   75.31   11.67\n",
      "   82        474I 474I N   CA   3699 3700   28.25  -43.60                  -43.60\n",
      "   83  17696 476A 477A C   N    3718 3720 -106.36 -134.00   41.14    1.31  -62.50  163.42   25.02\n",
      "   83        477A 477A N   CA   3720 3721  116.52  147.00                  -40.90\n",
      "   84  17697 477A 478E C   N    3723 3725 -125.21 -117.80   32.86    1.49  -63.60  162.99   25.20\n",
      "   84        478E 478E N   CA   3725 3726  168.81  136.80                  -40.30\n",
      "   85  17699 479R 480G C   N    3743 3745   67.13   78.70   47.43    0.90  -62.40  151.67   22.95\n",
      "   85        480G 480G N   CA   3745 3746 -120.10 -166.10                  -41.20\n",
      "   86  17700 480G 481F C   N    3747 3749  -57.31  -71.40   32.08    2.13  -63.20  156.29   22.10\n",
      "   86        481F 481F N   CA   3749 3750  111.88  140.70                  -44.30\n",
      "   87  17701 481F 482V C   N    3758 3760 -161.47 -125.40   96.46    5.45  -62.40  138.12   16.00\n",
      "   87        482V 482V N   CA   3760 3761   53.84  143.30                  -42.40\n",
      "   88  17702 482V 483N C   N    3765 3767   71.79   55.90   48.69    2.44 -119.90 -138.80   11.90\n",
      "   88        483N 483N N   CA   3767 3768   -6.53   39.50                  137.00\n",
      "   89  17718 498L 499K C   N    3900 3902  164.16  -62.90  134.12   22.66   56.60  144.92    9.76\n",
      "   89        499K 499K N   CA   3902 3903  -58.52  -40.80                   38.60\n",
      "   90  17721 501F 502G C   N    3928 3930  -62.53  -62.40    4.44    0.67   82.20  154.53   11.82\n",
      "   90        502G 502G N   CA   3930 3931  -45.64  -41.20                    8.50\n",
      "   91  17722 502G 503T C   N    3932 3934  -87.43  -78.10   29.76    1.59  -63.20  165.42   19.70\n",
      "   91        503T 503T N   CA   3934 3935  121.54  149.80                  -42.10\n",
      "   92  17723 503T 504E C   N    3939 3941  -74.95  -69.30   53.28    4.26  -63.60  130.32   16.78\n",
      "   92        504E 504E N   CA   3941 3942   89.52  142.50                  -40.30\n",
      "   93  17740 520G 521K C   N    4065 4067  -68.61  -70.20   13.60    1.02  -62.90  165.40   21.79\n",
      "   93        521K 521K N   CA   4067 4068  153.90  140.40                  -40.80\n",
      "   94  17741 521K 522D C   N    4074 4076   62.94   54.50   28.97    1.54  -63.30  136.98   23.33\n",
      "   94        522D 522D N   CA   4076 4077   13.19   40.90                  -40.00\n",
      "   95  17743 523K 524G C   N    4091 4093   59.10   78.70   37.48    0.58   82.20  144.51    8.16\n",
      "   95        524G 524G N   CA   4093 4094 -134.15 -166.10                    8.50\n",
      "   96  17746 526K 527V C   N    4113 4115  -62.26  -73.50   16.75    1.20  -62.40  169.18   21.42\n",
      "   96        527V 527V N   CA   4115 4116  126.78  139.20                  -42.40\n",
      "   97  17747 527V 528N C   N    4120 4122  -51.28  -71.20   27.93    2.25  -63.20  156.98   18.99\n",
      "   97        528N 528N N   CA   4122 4123  162.37  142.80                  -41.10\n",
      "   98  17752 532T 533S C   N    4157 4159  -74.96  -72.40   13.54    0.71  -64.10  159.67   12.25\n",
      "   98        533S 533S N   CA   4159 4160  165.69  152.40                  -35.00\n",
      "   99  17753 533S 534N C   N    4163 4165   74.06   55.90   78.26    4.26  -63.20  137.33   21.00\n",
      "   99        534N 534N N   CA   4165 4166  -36.62   39.50                  -41.10\n",
      "  100  17758 538K 539S C   N    4200 4202  -78.60  -72.40   38.56    2.06  -64.10  135.32   10.73\n",
      "  100        539S 539S N   CA   4202 4203 -169.54  152.40                  -35.00\n",
      "  101  17759 539S 540D C   N    4206 4208   57.92   54.50   75.14    5.80  -70.90  133.32    8.74\n",
      "  101        540D 540D N   CA   4208 4209  115.96   40.90                  150.30\n",
      "  102  17760 540D 541M C   N    4214 4216 -139.78 -125.60   27.70    1.51  -63.40  174.78   21.74\n",
      "  102        541M 541M N   CA   4216 4217  116.70  140.50                  -40.50\n",
      "  103  17766 546V 547T C   N    4255 4257  -66.49  -63.20    5.03    0.84 -124.80 -179.71    9.41\n",
      "  103        547T 547T N   CA   4257 4258  -45.90  -42.10                  143.50\n",
      "  104  17767 547T 548I C   N    4262 4264    6.66  -97.30  104.28    4.52 -120.60  127.77    8.49\n",
      "  104        548I 548I N   CA   4264 4265  118.93  127.20                  130.30\n",
      "  105  17770 550L 551G C   N    4286 4288  -73.41  -80.20   41.18    1.29   82.20 -160.42    7.11\n",
      "  105        551G 551G N   CA   4288 4289  133.48  174.10                    8.50\n",
      "  106  17784 564V 565I C   N    4397 4399 -129.08 -120.60   18.93    1.39  -63.40  170.17   26.09\n",
      "  106        565I 565I N   CA   4399 4400  113.38  130.30                  -43.60\n",
      "  107  17794 574G 575I C   N    4479 4481  -58.06  -63.40    5.90    1.11 -120.60 -177.54    8.20\n",
      "  107        575I 575I N   CA   4481 4482  -41.10  -43.60                  130.30\n",
      "  108  17795 575I 576V C   N    4487 4489   66.52 -125.40  168.64    6.16  -62.40 -154.02   23.00\n",
      "  108        576V 576V N   CA   4489 4490  156.96  143.30                  -42.40\n",
      "  109  17796 576V 577K C   N    4494 4496   63.57   56.60   24.86    1.33  -62.90  138.13   24.11\n",
      "  109        577K 577K N   CA   4496 4497   14.74   38.60                  -40.80\n",
      "  110  17807 587A 588I C   N    4571 4573  -98.12  -97.30    0.87    0.05  -63.40  174.58   27.50\n",
      "  110        588I 588I N   CA   4573 4574  127.49  127.20                  -43.60\n",
      "  111  17808 588I 589T C   N    4579 4581  -60.32  -63.20   14.38    1.67 -124.80  172.79    9.24\n",
      "  111        589T 589T N   CA   4581 4582  -56.19  -42.10                  143.50\n",
      "  112  17809 589T 590L C   N    4586 4588 -120.06 -108.50   18.09    0.83  -63.50 -178.58   28.68\n",
      "  112        590L 590L N   CA   4588 4589  146.42  132.50                  -41.20\n",
      "  113  17823 603R 604L C   N    4706 4708   68.01   60.20    7.82    0.88  -63.50  149.50   27.57\n",
      "  113        604L 604L N   CA   4708 4709   29.89   29.60                  -41.20\n",
      "  114  17825 605V 606R C   N    4721 4723 -117.84 -125.20   83.48    3.87  -63.00  112.77   12.77\n",
      "  114        606R 606R N   CA   4723 4724   57.45  140.60                  -41.10\n",
      "  115  17833 613K 614A C   N    4791 4793  -99.30  -68.20   91.58    8.40  -62.50  106.62   16.12\n",
      "  115        614A 614A N   CA   4793 4794   59.17  145.30                  -40.90\n",
      "  116  17834 614A 615D C   N    4796 4798 -112.92  -63.30   93.09   14.65  -63.30   93.09   14.65\n",
      "  116        615D 615D N   CA   4798 4799 -118.76  -40.00                  -40.00\n",
      "  117  17837 617L 618L C   N    4820 4822 -136.05 -108.50   53.44    2.54  -63.50  158.13   26.44\n",
      "  117        618L 618L N   CA   4822 4823  178.30  132.50                  -41.20\n",
      "  118  17847 627Q 628R C   N    4897 4899 -140.19 -125.20   58.36    2.45  -63.00  144.28   24.63\n",
      "  118        628R 628R N   CA   4899 4900 -163.00  140.60                  -41.10\n",
      "  119  17851 631T 632K C   N    4929 4931  -80.57 -118.00   43.62    2.06  -62.90  158.69   21.62\n",
      "  119        632K 632K N   CA   4931 4932  161.50  139.10                  -40.80\n",
      "  120  17865 645L 646E C   N    5036 5038  -57.51  -63.60    8.28    1.00  -69.30  171.99   13.54\n",
      "  120        646E 646E N   CA   5038 5039  -45.92  -40.30                  142.50\n",
      "  121  17870 650E 651F C   N    5078 5080  -85.98 -124.20   67.04    2.03  -63.20  134.47   17.20\n",
      "  121        651F 651F N   CA   5080 5081   88.22  143.30                  -44.30\n",
      "  122  17871 651F 652T C   N    5089 5091 -158.99 -124.80   66.11    2.06  -63.20  151.85   25.33\n",
      "  122        652T 652T N   CA   5091 5092 -159.92  143.50                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   45   91  349  414  513  507  621  743  781  755\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00183.B99990001.pdb\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "wrpdb___568_> Residues, atoms, selected atoms:      653     5106     5106\n",
      "openf___224_> Open           EAO00183.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     22298699   21776.074    21.266\n",
      "read_mo_297_> Segments, residues, atoms:        1      653     5106\n",
      "read_mo_298_> Segment:        1  1:A  653:A     5106\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5106     5106        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    27134\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    30588\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    32747\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    34959\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    37150\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    38832\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    40673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    42690\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    43535\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45692\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45976\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "iupac_m_485_> OD1/2 will be swapped:      130.9272       13  13\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.9187       81  81\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.1403       82  82\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      100.7551      115  115\n",
      "iupac_m_485_> OD1/2 will be swapped:      -93.9920      149  149\n",
      "iupac_m_485_> OD1/2 will be swapped:     -108.7067      151  151\n",
      "iupac_m_487_> NH1/2 swapped:      175.9501      152  152\n",
      "iupac_m_487_> NH1/2 swapped:      179.2659      160  160\n",
      "iupac_m_487_> NH1/2 swapped:      178.3871      197  197\n",
      "iupac_m_485_> OD1/2 will be swapped:      137.5173      206  206\n",
      "iupac_m_487_> NH1/2 swapped:     -177.6010      210  210\n",
      "iupac_m_485_> OD1/2 will be swapped:      104.9867      213  213\n",
      "iupac_m_487_> NH1/2 swapped:      179.0917      226  226\n",
      "iupac_m_485_> OD1/2 will be swapped:     -134.4874      239  239\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6063      250  250\n",
      "iupac_m_485_> OD1/2 will be swapped:      143.7399      252  252\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.2743      273  273\n",
      "iupac_m_485_> OD1/2 will be swapped:      148.9653      299  299\n",
      "iupac_m_485_> OD1/2 will be swapped:     -177.0521      316  316\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5371      332  332\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.0884      333  333\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -178.3251      359  359\n",
      "iupac_m_487_> NH1/2 swapped:      178.2242      385  385\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.6406      399  399\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.6756      410  410\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3249      415  415\n",
      "iupac_m_485_> OD1/2 will be swapped:     -166.7216      443  443\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.0812      445  445\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -115.1034      454  454\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.4213      462  462\n",
      "iupac_m_487_> NH1/2 swapped:     -175.4606      464  464\n",
      "iupac_m_487_> NH1/2 swapped:     -176.3831      472  472\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9448      495  495\n",
      "iupac_m_486_> OE1/2 will be swapped:      155.6966      504  504\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.9346      506  506\n",
      "iupac_m_486_> OE1/2 will be swapped:      -99.7155      530  530\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.9618      567  567\n",
      "iupac_m_487_> NH1/2 swapped:      179.9477      606  606\n",
      "iupac_m_485_> OD1/2 will be swapped:     -175.2025      616  616\n",
      "iupac_m_486_> OE1/2 will be swapped:      130.5403      623  623\n",
      "iupac_m_487_> NH1/2 swapped:      176.5866      628  628\n",
      "iupac_m_486_> OE1/2 will be swapped:     -158.4676      646  646\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   819708\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -47605.5117\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -47605.511719\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    21.746000\n",
      "Sequence length          :          653\n",
      "Compactness              :     0.049333\n",
      "Native energy (pair)     :    65.369541\n",
      "Native energy (surface)  :    24.604567\n",
      "Native energy (combined) :     4.693341\n",
      "Z score (pair)           :    -2.173679\n",
      "Z score (surface)        :    -2.937819\n",
      "Z score (combined)       :    -3.262295\n",
      "GA341 score              :     0.152968\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10289\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4317.1641\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5180       0      0   0.005   0.005      33.107       1.000\n",
      " 2 Bond angle potential               :    6986       0      9   1.874   1.874      468.21       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3019       0    105  48.892  48.892      1132.0       1.000\n",
      " 4 Stereochemical improper torsion pot:    2035       0      0   1.174   1.174      54.649       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10289       0      0   0.003   0.003      8.5475       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10350       0     13   0.752   0.752      646.84       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10301       2     28   0.808   0.808      754.62       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     652       0      5   4.054   4.054      123.83       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     570       0      5  83.818  83.818      182.17       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     437       0      5  77.691  77.691      193.64       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     206       0      0  73.444  73.444      144.29       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      82       0      0  91.027  91.027      54.175       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5331       0      0   0.621   0.621      97.134       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     651     123     99  33.237  74.792      324.15       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     776       0      5   2.025   2.025      99.808       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00183.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00183.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   80288.7188\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17223   3Q   4L C   N      24   26  -66.22  -63.50    2.73    0.46 -108.50  178.57    9.15\n",
      "    1          4L   4L N   CA     26   27  -40.99  -41.20                  132.50\n",
      "    2  17224   4L   5I C   N      32   34  -60.76  -63.40    8.42    1.30 -120.60 -172.12   12.97\n",
      "    2          5I   5I N   CA     34   35  -51.60  -43.60                  130.30\n",
      "    3  17227   7V   8V C   N      52   54 -169.94 -125.40   52.86    1.39  -62.40 -178.80   29.35\n",
      "    3          8V   8V N   CA     54   55  171.77  143.30                  -42.40\n",
      "    4  17230  10E  11L C   N      76   78  -66.36  -63.50    9.67    1.52  -70.70  168.01   12.89\n",
      "    4         11L  11L N   CA     78   79  -50.44  -41.20                  141.60\n",
      "    5  17231  11L  12H C   N      84   86  164.31 -125.60  142.03    3.95  -63.20  143.60   24.57\n",
      "    5         12H  12H N   CA     86   87  -97.67  138.80                  -42.30\n",
      "    6  17234  14A  15F C   N     107  109   55.91   58.10   11.46    0.74 -124.20 -154.60   10.60\n",
      "    6         15F  15F N   CA    109  110   44.15   32.90                  143.30\n",
      "    7  17239  19K  20M C   N     147  149  -74.90  -73.00   25.22    1.67  -63.40  151.79   23.57\n",
      "    7         20M  20M N   CA    149  150  168.15  143.00                  -40.50\n",
      "    8  17243  23K  24L C   N     180  182  -92.36 -108.50   71.91    3.70  -63.50  107.57   13.58\n",
      "    8         24L  24L N   CA    182  183   62.42  132.50                  -41.20\n",
      "    9  17245  25N  26L C   N     196  198  -66.64  -70.70   14.93    0.99  -63.50  168.47   23.21\n",
      "    9         26L  26L N   CA    198  199  127.24  141.60                  -41.20\n",
      "   10  17247  27P  28Q C   N     211  213  -67.00  -73.00    6.76    0.42  -63.80  177.91   25.99\n",
      "   10         28Q  28Q N   CA    213  214  137.58  140.70                  -40.30\n",
      "   11  17248  28Q  29I C   N     220  222 -135.83 -120.60   36.53    1.57  -63.40  169.18   30.37\n",
      "   11         29I  29I N   CA    222  223  163.51  130.30                  -43.60\n",
      "   12  17252  32V  33G C   N     247  249  -61.26  -62.40    2.21    0.30   82.20  152.46   11.63\n",
      "   12         33G  33G N   CA    249  250  -43.10  -41.20                    8.50\n",
      "   13  17265  45A  46I C   N     332  334   62.14  -97.30  165.20    8.47 -120.60 -176.79   13.97\n",
      "   13         46I  46I N   CA    334  335   83.97  127.20                  130.30\n",
      "   14  17275  55G  56S C   N     409  411 -123.02 -136.60   15.43    0.80  -64.10  176.59   16.93\n",
      "   14         56S  56S N   CA    411  412  158.53  151.20                  -35.00\n",
      "   15  17293  73K  74T C   N     545  547 -147.64 -124.80   46.31    1.45  -63.20  158.49   25.68\n",
      "   15         74T  74T N   CA    547  548 -176.22  143.50                  -42.10\n",
      "   16  17294  74T  75S C   N     552  554  112.10 -136.60  114.23    3.91  -64.10 -129.82   20.04\n",
      "   16         75S  75S N   CA    554  555  176.89  151.20                  -35.00\n",
      "   17  17311  91F  92D C   N     706  708   74.35   54.50  104.62    9.30  -63.30 -136.26   24.28\n",
      "   17         92D  92D N   CA    708  709  143.62   40.90                  -40.00\n",
      "   18  17313  93F  94L C   N     725  727 -161.41 -108.50   81.81    3.74  -70.70  105.21    7.49\n",
      "   18         94L  94L N   CA    727  728 -165.11  132.50                  141.60\n",
      "   19  17316  96I  97N C   N     750  752  -87.33  -71.20   49.34    3.60  -63.20  139.37   16.21\n",
      "   19         97N  97N N   CA    752  753   96.17  142.80                  -41.10\n",
      "   20  17317  97N  98E C   N     758  760 -157.22 -117.80   48.90    1.46  -63.60 -179.82   29.17\n",
      "   20         98E  98E N   CA    760  761  165.75  136.80                  -40.30\n",
      "   21  17348 128L 129N C   N    1004 1006  -67.95 -119.90   71.28    2.07  -63.20  129.38   15.92\n",
      "   21        129N 129N N   CA   1006 1007   88.20  137.00                  -41.10\n",
      "   22  17349 129N 130L C   N    1012 1014 -153.05 -108.50   88.42    4.21  -63.50  141.78   25.05\n",
      "   22        130L 130L N   CA   1014 1015 -151.12  132.50                  -41.20\n",
      "   23  17350 130L 131T C   N    1020 1022  -64.79  -78.10   14.57    0.70  -63.20  174.04   22.42\n",
      "   23        131T 131T N   CA   1022 1023  143.87  149.80                  -42.10\n",
      "   24  17351 131T 132L C   N    1027 1029  -67.03  -70.70    6.24    0.60  -63.50  172.18   24.11\n",
      "   24        132L 132L N   CA   1029 1030  146.65  141.60                  -41.20\n",
      "   25  17352 132L 133V C   N    1035 1037  -98.75  -73.50   41.53    3.61  -62.40  153.01   17.55\n",
      "   25        133V 133V N   CA   1037 1038  106.23  139.20                  -42.40\n",
      "   26  17355 135L 136P C   N    1058 1060  -71.33  -58.70   39.25    2.40  -64.50  140.70   10.86\n",
      "   26        136P 136P N   CA   1060 1061    6.66  -30.50                  147.20\n",
      "   27  17360 140M 141N C   N    1092 1094  -95.17 -119.90   30.27    0.89  -63.20  163.78   18.89\n",
      "   27        141N 141N N   CA   1094 1095  119.53  137.00                  -41.10\n",
      "   28  17361 141N 142A C   N    1100 1102 -165.48 -134.00   61.09    2.41  -62.50  157.94   31.22\n",
      "   28        142A 142A N   CA   1102 1103 -160.65  147.00                  -40.90\n",
      "   29  17362 142A 143V C   N    1105 1107  -50.25  -73.50   27.92    2.30  -62.40  163.38   20.00\n",
      "   29        143V 143V N   CA   1107 1108  154.67  139.20                  -42.40\n",
      "   30  17363 143V 144G C   N    1112 1114  133.50   82.20   76.17    2.27  -62.40  164.23   29.60\n",
      "   30        144G 144G N   CA   1114 1115  -47.81    8.50                  -41.20\n",
      "   31  17371 151D 152R C   N    1173 1175 -155.15 -125.20   63.93    2.32   57.30 -143.03   25.01\n",
      "   31        152R 152R N   CA   1175 1176 -162.92  140.60                   38.00\n",
      "   32  17388 168I 169I C   N    1313 1315  -93.89 -120.60   53.61    2.23  -63.40  131.02   20.52\n",
      "   32        169I 169I N   CA   1315 1316   83.82  130.30                  -43.60\n",
      "   33  17390 170L 171A C   N    1329 1331  -57.11  -68.20   12.25    1.18  -62.50  168.68   27.38\n",
      "   33        171A 171A N   CA   1331 1332  150.51  145.30                  -40.90\n",
      "   34  17399 179L 180A C   N    1391 1393   76.25   55.40   31.38    1.52  -62.50  149.49   30.30\n",
      "   34        180A 180A N   CA   1393 1394   14.75   38.20                  -40.90\n",
      "   35  17411 191L 192D C   N    1479 1481 -123.77  -96.50   61.78    2.56  -63.30  115.80   12.32\n",
      "   35        192D 192D N   CA   1481 1482   58.76  114.20                  -40.00\n",
      "   36  17417 197R 198T C   N    1526 1528  -61.86  -63.20    3.46    0.53 -124.80 -171.59    9.88\n",
      "   36        198T 198T N   CA   1528 1529  -38.91  -42.10                  143.50\n",
      "   37  17425 205L 206D C   N    1583 1585  -85.42  -70.90   16.50    0.89  -63.30  163.38   21.56\n",
      "   37        206D 206D N   CA   1585 1586  158.12  150.30                  -40.00\n",
      "   38  17445 225L 226R C   N    1728 1730 -101.13 -125.20   93.75    4.96  -63.00   95.63   15.53\n",
      "   38        226R 226R N   CA   1730 1731 -128.80  140.60                  -41.10\n",
      "   39  17447 227H 228G C   N    1749 1751  -58.30  -62.40    5.01    0.94   82.20  148.10   11.24\n",
      "   39        228G 228G N   CA   1751 1752  -38.32  -41.20                    8.50\n",
      "   40  17448 228G 229F C   N    1753 1755 -144.99  -63.20  104.63   18.86  -63.20  104.63   18.86\n",
      "   40        229F 229F N   CA   1755 1756 -109.56  -44.30                  -44.30\n",
      "   41  17459 239D 240I C   N    1840 1842 -104.74  -97.30    8.50    0.50  -63.40  171.76   26.86\n",
      "   41        240I 240I N   CA   1842 1843  123.10  127.20                  -43.60\n",
      "   42  17460 240I 241N C   N    1848 1850  -62.63  -63.20    5.63    0.67 -119.90 -174.63    9.53\n",
      "   42        241N 241N N   CA   1850 1851  -46.70  -41.10                  137.00\n",
      "   43  17461 241N 242D C   N    1856 1858   55.59   54.50    6.75    0.58  -63.30  147.65   25.05\n",
      "   43        242D 242D N   CA   1858 1859   47.56   40.90                  -40.00\n",
      "   44  17462 242D 243S C   N    1864 1866  -74.88  -64.10   27.76    1.73 -136.60  172.07    7.21\n",
      "   44        243S 243S N   CA   1866 1867   -9.42  -35.00                  151.20\n",
      "   45  17464 244K 245G C   N    1879 1881  -68.30  -62.40    6.64    1.03   82.20  157.57   11.90\n",
      "   45        245G 245G N   CA   1881 1882  -38.16  -41.20                    8.50\n",
      "   46  17481 261P 262S C   N    2019 2021 -134.36 -136.60   40.71    2.11  -64.10  150.55   15.95\n",
      "   46        262S 262S N   CA   2021 2022 -168.15  151.20                  -35.00\n",
      "   47  17500 280N 281Q C   N    2168 2170   -0.24 -121.10  123.87    4.14  -63.80  165.52   27.69\n",
      "   47        281Q 281Q N   CA   2170 2171  112.53  139.70                  -40.30\n",
      "   48  17512 292P 293E C   N    2267 2269   58.40   54.60    4.28    0.36  -63.60  146.29   25.17\n",
      "   48        293E 293E N   CA   2269 2270   40.42   42.40                  -40.30\n",
      "   49  17523 303G 304S C   N    2356 2358  -60.03  -72.40   23.86    1.09  -64.10  167.05   12.35\n",
      "   49        304S 304S N   CA   2358 2359  132.00  152.40                  -35.00\n",
      "   50  17524 304S 305T C   N    2362 2364  -95.70 -124.80   85.71    2.89  -63.20  109.90   12.24\n",
      "   50        305T 305T N   CA   2364 2365   62.88  143.50                  -42.10\n",
      "   51  17529 309M 310E C   N    2403 2405  -73.33  -69.30   88.18    6.87  -63.60   95.21   12.19\n",
      "   51        310E 310E N   CA   2405 2406   54.41  142.50                  -40.30\n",
      "   52  17541 321D 322P C   N    2497 2499  -71.53  -64.50    9.81    0.63  -58.70  175.93   15.59\n",
      "   52        322P 322P N   CA   2499 2500  154.04  147.20                  -30.50\n",
      "   53  17542 322P 323S C   N    2504 2506   34.95  -72.40  132.45    6.76  -64.10  147.88   17.83\n",
      "   53        323S 323S N   CA   2506 2507   74.80  152.40                  -35.00\n",
      "   54  17555 335D 336T C   N    2608 2610 -113.64 -124.80   42.88    1.53  -63.20  152.76   16.80\n",
      "   54        336T 336T N   CA   2610 2611  102.09  143.50                  -42.10\n",
      "   55  17556 336T 337L C   N    2615 2617 -120.86 -108.50   30.79    1.51  -63.50  168.18   26.90\n",
      "   55        337L 337L N   CA   2617 2618  160.70  132.50                  -41.20\n",
      "   56  17574 354G 355G C   N    2742 2744 -174.40 -167.20   24.52    1.01   82.20  176.19   12.98\n",
      "   56        355G 355G N   CA   2744 2745  151.16  174.60                    8.50\n",
      "   57  17575 355G 356A C   N    2746 2748  126.85 -134.00  107.48    2.45  -62.50 -145.08   43.49\n",
      "   57        356A 356A N   CA   2748 2749 -171.54  147.00                  -40.90\n",
      "   58  17576 356A 357R C   N    2751 2753  179.66  -63.00  130.76   23.79  -63.00  130.76   23.79\n",
      "   58        357R 357R N   CA   2753 2754  -98.82  -41.10                  -41.10\n",
      "   59  17578 358L 359D C   N    2770 2772  -60.60  -70.90   17.31    1.33  -63.30  155.81   19.07\n",
      "   59        359D 359D N   CA   2772 2773  164.21  150.30                  -40.00\n",
      "   60  17581 361I 362F C   N    2798 2800  -70.04 -124.20   76.92    2.07  -63.20  133.17   18.03\n",
      "   60        362F 362F N   CA   2800 2801   88.69  143.30                  -44.30\n",
      "   61  17589 369Y 370V C   N    2865 2867   39.53  -62.40  138.19   23.24  -62.40  138.19   23.24\n",
      "   61        370V 370V N   CA   2867 2868   50.91  -42.40                  -42.40\n",
      "   62  17591 371N 372G C   N    2880 2882  -53.77  -62.40   12.31    2.28   82.20  142.00   10.70\n",
      "   62        372G 372G N   CA   2882 2883  -32.42  -41.20                    8.50\n",
      "   63  17592 372G 373L C   N    2884 2886  -56.66  -70.70   21.59    1.35  -63.50  166.54   23.52\n",
      "   63        373L 373L N   CA   2886 2887  125.20  141.60                  -41.20\n",
      "   64  17593 373L 374T C   N    2892 2894  -82.50  -78.10   56.16    2.55  -63.20  137.28   16.40\n",
      "   64        374T 374T N   CA   2894 2895   93.81  149.80                  -42.10\n",
      "   65  17599 379L 380T C   N    2939 2941   51.98   55.90   24.46    2.01 -124.80 -141.66    7.74\n",
      "   65        380T 380T N   CA   2941 2942   15.36   39.50                  143.50\n",
      "   66  17605 385R 386I C   N    2994 2996  -61.35  -97.30   38.20    1.89  -63.40  176.29   28.82\n",
      "   66        386I 386I N   CA   2996 2997  140.12  127.20                  -43.60\n",
      "   67  17606 386I 387N C   N    3002 3004   76.30   55.90   39.21    1.91  -63.20  147.24   23.90\n",
      "   67        387N 387N N   CA   3004 3005    6.02   39.50                  -41.10\n",
      "   68  17608 388A 389R C   N    3015 3017 -144.95 -125.20   55.97    2.21  -63.00  150.25   25.74\n",
      "   68        389R 389R N   CA   3017 3018 -167.03  140.60                  -41.10\n",
      "   69  17609 389R 390N C   N    3026 3028   83.14   55.90   62.65    2.99  -63.20  148.32   23.43\n",
      "   69        390N 390N N   CA   3028 3029  -16.92   39.50                  -41.10\n",
      "   70  17628 408L 409S C   N    3167 3169  -85.24  -72.40   46.15    2.31  -64.10  130.00   10.82\n",
      "   70        409S 409S N   CA   3169 3170 -163.27  152.40                  -35.00\n",
      "   71  17629 409S 410R C   N    3173 3175  -65.47  -72.10   24.98    2.06  -63.00  152.94   20.83\n",
      "   71        410R 410R N   CA   3175 3176  165.99  141.90                  -41.10\n",
      "   72  17633 413I 414A C   N    3210 3212  -89.88  -68.20   52.39    4.91  -62.50  141.19   21.96\n",
      "   72        414A 414A N   CA   3212 3213   97.61  145.30                  -40.90\n",
      "   73  17638 418E 419P C   N    3252 3254  -70.68  -64.50   12.21    0.71  -58.70  172.19   15.21\n",
      "   73        419P 419P N   CA   3254 3255  157.73  147.20                  -30.50\n",
      "   74  17639 419P 420S C   N    3259 3261  -93.46  -72.40   54.71    3.93  -64.10  140.02    8.84\n",
      "   74        420S 420S N   CA   3261 3262  101.90  152.40                  -35.00\n",
      "   75  17642 422K 423C C   N    3283 3285  -66.26  -63.00   13.26    1.86 -117.90  172.84    7.44\n",
      "   75        423C 423C N   CA   3285 3286  -53.95  -41.10                  141.10\n",
      "   76  17645 425Q 426F C   N    3305 3307   75.79   58.10   24.71    1.70  -63.20  151.36   26.97\n",
      "   76        426F 426F N   CA   3307 3308   15.65   32.90                  -44.30\n",
      "   77  17650 430E 431L C   N    3352 3354  -67.00  -70.70   46.66    3.40  -63.50  136.33   18.73\n",
      "   77        431L 431L N   CA   3354 3355   95.09  141.60                  -41.20\n",
      "   78  17652 432I 433K C   N    3368 3370 -165.01 -118.00   75.96    2.82  -62.90  157.90   26.44\n",
      "   78        433K 433K N   CA   3370 3371 -161.24  139.10                  -40.80\n",
      "   79  17653 433K 434I C   N    3377 3379  -53.86  -97.30   45.96    2.24  -63.40  174.45   28.26\n",
      "   79        434I 434I N   CA   3379 3380  142.21  127.20                  -43.60\n",
      "   80  17658 438C 439A C   N    3414 3416 -169.68 -134.00   44.43    1.13  -62.50 -179.17   35.27\n",
      "   80        439A 439A N   CA   3416 3417  173.47  147.00                  -40.90\n",
      "   81  17659 439A 440S C   N    3419 3421 -156.01 -136.60   41.83    1.63  -64.10  164.76   18.57\n",
      "   81        440S 440S N   CA   3421 3422 -171.75  151.20                  -35.00\n",
      "   82  17660 440S 441K C   N    3425 3427   70.17   56.60   38.83    1.99  -62.90  139.85   24.25\n",
      "   82        441K 441K N   CA   3427 3428    2.22   38.60                  -40.80\n",
      "   83  17663 443D 444N C   N    3450 3452  -80.05  -71.20   45.36    3.15   55.90  148.12   12.93\n",
      "   83        444N 444N N   CA   3452 3453   98.31  142.80                   39.50\n",
      "   84  17666 446P 447R C   N    3476 3478  -76.12  -72.10   13.39    1.11  -63.00  170.72   22.21\n",
      "   84        447R 447R N   CA   3478 3479  129.12  141.90                  -41.10\n",
      "   85  17667 447R 448L C   N    3487 3489 -140.41 -108.50   99.67    5.00  -70.70  110.18    6.88\n",
      "   85        448L 448L N   CA   3489 3490 -133.07  132.50                  141.60\n",
      "   86  17668 448L 449K C   N    3495 3497 -116.26 -118.00   83.08    4.06  -62.90  110.74   17.57\n",
      "   86        449K 449K N   CA   3497 3498 -137.83  139.10                  -40.80\n",
      "   87  17675 455I 456C C   N    3548 3550  -65.35  -63.00    2.42    0.37 -117.90 -174.00    7.98\n",
      "   87        456C 456C N   CA   3550 3551  -40.47  -41.10                  141.10\n",
      "   88  17676 456C 457R C   N    3554 3556  -62.76  -72.10   17.00    1.47  -63.00  162.80   21.98\n",
      "   88        457R 457R N   CA   3556 3557  156.10  141.90                  -41.10\n",
      "   89  17685 465A 466P C   N    3634 3636  -62.07  -58.70   32.33    2.41  -64.50  145.56   10.73\n",
      "   89        466P 466P N   CA   3636 3637    1.65  -30.50                  147.20\n",
      "   90  17690 470H 471V C   N    3672 3674  -66.82  -73.50    8.37    0.58  -62.40  176.61   22.10\n",
      "   90        471V 471V N   CA   3674 3675  134.15  139.20                  -42.40\n",
      "   91  17691 471V 472R C   N    3679 3681 -157.97 -125.20   68.54    2.47  -63.00  151.55   26.61\n",
      "   91        472R 472R N   CA   3681 3682 -159.20  140.60                  -41.10\n",
      "   92  17692 472R 473T C   N    3690 3692  -84.37  -63.20   72.50    8.09  -63.20   72.50    8.09\n",
      "   92        473T 473T N   CA   3692 3693   27.24  -42.10                  -42.10\n",
      "   93  17695 475I 476A C   N    3713 3715  -79.77  -68.20   59.33    5.18  -62.50  129.17   20.39\n",
      "   93        476A 476A N   CA   3715 3716   87.11  145.30                  -40.90\n",
      "   94  17696 476A 477A C   N    3718 3720  -67.84  -68.20   22.80    1.85  -62.50  151.09   25.03\n",
      "   94        477A 477A N   CA   3720 3721  168.10  145.30                  -40.90\n",
      "   95  17697 477A 478E C   N    3723 3725  -54.56  -69.30   15.96    0.99  -63.60  176.91   24.09\n",
      "   95        478E 478E N   CA   3725 3726  136.38  142.50                  -40.30\n",
      "   96  17698 478E 479R C   N    3732 3734 -114.34 -125.20   26.63    1.46  -63.00  162.32   25.47\n",
      "   96        479R 479R N   CA   3734 3735  164.92  140.60                  -41.10\n",
      "   97  17699 479R 480G C   N    3743 3745  101.07   78.70   25.23    0.88  -62.40 -146.99   30.76\n",
      "   97        480G 480G N   CA   3745 3746 -177.77 -166.10                  -41.20\n",
      "   98  17700 480G 481F C   N    3747 3749  -92.67  -71.40   46.51    4.12  -63.20  146.63   18.48\n",
      "   98        481F 481F N   CA   3749 3750   99.34  140.70                  -44.30\n",
      "   99  17704 484V 485K C   N    3780 3782   80.50  -62.90  151.11   22.22  -62.90  151.11   22.22\n",
      "   99        485K 485K N   CA   3782 3783  -88.45  -40.80                  -40.80\n",
      "  100  17721 501F 502G C   N    3928 3930  -67.31  -62.40    5.34    0.85   82.20  156.91   11.87\n",
      "  100        502G 502G N   CA   3930 3931  -39.10  -41.20                    8.50\n",
      "  101  17723 503T 504E C   N    3939 3941   60.41   54.60   11.16    1.30  -63.60  154.55   26.54\n",
      "  101        504E 504E N   CA   3941 3942   51.92   42.40                  -40.30\n",
      "  102  17738 518N 519E C   N    4052 4054  -69.20  -69.30    4.10    0.31  -63.60  178.79   23.54\n",
      "  102        519E 519E N   CA   4054 4055  138.40  142.50                  -40.30\n",
      "  103  17739 519E 520G C   N    4061 4063  109.91   82.20   55.95    1.56  -62.40  172.31   30.83\n",
      "  103        520G 520G N   CA   4063 4064  -40.10    8.50                  -41.20\n",
      "  104  17742 522D 523K C   N    4082 4084  -72.24  -70.20   28.82    2.14  -62.90  152.74   19.34\n",
      "  104        523K 523K N   CA   4084 4085  111.65  140.40                  -40.80\n",
      "  105  17743 523K 524G C   N    4091 4093  -68.49  -62.40    6.14    1.12   82.20  158.92   12.06\n",
      "  105        524G 524G N   CA   4093 4094  -41.96  -41.20                    8.50\n",
      "  106  17746 526K 527V C   N    4113 4115  -64.49  -73.50   18.42    1.40  -62.40  165.55   20.83\n",
      "  106        527V 527V N   CA   4115 4116  123.14  139.20                  -42.40\n",
      "  107  17747 527V 528N C   N    4120 4122 -125.12 -119.90   24.40    0.98  -63.20  169.76   24.74\n",
      "  107        528N 528N N   CA   4122 4123  160.84  137.00                  -41.10\n",
      "  108  17748 528N 529S C   N    4128 4130   88.74 -136.60  140.17    4.65  -64.10 -156.15   17.40\n",
      "  108        529S 529S N   CA   4130 4131 -169.88  151.20                  -35.00\n",
      "  109  17758 538K 539S C   N    4200 4202  -72.98  -72.40   14.77    0.89  -64.10  172.87   12.02\n",
      "  109        539S 539S N   CA   4202 4203  137.64  152.40                  -35.00\n",
      "  110  17760 540D 541M C   N    4214 4216 -116.19 -125.60   30.13    1.22  -63.40  161.26   20.66\n",
      "  110        541M 541M N   CA   4216 4217  111.87  140.50                  -40.50\n",
      "  111  17770 550L 551G C   N    4286 4288  -70.04  -80.20   21.63    0.50   82.20 -148.71    6.75\n",
      "  111        551G 551G N   CA   4288 4289  155.01  174.10                    8.50\n",
      "  112  17784 564V 565I C   N    4397 4399 -129.22 -120.60   23.42    1.66  -63.40  165.76   25.39\n",
      "  112        565I 565I N   CA   4399 4400  108.53  130.30                  -43.60\n",
      "  113  17807 587A 588I C   N    4571 4573 -108.83 -120.60   12.25    0.94  -63.40 -176.99   28.59\n",
      "  113        588I 588I N   CA   4573 4574  133.68  130.30                  -43.60\n",
      "  114  17808 588I 589T C   N    4579 4581  -60.76  -63.20    9.69    1.10 -124.80  177.01    9.41\n",
      "  114        589T 589T N   CA   4581 4582  -51.48  -42.10                  143.50\n",
      "  115  17809 589T 590L C   N    4586 4588 -123.63 -108.50   21.12    0.95  -63.50 -178.21   28.96\n",
      "  115        590L 590L N   CA   4588 4589  147.24  132.50                  -41.20\n",
      "  116  17823 603R 604L C   N    4706 4708   65.95   60.20    5.76    0.63  -63.50  147.43   27.19\n",
      "  116        604L 604L N   CA   4708 4709   29.35   29.60                  -41.20\n",
      "  117  17825 605V 606R C   N    4721 4723 -118.34 -125.20   91.42    4.26  -63.00  106.11   12.01\n",
      "  117        606R 606R N   CA   4723 4724   49.43  140.60                  -41.10\n",
      "  118  17833 613K 614A C   N    4791 4793  -97.08  -68.20   87.44    8.00  -62.50  109.28   16.60\n",
      "  118        614A 614A N   CA   4793 4794   62.77  145.30                  -40.90\n",
      "  119  17834 614A 615D C   N    4796 4798 -116.79  -63.30  102.88   16.12  -63.30  102.88   16.12\n",
      "  119        615D 615D N   CA   4798 4799 -127.88  -40.00                  -40.00\n",
      "  120  17837 617L 618L C   N    4820 4822 -136.20 -108.50   47.76    2.23  -63.50  164.34   27.31\n",
      "  120        618L 618L N   CA   4822 4823  171.41  132.50                  -41.20\n",
      "  121  17845 625E 626T C   N    4881 4883  -77.74 -124.80   47.10    2.44  -63.20  177.00   23.66\n",
      "  121        626T 626T N   CA   4883 4884  141.50  143.50                  -42.10\n",
      "  122  17847 627Q 628R C   N    4897 4899 -122.67 -125.20   70.32    3.45  -63.00  123.41   20.70\n",
      "  122        628R 628R N   CA   4899 4900 -149.12  140.60                  -41.10\n",
      "  123  17851 631T 632K C   N    4929 4931  -86.42 -118.00   42.09    2.09  -62.90  154.07   21.37\n",
      "  123        632K 632K N   CA   4931 4932  166.93  139.10                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   36   64  363  447  527  514  637  738  779  713\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00183.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      653     5106     5106\n",
      "openf___224_> Open           EAO00183.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     22298699   21776.074    21.266\n",
      "read_mo_297_> Segments, residues, atoms:        1      653     5106\n",
      "read_mo_298_> Segment:        1  1:A  653:A     5106\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5106     5106        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    27134\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    30588\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    32747\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    34959\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    37150\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    38832\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    40673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    42690\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    43535\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45692\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45976\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "iupac_m_485_> OD1/2 will be swapped:      126.4085       13  13\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.6923       44  44\n",
      "iupac_m_487_> NH1/2 swapped:      179.5390       61  61\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.5051       78  78\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.9194       91  91\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.6188       93  93\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.4935       95  95\n",
      "iupac_m_487_> NH1/2 swapped:      179.6439      103  103\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.6712      106  106\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.2214      115  115\n",
      "iupac_m_487_> NH1/2 swapped:      179.6557      152  152\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.2991      178  178\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8953      186  186\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.4877      192  192\n",
      "iupac_m_485_> OD1/2 will be swapped:      158.9569      206  206\n",
      "iupac_m_487_> NH1/2 swapped:     -175.6130      226  226\n",
      "iupac_m_485_> OD1/2 will be swapped:      167.3416      239  239\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4426      250  250\n",
      "iupac_m_487_> NH1/2 swapped:      179.4550      254  254\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.6726      268  268\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5154      269  269\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.2721      273  273\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.3295      299  299\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.4189      316  316\n",
      "iupac_m_485_> OD1/2 will be swapped:      151.4019      342  342\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.6131      351  351\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9657      357  357\n",
      "iupac_m_485_> OD1/2 will be swapped:      169.0618      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:      130.7023      364  364\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.5749      366  366\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.2177      385  385\n",
      "iupac_m_487_> NH1/2 swapped:      179.5078      410  410\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7888      415  415\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.7483      443  443\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1899      457  457\n",
      "iupac_m_487_> NH1/2 swapped:      179.7973      464  464\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.4890      490  490\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7684      495  495\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.7749      497  497\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.1057      504  504\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.9242      506  506\n",
      "iupac_m_485_> OD1/2 will be swapped:      143.2599      522  522\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.2182      558  558\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6113      566  566\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.9209      567  567\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1013      597  597\n",
      "iupac_m_487_> NH1/2 swapped:      179.6320      603  603\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.5893      616  616\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.2094      620  620\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.8718      650  650\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   819602\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -45738.8516\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -45738.851562\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    21.746000\n",
      "Sequence length          :          653\n",
      "Compactness              :     0.046716\n",
      "Native energy (pair)     :   134.037552\n",
      "Native energy (surface)  :    26.316633\n",
      "Native energy (combined) :     6.019701\n",
      "Z score (pair)           :    -1.535972\n",
      "Z score (surface)        :    -2.536460\n",
      "Z score (combined)       :    -2.517281\n",
      "GA341 score              :     0.075072\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10734\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4355.4248\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5180       0      0   0.004   0.004      26.715       1.000\n",
      " 2 Bond angle potential               :    6986       0      9   1.833   1.833      445.86       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3019       0    123  49.758  49.758      1163.1       1.000\n",
      " 4 Stereochemical improper torsion pot:    2035       0      0   1.140   1.140      52.402       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10734       0      0   0.003   0.003      11.081       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10350       0     20   0.746   0.746      617.04       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10301       2     34   0.836   0.836      794.18       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     652       2      5  10.675  10.675      135.32       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     570       0      7  83.137  83.137      179.69       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     437       0      1  78.043  78.043      162.41       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     206       0      0  71.737  71.737      133.19       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      82       0      0  87.204  87.204      51.837       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5331       0      0   0.583   0.583      87.480       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     651     128    104  34.107  76.592      407.33       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     776       0      3   1.737   1.737      87.800       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00183.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00183.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   79025.9922\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  38124 514Q 564V N   O    4020 4398    6.44    3.47    2.97    4.53    3.47    2.97    4.53\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  18411 540D 540D CA  C    4209 4214   -2.63 -179.18  176.56   41.03 -179.18  176.56   41.03\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17223   3Q   4L C   N      24   26   77.38  -70.70  148.41   13.80 -108.50  175.16    8.13\n",
      "    1          4L   4L N   CA     26   27  151.58  141.60                  132.50\n",
      "    2  17227   7V   8V C   N      52   54 -158.72 -125.40   43.93    1.18  -62.40  174.64   27.91\n",
      "    2          8V   8V N   CA     54   55  171.92  143.30                  -42.40\n",
      "    3  17239  19K  20M C   N     147  149  -83.94  -73.00   11.08    0.86  -63.40  179.39   28.41\n",
      "    3         20M  20M N   CA    149  150  141.29  143.00                  -40.50\n",
      "    4  17243  23K  24L C   N     180  182  -96.22 -108.50   73.98    3.86  -63.50  105.93   13.24\n",
      "    4         24L  24L N   CA    182  183   59.55  132.50                  -41.20\n",
      "    5  17244  24L  25N C   N     188  190  -67.13  -63.20    6.66    1.05 -119.90 -175.76    9.41\n",
      "    5         25N  25N N   CA    190  191  -46.47  -41.10                  137.00\n",
      "    6  17245  25N  26L C   N     196  198 -123.54 -108.50   35.73    2.03  -63.50  153.52   18.91\n",
      "    6         26L  26L N   CA    198  199  100.09  132.50                  -41.20\n",
      "    7  17247  27P  28Q C   N     211  213 -122.60 -121.10    7.82    0.34  -63.80 -177.92   29.90\n",
      "    7         28Q  28Q N   CA    213  214  147.38  139.70                  -40.30\n",
      "    8  17248  28Q  29I C   N     220  222 -134.86 -120.60   24.88    1.02  -63.40 -179.54   32.20\n",
      "    8         29I  29I N   CA    222  223  150.69  130.30                  -43.60\n",
      "    9  17252  32V  33G C   N     247  249  -61.40  -62.40    2.71    0.37   82.20  152.80   11.66\n",
      "    9         33G  33G N   CA    249  250  -43.72  -41.20                    8.50\n",
      "   10  17265  45A  46I C   N     332  334   48.60  -97.30  151.67    7.88 -120.60  174.96   10.54\n",
      "   10         46I  46I N   CA    334  335   85.77  127.20                  130.30\n",
      "   11  17276  56S  57G C   N     415  417 -117.62  -80.20  105.92    6.01   82.20  173.44   13.35\n",
      "   11         57G  57G N   CA    417  418   75.01  174.10                    8.50\n",
      "   12  17293  73K  74T C   N     545  547 -144.72  -63.20   81.90   13.19  -63.20   81.90   13.19\n",
      "   12         74T  74T N   CA    547  548  -49.98  -42.10                  -42.10\n",
      "   13  17294  74T  75S C   N     552  554  -55.92  -72.40   44.53    3.18  -64.10  131.48    9.07\n",
      "   13         75S  75S N   CA    554  555 -166.23  152.40                  -35.00\n",
      "   14  17311  91F  92D C   N     706  708   71.12   54.50  102.32    8.87  -63.30 -136.84   24.12\n",
      "   14         92D  92D N   CA    708  709  141.86   40.90                  -40.00\n",
      "   15  17313  93F  94L C   N     725  727 -170.29 -108.50   89.70    4.06  -70.70  114.22    8.22\n",
      "   15         94L  94L N   CA    727  728 -162.48  132.50                  141.60\n",
      "   16  17316  96I  97N C   N     750  752  -95.23 -119.90   39.51    1.20  -63.20  150.67   17.26\n",
      "   16         97N  97N N   CA    752  753  106.13  137.00                  -41.10\n",
      "   17  17317  97N  98E C   N     758  760 -171.95 -117.80   60.76    1.78  -63.60 -170.62   31.12\n",
      "   17         98E  98E N   CA    760  761  164.38  136.80                  -40.30\n",
      "   18  17341 121K 122V C   N     947  949  -72.68  -62.40   13.35    2.26 -125.40  173.95    9.67\n",
      "   18        122V 122V N   CA    949  950  -50.93  -42.40                  143.30\n",
      "   19  17343 123Y 124S C   N     966  968  -58.88  -72.40   18.83    0.90  -64.10  174.37   12.95\n",
      "   19        124S 124S N   CA    968  969  139.29  152.40                  -35.00\n",
      "   20  17345 125K 126N C   N     981  983   76.95   55.90   87.07    4.69  -63.20  140.20   21.06\n",
      "   20        126N 126N N   CA    983  984  -44.99   39.50                  -41.10\n",
      "   21  17348 128L 129N C   N    1004 1006   45.92   55.90   46.45    2.57  -63.20  166.65   26.62\n",
      "   21        129N 129N N   CA   1006 1007   84.86   39.50                  -41.10\n",
      "   22  17349 129N 130L C   N    1012 1014  -38.74  -70.70   50.92    3.18  -63.50  145.29   21.67\n",
      "   22        130L 130L N   CA   1014 1015  101.96  141.60                  -41.20\n",
      "   23  17350 130L 131T C   N    1020 1022 -150.05 -124.80   47.53    1.48  -63.20  159.80   25.99\n",
      "   23        131T 131T N   CA   1022 1023 -176.24  143.50                  -42.10\n",
      "   24  17351 131T 132L C   N    1027 1029  -78.61  -70.70   13.65    1.32  -63.50  172.34   23.10\n",
      "   24        132L 132L N   CA   1029 1030  130.48  141.60                  -41.20\n",
      "   25  17352 132L 133V C   N    1035 1037 -126.30 -125.40   49.46    2.48  -62.40  150.49   16.52\n",
      "   25        133V 133V N   CA   1037 1038   93.86  143.30                  -42.40\n",
      "   26  17361 141N 142A C   N    1100 1102  -38.31  -68.20   30.19    2.70  -62.50  171.26   26.98\n",
      "   26        142A 142A N   CA   1102 1103  149.56  145.30                  -40.90\n",
      "   27  17362 142A 143V C   N    1105 1107  -90.23  -73.50   46.45    4.09  -62.40  141.03   16.41\n",
      "   27        143V 143V N   CA   1107 1108   95.86  139.20                  -42.40\n",
      "   28  17371 151D 152R C   N    1173 1175 -158.96 -125.20   67.96    2.41   57.30 -143.74   24.74\n",
      "   28        152R 152R N   CA   1175 1176 -160.42  140.60                   38.00\n",
      "   29  17373 153Q 154I C   N    1193 1195 -109.36 -120.60   40.48    2.74  -63.40  154.21   26.97\n",
      "   29        154I 154I N   CA   1195 1196  169.19  130.30                  -43.60\n",
      "   30  17374 154I 155K C   N    1201 1203   76.31   56.60   67.63    3.60  -62.90  139.98   23.52\n",
      "   30        155K 155K N   CA   1203 1204  -26.10   38.60                  -40.80\n",
      "   31  17388 168I 169I C   N    1313 1315  -97.22 -120.60   46.54    1.93  -63.40  137.88   21.55\n",
      "   31        169I 169I N   CA   1315 1316   90.07  130.30                  -43.60\n",
      "   32  17390 170L 171A C   N    1329 1331  -49.78  -68.20   21.21    1.43  -62.50  176.16   29.51\n",
      "   32        171A 171A N   CA   1331 1332  134.80  145.30                  -40.90\n",
      "   33  17399 179L 180A C   N    1391 1393   71.27   55.40   17.08    1.46  -62.50  152.30   30.97\n",
      "   33        180A 180A N   CA   1393 1394   31.90   38.20                  -40.90\n",
      "   34  17411 191L 192D C   N    1479 1481 -134.60  -96.50   53.46    2.19  -63.30  136.76   14.55\n",
      "   34        192D 192D N   CA   1481 1482   76.71  114.20                  -40.00\n",
      "   35  17417 197R 198T C   N    1526 1528  -56.62  -63.20   10.82    1.15 -124.80  179.28    9.62\n",
      "   35        198T 198T N   CA   1528 1529  -50.70  -42.10                  143.50\n",
      "   36  17418 198T 199V C   N    1533 1535 -130.16 -125.40   38.71    2.03  -62.40  162.13   17.81\n",
      "   36        199V 199V N   CA   1535 1536  104.89  143.30                  -42.40\n",
      "   37  17420 200G 201V C   N    1544 1546  -74.94  -62.40   18.84    2.13 -125.40  178.91    7.46\n",
      "   37        201V 201V N   CA   1546 1547  -28.35  -42.40                  143.30\n",
      "   38  17425 205L 206D C   N    1583 1585  -88.20  -70.90   17.85    1.18  -63.30  167.20   22.21\n",
      "   38        206D 206D N   CA   1585 1586  154.66  150.30                  -40.00\n",
      "   39  17445 225L 226R C   N    1728 1730  -34.11 -125.20   91.60    3.93  -63.00  171.10   21.31\n",
      "   39        226R 226R N   CA   1730 1731  150.26  140.60                  -41.10\n",
      "   40  17447 227H 228G C   N    1749 1751  -54.48  -62.40   10.47    1.96   82.20  143.24   10.82\n",
      "   40        228G 228G N   CA   1751 1752  -34.35  -41.20                    8.50\n",
      "   41  17448 228G 229F C   N    1753 1755 -103.94  -63.20   82.82   14.08  -63.20   82.82   14.08\n",
      "   41        229F 229F N   CA   1755 1756 -116.41  -44.30                  -44.30\n",
      "   42  17461 241N 242D C   N    1856 1858 -152.74  -96.50   84.99    3.56  -63.30  167.88   26.42\n",
      "   42        242D 242D N   CA   1858 1859  177.93  114.20                  -40.00\n",
      "   43  17464 244K 245G C   N    1879 1881  -68.91  -62.40    7.34    1.39   82.20  160.17   12.20\n",
      "   43        245G 245G N   CA   1881 1882  -44.59  -41.20                    8.50\n",
      "   44  17481 261P 262S C   N    2019 2021 -141.35 -136.60   42.36    2.05  -64.10  152.69   16.63\n",
      "   44        262S 262S N   CA   2021 2022 -166.71  151.20                  -35.00\n",
      "   45  17500 280N 281Q C   N    2168 2170  -31.24 -121.10   95.66    3.04  -63.80  150.74   23.90\n",
      "   45        281Q 281Q N   CA   2170 2171  106.88  139.70                  -40.30\n",
      "   46  17512 292P 293E C   N    2267 2269   60.86   54.60    6.65    0.86  -63.60  150.68   25.91\n",
      "   46        293E 293E N   CA   2269 2270   44.63   42.40                  -40.30\n",
      "   47  17522 302L 303G C   N    2352 2354   19.36   78.70   91.27    1.79  -62.40   98.85   14.72\n",
      "   47        303G 303G N   CA   2354 2355  -96.76 -166.10                  -41.20\n",
      "   48  17523 303G 304S C   N    2356 2358  -60.49  -72.40   13.56    0.75  -64.10  179.11   12.77\n",
      "   48        304S 304S N   CA   2358 2359  145.92  152.40                  -35.00\n",
      "   49  17524 304S 305T C   N    2362 2364  -92.55 -124.80   79.21    2.57  -63.20  117.00   13.28\n",
      "   49        305T 305T N   CA   2364 2365   71.15  143.50                  -42.10\n",
      "   50  17526 306M 307K C   N    2377 2379  -82.52  -70.20   52.32    4.07  -62.90  131.82   16.14\n",
      "   50        307K 307K N   CA   2379 2380   89.55  140.40                  -40.80\n",
      "   51  17527 307K 308Q C   N    2386 2388  167.58  -63.80  128.86   21.11  -63.80  128.86   21.11\n",
      "   51        308Q 308Q N   CA   2388 2389  -32.51  -40.30                  -40.30\n",
      "   52  17529 309M 310E C   N    2403 2405  -75.53  -69.30   91.76    7.22  -63.60   92.03   11.66\n",
      "   52        310E 310E N   CA   2405 2406   50.95  142.50                  -40.30\n",
      "   53  17541 321D 322P C   N    2497 2499  -71.78  -64.50    9.12    0.64  -58.70  177.29   15.72\n",
      "   53        322P 322P N   CA   2499 2500  152.69  147.20                  -30.50\n",
      "   54  17542 322P 323S C   N    2504 2506   23.79  -72.40  123.52    6.14  -64.10  140.73   16.53\n",
      "   54        323S 323S N   CA   2506 2507   74.91  152.40                  -35.00\n",
      "   55  17555 335D 336T C   N    2608 2610 -107.77 -124.80   47.08    1.57  -63.20  148.56   16.52\n",
      "   55        336T 336T N   CA   2610 2611   99.61  143.50                  -42.10\n",
      "   56  17556 336T 337L C   N    2615 2617 -115.33 -108.50   23.48    1.19  -63.50  171.84   27.06\n",
      "   56        337L 337L N   CA   2617 2618  154.97  132.50                  -41.20\n",
      "   57  17574 354G 355G C   N    2742 2744  -89.12  -80.20   13.79    0.39   82.20 -114.30    7.57\n",
      "   57        355G 355G N   CA   2744 2745 -175.39  174.10                    8.50\n",
      "   58  17575 355G 356A C   N    2746 2748 -135.67 -134.00    7.04    0.43  -62.50 -166.68   35.60\n",
      "   58        356A 356A N   CA   2748 2749  140.16  147.00                  -40.90\n",
      "   59  17576 356A 357R C   N    2751 2753 -149.68 -125.20   25.08    0.87  -63.00 -166.66   32.01\n",
      "   59        357R 357R N   CA   2753 2754  146.08  140.60                  -41.10\n",
      "   60  17577 357R 358L C   N    2762 2764 -129.02 -108.50   84.73    4.75  -63.50  112.54   13.93\n",
      "   60        358L 358L N   CA   2764 2765   50.30  132.50                  -41.20\n",
      "   61  17578 358L 359D C   N    2770 2772 -132.79  -96.50   57.33    2.40  -63.30  175.74   26.18\n",
      "   61        359D 359D N   CA   2772 2773  158.59  114.20                  -40.00\n",
      "   62  17581 361I 362F C   N    2798 2800  -71.13 -124.20   71.88    1.90  -63.20  139.35   18.82\n",
      "   62        362F 362F N   CA   2800 2801   94.82  143.30                  -44.30\n",
      "   63  17582 362F 363N C   N    2809 2811 -110.62  -63.20   65.30    7.35  -63.20   65.30    7.35\n",
      "   63        363N 363N N   CA   2811 2812    3.79  -41.10                  -41.10\n",
      "   64  17585 365C 366F C   N    2832 2834  -78.03  -63.20   62.30    7.72  -63.20   62.30    7.72\n",
      "   64        366F 366F N   CA   2834 2835   16.21  -44.30                  -44.30\n",
      "   65  17589 369Y 370V C   N    2865 2867  130.44 -125.40  104.17    4.03  -62.40 -119.39   40.08\n",
      "   65        370V 370V N   CA   2867 2868  144.54  143.30                  -42.40\n",
      "   66  17590 370V 371N C   N    2872 2874  -90.17 -119.90   58.38    1.91  -63.20  130.67   15.01\n",
      "   66        371N 371N N   CA   2874 2875   86.75  137.00                  -41.10\n",
      "   67  17592 372G 373L C   N    2884 2886  -78.52  -70.70   49.73    4.07  -63.50  134.53   17.87\n",
      "   67        373L 373L N   CA   2886 2887   92.49  141.60                  -41.20\n",
      "   68  17593 373L 374T C   N    2892 2894  -68.08  -78.10   52.03    1.92  -63.20  140.93   17.76\n",
      "   68        374T 374T N   CA   2894 2895   98.74  149.80                  -42.10\n",
      "   69  17608 388A 389R C   N    3015 3017  -51.17 -125.20   74.08    3.04  -63.00  176.07   23.01\n",
      "   69        389R 389R N   CA   3017 3018  143.23  140.60                  -41.10\n",
      "   70  17609 389R 390N C   N    3026 3028 -106.16 -119.90   21.15    0.63  -63.20  167.62   18.91\n",
      "   70        390N 390N N   CA   3028 3029  120.92  137.00                  -41.10\n",
      "   71  17626 406V 407A C   N    3154 3156  -73.13  -62.50   13.21    2.68 -134.00  175.19   10.78\n",
      "   71        407A 407A N   CA   3156 3157  -48.73  -40.90                  147.00\n",
      "   72  17628 408L 409S C   N    3167 3169 -140.19 -136.60   12.75    0.56  -64.10  178.59   18.38\n",
      "   72        409S 409S N   CA   3169 3170  163.43  151.20                  -35.00\n",
      "   73  17629 409S 410R C   N    3173 3175 -107.23 -125.20   31.49    1.75  -63.00  158.73   24.50\n",
      "   73        410R 410R N   CA   3175 3176  166.46  140.60                  -41.10\n",
      "   74  17631 411Q 412Q C   N    3193 3195  -63.03  -63.80    2.21    0.29 -121.10 -172.83    9.63\n",
      "   74        412Q 412Q N   CA   3195 3196  -42.37  -40.30                  139.70\n",
      "   75  17633 413I 414A C   N    3210 3212  -59.25  -68.20   26.92    1.86  -62.50  160.84   26.52\n",
      "   75        414A 414A N   CA   3212 3213  119.91  145.30                  -40.90\n",
      "   76  17639 419P 420S C   N    3259 3261  174.24 -136.60   88.16    5.00  -64.10  166.05   13.87\n",
      "   76        420S 420S N   CA   3261 3262   78.01  151.20                  -35.00\n",
      "   77  17642 422K 423C C   N    3283 3285  -65.21  -63.00    8.44    1.19 -117.90  177.64    7.64\n",
      "   77        423C 423C N   CA   3285 3286  -49.25  -41.10                  141.10\n",
      "   78  17645 425Q 426F C   N    3305 3307   74.65   58.10   25.50    1.55  -63.20  149.48   26.60\n",
      "   78        426F 426F N   CA   3307 3308   13.50   32.90                  -44.30\n",
      "   79  17650 430E 431L C   N    3352 3354  -60.91  -70.70   27.88    1.79  -63.50  156.72   21.91\n",
      "   79        431L 431L N   CA   3354 3355  115.50  141.60                  -41.20\n",
      "   80  17651 431L 432I C   N    3360 3362 -123.27 -120.60   25.63    1.36  -63.40  171.40   30.29\n",
      "   80        432I 432I N   CA   3362 3363  155.80  130.30                  -43.60\n",
      "   81  17652 432I 433K C   N    3368 3370  -55.08  -70.20   21.63    1.29  -62.90  165.91   21.98\n",
      "   81        433K 433K N   CA   3370 3371  124.92  140.40                  -40.80\n",
      "   82  17653 433K 434I C   N    3377 3379 -143.38 -120.60   29.16    1.32  -63.40 -174.03   33.40\n",
      "   82        434I 434I N   CA   3379 3380  148.50  130.30                  -43.60\n",
      "   83  17658 438C 439A C   N    3414 3416  -79.26  -68.20   15.79    0.99  -62.50  163.39   27.63\n",
      "   83        439A 439A N   CA   3416 3417  156.58  145.30                  -40.90\n",
      "   84  17659 439A 440S C   N    3419 3421 -137.56 -136.60    8.28    0.40  -64.10 -178.86   18.36\n",
      "   84        440S 440S N   CA   3421 3422  159.42  151.20                  -35.00\n",
      "   85  17660 440S 441K C   N    3425 3427 -102.80  -62.90   79.82    9.18  -62.90   79.82    9.18\n",
      "   85        441K 441K N   CA   3427 3428   28.33  -40.80                  -40.80\n",
      "   86  17663 443D 444N C   N    3450 3452  -66.96  -63.20    7.71    0.83   55.90  143.36   17.65\n",
      "   86        444N 444N N   CA   3452 3453  -34.37  -41.10                   39.50\n",
      "   87  17668 448L 449K C   N    3495 3497 -120.19 -118.00    6.04    0.26  -62.90 -176.36   27.26\n",
      "   87        449K 449K N   CA   3497 3498  144.73  139.10                  -40.80\n",
      "   88  17669 449K 450Q C   N    3504 3506 -162.83 -121.10  117.75    4.77  -63.80  121.22   22.01\n",
      "   88        450Q 450Q N   CA   3506 3507 -110.19  139.70                  -40.30\n",
      "   89  17676 456C 457R C   N    3554 3556 -136.83 -125.20   18.57    0.60  -63.00  179.69   29.32\n",
      "   89        457R 457R N   CA   3556 3557  155.08  140.60                  -41.10\n",
      "   90  17689 469S 470H C   N    3662 3664  -74.28 -125.60   57.79    1.51  -63.20  154.94   18.70\n",
      "   90        470H 470H N   CA   3664 3665  112.24  138.80                  -42.30\n",
      "   91  17690 470H 471V C   N    3672 3674 -119.02 -125.40   41.00    2.18  -62.40  145.28   21.87\n",
      "   91        471V 471V N   CA   3674 3675 -176.19  143.30                  -42.40\n",
      "   92  17691 471V 472R C   N    3679 3681  -63.35  -72.10   30.48    2.15  -63.00  153.80   20.75\n",
      "   92        472R 472R N   CA   3681 3682  112.70  141.90                  -41.10\n",
      "   93  17695 475I 476A C   N    3713 3715  -58.57  -68.20   19.72    1.88  -62.50  156.63   25.48\n",
      "   93        476A 476A N   CA   3715 3716  162.52  145.30                  -40.90\n",
      "   94  17696 476A 477A C   N    3718 3720  -75.54  -68.20   59.18    5.03  -62.50  128.14   20.40\n",
      "   94        477A 477A N   CA   3720 3721   86.58  145.30                  -40.90\n",
      "   95  17698 478E 479R C   N    3732 3734  -53.80 -125.20   73.19    3.35  -63.00  162.51   21.35\n",
      "   95        479R 479R N   CA   3734 3735  156.66  140.60                  -41.10\n",
      "   96  17700 480G 481F C   N    3747 3749   52.11   58.10   12.85    0.65  -63.20  145.40   26.22\n",
      "   96        481F 481F N   CA   3749 3750   44.27   32.90                  -44.30\n",
      "   97  17701 481F 482V C   N    3758 3760 -124.68 -125.40   40.12    2.01  -62.40  147.93   22.55\n",
      "   97        482V 482V N   CA   3760 3761 -176.59  143.30                  -42.40\n",
      "   98  17702 482V 483N C   N    3765 3767  -67.46  -63.20   14.13    2.01 -119.90  176.40    9.04\n",
      "   98        483N 483N N   CA   3767 3768  -54.57  -41.10                  137.00\n",
      "   99  17704 484V 485K C   N    3780 3782    7.53  -62.90  117.33   19.37  -62.90  117.33   19.37\n",
      "   99        485K 485K N   CA   3782 3783   53.04  -40.80                  -40.80\n",
      "  100  17721 501F 502G C   N    3928 3930  -62.44  -62.40    4.13    0.62   82.20  154.33   11.80\n",
      "  100        502G 502G N   CA   3930 3931  -45.33  -41.20                    8.50\n",
      "  101  17722 502G 503T C   N    3932 3934  -92.01  -78.10   30.39    1.76  -63.20  167.38   19.69\n",
      "  101        503T 503T N   CA   3934 3935  122.78  149.80                  -42.10\n",
      "  102  17723 503T 504E C   N    3939 3941  -72.19  -69.30   48.96    3.84  -63.60  134.20   17.44\n",
      "  102        504E 504E N   CA   3941 3942   93.62  142.50                  -40.30\n",
      "  103  17741 521K 522D C   N    4074 4076  -90.86  -96.50   58.96    2.48  -63.30   99.41   10.90\n",
      "  103        522D 522D N   CA   4076 4077   55.52  114.20                  -40.00\n",
      "  104  17742 522D 523K C   N    4082 4084 -129.47 -118.00   43.84    2.24  -62.90  152.85   17.56\n",
      "  104        523K 523K N   CA   4084 4085   96.79  139.10                  -40.80\n",
      "  105  17743 523K 524G C   N    4091 4093  -65.83  -62.40    3.70    0.70   82.20  156.60   11.91\n",
      "  105        524G 524G N   CA   4093 4094  -42.60  -41.20                    8.50\n",
      "  106  17746 526K 527V C   N    4113 4115  -59.92  -73.50   16.15    1.10  -62.40  172.87   22.02\n",
      "  106        527V 527V N   CA   4115 4116  130.46  139.20                  -42.40\n",
      "  107  17747 527V 528N C   N    4120 4122  -51.40  -71.20   26.49    2.14  -63.20  158.95   19.25\n",
      "  107        528N 528N N   CA   4122 4123  160.39  142.80                  -41.10\n",
      "  108  17758 538K 539S C   N    4200 4202 -172.20 -136.60   35.71    1.35  -64.10 -157.63   22.46\n",
      "  108        539S 539S N   CA   4202 4203  153.92  151.20                  -35.00\n",
      "  109  17759 539S 540D C   N    4206 4208 -166.57  -96.50   70.15    2.87  -70.90  101.12    8.40\n",
      "  109        540D 540D N   CA   4208 4209  117.55  114.20                  150.30\n",
      "  110  17760 540D 541M C   N    4214 4216 -134.23 -125.60    9.91    0.50  -63.40 -170.16   23.97\n",
      "  110        541M 541M N   CA   4216 4217  135.63  140.50                  -40.50\n",
      "  111  17770 550L 551G C   N    4286 4288  -70.33  -80.20   33.07    0.86   82.20 -156.95    6.85\n",
      "  111        551G 551G N   CA   4288 4289  142.54  174.10                    8.50\n",
      "  112  17772 552N 553N C   N    4298 4300  -64.52  -63.20    6.07    0.83 -119.90 -175.51    9.46\n",
      "  112        553N 553N N   CA   4300 4301  -47.02  -41.10                  137.00\n",
      "  113  17784 564V 565I C   N    4397 4399 -124.66 -120.60   22.55    1.44  -63.40  163.63   25.12\n",
      "  113        565I 565I N   CA   4399 4400  108.12  130.30                  -43.60\n",
      "  114  17796 576V 577K C   N    4494 4496  -83.46  -62.90   80.15    9.49  -62.90   80.15    9.49\n",
      "  114        577K 577K N   CA   4496 4497   36.67  -40.80                  -40.80\n",
      "  115  17809 589T 590L C   N    4586 4588  -68.33  -70.70   26.79    2.12  -63.50  150.59   21.19\n",
      "  115        590L 590L N   CA   4588 4589  168.29  141.60                  -41.20\n",
      "  116  17821 601Y 602S C   N    4689 4691  -69.43  -72.40    3.31    0.27  -64.10  171.21   12.73\n",
      "  116        602S 602S N   CA   4691 4692  153.87  152.40                  -35.00\n",
      "  117  17822 602S 603R C   N    4695 4697   44.23  -63.00  153.56   27.44  -63.00  153.56   27.44\n",
      "  117        603R 603R N   CA   4697 4698   68.82  -41.10                  -41.10\n",
      "  118  17823 603R 604L C   N    4706 4708  -45.84  -70.70   58.85    3.70  -63.50  130.66   19.20\n",
      "  118        604L 604L N   CA   4708 4709   88.26  141.60                  -41.20\n",
      "  119  17825 605V 606R C   N    4721 4723 -116.40 -125.20   93.27    4.31  -63.00  103.66   11.73\n",
      "  119        606R 606R N   CA   4723 4724   47.75  140.60                  -41.10\n",
      "  120  17834 614A 615D C   N    4796 4798  -53.59  -96.50   50.21    2.08  -63.30  128.49   16.46\n",
      "  120        615D 615D N   CA   4798 4799   88.12  114.20                  -40.00\n",
      "  121  17835 615D 616D C   N    4804 4806   55.73   54.50   28.01    1.96  -63.30  130.26   22.21\n",
      "  121        616D 616D N   CA   4806 4807   12.91   40.90                  -40.00\n",
      "  122  17837 617L 618L C   N    4820 4822 -131.00 -108.50   60.93    3.01  -63.50  146.19   24.47\n",
      "  122        618L 618L N   CA   4822 4823 -170.88  132.50                  -41.20\n",
      "  123  17845 625E 626T C   N    4881 4883  -72.70 -124.80   57.12    2.34  -63.20  162.47   20.22\n",
      "  123        626T 626T N   CA   4883 4884  120.09  143.50                  -42.10\n",
      "  124  17847 627Q 628R C   N    4897 4899 -117.27 -125.20   46.47    2.06  -63.00  146.34   16.90\n",
      "  124        628R 628R N   CA   4899 4900   94.81  140.60                  -41.10\n",
      "  125  17848 628R 629K C   N    4908 4910   59.65   56.60   11.55    0.63 -118.00 -150.19    6.98\n",
      "  125        629K 629K N   CA   4910 4911   27.46   38.60                  139.10\n",
      "  126  17849 629K 630A C   N    4917 4919  -70.70  -62.50    9.76    1.59 -134.00 -171.66   11.58\n",
      "  126        630A 630A N   CA   4919 4920  -35.61  -40.90                  147.00\n",
      "  127  17850 630A 631T C   N    4922 4924   73.80  -63.20  144.68   19.67  -63.20  144.68   19.67\n",
      "  127        631T 631T N   CA   4924 4925  -88.62  -42.10                  -42.10\n",
      "  128  17871 651F 652T C   N    5089 5091  -69.03  -78.10   10.11    0.67  -63.20  163.74   21.38\n",
      "  128        652T 652T N   CA   5091 5092  154.27  149.80                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   44   94  380  512  540  525  644  728  756  782\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00183.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      653     5106     5106\n",
      "openf___224_> Open           EAO00183.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     22298699   21776.074    21.266\n",
      "read_mo_297_> Segments, residues, atoms:        1      653     5106\n",
      "read_mo_298_> Segment:        1  1:A  653:A     5106\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5106     5106        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    27134\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    30588\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    32747\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    34959\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    37150\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    38832\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    40673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    42690\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    43535\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45692\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45976\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "iupac_m_487_> NH1/2 swapped:      178.4524       54  54\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8520       61  61\n",
      "iupac_m_486_> OE1/2 will be swapped:      -93.9329       78  78\n",
      "iupac_m_486_> OE1/2 will be swapped:     -110.6521       98  98\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.2090       99  99\n",
      "iupac_m_487_> NH1/2 swapped:      179.8012      103  103\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.3306      115  115\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.3029      145  145\n",
      "iupac_m_485_> OD1/2 will be swapped:     -173.5305      151  151\n",
      "iupac_m_487_> NH1/2 swapped:      179.9344      152  152\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4217      186  186\n",
      "iupac_m_487_> NH1/2 swapped:     -173.7949      197  197\n",
      "iupac_m_485_> OD1/2 will be swapped:      177.4679      209  209\n",
      "iupac_m_485_> OD1/2 will be swapped:     -172.8212      213  213\n",
      "iupac_m_485_> OD1/2 will be swapped:       98.2285      216  216\n",
      "iupac_m_487_> NH1/2 swapped:      178.0417      226  226\n",
      "iupac_m_485_> OD1/2 will be swapped:      133.8251      242  242\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.5493      252  252\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -93.0845      256  256\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      174.8604      269  269\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.6935      293  293\n",
      "iupac_m_485_> OD1/2 will be swapped:      141.9119      299  299\n",
      "iupac_m_487_> NH1/2 swapped:      178.1000      332  332\n",
      "iupac_m_485_> OD1/2 will be swapped:      138.9547      335  335\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.2607      364  364\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.9070      378  378\n",
      "iupac_m_486_> OE1/2 will be swapped:       99.2735      382  382\n",
      "iupac_m_487_> NH1/2 swapped:      178.8615      385  385\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.8608      399  399\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -156.2796      429  429\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.2688      430  430\n",
      "iupac_m_485_> OD1/2 will be swapped:      117.4385      443  443\n",
      "iupac_m_485_> OD1/2 will be swapped:     -172.4283      454  454\n",
      "iupac_m_487_> NH1/2 swapped:      178.3114      457  457\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1439      464  464\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3694      472  472\n",
      "iupac_m_486_> OE1/2 will be swapped:      165.4369      478  478\n",
      "iupac_m_485_> OD1/2 will be swapped:     -148.6843      522  522\n",
      "iupac_m_486_> OE1/2 will be swapped:      110.7713      537  537\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.9541      543  543\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.9904      573  573\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -109.2093      599  599\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5869      603  603\n",
      "iupac_m_487_> NH1/2 swapped:      179.1023      606  606\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.0158      609  609\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.0455\n",
      "iupac_m_487_> NH1/2 swapped:     -177.1354      628  628\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.1461      643  643\n",
      "iupac_m_487_> NH1/2 swapped:      178.8086      649  649\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   829123\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -47037.3516\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -47037.351562\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    21.746000\n",
      "Sequence length          :          653\n",
      "Compactness              :     0.054462\n",
      "Native energy (pair)     :   150.190993\n",
      "Native energy (surface)  :    27.793671\n",
      "Native energy (combined) :     6.434222\n",
      "Z score (pair)           :    -1.651020\n",
      "Z score (surface)        :    -2.424247\n",
      "Z score (combined)       :    -2.576095\n",
      "GA341 score              :     0.081666\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10380\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4289.0005\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5180       0      0   0.004   0.004      29.645       1.000\n",
      " 2 Bond angle potential               :    6986       0      6   1.798   1.798      437.49       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3019       0    115  49.169  49.169      1142.5       1.000\n",
      " 4 Stereochemical improper torsion pot:    2035       0      0   1.130   1.130      51.230       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10380       0      0   0.003   0.003      9.7770       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10350       0     16   0.769   0.769      668.30       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10301       1     37   0.846   0.846      815.24       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     652       1      7   8.138   8.138      144.20       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     570       0      3  85.987  85.987      171.81       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     437       0      2  76.925  76.925      170.78       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     206       0      2  75.125  75.125      141.67       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      82       0      0  95.123  95.123      57.221       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5331       0      1   0.608   0.608      99.520       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     651     123     90  32.032  74.917      272.84       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     776       0      2   1.748   1.748      76.727       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00183.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00183.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   79897.7500\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  18411 540D 540D CA  C    4209 4214   -3.37 -179.18  175.81   40.86 -179.18  175.81   40.86\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17227   7V   8V C   N      52   54 -130.92 -125.40    7.12    0.19  -62.40 -176.89   27.39\n",
      "    1          8V   8V N   CA     54   55  147.80  143.30                  -42.40\n",
      "    2  17239  19K  20M C   N     147  149  -68.46  -73.00    4.85    0.39  -63.40  174.87   26.50\n",
      "    2         20M  20M N   CA    149  150  144.70  143.00                  -40.50\n",
      "    3  17243  23K  24L C   N     180  182  -93.16 -108.50   89.70    4.67  -63.50   90.33   11.24\n",
      "    3         24L  24L N   CA    182  183   44.12  132.50                  -41.20\n",
      "    4  17245  25N  26L C   N     196  198  -70.37  -70.70   19.14    1.46  -63.50  158.21   22.37\n",
      "    4         26L  26L N   CA    198  199  160.73  141.60                  -41.20\n",
      "    5  17246  26L  27P C   N     204  206  -45.86  -58.70   14.30    1.38  -64.50  176.98   14.21\n",
      "    5         27P  27P N   CA    206  207  -36.81  -30.50                  147.20\n",
      "    6  17247  27P  28Q C   N     211  213   45.44   55.10   27.22    1.31  -63.80  152.24   27.44\n",
      "    6         28Q  28Q N   CA    213  214   65.74   40.30                  -40.30\n",
      "    7  17248  28Q  29I C   N     220  222 -119.05 -120.60   53.94    3.14  -63.40  143.41   25.55\n",
      "    7         29I  29I N   CA    222  223 -175.78  130.30                  -43.60\n",
      "    8  17252  32V  33G C   N     247  249  -61.31  -62.40    3.05    0.42   82.20  152.83   11.67\n",
      "    8         33G  33G N   CA    249  250  -44.05  -41.20                    8.50\n",
      "    9  17265  45A  46I C   N     332  334   61.68  -97.30  165.28    8.59 -120.60 -175.83   14.08\n",
      "    9         46I  46I N   CA    334  335   82.01  127.20                  130.30\n",
      "   10  17274  54R  55G C   N     405  407   89.62   78.70   10.97    0.58   82.20  173.74    8.06\n",
      "   10         55G  55G N   CA    407  408 -165.08 -166.10                    8.50\n",
      "   11  17275  55G  56S C   N     409  411  -82.17  -72.40   11.78    0.61  -64.10  167.00   13.26\n",
      "   11         56S  56S N   CA    411  412  158.98  152.40                  -35.00\n",
      "   12  17291  71L  72P C   N     529  531  -79.95  -64.50   38.82    2.28  -58.70  148.22   14.00\n",
      "   12         72P  72P N   CA    531  532 -177.19  147.20                  -30.50\n",
      "   13  17293  73K  74T C   N     545  547 -159.38 -124.80   36.60    1.60  -63.20 -171.25   30.35\n",
      "   13         74T  74T N   CA    547  548  155.50  143.50                  -42.10\n",
      "   14  17310  90Y  91F C   N     695  697 -113.39 -124.20   39.43    1.56  -63.20  157.87   18.90\n",
      "   14         91F  91F N   CA    697  698  105.38  143.30                  -44.30\n",
      "   15  17311  91F  92D C   N     706  708  -81.44  -70.90   40.19    1.51  -63.30  132.17   17.46\n",
      "   15         92D  92D N   CA    708  709 -170.92  150.30                  -40.00\n",
      "   16  17317  97N  98E C   N     758  760  -46.83  -69.30   23.43    1.99  -63.60  171.37   21.97\n",
      "   16         98E  98E N   CA    760  761  149.15  142.50                  -40.30\n",
      "   17  17318  98E  99E C   N     767  769  -65.85  -63.60    7.14    1.08 -117.80 -176.38    9.92\n",
      "   17         99E  99E N   CA    769  770  -47.08  -40.30                  136.80\n",
      "   18  17319  99E 100I C   N     776  778   72.38 -120.60  167.03   11.56 -120.60  167.03   11.56\n",
      "   18        100I 100I N   CA    778  779  131.46  130.30                  130.30\n",
      "   19  17342 122V 123Y C   N     954  956  -83.02  -98.40   73.91    7.78  -63.50  101.40   15.13\n",
      "   19        123Y 123Y N   CA    956  957   56.10  128.40                  -43.40\n",
      "   20  17350 130L 131T C   N    1020 1022 -134.96 -124.80   10.34    0.50  -63.20 -173.16   28.68\n",
      "   20        131T 131T N   CA   1022 1023  145.39  143.50                  -42.10\n",
      "   21  17351 131T 132L C   N    1027 1029  -72.63  -70.70    3.95    0.37  -63.50  179.59   24.43\n",
      "   21        132L 132L N   CA   1029 1030  138.16  141.60                  -41.20\n",
      "   22  17352 132L 133V C   N    1035 1037 -126.83 -125.40   18.23    0.94  -62.40  179.49   19.90\n",
      "   22        133V 133V N   CA   1037 1038  125.13  143.30                  -42.40\n",
      "   23  17357 137G 138L C   N    1069 1071  -59.35  -63.50    6.39    1.14 -108.50  175.85    8.91\n",
      "   23        138L 138L N   CA   1071 1072  -36.34  -41.20                  132.50\n",
      "   24  17371 151D 152R C   N    1173 1175 -162.27 -125.20   67.53    2.30   57.30 -147.83   24.23\n",
      "   24        152R 152R N   CA   1175 1176 -162.96  140.60                   38.00\n",
      "   25  17388 168I 169I C   N    1313 1315  -92.86 -120.60   48.81    2.01  -63.40  136.95   21.51\n",
      "   25        169I 169I N   CA   1315 1316   90.14  130.30                  -43.60\n",
      "   26  17390 170L 171A C   N    1329 1331  -55.10  -68.20   15.14    1.02  -62.50  178.76   29.66\n",
      "   26        171A 171A N   CA   1331 1332  137.71  145.30                  -40.90\n",
      "   27  17399 179L 180A C   N    1391 1393   75.42   55.40   28.60    1.49  -62.50  149.88   30.41\n",
      "   27        180A 180A N   CA   1393 1394   17.77   38.20                  -40.90\n",
      "   28  17411 191L 192D C   N    1479 1481 -125.60  -96.50   57.92    2.40  -63.30  121.34   12.90\n",
      "   28        192D 192D N   CA   1481 1482   64.13  114.20                  -40.00\n",
      "   29  17417 197R 198T C   N    1526 1528  -58.00  -63.20    5.86    0.74 -124.80 -175.76    9.80\n",
      "   29        198T 198T N   CA   1528 1529  -44.80  -42.10                  143.50\n",
      "   30  17418 198T 199V C   N    1533 1535 -129.34 -125.40   26.39    1.40  -62.40  173.08   19.09\n",
      "   30        199V 199V N   CA   1535 1536  117.21  143.30                  -42.40\n",
      "   31  17425 205L 206D C   N    1583 1585 -102.05  -70.90   31.31    2.26  -63.30  170.96   23.59\n",
      "   31        206D 206D N   CA   1585 1586  153.49  150.30                  -40.00\n",
      "   32  17445 225L 226R C   N    1728 1730  -70.56 -125.20   64.75    3.40  -63.00  143.77   19.93\n",
      "   32        226R 226R N   CA   1730 1731  175.33  140.60                  -41.10\n",
      "   33  17446 226R 227H C   N    1739 1741  -54.99  -67.60   35.07    2.31  -63.20  149.81   19.25\n",
      "   33        227H 227H N   CA   1741 1742  107.28  140.00                  -42.30\n",
      "   34  17449 229F 230V C   N    1764 1766 -114.74 -125.40   30.32    1.70  -62.40  155.02   22.84\n",
      "   34        230V 230V N   CA   1766 1767  171.69  143.30                  -42.40\n",
      "   35  17450 230V 231G C   N    1771 1773   85.16   82.20   57.54    2.64  -62.40  147.76   26.07\n",
      "   35        231G 231G N   CA   1773 1774  -48.96    8.50                  -41.20\n",
      "   36  17454 234N 235R C   N    1797 1799  -76.18 -125.20   52.43    2.57  -63.00  160.24   22.54\n",
      "   36        235R 235R N   CA   1799 1800  159.20  140.60                  -41.10\n",
      "   37  17455 235R 236S C   N    1808 1810   73.33   56.90   64.34    2.90  -64.10  137.74   17.94\n",
      "   37        236S 236S N   CA   1810 1811  -25.81   36.40                  -35.00\n",
      "   38  17459 239D 240I C   N    1840 1842  -70.94  -97.30   32.58    2.16  -63.40  170.21   28.16\n",
      "   38        240I 240I N   CA   1842 1843  146.36  127.20                  -43.60\n",
      "   39  17481 261P 262S C   N    2019 2021 -133.80 -136.60   40.71    2.12  -64.10  150.32   15.90\n",
      "   39        262S 262S N   CA   2021 2022 -168.19  151.20                  -35.00\n",
      "   40  17500 280N 281Q C   N    2168 2170  -40.55 -121.10   88.27    2.75  -63.80  145.77   22.68\n",
      "   40        281Q 281Q N   CA   2170 2171  103.60  139.70                  -40.30\n",
      "   41  17512 292P 293E C   N    2267 2269   55.50   54.60    1.33    0.17  -63.60  145.56   25.02\n",
      "   41        293E 293E N   CA   2269 2270   43.38   42.40                  -40.30\n",
      "   42  17523 303G 304S C   N    2356 2358  -54.96  -72.40   24.92    1.18  -64.10  169.84   12.87\n",
      "   42        304S 304S N   CA   2358 2359  134.59  152.40                  -35.00\n",
      "   43  17524 304S 305T C   N    2362 2364  -81.32 -124.80   96.53    3.07  -63.20  101.05   11.85\n",
      "   43        305T 305T N   CA   2364 2365   57.31  143.50                  -42.10\n",
      "   44  17529 309M 310E C   N    2403 2405  -76.52  -63.60   85.84   10.79  -63.60   85.84   10.79\n",
      "   44        310E 310E N   CA   2405 2406   44.56  -40.30                  -40.30\n",
      "   45  17539 319N 320I C   N    2481 2483 -105.91 -120.60   79.55    3.99  -63.40  104.73   16.03\n",
      "   45        320I 320I N   CA   2483 2484   52.12  130.30                  -43.60\n",
      "   46  17542 322P 323S C   N    2504 2506 -113.22 -136.60   72.12    3.09  -64.10  127.79    7.94\n",
      "   46        323S 323S N   CA   2506 2507   82.97  151.20                  -35.00\n",
      "   47  17556 336T 337L C   N    2615 2617  -67.36  -70.70   21.47    1.76  -63.50  156.04   21.89\n",
      "   47        337L 337L N   CA   2617 2618  162.81  141.60                  -41.20\n",
      "   48  17565 345A 346T C   N    2677 2679  -61.59  -63.20    9.12    1.07 -124.80  177.09    9.39\n",
      "   48        346T 346T N   CA   2679 2680  -51.08  -42.10                  143.50\n",
      "   49  17566 346T 347D C   N    2684 2686   54.82   54.50  102.45    7.56  -70.90  125.91    9.30\n",
      "   49        347D 347D N   CA   2686 2687  143.34   40.90                  150.30\n",
      "   50  17574 354G 355G C   N    2742 2744 -117.02  -80.20   36.89    2.39   82.20 -127.51   17.76\n",
      "   50        355G 355G N   CA   2744 2745  176.44  174.10                    8.50\n",
      "   51  17575 355G 356A C   N    2746 2748 -128.33 -134.00    7.77    0.23  -62.50 -170.76   34.53\n",
      "   51        356A 356A N   CA   2748 2749  141.68  147.00                  -40.90\n",
      "   52  17576 356A 357R C   N    2751 2753 -143.86 -125.20   21.65    1.13  -63.00 -171.11   21.54\n",
      "   52        357R 357R N   CA   2753 2754  129.61  140.60                  -41.10\n",
      "   53  17578 358L 359D C   N    2770 2772   93.01   54.50   97.79   10.24  -63.30 -128.48   38.17\n",
      "   53        359D 359D N   CA   2772 2773  130.78   40.90                  -40.00\n",
      "   54  17581 361I 362F C   N    2798 2800  -81.24 -124.20   70.92    2.08  -63.20  132.41   17.21\n",
      "   54        362F 362F N   CA   2800 2801   86.88  143.30                  -44.30\n",
      "   55  17589 369Y 370V C   N    2865 2867   54.28  -62.40  151.35   25.61  -62.40  151.35   25.61\n",
      "   55        370V 370V N   CA   2867 2868   54.00  -42.40                  -42.40\n",
      "   56  17590 370V 371N C   N    2872 2874  -75.35  -71.20   26.77    1.60  -63.20  150.15   19.45\n",
      "   56        371N 371N N   CA   2874 2875  169.25  142.80                  -41.10\n",
      "   57  17592 372G 373L C   N    2884 2886  -60.18  -70.70   28.35    1.81  -63.50  156.51   21.92\n",
      "   57        373L 373L N   CA   2886 2887  115.28  141.60                  -41.20\n",
      "   58  17594 374T 375A C   N    2899 2901  -63.13  -68.20   36.39    2.75  -62.50  150.17   24.58\n",
      "   58        375A 375A N   CA   2901 2902  109.27  145.30                  -40.90\n",
      "   59  17599 379L 380T C   N    2939 2941  -58.61  -63.20    8.34    0.88 -124.80 -179.95    9.60\n",
      "   59        380T 380T N   CA   2941 2942  -49.06  -42.10                  143.50\n",
      "   60  17600 380T 381D C   N    2946 2948 -144.94  -96.50   51.12    2.08  -70.90   90.73    7.56\n",
      "   60        381D 381D N   CA   2948 2949   97.87  114.20                  150.30\n",
      "   61  17601 381D 382E C   N    2954 2956 -166.94 -117.80   50.63    1.59  -69.30   97.86    7.18\n",
      "   61        382E 382E N   CA   2956 2957  149.01  136.80                  142.50\n",
      "   62  17605 385R 386I C   N    2994 2996  -65.57  -97.30   32.64    1.46  -63.40  178.45   29.17\n",
      "   62        386I 386I N   CA   2996 2997  134.83  127.20                  -43.60\n",
      "   63  17606 386I 387N C   N    3002 3004   74.69   55.90   38.50    1.85  -63.20  145.68   23.65\n",
      "   63        387N 387N N   CA   3004 3005    5.90   39.50                  -41.10\n",
      "   64  17608 388A 389R C   N    3015 3017 -116.61 -125.20   10.22    0.54  -63.00 -179.10   28.14\n",
      "   64        389R 389R N   CA   3017 3018  146.13  140.60                  -41.10\n",
      "   65  17609 389R 390N C   N    3026 3028 -175.07 -119.90  108.43    3.54  -63.20  142.68   23.35\n",
      "   65        390N 390N N   CA   3028 3029 -129.65  137.00                  -41.10\n",
      "   66  17628 408L 409S C   N    3167 3169 -108.19 -136.60   58.49    3.32  -64.10  130.35   12.54\n",
      "   66        409S 409S N   CA   3169 3170 -157.67  151.20                  -35.00\n",
      "   67  17629 409S 410R C   N    3173 3175  -63.54  -72.10   28.05    2.33  -63.00  150.29   20.34\n",
      "   67        410R 410R N   CA   3175 3176  168.61  141.90                  -41.10\n",
      "   68  17631 411Q 412Q C   N    3193 3195  -58.58  -63.80    5.22    0.88 -121.10 -169.73    7.82\n",
      "   68        412Q 412Q N   CA   3195 3196  -40.01  -40.30                  139.70\n",
      "   69  17633 413I 414A C   N    3210 3212  -59.33  -68.20   18.13    1.18  -62.50  170.42   28.08\n",
      "   69        414A 414A N   CA   3212 3213  129.49  145.30                  -40.90\n",
      "   70  17635 415R 416L C   N    3226 3228  -64.50  -63.50    1.72    0.30 -108.50 -179.45    9.24\n",
      "   70        416L 416L N   CA   3228 3229  -42.60  -41.20                  132.50\n",
      "   71  17639 419P 420S C   N    3259 3261 -106.05 -136.60   30.67    1.15  -64.10 -178.56   16.00\n",
      "   71        420S 420S N   CA   3261 3262  148.47  151.20                  -35.00\n",
      "   72  17640 420S 421Q C   N    3265 3267   63.72   55.10   35.26    1.87  -63.80  135.70   24.28\n",
      "   72        421Q 421Q N   CA   3267 3268    6.11   40.30                  -40.30\n",
      "   73  17642 422K 423C C   N    3283 3285  -68.33  -63.00    6.68    1.18 -117.90 -179.30    7.74\n",
      "   73        423C 423C N   CA   3285 3286  -45.13  -41.10                  141.10\n",
      "   74  17645 425Q 426F C   N    3305 3307   73.93   58.10   22.93    1.50  -63.20  149.93   26.74\n",
      "   74        426F 426F N   CA   3307 3308   16.31   32.90                  -44.30\n",
      "   75  17651 431L 432I C   N    3360 3362  -73.80 -120.60   59.59    2.71  -63.40  137.41   22.17\n",
      "   75        432I 432I N   CA   3362 3363   93.42  130.30                  -43.60\n",
      "   76  17652 432I 433K C   N    3368 3370 -163.25 -118.00   52.14    1.73  -62.90 -176.03   29.84\n",
      "   76        433K 433K N   CA   3370 3371  165.01  139.10                  -40.80\n",
      "   77  17653 433K 434I C   N    3377 3379 -122.45 -120.60   13.82    0.72  -63.40 -177.76   32.04\n",
      "   77        434I 434I N   CA   3379 3380  144.00  130.30                  -43.60\n",
      "   78  17658 438C 439A C   N    3414 3416 -128.94 -134.00    5.98    0.14  -62.50 -172.54   34.28\n",
      "   78        439A 439A N   CA   3416 3417  143.81  147.00                  -40.90\n",
      "   79  17659 439A 440S C   N    3419 3421  -96.70  -72.40   32.72    1.58  -64.10  154.18   13.37\n",
      "   79        440S 440S N   CA   3421 3422  174.31  152.40                  -35.00\n",
      "   80  17660 440S 441K C   N    3425 3427  -91.37  -70.20   79.40    6.22  -62.90  108.47   12.82\n",
      "   80        441K 441K N   CA   3427 3428   63.87  140.40                  -40.80\n",
      "   81  17672 452V 453V C   N    3525 3527  -63.36  -73.50   10.22    0.71  -62.40  179.64   22.79\n",
      "   81        453V 453V N   CA   3527 3528  137.97  139.20                  -42.40\n",
      "   82  17673 453V 454D C   N    3532 3534   63.43   54.50   31.54    1.69  -63.30  136.48   23.22\n",
      "   82        454D 454D N   CA   3534 3535   10.65   40.90                  -40.00\n",
      "   83  17676 456C 457R C   N    3554 3556 -115.73 -125.20   21.97    1.21  -63.00  167.03   26.20\n",
      "   83        457R 457R N   CA   3556 3557  160.42  140.60                  -41.10\n",
      "   84  17684 464R 465A C   N    3629 3631  -64.43  -62.50    2.82    0.56 -134.00 -176.27   11.33\n",
      "   84        465A 465A N   CA   3631 3632  -42.95  -40.90                  147.00\n",
      "   85  17685 465A 466P C   N    3634 3636  -73.15  -58.70   24.80    1.52  -64.50  157.78   12.24\n",
      "   85        466P 466P N   CA   3636 3637  -10.34  -30.50                  147.20\n",
      "   86  17689 469S 470H C   N    3662 3664  -68.31 -125.60   68.74    1.72  -63.20  143.20   17.59\n",
      "   86        470H 470H N   CA   3664 3665  100.81  138.80                  -42.30\n",
      "   87  17690 470H 471V C   N    3672 3674 -152.59 -125.40   40.13    1.15  -62.40  170.59   27.05\n",
      "   87        471V 471V N   CA   3674 3675  172.81  143.30                  -42.40\n",
      "   88  17691 471V 472R C   N    3679 3681  -63.26  -72.10   13.18    1.15  -63.00  167.23   22.61\n",
      "   88        472R 472R N   CA   3681 3682  151.67  141.90                  -41.10\n",
      "   89  17693 473T 474I C   N    3697 3699 -134.10 -120.60   14.78    0.80  -63.40 -166.68   29.71\n",
      "   89        474I 474I N   CA   3699 3700  136.32  130.30                  -43.60\n",
      "   90  17695 475I 476A C   N    3713 3715  -75.95  -68.20    8.89    0.87  -62.50  178.67   29.96\n",
      "   90        476A 476A N   CA   3715 3716  140.94  145.30                  -40.90\n",
      "   91  17696 476A 477A C   N    3718 3720  -67.64  -68.20   17.31    1.38  -62.50  168.98   27.44\n",
      "   91        477A 477A N   CA   3720 3721  128.00  145.30                  -40.90\n",
      "   92  17698 478E 479R C   N    3732 3734  -94.56 -125.20   55.46    3.07  -63.00  135.80   20.53\n",
      "   92        479R 479R N   CA   3734 3735 -173.18  140.60                  -41.10\n",
      "   93  17699 479R 480G C   N    3743 3745  118.11   82.20   55.97    1.61  -62.40  179.61   31.80\n",
      "   93        480G 480G N   CA   3745 3746  -34.43    8.50                  -41.20\n",
      "   94  17700 480G 481F C   N    3747 3749  -87.36  -71.40   43.51    3.82  -63.20  146.53   18.78\n",
      "   94        481F 481F N   CA   3749 3750  100.22  140.70                  -44.30\n",
      "   95  17701 481F 482V C   N    3758 3760 -166.87 -125.40   74.76    2.43  -62.40  153.23   25.44\n",
      "   95        482V 482V N   CA   3760 3761 -154.50  143.30                  -42.40\n",
      "   96  17702 482V 483N C   N    3765 3767  -71.12  -63.20   10.35    1.19 -119.90  178.25    6.89\n",
      "   96        483N 483N N   CA   3767 3768  -34.44  -41.10                  137.00\n",
      "   97  17704 484V 485K C   N    3780 3782   68.12  -62.90  139.01   20.28  -62.90  139.01   20.28\n",
      "   97        485K 485K N   CA   3782 3783  -87.24  -40.80                  -40.80\n",
      "   98  17715 495R 496S C   N    3875 3877  -74.02  -72.40   62.01    3.59  -64.10  111.05    8.67\n",
      "   98        496S 496S N   CA   3877 3878 -145.61  152.40                  -35.00\n",
      "   99  17716 496S 497F C   N    3881 3883   65.55   58.10   22.78    1.25  -63.20  140.27   25.00\n",
      "   99        497F 497F N   CA   3883 3884   11.37   32.90                  -44.30\n",
      "  100  17718 498L 499K C   N    3900 3902  160.58   56.60  130.19    9.64   56.60  130.19    9.64\n",
      "  100        499K 499K N   CA   3902 3903  -39.75   38.60                   38.60\n",
      "  101  17721 501F 502G C   N    3928 3930  -66.32  -62.40    5.42    0.77   82.20  155.46   11.74\n",
      "  101        502G 502G N   CA   3930 3931  -37.45  -41.20                    8.50\n",
      "  102  17723 503T 504E C   N    3939 3941   56.83   54.60    8.92    0.86  -63.60  151.15   25.94\n",
      "  102        504E 504E N   CA   3941 3942   51.04   42.40                  -40.30\n",
      "  103  17742 522D 523K C   N    4082 4084  -79.53  -70.20   32.45    2.55  -62.90  151.04   18.76\n",
      "  103        523K 523K N   CA   4084 4085  109.32  140.40                  -40.80\n",
      "  104  17743 523K 524G C   N    4091 4093  -66.77  -62.40    4.39    0.80   82.20  157.17   11.93\n",
      "  104        524G 524G N   CA   4093 4094  -41.60  -41.20                    8.50\n",
      "  105  17746 526K 527V C   N    4113 4115  -62.91  -73.50   11.13    0.75  -62.40  178.19   22.52\n",
      "  105        527V 527V N   CA   4115 4116  135.79  139.20                  -42.40\n",
      "  106  17747 527V 528N C   N    4120 4122  -48.50  -71.20   28.14    2.28  -63.20  160.15   19.25\n",
      "  106        528N 528N N   CA   4122 4123  159.43  142.80                  -41.10\n",
      "  107  17758 538K 539S C   N    4200 4202 -159.16 -136.60   22.64    0.95  -64.10 -160.29   21.30\n",
      "  107        539S 539S N   CA   4202 4203  149.36  151.20                  -35.00\n",
      "  108  17759 539S 540D C   N    4206 4208 -172.40  -96.50   75.90    3.10  -70.90  107.63    8.95\n",
      "  108        540D 540D N   CA   4208 4209  114.51  114.20                  150.30\n",
      "  109  17760 540D 541M C   N    4214 4216 -131.76 -125.60    7.96    0.42  -63.40 -171.24   23.91\n",
      "  109        541M 541M N   CA   4216 4217  135.45  140.50                  -40.50\n",
      "  110  17770 550L 551G C   N    4286 4288  -89.90  -80.20   54.69    2.64   82.20 -154.79    8.28\n",
      "  110        551G 551G N   CA   4288 4289  120.28  174.10                    8.50\n",
      "  111  17772 552N 553N C   N    4298 4300  -64.76  -63.20   10.64    1.42 -119.90 -179.97    9.25\n",
      "  111        553N 553N N   CA   4300 4301  -51.62  -41.10                  137.00\n",
      "  112  17784 564V 565I C   N    4397 4399 -128.52 -120.60   26.90    1.83  -63.40  161.87   24.78\n",
      "  112        565I 565I N   CA   4399 4400  104.59  130.30                  -43.60\n",
      "  113  17807 587A 588I C   N    4571 4573  -89.87  -97.30    9.64    0.74  -63.40  166.78   26.47\n",
      "  113        588I 588I N   CA   4573 4574  121.06  127.20                  -43.60\n",
      "  114  17808 588I 589T C   N    4579 4581  -63.35  -63.20   11.26    1.45 -124.80  174.33    9.23\n",
      "  114        589T 589T N   CA   4581 4582  -53.36  -42.10                  143.50\n",
      "  115  17809 589T 590L C   N    4586 4588  -82.45  -70.70   27.37    1.72  -63.50  153.66   22.47\n",
      "  115        590L 590L N   CA   4588 4589  166.32  141.60                  -41.20\n",
      "  116  17823 603R 604L C   N    4706 4708   65.67   60.20    6.09    0.74  -63.50  148.61   27.40\n",
      "  116        604L 604L N   CA   4708 4709   32.28   29.60                  -41.20\n",
      "  117  17825 605V 606R C   N    4721 4723 -120.71 -125.20   84.70    3.99  -63.00  112.96   12.78\n",
      "  117        606R 606R N   CA   4723 4724   56.02  140.60                  -41.10\n",
      "  118  17837 617L 618L C   N    4820 4822 -157.91 -108.50   65.72    2.94  -63.50  171.32   29.55\n",
      "  118        618L 618L N   CA   4822 4823  175.84  132.50                  -41.20\n",
      "  119  17840 620E 621P C   N    4842 4844  -73.44  -64.50   44.68    3.82  -58.70  134.73    9.98\n",
      "  119        621P 621P N   CA   4844 4845  103.42  147.20                  -30.50\n",
      "  120  17845 625E 626T C   N    4881 4883  -79.77 -124.80   45.18    2.30  -63.20  178.85   24.04\n",
      "  120        626T 626T N   CA   4883 4884  139.82  143.50                  -42.10\n",
      "  121  17847 627Q 628R C   N    4897 4899 -138.98 -125.20   95.40    4.88  -63.00  115.73   13.40\n",
      "  121        628R 628R N   CA   4899 4900   46.20  140.60                  -41.10\n",
      "  122  17865 645L 646E C   N    5036 5038   62.39  -63.60  153.04   19.37  -69.30  159.68   14.51\n",
      "  122        646E 646E N   CA   5038 5039 -127.19  -40.30                  142.50\n",
      "  123  17870 650E 651F C   N    5078 5080 -105.36  -63.20   90.59   10.35  -63.20   90.59   10.35\n",
      "  123        651F 651F N   CA   5080 5081   35.88  -44.30                  -44.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   42   96  380  493  489  509  640  739  760  746\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00183.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      653     5106     5106\n",
      "openf___224_> Open           EAO00183.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     22298699   21776.074    21.266\n",
      "read_mo_297_> Segments, residues, atoms:        1      653     5106\n",
      "read_mo_298_> Segment:        1  1:A  653:A     5106\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5106     5106        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    27134\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    30588\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    32747\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    34959\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    37150\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    38832\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    40673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    42690\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    43535\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45692\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    45976\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5106    46576\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.8816       13  13\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.3232       15  15\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.1891       76  76\n",
      "iupac_m_486_> OE1/2 will be swapped:      -94.3553       78  78\n",
      "iupac_m_486_> OE1/2 will be swapped:     -138.8013       81  81\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -142.4603       91  91\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1608      103  103\n",
      "iupac_m_486_> OE1/2 will be swapped:      112.2421      106  106\n",
      "iupac_m_487_> NH1/2 swapped:      175.9659      116  116\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.8966      123  123\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.6400      152  152\n",
      "iupac_m_487_> NH1/2 swapped:      172.7062      160  160\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8493      186  186\n",
      "iupac_m_485_> OD1/2 will be swapped:      138.8410      192  192\n",
      "iupac_m_485_> OD1/2 will be swapped:     -178.0854      206  206\n",
      "iupac_m_485_> OD1/2 will be swapped:      137.7817      209  209\n",
      "iupac_m_485_> OD1/2 will be swapped:      154.7425      213  213\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.0027      229  229\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2854      235  235\n",
      "iupac_m_485_> OD1/2 will be swapped:     -117.2392      242  242\n",
      "iupac_m_485_> OD1/2 will be swapped:     -180.0000      252  252\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.4508      253  253\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      119.6778      257  257\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -174.7325      269  269\n",
      "iupac_m_485_> OD1/2 will be swapped:      143.2274      289  289\n",
      "iupac_m_485_> OD1/2 will be swapped:     -150.2616      299  299\n",
      "iupac_m_485_> OD1/2 will be swapped:      131.9759      316  316\n",
      "iupac_m_487_> NH1/2 swapped:      175.9792      332  332\n",
      "iupac_m_485_> OD1/2 will be swapped:      158.3983      335  335\n",
      "iupac_m_486_> OE1/2 will be swapped:      -98.0219      351  351\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7120      357  357\n",
      "iupac_m_486_> OE1/2 will be swapped:      146.8622      364  364\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -102.3084      369  369\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.5615      381  381\n",
      "iupac_m_487_> NH1/2 swapped:      179.8481      385  385\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.2421      399  399\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      175.5572      410  410\n",
      "iupac_m_487_> NH1/2 swapped:      177.6262      415  415\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.8110      426  426\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      159.8511      436  436\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.4185      445  445\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.1443      457  457\n",
      "iupac_m_487_> NH1/2 swapped:      176.5688      464  464\n",
      "iupac_m_486_> OE1/2 will be swapped:     -100.8771      478  478\n",
      "iupac_m_487_> NH1/2 swapped:      179.5463      495  495\n",
      "iupac_m_486_> OE1/2 will be swapped:      153.1518      506  506\n",
      "iupac_m_486_> OE1/2 will be swapped:      102.0713      519  519\n",
      "iupac_m_485_> OD1/2 will be swapped:     -163.1976      522  522\n",
      "iupac_m_486_> OE1/2 will be swapped:      -95.9385      530  530\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.0699      543  543\n",
      "iupac_m_486_> OE1/2 will be swapped:     -162.4080      570  570\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.0085      572  572\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      172.8666      597  597\n",
      "iupac_m_487_> NH1/2 swapped:      177.6580      603  603\n",
      "iupac_m_487_> NH1/2 swapped:      174.1818      606  606\n",
      "iupac_m_486_> OE1/2 will be swapped:     -106.6025      611  611\n",
      "iupac_m_485_> OD1/2 will be swapped:      139.0025      615  615\n",
      "iupac_m_485_> OD1/2 will be swapped:     -136.4436      616  616\n",
      "iupac_m_487_> NH1/2 swapped:      178.5609      649  649\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   837936\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -48926.7031\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -48926.703125\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    21.746000\n",
      "Sequence length          :          653\n",
      "Compactness              :     0.049490\n",
      "Native energy (pair)     :    87.636776\n",
      "Native energy (surface)  :    22.358610\n",
      "Native energy (combined) :     4.668736\n",
      "Z score (pair)           :    -2.299401\n",
      "Z score (surface)        :    -3.103593\n",
      "Z score (combined)       :    -3.368468\n",
      "GA341 score              :     0.168664\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      653\n",
      "Number of all, selected real atoms                :     5106    5106\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    46576   46576\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10313\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4511.7144\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5180       0      0   0.005   0.005      33.608       1.000\n",
      " 2 Bond angle potential               :    6986       0      7   1.937   1.937      499.90       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3019       0    114  48.695  48.695      1124.5       1.000\n",
      " 4 Stereochemical improper torsion pot:    2035       0      0   1.238   1.238      60.566       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10313       0      0   0.003   0.003      10.485       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10350       0     15   0.776   0.776      684.79       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10301       2     38   0.830   0.830      838.72       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     652       0      6   4.575   4.575      158.36       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     570       0      3  85.684  85.684      193.40       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     437       0      3  77.749  77.749      199.84       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     206       0      3  84.236  84.236      159.81       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      82       0      0  91.815  91.815      57.222       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5331       0      2   0.625   0.625      96.544       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     651     124     99  32.329  74.099      317.55       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     776       0      0   1.925   1.925      76.381       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00183.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00183.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   80887.0547\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17223   3Q   4L C   N      24   26   52.84  -70.70  131.23   13.01 -108.50  169.94    8.78\n",
      "    1          4L   4L N   CA     26   27 -174.12  141.60                  132.50\n",
      "    2  17224   4L   5I C   N      32   34  -60.24  -63.40    4.84    0.74 -120.60 -172.45    8.54\n",
      "    2          5I   5I N   CA     34   35  -47.27  -43.60                  130.30\n",
      "    3  17227   7V   8V C   N      52   54 -133.74 -125.40   22.79    0.89  -62.40  168.90   25.76\n",
      "    3          8V   8V N   CA     54   55  164.51  143.30                  -42.40\n",
      "    4  17232  12H  13D C   N      94   96 -103.19  -63.30   77.36   12.10  -63.30   77.36   12.10\n",
      "    4         13D  13D N   CA     96   97 -106.27  -40.00                  -40.00\n",
      "    5  17239  19K  20M C   N     147  149  -78.84  -73.00   10.21    0.64  -63.40  168.84   26.43\n",
      "    5         20M  20M N   CA    149  150  151.37  143.00                  -40.50\n",
      "    6  17245  25N  26L C   N     196  198  -66.21  -70.70    7.45    0.46  -63.50  176.87   24.40\n",
      "    6         26L  26L N   CA    198  199  135.65  141.60                  -41.20\n",
      "    7  17252  32V  33G C   N     247  249  -60.85  -62.40    6.20    0.86   82.20  153.51   11.77\n",
      "    7         33G  33G N   CA    249  250  -47.21  -41.20                    8.50\n",
      "    8  17265  45A  46I C   N     332  334   49.29  -97.30  152.74    8.00 -120.60  176.01   10.55\n",
      "    8         46I  46I N   CA    334  335   84.30  127.20                  130.30\n",
      "    9  17274  54R  55G C   N     405  407   51.07   78.70   28.11    1.28   82.20  172.27    9.89\n",
      "    9         55G  55G N   CA    407  408 -160.93 -166.10                    8.50\n",
      "   10  17275  55G  56S C   N     409  411   49.70   56.90   29.49    1.35  -64.10  151.49   19.06\n",
      "   10         56S  56S N   CA    411  412   64.99   36.40                  -35.00\n",
      "   11  17286  66L  67Q C   N     488  490  -70.19  -63.80   66.00    9.38  -63.80   66.00    9.38\n",
      "   11         67Q  67Q N   CA    490  491   25.39  -40.30                  -40.30\n",
      "   12  17291  71L  72P C   N     529  531  -88.43  -64.50   24.81    2.73  -58.70  173.72   12.11\n",
      "   12         72P  72P N   CA    531  532  140.65  147.20                  -30.50\n",
      "   13  17293  73K  74T C   N     545  547 -144.94 -124.80   53.11    1.75  -63.20  149.57   24.35\n",
      "   13         74T  74T N   CA    547  548 -167.36  143.50                  -42.10\n",
      "   14  17294  74T  75S C   N     552  554  -80.33  -72.40   74.59    4.12  -64.10   99.76    8.30\n",
      "   14         75S  75S N   CA    554  555 -133.43  152.40                  -35.00\n",
      "   15  17311  91F  92D C   N     706  708 -175.33  -70.90  133.64    6.06  -63.30  141.43   23.95\n",
      "   15         92D  92D N   CA    708  709 -126.32  150.30                  -40.00\n",
      "   16  17312  92D  93F C   N     714  716   75.95   58.10   23.95    1.75 -124.20 -156.23   11.00\n",
      "   16         93F  93F N   CA    716  717   16.94   32.90                  143.30\n",
      "   17  17317  97N  98E C   N     758  760  -77.90  -69.30   30.38    2.01  -63.60  148.75   20.63\n",
      "   17         98E  98E N   CA    760  761  171.64  142.50                  -40.30\n",
      "   18  17342 122V 123Y C   N     954  956  -82.33  -98.40   80.05    8.44  -63.50   95.26   14.20\n",
      "   18        123Y 123Y N   CA    956  957   49.98  128.40                  -43.40\n",
      "   19  17349 129N 130L C   N    1012 1014  -88.78 -108.50   68.92    3.49  -63.50  110.59   14.12\n",
      "   19        130L 130L N   CA   1014 1015   66.46  132.50                  -41.20\n",
      "   20  17350 130L 131T C   N    1020 1022 -176.56 -124.80   65.11    2.27  -63.20  176.21   29.48\n",
      "   20        131T 131T N   CA   1022 1023 -177.00  143.50                  -42.10\n",
      "   21  17351 131T 132L C   N    1027 1029 -100.62 -108.50   20.11    0.99  -63.50  159.58   20.35\n",
      "   21        132L 132L N   CA   1029 1030  114.01  132.50                  -41.20\n",
      "   22  17361 141N 142A C   N    1100 1102  -36.68  -68.20   31.53    2.65  -62.50  176.43   27.76\n",
      "   22        142A 142A N   CA   1102 1103  144.57  145.30                  -40.90\n",
      "   23  17362 142A 143V C   N    1105 1107  -89.74  -73.50   44.27    3.90  -62.40  143.06   16.69\n",
      "   23        143V 143V N   CA   1107 1108   98.02  139.20                  -42.40\n",
      "   24  17371 151D 152R C   N    1173 1175 -153.69 -125.20   57.41    2.04   57.30 -146.83   24.81\n",
      "   24        152R 152R N   CA   1175 1176 -169.56  140.60                   38.00\n",
      "   25  17388 168I 169I C   N    1313 1315  -88.63 -120.60   58.23    2.40  -63.40  127.75   20.13\n",
      "   25        169I 169I N   CA   1315 1316   81.63  130.30                  -43.60\n",
      "   26  17390 170L 171A C   N    1329 1331  -52.64  -68.20   20.80    1.32  -62.50  172.68   28.79\n",
      "   26        171A 171A N   CA   1331 1332  131.50  145.30                  -40.90\n",
      "   27  17399 179L 180A C   N    1391 1393   71.74   55.40   16.88    1.59  -62.50  153.70   31.26\n",
      "   27        180A 180A N   CA   1393 1394   33.97   38.20                  -40.90\n",
      "   28  17411 191L 192D C   N    1479 1481 -139.61  -96.50   56.81    2.32  -63.30  139.86   14.92\n",
      "   28        192D 192D N   CA   1481 1482   77.21  114.20                  -40.00\n",
      "   29  17417 197R 198T C   N    1526 1528  -56.00  -63.20    9.74    1.09 -124.80 -178.62    9.72\n",
      "   29        198T 198T N   CA   1528 1529  -48.67  -42.10                  143.50\n",
      "   30  17418 198T 199V C   N    1533 1535 -140.23 -125.40   29.60    1.70  -62.40  178.01   19.51\n",
      "   30        199V 199V N   CA   1535 1536  117.69  143.30                  -42.40\n",
      "   31  17425 205L 206D C   N    1583 1585 -103.02  -70.90   32.17    2.49  -63.30  175.98   24.27\n",
      "   31        206D 206D N   CA   1585 1586  148.56  150.30                  -40.00\n",
      "   32  17445 225L 226R C   N    1728 1730  -96.12  -63.00   93.06   14.85  -63.00   93.06   14.85\n",
      "   32        226R 226R N   CA   1730 1731 -128.06  -41.10                  -41.10\n",
      "   33  17447 227H 228G C   N    1749 1751  -59.59  -62.40    7.51    1.27   82.20  148.10   11.16\n",
      "   33        228G 228G N   CA   1751 1752  -34.24  -41.20                    8.50\n",
      "   34  17448 228G 229F C   N    1753 1755 -151.74  -63.20  103.64   18.68  -63.20  103.64   18.68\n",
      "   34        229F 229F N   CA   1755 1756  -98.18  -44.30                  -44.30\n",
      "   35  17459 239D 240I C   N    1840 1842  -69.27  -97.30   35.09    2.36  -63.40  168.19   27.77\n",
      "   35        240I 240I N   CA   1842 1843  148.31  127.20                  -43.60\n",
      "   36  17464 244K 245G C   N    1879 1881  -66.28  -62.40    6.82    1.22   82.20  158.45   12.11\n",
      "   36        245G 245G N   CA   1881 1882  -46.82  -41.20                    8.50\n",
      "   37  17481 261P 262S C   N    2019 2021 -119.75 -136.60   45.23    2.54  -64.10  143.09   14.32\n",
      "   37        262S 262S N   CA   2021 2022 -166.83  151.20                  -35.00\n",
      "   38  17499 279L 280N C   N    2160 2162  -82.34 -119.90   73.97    2.42  -63.20  115.97   13.53\n",
      "   38        280N 280N N   CA   2162 2163   73.28  137.00                  -41.10\n",
      "   39  17500 280N 281Q C   N    2168 2170  -86.42 -121.10   54.52    1.85  -63.80  139.77   19.43\n",
      "   39        281Q 281Q N   CA   2170 2171   97.63  139.70                  -40.30\n",
      "   40  17511 291I 292P C   N    2260 2262  -77.97  -64.50   39.81    2.39  -58.70  146.12   13.66\n",
      "   40        292P 292P N   CA   2262 2263 -175.34  147.20                  -30.50\n",
      "   41  17512 292P 293E C   N    2267 2269 -110.04 -117.80   36.64    1.68  -63.60  148.73   17.72\n",
      "   41        293E 293E N   CA   2269 2270  100.99  136.80                  -40.30\n",
      "   42  17523 303G 304S C   N    2356 2358  -60.64  -72.40   19.86    0.91  -64.10  171.43   12.63\n",
      "   42        304S 304S N   CA   2358 2359  136.40  152.40                  -35.00\n",
      "   43  17524 304S 305T C   N    2362 2364 -101.22 -124.80   75.65    2.60  -63.20  119.91   13.25\n",
      "   43        305T 305T N   CA   2364 2365   71.61  143.50                  -42.10\n",
      "   44  17529 309M 310E C   N    2403 2405  -78.52  -63.60   91.83   11.49  -63.60   91.83   11.49\n",
      "   44        310E 310E N   CA   2405 2406   50.31  -40.30                  -40.30\n",
      "   45  17555 335D 336T C   N    2608 2610  -98.82 -124.80   60.14    1.93  -63.20  136.10   15.37\n",
      "   45        336T 336T N   CA   2610 2611   89.26  143.50                  -42.10\n",
      "   46  17556 336T 337L C   N    2615 2617  -88.84  -70.70   25.02    1.61  -63.50  161.96   24.02\n",
      "   46        337L 337L N   CA   2617 2618  158.84  141.60                  -41.20\n",
      "   47  17564 344G 345A C   N    2672 2674  -70.30  -62.50   10.47    1.64 -134.00 -169.93   11.69\n",
      "   47        345A 345A N   CA   2674 2675  -33.92  -40.90                  147.00\n",
      "   48  17565 345A 346T C   N    2677 2679   26.50  -78.10  123.02    5.37 -124.80  162.20    6.92\n",
      "   48        346T 346T N   CA   2679 2680   85.05  149.80                  143.50\n",
      "   49  17574 354G 355G C   N    2742 2744 -109.13  -80.20   34.38    2.53   82.20 -136.25   17.34\n",
      "   49        355G 355G N   CA   2744 2745  155.53  174.10                    8.50\n",
      "   50  17576 356A 357R C   N    2751 2753   18.80  -72.10  105.27    6.97  -63.00  153.51   26.19\n",
      "   50        357R 357R N   CA   2753 2754   88.80  141.90                  -41.10\n",
      "   51  17578 358L 359D C   N    2770 2772  -55.10  -70.90   24.57    1.94  -63.30  151.11   18.17\n",
      "   51        359D 359D N   CA   2772 2773  169.12  150.30                  -40.00\n",
      "   52  17581 361I 362F C   N    2798 2800  -67.60 -124.20   79.70    2.14  -63.20  131.57   17.97\n",
      "   52        362F 362F N   CA   2800 2801   87.19  143.30                  -44.30\n",
      "   53  17589 369Y 370V C   N    2865 2867    3.72  -73.50  107.63    7.62  -62.40  125.46   19.88\n",
      "   53        370V 370V N   CA   2867 2868   64.22  139.20                  -42.40\n",
      "   54  17591 371N 372G C   N    2880 2882  -57.32  -62.40    7.14    1.32   82.20  146.50   11.09\n",
      "   54        372G 372G N   CA   2882 2883  -36.19  -41.20                    8.50\n",
      "   55  17592 372G 373L C   N    2884 2886  -59.45  -70.70   31.20    2.00  -63.50  153.76   21.58\n",
      "   55        373L 373L N   CA   2886 2887  112.50  141.60                  -41.20\n",
      "   56  17593 373L 374T C   N    2892 2894 -104.68 -124.80   29.47    0.94  -63.20  169.23   19.25\n",
      "   56        374T 374T N   CA   2894 2895  121.96  143.50                  -42.10\n",
      "   57  17596 376K 377K C   N    2913 2915  -92.69  -62.90   83.24   12.59  -62.90   83.24   12.59\n",
      "   57        377K 377K N   CA   2915 2916 -118.52  -40.80                  -40.80\n",
      "   58  17599 379L 380T C   N    2939 2941   57.60   55.90   18.93    1.20 -124.80 -144.05   13.10\n",
      "   58        380T 380T N   CA   2941 2942   20.65   39.50                  143.50\n",
      "   59  17608 388A 389R C   N    3015 3017  -56.05 -125.20   69.17    2.73  -63.00 -179.88   23.87\n",
      "   59        389R 389R N   CA   3017 3018  138.91  140.60                  -41.10\n",
      "   60  17609 389R 390N C   N    3026 3028 -100.91 -119.90   20.99    0.66  -63.20  173.30   19.82\n",
      "   60        390N 390N N   CA   3028 3029  128.05  137.00                  -41.10\n",
      "   61  17628 408L 409S C   N    3167 3169 -116.81 -136.60   38.98    1.47  -64.10  161.46    9.95\n",
      "   61        409S 409S N   CA   3169 3170  117.62  151.20                  -35.00\n",
      "   62  17633 413I 414A C   N    3210 3212  -89.97  -68.20   48.31    4.57  -62.50  145.69   22.69\n",
      "   62        414A 414A N   CA   3212 3213  102.18  145.30                  -40.90\n",
      "   63  17638 418E 419P C   N    3252 3254  -76.60  -64.50   12.10    1.23  -58.70  178.67   13.36\n",
      "   63        419P 419P N   CA   3254 3255  147.27  147.20                  -30.50\n",
      "   64  17639 419P 420S C   N    3259 3261 -171.05 -136.60   35.42    1.62  -64.10 -152.38   13.82\n",
      "   64        420S 420S N   CA   3261 3262  142.96  151.20                  -35.00\n",
      "   65  17640 420S 421Q C   N    3265 3267   51.55   55.10    4.32    0.33  -63.80  142.15   25.81\n",
      "   65        421Q 421Q N   CA   3267 3268   42.77   40.30                  -40.30\n",
      "   66  17642 422K 423C C   N    3283 3285  -65.79  -63.00   11.26    1.58 -117.90  174.84    7.53\n",
      "   66        423C 423C N   CA   3285 3286  -52.01  -41.10                  141.10\n",
      "   67  17645 425Q 426F C   N    3305 3307   74.13   58.10   27.59    1.52  -63.20  147.84   26.24\n",
      "   67        426F 426F N   CA   3307 3308   10.45   32.90                  -44.30\n",
      "   68  17649 429E 430E C   N    3343 3345  -58.18  -69.30   19.28    1.10  -63.60  167.14   22.58\n",
      "   68        430E 430E N   CA   3345 3346  126.75  142.50                  -40.30\n",
      "   69  17650 430E 431L C   N    3352 3354  -64.76  -70.70    8.44    0.84  -63.50  171.20   23.84\n",
      "   69        431L 431L N   CA   3354 3355  147.60  141.60                  -41.20\n",
      "   70  17651 431L 432I C   N    3360 3362 -109.89 -120.60   39.95    2.69  -63.40  154.75   27.07\n",
      "   70        432I 432I N   CA   3362 3363  168.79  130.30                  -43.60\n",
      "   71  17652 432I 433K C   N    3368 3370  -69.08  -70.20   14.47    1.07  -62.90  164.49   21.70\n",
      "   71        433K 433K N   CA   3370 3371  154.83  140.40                  -40.80\n",
      "   72  17653 433K 434I C   N    3377 3379 -105.90  -97.30    8.61    0.36  -63.40  175.64   27.47\n",
      "   72        434I 434I N   CA   3379 3380  126.82  127.20                  -43.60\n",
      "   73  17658 438C 439A C   N    3414 3416  -75.16  -68.20   11.56    0.73  -62.50  165.05   27.69\n",
      "   73        439A 439A N   CA   3416 3417  154.54  145.30                  -40.90\n",
      "   74  17659 439A 440S C   N    3419 3421 -139.83 -136.60   25.05    1.20  -64.10  167.10   17.54\n",
      "   74        440S 440S N   CA   3421 3422  176.04  151.20                  -35.00\n",
      "   75  17660 440S 441K C   N    3425 3427  -89.46  -62.90   92.22   10.83  -62.90   92.22   10.83\n",
      "   75        441K 441K N   CA   3427 3428   47.51  -40.80                  -40.80\n",
      "   76  17663 443D 444N C   N    3450 3452  -65.96  -63.20    2.84    0.40   55.90  145.73   17.89\n",
      "   76        444N 444N N   CA   3452 3453  -40.42  -41.10                   39.50\n",
      "   77  17665 445F 446P C   N    3469 3471  -50.55  -58.70   34.13    2.23  -64.50  149.81   11.93\n",
      "   77        446P 446P N   CA   3471 3472  -63.65  -30.50                  147.20\n",
      "   78  17666 446P 447R C   N    3476 3478 -163.90 -125.20   43.46    2.22  -63.00 -169.23   21.59\n",
      "   78        447R 447R N   CA   3478 3479  120.81  140.60                  -41.10\n",
      "   79  17667 447R 448L C   N    3487 3489 -161.54 -108.50  102.25    4.85  -70.70  119.94    7.87\n",
      "   79        448L 448L N   CA   3489 3490 -140.08  132.50                  141.60\n",
      "   80  17668 448L 449K C   N    3495 3497 -114.26 -118.00   91.03    4.47  -62.90  102.89   16.42\n",
      "   80        449K 449K N   CA   3497 3498 -129.95  139.10                  -40.80\n",
      "   81  17676 456C 457R C   N    3554 3556 -152.21 -125.20   46.85    1.56  -63.00  166.01   28.37\n",
      "   81        457R 457R N   CA   3556 3557  178.89  140.60                  -41.10\n",
      "   82  17684 464R 465A C   N    3629 3631  -62.96  -62.50    1.03    0.15 -134.00 -172.96   11.53\n",
      "   82        465A 465A N   CA   3631 3632  -39.97  -40.90                  147.00\n",
      "   83  17685 465A 466P C   N    3634 3636  -68.41  -58.70   17.66    1.06  -64.50  162.99   12.37\n",
      "   83        466P 466P N   CA   3636 3637  -15.75  -30.50                  147.20\n",
      "   84  17690 470H 471V C   N    3672 3674  -85.02  -73.50   24.50    1.88  -62.40  158.40   21.41\n",
      "   84        471V 471V N   CA   3674 3675  160.83  139.20                  -42.40\n",
      "   85  17691 471V 472R C   N    3679 3681  -64.75  -72.10   43.37    3.17  -63.00  140.26   18.83\n",
      "   85        472R 472R N   CA   3681 3682   99.15  141.90                  -41.10\n",
      "   86  17693 473T 474I C   N    3697 3699 -113.14 -120.60   27.49    1.29  -63.40  155.61   24.05\n",
      "   86        474I 474I N   CA   3699 3700  103.85  130.30                  -43.60\n",
      "   87  17698 478E 479R C   N    3732 3734  -85.32 -125.20   56.06    3.08  -63.00  140.68   20.54\n",
      "   87        479R 479R N   CA   3734 3735 -180.00  140.60                  -41.10\n",
      "   88  17699 479R 480G C   N    3743 3745  127.35   78.70   64.76    1.62  -62.40 -121.07   44.31\n",
      "   88        480G 480G N   CA   3745 3746  151.15 -166.10                  -41.20\n",
      "   89  17700 480G 481F C   N    3747 3749  -71.98  -71.40    1.02    0.06  -63.20  174.37   24.80\n",
      "   89        481F 481F N   CA   3749 3750  141.55  140.70                  -44.30\n",
      "   90  17701 481F 482V C   N    3758 3760 -169.04 -125.40   78.22    2.54  -62.40  152.77   25.47\n",
      "   90        482V 482V N   CA   3760 3761 -151.79  143.30                  -42.40\n",
      "   91  17702 482V 483N C   N    3765 3767  -69.43  -63.20   13.47    1.45 -119.90  173.65    6.63\n",
      "   91        483N 483N N   CA   3767 3768  -29.16  -41.10                  137.00\n",
      "   92  17704 484V 485K C   N    3780 3782   71.72  -62.90  144.78   20.81  -62.90  144.78   20.81\n",
      "   92        485K 485K N   CA   3782 3783  -94.08  -40.80                  -40.80\n",
      "   93  17718 498L 499K C   N    3900 3902  155.95   56.60  102.69   10.59   56.60  102.69   10.59\n",
      "   93        499K 499K N   CA   3902 3903   12.61   38.60                   38.60\n",
      "   94  17721 501F 502G C   N    3928 3930  -67.24  -62.40    5.58    0.85   82.20  156.64   11.84\n",
      "   94        502G 502G N   CA   3930 3931  -38.42  -41.20                    8.50\n",
      "   95  17723 503T 504E C   N    3939 3941   58.39   54.60    6.80    0.81  -63.60  150.62   25.88\n",
      "   95        504E 504E N   CA   3941 3942   48.04   42.40                  -40.30\n",
      "   96  17738 518N 519E C   N    4052 4054  -66.47  -69.30    7.14    0.44  -63.60  176.27   23.35\n",
      "   96        519E 519E N   CA   4054 4055  135.94  142.50                  -40.30\n",
      "   97  17739 519E 520G C   N    4061 4063  114.47   82.20   62.78    1.73  -62.40  176.92   31.43\n",
      "   97        520G 520G N   CA   4063 4064  -45.35    8.50                  -41.20\n",
      "   98  17742 522D 523K C   N    4082 4084  -76.64  -70.20   41.17    3.13  -62.90  141.21   17.63\n",
      "   98        523K 523K N   CA   4084 4085   99.74  140.40                  -40.80\n",
      "   99  17743 523K 524G C   N    4091 4093  -66.63  -62.40    5.34    1.00   82.20  157.98   12.04\n",
      "   99        524G 524G N   CA   4093 4094  -44.47  -41.20                    8.50\n",
      "  100  17746 526K 527V C   N    4113 4115  -70.09  -73.50    6.80    0.52  -62.40  175.89   21.83\n",
      "  100        527V 527V N   CA   4115 4116  133.32  139.20                  -42.40\n",
      "  101  17747 527V 528N C   N    4120 4122  -48.73  -71.20   24.62    1.97  -63.20  166.66   20.07\n",
      "  101        528N 528N N   CA   4122 4123  152.87  142.80                  -41.10\n",
      "  102  17758 538K 539S C   N    4200 4202  -91.74  -72.40   34.61    1.58  -64.10  146.52   12.46\n",
      "  102        539S 539S N   CA   4202 4203 -178.90  152.40                  -35.00\n",
      "  103  17760 540D 541M C   N    4214 4216 -119.06 -125.60   30.05    1.29  -63.40  161.56   20.58\n",
      "  103        541M 541M N   CA   4216 4217  111.17  140.50                  -40.50\n",
      "  104  17770 550L 551G C   N    4286 4288  -96.96  -80.20   67.92    1.89   82.20 -139.47   17.22\n",
      "  104        551G 551G N   CA   4288 4289 -120.08  174.10                    8.50\n",
      "  105  17772 552N 553N C   N    4298 4300   63.06   55.90   15.76    0.75 -119.90 -150.76   10.99\n",
      "  105        553N 553N N   CA   4300 4301   25.46   39.50                  137.00\n",
      "  106  17773 553N 554M C   N    4306 4308 -128.24 -125.60   66.51    3.24  -73.00   88.35    6.92\n",
      "  106        554M 554M N   CA   4308 4309   74.04  140.50                  143.00\n",
      "  107  17774 554M 555S C   N    4314 4316  -83.50  -64.10   30.85    2.28 -136.60  170.69    7.38\n",
      "  107        555S 555S N   CA   4316 4317  -11.02  -35.00                  151.20\n",
      "  108  17784 564V 565I C   N    4397 4399 -132.97 -120.60   21.43    1.64  -63.40  171.18   26.20\n",
      "  108        565I 565I N   CA   4399 4400  112.81  130.30                  -43.60\n",
      "  109  17794 574G 575I C   N    4479 4481  -57.80  -63.40    5.60    1.00 -120.60 -175.21    8.32\n",
      "  109        575I 575I N   CA   4481 4482  -43.49  -43.60                  130.30\n",
      "  110  17795 575I 576V C   N    4487 4489   83.87  -62.40  146.32   23.03  -62.40  146.32   23.03\n",
      "  110        576V 576V N   CA   4489 4490  -46.35  -42.40                  -42.40\n",
      "  111  17807 587A 588I C   N    4571 4573  -95.37  -97.30    7.32    0.73  -63.40 -179.29   28.58\n",
      "  111        588I 588I N   CA   4573 4574  134.26  127.20                  -43.60\n",
      "  112  17808 588I 589T C   N    4579 4581  -54.11  -63.20   25.81    2.82 -124.80  166.04    9.09\n",
      "  112        589T 589T N   CA   4581 4582  -66.26  -42.10                  143.50\n",
      "  113  17809 589T 590L C   N    4586 4588 -131.80 -108.50   29.76    1.32  -63.50 -178.85   29.38\n",
      "  113        590L 590L N   CA   4588 4589  151.02  132.50                  -41.20\n",
      "  114  17823 603R 604L C   N    4706 4708   62.19   60.20    6.35    0.55  -63.50  147.31   27.15\n",
      "  114        604L 604L N   CA   4708 4709   35.63   29.60                  -41.20\n",
      "  115  17825 605V 606R C   N    4721 4723 -122.00 -125.20   77.73    3.68  -63.00  119.59   13.54\n",
      "  115        606R 606R N   CA   4723 4724   62.93  140.60                  -41.10\n",
      "  116  17834 614A 615D C   N    4796 4798  -66.86  -96.50   34.88    1.45  -63.30  135.85   16.56\n",
      "  116        615D 615D N   CA   4798 4799   95.81  114.20                  -40.00\n",
      "  117  17835 615D 616D C   N    4804 4806   61.69   54.50   38.08    2.26  -63.30  132.34   22.43\n",
      "  117        616D 616D N   CA   4806 4807    3.50   40.90                  -40.00\n",
      "  118  17837 617L 618L C   N    4820 4822 -134.21 -108.50   48.30    2.28  -63.50  161.70   26.83\n",
      "  118        618L 618L N   CA   4822 4823  173.38  132.50                  -41.20\n",
      "  119  17847 627Q 628R C   N    4897 4899 -125.43 -125.20   66.48    3.21  -63.00  128.07   21.51\n",
      "  119        628R 628R N   CA   4899 4900 -152.93  140.60                  -41.10\n",
      "  120  17851 631T 632K C   N    4929 4931  -82.38 -118.00   39.03    1.77  -62.90  165.31   22.58\n",
      "  120        632K 632K N   CA   4931 4932  155.05  139.10                  -40.80\n",
      "  121  17866 646E 647S C   N    5045 5047   60.66  -72.40  141.05    8.52 -136.60  169.01    7.90\n",
      "  121        647S 647S N   CA   5047 5048  105.60  152.40                  151.20\n",
      "  122  17867 647S 648V C   N    5051 5053  -72.06  -62.40   10.07    1.70 -125.40  179.56    9.97\n",
      "  122        648V 648V N   CA   5053 5054  -45.24  -42.40                  143.30\n",
      "  123  17868 648V 649R C   N    5058 5060  -63.53  -63.00    2.47    0.30   57.30  143.11   17.62\n",
      "  123        649R 649R N   CA   5060 5061  -38.69  -41.10                   38.00\n",
      "  124  17871 651F 652T C   N    5089 5091  -76.22  -78.10   17.53    0.81  -63.20  151.23   20.26\n",
      "  124        652T 652T N   CA   5091 5092  167.23  149.80                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   23   81  355  417  476  511  659  736  773  736\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00183.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      653     5106     5106\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00183.B99990001.pdb        4332.37695   -47757.87109        0.25108\n",
      "EAO00183.B99990002.pdb        4317.16406   -47605.51172        0.15297\n",
      "EAO00183.B99990003.pdb        4355.42480   -45738.85156        0.07507\n",
      "EAO00183.B99990004.pdb        4289.00049   -47037.35156        0.08167\n",
      "EAO00183.B99990005.pdb        4511.71436   -48926.70312        0.16866\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22821735   22286.852    21.765\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26382858   25764.510    25.161\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26385100   25766.699    25.163\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26386550   25768.115    25.164\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26389450   25770.947    25.167\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26395250   25776.611    25.172\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     39833649   38900.047    37.988\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39845249   38911.375    37.999\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39868449   38934.031    38.022\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39914849   38979.344    38.066\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40007649   39069.969    38.154\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     46726849   45631.688    44.562\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00184.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22824055   22289.117    21.767\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22825505   22290.531    21.768\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22828405   22293.363    21.771\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22834205   22299.027    21.776\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22835729   22300.516    21.778\n",
      "\n",
      "Read the alignment from file       : ali/EAO00184.ali\n",
      "\n",
      "Total number of alignment positions:   382\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00184     382      1    EAO00184\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     22837590   22302.334    21.780\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23162765   22619.887    22.090\n",
      "profile_iteration_> processing sequence:       1    382      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23163843   22620.941    22.091\n",
      "profile_iteration_> processing sequence:    6720    382      1     3.7656250     0.0005604     0.0000022  1785\n",
      "profile_iteration_> processing sequence:   13439    382      1     7.5000000     0.0005581     0.0000022  1792\n",
      "profile_iteration_> processing sequence:   20158    382      1    11.3281250     0.0005620     0.0000022  1779\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23164218   22621.307    22.091\n",
      "profile_iteration_> processing sequence:   26877    382      1    15.0156250     0.0005587     0.0000022  1790\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23164454   22621.537    22.091\n",
      "profile_iteration_> processing sequence:   33596    382      1    18.7500000     0.0005581     0.0000022  1792\n",
      "profile_iteration_> processing sequence:   40315    382      1    22.4218750     0.0005562     0.0000022  1798\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23167404   22624.418    22.094\n",
      "profile_iteration_> processing sequence:   47034    382      1    26.1718750     0.0005564     0.0000022  1797\n",
      "profile_iteration_> processing sequence:   53753    382      1    30.0000000     0.0005581     0.0000022  1792\n",
      "profile_iteration_> processing sequence:   60472    382      1    33.8281250     0.0005594     0.0000022  1788\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23170914   22627.846    22.098\n",
      "profile_iteration_> processing sequence:   67191    382      1    37.7500000     0.0005618     0.0000022  1780\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    49.65237\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    278.7492\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0306,      121,        0.0077,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00015   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00015   0.00000\n",
      "Z:         1  -2.85000   0.00045   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00015   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00045   0.00016\n",
      "Z:         1  -2.25000   0.00045   0.00055\n",
      "Z:         1  -2.15000   0.00193   0.00163\n",
      "Z:         1  -2.05000   0.00387   0.00416\n",
      "Z:         1  -1.95000   0.00878   0.00933\n",
      "Z:         1  -1.85000   0.01250   0.01872\n",
      "Z:         1  -1.75000   0.03125   0.03400\n",
      "Z:         1  -1.65000   0.06757   0.05658\n",
      "Z:         1  -1.55000   0.08185   0.08721\n",
      "Z:         1  -1.45000   0.17383   0.12565\n",
      "Z:         1  -1.35000   0.16401   0.17059\n",
      "Z:         1  -1.25000   0.21580   0.21981\n",
      "Z:         1  -1.15000   0.23797   0.27052\n",
      "Z:         1  -1.05000   0.30242   0.31973\n",
      "Z:         1  -0.95000   0.33903   0.36469\n",
      "Z:         1  -0.85000   0.41850   0.40316\n",
      "Z:         1  -0.75000   0.44246   0.43360\n",
      "Z:         1  -0.65000   0.43949   0.45519\n",
      "Z:         1  -0.55000   0.46330   0.46779\n",
      "Z:         1  -0.45000   0.47119   0.47182\n",
      "Z:         1  -0.35000   0.44842   0.46811\n",
      "Z:         1  -0.25000   0.45437   0.45775\n",
      "Z:         1  -0.15000   0.42862   0.44196\n",
      "Z:         1  -0.05000   0.43443   0.42194\n",
      "Z:         1   0.05000   0.43651   0.39889\n",
      "Z:         1   0.15000   0.36790   0.37383\n",
      "Z:         1   0.25000   0.35317   0.34769\n",
      "Z:         1   0.35000   0.32057   0.32121\n",
      "Z:         1   0.45000   0.29661   0.29500\n",
      "Z:         1   0.55000   0.26581   0.26953\n",
      "Z:         1   0.65000   0.23872   0.24513\n",
      "Z:         1   0.75000   0.21952   0.22205\n",
      "Z:         1   0.85000   0.19065   0.20043\n",
      "Z:         1   0.95000   0.18722   0.18036\n",
      "Z:         1   1.05000   0.16684   0.16185\n",
      "Z:         1   1.15000   0.13915   0.14489\n",
      "Z:         1   1.25000   0.12487   0.12944\n",
      "Z:         1   1.35000   0.11579   0.11542\n",
      "Z:         1   1.45000   0.10135   0.10275\n",
      "Z:         1   1.55000   0.08915   0.09133\n",
      "Z:         1   1.65000   0.07873   0.08109\n",
      "Z:         1   1.75000   0.07025   0.07191\n",
      "Z:         1   1.85000   0.07084   0.06371\n",
      "Z:         1   1.95000   0.05879   0.05639\n",
      "Z:         1   2.05000   0.05998   0.04988\n",
      "Z:         1   2.15000   0.04509   0.04409\n",
      "Z:         1   2.25000   0.03974   0.03895\n",
      "Z:         1   2.35000   0.03215   0.03439\n",
      "Z:         1   2.45000   0.03200   0.03035\n",
      "Z:         1   2.55000   0.02604   0.02678\n",
      "Z:         1   2.65000   0.02530   0.02361\n",
      "Z:         1   2.75000   0.02054   0.02082\n",
      "Z:         1   2.85000   0.01682   0.01835\n",
      "Z:         1   2.95000   0.01563   0.01617\n",
      "Z:         1   3.05000   0.01459   0.01424\n",
      "Z:         1   3.15000   0.01250   0.01255\n",
      "Z:         1   3.25000   0.00997   0.01105\n",
      "Z:         1   3.35000   0.01146   0.00973\n",
      "Z:         1   3.45000   0.00923   0.00857\n",
      "Z:         1   3.55000   0.00729   0.00754\n",
      "Z:         1   3.65000   0.00878   0.00664\n",
      "Z:         1   3.75000   0.00655   0.00584\n",
      "Z:         1   3.85000   0.00640   0.00514\n",
      "Z:         1   3.95000   0.00536   0.00453\n",
      "Z:         1   4.05000   0.00476   0.00398\n",
      "Z:         1   4.15000   0.00417   0.00350\n",
      "Z:         1   4.25000   0.00283   0.00308\n",
      "Z:         1   4.35000   0.00313   0.00271\n",
      "Z:         1   4.45000   0.00372   0.00239\n",
      "Z:         1   4.55000   0.00268   0.00210\n",
      "Z:         1   4.65000   0.00298   0.00185\n",
      "Z:         1   4.75000   0.00387   0.00163\n",
      "Z:         1   4.85000   0.00208   0.00143\n",
      "Z:         1   4.95000   0.00268   0.00126\n",
      "Z:         1   5.05000   0.00089   0.00111\n",
      "Z:         1   5.15000   0.00089   0.00097\n",
      "Z:         1   5.25000   0.00164   0.00086\n",
      "Z:         1   5.35000   0.00089   0.00075\n",
      "Z:         1   5.45000   0.00060   0.00066\n",
      "Z:         1   5.55000   0.00193   0.00058\n",
      "Z:         1   5.65000   0.00030   0.00051\n",
      "Z:         1   5.75000   0.00104   0.00045\n",
      "Z:         1   5.85000   0.00119   0.00040\n",
      "Z:         1   5.95000   0.00119   0.00035\n",
      "Z:         1   6.05000   0.00104   0.00031\n",
      "Z:         1   6.15000   0.00104   0.00027\n",
      "Z:         1   6.25000   0.00074   0.00024\n",
      "Z:         1   6.35000   0.00030   0.00021\n",
      "Z:         1   6.45000   0.00060   0.00018\n",
      "Z:         1   6.55000   0.00030   0.00016\n",
      "Z:         1   6.65000   0.00060   0.00014\n",
      "Z:         1   6.75000   0.00074   0.00013\n",
      "Z:         1   6.85000   0.00015   0.00011\n",
      "Z:         1   6.95000   0.00030   0.00010\n",
      "Z:         1   7.05000   0.00030   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00060   0.00006\n",
      "Z:         1   7.45000   0.00074   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00045   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00045   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     23493514   22942.885    22.405\n",
      "> 6f4jA                      1    8792    7400     175     382   30.00    0.98E-07       2   158     1   160     1   160\n",
      "> 5yxmA                      1   11036    6850     194     382   32.90    0.14E-05       3   144    14   158    36   190\n",
      "> 6dbfA                      1   13538    5000     208     382   33.33    0.73E-02       4   105    20   128    62   166\n",
      "> 6dbgA                      1   13539    5150     286     382   34.31    0.50E-02       5   102    20   125   106   207\n",
      "> 6mkyA                      1   13811    7950     259     382   31.01    0.12E-07       6   136    10   146    98   255\n",
      "> 4wp6A                      1   15769    5200     151     382   35.48    0.21E-02       7    61    90   150    81   142\n",
      "> 6hkwA                      1   17842    7950     306     382   31.01    0.14E-07       8   136    10   146   143   300\n",
      "> 2rr6A                      1   23534    5100     168     382   33.80    0.37E-02       9   137     3   147    13   154\n",
      "> 6hluA                      1   30107    5550     908     382   33.33    0.25E-02      10   105    20   128   149   253\n",
      "> 6hluB                      1   30108    5550     896     382   33.33    0.25E-02      11   105    20   128   149   253\n",
      "> 5il7A                      1   34086    5550     440     382   33.33    0.12E-02      12   105    20   128   149   253\n",
      "> 4z0cA                      1   36939    5300     709     382   30.43    0.63E-02      13   128    20   153    82   219\n",
      "> 6obnC                      1   37928    7950     285     382   31.01    0.13E-07      14   136    10   146   122   279\n",
      "> 1a9nA                      1   40295    8100     162     382   32.28    0.36E-08      15   156     3   160     2   159\n",
      "> 1a9nC                      1   40296    8100     174     382   32.28    0.39E-08      16   156     3   160     2   159\n",
      "> 2jqdA                      1   42829    7100     164     382   35.92    0.37E-06      17   138     3   147     6   147\n",
      "> 4aw4A                      1   49807    5150     301     382   34.31    0.53E-02      18   102    20   125   128   229\n",
      "> 4cilA                      1   51615    5150     279     382   34.31    0.49E-02      19   102    20   125    99   200\n",
      "> 1m9sA                      1   53995    5150     523     382   34.31    0.92E-02      20   102    20   125   106   207\n",
      "> 4r58A                      1   54411    5650     220     382   31.19    0.39E-03      21   106    20   132    40   148\n",
      "> 4r5cA                      1   54412    5250     304     382   29.73    0.34E-02      22   110    20   132   179   289\n",
      "> 1o6vA                      1   58470    5250     464     382   30.93    0.52E-02      23    97    18   115   216   312\n",
      "> 1xeuA                      1   64219    5350     263     382   31.68    0.18E-02      24   101    20   125    84   184\n",
      "> 2wquB                      1   65109    5150     280     382   34.31    0.49E-02      25   102    20   125   106   207\n",
      "> 2wquC                      1   65110    5150     264     382   34.31    0.47E-02      26   102    20   125   106   207\n",
      "> 4xosA                      1   67083    6700     152     382   33.80    0.21E-05      27   138     3   147     9   150\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    37.734375\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000562\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1785.092529\n",
      "profile_iteration__> Statistics              [sec]:     0.062500\n",
      "profile_iteration__> Add to profile          [sec]:     0.062500\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000564\n",
      "profile_iteration__> Overall performance [seq/sec]:  1772.583618\n",
      "openf___224_> Open           buildprofile/EAO00184build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23276376   22730.836    22.198\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23277900   22732.324    22.200\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23278528   22732.938    22.200\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23279100   22733.496    22.201\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23279516   22733.902    22.201\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23279920   22734.297    22.201\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23280460   22734.824    22.202\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23280700   22735.059    22.202\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23281240   22735.586    22.203\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23281784   22736.117    22.203\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23282200   22736.523    22.204\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23282616   22736.930    22.204\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23283032   22737.336    22.204\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23283540   22737.832    22.205\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23284080   22738.359    22.205\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23284700   22738.965    22.206\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23285320   22739.570    22.207\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23285868   22740.105    22.207\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23286272   22740.500    22.208\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23286676   22740.895    22.208\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23287080   22741.289    22.208\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23287500   22741.699    22.209\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23287936   22742.125    22.209\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23288320   22742.500    22.209\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23288720   22742.891    22.210\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23289124   22743.285    22.210\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23289528   22743.680    22.211\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23290076   22744.215    22.211\n",
      "openf___224_> Open           buildprofile/EAO00184build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           4wp6.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22564679   22035.820    21.519\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22564755   22035.895    21.519\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22564926   22036.061    21.520\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22565776   22036.891    21.520\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22567034   22038.119    21.522\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22568938   22039.979    21.523\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22568978   22040.018    21.523\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22569068   22040.105    21.524\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22571924   22042.895    21.526\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22571984   22042.953    21.526\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22572119   22043.086    21.526\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22576403   22047.270    21.531\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22576491   22047.355    21.531\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22576689   22047.547    21.531\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22583115   22053.824    21.537\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22583247   22053.953    21.537\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22583544   22054.242    21.537\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22593166   22063.639    21.547\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22593366   22063.834    21.547\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22593816   22064.273    21.547\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22608266   22078.385    21.561\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22608566   22078.678    21.561\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22609241   22079.336    21.562\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     22723033   22190.461    21.670\n",
      "read_mo_297_> Segments, residues, atoms:        1      151     1209\n",
      "read_mo_298_> Segment:        1  196:A  359:A     1209\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22727383   22194.711    21.675\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     22815764   22281.020    21.759\n",
      "openf___224_> Open           ali/EAO00184.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22875501   22339.355    21.816\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22922009   22384.773    21.860\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23015025   22475.609    21.949\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23016549   22477.098    21.950\n",
      "\n",
      "Read the alignment from file       : ali/EAO00184.ali\n",
      "\n",
      "Total number of alignment positions:   382\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      4wp6A     151      1    4wp6.pdb MOL_ID  1; MOLECULE  MRNA EXPORT PROTEIN; CHAIN  A; FRAGMENT  UNP RESIDUES 191-3\n",
      "  2   EAO00184     382      1    EAO00184\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          382\n",
      "  Score                    :   85939.8750\n",
      "openf___224_> Open           ali/EAO00184-4wp6A.ali\n",
      "openf___224_> Open           ali/EAO00184-4wp6A.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00184-4wp6A.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22564434   22035.580    21.519\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22565884   22036.996    21.521\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22568784   22039.828    21.523\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22574584   22045.492    21.529\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22703792   22171.672    21.652\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22704392   22172.258    21.653\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22705916   22173.746    21.654\n",
      "\n",
      "Read the alignment from file       : ali/EAO00184-4wp6A.ali\n",
      "\n",
      "Total number of alignment positions:   382\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      4wp6A     151      1    4wp6.pdb MOL_ID  1; MOLECULE  MRNA EXPORT PROTEIN; CHAIN  A; FRAGMENT  UNP RESIDUES 191-3\n",
      "  2   EAO00184     382      1    EAO00184\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           4wp6.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22752941   22219.668    21.699\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22752941   22219.668    21.699\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22753112   22219.836    21.699\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22753962   22220.666    21.700\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22755220   22221.895    21.701\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22757124   22223.754    21.703\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22757124   22223.754    21.703\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22757214   22223.842    21.703\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22760070   22226.631    21.706\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22760070   22226.631    21.706\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22760205   22226.762    21.706\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22764489   22230.945    21.710\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22764489   22230.945    21.710\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22764687   22231.141    21.710\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22771113   22237.414    21.716\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22771113   22237.414    21.716\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22771410   22237.705    21.717\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22781032   22247.102    21.726\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22781032   22247.102    21.726\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22781482   22247.541    21.726\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22795932   22261.652    21.740\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22795932   22261.652    21.740\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     22796607   22262.312    21.741\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     22795941   22261.660    21.740\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     94     1  223   237      D     L   11.382\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22935193   22397.648    21.873\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22936793   22399.211    21.874\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22940773   22403.098    21.878\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22942773   22405.051    21.880\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22945773   22407.980    21.883\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22950273   22412.375    21.887\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22955845   22417.816    21.892\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22958645   22420.551    21.895\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22962845   22424.652    21.899\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22969145   22430.805    21.905\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22971917   22433.512    21.908\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     22973317   22434.879    21.909\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23033134   22493.295    21.966\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23337130   22790.166    22.256\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23381602   22833.596    22.298\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      382   3083   3121      0         0      1219\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23509826   22958.814    22.421\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      382   3083   3121   4209      4859      1219\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23642402   23088.283    22.547\n",
      "patch_______> segment topology patched using RTF:  382 ;  GLU  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      382   3084   3122   4211      4861      1220\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                     1      382   3084   3122   4211      4861      1220\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1007     2077\n",
      "openf___224_> Open           EAO00184.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      382     3084     3084\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23749851   23193.215    22.650\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23758043   23201.215    22.657\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23774427   23217.215    22.673\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23807195   23249.215    22.704\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23839963   23281.215    22.736\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23905499   23345.215    22.798\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23938267   23377.215    22.829\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23971035   23409.215    22.861\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24036571   23473.215    22.923\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24167643   23601.215    23.048\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24233179   23665.215    23.111\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24298715   23729.215    23.173\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24429787   23857.215    23.298\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24691931   24113.215    23.548\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24823003   24241.215    23.673\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     3122    4211    4565    1207\n",
      "              Total number of restraints before, now         :        0    13105\n",
      "make_re_422_> Number of previous, current restraints         :        0    13105\n",
      "make_re_423_> Number of previous, current selected restraints:        0    13105\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     25044932   24457.941    23.885\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  4wp6A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    13105    13485\n",
      "make_re_423_> Number of previous, current selected restraints:    13105    13485\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     25044948   24457.957    23.885\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  4wp6A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      394\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25176020   24585.957    24.010\n",
      "make_re_422_> Number of previous, current restraints         :    13485    13866\n",
      "make_re_423_> Number of previous, current selected restraints:    13485    13472\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     25176020   24585.957    24.010\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  4wp6A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    13866    14204\n",
      "make_re_423_> Number of previous, current selected restraints:    13472    13810\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     25176020   24585.957    24.010\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  4wp6A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    14204    14476\n",
      "make_re_423_> Number of previous, current selected restraints:    13810    14082\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     25176020   24585.957    24.010\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  4wp6A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25438164   24841.957    24.260\n",
      "make_re_422_> Number of previous, current restraints         :    14476    14620\n",
      "make_re_423_> Number of previous, current selected restraints:    14082    14226\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     25438164   24841.957    24.260\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  4wp6A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    14620    14675\n",
      "make_re_423_> Number of previous, current selected restraints:    14226    14281\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     26326320   25709.297    25.107\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     26588464   25965.297    25.357\n",
      "make_re_422_> Number of previous, current restraints         :    14675    17376\n",
      "make_re_423_> Number of previous, current selected restraints:    14281    16982\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27112752   26477.297    25.857\n",
      "make_re_422_> Number of previous, current restraints         :    17376    20252\n",
      "make_re_423_> Number of previous, current selected restraints:    16982    19858\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    20252    21440\n",
      "make_re_423_> Number of previous, current selected restraints:    19858    21046\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    21440    21917\n",
      "make_re_423_> Number of previous, current selected restraints:    21046    21523\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    21917    21917\n",
      "make_re_423_> Number of previous, current selected restraints:    21523    21523\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    21917    21917\n",
      "make_re_423_> Number of previous, current selected restraints:    21523    21523\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    21917    21917\n",
      "make_re_423_> Number of previous, current selected restraints:    21523    21523\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    21917    21917\n",
      "make_re_423_> Number of previous, current selected restraints:    21523    21523\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    21917    21917\n",
      "make_re_423_> Number of previous, current selected restraints:    21523    21523\n",
      "rmdupl__427_>     2374 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    21917    19149\n",
      "openf___224_> Open           EAO00184.rsr\n",
      "openf___224_> Open           EAO00184.rsr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27180205   26543.168    25.921\n",
      "rdcsr2__307_> Number of restraints read    :    19149\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    19149    19149\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00184.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     27180237   26543.199    25.921\n",
      "read_mo_297_> Segments, residues, atoms:        1      382     3084\n",
      "read_mo_298_> Segment:        1  1:A  382:A     3084\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3084     3084        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13392\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13854\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14132\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14626\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15344\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    16575\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    17868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    18897\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27546825   26901.195    26.271\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27620841   26973.477    26.341\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19140\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "iupac_m_487_> NH1/2 swapped:      179.6412        3  3\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.4278       14  14\n",
      "iupac_m_486_> OE1/2 will be swapped:     -148.0480       34  34\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.3274       37  37\n",
      "iupac_m_487_> NH1/2 swapped:      179.2545       44  44\n",
      "iupac_m_486_> OE1/2 will be swapped:      122.9114       45  45\n",
      "iupac_m_486_> OE1/2 will be swapped:      136.5184       47  47\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.3023       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.6665       55  55\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.4637       58  58\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.1822       60  60\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.7224       70  70\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -113.8482       85  85\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0823       92  92\n",
      "iupac_m_487_> NH1/2 swapped:     -175.6131      108  108\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1386      110  110\n",
      "iupac_m_486_> OE1/2 will be swapped:      135.4951      129  129\n",
      "iupac_m_487_> NH1/2 swapped:      179.8082      143  143\n",
      "iupac_m_485_> OD1/2 will be swapped:      169.9729      146  146\n",
      "iupac_m_487_> NH1/2 swapped:      176.8465      159  159\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9115      164  164\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1637      175  175\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0723      177  177\n",
      "iupac_m_487_> NH1/2 swapped:      179.6551      181  181\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.1938      192  192\n",
      "iupac_m_487_> NH1/2 swapped:      178.7733      197  197\n",
      "iupac_m_486_> OE1/2 will be swapped:      128.4451      200  200\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.4743      203  203\n",
      "iupac_m_486_> OE1/2 will be swapped:      122.0341      222  222\n",
      "iupac_m_486_> OE1/2 will be swapped:      159.2030      223  223\n",
      "iupac_m_486_> OE1/2 will be swapped:     -165.4382      224  224\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8475      227  227\n",
      "iupac_m_486_> OE1/2 will be swapped:     -147.0497      234  234\n",
      "iupac_m_487_> NH1/2 swapped:      173.2729      235  235\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.3784      236  236\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.0252      257  257\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.9416      263  263\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9585      265  265\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9590      269  269\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.2617      271  271\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.9802      279  279\n",
      "iupac_m_486_> OE1/2 will be swapped:      130.5706      284  284\n",
      "iupac_m_485_> OD1/2 will be swapped:      122.0677      292  292\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.4775      296  296\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -163.3336      303  303\n",
      "iupac_m_487_> NH1/2 swapped:     -178.2406      312  312\n",
      "iupac_m_487_> NH1/2 swapped:      178.3036      326  326\n",
      "iupac_m_487_> NH1/2 swapped:     -176.1278      334  334\n",
      "iupac_m_487_> NH1/2 swapped:     -177.1277      337  337\n",
      "iupac_m_486_> OE1/2 will be swapped:     -162.7284      360  360\n",
      "iupac_m_487_> NH1/2 swapped:     -177.4245      364  364\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9802      367  367\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.0518      379  379\n",
      "iupac_m_486_> OE1/2 will be swapped:     -150.3456      382  382\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLU:OXT  GLU\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28061645   27403.949    26.762\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28357709   27693.074    27.044\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28949837   28271.324    27.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     30134093   29427.824    28.738\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     32502605   31740.824    30.997\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     37239629   36366.824    35.514\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   637473\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -23494.6426\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -23494.642578\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.383999\n",
      "Sequence length          :          382\n",
      "Compactness              :     0.109388\n",
      "Native energy (pair)     :   731.484288\n",
      "Native energy (surface)  :    54.542489\n",
      "Native energy (combined) :    23.110847\n",
      "Z score (pair)           :    -1.389325\n",
      "Z score (surface)        :    -0.385142\n",
      "Z score (combined)       :    -1.244635\n",
      "GA341 score              :     0.157604\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7763\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3464.7781\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3122       0      0   0.005   0.005      20.710       1.000\n",
      " 2 Bond angle potential               :    4211       2     15   2.198   2.198      384.05       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1797       0    109  52.243  52.243      789.45       1.000\n",
      " 4 Stereochemical improper torsion pot:    1207       0      0   1.375   1.375      44.942       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7763       0      0   0.004   0.004      17.374       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2701       0     10   1.594   1.594      423.44       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2876       4     17   1.927   1.927      587.41       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     381       2     12   5.564   5.564      139.08       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     338       0      7  81.227  81.227      135.81       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     272       0      1  72.519  72.519      104.43       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     144       0      0  97.900  97.900      112.76       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      55       0      0 100.437 100.437      44.895       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1188       0      0   0.718   0.718      15.474       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     380     200    100  45.376 126.108      575.49       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     477       0      1   3.154   3.154      69.452       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00184.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00184.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   71039.0469\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6009 263Y 264G C   N    2122 2124  143.85  120.00   23.84    5.42  120.00   23.84    5.42\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16646 285T 217L N   O    2301 1741    7.83    4.82    3.01    5.21    4.82    3.01    5.21\n",
      "    2  16647 285T 218R N   O    2301 1752    6.95    2.73    4.22    4.57    2.73    4.22    4.57\n",
      "    3  17101 330P 215Q N   O    2663 1725   14.95   10.63    4.32    4.66   10.63    4.32    4.66\n",
      "    4  17102 330P 217L N   O    2663 1741   13.94   10.04    3.90    5.15   10.04    3.90    5.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10983 266L 266L CA  C    2140 2145 -156.87 -180.00   23.13    4.62 -180.00   23.13    4.62\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10338   1M   2G C   N       7    9   66.29   78.70   53.23    1.03   82.20  123.87    6.80\n",
      "    1          2G   2G N   CA      9   10 -114.34 -166.10                    8.50\n",
      "    2  10339   2G   3R C   N      11   13  -70.46  -72.10   10.36    0.83  -63.00  166.94   23.06\n",
      "    2          3R   3R N   CA     13   14  152.13  141.90                  -41.10\n",
      "    3  10341   4I   5T C   N      30   32  -93.02  -78.10   33.11    1.08  -63.20  141.71   20.16\n",
      "    3          5T   5T N   CA     32   33  179.36  149.80                  -42.10\n",
      "    4  10347  10R  11R C   N      81   83 -134.00 -125.20   25.27    1.00  -63.00  170.14   27.83\n",
      "    4         11R  11R N   CA     83   84  164.28  140.60                  -41.10\n",
      "    5  10348  11R  12R C   N      92   94  -79.73  -72.10   18.62    1.27  -63.00  160.89   22.88\n",
      "    5         12R  12R N   CA     94   95  158.88  141.90                  -41.10\n",
      "    6  10349  12R  13A C   N     103  105 -152.25 -134.00   49.21    2.24  -62.50  155.02   30.14\n",
      "    6         13A  13A N   CA    105  106 -167.30  147.00                  -40.90\n",
      "    7  10350  13A  14E C   N     108  110  -76.95  -69.30   76.21    6.08  -63.60  107.81   13.68\n",
      "    7         14E  14E N   CA    110  111   66.68  142.50                  -40.30\n",
      "    8  10351  14E  15H C   N     117  119 -175.67 -125.60   72.47    1.80  -63.20  169.29   28.09\n",
      "    8         15H  15H N   CA    119  120 -168.82  138.80                  -42.30\n",
      "    9  10352  15H  16N C   N     127  129 -107.15 -119.90   27.35    0.92  -63.20  160.05   17.95\n",
      "    9         16N  16N N   CA    129  130  112.80  137.00                  -41.10\n",
      "   10  10353  16N  17E C   N     135  137 -152.52 -117.80   58.45    2.06  -63.60  162.39   26.50\n",
      "   10         17E  17E N   CA    137  138 -176.18  136.80                  -40.30\n",
      "   11  10354  17E  18G C   N     144  146   74.98   82.20   24.57    0.84  -62.40  155.66   29.44\n",
      "   11         18G  18G N   CA    146  147   31.99    8.50                  -41.20\n",
      "   12  10361  24K  25E C   N     193  195 -112.62 -117.80    6.59    0.34  -63.60 -174.58   27.50\n",
      "   12         25E  25E N   CA    195  196  140.88  136.80                  -40.30\n",
      "   13  10363  26I  27A C   N     210  212  -57.76  -68.20   10.45    0.90  -62.50  173.58   28.22\n",
      "   13         27A  27A N   CA    212  213  145.59  145.30                  -40.90\n",
      "   14  10364  27A  28L C   N     215  217 -126.57 -108.50   20.88    0.92  -63.50 -173.20   29.84\n",
      "   14         28L  28L N   CA    217  218  142.97  132.50                  -41.20\n",
      "   15  10365  28L  29H C   N     223  225  -62.51  -67.60    7.94    0.47  -63.20  176.21   22.07\n",
      "   15         29H  29H N   CA    225  226  133.90  140.00                  -42.30\n",
      "   16  10366  29H  30Q C   N     233  235 -137.67 -121.10   34.07    1.27  -63.80  167.41   28.46\n",
      "   16         30Q  30Q N   CA    235  236  169.47  139.70                  -40.30\n",
      "   17  10367  30Q  31Q C   N     242  244 -100.02 -121.10   31.16    1.61  -63.80  161.18   25.64\n",
      "   17         31Q  31Q N   CA    244  245  162.64  139.70                  -40.30\n",
      "   18  10368  31Q  32D C   N     251  253   47.36  -70.90  128.97   10.80  -63.30  161.95   18.07\n",
      "   18         32D  32D N   CA    253  254 -158.24  150.30                  -40.00\n",
      "   19  10370  33I  34E C   N     267  269  -63.83  -69.30    8.71    0.80  -63.60  170.42   22.74\n",
      "   19         34E  34E N   CA    269  270  149.28  142.50                  -40.30\n",
      "   20  10371  34E  35K C   N     276  278  -95.52 -118.00   30.48    1.52  -62.90  162.82   23.05\n",
      "   20         35K  35K N   CA    278  279  159.68  139.10                  -40.80\n",
      "   21  10372  35K  36I C   N     285  287  -65.51  -63.40    6.21    0.96 -120.60  176.87    8.11\n",
      "   21         36I  36I N   CA    287  288  -37.76  -43.60                  130.30\n",
      "   22  10376  39V  40G C   N     316  318 -136.08 -167.20   49.96    1.27   82.20 -169.69   14.72\n",
      "   22         40G  40G N   CA    318  319  135.52  174.60                    8.50\n",
      "   23  10380  43C  44R C   N     339  341  -61.65  -72.10   12.26    0.81  -63.00  176.59   23.95\n",
      "   23         44R  44R N   CA    341  342  135.49  141.90                  -41.10\n",
      "   24  10382  45E  46L C   N     359  361  -73.55  -70.70    6.73    0.62  -63.50  177.00   24.02\n",
      "   24         46L  46L N   CA    361  362  135.51  141.60                  -41.20\n",
      "   25  10383  46L  47E C   N     367  369  -96.91  -69.30   42.66    2.40  -63.60  148.46   21.68\n",
      "   25         47E  47E N   CA    369  370  175.02  142.50                  -40.30\n",
      "   26  10386  49L  50Y C   N     392  394 -101.88  -98.40    4.22    0.27  -63.50  173.71   25.71\n",
      "   26         50Y  50Y N   CA    394  395  126.01  128.40                  -43.40\n",
      "   27  10387  50Y  51L C   N     404  406  -94.43 -108.50   14.77    0.66  -63.50  172.03   22.30\n",
      "   27         51L  51L N   CA    406  407  128.02  132.50                  -41.20\n",
      "   28  10389  52C  53N C   N     418  420  -63.91  -71.20    7.64    0.60  -63.20  173.83   21.77\n",
      "   28         53N  53N N   CA    420  421  145.07  142.80                  -41.10\n",
      "   29  10390  53N  54N C   N     426  428  -88.03  -71.20   34.15    2.63  -63.20  156.17   18.26\n",
      "   29         54N  54N N   CA    428  429  113.09  142.80                  -41.10\n",
      "   30  10391  54N  55Y C   N     434  436 -125.89 -124.30   12.34    0.59  -63.50 -179.89   31.71\n",
      "   30         55Y  55Y N   CA    436  437  147.64  135.40                  -43.40\n",
      "   31  10393  56I  57S C   N     454  456  -89.14  -72.40   29.22    1.34  -64.10  150.75   12.58\n",
      "   31         57S  57S N   CA    456  457  176.35  152.40                  -35.00\n",
      "   32  10394  57S  58R C   N     460  462  -79.00  -72.10   21.53    1.79  -63.00  163.38   21.04\n",
      "   32         58R  58R N   CA    462  463  121.50  141.90                  -41.10\n",
      "   33  10396  59I  60E C   N     479  481  -92.12  -69.30   26.30    1.55  -63.60  166.60   23.82\n",
      "   33         60E  60E N   CA    481  482  155.56  142.50                  -40.30\n",
      "   34  10397  60E  61G C   N     488  490 -145.95 -167.20   21.51    0.34   82.20 -150.53   15.68\n",
      "   34         61G  61G N   CA    490  491  171.28  174.60                    8.50\n",
      "   35  10398  61G  62L C   N     492  494 -117.35 -108.50   30.96    1.57  -63.50  165.63   26.33\n",
      "   35         62L  62L N   CA    494  495  162.17  132.50                  -41.20\n",
      "   36  10400  63Q  64H C   N     509  511  -75.43  -63.20   73.69   10.00  -63.20   73.69   10.00\n",
      "   36         64H  64H N   CA    511  512 -114.97  -42.30                  -42.30\n",
      "   37  10401  64H  65L C   N     519  521  -64.35  -70.70   12.01    1.14  -63.50  167.00   23.24\n",
      "   37         65L  65L N   CA    521  522  151.80  141.60                  -41.20\n",
      "   38  10402  65L  66K C   N     527  529   54.17   56.60   15.97    0.97  -62.90  150.88   26.05\n",
      "   38         66K  66K N   CA    529  530   54.38   38.60                  -40.80\n",
      "   39  10403  66K  67R C   N     536  538  -44.83  -72.10   32.79    2.15  -63.00  165.80   23.64\n",
      "   39         67R  67R N   CA    538  539  123.70  141.90                  -41.10\n",
      "   40  10404  67R  68L C   N     547  549  -89.56  -70.70   35.75    2.21  -63.50  149.12   22.28\n",
      "   40         68L  68L N   CA    549  550  171.97  141.60                  -41.20\n",
      "   41  10405  68L  69K C   N     555  557  -72.32  -70.20   32.19    2.38   56.60  146.55   12.56\n",
      "   41         69K  69K N   CA    557  558  108.28  140.40                   38.60\n",
      "   42  10412  75V  76N C   N     612  614 -124.02 -119.90   13.55    0.51  -63.20  179.61   25.92\n",
      "   42         76N  76N N   CA    614  615  149.91  137.00                  -41.10\n",
      "   43  10413  76N  77N C   N     620  622   57.06   55.90    4.43    0.23  -63.20  142.43   23.41\n",
      "   43         77N  77N N   CA    622  623   35.23   39.50                  -41.10\n",
      "   44  10415  78I  79A C   N     636  638  -61.00  -68.20   24.64    1.74  -62.50  162.64   26.73\n",
      "   44         79A  79A N   CA    638  639  121.74  145.30                  -40.90\n",
      "   45  10418  81I  82E C   N     655  657  -67.81  -69.30   17.68    1.40  -63.60  159.64   21.52\n",
      "   45         82E  82E N   CA    657  658  160.11  142.50                  -40.30\n",
      "   46  10421  84L  85E C   N     676  678   48.33  -69.30  121.43    8.11  -63.60 -170.68   31.27\n",
      "   46         85E  85E N   CA    678  679  112.38  142.50                  -40.30\n",
      "   47  10423  86G  87C C   N     689  691  -62.98  -63.00   18.43    2.25 -117.90  168.57    7.30\n",
      "   47         87C  87C N   CA    691  692  -59.53  -41.10                  141.10\n",
      "   48  10425  88E  89S C   N     704  706   72.06 -136.60  152.22    5.63  -64.10 -151.84   15.84\n",
      "   48         89S  89S N   CA    706  707  167.54  151.20                  -35.00\n",
      "   49  10426  89S  90L C   N     710  712  -93.70  -70.70   70.95    4.61  -63.50  114.15   17.71\n",
      "   49         90L  90L N   CA    712  713 -151.28  141.60                  -41.20\n",
      "   50  10427  90L  91E C   N     718  720  -77.00  -69.30   55.18    4.47  -63.60  128.86   16.48\n",
      "   50         91E  91E N   CA    720  721   87.86  142.50                  -40.30\n",
      "   51  10428  91E  92R C   N     727  729   58.67   57.30    2.38    0.13  -63.00  144.07   26.26\n",
      "   51         92R  92R N   CA    729  730   36.05   38.00                  -41.10\n",
      "   52  10429  92R  93L C   N     738  740 -121.81 -108.50   22.28    1.27  -63.50  166.38   20.62\n",
      "   52         93L  93L N   CA    740  741  114.63  132.50                  -41.20\n",
      "   53  10430  93L  94D C   N     746  748 -111.47  -70.90   49.33    2.40   54.50 -144.50   14.65\n",
      "   53         94D  94D N   CA    748  749  178.36  150.30                   40.90\n",
      "   54  10432  95L  96T C   N     762  764 -159.80 -124.80   47.32    1.57  -63.20  172.20   28.16\n",
      "   54         96T  96T N   CA    764  765  175.35  143.50                  -42.10\n",
      "   55  10433  96T  97L C   N     769  771 -117.79 -108.50   87.03    4.60  -63.50  113.58   19.12\n",
      "   55         97L  97L N   CA    771  772 -140.96  132.50                  -41.20\n",
      "   56  10435  98N  99F C   N     785  787 -106.30 -124.20   18.60    0.87  -63.20  172.81   26.88\n",
      "   56         99F  99F N   CA    787  788  148.35  143.30                  -44.30\n",
      "   57  10437 100V 101A C   N     803  805 -170.38 -134.00   42.28    0.99  -68.20  104.79    8.06\n",
      "   57        101A 101A N   CA    805  806  168.54  147.00                  145.30\n",
      "   58  10439 102E 103M C   N     817  819 -173.42 -125.60   49.90    2.32  -63.40 -160.22   24.70\n",
      "   58        103M 103M N   CA    819  820  126.26  140.50                  -40.50\n",
      "   59  10440 103M 104T C   N     825  827 -151.10 -124.80   26.43    1.46  -63.20 -162.37   31.06\n",
      "   59        104T 104T N   CA    827  828  140.90  143.50                  -42.10\n",
      "   60  10442 105C 106V C   N     838  840  -61.94  -62.40    3.04    0.36 -125.40 -177.33   10.26\n",
      "   60        106V 106V N   CA    840  841  -45.40  -42.40                  143.30\n",
      "   61  10444 107Q 108R C   N     854  856 -124.32 -125.20    3.65    0.15  -63.00 -171.59   22.05\n",
      "   61        108R 108R N   CA    856  857  137.06  140.60                  -41.10\n",
      "   62  10445 108R 109L C   N     865  867 -124.20 -108.50   27.53    1.57  -63.50  162.83   20.10\n",
      "   62        109L 109L N   CA    867  868  109.89  132.50                  -41.20\n",
      "   63  10446 109L 110R C   N     873  875 -158.33 -125.20   34.14    1.17  -63.00 -165.05   32.76\n",
      "   63        110R 110R N   CA    875  876  148.84  140.60                  -41.10\n",
      "   64  10447 110R 111A C   N     884  886  -67.47  -68.20    0.98    0.10  -62.50  173.22   28.63\n",
      "   64        111A 111A N   CA    886  887  145.96  145.30                  -40.90\n",
      "   65  10448 111A 112N C   N     889  891 -150.90 -119.90  107.74    4.12  -63.20  117.85   19.17\n",
      "   65        112N 112N N   CA    891  892 -119.81  137.00                  -41.10\n",
      "   66  10459 122G 123N C   N     977  979 -109.21 -119.90   19.12    1.03  -63.20  172.31   24.16\n",
      "   66        123N 123N N   CA    979  980  152.85  137.00                  -41.10\n",
      "   67  10460 123N 124P C   N     985  987  -62.48  -58.70   13.63    0.86  -64.50  164.61   12.18\n",
      "   67        124P 124P N   CA    987  988  -17.40  -30.50                  147.20\n",
      "   68  10461 124P 125C C   N     992  994  -59.83  -63.00    3.31    0.49 -117.90 -173.85    8.04\n",
      "   68        125C 125C N   CA    994  995  -42.03  -41.10                  141.10\n",
      "   69  10462 125C 126T C   N     998 1000  -75.90  -78.10   11.47    0.56  -63.20  157.35   21.02\n",
      "   69        126T 126T N   CA   1000 1001  161.06  149.80                  -42.10\n",
      "   70  10465 128I 129E C   N    1022 1024   78.52  -63.60  142.37   22.96  -63.60  142.37   22.96\n",
      "   70        129E 129E N   CA   1024 1025  -31.91  -40.30                  -40.30\n",
      "   71  10469 132R 133A C   N    1058 1060 -150.95 -134.00   23.30    0.68  -62.50  179.44   34.13\n",
      "   71        133A 133A N   CA   1060 1061  162.98  147.00                  -40.90\n",
      "   72  10470 133A 134Y C   N    1063 1065  -74.61  -98.40   24.65    1.21  -63.50  165.71   25.73\n",
      "   72        134Y 134Y N   CA   1065 1066  121.94  128.40                  -43.40\n",
      "   73  10472 135V 136I C   N    1082 1084  -65.40  -63.40    3.52    0.65 -120.60 -174.79    8.57\n",
      "   73        136I 136I N   CA   1084 1085  -46.49  -43.60                  130.30\n",
      "   74  10473 136I 137H C   N    1090 1092 -153.10 -125.60   28.50    0.84  -63.20 -166.44   30.03\n",
      "   74        137H 137H N   CA   1092 1093  146.29  138.80                  -42.30\n",
      "   75  10474 137H 138A C   N    1100 1102 -151.89 -134.00   20.21    0.46  -62.50 -174.35   35.20\n",
      "   75        138A 138A N   CA   1102 1103  156.39  147.00                  -40.90\n",
      "   76  10475 138A 139L C   N    1105 1107 -100.78 -108.50    8.76    0.39  -63.50  173.61   22.26\n",
      "   76        139L 139L N   CA   1107 1108  128.36  132.50                  -41.20\n",
      "   77  10477 140P 141Q C   N    1120 1122 -105.17 -121.10   43.66    2.26  -63.80  145.37   23.58\n",
      "   77        141Q 141Q N   CA   1122 1123 -179.66  139.70                  -40.30\n",
      "   78  10479 142L 143R C   N    1137 1139 -129.92 -125.20   19.58    0.83  -63.00  172.78   27.93\n",
      "   78        143R 143R N   CA   1139 1140  159.60  140.60                  -41.10\n",
      "   79  10480 143R 144D C   N    1148 1150 -102.58  -96.50   11.13    0.46  -63.30  150.11   16.55\n",
      "   79        144D 144D N   CA   1150 1151  104.88  114.20                  -40.00\n",
      "   80  10483 146D 147G C   N    1172 1174  114.20   82.20   53.51    1.49  -62.40  176.74   31.84\n",
      "   80        147G 147G N   CA   1174 1175  -34.38    8.50                  -41.20\n",
      "   81  10487 150V 151V C   N    1201 1203  -56.39  -62.40   21.59    3.10 -125.40  178.82    6.85\n",
      "   81        151V 151V N   CA   1203 1204  -21.67  -42.40                  143.30\n",
      "   82  10495 158A 159R C   N    1267 1269 -143.02 -125.20   73.81    3.13  -63.00  133.35   23.23\n",
      "   82        159R 159R N   CA   1269 1270 -147.77  140.60                  -41.10\n",
      "   83  10496 159R 160Q C   N    1278 1280 -169.76 -121.10   69.41    2.27  -63.80  168.11   29.88\n",
      "   83        160Q 160Q N   CA   1280 1281 -170.81  139.70                  -40.30\n",
      "   84  10498 161C 162N C   N    1293 1295 -147.28 -119.90   30.30    0.95  -63.20 -171.30   28.32\n",
      "   84        162N 162N N   CA   1295 1296  149.97  137.00                  -41.10\n",
      "   85  10499 162N 163S C   N    1301 1303 -174.79 -136.60   81.41    3.15  -64.10  150.45   18.78\n",
      "   85        163S 163S N   CA   1303 1304 -136.90  151.20                  -35.00\n",
      "   86  10502 165V 166A C   N    1323 1325 -177.16 -134.00   56.02    1.51  -62.50  178.18   35.15\n",
      "   86        166A 166A N   CA   1325 1326 -177.29  147.00                  -40.90\n",
      "   87  10503 166A 167A C   N    1328 1330 -176.36 -134.00   45.96    1.05  -62.50 -168.24   37.41\n",
      "   87        167A 167A N   CA   1330 1331  164.81  147.00                  -40.90\n",
      "   88  10505 168V 169V C   N    1340 1342   47.81  -73.50  141.73   11.54 -125.40 -173.49    9.32\n",
      "   88        169V 169V N   CA   1342 1343 -147.51  139.20                  143.30\n",
      "   89  10509 172A 173A C   N    1369 1371 -175.17 -134.00   52.05    1.36  -62.50  179.91   35.36\n",
      "   89        173A 173A N   CA   1371 1372  178.84  147.00                  -40.90\n",
      "   90  10511 174L 175R C   N    1382 1384  -91.33  -72.10   45.82    3.12  -63.00  138.34   20.65\n",
      "   90        175R 175R N   CA   1384 1385 -176.51  141.90                  -41.10\n",
      "   91  10513 176V 177R C   N    1400 1402 -138.95 -125.20   81.48    3.61   57.30 -118.81   14.97\n",
      "   91        177R 177R N   CA   1402 1403 -139.09  140.60                   38.00\n",
      "   92  10515 178E 179A C   N    1420 1422 -138.18 -134.00    5.69    0.33   55.40 -163.26   23.23\n",
      "   92        179A 179A N   CA   1422 1423  143.13  147.00                   38.20\n",
      "   93  10517 180E 181R C   N    1434 1436 -137.24 -125.20   52.12    2.74  -63.00  150.55   17.04\n",
      "   93        181R 181R N   CA   1436 1437   89.88  140.60                  -41.10\n",
      "   94  10519 182I 183K C   N    1453 1455 -112.57 -118.00   34.66    1.74  -62.90  154.09   22.97\n",
      "   94        183K 183K N   CA   1455 1456  173.33  139.10                  -40.80\n",
      "   95  10520 183K 184A C   N    1462 1464  -70.35  -68.20    3.52    0.34  -62.50  176.77   29.36\n",
      "   95        184A 184A N   CA   1464 1465  142.51  145.30                  -40.90\n",
      "   96  10521 184A 185E C   N    1467 1469  -77.28  -69.30    8.27    0.55  -63.60  175.56   24.17\n",
      "   96        185E 185E N   CA   1469 1470  144.67  142.50                  -40.30\n",
      "   97  10522 185E 186M C   N    1476 1478  177.96 -125.60   90.69    2.93  -63.40  160.45   32.26\n",
      "   97        186M 186M N   CA   1478 1479 -148.52  140.50                  -40.50\n",
      "   98  10523 186M 187I C   N    1484 1486  -60.24  -63.40    4.40    0.68 -120.60 -173.02    8.51\n",
      "   98        187I 187I N   CA   1486 1487  -46.67  -43.60                  130.30\n",
      "   99  10524 187I 188A C   N    1492 1494  -74.02  -68.20    7.12    0.46  -62.50  170.10   28.45\n",
      "   99        188A 188A N   CA   1494 1495  149.40  145.30                  -40.90\n",
      "  100  10525 188A 189Q C   N    1497 1499 -100.28 -121.10   23.32    0.72  -63.80  173.38   23.73\n",
      "  100        189Q 189Q N   CA   1499 1500  129.20  139.70                  -40.30\n",
      "  101  10526 189Q 190G C   N    1506 1508 -132.36 -167.20   52.48    1.25   82.20 -167.01   14.95\n",
      "  101        190G 190G N   CA   1508 1509  135.36  174.60                    8.50\n",
      "  102  10528 191I 192D C   N    1518 1520   53.48  -70.90  136.27   11.42  -96.50  175.83    7.17\n",
      "  102        192D 192D N   CA   1520 1521 -154.01  150.30                  114.20\n",
      "  103  10530 193V 194F C   N    1533 1535   95.64 -124.20  143.09    4.75  -63.20 -145.88   25.07\n",
      "  103        194F 194F N   CA   1535 1536  172.12  143.30                  -44.30\n",
      "  104  10532 195P 196P C   N    1551 1553  -56.95  -58.70   19.83    1.78  -64.50  158.12   11.41\n",
      "  104        196P 196P N   CA   1553 1554  -10.74  -30.50                  147.20\n",
      "  105  10533 196P 197R C   N    1558 1560  -52.80  -72.10   19.49    1.55  -63.00  174.61   22.92\n",
      "  105        197R 197R N   CA   1560 1561  144.59  141.90                  -41.10\n",
      "  106  10534 197R 198Y C   N    1569 1571  -76.46  -98.40   21.94    0.85  -63.50  171.86   26.61\n",
      "  106        198Y 198Y N   CA   1571 1572  127.97  128.40                  -43.40\n",
      "  107  10535 198Y 199N C   N    1581 1583 -179.56 -119.90   71.69    2.11  -63.20 -176.29   29.20\n",
      "  107        199N 199N N   CA   1583 1584  176.75  137.00                  -41.10\n",
      "  108  10536 199N 200E C   N    1589 1591 -151.83 -117.80   63.08    2.33  -63.60  156.94   25.72\n",
      "  108        200E 200E N   CA   1591 1592 -170.09  136.80                  -40.30\n",
      "  109  10537 200E 201K C   N    1598 1600   60.79   56.60    6.27    0.75  -62.90  149.55   26.03\n",
      "  109        201K 201K N   CA   1600 1601   43.27   38.60                  -40.80\n",
      "  110  10538 201K 202G C   N    1607 1609  102.05   82.20   59.54    1.93  -62.40  164.58   29.13\n",
      "  110        202G 202G N   CA   1609 1610  -47.63    8.50                  -41.20\n",
      "  111  10540 203E 204R C   N    1620 1622 -131.47 -125.20   63.30    3.19  -63.00  137.04   15.51\n",
      "  111        204R 204R N   CA   1622 1623   77.61  140.60                  -41.10\n",
      "  112  10543 206Y 207G C   N    1650 1652   45.53   78.70   70.58    1.05   82.20  118.14    7.48\n",
      "  112        207G 207G N   CA   1652 1653 -103.81 -166.10                    8.50\n",
      "  113  10544 207G 208H C   N    1654 1656 -164.49  -63.20  102.62   15.39  -63.20  102.62   15.39\n",
      "  113        208H 208H N   CA   1656 1657  -25.89  -42.30                  -42.30\n",
      "  114  10559 222E 223E C   N    1787 1789   38.11   54.60   32.78    1.67  -69.30  129.18    7.42\n",
      "  114        223E 223E N   CA   1789 1790   70.73   42.40                  142.50\n",
      "  115  10560 223E 224E C   N    1796 1798  -75.84  -69.30   30.94    2.13  -63.60  147.47   20.34\n",
      "  115        224E 224E N   CA   1798 1799  172.74  142.50                  -40.30\n",
      "  116  10561 224E 225A C   N    1805 1807 -173.42 -134.00   58.49    1.86  -62.50  170.05   33.61\n",
      "  116        225A 225A N   CA   1807 1808 -169.80  147.00                  -40.90\n",
      "  117  10563 226E 227R C   N    1819 1821 -130.90 -125.20   63.29    3.17  -63.00  136.71   15.47\n",
      "  117        227R 227R N   CA   1821 1822   77.56  140.60                  -41.10\n",
      "  118  10564 227R 228E C   N    1830 1832 -149.84  -63.60   94.50   12.86   54.60  161.69   16.19\n",
      "  118        228E 228E N   CA   1832 1833   -1.67  -40.30                   42.40\n",
      "  119  10566 229K 230K C   N    1848 1850 -137.90 -118.00   93.95    4.77  -62.90  115.69   13.72\n",
      "  119        230K 230K N   CA   1850 1851   47.28  139.10                  -40.80\n",
      "  120  10568 231L 232Q C   N    1865 1867 -106.09 -121.10   33.27    1.74  -63.80  156.14   25.22\n",
      "  120        232Q 232Q N   CA   1867 1868  169.40  139.70                  -40.30\n",
      "  121  10569 232Q 233K C   N    1874 1876 -123.98 -118.00    8.67    0.31  -62.90 -175.76   27.57\n",
      "  121        233K 233K N   CA   1876 1877  145.38  139.10                  -40.80\n",
      "  122  10570 233K 234E C   N    1883 1885 -127.70 -117.80   15.84    0.54  -63.60 -177.82   27.91\n",
      "  122        234E 234E N   CA   1885 1886  149.17  136.80                  -40.30\n",
      "  123  10571 234E 235R C   N    1892 1894   91.62   57.30  137.38   11.65  -63.00 -146.04   25.50\n",
      "  123        235R 235R N   CA   1894 1895  171.02   38.00                  -41.10\n",
      "  124  10574 237T 238G C   N    1919 1921   87.26   78.70   14.39    0.25   82.20  173.91    8.70\n",
      "  124        238G 238G N   CA   1921 1922 -177.66 -166.10                    8.50\n",
      "  125  10580 243V 244S C   N    1959 1961 -126.65 -136.60   10.00    0.37  -64.10 -174.29   17.85\n",
      "  125        244S 244S N   CA   1961 1962  150.14  151.20                  -35.00\n",
      "  126  10581 244S 245A C   N    1965 1967 -123.52 -134.00   21.05    0.85  -62.50 -179.72   27.29\n",
      "  126        245A 245A N   CA   1967 1968  128.74  147.00                  -40.90\n",
      "  127  10582 245A 246E C   N    1970 1972  144.85 -117.80  104.12    3.11  -63.60 -149.59   35.74\n",
      "  127        246E 246E N   CA   1972 1973  173.74  136.80                  -40.30\n",
      "  128  10583 246E 247V C   N    1979 1981  -63.42  -62.40    7.95    1.07 -125.40  177.58    9.98\n",
      "  128        247V 247V N   CA   1981 1982  -50.28  -42.40                  143.30\n",
      "  129  10584 247V 248N C   N    1986 1988 -104.13 -119.90   49.03    1.83  -63.20  137.89   15.36\n",
      "  129        248N 248N N   CA   1988 1989   90.57  137.00                  -41.10\n",
      "  130  10589 252Q 253R C   N    2030 2032 -134.68 -125.20   16.12    0.53  -63.00 -179.85   29.25\n",
      "  130        253R 253R N   CA   2032 2033  153.63  140.60                  -41.10\n",
      "  131  10591 254L 255T C   N    2049 2051  -74.77  -63.20   83.82   11.52  -63.20   83.82   11.52\n",
      "  131        255T 255T N   CA   2051 2052 -125.12  -42.10                  -42.10\n",
      "  132  10592 255T 256A C   N    2056 2058  -56.02  -68.20   12.20    1.01  -62.50  174.60   28.30\n",
      "  132        256A 256A N   CA   2058 2059  144.63  145.30                  -40.90\n",
      "  133  10593 256A 257E C   N    2061 2063  -86.14  -69.30   32.20    2.91  -63.60  156.99   19.78\n",
      "  133        257E 257E N   CA   2063 2064  115.06  142.50                  -40.30\n",
      "  134  10594 257E 258E C   N    2070 2072 -159.79 -117.80   42.38    1.42  -63.60 -158.40   32.22\n",
      "  134        258E 258E N   CA   2072 2073  142.53  136.80                  -40.30\n",
      "  135  10596 259E 260I C   N    2088 2090  -51.79  -97.30   46.48    2.03  -63.40 -179.91   29.12\n",
      "  135        260I 260I N   CA   2090 2091  136.68  127.20                  -43.60\n",
      "  136  10597 260I 261A C   N    2096 2098  -48.36  -68.20   20.32    1.86  -62.50  170.00   27.20\n",
      "  136        261A 261A N   CA   2098 2099  149.69  145.30                  -40.90\n",
      "  137  10598 261A 262K C   N    2101 2103   64.68  -70.20  141.12   11.19  -62.90 -172.58   22.51\n",
      "  137        262K 262K N   CA   2103 2104 -178.09  140.40                  -40.80\n",
      "  138  10599 262K 263Y C   N    2110 2112 -126.96 -124.30    8.10    0.33  -63.50 -175.21   32.51\n",
      "  138        263Y 263Y N   CA   2112 2113  143.05  135.40                  -43.40\n",
      "  139  10604 267L 268L C   N    2153 2155 -112.77 -108.50   12.27    0.70  -63.50  169.52   21.27\n",
      "  139        268L 268L N   CA   2155 2156  121.00  132.50                  -41.20\n",
      "  140  10605 268L 269R C   N    2161 2163 -115.28 -125.20    9.96    0.42  -63.00 -174.99   28.61\n",
      "  140        269R 269R N   CA   2163 2164  141.43  140.60                  -41.10\n",
      "  141  10606 269R 270N C   N    2172 2174 -121.06 -119.90   13.98    0.66  -63.20  174.07   19.20\n",
      "  141        270N 270N N   CA   2174 2175  123.07  137.00                  -41.10\n",
      "  142  10607 270N 271E C   N    2180 2182 -126.97 -117.80   33.48    1.81  -63.60  158.16   18.47\n",
      "  142        271E 271E N   CA   2182 2183  104.60  136.80                  -40.30\n",
      "  143  10608 271E 272P C   N    2189 2191  -61.84  -58.70    6.80    0.40  -64.50  171.68   12.67\n",
      "  143        272P 272P N   CA   2191 2192  -24.46  -30.50                  147.20\n",
      "  144  10609 272P 273K C   N    2196 2198  -86.52  -70.20   27.01    2.25  -62.90  161.42   19.79\n",
      "  144        273K 273K N   CA   2198 2199  118.88  140.40                  -40.80\n",
      "  145  10610 273K 274F C   N    2205 2207 -103.40 -124.20   22.33    0.65  -63.20 -176.08   22.99\n",
      "  145        274F 274F N   CA   2207 2208  135.18  143.30                  -44.30\n",
      "  146  10613 276M 277R C   N    2231 2233 -151.17 -125.20   25.98    1.07  -63.00 -160.23   32.99\n",
      "  146        277R 277R N   CA   2233 2234  139.64  140.60                  -41.10\n",
      "  147  10614 277R 278L C   N    2242 2244 -140.38 -108.50   35.28    1.87  -63.50  176.22   21.61\n",
      "  147        278L 278L N   CA   2244 2245  117.37  132.50                  -41.20\n",
      "  148  10616 279D 280Q C   N    2258 2260 -103.72 -121.10   18.55    0.84  -63.80  178.05   28.32\n",
      "  148        280Q 280Q N   CA   2260 2261  146.18  139.70                  -40.30\n",
      "  149  10617 280Q 281E C   N    2267 2269 -129.59 -117.80   31.17    1.29  -63.60  167.59   26.06\n",
      "  149        281E 281E N   CA   2269 2270  165.65  136.80                  -40.30\n",
      "  150  10618 281E 282S C   N    2276 2278 -148.99 -136.60   12.50    0.45  -64.10 -168.08   19.99\n",
      "  150        282S 282S N   CA   2278 2279  152.87  151.20                  -35.00\n",
      "  151  10619 282S 283D C   N    2282 2284 -126.19  -96.50   41.66    1.74  -63.30 -172.55   27.20\n",
      "  151        283D 283D N   CA   2284 2285  143.42  114.20                  -40.00\n",
      "  152  10620 283D 284E C   N    2290 2292 -171.73 -117.80   73.33    2.30  -63.60  171.57   28.61\n",
      "  152        284E 284E N   CA   2292 2293 -173.51  136.80                  -40.30\n",
      "  153  10627 290T 291V C   N    2341 2343  -64.77  -62.40    2.38    0.37 -125.40 -175.28   10.33\n",
      "  153        291V 291V N   CA   2343 2344  -42.21  -42.40                  143.30\n",
      "  154  10628 291V 292D C   N    2348 2350 -159.81  -96.50   63.35    2.58   54.50  162.16   21.05\n",
      "  154        292D 292D N   CA   2350 2351  112.11  114.20                   40.90\n",
      "  155  10630 293V 294P C   N    2363 2365  -60.76  -64.50    4.80    0.53  -58.70  179.30   14.98\n",
      "  155        294P 294P N   CA   2365 2366  150.22  147.20                  -30.50\n",
      "  156  10631 294P 295K C   N    2370 2372  -62.76  -70.20   12.44    1.04  -62.90  168.83   21.92\n",
      "  156        295K 295K N   CA   2372 2373  150.37  140.40                  -40.80\n",
      "  157  10632 295K 296F C   N    2379 2381  110.95 -124.20  132.89    3.91  -63.20 -144.54   26.22\n",
      "  157        296F 296F N   CA   2381 2382 -171.16  143.30                  -44.30\n",
      "  158  10633 296F 297L C   N    2390 2392  -63.95  -70.70   12.31    0.76  -63.50  172.50   23.93\n",
      "  158        297L 297L N   CA   2392 2393  131.30  141.60                  -41.20\n",
      "  159  10634 297L 298S C   N    2398 2400 -179.67 -136.60   72.59    2.55  -64.10  163.29   20.10\n",
      "  159        298S 298S N   CA   2400 2401 -150.37  151.20                  -35.00\n",
      "  160  10639 302I 303D C   N    2434 2436  -81.83  -70.90   11.07    0.89  -63.30  172.46   22.45\n",
      "  160        303D 303D N   CA   2436 2437  148.54  150.30                  -40.00\n",
      "  161  10641 304V 305Q C   N    2449 2451 -170.69 -121.10   69.90    2.27  -63.80  168.87   30.03\n",
      "  161        305Q 305Q N   CA   2451 2452 -171.03  139.70                  -40.30\n",
      "  162  10643 306I 307E C   N    2466 2468 -149.68 -117.80   32.10    1.09  -63.60 -161.23   31.32\n",
      "  162        307E 307E N   CA   2468 2469  140.53  136.80                  -40.30\n",
      "  163  10644 307E 308V C   N    2475 2477 -136.93 -125.40   23.84    0.83  -62.40  170.58   26.16\n",
      "  163        308V 308V N   CA   2477 2478  164.17  143.30                  -42.40\n",
      "  164  10645 308V 309N C   N    2482 2484 -133.15  -63.20   85.12   13.98  -63.20   85.12   13.98\n",
      "  164        309N 309N N   CA   2484 2485  -89.59  -41.10                  -41.10\n",
      "  165  10655 318K 319L C   N    2571 2573 -141.65 -108.50   33.90    1.54  -63.50 -164.50   31.98\n",
      "  165        319L 319L N   CA   2573 2574  139.60  132.50                  -41.20\n",
      "  166  10657 320I 321Q C   N    2587 2589 -127.87 -121.10   26.08    1.33  -63.80  167.55   21.90\n",
      "  166        321Q 321Q N   CA   2589 2590  114.51  139.70                  -40.30\n",
      "  167  10660 323P 324L C   N    2610 2612 -148.27 -108.50   56.06    2.53  -63.50  169.51   28.75\n",
      "  167        324L 324L N   CA   2612 2613  172.01  132.50                  -41.20\n",
      "  168  10662 325Q 326R C   N    2627 2629 -133.53 -125.20   34.41    1.46  -63.00  161.16   26.57\n",
      "  168        326R 326R N   CA   2629 2630  173.99  140.60                  -41.10\n",
      "  169  10663 326R 327E C   N    2638 2640  -86.29  -69.30   25.94    2.38  -63.60  164.76   20.80\n",
      "  169        327E 327E N   CA   2640 2641  122.89  142.50                  -40.30\n",
      "  170  10665 328I 329S C   N    2655 2657  -96.38  -72.40   48.01    2.21  -64.10  134.92   11.98\n",
      "  170        329S 329S N   CA   2657 2658 -166.00  152.40                  -35.00\n",
      "  171  10668 331S 332A C   N    2674 2676   67.46  -62.50  149.44   24.83  -62.50  149.44   24.83\n",
      "  171        332A 332A N   CA   2676 2677 -114.68  -40.90                  -40.90\n",
      "  172  10670 333A 334R C   N    2684 2686  145.84 -125.20   96.84    4.82  -63.00 -151.63   24.87\n",
      "  172        334R 334R N   CA   2686 2687  102.32  140.60                  -41.10\n",
      "  173  10671 334R 335V C   N    2695 2697  -66.22  -62.40    6.81    1.09  -73.50  172.92   14.87\n",
      "  173        335V 335V N   CA   2697 2698  -48.03  -42.40                  139.20\n",
      "  174  10673 336Q 337R C   N    2711 2713   55.05   57.30   21.73    1.60  -63.00  131.30   23.88\n",
      "  174        337R 337R N   CA   2713 2714   16.39   38.00                  -41.10\n",
      "  175  10674 337R 338S C   N    2722 2724  148.45 -136.60  142.78    5.30  -64.10  156.44   20.93\n",
      "  175        338S 338S N   CA   2724 2725  -87.28  151.20                  -35.00\n",
      "  176  10677 340V 341T C   N    2742 2744  -43.14  -78.10   63.57    2.10   55.90  114.38    9.00\n",
      "  176        341T 341T N   CA   2744 2745   96.71  149.80                   39.50\n",
      "  177  10686 349P 350Y C   N    2804 2806   62.09  -98.40  160.60    6.30  -63.50 -151.93   38.73\n",
      "  177        350Y 350Y N   CA   2806 2807  122.48  128.40                  -43.40\n",
      "  178  10688 351A 352S C   N    2821 2823  148.35 -136.60   75.12    2.90  -64.10 -134.51   26.96\n",
      "  178        352S 352S N   CA   2823 2824  154.48  151.20                  -35.00\n",
      "  179  10689 352S 353H C   N    2827 2829  143.09 -125.60  127.47    3.15  -63.20  178.09   30.59\n",
      "  179        353H 353H N   CA   2829 2830 -132.25  138.80                  -42.30\n",
      "  180  10690 353H 354V C   N    2837 2839  -63.07  -62.40    1.22    0.13 -125.40 -173.93   10.42\n",
      "  180        354V 354V N   CA   2839 2840  -41.38  -42.40                  143.30\n",
      "  181  10691 354V 355L C   N    2844 2846  -61.78  -70.70   16.70    1.03  -63.50  168.69   23.52\n",
      "  181        355L 355L N   CA   2846 2847  127.48  141.60                  -41.20\n",
      "  182  10697 360E 361A C   N    2896 2898  -65.49  -68.20   11.84    0.86  -62.50  174.70   28.48\n",
      "  182        361A 361A N   CA   2898 2899  133.77  145.30                  -40.90\n",
      "  183  10698 361A 362K C   N    2901 2903 -152.46 -118.00   37.34    1.24  -62.90 -171.64   29.81\n",
      "  183        362K 362K N   CA   2903 2904  153.50  139.10                  -40.80\n",
      "  184  10699 362K 363K C   N    2910 2912 -135.16 -118.00   44.94    2.32  -62.90  156.10   17.93\n",
      "  184        363K 363K N   CA   2912 2913   97.57  139.10                  -40.80\n",
      "  185  10700 363K 364R C   N    2919 2921 -162.85 -125.20   37.67    1.48  -63.00 -156.89   34.16\n",
      "  185        364R 364R N   CA   2921 2922  142.03  140.60                  -41.10\n",
      "  186  10701 364R 365R C   N    2930 2932 -151.70 -125.20   28.48    0.90  -63.00 -170.14   31.65\n",
      "  186        365R 365R N   CA   2932 2933  151.03  140.60                  -41.10\n",
      "  187  10702 365R 366Q C   N    2941 2943  -84.49  -73.00   63.30    4.37  -63.80  120.55   16.70\n",
      "  187        366Q 366Q N   CA   2943 2944   78.46  140.70                  -40.30\n",
      "  188  10703 366Q 367R C   N    2950 2952 -172.94 -125.20   63.79    1.91  -63.00  174.87   30.72\n",
      "  188        367R 367R N   CA   2952 2953 -177.09  140.60                  -41.10\n",
      "  189  10704 367R 368L C   N    2961 2963  -95.87 -108.50   18.95    0.86  -63.50  162.83   20.98\n",
      "  189        368L 368L N   CA   2963 2964  118.37  132.50                  -41.20\n",
      "  190  10705 368L 369H C   N    2969 2971  -80.16  -67.60   14.53    1.27  -63.20  175.83   20.98\n",
      "  190        369H 369H N   CA   2971 2972  132.71  140.00                  -42.30\n",
      "  191  10706 369H 370G C   N    2979 2981   64.23   78.70   18.02    0.51   82.20  164.85    8.90\n",
      "  191        370G 370G N   CA   2981 2982 -155.37 -166.10                    8.50\n",
      "  192  10707 370G 371T C   N    2983 2985 -114.39 -124.80   12.28    0.74  -63.20  175.51   25.91\n",
      "  192        371T 371T N   CA   2985 2986  150.02  143.50                  -42.10\n",
      "  193  10708 371T 372W C   N    2990 2992 -116.97 -124.90    8.47    0.44  -63.00  177.81   25.82\n",
      "  193        372W 372W N   CA   2992 2993  146.38  143.40                  -44.20\n",
      "  194  10709 372W 373N C   N    3004 3006 -103.50 -119.90   23.79    0.70  -63.20  165.84   18.80\n",
      "  194        373N 373N N   CA   3006 3007  119.77  137.00                  -41.10\n",
      "  195  10710 373N 374D C   N    3012 3014 -115.50  -96.50   19.33    0.79  -63.30  159.47   17.25\n",
      "  195        374D 374D N   CA   3014 3015  110.69  114.20                  -40.00\n",
      "  196  10711 374D 375D C   N    3020 3022  -98.12  -96.50    7.98    0.33  -63.30  150.47   16.77\n",
      "  196        375D 375D N   CA   3022 3023  106.38  114.20                  -40.00\n",
      "  197  10712 375D 376E C   N    3028 3030 -144.77 -117.80   43.05    1.48  -63.60  169.97   27.17\n",
      "  197        376E 376E N   CA   3030 3031  170.36  136.80                  -40.30\n",
      "  198  10714 377C 378S C   N    3043 3045  -73.35  -72.40    0.95    0.07  -64.10  172.90   13.10\n",
      "  198        378S 378S N   CA   3045 3046  152.34  152.40                  -35.00\n",
      "  199  10715 378S 379E C   N    3049 3051 -108.77 -117.80   10.83    0.55  -63.60 -177.41   26.90\n",
      "  199        379E 379E N   CA   3051 3052  142.78  136.80                  -40.30\n",
      "  200  10717 380P 381L C   N    3065 3067  -78.71  -70.70    9.34    0.66  -63.50  173.07   24.93\n",
      "  200        381L 381L N   CA   3067 3068  146.39  141.60                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   51   74  313  287  452  367  415  481  530  582\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00184.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      382     3084     3084\n",
      "openf___224_> Open           EAO00184.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     36946889   36080.945    35.235\n",
      "read_mo_297_> Segments, residues, atoms:        1      382     3084\n",
      "read_mo_298_> Segment:        1  1:A  382:A     3084\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3084     3084        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13392\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13854\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14132\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14626\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15344\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    16575\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    17868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    18897\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19140\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "iupac_m_487_> NH1/2 swapped:      179.1729       12  12\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.8052       14  14\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.8002       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.1237       60  60\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.2592       82  82\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.0154       85  85\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.3051       88  88\n",
      "iupac_m_487_> NH1/2 swapped:      179.8455       92  92\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.5201      102  102\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2978      108  108\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.1983      114  114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.0870      143  143\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.2488      144  144\n",
      "iupac_m_485_> OD1/2 will be swapped:     -178.8150      146  146\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.6715      149  149\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8092      155  155\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7295      159  159\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.7039      180  180\n",
      "iupac_m_487_> NH1/2 swapped:      177.6637      181  181\n",
      "iupac_m_486_> OE1/2 will be swapped:     -140.4482      185  185\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3239      197  197\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.1037      200  200\n",
      "iupac_m_487_> NH1/2 swapped:      178.7070      213  213\n",
      "iupac_m_487_> NH1/2 swapped:     -177.1594      218  218\n",
      "iupac_m_486_> OE1/2 will be swapped:       90.3961      219  219\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.3321      224  224\n",
      "iupac_m_486_> OE1/2 will be swapped:      143.4188      228  228\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3867      235  235\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.9084      236  236\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8629      253  253\n",
      "iupac_m_486_> OE1/2 will be swapped:     -136.0568      257  257\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.7779      258  258\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1588      265  265\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0184      269  269\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1127      277  277\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.8964      279  279\n",
      "iupac_m_486_> OE1/2 will be swapped:     -137.8607      281  281\n",
      "iupac_m_486_> OE1/2 will be swapped:     -152.5183      284  284\n",
      "iupac_m_486_> OE1/2 will be swapped:      138.8682      307  307\n",
      "iupac_m_487_> NH1/2 swapped:      179.3538      312  312\n",
      "iupac_m_486_> OE1/2 will be swapped:      140.2209      317  317\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8899      337  337\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.9511      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:      113.9250      360  360\n",
      "iupac_m_485_> OD1/2 will be swapped:      100.9453      375  375\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.1840      379  379\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.3184      382  382\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLU:OXT  GLU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   629791\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22821.5195\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22821.519531\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.383999\n",
      "Sequence length          :          382\n",
      "Compactness              :     0.120475\n",
      "Native energy (pair)     :   718.874624\n",
      "Native energy (surface)  :    50.255441\n",
      "Native energy (combined) :    24.757952\n",
      "Z score (pair)           :    -1.547535\n",
      "Z score (surface)        :    -0.614577\n",
      "Z score (combined)       :    -1.378495\n",
      "GA341 score              :     0.181389\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7624\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3228.3455\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3122       0      0   0.004   0.004      14.837       1.000\n",
      " 2 Bond angle potential               :    4211       0      6   1.885   1.885      283.36       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1797       0    102  52.263  52.263      779.14       1.000\n",
      " 4 Stereochemical improper torsion pot:    1207       0      0   1.265   1.265      39.319       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7624       0      0   0.004   0.004      18.097       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2701       1      8   1.482   1.482      364.43       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2876       1     17   1.851   1.851      562.30       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     381       0      5   4.491   4.491      90.623       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     338       0      6  83.838  83.838      120.03       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     272       0      1  72.554  72.554      125.73       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     144       0      0 102.681 102.681      117.74       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      55       0      0  96.478  96.478      36.892       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1188       0      0   0.681   0.681      14.135       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     380     194    105  45.464 122.291      610.43       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     477       0      0   2.985   2.985      51.278       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00184.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00184.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   68833.6172\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13620 218R 358I CA  CA   1743 2873   16.50   12.91    3.59    4.65   12.91    3.59    4.65\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16646 285T 217L N   O    2301 1741    7.56    4.82    2.74    4.74    4.82    2.74    4.74\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10338   1M   2G C   N       7    9   76.17   78.70    9.57    0.18   82.20  165.48    8.34\n",
      "    1          2G   2G N   CA      9   10 -156.87 -166.10                    8.50\n",
      "    2  10339   2G   3R C   N      11   13  -73.09  -72.10   25.37    1.93  -63.00  151.99   21.22\n",
      "    2          3R   3R N   CA     13   14  167.25  141.90                  -41.10\n",
      "    3  10347  10R  11R C   N      81   83   52.79   57.30   24.37    1.32  -63.00  155.01   27.96\n",
      "    3         11R  11R N   CA     83   84   61.95   38.00                  -41.10\n",
      "    4  10348  11R  12R C   N      92   94  -82.20  -72.10   18.97    1.63  -63.00  168.04   21.47\n",
      "    4         12R  12R N   CA     94   95  125.84  141.90                  -41.10\n",
      "    5  10349  12R  13A C   N     103  105 -170.86 -134.00   43.22    1.02  -62.50 -175.31   35.97\n",
      "    5         13A  13A N   CA    105  106  169.55  147.00                  -40.90\n",
      "    6  10350  13A  14E C   N     108  110  -83.32  -69.30   54.50    4.61  -63.60  131.63   16.54\n",
      "    6         14E  14E N   CA    110  111   89.84  142.50                  -40.30\n",
      "    7  10351  14E  15H C   N     117  119  175.32 -125.60   65.93    1.74  -63.20 -167.26   31.68\n",
      "    7         15H  15H N   CA    119  120  168.06  138.80                  -42.30\n",
      "    8  10352  15H  16N C   N     127  129  -90.41 -119.90   53.58    1.70  -63.20  136.11   15.67\n",
      "    8         16N  16N N   CA    129  130   92.27  137.00                  -41.10\n",
      "    9  10353  16N  17E C   N     135  137 -156.11 -117.80   62.21    2.16  -63.60  162.73   26.71\n",
      "    9         17E  17E N   CA    137  138 -174.19  136.80                  -40.30\n",
      "   10  10354  17E  18G C   N     144  146  101.71   78.70   87.73    1.61  -62.40 -137.37   41.48\n",
      "   10         18G  18G N   CA    146  147  109.24 -166.10                  -41.20\n",
      "   11  10361  24K  25E C   N     193  195  -52.66  -69.30   17.39    1.13  -63.60  178.08   24.35\n",
      "   11         25E  25E N   CA    195  196  137.45  142.50                  -40.30\n",
      "   12  10363  26I  27A C   N     210  212  -59.83  -68.20    8.60    0.66  -62.50  175.76   28.67\n",
      "   12         27A  27A N   CA    212  213  143.36  145.30                  -40.90\n",
      "   13  10364  27A  28L C   N     215  217 -106.66 -108.50   18.76    1.03  -63.50  173.10   26.68\n",
      "   13         28L  28L N   CA    217  218  151.17  132.50                  -41.20\n",
      "   14  10365  28L  29H C   N     223  225   49.73   56.30   11.68    0.93  -63.20  146.14   24.86\n",
      "   14         29H  29H N   CA    225  226   50.46   40.80                  -42.30\n",
      "   15  10366  29H  30Q C   N     233  235  -68.73  -73.00    6.83    0.52  -63.80  173.73   25.80\n",
      "   15         30Q  30Q N   CA    235  236  146.03  140.70                  -40.30\n",
      "   16  10367  30Q  31Q C   N     242  244  -92.77  -73.00   31.67    1.82  -63.80  156.95   24.63\n",
      "   16         31Q  31Q N   CA    244  245  165.44  140.70                  -40.30\n",
      "   17  10370  33I  34E C   N     267  269 -145.09 -117.80   35.30    1.08  -63.60 -179.98   28.55\n",
      "   17         34E  34E N   CA    269  270  159.18  136.80                  -40.30\n",
      "   18  10371  34E  35K C   N     276  278 -154.20  -62.90  107.08   18.68  -62.90  107.08   18.68\n",
      "   18         35K  35K N   CA    278  279  -96.75  -40.80                  -40.80\n",
      "   19  10372  35K  36I C   N     285  287  -58.90  -63.40    4.98    0.81 -120.60 -173.46    8.44\n",
      "   19         36I  36I N   CA    287  288  -45.73  -43.60                  130.30\n",
      "   20  10373  36I  37E C   N     293  295  -62.76  -69.30   17.34    1.09  -63.60  166.74   22.28\n",
      "   20         37E  37E N   CA    295  296  126.44  142.50                  -40.30\n",
      "   21  10375  38V  39V C   N     309  311  -65.71  -62.40    3.48    0.59 -125.40 -176.78   10.24\n",
      "   21         39V  39V N   CA    311  312  -43.48  -42.40                  143.30\n",
      "   22  10376  39V  40G C   N     316  318   95.53   78.70   35.16    0.52   82.20  155.10    8.20\n",
      "   22         40G  40G N   CA    318  319  163.03 -166.10                    8.50\n",
      "   23  10382  45E  46L C   N     359  361 -115.03 -108.50   31.66    1.77  -63.50  151.74   18.84\n",
      "   23         46L  46L N   CA    361  362  101.52  132.50                  -41.20\n",
      "   24  10383  46L  47E C   N     367  369 -133.91 -117.80   26.41    1.44  -63.60  171.28   19.97\n",
      "   24         47E  47E N   CA    369  370  115.88  136.80                  -40.30\n",
      "   25  10385  48I  49L C   N     384  386  -92.09  -70.70   29.21    1.89  -63.50  159.87   23.93\n",
      "   25         49L  49L N   CA    386  387  161.50  141.60                  -41.20\n",
      "   26  10386  49L  50Y C   N     392  394  -81.19  -98.40   17.36    0.74  -63.50  170.39   26.15\n",
      "   26         50Y  50Y N   CA    394  395  126.06  128.40                  -43.40\n",
      "   27  10387  50Y  51L C   N     404  406 -139.35 -108.50   42.87    1.93  -63.50  173.94   28.84\n",
      "   27         51L  51L N   CA    406  407  162.27  132.50                  -41.20\n",
      "   28  10389  52C  53N C   N     418  420  -61.56  -71.20   15.56    0.83  -63.20  171.69   21.56\n",
      "   28         53N  53N N   CA    420  421  130.58  142.80                  -41.10\n",
      "   29  10390  53N  54N C   N     426  428 -131.36 -119.90   37.41    1.41  -63.20  161.39   24.02\n",
      "   29         54N  54N N   CA    428  429  172.61  137.00                  -41.10\n",
      "   30  10391  54N  55Y C   N     434  436  -73.13  -98.40   26.73    1.26  -63.50  179.75   29.02\n",
      "   30         55Y  55Y N   CA    436  437  137.10  128.40                  -43.40\n",
      "   31  10393  56I  57S C   N     454  456  -90.31  -72.40   36.16    1.66  -64.10  143.60   12.15\n",
      "   31         57S  57S N   CA    456  457 -176.19  152.40                  -35.00\n",
      "   32  10394  57S  58R C   N     460  462  -65.42  -72.10   17.14    1.45  -63.00  161.23   21.94\n",
      "   32         58R  58R N   CA    462  463  157.69  141.90                  -41.10\n",
      "   33  10397  60E  61G C   N     488  490  106.69   78.70   49.22    0.82   82.20  146.96    8.34\n",
      "   33         61G  61G N   CA    490  491  153.41 -166.10                    8.50\n",
      "   34  10398  61G  62L C   N     492  494 -118.10 -108.50   24.07    1.18  -63.50  173.07   27.40\n",
      "   34         62L  62L N   CA    494  495  154.57  132.50                  -41.20\n",
      "   35  10402  65L  66K C   N     527  529 -137.49 -118.00   20.55    0.69  -62.90 -171.07   29.00\n",
      "   35         66K  66K N   CA    529  530  145.62  139.10                  -40.80\n",
      "   36  10403  66K  67R C   N     536  538   57.45   57.30   24.16    1.60  -63.00  158.65   28.68\n",
      "   36         67R  67R N   CA    538  539   62.16   38.00                  -41.10\n",
      "   37  10404  67R  68L C   N     547  549  -43.61  -70.70   28.29    2.78  -63.50  170.17   22.56\n",
      "   37         68L  68L N   CA    549  550  149.79  141.60                  -41.20\n",
      "   38  10405  68L  69K C   N     555  557  -44.04  -70.20   27.75    2.22   56.60  149.89    9.07\n",
      "   38         69K  69K N   CA    557  558  149.67  140.40                   38.60\n",
      "   39  10412  75V  76N C   N     612  614 -120.86 -119.90    8.86    0.42  -63.20  178.84   19.78\n",
      "   39         76N  76N N   CA    614  615  128.19  137.00                  -41.10\n",
      "   40  10413  76N  77N C   N     620  622 -150.01 -119.90   39.95    1.16  -63.20  178.22   27.13\n",
      "   40         77N  77N N   CA    622  623  163.25  137.00                  -41.10\n",
      "   41  10415  78I  79A C   N     636  638  167.48 -134.00   72.77    1.86  -62.50 -176.94   36.59\n",
      "   41         79A  79A N   CA    638  639 -169.75  147.00                  -40.90\n",
      "   42  10418  81I  82E C   N     655  657 -149.16 -117.80   58.62    2.17  -63.60  158.46   25.81\n",
      "   42         82E  82E N   CA    657  658 -173.68  136.80                  -40.30\n",
      "   43  10421  84L  85E C   N     676  678 -165.03  -63.60  104.38   15.28  -63.60  104.38   15.28\n",
      "   43         85E  85E N   CA    678  679  -15.67  -40.30                  -40.30\n",
      "   44  10422  85E  86G C   N     685  687   98.79   82.20   38.55    1.12  -62.40  161.87   29.47\n",
      "   44         86G  86G N   CA    687  688  -26.30    8.50                  -41.20\n",
      "   45  10425  88E  89S C   N     704  706  161.66 -136.60   61.76    2.41  -64.10 -141.70   25.50\n",
      "   45         89S  89S N   CA    706  707  152.84  151.20                  -35.00\n",
      "   46  10427  90L  91E C   N     718  720 -132.96 -117.80   82.90    3.85  -63.60  122.85   20.15\n",
      "   46         91E  91E N   CA    720  721 -141.70  136.80                  -40.30\n",
      "   47  10428  91E  92R C   N     727  729  -86.70  -72.10   41.22    3.45  -63.00  146.38   18.29\n",
      "   47         92R  92R N   CA    729  730  103.35  141.90                  -41.10\n",
      "   48  10429  92R  93L C   N     738  740 -132.67 -108.50   48.90    2.33  -63.50  159.56   26.43\n",
      "   48         93L  93L N   CA    740  741  175.01  132.50                  -41.20\n",
      "   49  10430  93L  94D C   N     746  748  -65.41  -70.90    9.00    0.32   54.50  157.60   10.54\n",
      "   49         94D  94D N   CA    748  749  143.16  150.30                   40.90\n",
      "   50  10432  95L  96T C   N     762  764 -110.19 -124.80   46.89    1.61  -63.20  148.67   16.43\n",
      "   50         96T  96T N   CA    764  765   98.94  143.50                  -42.10\n",
      "   51  10434  97L  98N C   N     777  779 -117.31 -119.90   80.62    3.72  -63.20  114.87   17.39\n",
      "   51         98N  98N N   CA    779  780 -142.42  137.00                  -41.10\n",
      "   52  10437 100V 101A C   N     803  805  -63.71  -68.20   85.14    7.03  -68.20   85.14    7.03\n",
      "   52        101A 101A N   CA    805  806 -129.68  145.30                  145.30\n",
      "   53  10439 102E 103M C   N     817  819 -174.45 -125.60   48.95    1.86  -63.40 -151.96   39.58\n",
      "   53        103M 103M N   CA    819  820  143.58  140.50                  -40.50\n",
      "   54  10440 103M 104T C   N     825  827 -158.65 -124.80   54.59    1.70  -63.20  162.56   26.79\n",
      "   54        104T 104T N   CA    827  828 -173.68  143.50                  -42.10\n",
      "   55  10444 107Q 108R C   N     854  856 -161.19 -125.20   60.27    3.34  -63.00  165.60   18.86\n",
      "   55        108R 108R N   CA    856  857   92.26  140.60                  -41.10\n",
      "   56  10445 108R 109L C   N     865  867 -143.34 -108.50   45.95    2.05  -63.50  175.54   29.31\n",
      "   56        109L 109L N   CA    867  868  162.46  132.50                  -41.20\n",
      "   57  10446 109L 110R C   N     873  875  -83.59  -72.10   29.81    2.05  -63.00  150.91   21.80\n",
      "   57        110R 110R N   CA    875  876  169.41  141.90                  -41.10\n",
      "   58  10447 110R 111A C   N     884  886  -62.96  -68.20   16.69    1.52  -62.50  157.96   25.91\n",
      "   58        111A 111A N   CA    886  887  161.14  145.30                  -40.90\n",
      "   59  10448 111A 112N C   N     889  891  -78.59 -119.90   41.68    1.87  -63.20  177.01   22.99\n",
      "   59        112N 112N N   CA    891  892  142.56  137.00                  -41.10\n",
      "   60  10459 122G 123N C   N     977  979 -117.24 -119.90   22.49    1.08  -63.20  168.47   24.14\n",
      "   60        123N 123N N   CA    979  980  159.33  137.00                  -41.10\n",
      "   61  10462 125C 126T C   N     998 1000  -81.81  -78.10   57.27    2.33  -63.20  112.50   15.67\n",
      "   61        126T 126T N   CA   1000 1001 -153.05  149.80                  -42.10\n",
      "   62  10465 128I 129E C   N    1022 1024  123.80 -117.80  119.06    4.07  -63.60 -117.44   41.14\n",
      "   62        129E 129E N   CA   1024 1025  149.27  136.80                  -40.30\n",
      "   63  10466 129E 130G C   N    1031 1033   59.83   82.20   73.37    2.47  -62.40  171.00   31.73\n",
      "   63        130G 130G N   CA   1033 1034   78.39    8.50                  -41.20\n",
      "   64  10469 132R 133A C   N    1058 1060   70.84  -68.20  139.41   11.47  -62.50 -139.15   43.19\n",
      "   64        133A 133A N   CA   1060 1061  135.16  145.30                  -40.90\n",
      "   65  10470 133A 134Y C   N    1063 1065 -137.30 -124.30   27.85    2.03  -63.50  170.92   23.99\n",
      "   65        134Y 134Y N   CA   1065 1066  110.77  135.40                  -43.40\n",
      "   66  10473 136I 137H C   N    1090 1092 -158.59  -63.20  111.90   19.21  -63.20  111.90   19.21\n",
      "   66        137H 137H N   CA   1092 1093 -100.81  -42.30                  -42.30\n",
      "   67  10474 137H 138A C   N    1100 1102  -61.25  -68.20   21.97    2.00  -62.50  152.97   25.01\n",
      "   67        138A 138A N   CA   1102 1103  166.14  145.30                  -40.90\n",
      "   68  10475 138A 139L C   N    1105 1107  -55.22  -70.70   17.81    1.28  -63.50  174.17   24.67\n",
      "   68        139L 139L N   CA   1107 1108  132.77  141.60                  -41.20\n",
      "   69  10477 140P 141Q C   N    1120 1122 -111.78 -121.10   22.01    1.15  -63.80  167.10   27.13\n",
      "   69        141Q 141Q N   CA   1122 1123  159.63  139.70                  -40.30\n",
      "   70  10479 142L 143R C   N    1137 1139  -58.77  -72.10   15.59    1.03  -63.00  174.98   23.92\n",
      "   70        143R 143R N   CA   1139 1140  133.83  141.90                  -41.10\n",
      "   71  10481 144D 145L C   N    1156 1158  -86.79  -70.70   17.24    1.33  -63.50  172.60   25.36\n",
      "   71        145L 145L N   CA   1158 1159  147.78  141.60                  -41.20\n",
      "   72  10483 146D 147G C   N    1172 1174  103.83   82.20   34.31    0.98  -62.40  167.83   30.81\n",
      "   72        147G 147G N   CA   1174 1175  -18.13    8.50                  -41.20\n",
      "   73  10487 150V 151V C   N    1201 1203  -53.93  -62.40   14.35    2.32 -125.40 -171.78    7.25\n",
      "   73        151V 151V N   CA   1203 1204  -30.82  -42.40                  143.30\n",
      "   74  10495 158A 159R C   N    1267 1269  -81.60  -72.10   86.13    6.83  -63.00   99.15   12.25\n",
      "   74        159R 159R N   CA   1269 1270   56.29  141.90                  -41.10\n",
      "   75  10496 159R 160Q C   N    1278 1280  -66.68  -73.00   58.85    3.98  -63.80  120.52   17.87\n",
      "   75        160Q 160Q N   CA   1280 1281 -160.79  140.70                  -40.30\n",
      "   76  10498 161C 162N C   N    1293 1295  -84.93  -71.20   42.87    3.12  -63.20  144.92   17.01\n",
      "   76        162N 162N N   CA   1295 1296  102.18  142.80                  -41.10\n",
      "   77  10499 162N 163S C   N    1301 1303 -158.63 -136.60   27.79    0.86  -64.10 -176.86   20.08\n",
      "   77        163S 163S N   CA   1303 1304  168.14  151.20                  -35.00\n",
      "   78  10502 165V 166A C   N    1323 1325 -150.21 -134.00   24.47    0.79  -62.50  177.03   33.69\n",
      "   78        166A 166A N   CA   1325 1326  165.33  147.00                  -40.90\n",
      "   79  10503 166A 167A C   N    1328 1330 -119.77 -134.00   19.18    0.54  -62.50 -175.83   27.99\n",
      "   79        167A 167A N   CA   1330 1331  134.15  147.00                  -40.90\n",
      "   80  10508 171E 172A C   N    1364 1366   81.71 -134.00  144.49    5.08  -68.20  150.03   12.51\n",
      "   80        172A 172A N   CA   1366 1367  139.31  147.00                  145.30\n",
      "   81  10509 172A 173A C   N    1369 1371 -138.47 -134.00   15.75    0.96  -62.50 -171.24   28.37\n",
      "   81        173A 173A N   CA   1371 1372  131.90  147.00                  -40.90\n",
      "   82  10510 173A 174L C   N    1374 1376  163.74 -108.50   91.87    4.12  -63.50 -152.77   36.71\n",
      "   82        174L 174L N   CA   1376 1377  159.68  132.50                  -41.20\n",
      "   83  10512 175R 176V C   N    1393 1395  -63.82  -62.40    4.40    0.65 -125.40 -179.07   10.15\n",
      "   83        176V 176V N   CA   1395 1396  -46.57  -42.40                  143.30\n",
      "   84  10513 176V 177R C   N    1400 1402  -83.46  -72.10   44.87    3.20   57.30 -156.26   12.88\n",
      "   84        177R 177R N   CA   1402 1403 -174.70  141.90                   38.00\n",
      "   85  10515 178E 179A C   N    1420 1422  -60.22  -62.50    5.72    1.06   55.40  137.20   16.19\n",
      "   85        179A 179A N   CA   1422 1423  -35.65  -40.90                   38.20\n",
      "   86  10516 179A 180E C   N    1425 1427   65.83  -69.30  152.91   13.65  -63.60  167.06   20.54\n",
      "   86        180E 180E N   CA   1427 1428 -145.94  142.50                  -40.30\n",
      "   87  10517 180E 181R C   N    1434 1436  -75.13  -72.10   52.47    4.10  -63.00  131.17   16.93\n",
      "   87        181R 181R N   CA   1436 1437   89.51  141.90                  -41.10\n",
      "   88  10519 182I 183K C   N    1453 1455 -158.83 -118.00   49.27    1.65  -62.90 -179.83   29.10\n",
      "   88        183K 183K N   CA   1455 1456  166.69  139.10                  -40.80\n",
      "   89  10520 183K 184A C   N    1462 1464  -63.51  -68.20    8.90    0.57  -62.50  178.63   29.22\n",
      "   89        184A 184A N   CA   1464 1465  137.73  145.30                  -40.90\n",
      "   90  10521 184A 185E C   N    1467 1469 -178.15 -117.80   71.84    2.12  -63.60 -176.05   30.62\n",
      "   90        185E 185E N   CA   1469 1470  175.77  136.80                  -40.30\n",
      "   91  10522 185E 186M C   N    1476 1478  -80.84  -73.00   83.50    5.78  -63.40  101.87   13.94\n",
      "   91        186M 186M N   CA   1478 1479   59.86  143.00                  -40.50\n",
      "   92  10524 187I 188A C   N    1492 1494  -61.86  -68.20   13.51    0.88  -62.50  174.27   28.59\n",
      "   92        188A 188A N   CA   1494 1495  133.37  145.30                  -40.90\n",
      "   93  10530 193V 194F C   N    1533 1535   42.08  -71.40  114.89    7.22  -63.20 -162.51   33.98\n",
      "   93        194F 194F N   CA   1535 1536  122.78  140.70                  -44.30\n",
      "   94  10533 196P 197R C   N    1558 1560 -124.55 -125.20   96.52    4.68  -63.00  102.36   17.84\n",
      "   94        197R 197R N   CA   1560 1561 -122.88  140.60                  -41.10\n",
      "   95  10534 197R 198Y C   N    1569 1571 -112.10  -98.40   14.03    0.59  -63.50  175.62   25.57\n",
      "   95        198Y 198Y N   CA   1571 1572  125.36  128.40                  -43.40\n",
      "   96  10535 198Y 199N C   N    1581 1583  112.26 -119.90  137.32    4.47  -63.20 -140.56   25.74\n",
      "   96        199N 199N N   CA   1583 1584 -172.89  137.00                  -41.10\n",
      "   97  10536 199N 200E C   N    1589 1591 -109.22 -117.80    9.82    0.29  -63.60  178.27   21.60\n",
      "   97        200E 200E N   CA   1591 1592  132.03  136.80                  -40.30\n",
      "   98  10537 200E 201K C   N    1598 1600 -154.75  -62.90   92.17   14.75  -62.90   92.17   14.75\n",
      "   98        201K 201K N   CA   1600 1601  -33.18  -40.80                  -40.80\n",
      "   99  10538 201K 202G C   N    1607 1609  112.22   82.20   59.31    1.64  -62.40  174.63   31.14\n",
      "   99        202G 202G N   CA   1609 1610  -42.65    8.50                  -41.20\n",
      "  100  10540 203E 204R C   N    1620 1622 -131.19 -125.20   91.64    4.55  -63.00  113.12   12.91\n",
      "  100        204R 204R N   CA   1622 1623   49.15  140.60                  -41.10\n",
      "  101  10543 206Y 207G C   N    1650 1652   41.12   78.70   76.02    1.15   82.20  116.04    7.56\n",
      "  101        207G 207G N   CA   1652 1653 -100.02 -166.10                    8.50\n",
      "  102  10544 207G 208H C   N    1654 1656 -160.59  -63.20  100.95   14.48  -63.20  100.95   14.48\n",
      "  102        208H 208H N   CA   1656 1657  -15.71  -42.30                  -42.30\n",
      "  103  10560 223E 224E C   N    1796 1798 -160.90 -117.80   61.04    1.96  -63.60  170.22   27.97\n",
      "  103        224E 224E N   CA   1798 1799 -179.97  136.80                  -40.30\n",
      "  104  10561 224E 225A C   N    1805 1807  -71.70  -68.20   33.44    2.57  -62.50  140.84   23.54\n",
      "  104        225A 225A N   CA   1807 1808  178.56  145.30                  -40.90\n",
      "  105  10562 225A 226E C   N    1810 1812  -55.32  -69.30   16.34    1.47  -63.60  168.94   22.08\n",
      "  105        226E 226E N   CA   1812 1813  150.96  142.50                  -40.30\n",
      "  106  10563 226E 227R C   N    1819 1821   64.97   57.30    7.85    0.83  -63.00  149.55   27.26\n",
      "  106        227R 227R N   CA   1821 1822   36.29   38.00                  -41.10\n",
      "  107  10564 227R 228E C   N    1830 1832  -80.82  -69.30   54.11    3.71   54.60 -155.70   11.75\n",
      "  107        228E 228E N   CA   1832 1833 -164.64  142.50                   42.40\n",
      "  108  10566 229K 230K C   N    1848 1850  -96.62 -118.00   45.61    1.84  -62.90  143.63   17.10\n",
      "  108        230K 230K N   CA   1850 1851   98.81  139.10                  -40.80\n",
      "  109  10568 231L 232Q C   N    1865 1867 -154.47 -121.10   81.66    3.21  -63.80  139.08   24.82\n",
      "  109        232Q 232Q N   CA   1867 1868 -145.77  139.70                  -40.30\n",
      "  110  10569 232Q 233K C   N    1874 1876 -120.30 -118.00   31.31    1.49  -62.90  159.55   24.15\n",
      "  110        233K 233K N   CA   1876 1877  170.33  139.10                  -40.80\n",
      "  111  10570 233K 234E C   N    1883 1885  -75.15  -69.30   49.95    3.60  -63.60  128.11   17.72\n",
      "  111        234E 234E N   CA   1885 1886 -167.89  142.50                  -40.30\n",
      "  112  10571 234E 235R C   N    1892 1894  -50.17  -72.10   39.55    2.61  -63.00  150.63   21.22\n",
      "  112        235R 235R N   CA   1894 1895  108.99  141.90                  -41.10\n",
      "  113  10572 235R 236E C   N    1903 1905  140.22 -117.80  115.17    3.37  -63.60 -157.19   34.71\n",
      "  113        236E 236E N   CA   1905 1906 -169.68  136.80                  -40.30\n",
      "  114  10573 236E 237T C   N    1912 1914  -68.07  -78.10   10.49    0.55  -63.20  171.24   22.28\n",
      "  114        237T 237T N   CA   1914 1915  146.73  149.80                  -42.10\n",
      "  115  10574 237T 238G C   N    1919 1921  -78.15  -80.20    2.86    0.20   82.20 -128.06    7.09\n",
      "  115        238G 238G N   CA   1921 1922  176.09  174.10                    8.50\n",
      "  116  10580 243V 244S C   N    1959 1961 -130.50 -136.60   39.28    1.86  -64.10  161.66   10.13\n",
      "  116        244S 244S N   CA   1961 1962  112.40  151.20                  -35.00\n",
      "  117  10581 244S 245A C   N    1965 1967 -121.71 -134.00   27.16    1.15  -62.50  174.06   26.34\n",
      "  117        245A 245A N   CA   1967 1968  122.78  147.00                  -40.90\n",
      "  118  10582 245A 246E C   N    1970 1972  127.63 -117.80  131.20    3.84  -63.60 -153.51   35.49\n",
      "  118        246E 246E N   CA   1972 1973 -159.28  136.80                  -40.30\n",
      "  119  10584 247V 248N C   N    1986 1988   53.84   55.90    6.94    0.36  -63.20  145.97   23.93\n",
      "  119        248N 248N N   CA   1988 1989   46.12   39.50                  -41.10\n",
      "  120  10589 252Q 253R C   N    2030 2032 -154.34 -125.20   34.30    1.02  -63.00 -175.57   31.06\n",
      "  120        253R 253R N   CA   2032 2033  158.68  140.60                  -41.10\n",
      "  121  10590 253R 254L C   N    2041 2043 -124.45 -108.50   59.80    3.04  -63.50  142.36   23.54\n",
      "  121        254L 254L N   CA   2043 2044 -169.86  132.50                  -41.20\n",
      "  122  10592 255T 256A C   N    2056 2058  -81.33  -68.20   28.57    1.87  -62.50  149.62   25.49\n",
      "  122        256A 256A N   CA   2058 2059  170.67  145.30                  -40.90\n",
      "  123  10593 256A 257E C   N    2061 2063  -57.25  -69.30   12.20    0.99  -63.60  175.41   23.05\n",
      "  123        257E 257E N   CA   2063 2064  144.40  142.50                  -40.30\n",
      "  124  10594 257E 258E C   N    2070 2072 -138.92 -117.80   89.02    4.02  -63.60  122.35   20.33\n",
      "  124        258E 258E N   CA   2072 2073 -136.72  136.80                  -40.30\n",
      "  125  10596 259E 260I C   N    2088 2090  -93.55  -97.30   40.84    4.26  -63.40  133.59   20.95\n",
      "  125        260I 260I N   CA   2090 2091   86.54  127.20                  -43.60\n",
      "  126  10597 260I 261A C   N    2096 2098 -151.90 -134.00   27.36    1.63  -62.50 -170.39   28.41\n",
      "  126        261A 261A N   CA   2098 2099  126.31  147.00                  -40.90\n",
      "  127  10598 261A 262K C   N    2101 2103   50.71   56.60    7.24    0.55  -62.90  141.06   24.48\n",
      "  127        262K 262K N   CA   2103 2104   42.82   38.60                  -40.80\n",
      "  128  10599 262K 263Y C   N    2110 2112 -153.53 -124.30   47.19    1.94  -63.50  169.96   31.23\n",
      "  128        263Y 263Y N   CA   2112 2113  172.45  135.40                  -43.40\n",
      "  129  10600 263Y 264G C   N    2122 2124  -64.34  -62.40   11.70    1.66   82.20  151.43   11.29\n",
      "  129        264G 264G N   CA   2124 2125  -29.66  -41.20                    8.50\n",
      "  130  10604 267L 268L C   N    2153 2155 -116.19 -108.50   73.47    3.89  -63.50  124.89   20.60\n",
      "  130        268L 268L N   CA   2155 2156 -154.43  132.50                  -41.20\n",
      "  131  10605 268L 269R C   N    2161 2163  -49.85  -72.10   24.62    2.08  -63.00  166.98   21.70\n",
      "  131        269R 269R N   CA   2163 2164  152.44  141.90                  -41.10\n",
      "  132  10606 269R 270N C   N    2172 2174 -138.18 -119.90   63.40    2.42  -63.20  142.52   21.97\n",
      "  132        270N 270N N   CA   2174 2175 -162.30  137.00                  -41.10\n",
      "  133  10607 270N 271E C   N    2180 2182  -66.44  -69.30   16.32    1.34  -63.60  161.16   21.64\n",
      "  133        271E 271E N   CA   2182 2183  158.57  142.50                  -40.30\n",
      "  134  10609 272P 273K C   N    2196 2198  -55.50  -70.20   14.77    1.11  -62.90  177.47   22.65\n",
      "  134        273K 273K N   CA   2198 2199  141.88  140.40                  -40.80\n",
      "  135  10610 273K 274F C   N    2205 2207  -75.46  -71.40   13.65    0.98  -63.20  162.43   23.38\n",
      "  135        274F 274F N   CA   2207 2208  153.73  140.70                  -44.30\n",
      "  136  10611 274F 275P C   N    2216 2218  -63.39  -58.70    6.22    0.47  -64.50  173.62   12.90\n",
      "  136        275P 275P N   CA   2218 2219  -26.41  -30.50                  147.20\n",
      "  137  10612 275P 276M C   N    2223 2225  -73.64  -73.00   18.43    1.26  -63.40  165.40   23.91\n",
      "  137        276M 276M N   CA   2225 2226  124.58  143.00                  -40.50\n",
      "  138  10613 276M 277R C   N    2231 2233  -89.84  -72.10   20.00    1.70  -63.00  175.81   22.06\n",
      "  138        277R 277R N   CA   2233 2234  132.65  141.90                  -41.10\n",
      "  139  10614 277R 278L C   N    2242 2244 -132.33 -108.50   24.58    1.11  -63.50 -167.52   23.82\n",
      "  139        278L 278L N   CA   2244 2245  138.55  132.50                  -41.20\n",
      "  140  10615 278L 279D C   N    2250 2252  -95.39  -96.50   17.66    0.74  -63.30  140.30   15.65\n",
      "  140        279D 279D N   CA   2252 2253   96.58  114.20                  -40.00\n",
      "  141  10616 279D 280Q C   N    2258 2260 -132.85 -121.10   15.52    0.49  -63.80 -176.64   30.60\n",
      "  141        280Q 280Q N   CA   2260 2261  149.84  139.70                  -40.30\n",
      "  142  10617 280Q 281E C   N    2267 2269 -103.14 -117.80   15.02    0.48  -63.60  178.26   21.84\n",
      "  142        281E 281E N   CA   2269 2270  133.52  136.80                  -40.30\n",
      "  143  10618 281E 282S C   N    2276 2278 -168.91 -136.60   33.01    1.14  -64.10 -162.82   21.85\n",
      "  143        282S 282S N   CA   2278 2279  157.98  151.20                  -35.00\n",
      "  144  10619 282S 283D C   N    2282 2284 -160.31  -96.50   64.13    2.62  -63.30 -172.45   19.94\n",
      "  144        283D 283D N   CA   2284 2285  120.51  114.20                  -40.00\n",
      "  145  10620 283D 284E C   N    2290 2292  172.52 -117.80   94.51    2.96  -63.60  171.82   29.19\n",
      "  145        284E 284E N   CA   2292 2293 -159.36  136.80                  -40.30\n",
      "  146  10626 289L 290T C   N    2334 2336  158.00 -124.80   81.13    3.59  -78.10  125.30    7.11\n",
      "  146        290T 290T N   CA   2336 2337  168.42  143.50                  149.80\n",
      "  147  10627 290T 291V C   N    2341 2343  -64.47  -62.40    2.29    0.39 -125.40 -176.30   10.28\n",
      "  147        291V 291V N   CA   2343 2344  -43.39  -42.40                  143.30\n",
      "  148  10628 291V 292D C   N    2348 2350 -122.52  -96.50   41.05    1.69   54.50 -178.16   18.87\n",
      "  148        292D 292D N   CA   2350 2351   82.46  114.20                   40.90\n",
      "  149  10630 293V 294P C   N    2363 2365  -55.69  -64.50   25.76    1.55  -58.70  153.52   12.93\n",
      "  149        294P 294P N   CA   2365 2366  122.99  147.20                  -30.50\n",
      "  150  10633 296F 297L C   N    2390 2392 -152.72 -108.50   44.29    2.14  -63.50 -167.02   23.64\n",
      "  150        297L 297L N   CA   2392 2393  129.91  132.50                  -41.20\n",
      "  151  10634 297L 298S C   N    2398 2400  166.73 -136.60  108.32    4.03  -64.10  152.72   19.99\n",
      "  151        298S 298S N   CA   2400 2401 -116.49  151.20                  -35.00\n",
      "  152  10637 300T 301L C   N    2418 2420  -53.75  -63.50   13.05    2.37  -70.70  174.95   12.62\n",
      "  152        301L 301L N   CA   2420 2421  -32.53  -41.20                  141.60\n",
      "  153  10642 305Q 306I C   N    2458 2460  -62.90  -63.40    1.91    0.33 -120.60 -178.52    8.28\n",
      "  153        306I 306I N   CA   2460 2461  -41.76  -43.60                  130.30\n",
      "  154  10643 306I 307E C   N    2466 2468  -76.80  -69.30   34.58    2.37  -63.60  144.05   19.94\n",
      "  154        307E 307E N   CA   2468 2469  176.25  142.50                  -40.30\n",
      "  155  10644 307E 308V C   N    2475 2477  -59.03 -125.40   69.54    2.11  -62.40  164.97   21.08\n",
      "  155        308V 308V N   CA   2477 2478  122.54  143.30                  -42.40\n",
      "  156  10645 308V 309N C   N    2482 2484 -112.18 -119.90   14.64    0.47  -63.20  172.74   19.32\n",
      "  156        309N 309N N   CA   2484 2485  124.55  137.00                  -41.10\n",
      "  157  10655 318K 319L C   N    2571 2573 -163.52 -108.50   55.90    2.79  -63.50 -168.04   23.56\n",
      "  157        319L 319L N   CA   2573 2574  122.64  132.50                  -41.20\n",
      "  158  10657 320I 321Q C   N    2587 2589 -173.42 -121.10   56.41    1.77  -63.80 -166.95   33.88\n",
      "  158        321Q 321Q N   CA   2589 2590  160.79  139.70                  -40.30\n",
      "  159  10660 323P 324L C   N    2610 2612 -171.73 -108.50   70.24    3.10  -63.50 -170.37   32.91\n",
      "  159        324L 324L N   CA   2612 2613  163.09  132.50                  -41.20\n",
      "  160  10661 324L 325Q C   N    2618 2620 -103.52 -121.10   24.84    0.81  -63.80  167.24   22.70\n",
      "  160        325Q 325Q N   CA   2620 2621  122.15  139.70                  -40.30\n",
      "  161  10662 325Q 326R C   N    2627 2629 -138.75 -125.20   23.33    0.78  -63.00  176.40   28.99\n",
      "  161        326R 326R N   CA   2629 2630  159.60  140.60                  -41.10\n",
      "  162  10663 326R 327E C   N    2638 2640  -85.79  -69.30   52.86    4.55  -63.60  134.43   16.80\n",
      "  162        327E 327E N   CA   2640 2641   92.28  142.50                  -40.30\n",
      "  163  10665 328I 329S C   N    2655 2657 -154.52 -136.60   20.92    0.64  -64.10 -173.60   20.01\n",
      "  163        329S 329S N   CA   2657 2658  162.00  151.20                  -35.00\n",
      "  164  10666 329S 330P C   N    2661 2663  -56.69  -64.50   11.08    1.21  -58.70  174.46   14.25\n",
      "  164        330P 330P N   CA   2663 2664  155.06  147.20                  -30.50\n",
      "  165  10667 330P 331S C   N    2668 2670  -53.01 -136.60   87.38    4.13  -64.10  148.78   10.17\n",
      "  165        331S 331S N   CA   2670 2671  176.64  151.20                  -35.00\n",
      "  166  10668 331S 332A C   N    2674 2676  -38.98  -68.20   29.47    2.62  -62.50  171.55   27.06\n",
      "  166        332A 332A N   CA   2676 2677  149.17  145.30                  -40.90\n",
      "  167  10669 332A 333A C   N    2679 2681  -47.87  -68.20   46.27    3.06  -62.50  145.37   24.57\n",
      "  167        333A 333A N   CA   2681 2682  103.73  145.30                  -40.90\n",
      "  168  10670 333A 334R C   N    2684 2686 -170.93  -63.00  111.57   19.83  -63.00  111.57   19.83\n",
      "  168        334R 334R N   CA   2686 2687  -69.37  -41.10                  -41.10\n",
      "  169  10671 334R 335V C   N    2695 2697  -57.43  -62.40    5.82    0.99  -73.50  179.29   15.09\n",
      "  169        335V 335V N   CA   2697 2698  -39.37  -42.40                  139.20\n",
      "  170  10674 337R 338S C   N    2722 2724  167.80 -136.60   82.35    4.62  -64.10  179.31   14.65\n",
      "  170        338S 338S N   CA   2724 2725   90.46  151.20                  -35.00\n",
      "  171  10680 343A 344L C   N    2758 2760  -53.49  -70.70   20.39    2.06  -63.50  166.56   22.57\n",
      "  171        344L 344L N   CA   2760 2761  152.54  141.60                  -41.20\n",
      "  172  10685 348I 349P C   N    2797 2799  -53.90  -64.50   39.27    2.44  -58.70  139.96   11.96\n",
      "  172        349P 349P N   CA   2799 2800  109.38  147.20                  -30.50\n",
      "  173  10686 349P 350Y C   N    2804 2806 -139.78 -124.30   15.96    1.29  -63.50 -169.17   26.96\n",
      "  173        350Y 350Y N   CA   2806 2807  131.52  135.40                  -43.40\n",
      "  174  10687 350Y 351A C   N    2816 2818 -159.61 -134.00   30.47    0.73  -62.50 -176.60   35.22\n",
      "  174        351A 351A N   CA   2818 2819  163.52  147.00                  -40.90\n",
      "  175  10691 354V 355L C   N    2844 2846  -73.01  -70.70   62.13    4.80  -63.50  121.09   16.29\n",
      "  175        355L 355L N   CA   2846 2847   79.52  141.60                  -41.20\n",
      "  176  10697 360E 361A C   N    2896 2898  -58.88  -68.20   30.11    2.10  -62.50  157.61   26.01\n",
      "  176        361A 361A N   CA   2898 2899  116.67  145.30                  -40.90\n",
      "  177  10698 361A 362K C   N    2901 2903 -155.30 -118.00   39.57    1.33  -62.90 -169.23   30.29\n",
      "  177        362K 362K N   CA   2903 2904  152.30  139.10                  -40.80\n",
      "  178  10699 362K 363K C   N    2910 2912 -118.72 -118.00    9.31    0.46  -62.90  179.52   20.97\n",
      "  178        363K 363K N   CA   2912 2913  129.82  139.10                  -40.80\n",
      "  179  10700 363K 364R C   N    2919 2921 -121.41 -125.20   17.47    0.75  -63.00  174.69   20.39\n",
      "  179        364R 364R N   CA   2921 2922  123.54  140.60                  -41.10\n",
      "  180  10701 364R 365R C   N    2930 2932 -130.54 -125.20   21.23    1.12  -63.00  174.73   20.10\n",
      "  180        365R 365R N   CA   2932 2933  120.05  140.60                  -41.10\n",
      "  181  10702 365R 366Q C   N    2941 2943 -109.71 -121.10   69.83    3.12  -63.80  120.22   15.72\n",
      "  181        366Q 366Q N   CA   2943 2944   70.80  139.70                  -40.30\n",
      "  182  10703 366Q 367R C   N    2950 2952 -153.59 -125.20   36.98    1.10  -63.00  179.18   30.29\n",
      "  182        367R 367R N   CA   2952 2953  164.31  140.60                  -41.10\n",
      "  183  10704 367R 368L C   N    2961 2963  -68.45  -70.70    2.51    0.18  -63.50  178.38   25.05\n",
      "  183        368L 368L N   CA   2963 2964  140.49  141.60                  -41.20\n",
      "  184  10705 368L 369H C   N    2969 2971 -118.78 -125.60   14.19    0.63  -63.20  175.49   25.58\n",
      "  184        369H 369H N   CA   2971 2972  151.24  138.80                  -42.30\n",
      "  185  10706 369H 370G C   N    2979 2981   97.59   78.70   67.38    1.20   82.20  121.69    6.67\n",
      "  185        370G 370G N   CA   2981 2982  129.22 -166.10                    8.50\n",
      "  186  10707 370G 371T C   N    2983 2985 -138.10 -124.80   19.42    0.62  -63.20  176.89   27.57\n",
      "  186        371T 371T N   CA   2985 2986  157.65  143.50                  -42.10\n",
      "  187  10708 371T 372W C   N    2990 2992 -102.82 -124.90   22.35    0.90  -63.00 -179.71   25.15\n",
      "  187        372W 372W N   CA   2992 2993  139.96  143.40                  -44.20\n",
      "  188  10709 372W 373N C   N    3004 3006  -92.91 -119.90   35.03    1.02  -63.20  158.57   18.34\n",
      "  188        373N 373N N   CA   3006 3007  114.66  137.00                  -41.10\n",
      "  189  10710 373N 374D C   N    3012 3014  -69.11  -70.90   14.09    0.61  -63.30  176.41   21.43\n",
      "  189        374D 374D N   CA   3014 3015  136.32  150.30                  -40.00\n",
      "  190  10711 374D 375D C   N    3020 3022  -75.65  -70.90    8.74    0.30  -63.30  162.83   20.86\n",
      "  190        375D 375D N   CA   3022 3023  157.64  150.30                  -40.00\n",
      "  191  10712 375D 376E C   N    3028 3030 -104.40 -117.80   20.69    0.70  -63.60  166.41   20.23\n",
      "  191        376E 376E N   CA   3030 3031  121.03  136.80                  -40.30\n",
      "  192  10714 377C 378S C   N    3043 3045 -109.17 -136.60   39.76    1.30  -64.10  163.74   10.13\n",
      "  192        378S 378S N   CA   3045 3046  122.41  151.20                  -35.00\n",
      "  193  10715 378S 379E C   N    3049 3051 -154.04 -117.80   41.33    1.21  -63.60 -173.57   29.87\n",
      "  193        379E 379E N   CA   3051 3052  156.67  136.80                  -40.30\n",
      "  194  10717 380P 381L C   N    3065 3067  -74.14  -70.70    5.01    0.32  -63.50  173.88   24.77\n",
      "  194        381L 381L N   CA   3067 3068  145.25  141.60                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   63   70  360  326  415  380  423  472  510  540\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00184.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      382     3084     3084\n",
      "openf___224_> Open           EAO00184.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     36946889   36080.945    35.235\n",
      "read_mo_297_> Segments, residues, atoms:        1      382     3084\n",
      "read_mo_298_> Segment:        1  1:A  382:A     3084\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3084     3084        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13392\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13854\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14132\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14626\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15344\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    16575\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    17868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    18897\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19140\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "iupac_m_486_> OE1/2 will be swapped:     -132.0249        7  7\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.7722       25  25\n",
      "iupac_m_485_> OD1/2 will be swapped:      161.8315       32  32\n",
      "iupac_m_486_> OE1/2 will be swapped:     -132.2347       34  34\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.8224       37  37\n",
      "iupac_m_485_> OD1/2 will be swapped:      110.7356       41  41\n",
      "iupac_m_486_> OE1/2 will be swapped:      128.3467       45  45\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.9957       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3216       58  58\n",
      "iupac_m_486_> OE1/2 will be swapped:      161.8097       85  85\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.5743      102  102\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9604      108  108\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.0995      116  116\n",
      "iupac_m_486_> OE1/2 will be swapped:      158.3981      129  129\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.0130      131  131\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8953      132  132\n",
      "iupac_m_487_> NH1/2 swapped:     -177.5446      143  143\n",
      "iupac_m_485_> OD1/2 will be swapped:      124.0359      144  144\n",
      "iupac_m_485_> OD1/2 will be swapped:     -166.5108      146  146\n",
      "iupac_m_486_> OE1/2 will be swapped:      109.9422      148  148\n",
      "iupac_m_486_> OE1/2 will be swapped:      105.9553      149  149\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4923      152  152\n",
      "iupac_m_486_> OE1/2 will be swapped:      101.5660      154  154\n",
      "iupac_m_486_> OE1/2 will be swapped:     -129.4262      164  164\n",
      "iupac_m_486_> OE1/2 will be swapped:      -94.1571      180  180\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.9085      185  185\n",
      "iupac_m_487_> NH1/2 swapped:     -176.1479      197  197\n",
      "iupac_m_486_> OE1/2 will be swapped:      127.0130      200  200\n",
      "iupac_m_486_> OE1/2 will be swapped:      -92.7472      203  203\n",
      "iupac_m_487_> NH1/2 swapped:     -167.0169      204  204\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.0392      211  211\n",
      "iupac_m_486_> OE1/2 will be swapped:      117.5460      212  212\n",
      "iupac_m_487_> NH1/2 swapped:      178.4916      213  213\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.4161      221  221\n",
      "iupac_m_486_> OE1/2 will be swapped:      -95.5081      223  223\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4670      227  227\n",
      "iupac_m_486_> OE1/2 will be swapped:      162.1635      228  228\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.3737      246  246\n",
      "iupac_m_487_> NH1/2 swapped:      179.1448      250  250\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.8503      257  257\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.2947      258  258\n",
      "iupac_m_486_> OE1/2 will be swapped:     -151.9985      259  259\n",
      "iupac_m_487_> NH1/2 swapped:      180.0000      269  269\n",
      "iupac_m_486_> OE1/2 will be swapped:      132.6224      271  271\n",
      "iupac_m_485_> OD1/2 will be swapped:      167.8007      303  303\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.6046      307  307\n",
      "iupac_m_487_> NH1/2 swapped:      179.3167      312  312\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.8534      317  317\n",
      "iupac_m_487_> NH1/2 swapped:      171.1201      337  337\n",
      "iupac_m_486_> OE1/2 will be swapped:     -126.8721      359  359\n",
      "iupac_m_487_> NH1/2 swapped:      177.9692      364  364\n",
      "iupac_m_487_> NH1/2 swapped:      179.3871      365  365\n",
      "iupac_m_487_> NH1/2 swapped:      179.8264      367  367\n",
      "iupac_m_485_> OD1/2 will be swapped:      167.9308      374  374\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.7913      375  375\n",
      "iupac_m_486_> OE1/2 will be swapped:     -159.4488      376  376\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLU:OXT  GLU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   622008\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -21988.7305\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -21988.730469\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.383999\n",
      "Sequence length          :          382\n",
      "Compactness              :     0.144294\n",
      "Native energy (pair)     :   725.806252\n",
      "Native energy (surface)  :    43.975382\n",
      "Native energy (combined) :    22.180796\n",
      "Z score (pair)           :    -0.142965\n",
      "Z score (surface)        :    -0.600766\n",
      "Z score (combined)       :    -0.480924\n",
      "GA341 score              :     0.081617\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7391\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3235.3833\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3122       0      0   0.004   0.004      16.260       1.000\n",
      " 2 Bond angle potential               :    4211       0      3   1.936   1.936      301.41       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1797       0    120  52.817  52.817      792.64       1.000\n",
      " 4 Stereochemical improper torsion pot:    1207       0      0   1.269   1.269      38.936       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7391       0      0   0.004   0.004      13.424       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2701       0      8   1.508   1.508      382.79       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2876       2     22   1.849   1.849      559.04       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     381       0      6   4.888   4.888      107.34       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     338       0      3  81.435  81.435      107.28       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     272       0      1  77.730  77.730      120.50       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     144       0      0  94.005  94.005      115.71       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      55       0      0 101.587 101.587      36.270       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1188       0      0   0.673   0.673      15.339       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     380     187    106  44.466 122.195      577.84       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     477       0      1   2.529   2.529      50.611       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00184.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00184.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   69194.6328\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16645 285T 216M N   O    2301 1733   10.84    7.29    3.55    4.51    7.29    3.55    4.51\n",
      "    2  16646 285T 217L N   O    2301 1741    8.14    4.82    3.33    5.76    4.82    3.33    5.76\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10338   1M   2G C   N       7    9   70.56   78.70   34.03    0.65   82.20  142.04    7.48\n",
      "    1          2G   2G N   CA      9   10 -133.06 -166.10                    8.50\n",
      "    2  10339   2G   3R C   N      11   13  -60.66  -72.10   15.19    1.32  -63.00  167.03   22.41\n",
      "    2          3R   3R N   CA     13   14  151.89  141.90                  -41.10\n",
      "    3  10347  10R  11R C   N      81   83   43.68   57.30   33.04    1.60  -63.00  152.66   27.28\n",
      "    3         11R  11R N   CA     83   84   68.10   38.00                  -41.10\n",
      "    4  10348  11R  12R C   N      92   94  -96.75  -72.10   39.29    3.41  -63.00  156.09   19.04\n",
      "    4         12R  12R N   CA     94   95  111.30  141.90                  -41.10\n",
      "    5  10349  12R  13A C   N     103  105 -172.35 -134.00   53.00    1.55  -62.50  174.46   34.31\n",
      "    5         13A  13A N   CA    105  106 -176.42  147.00                  -40.90\n",
      "    6  10351  14E  15H C   N     117  119  -67.67  -67.60    5.13    0.40  -63.20  177.24   21.88\n",
      "    6         15H  15H N   CA    119  120  134.88  140.00                  -42.30\n",
      "    7  10352  15H  16N C   N     127  129  -97.19 -119.90   36.66    1.11  -63.20  153.14   17.49\n",
      "    7         16N  16N N   CA    129  130  108.22  137.00                  -41.10\n",
      "    8  10353  16N  17E C   N     135  137 -134.81 -117.80   95.96    4.47  -63.60  113.55   18.92\n",
      "    8         17E  17E N   CA    137  138 -128.76  136.80                  -40.30\n",
      "    9  10354  17E  18G C   N     144  146   74.42   82.20   40.66    1.58  -62.40  163.55   30.88\n",
      "    9         18G  18G N   CA    146  147   48.41    8.50                  -41.20\n",
      "   10  10361  24K  25E C   N     193  195 -146.37 -117.80   29.68    0.92  -63.60 -166.52   30.43\n",
      "   10         25E  25E N   CA    195  196  144.82  136.80                  -40.30\n",
      "   11  10363  26I  27A C   N     210  212 -116.20 -134.00   22.04    0.56  -62.50 -177.04   27.89\n",
      "   11         27A  27A N   CA    212  213  134.01  147.00                  -40.90\n",
      "   12  10364  27A  28L C   N     215  217 -140.96 -108.50   33.58    1.51  -63.50 -166.13   31.72\n",
      "   12         28L  28L N   CA    217  218  141.08  132.50                  -41.20\n",
      "   13  10365  28L  29H C   N     223  225 -123.40 -125.60   19.20    0.69  -63.20  172.85   18.72\n",
      "   13         29H  29H N   CA    225  226  119.73  138.80                  -42.30\n",
      "   14  10367  30Q  31Q C   N     242  244  -57.44  -73.00   18.64    1.45  -63.80  168.86   24.50\n",
      "   14         31Q  31Q N   CA    244  245  150.96  140.70                  -40.30\n",
      "   15  10368  31Q  32D C   N     251  253  -88.07  -63.30   66.35    7.10  -63.30   66.35    7.10\n",
      "   15         32D  32D N   CA    253  254   21.56  -40.00                  -40.00\n",
      "   16  10370  33I  34E C   N     267  269  -72.73  -69.30    4.30    0.39  -63.60 -179.99   24.50\n",
      "   16         34E  34E N   CA    269  270  139.92  142.50                  -40.30\n",
      "   17  10371  34E  35K C   N     276  278 -121.87 -118.00   35.12    1.65  -62.90  156.71   23.88\n",
      "   17         35K  35K N   CA    278  279  174.01  139.10                  -40.80\n",
      "   18  10373  36I  37E C   N     293  295   57.35  -69.30  128.06    9.02  -63.60 -156.34   33.66\n",
      "   18         37E  37E N   CA    295  296  123.55  142.50                  -40.30\n",
      "   19  10376  39V  40G C   N     316  318 -162.47 -167.20   37.81    1.42   82.20  172.72   13.10\n",
      "   19         40G  40G N   CA    318  319  137.08  174.60                    8.50\n",
      "   20  10380  43C  44R C   N     339  341  -59.63  -72.10   23.82    1.58  -63.00  162.73   22.21\n",
      "   20         44R  44R N   CA    341  342  121.60  141.90                  -41.10\n",
      "   21  10382  45E  46L C   N     359  361  -79.81  -70.70   17.23    1.07  -63.50  163.39   23.66\n",
      "   21         46L  46L N   CA    361  362  156.22  141.60                  -41.20\n",
      "   22  10383  46L  47E C   N     367  369 -110.30 -117.80   36.89    1.96  -63.60  154.03   23.18\n",
      "   22         47E  47E N   CA    369  370  172.92  136.80                  -40.30\n",
      "   23  10385  48I  49L C   N     384  386  -62.35  -70.70   10.75    0.71  -63.50  176.03   24.51\n",
      "   23         49L  49L N   CA    386  387  134.83  141.60                  -41.20\n",
      "   24  10387  50Y  51L C   N     404  406  -55.02  -70.70   35.26    2.21  -63.50  151.45   21.52\n",
      "   24         51L  51L N   CA    406  407  110.01  141.60                  -41.20\n",
      "   25  10388  51L  52C C   N     412  414  -65.07  -63.00    5.99    0.89 -117.90 -179.90    7.75\n",
      "   25         52C  52C N   CA    414  415  -46.72  -41.10                  141.10\n",
      "   26  10389  52C  53N C   N     418  420  -75.53  -71.20    6.30    0.50  -63.20  179.74   21.82\n",
      "   26         53N  53N N   CA    420  421  138.22  142.80                  -41.10\n",
      "   27  10390  53N  54N C   N     426  428 -114.20 -119.90   52.64    2.26  -63.20  135.72   14.81\n",
      "   27         54N  54N N   CA    428  429   84.67  137.00                  -41.10\n",
      "   28  10394  57S  58R C   N     460  462 -147.31 -125.20   26.05    0.77  -63.00 -175.14   30.69\n",
      "   28         58R  58R N   CA    462  463  154.38  140.60                  -41.10\n",
      "   29  10396  59I  60E C   N     479  481 -134.03 -117.80   27.17    0.96  -63.60  175.83   27.40\n",
      "   29         60E  60E N   CA    481  482  158.59  136.80                  -40.30\n",
      "   30  10397  60E  61G C   N     488  490  -76.53  -80.20   27.69    0.91   82.20 -149.56    7.14\n",
      "   30         61G  61G N   CA    490  491  146.66  174.10                    8.50\n",
      "   31  10400  63Q  64H C   N     509  511   62.87   56.30   19.80    2.36  -63.20  162.02   27.59\n",
      "   31         64H  64H N   CA    511  512   59.48   40.80                  -42.30\n",
      "   32  10401  64H  65L C   N     519  521  -57.89  -70.70   20.77    2.03  -63.50  160.94   22.03\n",
      "   32         65L  65L N   CA    521  522  157.96  141.60                  -41.20\n",
      "   33  10402  65L  66K C   N     527  529   64.81   56.60   20.66    1.04  -62.90  141.29   24.68\n",
      "   33         66K  66K N   CA    529  530   19.64   38.60                  -40.80\n",
      "   34  10403  66K  67R C   N     536  538 -158.87 -125.20   98.89    3.93  -63.00  128.33   23.15\n",
      "   34         67R  67R N   CA    538  539 -126.42  140.60                  -41.10\n",
      "   35  10404  67R  68L C   N     547  549  -60.35  -70.70   11.65    1.17  -63.50  171.87   23.69\n",
      "   35         68L  68L N   CA    549  550  146.95  141.60                  -41.20\n",
      "   36  10405  68L  69K C   N     555  557 -148.27 -118.00   38.73    1.32   56.60 -160.99   24.92\n",
      "   36         69K  69K N   CA    557  558  163.26  139.10                   38.60\n",
      "   37  10412  75V  76N C   N     612  614 -101.84 -119.90   31.35    0.98  -63.20  157.30   17.81\n",
      "   37         76N  76N N   CA    614  615  111.38  137.00                  -41.10\n",
      "   38  10413  76N  77N C   N     620  622 -139.17 -119.90   24.66    0.72  -63.20 -176.97   27.18\n",
      "   38         77N  77N N   CA    622  623  152.38  137.00                  -41.10\n",
      "   39  10415  78I  79A C   N     636  638 -177.30 -134.00   60.83    1.82  -62.50  172.98   34.27\n",
      "   39         79A  79A N   CA    638  639 -170.29  147.00                  -40.90\n",
      "   40  10417  80C  81I C   N     647  649  -59.91  -63.40    4.98    0.76 -120.60 -172.46    8.53\n",
      "   40         81I  81I N   CA    649  650  -47.15  -43.60                  130.30\n",
      "   41  10418  81I  82E C   N     655  657   70.42  -69.30  160.09   14.35  -63.60  166.65   20.84\n",
      "   41         82E  82E N   CA    657  658 -139.34  142.50                  -40.30\n",
      "   42  10421  84L  85E C   N     676  678 -117.34 -117.80    5.63    0.29  -63.60 -174.74   27.74\n",
      "   42         85E  85E N   CA    678  679  142.41  136.80                  -40.30\n",
      "   43  10422  85E  86G C   N     685  687  102.23   82.20   45.66    1.32  -62.40  164.86   29.79\n",
      "   43         86G  86G N   CA    687  688  -32.53    8.50                  -41.20\n",
      "   44  10424  87C  88E C   N     695  697  -79.58  -69.30   64.86    5.29  -63.60  119.84   15.16\n",
      "   44         88E  88E N   CA    697  698   78.46  142.50                  -40.30\n",
      "   45  10426  89S  90L C   N     710  712 -135.27 -108.50   37.74    1.70  -63.50  175.08   28.75\n",
      "   45         90L  90L N   CA    712  713  159.11  132.50                  -41.20\n",
      "   46  10427  90L  91E C   N     718  720  -74.48  -69.30    9.10    0.52  -63.60  170.07   23.28\n",
      "   46         91E  91E N   CA    720  721  149.98  142.50                  -40.30\n",
      "   47  10428  91E  92R C   N     727  729   64.69   57.30    7.77    0.96  -63.00  151.48   27.61\n",
      "   47         92R  92R N   CA    729  730   40.40   38.00                  -41.10\n",
      "   48  10429  92R  93L C   N     738  740 -112.90 -108.50   44.02    2.33  -63.50  150.82   24.00\n",
      "   48         93L  93L N   CA    740  741  176.30  132.50                  -41.20\n",
      "   49  10430  93L  94D C   N     746  748  -64.75  -70.90   11.53    0.86   54.50  168.58   10.30\n",
      "   49         94D  94D N   CA    748  749  160.05  150.30                   40.90\n",
      "   50  10432  95L  96T C   N     762  764 -128.34 -124.80  101.48    4.46  -63.20  106.44   11.33\n",
      "   50         96T  96T N   CA    764  765   42.08  143.50                  -42.10\n",
      "   51  10433  96T  97L C   N     769  771  -99.27  -70.70   52.36    3.24  -63.50  138.03   21.36\n",
      "   51         97L  97L N   CA    771  772 -174.52  141.60                  -41.20\n",
      "   52  10437 100V 101A C   N     803  805  170.45 -134.00   69.58    1.79  -68.20  128.94    9.37\n",
      "   52        101A 101A N   CA    805  806 -171.11  147.00                  145.30\n",
      "   53  10439 102E 103M C   N     817  819 -141.60 -125.60   24.37    0.77  -63.40  178.64   32.81\n",
      "   53        103M 103M N   CA    819  820  158.88  140.50                  -40.50\n",
      "   54  10440 103M 104T C   N     825  827 -136.62 -124.80   29.87    0.97  -63.20  164.29   25.84\n",
      "   54        104T 104T N   CA    827  828  170.93  143.50                  -42.10\n",
      "   55  10443 106V 107Q C   N     845  847 -141.24 -121.10   34.89    1.82  -63.80  170.15   21.97\n",
      "   55        107Q 107Q N   CA    847  848  111.20  139.70                  -40.30\n",
      "   56  10446 109L 110R C   N     873  875 -148.71 -125.20   48.35    1.73  -63.00  160.79   27.43\n",
      "   56        110R 110R N   CA    875  876 -177.15  140.60                  -41.10\n",
      "   57  10447 110R 111A C   N     884  886 -135.32 -134.00   39.73    2.23  -62.50  151.10   28.65\n",
      "   57        111A 111A N   CA    886  887 -173.29  147.00                  -40.90\n",
      "   58  10448 111A 112N C   N     889  891  -60.98 -119.90   65.61    2.04  -63.20  149.26   18.78\n",
      "   58        112N 112N N   CA    891  892  108.14  137.00                  -41.10\n",
      "   59  10459 122G 123N C   N     977  979  -92.01 -119.90   35.81    1.92  -63.20  162.02   21.89\n",
      "   59        123N 123N N   CA    979  980  159.46  137.00                  -41.10\n",
      "   60  10460 123N 124P C   N     985  987  -58.10  -58.70    1.22    0.07  -64.50  178.88   13.01\n",
      "   60        124P 124P N   CA    987  988  -31.57  -30.50                  147.20\n",
      "   61  10463 126T 127K C   N    1005 1007  -82.08  -70.20   43.03    2.84  -62.90  138.78   19.13\n",
      "   61        127K 127K N   CA   1007 1008 -178.25  140.40                  -40.80\n",
      "   62  10465 128I 129E C   N    1022 1024 -153.07 -117.80   58.61    2.05  -63.60  162.86   26.59\n",
      "   62        129E 129E N   CA   1024 1025 -176.39  136.80                  -40.30\n",
      "   63  10466 129E 130G C   N    1031 1033   59.00   82.20   65.99    2.10  -62.40  164.81   30.70\n",
      "   63        130G 130G N   CA   1033 1034   70.27    8.50                  -41.20\n",
      "   64  10469 132R 133A C   N    1058 1060  169.93 -134.00   58.21    1.43  -62.50 -158.10   39.73\n",
      "   64        133A 133A N   CA   1060 1061  162.61  147.00                  -40.90\n",
      "   65  10470 133A 134Y C   N    1063 1065  167.10 -124.30   72.28    5.93  -63.50 -157.27   27.98\n",
      "   65        134Y 134Y N   CA   1065 1066  112.66  135.40                  -43.40\n",
      "   66  10473 136I 137H C   N    1090 1092 -158.01 -125.60   85.47    2.58  -63.20  137.66   22.99\n",
      "   66        137H 137H N   CA   1092 1093 -142.11  138.80                  -42.30\n",
      "   67  10474 137H 138A C   N    1100 1102  -64.59  -68.20   13.10    1.18  -62.50  161.22   26.52\n",
      "   67        138A 138A N   CA   1102 1103  157.89  145.30                  -40.90\n",
      "   68  10475 138A 139L C   N    1105 1107  -59.90  -70.70   14.57    0.95  -63.50  173.06   24.24\n",
      "   68        139L 139L N   CA   1107 1108  131.82  141.60                  -41.20\n",
      "   69  10477 140P 141Q C   N    1120 1122 -115.24 -121.10   39.96    1.99  -63.80  149.59   24.72\n",
      "   69        141Q 141Q N   CA   1122 1123  179.23  139.70                  -40.30\n",
      "   70  10478 141Q 142L C   N    1129 1131  -82.13 -108.50   46.39    2.17  -63.50  136.81   18.01\n",
      "   70        142L 142L N   CA   1131 1132   94.34  132.50                  -41.20\n",
      "   71  10479 142L 143R C   N    1137 1139  179.25 -125.20   65.60    1.94  -63.00 -174.44   32.65\n",
      "   71        143R 143R N   CA   1139 1140  175.49  140.60                  -41.10\n",
      "   72  10480 143R 144D C   N    1148 1150  -98.89  -96.50    9.28    0.39  -63.30  149.54   16.62\n",
      "   72        144D 144D N   CA   1150 1151  105.24  114.20                  -40.00\n",
      "   73  10483 146D 147G C   N    1172 1174  100.41   82.20   27.70    0.81  -62.40  165.34   30.53\n",
      "   73        147G 147G N   CA   1174 1175  -12.37    8.50                  -41.20\n",
      "   74  10487 150V 151V C   N    1201 1203  -54.68  -62.40   13.96    2.23 -125.40 -172.12    7.26\n",
      "   74        151V 151V N   CA   1203 1204  -30.76  -42.40                  143.30\n",
      "   75  10495 158A 159R C   N    1267 1269 -140.86 -125.20   21.50    1.18  -63.00 -175.78   21.03\n",
      "   75        159R 159R N   CA   1269 1270  125.86  140.60                  -41.10\n",
      "   76  10496 159R 160Q C   N    1278 1280 -157.43 -121.10   36.94    1.26  -63.80 -163.02   33.80\n",
      "   76        160Q 160Q N   CA   1280 1281  146.39  139.70                  -40.30\n",
      "   77  10498 161C 162N C   N    1293 1295 -108.84 -119.90   15.86    0.46  -63.20  172.87   19.46\n",
      "   77        162N 162N N   CA   1295 1296  125.64  137.00                  -41.10\n",
      "   78  10499 162N 163S C   N    1301 1303 -177.49 -136.60   47.82    1.46  -64.10 -172.76   21.75\n",
      "   78        163S 163S N   CA   1303 1304  175.99  151.20                  -35.00\n",
      "   79  10500 163S 164E C   N    1307 1309 -104.07 -117.80   48.95    2.16  -63.60  136.27   16.31\n",
      "   79        164E 164E N   CA   1309 1310   89.82  136.80                  -40.30\n",
      "   80  10502 165V 166A C   N    1323 1325 -159.72 -134.00   29.63    1.54  -62.50 -161.40   29.75\n",
      "   80        166A 166A N   CA   1325 1326  132.28  147.00                  -40.90\n",
      "   81  10503 166A 167A C   N    1328 1330 -131.09 -134.00   16.39    0.98  -62.50  170.39   31.60\n",
      "   81        167A 167A N   CA   1330 1331  163.13  147.00                  -40.90\n",
      "   82  10508 171E 172A C   N    1364 1366 -177.98 -134.00   46.41    1.09  -68.20  111.01    8.83\n",
      "   82        172A 172A N   CA   1366 1367  161.81  147.00                  145.30\n",
      "   83  10509 172A 173A C   N    1369 1371   52.21  -68.20  128.82    9.28  -62.50 -178.67   35.69\n",
      "   83        173A 173A N   CA   1371 1372   99.54  145.30                  -40.90\n",
      "   84  10510 173A 174L C   N    1374 1376  174.79 -108.50  116.51    5.32  -63.50  156.64   28.61\n",
      "   84        174L 174L N   CA   1376 1377 -139.80  132.50                  -41.20\n",
      "   85  10513 176V 177R C   N    1400 1402  169.41 -125.20   92.47    2.82   57.30 -158.05   21.68\n",
      "   85        177R 177R N   CA   1402 1403 -154.03  140.60                   38.00\n",
      "   86  10516 179A 180E C   N    1425 1427  -56.33  -69.30   17.41    1.60  -63.60  165.74   21.71\n",
      "   86        180E 180E N   CA   1427 1428  154.12  142.50                  -40.30\n",
      "   87  10517 180E 181R C   N    1434 1436  -66.91  -72.10   46.54    3.46  -63.00  136.81   18.22\n",
      "   87        181R 181R N   CA   1436 1437   95.65  141.90                  -41.10\n",
      "   88  10519 182I 183K C   N    1453 1455 -157.02 -118.00   39.67    1.39  -62.90 -163.14   31.19\n",
      "   88        183K 183K N   CA   1455 1456  146.30  139.10                  -40.80\n",
      "   89  10520 183K 184A C   N    1462 1464  -68.27  -68.20    0.41    0.03  -62.50  173.50   28.72\n",
      "   89        184A 184A N   CA   1464 1465  145.70  145.30                  -40.90\n",
      "   90  10521 184A 185E C   N    1467 1469 -157.82 -117.80   54.57    1.72  -63.60  173.60   28.29\n",
      "   90        185E 185E N   CA   1469 1470  173.89  136.80                  -40.30\n",
      "   91  10524 187I 188A C   N    1492 1494  -75.92  -68.20    9.67    0.63  -62.50  168.52   28.29\n",
      "   91        188A 188A N   CA   1494 1495  151.12  145.30                  -40.90\n",
      "   92  10525 188A 189Q C   N    1497 1499 -135.64 -121.10   45.58    1.88  -63.80  154.52   26.44\n",
      "   92        189Q 189Q N   CA   1499 1500 -177.10  139.70                  -40.30\n",
      "   93  10526 189Q 190G C   N    1506 1508  -90.57  -80.20   99.51    4.44   82.20 -174.83    9.40\n",
      "   93        190G 190G N   CA   1508 1509   75.13  174.10                    8.50\n",
      "   94  10530 193V 194F C   N    1533 1535   68.96  -71.40  142.78   10.50  -63.20 -160.95   22.69\n",
      "   94        194F 194F N   CA   1535 1536  166.85  140.70                  -44.30\n",
      "   95  10533 196P 197R C   N    1558 1560 -149.69 -125.20   55.78    2.07  -63.00  154.75   26.64\n",
      "   95        197R 197R N   CA   1560 1561 -169.29  140.60                  -41.10\n",
      "   96  10535 198Y 199N C   N    1581 1583  136.88   55.90   88.14   10.86  -63.20 -162.79   23.31\n",
      "   96        199N 199N N   CA   1583 1584   74.29   39.50                  -41.10\n",
      "   97  10536 199N 200E C   N    1589 1591  159.93 -117.80   84.54    2.67  -63.60 -147.06   35.60\n",
      "   97        200E 200E N   CA   1591 1592  156.23  136.80                  -40.30\n",
      "   98  10537 200E 201K C   N    1598 1600   53.95   56.60   32.62    2.14  -62.90  161.80   27.63\n",
      "   98        201K 201K N   CA   1600 1601   71.11   38.60                  -40.80\n",
      "   99  10538 201K 202G C   N    1607 1609   85.50   82.20   50.47    2.62  -62.40  178.57   33.69\n",
      "   99        202G 202G N   CA   1609 1610   58.86    8.50                  -41.20\n",
      "  100  10540 203E 204R C   N    1620 1622   65.06   57.30   15.88    0.79  -63.00  143.73   26.16\n",
      "  100        204R 204R N   CA   1622 1623   24.14   38.00                  -41.10\n",
      "  101  10560 223E 224E C   N    1796 1798  -72.15  -69.30   42.23    3.12  -63.60  135.34   18.52\n",
      "  101        224E 224E N   CA   1798 1799 -175.37  142.50                  -40.30\n",
      "  102  10561 224E 225A C   N    1805 1807 -162.06 -134.00   81.10    3.77  -62.50  138.31   27.69\n",
      "  102        225A 225A N   CA   1807 1808 -136.91  147.00                  -40.90\n",
      "  103  10562 225A 226E C   N    1810 1812  -73.25  -69.30   52.17    4.12  -63.60  131.13   16.97\n",
      "  103        226E 226E N   CA   1812 1813   90.48  142.50                  -40.30\n",
      "  104  10563 226E 227R C   N    1819 1821   57.64   57.30    0.34    0.04  -63.00  144.28   26.29\n",
      "  104        227R 227R N   CA   1821 1822   38.04   38.00                  -41.10\n",
      "  105  10564 227R 228E C   N    1830 1832  -65.75  -63.60   13.07    1.86   54.60  153.69   19.89\n",
      "  105        228E 228E N   CA   1832 1833  -53.19  -40.30                   42.40\n",
      "  106  10566 229K 230K C   N    1848 1850 -139.75 -118.00   22.85    0.99  -62.90 -170.81   21.78\n",
      "  106        230K 230K N   CA   1850 1851  132.08  139.10                  -40.80\n",
      "  107  10568 231L 232Q C   N    1865 1867 -109.34 -121.10   69.18    3.46  -63.80  120.74   20.16\n",
      "  107        232Q 232Q N   CA   1867 1868 -152.13  139.70                  -40.30\n",
      "  108  10569 232Q 233K C   N    1874 1876  -71.04  -70.20    3.24    0.21  -62.90  175.86   23.29\n",
      "  108        233K 233K N   CA   1876 1877  143.53  140.40                  -40.80\n",
      "  109  10570 233K 234E C   N    1883 1885 -161.24 -117.80   97.48    3.86  -63.60  136.68   23.19\n",
      "  109        234E 234E N   CA   1885 1886 -135.94  136.80                  -40.30\n",
      "  110  10572 235R 236E C   N    1903 1905 -108.54 -117.80   16.83    0.92  -63.60  174.72   25.84\n",
      "  110        236E 236E N   CA   1905 1906  150.86  136.80                  -40.30\n",
      "  111  10573 236E 237T C   N    1912 1914  -62.95  -78.10   28.17    0.93  -63.20  168.14   21.58\n",
      "  111        237T 237T N   CA   1914 1915  126.04  149.80                  -42.10\n",
      "  112  10574 237T 238G C   N    1919 1921   70.31   78.70   20.42    0.91   82.20  167.21    7.52\n",
      "  112        238G 238G N   CA   1921 1922  175.29 -166.10                    8.50\n",
      "  113  10581 244S 245A C   N    1965 1967   85.80 -134.00  158.40    3.63  -62.50  177.97   28.97\n",
      "  113        245A 245A N   CA   1967 1968 -139.29  147.00                  -40.90\n",
      "  114  10582 245A 246E C   N    1970 1972  -67.42  -69.30   32.67    2.56  -63.60  144.63   19.49\n",
      "  114        246E 246E N   CA   1972 1973  175.12  142.50                  -40.30\n",
      "  115  10584 247V 248N C   N    1986 1988   49.19   55.90    9.84    0.57  -63.20  142.62   23.35\n",
      "  115        248N 248N N   CA   1988 1989   46.70   39.50                  -41.10\n",
      "  116  10587 250R 251P C   N    2014 2016  -67.19  -58.70   10.52    0.86  -64.50  171.51   12.94\n",
      "  116        251P 251P N   CA   2016 2017  -24.29  -30.50                  147.20\n",
      "  117  10589 252Q 253R C   N    2030 2032 -127.28 -125.20    7.11    0.29  -63.00 -176.85   29.17\n",
      "  117        253R 253R N   CA   2032 2033  147.40  140.60                  -41.10\n",
      "  118  10592 255T 256A C   N    2056 2058 -128.55 -134.00    7.40    0.43  -62.50  179.68   32.98\n",
      "  118        256A 256A N   CA   2058 2059  152.00  147.00                  -40.90\n",
      "  119  10594 257E 258E C   N    2070 2072  -77.61  -69.30    9.10    0.56  -63.60  174.06   23.99\n",
      "  119        258E 258E N   CA   2072 2073  146.20  142.50                  -40.30\n",
      "  120  10596 259E 260I C   N    2088 2090  -47.61  -97.30   51.01    2.52  -63.40  160.02   26.79\n",
      "  120        260I 260I N   CA   2090 2091  115.64  127.20                  -43.60\n",
      "  121  10597 260I 261A C   N    2096 2098 -151.29 -134.00   20.78    0.51  -62.50 -176.51   34.81\n",
      "  121        261A 261A N   CA   2098 2099  158.53  147.00                  -40.90\n",
      "  122  10598 261A 262K C   N    2101 2103  148.45 -118.00  111.75    3.73  -62.90 -169.60   32.90\n",
      "  122        262K 262K N   CA   2103 2104 -159.77  139.10                  -40.80\n",
      "  123  10599 262K 263Y C   N    2110 2112  -61.15  -98.40   37.32    1.49  -63.50  169.41   27.00\n",
      "  123        263Y 263Y N   CA   2112 2113  126.00  128.40                  -43.40\n",
      "  124  10603 266L 267L C   N    2145 2147 -154.87 -108.50   49.19    2.19  -70.70   84.49    7.43\n",
      "  124        267L 267L N   CA   2147 2148  148.92  132.50                  141.60\n",
      "  125  10604 267L 268L C   N    2153 2155 -160.56 -108.50   62.11    3.39  -63.50  170.22   21.02\n",
      "  125        268L 268L N   CA   2155 2156   98.63  132.50                  -41.20\n",
      "  126  10605 268L 269R C   N    2161 2163 -112.39 -125.20   14.94    0.44  -63.00 -179.12   21.56\n",
      "  126        269R 269R N   CA   2163 2164  132.91  140.60                  -41.10\n",
      "  127  10606 269R 270N C   N    2172 2174 -112.86 -119.90   34.16    1.38  -63.20  152.96   16.90\n",
      "  127        270N 270N N   CA   2174 2175  103.58  137.00                  -41.10\n",
      "  128  10607 270N 271E C   N    2180 2182 -136.62 -117.80   20.54    0.96  -63.60 -176.02   21.51\n",
      "  128        271E 271E N   CA   2182 2183  128.57  136.80                  -40.30\n",
      "  129  10609 272P 273K C   N    2196 2198 -150.23 -118.00   35.04    1.16  -62.90 -172.13   29.62\n",
      "  129        273K 273K N   CA   2198 2199  152.86  139.10                  -40.80\n",
      "  130  10610 273K 274F C   N    2205 2207  -75.20  -71.40    5.67    0.50  -63.20  179.60   25.75\n",
      "  130        274F 274F N   CA   2207 2208  136.50  140.70                  -44.30\n",
      "  131  10612 275P 276M C   N    2223 2225 -150.62 -125.60   33.33    1.00  -63.40  179.57   33.58\n",
      "  131        276M 276M N   CA   2225 2226  162.53  140.50                  -40.50\n",
      "  132  10613 276M 277R C   N    2231 2233  -66.32  -72.10   10.45    0.69  -63.00  174.32   23.33\n",
      "  132        277R 277R N   CA   2233 2234  133.19  141.90                  -41.10\n",
      "  133  10614 277R 278L C   N    2242 2244 -102.94 -108.50   10.57    0.60  -63.50 -178.36   27.63\n",
      "  133        278L 278L N   CA   2244 2245  141.50  132.50                  -41.20\n",
      "  134  10615 278L 279D C   N    2250 2252  -57.99  -70.90   14.41    0.81  -63.30  176.19   21.43\n",
      "  134        279D 279D N   CA   2252 2253  143.89  150.30                  -40.00\n",
      "  135  10616 279D 280Q C   N    2258 2260 -101.49 -121.10   24.96    1.25  -63.80  168.81   26.84\n",
      "  135        280Q 280Q N   CA   2260 2261  155.15  139.70                  -40.30\n",
      "  136  10617 280Q 281E C   N    2267 2269   86.63 -117.80  158.19    6.53  -63.60 -148.82   35.82\n",
      "  136        281E 281E N   CA   2269 2270  108.12  136.80                  -40.30\n",
      "  137  10618 281E 282S C   N    2276 2278  151.57 -136.60   72.74    2.60  -64.10 -142.81   26.11\n",
      "  137        282S 282S N   CA   2278 2279  162.69  151.20                  -35.00\n",
      "  138  10619 282S 283D C   N    2282 2284 -151.57  -96.50   61.68    2.55  -63.30 -161.29   30.22\n",
      "  138        283D 283D N   CA   2284 2285  141.98  114.20                  -40.00\n",
      "  139  10620 283D 284E C   N    2290 2292 -162.66 -117.80   46.08    1.46  -63.60 -161.18   31.98\n",
      "  139        284E 284E N   CA   2292 2293  147.31  136.80                  -40.30\n",
      "  140  10622 285T 286A C   N    2306 2308  -63.36  -68.20   89.97    7.43 -134.00  112.95    6.79\n",
      "  140        286A 286A N   CA   2308 2309 -124.86  145.30                  147.00\n",
      "  141  10628 291V 292D C   N    2348 2350  -79.34  -70.90   41.14    1.63   54.50 -159.22   11.53\n",
      "  141        292D 292D N   CA   2350 2351 -169.44  150.30                   40.90\n",
      "  142  10630 293V 294P C   N    2363 2365  -64.32  -64.50   11.10    0.84  -58.70  171.29   14.61\n",
      "  142        294P 294P N   CA   2365 2366  158.30  147.20                  -30.50\n",
      "  143  10631 294P 295K C   N    2370 2372  -69.80  -70.20   10.18    0.73  -62.90  171.16   21.86\n",
      "  143        295K 295K N   CA   2372 2373  130.22  140.40                  -40.80\n",
      "  144  10632 295K 296F C   N    2379 2381  154.80 -124.20  118.27    3.23  -63.20  166.13   29.94\n",
      "  144        296F 296F N   CA   2381 2382 -130.52  143.30                  -44.30\n",
      "  145  10633 296F 297L C   N    2390 2392  -95.97 -108.50   25.61    1.22  -63.50  154.81   19.88\n",
      "  145        297L 297L N   CA   2392 2393  110.16  132.50                  -41.20\n",
      "  146  10634 297L 298S C   N    2398 2400 -178.65 -136.60   76.28    2.77  -64.10  158.93   19.69\n",
      "  146        298S 298S N   CA   2400 2401 -145.16  151.20                  -35.00\n",
      "  147  10639 302I 303D C   N    2434 2436  -64.05  -70.90    6.86    0.53  -63.30  169.26   20.94\n",
      "  147        303D 303D N   CA   2436 2437  150.75  150.30                  -40.00\n",
      "  148  10641 304V 305Q C   N    2449 2451  -67.81  -63.80   75.48   11.31  -63.80   75.48   11.31\n",
      "  148        305Q 305Q N   CA   2451 2452 -115.67  -40.30                  -40.30\n",
      "  149  10643 306I 307E C   N    2466 2468 -149.56 -117.80   49.33    1.67  -63.60  168.70   27.23\n",
      "  149        307E 307E N   CA   2468 2469  174.54  136.80                  -40.30\n",
      "  150  10644 307E 308V C   N    2475 2477  -99.52 -125.40   33.07    0.88  -62.40  169.24   19.54\n",
      "  150        308V 308V N   CA   2477 2478  122.72  143.30                  -42.40\n",
      "  151  10645 308V 309N C   N    2482 2484 -112.28 -119.90   10.50    0.31  -63.20  177.78   19.93\n",
      "  151        309N 309N N   CA   2484 2485  129.77  137.00                  -41.10\n",
      "  152  10655 318K 319L C   N    2571 2573 -167.10 -108.50   62.27    3.24  -63.50 -175.51   22.75\n",
      "  152        319L 319L N   CA   2573 2574  111.45  132.50                  -41.20\n",
      "  153  10657 320I 321Q C   N    2587 2589 -148.25 -121.10   41.81    1.41  -63.80  170.58   29.42\n",
      "  153        321Q 321Q N   CA   2589 2590  171.49  139.70                  -40.30\n",
      "  154  10660 323P 324L C   N    2610 2612  160.55 -108.50   96.38    4.30  -63.50 -154.28   36.64\n",
      "  154        324L 324L N   CA   2612 2613  164.40  132.50                  -41.20\n",
      "  155  10662 325Q 326R C   N    2627 2629  -69.30  -72.10    8.61    0.72  -63.00  168.98   23.25\n",
      "  155        326R 326R N   CA   2629 2630  150.04  141.90                  -41.10\n",
      "  156  10663 326R 327E C   N    2638 2640 -131.84 -117.80   40.36    1.71  -63.60  160.31   25.20\n",
      "  156        327E 327E N   CA   2640 2641  174.63  136.80                  -40.30\n",
      "  157  10665 328I 329S C   N    2655 2657  -74.66  -72.40   21.94    1.37  -64.10  165.91   11.43\n",
      "  157        329S 329S N   CA   2657 2658  130.58  152.40                  -35.00\n",
      "  158  10667 330P 331S C   N    2668 2670 -157.49 -136.60   61.49    3.44  -64.10  158.74   11.27\n",
      "  158        331S 331S N   CA   2670 2671   93.36  151.20                  -35.00\n",
      "  159  10668 331S 332A C   N    2674 2676  -68.52  -68.20    5.97    0.47  -62.50  167.95   27.82\n",
      "  159        332A 332A N   CA   2676 2677  151.26  145.30                  -40.90\n",
      "  160  10669 332A 333A C   N    2679 2681   98.82 -134.00  145.71    3.37  -62.50 -169.67   31.22\n",
      "  160        333A 333A N   CA   2681 2682 -141.89  147.00                  -40.90\n",
      "  161  10673 336Q 337R C   N    2711 2713   82.73   57.30   37.66    2.33  -63.00  154.50   27.81\n",
      "  161        337R 337R N   CA   2713 2714   10.22   38.00                  -41.10\n",
      "  162  10677 340V 341T C   N    2742 2744  -68.57  -78.10   16.95    0.56   55.90  157.36   10.79\n",
      "  162        341T 341T N   CA   2744 2745  135.79  149.80                   39.50\n",
      "  163  10685 348I 349P C   N    2797 2799  -68.18  -64.50   19.98    1.31  -58.70  162.94   14.24\n",
      "  163        349P 349P N   CA   2799 2800  166.84  147.20                  -30.50\n",
      "  164  10686 349P 350Y C   N    2804 2806 -123.61 -124.30   46.74    2.50  -63.50  147.28   26.34\n",
      "  164        350Y 350Y N   CA   2806 2807 -177.86  135.40                  -43.40\n",
      "  165  10687 350Y 351A C   N    2816 2818  -63.04  -68.20   28.84    2.50  -62.50  145.42   23.86\n",
      "  165        351A 351A N   CA   2818 2819  173.68  145.30                  -40.90\n",
      "  166  10688 351A 352S C   N    2821 2823  -48.95  -72.40   45.18    2.07  -64.10  149.55   11.80\n",
      "  166        352S 352S N   CA   2823 2824  113.78  152.40                  -35.00\n",
      "  167  10689 352S 353H C   N    2827 2829  156.35 -125.60  101.34    2.49  -63.20 -178.93   30.82\n",
      "  167        353H 353H N   CA   2829 2830 -156.57  138.80                  -42.30\n",
      "  168  10691 354V 355L C   N    2844 2846  -71.20  -70.70   22.71    1.74  -63.50  160.28   21.82\n",
      "  168        355L 355L N   CA   2846 2847  118.89  141.60                  -41.20\n",
      "  169  10697 360E 361A C   N    2896 2898   64.95  -68.20  137.13   12.63  -62.50 -169.94   29.09\n",
      "  169        361A 361A N   CA   2898 2899  178.11  145.30                  -40.90\n",
      "  170  10698 361A 362K C   N    2901 2903  -73.51  -70.20    5.44    0.33  -62.90  174.81   23.30\n",
      "  170        362K 362K N   CA   2903 2904  144.72  140.40                  -40.80\n",
      "  171  10699 362K 363K C   N    2910 2912  -95.01 -118.00   24.20    0.81  -62.90  175.34   21.22\n",
      "  171        363K 363K N   CA   2912 2913  131.57  139.10                  -40.80\n",
      "  172  10700 363K 364R C   N    2919 2921 -130.32 -125.20   23.11    1.21  -63.00  172.81   19.86\n",
      "  172        364R 364R N   CA   2921 2922  118.06  140.60                  -41.10\n",
      "  173  10701 364R 365R C   N    2930 2932 -141.74 -125.20   17.07    0.79  -63.00 -165.86   22.23\n",
      "  173        365R 365R N   CA   2932 2933  136.36  140.60                  -41.10\n",
      "  174  10702 365R 366Q C   N    2941 2943 -161.61 -121.10   46.74    1.44  -63.80 -175.30   32.13\n",
      "  174        366Q 366Q N   CA   2943 2944  163.02  139.70                  -40.30\n",
      "  175  10703 366Q 367R C   N    2950 2952  -79.86  -72.10   13.70    0.90  -63.00  166.57   23.65\n",
      "  175        367R 367R N   CA   2952 2953  153.19  141.90                  -41.10\n",
      "  176  10704 367R 368L C   N    2961 2963  -77.96  -70.70    7.30    0.64  -63.50  177.07   25.44\n",
      "  176        368L 368L N   CA   2963 2964  142.32  141.60                  -41.20\n",
      "  177  10705 368L 369H C   N    2969 2971 -151.25 -125.60   44.30    1.16  -63.20  167.75   26.62\n",
      "  177        369H 369H N   CA   2971 2972  174.91  138.80                  -42.30\n",
      "  178  10706 369H 370G C   N    2979 2981 -107.07  -80.20   45.39    3.03   82.20 -146.01   16.69\n",
      "  178        370G 370G N   CA   2981 2982  137.51  174.10                    8.50\n",
      "  179  10707 370G 371T C   N    2983 2985 -103.15 -124.80   25.68    1.56  -63.20  165.48   23.89\n",
      "  179        371T 371T N   CA   2985 2986  157.31  143.50                  -42.10\n",
      "  180  10708 371T 372W C   N    2990 2992 -122.94 -124.90    2.71    0.15  -63.00 -179.24   26.60\n",
      "  180        372W 372W N   CA   2992 2993  145.27  143.40                  -44.20\n",
      "  181  10709 372W 373N C   N    3004 3006  -67.21  -71.20   12.18    0.68  -63.20  172.44   21.34\n",
      "  181        373N 373N N   CA   3006 3007  131.30  142.80                  -41.10\n",
      "  182  10710 373N 374D C   N    3012 3014  -93.95  -96.50    3.88    0.16  -63.30  154.35   17.41\n",
      "  182        374D 374D N   CA   3014 3015  111.28  114.20                  -40.00\n",
      "  183  10711 374D 375D C   N    3020 3022  -98.31  -96.50    7.25    0.30  -63.30  151.29   16.86\n",
      "  183        375D 375D N   CA   3022 3023  107.18  114.20                  -40.00\n",
      "  184  10712 375D 376E C   N    3028 3030  -70.15  -69.30   11.61    0.92  -63.60  171.35   22.49\n",
      "  184        376E 376E N   CA   3030 3031  130.92  142.50                  -40.30\n",
      "  185  10714 377C 378S C   N    3043 3045 -125.76 -136.60   14.84    0.47  -64.10 -173.45   11.49\n",
      "  185        378S 378S N   CA   3045 3046  141.06  151.20                  -35.00\n",
      "  186  10715 378S 379E C   N    3049 3051  -78.97  -69.30   22.23    1.36  -63.60  157.93   21.91\n",
      "  186        379E 379E N   CA   3051 3052  162.52  142.50                  -40.30\n",
      "  187  10717 380P 381L C   N    3065 3067  -60.39  -70.70   11.37    0.85  -63.50  178.04   24.90\n",
      "  187        381L 381L N   CA   3067 3068  136.81  141.60                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   53   71  316  312  402  355  399  486  517  565\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00184.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      382     3084     3084\n",
      "openf___224_> Open           EAO00184.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     36946889   36080.945    35.235\n",
      "read_mo_297_> Segments, residues, atoms:        1      382     3084\n",
      "read_mo_298_> Segment:        1  1:A  382:A     3084\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3084     3084        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13392\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13854\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14132\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14626\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15344\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    16575\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    17868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    18897\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19140\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0704       10  10\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.5117       14  14\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.2734       17  17\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.8183       25  25\n",
      "iupac_m_487_> NH1/2 swapped:      178.8431       44  44\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.1305       47  47\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.5593       55  55\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.5936       60  60\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       99.1749       70  70\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.4966       82  82\n",
      "iupac_m_486_> OE1/2 will be swapped:      143.2578       91  91\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0889      108  108\n",
      "iupac_m_487_> NH1/2 swapped:      175.4081      110  110\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.9768      114  114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:       98.5381      129  129\n",
      "iupac_m_487_> NH1/2 swapped:      176.8549      132  132\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.0753      144  144\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.2578      146  146\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.4744      148  148\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.1176      149  149\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7317      152  152\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5719      155  155\n",
      "iupac_m_486_> OE1/2 will be swapped:     -119.7652      164  164\n",
      "iupac_m_485_> OD1/2 will be swapped:      136.5212      170  170\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.7865      185  185\n",
      "iupac_m_486_> OE1/2 will be swapped:       98.0923      203  203\n",
      "iupac_m_487_> NH1/2 swapped:      177.8134      204  204\n",
      "iupac_m_486_> OE1/2 will be swapped:      132.9195      211  211\n",
      "iupac_m_486_> OE1/2 will be swapped:     -168.9173      212  212\n",
      "iupac_m_486_> OE1/2 will be swapped:     -151.4346      219  219\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.4593      221  221\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.6508      222  222\n",
      "iupac_m_486_> OE1/2 will be swapped:      112.2719      223  223\n",
      "iupac_m_486_> OE1/2 will be swapped:      128.5167      224  224\n",
      "iupac_m_487_> NH1/2 swapped:      179.6221      227  227\n",
      "iupac_m_486_> OE1/2 will be swapped:     -129.3110      228  228\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.0612      234  234\n",
      "iupac_m_486_> OE1/2 will be swapped:     -168.5870      236  236\n",
      "iupac_m_487_> NH1/2 swapped:     -176.4305      253  253\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.6917      258  258\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.9718      259  259\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4422      269  269\n",
      "iupac_m_487_> NH1/2 swapped:      175.0849      277  277\n",
      "iupac_m_485_> OD1/2 will be swapped:     -144.5656      279  279\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.2275      284  284\n",
      "iupac_m_485_> OD1/2 will be swapped:      135.1637      303  303\n",
      "iupac_m_487_> NH1/2 swapped:     -177.0524      312  312\n",
      "iupac_m_486_> OE1/2 will be swapped:      112.5288      317  317\n",
      "iupac_m_487_> NH1/2 swapped:     -177.9710      334  334\n",
      "iupac_m_487_> NH1/2 swapped:      177.8205      364  364\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3592      365  365\n",
      "iupac_m_487_> NH1/2 swapped:      179.3719      367  367\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.5168      374  374\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.3436      376  376\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.0777      382  382\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLU:OXT  GLU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   635503\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -23696.5078\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -23696.507812\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.383999\n",
      "Sequence length          :          382\n",
      "Compactness              :     0.172298\n",
      "Native energy (pair)     :   719.757375\n",
      "Native energy (surface)  :    53.357874\n",
      "Native energy (combined) :    24.636935\n",
      "Z score (pair)           :     0.178208\n",
      "Z score (surface)        :    -0.096748\n",
      "Z score (combined)       :     0.050633\n",
      "GA341 score              :     0.051148\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7332\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3004.5254\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3122       0      0   0.004   0.004      13.880       1.000\n",
      " 2 Bond angle potential               :    4211       0      4   1.841   1.841      271.55       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1797       0    104  51.952  51.952      761.48       1.000\n",
      " 4 Stereochemical improper torsion pot:    1207       0      0   1.194   1.194      34.888       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7332       0      0   0.004   0.004      11.997       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2701       1     10   1.517   1.517      395.26       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2876       3     14   1.796   1.796      504.40       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     381       1      4   4.586   4.586      94.508       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     338       0      3  78.758  78.758      93.487       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     272       0      0  71.046  71.046      95.417       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     144       0      0  96.798  96.798      113.77       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      55       0      0  78.167  78.167      35.951       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1188       0      0   0.754   0.754      18.240       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     380     181     99  41.268 120.032      507.09       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     477       0      0   2.949   2.949      52.608       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00184.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00184.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   66001.2188\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13620 218R 358I CA  CA   1743 2873   16.42   12.91    3.51    4.54   12.91    3.51    4.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16645 285T 216M N   O    2301 1733   11.07    7.29    3.78    4.80    7.29    3.78    4.80\n",
      "    2  16646 285T 217L N   O    2301 1741    8.33    4.82    3.51    6.08    4.82    3.51    6.08\n",
      "    3  16647 285T 218R N   O    2301 1752    7.14    2.73    4.41    4.78    2.73    4.41    4.78\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10982 265R 265R CA  C    2129 2137 -154.98 -180.00   25.02    5.00 -180.00   25.02    5.00\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10338   1M   2G C   N       7    9 -104.55  -80.20   24.91    1.47   82.20 -116.67   18.68\n",
      "    1          2G   2G N   CA      9   10  179.36  174.10                    8.50\n",
      "    2  10339   2G   3R C   N      11   13 -113.57 -125.20   24.80    1.37  -63.00  164.38   25.70\n",
      "    2          3R   3R N   CA     13   14  162.50  140.60                  -41.10\n",
      "    3  10341   4I   5T C   N      30   32  -98.81  -78.10   24.54    1.06  -63.20  158.97   22.76\n",
      "    3          5T   5T N   CA     32   33  162.97  149.80                  -42.10\n",
      "    4  10347  10R  11R C   N      81   83   51.35   57.30   38.13    2.12  -63.00  163.43   29.21\n",
      "    4         11R  11R N   CA     83   84   75.66   38.00                  -41.10\n",
      "    5  10348  11R  12R C   N      92   94  -57.05  -72.10   17.79    1.17  -63.00  173.62   23.86\n",
      "    5         12R  12R N   CA     94   95  132.42  141.90                  -41.10\n",
      "    6  10349  12R  13A C   N     103  105  -91.74  -68.20   44.70    4.30  -62.50  151.06   23.50\n",
      "    6         13A  13A N   CA    105  106  107.30  145.30                  -40.90\n",
      "    7  10351  14E  15H C   N     117  119 -109.91 -125.60   81.27    3.36  -63.20  109.61   16.75\n",
      "    7         15H  15H N   CA    119  120 -141.46  138.80                  -42.30\n",
      "    8  10352  15H  16N C   N     127  129 -112.55 -119.90   28.42    1.11  -63.20  158.52   17.58\n",
      "    8         16N  16N N   CA    129  130  109.54  137.00                  -41.10\n",
      "    9  10353  16N  17E C   N     135  137 -170.64 -117.80   81.66    2.76  -63.60  161.28   27.10\n",
      "    9         17E  17E N   CA    137  138 -160.94  136.80                  -40.30\n",
      "   10  10354  17E  18G C   N     144  146  117.34   78.70   97.81    1.48  -62.40 -128.91   43.34\n",
      "   10         18G  18G N   CA    146  147  104.04 -166.10                  -41.20\n",
      "   11  10361  24K  25E C   N     193  195  -72.12  -69.30   27.40    2.19  -63.60  155.78   20.32\n",
      "   11         25E  25E N   CA    195  196  115.25  142.50                  -40.30\n",
      "   12  10363  26I  27A C   N     210  212 -142.80 -134.00   15.99    0.98  -62.50 -167.87   28.85\n",
      "   12         27A  27A N   CA    212  213  133.64  147.00                  -40.90\n",
      "   13  10364  27A  28L C   N     215  217 -143.31 -108.50   37.34    1.66  -63.50 -169.68   31.37\n",
      "   13         28L  28L N   CA    217  218  146.02  132.50                  -41.20\n",
      "   14  10365  28L  29H C   N     223  225  -76.74  -67.60    9.55    0.63  -63.20  175.44   22.79\n",
      "   14         29H  29H N   CA    225  226  142.79  140.00                  -42.30\n",
      "   15  10366  29H  30Q C   N     233  235 -122.83 -121.10   13.51    0.61  -63.80  176.75   29.12\n",
      "   15         30Q  30Q N   CA    235  236  153.09  139.70                  -40.30\n",
      "   16  10367  30Q  31Q C   N     242  244  -94.71  -73.00   64.78    3.85  -63.80  121.95   19.58\n",
      "   16         31Q  31Q N   CA    244  245 -158.26  140.70                  -40.30\n",
      "   17  10368  31Q  32D C   N     251  253  -77.80  -96.50   56.98    2.40  -63.30  101.42   11.71\n",
      "   17         32D  32D N   CA    253  254   60.38  114.20                  -40.00\n",
      "   18  10370  33I  34E C   N     267  269  -59.44  -69.30    9.93    0.71  -63.60  178.45   23.57\n",
      "   18         34E  34E N   CA    269  270  141.30  142.50                  -40.30\n",
      "   19  10371  34E  35K C   N     276  278  -81.77  -70.20   26.14    1.63  -62.90  156.50   21.40\n",
      "   19         35K  35K N   CA    278  279  163.84  140.40                  -40.80\n",
      "   20  10373  36I  37E C   N     293  295  -77.36  -69.30   26.67    1.75  -63.60  152.40   21.08\n",
      "   20         37E  37E N   CA    295  296  167.92  142.50                  -40.30\n",
      "   21  10376  39V  40G C   N     316  318  -87.17  -80.20   47.26    1.52   82.20 -135.30   17.41\n",
      "   21         40G  40G N   CA    318  319 -139.16  174.10                    8.50\n",
      "   22  10379  42A  43C C   N     333  335  -61.52  -63.00   11.37    1.29 -117.90  175.81    7.60\n",
      "   22         43C  43C N   CA    335  336  -52.37  -41.10                  141.10\n",
      "   23  10380  43C  44R C   N     339  341 -119.78 -125.20   31.64    1.40  -63.00  160.88   18.67\n",
      "   23         44R  44R N   CA    341  342  109.43  140.60                  -41.10\n",
      "   24  10382  45E  46L C   N     359  361 -126.98 -108.50   22.49    1.23  -63.50  172.96   21.37\n",
      "   24         46L  46L N   CA    361  362  119.68  132.50                  -41.20\n",
      "   25  10383  46L  47E C   N     367  369  -93.28 -117.80   73.16    3.12  -63.60  112.17   13.55\n",
      "   25         47E  47E N   CA    369  370   67.87  136.80                  -40.30\n",
      "   26  10385  48I  49L C   N     384  386 -117.24 -108.50   40.23    2.07  -63.50  156.54   25.06\n",
      "   26         49L  49L N   CA    386  387  171.77  132.50                  -41.20\n",
      "   27  10386  49L  50Y C   N     392  394  -59.37  -98.40   40.31    1.95  -63.50  161.79   25.88\n",
      "   27         50Y  50Y N   CA    394  395  118.34  128.40                  -43.40\n",
      "   28  10387  50Y  51L C   N     404  406 -151.08 -108.50   51.57    2.28  -63.50  179.94   30.38\n",
      "   28         51L  51L N   CA    406  407  161.60  132.50                  -41.20\n",
      "   29  10389  52C  53N C   N     418  420  -81.24  -71.20   26.10    1.94  -63.20  160.82   19.17\n",
      "   29         53N  53N N   CA    420  421  118.71  142.80                  -41.10\n",
      "   30  10390  53N  54N C   N     426  428 -159.18 -119.90   55.05    1.61  -63.20  172.50   26.84\n",
      "   30         54N  54N N   CA    428  429  175.57  137.00                  -41.10\n",
      "   31  10391  54N  55Y C   N     434  436 -106.92  -98.40    9.21    0.47  -63.50  173.82   25.51\n",
      "   31         55Y  55Y N   CA    436  437  124.91  128.40                  -43.40\n",
      "   32  10396  59I  60E C   N     479  481 -140.21 -117.80   34.82    1.18  -63.60  174.01   27.48\n",
      "   32         60E  60E N   CA    481  482  163.46  136.80                  -40.30\n",
      "   33  10397  60E  61G C   N     488  490  105.76   78.70   74.15    1.16   82.20  118.72    6.92\n",
      "   33         61G  61G N   CA    490  491  124.86 -166.10                    8.50\n",
      "   34  10398  61G  62L C   N     492  494 -146.15 -108.50   89.13    4.34  -63.50  134.02   23.58\n",
      "   34         62L  62L N   CA    494  495 -146.71  132.50                  -41.20\n",
      "   35  10400  63Q  64H C   N     509  511 -118.11  -63.20   55.08    8.51  -63.20   55.08    8.51\n",
      "   35         64H  64H N   CA    511  512  -37.96  -42.30                  -42.30\n",
      "   36  10403  66K  67R C   N     536  538 -158.83 -125.20   67.87    2.41  -63.00  153.07   26.87\n",
      "   36         67R  67R N   CA    538  539 -160.45  140.60                  -41.10\n",
      "   37  10404  67R  68L C   N     547  549  -86.69  -70.70   26.76    1.66  -63.50  157.46   23.26\n",
      "   37         68L  68L N   CA    549  550  163.06  141.60                  -41.20\n",
      "   38  10405  68L  69K C   N     555  557  -76.62  -70.20   50.56    3.81   56.60  142.88   13.59\n",
      "   38         69K  69K N   CA    557  558   90.24  140.40                   38.60\n",
      "   39  10412  75V  76N C   N     612  614  -86.05  -71.20   15.03    1.14  -63.20  179.87   23.77\n",
      "   39         76N  76N N   CA    614  615  140.49  142.80                  -41.10\n",
      "   40  10413  76N  77N C   N     620  622 -154.48 -119.90   62.60    1.98  -63.20  158.62   24.82\n",
      "   40         77N  77N N   CA    622  623 -170.82  137.00                  -41.10\n",
      "   41  10415  78I  79A C   N     636  638 -175.86 -134.00   44.31    1.04  -62.50 -165.89   37.78\n",
      "   41         79A  79A N   CA    638  639  161.53  147.00                  -40.90\n",
      "   42  10416  79A  80C C   N     641  643  -65.10  -63.00    2.76    0.48 -117.90 -176.25    7.89\n",
      "   42         80C  80C N   CA    643  644  -42.90  -41.10                  141.10\n",
      "   43  10418  81I  82E C   N     655  657 -159.18 -117.80   61.65    2.04  -63.60  167.20   27.47\n",
      "   43         82E  82E N   CA    657  658 -177.49  136.80                  -40.30\n",
      "   44  10421  84L  85E C   N     676  678   98.63 -117.80  152.89    4.58  -63.60 -151.90   25.65\n",
      "   44         85E  85E N   CA    678  679 -170.64  136.80                  -40.30\n",
      "   45  10422  85E  86G C   N     685  687   99.96   82.20   29.26    0.82  -62.40  164.51   30.31\n",
      "   45         86G  86G N   CA    687  688  -14.75    8.50                  -41.20\n",
      "   46  10423  86G  87C C   N     689  691  -60.09  -63.00   17.37    1.94 -117.90  170.76    7.41\n",
      "   46         87C  87C N   CA    691  692  -58.22  -41.10                  141.10\n",
      "   47  10425  88E  89S C   N     704  706  168.64 -136.60   57.15    1.89  -64.10 -157.56   23.84\n",
      "   47         89S  89S N   CA    706  707  167.56  151.20                  -35.00\n",
      "   48  10426  89S  90L C   N     710  712 -125.48 -108.50   17.24    0.79  -63.50 -172.76   23.29\n",
      "   48         90L  90L N   CA    712  713  135.48  132.50                  -41.20\n",
      "   49  10427  90L  91E C   N     718  720 -143.43 -117.80   28.73    1.38  -63.60 -177.49   21.20\n",
      "   49         91E  91E N   CA    720  721  123.83  136.80                  -40.30\n",
      "   50  10428  91E  92R C   N     727  729   56.28   57.30    8.12    0.61  -63.00  138.84   25.31\n",
      "   50         92R  92R N   CA    729  730   29.94   38.00                  -41.10\n",
      "   51  10429  92R  93L C   N     738  740 -133.56 -108.50   54.90    2.65  -63.50  154.27   25.75\n",
      "   51         93L  93L N   CA    740  741 -178.65  132.50                  -41.20\n",
      "   52  10430  93L  94D C   N     746  748  -68.67  -70.90    2.29    0.19   54.50  165.07   10.76\n",
      "   52         94D  94D N   CA    748  749  150.80  150.30                   40.90\n",
      "   53  10433  96T  97L C   N     769  771  -58.82  -70.70   20.62    1.28  -63.50  166.02   23.32\n",
      "   53         97L  97L N   CA    771  772  124.75  141.60                  -41.20\n",
      "   54  10434  97L  98N C   N     777  779  123.59 -119.90  117.12    4.56  -63.20 -117.31   39.27\n",
      "   54         98N  98N N   CA    779  780  148.92  137.00                  -41.10\n",
      "   55  10439 102E 103M C   N     817  819 -131.64 -125.60   28.07    1.20  -63.40  166.23   30.20\n",
      "   55        103M 103M N   CA    819  820  167.92  140.50                  -40.50\n",
      "   56  10440 103M 104T C   N     825  827  -90.07  -78.10   27.79    0.91  -63.20  145.52   20.45\n",
      "   56        104T 104T N   CA    827  828  174.88  149.80                  -42.10\n",
      "   57  10443 106V 107Q C   N     845  847  -49.96  -73.00   26.27    2.04  -63.80  166.94   23.84\n",
      "   57        107Q 107Q N   CA    847  848  153.33  140.70                  -40.30\n",
      "   58  10445 108R 109L C   N     865  867  -70.72  -70.70    0.27    0.02  -63.50  177.08   25.01\n",
      "   58        109L 109L N   CA    867  868  141.87  141.60                  -41.20\n",
      "   59  10446 109L 110R C   N     873  875 -106.61 -125.20   32.06    1.07  -63.00  161.58   19.29\n",
      "   59        110R 110R N   CA    875  876  114.48  140.60                  -41.10\n",
      "   60  10447 110R 111A C   N     884  886  -81.45  -68.20   21.92    1.38  -62.50  157.47   26.78\n",
      "   60        111A 111A N   CA    886  887  162.77  145.30                  -40.90\n",
      "   61  10448 111A 112N C   N     889  891  -92.88 -119.90   46.54    1.45  -63.20  143.32   16.45\n",
      "   61        112N 112N N   CA    891  892   99.11  137.00                  -41.10\n",
      "   62  10459 122G 123N C   N     977  979 -103.26 -119.90   22.30    1.20  -63.20  171.79   23.76\n",
      "   62        123N 123N N   CA    979  980  151.85  137.00                  -41.10\n",
      "   63  10463 126T 127K C   N    1005 1007  -70.38  -70.20   22.67    1.64  -62.90  156.31   20.72\n",
      "   63        127K 127K N   CA   1007 1008  163.07  140.40                  -40.80\n",
      "   64  10465 128I 129E C   N    1022 1024 -166.34 -117.80   55.97    1.64  -63.60 -174.00   30.40\n",
      "   64        129E 129E N   CA   1024 1025  164.66  136.80                  -40.30\n",
      "   65  10466 129E 130G C   N    1031 1033   62.06   82.20   61.39    2.01  -62.40  164.58   30.76\n",
      "   65        130G 130G N   CA   1033 1034   66.49    8.50                  -41.20\n",
      "   66  10469 132R 133A C   N    1058 1060  173.99 -134.00   61.70    1.48  -62.50 -174.13   36.83\n",
      "   66        133A 133A N   CA   1060 1061 -179.80  147.00                  -40.90\n",
      "   67  10470 133A 134Y C   N    1063 1065 -115.35 -124.30    8.98    0.65  -63.50 -174.65   26.97\n",
      "   67        134Y 134Y N   CA   1065 1066  134.55  135.40                  -43.40\n",
      "   68  10473 136I 137H C   N    1090 1092 -104.32 -125.60   21.56    0.82  -63.20 -179.83   25.21\n",
      "   68        137H 137H N   CA   1092 1093  142.28  138.80                  -42.30\n",
      "   69  10474 137H 138A C   N    1100 1102  -90.50  -68.20   38.86    3.77  -62.50  156.89   24.49\n",
      "   69        138A 138A N   CA   1102 1103  113.48  145.30                  -40.90\n",
      "   70  10475 138A 139L C   N    1105 1107 -152.47 -108.50   56.97    2.54  -63.50  174.46   29.69\n",
      "   70        139L 139L N   CA   1107 1108  168.72  132.50                  -41.20\n",
      "   71  10477 140P 141Q C   N    1120 1122   -9.46  -73.00   71.78    4.44  -63.80  157.29   26.01\n",
      "   71        141Q 141Q N   CA   1122 1123  107.31  140.70                  -40.30\n",
      "   72  10478 141Q 142L C   N    1129 1131 -113.02 -108.50    8.39    0.48  -63.50  173.84   21.85\n",
      "   72        142L 142L N   CA   1131 1132  125.44  132.50                  -41.20\n",
      "   73  10479 142L 143R C   N    1137 1139  -86.59  -72.10   16.98    1.12  -63.00  169.80   24.56\n",
      "   73        143R 143R N   CA   1139 1140  150.75  141.90                  -41.10\n",
      "   74  10480 143R 144D C   N    1148 1150  -88.66  -96.50   30.81    1.30  -63.30  126.96   14.32\n",
      "   74        144D 144D N   CA   1150 1151   84.41  114.20                  -40.00\n",
      "   75  10483 146D 147G C   N    1172 1174  106.48   82.20   32.92    1.08  -62.40  171.10   31.53\n",
      "   75        147G 147G N   CA   1174 1175  -13.73    8.50                  -41.20\n",
      "   76  10487 150V 151V C   N    1201 1203  -55.41  -62.40   15.68    2.41 -125.40 -174.61    7.15\n",
      "   76        151V 151V N   CA   1203 1204  -28.37  -42.40                  143.30\n",
      "   77  10495 158A 159R C   N    1267 1269 -128.38 -125.20   20.96    0.94  -63.00  170.61   27.53\n",
      "   77        159R 159R N   CA   1269 1270  161.31  140.60                  -41.10\n",
      "   78  10496 159R 160Q C   N    1278 1280 -132.22 -121.10   31.35    1.62  -63.80  165.50   21.51\n",
      "   78        160Q 160Q N   CA   1280 1281  110.39  139.70                  -40.30\n",
      "   79  10498 161C 162N C   N    1293 1295 -121.29 -119.90   35.23    1.63  -63.20  154.25   16.83\n",
      "   79        162N 162N N   CA   1295 1296  101.80  137.00                  -41.10\n",
      "   80  10500 163S 164E C   N    1307 1309 -150.95 -117.80   36.66    1.74  -63.60 -176.45   21.24\n",
      "   80        164E 164E N   CA   1309 1310  121.13  136.80                  -40.30\n",
      "   81  10502 165V 166A C   N    1323 1325 -141.77 -134.00    9.32    0.50  -62.50 -165.82   36.07\n",
      "   81        166A 166A N   CA   1325 1326  141.84  147.00                  -40.90\n",
      "   82  10503 166A 167A C   N    1328 1330  167.73 -134.00   99.66    3.61  -62.50  158.64   32.18\n",
      "   82        167A 167A N   CA   1330 1331 -132.15  147.00                  -40.90\n",
      "   83  10506 169V 170D C   N    1347 1349  178.20  -63.30  120.21   19.89  -96.50 -165.83    8.05\n",
      "   83        170D 170D N   CA   1349 1350  -60.23  -40.00                  114.20\n",
      "   84  10508 171E 172A C   N    1364 1366  112.88 -134.00  113.95    3.28  -68.20  179.58   14.69\n",
      "   84        172A 172A N   CA   1366 1367  160.66  147.00                  145.30\n",
      "   85  10509 172A 173A C   N    1369 1371  -68.49  -68.20   29.39    2.36  -62.50  144.54   23.98\n",
      "   85        173A 173A N   CA   1371 1372  174.69  145.30                  -40.90\n",
      "   86  10510 173A 174L C   N    1374 1376  -63.85  -70.70   45.07    3.16  -63.50  138.26   19.18\n",
      "   86        174L 174L N   CA   1376 1377   97.06  141.60                  -41.20\n",
      "   87  10511 174L 175R C   N    1382 1384 -169.97 -125.20   59.83    1.79  -63.00  175.08   30.60\n",
      "   87        175R 175R N   CA   1384 1385 -179.70  140.60                  -41.10\n",
      "   88  10513 176V 177R C   N    1400 1402 -128.96 -125.20    5.20    0.28   57.30 -160.03   24.76\n",
      "   88        177R 177R N   CA   1402 1403  137.01  140.60                   38.00\n",
      "   89  10519 182I 183K C   N    1453 1455 -140.95 -118.00   38.59    1.45  -62.90  168.26   26.52\n",
      "   89        183K 183K N   CA   1455 1456  170.13  139.10                  -40.80\n",
      "   90  10520 183K 184A C   N    1462 1464  -98.30  -68.20   31.46    2.94  -62.50 -179.39   28.06\n",
      "   90        184A 184A N   CA   1464 1465  136.13  145.30                  -40.90\n",
      "   91  10521 184A 185E C   N    1467 1469 -155.92 -117.80   85.82    3.40  -63.60  140.57   23.58\n",
      "   91        185E 185E N   CA   1469 1470 -146.31  136.80                  -40.30\n",
      "   92  10522 185E 186M C   N    1476 1478  -62.83  -73.00   29.44    2.17  -63.40  148.88   22.18\n",
      "   92        186M 186M N   CA   1478 1479  170.62  143.00                  -40.50\n",
      "   93  10524 187I 188A C   N    1492 1494 -133.36 -134.00   11.68    0.68  -62.50  175.40   32.54\n",
      "   93        188A 188A N   CA   1494 1495  158.66  147.00                  -40.90\n",
      "   94  10526 189Q 190G C   N    1506 1508  175.68 -167.20   55.19    2.27   82.20  147.13   11.04\n",
      "   94        190G 190G N   CA   1508 1509  122.13  174.60                    8.50\n",
      "   95  10530 193V 194F C   N    1533 1535   94.05 -124.20  142.20    5.22  -63.20 -134.90   25.91\n",
      "   95        194F 194F N   CA   1535 1536  154.63  143.30                  -44.30\n",
      "   96  10533 196P 197R C   N    1558 1560 -144.70 -125.20   53.57    2.10  -63.00  152.19   26.00\n",
      "   96        197R 197R N   CA   1560 1561 -169.50  140.60                  -41.10\n",
      "   97  10535 198Y 199N C   N    1581 1583  -73.99  -71.20    4.62    0.37  -63.20 -179.88   23.12\n",
      "   97        199N 199N N   CA   1583 1584  139.11  142.80                  -41.10\n",
      "   98  10536 199N 200E C   N    1589 1591  -80.66  -69.30   27.00    2.39  -63.60  159.22   20.34\n",
      "   98        200E 200E N   CA   1591 1592  118.00  142.50                  -40.30\n",
      "   99  10537 200E 201K C   N    1598 1600 -130.30 -118.00   73.75    3.39  -62.90  126.79   20.47\n",
      "   99        201K 201K N   CA   1600 1601 -148.19  139.10                  -40.80\n",
      "  100  10538 201K 202G C   N    1607 1609   72.75   82.20  104.98    4.62  -62.40 -154.91   37.61\n",
      "  100        202G 202G N   CA   1609 1610  113.06    8.50                  -41.20\n",
      "  101  10540 203E 204R C   N    1620 1622   69.27   57.30   21.46    1.14  -63.00  145.78   26.48\n",
      "  101        204R 204R N   CA   1622 1623   20.19   38.00                  -41.10\n",
      "  102  10543 206Y 207G C   N    1650 1652  -58.29  -62.40    4.19    0.78   82.20  148.76   11.32\n",
      "  102        207G 207G N   CA   1652 1653  -40.39  -41.20                    8.50\n",
      "  103  10544 207G 208H C   N    1654 1656 -102.82  -63.20   70.87   11.38  -63.20   70.87   11.38\n",
      "  103        208H 208H N   CA   1656 1657 -101.06  -42.30                  -42.30\n",
      "  104  10560 223E 224E C   N    1796 1798 -160.12 -117.80   52.97    1.59  -63.60  179.25   29.18\n",
      "  104        224E 224E N   CA   1798 1799  168.65  136.80                  -40.30\n",
      "  105  10561 224E 225A C   N    1805 1807 -126.58 -134.00   12.74    0.77  -62.50  173.98   31.94\n",
      "  105        225A 225A N   CA   1807 1808  157.36  147.00                  -40.90\n",
      "  106  10562 225A 226E C   N    1810 1812   64.27  -69.30  148.62   13.18  -63.60  170.00   20.67\n",
      "  106        226E 226E N   CA   1812 1813 -152.33  142.50                  -40.30\n",
      "  107  10563 226E 227R C   N    1819 1821  -60.62  -72.10   16.35    1.06  -63.00  171.38   23.31\n",
      "  107        227R 227R N   CA   1821 1822  130.27  141.90                  -41.10\n",
      "  108  10564 227R 228E C   N    1830 1832 -118.51 -117.80   41.84    2.09   54.60 -139.72   15.33\n",
      "  108        228E 228E N   CA   1832 1833  178.63  136.80                   42.40\n",
      "  109  10566 229K 230K C   N    1848 1850 -127.79 -118.00   45.96    2.34  -62.90  149.78   17.21\n",
      "  109        230K 230K N   CA   1850 1851   94.19  139.10                  -40.80\n",
      "  110  10568 231L 232Q C   N    1865 1867  -94.32 -121.10   69.94    3.63  -63.80  119.36   19.18\n",
      "  110        232Q 232Q N   CA   1867 1868 -155.69  139.70                  -40.30\n",
      "  111  10569 232Q 233K C   N    1874 1876  -79.67  -70.20   14.87    0.89  -62.90  168.18   22.79\n",
      "  111        233K 233K N   CA   1876 1877  151.86  140.40                  -40.80\n",
      "  112  10570 233K 234E C   N    1883 1885 -113.16 -117.80   38.27    2.00  -63.60  153.15   23.22\n",
      "  112        234E 234E N   CA   1885 1886  174.78  136.80                  -40.30\n",
      "  113  10573 236E 237T C   N    1912 1914 -148.04 -124.80   35.24    1.11  -63.20  170.51   27.31\n",
      "  113        237T 237T N   CA   1914 1915  169.99  143.50                  -42.10\n",
      "  114  10574 237T 238G C   N    1919 1921  -75.17  -80.20    7.48    0.52   82.20 -127.51    6.96\n",
      "  114        238G 238G N   CA   1921 1922  179.63  174.10                    8.50\n",
      "  115  10581 244S 245A C   N    1965 1967  148.46 -134.00   93.47    2.29  -62.50 -168.72   38.64\n",
      "  115        245A 245A N   CA   1967 1968 -160.80  147.00                  -40.90\n",
      "  116  10584 247V 248N C   N    1986 1988   42.13   55.90   14.94    1.31  -63.20  136.23   22.26\n",
      "  116        248N 248N N   CA   1988 1989   45.30   39.50                  -41.10\n",
      "  117  10589 252Q 253R C   N    2030 2032 -134.12 -125.20   22.53    0.86  -63.00  172.92   28.22\n",
      "  117        253R 253R N   CA   2032 2033  161.29  140.60                  -41.10\n",
      "  118  10591 254L 255T C   N    2049 2051  -97.83  -78.10   86.10    3.10  -63.20   91.13   13.97\n",
      "  118        255T 255T N   CA   2051 2052 -126.39  149.80                  -42.10\n",
      "  119  10592 255T 256A C   N    2056 2058  -62.51  -68.20   12.20    1.16  -62.50  163.01   26.72\n",
      "  119        256A 256A N   CA   2058 2059  156.09  145.30                  -40.90\n",
      "  120  10593 256A 257E C   N    2061 2063  -61.54  -69.30   21.37    1.35  -63.60  162.90   21.83\n",
      "  120        257E 257E N   CA   2063 2064  122.59  142.50                  -40.30\n",
      "  121  10594 257E 258E C   N    2070 2072 -163.93 -117.80   48.22    2.13  -63.60 -168.55   22.11\n",
      "  121        258E 258E N   CA   2072 2073  122.75  136.80                  -40.30\n",
      "  122  10596 259E 260I C   N    2088 2090  -71.50  -97.30   26.10    1.20  -63.40  167.06   27.10\n",
      "  122        260I 260I N   CA   2090 2091  123.26  127.20                  -43.60\n",
      "  123  10597 260I 261A C   N    2096 2098  147.39 -134.00   93.34    2.24  -62.50 -166.70   39.03\n",
      "  123        261A 261A N   CA   2098 2099 -162.68  147.00                  -40.90\n",
      "  124  10598 261A 262K C   N    2101 2103  -93.69 -118.00   91.32    4.04  -62.90   96.90   11.30\n",
      "  124        262K 262K N   CA   2103 2104   51.08  139.10                  -40.80\n",
      "  125  10599 262K 263Y C   N    2110 2112 -118.36 -124.30    6.10    0.41  -63.50 -174.29   26.91\n",
      "  125        263Y 263Y N   CA   2112 2113  134.02  135.40                  -43.40\n",
      "  126  10600 263Y 264G C   N    2122 2124   75.61   78.70    9.27    0.39   82.20  176.78    8.25\n",
      "  126        264G 264G N   CA   2124 2125 -174.84 -166.10                    8.50\n",
      "  127  10604 267L 268L C   N    2153 2155  -89.07  -70.70   32.21    1.99  -63.50  152.89   22.77\n",
      "  127        268L 268L N   CA   2155 2156  168.06  141.60                  -41.20\n",
      "  128  10605 268L 269R C   N    2161 2163  -82.52  -72.10   17.62    1.15  -63.00  163.95   23.49\n",
      "  128        269R 269R N   CA   2163 2164  156.12  141.90                  -41.10\n",
      "  129  10606 269R 270N C   N    2172 2174 -133.68 -119.90   24.95    0.79  -63.20  175.85   25.98\n",
      "  129        270N 270N N   CA   2174 2175  157.79  137.00                  -41.10\n",
      "  130  10607 270N 271E C   N    2180 2182  -72.82  -69.30    4.47    0.25  -63.60  174.69   23.80\n",
      "  130        271E 271E N   CA   2182 2183  145.25  142.50                  -40.30\n",
      "  131  10609 272P 273K C   N    2196 2198 -118.95 -118.00   12.84    0.64  -62.90  176.25   20.56\n",
      "  131        273K 273K N   CA   2198 2199  126.30  139.10                  -40.80\n",
      "  132  10610 273K 274F C   N    2205 2207   60.99  -71.40  133.76    9.66  -63.20 -160.68   22.56\n",
      "  132        274F 274F N   CA   2207 2208  159.79  140.70                  -44.30\n",
      "  133  10612 275P 276M C   N    2223 2225 -162.69 -125.60   38.41    1.28  -63.40 -163.99   36.94\n",
      "  133        276M 276M N   CA   2225 2226  150.50  140.50                  -40.50\n",
      "  134  10613 276M 277R C   N    2231 2233 -140.80 -125.20   16.37    0.78  -63.00 -166.90   22.13\n",
      "  134        277R 277R N   CA   2233 2234  135.64  140.60                  -41.10\n",
      "  135  10614 277R 278L C   N    2242 2244 -120.93 -108.50   13.16    0.68  -63.50  178.85   22.29\n",
      "  135        278L 278L N   CA   2244 2245  128.18  132.50                  -41.20\n",
      "  136  10616 279D 280Q C   N    2258 2260  -80.36  -73.00    7.39    0.54  -63.80 -179.58   27.41\n",
      "  136        280Q 280Q N   CA   2260 2261  140.04  140.70                  -40.30\n",
      "  137  10617 280Q 281E C   N    2267 2269 -114.74 -117.80    7.56    0.31  -63.60  177.71   21.32\n",
      "  137        281E 281E N   CA   2269 2270  129.89  136.80                  -40.30\n",
      "  138  10618 281E 282S C   N    2276 2278 -113.76 -136.60   22.88    0.94  -64.10  179.49   16.44\n",
      "  138        282S 282S N   CA   2278 2279  152.52  151.20                  -35.00\n",
      "  139  10619 282S 283D C   N    2282 2284 -116.77  -70.90   48.66    3.01  -63.30  162.53   23.52\n",
      "  139        283D 283D N   CA   2284 2285  166.52  150.30                  -40.00\n",
      "  140  10620 283D 284E C   N    2290 2292  176.30 -117.80   81.71    2.44  -63.60 -179.61   30.32\n",
      "  140        284E 284E N   CA   2292 2293 -174.90  136.80                  -40.30\n",
      "  141  10628 291V 292D C   N    2348 2350  -64.26  -63.30   18.52    2.23   54.50  134.16   17.42\n",
      "  141        292D 292D N   CA   2350 2351  -21.50  -40.00                   40.90\n",
      "  142  10631 294P 295K C   N    2370 2372   20.13   56.60   85.53    4.30  -62.90  177.40   28.00\n",
      "  142        295K 295K N   CA   2372 2373  115.97   38.60                  -40.80\n",
      "  143  10636 299T 300T C   N    2411 2413 -111.60 -124.80   58.84    2.15  -63.20  137.09   14.96\n",
      "  143        300T 300T N   CA   2413 2414   86.15  143.50                  -42.10\n",
      "  144  10639 302I 303D C   N    2434 2436  -86.25  -70.90   21.23    0.88  -63.30  156.73   20.80\n",
      "  144        303D 303D N   CA   2436 2437  164.96  150.30                  -40.00\n",
      "  145  10640 303D 304V C   N    2442 2444  -66.02  -62.40    4.07    0.70 -125.40 -177.64   10.19\n",
      "  145        304V 304V N   CA   2444 2445  -44.28  -42.40                  143.30\n",
      "  146  10643 306I 307E C   N    2466 2468  -72.81  -69.30   31.54    2.28  -63.60  146.15   20.00\n",
      "  146        307E 307E N   CA   2468 2469  173.84  142.50                  -40.30\n",
      "  147  10644 307E 308V C   N    2475 2477  -52.58 -125.40   79.34    2.23  -62.40  154.52   20.14\n",
      "  147        308V 308V N   CA   2477 2478  111.81  143.30                  -42.40\n",
      "  148  10655 318K 319L C   N    2571 2573 -158.55 -108.50   52.76    2.36  -63.50 -165.57   32.85\n",
      "  148        319L 319L N   CA   2573 2574  149.19  132.50                  -41.20\n",
      "  149  10657 320I 321Q C   N    2587 2589 -110.88 -121.10   19.60    0.72  -63.80  169.93   22.80\n",
      "  149        321Q 321Q N   CA   2589 2590  122.97  139.70                  -40.30\n",
      "  150  10660 323P 324L C   N    2610 2612  155.84 -108.50  101.14    4.51  -63.50 -151.85   37.21\n",
      "  150        324L 324L N   CA   2612 2613  165.36  132.50                  -41.20\n",
      "  151  10662 325Q 326R C   N    2627 2629  -56.45  -72.10   15.89    1.27  -63.00  174.36   23.12\n",
      "  151        326R 326R N   CA   2629 2630  144.67  141.90                  -41.10\n",
      "  152  10663 326R 327E C   N    2638 2640 -125.87 -117.80   18.83    0.76  -63.60  177.19   27.14\n",
      "  152        327E 327E N   CA   2640 2641  153.81  136.80                  -40.30\n",
      "  153  10665 328I 329S C   N    2655 2657 -134.65 -136.60    7.21    0.40  -64.10 -178.84   18.14\n",
      "  153        329S 329S N   CA   2657 2658  158.14  151.20                  -35.00\n",
      "  154  10666 329S 330P C   N    2661 2663  -70.65  -64.50    6.35    0.58  -58.70  179.68   13.90\n",
      "  154        330P 330P N   CA   2663 2664  148.78  147.20                  -30.50\n",
      "  155  10667 330P 331S C   N    2668 2670 -155.29 -136.60   38.28    1.46  -64.10  167.41   18.71\n",
      "  155        331S 331S N   CA   2670 2671 -175.40  151.20                  -35.00\n",
      "  156  10668 331S 332A C   N    2674 2676 -178.73 -134.00  100.02    4.24  -62.50  142.62   28.92\n",
      "  156        332A 332A N   CA   2676 2677 -123.55  147.00                  -40.90\n",
      "  157  10669 332A 333A C   N    2679 2681  -78.30  -68.20   63.24    5.45  -62.50  124.77   19.73\n",
      "  157        333A 333A N   CA   2681 2682   82.87  145.30                  -40.90\n",
      "  158  10670 333A 334R C   N    2684 2686   46.19   57.30   24.27    1.19  -63.00  148.52   26.74\n",
      "  158        334R 334R N   CA   2686 2687   59.58   38.00                  -41.10\n",
      "  159  10673 336Q 337R C   N    2711 2713 -142.33 -125.20   42.70    2.33  -63.00  163.17   18.48\n",
      "  159        337R 337R N   CA   2713 2714  101.48  140.60                  -41.10\n",
      "  160  10680 343A 344L C   N    2758 2760  -59.03  -70.70   14.33    0.98  -63.50  174.54   24.49\n",
      "  160        344L 344L N   CA   2760 2761  133.28  141.60                  -41.20\n",
      "  161  10686 349P 350Y C   N    2804 2806  -74.75  -98.40   24.12    0.99  -63.50  176.91   27.50\n",
      "  161        350Y 350Y N   CA   2806 2807  133.15  128.40                  -43.40\n",
      "  162  10687 350Y 351A C   N    2816 2818  -90.47  -68.20   57.39    5.34  -62.50  136.21   21.13\n",
      "  162        351A 351A N   CA   2818 2819   92.41  145.30                  -40.90\n",
      "  163  10691 354V 355L C   N    2844 2846  -68.27  -70.70   46.47    3.43  -63.50  136.48   18.68\n",
      "  163        355L 355L N   CA   2846 2847   95.20  141.60                  -41.20\n",
      "  164  10697 360E 361A C   N    2896 2898  -49.82  -68.20   21.58    1.44  -62.50  175.35   29.37\n",
      "  164        361A 361A N   CA   2898 2899  133.99  145.30                  -40.90\n",
      "  165  10698 361A 362K C   N    2901 2903   47.29   56.60   26.33    1.35  -62.90  151.54   25.90\n",
      "  165        362K 362K N   CA   2903 2904   63.23   38.60                  -40.80\n",
      "  166  10699 362K 363K C   N    2910 2912  -63.04  -70.20    8.84    0.74  -62.90  173.62   22.55\n",
      "  166        363K 363K N   CA   2912 2913  145.58  140.40                  -40.80\n",
      "  167  10700 363K 364R C   N    2919 2921 -115.82 -125.20   12.18    0.66  -63.00  178.53   27.77\n",
      "  167        364R 364R N   CA   2921 2922  148.36  140.60                  -41.10\n",
      "  168  10701 364R 365R C   N    2930 2932 -139.72 -125.20   26.95    0.93  -63.00  173.49   28.66\n",
      "  168        365R 365R N   CA   2932 2933  163.30  140.60                  -41.10\n",
      "  169  10702 365R 366Q C   N    2941 2943  -79.79  -73.00   30.63    2.14  -63.80  151.98   21.54\n",
      "  169        366Q 366Q N   CA   2943 2944  110.83  140.70                  -40.30\n",
      "  170  10703 366Q 367R C   N    2950 2952 -162.26 -125.20   47.68    1.41  -63.00  178.46   30.68\n",
      "  170        367R 367R N   CA   2952 2953  170.60  140.60                  -41.10\n",
      "  171  10704 367R 368L C   N    2961 2963 -105.66 -108.50    6.44    0.31  -63.50  173.14   22.00\n",
      "  171        368L 368L N   CA   2963 2964  126.73  132.50                  -41.20\n",
      "  172  10705 368L 369H C   N    2969 2971 -151.25 -125.60   35.03    0.86  -63.20  178.31   27.98\n",
      "  172        369H 369H N   CA   2971 2972  162.65  138.80                  -42.30\n",
      "  173  10706 369H 370G C   N    2979 2981 -142.84 -167.20   26.17    0.38   82.20 -153.34   15.60\n",
      "  173        370G 370G N   CA   2981 2982  165.02  174.60                    8.50\n",
      "  174  10707 370G 371T C   N    2983 2985  -68.53  -78.10   20.00    0.65  -63.20  174.42   22.02\n",
      "  174        371T 371T N   CA   2985 2986  132.24  149.80                  -42.10\n",
      "  175  10708 371T 372W C   N    2990 2992 -114.46 -124.90   17.80    0.60  -63.00 -179.34   20.21\n",
      "  175        372W 372W N   CA   2992 2993  128.98  143.40                  -44.20\n",
      "  176  10709 372W 373N C   N    3004 3006 -115.96 -119.90   18.51    0.74  -63.20  168.49   18.68\n",
      "  176        373N 373N N   CA   3006 3007  118.91  137.00                  -41.10\n",
      "  177  10710 373N 374D C   N    3012 3014 -113.87  -96.50   80.63    3.37  -63.30   90.84    9.71\n",
      "  177        374D 374D N   CA   3014 3015   35.47  114.20                  -40.00\n",
      "  178  10712 375D 376E C   N    3028 3030 -127.23 -117.80   18.04    0.67  -63.60  179.20   27.48\n",
      "  178        376E 376E N   CA   3030 3031  152.17  136.80                  -40.30\n",
      "  179  10714 377C 378S C   N    3043 3045 -138.07 -136.60    3.46    0.20  -64.10 -168.23   19.15\n",
      "  179        378S 378S N   CA   3045 3046  148.06  151.20                  -35.00\n",
      "  180  10715 378S 379E C   N    3049 3051 -134.26 -117.80   30.12    1.10  -63.60  172.79   27.00\n",
      "  180        379E 379E N   CA   3051 3052  162.02  136.80                  -40.30\n",
      "  181  10717 380P 381L C   N    3065 3067  -62.39  -70.70    9.26    0.68  -63.50  178.71   24.88\n",
      "  181        381L 381L N   CA   3067 3068  137.51  141.60                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   45   66  325  296  376  359  383  445  499  570\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00184.B99990004.pdb\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "wrpdb___568_> Residues, atoms, selected atoms:      382     3084     3084\n",
      "openf___224_> Open           EAO00184.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     36946889   36080.945    35.235\n",
      "read_mo_297_> Segments, residues, atoms:        1      382     3084\n",
      "read_mo_298_> Segment:        1  1:A  382:A     3084\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3084     3084        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13392\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    13854\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14132\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    14626\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15344\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    15816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    16575\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    17868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    18897\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19140\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3084    19149\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9407        3  3\n",
      "iupac_m_487_> NH1/2 swapped:      179.0406       10  10\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.4890       14  14\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.0518       47  47\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.7149       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.0874       58  58\n",
      "iupac_m_486_> OE1/2 will be swapped:      141.8325       91  91\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0779       92  92\n",
      "iupac_m_486_> OE1/2 will be swapped:      164.7270      102  102\n",
      "iupac_m_487_> NH1/2 swapped:      179.2514      108  108\n",
      "iupac_m_487_> NH1/2 swapped:      177.4066      110  110\n",
      "iupac_m_487_> NH1/2 swapped:      179.2491      132  132\n",
      "iupac_m_487_> NH1/2 swapped:      178.8665      143  143\n",
      "iupac_m_485_> OD1/2 will be swapped:     -163.3055      144  144\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.9441      146  146\n",
      "iupac_m_487_> NH1/2 swapped:      177.0487      152  152\n",
      "iupac_m_486_> OE1/2 will be swapped:      120.1705      164  164\n",
      "iupac_m_486_> OE1/2 will be swapped:      155.3342      180  180\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5799      181  181\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.9367      185  185\n",
      "iupac_m_485_> OD1/2 will be swapped:      143.9530      192  192\n",
      "iupac_m_487_> NH1/2 swapped:      176.8878      197  197\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.8100      211  211\n",
      "iupac_m_487_> NH1/2 swapped:      175.4962      213  213\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.9250      219  219\n",
      "iupac_m_486_> OE1/2 will be swapped:     -102.2888      223  223\n",
      "iupac_m_486_> OE1/2 will be swapped:     -153.7171      224  224\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.9904      228  228\n",
      "iupac_m_487_> NH1/2 swapped:     -177.6558      235  235\n",
      "iupac_m_486_> OE1/2 will be swapped:       99.0464      246  246\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1786      250  250\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.0309      257  257\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9720      258  258\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5082      265  265\n",
      "iupac_m_487_> NH1/2 swapped:     -178.6072      269  269\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.2468      271  271\n",
      "iupac_m_487_> NH1/2 swapped:      177.4218      277  277\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.2425      307  307\n",
      "iupac_m_487_> NH1/2 swapped:      176.0987      312  312\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5707      326  326\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.7630      327  327\n",
      "iupac_m_487_> NH1/2 swapped:      174.9042      334  334\n",
      "iupac_m_487_> NH1/2 swapped:      165.2666      337  337\n",
      "iupac_m_486_> OE1/2 will be swapped:     -104.9962      357  357\n",
      "iupac_m_486_> OE1/2 will be swapped:      -92.4168      359  359\n",
      "iupac_m_487_> NH1/2 swapped:      179.4178      364  364\n",
      "iupac_m_487_> NH1/2 swapped:      177.6773      365  365\n",
      "iupac_m_487_> NH1/2 swapped:      177.3854      367  367\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLU:OXT  GLU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   645508\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22807.3750\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22807.375000\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.383999\n",
      "Sequence length          :          382\n",
      "Compactness              :     0.246498\n",
      "Native energy (pair)     :   789.764879\n",
      "Native energy (surface)  :    70.283162\n",
      "Native energy (combined) :    25.585614\n",
      "Z score (pair)           :     0.175102\n",
      "Z score (surface)        :     0.850153\n",
      "Z score (combined)       :     0.650931\n",
      "GA341 score              :     0.031872\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      382\n",
      "Number of all, selected real atoms                :     3084    3084\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19149   19149\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7941\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3387.7305\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3122       0      0   0.006   0.006      30.027       1.000\n",
      " 2 Bond angle potential               :    4211       1     18   2.110   2.110      364.40       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1797       0    113  53.311  53.311      796.28       1.000\n",
      " 4 Stereochemical improper torsion pot:    1207       0      0   1.446   1.446      49.237       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7941       0      1   0.005   0.005      22.884       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2701       0     10   1.511   1.511      385.97       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2876       2     23   1.878   1.878      573.94       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     381       1      8   5.129   5.129      118.18       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     338       0      6  82.848  82.848      114.65       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     272       0      3  72.772  72.772      133.82       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     144       0      1  96.002  96.002      115.16       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      55       0      0 102.928 102.928      40.101       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1188       0      0   0.613   0.613      12.563       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     380     189    103  43.601 123.254      577.75       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     477       0      0   2.843   2.843      52.773       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00184.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00184.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   68774.2344\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5048 178E 178E N   CA   1413 1414  129.27  107.00   22.27    6.40  107.00   22.27    6.40\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16646 285T 217L N   O    2301 1741    8.15    4.82    3.34    5.78    4.82    3.34    5.78\n",
      "    2  17174 335V 313L N   O    2697 2530   15.63    9.93    5.70    5.19    9.93    5.70    5.19\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10894 177R 177R CA  C    1403 1411 -155.65 -180.00   24.35    4.87 -180.00   24.35    4.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10339   2G   3R C   N      11   13  -66.13  -72.10    9.35    0.81  -63.00  169.84   23.15\n",
      "    1          3R   3R N   CA     13   14  149.10  141.90                  -41.10\n",
      "    2  10341   4I   5T C   N      30   32  -94.03  -78.10   18.27    0.82  -63.20  162.09   22.84\n",
      "    2          5T   5T N   CA     32   33  158.76  149.80                  -42.10\n",
      "    3  10347  10R  11R C   N      81   83   40.99   57.30   29.32    1.55  -63.00  146.69   26.27\n",
      "    3         11R  11R N   CA     83   84   62.36   38.00                  -41.10\n",
      "    4  10348  11R  12R C   N      92   94  -87.78  -72.10   25.69    2.23  -63.00  164.52   20.66\n",
      "    4         12R  12R N   CA     94   95  121.54  141.90                  -41.10\n",
      "    5  10349  12R  13A C   N     103  105  177.52 -134.00   78.91    2.74  -62.50  162.66   32.66\n",
      "    5         13A  13A N   CA    105  106 -150.73  147.00                  -40.90\n",
      "    6  10350  13A  14E C   N     108  110  -51.45  -69.30   26.28    2.42  -63.60  158.38   20.47\n",
      "    6         14E  14E N   CA    110  111  161.79  142.50                  -40.30\n",
      "    7  10351  14E  15H C   N     117  119  -52.61  -67.60   22.46    2.02  -63.20  161.32   19.53\n",
      "    7         15H  15H N   CA    119  120  156.73  140.00                  -42.30\n",
      "    8  10352  15H  16N C   N     127  129   60.95   55.90    8.05    0.43  -63.20  144.70   23.79\n",
      "    8         16N  16N N   CA    129  130   33.24   39.50                  -41.10\n",
      "    9  10353  16N  17E C   N     135  137 -133.27 -117.80   15.63    0.52  -63.60 -167.64   22.64\n",
      "    9         17E  17E N   CA    137  138  138.99  136.80                  -40.30\n",
      "   10  10354  17E  18G C   N     144  146   99.23   82.20   91.99    5.30  -62.40 -146.10   39.98\n",
      "   10         18G  18G N   CA    146  147   98.90    8.50                  -41.20\n",
      "   11  10361  24K  25E C   N     193  195  -67.58  -69.30    2.88    0.26  -63.60  174.94   23.54\n",
      "   11         25E  25E N   CA    195  196  144.80  142.50                  -40.30\n",
      "   12  10363  26I  27A C   N     210  212 -116.33 -134.00   22.75    0.61  -62.50 -178.28   27.69\n",
      "   12         27A  27A N   CA    212  213  132.67  147.00                  -40.90\n",
      "   13  10365  28L  29H C   N     223  225 -146.95 -125.60   37.66    1.00  -63.20  169.94   26.65\n",
      "   13         29H  29H N   CA    225  226  169.83  138.80                  -42.30\n",
      "   14  10367  30Q  31Q C   N     242  244 -115.14 -121.10    6.01    0.24  -63.80 -173.55   30.15\n",
      "   14         31Q  31Q N   CA    244  245  140.46  139.70                  -40.30\n",
      "   15  10368  31Q  32D C   N     251  253 -121.92  -70.90   52.06    3.55  -63.30  169.77   24.75\n",
      "   15         32D  32D N   CA    253  254  160.67  150.30                  -40.00\n",
      "   16  10369  32D  33I C   N     259  261  -64.21  -63.40    2.14    0.33 -120.60 -179.07    8.29\n",
      "   16         33I  33I N   CA    261  262  -41.62  -43.60                  130.30\n",
      "   17  10370  33I  34E C   N     267  269  -96.63 -117.80   21.63    0.91  -63.60 -178.49   26.06\n",
      "   17         34E  34E N   CA    269  270  141.22  136.80                  -40.30\n",
      "   18  10371  34E  35K C   N     276  278  -90.63  -70.20   20.44    1.48  -62.90 -179.67   25.02\n",
      "   18         35K  35K N   CA    278  279  141.02  140.40                  -40.80\n",
      "   19  10373  36I  37E C   N     293  295  -73.94  -69.30    4.68    0.33  -63.60  176.85   24.15\n",
      "   19         37E  37E N   CA    295  296  143.15  142.50                  -40.30\n",
      "   20  10376  39V  40G C   N     316  318  -92.39  -80.20   49.80    2.57   82.20 -149.66    8.32\n",
      "   20         40G  40G N   CA    318  319  125.82  174.10                    8.50\n",
      "   21  10379  42A  43C C   N     333  335  -58.35  -63.00   11.26    1.17 -117.90  177.81    7.71\n",
      "   21         43C  43C N   CA    335  336  -51.35  -41.10                  141.10\n",
      "   22  10380  43C  44R C   N     339  341 -110.33 -125.20   42.04    1.66  -63.00  150.04   17.62\n",
      "   22         44R  44R N   CA    341  342  101.28  140.60                  -41.10\n",
      "   23  10382  45E  46L C   N     359  361  -72.70  -70.70    7.14    0.62  -63.50  176.19   23.95\n",
      "   23         46L  46L N   CA    361  362  134.75  141.60                  -41.20\n",
      "   24  10383  46L  47E C   N     367  369  -70.93  -69.30   13.31    0.96  -63.60  164.15   22.29\n",
      "   24         47E  47E N   CA    369  370  155.71  142.50                  -40.30\n",
      "   25  10385  48I  49L C   N     384  386  -94.38 -108.50   25.89    1.22  -63.50  155.11   19.98\n",
      "   25         49L  49L N   CA    386  387  110.81  132.50                  -41.20\n",
      "   26  10386  49L  50Y C   N     392  394 -107.50  -98.40   31.59    3.19  -63.50  148.24   21.47\n",
      "   26         50Y  50Y N   CA    394  395   98.15  128.40                  -43.40\n",
      "   27  10387  50Y  51L C   N     404  406 -141.05 -108.50   60.61    2.86  -63.50  155.84   26.41\n",
      "   27         51L  51L N   CA    406  407 -176.37  132.50                  -41.20\n",
      "   28  10389  52C  53N C   N     418  420  -61.99  -71.20   13.85    0.74  -63.20  173.56   21.77\n",
      "   28         53N  53N N   CA    420  421  132.46  142.80                  -41.10\n",
      "   29  10390  53N  54N C   N     426  428  -91.47  -71.20   27.04    1.48  -63.20  160.71   21.69\n",
      "   29         54N  54N N   CA    428  429  160.69  142.80                  -41.10\n",
      "   30  10393  56I  57S C   N     454  456 -156.52 -136.60   55.46    2.32  -64.10  153.09   17.76\n",
      "   30         57S  57S N   CA    456  457 -157.05  151.20                  -35.00\n",
      "   31  10394  57S  58R C   N     460  462   58.48   57.30    6.67    0.53  -63.00  148.64   27.06\n",
      "   31         58R  58R N   CA    462  463   44.56   38.00                  -41.10\n",
      "   32  10396  59I  60E C   N     479  481  -74.92  -69.30    6.80    0.62  -63.60  179.33   23.31\n",
      "   32         60E  60E N   CA    481  482  138.67  142.50                  -40.30\n",
      "   33  10397  60E  61G C   N     488  490  -84.59  -80.20    5.40    0.39   82.20 -127.18    7.39\n",
      "   33         61G  61G N   CA    490  491  170.95  174.10                    8.50\n",
      "   34  10398  61G  62L C   N     492  494 -110.76 -108.50   48.98    2.67  -63.50  133.42   16.52\n",
      "   34         62L  62L N   CA    494  495   83.57  132.50                  -41.20\n",
      "   35  10401  64H  65L C   N     519  521  -86.88  -63.50   68.64   10.98  -63.50   68.64   10.98\n",
      "   35         65L  65L N   CA    521  522 -105.74  -41.20                  -41.20\n",
      "   36  10402  65L  66K C   N     527  529 -128.82 -118.00   48.95    2.49  -62.90  147.69   16.97\n",
      "   36         66K  66K N   CA    529  530   91.36  139.10                  -40.80\n",
      "   37  10403  66K  67R C   N     536  538  148.78 -125.20   97.32    2.93  -63.00 -161.00   35.89\n",
      "   37         67R  67R N   CA    538  539 -173.89  140.60                  -41.10\n",
      "   38  10404  67R  68L C   N     547  549 -152.63 -108.50   85.40    4.05  -63.50  144.07   25.37\n",
      "   38         68L  68L N   CA    549  550 -154.39  132.50                  -41.20\n",
      "   39  10405  68L  69K C   N     555  557  -69.06  -70.20   16.29    1.21   56.60  172.42   11.37\n",
      "   39         69K  69K N   CA    557  558  156.65  140.40                   38.60\n",
      "   40  10412  75V  76N C   N     612  614  -60.20  -71.20   21.99    1.69  -63.20  157.10   19.48\n",
      "   40         76N  76N N   CA    614  615  161.83  142.80                  -41.10\n",
      "   41  10413  76N  77N C   N     620  622  -74.19  -71.20   64.95    4.25  -63.20  119.52   14.36\n",
      "   41         77N  77N N   CA    622  623   77.91  142.80                  -41.10\n",
      "   42  10415  78I  79A C   N     636  638   66.49  -68.20  135.01   11.76  -62.50 -151.06   31.64\n",
      "   42         79A  79A N   CA    638  639  154.73  145.30                  -40.90\n",
      "   43  10417  80C  81I C   N     647  649  -60.06  -63.40    3.77    0.61 -120.60 -174.20    8.44\n",
      "   43         81I  81I N   CA    649  650  -45.36  -43.60                  130.30\n",
      "   44  10418  81I  82E C   N     655  657   74.85  -69.30  157.71   13.87  -63.60  178.84   21.98\n",
      "   44         82E  82E N   CA    657  658 -153.52  142.50                  -40.30\n",
      "   45  10421  84L  85E C   N     676  678  -32.79  -69.30   40.00    2.45  -63.60  169.29   24.31\n",
      "   45         85E  85E N   CA    678  679  126.16  142.50                  -40.30\n",
      "   46  10423  86G  87C C   N     689  691  -65.69  -63.00    2.79    0.42 -117.90 -173.97    7.98\n",
      "   46         87C  87C N   CA    691  692  -40.34  -41.10                  141.10\n",
      "   47  10425  88E  89S C   N     704  706  -97.98 -136.60   38.74    1.46  -64.10 -179.95   15.31\n",
      "   47         89S  89S N   CA    706  707  148.16  151.20                  -35.00\n",
      "   48  10428  91E  92R C   N     727  729   53.51   57.30   19.76    1.06  -63.00  152.57   27.60\n",
      "   48         92R  92R N   CA    729  730   57.39   38.00                  -41.10\n",
      "   49  10429  92R  93L C   N     738  740  -83.16  -70.70   12.76    1.23  -63.50 -179.00   26.30\n",
      "   49         93L  93L N   CA    740  741  138.87  141.60                  -41.20\n",
      "   50  10430  93L  94D C   N     746  748  -82.81  -70.90   21.00    0.73   54.50 -173.17   11.95\n",
      "   50         94D  94D N   CA    748  749  167.59  150.30                   40.90\n",
      "   51  10432  95L  96T C   N     762  764  -62.10  -78.10   35.41    1.16  -63.20  160.31   20.63\n",
      "   51         96T  96T N   CA    764  765  118.21  149.80                  -42.10\n",
      "   52  10433  96T  97L C   N     769  771 -132.58 -108.50   46.24    2.19  -63.50  162.26   26.80\n",
      "   52         97L  97L N   CA    771  772  171.98  132.50                  -41.20\n",
      "   53  10435  98N  99F C   N     785  787  -90.15  -63.20   78.53   12.68  -63.20   78.53   12.68\n",
      "   53         99F  99F N   CA    787  788 -118.06  -44.30                  -44.30\n",
      "   54  10437 100V 101A C   N     803  805  174.82 -134.00   74.66    2.33  -68.20  129.71    9.01\n",
      "   54        101A 101A N   CA    805  806 -158.64  147.00                  145.30\n",
      "   55  10439 102E 103M C   N     817  819 -113.26 -125.60   13.16    0.63  -63.40 -178.59   31.07\n",
      "   55        103M 103M N   CA    819  820  145.07  140.50                  -40.50\n",
      "   56  10440 103M 104T C   N     825  827 -119.84 -124.80    5.01    0.28  -63.20 -177.25   27.20\n",
      "   56        104T 104T N   CA    827  828  144.15  143.50                  -42.10\n",
      "   57  10443 106V 107Q C   N     845  847  -89.30  -73.00   79.43    5.53  -63.80  106.36   14.43\n",
      "   57        107Q 107Q N   CA    847  848   62.96  140.70                  -40.30\n",
      "   58  10444 107Q 108R C   N     854  856 -175.21 -125.20   50.05    1.96  -63.00 -150.92   35.72\n",
      "   58        108R 108R N   CA    856  857  142.48  140.60                  -41.10\n",
      "   59  10445 108R 109L C   N     865  867 -115.61 -108.50   12.12    0.69  -63.50  171.98   21.52\n",
      "   59        109L 109L N   CA    867  868  122.69  132.50                  -41.20\n",
      "   60  10446 109L 110R C   N     873  875 -160.51 -125.20   53.53    1.68  -63.00  169.03   29.25\n",
      "   60        110R 110R N   CA    875  876 -179.17  140.60                  -41.10\n",
      "   61  10447 110R 111A C   N     884  886  -63.69  -68.20    7.80    0.76  -62.50  167.44   27.50\n",
      "   61        111A 111A N   CA    886  887  151.67  145.30                  -40.90\n",
      "   62  10448 111A 112N C   N     889  891 -109.86 -119.90   35.32    1.80  -63.20  155.22   22.06\n",
      "   62        112N 112N N   CA    891  892  170.86  137.00                  -41.10\n",
      "   63  10459 122G 123N C   N     977  979 -118.67 -119.90   18.62    0.87  -63.20  172.49   24.72\n",
      "   63        123N 123N N   CA    979  980  155.57  137.00                  -41.10\n",
      "   64  10462 125C 126T C   N     998 1000  -61.54  -78.10   17.19    1.12  -63.20  163.48   20.86\n",
      "   64        126T 126T N   CA   1000 1001  154.42  149.80                  -42.10\n",
      "   65  10463 126T 127K C   N    1005 1007   59.46   56.60    9.71    0.92  -62.90  151.11   26.24\n",
      "   65        127K 127K N   CA   1007 1008   47.88   38.60                  -40.80\n",
      "   66  10465 128I 129E C   N    1022 1024 -159.99 -117.80   50.20    1.48  -63.60 -176.89   29.70\n",
      "   66        129E 129E N   CA   1024 1025  164.01  136.80                  -40.30\n",
      "   67  10466 129E 130G C   N    1031 1033  117.47   82.20  102.95    3.31  -62.40 -174.09   30.98\n",
      "   67        130G 130G N   CA   1033 1034  -88.21    8.50                  -41.20\n",
      "   68  10469 132R 133A C   N    1058 1060   85.10 -134.00  141.19    5.02  -62.50 -128.01   45.70\n",
      "   68        133A 133A N   CA   1060 1061  138.08  147.00                  -40.90\n",
      "   69  10470 133A 134Y C   N    1063 1065 -134.08 -124.30   11.74    0.97  -63.50 -173.80   26.42\n",
      "   69        134Y 134Y N   CA   1065 1066  128.91  135.40                  -43.40\n",
      "   70  10473 136I 137H C   N    1090 1092 -162.09 -125.60   40.76    1.07  -63.20 -171.26   29.94\n",
      "   70        137H 137H N   CA   1092 1093  156.94  138.80                  -42.30\n",
      "   71  10474 137H 138A C   N    1100 1102 -134.87 -134.00   10.83    0.63  -62.50 -168.68   28.82\n",
      "   71        138A 138A N   CA   1102 1103  136.21  147.00                  -40.90\n",
      "   72  10475 138A 139L C   N    1105 1107 -112.71 -108.50   11.59    0.66  -63.50  170.18   21.36\n",
      "   72        139L 139L N   CA   1107 1108  121.71  132.50                  -41.20\n",
      "   73  10477 140P 141Q C   N    1120 1122 -141.03 -121.10   21.28    0.67  -63.80 -170.96   31.85\n",
      "   73        141Q 141Q N   CA   1122 1123  147.16  139.70                  -40.30\n",
      "   74  10479 142L 143R C   N    1137 1139 -105.23 -125.20   23.08    0.69  -63.00  175.30   21.16\n",
      "   74        143R 143R N   CA   1139 1140  129.04  140.60                  -41.10\n",
      "   75  10481 144D 145L C   N    1156 1158  -58.43  -70.70   22.14    1.37  -63.50  164.46   23.13\n",
      "   75        145L 145L N   CA   1158 1159  123.18  141.60                  -41.20\n",
      "   76  10483 146D 147G C   N    1172 1174  106.53   82.20   51.67    1.46  -62.40  168.99   30.35\n",
      "   76        147G 147G N   CA   1174 1175  -37.08    8.50                  -41.20\n",
      "   77  10487 150V 151V C   N    1201 1203  -58.44  -62.40   20.41    2.82 -125.40  178.70    6.91\n",
      "   77        151V 151V N   CA   1203 1204  -22.38  -42.40                  143.30\n",
      "   78  10495 158A 159R C   N    1267 1269 -156.26 -125.20   33.81    1.68  -63.00 -167.56   21.80\n",
      "   78        159R 159R N   CA   1269 1270  127.24  140.60                  -41.10\n",
      "   79  10496 159R 160Q C   N    1278 1280 -118.16 -121.10    2.94    0.11  -63.80 -172.12   25.12\n",
      "   79        160Q 160Q N   CA   1280 1281  139.54  139.70                  -40.30\n",
      "   80  10498 161C 162N C   N    1293 1295 -150.51 -119.90   31.75    1.52  -63.20 -169.17   20.56\n",
      "   80        162N 162N N   CA   1295 1296  128.59  137.00                  -41.10\n",
      "   81  10499 162N 163S C   N    1301 1303 -177.55 -136.60   56.83    1.82  -64.10  175.88   20.91\n",
      "   81        163S 163S N   CA   1303 1304 -169.40  151.20                  -35.00\n",
      "   82  10501 164E 165V C   N    1316 1318  -88.42  -62.40   29.87    5.11 -125.40  163.85    8.92\n",
      "   82        165V 165V N   CA   1318 1319  -57.07  -42.40                  143.30\n",
      "   83  10502 165V 166A C   N    1323 1325  150.28 -134.00   85.34    4.33  -62.50 -150.86   32.30\n",
      "   83        166A 166A N   CA   1325 1326  107.64  147.00                  -40.90\n",
      "   84  10503 166A 167A C   N    1328 1330  131.86 -134.00   94.79    2.74  -62.50 -128.98   46.23\n",
      "   84        167A 167A N   CA   1330 1331  158.06  147.00                  -40.90\n",
      "   85  10505 168V 169V C   N    1340 1342  -71.88  -73.50  158.45   13.52 -125.40  171.13    6.97\n",
      "   85        169V 169V N   CA   1342 1343  -19.24  139.20                  143.30\n",
      "   86  10509 172A 173A C   N    1369 1371 -149.46 -134.00   37.27    1.63  -62.50  163.28   31.38\n",
      "   86        173A 173A N   CA   1371 1372 -179.09  147.00                  -40.90\n",
      "   87  10510 173A 174L C   N    1374 1376   76.59  -70.70  154.67   15.25  -63.50 -168.88   24.91\n",
      "   87        174L 174L N   CA   1376 1377 -171.20  141.60                  -41.20\n",
      "   88  10512 175R 176V C   N    1393 1395  -63.47  -62.40    9.37    1.13 -125.40 -173.06    7.50\n",
      "   88        176V 176V N   CA   1395 1396  -33.09  -42.40                  143.30\n",
      "   89  10513 176V 177R C   N    1400 1402 -131.56 -125.20   40.77    1.82   57.30 -137.07   26.75\n",
      "   89        177R 177R N   CA   1402 1403 -179.13  140.60                   38.00\n",
      "   90  10515 178E 179A C   N    1420 1422  -66.68  -68.20   12.64    0.97   55.40  154.42    9.71\n",
      "   90        179A 179A N   CA   1422 1423  132.75  145.30                   38.20\n",
      "   91  10516 179A 180E C   N    1425 1427  -76.71  -69.30   14.11    0.82  -63.60  165.71   22.82\n",
      "   91        180E 180E N   CA   1427 1428  154.51  142.50                  -40.30\n",
      "   92  10517 180E 181R C   N    1434 1436   71.17   57.30   27.71    1.40  -63.00  145.05   26.25\n",
      "   92        181R 181R N   CA   1436 1437   14.01   38.00                  -41.10\n",
      "   93  10519 182I 183K C   N    1453 1455 -148.25 -118.00   31.92    1.07  -62.90 -169.89   29.79\n",
      "   93        183K 183K N   CA   1455 1456  149.32  139.10                  -40.80\n",
      "   94  10520 183K 184A C   N    1462 1464  -85.41  -68.20   18.79    1.32  -62.50  167.82   28.68\n",
      "   94        184A 184A N   CA   1464 1465  152.85  145.30                  -40.90\n",
      "   95  10521 184A 185E C   N    1467 1469 -157.31 -117.80   54.93    1.75  -63.60  172.42   28.11\n",
      "   95        185E 185E N   CA   1469 1470  174.97  136.80                  -40.30\n",
      "   96  10524 187I 188A C   N    1492 1494 -132.35 -134.00    7.18    0.44  -62.50  179.29   33.12\n",
      "   96        188A 188A N   CA   1494 1495  153.98  147.00                  -40.90\n",
      "   97  10525 188A 189Q C   N    1497 1499  -76.34  -73.00   24.44    1.53  -63.80  155.29   23.50\n",
      "   97        189Q 189Q N   CA   1499 1500  164.91  140.70                  -40.30\n",
      "   98  10530 193V 194F C   N    1533 1535   89.90 -124.20  146.17    5.44  -63.20 -136.08   25.65\n",
      "   98        194F 194F N   CA   1535 1536  152.29  143.30                  -44.30\n",
      "   99  10532 195P 196P C   N    1551 1553  -71.85  -58.70   23.98    1.44  -64.50  157.81   12.17\n",
      "   99        196P 196P N   CA   1553 1554  -10.44  -30.50                  147.20\n",
      "  100  10533 196P 197R C   N    1558 1560  -54.76  -72.10   19.62    1.67  -63.00  168.01   22.15\n",
      "  100        197R 197R N   CA   1560 1561  151.09  141.90                  -41.10\n",
      "  101  10534 197R 198Y C   N    1569 1571  -61.95  -98.40   39.54    2.28  -63.50  156.49   24.91\n",
      "  101        198Y 198Y N   CA   1571 1572  113.08  128.40                  -43.40\n",
      "  102  10535 198Y 199N C   N    1581 1583 -164.90 -119.90   50.31    1.56  -63.20 -170.92   29.25\n",
      "  102        199N 199N N   CA   1583 1584  159.50  137.00                  -41.10\n",
      "  103  10537 200E 201K C   N    1598 1600  -65.70  -70.20   58.82    4.37  -62.90  120.19   15.76\n",
      "  103        201K 201K N   CA   1600 1601 -160.96  140.40                  -40.80\n",
      "  104  10538 201K 202G C   N    1607 1609   77.33   82.20   52.17    2.29  -62.40  172.79   32.53\n",
      "  104        202G 202G N   CA   1609 1610   60.44    8.50                  -41.20\n",
      "  105  10540 203E 204R C   N    1620 1622   77.41   57.30   28.65    1.84  -63.00  152.18   27.56\n",
      "  105        204R 204R N   CA   1622 1623   17.58   38.00                  -41.10\n",
      "  106  10559 222E 223E C   N    1787 1789   45.73   54.60   30.28    1.70  -69.30  135.25    7.85\n",
      "  106        223E 223E N   CA   1789 1790   71.35   42.40                  142.50\n",
      "  107  10560 223E 224E C   N    1796 1798 -106.03 -117.80   38.67    1.68  -63.60  146.54   17.57\n",
      "  107        224E 224E N   CA   1798 1799   99.96  136.80                  -40.30\n",
      "  108  10562 225A 226E C   N    1810 1812 -140.00 -117.80   45.07    2.47  -63.60  157.64   18.23\n",
      "  108        226E 226E N   CA   1812 1813   97.58  136.80                  -40.30\n",
      "  109  10563 226E 227R C   N    1819 1821   49.62   57.30   10.20    0.71  -63.00  141.59   25.72\n",
      "  109        227R 227R N   CA   1821 1822   44.71   38.00                  -41.10\n",
      "  110  10564 227R 228E C   N    1830 1832  -89.83 -117.80   77.51    3.26   54.60  146.11   15.88\n",
      "  110        228E 228E N   CA   1832 1833   64.51  136.80                   42.40\n",
      "  111  10566 229K 230K C   N    1848 1850 -130.88 -118.00   45.61    2.34  -62.90  152.17   17.48\n",
      "  111        230K 230K N   CA   1850 1851   95.35  139.10                  -40.80\n",
      "  112  10568 231L 232Q C   N    1865 1867 -100.64 -121.10   49.48    2.58  -63.80  139.89   22.54\n",
      "  112        232Q 232Q N   CA   1867 1868 -175.26  139.70                  -40.30\n",
      "  113  10569 232Q 233K C   N    1874 1876  -99.75 -118.00   21.61    1.03  -62.90  172.51   24.56\n",
      "  113        233K 233K N   CA   1876 1877  150.67  139.10                  -40.80\n",
      "  114  10570 233K 234E C   N    1883 1885 -134.61 -117.80   48.75    2.07  -63.60  154.43   24.54\n",
      "  114        234E 234E N   CA   1885 1886 -177.44  136.80                  -40.30\n",
      "  115  10571 234E 235R C   N    1892 1894  -70.62  -72.10   45.12    3.44  -63.00  138.11   18.16\n",
      "  115        235R 235R N   CA   1894 1895   96.81  141.90                  -41.10\n",
      "  116  10572 235R 236E C   N    1903 1905 -165.43 -117.80   56.93    1.68  -63.60 -177.28   29.90\n",
      "  116        236E 236E N   CA   1905 1906  167.98  136.80                  -40.30\n",
      "  117  10574 237T 238G C   N    1919 1921 -153.40 -167.20   16.22    0.27   82.20 -159.26   14.98\n",
      "  117        238G 238G N   CA   1921 1922  166.06  174.60                    8.50\n",
      "  118  10580 243V 244S C   N    1959 1961  -94.41 -136.60   85.52    4.85  -64.10  103.93    9.64\n",
      "  118        244S 244S N   CA   1961 1962 -134.42  151.20                  -35.00\n",
      "  119  10581 244S 245A C   N    1965 1967 -179.56 -134.00   59.13    1.60  -62.50  178.25   35.26\n",
      "  119        245A 245A N   CA   1967 1968 -175.32  147.00                  -40.90\n",
      "  120  10584 247V 248N C   N    1986 1988  -61.50  -71.20   40.02    2.31  -63.20  145.08   18.23\n",
      "  120        248N 248N N   CA   1988 1989  103.98  142.80                  -41.10\n",
      "  121  10588 251P 252Q C   N    2021 2023   33.00  -73.00  119.89    9.30 -121.10  164.31    7.39\n",
      "  121        252Q 252Q N   CA   2023 2024 -163.28  140.70                  139.70\n",
      "  122  10589 252Q 253R C   N    2030 2032 -172.48 -125.20   71.97    2.26  -63.00  165.45   29.31\n",
      "  122        253R 253R N   CA   2032 2033 -165.14  140.60                  -41.10\n",
      "  123  10591 254L 255T C   N    2049 2051 -113.61 -124.80   31.51    1.67  -63.20  153.46   23.02\n",
      "  123        255T 255T N   CA   2051 2052  172.95  143.50                  -42.10\n",
      "  124  10592 255T 256A C   N    2056 2058  -68.73  -68.20   16.52    1.31  -62.50  157.41   26.11\n",
      "  124        256A 256A N   CA   2058 2059  161.82  145.30                  -40.90\n",
      "  125  10593 256A 257E C   N    2061 2063   41.91  -69.30  121.07    7.47  -63.60  171.28   28.46\n",
      "  125        257E 257E N   CA   2063 2064   94.62  142.50                  -40.30\n",
      "  126  10594 257E 258E C   N    2070 2072 -151.74  -63.60  108.59   18.66  -63.60  108.59   18.66\n",
      "  126        258E 258E N   CA   2072 2073 -103.72  -40.30                  -40.30\n",
      "  127  10597 260I 261A C   N    2096 2098  -93.02  -68.20   27.51    2.65  -62.50  176.98   27.67\n",
      "  127        261A 261A N   CA   2098 2099  133.43  145.30                  -40.90\n",
      "  128  10598 261A 262K C   N    2101 2103  154.73 -118.00  140.73    5.21  -62.90  158.51   27.69\n",
      "  128        262K 262K N   CA   2103 2104 -110.49  139.10                  -40.80\n",
      "  129  10604 267L 268L C   N    2153 2155 -129.16 -108.50   23.63    1.27  -63.50  175.01   21.60\n",
      "  129        268L 268L N   CA   2155 2156  121.03  132.50                  -41.20\n",
      "  130  10605 268L 269R C   N    2161 2163 -104.89 -125.20   25.06    0.74  -63.00  172.19   20.77\n",
      "  130        269R 269R N   CA   2163 2164  125.92  140.60                  -41.10\n",
      "  131  10606 269R 270N C   N    2172 2174  -98.26 -119.90   30.53    1.65  -63.20  164.15   22.52\n",
      "  131        270N 270N N   CA   2174 2175  158.54  137.00                  -41.10\n",
      "  132  10607 270N 271E C   N    2180 2182  -69.45  -69.30    0.19    0.01  -63.60  177.18   23.94\n",
      "  132        271E 271E N   CA   2182 2183  142.62  142.50                  -40.30\n",
      "  133  10609 272P 273K C   N    2196 2198  -82.37  -70.20   15.53    0.94  -62.90  170.28   23.23\n",
      "  133        273K 273K N   CA   2198 2199  150.04  140.40                  -40.80\n",
      "  134  10610 273K 274F C   N    2205 2207   75.69  -71.40  147.47   10.31  -63.20 -144.83   24.43\n",
      "  134        274F 274F N   CA   2207 2208  151.36  140.70                  -44.30\n",
      "  135  10612 275P 276M C   N    2223 2225 -116.04 -125.60   21.97    0.82  -63.40  169.59   21.86\n",
      "  135        276M 276M N   CA   2225 2226  120.71  140.50                  -40.50\n",
      "  136  10614 277R 278L C   N    2242 2244 -119.63 -108.50   11.44    0.58  -63.50 -179.98   22.48\n",
      "  136        278L 278L N   CA   2244 2245  129.85  132.50                  -41.20\n",
      "  137  10615 278L 279D C   N    2250 2252 -125.92  -96.50   32.83    1.36  -63.30 -179.99   19.37\n",
      "  137        279D 279D N   CA   2252 2253  128.77  114.20                  -40.00\n",
      "  138  10616 279D 280Q C   N    2258 2260 -111.03 -121.10   31.79    1.64  -63.80  157.12   25.62\n",
      "  138        280Q 280Q N   CA   2260 2261  169.85  139.70                  -40.30\n",
      "  139  10617 280Q 281E C   N    2267 2269 -113.06 -117.80    8.45    0.46  -63.60 -177.28   27.16\n",
      "  139        281E 281E N   CA   2269 2270  143.79  136.80                  -40.30\n",
      "  140  10618 281E 282S C   N    2276 2278 -131.05 -136.60   28.94    1.57  -64.10  160.07   16.38\n",
      "  140        282S 282S N   CA   2278 2279  179.60  151.20                  -35.00\n",
      "  141  10619 282S 283D C   N    2282 2284 -133.04  -96.50   38.16    1.57  -63.30  179.30   19.15\n",
      "  141        283D 283D N   CA   2284 2285  125.18  114.20                  -40.00\n",
      "  142  10620 283D 284E C   N    2290 2292 -168.77 -117.80   52.25    1.66  -63.60 -158.92   32.59\n",
      "  142        284E 284E N   CA   2292 2293  148.31  136.80                  -40.30\n",
      "  143  10626 289L 290T C   N    2334 2336   60.04  -78.10  138.21    8.27  -78.10  138.21    8.27\n",
      "  143        290T 290T N   CA   2336 2337  145.38  149.80                  149.80\n",
      "  144  10627 290T 291V C   N    2341 2343  -65.99  -62.40   25.87    3.08 -125.40  170.75    6.76\n",
      "  144        291V 291V N   CA   2343 2344  -16.78  -42.40                  143.30\n",
      "  145  10628 291V 292D C   N    2348 2350  -55.65  -63.30   12.62    2.04   54.50  130.97   16.94\n",
      "  145        292D 292D N   CA   2350 2351  -29.95  -40.00                   40.90\n",
      "  146  10630 293V 294P C   N    2363 2365  -61.22  -64.50    3.40    0.37  -58.70  178.62   14.56\n",
      "  146        294P 294P N   CA   2365 2366  148.10  147.20                  -30.50\n",
      "  147  10631 294P 295K C   N    2370 2372  -76.15  -70.20   18.79    1.49  -62.90  163.91   20.59\n",
      "  147        295K 295K N   CA   2372 2373  122.57  140.40                  -40.80\n",
      "  148  10632 295K 296F C   N    2379 2381  165.44 -124.20   91.39    2.38  -63.20  173.98   31.30\n",
      "  148        296F 296F N   CA   2381 2382 -158.38  143.30                  -44.30\n",
      "  149  10633 296F 297L C   N    2390 2392  -78.16  -70.70   44.66    3.67  -63.50  139.54   18.58\n",
      "  149        297L 297L N   CA   2392 2393   97.57  141.60                  -41.20\n",
      "  150  10634 297L 298S C   N    2398 2400 -148.85 -136.60   58.59    2.69  -64.10  144.07   16.56\n",
      "  150        298S 298S N   CA   2400 2401 -151.51  151.20                  -35.00\n",
      "  151  10639 302I 303D C   N    2434 2436 -137.13  -96.50   46.83    1.94  -63.30 -167.77   20.54\n",
      "  151        303D 303D N   CA   2436 2437  137.49  114.20                  -40.00\n",
      "  152  10641 304V 305Q C   N    2449 2451  146.64 -121.10  100.74    3.15  -63.80 -155.46   36.92\n",
      "  152        305Q 305Q N   CA   2451 2452 -179.84  139.70                  -40.30\n",
      "  153  10643 306I 307E C   N    2466 2468 -137.10 -117.80   56.39    2.40  -63.60  149.27   23.97\n",
      "  153        307E 307E N   CA   2468 2469 -170.22  136.80                  -40.30\n",
      "  154  10644 307E 308V C   N    2475 2477  -71.60 -125.40   67.47    1.78  -62.40  145.28   17.87\n",
      "  154        308V 308V N   CA   2477 2478  102.58  143.30                  -42.40\n",
      "  155  10645 308V 309N C   N    2482 2484 -108.77 -119.90   24.11    0.81  -63.20  163.21   18.28\n",
      "  155        309N 309N N   CA   2484 2485  115.62  137.00                  -41.10\n",
      "  156  10655 318K 319L C   N    2571 2573 -177.17 -108.50   71.58    3.21  -63.50 -158.74   34.87\n",
      "  156        319L 319L N   CA   2573 2574  152.71  132.50                  -41.20\n",
      "  157  10657 320I 321Q C   N    2587 2589 -147.79 -121.10   28.15    1.25  -63.80 -169.42   24.65\n",
      "  157        321Q 321Q N   CA   2589 2590  130.77  139.70                  -40.30\n",
      "  158  10659 322V 323P C   N    2603 2605  -64.53  -58.70    9.28    0.59  -64.50  170.48   12.72\n",
      "  158        323P 323P N   CA   2605 2606  -23.28  -30.50                  147.20\n",
      "  159  10660 323P 324L C   N    2610 2612  -69.96  -70.70   19.60    1.46  -63.50  163.35   22.32\n",
      "  159        324L 324L N   CA   2612 2613  122.02  141.60                  -41.20\n",
      "  160  10661 324L 325Q C   N    2618 2620  -89.05  -73.00   27.98    2.13  -63.80  160.09   22.29\n",
      "  160        325Q 325Q N   CA   2620 2621  117.78  140.70                  -40.30\n",
      "  161  10662 325Q 326R C   N    2627 2629 -152.16 -125.20   29.10    0.91  -63.00 -170.38   31.65\n",
      "  161        326R 326R N   CA   2629 2630  151.55  140.60                  -41.10\n",
      "  162  10663 326R 327E C   N    2638 2640 -115.88 -117.80   36.91    1.82  -63.60  149.67   17.67\n",
      "  162        327E 327E N   CA   2640 2641   99.94  136.80                  -40.30\n",
      "  163  10665 328I 329S C   N    2655 2657 -132.14 -136.60   11.08    0.62  -64.10  177.24   17.67\n",
      "  163        329S 329S N   CA   2657 2658  161.34  151.20                  -35.00\n",
      "  164  10666 329S 330P C   N    2661 2663  -54.17  -64.50   10.68    0.96  -58.70  175.03   14.83\n",
      "  164        330P 330P N   CA   2663 2664  144.47  147.20                  -30.50\n",
      "  165  10667 330P 331S C   N    2668 2670 -155.46 -136.60   57.75    2.47  -64.10  150.19   17.47\n",
      "  165        331S 331S N   CA   2670 2671 -154.22  151.20                  -35.00\n",
      "  166  10668 331S 332A C   N    2674 2676  121.20 -134.00  115.87    2.63  -62.50 -145.22   43.57\n",
      "  166        332A 332A N   CA   2676 2677 -163.57  147.00                  -40.90\n",
      "  167  10670 333A 334R C   N    2684 2686 -132.81  -63.00   73.74   10.76  -63.00   73.74   10.76\n",
      "  167        334R 334R N   CA   2686 2687  -17.33  -41.10                  -41.10\n",
      "  168  10673 336Q 337R C   N    2711 2713   87.65   57.30   37.42    2.87  -63.00  161.15   29.09\n",
      "  168        337R 337R N   CA   2713 2714   16.11   38.00                  -41.10\n",
      "  169  10686 349P 350Y C   N    2804 2806 -133.91 -124.30   40.64    1.78  -63.50  158.24   28.55\n",
      "  169        350Y 350Y N   CA   2806 2807  174.89  135.40                  -43.40\n",
      "  170  10687 350Y 351A C   N    2816 2818  -54.31  -68.20   14.45    1.08  -62.50  177.98   28.78\n",
      "  170        351A 351A N   CA   2818 2819  141.31  145.30                  -40.90\n",
      "  171  10688 351A 352S C   N    2821 2823 -150.35 -136.60   69.84    3.23  -64.10  136.13   16.08\n",
      "  171        352S 352S N   CA   2823 2824 -140.33  151.20                  -35.00\n",
      "  172  10691 354V 355L C   N    2844 2846 -162.43 -108.50   55.91    2.85  -63.50 -172.80   22.99\n",
      "  172        355L 355L N   CA   2846 2847  117.72  132.50                  -41.20\n",
      "  173  10698 361A 362K C   N    2901 2903  -88.34  -70.20   35.91    2.95  -62.90  152.36   18.54\n",
      "  173        362K 362K N   CA   2903 2904  109.42  140.40                  -40.80\n",
      "  174  10699 362K 363K C   N    2910 2912 -147.81 -118.00   49.30    1.84  -62.90  164.45   26.41\n",
      "  174        363K 363K N   CA   2912 2913  178.37  139.10                  -40.80\n",
      "  175  10700 363K 364R C   N    2919 2921  -56.54  -72.10   15.68    1.16  -63.00  179.01   23.75\n",
      "  175        364R 364R N   CA   2921 2922  140.01  141.90                  -41.10\n",
      "  176  10701 364R 365R C   N    2930 2932  -96.14  -72.10   33.00    2.87  -63.00  163.77   20.10\n",
      "  176        365R 365R N   CA   2932 2933  119.28  141.90                  -41.10\n",
      "  177  10702 365R 366Q C   N    2941 2943 -102.27 -121.10   21.54    0.66  -63.80  173.84   23.71\n",
      "  177        366Q 366Q N   CA   2943 2944  129.23  139.70                  -40.30\n",
      "  178  10703 366Q 367R C   N    2950 2952  -65.74  -72.10   15.16    1.03  -63.00  169.26   22.68\n",
      "  178        367R 367R N   CA   2952 2953  128.14  141.90                  -41.10\n",
      "  179  10704 367R 368L C   N    2961 2963 -129.12 -108.50   26.97    1.20  -63.50 -178.79   29.22\n",
      "  179        368L 368L N   CA   2963 2964  149.88  132.50                  -41.20\n",
      "  180  10705 368L 369H C   N    2969 2971  -81.01  -67.60   26.36    1.64  -63.20  156.03   20.65\n",
      "  180        369H 369H N   CA   2971 2972  162.69  140.00                  -42.30\n",
      "  181  10706 369H 370G C   N    2979 2981  106.32   78.70   59.72    0.89   82.20  134.63    7.72\n",
      "  181        370G 370G N   CA   2981 2982  140.95 -166.10                    8.50\n",
      "  182  10707 370G 371T C   N    2983 2985 -131.83 -124.80   21.47    0.73  -63.20  168.71   26.13\n",
      "  182        371T 371T N   CA   2985 2986  163.78  143.50                  -42.10\n",
      "  183  10708 371T 372W C   N    2990 2992  -93.33  -71.30   22.26    1.50  -63.00  176.26   24.00\n",
      "  183        372W 372W N   CA   2992 2993  142.17  139.00                  -44.20\n",
      "  184  10709 372W 373N C   N    3004 3006 -101.38 -119.90   21.92    1.15  -63.20  174.41   23.98\n",
      "  184        373N 373N N   CA   3006 3007  148.72  137.00                  -41.10\n",
      "  185  10710 373N 374D C   N    3012 3014  -89.86  -70.90   63.45    2.31  -63.30  112.34   15.57\n",
      "  185        374D 374D N   CA   3014 3015 -149.15  150.30                  -40.00\n",
      "  186  10712 375D 376E C   N    3028 3030  -81.19  -69.30   14.07    0.81  -63.60  170.58   23.72\n",
      "  186        376E 376E N   CA   3030 3031  150.02  142.50                  -40.30\n",
      "  187  10714 377C 378S C   N    3043 3045  -81.46  -72.40   12.81    0.61  -64.10  164.46   13.03\n",
      "  187        378S 378S N   CA   3045 3046  161.45  152.40                  -35.00\n",
      "  188  10715 378S 379E C   N    3049 3051  -75.44  -69.30    7.17    0.42  -63.60  173.91   23.84\n",
      "  188        379E 379E N   CA   3051 3052  146.19  142.50                  -40.30\n",
      "  189  10716 379E 380P C   N    3058 3060  -64.14  -58.70    9.45    0.58  -64.50  169.97   12.66\n",
      "  189        380P 380P N   CA   3060 3061  -22.77  -30.50                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   54   77  351  322  430  368  407  508  567  586\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00184.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      382     3084     3084\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00184.B99990001.pdb        3464.77808   -23494.64258        0.15760\n",
      "EAO00184.B99990002.pdb        3228.34546   -22821.51953        0.18139\n",
      "EAO00184.B99990003.pdb        3235.38330   -21988.73047        0.08162\n",
      "EAO00184.B99990004.pdb        3004.52539   -23696.50781        0.05115\n",
      "EAO00184.B99990005.pdb        3387.73047   -22807.37500        0.03187\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     37138231   36267.805    35.418\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     37138759   36268.320    35.418\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     37187359   36315.781    35.465\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     37200659   36328.770    35.477\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37469925   36591.723    35.734\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41031048   40069.383    39.130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41033290   40071.570    39.132\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41034740   40072.988    39.134\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41037640   40075.820    39.137\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41043440   40081.484    39.142\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     54481839   53204.922    51.958\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     54493439   53216.250    51.969\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     54516639   53238.906    51.991\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     54563039   53284.219    52.035\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     54655839   53374.844    52.124\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     61375039   59936.562    58.532\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00185.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     37472245   36593.988    35.736\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     37473695   36595.406    35.738\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     37476595   36598.238    35.740\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     37482395   36603.902    35.746\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     37493995   36615.230    35.757\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     37495735   36616.930    35.759\n",
      "\n",
      "Read the alignment from file       : ali/EAO00185.ali\n",
      "\n",
      "Total number of alignment positions:   436\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00185     436      1    EAO00185\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     37497812   36618.957    35.761\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37822987   36936.512    36.071\n",
      "profile_iteration_> processing sequence:       1    436      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37824065   36937.562    36.072\n",
      "profile_iteration_> processing sequence:    6720    436      1     4.5156250     0.0006720     0.0000026  1488\n",
      "profile_iteration_> processing sequence:   13439    436      1     8.9375000     0.0006650     0.0000026  1504\n",
      "profile_iteration_> processing sequence:   20158    436      1    13.3281250     0.0006612     0.0000026  1512\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37824440   36937.930    36.072\n",
      "profile_iteration_> processing sequence:   26877    436      1    17.7500000     0.0006604     0.0000026  1514\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37824676   36938.160    36.072\n",
      "profile_iteration_> processing sequence:   33596    436      1    21.8906250     0.0006516     0.0000026  1535\n",
      "profile_iteration_> processing sequence:   40315    436      1    26.0312500     0.0006457     0.0000025  1549\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37827626   36941.039    36.075\n",
      "profile_iteration_> processing sequence:   47034    436      1    30.2031250     0.0006422     0.0000025  1557\n",
      "profile_iteration_> processing sequence:   53753    436      1    34.4843750     0.0006415     0.0000025  1559\n",
      "profile_iteration_> processing sequence:   60472    436      1    38.6250000     0.0006387     0.0000025  1566\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     37831136   36944.469    36.079\n",
      "profile_iteration_> processing sequence:   67191    436      1    42.9687500     0.0006395     0.0000025  1564\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    24.01241\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    238.5901\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0253,      121,        0.0093,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00015   0.00000\n",
      "Z:         1  -2.75000   0.00045   0.00000\n",
      "Z:         1  -2.65000   0.00015   0.00000\n",
      "Z:         1  -2.55000   0.00015   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00074   0.00016\n",
      "Z:         1  -2.25000   0.00164   0.00055\n",
      "Z:         1  -2.15000   0.00208   0.00163\n",
      "Z:         1  -2.05000   0.00640   0.00416\n",
      "Z:         1  -1.95000   0.01042   0.00933\n",
      "Z:         1  -1.85000   0.02188   0.01872\n",
      "Z:         1  -1.75000   0.03483   0.03400\n",
      "Z:         1  -1.65000   0.06042   0.05658\n",
      "Z:         1  -1.55000   0.08662   0.08721\n",
      "Z:         1  -1.45000   0.11564   0.12565\n",
      "Z:         1  -1.35000   0.17234   0.17059\n",
      "Z:         1  -1.25000   0.21029   0.21981\n",
      "Z:         1  -1.15000   0.29289   0.27052\n",
      "Z:         1  -1.05000   0.31656   0.31973\n",
      "Z:         1  -0.95000   0.36626   0.36469\n",
      "Z:         1  -0.85000   0.39633   0.40316\n",
      "Z:         1  -0.75000   0.41954   0.43360\n",
      "Z:         1  -0.65000   0.45764   0.45519\n",
      "Z:         1  -0.55000   0.47952   0.46779\n",
      "Z:         1  -0.45000   0.45050   0.47182\n",
      "Z:         1  -0.35000   0.44529   0.46811\n",
      "Z:         1  -0.25000   0.47015   0.45775\n",
      "Z:         1  -0.15000   0.43234   0.44196\n",
      "Z:         1  -0.05000   0.41300   0.42194\n",
      "Z:         1   0.05000   0.40496   0.39889\n",
      "Z:         1   0.15000   0.36537   0.37383\n",
      "Z:         1   0.25000   0.34617   0.34769\n",
      "Z:         1   0.35000   0.31477   0.32121\n",
      "Z:         1   0.45000   0.26804   0.29500\n",
      "Z:         1   0.55000   0.26685   0.26953\n",
      "Z:         1   0.65000   0.23842   0.24513\n",
      "Z:         1   0.75000   0.23158   0.22205\n",
      "Z:         1   0.85000   0.19749   0.20043\n",
      "Z:         1   0.95000   0.17115   0.18036\n",
      "Z:         1   1.05000   0.16088   0.16185\n",
      "Z:         1   1.15000   0.19571   0.14489\n",
      "Z:         1   1.25000   0.12695   0.12944\n",
      "Z:         1   1.35000   0.13722   0.11542\n",
      "Z:         1   1.45000   0.10552   0.10275\n",
      "Z:         1   1.55000   0.10641   0.09133\n",
      "Z:         1   1.65000   0.07873   0.08109\n",
      "Z:         1   1.75000   0.07620   0.07191\n",
      "Z:         1   1.85000   0.06548   0.06371\n",
      "Z:         1   1.95000   0.05507   0.05639\n",
      "Z:         1   2.05000   0.05120   0.04988\n",
      "Z:         1   2.15000   0.03825   0.04409\n",
      "Z:         1   2.25000   0.03736   0.03895\n",
      "Z:         1   2.35000   0.03408   0.03439\n",
      "Z:         1   2.45000   0.03036   0.03035\n",
      "Z:         1   2.55000   0.02590   0.02678\n",
      "Z:         1   2.65000   0.02203   0.02361\n",
      "Z:         1   2.75000   0.01652   0.02082\n",
      "Z:         1   2.85000   0.01920   0.01835\n",
      "Z:         1   2.95000   0.01637   0.01617\n",
      "Z:         1   3.05000   0.01220   0.01424\n",
      "Z:         1   3.15000   0.01339   0.01255\n",
      "Z:         1   3.25000   0.01220   0.01105\n",
      "Z:         1   3.35000   0.01116   0.00973\n",
      "Z:         1   3.45000   0.00878   0.00857\n",
      "Z:         1   3.55000   0.00878   0.00754\n",
      "Z:         1   3.65000   0.00655   0.00664\n",
      "Z:         1   3.75000   0.00566   0.00584\n",
      "Z:         1   3.85000   0.00446   0.00514\n",
      "Z:         1   3.95000   0.00432   0.00453\n",
      "Z:         1   4.05000   0.00521   0.00398\n",
      "Z:         1   4.15000   0.00491   0.00350\n",
      "Z:         1   4.25000   0.00327   0.00308\n",
      "Z:         1   4.35000   0.00446   0.00271\n",
      "Z:         1   4.45000   0.00238   0.00239\n",
      "Z:         1   4.55000   0.00208   0.00210\n",
      "Z:         1   4.65000   0.00193   0.00185\n",
      "Z:         1   4.75000   0.00193   0.00163\n",
      "Z:         1   4.85000   0.00119   0.00143\n",
      "Z:         1   4.95000   0.00208   0.00126\n",
      "Z:         1   5.05000   0.00074   0.00111\n",
      "Z:         1   5.15000   0.00134   0.00097\n",
      "Z:         1   5.25000   0.00149   0.00086\n",
      "Z:         1   5.35000   0.00179   0.00075\n",
      "Z:         1   5.45000   0.00104   0.00066\n",
      "Z:         1   5.55000   0.00074   0.00058\n",
      "Z:         1   5.65000   0.00060   0.00051\n",
      "Z:         1   5.75000   0.00045   0.00045\n",
      "Z:         1   5.85000   0.00045   0.00040\n",
      "Z:         1   5.95000   0.00060   0.00035\n",
      "Z:         1   6.05000   0.00030   0.00031\n",
      "Z:         1   6.15000   0.00030   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00045   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00045   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00060   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00045   0.00007\n",
      "Z:         1   7.35000   0.00045   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00030   0.00004\n",
      "Z:         1   7.75000   0.00045   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    42.968750\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000639\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1564.879150\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000641\n",
      "profile_iteration__> Overall performance [seq/sec]:  1560.903076\n",
      "openf___224_> Open           buildprofile/EAO00185build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22863218   22327.361    21.804\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22864958   22329.061    21.806\n",
      "openf___224_> Open           buildprofile/EAO00185build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22821735   22286.852    21.765\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26382858   25764.510    25.161\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26385100   25766.699    25.163\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26386550   25768.115    25.164\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26389450   25770.947    25.167\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26395250   25776.611    25.172\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     39833649   38900.047    37.988\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39845249   38911.375    37.999\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39868449   38934.031    38.022\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39914849   38979.344    38.066\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40007649   39069.969    38.154\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     46726849   45631.688    44.562\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00186.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22824055   22289.117    21.767\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22825505   22290.531    21.768\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22828405   22293.363    21.771\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22834205   22299.027    21.776\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22845805   22310.355    21.787\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22869005   22333.012    21.810\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22872469   22336.395    21.813\n",
      "\n",
      "Read the alignment from file       : ali/EAO00186.ali\n",
      "\n",
      "Total number of alignment positions:   867\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00186     867      1    EAO00186\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     22876270   22340.107    21.817\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23201445   22657.660    22.127\n",
      "profile_iteration_> processing sequence:       1    867      1     0.0312500     0.0312500     0.0312500     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23202523   22658.715    22.128\n",
      "profile_iteration_> processing sequence:    6720    867      1     8.5468750     0.0012719     0.0000049   786\n",
      "profile_iteration_> processing sequence:   13439    867      1    16.7187500     0.0012440     0.0000049   804\n",
      "profile_iteration_> processing sequence:   20158    867      1    24.8750000     0.0012340     0.0000048   810\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23202898   22659.080    22.128\n",
      "profile_iteration_> processing sequence:   26877    867      1    33.0781250     0.0012307     0.0000048   813\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23203134   22659.311    22.128\n",
      "profile_iteration_> processing sequence:   33596    867      1    41.3281250     0.0012302     0.0000048   813\n",
      "profile_iteration_> processing sequence:   40315    867      1    49.4687500     0.0012271     0.0000048   815\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23206084   22662.191    22.131\n",
      "profile_iteration_> processing sequence:   47034    867      1    57.7343750     0.0012275     0.0000048   815\n",
      "profile_iteration_> processing sequence:   53753    867      1    66.2187500     0.0012319     0.0000048   812\n",
      "profile_iteration_> processing sequence:   60472    867      1    74.4843750     0.0012317     0.0000048   812\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23209594   22665.619    22.134\n",
      "profile_iteration_> processing sequence:   67191    867      1    83.0156250     0.0012355     0.0000048   809\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    64.96236\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    242.1066\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0191,      121,        0.0073,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00030   0.00001\n",
      "Z:         1  -2.45000   0.00000   0.00004\n",
      "Z:         1  -2.35000   0.00030   0.00016\n",
      "Z:         1  -2.25000   0.00283   0.00055\n",
      "Z:         1  -2.15000   0.00238   0.00163\n",
      "Z:         1  -2.05000   0.00551   0.00416\n",
      "Z:         1  -1.95000   0.01027   0.00933\n",
      "Z:         1  -1.85000   0.02485   0.01872\n",
      "Z:         1  -1.75000   0.03736   0.03400\n",
      "Z:         1  -1.65000   0.05447   0.05658\n",
      "Z:         1  -1.55000   0.08825   0.08721\n",
      "Z:         1  -1.45000   0.12219   0.12565\n",
      "Z:         1  -1.35000   0.17651   0.17059\n",
      "Z:         1  -1.25000   0.22324   0.21981\n",
      "Z:         1  -1.15000   0.31328   0.27052\n",
      "Z:         1  -1.05000   0.30629   0.31973\n",
      "Z:         1  -0.95000   0.35942   0.36469\n",
      "Z:         1  -0.85000   0.37832   0.40316\n",
      "Z:         1  -0.75000   0.43755   0.43360\n",
      "Z:         1  -0.65000   0.41032   0.45519\n",
      "Z:         1  -0.55000   0.47431   0.46779\n",
      "Z:         1  -0.45000   0.44514   0.47182\n",
      "Z:         1  -0.35000   0.46479   0.46811\n",
      "Z:         1  -0.25000   0.45854   0.45775\n",
      "Z:         1  -0.15000   0.45273   0.44196\n",
      "Z:         1  -0.05000   0.42550   0.42194\n",
      "Z:         1   0.05000   0.40734   0.39889\n",
      "Z:         1   0.15000   0.34692   0.37383\n",
      "Z:         1   0.25000   0.34811   0.34769\n",
      "Z:         1   0.35000   0.30405   0.32121\n",
      "Z:         1   0.45000   0.29736   0.29500\n",
      "Z:         1   0.55000   0.27101   0.26953\n",
      "Z:         1   0.65000   0.23931   0.24513\n",
      "Z:         1   0.75000   0.22785   0.22205\n",
      "Z:         1   0.85000   0.20583   0.20043\n",
      "Z:         1   0.95000   0.18246   0.18036\n",
      "Z:         1   1.05000   0.16564   0.16185\n",
      "Z:         1   1.15000   0.15463   0.14489\n",
      "Z:         1   1.25000   0.12754   0.12944\n",
      "Z:         1   1.35000   0.12338   0.11542\n",
      "Z:         1   1.45000   0.10224   0.10275\n",
      "Z:         1   1.55000   0.09451   0.09133\n",
      "Z:         1   1.65000   0.08572   0.08109\n",
      "Z:         1   1.75000   0.07293   0.07191\n",
      "Z:         1   1.85000   0.06593   0.06371\n",
      "Z:         1   1.95000   0.05358   0.05639\n",
      "Z:         1   2.05000   0.05581   0.04988\n",
      "Z:         1   2.15000   0.04108   0.04409\n",
      "Z:         1   2.25000   0.04093   0.03895\n",
      "Z:         1   2.35000   0.03453   0.03439\n",
      "Z:         1   2.45000   0.03587   0.03035\n",
      "Z:         1   2.55000   0.02709   0.02678\n",
      "Z:         1   2.65000   0.02381   0.02361\n",
      "Z:         1   2.75000   0.02113   0.02082\n",
      "Z:         1   2.85000   0.02054   0.01835\n",
      "Z:         1   2.95000   0.01801   0.01617\n",
      "Z:         1   3.05000   0.01339   0.01424\n",
      "Z:         1   3.15000   0.01533   0.01255\n",
      "Z:         1   3.25000   0.01131   0.01105\n",
      "Z:         1   3.35000   0.00863   0.00973\n",
      "Z:         1   3.45000   0.00938   0.00857\n",
      "Z:         1   3.55000   0.00893   0.00754\n",
      "Z:         1   3.65000   0.00759   0.00664\n",
      "Z:         1   3.75000   0.00461   0.00584\n",
      "Z:         1   3.85000   0.00446   0.00514\n",
      "Z:         1   3.95000   0.00506   0.00453\n",
      "Z:         1   4.05000   0.00506   0.00398\n",
      "Z:         1   4.15000   0.00357   0.00350\n",
      "Z:         1   4.25000   0.00402   0.00308\n",
      "Z:         1   4.35000   0.00313   0.00271\n",
      "Z:         1   4.45000   0.00268   0.00239\n",
      "Z:         1   4.55000   0.00119   0.00210\n",
      "Z:         1   4.65000   0.00119   0.00185\n",
      "Z:         1   4.75000   0.00149   0.00163\n",
      "Z:         1   4.85000   0.00193   0.00143\n",
      "Z:         1   4.95000   0.00179   0.00126\n",
      "Z:         1   5.05000   0.00179   0.00111\n",
      "Z:         1   5.15000   0.00074   0.00097\n",
      "Z:         1   5.25000   0.00119   0.00086\n",
      "Z:         1   5.35000   0.00074   0.00075\n",
      "Z:         1   5.45000   0.00074   0.00066\n",
      "Z:         1   5.55000   0.00074   0.00058\n",
      "Z:         1   5.65000   0.00045   0.00051\n",
      "Z:         1   5.75000   0.00089   0.00045\n",
      "Z:         1   5.85000   0.00000   0.00040\n",
      "Z:         1   5.95000   0.00060   0.00035\n",
      "Z:         1   6.05000   0.00060   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00000   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00030   0.00010\n",
      "Z:         1   7.05000   0.00030   0.00009\n",
      "Z:         1   7.15000   0.00030   0.00007\n",
      "Z:         1   7.25000   0.00015   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00015   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     23920194   23359.564    22.812\n",
      "> 5lg6A                      1     199   32900     889     867   27.21     0.0           2   740    80   854    95   873\n",
      "> 4wz9A                      1     627   24450     882     867   29.21     0.0           3   469    97   577   110   602\n",
      "> 4fkeA                      1    1510   33800     902     867   26.93     0.0           4   745    80   854    95   885\n",
      "> 3b34A                      1    1573   10250     866     867   27.36     0.0           5   304   142   454   129   446\n",
      "> 4fyqA                      1    2111   30350     892     867   27.19     0.0           6   695    98   824   115   846\n",
      "> 4fytA                      1    2112   31850     903     867   26.89     0.0           7   703    98   824   118   857\n",
      "> 6q4rA                      1    2743   37850     855     867   29.56     0.0           8   800     6   853     6   854\n",
      "> 4qhpA                      1    4761   11700     866     867   27.74     0.0           9   364    74   452    51   443\n",
      "> 3rjoA                      1    6863    6750     408     867   24.69    0.31E-06      10   367   448   853     1   397\n",
      "> 5dllA                      1   13591   10950     854     867   24.16     0.0          11   374   101   486    96   484\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 3mdjA                      1   14827   32200     818     867   29.59     0.0          12   749     6   820     5   788\n",
      "> 5zi7A                      1   16114    7350     850     867   24.52    0.32E-07      13   302   135   455   128   441\n",
      "> 3qnfA                      1   25433   33600     696     867   30.85     0.0          14   604     6   645     5   636\n",
      "> 3qnfB                      1   25434   34500     802     867   30.49     0.0          15   609     6   645     5   637\n",
      "> 4kx7A                      1   30347   34800     875     867   31.53     0.0          16   578    16   611    24   632\n",
      "> 2xq0A                      1   32532   10400     625     867   27.33     0.0          17   325    70   430    87   430\n",
      "> 5ab0A                      1   32942   33350     911     867   26.52     0.0          18   796     8   850    14   903\n",
      "> 5ab2A                      1   32943   34200     897     867   26.99     0.0          19   796     8   850    14   891\n",
      "> 4jbsA                      1   36736   35550     869     867   28.20     0.0          20   794     8   850    14   864\n",
      "> 4jbsB                      1   36737   35750     854     867   27.96     0.0          21   791     8   850    13   849\n",
      "> 1z5hA                      1   37002   31950     780     867   31.54     0.0          22   512    19   571     6   525\n",
      "> 6mgqC                      1   43134   35100     843     867   29.68     0.0          23   623     6   645     9   665\n",
      "> 4gaaA                      1   44928   12050     606     867   30.64     0.0          24   281   119   421   121   417\n",
      "> 6iffA                      1   45317    7300     429     867   28.90    0.18E-07      25   254   130   421   127   389\n",
      "> 3u9wA                      1   50957   12500     608     867   27.02     0.0          26   314    94   421    98   419\n",
      "> 3ciaA                      1   51580   11650     587     867   26.87     0.0          27   348    42   421    46   406\n",
      "> 5mj6A                      1   54712   34450     870     867   27.16     0.0          28   816     8   863    10   867\n",
      "> 4p8qB                      1   55700   33300     848     867   27.56     0.0          29   797     8   850     7   837\n",
      "> 6ea4A                      1   62769   34600     883     867   27.38     0.0          30   795     8   850    15   876\n",
      "> 4pj6B                      1   62971   33350     849     867   27.52     0.0          31   797     8   850     7   838\n",
      "> 6sbqA                      1   66947   10150     891     867   27.94     0.0          32   320    96   431    94   433\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.031250\n",
      "profile_iteration__> Score Collection        [sec]:    82.984375\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.001235\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:   810.152222\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.296875\n",
      "profile_iteration__> Overall performance    [/seq]:     0.001241\n",
      "profile_iteration__> Overall performance [seq/sec]:   805.750061\n",
      "openf___224_> Open           buildprofile/EAO00186build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23872466   23312.955    22.767\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23875930   23316.338    22.770\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23878886   23319.225    22.773\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23880758   23321.053    22.774\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23883734   23323.959    22.777\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23884946   23325.143    22.778\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23887722   23327.854    22.781\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23890530   23330.596    22.784\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23893726   23333.717    22.787\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23895178   23335.135    22.788\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23896642   23336.564    22.790\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23898134   23338.021    22.791\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23901126   23340.943    22.794\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23902330   23342.119    22.795\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23904742   23344.475    22.797\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23907174   23346.850    22.800\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23909482   23349.104    22.802\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23910778   23350.369    22.803\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23913958   23353.475    22.806\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23917138   23356.580    22.809\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23920310   23359.678    22.812\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23923470   23362.764    22.815\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23925514   23364.760    22.817\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23928002   23367.189    22.820\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23929122   23368.283    22.821\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23930134   23369.271    22.822\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23931386   23370.494    22.823\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23932774   23371.850    22.824\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23936034   23375.033    22.827\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23939218   23378.143    22.830\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23942394   23381.244    22.833\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23945578   23384.354    22.836\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23946854   23385.600    22.837\n",
      "openf___224_> Open           buildprofile/EAO00186build_profile.ali\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22821735   22286.852    21.765\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26382858   25764.510    25.161\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26385100   25766.699    25.163\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26386550   25768.115    25.164\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26389450   25770.947    25.167\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26395250   25776.611    25.172\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     39833649   38900.047    37.988\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39845249   38911.375    37.999\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39868449   38934.031    38.022\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39914849   38979.344    38.066\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40007649   39069.969    38.154\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     46726849   45631.688    44.562\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00187.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22824055   22289.117    21.767\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22825505   22290.531    21.768\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22828405   22293.363    21.771\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22834205   22299.027    21.776\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22845805   22310.355    21.787\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22848261   22312.754    21.790\n",
      "\n",
      "Read the alignment from file       : ali/EAO00187.ali\n",
      "\n",
      "Total number of alignment positions:   615\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00187     615      1    EAO00187\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     22851054   22315.482    21.792\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23176229   22633.035    22.103\n",
      "profile_iteration_> processing sequence:       1    615      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23177307   22634.090    22.104\n",
      "profile_iteration_> processing sequence:    6720    615      1     6.0781250     0.0009045     0.0000035  1106\n",
      "profile_iteration_> processing sequence:   13439    615      1    11.8906250     0.0008848     0.0000035  1130\n",
      "profile_iteration_> processing sequence:   20158    615      1    17.7343750     0.0008798     0.0000035  1137\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23177682   22634.455    22.104\n",
      "profile_iteration_> processing sequence:   26877    615      1    23.7031250     0.0008819     0.0000035  1134\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23177918   22634.686    22.104\n",
      "profile_iteration_> processing sequence:   33596    615      1    29.6250000     0.0008818     0.0000035  1134\n",
      "profile_iteration_> processing sequence:   40315    615      1    35.4375000     0.0008790     0.0000035  1138\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23180868   22637.566    22.107\n",
      "profile_iteration_> processing sequence:   47034    615      1    41.3281250     0.0008787     0.0000035  1138\n",
      "profile_iteration_> processing sequence:   53753    615      1    47.3437500     0.0008808     0.0000035  1135\n",
      "profile_iteration_> processing sequence:   60472    615      1    53.1875000     0.0008795     0.0000034  1137\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23184378   22640.994    22.110\n",
      "profile_iteration_> processing sequence:   67191    615      1    59.2968750     0.0008825     0.0000034  1133\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    32.28801\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    273.0611\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0235,      121,        0.0070,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00074   0.00004\n",
      "Z:         1  -2.35000   0.00015   0.00016\n",
      "Z:         1  -2.25000   0.00104   0.00055\n",
      "Z:         1  -2.15000   0.00238   0.00163\n",
      "Z:         1  -2.05000   0.00223   0.00416\n",
      "Z:         1  -1.95000   0.00714   0.00933\n",
      "Z:         1  -1.85000   0.01816   0.01872\n",
      "Z:         1  -1.75000   0.03438   0.03400\n",
      "Z:         1  -1.65000   0.05834   0.05658\n",
      "Z:         1  -1.55000   0.08275   0.08721\n",
      "Z:         1  -1.45000   0.12487   0.12565\n",
      "Z:         1  -1.35000   0.16743   0.17059\n",
      "Z:         1  -1.25000   0.20077   0.21981\n",
      "Z:         1  -1.15000   0.27176   0.27052\n",
      "Z:         1  -1.05000   0.33903   0.31973\n",
      "Z:         1  -0.95000   0.35168   0.36469\n",
      "Z:         1  -0.85000   0.37862   0.40316\n",
      "Z:         1  -0.75000   0.43636   0.43360\n",
      "Z:         1  -0.65000   0.44589   0.45519\n",
      "Z:         1  -0.55000   0.45839   0.46779\n",
      "Z:         1  -0.45000   0.47342   0.47182\n",
      "Z:         1  -0.35000   0.49649   0.46811\n",
      "Z:         1  -0.25000   0.44484   0.45775\n",
      "Z:         1  -0.15000   0.52596   0.44196\n",
      "Z:         1  -0.05000   0.41597   0.42194\n",
      "Z:         1   0.05000   0.40005   0.39889\n",
      "Z:         1   0.15000   0.35897   0.37383\n",
      "Z:         1   0.25000   0.34751   0.34769\n",
      "Z:         1   0.35000   0.31611   0.32121\n",
      "Z:         1   0.45000   0.29393   0.29500\n",
      "Z:         1   0.55000   0.24973   0.26953\n",
      "Z:         1   0.65000   0.23768   0.24513\n",
      "Z:         1   0.75000   0.22309   0.22205\n",
      "Z:         1   0.85000   0.19095   0.20043\n",
      "Z:         1   0.95000   0.17934   0.18036\n",
      "Z:         1   1.05000   0.15791   0.16185\n",
      "Z:         1   1.15000   0.14972   0.14489\n",
      "Z:         1   1.25000   0.11966   0.12944\n",
      "Z:         1   1.35000   0.11147   0.11542\n",
      "Z:         1   1.45000   0.10820   0.10275\n",
      "Z:         1   1.55000   0.08305   0.09133\n",
      "Z:         1   1.65000   0.08007   0.08109\n",
      "Z:         1   1.75000   0.07471   0.07191\n",
      "Z:         1   1.85000   0.06563   0.06371\n",
      "Z:         1   1.95000   0.05373   0.05639\n",
      "Z:         1   2.05000   0.04941   0.04988\n",
      "Z:         1   2.15000   0.04450   0.04409\n",
      "Z:         1   2.25000   0.04152   0.03895\n",
      "Z:         1   2.35000   0.03899   0.03439\n",
      "Z:         1   2.45000   0.02798   0.03035\n",
      "Z:         1   2.55000   0.02991   0.02678\n",
      "Z:         1   2.65000   0.02471   0.02361\n",
      "Z:         1   2.75000   0.02307   0.02082\n",
      "Z:         1   2.85000   0.01979   0.01835\n",
      "Z:         1   2.95000   0.01801   0.01617\n",
      "Z:         1   3.05000   0.01339   0.01424\n",
      "Z:         1   3.15000   0.01592   0.01255\n",
      "Z:         1   3.25000   0.01101   0.01105\n",
      "Z:         1   3.35000   0.00997   0.00973\n",
      "Z:         1   3.45000   0.00804   0.00857\n",
      "Z:         1   3.55000   0.00714   0.00754\n",
      "Z:         1   3.65000   0.00848   0.00664\n",
      "Z:         1   3.75000   0.00848   0.00584\n",
      "Z:         1   3.85000   0.00744   0.00514\n",
      "Z:         1   3.95000   0.00476   0.00453\n",
      "Z:         1   4.05000   0.00461   0.00398\n",
      "Z:         1   4.15000   0.00491   0.00350\n",
      "Z:         1   4.25000   0.00238   0.00308\n",
      "Z:         1   4.35000   0.00432   0.00271\n",
      "Z:         1   4.45000   0.00313   0.00239\n",
      "Z:         1   4.55000   0.00387   0.00210\n",
      "Z:         1   4.65000   0.00253   0.00185\n",
      "Z:         1   4.75000   0.00238   0.00163\n",
      "Z:         1   4.85000   0.00208   0.00143\n",
      "Z:         1   4.95000   0.00179   0.00126\n",
      "Z:         1   5.05000   0.00164   0.00111\n",
      "Z:         1   5.15000   0.00223   0.00097\n",
      "Z:         1   5.25000   0.00104   0.00086\n",
      "Z:         1   5.35000   0.00030   0.00075\n",
      "Z:         1   5.45000   0.00119   0.00066\n",
      "Z:         1   5.55000   0.00089   0.00058\n",
      "Z:         1   5.65000   0.00030   0.00051\n",
      "Z:         1   5.75000   0.00030   0.00045\n",
      "Z:         1   5.85000   0.00045   0.00040\n",
      "Z:         1   5.95000   0.00104   0.00035\n",
      "Z:         1   6.05000   0.00045   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00060   0.00024\n",
      "Z:         1   6.35000   0.00045   0.00021\n",
      "Z:         1   6.45000   0.00045   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00045   0.00013\n",
      "Z:         1   6.85000   0.00030   0.00011\n",
      "Z:         1   6.95000   0.00030   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00015   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00030   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00030   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00030   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.015625\n",
      "profile_iteration__> Score Collection        [sec]:    59.281250\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000882\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1134.341309\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000884\n",
      "profile_iteration__> Overall performance [seq/sec]:  1131.654785\n",
      "openf___224_> Open           buildprofile/EAO00187build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22869841   22333.828    21.810\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22872297   22336.227    21.813\n",
      "openf___224_> Open           buildprofile/EAO00187build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22821735   22286.852    21.765\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26382858   25764.510    25.161\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26385100   25766.699    25.163\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26386550   25768.115    25.164\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26389450   25770.947    25.167\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26395250   25776.611    25.172\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     39833649   38900.047    37.988\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39845249   38911.375    37.999\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39868449   38934.031    38.022\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39914849   38979.344    38.066\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40007649   39069.969    38.154\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     46726849   45631.688    44.562\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00188.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22824055   22289.117    21.767\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22825505   22290.531    21.768\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22828405   22293.363    21.771\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22834205   22299.027    21.776\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22835253   22300.051    21.777\n",
      "\n",
      "Read the alignment from file       : ali/EAO00188.ali\n",
      "\n",
      "Total number of alignment positions:   263\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00188     263      1    EAO00188\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     22836638   22301.404    21.779\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23161813   22618.957    22.089\n",
      "profile_iteration_> processing sequence:       1    263      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23162891   22620.012    22.090\n",
      "profile_iteration_> processing sequence:    6720    263      1     2.5625000     0.0003813     0.0000015  2622\n",
      "profile_iteration_> processing sequence:   13439    263      1     5.0625000     0.0003767     0.0000015  2655\n",
      "profile_iteration_> processing sequence:   20158    263      1     7.4843750     0.0003713     0.0000015  2693\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23163266   22620.377    22.090\n",
      "profile_iteration_> processing sequence:   26877    263      1     9.9687500     0.0003709     0.0000015  2696\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23163502   22620.607    22.090\n",
      "profile_iteration_> processing sequence:   33596    263      1    12.5156250     0.0003725     0.0000015  2684\n",
      "profile_iteration_> processing sequence:   40315    263      1    15.0156250     0.0003725     0.0000015  2685\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23166452   22623.488    22.093\n",
      "profile_iteration_> processing sequence:   47034    263      1    17.5312500     0.0003727     0.0000015  2683\n",
      "profile_iteration_> processing sequence:   53753    263      1    20.0468750     0.0003729     0.0000015  2681\n",
      "profile_iteration_> processing sequence:   60472    263      1    22.4843750     0.0003718     0.0000015  2690\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23169962   22626.916    22.097\n",
      "profile_iteration_> processing sequence:   67191    263      1    25.0468750     0.0003728     0.0000015  2683\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    35.08734\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    239.1664\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0217,      121,        0.0082,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00015   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00015   0.00000\n",
      "Z:         1  -2.65000   0.00030   0.00000\n",
      "Z:         1  -2.55000   0.00030   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00104   0.00016\n",
      "Z:         1  -2.25000   0.00149   0.00055\n",
      "Z:         1  -2.15000   0.00223   0.00163\n",
      "Z:         1  -2.05000   0.00655   0.00416\n",
      "Z:         1  -1.95000   0.01176   0.00933\n",
      "Z:         1  -1.85000   0.02366   0.01872\n",
      "Z:         1  -1.75000   0.03870   0.03400\n",
      "Z:         1  -1.65000   0.05968   0.05658\n",
      "Z:         1  -1.55000   0.09733   0.08721\n",
      "Z:         1  -1.45000   0.11847   0.12565\n",
      "Z:         1  -1.35000   0.17591   0.17059\n",
      "Z:         1  -1.25000   0.20821   0.21981\n",
      "Z:         1  -1.15000   0.26342   0.27052\n",
      "Z:         1  -1.05000   0.31105   0.31973\n",
      "Z:         1  -0.95000   0.34468   0.36469\n",
      "Z:         1  -0.85000   0.36701   0.40316\n",
      "Z:         1  -0.75000   0.41508   0.43360\n",
      "Z:         1  -0.65000   0.49619   0.45519\n",
      "Z:         1  -0.55000   0.45735   0.46779\n",
      "Z:         1  -0.45000   0.51822   0.47182\n",
      "Z:         1  -0.35000   0.47848   0.46811\n",
      "Z:         1  -0.25000   0.45139   0.45775\n",
      "Z:         1  -0.15000   0.46285   0.44196\n",
      "Z:         1  -0.05000   0.39648   0.42194\n",
      "Z:         1   0.05000   0.37371   0.39889\n",
      "Z:         1   0.15000   0.35585   0.37383\n",
      "Z:         1   0.25000   0.35287   0.34769\n",
      "Z:         1   0.35000   0.31224   0.32121\n",
      "Z:         1   0.45000   0.29721   0.29500\n",
      "Z:         1   0.55000   0.26476   0.26953\n",
      "Z:         1   0.65000   0.23768   0.24513\n",
      "Z:         1   0.75000   0.21803   0.22205\n",
      "Z:         1   0.85000   0.20761   0.20043\n",
      "Z:         1   0.95000   0.17904   0.18036\n",
      "Z:         1   1.05000   0.16505   0.16185\n",
      "Z:         1   1.15000   0.15835   0.14489\n",
      "Z:         1   1.25000   0.13246   0.12944\n",
      "Z:         1   1.35000   0.11623   0.11542\n",
      "Z:         1   1.45000   0.10239   0.10275\n",
      "Z:         1   1.55000   0.09644   0.09133\n",
      "Z:         1   1.65000   0.08215   0.08109\n",
      "Z:         1   1.75000   0.07412   0.07191\n",
      "Z:         1   1.85000   0.06757   0.06371\n",
      "Z:         1   1.95000   0.06251   0.05639\n",
      "Z:         1   2.05000   0.04956   0.04988\n",
      "Z:         1   2.15000   0.04361   0.04409\n",
      "Z:         1   2.25000   0.03870   0.03895\n",
      "Z:         1   2.35000   0.03542   0.03439\n",
      "Z:         1   2.45000   0.02902   0.03035\n",
      "Z:         1   2.55000   0.02530   0.02678\n",
      "Z:         1   2.65000   0.02143   0.02361\n",
      "Z:         1   2.75000   0.02441   0.02082\n",
      "Z:         1   2.85000   0.01905   0.01835\n",
      "Z:         1   2.95000   0.01741   0.01617\n",
      "Z:         1   3.05000   0.01339   0.01424\n",
      "Z:         1   3.15000   0.01444   0.01255\n",
      "Z:         1   3.25000   0.01250   0.01105\n",
      "Z:         1   3.35000   0.00952   0.00973\n",
      "Z:         1   3.45000   0.00893   0.00857\n",
      "Z:         1   3.55000   0.00863   0.00754\n",
      "Z:         1   3.65000   0.00774   0.00664\n",
      "Z:         1   3.75000   0.00833   0.00584\n",
      "Z:         1   3.85000   0.00595   0.00514\n",
      "Z:         1   3.95000   0.00595   0.00453\n",
      "Z:         1   4.05000   0.00580   0.00398\n",
      "Z:         1   4.15000   0.00402   0.00350\n",
      "Z:         1   4.25000   0.00298   0.00308\n",
      "Z:         1   4.35000   0.00372   0.00271\n",
      "Z:         1   4.45000   0.00208   0.00239\n",
      "Z:         1   4.55000   0.00164   0.00210\n",
      "Z:         1   4.65000   0.00149   0.00185\n",
      "Z:         1   4.75000   0.00134   0.00163\n",
      "Z:         1   4.85000   0.00208   0.00143\n",
      "Z:         1   4.95000   0.00104   0.00126\n",
      "Z:         1   5.05000   0.00089   0.00111\n",
      "Z:         1   5.15000   0.00208   0.00097\n",
      "Z:         1   5.25000   0.00104   0.00086\n",
      "Z:         1   5.35000   0.00089   0.00075\n",
      "Z:         1   5.45000   0.00030   0.00066\n",
      "Z:         1   5.55000   0.00015   0.00058\n",
      "Z:         1   5.65000   0.00060   0.00051\n",
      "Z:         1   5.75000   0.00015   0.00045\n",
      "Z:         1   5.85000   0.00030   0.00040\n",
      "Z:         1   5.95000   0.00030   0.00035\n",
      "Z:         1   6.05000   0.00015   0.00031\n",
      "Z:         1   6.15000   0.00000   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00045   0.00018\n",
      "Z:         1   6.55000   0.00030   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00045   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00015   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     23397362   22848.986    22.313\n",
      "> 2ehoA                      1   28973    6700     161     263   35.17    0.27E-07       2   143    44   218     4   148\n",
      "> 2e9xD                      1   31167    7400     197     263   30.41    0.80E-09       3   184    44   263     4   197\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    25.046875\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000373\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  2691.043701\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000374\n",
      "profile_iteration__> Overall performance [seq/sec]:  2672.646484\n",
      "openf___224_> Open           buildprofile/EAO00188build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23078487   22537.586    22.009\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23079535   22538.609    22.010\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23080103   22539.164    22.011\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23080835   22539.879    22.012\n",
      "openf___224_> Open           buildprofile/EAO00188build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490041   21962.930    21.448\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22490569   21963.445    21.449\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22539169   22010.906    21.495\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22552469   22023.895    21.508\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22821735   22286.852    21.765\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26382858   25764.510    25.161\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26385100   25766.699    25.163\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26386550   25768.115    25.164\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26389450   25770.947    25.167\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26395250   25776.611    25.172\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     39833649   38900.047    37.988\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39845249   38911.375    37.999\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39868449   38934.031    38.022\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     39914849   38979.344    38.066\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40007649   39069.969    38.154\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     46726849   45631.688    44.562\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00189.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22824055   22289.117    21.767\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22825505   22290.531    21.768\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22828405   22293.363    21.771\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22834205   22299.027    21.776\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22845805   22310.355    21.787\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     22847417   22311.930    21.789\n",
      "\n",
      "Read the alignment from file       : ali/EAO00189.ali\n",
      "\n",
      "Total number of alignment positions:   404\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00189     404      1    EAO00189\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     22849366   22313.834    21.791\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23174541   22631.387    22.101\n",
      "profile_iteration_> processing sequence:       1    404      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23175619   22632.441    22.102\n",
      "profile_iteration_> processing sequence:    6720    404      1     3.8593750     0.0005743     0.0000022  1741\n",
      "profile_iteration_> processing sequence:   13439    404      1     7.5937500     0.0005651     0.0000022  1770\n",
      "profile_iteration_> processing sequence:   20158    404      1    11.3125000     0.0005612     0.0000022  1782\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23175994   22632.807    22.102\n",
      "profile_iteration_> processing sequence:   26877    404      1    15.1093750     0.0005622     0.0000022  1779\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23176230   22633.037    22.103\n",
      "profile_iteration_> processing sequence:   33596    404      1    18.8906250     0.0005623     0.0000022  1778\n",
      "profile_iteration_> processing sequence:   40315    404      1    22.6093750     0.0005608     0.0000022  1783\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23179180   22635.918    22.105\n",
      "profile_iteration_> processing sequence:   47034    404      1    26.4062500     0.0005614     0.0000022  1781\n",
      "profile_iteration_> processing sequence:   53753    404      1    30.2812500     0.0005633     0.0000022  1775\n",
      "profile_iteration_> processing sequence:   60472    404      1    34.0312500     0.0005628     0.0000022  1777\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     23182690   22639.346    22.109\n",
      "profile_iteration_> processing sequence:   67191    404      1    37.9687500     0.0005651     0.0000022  1770\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    29.69237\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    238.6997\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0298,      121,        0.0100,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00015   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00015   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00030   0.00001\n",
      "Z:         1  -2.45000   0.00015   0.00004\n",
      "Z:         1  -2.35000   0.00074   0.00016\n",
      "Z:         1  -2.25000   0.00119   0.00055\n",
      "Z:         1  -2.15000   0.00446   0.00163\n",
      "Z:         1  -2.05000   0.00580   0.00416\n",
      "Z:         1  -1.95000   0.01265   0.00933\n",
      "Z:         1  -1.85000   0.02069   0.01872\n",
      "Z:         1  -1.75000   0.03497   0.03400\n",
      "Z:         1  -1.65000   0.06161   0.05658\n",
      "Z:         1  -1.55000   0.08662   0.08721\n",
      "Z:         1  -1.45000   0.12606   0.12565\n",
      "Z:         1  -1.35000   0.16966   0.17059\n",
      "Z:         1  -1.25000   0.21208   0.21981\n",
      "Z:         1  -1.15000   0.27295   0.27052\n",
      "Z:         1  -1.05000   0.32236   0.31973\n",
      "Z:         1  -0.95000   0.37475   0.36469\n",
      "Z:         1  -0.85000   0.42193   0.40316\n",
      "Z:         1  -0.75000   0.42624   0.43360\n",
      "Z:         1  -0.65000   0.45571   0.45519\n",
      "Z:         1  -0.55000   0.46345   0.46779\n",
      "Z:         1  -0.45000   0.46538   0.47182\n",
      "Z:         1  -0.35000   0.46330   0.46811\n",
      "Z:         1  -0.25000   0.44038   0.45775\n",
      "Z:         1  -0.15000   0.41374   0.44196\n",
      "Z:         1  -0.05000   0.41374   0.42194\n",
      "Z:         1   0.05000   0.38918   0.39889\n",
      "Z:         1   0.15000   0.34811   0.37383\n",
      "Z:         1   0.25000   0.33203   0.34769\n",
      "Z:         1   0.35000   0.32236   0.32121\n",
      "Z:         1   0.45000   0.30078   0.29500\n",
      "Z:         1   0.55000   0.27369   0.26953\n",
      "Z:         1   0.65000   0.24140   0.24513\n",
      "Z:         1   0.75000   0.21476   0.22205\n",
      "Z:         1   0.85000   0.19779   0.20043\n",
      "Z:         1   0.95000   0.19080   0.18036\n",
      "Z:         1   1.05000   0.15657   0.16185\n",
      "Z:         1   1.15000   0.14213   0.14489\n",
      "Z:         1   1.25000   0.15106   0.12944\n",
      "Z:         1   1.35000   0.11683   0.11542\n",
      "Z:         1   1.45000   0.10924   0.10275\n",
      "Z:         1   1.55000   0.08855   0.09133\n",
      "Z:         1   1.65000   0.13707   0.08109\n",
      "Z:         1   1.75000   0.06920   0.07191\n",
      "Z:         1   1.85000   0.06429   0.06371\n",
      "Z:         1   1.95000   0.06013   0.05639\n",
      "Z:         1   2.05000   0.04629   0.04988\n",
      "Z:         1   2.15000   0.04078   0.04409\n",
      "Z:         1   2.25000   0.04093   0.03895\n",
      "Z:         1   2.35000   0.03795   0.03439\n",
      "Z:         1   2.45000   0.02738   0.03035\n",
      "Z:         1   2.55000   0.02768   0.02678\n",
      "Z:         1   2.65000   0.02411   0.02361\n",
      "Z:         1   2.75000   0.02128   0.02082\n",
      "Z:         1   2.85000   0.01786   0.01835\n",
      "Z:         1   2.95000   0.01518   0.01617\n",
      "Z:         1   3.05000   0.01399   0.01424\n",
      "Z:         1   3.15000   0.01057   0.01255\n",
      "Z:         1   3.25000   0.01042   0.01105\n",
      "Z:         1   3.35000   0.00863   0.00973\n",
      "Z:         1   3.45000   0.01072   0.00857\n",
      "Z:         1   3.55000   0.00819   0.00754\n",
      "Z:         1   3.65000   0.00744   0.00664\n",
      "Z:         1   3.75000   0.00878   0.00584\n",
      "Z:         1   3.85000   0.00640   0.00514\n",
      "Z:         1   3.95000   0.00461   0.00453\n",
      "Z:         1   4.05000   0.00357   0.00398\n",
      "Z:         1   4.15000   0.00446   0.00350\n",
      "Z:         1   4.25000   0.00417   0.00308\n",
      "Z:         1   4.35000   0.00238   0.00271\n",
      "Z:         1   4.45000   0.00238   0.00239\n",
      "Z:         1   4.55000   0.00149   0.00210\n",
      "Z:         1   4.65000   0.00164   0.00185\n",
      "Z:         1   4.75000   0.00193   0.00163\n",
      "Z:         1   4.85000   0.00193   0.00143\n",
      "Z:         1   4.95000   0.00074   0.00126\n",
      "Z:         1   5.05000   0.00074   0.00111\n",
      "Z:         1   5.15000   0.00089   0.00097\n",
      "Z:         1   5.25000   0.00045   0.00086\n",
      "Z:         1   5.35000   0.00060   0.00075\n",
      "Z:         1   5.45000   0.00045   0.00066\n",
      "Z:         1   5.55000   0.00074   0.00058\n",
      "Z:         1   5.65000   0.00030   0.00051\n",
      "Z:         1   5.75000   0.00074   0.00045\n",
      "Z:         1   5.85000   0.00000   0.00040\n",
      "Z:         1   5.95000   0.00030   0.00035\n",
      "Z:         1   6.05000   0.00000   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00015   0.00018\n",
      "Z:         1   6.55000   0.00074   0.00016\n",
      "Z:         1   6.65000   0.00045   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00030   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00015   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00030   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00015   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00015   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    37.968750\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000565\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1771.123535\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000566\n",
      "profile_iteration__> Overall performance [seq/sec]:  1766.031982\n",
      "openf___224_> Open           buildprofile/EAO00189build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5833075    5696.362     5.563\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5834687    5697.937     5.564\n",
      "openf___224_> Open           buildprofile/EAO00189build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461082    5333.088     5.208\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461610    5333.604     5.209\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5510210    5381.064     5.255\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5523510    5394.053     5.268\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      5792776    5657.008     5.524\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      9353899    9134.667     8.921\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9356141    9136.856     8.923\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9357591    9138.272     8.924\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9360491    9141.104     8.927\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9366291    9146.769     8.932\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22804690   22270.205    21.748\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22816290   22281.533    21.759\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22839490   22304.189    21.781\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22885890   22349.502    21.826\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22978690   22440.127    21.914\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     29697890   29001.846    28.322\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00190.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5795096    5659.273     5.527\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5796546    5660.689     5.528\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5799446    5663.521     5.531\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5805246    5669.186     5.536\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5806446    5670.357     5.537\n",
      "\n",
      "Read the alignment from file       : ali/EAO00190.ali\n",
      "\n",
      "Total number of alignment positions:   301\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00190     301      1    EAO00190\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      5807983    5671.858     5.539\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6133158    5989.412     5.849\n",
      "profile_iteration_> processing sequence:       1    301      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6134236    5990.465     5.850\n",
      "profile_iteration_> processing sequence:    6720    301      1     2.9062500     0.0004325     0.0000017  2312\n",
      "profile_iteration_> processing sequence:   13439    301      1     5.7187500     0.0004255     0.0000017  2350\n",
      "profile_iteration_> processing sequence:   20158    301      1     8.5312500     0.0004232     0.0000017  2363\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6134611    5990.831     5.850\n",
      "profile_iteration_> processing sequence:   26877    301      1    11.3593750     0.0004226     0.0000017  2366\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6134847    5991.062     5.851\n",
      "profile_iteration_> processing sequence:   33596    301      1    14.2187500     0.0004232     0.0000017  2363\n",
      "profile_iteration_> processing sequence:   40315    301      1    17.0312500     0.0004225     0.0000017  2367\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6137797    5993.942     5.853\n",
      "profile_iteration_> processing sequence:   47034    301      1    19.8906250     0.0004229     0.0000017  2365\n",
      "profile_iteration_> processing sequence:   53753    301      1    22.8437500     0.0004250     0.0000017  2353\n",
      "profile_iteration_> processing sequence:   60472    301      1    25.7031250     0.0004250     0.0000017  2353\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6141307    5997.370     5.857\n",
      "profile_iteration_> processing sequence:   67191    301      1    28.6718750     0.0004267     0.0000017  2343\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    25.22938\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    226.3142\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0197,      121,        0.0053,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00015   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00015   0.00000\n",
      "Z:         1  -2.75000   0.00030   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00045   0.00016\n",
      "Z:         1  -2.25000   0.00164   0.00055\n",
      "Z:         1  -2.15000   0.00253   0.00163\n",
      "Z:         1  -2.05000   0.00625   0.00416\n",
      "Z:         1  -1.95000   0.01086   0.00933\n",
      "Z:         1  -1.85000   0.02337   0.01872\n",
      "Z:         1  -1.75000   0.04197   0.03400\n",
      "Z:         1  -1.65000   0.06801   0.05658\n",
      "Z:         1  -1.55000   0.07739   0.08721\n",
      "Z:         1  -1.45000   0.13052   0.12565\n",
      "Z:         1  -1.35000   0.15180   0.17059\n",
      "Z:         1  -1.25000   0.21952   0.21981\n",
      "Z:         1  -1.15000   0.25583   0.27052\n",
      "Z:         1  -1.05000   0.33278   0.31973\n",
      "Z:         1  -0.95000   0.37073   0.36469\n",
      "Z:         1  -0.85000   0.38561   0.40316\n",
      "Z:         1  -0.75000   0.43487   0.43360\n",
      "Z:         1  -0.65000   0.42579   0.45519\n",
      "Z:         1  -0.55000   0.51078   0.46779\n",
      "Z:         1  -0.45000   0.45437   0.47182\n",
      "Z:         1  -0.35000   0.47952   0.46811\n",
      "Z:         1  -0.25000   0.43889   0.45775\n",
      "Z:         1  -0.15000   0.43874   0.44196\n",
      "Z:         1  -0.05000   0.41880   0.42194\n",
      "Z:         1   0.05000   0.39707   0.39889\n",
      "Z:         1   0.15000   0.37504   0.37383\n",
      "Z:         1   0.25000   0.33144   0.34769\n",
      "Z:         1   0.35000   0.31328   0.32121\n",
      "Z:         1   0.45000   0.29438   0.29500\n",
      "Z:         1   0.55000   0.26908   0.26953\n",
      "Z:         1   0.65000   0.24810   0.24513\n",
      "Z:         1   0.75000   0.22399   0.22205\n",
      "Z:         1   0.85000   0.20568   0.20043\n",
      "Z:         1   0.95000   0.18589   0.18036\n",
      "Z:         1   1.05000   0.15567   0.16185\n",
      "Z:         1   1.15000   0.14570   0.14489\n",
      "Z:         1   1.25000   0.13781   0.12944\n",
      "Z:         1   1.35000   0.11445   0.11542\n",
      "Z:         1   1.45000   0.10760   0.10275\n",
      "Z:         1   1.55000   0.09942   0.09133\n",
      "Z:         1   1.65000   0.08096   0.08109\n",
      "Z:         1   1.75000   0.07397   0.07191\n",
      "Z:         1   1.85000   0.06414   0.06371\n",
      "Z:         1   1.95000   0.05953   0.05639\n",
      "Z:         1   2.05000   0.04926   0.04988\n",
      "Z:         1   2.15000   0.04480   0.04409\n",
      "Z:         1   2.25000   0.03706   0.03895\n",
      "Z:         1   2.35000   0.03378   0.03439\n",
      "Z:         1   2.45000   0.02857   0.03035\n",
      "Z:         1   2.55000   0.02619   0.02678\n",
      "Z:         1   2.65000   0.02292   0.02361\n",
      "Z:         1   2.75000   0.02530   0.02082\n",
      "Z:         1   2.85000   0.02158   0.01835\n",
      "Z:         1   2.95000   0.01652   0.01617\n",
      "Z:         1   3.05000   0.01131   0.01424\n",
      "Z:         1   3.15000   0.01220   0.01255\n",
      "Z:         1   3.25000   0.00848   0.01105\n",
      "Z:         1   3.35000   0.01101   0.00973\n",
      "Z:         1   3.45000   0.00967   0.00857\n",
      "Z:         1   3.55000   0.00997   0.00754\n",
      "Z:         1   3.65000   0.00878   0.00664\n",
      "Z:         1   3.75000   0.00670   0.00584\n",
      "Z:         1   3.85000   0.00551   0.00514\n",
      "Z:         1   3.95000   0.00357   0.00453\n",
      "Z:         1   4.05000   0.00491   0.00398\n",
      "Z:         1   4.15000   0.00402   0.00350\n",
      "Z:         1   4.25000   0.00298   0.00308\n",
      "Z:         1   4.35000   0.00268   0.00271\n",
      "Z:         1   4.45000   0.00342   0.00239\n",
      "Z:         1   4.55000   0.00357   0.00210\n",
      "Z:         1   4.65000   0.00313   0.00185\n",
      "Z:         1   4.75000   0.00238   0.00163\n",
      "Z:         1   4.85000   0.00193   0.00143\n",
      "Z:         1   4.95000   0.00089   0.00126\n",
      "Z:         1   5.05000   0.00089   0.00111\n",
      "Z:         1   5.15000   0.00164   0.00097\n",
      "Z:         1   5.25000   0.00089   0.00086\n",
      "Z:         1   5.35000   0.00104   0.00075\n",
      "Z:         1   5.45000   0.00015   0.00066\n",
      "Z:         1   5.55000   0.00089   0.00058\n",
      "Z:         1   5.65000   0.00074   0.00051\n",
      "Z:         1   5.75000   0.00030   0.00045\n",
      "Z:         1   5.85000   0.00030   0.00040\n",
      "Z:         1   5.95000   0.00060   0.00035\n",
      "Z:         1   6.05000   0.00045   0.00031\n",
      "Z:         1   6.15000   0.00030   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00000   0.00021\n",
      "Z:         1   6.45000   0.00030   0.00018\n",
      "Z:         1   6.55000   0.00074   0.00016\n",
      "Z:         1   6.65000   0.00060   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00045   0.00009\n",
      "Z:         1   7.15000   0.00060   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    28.671875\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000427\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  2347.318848\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000428\n",
      "profile_iteration__> Overall performance [seq/sec]:  2337.113037\n",
      "openf___224_> Open           buildprofile/EAO00190build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5817664    5681.312     5.548\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5818864    5682.484     5.549\n",
      "openf___224_> Open           buildprofile/EAO00190build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461082    5333.088     5.208\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461610    5333.604     5.209\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5510210    5381.064     5.255\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5523510    5394.053     5.268\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      5792776    5657.008     5.524\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      9353899    9134.667     8.921\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9356141    9136.856     8.923\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9357591    9138.272     8.924\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9360491    9141.104     8.927\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9366291    9146.769     8.932\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22804690   22270.205    21.748\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22816290   22281.533    21.759\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22839490   22304.189    21.781\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22885890   22349.502    21.826\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22978690   22440.127    21.914\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     29697890   29001.846    28.322\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00191.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5795096    5659.273     5.527\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5796546    5660.689     5.528\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5799446    5663.521     5.531\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5805246    5669.186     5.536\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5806102    5670.021     5.537\n",
      "\n",
      "Read the alignment from file       : ali/EAO00191.ali\n",
      "\n",
      "Total number of alignment positions:   215\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00191     215      1    EAO00191\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      5807295    5671.187     5.538\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6132470    5988.740     5.848\n",
      "profile_iteration_> processing sequence:       1    215      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6133548    5989.793     5.849\n",
      "profile_iteration_> processing sequence:    6720    215      1     2.0625000     0.0003069     0.0000012  3258\n",
      "profile_iteration_> processing sequence:   13439    215      1     4.0468750     0.0003011     0.0000012  3321\n",
      "profile_iteration_> processing sequence:   20158    215      1     6.0312500     0.0002992     0.0000012  3342\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6133923    5990.159     5.850\n",
      "profile_iteration_> processing sequence:   26877    215      1     8.0468750     0.0002994     0.0000012  3340\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6134159    5990.390     5.850\n",
      "profile_iteration_> processing sequence:   33596    215      1    10.0625000     0.0002995     0.0000012  3339\n",
      "profile_iteration_> processing sequence:   40315    215      1    12.0625000     0.0002992     0.0000012  3342\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6137109    5993.271     5.853\n",
      "profile_iteration_> processing sequence:   47034    215      1    14.0625000     0.0002990     0.0000012  3345\n",
      "profile_iteration_> processing sequence:   53753    215      1    16.1250000     0.0003000     0.0000012  3334\n",
      "profile_iteration_> processing sequence:   60472    215      1    18.1250000     0.0002997     0.0000012  3336\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6140619    5996.698     5.856\n",
      "profile_iteration_> processing sequence:   67191    215      1    20.2187500     0.0003009     0.0000012  3323\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    18.56558\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    231.6361\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0211,      121,        0.0139,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00015   0.00016\n",
      "Z:         1  -2.25000   0.00045   0.00055\n",
      "Z:         1  -2.15000   0.00268   0.00163\n",
      "Z:         1  -2.05000   0.00417   0.00416\n",
      "Z:         1  -1.95000   0.01027   0.00933\n",
      "Z:         1  -1.85000   0.01816   0.01872\n",
      "Z:         1  -1.75000   0.03691   0.03400\n",
      "Z:         1  -1.65000   0.05834   0.05658\n",
      "Z:         1  -1.55000   0.09108   0.08721\n",
      "Z:         1  -1.45000   0.12531   0.12565\n",
      "Z:         1  -1.35000   0.16535   0.17059\n",
      "Z:         1  -1.25000   0.22041   0.21981\n",
      "Z:         1  -1.15000   0.25941   0.27052\n",
      "Z:         1  -1.05000   0.30167   0.31973\n",
      "Z:         1  -0.95000   0.35957   0.36469\n",
      "Z:         1  -0.85000   0.38963   0.40316\n",
      "Z:         1  -0.75000   0.40347   0.43360\n",
      "Z:         1  -0.65000   0.44440   0.45519\n",
      "Z:         1  -0.55000   0.46270   0.46779\n",
      "Z:         1  -0.45000   0.47297   0.47182\n",
      "Z:         1  -0.35000   0.45303   0.46811\n",
      "Z:         1  -0.25000   0.47521   0.45775\n",
      "Z:         1  -0.15000   0.42728   0.44196\n",
      "Z:         1  -0.05000   0.42118   0.42194\n",
      "Z:         1   0.05000   0.39677   0.39889\n",
      "Z:         1   0.15000   0.36894   0.37383\n",
      "Z:         1   0.25000   0.33724   0.34769\n",
      "Z:         1   0.35000   0.32162   0.32121\n",
      "Z:         1   0.45000   0.28113   0.29500\n",
      "Z:         1   0.55000   0.28218   0.26953\n",
      "Z:         1   0.65000   0.24780   0.24513\n",
      "Z:         1   0.75000   0.23158   0.22205\n",
      "Z:         1   0.85000   0.20360   0.20043\n",
      "Z:         1   0.95000   0.17160   0.18036\n",
      "Z:         1   1.05000   0.16148   0.16185\n",
      "Z:         1   1.15000   0.15017   0.14489\n",
      "Z:         1   1.25000   0.13008   0.12944\n",
      "Z:         1   1.35000   0.11177   0.11542\n",
      "Z:         1   1.45000   0.11132   0.10275\n",
      "Z:         1   1.55000   0.09123   0.09133\n",
      "Z:         1   1.65000   0.09093   0.08109\n",
      "Z:         1   1.75000   0.07441   0.07191\n",
      "Z:         1   1.85000   0.07531   0.06371\n",
      "Z:         1   1.95000   0.06295   0.05639\n",
      "Z:         1   2.05000   0.07426   0.04988\n",
      "Z:         1   2.15000   0.05045   0.04409\n",
      "Z:         1   2.25000   0.04480   0.03895\n",
      "Z:         1   2.35000   0.03483   0.03439\n",
      "Z:         1   2.45000   0.03438   0.03035\n",
      "Z:         1   2.55000   0.02872   0.02678\n",
      "Z:         1   2.65000   0.02456   0.02361\n",
      "Z:         1   2.75000   0.02456   0.02082\n",
      "Z:         1   2.85000   0.01801   0.01835\n",
      "Z:         1   2.95000   0.02456   0.01617\n",
      "Z:         1   3.05000   0.01295   0.01424\n",
      "Z:         1   3.15000   0.01682   0.01255\n",
      "Z:         1   3.25000   0.01161   0.01105\n",
      "Z:         1   3.35000   0.01206   0.00973\n",
      "Z:         1   3.45000   0.01116   0.00857\n",
      "Z:         1   3.55000   0.01042   0.00754\n",
      "Z:         1   3.65000   0.00804   0.00664\n",
      "Z:         1   3.75000   0.00714   0.00584\n",
      "Z:         1   3.85000   0.00610   0.00514\n",
      "Z:         1   3.95000   0.00387   0.00453\n",
      "Z:         1   4.05000   0.00461   0.00398\n",
      "Z:         1   4.15000   0.00283   0.00350\n",
      "Z:         1   4.25000   0.00387   0.00308\n",
      "Z:         1   4.35000   0.00179   0.00271\n",
      "Z:         1   4.45000   0.00283   0.00239\n",
      "Z:         1   4.55000   0.00283   0.00210\n",
      "Z:         1   4.65000   0.00193   0.00185\n",
      "Z:         1   4.75000   0.00179   0.00163\n",
      "Z:         1   4.85000   0.00104   0.00143\n",
      "Z:         1   4.95000   0.00134   0.00126\n",
      "Z:         1   5.05000   0.00193   0.00111\n",
      "Z:         1   5.15000   0.00104   0.00097\n",
      "Z:         1   5.25000   0.00074   0.00086\n",
      "Z:         1   5.35000   0.00089   0.00075\n",
      "Z:         1   5.45000   0.00060   0.00066\n",
      "Z:         1   5.55000   0.00030   0.00058\n",
      "Z:         1   5.65000   0.00045   0.00051\n",
      "Z:         1   5.75000   0.00060   0.00045\n",
      "Z:         1   5.85000   0.00119   0.00040\n",
      "Z:         1   5.95000   0.00089   0.00035\n",
      "Z:         1   6.05000   0.00015   0.00031\n",
      "Z:         1   6.15000   0.00000   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00000   0.00021\n",
      "Z:         1   6.45000   0.00030   0.00018\n",
      "Z:         1   6.55000   0.00000   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    20.218750\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000301\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  3330.974365\n",
      "profile_iteration__> Statistics              [sec]:     0.062500\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000303\n",
      "profile_iteration__> Overall performance [seq/sec]:  3305.371338\n",
      "openf___224_> Open           buildprofile/EAO00191build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5814482    5678.205     5.545\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5815338    5679.041     5.546\n",
      "openf___224_> Open           buildprofile/EAO00191build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461082    5333.088     5.208\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461610    5333.604     5.209\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5510210    5381.064     5.255\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5523510    5394.053     5.268\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      5792776    5657.008     5.524\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      9353899    9134.667     8.921\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9356141    9136.856     8.923\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9357591    9138.272     8.924\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9360491    9141.104     8.927\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      9366291    9146.769     8.932\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22804690   22270.205    21.748\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22816290   22281.533    21.759\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22839490   22304.189    21.781\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22885890   22349.502    21.826\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22978690   22440.127    21.914\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     29697890   29001.846    28.322\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00192.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5795096    5659.273     5.527\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5796546    5660.689     5.528\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5799446    5663.521     5.531\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5805246    5669.186     5.536\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5816846    5680.514     5.547\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5819754    5683.354     5.550\n",
      "\n",
      "Read the alignment from file       : ali/EAO00192.ali\n",
      "\n",
      "Total number of alignment positions:   728\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00192     728      1    EAO00192\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      5822999    5686.522     5.553\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6148174    6004.076     5.863\n",
      "profile_iteration_> processing sequence:       1    728      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6149252    6005.129     5.864\n",
      "profile_iteration_> processing sequence:    6720    728      1     6.9687500     0.0010370     0.0000040   964\n",
      "profile_iteration_> processing sequence:   13439    728      1    13.7343750     0.0010220     0.0000040   978\n",
      "profile_iteration_> processing sequence:   20158    728      1    20.4843750     0.0010162     0.0000040   984\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6149627    6005.495     5.865\n",
      "profile_iteration_> processing sequence:   26877    728      1    27.3281250     0.0010168     0.0000040   983\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6149863    6005.726     5.865\n",
      "profile_iteration_> processing sequence:   33596    728      1    34.2187500     0.0010185     0.0000040   982\n",
      "profile_iteration_> processing sequence:   40315    728      1    41.0000000     0.0010170     0.0000040   983\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6152813    6008.606     5.868\n",
      "profile_iteration_> processing sequence:   47034    728      1    47.8593750     0.0010175     0.0000040   983\n",
      "profile_iteration_> processing sequence:   53753    728      1    54.8593750     0.0010206     0.0000040   980\n",
      "profile_iteration_> processing sequence:   60472    728      1    61.6875000     0.0010201     0.0000040   980\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      6156323    6012.034     5.871\n",
      "profile_iteration_> processing sequence:   67191    728      1    68.7656250     0.0010234     0.0000040   977\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    24.52570\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    245.5972\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0472,      121,        0.0175,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00045   0.00016\n",
      "Z:         1  -2.25000   0.00238   0.00055\n",
      "Z:         1  -2.15000   0.00342   0.00163\n",
      "Z:         1  -2.05000   0.00506   0.00416\n",
      "Z:         1  -1.95000   0.01146   0.00933\n",
      "Z:         1  -1.85000   0.02337   0.01872\n",
      "Z:         1  -1.75000   0.03914   0.03400\n",
      "Z:         1  -1.65000   0.05938   0.05658\n",
      "Z:         1  -1.55000   0.08706   0.08721\n",
      "Z:         1  -1.45000   0.13082   0.12565\n",
      "Z:         1  -1.35000   0.17576   0.17059\n",
      "Z:         1  -1.25000   0.22547   0.21981\n",
      "Z:         1  -1.15000   0.27116   0.27052\n",
      "Z:         1  -1.05000   0.33203   0.31973\n",
      "Z:         1  -0.95000   0.36373   0.36469\n",
      "Z:         1  -0.85000   0.38353   0.40316\n",
      "Z:         1  -0.75000   0.45214   0.43360\n",
      "Z:         1  -0.65000   0.45794   0.45519\n",
      "Z:         1  -0.55000   0.44886   0.46779\n",
      "Z:         1  -0.45000   0.46598   0.47182\n",
      "Z:         1  -0.35000   0.46434   0.46811\n",
      "Z:         1  -0.25000   0.44217   0.45775\n",
      "Z:         1  -0.15000   0.41687   0.44196\n",
      "Z:         1  -0.05000   0.41984   0.42194\n",
      "Z:         1   0.05000   0.36358   0.39889\n",
      "Z:         1   0.15000   0.36686   0.37383\n",
      "Z:         1   0.25000   0.31685   0.34769\n",
      "Z:         1   0.35000   0.30316   0.32121\n",
      "Z:         1   0.45000   0.27325   0.29500\n",
      "Z:         1   0.55000   0.28947   0.26953\n",
      "Z:         1   0.65000   0.22056   0.24513\n",
      "Z:         1   0.75000   0.21029   0.22205\n",
      "Z:         1   0.85000   0.18023   0.20043\n",
      "Z:         1   0.95000   0.20955   0.18036\n",
      "Z:         1   1.05000   0.16431   0.16185\n",
      "Z:         1   1.15000   0.14451   0.14489\n",
      "Z:         1   1.25000   0.17428   0.12944\n",
      "Z:         1   1.35000   0.19154   0.11542\n",
      "Z:         1   1.45000   0.11281   0.10275\n",
      "Z:         1   1.55000   0.08989   0.09133\n",
      "Z:         1   1.65000   0.08200   0.08109\n",
      "Z:         1   1.75000   0.07099   0.07191\n",
      "Z:         1   1.85000   0.05655   0.06371\n",
      "Z:         1   1.95000   0.05908   0.05639\n",
      "Z:         1   2.05000   0.04926   0.04988\n",
      "Z:         1   2.15000   0.04599   0.04409\n",
      "Z:         1   2.25000   0.03855   0.03895\n",
      "Z:         1   2.35000   0.03631   0.03439\n",
      "Z:         1   2.45000   0.03051   0.03035\n",
      "Z:         1   2.55000   0.02604   0.02678\n",
      "Z:         1   2.65000   0.02262   0.02361\n",
      "Z:         1   2.75000   0.02500   0.02082\n",
      "Z:         1   2.85000   0.01771   0.01835\n",
      "Z:         1   2.95000   0.01667   0.01617\n",
      "Z:         1   3.05000   0.01444   0.01424\n",
      "Z:         1   3.15000   0.01280   0.01255\n",
      "Z:         1   3.25000   0.00938   0.01105\n",
      "Z:         1   3.35000   0.00804   0.00973\n",
      "Z:         1   3.45000   0.00744   0.00857\n",
      "Z:         1   3.55000   0.00789   0.00754\n",
      "Z:         1   3.65000   0.00625   0.00664\n",
      "Z:         1   3.75000   0.00536   0.00584\n",
      "Z:         1   3.85000   0.00536   0.00514\n",
      "Z:         1   3.95000   0.00521   0.00453\n",
      "Z:         1   4.05000   0.00298   0.00398\n",
      "Z:         1   4.15000   0.00476   0.00350\n",
      "Z:         1   4.25000   0.00342   0.00308\n",
      "Z:         1   4.35000   0.00283   0.00271\n",
      "Z:         1   4.45000   0.00223   0.00239\n",
      "Z:         1   4.55000   0.00164   0.00210\n",
      "Z:         1   4.65000   0.00193   0.00185\n",
      "Z:         1   4.75000   0.00149   0.00163\n",
      "Z:         1   4.85000   0.00134   0.00143\n",
      "Z:         1   4.95000   0.00164   0.00126\n",
      "Z:         1   5.05000   0.00119   0.00111\n",
      "Z:         1   5.15000   0.00074   0.00097\n",
      "Z:         1   5.25000   0.00104   0.00086\n",
      "Z:         1   5.35000   0.00060   0.00075\n",
      "Z:         1   5.45000   0.00089   0.00066\n",
      "Z:         1   5.55000   0.00089   0.00058\n",
      "Z:         1   5.65000   0.00074   0.00051\n",
      "Z:         1   5.75000   0.00104   0.00045\n",
      "Z:         1   5.85000   0.00045   0.00040\n",
      "Z:         1   5.95000   0.00074   0.00035\n",
      "Z:         1   6.05000   0.00030   0.00031\n",
      "Z:         1   6.15000   0.00060   0.00027\n",
      "Z:         1   6.25000   0.00015   0.00024\n",
      "Z:         1   6.35000   0.00045   0.00021\n",
      "Z:         1   6.45000   0.00045   0.00018\n",
      "Z:         1   6.55000   0.00030   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00015   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00015   0.00009\n",
      "Z:         1   7.15000   0.00074   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00030   0.00004\n",
      "Z:         1   7.65000   0.00030   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00045   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      6755723    6597.386     6.443\n",
      "> 4fk5A                      1    1548    8450     437     728   29.74    0.45E-10       2   293   192   566    60   365\n",
      "> 5k1cA                      1    3339    5800     302     728   25.88    0.31E-04       3   223   306   578     4   231\n",
      "> 5k16A                      1    3394    6400     324     728   25.32    0.15E-05       4   232   306   578    20   256\n",
      "> 5k1bA                      1    3420    5150     264     728   28.89    0.80E-03       5   135   306   468     1   135\n",
      "> 6h4jA                      1    4156    5100     488     728   22.36    0.20E-02       6   349   306   710    16   422\n",
      "> 6gh9B                      1    5192    5900     309     728   30.93    0.19E-04       7   209   302   568     4   239\n",
      "> 6ghaA                      1    5193    6500     325     728   27.71    0.86E-06       8   222   302   568     3   251\n",
      "> 4w4uD                      1   10588    5800     409     728   27.40    0.42E-04       9   272   192   566    58   338\n",
      "> 3phdC                      1   11866    5300      85     728   35.14    0.11E-03      10    74   157   243     3    76\n",
      "> 6jlqA                      1   12824    5400     292     728   28.99    0.24E-03      11   202   306   565     2   208\n",
      "> 5g0fA                      1   14226    4850     110     728   30.77    0.15E-02      12    89   157   258     8    98\n",
      "> 2y6eA                      1   23882    6700     336     728   27.06    0.31E-06      13   226   302   561     3   257\n",
      "> 2y6eD                      1   23883    5950     327     728   27.87    0.15E-04      14   220   302   561     3   246\n",
      "> 6helA                      1   24305    5150     465     728   26.55    0.14E-02      15   158   519   710   219   395\n",
      "> 6helB                      1   24306    5100     463     728   26.40    0.19E-02      16   158   519   710   218   395\n",
      "> 5txkA                      1   24489    6050     317     728   27.02    0.88E-05      17   241   303   580     6   253\n",
      "> 2y5bA                      1   26462   10100     315     728   32.08     0.0          18   223   301   562     6   245\n",
      "> 3mhsA                      1   26781    9100     455     728   29.02     0.0          19   307   192   566    59   375\n",
      "> 6ml1A                      1   31144    6150     316     728   27.52    0.52E-05      20   227   302   569     4   261\n",
      "> 3n3kA                      1   31780    9550     355     728   30.66     0.0          21   237   295   565     9   282\n",
      "> 3ihpA                      1   32457   43700     673     728   37.10     0.0          22   606    23   726    18   672\n",
      "> 5xveA                      1   36131    5950     331     728   25.58    0.16E-04      23   232   303   565     3   260\n",
      "> 3i3tA                      1   37136    9050     303     728   28.88     0.0          24   216   306   562     3   234\n",
      "> 3i3tC                      1   37137    9200     310     728   28.87     0.0          25   223   306   562     3   241\n",
      "> 2i50A                      1   39772    4650     122     728   40.38    0.49E-02      26    49   192   247    51   102\n",
      "> 2l80A                      1   40166   11150     114     728   47.25     0.0          27    88   174   263    18   108\n",
      "> 5l8hA                      1   40243    6750     311     728   27.88    0.22E-06      28   220   306   565     6   231\n",
      "> 6cpmC                      1   41072    8500     339     728   28.09    0.30E-10      29   234   302   569    10   276\n",
      "> 6cpmD                      1   41073    8550     332     728   28.20    0.22E-10      30   233   302   569     7   272\n",
      "> 5b8dA                      1   48433    5850     100     728   34.38    0.75E-05      31    94   157   260     5   100\n",
      "> 2hd5A                      1   50321    6350     315     728   27.16    0.18E-05      32   229   303   565     2   244\n",
      "> 1wivA                      1   52252    4450      73     728   39.58    0.81E-02      33    48   591   638    20    67\n",
      "> 5wchA                      1   55397    5100     350     728   26.29    0.14E-02      34   226   303   566     1   232\n",
      "> 5wchB                      1   55398    5400     331     728   26.75    0.27E-03      35   225   303   566     1   228\n",
      "> 4wa6D                      1   55844    6550     415     728   29.71    0.86E-06      36   174   192   417    58   232\n",
      "> 5cvmA                      1   59437    6600     308     728   27.88    0.48E-06      37   220   306   565     4   229\n",
      "> 6nftA                      1   60868   11500     115     728   46.67     0.0          38   102   159   263     9   113\n",
      "> 2gfoA                      1   65004    9500     339     728   30.57     0.0          39   237   295   565     3   267\n",
      "> 2lbcA                      1   66336    4700     126     728   42.22    0.39E-02      40    45   601   645    79   123\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.015625\n",
      "profile_iteration__> Score Collection        [sec]:    68.750000\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.001023\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:   978.005005\n",
      "profile_iteration__> Statistics              [sec]:     0.031250\n",
      "profile_iteration__> Add to profile          [sec]:     0.109375\n",
      "profile_iteration__> Overall performance    [/seq]:     0.001026\n",
      "profile_iteration__> Overall performance [seq/sec]:   974.459106\n",
      "openf___224_> Open           buildprofile/EAO00192build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      6722143    6564.593     6.411\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6725051    6567.433     6.414\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6726219    6568.573     6.415\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6727107    6569.440     6.415\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6728031    6570.343     6.416\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6728567    6570.866     6.417\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6729959    6572.226     6.418\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6730791    6573.038     6.419\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6731675    6573.901     6.420\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6732759    6574.960     6.421\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6733051    6575.245     6.421\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6733855    6576.030     6.422\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6734207    6576.374     6.422\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6735107    6577.253     6.423\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6735983    6578.108     6.424\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6736611    6578.722     6.425\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6737239    6579.335     6.425\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6738199    6580.272     6.426\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6739087    6581.140     6.427\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6740311    6582.335     6.428\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6741215    6583.218     6.429\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6742159    6584.140     6.430\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6744579    6586.503     6.432\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6745503    6587.405     6.433\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6746363    6588.245     6.434\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6747251    6589.112     6.435\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6747443    6589.300     6.435\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6747791    6589.640     6.435\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6748667    6590.495     6.436\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6749599    6591.405     6.437\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6750527    6592.312     6.438\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6750899    6592.675     6.438\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6751811    6593.565     6.439\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6751999    6593.749     6.439\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6752899    6594.628     6.440\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6753795    6595.503     6.441\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6754487    6596.179     6.442\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6755363    6597.034     6.442\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6755767    6597.429     6.443\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6756711    6598.351     6.444\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6756887    6598.522     6.444\n",
      "openf___224_> Open           buildprofile/EAO00192build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461082    5333.088     5.208\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461610    5333.604     5.209\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5510210    5381.064     5.255\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5523510    5394.053     5.268\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           6nft.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5535720    5405.977     5.279\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5535796    5406.051     5.279\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5535967    5406.218     5.280\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5536817    5407.048     5.280\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5538075    5408.276     5.282\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5539979    5410.136     5.283\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5540019    5410.175     5.283\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5540109    5410.263     5.283\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5542965    5413.052     5.286\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5543025    5413.110     5.286\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5543160    5413.242     5.286\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5547444    5417.426     5.290\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5547532    5417.512     5.291\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5547730    5417.705     5.291\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5554156    5423.980     5.297\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5554288    5424.109     5.297\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5554585    5424.399     5.297\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5564207    5433.796     5.306\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5564407    5433.991     5.307\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5564857    5434.431     5.307\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5579307    5448.542     5.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      5682299    5549.120     5.419\n",
      "read_mo_297_> Segments, residues, atoms:        1      115      879\n",
      "read_mo_298_> Segment:        1  171:A  285:A      879\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5685605    5552.349     5.422\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5751498    5616.697     5.485\n",
      "openf___224_> Open           ali/EAO00192.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5801191    5665.226     5.532\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5836611    5699.815     5.566\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5907451    5768.995     5.634\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      6049131    5907.354     5.769\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      6052039    5910.194     5.772\n",
      "\n",
      "Read the alignment from file       : ali/EAO00192.ali\n",
      "\n",
      "Total number of alignment positions:   728\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      6nftA     115      1    6nft.pdb MOL_ID  1; MOLECULE  UBIQUITIN CARBOXYL-TERMINAL HYDROLASE 5; CHAIN  A, B; SYNON\n",
      "  2   EAO00192     728      1    EAO00192\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          731\n",
      "  Score                    :   74260.1016\n",
      "openf___224_> Open           ali/EAO00192-6nftA.ali\n",
      "openf___224_> Open           ali/EAO00192-6nftA.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461082    5333.088     5.208\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5461610    5333.604     5.209\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5510210    5381.064     5.255\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      5523510    5394.053     5.268\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00192-6nftA.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5535475    5405.737     5.279\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5536925    5407.153     5.280\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5539825    5409.985     5.283\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5545625    5415.649     5.289\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5557225    5426.978     5.300\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      5798033    5662.142     5.529\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5798489    5662.587     5.530\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5801397    5665.427     5.533\n",
      "\n",
      "Read the alignment from file       : ali/EAO00192-6nftA.ali\n",
      "\n",
      "Total number of alignment positions:   731\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      6nftA     115      1    6nft.pdb MOL_ID  1; MOLECULE  UBIQUITIN CARBOXYL-TERMINAL HYDROLASE 5; CHAIN  A, B; SYNON\n",
      "  2   EAO00192     728      1    EAO00192\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           6nft.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5837622    5700.803     5.567\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5837622    5700.803     5.567\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5837793    5700.970     5.567\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5838643    5701.800     5.568\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5839901    5703.028     5.569\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5841805    5704.888     5.571\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5841805    5704.888     5.571\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5841895    5704.976     5.571\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5844751    5707.765     5.574\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5844751    5707.765     5.574\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5844886    5707.896     5.574\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5849170    5712.080     5.578\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5849170    5712.080     5.578\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5849368    5712.273     5.578\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5855794    5718.549     5.585\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5855794    5718.549     5.585\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5856091    5718.839     5.585\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5865713    5728.235     5.594\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      5865713    5728.235     5.594\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5866163    5728.675     5.594\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      5880613    5742.786     5.608\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      5880298    5742.479     5.608\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6019550    5878.467     5.741\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6021150    5880.029     5.742\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6025130    5883.916     5.746\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6027130    5885.869     5.748\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6030130    5888.799     5.751\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6034630    5893.193     5.755\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6040202    5898.635     5.760\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6043002    5901.369     5.763\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6047202    5905.471     5.767\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6053502    5911.623     5.773\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6056274    5914.330     5.776\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      6057674    5915.697     5.777\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      6171813    6027.161     5.886\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      6727865    6570.181     6.416\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      6813321    6653.634     6.498\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      728   5655   5788      0         0      2447\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      7056321    6890.938     6.729\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      728   5655   5788   7843      9307      2447\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      7331097    7159.274     6.991\n",
      "patch_______> segment topology patched using RTF:  728 ;  LYS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      728   5656   5789   7845      9309      2448\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      728   5656   5789   7845      9309      2448\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      744     4912\n",
      "openf___224_> Open           EAO00192.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      728     5656     5656\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7438546    7264.205     7.094\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7446738    7272.205     7.102\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7463122    7288.205     7.117\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7495890    7320.205     7.149\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7528658    7352.205     7.180\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7594194    7416.205     7.242\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7626962    7448.205     7.274\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7692498    7512.205     7.336\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7725266    7544.205     7.367\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7856338    7672.205     7.492\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7921874    7736.205     7.555\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7987410    7800.205     7.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8118482    7928.205     7.742\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8380626    8184.205     7.992\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8511698    8312.205     8.117\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8642770    8440.205     8.242\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8904914    8696.205     8.492\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9429202    9208.205     8.992\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9691346    9464.205     9.242\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     5789    7845    8927    2409\n",
      "              Total number of restraints before, now         :        0    24970\n",
      "make_re_422_> Number of previous, current restraints         :        0    24970\n",
      "make_re_423_> Number of previous, current selected restraints:        0    24970\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     10107637    9870.739     9.639\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        2  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       14  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      112  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      115  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        2  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       14  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       21  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      112  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      115  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        2  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       14  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       21  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       31  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      115  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       14  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       21  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       31  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      115  LYS\n",
      "getdata_643_> Protein accepted:  6nftA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    24970    25696\n",
      "make_re_423_> Number of previous, current selected restraints:    24970    25696\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     10107653    9870.755     9.639\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  6nftA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      766\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    25696    26423\n",
      "make_re_423_> Number of previous, current selected restraints:    25696    25657\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     10107653    9870.755     9.639\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  6nftA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     10369797   10126.755     9.889\n",
      "make_re_422_> Number of previous, current restraints         :    26423    27050\n",
      "make_re_423_> Number of previous, current selected restraints:    25657    26284\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     10369797   10126.755     9.889\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  6nftA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    27050    27499\n",
      "make_re_423_> Number of previous, current selected restraints:    26284    26733\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     10369797   10126.755     9.889\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  6nftA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    27499    27713\n",
      "make_re_423_> Number of previous, current selected restraints:    26733    26947\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     10369797   10126.755     9.889\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  6nftA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     10894085   10638.755    10.389\n",
      "make_re_422_> Number of previous, current restraints         :    27713    27777\n",
      "make_re_423_> Number of previous, current selected restraints:    26947    27011\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     12522977   12229.470    11.943\n",
      "make_re_422_> Number of previous, current restraints         :    27777    29769\n",
      "make_re_423_> Number of previous, current selected restraints:    27011    29003\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "make_re_422_> Number of previous, current restraints         :    29769    31862\n",
      "make_re_423_> Number of previous, current selected restraints:    29003    31096\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13047265   12741.470    12.443\n",
      "make_re_422_> Number of previous, current restraints         :    31862    32974\n",
      "make_re_423_> Number of previous, current selected restraints:    31096    32208\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    32974    33445\n",
      "make_re_423_> Number of previous, current selected restraints:    32208    32679\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    33445    33445\n",
      "make_re_423_> Number of previous, current selected restraints:    32679    32679\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    33445    33445\n",
      "make_re_423_> Number of previous, current selected restraints:    32679    32679\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    33445    33445\n",
      "make_re_423_> Number of previous, current selected restraints:    32679    32679\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    33445    33445\n",
      "make_re_423_> Number of previous, current selected restraints:    32679    32679\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    33445    33445\n",
      "make_re_423_> Number of previous, current selected restraints:    32679    32679\n",
      "rmdupl__427_>     4485 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    33445    28194\n",
      "openf___224_> Open           EAO00192.rsr\n",
      "openf___224_> Open           EAO00192.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     12807040   12506.875    12.214\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13593472   13274.875    12.964\n",
      "rdcsr2__307_> Number of restraints read    :    28194\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    28194    28194\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00192.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13593504   13274.906    12.964\n",
      "read_mo_297_> Segments, residues, atoms:        1      728     5656\n",
      "read_mo_298_> Segment:        1  1:A  728:A     5656\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5656     5656        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    23948\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24395\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26848\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27084\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27270\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28042\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14021820   13693.184    13.372\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28044\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28188\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28194\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.1449        5  5\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -175.1613        9  9\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.7415       10  10\n",
      "iupac_m_485_> OD1/2 will be swapped:      143.1378       17  17\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.8813       18  18\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9802       33  33\n",
      "iupac_m_486_> OE1/2 will be swapped:      146.9785       34  34\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.0533       37  37\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.2526       43  43\n",
      "iupac_m_486_> OE1/2 will be swapped:      130.5620       61  61\n",
      "iupac_m_486_> OE1/2 will be swapped:      101.6007       87  87\n",
      "iupac_m_485_> OD1/2 will be swapped:     -164.3902       88  88\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.2074       94  94\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.9128      106  106\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -144.3491      107  107\n",
      "iupac_m_485_> OD1/2 will be swapped:     -163.8579      128  128\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.8499      148  148\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.5788      164  164\n",
      "iupac_m_485_> OD1/2 will be swapped:     -174.8844      178  178\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.3399      183  183\n",
      "iupac_m_486_> OE1/2 will be swapped:     -142.0067      186  186\n",
      "iupac_m_487_> NH1/2 swapped:     -170.8707      188  188\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      129.2132      252  252\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -142.3361      262  262\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.0764      275  275\n",
      "iupac_m_485_> OD1/2 will be swapped:      142.5667      285  285\n",
      "iupac_m_487_> NH1/2 swapped:      178.4471      288  288\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.1264      291  291\n",
      "iupac_m_485_> OD1/2 will be swapped:      159.6405      294  294\n",
      "iupac_m_487_> NH1/2 swapped:      179.2923      304  304\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.0627      327  327\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9657      331  331\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.8712      338  338\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6179      340  340\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9720      358  358\n",
      "iupac_m_487_> NH1/2 swapped:      179.8847      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.9778      368  368\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.8417      372  372\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.4367      375  375\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.4599      384  384\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.8253      413  413\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9657      416  416\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7897      453  453\n",
      "iupac_m_487_> NH1/2 swapped:      179.6749      456  456\n",
      "iupac_m_485_> OD1/2 will be swapped:      177.6049      459  459\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.3695      469  469\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.1061      476  476\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.4088      483  483\n",
      "iupac_m_487_> NH1/2 swapped:      179.4637      490  490\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.4159      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.1375      507  507\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.2830      509  509\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9802      526  526\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6534      528  528\n",
      "iupac_m_487_> NH1/2 swapped:      179.4236      540  540\n",
      "iupac_m_485_> OD1/2 will be swapped:     -155.9765      544  544\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      114.2936      556  556\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -146.1263      562  562\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      107.3005      567  567\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.3904      576  576\n",
      "iupac_m_486_> OE1/2 will be swapped:      -96.6727      585  585\n",
      "iupac_m_484W> Dihedral still outside +-90:       95.3971\n",
      "iupac_m_485_> OD1/2 will be swapped:      108.8500      600  600\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.9610      613  613\n",
      "iupac_m_486_> OE1/2 will be swapped:     -126.6799      614  614\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.4286      629  629\n",
      "iupac_m_487_> NH1/2 swapped:      179.8481      630  630\n",
      "iupac_m_485_> OD1/2 will be swapped:     -159.2433      633  633\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.1536      639  639\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.3928      643  643\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.4521      649  649\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9720      656  656\n",
      "iupac_m_487_> NH1/2 swapped:      179.8830      670  670\n",
      "iupac_m_485_> OD1/2 will be swapped:      160.0756      704  704\n",
      "iupac_m_486_> OE1/2 will be swapped:      153.0594      705  705\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      117.2140      722  722\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:     -119.4381\n",
      "iupac_m_487_> NH1/2 swapped:      135.4717      725  725\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14450336   14111.656    13.781\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14721824   14376.781    14.040\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15264800   14907.031    14.558\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16350752   15967.531    15.593\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     18522656   18088.531    17.665\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   486085\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -17505.7949\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -17505.794922\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    53.912998\n",
      "Sequence length          :          728\n",
      "Compactness              :     0.000338\n",
      "Native energy (pair)     :  1043.213114\n",
      "Native energy (surface)  :   119.290922\n",
      "Native energy (combined) :    32.067137\n",
      "Z score (pair)           :    -2.079209\n",
      "Z score (surface)        :    -0.471354\n",
      "Z score (combined)       :    -1.830575\n",
      "GA341 score              :     0.402338\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     9310\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      376139.3750\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5789     434    552   0.425   0.425     0.30940E+06   1.000\n",
      " 2 Bond angle potential               :    7845     511    873  10.199  10.199      17140.       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3676       2    300  56.844  56.844      2056.7       1.000\n",
      " 4 Stereochemical improper torsion pot:    2409      19     49   3.229   3.229      529.51       1.000\n",
      " 5 Soft-sphere overlap restraints     :    9310       1      6   0.009   0.009      86.614       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    1992     223    400   4.392   4.392      22959.       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2093     194    386   4.848   4.848      20565.       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     727      10     35   6.016   6.016      310.31       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     627       0     42  91.019  91.019      418.39       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     449       0     28  84.309  84.309      321.70       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     214       0      4 109.894 109.894      183.11       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      2  88.749  88.749      49.061       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1112       0      0   0.849   0.849      72.591       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     726     505    241  53.401 152.896      1676.7       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     471       6     22   2.988   2.988      367.79       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00192.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00192.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  234694.0625\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     84  10E  10E C   CA     81   75    1.65    1.49    0.16    4.63    1.49    0.16    4.63\n",
      "    2     87  11A  11A N   CA     83   84    1.57    1.43    0.14    4.51    1.43    0.14    4.51\n",
      "    3     89  11A  11A C   CA     86   84    1.66    1.49    0.17    5.02    1.49    0.17    5.02\n",
      "    4     90  11A  12C C   N      86   88    1.47    1.35    0.13    4.55    1.35    0.13    4.55\n",
      "    5     93  12C  12C N   CA     88   89    1.58    1.43    0.15    4.87    1.43    0.15    4.87\n",
      "    6     95  12C  12C C   CA     92   89    1.68    1.49    0.19    5.43    1.49    0.19    5.43\n",
      "    7     96  12C  13A C   N      92   94    1.48    1.35    0.14    4.87    1.35    0.14    4.87\n",
      "    8     98  13A  13A N   CA     94   95    1.59    1.43    0.16    5.21    1.43    0.16    5.21\n",
      "    9    100  13A  13A C   CA     97   95    1.68    1.49    0.19    5.62    1.49    0.19    5.62\n",
      "   10    101  13A  14H C   N      97   99    1.49    1.35    0.15    5.15    1.35    0.15    5.15\n",
      "   11    108  14H  14H N   CA     99  100    1.60    1.43    0.17    5.49    1.43    0.17    5.49\n",
      "   12    110  14H  14H C   CA    107  100    1.72    1.49    0.23    6.58    1.49    0.23    6.58\n",
      "   13    111  14H  15C C   N     107  109    1.51    1.35    0.16    5.71    1.35    0.16    5.71\n",
      "   14    115  15C  15C N   CA    109  110    1.62    1.43    0.19    6.17    1.43    0.19    6.17\n",
      "   15    117  15C  15C C   CA    113  110    1.72    1.49    0.23    6.76    1.49    0.23    6.76\n",
      "   16    118  15C  16W C   N     113  115    1.52    1.35    0.17    6.20    1.35    0.17    6.20\n",
      "   17    128  16W  16W N   CA    115  116    1.63    1.43    0.20    6.54    1.43    0.20    6.54\n",
      "   18    130  16W  16W C   CA    127  116    1.76    1.49    0.27    7.73    1.49    0.27    7.73\n",
      "   19    131  16W  17D C   N     127  129    1.54    1.35    0.19    6.88    1.35    0.19    6.88\n",
      "   20    139  17D  17D N   CA    129  130    1.66    1.43    0.23    7.45    1.43    0.23    7.45\n",
      "   21    141  17D  17D C   CA    135  130    1.77    1.49    0.28    8.16    1.49    0.28    8.16\n",
      "   22    142  17D  18E C   N     135  137    1.55    1.35    0.21    7.31    1.35    0.21    7.31\n",
      "   23    148  18E  18E N   CA    137  138    1.67    1.43    0.24    7.86    1.43    0.24    7.86\n",
      "   24    150  18E  18E C   CA    144  138    1.80    1.49    0.31    8.89    1.49    0.31    8.89\n",
      "   25    151  18E  19S C   N     144  146    1.56    1.35    0.21    7.61    1.35    0.21    7.61\n",
      "   26    154  19S  19S N   CA    146  147    1.68    1.43    0.25    8.21    1.43    0.25    8.21\n",
      "   27    156  19S  19S C   CA    150  147    1.81    1.49    0.32    9.20    1.49    0.32    9.20\n",
      "   28    157  19S  20V C   N     150  152    1.57    1.35    0.23    8.01    1.35    0.23    8.01\n",
      "   29    161  20V  20V N   CA    152  153    1.69    1.43    0.26    8.63    1.43    0.26    8.63\n",
      "   30    163  20V  20V C   CA    157  153    1.82    1.49    0.33    9.66    1.49    0.33    9.66\n",
      "   31    164  20V  21M C   N     157  159    1.59    1.35    0.24    8.65    1.35    0.24    8.65\n",
      "   32    169  21M  21M N   CA    159  160    1.71    1.43    0.28    9.35    1.43    0.28    9.35\n",
      "   33    171  21M  21M C   CA    165  160    1.84    1.49    0.35   10.23    1.49    0.35   10.23\n",
      "   34    172  21M  22L C   N     165  167    1.60    1.35    0.25    8.90    1.35    0.25    8.90\n",
      "   35    177  22L  22L N   CA    167  168    1.72    1.43    0.29    9.56    1.43    0.29    9.56\n",
      "   36    179  22L  22L C   CA    173  168    1.86    1.49    0.37   10.85    1.49    0.37   10.85\n",
      "   37    180  22L  23V C   N     173  175    1.61    1.35    0.26    9.39    1.35    0.26    9.39\n",
      "   38    184  23V  23V N   CA    175  176    1.74    1.43    0.31   10.11    1.43    0.31   10.11\n",
      "   39    186  23V  23V C   CA    180  176    1.88    1.49    0.39   11.40    1.49    0.39   11.40\n",
      "   40    187  23V  24P C   N     180  182    1.68    1.30    0.38   11.18    1.30    0.38   11.18\n",
      "   41    189  24P  24P C   CA    187  183    1.89    1.49    0.40   11.55    1.49    0.40   11.55\n",
      "   42    190  24P  25N C   N     187  189    1.63    1.35    0.28    9.97    1.35    0.28    9.97\n",
      "   43    191  24P  24P N   CA    182  183    1.75    1.43    0.32   10.46    1.43    0.32   10.46\n",
      "   44    200  25N  25N N   CA    189  190    1.76    1.43    0.33   10.79    1.43    0.33   10.79\n",
      "   45    202  25N  25N C   CA    195  190    1.91    1.49    0.42   12.07    1.49    0.42   12.07\n",
      "   46    203  25N  26H C   N     195  197    1.64    1.35    0.29   10.37    1.35    0.29   10.37\n",
      "   47    210  26H  26H N   CA    197  198    1.77    1.43    0.34   11.24    1.43    0.34   11.24\n",
      "   48    212  26H  26H C   CA    205  198    1.93    1.49    0.44   12.75    1.49    0.44   12.75\n",
      "   49    213  26H  27R C   N     205  207    1.65    1.35    0.31   10.97    1.35    0.31   10.97\n",
      "   50    222  27R  27R N   CA    207  208    1.79    1.43    0.36   11.95    1.43    0.36   11.95\n",
      "   51    224  27R  27R C   CA    216  208    1.95    1.49    0.46   13.36    1.49    0.46   13.36\n",
      "   52    225  27R  28S C   N     216  218    1.66    1.35    0.32   11.34    1.35    0.32   11.34\n",
      "   53    228  28S  28S N   CA    218  219    1.80    1.43    0.37   12.30    1.43    0.37   12.30\n",
      "   54    230  28S  28S C   CA    222  219    1.96    1.49    0.47   13.63    1.49    0.47   13.63\n",
      "   55    231  28S  29A C   N     222  224    1.67    1.35    0.33   11.60    1.35    0.33   11.60\n",
      "   56    233  29A  29A N   CA    224  225    1.81    1.43    0.38   12.58    1.43    0.38   12.58\n",
      "   57    235  29A  29A C   CA    227  225    1.97    1.49    0.48   14.03    1.49    0.48   14.03\n",
      "   58    236  29A  30V C   N     227  229    1.68    1.35    0.34   11.95    1.35    0.34   11.95\n",
      "   59    240  30V  30V N   CA    229  230    1.82    1.43    0.39   12.97    1.43    0.39   12.97\n",
      "   60    242  30V  30V C   CA    234  230    1.99    1.49    0.50   14.44    1.49    0.50   14.44\n",
      "   61    243  30V  31H C   N     234  236    1.69    1.35    0.35   12.34    1.35    0.35   12.34\n",
      "   62    250  31H  31H N   CA    236  237    1.84    1.43    0.41   13.37    1.43    0.41   13.37\n",
      "   63    252  31H  31H C   CA    244  237    2.02    1.49    0.53   15.36    1.49    0.53   15.36\n",
      "   64    253  31H  32K C   N     244  246    1.71    1.35    0.36   12.94    1.35    0.36   12.94\n",
      "   65    260  32K  32K N   CA    246  247    1.86    1.43    0.43   14.06    1.43    0.43   14.06\n",
      "   66    262  32K  32K C   CA    253  247    2.04    1.49    0.55   15.88    1.49    0.55   15.88\n",
      "   67    263  32K  33E C   N     253  255    1.73    1.35    0.38   13.61    1.35    0.38   13.61\n",
      "   68    269  33E  33E N   CA    255  256    1.88    1.43    0.45   14.81    1.43    0.45   14.81\n",
      "   69    271  33E  33E C   CA    262  256    2.06    1.49    0.57   16.64    1.49    0.57   16.64\n",
      "   70    272  33E  34E C   N     262  264    1.74    1.35    0.40   14.03    1.35    0.40   14.03\n",
      "   71    278  34E  34E N   CA    264  265    1.89    1.43    0.46   15.24    1.43    0.46   15.24\n",
      "   72    280  34E  34E C   CA    271  265    2.08    1.49    0.59   17.17    1.49    0.59   17.17\n",
      "   73    281  34E  35C C   N     271  273    1.75    1.35    0.40   14.35    1.35    0.40   14.35\n",
      "   74    284  35C  35C N   CA    273  274    1.90    1.43    0.47   15.56    1.43    0.47   15.56\n",
      "   75    286  35C  35C C   CA    277  274    2.08    1.49    0.59   17.29    1.49    0.59   17.29\n",
      "   76    287  35C  36A C   N     277  279    1.76    1.35    0.41   14.57    1.35    0.41   14.57\n",
      "   77    289  36A  36A N   CA    279  280    1.91    1.43    0.48   15.83    1.43    0.48   15.83\n",
      "   78    291  36A  36A C   CA    282  280    2.09    1.49    0.60   17.53    1.49    0.60   17.53\n",
      "   79    292  36A  37Y C   N     282  284    1.76    1.35    0.42   14.79    1.35    0.42   14.79\n",
      "   80    302  37Y  37Y N   CA    284  285    1.92    1.43    0.49   16.06    1.43    0.49   16.06\n",
      "   81    304  37Y  37Y C   CA    294  285    2.11    1.49    0.62   18.15    1.49    0.62   18.15\n",
      "   82    305  37Y  38C C   N     294  296    1.78    1.35    0.43   15.25    1.35    0.43   15.25\n",
      "   83    308  38C  38C N   CA    296  297    1.93    1.43    0.50   16.59    1.43    0.50   16.59\n",
      "   84    310  38C  38C C   CA    300  297    2.13    1.49    0.64   18.51    1.49    0.64   18.51\n",
      "   85    311  38C  39C C   N     300  302    1.78    1.35    0.44   15.57    1.35    0.44   15.57\n",
      "   86    314  39C  39C N   CA    302  303    1.94    1.43    0.51   16.90    1.43    0.51   16.90\n",
      "   87    316  39C  39C C   CA    306  303    2.14    1.49    0.65   18.90    1.49    0.65   18.90\n",
      "   88    317  39C  40R C   N     306  308    1.79    1.35    0.45   15.85    1.35    0.45   15.85\n",
      "   89    325  40R  40R N   CA    308  309    1.95    1.43    0.52   17.16    1.43    0.52   17.16\n",
      "   90    327  40R  40R C   CA    317  309    2.16    1.49    0.67   19.58    1.49    0.67   19.58\n",
      "   91    328  40R  41T C   N     317  319    1.80    1.35    0.46   16.30    1.35    0.46   16.30\n",
      "   92    332  41T  41T N   CA    319  320    1.97    1.43    0.54   17.70    1.43    0.54   17.70\n",
      "   93    334  41T  41T C   CA    324  320    2.17    1.49    0.68   19.88    1.49    0.68   19.88\n",
      "   94    335  41T  42C C   N     324  326    1.81    1.35    0.47   16.65    1.35    0.47   16.65\n",
      "   95    338  42C  42C N   CA    326  327    1.98    1.43    0.55   18.09    1.43    0.55   18.09\n",
      "   96    340  42C  42C C   CA    330  327    2.18    1.49    0.69   20.17    1.49    0.69   20.17\n",
      "   97    341  42C  43R C   N     330  332    1.82    1.35    0.48   16.95    1.35    0.48   16.95\n",
      "   98    349  43R  43R N   CA    332  333    1.99    1.43    0.56   18.36    1.43    0.56   18.36\n",
      "   99    351  43R  43R C   CA    341  333    2.21    1.49    0.72   20.83    1.49    0.72   20.83\n",
      "  100    352  43R  44H C   N     341  343    1.83    1.35    0.49   17.36    1.35    0.49   17.36\n",
      "  101    359  44H  44H N   CA    343  344    2.00    1.43    0.57   18.82    1.43    0.57   18.82\n",
      "  102    361  44H  44H C   CA    351  344    2.23    1.49    0.74   21.45    1.49    0.74   21.45\n",
      "  103    362  44H  45E C   N     351  353    1.85    1.35    0.51   18.04    1.35    0.51   18.04\n",
      "  104    369  45E  45E N   CA    353  354    2.03    1.43    0.60   19.62    1.43    0.60   19.62\n",
      "  105    371  45E  45E C   CA    360  354    2.25    1.49    0.76   22.13    1.49    0.76   22.13\n",
      "  106    372  45E  46M C   N     360  362    1.87    1.35    0.52   18.49    1.35    0.52   18.49\n",
      "  107    377  46M  46M N   CA    362  363    2.04    1.43    0.61   20.05    1.43    0.61   20.05\n",
      "  108    379  46M  46M C   CA    368  363    2.27    1.49    0.78   22.56    1.49    0.78   22.56\n",
      "  109    380  46M  47G C   N     368  370    1.87    1.35    0.53   18.73    1.35    0.53   18.73\n",
      "  110    381  47G  47G N   CA    370  371    1.97    1.43    0.54   17.91    1.43    0.54   17.91\n",
      "  111    383  47G  47G C   CA    372  371    2.01    1.49    0.52   15.19    1.49    0.52   15.19\n",
      "  112    384  47G  48T C   N     372  374    1.58    1.35    0.24    8.44    1.35    0.24    8.44\n",
      "  113    388  48T  48T N   CA    374  375    1.70    1.43    0.27    9.04    1.43    0.27    9.04\n",
      "  114   1825 236A 236A C   CA   1780 1778    1.65    1.49    0.16    4.64    1.49    0.16    4.64\n",
      "  115   1845 238F 238F C   CA   1799 1791    1.66    1.49    0.17    4.91    1.49    0.17    4.91\n",
      "  116   2041 261I 261I N   CA   1984 1985    1.59    1.43    0.16    5.11    1.43    0.16    5.11\n",
      "  117   2043 261I 261I C   CA   1990 1985    1.70    1.49    0.21    6.07    1.49    0.21    6.07\n",
      "  118   2044 261I 262D C   N    1990 1992    1.55    1.35    0.21    7.41    1.35    0.21    7.41\n",
      "  119   2049 262D 262D N   CA   1992 1993    1.71    1.43    0.28    9.15    1.43    0.28    9.15\n",
      "  120   2051 262D 262D C   CA   1998 1993    1.79    1.49    0.30    8.80    1.49    0.30    8.80\n",
      "  121   2052 262D 263I C   N    1998 2000    1.59    1.35    0.24    8.63    1.35    0.24    8.63\n",
      "  122   2057 263I 263I N   CA   2000 2001    1.73    1.43    0.30    9.87    1.43    0.30    9.87\n",
      "  123   2059 263I 263I C   CA   2006 2001    1.82    1.49    0.33    9.56    1.49    0.33    9.56\n",
      "  124   2060 263I 264Q C   N    2006 2008    1.60    1.35    0.25    8.89    1.35    0.25    8.89\n",
      "  125   2066 264Q 264Q N   CA   2008 2009    1.72    1.43    0.29    9.47    1.43    0.29    9.47\n",
      "  126   2068 264Q 264Q C   CA   2015 2009    1.81    1.49    0.32    9.18    1.49    0.32    9.18\n",
      "  127   2069 264Q 265T C   N    2015 2017    1.57    1.35    0.22    7.96    1.35    0.22    7.96\n",
      "  128   2073 265T 265T N   CA   2017 2018    1.69    1.43    0.26    8.53    1.43    0.26    8.53\n",
      "  129   2075 265T 265T C   CA   2022 2018    1.82    1.49    0.33    9.46    1.49    0.33    9.46\n",
      "  130   2076 265T 266A C   N    2022 2024    1.57    1.35    0.22    7.88    1.35    0.22    7.88\n",
      "  131   2078 266A 266A N   CA   2024 2025    1.68    1.43    0.25    8.23    1.43    0.25    8.23\n",
      "  132   2080 266A 266A C   CA   2027 2025    1.79    1.49    0.30    8.74    1.49    0.30    8.74\n",
      "  133   2081 266A 267K C   N    2027 2029    1.55    1.35    0.21    7.43    1.35    0.21    7.43\n",
      "  134   2087 267K 267K N   CA   2029 2030    1.67    1.43    0.24    7.98    1.43    0.24    7.98\n",
      "  135   2089 267K 267K C   CA   2036 2030    1.78    1.49    0.29    8.41    1.49    0.29    8.41\n",
      "  136   2090 267K 268K C   N    2036 2038    1.55    1.35    0.20    7.15    1.35    0.20    7.15\n",
      "  137   2096 268K 268K N   CA   2038 2039    1.66    1.43    0.23    7.54    1.43    0.23    7.54\n",
      "  138   2098 268K 268K C   CA   2045 2039    1.78    1.49    0.29    8.36    1.49    0.29    8.36\n",
      "  139   2099 268K 269T C   N    2045 2047    1.54    1.35    0.20    7.02    1.35    0.20    7.02\n",
      "  140   2103 269T 269T N   CA   2047 2048    1.65    1.43    0.22    7.38    1.43    0.22    7.38\n",
      "  141   2105 269T 269T C   CA   2052 2048    1.77    1.49    0.28    8.19    1.49    0.28    8.19\n",
      "  142   2106 269T 270A C   N    2052 2054    1.54    1.35    0.19    6.76    1.35    0.19    6.76\n",
      "  143   2108 270A 270A N   CA   2054 2055    1.64    1.43    0.21    6.96    1.43    0.21    6.96\n",
      "  144   2110 270A 270A C   CA   2057 2055    1.75    1.49    0.26    7.56    1.49    0.26    7.56\n",
      "  145   2111 270A 271K C   N    2057 2059    1.53    1.35    0.18    6.40    1.35    0.18    6.40\n",
      "  146   2117 271K 271K N   CA   2059 2060    1.64    1.43    0.21    6.91    1.43    0.21    6.91\n",
      "  147   2119 271K 271K C   CA   2066 2060    1.74    1.49    0.25    7.24    1.49    0.25    7.24\n",
      "  148   2120 271K 272T C   N    2066 2068    1.52    1.35    0.18    6.21    1.35    0.18    6.21\n",
      "  149   2124 272T 272T N   CA   2068 2069    1.63    1.43    0.20    6.58    1.43    0.20    6.58\n",
      "  150   2126 272T 272T C   CA   2073 2069    1.74    1.49    0.25    7.18    1.49    0.25    7.18\n",
      "  151   2127 272T 273L C   N    2073 2075    1.52    1.35    0.17    6.17    1.35    0.17    6.17\n",
      "  152   2132 273L 273L N   CA   2075 2076    1.62    1.43    0.19    6.38    1.43    0.19    6.38\n",
      "  153   2134 273L 273L C   CA   2081 2076    1.73    1.49    0.24    6.90    1.49    0.24    6.90\n",
      "  154   2135 273L 274G C   N    2081 2083    1.51    1.35    0.17    5.88    1.35    0.17    5.88\n",
      "  155   2136 274G 274G N   CA   2083 2084    1.62    1.43    0.19    6.17    1.43    0.19    6.17\n",
      "  156   2138 274G 274G C   CA   2085 2084    1.72    1.49    0.23    6.60    1.49    0.23    6.60\n",
      "  157   2139 274G 275E C   N    2085 2087    1.51    1.35    0.16    5.68    1.35    0.16    5.68\n",
      "  158   2145 275E 275E N   CA   2087 2088    1.61    1.43    0.18    5.97    1.43    0.18    5.97\n",
      "  159   2147 275E 275E C   CA   2094 2088    1.70    1.49    0.21    6.13    1.49    0.21    6.13\n",
      "  160   2148 275E 276M C   N    2094 2096    1.49    1.35    0.15    5.23    1.35    0.15    5.23\n",
      "  161   2153 276M 276M N   CA   2096 2097    1.60    1.43    0.17    5.50    1.43    0.17    5.50\n",
      "  162   2155 276M 276M C   CA   2102 2097    1.69    1.49    0.20    5.87    1.49    0.20    5.87\n",
      "  163   2156 276M 277Q C   N    2102 2104    1.49    1.35    0.14    5.02    1.35    0.14    5.02\n",
      "  164   2162 277Q 277Q N   CA   2104 2105    1.59    1.43    0.16    5.25    1.43    0.16    5.25\n",
      "  165   2164 277Q 277Q C   CA   2111 2105    1.68    1.49    0.19    5.45    1.49    0.19    5.45\n",
      "  166   2165 277Q 278Y C   N    2111 2113    1.48    1.35    0.13    4.62    1.35    0.13    4.62\n",
      "  167   2175 278Y 278Y N   CA   2113 2114    1.58    1.43    0.15    4.87    1.43    0.15    4.87\n",
      "  168   2177 278Y 278Y C   CA   2123 2114    1.66    1.49    0.17    4.98    1.49    0.17    4.98\n",
      "  169   2185 279D 279D C   CA   2131 2126    1.64    1.49    0.15    4.50    1.49    0.15    4.50\n",
      "  170   4422 555V 555V C   CA   4318 4314    1.67    1.49    0.18    5.17    1.49    0.18    5.17\n",
      "  171   4423 555V 556F C   N    4318 4320    1.48    1.35    0.13    4.70    1.35    0.13    4.70\n",
      "  172   4432 556F 556F N   CA   4320 4321    1.59    1.43    0.16    5.33    1.43    0.16    5.33\n",
      "  173   4434 556F 556F C   CA   4329 4321    1.70    1.49    0.21    6.15    1.49    0.21    6.15\n",
      "  174   4435 556F 557V C   N    4329 4331    1.50    1.35    0.15    5.38    1.35    0.15    5.38\n",
      "  175   4439 557V 557V N   CA   4331 4332    1.61    1.43    0.18    5.82    1.43    0.18    5.82\n",
      "  176   4441 557V 557V C   CA   4336 4332    1.71    1.49    0.22    6.48    1.49    0.22    6.48\n",
      "  177   4442 557V 558D C   N    4336 4338    1.50    1.35    0.16    5.50    1.35    0.16    5.50\n",
      "  178   4447 558D 558D N   CA   4338 4339    1.61    1.43    0.18    5.78    1.43    0.18    5.78\n",
      "  179   4449 558D 558D C   CA   4344 4339    1.73    1.49    0.24    6.95    1.49    0.24    6.95\n",
      "  180   4450 558D 559V C   N    4344 4346    1.51    1.35    0.17    5.99    1.35    0.17    5.99\n",
      "  181   4454 559V 559V N   CA   4346 4347    1.62    1.43    0.19    6.41    1.43    0.19    6.41\n",
      "  182   4456 559V 559V C   CA   4351 4347    1.76    1.49    0.27    7.87    1.49    0.27    7.87\n",
      "  183   4457 559V 560P C   N    4351 4353    1.56    1.30    0.26    7.73    1.30    0.26    7.73\n",
      "  184   4459 560P 560P C   CA   4358 4354    1.77    1.49    0.28    8.27    1.49    0.28    8.27\n",
      "  185   4460 560P 561L C   N    4358 4360    1.55    1.35    0.20    7.26    1.35    0.20    7.26\n",
      "  186   4461 560P 560P N   CA   4353 4354    1.67    1.43    0.24    7.87    1.43    0.24    7.87\n",
      "  187   4470 561L 561L N   CA   4360 4361    1.67    1.43    0.24    7.80    1.43    0.24    7.80\n",
      "  188   4472 561L 561L C   CA   4366 4361    1.80    1.49    0.31    8.97    1.49    0.31    8.97\n",
      "  189   4473 561L 562E C   N    4366 4368    1.56    1.35    0.22    7.64    1.35    0.22    7.64\n",
      "  190   4479 562E 562E N   CA   4368 4369    1.69    1.43    0.26    8.42    1.43    0.26    8.42\n",
      "  191   4481 562E 562E C   CA   4375 4369    1.84    1.49    0.35   10.16    1.49    0.35   10.16\n",
      "  192   4482 562E 563L C   N    4375 4377    1.60    1.35    0.25    8.91    1.35    0.25    8.91\n",
      "  193   4487 563L 563L N   CA   4377 4378    1.73    1.43    0.30    9.79    1.43    0.30    9.79\n",
      "  194   4489 563L 563L C   CA   4383 4378    1.88    1.49    0.39   11.32    1.49    0.39   11.32\n",
      "  195   4490 563L 564D C   N    4383 4385    1.62    1.35    0.28    9.84    1.35    0.28    9.84\n",
      "  196   4495 564D 564D N   CA   4385 4386    1.76    1.43    0.33   10.71    1.43    0.33   10.71\n",
      "  197   4497 564D 564D C   CA   4391 4386    1.91    1.49    0.42   12.08    1.49    0.42   12.08\n",
      "  198   4498 564D 565L C   N    4391 4393    1.65    1.35    0.30   10.76    1.35    0.30   10.76\n",
      "  199   4503 565L 565L N   CA   4393 4394    1.78    1.43    0.35   11.61    1.43    0.35   11.61\n",
      "  200   4505 565L 565L C   CA   4399 4394    1.95    1.49    0.46   13.40    1.49    0.46   13.40\n",
      "  201   4506 565L 566E C   N    4399 4401    1.67    1.35    0.32   11.41    1.35    0.32   11.41\n",
      "  202   4512 566E 566E N   CA   4401 4402    1.80    1.43    0.37   12.31    1.43    0.37   12.31\n",
      "  203   4514 566E 566E C   CA   4408 4402    1.99    1.49    0.50   14.57    1.49    0.50   14.57\n",
      "  204   4515 566E 567F C   N    4408 4410    1.70    1.35    0.35   12.57    1.35    0.35   12.57\n",
      "  205   4524 567F 567F N   CA   4410 4411    1.84    1.43    0.41   13.64    1.43    0.41   13.64\n",
      "  206   4526 567F 567F C   CA   4419 4411    2.04    1.49    0.55   15.98    1.49    0.55   15.98\n",
      "  207   4527 567F 568L C   N    4419 4421    1.73    1.35    0.39   13.82    1.35    0.39   13.82\n",
      "  208   4532 568L 568L N   CA   4421 4422    1.88    1.43    0.45   14.89    1.43    0.45   14.89\n",
      "  209   4534 568L 568L C   CA   4427 4422    2.08    1.49    0.59   17.25    1.49    0.59   17.25\n",
      "  210   4535 568L 569R C   N    4427 4429    1.76    1.35    0.42   14.74    1.35    0.42   14.74\n",
      "  211   4543 569R 569R N   CA   4429 4430    1.92    1.43    0.49   16.10    1.43    0.49   16.10\n",
      "  212   4545 569R 569R C   CA   4438 4430    2.14    1.49    0.65   19.01    1.49    0.65   19.01\n",
      "  213   4546 569R 570G C   N    4438 4440    1.79    1.35    0.45   15.84    1.35    0.45   15.84\n",
      "  214   4547 570G 570G N   CA   4440 4441    1.96    1.43    0.53   17.31    1.43    0.53   17.31\n",
      "  215   4549 570G 570G C   CA   4442 4441    2.16    1.49    0.67   19.35    1.49    0.67   19.35\n",
      "  216   4550 570G 571K C   N    4442 4444    1.80    1.35    0.46   16.15    1.35    0.46   16.15\n",
      "  217   4556 571K 571K N   CA   4444 4445    1.96    1.43    0.53   17.48    1.43    0.53   17.48\n",
      "  218   4558 571K 571K C   CA   4451 4445    2.19    1.49    0.70   20.28    1.49    0.70   20.28\n",
      "  219   4559 571K 572G C   N    4451 4453    1.81    1.35    0.47   16.58    1.35    0.47   16.58\n",
      "  220   4560 572G 572G N   CA   4453 4454    1.98    1.43    0.55   18.17    1.43    0.55   18.17\n",
      "  221   4562 572G 572G C   CA   4455 4454    2.19    1.49    0.70   20.40    1.49    0.70   20.40\n",
      "  222   4563 572G 573L C   N    4455 4457    1.81    1.35    0.46   16.33    1.35    0.46   16.33\n",
      "  223   4568 573L 573L N   CA   4457 4458    1.97    1.43    0.54   17.92    1.43    0.54   17.92\n",
      "  224   4570 573L 573L C   CA   4463 4458    2.14    1.49    0.65   19.00    1.49    0.65   19.00\n",
      "  225   4571 573L 574Q C   N    4463 4465    1.81    1.35    0.47   16.53    1.35    0.47   16.53\n",
      "  226   4577 574Q 574Q N   CA   4465 4466    1.97    1.43    0.54   17.76    1.43    0.54   17.76\n",
      "  227   4579 574Q 574Q C   CA   4472 4466    2.18    1.49    0.69   20.08    1.49    0.69   20.08\n",
      "  228   4580 574Q 575S C   N    4472 4474    1.80    1.35    0.45   16.09    1.35    0.45   16.09\n",
      "  229   4583 575S 575S N   CA   4474 4475    1.97    1.43    0.54   17.84    1.43    0.54   17.84\n",
      "  230   4585 575S 575S C   CA   4478 4475    2.21    1.49    0.72   20.84    1.49    0.72   20.84\n",
      "  231   4586 575S 576D C   N    4478 4480    1.87    1.35    0.52   18.62    1.35    0.52   18.62\n",
      "  232   4591 576D 576D N   CA   4480 4481    2.05    1.43    0.62   20.30    1.43    0.62   20.30\n",
      "  233   4593 576D 576D C   CA   4486 4481    2.31    1.49    0.82   23.76    1.49    0.82   23.76\n",
      "  234   4594 576D 577E C   N    4486 4488    1.88    1.35    0.53   18.93    1.35    0.53   18.93\n",
      "  235   4600 577E 577E N   CA   4488 4489    2.07    1.43    0.64   21.05    1.43    0.64   21.05\n",
      "  236   4602 577E 577E C   CA   4495 4489    2.39    1.49    0.90   26.14    1.49    0.90   26.14\n",
      "  237   4603 577E 578V C   N    4495 4497    2.03    1.35    0.69   24.42    1.35    0.69   24.42\n",
      "  238   4607 578V 578V N   CA   4497 4498    2.24    1.43    0.81   26.52    1.43    0.81   26.52\n",
      "  239   4609 578V 578V C   CA   4502 4498    2.68    1.49    1.19   34.67    1.49    1.19   34.67\n",
      "  240   4610 578V 579S C   N    4502 4504    2.10    1.35    0.75   26.76    1.35    0.75   26.76\n",
      "  241   4613 579S 579S N   CA   4504 4505    2.30    1.43    0.87   28.64    1.43    0.87   28.64\n",
      "  242   4615 579S 579S C   CA   4508 4505    2.52    1.49    1.03   30.06    1.49    1.03   30.06\n",
      "  243   4616 579S 580M C   N    4508 4510    2.04    1.35    0.69   24.53    1.35    0.69   24.53\n",
      "  244   4621 580M 580M N   CA   4510 4511    2.23    1.43    0.80   26.44    1.43    0.80   26.44\n",
      "  245   4623 580M 580M C   CA   4516 4511    2.63    1.49    1.14   33.18    1.49    1.14   33.18\n",
      "  246   4624 580M 581P C   N    4516 4518    2.44    1.30    1.14   33.77    1.30    1.14   33.77\n",
      "  247   4626 581P 581P C   CA   4523 4519    2.77    1.49    1.28   37.13    1.49    1.28   37.13\n",
      "  248   4627 581P 582E C   N    4523 4525    2.26    1.35    0.92   32.59    1.35    0.92   32.59\n",
      "  249   4628 581P 581P N   CA   4518 4519    2.36    1.43    0.93   30.45    1.43    0.93   30.45\n",
      "  250   4638 582E 582E N   CA   4525 4526    2.55    1.43    1.12   36.95    1.43    1.12   36.95\n",
      "  251   4640 582E 582E C   CA   4532 4526    2.89    1.49    1.40   40.74    1.49    1.40   40.74\n",
      "  252   4641 582E 583A C   N    4532 4534    2.39    1.35    1.05   37.08    1.35    1.05   37.08\n",
      "  253   4643 583A 583A N   CA   4534 4535    2.71    1.43    1.28   42.07    1.43    1.28   42.07\n",
      "  254   4645 583A 583A C   CA   4537 4535    3.32    1.49    1.83   53.17    1.49    1.83   53.17\n",
      "  255   4646 583A 584Q C   N    4537 4539    2.67    1.35    1.32   46.83    1.35    1.32   46.83\n",
      "  256   4652 584Q 584Q N   CA   4539 4540    2.99    1.43    1.56   51.22    1.43    1.56   51.22\n",
      "  257   4654 584Q 584Q C   CA   4546 4540    3.50    1.49    2.01   58.55    1.49    2.01   58.55\n",
      "  258   4655 584Q 585E C   N    4546 4548    2.74    1.35    1.39   49.29    1.35    1.39   49.29\n",
      "  259   4661 585E 585E N   CA   4548 4549    3.08    1.43    1.65   54.34    1.43    1.65   54.34\n",
      "  260   4663 585E 585E C   CA   4555 4549    3.72    1.49    2.23   64.81    1.49    2.23   64.81\n",
      "  261   4664 585E 586D C   N    4555 4557    2.89    1.35    1.55   54.86    1.35    1.55   54.86\n",
      "  262   4669 586D 586D N   CA   4557 4558    3.27    1.43    1.84   60.67    1.43    1.84   60.67\n",
      "  263   4671 586D 586D C   CA   4563 4558    3.81    1.49    2.32   67.37    1.49    2.32   67.37\n",
      "  264   4672 586D 587L C   N    4563 4565    2.97    1.35    1.62   57.54    1.35    1.62   57.54\n",
      "  265   4677 587L 587L N   CA   4565 4566    3.34    1.43    1.91   62.83    1.43    1.91   62.83\n",
      "  266   4679 587L 587L C   CA   4571 4566    4.00    1.49    2.51   72.83    1.49    2.51   72.83\n",
      "  267   4680 587L 588P C   N    4571 4573    3.84    1.30    2.54   75.28    1.30    2.54   75.28\n",
      "  268   4682 588P 588P C   CA   4578 4574    4.42    1.49    2.93   85.22    1.49    2.93   85.22\n",
      "  269   4683 588P 589H C   N    4578 4580    3.38    1.35    2.04   72.26    1.35    2.04   72.26\n",
      "  270   4684 588P 588P N   CA   4573 4574    3.53    1.43    2.10   69.01    1.43    2.10   69.01\n",
      "  271   4686 588P 588P CB  CG   4576 4577    1.73    1.54    0.19    5.19    1.54    0.19    5.19\n",
      "  272   4689 589H 589H CB  CA   4584 4581    1.74    1.54    0.20    5.42    1.54    0.20    5.42\n",
      "  273   4690 589H 589H CG  CB   4583 4584    1.69    1.50    0.19    5.40    1.50    0.19    5.40\n",
      "  274   4695 589H 589H N   CA   4580 4581    3.84    1.43    2.41   79.25    1.43    2.41   79.25\n",
      "  275   4697 589H 589H C   CA   4588 4581    4.61    1.49    3.12   90.79    1.49    3.12   90.79\n",
      "  276   4698 589H 590K C   N    4588 4590    3.48    1.35    2.13   75.69    1.35    2.13   75.69\n",
      "  277   4705 590K 590K N   CA   4590 4591    3.93    1.43    2.50   82.38    1.43    2.50   82.38\n",
      "  278   4707 590K 590K C   CA   4597 4591    4.73    1.49    3.24   94.13    1.49    3.24   94.13\n",
      "  279   4708 590K 591P C   N    4597 4599    4.63    1.30    3.33   98.78    1.30    3.33   98.78\n",
      "  280   4709 591P 591P O   C    4605 4604    1.40    1.23    0.17    7.85    1.23    0.17    7.85\n",
      "  281   4710 591P 591P C   CA   4604 4600    5.14    1.49    3.65  106.02    1.49    3.65  106.02\n",
      "  282   4711 591P 592T C   N    4604 4606    3.84    1.35    2.50   88.60    1.35    2.50   88.60\n",
      "  283   4712 591P 591P N   CA   4599 4600    4.07    1.43    2.64   86.83    1.43    2.64   86.83\n",
      "  284   4714 591P 591P CB  CG   4602 4603    1.75    1.54    0.22    5.96    1.54    0.22    5.96\n",
      "  285   4720 592T 592T N   CA   4606 4607    4.37    1.43    2.94   96.82    1.43    2.94   96.82\n",
      "  286   4722 592T 592T C   CA   4611 4607    5.36    1.49    3.87  112.55    1.49    3.87  112.55\n",
      "  287   4723 592T 593S C   N    4611 4613    4.05    1.35    2.71   95.97    1.35    2.71   95.97\n",
      "  288   4726 593S 593S N   CA   4613 4614    4.61    1.43    3.18  104.66    1.43    3.18  104.66\n",
      "  289   4728 593S 593S C   CA   4617 4614    5.68    1.49    4.19  121.76    1.49    4.19  121.76\n",
      "  290   4729 593S 594S C   N    4617 4619    4.24    1.35    2.89  102.56    1.35    2.89  102.56\n",
      "  291   4732 594S 594S N   CA   4619 4620    4.83    1.43    3.40  111.93    1.43    3.40  111.93\n",
      "  292   4734 594S 594S C   CA   4623 4620    6.04    1.49    4.55  132.25    1.49    4.55  132.25\n",
      "  293   4735 594S 595S C   N    4623 4625    4.40    1.35    3.06  108.38    1.35    3.06  108.38\n",
      "  294   4738 595S 595S N   CA   4625 4626    5.04    1.43    3.61  118.84    1.43    3.61  118.84\n",
      "  295   4740 595S 595S C   CA   4629 4626    6.20    1.49    4.71  136.81    1.49    4.71  136.81\n",
      "  296   4741 595S 596M C   N    4629 4631    4.60    1.35    3.26  115.45    1.35    3.26  115.45\n",
      "  297   4746 596M 596M N   CA   4631 4632    5.25    1.43    3.82  125.79    1.43    3.82  125.79\n",
      "  298   4747 596M 596M O   C    4638 4637    1.33    1.23    0.10    4.58    1.23    0.10    4.58\n",
      "  299   4748 596M 596M C   CA   4637 4632    6.47    1.49    4.98  144.63    1.49    4.98  144.63\n",
      "  300   4749 596M 597K C   N    4637 4639    4.77    1.35    3.43  121.47    1.35    3.43  121.47\n",
      "  301   4755 597K 597K N   CA   4639 4640    5.46    1.43    4.03  132.59    1.43    4.03  132.59\n",
      "  302   4757 597K 597K C   CA   4646 4640    6.72    1.49    5.23  152.13    1.49    5.23  152.13\n",
      "  303   4758 597K 598T C   N    4646 4648    4.82    1.35    3.47  123.19    1.35    3.47  123.19\n",
      "  304   4762 598T 598T N   CA   4648 4649    5.47    1.43    4.04  132.93    1.43    4.04  132.93\n",
      "  305   4764 598T 598T C   CA   4653 4649    6.89    1.49    5.40  157.01    1.49    5.40  157.01\n",
      "  306   4765 598T 599V C   N    4653 4655    4.99    1.35    3.65  129.26    1.35    3.65  129.26\n",
      "  307   4769 599V 599V N   CA   4655 4656    5.70    1.43    4.27  140.57    1.43    4.27  140.57\n",
      "  308   4771 599V 599V C   CA   4660 4656    6.98    1.49    5.49  159.55    1.49    5.49  159.55\n",
      "  309   4772 599V 600D C   N    4660 4662    5.05    1.35    3.70  131.26    1.35    3.70  131.26\n",
      "  310   4777 600D 600D N   CA   4662 4663    5.78    1.43    4.35  143.08    1.43    4.35  143.08\n",
      "  311   4779 600D 600D C   CA   4668 4663    7.10    1.49    5.61  163.12    1.49    5.61  163.12\n",
      "  312   4780 600D 601E C   N    4668 4670    5.19    1.35    3.84  136.34    1.35    3.84  136.34\n",
      "  313   4786 601E 601E N   CA   4670 4671    5.90    1.43    4.47  147.01    1.43    4.47  147.01\n",
      "  314   4788 601E 601E C   CA   4677 4671    7.29    1.49    5.80  168.53    1.49    5.80  168.53\n",
      "  315   4789 601E 602E C   N    4677 4679    5.27    1.35    3.92  139.18    1.35    3.92  139.18\n",
      "  316   4795 602E 602E N   CA   4679 4680    6.01    1.43    4.58  150.53    1.43    4.58  150.53\n",
      "  317   4797 602E 602E C   CA   4686 4680    7.25    1.49    5.76  167.44    1.49    5.76  167.44\n",
      "  318   4798 602E 603A C   N    4686 4688    5.27    1.35    3.92  139.01    1.35    3.92  139.01\n",
      "  319   4800 603A 603A N   CA   4688 4689    6.00    1.43    4.57  150.26    1.43    4.57  150.26\n",
      "  320   4802 603A 603A C   CA   4691 4689    7.34    1.49    5.85  169.95    1.49    5.85  169.95\n",
      "  321   4803 603A 604L C   N    4691 4693    5.24    1.35    3.90  138.21    1.35    3.90  138.21\n",
      "  322   4808 604L 604L N   CA   4693 4694    5.96    1.43    4.53  149.08    1.43    4.53  149.08\n",
      "  323   4810 604L 604L C   CA   4699 4694    7.24    1.49    5.75  167.19    1.49    5.75  167.19\n",
      "  324   4811 604L 605A C   N    4699 4701    5.17    1.35    3.83  135.65    1.35    3.83  135.65\n",
      "  325   4813 605A 605A N   CA   4701 4702    5.85    1.43    4.42  145.34    1.43    4.42  145.34\n",
      "  326   4815 605A 605A C   CA   4704 4702    7.05    1.49    5.56  161.55    1.49    5.56  161.55\n",
      "  327   4816 605A 606M C   N    4704 4706    5.04    1.35    3.69  131.01    1.35    3.69  131.01\n",
      "  328   4821 606M 606M N   CA   4706 4707    4.69    1.43    3.26  107.27    1.43    3.26  107.27\n",
      "  329   4822 606M 606M O   C    4713 4712    1.50    1.23    0.27   12.23    1.23    0.27   12.23\n",
      "  330   4823 606M 606M C   CA   4712 4707    3.27    1.49    1.78   51.63    1.49    1.78   51.63\n",
      "  331   4824 606M 607L C   N    4712 4714    1.95    1.35    0.60   21.29    1.35    0.60   21.29\n",
      "  332   4829 607L 607L N   CA   4714 4715    1.93    1.43    0.50   16.33    1.43    0.50   16.33\n",
      "  333   4831 607L 607L C   CA   4720 4715    1.69    1.49    0.20    5.81    1.49    0.20    5.81\n",
      "  334   4846 609S 610M C   N    4734 4736    1.47    1.35    0.13    4.58    1.35    0.13    4.58\n",
      "  335   4851 610M 610M N   CA   4736 4737    1.58    1.43    0.15    4.84    1.43    0.15    4.84\n",
      "  336   4853 610M 610M C   CA   4742 4737    1.67    1.49    0.18    5.21    1.49    0.18    5.21\n",
      "  337   4854 610M 611G C   N    4742 4744    1.48    1.35    0.14    4.91    1.35    0.14    4.91\n",
      "  338   4855 611G 611G N   CA   4744 4745    1.58    1.43    0.15    4.88    1.43    0.15    4.88\n",
      "  339   4857 611G 611G C   CA   4746 4745    1.66    1.49    0.17    5.03    1.49    0.17    5.03\n",
      "  340   4872 613E 613E N   CA   4756 4757    1.57    1.43    0.14    4.72    1.43    0.14    4.72\n",
      "  341   4874 613E 613E C   CA   4763 4757    1.65    1.49    0.16    4.70    1.49    0.16    4.70\n",
      "  342   4883 614E 614E C   CA   4772 4766    1.66    1.49    0.17    4.92    1.49    0.17    4.92\n",
      "  343   4902 617A 617A C   CA   4791 4789    1.65    1.49    0.16    4.79    1.49    0.16    4.79\n",
      "  344   4911 618R 618R N   CA   4793 4794    1.57    1.43    0.14    4.52    1.43    0.14    4.52\n",
      "  345   4924 619Y 619Y N   CA   4804 4805    1.57    1.43    0.14    4.64    1.43    0.14    4.64\n",
      "  346   4926 619Y 619Y C   CA   4814 4805    1.65    1.49    0.16    4.54    1.49    0.16    4.54\n",
      "  347   5546 700L 700L C   CA   5420 5415    1.65    1.49    0.16    4.66    1.49    0.16    4.66\n",
      "  348   5552 701L 701L N   CA   5422 5423    1.57    1.43    0.14    4.76    1.43    0.14    4.76\n",
      "  349   5554 701L 701L C   CA   5428 5423    1.68    1.49    0.19    5.64    1.49    0.19    5.64\n",
      "  350   5555 701L 702F C   N    5428 5430    1.49    1.35    0.14    5.00    1.35    0.14    5.00\n",
      "  351   5564 702F 702F N   CA   5430 5431    1.60    1.43    0.17    5.50    1.43    0.17    5.50\n",
      "  352   5566 702F 702F C   CA   5439 5431    1.71    1.49    0.22    6.46    1.49    0.22    6.46\n",
      "  353   5567 702F 703N C   N    5439 5441    1.52    1.35    0.18    6.23    1.35    0.18    6.23\n",
      "  354   5572 703N 703N N   CA   5441 5442    1.65    1.43    0.22    7.08    1.43    0.22    7.08\n",
      "  355   5574 703N 703N C   CA   5447 5442    1.75    1.49    0.26    7.69    1.49    0.26    7.69\n",
      "  356   5575 703N 704D C   N    5447 5449    1.54    1.35    0.19    6.88    1.35    0.19    6.88\n",
      "  357   5580 704D 704D N   CA   5449 5450    1.65    1.43    0.22    7.34    1.43    0.22    7.34\n",
      "  358   5582 704D 704D C   CA   5455 5450    1.78    1.49    0.29    8.51    1.49    0.29    8.51\n",
      "  359   5583 704D 705E C   N    5455 5457    1.54    1.35    0.19    6.74    1.35    0.19    6.74\n",
      "  360   5589 705E 705E N   CA   5457 5458    1.65    1.43    0.22    7.14    1.43    0.22    7.14\n",
      "  361   5591 705E 705E C   CA   5464 5458    1.76    1.49    0.27    7.76    1.49    0.27    7.76\n",
      "  362   5592 705E 706K C   N    5464 5466    1.56    1.35    0.21    7.45    1.35    0.21    7.45\n",
      "  363   5598 706K 706K N   CA   5466 5467    1.66    1.43    0.23    7.55    1.43    0.23    7.55\n",
      "  364   5600 706K 706K C   CA   5473 5467    1.78    1.49    0.29    8.46    1.49    0.29    8.46\n",
      "  365   5601 706K 707V C   N    5473 5475    1.57    1.35    0.23    8.11    1.35    0.23    8.11\n",
      "  366   5605 707V 707V N   CA   5475 5476    1.70    1.43    0.27    8.96    1.43    0.27    8.96\n",
      "  367   5607 707V 707V C   CA   5480 5476    1.82    1.49    0.33    9.58    1.49    0.33    9.58\n",
      "  368   5608 707V 708A C   N    5480 5482    1.58    1.35    0.24    8.41    1.35    0.24    8.41\n",
      "  369   5610 708A 708A N   CA   5482 5483    1.71    1.43    0.28    9.36    1.43    0.28    9.36\n",
      "  370   5612 708A 708A C   CA   5485 5483    1.83    1.49    0.34    9.79    1.49    0.34    9.79\n",
      "  371   5613 708A 709E C   N    5485 5487    1.58    1.35    0.23    8.24    1.35    0.23    8.24\n",
      "  372   5619 709E 709E N   CA   5487 5488    1.70    1.43    0.27    9.04    1.43    0.27    9.04\n",
      "  373   5621 709E 709E C   CA   5494 5488    1.76    1.49    0.27    7.72    1.49    0.27    7.72\n",
      "  374   5622 709E 710S C   N    5494 5496    1.55    1.35    0.20    7.23    1.35    0.20    7.23\n",
      "  375   5625 710S 710S N   CA   5496 5497    1.65    1.43    0.22    7.17    1.43    0.22    7.17\n",
      "  376   5627 710S 710S C   CA   5500 5497    1.82    1.49    0.33    9.50    1.49    0.33    9.50\n",
      "  377   5628 710S 711L C   N    5500 5502    1.55    1.35    0.21    7.43    1.35    0.21    7.43\n",
      "  378   5633 711L 711L N   CA   5502 5503    1.68    1.43    0.25    8.20    1.43    0.25    8.20\n",
      "  379   5635 711L 711L C   CA   5508 5503    1.76    1.49    0.27    7.79    1.49    0.27    7.79\n",
      "  380   5636 711L 712N C   N    5508 5510    1.61    1.35    0.27    9.52    1.35    0.27    9.52\n",
      "  381   5641 712N 712N N   CA   5510 5511    1.75    1.43    0.32   10.67    1.43    0.32   10.67\n",
      "  382   5643 712N 712N C   CA   5516 5511    1.83    1.49    0.34    9.78    1.49    0.34    9.78\n",
      "  383   5644 712N 713P C   N    5516 5518    1.60    1.30    0.30    9.03    1.30    0.30    9.03\n",
      "  384   5646 713P 713P C   CA   5523 5519    1.85    1.49    0.36   10.37    1.49    0.36   10.37\n",
      "  385   5647 713P 714P C   N    5523 5525    1.72    1.30    0.42   12.38    1.30    0.42   12.38\n",
      "  386   5648 713P 713P N   CA   5518 5519    1.73    1.43    0.30    9.85    1.43    0.30    9.85\n",
      "  387   5654 714P 714P C   CA   5530 5526    1.99    1.49    0.50   14.45    1.49    0.50   14.45\n",
      "  388   5655 714P 715F C   N    5530 5532    1.78    1.35    0.43   15.29    1.35    0.43   15.29\n",
      "  389   5656 714P 714P N   CA   5525 5526    1.78    1.43    0.34   11.23    1.43    0.34   11.23\n",
      "  390   5669 715F 715F N   CA   5532 5533    1.93    1.43    0.50   16.49    1.43    0.50   16.49\n",
      "  391   5671 715F 715F C   CA   5541 5533    1.98    1.49    0.49   14.35    1.49    0.49   14.35\n",
      "  392   5672 715F 716S C   N    5541 5543    1.64    1.35    0.30   10.50    1.35    0.30   10.50\n",
      "  393   5675 716S 716S N   CA   5543 5544    1.79    1.43    0.36   11.75    1.43    0.36   11.75\n",
      "  394   5677 716S 716S C   CA   5547 5544    2.03    1.49    0.54   15.63    1.49    0.54   15.63\n",
      "  395   5678 716S 717M C   N    5547 5549    1.80    1.35    0.46   16.18    1.35    0.46   16.18\n",
      "  396   5683 717M 717M N   CA   5549 5550    1.97    1.43    0.54   17.70    1.43    0.54   17.70\n",
      "  397   5685 717M 717M C   CA   5555 5550    2.22    1.49    0.73   21.32    1.49    0.73   21.32\n",
      "  398   5686 717M 718A C   N    5555 5557    1.84    1.35    0.50   17.63    1.35    0.50   17.63\n",
      "  399   5688 718A 718A N   CA   5557 5558    2.01    1.43    0.58   19.21    1.43    0.58   19.21\n",
      "  400   5690 718A 718A C   CA   5560 5558    2.17    1.49    0.68   19.72    1.49    0.68   19.72\n",
      "  401   5691 718A 719F C   N    5560 5562    1.83    1.35    0.48   17.09    1.35    0.48   17.09\n",
      "  402   5700 719F 719F N   CA   5562 5563    1.99    1.43    0.56   18.56    1.43    0.56   18.56\n",
      "  403   5702 719F 719F C   CA   5571 5563    2.40    1.49    0.91   26.57    1.49    0.91   26.57\n",
      "  404   5703 719F 720L C   N    5571 5573    1.93    1.35    0.58   20.73    1.35    0.58   20.73\n",
      "  405   5708 720L 720L N   CA   5573 5574    2.14    1.43    0.71   23.27    1.43    0.71   23.27\n",
      "  406   5710 720L 720L C   CA   5579 5574    2.38    1.49    0.89   25.76    1.49    0.89   25.76\n",
      "  407   5711 720L 721Y C   N    5579 5581    1.96    1.35    0.62   21.95    1.35    0.62   21.95\n",
      "  408   5721 721Y 721Y N   CA   5581 5582    2.16    1.43    0.73   24.15    1.43    0.73   24.15\n",
      "  409   5723 721Y 721Y C   CA   5591 5582    2.59    1.49    1.10   32.07    1.49    1.10   32.07\n",
      "  410   5724 721Y 722F C   N    5591 5593    2.08    1.35    0.74   26.18    1.35    0.74   26.18\n",
      "  411   5733 722F 722F N   CA   5593 5594    2.32    1.43    0.89   29.25    1.43    0.89   29.25\n",
      "  412   5735 722F 722F C   CA   5602 5594    2.68    1.49    1.19   34.68    1.49    1.19   34.68\n",
      "  413   5736 722F 723Y C   N    5602 5604    2.16    1.35    0.81   28.73    1.35    0.81   28.73\n",
      "  414   5737 723Y 723Y CB  CA   5606 5605    1.75    1.54    0.21    5.71    1.54    0.21    5.71\n",
      "  415   5738 723Y 723Y CG  CB   5607 5606    1.67    1.49    0.18    5.00    1.49    0.18    5.00\n",
      "  416   5746 723Y 723Y N   CA   5604 5605    2.40    1.43    0.97   31.77    1.43    0.97   31.77\n",
      "  417   5748 723Y 723Y C   CA   5614 5605    2.93    1.49    1.44   41.72    1.49    1.44   41.72\n",
      "  418   5749 723Y 724K C   N    5614 5616    2.29    1.35    0.95   33.56    1.35    0.95   33.56\n",
      "  419   5755 724K 724K N   CA   5616 5617    2.56    1.43    1.13   37.07    1.43    1.13   37.07\n",
      "  420   5757 724K 724K C   CA   5623 5617    2.95    1.49    1.46   42.39    1.49    1.46   42.39\n",
      "  421   5758 724K 725R C   N    5623 5625    2.38    1.35    1.04   36.88    1.35    1.04   36.88\n",
      "  422   5766 725R 725R N   CA   5625 5626    2.69    1.43    1.26   41.39    1.43    1.26   41.39\n",
      "  423   5768 725R 725R C   CA   5634 5626    3.18    1.49    1.69   49.03    1.49    1.69   49.03\n",
      "  424   5769 725R 726V C   N    5634 5636    2.48    1.35    1.14   40.38    1.35    1.14   40.38\n",
      "  425   5773 726V 726V N   CA   5636 5637    2.74    1.43    1.31   43.15    1.43    1.31   43.15\n",
      "  426   5775 726V 726V C   CA   5641 5637    3.27    1.49    1.78   51.60    1.49    1.78   51.60\n",
      "  427   5776 726V 727G C   N    5641 5643    2.47    1.35    1.13   40.02    1.35    1.13   40.02\n",
      "  428   5777 727G 727G N   CA   5643 5644    2.75    1.43    1.32   43.29    1.43    1.32   43.29\n",
      "  429   5779 727G 727G C   CA   5645 5644    3.17    1.49    1.68   48.92    1.49    1.68   48.92\n",
      "  430   5780 727G 728K C   N    5645 5647    2.54    1.35    1.20   42.53    1.35    1.20   42.53\n",
      "  431   5786 728K 728K N   CA   5647 5648    2.42    1.43    0.99   32.54    1.43    0.99   32.54\n",
      "  432   5787 728K 728K O   C    5655 5654    1.50    1.26    0.24    9.92    1.26    0.24    9.92\n",
      "  433   5788 728K 728K C   CA   5654 5648    2.05    1.52    0.53   13.85    1.52    0.53   13.85\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5895  10E  10E N   CA     74   75  123.20  107.00   16.20    4.66  107.00   16.20    4.66\n",
      "    2   5907  11A  11A N   CA     83   84  123.34  107.00   16.34    4.70  107.00   16.34    4.70\n",
      "    3   5914  12C  12C N   CA     88   89  124.23  107.00   17.23    4.96  107.00   17.23    4.96\n",
      "    4   5922  13A  13A N   CA     94   95  124.38  107.00   17.38    5.00  107.00   17.38    5.00\n",
      "    5   5929  14H  14H N   CA     99  100  124.72  107.00   17.72    5.09  107.00   17.72    5.09\n",
      "    6   5944  15C  15C N   CA    109  110  127.39  107.00   20.39    5.86  107.00   20.39    5.86\n",
      "    7   5952  16W  16W N   CA    115  116  126.76  107.00   19.76    5.68  107.00   19.76    5.68\n",
      "    8   5975  17D  17D N   CA    129  130  129.70  107.00   22.70    6.53  107.00   22.70    6.53\n",
      "    9   5986  18E  18E N   CA    137  138  130.29  107.00   23.29    6.70  107.00   23.29    6.70\n",
      "   10   5989  18E  18E CA  C     138  144  132.45  116.50   15.96    4.59  116.50   15.96    4.59\n",
      "   11   5998  19S  19S N   CA    146  147  132.02  107.00   25.02    7.20  107.00   25.02    7.20\n",
      "   12   6001  19S  19S CA  C     147  150  133.08  116.50   16.58    4.77  116.50   16.58    4.77\n",
      "   13   6006  20V  20V N   CA    152  153  132.74  107.00   25.74    7.40  107.00   25.74    7.40\n",
      "   14   6010  20V  20V CA  C     153  157  134.01  116.50   17.51    5.04  116.50   17.51    5.04\n",
      "   15   6013  20V  21M C   N     157  159  139.96  120.00   19.96    4.54  120.00   19.96    4.54\n",
      "   16   6016  21M  21M N   CA    159  160  133.60  107.00   26.60    7.65  107.00   26.60    7.65\n",
      "   17   6019  21M  21M CA  C     160  165  134.56  116.50   18.06    5.19  116.50   18.06    5.19\n",
      "   18   6026  22L  22L N   CA    167  168  134.42  107.00   27.42    7.88  107.00   27.42    7.88\n",
      "   19   6029  22L  22L CA  C     168  173  135.53  116.50   19.03    5.47  116.50   19.03    5.47\n",
      "   20   6034  22L  23V C   N     173  175  140.03  120.00   20.03    4.55  120.00   20.03    4.55\n",
      "   21   6037  23V  23V N   CA    175  176  138.03  107.00   31.03    8.92  107.00   31.03    8.92\n",
      "   22   6044  23V  24P C   N     180  182  145.64  117.00   28.64    7.13  117.00   28.64    7.13\n",
      "   23   6047  24P  24P N   CA    182  183  144.19  108.20   36.00    8.18  108.20   36.00    8.18\n",
      "   24   6051  24P  24P CA  C     183  187  137.31  116.50   20.81    5.98  116.50   20.81    5.98\n",
      "   25   6056  24P  25N C   N     187  189  141.41  120.00   21.41    4.86  120.00   21.41    4.86\n",
      "   26   6059  25N  25N N   CA    189  190  136.52  107.00   29.53    8.49  107.00   29.53    8.49\n",
      "   27   6062  25N  25N CA  C     190  195  137.18  116.50   20.68    5.95  116.50   20.68    5.95\n",
      "   28   6067  25N  26H C   N     195  197  141.84  120.00   21.84    4.96  120.00   21.84    4.96\n",
      "   29   6070  26H  26H N   CA    197  198  137.91  107.00   30.91    8.89  107.00   30.91    8.89\n",
      "   30   6072  26H  26H CA  C     198  205  137.87  116.50   21.38    6.15  116.50   21.38    6.15\n",
      "   31   6082  26H  27R C   N     205  207  144.19  120.00   24.19    5.50  120.00   24.19    5.50\n",
      "   32   6085  27R  27R N   CA    207  208  139.73  107.00   32.73    9.41  107.00   32.73    9.41\n",
      "   33   6088  27R  27R CA  C     208  216  138.82  116.50   22.32    6.42  116.50   22.32    6.42\n",
      "   34   6096  27R  28S C   N     216  218  143.18  120.00   23.18    5.27  120.00   23.18    5.27\n",
      "   35   6099  28S  28S N   CA    218  219  139.58  107.00   32.58    9.37  107.00   32.58    9.37\n",
      "   36   6102  28S  28S CA  C     219  222  138.76  116.50   22.26    6.40  116.50   22.26    6.40\n",
      "   37   6104  28S  29A C   N     222  224  143.97  120.00   23.97    5.45  120.00   23.97    5.45\n",
      "   38   6107  29A  29A N   CA    224  225  139.82  107.00   32.82    9.44  107.00   32.82    9.44\n",
      "   39   6109  29A  29A CA  C     225  227  139.52  116.50   23.02    6.62  116.50   23.02    6.62\n",
      "   40   6111  29A  30V C   N     227  229  144.28  120.00   24.28    5.52  120.00   24.28    5.52\n",
      "   41   6114  30V  30V N   CA    229  230  140.25  107.00   33.25    9.56  107.00   33.25    9.56\n",
      "   42   6118  30V  30V CA  C     230  234  140.24  116.50   23.74    6.83  116.50   23.74    6.83\n",
      "   43   6121  30V  31H C   N     234  236  144.69  120.00   24.69    5.61  120.00   24.69    5.61\n",
      "   44   6124  31H  31H N   CA    236  237  140.38  107.00   33.38    9.60  107.00   33.38    9.60\n",
      "   45   6126  31H  31H CA  C     237  244  141.41  116.50   24.91    7.16  116.50   24.91    7.16\n",
      "   46   6136  31H  32K C   N     244  246  145.19  120.00   25.19    5.72  120.00   25.19    5.72\n",
      "   47   6139  32K  32K N   CA    246  247  142.21  107.00   35.21   10.12  107.00   35.21   10.12\n",
      "   48   6142  32K  32K CA  C     247  253  141.96  116.50   25.46    7.32  116.50   25.46    7.32\n",
      "   49   6147  32K  33E C   N     253  255  147.07  120.00   27.07    6.15  120.00   27.07    6.15\n",
      "   50   6150  33E  33E N   CA    255  256  142.81  107.00   35.81   10.30  107.00   35.81   10.30\n",
      "   51   6153  33E  33E CA  C     256  262  142.68  116.50   26.18    7.53  116.50   26.18    7.53\n",
      "   52   6159  33E  34E C   N     262  264  146.86  120.00   26.86    6.11  120.00   26.86    6.11\n",
      "   53   6162  34E  34E N   CA    264  265  144.06  107.00   37.06   10.66  107.00   37.06   10.66\n",
      "   54   6165  34E  34E CA  C     265  271  143.16  116.50   26.66    7.67  116.50   26.66    7.67\n",
      "   55   6171  34E  35C C   N     271  273  147.24  120.00   27.24    6.19  120.00   27.24    6.19\n",
      "   56   6174  35C  35C N   CA    273  274  143.99  107.00   36.99   10.64  107.00   36.99   10.64\n",
      "   57   6177  35C  35C CA  C     274  277  143.39  116.50   26.89    7.73  116.50   26.89    7.73\n",
      "   58   6179  35C  36A C   N     277  279  147.76  120.00   27.76    6.31  120.00   27.76    6.31\n",
      "   59   6182  36A  36A N   CA    279  280  144.64  107.00   37.65   10.82  107.00   37.65   10.82\n",
      "   60   6184  36A  36A CA  C     280  282  143.48  116.50   26.98    7.76  116.50   26.98    7.76\n",
      "   61   6186  36A  37Y C   N     282  284  148.37  120.00   28.37    6.45  120.00   28.37    6.45\n",
      "   62   6189  37Y  37Y N   CA    284  285  144.93  107.00   37.93   10.91  107.00   37.93   10.91\n",
      "   63   6192  37Y  37Y CA  C     285  294  144.25  116.50   27.75    7.98  116.50   27.75    7.98\n",
      "   64   6204  37Y  38C C   N     294  296  148.79  120.00   28.79    6.54  120.00   28.79    6.54\n",
      "   65   6207  38C  38C N   CA    296  297  145.87  107.00   38.87   11.18  107.00   38.87   11.18\n",
      "   66   6210  38C  38C CA  C     297  300  144.45  116.50   27.95    8.04  116.50   27.95    8.04\n",
      "   67   6212  38C  39C C   N     300  302  149.21  120.00   29.21    6.64  120.00   29.21    6.64\n",
      "   68   6215  39C  39C N   CA    302  303  146.14  107.00   39.14   11.25  107.00   39.14   11.25\n",
      "   69   6218  39C  39C CA  C     303  306  145.23  116.50   28.73    8.26  116.50   28.73    8.26\n",
      "   70   6220  39C  40R C   N     306  308  148.37  120.00   28.37    6.45  120.00   28.37    6.45\n",
      "   71   6223  40R  40R N   CA    308  309  146.21  107.00   39.21   11.27  107.00   39.21   11.27\n",
      "   72   6226  40R  40R CA  C     309  317  145.94  116.50   29.44    8.47  116.50   29.44    8.47\n",
      "   73   6234  40R  41T C   N     317  319  149.45  120.00   29.45    6.69  120.00   29.45    6.69\n",
      "   74   6237  41T  41T N   CA    319  320  146.98  107.00   39.98   11.50  107.00   39.98   11.50\n",
      "   75   6241  41T  41T CA  C     320  324  146.10  116.50   29.60    8.51  116.50   29.60    8.51\n",
      "   76   6244  41T  42C C   N     324  326  150.19  120.00   30.19    6.86  120.00   30.19    6.86\n",
      "   77   6247  42C  42C N   CA    326  327  147.58  107.00   40.58   11.67  107.00   40.58   11.67\n",
      "   78   6250  42C  42C CA  C     327  330  146.48  116.50   29.98    8.62  116.50   29.98    8.62\n",
      "   79   6252  42C  43R C   N     330  332  150.20  120.00   30.20    6.86  120.00   30.20    6.86\n",
      "   80   6255  43R  43R N   CA    332  333  147.30  107.00   40.30   11.59  107.00   40.30   11.59\n",
      "   81   6258  43R  43R CA  C     333  341  147.36  116.50   30.86    8.87  116.50   30.86    8.87\n",
      "   82   6266  43R  44H C   N     341  343  150.22  120.00   30.22    6.87  120.00   30.22    6.87\n",
      "   83   6269  44H  44H N   CA    343  344  148.91  107.00   41.91   12.05  107.00   41.91   12.05\n",
      "   84   6271  44H  44H CA  C     344  351  147.83  116.50   31.33    9.01  116.50   31.33    9.01\n",
      "   85   6281  44H  45E C   N     351  353  151.84  120.00   31.84    7.23  120.00   31.84    7.23\n",
      "   86   6284  45E  45E N   CA    353  354  149.36  107.00   42.36   12.18  107.00   42.36   12.18\n",
      "   87   6287  45E  45E CA  C     354  360  148.43  116.50   31.93    9.18  116.50   31.93    9.18\n",
      "   88   6293  45E  46M C   N     360  362  151.72  120.00   31.72    7.21  120.00   31.72    7.21\n",
      "   89   6296  46M  46M N   CA    362  363  149.84  107.00   42.84   12.32  107.00   42.84   12.32\n",
      "   90   6299  46M  46M CA  C     363  368  148.81  116.50   32.31    9.29  116.50   32.31    9.29\n",
      "   91   6303  46M  47G C   N     368  370  151.24  120.00   31.24    7.10  120.00   31.24    7.10\n",
      "   92   6305  47G  47G N   CA    370  371  150.09  107.00   43.09   12.39  107.00   43.09   12.39\n",
      "   93   6307  47G  47G CA  C     371  372  134.58  116.50   18.08    5.20  116.50   18.08    5.20\n",
      "   94   6308  47G  48T C   N     372  374  146.62  120.00   26.62    6.05  120.00   26.62    6.05\n",
      "   95   6318  48T  49L C   N     379  381  145.69  120.00   25.69    5.84  120.00   25.69    5.84\n",
      "   96   7166 129C 130Y C   N     989  991  141.76  120.00   21.76    4.94  120.00   21.76    4.94\n",
      "   97   7170 130Y 130Y CA  CB    992  993  127.88  107.50   20.38    4.71  107.50   20.38    4.71\n",
      "   98   7460 158P 158P N   CA   1200 1201  136.23  108.20   28.03    6.37  108.20   28.03    6.37\n",
      "   99   7469 158P 159H C   N    1205 1207  146.16  120.00   26.16    5.95  120.00   26.16    5.95\n",
      "  100   8174 225V 226V C   N    1709 1711  141.06  120.00   21.06    4.79  120.00   21.06    4.79\n",
      "  101   8269 236A 236A N   CA   1777 1778  123.04  107.00   16.04    4.61  107.00   16.04    4.61\n",
      "  102   8558 260G 261I C   N    1982 1984  144.78  120.00   24.78    5.63  120.00   24.78    5.63\n",
      "  103   8561 261I 261I N   CA   1984 1985  125.40  107.00   18.40    5.29  107.00   18.40    5.29\n",
      "  104   8569 261I 262D C   N    1990 1992  148.86  120.00   28.86    6.56  120.00   28.86    6.56\n",
      "  105   8572 262D 262D N   CA   1992 1993  136.49  107.00   29.49    8.48  107.00   29.49    8.48\n",
      "  106   8575 262D 262D CA  C    1993 1998  136.73  116.50   20.23    5.82  116.50   20.23    5.82\n",
      "  107   8580 262D 263I C   N    1998 2000  143.57  120.00   23.57    5.36  120.00   23.57    5.36\n",
      "  108   8583 263I 263I N   CA   2000 2001  137.96  107.00   30.96    8.90  107.00   30.96    8.90\n",
      "  109   8587 263I 263I CA  C    2001 2006  136.39  116.50   19.89    5.72  116.50   19.89    5.72\n",
      "  110   8594 264Q 264Q N   CA   2008 2009  134.84  107.00   27.84    8.01  107.00   27.84    8.01\n",
      "  111   8606 265T 265T N   CA   2017 2018  131.45  107.00   24.45    7.03  107.00   24.45    7.03\n",
      "  112   8610 265T 265T CA  C    2018 2022  134.05  116.50   17.55    5.05  116.50   17.55    5.05\n",
      "  113   8616 266A 266A N   CA   2024 2025  131.72  107.00   24.72    7.11  107.00   24.72    7.11\n",
      "  114   8623 267K 267K N   CA   2029 2030  131.46  107.00   24.46    7.03  107.00   24.46    7.03\n",
      "  115   8634 268K 268K N   CA   2038 2039  129.33  107.00   22.33    6.42  107.00   22.33    6.42\n",
      "  116   8645 269T 269T N   CA   2047 2048  129.60  107.00   22.60    6.50  107.00   22.60    6.50\n",
      "  117   8655 270A 270A N   CA   2054 2055  129.27  107.00   22.27    6.40  107.00   22.27    6.40\n",
      "  118   8662 271K 271K N   CA   2059 2060  127.91  107.00   20.91    6.01  107.00   20.91    6.01\n",
      "  119   8673 272T 272T N   CA   2068 2069  126.84  107.00   19.84    5.71  107.00   19.84    5.71\n",
      "  120   8683 273L 273L N   CA   2075 2076  127.58  107.00   20.58    5.92  107.00   20.58    5.92\n",
      "  121   8693 274G 274G N   CA   2083 2084  127.99  107.00   20.99    6.03  107.00   20.99    6.03\n",
      "  122   8699 275E 275E N   CA   2087 2088  126.07  107.00   19.07    5.48  107.00   19.07    5.48\n",
      "  123   8711 276M 276M N   CA   2096 2097  125.41  107.00   18.41    5.29  107.00   18.41    5.29\n",
      "  124   8721 277Q 277Q N   CA   2104 2105  124.78  107.00   17.78    5.11  107.00   17.78    5.11\n",
      "  125   8751 279D 279D N   CA   2125 2126  122.66  107.00   15.66    4.50  107.00   15.66    4.50\n",
      "  126  11786 556F 556F N   CA   4320 4321  125.35  107.00   18.35    5.28  107.00   18.35    5.28\n",
      "  127  11802 557V 557V N   CA   4331 4332  126.58  107.00   19.58    5.63  107.00   19.58    5.63\n",
      "  128  11812 558D 558D N   CA   4338 4339  126.56  107.00   19.56    5.62  107.00   19.56    5.62\n",
      "  129  11823 559V 559V N   CA   4346 4347  129.33  107.00   22.33    6.42  107.00   22.33    6.42\n",
      "  130  11830 559V 560P C   N    4351 4353  142.63  117.00   25.63    6.38  117.00   25.63    6.38\n",
      "  131  11833 560P 560P N   CA   4353 4354  142.13  108.20   33.94    7.71  108.20   33.94    7.71\n",
      "  132  11845 561L 561L N   CA   4360 4361  130.70  107.00   23.70    6.81  107.00   23.70    6.81\n",
      "  133  11848 561L 561L CA  C    4361 4366  132.83  116.50   16.34    4.70  116.50   16.34    4.70\n",
      "  134  11856 562E 562E N   CA   4368 4369  132.77  107.00   25.78    7.41  107.00   25.78    7.41\n",
      "  135  11859 562E 562E CA  C    4369 4375  134.37  116.50   17.87    5.14  116.50   17.87    5.14\n",
      "  136  11865 562E 563L C   N    4375 4377  140.43  120.00   20.43    4.64  120.00   20.43    4.64\n",
      "  137  11868 563L 563L N   CA   4377 4378  133.26  107.00   26.26    7.55  107.00   26.26    7.55\n",
      "  138  11871 563L 563L CA  C    4378 4383  136.89  116.50   20.39    5.86  116.50   20.39    5.86\n",
      "  139  11876 563L 564D C   N    4383 4385  140.42  120.00   20.42    4.64  120.00   20.42    4.64\n",
      "  140  11879 564D 564D N   CA   4385 4386  136.19  107.00   29.19    8.39  107.00   29.19    8.39\n",
      "  141  11882 564D 564D CA  C    4386 4391  137.58  116.50   21.08    6.06  116.50   21.08    6.06\n",
      "  142  11887 564D 565L C   N    4391 4393  142.34  120.00   22.34    5.08  120.00   22.34    5.08\n",
      "  143  11890 565L 565L N   CA   4393 4394  138.62  107.00   31.62    9.09  107.00   31.62    9.09\n",
      "  144  11893 565L 565L CA  C    4394 4399  139.32  116.50   22.82    6.56  116.50   22.82    6.56\n",
      "  145  11898 565L 566E C   N    4399 4401  141.99  120.00   21.99    5.00  120.00   21.99    5.00\n",
      "  146  11901 566E 566E N   CA   4401 4402  139.46  107.00   32.46    9.33  107.00   32.46    9.33\n",
      "  147  11904 566E 566E CA  C    4402 4408  140.95  116.50   24.45    7.03  116.50   24.45    7.03\n",
      "  148  11910 566E 567F C   N    4408 4410  144.75  120.00   24.75    5.62  120.00   24.75    5.62\n",
      "  149  11913 567F 567F N   CA   4410 4411  141.22  107.00   34.22    9.84  107.00   34.22    9.84\n",
      "  150  11916 567F 567F CA  C    4411 4419  142.28  116.50   25.78    7.41  116.50   25.78    7.41\n",
      "  151  11926 567F 568L C   N    4419 4421  145.68  120.00   25.68    5.84  120.00   25.68    5.84\n",
      "  152  11929 568L 568L N   CA   4421 4422  143.27  107.00   36.27   10.43  107.00   36.27   10.43\n",
      "  153  11932 568L 568L CA  C    4422 4427  144.77  116.50   28.27    8.13  116.50   28.27    8.13\n",
      "  154  11937 568L 569R C   N    4427 4429  147.45  120.00   27.44    6.24  120.00   27.44    6.24\n",
      "  155  11940 569R 569R N   CA   4429 4430  142.62  107.00   35.62   10.24  107.00   35.62   10.24\n",
      "  156  11943 569R 569R CA  C    4430 4438  146.28  116.50   29.78    8.56  116.50   29.78    8.56\n",
      "  157  11951 569R 570G C   N    4438 4440  149.97  120.00   29.97    6.81  120.00   29.97    6.81\n",
      "  158  11953 570G 570G N   CA   4440 4441  147.79  107.00   40.80   11.73  107.00   40.80   11.73\n",
      "  159  11955 570G 570G CA  C    4441 4442  145.66  116.50   29.16    8.39  116.50   29.16    8.39\n",
      "  160  11956 570G 571K C   N    4442 4444  148.30  120.00   28.30    6.43  120.00   28.30    6.43\n",
      "  161  11959 571K 571K N   CA   4444 4445  147.22  107.00   40.22   11.57  107.00   40.22   11.57\n",
      "  162  11962 571K 571K CA  C    4445 4451  147.32  116.50   30.82    8.86  116.50   30.82    8.86\n",
      "  163  11967 571K 572G C   N    4451 4453  149.64  120.00   29.64    6.74  120.00   29.64    6.74\n",
      "  164  11969 572G 572G N   CA   4453 4454  148.47  107.00   41.47   11.92  107.00   41.47   11.92\n",
      "  165  11971 572G 572G CA  C    4454 4455  146.27  116.50   29.77    8.56  116.50   29.77    8.56\n",
      "  166  11972 572G 573L C   N    4455 4457  150.82  120.00   30.82    7.00  120.00   30.82    7.00\n",
      "  167  11975 573L 573L N   CA   4457 4458  147.81  107.00   40.81   11.73  107.00   40.81   11.73\n",
      "  168  11978 573L 573L CA  C    4458 4463  144.80  116.50   28.30    8.14  116.50   28.30    8.14\n",
      "  169  11983 573L 574Q C   N    4463 4465  151.78  120.00   31.78    7.22  120.00   31.78    7.22\n",
      "  170  11986 574Q 574Q N   CA   4465 4466  148.84  107.00   41.84   12.03  107.00   41.84   12.03\n",
      "  171  11989 574Q 574Q CA  C    4466 4472  144.00  116.50   27.50    7.91  116.50   27.50    7.91\n",
      "  172  11995 574Q 575S C   N    4472 4474  149.84  120.00   29.84    6.78  120.00   29.84    6.78\n",
      "  173  11998 575S 575S N   CA   4474 4475  147.35  107.00   40.35   11.60  107.00   40.35   11.60\n",
      "  174  12001 575S 575S CA  C    4475 4478  143.63  116.50   27.13    7.80  116.50   27.13    7.80\n",
      "  175  12003 575S 576D C   N    4478 4480  158.89  120.00   38.89    8.84  120.00   38.89    8.84\n",
      "  176  12006 576D 576D N   CA   4480 4481  154.17  107.00   47.17   13.56  107.00   47.17   13.56\n",
      "  177  12009 576D 576D CA  C    4481 4486  146.70  116.50   30.20    8.68  116.50   30.20    8.68\n",
      "  178  12014 576D 577E C   N    4486 4488  158.59  120.00   38.59    8.77  120.00   38.59    8.77\n",
      "  179  12017 577E 577E N   CA   4488 4489  152.52  107.00   45.52   13.09  107.00   45.52   13.09\n",
      "  180  12020 577E 577E CA  C    4489 4495  151.78  116.50   35.28   10.14  116.50   35.28   10.14\n",
      "  181  12026 577E 578V C   N    4495 4497  159.43  120.00   39.43    8.96  120.00   39.43    8.96\n",
      "  182  12029 578V 578V N   CA   4497 4498  151.97  107.00   44.97   12.93  107.00   44.97   12.93\n",
      "  183  12030 578V 578V CA  CB   4498 4499  130.94  108.50   22.44    5.26  108.50   22.44    5.26\n",
      "  184  12033 578V 578V CA  C    4498 4502  158.44  116.50   41.94   12.06  116.50   41.94   12.06\n",
      "  185  12036 578V 579S C   N    4502 4504  161.87  120.00   41.87    9.52  120.00   41.87    9.52\n",
      "  186  12039 579S 579S N   CA   4504 4505  162.20  107.00   55.20   15.87  107.00   55.20   15.87\n",
      "  187  12042 579S 579S CA  C    4505 4508  154.01  116.50   37.51   10.79  116.50   37.51   10.79\n",
      "  188  12044 579S 580M C   N    4508 4510  157.10  120.00   37.10    8.43  120.00   37.10    8.43\n",
      "  189  12047 580M 580M N   CA   4510 4511  160.76  107.00   53.76   15.46  107.00   53.76   15.46\n",
      "  190  12050 580M 580M CA  C    4511 4516  165.87  112.50   53.37    7.67  112.50   53.37    7.67\n",
      "  191  12054 580M 581P C   N    4516 4518  165.36  117.00   48.36   12.04  117.00   48.36   12.04\n",
      "  192  12057 581P 581P N   CA   4518 4519  175.97  108.20   67.78   15.40  108.20   67.78   15.40\n",
      "  193  12061 581P 581P CA  C    4519 4523  164.61  116.50   48.11   13.83  116.50   48.11   13.83\n",
      "  194  12066 581P 582E C   N    4523 4525  163.31  120.00   43.31    9.84  120.00   43.31    9.84\n",
      "  195  12069 582E 582E N   CA   4525 4526  161.70  107.00   54.70   15.73  107.00   54.70   15.73\n",
      "  196  12070 582E 582E CA  CB   4526 4527  134.25  113.50   20.75    5.09  113.50   20.75    5.09\n",
      "  197  12072 582E 582E CA  C    4526 4532  161.07  116.50   44.57   12.82  116.50   44.57   12.82\n",
      "  198  12078 582E 583A C   N    4532 4534  166.64  120.00   46.64   10.60  120.00   46.64   10.60\n",
      "  199  12081 583A 583A N   CA   4534 4535  169.99  107.00   62.99   18.11  107.00   62.99   18.11\n",
      "  200  12083 583A 583A CA  C    4535 4537  168.13  116.50   51.63   14.85  116.50   51.63   14.85\n",
      "  201  12085 583A 584Q C   N    4537 4539  167.56  120.00   47.56   10.81  120.00   47.56   10.81\n",
      "  202  12088 584Q 584Q N   CA   4539 4540  173.28  107.00   66.28   19.06  107.00   66.28   19.06\n",
      "  203  12091 584Q 584Q CA  C    4540 4546  168.05  116.50   51.55   14.82  116.50   51.55   14.82\n",
      "  204  12097 584Q 585E C   N    4546 4548  174.14  120.00   54.14   12.30  120.00   54.14   12.30\n",
      "  205  12100 585E 585E N   CA   4548 4549  177.27  107.00   70.27   20.20  107.00   70.27   20.20\n",
      "  206  12103 585E 585E CA  C    4549 4555  172.31  116.50   55.81   16.05  116.50   55.81   16.05\n",
      "  207  12109 585E 586D C   N    4555 4557  171.85  120.00   51.85   11.78  120.00   51.85   11.78\n",
      "  208  12112 586D 586D N   CA   4557 4558  171.89  107.00   64.89   18.66  107.00   64.89   18.66\n",
      "  209  12115 586D 586D CA  C    4558 4563  172.24  116.50   55.74   16.03  116.50   55.74   16.03\n",
      "  210  12120 586D 587L C   N    4563 4565  178.59  120.00   58.59   13.31  120.00   58.59   13.31\n",
      "  211  12123 587L 587L N   CA   4565 4566  175.41  107.00   68.41   19.67  107.00   68.41   19.67\n",
      "  212  12126 587L 587L CA  C    4566 4571  169.61  112.50   57.11    8.20  112.50   57.11    8.20\n",
      "  213  12131 587L 588P C   N    4571 4573  174.08  117.00   57.08   14.21  117.00   57.08   14.21\n",
      "  214  12134 588P 588P N   CA   4573 4574  177.87  108.20   69.67   15.83  108.20   69.67   15.83\n",
      "  215  12138 588P 588P CA  C    4574 4578  175.17  116.50   58.67   16.87  116.50   58.67   16.87\n",
      "  216  12141 588P 588P CB  CG   4576 4577  134.10  108.50   25.60    6.88  108.50   25.60    6.88\n",
      "  217  12143 588P 589H C   N    4578 4580  173.96  120.00   53.96   12.26  120.00   53.96   12.26\n",
      "  218  12146 589H 589H N   CA   4580 4581  176.30  107.00   69.30   19.93  107.00   69.30   19.93\n",
      "  219  12148 589H 589H CA  C    4581 4588  175.52  116.50   59.02   16.97  116.50   59.02   16.97\n",
      "  220  12151 589H 589H CG  CB   4583 4584  141.10  113.00   28.10    6.90  113.00   28.10    6.90\n",
      "  221  12158 589H 590K C   N    4588 4590  177.29  120.00   57.29   13.02  120.00   57.29   13.02\n",
      "  222  12161 590K 590K N   CA   4590 4591  175.78  107.00   68.78   19.78  107.00   68.78   19.78\n",
      "  223  12164 590K 590K CA  C    4591 4597  176.99  112.50   64.49    9.26  112.50   64.49    9.26\n",
      "  224  12169 590K 591P C   N    4597 4599  176.37  117.00   59.37   14.78  117.00   59.37   14.78\n",
      "  225  12172 591P 591P N   CA   4599 4600  178.40  108.20   70.20   15.95  108.20   70.20   15.95\n",
      "  226  12176 591P 591P CA  C    4600 4604  176.59  116.50   60.09   17.28  116.50   60.09   17.28\n",
      "  227  12179 591P 591P CB  CG   4602 4603  139.41  108.50   30.91    8.31  108.50   30.91    8.31\n",
      "  228  12181 591P 592T C   N    4604 4606  175.21  120.00   55.21   12.55  120.00   55.21   12.55\n",
      "  229  12184 592T 592T N   CA   4606 4607  176.56  107.00   69.56   20.00  107.00   69.56   20.00\n",
      "  230  12188 592T 592T CA  C    4607 4611  175.27  116.50   58.77   16.90  116.50   58.77   16.90\n",
      "  231  12191 592T 593S C   N    4611 4613  178.86  120.00   58.86   13.38  120.00   58.86   13.38\n",
      "  232  12194 593S 593S N   CA   4613 4614  176.61  107.00   69.61   20.01  107.00   69.61   20.01\n",
      "  233  12197 593S 593S CA  C    4614 4617  176.78  116.50   60.28   17.33  116.50   60.28   17.33\n",
      "  234  12199 593S 594S C   N    4617 4619  176.62  120.00   56.62   12.87  120.00   56.62   12.87\n",
      "  235  12202 594S 594S N   CA   4619 4620  174.64  107.00   67.64   19.45  107.00   67.64   19.45\n",
      "  236  12205 594S 594S CA  C    4620 4623  175.37  116.50   58.87   16.93  116.50   58.87   16.93\n",
      "  237  12207 594S 595S C   N    4623 4625  175.79  120.00   55.79   12.68  120.00   55.79   12.68\n",
      "  238  12210 595S 595S N   CA   4625 4626  175.14  107.00   68.14   19.59  107.00   68.14   19.59\n",
      "  239  12213 595S 595S CA  C    4626 4629  174.79  116.50   58.29   16.76  116.50   58.29   16.76\n",
      "  240  12215 595S 596M C   N    4629 4631  176.93  120.00   56.92   12.94  120.00   56.92   12.94\n",
      "  241  12218 596M 596M N   CA   4631 4632  179.97  107.00   72.97   20.98  107.00   72.97   20.98\n",
      "  242  12221 596M 596M CA  C    4632 4637  178.54  116.50   62.04   17.84  116.50   62.04   17.84\n",
      "  243  12225 596M 597K C   N    4637 4639  178.65  120.00   58.65   13.33  120.00   58.65   13.33\n",
      "  244  12228 597K 597K N   CA   4639 4640  178.11  107.00   71.11   20.45  107.00   71.11   20.45\n",
      "  245  12231 597K 597K CA  C    4640 4646  176.53  116.50   60.03   17.26  116.50   60.03   17.26\n",
      "  246  12236 597K 598T C   N    4646 4648  178.26  120.00   58.26   13.24  120.00   58.26   13.24\n",
      "  247  12239 598T 598T N   CA   4648 4649  179.11  107.00   72.11   20.73  107.00   72.11   20.73\n",
      "  248  12243 598T 598T CA  C    4649 4653  176.79  116.50   60.29   17.33  116.50   60.29   17.33\n",
      "  249  12246 598T 599V C   N    4653 4655  178.87  120.00   58.87   13.38  120.00   58.87   13.38\n",
      "  250  12249 599V 599V N   CA   4655 4656  176.91  107.00   69.91   20.10  107.00   69.91   20.10\n",
      "  251  12253 599V 599V CA  C    4656 4660  177.63  116.50   61.13   17.58  116.50   61.13   17.58\n",
      "  252  12256 599V 600D C   N    4660 4662  176.47  120.00   56.47   12.83  120.00   56.47   12.83\n",
      "  253  12259 600D 600D N   CA   4662 4663  177.17  107.00   70.17   20.18  107.00   70.17   20.18\n",
      "  254  12262 600D 600D CA  C    4663 4668  176.59  116.50   60.09   17.28  116.50   60.09   17.28\n",
      "  255  12267 600D 601E C   N    4668 4670  177.23  120.00   57.22   13.00  120.00   57.22   13.00\n",
      "  256  12270 601E 601E N   CA   4670 4671  176.95  107.00   69.95   20.11  107.00   69.95   20.11\n",
      "  257  12273 601E 601E CA  C    4671 4677  175.88  116.50   59.38   17.07  116.50   59.38   17.07\n",
      "  258  12279 601E 602E C   N    4677 4679  178.13  120.00   58.13   13.21  120.00   58.13   13.21\n",
      "  259  12282 602E 602E N   CA   4679 4680  179.00  107.00   72.00   20.70  107.00   72.00   20.70\n",
      "  260  12285 602E 602E CA  C    4680 4686  177.47  116.50   60.97   17.53  116.50   60.97   17.53\n",
      "  261  12289 602E 602E OE2 CD   4685 4683  144.92  124.00   20.92    6.72  124.00   20.92    6.72\n",
      "  262  12291 602E 603A C   N    4686 4688  175.97  120.00   55.97   12.72  120.00   55.97   12.72\n",
      "  263  12294 603A 603A N   CA   4688 4689  177.08  107.00   70.08   20.15  107.00   70.08   20.15\n",
      "  264  12296 603A 603A CA  C    4689 4691  177.88  116.50   61.39   17.65  116.50   61.39   17.65\n",
      "  265  12298 603A 604L C   N    4691 4693  177.66  120.00   57.66   13.10  120.00   57.66   13.10\n",
      "  266  12301 604L 604L N   CA   4693 4694  177.88  107.00   70.88   20.38  107.00   70.88   20.38\n",
      "  267  12304 604L 604L CA  C    4694 4699  177.80  116.50   61.30   17.63  116.50   61.30   17.63\n",
      "  268  12309 604L 605A C   N    4699 4701  179.34  120.00   59.34   13.48  120.00   59.34   13.48\n",
      "  269  12312 605A 605A N   CA   4701 4702  179.37  107.00   72.37   20.81  107.00   72.37   20.81\n",
      "  270  12314 605A 605A CA  C    4702 4704  174.04  116.50   57.54   16.55  116.50   57.54   16.55\n",
      "  271  12316 605A 606M C   N    4704 4706  174.70  120.00   54.70   12.43  120.00   54.70   12.43\n",
      "  272  12319 606M 606M N   CA   4706 4707  174.29  107.00   67.29   19.35  107.00   67.29   19.35\n",
      "  273  12321 606M 606M CA  C    4707 4712  149.98  121.00   28.98    8.33  121.00   28.98    8.33\n",
      "  274  12322 606M 606M CA  C    4707 4712  161.04  116.50   44.54   12.81  116.50   44.54   12.81\n",
      "  275  12323 606M 606M CB  CG   4708 4709  144.74  114.50   30.24    7.40  114.50   30.24    7.40\n",
      "  276  12324 606M 606M CG  SD   4709 4710  121.67   95.00   26.67    4.99   95.00   26.67    4.99\n",
      "  277  12326 606M 607L C   N    4712 4714  167.19  120.00   47.19   10.72  120.00   47.19   10.72\n",
      "  278  12329 607L 607L N   CA   4714 4715  145.60  107.00   38.60   11.10  107.00   38.60   11.10\n",
      "  279  12356 609S 610M C   N    4734 4736  144.15  120.00   24.15    5.49  120.00   24.15    5.49\n",
      "  280  12359 610M 610M N   CA   4736 4737  122.95  107.00   15.95    4.59  107.00   15.95    4.59\n",
      "  281  12374 612I 612I N   CA   4748 4749  122.90  107.00   15.90    4.57  107.00   15.90    4.57\n",
      "  282  12385 613E 613E N   CA   4756 4757  124.22  107.00   17.22    4.95  107.00   17.22    4.95\n",
      "  283  12397 614E 614E N   CA   4765 4766  124.31  107.00   17.31    4.98  107.00   17.31    4.98\n",
      "  284  12429 617A 617A N   CA   4788 4789  124.16  107.00   17.16    4.93  107.00   17.16    4.93\n",
      "  285  12436 618R 618R N   CA   4793 4794  124.56  107.00   17.56    5.05  107.00   17.56    5.05\n",
      "  286  12450 619Y 619Y N   CA   4804 4805  124.20  107.00   17.20    4.95  107.00   17.20    4.95\n",
      "  287  12468 620A 620A N   CA   4816 4817  123.35  107.00   16.35    4.70  107.00   16.35    4.70\n",
      "  288  13338 703N 703N N   CA   5441 5442  129.71  107.00   22.71    6.53  107.00   22.71    6.53\n",
      "  289  13349 704D 704D N   CA   5449 5450  130.48  107.00   23.48    6.75  107.00   23.48    6.75\n",
      "  290  13352 704D 704D CA  C    5450 5455  132.36  116.50   15.86    4.56  116.50   15.86    4.56\n",
      "  291  13360 705E 705E N   CA   5457 5458  127.63  107.00   20.63    5.93  107.00   20.63    5.93\n",
      "  292  13372 706K 706K N   CA   5466 5467  132.26  107.00   25.26    7.26  107.00   25.26    7.26\n",
      "  293  13375 706K 706K CA  C    5467 5473  132.41  116.50   15.91    4.58  116.50   15.91    4.58\n",
      "  294  13383 707V 707V N   CA   5475 5476  132.85  107.00   25.85    7.43  107.00   25.85    7.43\n",
      "  295  13390 707V 708A C   N    5480 5482  145.46  120.00   25.46    5.79  120.00   25.46    5.79\n",
      "  296  13393 708A 708A N   CA   5482 5483  133.35  107.00   26.35    7.58  107.00   26.35    7.58\n",
      "  297  13395 708A 708A CA  C    5483 5485  133.65  116.50   17.15    4.93  116.50   17.15    4.93\n",
      "  298  13397 708A 709E C   N    5485 5487  141.52  120.00   21.52    4.89  120.00   21.52    4.89\n",
      "  299  13400 709E 709E N   CA   5487 5488  134.16  107.00   27.16    7.81  107.00   27.16    7.81\n",
      "  300  13412 710S 710S N   CA   5496 5497  128.69  107.00   21.69    6.24  107.00   21.69    6.24\n",
      "  301  13415 710S 710S CA  C    5497 5500  134.16  116.50   17.66    5.08  116.50   17.66    5.08\n",
      "  302  13417 710S 711L C   N    5500 5502  141.50  120.00   21.50    4.89  120.00   21.50    4.89\n",
      "  303  13420 711L 711L N   CA   5502 5503  129.22  107.00   22.22    6.39  107.00   22.22    6.39\n",
      "  304  13428 711L 712N C   N    5508 5510  146.27  120.00   26.27    5.97  120.00   26.27    5.97\n",
      "  305  13431 712N 712N N   CA   5510 5511  132.63  107.00   25.63    7.37  107.00   25.63    7.37\n",
      "  306  13439 712N 713P C   N    5516 5518  149.44  117.00   32.44    8.08  117.00   32.44    8.08\n",
      "  307  13442 713P 713P N   CA   5518 5519  147.67  108.20   39.47    8.97  108.20   39.47    8.97\n",
      "  308  13446 713P 713P CA  C    5519 5523  144.52  112.50   32.02    4.60  112.50   32.02    4.60\n",
      "  309  13451 713P 714P C   N    5523 5525  164.03  117.00   47.03   11.71  117.00   47.03   11.71\n",
      "  310  13454 714P 714P N   CA   5525 5526  146.62  108.20   38.42    8.73  108.20   38.42    8.73\n",
      "  311  13458 714P 714P CA  C    5526 5530  143.29  116.50   26.79    7.70  116.50   26.79    7.70\n",
      "  312  13463 714P 715F C   N    5530 5532  154.33  120.00   34.33    7.80  120.00   34.33    7.80\n",
      "  313  13466 715F 715F N   CA   5532 5533  141.81  107.00   34.81   10.01  107.00   34.81   10.01\n",
      "  314  13469 715F 715F CA  C    5533 5541  137.42  116.50   20.92    6.01  116.50   20.92    6.01\n",
      "  315  13479 715F 716S C   N    5541 5543  143.09  120.00   23.09    5.25  120.00   23.09    5.25\n",
      "  316  13482 716S 716S N   CA   5543 5544  138.50  107.00   31.50    9.06  107.00   31.50    9.06\n",
      "  317  13485 716S 716S CA  C    5544 5547  145.69  116.50   29.19    8.39  116.50   29.19    8.39\n",
      "  318  13487 716S 717M C   N    5547 5549  151.05  120.00   31.05    7.06  120.00   31.05    7.06\n",
      "  319  13490 717M 717M N   CA   5549 5550  148.33  107.00   41.33   11.88  107.00   41.33   11.88\n",
      "  320  13493 717M 717M CA  C    5550 5555  147.64  116.50   31.14    8.95  116.50   31.14    8.95\n",
      "  321  13497 717M 718A C   N    5555 5557  154.19  120.00   34.19    7.77  120.00   34.19    7.77\n",
      "  322  13500 718A 718A N   CA   5557 5558  145.75  107.00   38.75   11.14  107.00   38.75   11.14\n",
      "  323  13502 718A 718A CA  C    5558 5560  152.52  116.50   36.02   10.36  116.50   36.02   10.36\n",
      "  324  13504 718A 719F C   N    5560 5562  149.72  120.00   29.72    6.75  120.00   29.72    6.75\n",
      "  325  13507 719F 719F N   CA   5562 5563  146.18  107.00   39.18   11.27  107.00   39.18   11.27\n",
      "  326  13508 719F 719F CA  CB   5563 5564  136.43  107.50   28.93    6.69  107.50   28.93    6.69\n",
      "  327  13510 719F 719F CA  C    5563 5571  155.14  116.50   38.64   11.11  116.50   38.64   11.11\n",
      "  328  13520 719F 720L C   N    5571 5573  152.10  120.00   32.10    7.30  120.00   32.10    7.30\n",
      "  329  13523 720L 720L N   CA   5573 5574  155.45  107.00   48.45   13.93  107.00   48.45   13.93\n",
      "  330  13526 720L 720L CA  C    5574 5579  153.11  116.50   36.61   10.53  116.50   36.61   10.53\n",
      "  331  13527 720L 720L CB  CG   5575 5576  134.58  114.00   20.58    4.83  114.00   20.58    4.83\n",
      "  332  13531 720L 721Y C   N    5579 5581  155.15  120.00   35.15    7.99  120.00   35.15    7.99\n",
      "  333  13534 721Y 721Y N   CA   5581 5582  166.31  107.00   59.31   17.05  107.00   59.31   17.05\n",
      "  334  13535 721Y 721Y CA  CB   5582 5583  132.19  107.50   24.69    5.71  107.50   24.69    5.71\n",
      "  335  13537 721Y 721Y CA  C    5582 5591  164.76  116.50   48.26   13.88  116.50   48.26   13.88\n",
      "  336  13549 721Y 722F C   N    5591 5593  157.10  120.00   37.10    8.43  120.00   37.10    8.43\n",
      "  337  13552 722F 722F N   CA   5593 5594  161.24  107.00   54.24   15.60  107.00   54.24   15.60\n",
      "  338  13555 722F 722F CA  C    5594 5602  159.40  116.50   42.90   12.34  116.50   42.90   12.34\n",
      "  339  13565 722F 723Y C   N    5602 5604  171.25  120.00   51.25   11.65  120.00   51.25   11.65\n",
      "  340  13568 723Y 723Y N   CA   5604 5605  165.28  107.00   58.28   16.76  107.00   58.28   16.76\n",
      "  341  13569 723Y 723Y CA  CB   5605 5606  128.16  107.50   20.66    4.78  107.50   20.66    4.78\n",
      "  342  13571 723Y 723Y CA  C    5605 5614  161.55  116.50   45.05   12.95  116.50   45.05   12.95\n",
      "  343  13583 723Y 724K C   N    5614 5616  162.39  120.00   42.39    9.63  120.00   42.39    9.63\n",
      "  344  13586 724K 724K N   CA   5616 5617  159.11  107.00   52.11   14.98  107.00   52.11   14.98\n",
      "  345  13589 724K 724K CA  C    5617 5623  158.05  116.50   41.55   11.95  116.50   41.55   11.95\n",
      "  346  13594 724K 725R C   N    5623 5625  166.93  120.00   46.93   10.67  120.00   46.93   10.67\n",
      "  347  13597 725R 725R N   CA   5625 5626  171.05  107.00   64.05   18.42  107.00   64.05   18.42\n",
      "  348  13600 725R 725R CA  C    5626 5634  161.13  116.50   44.63   12.83  116.50   44.63   12.83\n",
      "  349  13608 725R 726V C   N    5634 5636  163.36  120.00   43.36    9.85  120.00   43.36    9.85\n",
      "  350  13611 726V 726V N   CA   5636 5637  166.40  107.00   59.40   17.08  107.00   59.40   17.08\n",
      "  351  13615 726V 726V CA  C    5637 5641  165.99  116.50   49.49   14.23  116.50   49.49   14.23\n",
      "  352  13618 726V 727G C   N    5641 5643  165.15  120.00   45.15   10.26  120.00   45.15   10.26\n",
      "  353  13620 727G 727G N   CA   5643 5644  168.11  107.00   61.11   17.57  107.00   61.11   17.57\n",
      "  354  13622 727G 727G CA  C    5644 5645  164.73  116.50   48.23   13.87  116.50   48.23   13.87\n",
      "  355  13623 727G 728K C   N    5645 5647  166.52  120.00   46.52   10.57  120.00   46.52   10.57\n",
      "  356  13626 728K 728K N   CA   5647 5648  156.44  107.00   49.44   11.23  107.00   49.44   11.23\n",
      "  357  13628 728K 728K CA  C    5648 5654  157.05  118.00   39.05    7.93  118.00   39.05    7.93\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17268 722F 722F CE1 CD1  5599 5597  -76.33    0.00  -76.33    5.85    0.00  -76.33    5.85\n",
      "    2  17270 722F 722F CE2 CD2  5600 5598   80.98    0.00   80.98    6.05    0.00   80.98    6.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19222 577E 577E OE1 CG   4493 4491   24.20    0.00   24.20    7.63    0.00   24.20    7.63\n",
      "    2  19252 588P 588P CA  N    4574 4573   59.98   35.27   24.71    5.89   35.27   24.71    5.89\n",
      "    3  19303 607L 607L CA  N    4715 4714   70.60   35.27   35.33    8.42   35.27   35.33    8.42\n",
      "    4  19694 722F 722F CG  CD1  5596 5597   24.70    0.00   24.70    7.52    0.00   24.70    7.52\n",
      "    5  19695 722F 722F CD1 CE1  5597 5599   19.07    0.00   19.07    5.81    0.00   19.07    5.81\n",
      "    6  19715 726V 726V CA  N    5637 5636   54.36   35.27   19.10    4.55   35.27   19.10    4.55\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22527  47G  60V CA  CA    371  459   21.59    5.82   15.77    5.31    5.82   15.77    5.31\n",
      "    2  22528  47G  61E CA  CA    371  466   25.38    7.97   17.40    6.61    7.97   17.40    6.61\n",
      "    3  22529  47G  62H CA  CA    371  475   28.10    9.00   19.10    7.79    9.00   19.10    7.79\n",
      "    4  22530  47G  63V CA  CA    371  485   29.59   12.77   16.82    7.88   12.77   16.82    7.88\n",
      "    5  22531  47G 243C CA  CA    371 1840   19.60   13.26    6.34    9.20   13.26    6.34    9.20\n",
      "    6  22532  47G 244D CA  CA    371 1846   18.03   12.85    5.18    6.28   12.85    5.18    6.28\n",
      "    7  22533  47G 245D CA  CA    371 1854   20.75   10.17   10.58   14.28   10.17   10.58   14.28\n",
      "    8  22534  47G 246E CA  CA    371 1862   24.40   10.81   13.58   21.00   10.81   13.58   21.00\n",
      "    9  22535  47G 247V CA  CA    371 1871   26.68   11.44   15.24   24.81   11.44   15.24   24.81\n",
      "   10  22536  47G 248S CA  CA    371 1878   30.38   13.40   16.99   20.22   13.40   16.99   20.22\n",
      "   11  22537  48T  60V CA  CA    375  459   16.77    3.89   12.88    4.72    3.89   12.88    4.72\n",
      "   12  22538  48T  61E CA  CA    375  466   20.47    6.49   13.98    5.77    6.49   13.98    5.77\n",
      "   13  22539  48T  62H CA  CA    375  475   23.21    9.04   14.18    6.22    9.04   14.18    6.22\n",
      "   14  22540  48T  63V CA  CA    375  485   24.68   12.76   11.91    6.15   12.76   11.91    6.15\n",
      "   15  22541  48T 237D CA  CA    375 1783   28.87   13.73   15.14   24.98   13.73   15.14   24.98\n",
      "   16  22542  48T 239Y CA  CA    375 1802   21.66   13.74    7.92   14.61   13.74    7.92   14.61\n",
      "   17  22544  48T 245D CA  CA    375 1854   15.76   10.05    5.71    9.36   10.05    5.71    9.36\n",
      "   18  22545  48T 246E CA  CA    375 1862   19.43    9.43   10.01   20.25    9.43   10.01   20.25\n",
      "   19  22546  48T 247V CA  CA    375 1871   21.67   10.45   11.22   23.72   10.45   11.22   23.72\n",
      "   20  22547  48T 248S CA  CA    375 1878   25.37   12.09   13.28   20.57   12.09   13.28   20.57\n",
      "   21  22551  60V 237D CA  CA    459 1783   15.49   12.66    2.83    4.52   12.66    2.83    4.52\n",
      "   22  22566  61E 235G CA  CA    466 1774   17.85   12.03    5.82   10.66   12.03    5.82   10.66\n",
      "   23  22567  61E 236A CA  CA    466 1778   15.55   11.52    4.03    8.43   11.52    4.03    8.43\n",
      "   24  22580  61E 252F CA  CA    466 1909   16.37   13.72    2.65    4.74   13.72    2.65    4.74\n",
      "   25  22912  86D 254N CA  CA    676 1925   21.30   13.93    7.38    6.66   13.93    7.38    6.66\n",
      "   26  22913  86D 255H CA  CA    676 1933   17.93   12.25    5.68    7.05   12.25    5.68    7.05\n",
      "   27  22914  86D 256M CA  CA    676 1943   19.55   13.51    6.04    6.93   13.51    6.04    6.93\n",
      "   28  22915  86D 257Q CA  CA    676 1951   20.62   12.79    7.83    5.42   12.79    7.83    5.42\n",
      "   29  22916  86D 258H CA  CA    676 1960   17.13    8.96    8.18    5.43    8.96    8.18    5.43\n",
      "   30  22928  96N 255H CA  CA    744 1933   22.27   13.60    8.67    9.94   13.60    8.67    9.94\n",
      "   31  22929  96N 256M CA  CA    744 1943   23.21   13.80    9.41   10.31   13.80    9.41   10.31\n",
      "   32  22930  96N 257Q CA  CA    744 1951   24.60   12.80   11.80    7.94   12.80   11.80    7.94\n",
      "   33  22931  96N 258H CA  CA    744 1960   21.26    9.26   12.00    7.75    9.26   12.00    7.75\n",
      "   34  22932  96N 259F CA  CA    744 1970   19.72    9.20   10.52    5.75    9.20   10.52    5.75\n",
      "   35  22946 103P 254N CA  CA    793 1925   20.81   13.24    7.56    9.47   13.24    7.56    9.47\n",
      "   36  22947 103P 255H CA  CA    793 1933   17.33   11.11    6.22   11.14   11.11    6.22   11.14\n",
      "   37  22948 103P 256M CA  CA    793 1943   17.49   10.74    6.75   11.61   10.74    6.75   11.61\n",
      "   38  22949 103P 257Q CA  CA    793 1951   18.36   10.47    7.89    7.18   10.47    7.89    7.18\n",
      "   39  22950 103P 258H CA  CA    793 1960   15.04    7.11    7.93    6.33    7.11    7.93    6.33\n",
      "   40  22951 103P 259F CA  CA    793 1970   13.16    5.98    7.17    4.72    5.98    7.17    4.72\n",
      "   41  22990 117K 256M CA  CA    893 1943   12.83   10.10    2.73    4.77   10.10    2.73    4.77\n",
      "   42  22997 117K 606M CA  CA    893 4707   38.64   13.64   24.99   10.08   13.64   24.99   10.08\n",
      "   43  23034 118S 606M CA  CA    902 4707   35.10   11.87   23.23    8.58   11.87   23.23    8.58\n",
      "   44  23061 122P 606M CA  CA    932 4707   36.36   11.70   24.66    8.69   11.70   24.66    8.69\n",
      "   45  23080 123P 232T CA  CA    939 1754   14.00   10.94    3.07    4.91   10.94    3.07    4.91\n",
      "   46  23081 123P 233P CA  CA    939 1761   16.52   13.32    3.20    4.68   13.32    3.20    4.68\n",
      "   47  23090 123P 606M CA  CA    939 4707   36.01    9.60   26.41    9.11    9.60   26.41    9.11\n",
      "   48  23091 123P 607L CA  CA    939 4715   29.61   12.23   17.38    5.37   12.23   17.38    5.37\n",
      "   49  23108 129C 229G CA  CA    986 1736   12.13    7.57    4.56    8.69    7.57    4.56    8.69\n",
      "   50  23109 129C 230T CA  CA    986 1740   15.59   11.11    4.48    7.88   11.11    4.48    7.88\n",
      "   51  23110 129C 231V CA  CA    986 1747   17.70   10.93    6.77   12.49   10.93    6.77   12.49\n",
      "   52  23111 129C 232T CA  CA    986 1754   21.06   12.90    8.16   12.24   12.90    8.16   12.24\n",
      "   53  23116 129C 606M CA  CA    986 4707   44.79   12.81   31.99   10.68   12.81   31.99   10.68\n",
      "   54  23133 140G 227K CA  CA   1076 1719   16.99   13.32    3.66    5.87   13.32    3.66    5.87\n",
      "   55  23134 140G 228L CA  CA   1076 1728   15.34    9.80    5.54   11.45    9.80    5.54   11.45\n",
      "   56  23135 140G 229G CA  CA   1076 1736   18.29    9.76    8.53   14.37    9.76    8.53   14.37\n",
      "   57  23136 140G 230T CA  CA   1076 1740   21.45   13.58    7.87   12.97   13.58    7.87   12.97\n",
      "   58  23160 155L 228L CA  CA   1178 1728   11.97    8.19    3.77   10.36    8.19    3.77   10.36\n",
      "   59  23161 155L 229G CA  CA   1178 1736   15.53    9.59    5.94   12.23    9.59    5.94   12.23\n",
      "   60  23162 155L 230T CA  CA   1178 1740   17.94   13.09    4.85   10.28   13.09    4.85   10.28\n",
      "   61  23163 155L 231V CA  CA   1178 1747   19.88   13.69    6.19   13.66   13.69    6.19   13.66\n",
      "   62  23191 156Q 229G CA  CA   1186 1736   14.72   12.03    2.69    4.70   12.03    2.69    4.70\n",
      "   63  23222 157C 226V CA  CA   1195 1712   15.43   13.60    1.84    5.00   13.60    1.84    5.00\n",
      "   64  23223 157C 227K CA  CA   1195 1719   14.77   11.98    2.79    6.48   11.98    2.79    6.48\n",
      "   65  23224 157C 228L CA  CA   1195 1728   13.47   10.94    2.53    7.07   10.94    2.53    7.07\n",
      "   66  23225 157C 229G CA  CA   1195 1736   16.45   13.47    2.98    5.86   13.47    2.98    5.86\n",
      "   67  23233 168S 191N CA  CA   1280 1447   17.87    9.81    8.06    4.66    9.81    8.06    4.66\n",
      "   68  23234 168S 192W CA  CA   1280 1455   17.77   10.81    6.96    4.85   10.81    6.96    4.85\n",
      "   69  23235 168S 193M CA  CA   1280 1469   18.47   10.88    7.59    7.24   10.88    7.59    7.24\n",
      "   70  23236 168S 194C CA  CA   1280 1477   17.86   12.24    5.62    7.48   12.24    5.62    7.48\n",
      "   71  23237 168S 197C CA  CA   1280 1498   17.96   13.00    4.96    6.82   13.00    4.96    6.82\n",
      "   72  23238 168S 198G CA  CA   1280 1504   18.28   11.08    7.21   10.74   11.08    7.21   10.74\n",
      "   73  23239 168S 199T CA  CA   1280 1508   14.69    7.73    6.97   10.17    7.73    6.97   10.17\n",
      "   74  23240 168S 200I CA  CA   1280 1515   15.12    6.23    8.89    9.37    6.23    8.89    9.37\n",
      "   75  23241 168S 201G CA  CA   1280 1523   12.90    6.06    6.84    5.38    6.06    6.84    5.38\n",
      "   76  23252 168S 228L CA  CA   1280 1728   22.65   13.67    8.98   17.28   13.67    8.98   17.28\n",
      "   77  23261 179K 198G CA  CA   1359 1504   18.85   13.88    4.96    7.53   13.88    4.96    7.53\n",
      "   78  23262 179K 199T CA  CA   1359 1508   15.06   10.15    4.91    7.00   10.15    4.91    7.00\n",
      "   79  23263 179K 200I CA  CA   1359 1515   14.12    8.82    5.29    5.42    8.82    5.29    5.42\n",
      "   80  23284 180C 200I CA  CA   1368 1515   13.35   10.55    2.80    4.51   10.55    2.80    4.51\n",
      "   81  23335 182L 228L CA  CA   1379 1728   14.74   12.33    2.41    5.86   12.33    2.41    5.86\n",
      "   82  23336 182L 229G CA  CA   1379 1736   17.22   13.56    3.66    6.57   13.56    3.66    6.57\n",
      "   83  23526 193M 606M CA  CA   1469 4707   38.01   13.92   24.10   25.55   13.92   24.10   25.55\n",
      "   84  23685 198G 252F CA  CA   1504 1909   15.53   13.63    1.90    4.91   13.63    1.90    4.91\n",
      "   85  24135 226V 606M CA  CA   1712 4707   34.62   11.91   22.71   60.70   11.91   22.71   60.70\n",
      "   86  24158 227K 606M CA  CA   1719 4707   35.18   12.70   22.48   48.85   12.70   22.48   48.85\n",
      "   87  24175 228L 606M CA  CA   1728 4707   35.38   11.25   24.13   63.62   11.25   24.13   63.62\n",
      "   88  24187 229G 606M CA  CA   1736 4707   32.97   10.88   22.09   44.78   10.88   22.09   44.78\n",
      "   89  24188 229G 607L CA  CA   1736 4715   26.08   13.87   12.22   17.68   13.87   12.22   17.68\n",
      "   90  24189 229G 728K CA  CA   1736 5648   19.93   12.91    7.01   12.27   12.91    7.01   12.27\n",
      "   91  24198 230T 246E CA  CA   1740 1862   15.82   12.94    2.88    5.48   12.94    2.88    5.48\n",
      "   92  24206 230T 606M CA  CA   1740 4707   30.09    9.84   20.25   48.02    9.84   20.25   48.02\n",
      "   93  24207 230T 607L CA  CA   1740 4715   23.10   12.62   10.48   17.97   12.62   10.48   17.97\n",
      "   94  24208 230T 728K CA  CA   1740 5648   23.38   12.71   10.67   21.51   12.71   10.67   21.51\n",
      "   95  24214 231V 238F CA  CA   1747 1791   11.86    9.55    2.31    5.35    9.55    2.31    5.35\n",
      "   96  24215 231V 239Y CA  CA   1747 1802   15.40   13.01    2.39    5.04   13.01    2.39    5.04\n",
      "   97  24216 231V 248S CA  CA   1747 1878   15.49   12.98    2.50    4.56   12.98    2.50    4.56\n",
      "   98  24229 231V 606M CA  CA   1747 4707   27.22    6.06   21.16   57.46    6.06   21.16   57.46\n",
      "   99  24230 231V 607L CA  CA   1747 4715   20.34    9.09   11.26   22.71    9.09   11.26   22.71\n",
      "  100  24231 231V 728K CA  CA   1747 5648   25.60    9.49   16.11   36.85    9.49   16.11   36.85\n",
      "  101  24244 232T 262D CA  CA   1754 1993   13.01    9.01    4.00    7.57    9.01    4.00    7.57\n",
      "  102  24245 232T 606M CA  CA   1754 4707   24.21    5.37   18.84   46.58    5.37   18.84   46.58\n",
      "  103  24246 232T 607L CA  CA   1754 4715   17.25    6.72   10.53   21.42    6.72   10.53   21.42\n",
      "  104  24247 232T 728K CA  CA   1754 5648   28.11    7.50   20.62   44.12    7.50   20.62   44.12\n",
      "  105  24260 233P 261I CA  CA   1761 1985   13.62   11.18    2.44    4.72   11.18    2.44    4.72\n",
      "  106  24261 233P 262D CA  CA   1761 1993   15.65    8.58    7.06   13.43    8.58    7.06   13.43\n",
      "  107  24262 233P 606M CA  CA   1761 4707   20.76    5.20   15.56   37.85    5.20   15.56   37.85\n",
      "  108  24263 233P 607L CA  CA   1761 4715   13.85    4.36    9.49   22.69    4.36    9.49   22.69\n",
      "  109  24264 233P 728K CA  CA   1761 5648   31.34    6.73   24.61   53.53    6.73   24.61   53.53\n",
      "  110  24268 234S 248S CA  CA   1768 1878   15.39   11.90    3.49    4.94   11.90    3.49    4.94\n",
      "  111  24277 234S 606M CA  CA   1768 4707   21.11    8.18   12.93   24.24    8.18   12.93   24.24\n",
      "  112  24278 234S 607L CA  CA   1768 4715   14.05    8.02    6.04    9.87    8.02    6.04    9.87\n",
      "  113  24279 234S 728K CA  CA   1768 5648   32.32   10.59   21.73   34.54   10.59   21.73   34.54\n",
      "  114  24283 235G 246E CA  CA   1774 1862   16.83   12.57    4.26    7.94   12.57    4.26    7.94\n",
      "  115  24284 235G 247V CA  CA   1774 1871   14.81   11.30    3.51    7.26   11.30    3.51    7.26\n",
      "  116  24285 235G 248S CA  CA   1774 1878   11.81    8.95    2.85    5.31    8.95    2.85    5.31\n",
      "  117  24297 235G 606M CA  CA   1774 4707   23.95    7.52   16.43   38.24    7.52   16.43   38.24\n",
      "  118  24298 235G 607L CA  CA   1774 4715   16.99    9.05    7.94   14.66    9.05    7.94   14.66\n",
      "  119  24299 235G 728K CA  CA   1774 5648   30.96   11.48   19.48   37.85   11.48   19.48   37.85\n",
      "  120  24303 236A 245D CA  CA   1778 1854   17.18   13.89    3.29    5.62   13.89    3.29    5.62\n",
      "  121  24304 236A 246E CA  CA   1778 1862   13.59   10.23    3.36    7.19   10.23    3.36    7.19\n",
      "  122  24305 236A 247V CA  CA   1778 1871   12.18    9.89    2.28    5.37    9.89    2.28    5.37\n",
      "  123  24318 236A 606M CA  CA   1778 4707   27.91    8.77   19.14   48.95    8.77   19.14   48.95\n",
      "  124  24319 236A 607L CA  CA   1778 4715   20.94   11.42    9.52   18.00   11.42    9.52   18.00\n",
      "  125  24320 236A 728K CA  CA   1778 5648   27.88   12.98   14.90   31.98   12.98   14.90   31.98\n",
      "  126  24326 237D 246E CA  CA   1783 1862    9.67    7.05    2.62    5.36    7.05    2.62    5.36\n",
      "  127  24336 237D 606M CA  CA   1783 4707   31.44   12.37   19.07   39.03   12.37   19.07   39.03\n",
      "  128  24354 238F 606M CA  CA   1791 4707   33.53   13.44   20.09   48.89   13.44   20.09   48.89\n",
      "  129  24424 249D 606M CA  CA   1884 4707   29.99   13.37   16.61   39.97   13.37   16.61   39.97\n",
      "  130  24431 250V 606M CA  CA   1892 4707   26.96   13.43   13.53   24.85   13.43   13.53   24.85\n",
      "  131  24441 251H 606M CA  CA   1899 4707   26.30   12.08   14.22   28.61   12.08   14.22   28.61\n",
      "  132  24442 251H 607L CA  CA   1899 4715   20.60   13.68    6.93    9.85   13.68    6.93    9.85\n",
      "  133  24452 252F 606M CA  CA   1909 4707   25.91    8.96   16.94   39.68    8.96   16.94   39.68\n",
      "  134  24453 252F 607L CA  CA   1909 4715   19.63   11.27    8.36   13.40   11.27    8.36   13.40\n",
      "  135  24454 252F 728K CA  CA   1909 5648   33.36   13.91   19.46   34.79   13.91   19.46   34.79\n",
      "  136  24462 253A 262D CA  CA   1920 1993   12.92    7.50    5.43    8.90    7.50    5.43    8.90\n",
      "  137  24463 253A 606M CA  CA   1920 4707   22.81    6.73   16.08   31.96    6.73   16.08   31.96\n",
      "  138  24464 253A 607L CA  CA   1920 4715   16.63    8.40    8.23   11.89    8.40    8.23   11.89\n",
      "  139  24465 253A 728K CA  CA   1920 5648   34.40   11.13   23.27   34.42   11.13   23.27   34.42\n",
      "  140  24472 254N 262D CA  CA   1925 1993   13.57    9.29    4.28    5.58    9.29    4.28    5.58\n",
      "  141  24473 254N 606M CA  CA   1925 4707   25.10    9.84   15.26   22.67    9.84   15.26   22.67\n",
      "  142  24474 254N 607L CA  CA   1925 4715   19.42   11.52    7.90    8.59   11.52    7.90    8.59\n",
      "  143  24475 254N 728K CA  CA   1925 5648   34.87   13.83   21.03   24.76   13.83   21.03   24.76\n",
      "  144  24482 255H 606M CA  CA   1933 4707   28.74   10.39   18.35   35.41   10.39   18.35   35.41\n",
      "  145  24483 255H 607L CA  CA   1933 4715   22.83   13.12    9.71   12.99   13.12    9.71   12.99\n",
      "  146  24488 256M 262D CA  CA   1943 1993    9.80    7.00    2.80    4.86    7.00    2.80    4.86\n",
      "  147  24489 256M 606M CA  CA   1943 4707   28.22    7.34   20.88   42.20    7.34   20.88   42.20\n",
      "  148  24490 256M 607L CA  CA   1943 4715   21.93   10.55   11.37   15.80   10.55   11.37   15.80\n",
      "  149  24491 256M 728K CA  CA   1943 5648   29.55   11.69   17.86   28.43   11.69   17.86   28.43\n",
      "  150  24494 257Q 261I CA  CA   1951 1985    8.84    5.03    3.81    6.79    5.03    3.81    6.79\n",
      "  151  24495 257Q 262D CA  CA   1951 1993   13.34    6.08    7.26    7.14    6.08    7.26    7.14\n",
      "  152  24496 257Q 606M CA  CA   1951 4707   26.88    8.14   18.74   18.72    8.14   18.74   18.72\n",
      "  153  24497 257Q 607L CA  CA   1951 4715   20.97   10.33   10.65    8.48   10.33   10.65    8.48\n",
      "  154  24498 257Q 728K CA  CA   1951 5648   30.88   11.32   19.56   16.91   11.32   19.56   16.91\n",
      "  155  24502 258H 606M CA  CA   1960 4707   30.23   11.57   18.67   14.08   11.57   18.67   14.08\n",
      "  156  24505 259F 606M CA  CA   1970 4707   32.31   10.95   21.36   13.41   10.95   21.36   13.41\n",
      "  157  24506 259F 728K CA  CA   1970 5648   26.84   13.89   12.95    7.57   13.89   12.95    7.57\n",
      "  158  24508 260G 606M CA  CA   1981 4707   30.40    9.52   20.88    9.99    9.52   20.88    9.99\n",
      "  159  24509 260G 607L CA  CA   1981 4715   23.92   11.87   12.04    5.06   11.87   12.04    5.06\n",
      "  160  24510 260G 728K CA  CA   1981 5648   26.54   11.23   15.31    6.93   11.23   15.31    6.93\n",
      "  161  24511 261I 606M CA  CA   1985 4707   31.75    5.99   25.76   11.50    5.99   25.76   11.50\n",
      "  162  24512 261I 607L CA  CA   1985 4715   25.13    8.92   16.21    6.51    8.92   16.21    6.51\n",
      "  163  24513 261I 728K CA  CA   1985 5648   27.08    8.38   18.70    7.94    8.38   18.70    7.94\n",
      "  164  24514 262D 606M CA  CA   1993 4707   34.33    3.83   30.50   11.67    3.83   30.50   11.67\n",
      "  165  24515 262D 607L CA  CA   1993 4715   27.51    5.40   22.11    7.88    5.40   22.11    7.88\n",
      "  166  24516 262D 728K CA  CA   1993 5648   25.49    5.28   20.22    7.45    5.28   20.22    7.45\n",
      "  167  24517 606M 728K CA  CA   4707 5648   51.69    5.10   46.59   17.79    5.10   46.59   17.79\n",
      "  168  24518 607L 728K CA  CA   4715 5648   44.98    3.83   41.16   15.33    3.83   41.16   15.33\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24519  47G  60V N   O     370  464   24.92    5.46   19.46    6.05    5.46   19.46    6.05\n",
      "    2  24520  47G  61E N   O     370  473   27.31    9.39   17.92    6.20    9.39   17.92    6.20\n",
      "    3  24521  47G  62H N   O     370  483   29.75   10.24   19.51    7.04   10.24   19.51    7.04\n",
      "    4  24522  47G 243C N   O     370 1844   19.91   10.34    9.57   14.12   10.34    9.57   14.12\n",
      "    5  24523  47G 244D N   O     370 1852   19.87    9.80   10.07   11.89    9.80   10.07   11.89\n",
      "    6  24524  47G 246E N   O     370 1869   26.00    8.86   17.14   25.75    8.86   17.14   25.75\n",
      "    7  24525  48T  60V N   O     374  464   19.97    4.55   15.42    5.23    4.55   15.42    5.23\n",
      "    8  24526  48T  61E N   O     374  473   22.22    8.51   13.72    5.19    8.51   13.72    5.19\n",
      "    9  24527  48T  62H N   O     374  483   24.59   10.49   14.10    5.62   10.49   14.10    5.62\n",
      "   10  24528  48T 246E N   O     374 1869   20.83    8.70   12.13   22.85    8.70   12.13   22.85\n",
      "   11  24529  60V  47G N   O     458  373   17.88    3.45   14.42    4.68    3.45   14.42    4.68\n",
      "   12  24534  61E  47G N   O     465  373   21.56    5.39   16.17    6.03    5.39   16.17    6.03\n",
      "   13  24535  61E  48T N   O     465  380   17.62    3.58   14.04    5.79    3.58   14.04    5.79\n",
      "   14  24537  61E 236A N   O     465 1781   14.45    9.83    4.61    9.40    9.83    4.61    9.40\n",
      "   15  24543  62H  47G N   O     474  373   25.01    6.52   18.49    7.23    6.52   18.49    7.23\n",
      "   16  24544  62H  48T N   O     474  380   20.97    6.51   14.46    6.23    6.51   14.46    6.23\n",
      "   17  24557  63V  47G N   O     484  373   26.97    9.89   17.08    7.83    9.89   17.08    7.83\n",
      "   18  24558  63V  48T N   O     484  380   22.81   10.09   12.72    6.44   10.09   12.72    6.44\n",
      "   19  24681  79I 249D N   O     620 1890   12.73   10.58    2.15    5.45   10.58    2.15    5.45\n",
      "   20  24682  79I 250V N   O     620 1897   13.57   10.95    2.63    4.86   10.95    2.63    4.86\n",
      "   21  24683  79I 251H N   O     620 1907   12.11    8.69    3.42    7.25    8.69    3.42    7.25\n",
      "   22  24748  86D 255H N   O     675 1941   16.99   10.89    6.10    7.12   10.89    6.10    7.12\n",
      "   23  24749  86D 258H N   O     675 1968   16.28    8.43    7.85    4.86    8.43    7.85    4.86\n",
      "   24  24756  96N 195M N   O     743 1489   16.66   10.61    6.06    6.96   10.61    6.06    6.96\n",
      "   25  24757  96N 196T N   O     743 1496   13.84    8.31    5.53    7.32    8.31    5.53    7.32\n",
      "   26  24758  96N 197C N   O     743 1502   12.40    8.60    3.79    4.64    8.60    3.79    4.64\n",
      "   27  24759  96N 257Q N   O     743 1958   24.41   10.58   13.83    8.84   10.58   13.83    8.84\n",
      "   28  24760  96N 258H N   O     743 1968   21.62    7.30   14.31    8.95    7.30   14.31    8.95\n",
      "   29  24761  96N 259F N   O     743 1979   22.39   10.74   11.65    5.79   10.74   11.65    5.79\n",
      "   30  24774 103P 255H N   O     792 1941   15.83    9.83    6.00   10.08    9.83    6.00   10.08\n",
      "   31  24775 103P 256M N   O     792 1949   17.46   10.09    7.37   11.61   10.09    7.37   11.61\n",
      "   32  24776 103P 257Q N   O     792 1958   17.13    8.93    8.20    7.13    8.93    8.20    7.13\n",
      "   33  24777 103P 258H N   O     792 1968   14.31    5.45    8.86    6.92    5.45    8.86    6.92\n",
      "   34  24791 116A 255H N   O     887 1941   14.16    9.88    4.28    5.92    9.88    4.28    5.92\n",
      "   35  24792 116A 256M N   O     887 1949   14.28    9.39    4.90    6.56    9.39    4.90    6.56\n",
      "   36  24806 117K 256M N   O     892 1949   11.53    8.74    2.79    4.61    8.74    2.79    4.61\n",
      "   37  24870 129C 229G N   O     985 1738   13.82    9.63    4.19    6.22    9.63    4.19    6.22\n",
      "   38  24871 129C 231V N   O     985 1752   18.90    9.74    9.16   16.21    9.74    9.16   16.21\n",
      "   39  24887 140G 228L N   O    1075 1734   16.58    8.93    7.65   14.89    8.93    7.65   14.89\n",
      "   40  24888 140G 229G N   O    1075 1738   21.00   10.45   10.55   15.45   10.45   10.55   15.45\n",
      "   41  24906 155L 228L N   O    1177 1734   14.23    8.66    5.56   14.12    8.66    5.56   14.12\n",
      "   42  24926 156Q 228L N   O    1185 1734   12.52    9.87    2.65    5.83    9.87    2.65    5.83\n",
      "   43  24948 157C 228L N   O    1194 1734   13.22   10.65    2.57    7.01   10.65    2.57    7.01\n",
      "   44  24951 168S 156Q N   O    1279 1193   14.48    3.36   11.12    4.59    3.36   11.12    4.59\n",
      "   45  24960 168S 191N N   O    1279 1453   20.66   10.13   10.53    5.69   10.13   10.53    5.69\n",
      "   46  24961 168S 192W N   O    1279 1467   17.53    8.00    9.52    6.56    8.00    9.52    6.56\n",
      "   47  24962 168S 198G N   O    1279 1506   17.82    8.70    9.11   13.26    8.70    9.11   13.26\n",
      "   48  24963 168S 199T N   O    1279 1513   16.49    7.75    8.74   11.21    7.75    8.74   11.21\n",
      "   49  24964 168S 200I N   O    1279 1521   15.08    3.06   12.01   13.03    3.06   12.01   13.03\n",
      "   50  24965 168S 201G N   O    1279 1525   15.79    6.91    8.88    6.38    6.91    8.88    6.38\n",
      "   51  24979 179K 192W N   O    1358 1467   16.33    9.76    6.57    4.57    9.76    6.57    4.57\n",
      "   52  24980 179K 198G N   O    1358 1506   18.05   10.77    7.28   10.80   10.77    7.28   10.80\n",
      "   53  24981 179K 199T N   O    1358 1513   16.16    9.61    6.54    8.47    9.61    6.54    8.47\n",
      "   54  24982 179K 200I N   O    1358 1521   13.57    5.15    8.42    8.88    5.15    8.42    8.88\n",
      "   55  24997 180C 200I N   O    1367 1521   11.53    7.05    4.48    7.39    7.05    4.48    7.39\n",
      "   56  25224 195M 252F N   O    1482 1918   11.79    9.69    2.10    5.76    9.69    2.10    5.76\n",
      "   57  25254 196T 252F N   O    1490 1918   12.67   10.58    2.09    5.54   10.58    2.09    5.54\n",
      "   58  25316 199T 168S N   O    1507 1284   16.38   10.73    5.64    6.44   10.73    5.64    6.44\n",
      "   59  25341 200I 168S N   O    1514 1284   15.26    8.25    7.01    6.32    8.25    7.01    6.32\n",
      "   60  25892 228L 182L N   O    1727 1385   15.51   10.86    4.65   11.02   10.86    4.65   11.02\n",
      "   61  25909 228L 231V N   O    1727 1752   10.77    8.46    2.31    5.70    8.46    2.31    5.70\n",
      "   62  25918 229G 129C N   O    1735  990   12.90    9.03    3.87    5.87    9.03    3.87    5.87\n",
      "   63  25919 229G 140G N   O    1735 1078   18.04   10.74    7.30   10.59   10.74    7.30   10.59\n",
      "   64  25920 229G 155L N   O    1735 1184   13.31    9.45    3.87    7.34    9.45    3.87    7.34\n",
      "   65  25936 229G 606M N   O    1735 4713   29.17   10.86   18.31   34.52   10.86   18.31   34.52\n",
      "   66  25937 229G 728K N   O    1735 5655   16.52   10.98    5.54    9.58   10.98    5.54    9.58\n",
      "   67  25954 230T 606M N   O    1739 4713   26.55   10.05   16.50   36.34   10.05   16.50   36.34\n",
      "   68  25955 230T 728K N   O    1739 5655   18.80   10.90    7.90   15.57   10.90    7.90   15.57\n",
      "   69  25956 231V 122P N   O    1746  937   12.20    9.73    2.47    4.96    9.73    2.47    4.96\n",
      "   70  25957 231V 123P N   O    1746  944   11.10    8.38    2.72    4.97    8.38    2.72    4.97\n",
      "   71  25973 231V 606M N   O    1746 4713   23.92    7.57   16.34   39.93    7.57   16.34   39.93\n",
      "   72  25974 231V 728K N   O    1746 5655   20.99    9.06   11.94   25.61    9.06   11.94   25.61\n",
      "   73  25975 232T 123P N   O    1753  944   14.14    9.84    4.30    6.49    9.84    4.30    6.49\n",
      "   74  25988 232T 262D N   O    1753 1999   12.65    7.25    5.40    9.75    7.25    5.40    9.75\n",
      "   75  25989 232T 606M N   O    1753 4713   20.66    4.82   15.85   34.79    4.82   15.85   34.79\n",
      "   76  25990 232T 607L N   O    1753 4721   16.34    8.14    8.20   13.42    8.14    8.20   13.42\n",
      "   77  25991 232T 728K N   O    1753 5655   23.97    7.27   16.70   32.14    7.27   16.70   32.14\n",
      "   78  25999 233P 262D N   O    1760 1999   15.93    6.83    9.10   16.39    6.83    9.10   16.39\n",
      "   79  26000 233P 606M N   O    1760 4713   17.61    3.34   14.26   31.80    3.34   14.26   31.80\n",
      "   80  26001 233P 607L N   O    1760 4721   13.44    5.27    8.17   14.57    5.27    8.17   14.57\n",
      "   81  26002 233P 728K N   O    1760 5655   26.56    6.42   20.14   39.03    6.42   20.14   39.03\n",
      "   82  26009 234S 262D N   O    1767 1999   15.89    9.20    6.69    9.23    9.20    6.69    9.23\n",
      "   83  26010 234S 606M N   O    1767 4713   16.84    5.81   11.04   19.76    5.81   11.04   19.76\n",
      "   84  26011 234S 607L N   O    1767 4721   12.27    7.38    4.89    6.53    7.38    4.89    6.53\n",
      "   85  26012 234S 728K N   O    1767 5655   28.06    9.21   18.85   28.30    9.21   18.85   28.30\n",
      "   86  26020 235G 247V N   O    1773 1876   13.68   10.30    3.38    7.16   10.30    3.38    7.16\n",
      "   87  26025 235G 262D N   O    1773 1999   13.52    9.75    3.76    6.63    9.75    3.76    6.63\n",
      "   88  26026 235G 606M N   O    1773 4713   18.59    6.83   11.76   25.50    6.83   11.76   25.50\n",
      "   89  26027 235G 607L N   O    1773 4721   13.77    9.30    4.47    7.43    9.30    4.47    7.43\n",
      "   90  26028 235G 728K N   O    1773 5655   27.67   10.50   17.17   32.00   10.50   17.17   32.00\n",
      "   91  26039 236A 238F N   O    1777 1800   10.14    8.28    1.85    4.79    8.28    1.85    4.79\n",
      "   92  26040 236A 247V N   O    1777 1876   10.37    7.56    2.81    7.14    7.56    2.81    7.14\n",
      "   93  26048 236A 606M N   O    1777 4713   21.99    8.78   13.21   32.26    8.78   13.21   32.26\n",
      "   94  26064 237D 246E N   O    1782 1869   12.24    9.79    2.45    4.59    9.79    2.45    4.59\n",
      "   95  26082 238F 231V N   O    1790 1752   13.14   10.98    2.16    5.17   10.98    2.16    5.17\n",
      "   96  26210 245D  47G N   O    1853  373   17.66    9.49    8.17    9.38    9.49    8.17    9.38\n",
      "   97  26225 246E  47G N   O    1861  373   20.52    8.04   12.48   19.47    8.04   12.48   19.47\n",
      "   98  26226 246E  48T N   O    1861  380   15.97    8.02    7.94   15.40    8.02    7.94   15.40\n",
      "   99  26233 246E 236A N   O    1861 1781   12.82    9.53    3.29    7.21    9.53    3.29    7.21\n",
      "  100  26234 246E 237D N   O    1861 1789   11.07    8.66    2.41    4.60    8.66    2.41    4.60\n",
      "  101  26243 247V  47G N   O    1870  373   23.63    9.09   14.54   23.72    9.09   14.54   23.72\n",
      "  102  26244 247V  48T N   O    1870  380   19.04    8.35   10.68   22.31    8.35   10.68   22.31\n",
      "  103  26253 247V 236A N   O    1870 1781    9.91    7.22    2.69    6.87    7.22    2.69    6.87\n",
      "  104  26264 248S  47G N   O    1877  373   26.65   10.21   16.44   19.64   10.21   16.44   19.64\n",
      "  105  26265 248S  48T N   O    1877  380   22.11    9.22   12.88   20.38    9.22   12.88   20.38\n",
      "  106  26276 248S 234S N   O    1877 1772   15.20   10.72    4.48    5.99   10.72    4.48    5.99\n",
      "  107  26378 253A 262D N   O    1919 1999   14.28    9.70    4.58    5.91    9.70    4.58    5.91\n",
      "  108  26379 253A 606M N   O    1919 4713   19.17    9.45    9.72   15.08    9.45    9.72   15.08\n",
      "  109  26393 254N 261I N   O    1924 1991   10.15    6.99    3.16    4.94    6.99    3.16    4.94\n",
      "  110  26394 254N 262D N   O    1924 1999   15.17   10.28    4.89    5.37   10.28    4.89    5.37\n",
      "  111  26395 254N 606M N   O    1924 4713   19.79   10.87    8.93   11.40   10.87    8.93   11.40\n",
      "  112  26424 256M 116A N   O    1942  891   14.53    9.86    4.67    6.14    9.86    4.67    6.14\n",
      "  113  26446 256M 606M N   O    1942 4713   23.51   10.48   13.03   21.50   10.48   13.03   21.50\n",
      "  114  26461 257Q 261I N   O    1950 1991    8.79    3.50    5.30    8.95    3.50    5.30    8.95\n",
      "  115  26463 257Q 606M N   O    1950 4713   22.48    9.90   12.58   10.97    9.90   12.58   10.97\n",
      "  116  26468 258H  96N N   O    1959  750   22.20   10.98   11.22    6.40   10.98   11.22    6.40\n",
      "  117  26469 258H 103P N   O    1959  798   17.54    8.91    8.63    6.23    8.91    8.63    6.23\n",
      "  118  26483 258H 261I N   O    1959 1991   10.14    5.68    4.46    4.85    5.68    4.46    4.85\n",
      "  119  26489 259F  96N N   O    1969  750   19.90    9.14   10.76    5.51    9.14   10.76    5.51\n",
      "  120  26490 259F 103P N   O    1969  798   15.19    6.79    8.40    5.23    6.79    8.40    5.23\n",
      "  121  26539 261I 256M N   O    1984 1949    5.23    3.07    2.16    5.31    3.07    2.16    5.31\n",
      "  122  26540 261I 257Q N   O    1984 1958    7.81    4.22    3.58    5.80    4.22    3.58    5.80\n",
      "  123  26543 261I 606M N   O    1984 4713   26.51    9.30   17.22    7.11    9.30   17.22    7.11\n",
      "  124  26544 261I 728K N   O    1984 5655   23.82    9.61   14.20    5.67    9.61   14.20    5.67\n",
      "  125  26550 262D 231V N   O    1992 1752   12.13    8.91    3.22    6.39    8.91    3.22    6.39\n",
      "  126  26551 262D 232T N   O    1992 1759   12.42    8.79    3.63    6.26    8.79    3.63    6.26\n",
      "  127  26552 262D 233P N   O    1992 1766   15.24   10.81    4.44    7.19   10.81    4.44    7.19\n",
      "  128  26555 262D 253A N   O    1992 1923   11.74    6.80    4.94    7.30    6.80    4.94    7.30\n",
      "  129  26558 262D 256M N   O    1992 1949    9.49    5.39    4.10    6.84    5.39    4.10    6.84\n",
      "  130  26559 262D 257Q N   O    1992 1958   12.10    6.62    5.48    4.72    6.62    5.48    4.72\n",
      "  131  26563 262D 606M N   O    1992 4713   28.83    6.20   22.63    8.02    6.20   22.63    8.02\n",
      "  132  26564 262D 607L N   O    1992 4721   24.16    8.01   16.15    5.19    8.01   16.15    5.19\n",
      "  133  26565 262D 728K N   O    1992 5655   23.05    6.56   16.50    5.69    6.56   16.50    5.69\n",
      "  134  26566 606M 118S N   O    4706  906   38.93    9.51   29.42   10.64    9.51   29.42   10.64\n",
      "  135  26567 606M 122P N   O    4706  937   41.98   10.15   31.83   10.83   10.15   31.83   10.83\n",
      "  136  26568 606M 123P N   O    4706  944   42.40   10.43   31.97   10.37   10.43   31.97   10.37\n",
      "  137  26569 606M 230T N   O    4706 1745   32.17    9.71   22.46   49.82    9.71   22.46   49.82\n",
      "  138  26570 606M 231V N   O    4706 1752   30.60    7.12   23.48   57.88    7.12   23.48   57.88\n",
      "  139  26571 606M 232T N   O    4706 1759   27.28    5.78   21.50   46.02    5.78   21.50   46.02\n",
      "  140  26572 606M 233P N   O    4706 1766   23.93    7.57   16.36   32.91    7.57   16.36   32.91\n",
      "  141  26573 606M 234S N   O    4706 1772   25.61   10.52   15.09   23.73   10.52   15.09   23.73\n",
      "  142  26574 606M 235G N   O    4706 1776   29.93    7.86   22.07   46.90    7.86   22.07   46.90\n",
      "  143  26575 606M 251H N   O    4706 1907   29.86   10.34   19.52   37.76   10.34   19.52   37.76\n",
      "  144  26576 606M 252F N   O    4706 1918   29.68    7.37   22.31   51.28    7.37   22.31   51.28\n",
      "  145  26577 606M 253A N   O    4706 1923   28.13    6.24   21.89   39.29    6.24   21.89   39.29\n",
      "  146  26578 606M 254N N   O    4706 1931   31.19    9.32   21.87   29.62    9.32   21.87   29.62\n",
      "  147  26579 606M 255H N   O    4706 1941   34.24    9.46   24.78   44.84    9.46   24.78   44.84\n",
      "  148  26580 606M 256M N   O    4706 1949   32.96    6.87   26.09   49.70    6.87   26.09   49.70\n",
      "  149  26581 606M 257Q N   O    4706 1958   33.26    8.68   24.58   22.16    8.68   24.58   22.16\n",
      "  150  26582 606M 259F N   O    4706 1979   36.37    9.56   26.81   16.32    9.56   26.81   16.32\n",
      "  151  26583 606M 260G N   O    4706 1983   36.81    7.12   29.70   13.98    7.12   29.70   13.98\n",
      "  152  26584 606M 261I N   O    4706 1991   35.87    3.73   32.14   14.33    3.73   32.14   14.33\n",
      "  153  26585 606M 262D N   O    4706 1999   40.82    2.24   38.58   14.54    2.24   38.58   14.54\n",
      "  154  26586 606M 728K N   O    4706 5655   52.80    5.74   47.05   17.17    5.74   47.05   17.17\n",
      "  155  26587 607L 230T N   O    4714 1745   22.40   10.06   12.34   21.24   10.06   12.34   21.24\n",
      "  156  26588 607L 231V N   O    4714 1752   20.95    7.47   13.47   25.95    7.47   13.47   25.95\n",
      "  157  26589 607L 232T N   O    4714 1759   17.54    5.05   12.48   22.82    5.05   12.48   22.82\n",
      "  158  26590 607L 233P N   O    4714 1766   14.15    5.75    8.40   14.71    5.75    8.40   14.71\n",
      "  159  26591 607L 234S N   O    4714 1772   15.94    9.53    6.42    7.98    9.53    6.42    7.98\n",
      "  160  26592 607L 235G N   O    4714 1776   20.17    7.92   12.25   21.15    7.92   12.25   21.15\n",
      "  161  26593 607L 252F N   O    4714 1918   20.47    8.95   11.51   18.23    8.95   11.51   18.23\n",
      "  162  26594 607L 253A N   O    4714 1923   19.08    7.87   11.21   14.42    7.87   11.21   14.42\n",
      "  163  26595 607L 256M N   O    4714 1949   23.75    9.47   14.29   18.38    9.47   14.29   18.38\n",
      "  164  26596 607L 257Q N   O    4714 1958   24.41   10.98   13.43    9.51   10.98   13.43    9.51\n",
      "  165  26597 607L 260G N   O    4714 1983   27.27    9.44   17.84    7.32    9.44   17.84    7.32\n",
      "  166  26598 607L 261I N   O    4714 1991   26.26    6.26   20.00    7.95    6.26   20.00    7.95\n",
      "  167  26599 607L 262D N   O    4714 1999   31.16    3.40   27.76    9.67    3.40   27.76    9.67\n",
      "  168  26600 607L 728K N   O    4714 5655   43.33    5.32   38.01   13.07    5.32   38.01   13.07\n",
      "  169  26601 728K 231V N   O    5647 1752   28.45    8.18   20.27   44.33    8.18   20.27   44.33\n",
      "  170  26602 728K 232T N   O    5647 1759   32.42    6.93   25.50   49.56    6.93   25.50   49.56\n",
      "  171  26603 728K 233P N   O    5647 1766   35.62    7.68   27.94   52.37    7.68   27.94   52.37\n",
      "  172  26604 728K 235G N   O    5647 1776   31.16    9.89   21.27   40.14    9.89   21.27   40.14\n",
      "  173  26605 728K 253A N   O    5647 1923   35.86   10.01   25.85   35.91   10.01   25.85   35.91\n",
      "  174  26606 728K 256M N   O    5647 1949   30.91   10.79   20.13   29.36   10.79   20.13   29.36\n",
      "  175  26607 728K 260G N   O    5647 1983   27.13    9.33   17.80    7.82    9.33   17.80    7.82\n",
      "  176  26608 728K 261I N   O    5647 1991   28.71    7.36   21.34    8.77    7.36   21.34    8.77\n",
      "  177  26609 728K 262D N   O    5647 1999   25.59    3.27   22.31    8.11    3.27   22.31    8.11\n",
      "  178  26610 728K 606M N   O    5647 4713   49.60    3.30   46.30   17.49    3.30   46.30   17.49\n",
      "  179  26611 728K 607L N   O    5647 4721   45.76    2.29   43.47   15.79    2.29   43.47   15.79\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20515  70C  70C CA  C     543  546 -156.96 -180.00   23.04    4.61 -180.00   23.04    4.61\n",
      "    2  21025 580M 580M CA  C    4511 4516 -155.53 -180.00   24.47    4.89 -180.00   24.47    4.89\n",
      "    3  21032 587L 587L CA  C    4566 4571 -146.36 -180.00   33.64    6.73 -180.00   33.64    6.73\n",
      "    4  21051 606M 606M CA  C    4707 4712 -145.44 -180.00   34.56    6.91 -180.00   34.56    6.91\n",
      "    5  21166 721Y 721Y CA  C    5582 5591 -154.46 -180.00   25.54    5.11 -180.00   25.54    5.11\n",
      "    6  21169 724K 724K CA  C    5617 5623 -148.29 -180.00   31.71    6.34 -180.00   31.71    6.34\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19720   1M   2K C   N       7    9 -143.50 -118.00   25.50    0.97  -62.90 -163.22   22.65\n",
      "    1          2K   2K N   CA      9   10  138.72  139.10                  -40.80\n",
      "    2  19723   4P   5F C   N      30   32 -156.85 -124.20   42.74    1.12  -63.20  172.47   29.76\n",
      "    2          5F   5F N   CA     32   33  170.88  143.30                  -44.30\n",
      "    3  19724   5F   6N C   N      41   43 -166.16 -119.90   59.96    1.74  -63.20  176.83   27.72\n",
      "    3          6N   6N N   CA     43   44  175.14  137.00                  -41.10\n",
      "    4  19726   7I   8T C   N      57   59 -155.59 -124.80   46.97    1.48  -63.20  166.84   27.22\n",
      "    4          8T   8T N   CA     59   60  178.97  143.50                  -42.10\n",
      "    5  19727   8T   9D C   N      64   66 -165.78  -96.50   93.48    3.90  -63.30  175.96   28.18\n",
      "    5          9D   9D N   CA     66   67  176.96  114.20                  -40.00\n",
      "    6  19728   9D  10E C   N      72   74 -167.73 -117.80   62.32    1.87  -63.60  179.01   29.49\n",
      "    6         10E  10E N   CA     74   75  174.09  136.80                  -40.30\n",
      "    7  19729  10E  11A C   N      81   83 -167.34 -134.00   47.90    1.47  -62.50  173.08   33.86\n",
      "    7         11A  11A N   CA     83   84 -178.61  147.00                  -40.90\n",
      "    8  19731  12C  13A C   N      92   94 -173.82 -134.00   47.60    1.15  -62.50 -176.37   35.93\n",
      "    8         13A  13A N   CA     94   95  173.07  147.00                  -40.90\n",
      "    9  19732  13A  14H C   N      97   99 -163.20 -125.60   52.53    1.30  -63.20  173.85   28.08\n",
      "    9         14H  14H N   CA     99  100  175.49  138.80                  -42.30\n",
      "   10  19734  15C  16W C   N     113  115 -169.76 -124.90   53.39    1.72  -63.00  178.82   29.47\n",
      "   10         16W  16W N   CA    115  116  172.35  143.40                  -44.20\n",
      "   11  19735  16W  17D C   N     127  129 -175.67  -96.50   99.72    4.15  -63.30 -176.42   29.70\n",
      "   11         17D  17D N   CA    129  130  174.83  114.20                  -40.00\n",
      "   12  19736  17D  18E C   N     135  137 -175.40 -117.80   66.79    1.96  -63.60 -173.65   30.85\n",
      "   12         18E  18E N   CA    137  138  170.61  136.80                  -40.30\n",
      "   13  19737  18E  19S C   N     144  146  171.78 -136.60   55.65    1.76  -64.10 -162.97   23.22\n",
      "   13         19S  19S N   CA    146  147  171.98  151.20                  -35.00\n",
      "   14  19739  20V  21M C   N     157  159  175.76 -125.60   66.95    2.00  -63.40 -169.93   37.31\n",
      "   14         21M  21M N   CA    159  160  172.79  140.50                  -40.50\n",
      "   15  19740  21M  22L C   N     165  167  177.63 -108.50   81.82    3.62  -63.50 -167.74   33.85\n",
      "   15         22L  22L N   CA    167  168  167.69  132.50                  -41.20\n",
      "   16  19743  24P  25N C   N     187  189 -176.69 -119.90   69.65    2.04  -63.20 -178.54   28.78\n",
      "   16         25N  25N N   CA    189  190  177.31  137.00                  -41.10\n",
      "   17  19744  25N  26H C   N     195  197 -178.51 -125.60   64.95    1.61  -63.20 -177.66   29.99\n",
      "   17         26H  26H N   CA    197  198  176.46  138.80                  -42.30\n",
      "   18  19745  26H  27R C   N     205  207  151.73 -125.20   87.79    2.85  -63.00 -151.25   37.28\n",
      "   18         27R  27R N   CA    207  208  168.99  140.60                  -41.10\n",
      "   19  19746  27R  28S C   N     216  218 -179.78 -136.60   50.10    1.53  -64.10 -171.85   21.98\n",
      "   19         28S  28S N   CA    218  219  176.61  151.20                  -35.00\n",
      "   20  19747  28S  29A C   N     222  224 -178.50 -134.00   53.73    1.32  -62.50 -176.64   36.09\n",
      "   20         29A  29A N   CA    224  225  177.10  147.00                  -40.90\n",
      "   21  19749  30V  31H C   N     234  236 -176.47 -125.60   64.54    1.59  -63.20  179.44   29.50\n",
      "   21         31H  31H N   CA    236  237  178.52  138.80                  -42.30\n",
      "   22  19750  31H  32K C   N     244  246 -177.68 -118.00   67.88    2.25  -62.90 -172.92   31.02\n",
      "   22         32K  32K N   CA    246  247  171.46  139.10                  -40.80\n",
      "   23  19751  32K  33E C   N     253  255 -170.39 -117.80   67.05    2.03  -63.60  177.11   29.34\n",
      "   23         33E  33E N   CA    255  256  178.40  136.80                  -40.30\n",
      "   24  19752  33E  34E C   N     262  264 -173.88 -117.80   67.69    2.00  -63.60 -177.83   30.19\n",
      "   24         34E  34E N   CA    264  265  174.70  136.80                  -40.30\n",
      "   25  19754  35C  36A C   N     277  279  177.31 -134.00   54.07    1.23  -62.50 -168.89   37.58\n",
      "   25         36A  36A N   CA    279  280  170.52  147.00                  -40.90\n",
      "   26  19755  36A  37Y C   N     282  284 -161.68 -124.30   61.67    2.51  -63.50  164.63   30.61\n",
      "   26         37Y  37Y N   CA    284  285 -175.55  135.40                  -43.40\n",
      "   27  19758  39C  40R C   N     306  308 -175.52 -125.20   60.01    1.77  -63.00 -175.98   32.17\n",
      "   27         40R  40R N   CA    308  309  173.29  140.60                  -41.10\n",
      "   28  19759  40R  41T C   N     317  319 -179.17 -124.80   63.42    2.39  -63.20 -176.84   30.57\n",
      "   28         41T  41T N   CA    319  320  176.14  143.50                  -42.10\n",
      "   29  19761  42C  43R C   N     330  332  175.32 -125.20   68.45    2.05  -63.00 -171.16   33.31\n",
      "   29         43R  43R N   CA    332  333  174.49  140.60                  -41.10\n",
      "   30  19762  43R  44H C   N     341  343  173.51 -125.60   67.31    1.79  -63.20 -165.66   31.98\n",
      "   30         44H  44H N   CA    343  344  167.48  138.80                  -42.30\n",
      "   31  19763  44H  45E C   N     351  353  167.36 -117.80   81.43    2.40  -63.60 -161.53   33.25\n",
      "   31         45E  45E N   CA    353  354  168.90  136.80                  -40.30\n",
      "   32  19764  45E  46M C   N     360  362  169.73 -125.60   71.94    2.19  -63.40 -165.46   38.36\n",
      "   32         46M  46M N   CA    362  363  172.02  140.50                  -40.50\n",
      "   33  19765  46M  47G C   N     368  370  166.12 -167.20   28.82    0.84  -80.20  114.15    7.89\n",
      "   33         47G  47G N   CA    370  371  163.70  174.60                  174.10\n",
      "   34  19767  48T  49L C   N     379  381 -164.35 -108.50   58.49    2.62  -63.50 -163.26   33.51\n",
      "   34         49L  49L N   CA    381  382  149.87  132.50                  -41.20\n",
      "   35  19770  51C  52M C   N     400  402   -2.09  -73.00  101.13    6.40  -63.40  127.15   23.75\n",
      "   35         52M  52M N   CA    402  403   70.89  143.00                  -40.50\n",
      "   36  19773  54C  55H C   N     420  422 -160.93 -125.60   37.50    1.06  -63.20 -167.09   30.41\n",
      "   36         55H  55H N   CA    422  423  151.37  138.80                  -42.30\n",
      "   37  19774  55H  56M C   N     430  432 -179.08 -125.60   58.01    1.80  -63.40 -165.37   37.75\n",
      "   37         56M  56M N   CA    432  433  162.98  140.50                  -40.50\n",
      "   38  19775  56M  57G C   N     438  440  158.96 -167.20   45.81    1.68   82.20  155.48   10.98\n",
      "   38         57G  57G N   CA    440  441  143.72  174.60                    8.50\n",
      "   39  19776  57G  58L C   N     442  444 -158.05 -108.50   55.02    2.43  -63.50 -172.10   31.91\n",
      "   39         58L  58L N   CA    444  445  156.42  132.50                  -41.20\n",
      "   40  19778  59C  60V C   N     456  458  170.22 -125.40   92.91    2.63  -73.50  136.29    9.29\n",
      "   40         60V  60V N   CA    458  459 -149.71  143.30                  139.20\n",
      "   41  19779  60V  61E C   N     463  465 -176.10 -117.80   92.86    3.19  -69.30  125.86    7.30\n",
      "   41         61E  61E N   CA    465  466 -150.92  136.80                  142.50\n",
      "   42  19780  61E  62H C   N     472  474   78.27  -67.60  150.57   12.38  -67.60  150.57   12.38\n",
      "   42         62H  62H N   CA    474  475  177.35  140.00                  140.00\n",
      "   43  19787  68L  69N C   N     532  534 -172.33 -119.90   60.37    1.82  -63.20 -172.89   29.34\n",
      "   43         69N  69N N   CA    534  535  166.92  137.00                  -41.10\n",
      "   44  19790  71P  72T C   N     553  555 -147.51 -124.80   23.65    1.07  -63.20 -172.20   29.56\n",
      "   44         72T  72T N   CA    555  556  150.09  143.50                  -42.10\n",
      "   45  19791  72T  73H C   N     560  562 -170.72 -125.60   90.06    2.49  -63.20  147.48   24.87\n",
      "   45         73H  73H N   CA    562  563 -143.25  138.80                  -42.30\n",
      "   46  19793  74V  75M C   N     577  579   78.99  -73.00  153.28   11.88  -63.40 -148.27   26.94\n",
      "   46         75M  75M N   CA    579  580  162.80  143.00                  -40.50\n",
      "   47  19794  75M  76Y C   N     585  587   70.17 -124.30  166.49   13.01  -63.50 -150.77   39.16\n",
      "   47         76Y  76Y N   CA    587  588  117.55  135.40                  -43.40\n",
      "   48  19796  77V  78W C   N     604  606  117.39 -124.90  118.13    4.95  -63.00 -117.82   41.80\n",
      "   48         78W  78W N   CA    606  607  153.34  143.40                  -44.20\n",
      "   49  19803  84A  85K C   N     664  666 -100.35 -118.00   50.40    2.15  -62.90  137.88   16.25\n",
      "   49         85K  85K N   CA    666  667   91.90  139.10                  -40.80\n",
      "   50  19805  86D  87E C   N     681  683  133.30 -117.80  109.03    3.86  -63.60 -118.89   40.62\n",
      "   50         87E  87E N   CA    683  684  142.13  136.80                  -40.30\n",
      "   51  19806  87E  88D C   N     690  692 -151.25  -96.50   56.57    2.32  -63.30 -169.98   20.16\n",
      "   51         88D  88D N   CA    692  693  128.45  114.20                  -40.00\n",
      "   52  19807  88D  89A C   N     698  700 -153.79 -134.00   24.57    0.62  -62.50 -177.90   34.71\n",
      "   52         89A  89A N   CA    700  701  161.54  147.00                  -40.90\n",
      "   53  19808  89A  90G C   N     703  705 -176.20 -167.20   15.24    0.60   82.20 -175.67   13.37\n",
      "   53         90G  90G N   CA    705  706  162.30  174.60                    8.50\n",
      "   54  19809  90G  91G C   N     707  709 -103.33  -62.40   73.70   13.17   82.20 -153.21    8.44\n",
      "   54         91G  91G N   CA    709  710 -102.49  -41.20                    8.50\n",
      "   55  19811  92S  93K C   N     717  719 -164.65 -118.00   53.26    1.76  -62.90 -175.10   30.04\n",
      "   55         93K  93K N   CA    719  720  164.81  139.10                  -40.80\n",
      "   56  19812  93K  94D C   N     726  728  148.26  -96.50  125.76    5.20  -63.30 -145.05   35.59\n",
      "   56         94D  94D N   CA    728  729  164.54  114.20                  -40.00\n",
      "   57  19815  96N  97K C   N     749  751  142.87 -118.00  110.03    3.64  -62.90 -156.77   34.98\n",
      "   57         97K  97K N   CA    751  752 -173.14  139.10                  -40.80\n",
      "   58  19816  97K  98L C   N     758  760  -69.13  -70.70    7.60    0.52  -63.50  175.46   24.05\n",
      "   58         98L  98L N   CA    760  761  134.17  141.60                  -41.20\n",
      "   59  19817  98L  99G C   N     766  768   89.33   78.70   13.01    0.38   82.20  178.03    9.00\n",
      "   59         99G  99G N   CA    768  769 -173.61 -166.10                    8.50\n",
      "   60  19819 100V 101L C   N     777  779  116.60 -108.50  134.90    6.41  -63.50 -109.25   32.44\n",
      "   60        101L 101L N   CA    779  780  133.49  132.50                  -41.20\n",
      "   61  19820 101L 102A C   N     785  787 -173.13 -134.00   69.17    2.57  -62.50  159.62   31.81\n",
      "   61        102A 102A N   CA    787  788 -155.96  147.00                  -40.90\n",
      "   62  19822 103P 104K C   N     797  799  147.64 -118.00   98.42    3.34  -62.90 -146.75   36.22\n",
      "   62        104K 104K N   CA    799  800  167.09  139.10                  -40.80\n",
      "   63  19823 104K 105E C   N     806  808 -154.26 -117.80   46.72    1.42  -63.60  178.44   28.79\n",
      "   63        105E 105E N   CA    808  809  166.01  136.80                  -40.30\n",
      "   64  19824 105E 106Y C   N     815  817 -139.34 -124.30   28.54    1.13  -63.50  174.30   31.38\n",
      "   64        106Y 106Y N   CA    817  818  159.66  135.40                  -43.40\n",
      "   65  19825 106Y 107E C   N     827  829 -156.93 -117.80   47.61    1.41  -63.60 -178.40   29.35\n",
      "   65        107E 107E N   CA    829  830  163.92  136.80                  -40.30\n",
      "   66  19826 107E 108N C   N     836  838 -111.29 -119.90   13.35    0.40  -63.20  174.64   19.58\n",
      "   66        108N 108N N   CA    838  839  126.79  137.00                  -41.10\n",
      "   67  19827 108N 109A C   N     844  846   23.72  -68.20  101.33    7.07  -62.50  167.44   32.03\n",
      "   67        109A 109A N   CA    846  847  102.65  145.30                  -40.90\n",
      "   68  19828 109A 110L C   N     849  851  154.68 -108.50  119.99    5.31  -63.50 -177.15   33.39\n",
      "   68        110L 110L N   CA    851  852 -156.62  132.50                  -41.20\n",
      "   69  19831 112C 113A C   N     869  871  178.31 -134.00   50.91    1.18  -62.50 -165.04   38.19\n",
      "   69        113A 113A N   CA    871  872  164.82  147.00                  -40.90\n",
      "   70  19832 113A 114A C   N     874  876  164.59 -134.00   61.61    1.86  -62.50 -146.41   41.94\n",
      "   70        114A 114A N   CA    876  877  151.91  147.00                  -40.90\n",
      "   71  19836 117K 118S C   N     899  901  158.08 -136.60  145.67    8.25  -72.40 -175.49   14.65\n",
      "   71        118S 118S N   CA    901  902   20.99  151.20                  152.40\n",
      "   72  19837 118S 119F C   N     905  907 -150.04 -124.20   87.13    3.39  -63.20  124.48   22.23\n",
      "   72        119F 119F N   CA    907  908 -133.48  143.30                  -44.30\n",
      "   73  19839 120V 121S C   N     923  925  136.79 -136.60   89.04    3.04  -64.10 -139.15   27.36\n",
      "   73        121S 121S N   CA    925  926  171.83  151.20                  -35.00\n",
      "   74  19842 123P 124E C   N     943  945  -70.51  -69.30   25.27    1.89  -63.60  152.11   20.66\n",
      "   74        124E 124E N   CA    945  946  167.74  142.50                  -40.30\n",
      "   75  19843 124E 125L C   N     952  954  -62.89  -70.70    9.46    0.95  -63.50  171.85   23.83\n",
      "   75        125L 125L N   CA    954  955  146.95  141.60                  -41.20\n",
      "   76  19844 125L 126T C   N     960  962 -162.09 -124.80  100.10    3.31  -63.20  128.15   21.98\n",
      "   76        126T 126T N   CA    962  963 -123.60  143.50                  -42.10\n",
      "   77  19845 126T 127I C   N     967  969  -75.98  -63.40   13.25    2.22 -120.60  175.53    8.37\n",
      "   77        127I 127I N   CA    969  970  -39.46  -43.60                  130.30\n",
      "   78  19846 127I 128D C   N     975  977  -32.52  -70.90   54.26    4.39  -63.30  134.90   15.05\n",
      "   78        128D 128D N   CA    977  978 -171.34  150.30                  -40.00\n",
      "   79  19847 128D 129C C   N     983  985   46.04  -69.10  167.59   14.88 -117.90 -155.36    8.85\n",
      "   79        129C 129C N   CA    985  986  -96.43  141.80                  141.10\n",
      "   80  19848 129C 130Y C   N     989  991   36.32  -98.40  139.78    6.15  -63.50 -179.05   25.00\n",
      "   80        130Y 130Y N   CA    991  992  165.67  128.40                  -43.40\n",
      "   81  19849 130Y 131Q C   N    1001 1003 -140.01 -121.10   33.29    1.18  -63.80  170.57   29.04\n",
      "   81        131Q 131Q N   CA   1003 1004  167.10  139.70                  -40.30\n",
      "   82  19850 131Q 132W C   N    1010 1012  -32.92  -71.30   38.82    2.93  -63.00  173.59   20.12\n",
      "   82        132W 132W N   CA   1012 1013  144.84  139.00                  -44.20\n",
      "   83  19853 134I 135N C   N    1040 1042   47.05   55.90   12.13    0.75  -63.20  141.62   23.16\n",
      "   83        135N 135N N   CA   1042 1043   47.79   39.50                  -41.10\n",
      "   84  19854 135N 136A C   N    1048 1050 -153.01 -134.00   33.07    1.21  -62.50  170.96   32.83\n",
      "   84        136A 136A N   CA   1050 1051  174.06  147.00                  -40.90\n",
      "   85  19855 136A 137T C   N    1053 1055 -114.81 -124.80   17.25    0.53  -63.20  179.13   20.01\n",
      "   85        137T 137T N   CA   1055 1056  129.43  143.50                  -42.10\n",
      "   86  19856 137T 138S C   N    1060 1062  158.44 -136.60   70.30    2.21  -64.10 -158.79   24.44\n",
      "   86        138S 138S N   CA   1062 1063  178.06  151.20                  -35.00\n",
      "   87  19857 138S 139T C   N    1066 1068  124.40 -124.80  121.41    4.99  -63.20 -147.18   36.63\n",
      "   87        139T 139T N   CA   1068 1069 -166.87  143.50                  -42.10\n",
      "   88  19858 139T 140G C   N    1073 1075  -56.98  -62.40   14.52    2.46   82.20  143.82   10.73\n",
      "   88        140G 140G N   CA   1075 1076  -27.73  -41.20                    8.50\n",
      "   89  19859 140G 141A C   N    1077 1079 -118.97 -134.00   51.32    3.14  -62.50  135.38   25.21\n",
      "   89        141A 141A N   CA   1079 1080 -163.93  147.00                  -40.90\n",
      "   90  19860 141A 142Q C   N    1082 1084 -102.98 -121.10   74.21    3.77  -63.80  114.93   18.98\n",
      "   90        142Q 142Q N   CA   1084 1085 -148.34  139.70                  -40.30\n",
      "   91  19861 142Q 143A C   N    1091 1093 -129.53 -134.00   15.17    0.93  -62.50  171.27   31.66\n",
      "   91        143A 143A N   CA   1093 1094  161.50  147.00                  -40.90\n",
      "   92  19862 143A 144A C   N    1096 1098 -133.01 -134.00   43.91    2.52  -62.50  146.31   27.75\n",
      "   92        144A 144A N   CA   1098 1099 -169.10  147.00                  -40.90\n",
      "   93  19864 145I 146E C   N    1109 1111  -93.39  -63.60   85.45   13.08  -63.60   85.45   13.08\n",
      "   93        146E 146E N   CA   1111 1112 -120.39  -40.30                  -40.30\n",
      "   94  19866 147P 148E C   N    1125 1127   54.94  -69.30  128.02    8.58  -63.60 -167.30   32.02\n",
      "   94        148E 148E N   CA   1127 1128  111.62  142.50                  -40.30\n",
      "   95  19869 150V 151A C   N    1145 1147 -115.39 -134.00   30.20    1.82  -62.50  157.46   28.63\n",
      "   95        151A 151A N   CA   1147 1148  170.79  147.00                  -40.90\n",
      "   96  19870 151A 152S C   N    1150 1152 -142.67 -136.60   36.89    1.74  -64.10  158.29   17.13\n",
      "   96        152S 152S N   CA   1152 1153 -172.42  151.20                  -35.00\n",
      "   97  19871 152S 153M C   N    1156 1158 -154.70 -125.60   49.52    1.64  -63.40  166.25   31.79\n",
      "   97        153M 153M N   CA   1158 1159 -179.44  140.50                  -40.50\n",
      "   98  19872 153M 154R C   N    1164 1166 -178.72 -125.20   63.32    1.88  -63.00 -174.90   32.48\n",
      "   98        154R 154R N   CA   1166 1167  174.43  140.60                  -41.10\n",
      "   99  19873 154R 155L C   N    1175 1177 -129.61 -108.50  165.81    8.73  -70.70  166.16   10.67\n",
      "   99        155L 155L N   CA   1177 1178  -63.03  132.50                  141.60\n",
      "  100  19874 155L 156Q C   N    1183 1185  -31.42  -73.00  175.10   12.30 -121.10 -166.83   10.09\n",
      "  100        156Q 156Q N   CA   1185 1186  -49.20  140.70                  139.70\n",
      "  101  19877 158P 159H C   N    1205 1207   -6.40  -67.60   62.35    5.03  -63.20  175.22   19.10\n",
      "  101        159H 159H N   CA   1207 1208  151.94  140.00                  -42.30\n",
      "  102  19878 159H 160L C   N    1215 1217 -163.01 -108.50   54.81    2.68  -63.50 -164.77   23.94\n",
      "  102        160L 160L N   CA   1217 1218  126.76  132.50                  -41.20\n",
      "  103  19883 164E 165Q C   N    1253 1255 -168.14 -121.10   51.96    1.61  -63.80 -170.72   33.09\n",
      "  103        165Q 165Q N   CA   1255 1256  161.77  139.70                  -40.30\n",
      "  104  19884 165Q 166L C   N    1262 1264   30.26  -70.70  102.90    8.65  -63.50 -172.04   31.87\n",
      "  104        166L 166L N   CA   1264 1265  121.70  141.60                  -41.20\n",
      "  105  19888 169P 170F C   N    1290 1292  -77.88  -71.40   27.00    1.98  -63.20  149.51   21.75\n",
      "  105        170F 170F N   CA   1292 1293  166.91  140.70                  -44.30\n",
      "  106  19889 170F 171Q C   N    1301 1303  -60.86  -73.00   32.29    2.35  -63.80  149.10   21.77\n",
      "  106        171Q 171Q N   CA   1303 1304  170.62  140.70                  -40.30\n",
      "  107  19890 171Q 172T C   N    1310 1312 -115.45 -124.80   41.46    2.05  -63.20  143.84   21.89\n",
      "  107        172T 172T N   CA   1312 1313 -176.11  143.50                  -42.10\n",
      "  108  19891 172T 173A C   N    1317 1319 -157.86 -134.00   58.71    2.59  -62.50  152.07   29.89\n",
      "  108        173A 173A N   CA   1319 1320 -159.36  147.00                  -40.90\n",
      "  109  19893 174P 175T C   N    1329 1331 -169.08 -124.80   65.20    2.07  -63.20  165.00   27.60\n",
      "  109        175T 175T N   CA   1331 1332 -168.64  143.50                  -42.10\n",
      "  110  19894 175T 176S C   N    1336 1338 -159.30 -136.60   28.36    0.87  -64.10 -176.56   20.16\n",
      "  110        176S 176S N   CA   1338 1339  168.19  151.20                  -35.00\n",
      "  111  19895 176S 177S C   N    1342 1344 -162.01 -136.60   34.27    1.08  -64.10  179.81   20.09\n",
      "  111        177S 177S N   CA   1344 1345  174.18  151.20                  -35.00\n",
      "  112  19896 177S 178D C   N    1348 1350  165.53  -96.50  100.81    4.14  -63.30 -138.93   23.83\n",
      "  112        178D 178D N   CA   1350 1351  137.96  114.20                  -40.00\n",
      "  113  19899 180C 181A C   N    1371 1373  -72.10  -62.50   54.83    9.49  -62.50   54.83    9.49\n",
      "  113        181A 181A N   CA   1373 1374  -94.88  -40.90                  -40.90\n",
      "  114  19900 181A 182L C   N    1376 1378 -108.54 -108.50   47.42    2.56  -70.70   68.02    6.53\n",
      "  114        182L 182L N   CA   1378 1379   85.09  132.50                  141.60\n",
      "  115  19901 182L 183D C   N    1384 1386  -95.73  -70.90   48.76    1.64  -63.30  131.79   18.35\n",
      "  115        183D 183D N   CA   1386 1387 -167.74  150.30                  -40.00\n",
      "  116  19904 185C 186E C   N    1402 1404 -120.05 -117.80   38.51    1.97  -63.60  149.71   17.57\n",
      "  116        186E 186E N   CA   1404 1405   98.36  136.80                  -40.30\n",
      "  117  19906 187C 188R C   N    1417 1419 -132.73 -125.20   40.35    2.09  -63.00  158.24   18.01\n",
      "  117        188R 188R N   CA   1419 1420  100.95  140.60                  -41.10\n",
      "  118  19924 205E 206E C   N    1557 1559    6.10   54.60   82.08    4.37  -63.60  164.42   25.83\n",
      "  118        206E 206E N   CA   1559 1560  108.62   42.40                  -40.30\n",
      "  119  19925 206E 207A C   N    1566 1568   62.20  -68.20  131.05   10.65  -68.20  131.05   10.65\n",
      "  119        207A 207A N   CA   1568 1569  132.27  145.30                  145.30\n",
      "  120  19927 208G 209G C   N    1575 1577  115.16   78.70   59.32    1.08  -80.20  166.84   11.80\n",
      "  120        209G 209G N   CA   1577 1578  147.11 -166.10                  174.10\n",
      "  121  19948 229G 230T C   N    1737 1739 -107.44  -63.20   59.59    6.66  -63.20   59.59    6.66\n",
      "  121        230T 230T N   CA   1739 1740   -2.18  -42.10                  -42.10\n",
      "  122  19977 258H 259F C   N    1967 1969 -115.16  -63.20   52.31    8.07  -63.20   52.31    8.07\n",
      "  122        259F 259F N   CA   1969 1970  -38.27  -44.30                  -44.30\n",
      "  123  19981 262D 263I C   N    1998 2000  128.19 -120.60  113.35    7.10 -120.60  113.35    7.10\n",
      "  123        263I 263I N   CA   2000 2001  152.20  130.30                  130.30\n",
      "  124  19982 263I 264Q C   N    2006 2008 -161.45 -121.10   56.19    1.82  -63.80  171.42   30.09\n",
      "  124        264Q 264Q N   CA   2008 2009  178.81  139.70                  -40.30\n",
      "  125  19983 264Q 265T C   N    2015 2017 -175.16 -124.80   72.59    2.33  -63.20  165.67   27.93\n",
      "  125        265T 265T N   CA   2017 2018 -164.22  143.50                  -42.10\n",
      "  126  19984 265T 266A C   N    2022 2024 -176.98 -134.00   54.47    1.42  -62.50  179.80   35.42\n",
      "  126        266A 266A N   CA   2024 2025 -179.54  147.00                  -40.90\n",
      "  127  19985 266A 267K C   N    2027 2029 -164.11 -118.00   60.40    2.07  -62.90  173.63   28.51\n",
      "  127        267K 267K N   CA   2029 2030  178.12  139.10                  -40.80\n",
      "  128  19986 267K 268K C   N    2036 2038 -172.49 -118.00   66.14    2.22  -62.90  179.86   29.78\n",
      "  128        268K 268K N   CA   2038 2039  176.59  139.10                  -40.80\n",
      "  129  19987 268K 269T C   N    2045 2047 -162.07 -124.80   54.07    1.73  -63.20  167.51   27.63\n",
      "  129        269T 269T N   CA   2047 2048 -177.32  143.50                  -42.10\n",
      "  130  19988 269T 270A C   N    2052 2054  178.79 -134.00   54.77    1.28  -62.50 -173.10   36.81\n",
      "  130        270A 270A N   CA   2054 2055  174.75  147.00                  -40.90\n",
      "  131  19989 270A 271K C   N    2057 2059 -165.68 -118.00   58.01    1.94  -62.90  179.41   29.37\n",
      "  131        271K 271K N   CA   2059 2060  172.15  139.10                  -40.80\n",
      "  132  19990 271K 272T C   N    2066 2068 -168.34 -124.80   55.66    1.92  -63.20  174.87   28.93\n",
      "  132        272T 272T N   CA   2068 2069  178.17  143.50                  -42.10\n",
      "  133  19991 272T 273L C   N    2073 2075  177.61 -108.50   81.62    3.61  -63.50 -167.31   33.91\n",
      "  133        273L 273L N   CA   2075 2076  167.17  132.50                  -41.20\n",
      "  134  19992 273L 274G C   N    2081 2083  172.01 -167.20   22.14    0.62   82.20 -177.83   12.86\n",
      "  134        274G 274G N   CA   2083 2084  167.00  174.60                    8.50\n",
      "  135  19993 274G 275E C   N    2085 2087 -175.38 -117.80   65.53    1.92  -63.60 -171.63   31.13\n",
      "  135        275E 275E N   CA   2087 2088  168.08  136.80                  -40.30\n",
      "  136  19994 275E 276M C   N    2094 2096  175.11 -125.60   66.89    2.02  -63.40 -168.49   37.58\n",
      "  136        276M 276M N   CA   2096 2097  171.46  140.50                  -40.50\n",
      "  137  19995 276M 277Q C   N    2102 2104  176.43 -121.10   65.94    2.11  -63.80 -161.01   35.17\n",
      "  137        277Q 277Q N   CA   2104 2105  160.79  139.70                  -40.30\n",
      "  138  19996 277Q 278Y C   N    2111 2113 -169.89 -124.30   58.38    2.82  -63.50  179.63   33.38\n",
      "  138        278Y 278Y N   CA   2113 2114  171.86  135.40                  -43.40\n",
      "  139  19997 278Y 279D C   N    2123 2125 -175.27  -96.50   90.16    3.74  -63.30 -163.11   31.39\n",
      "  139        279D 279D N   CA   2125 2126  158.05  114.20                  -40.00\n",
      "  140  19998 279D 280Y C   N    2131 2133 -162.85 -124.30   56.54    2.44  -63.50  171.54   31.81\n",
      "  140        280Y 280Y N   CA   2133 2134  176.76  135.40                  -43.40\n",
      "  141  19999 280Y 281S C   N    2143 2145 -168.57 -136.60   39.06    1.19  -64.10 -176.08   20.87\n",
      "  141        281S 281S N   CA   2145 2146  173.63  151.20                  -35.00\n",
      "  142  20001 282C 283Q C   N    2155 2157 -166.45 -121.10   55.45    1.72  -63.80 -179.81   31.63\n",
      "  142        283Q 283Q N   CA   2157 2158  171.61  139.70                  -40.30\n",
      "  143  20002 283Q 284F C   N    2164 2166 -176.92 -124.20   59.81    1.61  -63.20 -176.40   32.29\n",
      "  143        284F 284F N   CA   2166 2167  171.56  143.30                  -44.30\n",
      "  144  20003 284F 285D C   N    2175 2177 -163.61  -96.50   85.84    3.58  -63.30 -177.66   28.88\n",
      "  144        285D 285D N   CA   2177 2178  167.73  114.20                  -40.00\n",
      "  145  20004 285D 286F C   N    2183 2185 -172.12 -124.20   58.16    1.51  -63.20  176.94   31.09\n",
      "  145        286F 286F N   CA   2185 2186  176.26  143.30                  -44.30\n",
      "  146  20005 286F 287K C   N    2194 2196 -151.69 -118.00   47.83    1.69  -62.90  171.00   27.49\n",
      "  146        287K 287K N   CA   2196 2197  173.05  139.10                  -40.80\n",
      "  147  20006 287K 288R C   N    2203 2205 -173.08 -125.20   63.93    1.91  -63.00  174.91   30.73\n",
      "  147        288R 288R N   CA   2205 2206 -177.04  140.60                  -41.10\n",
      "  148  20008 289I 290T C   N    2222 2224 -149.96 -124.80   48.23    1.50  -63.20  159.01   25.88\n",
      "  148        290T 290T N   CA   2224 2225 -175.36  143.50                  -42.10\n",
      "  149  20009 290T 291E C   N    2229 2231 -129.89 -117.80   42.66    1.88  -63.60  156.71   24.61\n",
      "  149        291E 291E N   CA   2231 2232  177.70  136.80                  -40.30\n",
      "  150  20010 291E 292N C   N    2238 2240 -143.31 -119.90   42.56    1.35  -63.20  166.84   25.34\n",
      "  150        292N 292N N   CA   2240 2241  172.55  137.00                  -41.10\n",
      "  151  20011 292N 293G C   N    2246 2248 -169.06 -167.20    8.59    0.35   82.20 -168.44   14.00\n",
      "  151        293G 293G N   CA   2248 2249  166.21  174.60                    8.50\n",
      "  152  20012 293G 294D C   N    2250 2252 -152.36  -96.50   72.88    3.04  -63.30 -177.77   28.21\n",
      "  152        294D 294D N   CA   2252 2253  161.01  114.20                  -40.00\n",
      "  153  20013 294D 295S C   N    2258 2260 -166.53 -136.60   32.49    1.02  -64.10 -169.05   21.23\n",
      "  153        295S 295S N   CA   2260 2261  163.84  151.20                  -35.00\n",
      "  154  20014 295S 296L C   N    2264 2266 -164.96 -108.50   58.15    2.63  -63.50 -159.98   34.01\n",
      "  154        296L 296L N   CA   2266 2267  146.42  132.50                  -41.20\n",
      "  155  20016 297V 298P C   N    2279 2281  -63.71  -58.70   29.40    2.84  -64.50  153.34   11.48\n",
      "  155        298P 298P N   CA   2281 2282  -59.46  -30.50                  147.20\n",
      "  156  20017 298P 299A C   N    2286 2288   74.37  -68.20  142.83   12.39  -62.50 -145.42   32.61\n",
      "  156        299A 299A N   CA   2288 2289  153.84  145.30                  -40.90\n",
      "  157  20018 299A 300F C   N    2291 2293 -174.01 -124.20   53.13    1.56  -63.20 -170.35   33.08\n",
      "  157        300F 300F N   CA   2293 2294  161.79  143.30                  -44.30\n",
      "  158  20019 300F 301G C   N    2302 2304  177.71 -167.20   48.47    1.99   82.20  153.40   11.46\n",
      "  158        301G 301G N   CA   2304 2305  128.53  174.60                    8.50\n",
      "  159  20021 302P 303G C   N    2313 2315 -174.49 -167.20   21.89    0.90   82.20  178.41   13.10\n",
      "  159        303G 303G N   CA   2315 2316  153.96  174.60                    8.50\n",
      "  160  20022 303G 304R C   N    2317 2319 -158.14 -125.20   42.18    1.25  -63.00  179.28   30.56\n",
      "  160        304R 304R N   CA   2319 2320  166.95  140.60                  -41.10\n",
      "  161  20023 304R 305T C   N    2328 2330 -154.75 -124.80   41.84    1.36  -63.20  171.64   27.82\n",
      "  161        305T 305T N   CA   2330 2331  172.71  143.50                  -42.10\n",
      "  162  20024 305T 306G C   N    2335 2337 -170.44 -167.20   13.09    0.54   82.20 -172.75   13.72\n",
      "  162        306G 306G N   CA   2337 2338  161.92  174.60                    8.50\n",
      "  163  20025 306G 307M C   N    2339 2341 -160.07 -125.60   44.47    1.32  -63.40  179.21   34.16\n",
      "  163        307M 307M N   CA   2341 2342  168.60  140.50                  -40.50\n",
      "  164  20026 307M 308H C   N    2347 2349 -151.59 -125.60   36.23    0.89  -63.20  177.27   27.86\n",
      "  164        308H 308H N   CA   2349 2350  164.04  138.80                  -42.30\n",
      "  165  20027 308H 309N C   N    2357 2359 -152.97 -119.90   40.64    1.19  -63.20 -178.03   27.76\n",
      "  165        309N 309N N   CA   2359 2360  160.62  137.00                  -41.10\n",
      "  166  20028 309N 310F C   N    2365 2367 -157.98 -124.20   39.45    1.04  -63.20  179.14   30.79\n",
      "  166        310F 310F N   CA   2367 2368  163.68  143.30                  -44.30\n",
      "  167  20029 310F 311G C   N    2376 2378 -179.96 -167.20   18.89    0.72   82.20 -179.09   13.06\n",
      "  167        311G 311G N   CA   2378 2379  160.67  174.60                    8.50\n",
      "  168  20030 311G 312N C   N    2380 2382 -157.84 -119.90   48.74    1.41  -63.20  178.47   27.55\n",
      "  168        312N 312N N   CA   2382 2383  167.59  137.00                  -41.10\n",
      "  169  20031 312N 313S C   N    2388 2390 -158.87 -136.60   27.34    0.84  -64.10 -175.81   20.18\n",
      "  169        313S 313S N   CA   2390 2391  167.06  151.20                  -35.00\n",
      "  170  20033 314C 315Y C   N    2400 2402 -143.19 -124.30   28.74    1.21  -63.50  178.33   32.19\n",
      "  170        315Y 315Y N   CA   2402 2403  157.06  135.40                  -43.40\n",
      "  171  20034 315Y 316M C   N    2412 2414 -153.77 -125.60   29.75    0.96  -63.40 -167.97   35.71\n",
      "  171        316M 316M N   CA   2414 2415  150.06  140.50                  -40.50\n",
      "  172  20035 316M 317A C   N    2420 2422 -159.43 -134.00   28.72    0.66  -62.50 -173.98   35.64\n",
      "  172        317A 317A N   CA   2422 2423  160.33  147.00                  -40.90\n",
      "  173  20036 317A 318S C   N    2425 2427 -159.01 -136.60   25.98    0.79  -64.10 -173.41   20.36\n",
      "  173        318S 318S N   CA   2427 2428  164.34  151.20                  -35.00\n",
      "  174  20038 319V 320L C   N    2438 2440 -150.65 -108.50   44.71    1.99  -63.50 -167.74   32.08\n",
      "  174        320L 320L N   CA   2440 2441  147.42  132.50                  -41.20\n",
      "  175  20039 320L 321Q C   N    2446 2448 -150.68 -121.10   34.79    1.07  -63.80 -176.45   31.48\n",
      "  175        321Q 321Q N   CA   2448 2449  158.02  139.70                  -40.30\n",
      "  176  20041 322C 323L C   N    2461 2463 -151.05 -108.50   43.97    1.98  -63.50 -164.14   32.61\n",
      "  176        323L 323L N   CA   2463 2464  143.59  132.50                  -41.20\n",
      "  177  20042 323L 324F C   N    2469 2471 -145.47 -124.20   24.19    0.65  -63.20 -179.30   30.37\n",
      "  177        324F 324F N   CA   2471 2472  154.82  143.30                  -44.30\n",
      "  178  20043 324F 325S C   N    2480 2482 -157.96 -136.60   22.40    0.73  -64.10 -168.38   20.64\n",
      "  178        325S 325S N   CA   2482 2483  157.93  151.20                  -35.00\n",
      "  179  20044 325S 326L C   N    2486 2488 -152.35 -108.50   48.45    2.14  -63.50 -171.99   31.59\n",
      "  179        326L 326L N   CA   2488 2489  153.11  132.50                  -41.20\n",
      "  180  20045 326L 327E C   N    2494 2496 -135.34 -117.80   18.63    0.84  -63.60 -174.72   21.70\n",
      "  180        327E 327E N   CA   2496 2497  130.52  136.80                  -40.30\n",
      "  181  20046 327E 328P C   N    2503 2505  -66.10  -58.70   14.92    1.82  -64.50  169.36   12.55\n",
      "  181        328P 328P N   CA   2505 2506  -43.45  -30.50                  147.20\n",
      "  182  20047 328P 329F C   N    2510 2512   62.57  -71.40  134.37    9.41  -63.20 -152.74   23.42\n",
      "  182        329F 329F N   CA   2512 2513  150.96  140.70                  -44.30\n",
      "  183  20048 329F 330K C   N    2521 2523 -159.55 -118.00   44.27    1.48  -62.90 -168.92   30.57\n",
      "  183        330K 330K N   CA   2523 2524  154.37  139.10                  -40.80\n",
      "  184  20049 330K 331E C   N    2530 2532 -162.28 -117.80   49.42    1.45  -63.60 -170.85   30.64\n",
      "  184        331E 331E N   CA   2532 2533  158.33  136.80                  -40.30\n",
      "  185  20050 331E 332A C   N    2539 2541 -161.23 -134.00   29.02    0.67  -62.50 -170.20   36.36\n",
      "  185        332A 332A N   CA   2541 2542  157.01  147.00                  -40.90\n",
      "  186  20051 332A 333F C   N    2544 2546 -156.24 -124.20   33.94    1.01  -63.20 -173.87   31.72\n",
      "  186        333F 333F N   CA   2546 2547  154.49  143.30                  -44.30\n",
      "  187  20052 333F 334Y C   N    2555 2557 -139.02 -124.30   18.19    0.91  -63.50 -173.52   33.33\n",
      "  187        334Y 334Y N   CA   2557 2558  146.09  135.40                  -43.40\n",
      "  188  20053 334Y 335Y C   N    2567 2569 -141.67 -124.30   23.21    1.08  -63.50 -176.68   32.93\n",
      "  188        335Y 335Y N   CA   2569 2570  150.78  135.40                  -43.40\n",
      "  189  20054 335Y 336N C   N    2579 2581 -157.20 -119.90   43.32    1.30  -63.20 -174.54   28.42\n",
      "  189        336N 336N N   CA   2581 2582  159.03  137.00                  -41.10\n",
      "  190  20056 337C 338E C   N    2593 2595 -156.69 -117.80   41.12    1.24  -63.60 -166.58   30.95\n",
      "  190        338E 338E N   CA   2595 2596  150.16  136.80                  -40.30\n",
      "  191  20057 338E 339T C   N    2602 2604 -149.25 -124.80   28.44    1.08  -63.20 -178.43   28.84\n",
      "  191        339T 339T N   CA   2604 2605  158.02  143.50                  -42.10\n",
      "  192  20058 339T 340R C   N    2609 2611 -156.34 -125.20   36.60    1.09  -63.00 -175.56   31.18\n",
      "  192        340R 340R N   CA   2611 2612  159.83  140.60                  -41.10\n",
      "  193  20059 340R 341H C   N    2620 2622 -156.72 -125.60   36.70    0.93  -63.20 -175.15   29.13\n",
      "  193        341H 341H N   CA   2622 2623  158.25  138.80                  -42.30\n",
      "  194  20060 341H 342Q C   N    2630 2632 -155.60 -121.10   38.99    1.20  -63.80 -173.95   32.07\n",
      "  194        342Q 342Q N   CA   2632 2633  157.87  139.70                  -40.30\n",
      "  195  20061 342Q 343N C   N    2639 2641 -151.32 -119.90   38.31    1.12  -63.20 -177.34   27.76\n",
      "  195        343N 343N N   CA   2641 2642  158.91  137.00                  -41.10\n",
      "  196  20062 343N 344A C   N    2647 2649 -161.19 -134.00   29.29    0.67  -62.50 -170.96   36.23\n",
      "  196        344A 344A N   CA   2649 2650  157.87  147.00                  -40.90\n",
      "  197  20064 345C 346Q C   N    2658 2660 -156.07 -121.10   39.24    1.21  -63.80 -173.40   32.18\n",
      "  197        346Q 346Q N   CA   2660 2661  157.50  139.70                  -40.30\n",
      "  198  20065 346Q 347E C   N    2667 2669 -154.18 -117.80   40.56    1.19  -63.60 -171.79   30.12\n",
      "  198        347E 347E N   CA   2669 2670  154.72  136.80                  -40.30\n",
      "  199  20066 347E 348N C   N    2676 2678 -133.21 -119.90   13.72    0.65  -63.20 -171.70   20.57\n",
      "  199        348N 348N N   CA   2678 2679  133.70  137.00                  -41.10\n",
      "  200  20067 348N 349P C   N    2684 2686  -65.68  -58.70   15.65    1.86  -64.50  168.30   12.50\n",
      "  200        349P 349P N   CA   2686 2687  -44.50  -30.50                  147.20\n",
      "  201  20068 349P 350Y C   N    2691 2693   60.62  -98.40  159.09    6.10  -63.50 -144.30   39.97\n",
      "  201        350Y 350Y N   CA   2693 2694  133.00  128.40                  -43.40\n",
      "  202  20069 350Y 351K C   N    2703 2705 -146.17 -118.00   32.40    1.07  -62.90 -176.00   28.88\n",
      "  202        351K 351K N   CA   2705 2706  155.12  139.10                  -40.80\n",
      "  203  20071 352C 353H C   N    2718 2720 -154.41 -125.60   34.99    0.87  -63.20 -176.65   28.81\n",
      "  203        353H 353H N   CA   2720 2721  158.65  138.80                  -42.30\n",
      "  204  20074 355C 356Q C   N    2740 2742 -158.85 -121.10   39.47    1.28  -63.80 -166.56   33.33\n",
      "  204        356Q 356Q N   CA   2742 2743  151.23  139.70                  -40.30\n",
      "  205  20075 356Q 357T C   N    2749 2751 -149.67 -124.80   32.46    1.10  -63.20  176.20   28.15\n",
      "  205        357T 357T N   CA   2751 2752  164.37  143.50                  -42.10\n",
      "  206  20076 357T 358E C   N    2756 2758 -155.41 -117.80   43.98    1.29  -63.60 -175.44   29.68\n",
      "  206        358E 358E N   CA   2758 2759  159.59  136.80                  -40.30\n",
      "  207  20077 358E 359R C   N    2765 2767 -150.23 -125.20   29.94    0.89  -63.00 -176.12   30.74\n",
      "  207        359R 359R N   CA   2767 2768  157.03  140.60                  -41.10\n",
      "  208  20079 360V 361A C   N    2783 2785 -160.10 -134.00   28.74    0.65  -62.50 -172.52   35.92\n",
      "  208        361A 361A N   CA   2785 2786  159.03  147.00                  -40.90\n",
      "  209  20080 361A 362S C   N    2788 2790 -164.64 -136.60   34.21    1.05  -64.10 -175.92   20.59\n",
      "  209        362S 362S N   CA   2790 2791  170.80  151.20                  -35.00\n",
      "  210  20081 362S 363G C   N    2794 2796  176.66 -167.20   28.58    1.14   82.20  170.97   12.41\n",
      "  210        363G 363G N   CA   2796 2797  151.01  174.60                    8.50\n",
      "  211  20082 363G 364L C   N    2798 2800 -139.75 -108.50   36.95    1.63  -63.50 -176.81   30.16\n",
      "  211        364L 364L N   CA   2800 2801  152.23  132.50                  -41.20\n",
      "  212  20083 364L 365L C   N    2806 2808 -138.92 -108.50   31.46    1.42  -63.50 -166.45   31.55\n",
      "  212        365L 365L N   CA   2808 2809  140.54  132.50                  -41.20\n",
      "  213  20084 365L 366S C   N    2814 2816 -153.10 -136.60   23.13    0.74  -64.10 -179.02   19.52\n",
      "  213        366S 366S N   CA   2816 2817  167.41  151.20                  -35.00\n",
      "  214  20085 366S 367G C   N    2820 2822  173.77 -167.20   26.69    1.00   82.20  173.51   12.45\n",
      "  214        367G 367G N   CA   2822 2823  155.89  174.60                    8.50\n",
      "  215  20086 367G 368E C   N    2824 2826 -157.15 -117.80   42.96    1.27  -63.60 -169.75   30.54\n",
      "  215        368E 368E N   CA   2826 2827  154.03  136.80                  -40.30\n",
      "  216  20087 368E 369F C   N    2833 2835 -147.08 -124.20   26.65    0.70  -63.20  179.55   30.29\n",
      "  216        369F 369F N   CA   2835 2836  156.95  143.30                  -44.30\n",
      "  217  20088 369F 370S C   N    2844 2846 -153.09 -136.60   19.34    0.59  -64.10 -173.69   19.90\n",
      "  217        370S 370S N   CA   2846 2847  161.31  151.20                  -35.00\n",
      "  218  20090 371V 372E C   N    2857 2859 -158.44 -117.80   49.83    1.48  -63.60 -179.08   29.33\n",
      "  218        372E 372E N   CA   2859 2860  165.63  136.80                  -40.30\n",
      "  219  20091 372E 373G C   N    2866 2868  177.99 -167.20   24.31    0.95   82.20  175.30   12.69\n",
      "  219        373G 373G N   CA   2868 2869  155.32  174.60                    8.50\n",
      "  220  20092 373G 374Q C   N    2870 2872 -157.02 -121.10   41.04    1.27  -63.80 -174.71   32.02\n",
      "  220        374Q 374Q N   CA   2872 2873  159.56  139.70                  -40.30\n",
      "  221  20093 374Q 375E C   N    2879 2881 -146.76 -117.80   32.08    0.94  -63.60 -171.58   29.77\n",
      "  221        375E 375E N   CA   2881 2882  150.62  136.80                  -40.30\n",
      "  222  20094 375E 376L C   N    2888 2890 -143.17 -108.50   37.06    1.65  -63.50 -169.35   31.40\n",
      "  222        376L 376L N   CA   2890 2891  145.60  132.50                  -41.20\n",
      "  223  20095 376L 377T C   N    2896 2898 -140.08 -124.80   19.56    0.67  -63.20  179.50   28.03\n",
      "  223        377T 377T N   CA   2898 2899  155.70  143.50                  -42.10\n",
      "  224  20096 377T 378N C   N    2903 2905 -157.91 -119.90   46.22    1.36  -63.20 -177.83   28.03\n",
      "  224        378N 378N N   CA   2905 2906  163.29  137.00                  -41.10\n",
      "  225  20097 378N 379G C   N    2911 2913  178.21 -167.20   34.47    1.41   82.20  165.54   12.16\n",
      "  225        379G 379G N   CA   2913 2914  143.37  174.60                    8.50\n",
      "  226  20099 380I 381T C   N    2923 2925 -142.57 -124.80   23.39    0.79  -63.20  177.87   27.97\n",
      "  226        381T 381T N   CA   2925 2926  158.72  143.50                  -42.10\n",
      "  227  20100 381T 382A C   N    2930 2932 -161.84 -134.00   30.42    0.69  -62.50 -171.79   36.13\n",
      "  227        382A 382A N   CA   2932 2933  159.25  147.00                  -40.90\n",
      "  228  20101 382A 383R C   N    2935 2937 -154.43 -125.20   36.16    1.07  -63.00 -178.31   30.69\n",
      "  228        383R 383R N   CA   2937 2938  161.89  140.60                  -41.10\n",
      "  229  20102 383R 384E C   N    2946 2948 -148.45 -117.80   35.58    1.04  -63.60 -174.60   29.45\n",
      "  229        384E 384E N   CA   2948 2949  154.86  136.80                  -40.30\n",
      "  230  20103 384E 385F C   N    2955 2957 -144.81 -124.20   24.96    0.65  -63.20  178.12   29.96\n",
      "  230        385F 385F N   CA   2957 2958  157.38  143.30                  -44.30\n",
      "  231  20104 385F 386K C   N    2966 2968 -146.42 -118.00   32.52    1.08  -62.90 -175.70   28.93\n",
      "  231        386K 386K N   CA   2968 2969  154.91  139.10                  -40.80\n",
      "  232  20105 386K 387R C   N    2975 2977 -142.38 -125.20   29.14    0.96  -63.00  173.94   28.88\n",
      "  232        387R 387R N   CA   2977 2978  164.14  140.60                  -41.10\n",
      "  233  20107 388V 389F C   N    2993 2995 -109.10 -124.20   15.11    0.58  -63.20  178.89   27.91\n",
      "  233        389F 389F N   CA   2995 2996  142.80  143.30                  -44.30\n",
      "  234  20109 390A 391Q C   N    3009 3011  167.02 -121.10   78.76    2.46  -63.80 -163.71   35.06\n",
      "  234        391Q 391Q N   CA   3011 3012  171.91  139.70                  -40.30\n",
      "  235  20110 391Q 392K C   N    3018 3020 -127.04 -118.00   25.84    1.10  -62.90  168.57   25.73\n",
      "  235        392K 392K N   CA   3020 3021  163.31  139.10                  -40.80\n",
      "  236  20111 392K 393H C   N    3027 3029 -127.19 -125.60    5.18    0.16  -63.20 -174.63   27.36\n",
      "  236        393H 393H N   CA   3029 3030  143.73  138.80                  -42.30\n",
      "  237  20113 394P 395D C   N    3044 3046 -124.49  -96.50   47.61    2.00  -63.30  178.12   25.94\n",
      "  237        395D 395D N   CA   3046 3047  152.72  114.20                  -40.00\n",
      "  238  20114 395D 396F C   N    3052 3054 -132.01 -124.20    9.51    0.25  -63.20 -179.41   29.58\n",
      "  238        396F 396F N   CA   3054 3055  148.73  143.30                  -44.30\n",
      "  239  20115 396F 397S C   N    3063 3065 -151.27 -136.60   16.72    0.51  -64.10 -172.69   19.83\n",
      "  239        397S 397S N   CA   3065 3066  159.21  151.20                  -35.00\n",
      "  240  20116 397S 398T C   N    3069 3071 -142.22 -124.80   24.70    0.80  -63.20  175.67   27.66\n",
      "  240        398T 398T N   CA   3071 3072  161.01  143.50                  -42.10\n",
      "  241  20117 398T 399A C   N    3076 3078 -154.50 -134.00   22.48    0.51  -62.50 -172.94   35.57\n",
      "  241        399A 399A N   CA   3078 3079  156.23  147.00                  -40.90\n",
      "  242  20118 399A 400E C   N    3081 3083 -148.16 -117.80   33.71    0.99  -63.60 -171.69   29.83\n",
      "  242        400E 400E N   CA   3083 3084  151.44  136.80                  -40.30\n",
      "  243  20119 400E 401Q C   N    3090 3092 -150.88 -121.10   32.68    1.02  -63.80 -172.07   32.14\n",
      "  243        401Q 401Q N   CA   3092 3093  153.16  139.70                  -40.30\n",
      "  244  20120 401Q 402Q C   N    3099 3101 -145.84 -121.10   25.21    0.85  -63.80 -166.63   32.72\n",
      "  244        402Q 402Q N   CA   3101 3102  144.59  139.70                  -40.30\n",
      "  245  20121 402Q 403D C   N    3108 3110 -148.77  -96.50   55.31    2.28  -63.30 -167.69   20.42\n",
      "  245        403D 403D N   CA   3110 3111  132.27  114.20                  -40.00\n",
      "  246  20122 403D 404A C   N    3116 3118 -148.11 -134.00   14.32    0.39  -62.50 -169.97   35.73\n",
      "  246        404A 404A N   CA   3118 3119  149.44  147.00                  -40.90\n",
      "  247  20123 404A 405Q C   N    3121 3123 -159.99 -121.10   40.70    1.32  -63.80 -166.42   33.40\n",
      "  247        405Q 405Q N   CA   3123 3124  151.71  139.70                  -40.30\n",
      "  248  20124 405Q 406E C   N    3130 3132 -152.08 -117.80   35.27    1.11  -63.60 -164.27   31.03\n",
      "  248        406E 406E N   CA   3132 3133  145.11  136.80                  -40.30\n",
      "  249  20125 406E 407Y C   N    3139 3141 -141.18 -124.30   26.57    1.10  -63.50  178.46   32.13\n",
      "  249        407Y 407Y N   CA   3141 3142  155.93  135.40                  -43.40\n",
      "  250  20126 407Y 408F C   N    3151 3153 -142.65 -124.20   20.32    0.56  -63.20 -177.87   30.41\n",
      "  250        408F 408F N   CA   3153 3154  151.82  143.30                  -44.30\n",
      "  251  20127 408F 409L C   N    3162 3164 -131.99 -108.50   24.11    1.09  -63.50 -168.22   23.74\n",
      "  251        409L 409L N   CA   3164 3165  137.93  132.50                  -41.20\n",
      "  252  20128 409L 410Y C   N    3170 3172 -132.68 -124.30    8.59    0.69  -63.50 -170.03   27.05\n",
      "  252        410Y 410Y N   CA   3172 3173  133.52  135.40                  -43.40\n",
      "  253  20129 410Y 411L C   N    3182 3184 -140.64 -108.50   34.89    1.55  -63.50 -170.85   31.04\n",
      "  253        411L 411L N   CA   3184 3185  146.10  132.50                  -41.20\n",
      "  254  20131 412V 413E C   N    3197 3199 -155.19 -117.80   39.22    1.20  -63.60 -165.98   30.96\n",
      "  254        413E 413E N   CA   3199 3200  148.66  136.80                  -40.30\n",
      "  255  20132 413E 414Q C   N    3206 3208 -155.28 -121.10   35.66    1.16  -63.80 -167.07   33.09\n",
      "  255        414Q 414Q N   CA   3208 3209  149.84  139.70                  -40.30\n",
      "  256  20133 414Q 415M C   N    3215 3217 -154.19 -125.60   30.32    0.97  -63.40 -168.23   35.70\n",
      "  256        415M 415M N   CA   3217 3218  150.58  140.50                  -40.50\n",
      "  257  20134 415M 416R C   N    3223 3225 -153.48 -125.20   30.22    0.96  -63.00 -169.50   31.85\n",
      "  257        416R 416R N   CA   3225 3226  151.26  140.60                  -41.10\n",
      "  258  20135 416R 417R C   N    3234 3236 -145.98 -125.20   20.92    0.78  -63.00 -165.54   31.93\n",
      "  258        417R 417R N   CA   3236 3237  143.04  140.60                  -41.10\n",
      "  259  20136 417R 418Y C   N    3245 3247 -139.82 -124.30   19.78    0.96  -63.50 -174.63   33.18\n",
      "  259        418Y 418Y N   CA   3247 3248  147.66  135.40                  -43.40\n",
      "  260  20138 419V 420K C   N    3264 3266 -142.37 -118.00   24.77    1.02  -62.90 -167.37   22.17\n",
      "  260        420K 420K N   CA   3266 3267  134.67  139.10                  -40.80\n",
      "  261  20140 421P 422S C   N    3280 3282 -147.64 -136.60   11.07    0.42  -64.10 -167.94   19.89\n",
      "  261        422S 422S N   CA   3282 3283  152.06  151.20                  -35.00\n",
      "  262  20141 422S 423N C   N    3286 3288 -141.34 -119.90   23.67    0.74  -63.20 -171.18   28.02\n",
      "  262        423N 423N N   CA   3288 3289  147.01  137.00                  -41.10\n",
      "  263  20144 425V 426G C   N    3309 3311  172.70 -167.20   27.11    0.99   82.20  173.39   12.41\n",
      "  263        426G 426G N   CA   3311 3312  156.41  174.60                    8.50\n",
      "  264  20145 426G 427A C   N    3313 3315 -156.94 -134.00   24.93    0.57  -62.50 -172.18   35.82\n",
      "  264        427A 427A N   CA   3315 3316  156.75  147.00                  -40.90\n",
      "  265  20146 427A 428N C   N    3318 3320 -139.97 -119.90   23.50    0.70  -63.20 -173.77   27.62\n",
      "  265        428N 428N N   CA   3320 3321  149.23  137.00                  -41.10\n",
      "  266  20148 429I 430S C   N    3334 3336 -142.52 -136.60    7.00    0.21  -64.10 -172.72   19.17\n",
      "  266        430S 430S N   CA   3336 3337  154.92  151.20                  -35.00\n",
      "  267  20149 430S 431H C   N    3340 3342 -138.10 -125.60   13.40    0.56  -63.20 -168.47   20.55\n",
      "  267        431H 431H N   CA   3342 3343  133.98  138.80                  -42.30\n",
      "  268  20152 433V 434D C   N    3364 3366 -125.49  -96.50   33.53    1.39  -63.30 -178.01   19.61\n",
      "  268        434D 434D N   CA   3366 3367  131.04  114.20                  -40.00\n",
      "  269  20154 435I 436F C   N    3380 3382 -134.60 -124.20   10.63    0.35  -63.20 -175.42   30.30\n",
      "  269        436F 436F N   CA   3382 3383  145.49  143.30                  -44.30\n",
      "  270  20155 436F 437K C   N    3391 3393 -151.19 -118.00   35.43    1.18  -62.90 -170.48   29.89\n",
      "  270        437K 437K N   CA   3393 3394  151.50  139.10                  -40.80\n",
      "  271  20156 437K 438M C   N    3400 3402 -143.78 -125.60   19.10    0.62  -63.40 -169.08   34.83\n",
      "  271        438M 438M N   CA   3402 3403  146.33  140.50                  -40.50\n",
      "  272  20157 438M 439T C   N    3408 3410 -140.74 -124.80   19.62    0.70  -63.20 -179.53   28.20\n",
      "  272        439T 439T N   CA   3410 3411  154.93  143.50                  -42.10\n",
      "  273  20159 440V 441E C   N    3422 3424 -147.90 -117.80   32.32    0.96  -63.60 -169.25   30.14\n",
      "  273        441E 441E N   CA   3424 3425  148.58  136.80                  -40.30\n",
      "  274  20160 441E 442H C   N    3431 3433 -148.58 -125.60   24.73    0.68  -63.20 -169.99   29.30\n",
      "  274        442H 442H N   CA   3433 3434  147.95  138.80                  -42.30\n",
      "  275  20161 442H 443R C   N    3441 3443 -147.54 -125.20   24.58    0.76  -63.00 -171.89   31.16\n",
      "  275        443R 443R N   CA   3443 3444  150.85  140.60                  -41.10\n",
      "  276  20163 444V 445Q C   N    3459 3461 -144.80 -121.10   25.01    0.80  -63.80 -169.88   32.19\n",
      "  276        445Q 445Q N   CA   3461 3462  147.69  139.70                  -40.30\n",
      "  277  20165 446C 447G C   N    3474 3476  168.03 -167.20   33.42    1.22   82.20  167.35   11.93\n",
      "  277        447G 447G N   CA   3476 3477  152.16  174.60                    8.50\n",
      "  278  20166 447G 448S C   N    3478 3480 -156.52 -136.60   21.29    0.68  -64.10 -169.77   20.43\n",
      "  278        448S 448S N   CA   3480 3481  158.72  151.20                  -35.00\n",
      "  279  20168 449C 450H C   N    3490 3492 -145.98 -125.60   22.23    0.60  -63.20 -170.89   29.03\n",
      "  279        450H 450H N   CA   3492 3493  147.67  138.80                  -42.30\n",
      "  280  20169 450H 451K C   N    3500 3502 -143.24 -118.00   26.55    0.89  -62.90 -170.28   29.45\n",
      "  280        451K 451K N   CA   3502 3503  147.33  139.10                  -40.80\n",
      "  281  20171 452V 453R C   N    3516 3518 -142.30 -125.20   17.22    0.64  -63.00 -166.73   31.53\n",
      "  281        453R 453R N   CA   3518 3519  142.64  140.60                  -41.10\n",
      "  282  20172 453R 454Y C   N    3527 3529 -132.72 -124.30    9.83    0.53  -63.50 -170.76   33.49\n",
      "  282        454Y 454Y N   CA   3529 3530  140.48  135.40                  -43.40\n",
      "  283  20173 454Y 455T C   N    3539 3541 -135.12 -124.80   10.56    0.50  -63.20 -173.43   28.65\n",
      "  283        455T 455T N   CA   3541 3542  145.75  143.50                  -42.10\n",
      "  284  20174 455T 456R C   N    3546 3548 -144.76 -125.20   20.27    0.68  -63.00 -168.66   31.43\n",
      "  284        456R 456R N   CA   3548 3549  145.91  140.60                  -41.10\n",
      "  285  20175 456R 457E C   N    3557 3559 -143.97 -117.80   26.95    0.85  -63.60 -166.08   30.36\n",
      "  285        457E 457E N   CA   3559 3560  143.22  136.80                  -40.30\n",
      "  286  20176 457E 458T C   N    3566 3568 -131.81 -124.80    7.61    0.45  -63.20 -169.82   28.92\n",
      "  286        458T 458T N   CA   3568 3569  140.53  143.50                  -42.10\n",
      "  287  20177 458T 459D C   N    3573 3575 -130.62  -96.50   34.17    1.40  -63.30  169.92   18.12\n",
      "  287        459D 459D N   CA   3575 3576  116.01  114.20                  -40.00\n",
      "  288  20180 461C 462L C   N    3593 3595 -129.10 -108.50   22.00    0.98  -63.50 -169.76   30.47\n",
      "  288        462L 462L N   CA   3595 3596  140.22  132.50                  -41.20\n",
      "  289  20181 462L 463S C   N    3601 3603 -135.26 -136.60    6.77    0.31  -64.10 -166.85   11.95\n",
      "  289        463S 463S N   CA   3603 3604  144.56  151.20                  -35.00\n",
      "  290  20182 463S 464L C   N    3607 3609 -125.83 -108.50   19.20    0.85  -63.50 -171.38   30.04\n",
      "  290        464L 464L N   CA   3609 3610  140.77  132.50                  -41.20\n",
      "  291  20186 467P 468L C   N    3637 3639 -115.01 -108.50    6.50    0.31  -63.50 -178.78   22.78\n",
      "  291        468L 468L N   CA   3639 3640  132.55  132.50                  -41.20\n",
      "  292  20187 468L 469D C   N    3645 3647 -114.78  -96.50   20.23    0.84  -63.30  170.79   18.60\n",
      "  292        469D 469D N   CA   3647 3648  122.85  114.20                  -40.00\n",
      "  293  20189 470P 471N C   N    3660 3662 -138.40 -119.90   19.12    0.67  -63.20 -167.64   28.31\n",
      "  293        471N 471N N   CA   3662 3663  141.85  137.00                  -41.10\n",
      "  294  20190 471N 472L C   N    3668 3670 -112.01 -108.50    4.33    0.24  -63.50  177.90   22.43\n",
      "  294        472L 472L N   CA   3670 3671  129.96  132.50                  -41.20\n",
      "  295  20191 472L 473K C   N    3676 3678 -132.76 -118.00   14.82    0.58  -62.90 -168.24   22.18\n",
      "  295        473K 473K N   CA   3678 3679  137.78  139.10                  -40.80\n",
      "  296  20192 473K 474N C   N    3685 3687 -139.29 -119.90   21.96    0.67  -63.20 -172.29   27.77\n",
      "  296        474N 474N N   CA   3687 3688  147.30  137.00                  -41.10\n",
      "  297  20193 474N 475S C   N    3693 3695 -132.96 -136.60   12.82    0.56  -64.10 -172.95   11.57\n",
      "  297        475S 475S N   CA   3695 3696  138.91  151.20                  -35.00\n",
      "  298  20194 475S 476D C   N    3699 3701 -137.44  -96.50   45.05    1.86  -63.30 -171.78   20.08\n",
      "  298        476D 476D N   CA   3701 3702  133.00  114.20                  -40.00\n",
      "  299  20195 476D 477G C   N    3707 3709 -163.60 -167.20   26.94    1.01   82.20 -179.80   13.51\n",
      "  299        477G 477G N   CA   3709 3710  147.90  174.60                    8.50\n",
      "  300  20196 477G 478N C   N    3711 3713 -134.79 -119.90   16.18    0.52  -63.20 -170.38   27.77\n",
      "  300        478N 478N N   CA   3713 3714  143.32  137.00                  -41.10\n",
      "  301  20198 479P 480K C   N    3726 3728 -137.10 -118.00   19.12    0.71  -62.90 -166.11   29.62\n",
      "  301        480K 480K N   CA   3728 3729  140.06  139.10                  -40.80\n",
      "  302  20199 480K 481Q C   N    3735 3737 -126.58 -121.10    5.57    0.23  -63.80 -170.33   25.09\n",
      "  302        481Q 481Q N   CA   3737 3738  138.68  139.70                  -40.30\n",
      "  303  20200 481Q 482T C   N    3744 3746 -131.05 -124.80    9.37    0.30  -63.20 -179.36   27.63\n",
      "  303        482T 482T N   CA   3746 3747  150.48  143.50                  -42.10\n",
      "  304  20201 482T 483E C   N    3751 3753 -134.65 -117.80   17.11    0.56  -63.60 -166.54   29.80\n",
      "  304        483E 483E N   CA   3753 3754  139.75  136.80                  -40.30\n",
      "  305  20202 483E 484E C   N    3760 3762 -140.53 -117.80   24.37    0.73  -63.60 -169.66   29.70\n",
      "  305        484E 484E N   CA   3762 3763  145.59  136.80                  -40.30\n",
      "  306  20203 484E 485E C   N    3769 3771 -134.28 -117.80   16.49    0.59  -63.60 -168.81   22.47\n",
      "  306        485E 485E N   CA   3771 3772  137.34  136.80                  -40.30\n",
      "  307  20205 486I 487F C   N    3786 3788 -138.38 -124.20   14.47    0.48  -63.20 -174.55   30.64\n",
      "  307        487F 487F N   CA   3788 3789  146.17  143.30                  -44.30\n",
      "  308  20206 487F 488A C   N    3797 3799 -159.06 -134.00   25.81    0.65  -62.50 -167.99   36.62\n",
      "  308        488A 488A N   CA   3799 3800  153.14  147.00                  -40.90\n",
      "  309  20207 488A 489S C   N    3802 3804 -159.63 -136.60   23.56    0.81  -64.10 -165.99   20.93\n",
      "  309        489S 489S N   CA   3804 3805  156.13  151.20                  -35.00\n",
      "  310  20208 489S 490R C   N    3808 3810 -132.78 -125.20    9.74    0.53  -63.00 -171.06   21.83\n",
      "  310        490R 490R N   CA   3810 3811  134.48  140.60                  -41.10\n",
      "  311  20209 490R 491P C   N    3819 3821  -66.37  -58.70    8.27    1.16  -64.50  179.21   13.27\n",
      "  311        491P 491P N   CA   3821 3822  -33.60  -30.50                  147.20\n",
      "  312  20210 491P 492H C   N    3826 3828   65.88   56.30  102.20    9.87  -63.20 -142.42   23.42\n",
      "  312        492H 492H N   CA   3828 3829  142.55   40.80                  -42.30\n",
      "  313  20212 493I 494S C   N    3844 3846 -143.58 -136.60    7.62    0.38  -64.10 -166.12   19.71\n",
      "  313        494S 494S N   CA   3846 3847  148.16  151.20                  -35.00\n",
      "  314  20213 494S 495L C   N    3850 3852 -132.11 -108.50   23.93    1.10  -63.50 -169.62   23.55\n",
      "  314        495L 495L N   CA   3852 3853  136.39  132.50                  -41.20\n",
      "  315  20214 495L 496D C   N    3858 3860 -130.35  -96.50   38.31    1.59  -63.30 -175.27   19.82\n",
      "  315        496D 496D N   CA   3860 3861  132.14  114.20                  -40.00\n",
      "  316  20215 496D 497A C   N    3866 3868 -143.40 -134.00    9.67    0.25  -62.50 -171.84   35.17\n",
      "  316        497A 497A N   CA   3868 3869  149.23  147.00                  -40.90\n",
      "  317  20217 498C 499L C   N    3877 3879 -140.84 -108.50   37.73    1.66  -63.50 -176.09   30.32\n",
      "  317        499L 499L N   CA   3879 3880  151.94  132.50                  -41.20\n",
      "  318  20218 499L 500G C   N    3885 3887  173.20 -167.20   32.35    1.27   82.20  167.28   12.10\n",
      "  318        500G 500G N   CA   3887 3888  148.86  174.60                    8.50\n",
      "  319  20219 500G 501S C   N    3889 3891 -156.17 -136.60   19.98    0.69  -64.10 -166.84   20.61\n",
      "  319        501S 501S N   CA   3891 3892  155.20  151.20                  -35.00\n",
      "  320  20220 501S 502L C   N    3895 3897 -136.89 -108.50   30.19    1.34  -63.50 -169.28   31.03\n",
      "  320        502L 502L N   CA   3897 3898  142.76  132.50                  -41.20\n",
      "  321  20221 502L 503M C   N    3903 3905 -141.86 -125.60   16.77    0.57  -63.40 -168.30   34.81\n",
      "  321        503M 503M N   CA   3905 3906  144.59  140.50                  -40.50\n",
      "  322  20222 503M 504Q C   N    3911 3913 -151.75 -121.10   32.00    1.04  -63.80 -167.90   32.81\n",
      "  322        504Q 504Q N   CA   3913 3914  148.91  139.70                  -40.30\n",
      "  323  20223 504Q 505A C   N    3920 3922 -154.92 -134.00   21.38    0.56  -62.50 -168.53   36.32\n",
      "  323        505A 505A N   CA   3922 3923  151.41  147.00                  -40.90\n",
      "  324  20224 505A 506T C   N    3925 3927 -140.08 -124.80   15.77    0.73  -63.20 -172.96   29.02\n",
      "  324        506T 506T N   CA   3927 3928  147.39  143.50                  -42.10\n",
      "  325  20225 506T 507D C   N    3932 3934 -138.98  -96.50   44.50    1.83  -63.30 -176.25   19.56\n",
      "  325        507D 507D N   CA   3934 3935  127.44  114.20                  -40.00\n",
      "  326  20227 508I 509D C   N    3948 3950 -141.20  -96.50   46.26    1.90  -63.30 -176.52   19.51\n",
      "  326        509D 509D N   CA   3950 3951  126.12  114.20                  -40.00\n",
      "  327  20229 510C 511R C   N    3962 3964 -149.20 -125.20   26.59    0.81  -63.00 -172.19   31.22\n",
      "  327        511R 511R N   CA   3964 3965  152.04  140.60                  -41.10\n",
      "  328  20231 512C 513S C   N    3979 3981 -150.96 -136.60   16.10    0.50  -64.10 -172.18   19.84\n",
      "  328        513S 513S N   CA   3981 3982  158.47  151.20                  -35.00\n",
      "  329  20232 513S 514A C   N    3985 3987 -150.38 -134.00   17.41    0.40  -62.50 -172.00   35.51\n",
      "  329        514A 514A N   CA   3987 3988  152.90  147.00                  -40.90\n",
      "  330  20234 515C 516G C   N    3996 3998  174.43 -167.20   31.29    1.24   82.20  168.29   12.19\n",
      "  330        516G 516G N   CA   3998 3999  149.27  174.60                    8.50\n",
      "  331  20238 519V 520T C   N    4022 4024 -134.98 -124.80   10.97    0.46  -63.20 -175.19   28.42\n",
      "  331        520T 520T N   CA   4024 4025  147.60  143.50                  -42.10\n",
      "  332  20239 520T 521Y C   N    4029 4031 -139.59 -124.30   23.61    0.99  -63.50 -179.92   32.32\n",
      "  332        521Y 521Y N   CA   4031 4032  153.38  135.40                  -43.40\n",
      "  333  20241 522C 523K C   N    4047 4049 -147.90 -118.00   32.54    1.08  -62.90 -172.38   29.45\n",
      "  333        523K 523K N   CA   4049 4050  151.94  139.10                  -40.80\n",
      "  334  20242 523K 524T C   N    4056 4058 -142.56 -124.80   20.30    0.78  -63.20 -177.31   28.60\n",
      "  334        524T 524T N   CA   4058 4059  153.34  143.50                  -42.10\n",
      "  335  20243 524T 525T C   N    4063 4065 -148.57 -124.80   29.49    1.04  -63.20  178.66   28.42\n",
      "  335        525T 525T N   CA   4065 4066  160.95  143.50                  -42.10\n",
      "  336  20244 525T 526R C   N    4070 4072 -153.68 -125.20   29.74    0.99  -63.00 -167.55   32.13\n",
      "  336        526R 526R N   CA   4072 4073  149.16  140.60                  -41.10\n",
      "  337  20245 526R 527L C   N    4081 4083 -152.86 -108.50   46.61    2.09  -63.50 -166.17   32.43\n",
      "  337        527L 527L N   CA   4083 4084  146.80  132.50                  -41.20\n",
      "  338  20246 527L 528R C   N    4089 4091 -153.72 -125.20   30.62    0.97  -63.00 -169.81   31.82\n",
      "  338        528R 528R N   CA   4091 4092  151.75  140.60                  -41.10\n",
      "  339  20247 528R 529T C   N    4100 4102 -149.08 -124.80   35.52    1.13  -63.20  171.53   27.50\n",
      "  339        529T 529T N   CA   4102 4103  169.42  143.50                  -42.10\n",
      "  340  20248 529T 530F C   N    4107 4109 -146.75 -124.20   23.22    0.75  -63.20 -173.37   31.28\n",
      "  340        530F 530F N   CA   4109 4110  148.82  143.30                  -44.30\n",
      "  341  20250 531P 532D C   N    4125 4127 -150.09  -96.50   69.71    2.91  -63.30 -176.90   28.18\n",
      "  341        532D 532D N   CA   4127 4128  158.78  114.20                  -40.00\n",
      "  342  20252 533V 534L C   N    4140 4142 -152.93 -108.50   45.41    2.07  -63.50 -161.79   33.05\n",
      "  342        534L 534L N   CA   4142 4143  141.90  132.50                  -41.20\n",
      "  343  20255 536I 537F C   N    4163 4165 -146.34 -124.20   28.75    0.75  -63.20  175.05   29.60\n",
      "  343        537F 537F N   CA   4165 4166  161.65  143.30                  -44.30\n",
      "  344  20256 537F 538L C   N    4174 4176 -153.31 -108.50   54.83    2.42  -63.50  178.88   30.36\n",
      "  344        538L 538L N   CA   4176 4177  164.10  132.50                  -41.20\n",
      "  345  20257 538L 539R C   N    4182 4184 -157.30 -125.20   41.10    1.22  -63.00  179.41   30.53\n",
      "  345        539R 539R N   CA   4184 4185  166.27  140.60                  -41.10\n",
      "  346  20258 539R 540R C   N    4193 4195 -160.76 -125.20   47.17    1.41  -63.00  176.79   30.36\n",
      "  346        540R 540R N   CA   4195 4196  171.60  140.60                  -41.10\n",
      "  347  20259 540R 541A C   N    4204 4206 -163.33 -134.00   35.90    0.90  -62.50 -178.10   35.15\n",
      "  347        541A 541A N   CA   4206 4207  167.70  147.00                  -40.90\n",
      "  348  20260 541A 542Q C   N    4209 4211 -165.34 -121.10   48.74    1.52  -63.80 -170.88   32.95\n",
      "  348        542Q 542Q N   CA   4211 4212  160.15  139.70                  -40.30\n",
      "  349  20261 542Q 543F C   N    4218 4220 -161.58 -124.20   43.08    1.14  -63.20 -179.79   31.12\n",
      "  349        543F 543F N   CA   4220 4221  164.71  143.30                  -44.30\n",
      "  350  20262 543F 544D C   N    4229 4231 -164.66  -96.50   85.74    3.57  -63.30 -175.81   29.17\n",
      "  350        544D 544D N   CA   4231 4232  166.21  114.20                  -40.00\n",
      "  351  20263 544D 545M C   N    4237 4239 -170.22 -125.60   53.20    1.57  -63.40 -175.83   35.57\n",
      "  351        545M 545M N   CA   4239 4240  169.47  140.50                  -40.50\n",
      "  352  20264 545M 546G C   N    4245 4247  172.90 -167.20   20.97    0.57   82.20 -176.52   12.96\n",
      "  352        546G 546G N   CA   4247 4248  168.00  174.60                    8.50\n",
      "  353  20265 546G 547T C   N    4249 4251 -163.02 -124.80   41.89    1.72  -63.20 -173.74   30.22\n",
      "  353        547T 547T N   CA   4251 4252  160.64  143.50                  -42.10\n",
      "  354  20266 547T 548M C   N    4256 4258  178.48 -125.60   68.27    2.02  -63.40 -176.95   36.03\n",
      "  354        548M 548M N   CA   4258 4259  179.66  140.50                  -40.50\n",
      "  355  20267 548M 549S C   N    4264 4266 -155.36 -136.60   22.77    0.70  -64.10 -175.03   19.97\n",
      "  355        549S 549S N   CA   4266 4267  164.10  151.20                  -35.00\n",
      "  356  20269 550V 551K C   N    4277 4279 -168.09 -118.00   50.64    1.80  -62.90 -157.84   32.51\n",
      "  356        551K 551K N   CA   4279 4280  146.57  139.10                  -40.80\n",
      "  357  20270 551K 552K C   N    4286 4288 -179.94 -118.00   76.67    2.58  -62.90  178.60   29.97\n",
      "  357        552K 552K N   CA   4288 4289 -175.71  139.10                  -40.80\n",
      "  358  20271 552K 553L C   N    4295 4297 -158.21 -108.50   52.82    2.35  -63.50 -166.75   32.67\n",
      "  358        553L 553L N   CA   4297 4298  150.35  132.50                  -41.20\n",
      "  359  20272 553L 554D C   N    4303 4305 -178.43  -96.50  104.58    4.36  -63.30 -178.12   29.62\n",
      "  359        554D 554D N   CA   4305 4306  179.19  114.20                  -40.00\n",
      "  360  20274 555V 556F C   N    4318 4320  145.19 -124.20   95.79    2.86  -63.20 -152.74   37.19\n",
      "  360        556F 556F N   CA   4320 4321  174.38  143.30                  -44.30\n",
      "  361  20276 557V 558D C   N    4336 4338 -154.62  -70.90   92.09    5.29  -63.30  159.98   25.52\n",
      "  361        558D 558D N   CA   4338 4339 -171.35  150.30                  -40.00\n",
      "  362  20279 560P 561L C   N    4358 4360 -158.43 -108.50   65.87    2.94  -63.50  171.92   29.67\n",
      "  362        561L 561L N   CA   4360 4361  175.47  132.50                  -41.20\n",
      "  363  20280 561L 562E C   N    4366 4368 -158.44 -117.80   56.01    1.77  -63.60  172.73   28.20\n",
      "  363        562E 562E N   CA   4368 4369  175.34  136.80                  -40.30\n",
      "  364  20281 562E 563L C   N    4375 4377 -174.16 -108.50   80.19    3.55  -63.50  178.66   31.46\n",
      "  364        563L 563L N   CA   4377 4378  178.53  132.50                  -41.20\n",
      "  365  20282 563L 564D C   N    4383 4385 -172.82  -96.50   96.10    4.00  -63.30 -176.36   29.55\n",
      "  365        564D 564D N   CA   4385 4386  172.60  114.20                  -40.00\n",
      "  366  20283 564D 565L C   N    4391 4393  176.55 -108.50   83.27    3.68  -63.50 -167.93   33.87\n",
      "  366        565L 565L N   CA   4393 4394  168.79  132.50                  -41.20\n",
      "  367  20284 565L 566E C   N    4399 4401 -172.47 -117.80   60.58    1.78  -63.60 -169.11   31.35\n",
      "  367        566E 566E N   CA   4401 4402  162.90  136.80                  -40.30\n",
      "  368  20285 566E 567F C   N    4408 4410 -170.44 -124.20   55.79    1.45  -63.20  177.30   31.08\n",
      "  368        567F 567F N   CA   4410 4411  174.52  143.30                  -44.30\n",
      "  369  20286 567F 568L C   N    4419 4421  179.13 -108.50   80.08    3.54  -63.50 -167.94   33.74\n",
      "  369        568L 568L N   CA   4421 4422  166.78  132.50                  -41.20\n",
      "  370  20287 568L 569R C   N    4427 4429 -177.35 -125.20   65.83    1.95  -63.00  179.31   31.57\n",
      "  370        569R 569R N   CA   4429 4430 -179.21  140.60                  -41.10\n",
      "  371  20288 569R 570G C   N    4438 4440  177.31 -167.20   15.57    0.26   82.20 -167.18   13.61\n",
      "  371        570G 570G N   CA   4440 4441  176.23  174.60                    8.50\n",
      "  372  20289 570G 571K C   N    4442 4444 -177.71 -118.00   70.23    2.34  -62.90 -176.52   30.53\n",
      "  372        571K 571K N   CA   4444 4445  176.07  139.10                  -40.80\n",
      "  373  20290 571K 572G C   N    4451 4453 -166.39 -167.20    4.82    0.20   82.20 -156.04   14.76\n",
      "  373        572G 572G N   CA   4453 4454  179.35  174.60                    8.50\n",
      "  374  20291 572G 573L C   N    4455 4457 -151.89 -108.50   62.50    2.83  -63.50  166.68   28.56\n",
      "  374        573L 573L N   CA   4457 4458  177.49  132.50                  -41.20\n",
      "  375  20292 573L 574Q C   N    4463 4465 -145.37 -121.10   38.41    1.31  -63.80  170.94   29.35\n",
      "  375        574Q 574Q N   CA   4465 4466  169.47  139.70                  -40.30\n",
      "  376  20293 574Q 575S C   N    4472 4474  169.38 -136.60   55.13    1.92  -64.10 -153.84   24.06\n",
      "  376        575S 575S N   CA   4474 4475  162.23  151.20                  -35.00\n",
      "  377  20294 575S 576D C   N    4478 4480 -125.48  -70.90   56.61    3.69  -63.30  166.72   24.60\n",
      "  377        576D 576D N   CA   4480 4481  165.31  150.30                  -40.00\n",
      "  378  20295 576D 577E C   N    4486 4488 -114.23 -117.80   37.03    1.92  -63.60  154.57   23.47\n",
      "  378        577E 577E N   CA   4488 4489  173.66  136.80                  -40.30\n",
      "  379  20297 578V 579S C   N    4502 4504 -116.17 -136.60   62.79    3.50  -64.10  125.71   12.82\n",
      "  379        579S 579S N   CA   4504 4505 -149.43  151.20                  -35.00\n",
      "  380  20298 579S 580M C   N    4508 4510 -157.06 -125.60   91.98    3.64  -63.40  131.68   26.33\n",
      "  380        580M 580M N   CA   4510 4511 -133.06  140.50                  -40.50\n",
      "  381  20299 580M 581P C   N    4516 4518  161.75  -58.70  142.63   16.47  -64.50 -160.37   21.56\n",
      "  381        581P 581P N   CA   4518 4519   -1.00  -30.50                  147.20\n",
      "  382  20300 581P 582E C   N    4523 4525  147.33 -117.80   95.38    3.26  -63.60 -131.64   38.28\n",
      "  382        582E 582E N   CA   4525 4526  146.70  136.80                  -40.30\n",
      "  383  20301 582E 583A C   N    4532 4534  144.34 -134.00   98.46    2.41  -62.50 -167.21   39.01\n",
      "  383        583A 583A N   CA   4534 4535 -157.99  147.00                  -40.90\n",
      "  384  20302 583A 584Q C   N    4537 4539 -162.64 -121.10   96.95    3.76  -63.80  135.31   24.42\n",
      "  384        584Q 584Q N   CA   4539 4540 -132.70  139.70                  -40.30\n",
      "  385  20303 584Q 585E C   N    4546 4548  -65.56  -63.60   80.17   10.58  -63.60   80.17   10.58\n",
      "  385        585E 585E N   CA   4548 4549   39.85  -40.30                  -40.30\n",
      "  386  20304 585E 586D C   N    4555 4557 -167.14  -96.50   71.11    2.91  -63.30 -167.32   20.54\n",
      "  386        586D 586D N   CA   4557 4558  122.31  114.20                  -40.00\n",
      "  387  20305 586D 587L C   N    4563 4565   44.35  -70.70  133.23   13.43  -63.50  154.06   19.77\n",
      "  387        587L 587L N   CA   4565 4566 -151.21  141.60                  -41.20\n",
      "  388  20306 587L 588P C   N    4571 4573 -132.00  -64.50   67.51    6.87  -64.50   67.51    6.87\n",
      "  388        588P 588P N   CA   4573 4574  147.76  147.20                  147.20\n",
      "  389  20307 588P 589H C   N    4578 4580  113.56   56.30   65.11    7.87  -63.20 -175.73   31.17\n",
      "  389        589H 589H N   CA   4580 4581    9.80   40.80                  -42.30\n",
      "  390  20308 589H 590K C   N    4588 4590  -64.15  -70.20   44.52    3.36  -62.90  134.70   17.56\n",
      "  390        590K 590K N   CA   4590 4591 -175.49  140.40                  -40.80\n",
      "  391  20309 590K 591P C   N    4597 4599 -141.16  -64.50  113.77    7.06  -64.50  113.77    7.06\n",
      "  391        591P 591P N   CA   4599 4600 -128.74  147.20                  147.20\n",
      "  392  20310 591P 592T C   N    4604 4606 -173.32 -124.80   54.42    2.16  -63.20 -174.10   30.68\n",
      "  392        592T 592T N   CA   4606 4607  168.13  143.50                  -42.10\n",
      "  393  20311 592T 593S C   N    4611 4613   18.68  -72.40   93.42    7.18  -64.10  172.89   11.22\n",
      "  393        593S 593S N   CA   4613 4614  173.21  152.40                  -35.00\n",
      "  394  20312 593S 594S C   N    4617 4619  169.10 -136.60   67.37    2.07  -64.10 -175.57   22.46\n",
      "  394        594S 594S N   CA   4619 4620 -168.93  151.20                  -35.00\n",
      "  395  20313 594S 595S C   N    4623 4625 -172.27 -136.60   46.46    1.45  -64.10 -179.87   20.86\n",
      "  395        595S 595S N   CA   4625 4626 -179.04  151.20                  -35.00\n",
      "  396  20314 595S 596M C   N    4629 4631  -60.52  -73.00   47.88    3.46  -63.40  130.30   19.23\n",
      "  396        596M 596M N   CA   4631 4632 -170.77  143.00                  -40.50\n",
      "  397  20315 596M 597K C   N    4637 4639 -135.66 -118.00   37.21    1.93  -62.90  164.15   18.86\n",
      "  397        597K 597K N   CA   4639 4640  106.35  139.10                  -40.80\n",
      "  398  20316 597K 598T C   N    4646 4648   24.93  -63.20   89.57   13.05  -63.20   89.57   13.05\n",
      "  398        598T 598T N   CA   4648 4649  -58.11  -42.10                  -42.10\n",
      "  399  20318 599V 600D C   N    4660 4662  158.45  -96.50  119.89    4.97  -63.30 -157.45   33.46\n",
      "  399        600D 600D N   CA   4662 4663  171.97  114.20                  -40.00\n",
      "  400  20319 600D 601E C   N    4668 4670  171.43 -117.80   73.15    2.28  -63.60 -153.50   34.23\n",
      "  400        601E 601E N   CA   4670 4671  155.30  136.80                  -40.30\n",
      "  401  20320 601E 602E C   N    4677 4679 -164.09 -117.80   50.06    2.32  -63.60 -172.71   21.64\n",
      "  401        602E 602E N   CA   4679 4680  117.75  136.80                  -40.30\n",
      "  402  20321 602E 603A C   N    4686 4688 -175.31 -134.00   61.32    1.95  -62.50  169.72   33.63\n",
      "  402        603A 603A N   CA   4688 4689 -167.69  147.00                  -40.90\n",
      "  403  20322 603A 604L C   N    4691 4693  168.83 -108.50   90.26    4.00  -63.50 -162.97   34.97\n",
      "  403        604L 604L N   CA   4693 4694  168.73  132.50                  -41.20\n",
      "  404  20323 604L 605A C   N    4699 4701    5.08  -68.20   88.84    8.72  -62.50  140.85   21.10\n",
      "  404        605A 605A N   CA   4701 4702 -164.47  145.30                  -40.90\n",
      "  405  20325 606M 607L C   N    4712 4714  -10.30  -63.50   68.88    9.17  -63.50   68.88    9.17\n",
      "  405        607L 607L N   CA   4714 4715  -84.95  -41.20                  -41.20\n",
      "  406  20326 607L 608L C   N    4720 4722  131.85 -108.50  123.51    5.58  -63.50 -133.43   40.99\n",
      "  406        608L 608L N   CA   4722 4723  163.16  132.50                  -41.20\n",
      "  407  20327 608L 609S C   N    4728 4730  163.73 -136.60   60.59    2.14  -64.10 -149.92   24.75\n",
      "  407        609S 609S N   CA   4730 4731  161.71  151.20                  -35.00\n",
      "  408  20328 609S 610M C   N    4734 4736 -142.42 -125.60   67.30    2.83  -63.40  138.57   26.68\n",
      "  408        610M 610M N   CA   4736 4737 -154.33  140.50                  -40.50\n",
      "  409  20329 610M 611G C   N    4742 4744 -179.62 -167.20   16.94    0.36   82.20 -157.06   14.26\n",
      "  409        611G 611G N   CA   4744 4745 -173.88  174.60                    8.50\n",
      "  410  20331 612I 613E C   N    4754 4756 -148.80 -117.80   42.15    1.32  -63.60  176.28   28.23\n",
      "  410        613E 613E N   CA   4756 4757  165.37  136.80                  -40.30\n",
      "  411  20332 613E 614E C   N    4763 4765 -147.21 -117.80   85.02    3.60  -63.60  132.77   22.14\n",
      "  411        614E 614E N   CA   4765 4766 -143.43  136.80                  -40.30\n",
      "  412  20333 614E 615T C   N    4772 4774 -167.39 -124.80   46.91    1.91  -63.20 -173.45   30.48\n",
      "  412        615T 615T N   CA   4774 4775  163.16  143.50                  -42.10\n",
      "  413  20335 616V 617A C   N    4786 4788  156.72 -134.00   82.01    1.96  -62.50 -169.58   38.25\n",
      "  413        617A 617A N   CA   4788 4789 -169.12  147.00                  -40.90\n",
      "  414  20336 617A 618R C   N    4791 4793 -177.87 -125.20   55.99    1.80  -63.00 -163.59   34.07\n",
      "  414        618R 618R N   CA   4793 4794  159.58  140.60                  -41.10\n",
      "  415  20337 618R 619Y C   N    4802 4804 -153.14 -124.30   70.58    2.81  -63.50  147.21   27.43\n",
      "  415        619Y 619Y N   CA   4804 4805 -160.18  135.40                  -43.40\n",
      "  416  20338 619Y 620A C   N    4814 4816 -140.87 -134.00   32.97    1.69  -62.50  160.33   30.46\n",
      "  416        620A 620A N   CA   4816 4817  179.24  147.00                  -40.90\n",
      "  417  20339 620A 621L C   N    4819 4821 -127.20 -108.50   45.70    2.24  -63.50  158.01   25.88\n",
      "  417        621L 621L N   CA   4821 4822  174.20  132.50                  -41.20\n",
      "  418  20340 621L 622L C   N    4827 4829 -152.69 -108.50   57.59    2.57  -63.50  173.97   29.63\n",
      "  418        622L 622L N   CA   4829 4830  169.44  132.50                  -41.20\n",
      "  419  20341 622L 623Q C   N    4835 4837 -151.33 -121.10   53.04    1.88  -63.80  162.08   28.26\n",
      "  419        623Q 623Q N   CA   4837 4838 -176.71  139.70                  -40.30\n",
      "  420  20342 623Q 624T C   N    4844 4846 -168.97 -124.80   60.16    1.98  -63.20  170.36   28.34\n",
      "  420        624T 624T N   CA   4846 4847 -175.66  143.50                  -42.10\n",
      "  421  20343 624T 625G C   N    4851 4853  172.92 -167.20   21.01    0.58   82.20 -176.70   12.95\n",
      "  421        625G 625G N   CA   4853 4854  167.78  174.60                    8.50\n",
      "  422  20344 625G 626M C   N    4855 4857 -178.93 -125.60   57.37    1.80  -63.40 -164.39   37.90\n",
      "  422        626M 626M N   CA   4857 4858  161.66  140.50                  -40.50\n",
      "  423  20345 626M 627N C   N    4863 4865 -167.95 -119.90   58.95    1.72  -63.20 -178.88   28.36\n",
      "  423        627N 627N N   CA   4865 4866  171.15  137.00                  -41.10\n",
      "  424  20346 627N 628A C   N    4871 4873 -169.25 -134.00   43.10    1.07  -62.50 -178.08   35.43\n",
      "  424        628A 628A N   CA   4873 4874  171.80  147.00                  -40.90\n",
      "  425  20347 628A 629E C   N    4876 4878 -168.30 -117.80   67.01    2.08  -63.60  173.90   28.80\n",
      "  425        629E 629E N   CA   4878 4879 -179.15  136.80                  -40.30\n",
      "  426  20348 629E 630R C   N    4885 4887 -148.43 -125.20   34.41    1.07  -63.00  175.16   29.42\n",
      "  426        630R 630R N   CA   4887 4888  165.99  140.60                  -41.10\n",
      "  427  20349 630R 631A C   N    4896 4898 -172.05 -134.00   44.36    1.04  -62.50 -174.81   36.11\n",
      "  427        631A 631A N   CA   4898 4899  169.79  147.00                  -40.90\n",
      "  428  20351 632V 633D C   N    4908 4910 -160.26  -96.50   85.19    3.56  -63.30  178.03   28.14\n",
      "  428        633D 633D N   CA   4910 4911  170.69  114.20                  -40.00\n",
      "  429  20352 633D 634Y C   N    4916 4918 -135.28 -124.30   14.73    0.68  -63.50 -174.20   33.05\n",
      "  429        634Y 634Y N   CA   4918 4919  145.22  135.40                  -43.40\n",
      "  430  20354 635V 636F C   N    4935 4937 -135.30 -124.20   13.18    0.34  -63.20 -179.68   29.73\n",
      "  430        636F 636F N   CA   4937 4938  150.42  143.30                  -44.30\n",
      "  431  20355 636F 637S C   N    4946 4948 -169.12 -136.60   40.62    1.25  -64.10 -177.32   20.82\n",
      "  431        637S 637S N   CA   4948 4949  175.53  151.20                  -35.00\n",
      "  432  20356 637S 638H C   N    4952 4954 -128.58 -125.60    8.87    0.38  -63.20 -175.29   19.97\n",
      "  432        638H 638H N   CA   4954 4955  130.44  138.80                  -42.30\n",
      "  433  20358 639E 640N C   N    4971 4973   40.46   55.90   29.08    1.42  -63.20  147.71   23.84\n",
      "  433        640N 640N N   CA   4973 4974   64.14   39.50                  -41.10\n",
      "  434  20360 641I 642A C   N    4987 4989 -153.47 -134.00   20.33    0.49  -62.50 -170.49   35.92\n",
      "  434        642A 642A N   CA   4989 4990  152.86  147.00                  -40.90\n",
      "  435  20361 642A 643E C   N    4992 4994 -155.96 -117.80   41.15    1.22  -63.60 -168.73   30.62\n",
      "  435        643E 643E N   CA   4994 4995  152.20  136.80                  -40.30\n",
      "  436  20362 643E 644E C   N    5001 5003 -149.33 -117.80   36.59    1.07  -63.60 -174.64   29.49\n",
      "  436        644E 644E N   CA   5003 5004  155.36  136.80                  -40.30\n",
      "  437  20363 644E 645A C   N    5010 5012 -162.50 -134.00   34.73    0.86  -62.50 -177.83   35.16\n",
      "  437        645A 645A N   CA   5012 5013  166.84  147.00                  -40.90\n",
      "  438  20364 645A 646G C   N    5015 5017  171.04 -167.20   35.23    1.38   82.20  164.45   11.87\n",
      "  438        646G 646G N   CA   5017 5018  146.90  174.60                    8.50\n",
      "  439  20365 646G 647L C   N    5019 5021 -151.29 -108.50   49.52    2.18  -63.50 -176.30   30.92\n",
      "  439        647L 647L N   CA   5021 5022  157.44  132.50                  -41.20\n",
      "  440  20366 647L 648A C   N    5027 5029 -154.94 -134.00   23.03    0.52  -62.50 -173.03   35.58\n",
      "  440        648A 648A N   CA   5029 5030  156.58  147.00                  -40.90\n",
      "  441  20367 648A 649E C   N    5032 5034 -151.32 -117.80   36.00    1.07  -63.60 -168.90   30.37\n",
      "  441        649E 649E N   CA   5034 5035  149.92  136.80                  -40.30\n",
      "  442  20369 650I 651S C   N    5049 5051 -153.77 -136.60   18.01    0.59  -64.10 -169.24   20.26\n",
      "  442        651S 651S N   CA   5051 5052  156.63  151.20                  -35.00\n",
      "  443  20370 651S 652A C   N    5055 5057 -161.49 -134.00   28.81    0.69  -62.50 -168.87   36.59\n",
      "  443        652A 652A N   CA   5057 5058  155.61  147.00                  -40.90\n",
      "  444  20371 652A 653T C   N    5060 5062 -144.45 -124.80   26.18    0.87  -63.20  176.87   27.95\n",
      "  444        653T 653T N   CA   5062 5063  160.79  143.50                  -42.10\n",
      "  445  20372 653T 654A C   N    5067 5069 -161.96 -134.00   29.28    0.70  -62.50 -168.70   36.65\n",
      "  445        654A 654A N   CA   5069 5070  155.69  147.00                  -40.90\n",
      "  446  20373 654A 655S C   N    5072 5074 -155.76 -136.60   19.44    0.69  -64.10 -166.41   20.61\n",
      "  446        655S 655S N   CA   5074 5075  154.48  151.20                  -35.00\n",
      "  447  20374 655S 656E C   N    5078 5080 -158.21 -117.80   44.34    1.30  -63.60 -170.10   30.55\n",
      "  447        656E 656E N   CA   5080 5081  155.05  136.80                  -40.30\n",
      "  448  20375 656E 657S C   N    5087 5089 -149.41 -136.60   13.45    0.44  -64.10 -170.08   19.88\n",
      "  448        657S 657S N   CA   5089 5090  155.31  151.20                  -35.00\n",
      "  449  20376 657S 658Q C   N    5093 5095 -145.23 -121.10   24.29    0.86  -63.80 -165.01   32.93\n",
      "  449        658Q 658Q N   CA   5095 5096  142.53  139.70                  -40.30\n",
      "  450  20378 659P 660A C   N    5109 5111 -141.15 -134.00    9.35    0.26  -62.50 -176.24   34.33\n",
      "  450        660A 660A N   CA   5111 5112  153.02  147.00                  -40.90\n",
      "  451  20379 660A 661H C   N    5114 5116 -142.40 -125.60   25.03    0.63  -63.20  178.84   27.51\n",
      "  451        661H 661H N   CA   5116 5117  157.35  138.80                  -42.30\n",
      "  452  20381 662V 663L C   N    5131 5133 -127.92 -108.50   19.66    0.90  -63.50 -171.84   23.35\n",
      "  452        663L 663L N   CA   5133 5134  135.58  132.50                  -41.20\n",
      "  453  20382 663L 664D C   N    5139 5141 -118.93  -96.50   38.95    1.64  -63.30 -177.37   26.13\n",
      "  453        664D 664D N   CA   5141 5142  146.05  114.20                  -40.00\n",
      "  454  20383 664D 665G C   N    5147 5149 -153.02 -167.20   45.52    1.55   82.20  175.10   13.45\n",
      "  454        665G 665G N   CA   5149 5150  131.34  174.60                    8.50\n",
      "  455  20385 666P 667A C   N    5158 5160 -152.59 -134.00   19.17    0.48  -62.50 -169.84   35.98\n",
      "  455        667A 667A N   CA   5160 5161  151.64  147.00                  -40.90\n",
      "  456  20386 667A 668K C   N    5163 5165 -145.88 -118.00   28.27    1.00  -62.90 -165.90   30.17\n",
      "  456        668K 668K N   CA   5165 5166  143.74  139.10                  -40.80\n",
      "  457  20387 668K 669Y C   N    5172 5174 -139.19 -124.30   23.61    0.98  -63.50  179.61   32.23\n",
      "  457        669Y 669Y N   CA   5174 5175  153.72  135.40                  -43.40\n",
      "  458  20388 669Y 670R C   N    5184 5186 -139.53 -125.20   14.35    0.56  -63.00 -166.69   31.36\n",
      "  458        670R 670R N   CA   5186 5187  141.38  140.60                  -41.10\n",
      "  459  20389 670R 671L C   N    5195 5197 -144.20 -108.50   37.92    1.69  -63.50 -168.64   31.56\n",
      "  459        671L 671L N   CA   5197 5198  145.29  132.50                  -41.20\n",
      "  460  20390 671L 672H C   N    5203 5205 -151.70 -125.60   28.36    0.77  -63.20 -170.29   29.45\n",
      "  460        672H 672H N   CA   5205 5206  149.89  138.80                  -42.30\n",
      "  461  20391 672H 673A C   N    5213 5215 -164.65 -134.00   32.33    0.76  -62.50 -168.64   36.79\n",
      "  461        673A 673A N   CA   5215 5216  157.29  147.00                  -40.90\n",
      "  462  20392 673A 674M C   N    5218 5220 -152.97 -125.60   28.55    0.94  -63.40 -167.07   35.79\n",
      "  462        674M 674M N   CA   5220 5221  148.63  140.50                  -40.50\n",
      "  463  20394 675I 676S C   N    5234 5236 -152.77 -136.60   17.15    0.55  -64.10 -169.95   20.14\n",
      "  463        676S 676S N   CA   5236 5237  156.90  151.20                  -35.00\n",
      "  464  20395 676S 677H C   N    5240 5242 -155.16 -125.60   33.86    0.87  -63.20 -173.37   29.27\n",
      "  464        677H 677H N   CA   5242 5243  155.30  138.80                  -42.30\n",
      "  465  20397 678V 679G C   N    5257 5259  177.98 -167.20   26.61    1.06   82.20  172.94   12.56\n",
      "  465        679G 679G N   CA   5259 5260  152.50  174.60                    8.50\n",
      "  466  20398 679G 680A C   N    5261 5263 -165.01 -134.00   33.27    0.77  -62.50 -169.92   36.59\n",
      "  466        680A 680A N   CA   5263 5264  159.03  147.00                  -40.90\n",
      "  467  20399 680A 681S C   N    5266 5268 -155.66 -136.60   20.51    0.65  -64.10 -170.22   20.34\n",
      "  467        681S 681S N   CA   5268 5269  158.77  151.20                  -35.00\n",
      "  468  20400 681S 682A C   N    5272 5274 -157.26 -134.00   24.86    0.57  -62.50 -171.16   36.00\n",
      "  468        682A 682A N   CA   5274 5275  155.76  147.00                  -40.90\n",
      "  469  20401 682A 683K C   N    5277 5279 -145.94 -118.00   31.35    1.04  -62.90 -174.51   29.06\n",
      "  469        683K 683K N   CA   5279 5280  153.33  139.10                  -40.80\n",
      "  470  20402 683K 684T C   N    5286 5288 -145.52 -124.80   31.50    0.99  -63.20  171.70   27.33\n",
      "  470        684T 684T N   CA   5288 5289  167.22  143.50                  -42.10\n",
      "  471  20403 684T 685G C   N    5293 5295 -178.69 -167.20   23.96    0.97   82.20  175.69   12.82\n",
      "  471        685G 685G N   CA   5295 5296  153.57  174.60                    8.50\n",
      "  472  20404 685G 686H C   N    5297 5299 -151.99 -125.60   27.29    0.81  -63.20 -166.48   29.96\n",
      "  472        686H 686H N   CA   5299 5300  145.75  138.80                  -42.30\n",
      "  473  20405 686H 687Y C   N    5307 5309 -143.07 -124.30   27.34    1.19  -63.50  179.89   32.44\n",
      "  473        687Y 687Y N   CA   5309 5310  155.27  135.40                  -43.40\n",
      "  474  20408 689C 690H C   N    5332 5334 -135.78 -125.60   99.67    3.97  -63.20  109.48   12.06\n",
      "  474        690H 690H N   CA   5334 5335   39.65  138.80                  -42.30\n",
      "  475  20411 692C 693D C   N    5356 5358 -167.68  -96.50   98.79    4.13  -63.30  172.47   27.84\n",
      "  475        693D 693D N   CA   5358 5359 -177.29  114.20                  -40.00\n",
      "  476  20412 693D 694A C   N    5364 5366  179.55 -134.00   50.96    1.16  -62.50 -168.28   37.59\n",
      "  476        694A 694A N   CA   5366 5367  167.96  147.00                  -40.90\n",
      "  477  20413 694A 695Q C   N    5369 5371 -160.30 -121.10   56.68    1.86  -63.80  169.27   29.72\n",
      "  477        695Q 695Q N   CA   5371 5372 -179.37  139.70                  -40.30\n",
      "  478  20414 695Q 696T C   N    5378 5380 -155.02 -124.80   46.07    1.45  -63.20  167.12   27.23\n",
      "  478        696T 696T N   CA   5380 5381  178.27  143.50                  -42.10\n",
      "  479  20415 696T 697G C   N    5385 5387  179.14 -167.20   13.95    0.21   82.20 -165.24   13.78\n",
      "  479        697G 697G N   CA   5387 5388  177.42  174.60                    8.50\n",
      "  480  20416 697G 698K C   N    5389 5391  173.52 -118.00   69.00    2.47  -62.90 -148.47   34.75\n",
      "  480        698K 698K N   CA   5391 5392  147.53  139.10                  -40.80\n",
      "  481  20417 698K 699W C   N    5398 5400 -163.94 -124.90   52.13    1.66  -63.00  170.86   28.09\n",
      "  481        699W 699W N   CA   5400 5401  177.94  143.40                  -44.20\n",
      "  482  20418 699W 700L C   N    5412 5414 -157.28 -108.50   64.52    2.88  -63.50  171.90   29.60\n",
      "  482        700L 700L N   CA   5414 5415  174.73  132.50                  -41.20\n",
      "  483  20419 700L 701L C   N    5420 5422 -154.21 -108.50   68.12    3.10  -63.50  163.29   28.21\n",
      "  483        701L 701L N   CA   5422 5423 -176.99  132.50                  -41.20\n",
      "  484  20420 701L 702F C   N    5428 5430 -167.72 -124.20   56.10    1.46  -63.20  172.31   30.22\n",
      "  484        702F 702F N   CA   5430 5431  178.71  143.30                  -44.30\n",
      "  485  20421 702F 703N C   N    5439 5441 -163.37 -119.90   70.77    2.16  -63.20  161.01   25.52\n",
      "  485        703N 703N N   CA   5441 5442 -167.16  137.00                  -41.10\n",
      "  486  20422 703N 704D C   N    5447 5449  175.31  -96.50   96.57    3.99  -63.30 -153.98   33.08\n",
      "  486        704D 704D N   CA   5449 5450  153.54  114.20                  -40.00\n",
      "  487  20423 704D 705E C   N    5455 5457 -168.34 -117.80   62.45    1.87  -63.60  179.87   29.64\n",
      "  487        705E 705E N   CA   5457 5458  173.48  136.80                  -40.30\n",
      "  488  20424 705E 706K C   N    5464 5466  177.86 -118.00   65.53    2.27  -62.90 -155.07   33.64\n",
      "  488        706K 706K N   CA   5466 5467  152.53  139.10                  -40.80\n",
      "  489  20426 707V 708A C   N    5480 5482  167.47 -134.00   74.43    1.96  -62.50 -178.85   36.26\n",
      "  489        708A 708A N   CA   5482 5483 -167.02  147.00                  -40.90\n",
      "  490  20427 708A 709E C   N    5485 5487  156.97 -117.80   90.83    2.72  -63.60 -154.12   34.69\n",
      "  490        709E 709E N   CA   5487 5488  168.21  136.80                  -40.30\n",
      "  491  20428 709E 710S C   N    5494 5496 -173.86 -136.60   54.23    1.78  -64.10  173.52   20.48\n",
      "  491        710S 710S N   CA   5496 5497 -169.40  151.20                  -35.00\n",
      "  492  20429 710S 711L C   N    5500 5502 -162.98 -108.50   86.69    3.99  -63.50  154.99   27.47\n",
      "  492        711L 711L N   CA   5502 5503 -160.06  132.50                  -41.20\n",
      "  493  20430 711L 712N C   N    5508 5510 -140.44 -119.90   28.35    0.82  -63.20  179.80   26.84\n",
      "  493        712N 712N N   CA   5510 5511  156.54  137.00                  -41.10\n",
      "  494  20433 714P 715F C   N    5530 5532 -150.17 -124.20   26.07    0.95  -63.20 -168.98   32.10\n",
      "  494        715F 715F N   CA   5532 5533  145.62  143.30                  -44.30\n",
      "  495  20434 715F 716S C   N    5541 5543 -168.80 -136.60   40.61    1.25  -64.10 -177.84   20.76\n",
      "  495        716S 716S N   CA   5543 5544  175.93  151.20                  -35.00\n",
      "  496  20435 716S 717M C   N    5547 5549 -173.55 -125.60   84.34    2.84  -63.40  155.39   31.05\n",
      "  496        717M 717M N   CA   5549 5550 -150.11  140.50                  -40.50\n",
      "  497  20436 717M 718A C   N    5555 5557 -175.35 -134.00   67.97    2.38  -62.50  163.40   32.54\n",
      "  497        718A 718A N   CA   5557 5558 -159.07  147.00                  -40.90\n",
      "  498  20437 718A 719F C   N    5560 5562  171.11 -124.20   66.15    2.18  -63.20 -157.68   35.60\n",
      "  498        719F 719F N   CA   5562 5563  157.15  143.30                  -44.30\n",
      "  499  20438 719F 720L C   N    5571 5573  175.66 -108.50  115.13    5.25  -63.50  156.64   28.58\n",
      "  499        720L 720L N   CA   5573 5574 -140.88  132.50                  -41.20\n",
      "  500  20439 720L 721Y C   N    5579 5581 -177.46  -63.50  114.03   19.86  -63.50  114.03   19.86\n",
      "  500        721Y 721Y N   CA   5581 5582  -47.53  -43.40                  -43.40\n",
      "  501  20440 721Y 722F C   N    5591 5593 -167.49 -124.20   50.13    1.33  -63.20 -179.68   31.41\n",
      "  501        722F 722F N   CA   5593 5594  168.59  143.30                  -44.30\n",
      "  502  20441 722F 723Y C   N    5602 5604  -16.45  -98.40   85.09    4.22  -63.50  156.16   27.16\n",
      "  502        723Y 723Y N   CA   5604 5605  105.51  128.40                  -43.40\n",
      "  503  20442 723Y 724K C   N    5614 5616 -175.18 -118.00   72.93    2.48  -62.90  175.46   29.32\n",
      "  503        724K 724K N   CA   5616 5617 -175.63  139.10                  -40.80\n",
      "  504  20443 724K 725R C   N    5623 5625 -148.28 -125.20   50.33    1.84  -63.00  158.47   27.08\n",
      "  504        725R 725R N   CA   5625 5626 -174.67  140.60                  -41.10\n",
      "  505  20445 726V 727G C   N    5641 5643  151.71 -167.20   57.64    2.15   82.20  143.62   10.09\n",
      "  505        727G 727G N   CA   5643 5644  134.18  174.60                    8.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  27953 192W 237D CZ3 CB   1464 1784   11.50    4.77    6.73    4.64    4.77    6.73    4.64\n",
      "    2  27959 192W 237D CH2 CB   1465 1784   11.62    4.49    7.13    4.93    4.49    7.13    4.93\n",
      "    3  28118 233P 607L CB  CD2  1763 4719   12.56    4.45    8.11    5.11    4.45    8.11    5.11\n",
      "    4  28184 252F 606M CB  SD   1910 4710   26.02    4.79   21.23   16.79    4.79   21.23   16.79\n",
      "    5  28185 253A 606M CB  CG   1921 4709   21.47    4.50   16.97   11.23    4.50   16.97   11.23\n",
      "    6  28186 253A 606M CB  SD   1921 4710   22.12    3.87   18.25   12.40    3.87   18.25   12.40\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   62  119  394  456  551  461  475  597  618  679\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00192.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      728     5656     5656\n",
      "openf___224_> Open           EAO00192.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18229916   17802.652    17.385\n",
      "read_mo_297_> Segments, residues, atoms:        1      728     5656\n",
      "read_mo_298_> Segment:        1  1:A  728:A     5656\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5656     5656        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    23948\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24395\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26848\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27084\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27270\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28042\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28044\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28188\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28194\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.5248       10  10\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.5633       17  17\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.8356       33  33\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.1134       37  37\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -176.6254       40  40\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.7906       45  45\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.4964       86  86\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.8766       94  94\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.8234      119  119\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2132\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.0352      124  124\n",
      "iupac_m_485_> OD1/2 will be swapped:      158.6366      128  128\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -100.9314      130  130\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.5203      146  146\n",
      "iupac_m_487_> NH1/2 swapped:      179.8082      154  154\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.4826      164  164\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.7649      170  170\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.8512      186  186\n",
      "iupac_m_487_> NH1/2 swapped:      175.5791      188  188\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      109.6923      239  239\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.7892      252  252\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -134.8556      279  279\n",
      "iupac_m_485_> OD1/2 will be swapped:     -140.8291      285  285\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -104.4152      286  286\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.7557      288  288\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9287      291  291\n",
      "iupac_m_487_> NH1/2 swapped:      179.7824      304  304\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      327  327\n",
      "iupac_m_486_> OE1/2 will be swapped:      180.0000      331  331\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.6025      347  347\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.8953      372  372\n",
      "iupac_m_487_> NH1/2 swapped:      180.0000      387  387\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.7603      400  400\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9374      416  416\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.6535      434  434\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9657      456  456\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9720      457  457\n",
      "iupac_m_485_> OD1/2 will be swapped:      129.1309      476  476\n",
      "iupac_m_486_> OE1/2 will be swapped:      180.0000      483  483\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.4914      484  484\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6289      490  490\n",
      "iupac_m_485_> OD1/2 will be swapped:      144.3344      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.1404      507  507\n",
      "iupac_m_487_> NH1/2 swapped:      179.8881      526  526\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6897      528  528\n",
      "iupac_m_485_> OD1/2 will be swapped:     -156.8173      544  544\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.8529      554  554\n",
      "iupac_m_485_> OD1/2 will be swapped:     -175.9756      558  558\n",
      "iupac_m_485_> OD1/2 will be swapped:      125.3316      564  564\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.3202      566  566\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      102.6057      567  567\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -176.3563      569  569\n",
      "iupac_m_486_> OE1/2 will be swapped:     -135.7541      577  577\n",
      "iupac_m_486_> OE1/2 will be swapped:     -122.3961      602  602\n",
      "iupac_m_486_> OE1/2 will be swapped:      104.9668      613  613\n",
      "iupac_m_486_> OE1/2 will be swapped:      120.2834      614  614\n",
      "iupac_m_487_> NH1/2 swapped:     -175.6887      618  618\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      110.3313      619  619\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.3984      630  630\n",
      "iupac_m_485_> OD1/2 will be swapped:     -156.1288      633  633\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.1536      639  639\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.6231      644  644\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.1589      649  649\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.9933      656  656\n",
      "iupac_m_485_> OD1/2 will be swapped:     -167.0205      664  664\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6190      670  670\n",
      "iupac_m_485_> OD1/2 will be swapped:      150.1787      693  693\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.0838      705  705\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -114.4559      715  715\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -121.7699      722  722\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   472154\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -18096.4277\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -18096.427734\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    53.912998\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sequence length          :          728\n",
      "Compactness              :     0.000340\n",
      "Native energy (pair)     :   983.079155\n",
      "Native energy (surface)  :   114.755058\n",
      "Native energy (combined) :    30.201557\n",
      "Z score (pair)           :    -2.193692\n",
      "Z score (surface)        :    -0.538916\n",
      "Z score (combined)       :    -1.856146\n",
      "GA341 score              :     0.410254\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     8863\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      373309.8750\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5789     447    559   0.423   0.423     0.30679E+06   1.000\n",
      " 2 Bond angle potential               :    7845     512    882  10.124  10.124      17005.       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3676       0    287  56.352  56.352      2025.1       1.000\n",
      " 4 Stereochemical improper torsion pot:    2409       6     29   2.501   2.501      291.31       1.000\n",
      " 5 Soft-sphere overlap restraints     :    8863       2      5   0.008   0.008      65.981       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    1992     217    401   4.386   4.386      23042.       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2093     185    373   4.858   4.858      20788.       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     727       7     37   6.248   6.248      334.69       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     627       0     57  88.798  88.798      471.84       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     449       0     17  81.260  81.260      277.19       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     214       0      2 110.939 110.939      166.35       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0  95.532  95.532      47.158       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1112       0      1   0.828   0.828      75.855       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     726     506    229  51.492 153.432      1675.9       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     471       5     13   2.735   2.735      252.25       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00192.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00192.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  231470.1094\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     84  10E  10E C   CA     81   75    1.66    1.49    0.17    4.83    1.49    0.17    4.83\n",
      "    2     87  11A  11A N   CA     83   84    1.57    1.43    0.14    4.67    1.43    0.14    4.67\n",
      "    3     89  11A  11A C   CA     86   84    1.66    1.49    0.17    5.05    1.49    0.17    5.05\n",
      "    4     90  11A  12C C   N      86   88    1.47    1.35    0.13    4.57    1.35    0.13    4.57\n",
      "    5     93  12C  12C N   CA     88   89    1.58    1.43    0.15    4.84    1.43    0.15    4.84\n",
      "    6     95  12C  12C C   CA     92   89    1.68    1.49    0.19    5.62    1.49    0.19    5.62\n",
      "    7     98  13A  13A N   CA     94   95    1.59    1.43    0.16    5.28    1.43    0.16    5.28\n",
      "    8    100  13A  13A C   CA     97   95    1.71    1.49    0.22    6.29    1.49    0.22    6.29\n",
      "    9    108  14H  14H N   CA     99  100    1.60    1.43    0.17    5.64    1.43    0.17    5.64\n",
      "   10    110  14H  14H C   CA    107  100    1.70    1.49    0.21    6.19    1.49    0.21    6.19\n",
      "   11    111  14H  15C C   N     107  109    1.50    1.35    0.16    5.58    1.35    0.16    5.58\n",
      "   12    115  15C  15C N   CA    109  110    1.61    1.43    0.18    5.94    1.43    0.18    5.94\n",
      "   13    117  15C  15C C   CA    113  110    1.72    1.49    0.23    6.64    1.49    0.23    6.64\n",
      "   14    118  15C  16W C   N     113  115    1.51    1.35    0.17    5.98    1.35    0.17    5.98\n",
      "   15    128  16W  16W N   CA    115  116    1.62    1.43    0.19    6.30    1.43    0.19    6.30\n",
      "   16    130  16W  16W C   CA    127  116    1.76    1.49    0.27    7.71    1.49    0.27    7.71\n",
      "   17    131  16W  17D C   N     127  129    1.54    1.35    0.19    6.86    1.35    0.19    6.86\n",
      "   18    139  17D  17D N   CA    129  130    1.66    1.43    0.23    7.43    1.43    0.23    7.43\n",
      "   19    141  17D  17D C   CA    135  130    1.77    1.49    0.28    8.06    1.49    0.28    8.06\n",
      "   20    142  17D  18E C   N     135  137    1.55    1.35    0.20    7.24    1.35    0.20    7.24\n",
      "   21    148  18E  18E N   CA    137  138    1.67    1.43    0.24    7.77    1.43    0.24    7.77\n",
      "   22    150  18E  18E C   CA    144  138    1.79    1.49    0.30    8.72    1.49    0.30    8.72\n",
      "   23    151  18E  19S C   N     144  146    1.56    1.35    0.22    7.67    1.35    0.22    7.67\n",
      "   24    154  19S  19S N   CA    146  147    1.68    1.43    0.25    8.31    1.43    0.25    8.31\n",
      "   25    156  19S  19S C   CA    150  147    1.80    1.49    0.31    9.13    1.49    0.31    9.13\n",
      "   26    157  19S  20V C   N     150  152    1.58    1.35    0.23    8.29    1.35    0.23    8.29\n",
      "   27    161  20V  20V N   CA    152  153    1.70    1.43    0.27    8.95    1.43    0.27    8.95\n",
      "   28    163  20V  20V C   CA    157  153    1.83    1.49    0.34    9.97    1.49    0.34    9.97\n",
      "   29    164  20V  21M C   N     157  159    1.59    1.35    0.24    8.58    1.35    0.24    8.58\n",
      "   30    169  21M  21M N   CA    159  160    1.71    1.43    0.28    9.09    1.43    0.28    9.09\n",
      "   31    171  21M  21M C   CA    165  160    1.84    1.49    0.35   10.29    1.49    0.35   10.29\n",
      "   32    172  21M  22L C   N     165  167    1.59    1.35    0.25    8.82    1.35    0.25    8.82\n",
      "   33    177  22L  22L N   CA    167  168    1.72    1.43    0.29    9.51    1.43    0.29    9.51\n",
      "   34    179  22L  22L C   CA    173  168    1.86    1.49    0.37   10.69    1.49    0.37   10.69\n",
      "   35    180  22L  23V C   N     173  175    1.61    1.35    0.26    9.36    1.35    0.26    9.36\n",
      "   36    184  23V  23V N   CA    175  176    1.74    1.43    0.31   10.06    1.43    0.31   10.06\n",
      "   37    186  23V  23V C   CA    180  176    1.88    1.49    0.39   11.48    1.49    0.39   11.48\n",
      "   38    187  23V  24P C   N     180  182    1.68    1.30    0.38   11.21    1.30    0.38   11.21\n",
      "   39    189  24P  24P C   CA    187  183    1.89    1.49    0.40   11.73    1.49    0.40   11.73\n",
      "   40    190  24P  25N C   N     187  189    1.63    1.35    0.28   10.06    1.35    0.28   10.06\n",
      "   41    191  24P  24P N   CA    182  183    1.76    1.43    0.33   10.80    1.43    0.33   10.80\n",
      "   42    200  25N  25N N   CA    189  190    1.76    1.43    0.33   10.89    1.43    0.33   10.89\n",
      "   43    202  25N  25N C   CA    195  190    1.91    1.49    0.42   12.26    1.49    0.42   12.26\n",
      "   44    203  25N  26H C   N     195  197    1.64    1.35    0.30   10.52    1.35    0.30   10.52\n",
      "   45    210  26H  26H N   CA    197  198    1.78    1.43    0.35   11.40    1.43    0.35   11.40\n",
      "   46    212  26H  26H C   CA    205  198    1.94    1.49    0.45   12.96    1.49    0.45   12.96\n",
      "   47    213  26H  27R C   N     205  207    1.66    1.35    0.31   11.05    1.35    0.31   11.05\n",
      "   48    222  27R  27R N   CA    207  208    1.79    1.43    0.36   11.96    1.43    0.36   11.96\n",
      "   49    224  27R  27R C   CA    216  208    1.96    1.49    0.47   13.72    1.49    0.47   13.72\n",
      "   50    225  27R  28S C   N     216  218    1.67    1.35    0.33   11.63    1.35    0.33   11.63\n",
      "   51    228  28S  28S N   CA    218  219    1.81    1.43    0.38   12.64    1.43    0.38   12.64\n",
      "   52    230  28S  28S C   CA    222  219    1.97    1.49    0.48   14.06    1.49    0.48   14.06\n",
      "   53    231  28S  29A C   N     222  224    1.68    1.35    0.34   11.97    1.35    0.34   11.97\n",
      "   54    233  29A  29A N   CA    224  225    1.83    1.43    0.40   13.02    1.43    0.40   13.02\n",
      "   55    235  29A  29A C   CA    227  225    1.99    1.49    0.50   14.44    1.49    0.50   14.44\n",
      "   56    236  29A  30V C   N     227  229    1.69    1.35    0.35   12.28    1.35    0.35   12.28\n",
      "   57    240  30V  30V N   CA    229  230    1.84    1.43    0.41   13.35    1.43    0.41   13.35\n",
      "   58    242  30V  30V C   CA    234  230    2.00    1.49    0.51   14.90    1.49    0.51   14.90\n",
      "   59    243  30V  31H C   N     234  236    1.70    1.35    0.36   12.66    1.35    0.36   12.66\n",
      "   60    250  31H  31H N   CA    236  237    1.85    1.43    0.42   13.71    1.43    0.42   13.71\n",
      "   61    252  31H  31H C   CA    244  237    2.03    1.49    0.54   15.62    1.49    0.54   15.62\n",
      "   62    253  31H  32K C   N     244  246    1.71    1.35    0.37   13.09    1.35    0.37   13.09\n",
      "   63    260  32K  32K N   CA    246  247    1.86    1.43    0.43   14.28    1.43    0.43   14.28\n",
      "   64    262  32K  32K C   CA    253  247    2.04    1.49    0.55   15.98    1.49    0.55   15.98\n",
      "   65    263  32K  33E C   N     253  255    1.73    1.35    0.38   13.55    1.35    0.38   13.55\n",
      "   66    269  33E  33E N   CA    255  256    1.88    1.43    0.45   14.75    1.43    0.45   14.75\n",
      "   67    271  33E  33E C   CA    262  256    2.06    1.49    0.57   16.57    1.49    0.57   16.57\n",
      "   68    272  33E  34E C   N     262  264    1.74    1.35    0.39   13.92    1.35    0.39   13.92\n",
      "   69    278  34E  34E N   CA    264  265    1.89    1.43    0.46   15.18    1.43    0.46   15.18\n",
      "   70    280  34E  34E C   CA    271  265    2.07    1.49    0.58   17.00    1.49    0.58   17.00\n",
      "   71    281  34E  35C C   N     271  273    1.75    1.35    0.40   14.21    1.35    0.40   14.21\n",
      "   72    284  35C  35C N   CA    273  274    1.90    1.43    0.47   15.58    1.43    0.47   15.58\n",
      "   73    286  35C  35C C   CA    277  274    2.08    1.49    0.59   17.28    1.49    0.59   17.28\n",
      "   74    287  35C  36A C   N     277  279    1.76    1.35    0.41   14.54    1.35    0.41   14.54\n",
      "   75    289  36A  36A N   CA    279  280    1.91    1.43    0.48   15.87    1.43    0.48   15.87\n",
      "   76    291  36A  36A C   CA    282  280    2.09    1.49    0.60   17.57    1.49    0.60   17.57\n",
      "   77    292  36A  37Y C   N     282  284    1.76    1.35    0.42   14.75    1.35    0.42   14.75\n",
      "   78    302  37Y  37Y N   CA    284  285    1.92    1.43    0.49   16.08    1.43    0.49   16.08\n",
      "   79    304  37Y  37Y C   CA    294  285    2.12    1.49    0.63   18.26    1.49    0.63   18.26\n",
      "   80    305  37Y  38C C   N     294  296    1.78    1.35    0.43   15.29    1.35    0.43   15.29\n",
      "   81    308  38C  38C N   CA    296  297    1.94    1.43    0.51   16.71    1.43    0.51   16.71\n",
      "   82    310  38C  38C C   CA    300  297    2.13    1.49    0.64   18.50    1.49    0.64   18.50\n",
      "   83    311  38C  39C C   N     300  302    1.78    1.35    0.44   15.57    1.35    0.44   15.57\n",
      "   84    314  39C  39C N   CA    302  303    1.94    1.43    0.51   16.91    1.43    0.51   16.91\n",
      "   85    316  39C  39C C   CA    306  303    2.14    1.49    0.65   18.92    1.49    0.65   18.92\n",
      "   86    317  39C  40R C   N     306  308    1.79    1.35    0.45   15.83    1.35    0.45   15.83\n",
      "   87    325  40R  40R N   CA    308  309    1.95    1.43    0.52   17.22    1.43    0.52   17.22\n",
      "   88    327  40R  40R C   CA    317  309    2.16    1.49    0.67   19.53    1.49    0.67   19.53\n",
      "   89    328  40R  41T C   N     317  319    1.81    1.35    0.46   16.36    1.35    0.46   16.36\n",
      "   90    332  41T  41T N   CA    319  320    1.97    1.43    0.54   17.78    1.43    0.54   17.78\n",
      "   91    334  41T  41T C   CA    324  320    2.18    1.49    0.69   19.96    1.49    0.69   19.96\n",
      "   92    335  41T  42C C   N     324  326    1.82    1.35    0.47   16.75    1.35    0.47   16.75\n",
      "   93    338  42C  42C N   CA    326  327    1.98    1.43    0.55   18.21    1.43    0.55   18.21\n",
      "   94    340  42C  42C C   CA    330  327    2.19    1.49    0.70   20.40    1.49    0.70   20.40\n",
      "   95    341  42C  43R C   N     330  332    1.83    1.35    0.49   17.23    1.35    0.49   17.23\n",
      "   96    349  43R  43R N   CA    332  333    2.00    1.43    0.57   18.66    1.43    0.57   18.66\n",
      "   97    351  43R  43R C   CA    341  333    2.22    1.49    0.73   21.21    1.49    0.73   21.21\n",
      "   98    352  43R  44H C   N     341  343    1.84    1.35    0.50   17.59    1.35    0.50   17.59\n",
      "   99    359  44H  44H N   CA    343  344    2.01    1.43    0.58   19.17    1.43    0.58   19.17\n",
      "  100    361  44H  44H C   CA    351  344    2.23    1.49    0.74   21.60    1.49    0.74   21.60\n",
      "  101    362  44H  45E C   N     351  353    1.85    1.35    0.51   18.03    1.35    0.51   18.03\n",
      "  102    369  45E  45E N   CA    353  354    2.03    1.43    0.60   19.63    1.43    0.60   19.63\n",
      "  103    371  45E  45E C   CA    360  354    2.25    1.49    0.76   22.15    1.49    0.76   22.15\n",
      "  104    372  45E  46M C   N     360  362    1.87    1.35    0.52   18.53    1.35    0.52   18.53\n",
      "  105    377  46M  46M N   CA    362  363    2.04    1.43    0.61   20.19    1.43    0.61   20.19\n",
      "  106    379  46M  46M C   CA    368  363    2.27    1.49    0.78   22.70    1.49    0.78   22.70\n",
      "  107    380  46M  47G C   N     368  370    1.88    1.35    0.53   18.94    1.35    0.53   18.94\n",
      "  108    381  47G  47G N   CA    370  371    1.98    1.43    0.55   18.16    1.43    0.55   18.16\n",
      "  109    383  47G  47G C   CA    372  371    2.02    1.49    0.53   15.43    1.49    0.53   15.43\n",
      "  110    384  47G  48T C   N     372  374    1.58    1.35    0.24    8.50    1.35    0.24    8.50\n",
      "  111    388  48T  48T N   CA    374  375    1.71    1.43    0.28    9.14    1.43    0.28    9.14\n",
      "  112   1825 236A 236A C   CA   1780 1778    1.70    1.49    0.21    5.96    1.49    0.21    5.96\n",
      "  113   2036 260G 261I C   N    1982 1984    1.49    1.35    0.15    5.23    1.35    0.15    5.23\n",
      "  114   2041 261I 261I N   CA   1984 1985    1.61    1.43    0.18    5.84    1.43    0.18    5.84\n",
      "  115   2043 261I 261I C   CA   1990 1985    1.80    1.49    0.31    9.04    1.49    0.31    9.04\n",
      "  116   2044 261I 262D C   N    1990 1992    1.56    1.35    0.22    7.77    1.35    0.22    7.77\n",
      "  117   2049 262D 262D N   CA   1992 1993    1.74    1.43    0.31   10.12    1.43    0.31   10.12\n",
      "  118   2051 262D 262D C   CA   1998 1993    1.86    1.49    0.37   10.67    1.49    0.37   10.67\n",
      "  119   2052 262D 263I C   N    1998 2000    1.59    1.35    0.25    8.78    1.35    0.25    8.78\n",
      "  120   2057 263I 263I N   CA   2000 2001    1.70    1.43    0.27    9.04    1.43    0.27    9.04\n",
      "  121   2059 263I 263I C   CA   2006 2001    1.86    1.49    0.37   10.86    1.49    0.37   10.86\n",
      "  122   2060 263I 264Q C   N    2006 2008    1.58    1.35    0.23    8.33    1.35    0.23    8.33\n",
      "  123   2066 264Q 264Q N   CA   2008 2009    1.71    1.43    0.28    9.15    1.43    0.28    9.15\n",
      "  124   2068 264Q 264Q C   CA   2015 2009    1.83    1.49    0.34   10.03    1.49    0.34   10.03\n",
      "  125   2069 264Q 265T C   N    2015 2017    1.59    1.35    0.24    8.58    1.35    0.24    8.58\n",
      "  126   2073 265T 265T N   CA   2017 2018    1.71    1.43    0.28    9.15    1.43    0.28    9.15\n",
      "  127   2075 265T 265T C   CA   2022 2018    1.83    1.49    0.34    9.77    1.49    0.34    9.77\n",
      "  128   2076 265T 266A C   N    2022 2024    1.58    1.35    0.24    8.35    1.35    0.24    8.35\n",
      "  129   2078 266A 266A N   CA   2024 2025    1.70    1.43    0.27    8.94    1.43    0.27    8.94\n",
      "  130   2080 266A 266A C   CA   2027 2025    1.79    1.49    0.30    8.87    1.49    0.30    8.87\n",
      "  131   2081 266A 267K C   N    2027 2029    1.56    1.35    0.21    7.57    1.35    0.21    7.57\n",
      "  132   2087 267K 267K N   CA   2029 2030    1.68    1.43    0.25    8.11    1.43    0.25    8.11\n",
      "  133   2089 267K 267K C   CA   2036 2030    1.79    1.49    0.30    8.76    1.49    0.30    8.76\n",
      "  134   2090 267K 268K C   N    2036 2038    1.56    1.35    0.22    7.65    1.35    0.22    7.65\n",
      "  135   2096 268K 268K N   CA   2038 2039    1.68    1.43    0.25    8.10    1.43    0.25    8.10\n",
      "  136   2098 268K 268K C   CA   2045 2039    1.81    1.49    0.32    9.23    1.49    0.32    9.23\n",
      "  137   2099 268K 269T C   N    2045 2047    1.57    1.35    0.22    7.96    1.35    0.22    7.96\n",
      "  138   2103 269T 269T N   CA   2047 2048    1.69    1.43    0.26    8.63    1.43    0.26    8.63\n",
      "  139   2105 269T 269T C   CA   2052 2048    1.80    1.49    0.31    8.99    1.49    0.31    8.99\n",
      "  140   2106 269T 270A C   N    2052 2054    1.56    1.35    0.22    7.63    1.35    0.22    7.63\n",
      "  141   2108 270A 270A N   CA   2054 2055    1.67    1.43    0.24    7.91    1.43    0.24    7.91\n",
      "  142   2110 270A 270A C   CA   2057 2055    1.78    1.49    0.29    8.29    1.49    0.29    8.29\n",
      "  143   2111 270A 271K C   N    2057 2059    1.54    1.35    0.20    6.97    1.35    0.20    6.97\n",
      "  144   2117 271K 271K N   CA   2059 2060    1.66    1.43    0.23    7.45    1.43    0.23    7.45\n",
      "  145   2119 271K 271K C   CA   2066 2060    1.76    1.49    0.27    7.83    1.49    0.27    7.83\n",
      "  146   2120 271K 272T C   N    2066 2068    1.53    1.35    0.19    6.71    1.35    0.19    6.71\n",
      "  147   2124 272T 272T N   CA   2068 2069    1.65    1.43    0.22    7.09    1.43    0.22    7.09\n",
      "  148   2126 272T 272T C   CA   2073 2069    1.76    1.49    0.27    7.74    1.49    0.27    7.74\n",
      "  149   2127 272T 273L C   N    2073 2075    1.53    1.35    0.18    6.56    1.35    0.18    6.56\n",
      "  150   2132 273L 273L N   CA   2075 2076    1.64    1.43    0.21    7.02    1.43    0.21    7.02\n",
      "  151   2134 273L 273L C   CA   2081 2076    1.74    1.49    0.25    7.27    1.49    0.25    7.27\n",
      "  152   2135 273L 274G C   N    2081 2083    1.52    1.35    0.17    6.10    1.35    0.17    6.10\n",
      "  153   2136 274G 274G N   CA   2083 2084    1.63    1.43    0.20    6.43    1.43    0.20    6.43\n",
      "  154   2138 274G 274G C   CA   2085 2084    1.73    1.49    0.24    6.86    1.49    0.24    6.86\n",
      "  155   2139 274G 275E C   N    2085 2087    1.51    1.35    0.16    5.72    1.35    0.16    5.72\n",
      "  156   2145 275E 275E N   CA   2087 2088    1.61    1.43    0.18    6.05    1.43    0.18    6.05\n",
      "  157   2147 275E 275E C   CA   2094 2088    1.71    1.49    0.22    6.36    1.49    0.22    6.36\n",
      "  158   2148 275E 276M C   N    2094 2096    1.50    1.35    0.15    5.41    1.35    0.15    5.41\n",
      "  159   2153 276M 276M N   CA   2096 2097    1.61    1.43    0.18    5.79    1.43    0.18    5.79\n",
      "  160   2155 276M 276M C   CA   2102 2097    1.70    1.49    0.21    6.03    1.49    0.21    6.03\n",
      "  161   2156 276M 277Q C   N    2102 2104    1.49    1.35    0.15    5.28    1.35    0.15    5.28\n",
      "  162   2162 277Q 277Q N   CA   2104 2105    1.60    1.43    0.17    5.59    1.43    0.17    5.59\n",
      "  163   2164 277Q 277Q C   CA   2111 2105    1.68    1.49    0.19    5.64    1.49    0.19    5.64\n",
      "  164   2165 277Q 278Y C   N    2111 2113    1.48    1.35    0.14    4.92    1.35    0.14    4.92\n",
      "  165   2175 278Y 278Y N   CA   2113 2114    1.59    1.43    0.16    5.26    1.43    0.16    5.26\n",
      "  166   2177 278Y 278Y C   CA   2123 2114    1.68    1.49    0.19    5.46    1.49    0.19    5.46\n",
      "  167   2178 278Y 279D C   N    2123 2125    1.47    1.35    0.13    4.57    1.35    0.13    4.57\n",
      "  168   2183 279D 279D N   CA   2125 2126    1.57    1.43    0.14    4.70    1.43    0.14    4.70\n",
      "  169   2185 279D 279D C   CA   2131 2126    1.66    1.49    0.17    4.96    1.49    0.17    4.96\n",
      "  170   4434 556F 556F C   CA   4329 4321    1.67    1.49    0.18    5.33    1.49    0.18    5.33\n",
      "  171   4435 556F 557V C   N    4329 4331    1.49    1.35    0.14    5.03    1.35    0.14    5.03\n",
      "  172   4439 557V 557V N   CA   4331 4332    1.59    1.43    0.16    5.42    1.43    0.16    5.42\n",
      "  173   4441 557V 557V C   CA   4336 4332    1.70    1.49    0.21    6.17    1.49    0.21    6.17\n",
      "  174   4442 557V 558D C   N    4336 4338    1.50    1.35    0.15    5.50    1.35    0.15    5.50\n",
      "  175   4447 558D 558D N   CA   4338 4339    1.61    1.43    0.18    5.77    1.43    0.18    5.77\n",
      "  176   4449 558D 558D C   CA   4344 4339    1.73    1.49    0.24    6.84    1.49    0.24    6.84\n",
      "  177   4450 558D 559V C   N    4344 4346    1.52    1.35    0.17    6.13    1.35    0.17    6.13\n",
      "  178   4454 559V 559V N   CA   4346 4347    1.63    1.43    0.20    6.42    1.43    0.20    6.42\n",
      "  179   4456 559V 559V C   CA   4351 4347    1.77    1.49    0.28    8.21    1.49    0.28    8.21\n",
      "  180   4457 559V 560P C   N    4351 4353    1.57    1.30    0.27    7.96    1.30    0.27    7.96\n",
      "  181   4459 560P 560P C   CA   4358 4354    1.79    1.49    0.30    8.59    1.49    0.30    8.59\n",
      "  182   4460 560P 561L C   N    4358 4360    1.57    1.35    0.23    8.01    1.35    0.23    8.01\n",
      "  183   4461 560P 560P N   CA   4353 4354    1.68    1.43    0.25    8.23    1.43    0.25    8.23\n",
      "  184   4470 561L 561L N   CA   4360 4361    1.69    1.43    0.26    8.67    1.43    0.26    8.67\n",
      "  185   4472 561L 561L C   CA   4366 4361    1.83    1.49    0.34    9.85    1.49    0.34    9.85\n",
      "  186   4473 561L 562E C   N    4366 4368    1.59    1.35    0.24    8.60    1.35    0.24    8.60\n",
      "  187   4479 562E 562E N   CA   4368 4369    1.72    1.43    0.29    9.38    1.43    0.29    9.38\n",
      "  188   4481 562E 562E C   CA   4375 4369    1.88    1.49    0.39   11.47    1.49    0.39   11.47\n",
      "  189   4482 562E 563L C   N    4375 4377    1.63    1.35    0.28    9.94    1.35    0.28    9.94\n",
      "  190   4487 563L 563L N   CA   4377 4378    1.76    1.43    0.33   10.79    1.43    0.33   10.79\n",
      "  191   4489 563L 563L C   CA   4383 4378    1.92    1.49    0.43   12.44    1.49    0.43   12.44\n",
      "  192   4490 563L 564D C   N    4383 4385    1.64    1.35    0.30   10.61    1.35    0.30   10.61\n",
      "  193   4495 564D 564D N   CA   4385 4386    1.78    1.43    0.35   11.66    1.43    0.35   11.66\n",
      "  194   4497 564D 564D C   CA   4391 4386    1.94    1.49    0.45   13.18    1.49    0.45   13.18\n",
      "  195   4498 564D 565L C   N    4391 4393    1.67    1.35    0.32   11.42    1.35    0.32   11.42\n",
      "  196   4503 565L 565L N   CA   4393 4394    1.81    1.43    0.38   12.38    1.43    0.38   12.38\n",
      "  197   4505 565L 565L C   CA   4399 4394    1.98    1.49    0.49   14.37    1.49    0.49   14.37\n",
      "  198   4506 565L 566E C   N    4399 4401    1.69    1.35    0.34   12.12    1.35    0.34   12.12\n",
      "  199   4512 566E 566E N   CA   4401 4402    1.83    1.43    0.40   13.13    1.43    0.40   13.13\n",
      "  200   4514 566E 566E C   CA   4408 4402    2.01    1.49    0.52   15.25    1.49    0.52   15.25\n",
      "  201   4515 566E 567F C   N    4408 4410    1.70    1.35    0.36   12.76    1.35    0.36   12.76\n",
      "  202   4524 567F 567F N   CA   4410 4411    1.85    1.43    0.42   13.85    1.43    0.42   13.85\n",
      "  203   4526 567F 567F C   CA   4419 4411    2.06    1.49    0.57   16.45    1.49    0.57   16.45\n",
      "  204   4527 567F 568L C   N    4419 4421    1.74    1.35    0.39   13.85    1.35    0.39   13.85\n",
      "  205   4532 568L 568L N   CA   4421 4422    1.90    1.43    0.47   15.32    1.43    0.47   15.32\n",
      "  206   4534 568L 568L C   CA   4427 4422    2.07    1.49    0.58   16.76    1.49    0.58   16.76\n",
      "  207   4535 568L 569R C   N    4427 4429    1.76    1.35    0.41   14.61    1.35    0.41   14.61\n",
      "  208   4543 569R 569R N   CA   4429 4430    1.92    1.43    0.49   16.02    1.43    0.49   16.02\n",
      "  209   4545 569R 569R C   CA   4438 4430    2.12    1.49    0.63   18.45    1.49    0.63   18.45\n",
      "  210   4546 569R 570G C   N    4438 4440    1.78    1.35    0.44   15.52    1.35    0.44   15.52\n",
      "  211   4547 570G 570G N   CA   4440 4441    1.95    1.43    0.52   17.02    1.43    0.52   17.02\n",
      "  212   4549 570G 570G C   CA   4442 4441    2.14    1.49    0.65   18.85    1.49    0.65   18.85\n",
      "  213   4550 570G 571K C   N    4442 4444    1.79    1.35    0.44   15.71    1.35    0.44   15.71\n",
      "  214   4556 571K 571K N   CA   4444 4445    1.95    1.43    0.52   17.03    1.43    0.52   17.03\n",
      "  215   4558 571K 571K C   CA   4451 4445    2.15    1.49    0.66   19.22    1.49    0.66   19.22\n",
      "  216   4559 571K 572G C   N    4451 4453    1.79    1.35    0.45   15.88    1.35    0.45   15.88\n",
      "  217   4560 572G 572G N   CA   4453 4454    1.96    1.43    0.53   17.34    1.43    0.53   17.34\n",
      "  218   4562 572G 572G C   CA   4455 4454    2.15    1.49    0.66   19.14    1.49    0.66   19.14\n",
      "  219   4563 572G 573L C   N    4455 4457    1.79    1.35    0.44   15.76    1.35    0.44   15.76\n",
      "  220   4568 573L 573L N   CA   4457 4458    1.95    1.43    0.52   17.06    1.43    0.52   17.06\n",
      "  221   4570 573L 573L C   CA   4463 4458    2.14    1.49    0.65   18.83    1.49    0.65   18.83\n",
      "  222   4571 573L 574Q C   N    4463 4465    1.77    1.35    0.43   15.10    1.35    0.43   15.10\n",
      "  223   4577 574Q 574Q N   CA   4465 4466    1.93    1.43    0.50   16.45    1.43    0.50   16.45\n",
      "  224   4579 574Q 574Q C   CA   4472 4466    2.06    1.49    0.57   16.52    1.49    0.57   16.52\n",
      "  225   4580 574Q 575S C   N    4472 4474    1.73    1.35    0.38   13.54    1.35    0.38   13.54\n",
      "  226   4583 575S 575S N   CA   4474 4475    1.88    1.43    0.45   14.80    1.43    0.45   14.80\n",
      "  227   4585 575S 575S C   CA   4478 4475    2.07    1.49    0.58   16.93    1.49    0.58   16.93\n",
      "  228   4586 575S 576D C   N    4478 4480    1.79    1.35    0.44   15.62    1.35    0.44   15.62\n",
      "  229   4591 576D 576D N   CA   4480 4481    1.97    1.43    0.54   17.74    1.43    0.54   17.74\n",
      "  230   4593 576D 576D C   CA   4486 4481    2.21    1.49    0.72   21.03    1.49    0.72   21.03\n",
      "  231   4594 576D 577E C   N    4486 4488    1.87    1.35    0.53   18.77    1.35    0.53   18.77\n",
      "  232   4600 577E 577E N   CA   4488 4489    2.06    1.43    0.63   20.84    1.43    0.63   20.84\n",
      "  233   4602 577E 577E C   CA   4495 4489    2.32    1.49    0.83   24.07    1.49    0.83   24.07\n",
      "  234   4603 577E 578V C   N    4495 4497    1.90    1.35    0.56   19.75    1.35    0.56   19.75\n",
      "  235   4607 578V 578V N   CA   4497 4498    2.11    1.43    0.68   22.34    1.43    0.68   22.34\n",
      "  236   4609 578V 578V C   CA   4502 4498    2.37    1.49    0.88   25.48    1.49    0.88   25.48\n",
      "  237   4610 578V 579S C   N    4502 4504    1.97    1.35    0.62   22.13    1.35    0.62   22.13\n",
      "  238   4613 579S 579S N   CA   4504 4505    2.18    1.43    0.75   24.78    1.43    0.75   24.78\n",
      "  239   4615 579S 579S C   CA   4508 4505    2.48    1.49    0.99   28.70    1.49    0.99   28.70\n",
      "  240   4616 579S 580M C   N    4508 4510    2.04    1.35    0.69   24.60    1.35    0.69   24.60\n",
      "  241   4621 580M 580M N   CA   4510 4511    2.27    1.43    0.84   27.64    1.43    0.84   27.64\n",
      "  242   4623 580M 580M C   CA   4516 4511    2.55    1.49    1.06   30.84    1.49    1.06   30.84\n",
      "  243   4624 580M 581P C   N    4516 4518    2.43    1.30    1.13   33.65    1.30    1.13   33.65\n",
      "  244   4626 581P 581P C   CA   4523 4519    2.77    1.49    1.28   37.30    1.49    1.28   37.30\n",
      "  245   4627 581P 582E C   N    4523 4525    2.24    1.35    0.90   31.77    1.35    0.90   31.77\n",
      "  246   4628 581P 581P N   CA   4518 4519    2.38    1.43    0.95   31.20    1.43    0.95   31.20\n",
      "  247   4638 582E 582E N   CA   4525 4526    2.52    1.43    1.09   35.90    1.43    1.09   35.90\n",
      "  248   4640 582E 582E C   CA   4532 4526    2.97    1.49    1.48   42.98    1.49    1.48   42.98\n",
      "  249   4641 582E 583A C   N    4532 4534    2.39    1.35    1.04   36.89    1.35    1.04   36.89\n",
      "  250   4643 583A 583A N   CA   4534 4535    2.67    1.43    1.24   40.90    1.43    1.24   40.90\n",
      "  251   4645 583A 583A C   CA   4537 4535    3.16    1.49    1.67   48.52    1.49    1.67   48.52\n",
      "  252   4646 583A 584Q C   N    4537 4539    2.50    1.35    1.16   41.11    1.35    1.16   41.11\n",
      "  253   4652 584Q 584Q N   CA   4539 4540    2.81    1.43    1.38   45.29    1.43    1.38   45.29\n",
      "  254   4654 584Q 584Q C   CA   4546 4540    3.42    1.49    1.93   55.96    1.49    1.93   55.96\n",
      "  255   4655 584Q 585E C   N    4546 4548    2.69    1.35    1.34   47.54    1.35    1.34   47.54\n",
      "  256   4661 585E 585E N   CA   4548 4549    3.04    1.43    1.61   52.93    1.43    1.61   52.93\n",
      "  257   4663 585E 585E C   CA   4555 4549    3.61    1.49    2.12   61.71    1.49    2.12   61.71\n",
      "  258   4664 585E 586D C   N    4555 4557    2.87    1.35    1.52   53.94    1.35    1.52   53.94\n",
      "  259   4669 586D 586D N   CA   4557 4558    3.24    1.43    1.81   59.52    1.43    1.81   59.52\n",
      "  260   4671 586D 586D C   CA   4563 4558    3.96    1.49    2.47   71.69    1.49    2.47   71.69\n",
      "  261   4672 586D 587L C   N    4563 4565    3.05    1.35    1.70   60.36    1.35    1.70   60.36\n",
      "  262   4677 587L 587L N   CA   4565 4566    3.44    1.43    2.01   65.99    1.43    2.01   65.99\n",
      "  263   4679 587L 587L C   CA   4571 4566    4.09    1.49    2.60   75.52    1.49    2.60   75.52\n",
      "  264   4680 587L 588P C   N    4571 4573    3.85    1.30    2.55   75.74    1.30    2.55   75.74\n",
      "  265   4682 588P 588P C   CA   4578 4574    4.39    1.49    2.90   84.25    1.49    2.90   84.25\n",
      "  266   4683 588P 589H C   N    4578 4580    3.35    1.35    2.01   71.18    1.35    2.01   71.18\n",
      "  267   4684 588P 588P N   CA   4573 4574    3.51    1.43    2.07   68.20    1.43    2.07   68.20\n",
      "  268   4686 588P 588P CB  CG   4576 4577    1.70    1.54    0.16    4.53    1.54    0.16    4.53\n",
      "  269   4695 589H 589H N   CA   4580 4581    3.81    1.43    2.38   78.25    1.43    2.38   78.25\n",
      "  270   4697 589H 589H C   CA   4588 4581    4.58    1.49    3.09   89.82    1.49    3.09   89.82\n",
      "  271   4698 589H 590K C   N    4588 4590    3.50    1.35    2.15   76.25    1.35    2.15   76.25\n",
      "  272   4705 590K 590K N   CA   4590 4591    3.98    1.43    2.55   83.77    1.43    2.55   83.77\n",
      "  273   4707 590K 590K C   CA   4597 4591    4.90    1.49    3.41   99.19    1.49    3.41   99.19\n",
      "  274   4708 590K 591P C   N    4597 4599    4.64    1.30    3.34   99.08    1.30    3.34   99.08\n",
      "  275   4710 591P 591P C   CA   4604 4600    5.04    1.49    3.55  103.30    1.49    3.55  103.30\n",
      "  276   4711 591P 592T C   N    4604 4606    3.81    1.35    2.46   87.29    1.35    2.46   87.29\n",
      "  277   4712 591P 591P N   CA   4599 4600    4.05    1.43    2.61   85.90    1.43    2.61   85.90\n",
      "  278   4714 591P 591P CB  CG   4602 4603    1.73    1.54    0.20    5.37    1.54    0.20    5.37\n",
      "  279   4715 591P 591P CG  CD   4603 4601    1.73    1.54    0.19    5.18    1.54    0.19    5.18\n",
      "  280   4717 592T 592T CB  CA   4608 4607    1.66    1.50    0.16    4.53    1.50    0.16    4.53\n",
      "  281   4720 592T 592T N   CA   4606 4607    4.32    1.43    2.89   95.11    1.43    2.89   95.11\n",
      "  282   4722 592T 592T C   CA   4611 4607    5.35    1.49    3.86  112.11    1.49    3.86  112.11\n",
      "  283   4723 592T 593S C   N    4611 4613    4.01    1.35    2.66   94.39    1.35    2.66   94.39\n",
      "  284   4726 593S 593S N   CA   4613 4614    4.57    1.43    3.14  103.16    1.43    3.14  103.16\n",
      "  285   4728 593S 593S C   CA   4617 4614    5.60    1.49    4.11  119.49    1.49    4.11  119.49\n",
      "  286   4729 593S 594S C   N    4617 4619    4.19    1.35    2.85  101.05    1.35    2.85  101.05\n",
      "  287   4732 594S 594S N   CA   4619 4620    4.79    1.43    3.36  110.64    1.43    3.36  110.64\n",
      "  288   4734 594S 594S C   CA   4623 4620    5.85    1.49    4.36  126.86    1.49    4.36  126.86\n",
      "  289   4735 594S 595S C   N    4623 4625    4.38    1.35    3.04  107.68    1.35    3.04  107.68\n",
      "  290   4738 595S 595S N   CA   4625 4626    5.01    1.43    3.58  117.88    1.43    3.58  117.88\n",
      "  291   4740 595S 595S C   CA   4629 4626    6.24    1.49    4.75  138.11    1.49    4.75  138.11\n",
      "  292   4741 595S 596M C   N    4629 4631    4.59    1.35    3.25  115.09    1.35    3.25  115.09\n",
      "  293   4746 596M 596M N   CA   4631 4632    5.23    1.43    3.80  125.15    1.43    3.80  125.15\n",
      "  294   4748 596M 596M C   CA   4637 4632    6.38    1.49    4.89  142.07    1.49    4.89  142.07\n",
      "  295   4749 596M 597K C   N    4637 4639    4.65    1.35    3.30  117.07    1.35    3.30  117.07\n",
      "  296   4755 597K 597K N   CA   4639 4640    5.29    1.43    3.86  126.86    1.43    3.86  126.86\n",
      "  297   4757 597K 597K C   CA   4646 4640    6.49    1.49    5.00  145.33    1.49    5.00  145.33\n",
      "  298   4758 597K 598T C   N    4646 4648    4.74    1.35    3.39  120.38    1.35    3.39  120.38\n",
      "  299   4762 598T 598T N   CA   4648 4649    5.40    1.43    3.97  130.59    1.43    3.97  130.59\n",
      "  300   4764 598T 598T C   CA   4653 4649    6.71    1.49    5.22  151.70    1.49    5.22  151.70\n",
      "  301   4765 598T 599V C   N    4653 4655    4.88    1.35    3.54  125.51    1.35    3.54  125.51\n",
      "  302   4769 599V 599V N   CA   4655 4656    5.57    1.43    4.14  136.17    1.43    4.14  136.17\n",
      "  303   4771 599V 599V C   CA   4660 4656    6.92    1.49    5.43  157.86    1.49    5.43  157.86\n",
      "  304   4772 599V 600D C   N    4660 4662    5.04    1.35    3.69  131.02    1.35    3.69  131.02\n",
      "  305   4777 600D 600D N   CA   4662 4663    5.76    1.43    4.33  142.52    1.43    4.33  142.52\n",
      "  306   4779 600D 600D C   CA   4668 4663    7.05    1.49    5.56  161.70    1.49    5.56  161.70\n",
      "  307   4780 600D 601E C   N    4668 4670    5.11    1.35    3.76  133.34    1.35    3.76  133.34\n",
      "  308   4786 601E 601E N   CA   4670 4671    5.82    1.43    4.39  144.38    1.43    4.39  144.38\n",
      "  309   4788 601E 601E C   CA   4677 4671    7.20    1.49    5.71  166.03    1.49    5.71  166.03\n",
      "  310   4789 601E 602E C   N    4677 4679    5.21    1.35    3.87  137.10    1.35    3.87  137.10\n",
      "  311   4795 602E 602E N   CA   4679 4680    5.94    1.43    4.51  148.46    1.43    4.51  148.46\n",
      "  312   4797 602E 602E C   CA   4686 4680    7.35    1.49    5.86  170.27    1.49    5.86  170.27\n",
      "  313   4798 602E 603A C   N    4686 4688    5.30    1.35    3.95  140.23    1.35    3.95  140.23\n",
      "  314   4800 603A 603A N   CA   4688 4689    6.03    1.43    4.60  151.35    1.43    4.60  151.35\n",
      "  315   4802 603A 603A C   CA   4691 4689    7.35    1.49    5.86  170.46    1.49    5.86  170.46\n",
      "  316   4803 603A 604L C   N    4691 4693    5.28    1.35    3.94  139.57    1.35    3.94  139.57\n",
      "  317   4808 604L 604L N   CA   4693 4694    6.01    1.43    4.58  150.51    1.43    4.58  150.51\n",
      "  318   4810 604L 604L C   CA   4699 4694    7.19    1.49    5.70  165.63    1.49    5.70  165.63\n",
      "  319   4811 604L 605A C   N    4699 4701    5.18    1.35    3.84  136.13    1.35    3.84  136.13\n",
      "  320   4813 605A 605A N   CA   4701 4702    5.90    1.43    4.47  146.88    1.43    4.47  146.88\n",
      "  321   4815 605A 605A C   CA   4704 4702    7.19    1.49    5.70  165.83    1.49    5.70  165.83\n",
      "  322   4816 605A 606M C   N    4704 4706    5.18    1.35    3.84  136.02    1.35    3.84  136.02\n",
      "  323   4821 606M 606M N   CA   4706 4707    4.88    1.43    3.45  113.47    1.43    3.45  113.47\n",
      "  324   4822 606M 606M O   C    4713 4712    1.50    1.23    0.27   12.53    1.23    0.27   12.53\n",
      "  325   4823 606M 606M C   CA   4712 4707    3.37    1.49    1.88   54.51    1.49    1.88   54.51\n",
      "  326   4824 606M 607L C   N    4712 4714    2.03    1.35    0.69   24.44    1.35    0.69   24.44\n",
      "  327   4829 607L 607L N   CA   4714 4715    2.04    1.43    0.61   19.95    1.43    0.61   19.95\n",
      "  328   4831 607L 607L C   CA   4720 4715    1.77    1.49    0.28    8.21    1.49    0.28    8.21\n",
      "  329   4837 608L 608L N   CA   4722 4723    1.58    1.43    0.15    4.80    1.43    0.15    4.80\n",
      "  330   4839 608L 608L C   CA   4728 4723    1.69    1.49    0.20    5.68    1.49    0.20    5.68\n",
      "  331   4840 608L 609S C   N    4728 4730    1.48    1.35    0.13    4.64    1.35    0.13    4.64\n",
      "  332   4843 609S 609S N   CA   4730 4731    1.61    1.43    0.18    5.80    1.43    0.18    5.80\n",
      "  333   4845 609S 609S C   CA   4734 4731    1.68    1.49    0.19    5.45    1.49    0.19    5.45\n",
      "  334   4846 609S 610M C   N    4734 4736    1.50    1.35    0.16    5.60    1.35    0.16    5.60\n",
      "  335   4851 610M 610M N   CA   4736 4737    1.60    1.43    0.17    5.66    1.43    0.17    5.66\n",
      "  336   4853 610M 610M C   CA   4742 4737    1.71    1.49    0.22    6.47    1.49    0.22    6.47\n",
      "  337   4854 610M 611G C   N    4742 4744    1.49    1.35    0.15    5.22    1.35    0.15    5.22\n",
      "  338   4855 611G 611G N   CA   4744 4745    1.59    1.43    0.16    5.36    1.43    0.16    5.36\n",
      "  339   4857 611G 611G C   CA   4746 4745    1.68    1.49    0.19    5.50    1.49    0.19    5.50\n",
      "  340   4858 611G 612I C   N    4746 4748    1.47    1.35    0.13    4.55    1.35    0.13    4.55\n",
      "  341   4863 612I 612I N   CA   4748 4749    1.57    1.43    0.14    4.66    1.43    0.14    4.66\n",
      "  342   4865 612I 612I C   CA   4754 4749    1.68    1.49    0.19    5.55    1.49    0.19    5.55\n",
      "  343   4872 613E 613E N   CA   4756 4757    1.57    1.43    0.14    4.50    1.43    0.14    4.50\n",
      "  344   4874 613E 613E C   CA   4763 4757    1.66    1.49    0.17    4.96    1.49    0.17    4.96\n",
      "  345   4875 613E 614E C   N    4763 4765    1.48    1.35    0.13    4.67    1.35    0.13    4.67\n",
      "  346   4881 614E 614E N   CA   4765 4766    1.58    1.43    0.15    4.84    1.43    0.15    4.84\n",
      "  347   4883 614E 614E C   CA   4772 4766    1.68    1.49    0.19    5.62    1.49    0.19    5.62\n",
      "  348   4884 614E 615T C   N    4772 4774    1.49    1.35    0.14    5.08    1.35    0.14    5.08\n",
      "  349   4888 615T 615T N   CA   4774 4775    1.59    1.43    0.16    5.28    1.43    0.16    5.28\n",
      "  350   4890 615T 615T C   CA   4779 4775    1.69    1.49    0.20    5.86    1.49    0.20    5.86\n",
      "  351   4891 615T 616V C   N    4779 4781    1.49    1.35    0.14    5.00    1.35    0.14    5.00\n",
      "  352   4895 616V 616V N   CA   4781 4782    1.59    1.43    0.16    5.29    1.43    0.16    5.29\n",
      "  353   4897 616V 616V C   CA   4786 4782    1.67    1.49    0.18    5.33    1.49    0.18    5.33\n",
      "  354   4898 616V 617A C   N    4786 4788    1.47    1.35    0.13    4.56    1.35    0.13    4.56\n",
      "  355   4900 617A 617A N   CA   4788 4789    1.57    1.43    0.14    4.77    1.43    0.14    4.77\n",
      "  356   4902 617A 617A C   CA   4791 4789    1.66    1.49    0.17    5.09    1.49    0.17    5.09\n",
      "  357   4911 618R 618R N   CA   4793 4794    1.57    1.43    0.14    4.57    1.43    0.14    4.57\n",
      "  358   4913 618R 618R C   CA   4802 4794    1.65    1.49    0.16    4.56    1.49    0.16    4.56\n",
      "  359   5535 699W 699W C   CA   5412 5401    1.67    1.49    0.18    5.23    1.49    0.18    5.23\n",
      "  360   5536 699W 700L C   N    5412 5414    1.47    1.35    0.13    4.57    1.35    0.13    4.57\n",
      "  361   5544 700L 700L N   CA   5414 5415    1.58    1.43    0.15    5.06    1.43    0.15    5.06\n",
      "  362   5546 700L 700L C   CA   5420 5415    1.68    1.49    0.19    5.53    1.49    0.19    5.53\n",
      "  363   5547 700L 701L C   N    5420 5422    1.49    1.35    0.14    4.98    1.35    0.14    4.98\n",
      "  364   5552 701L 701L N   CA   5422 5423    1.59    1.43    0.16    5.33    1.43    0.16    5.33\n",
      "  365   5554 701L 701L C   CA   5428 5423    1.70    1.49    0.21    5.99    1.49    0.21    5.99\n",
      "  366   5555 701L 702F C   N    5428 5430    1.50    1.35    0.15    5.34    1.35    0.15    5.34\n",
      "  367   5564 702F 702F N   CA   5430 5431    1.61    1.43    0.18    5.80    1.43    0.18    5.80\n",
      "  368   5566 702F 702F C   CA   5439 5431    1.72    1.49    0.23    6.75    1.49    0.23    6.75\n",
      "  369   5567 702F 703N C   N    5439 5441    1.52    1.35    0.17    6.04    1.35    0.17    6.04\n",
      "  370   5572 703N 703N N   CA   5441 5442    1.63    1.43    0.20    6.66    1.43    0.20    6.66\n",
      "  371   5574 703N 703N C   CA   5447 5442    1.72    1.49    0.23    6.82    1.49    0.23    6.82\n",
      "  372   5575 703N 704D C   N    5447 5449    1.52    1.35    0.17    6.09    1.35    0.17    6.09\n",
      "  373   5580 704D 704D N   CA   5449 5450    1.63    1.43    0.20    6.48    1.43    0.20    6.48\n",
      "  374   5582 704D 704D C   CA   5455 5450    1.74    1.49    0.25    7.26    1.49    0.25    7.26\n",
      "  375   5583 704D 705E C   N    5455 5457    1.52    1.35    0.17    6.10    1.35    0.17    6.10\n",
      "  376   5589 705E 705E N   CA   5457 5458    1.63    1.43    0.20    6.65    1.43    0.20    6.65\n",
      "  377   5591 705E 705E C   CA   5464 5458    1.74    1.49    0.25    7.29    1.49    0.25    7.29\n",
      "  378   5592 705E 706K C   N    5464 5466    1.54    1.35    0.19    6.80    1.35    0.19    6.80\n",
      "  379   5598 706K 706K N   CA   5466 5467    1.65    1.43    0.22    7.27    1.43    0.22    7.27\n",
      "  380   5600 706K 706K C   CA   5473 5467    1.77    1.49    0.28    8.25    1.49    0.28    8.25\n",
      "  381   5601 706K 707V C   N    5473 5475    1.54    1.35    0.19    6.83    1.35    0.19    6.83\n",
      "  382   5605 707V 707V N   CA   5475 5476    1.65    1.43    0.22    7.34    1.43    0.22    7.34\n",
      "  383   5607 707V 707V C   CA   5480 5476    1.80    1.49    0.31    9.03    1.49    0.31    9.03\n",
      "  384   5608 707V 708A C   N    5480 5482    1.54    1.35    0.20    7.04    1.35    0.20    7.04\n",
      "  385   5610 708A 708A N   CA   5482 5483    1.66    1.43    0.23    7.57    1.43    0.23    7.57\n",
      "  386   5612 708A 708A C   CA   5485 5483    1.78    1.49    0.29    8.33    1.49    0.29    8.33\n",
      "  387   5613 708A 709E C   N    5485 5487    1.56    1.35    0.22    7.78    1.35    0.22    7.78\n",
      "  388   5619 709E 709E N   CA   5487 5488    1.69    1.43    0.26    8.41    1.43    0.26    8.41\n",
      "  389   5621 709E 709E C   CA   5494 5488    1.83    1.49    0.34   10.00    1.49    0.34   10.00\n",
      "  390   5622 709E 710S C   N    5494 5496    1.59    1.35    0.24    8.53    1.35    0.24    8.53\n",
      "  391   5625 710S 710S N   CA   5496 5497    1.71    1.43    0.28    9.27    1.43    0.28    9.27\n",
      "  392   5627 710S 710S C   CA   5500 5497    1.81    1.49    0.32    9.22    1.49    0.32    9.22\n",
      "  393   5628 710S 711L C   N    5500 5502    1.55    1.35    0.21    7.41    1.35    0.21    7.41\n",
      "  394   5633 711L 711L N   CA   5502 5503    1.69    1.43    0.26    8.41    1.43    0.26    8.41\n",
      "  395   5635 711L 711L C   CA   5508 5503    1.83    1.49    0.34    9.98    1.49    0.34    9.98\n",
      "  396   5636 711L 712N C   N    5508 5510    1.61    1.35    0.27    9.49    1.35    0.27    9.49\n",
      "  397   5641 712N 712N N   CA   5510 5511    1.75    1.43    0.32   10.38    1.43    0.32   10.38\n",
      "  398   5643 712N 712N C   CA   5516 5511    1.97    1.49    0.48   14.06    1.49    0.48   14.06\n",
      "  399   5644 712N 713P C   N    5516 5518    1.80    1.30    0.50   14.71    1.30    0.50   14.71\n",
      "  400   5646 713P 713P C   CA   5523 5519    2.01    1.49    0.52   15.08    1.49    0.52   15.08\n",
      "  401   5647 713P 714P C   N    5523 5525    1.84    1.30    0.54   15.98    1.30    0.54   15.98\n",
      "  402   5648 713P 713P N   CA   5518 5519    1.85    1.43    0.41   13.54    1.43    0.41   13.54\n",
      "  403   5654 714P 714P C   CA   5530 5526    2.06    1.49    0.57   16.51    1.49    0.57   16.51\n",
      "  404   5655 714P 715F C   N    5530 5532    1.70    1.35    0.35   12.47    1.35    0.35   12.47\n",
      "  405   5656 714P 714P N   CA   5525 5526    1.84    1.43    0.41   13.39    1.43    0.41   13.39\n",
      "  406   5669 715F 715F N   CA   5532 5533    1.84    1.43    0.41   13.60    1.43    0.41   13.60\n",
      "  407   5671 715F 715F C   CA   5541 5533    2.05    1.49    0.56   16.38    1.49    0.56   16.38\n",
      "  408   5672 715F 716S C   N    5541 5543    1.74    1.35    0.39   13.98    1.35    0.39   13.98\n",
      "  409   5675 716S 716S N   CA   5543 5544    1.91    1.43    0.48   15.81    1.43    0.48   15.81\n",
      "  410   5677 716S 716S C   CA   5547 5544    2.11    1.49    0.62   17.91    1.49    0.62   17.91\n",
      "  411   5678 716S 717M C   N    5547 5549    1.78    1.35    0.44   15.47    1.35    0.44   15.47\n",
      "  412   5683 717M 717M N   CA   5549 5550    1.93    1.43    0.50   16.42    1.43    0.50   16.42\n",
      "  413   5685 717M 717M C   CA   5555 5550    2.19    1.49    0.70   20.26    1.49    0.70   20.26\n",
      "  414   5686 717M 718A C   N    5555 5557    1.76    1.35    0.41   14.65    1.35    0.41   14.65\n",
      "  415   5688 718A 718A N   CA   5557 5558    1.93    1.43    0.50   16.51    1.43    0.50   16.51\n",
      "  416   5690 718A 718A C   CA   5560 5558    2.11    1.49    0.62   17.96    1.49    0.62   17.96\n",
      "  417   5691 718A 719F C   N    5560 5562    1.75    1.35    0.40   14.22    1.35    0.40   14.22\n",
      "  418   5700 719F 719F N   CA   5562 5563    1.93    1.43    0.50   16.31    1.43    0.50   16.31\n",
      "  419   5702 719F 719F C   CA   5571 5563    2.24    1.49    0.75   21.77    1.49    0.75   21.77\n",
      "  420   5703 719F 720L C   N    5571 5573    1.86    1.35    0.52   18.29    1.35    0.52   18.29\n",
      "  421   5708 720L 720L N   CA   5573 5574    2.07    1.43    0.64   20.90    1.43    0.64   20.90\n",
      "  422   5710 720L 720L C   CA   5579 5574    2.33    1.49    0.84   24.56    1.49    0.84   24.56\n",
      "  423   5711 720L 721Y C   N    5579 5581    1.97    1.35    0.63   22.19    1.35    0.63   22.19\n",
      "  424   5721 721Y 721Y N   CA   5581 5582    2.19    1.43    0.76   25.02    1.43    0.76   25.02\n",
      "  425   5723 721Y 721Y C   CA   5591 5582    2.53    1.49    1.04   30.22    1.49    1.04   30.22\n",
      "  426   5724 721Y 722F C   N    5591 5593    2.10    1.35    0.76   26.83    1.35    0.76   26.83\n",
      "  427   5733 722F 722F N   CA   5593 5594    2.36    1.43    0.93   30.58    1.43    0.93   30.58\n",
      "  428   5735 722F 722F C   CA   5602 5594    2.83    1.49    1.34   39.03    1.49    1.34   39.03\n",
      "  429   5736 722F 723Y C   N    5602 5604    2.31    1.35    0.97   34.29    1.35    0.97   34.29\n",
      "  430   5746 723Y 723Y N   CA   5604 5605    2.58    1.43    1.15   37.99    1.43    1.15   37.99\n",
      "  431   5748 723Y 723Y C   CA   5614 5605    3.05    1.49    1.56   45.32    1.49    1.56   45.32\n",
      "  432   5749 723Y 724K C   N    5614 5616    2.42    1.35    1.07   38.09    1.35    1.07   38.09\n",
      "  433   5755 724K 724K N   CA   5616 5617    2.69    1.43    1.26   41.40    1.43    1.26   41.40\n",
      "  434   5757 724K 724K C   CA   5623 5617    3.19    1.49    1.70   49.42    1.49    1.70   49.42\n",
      "  435   5758 724K 725R C   N    5623 5625    2.51    1.35    1.16   41.22    1.35    1.16   41.22\n",
      "  436   5766 725R 725R N   CA   5625 5626    2.80    1.43    1.37   45.07    1.43    1.37   45.07\n",
      "  437   5768 725R 725R C   CA   5634 5626    3.32    1.49    1.83   53.21    1.49    1.83   53.21\n",
      "  438   5769 725R 726V C   N    5634 5636    2.53    1.35    1.18   41.98    1.35    1.18   41.98\n",
      "  439   5773 726V 726V N   CA   5636 5637    2.82    1.43    1.39   45.72    1.43    1.39   45.72\n",
      "  440   5775 726V 726V C   CA   5641 5637    3.34    1.49    1.85   53.66    1.49    1.85   53.66\n",
      "  441   5776 726V 727G C   N    5641 5643    2.60    1.35    1.26   44.56    1.35    1.26   44.56\n",
      "  442   5777 727G 727G N   CA   5643 5644    2.91    1.43    1.48   48.62    1.43    1.48   48.62\n",
      "  443   5779 727G 727G C   CA   5645 5644    3.38    1.49    1.89   54.81    1.49    1.89   54.81\n",
      "  444   5780 727G 728K C   N    5645 5647    2.61    1.35    1.27   44.96    1.35    1.27   44.96\n",
      "  445   5786 728K 728K N   CA   5647 5648    2.47    1.43    1.04   34.08    1.43    1.04   34.08\n",
      "  446   5787 728K 728K O   C    5655 5654    1.45    1.26    0.19    8.20    1.26    0.19    8.20\n",
      "  447   5788 728K 728K C   CA   5654 5648    2.01    1.52    0.49   12.83    1.52    0.49   12.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5824   4P   4P N   CA     25   26  131.24  108.20   23.04    5.24  108.20   23.04    5.24\n",
      "    2   5895  10E  10E N   CA     74   75  123.42  107.00   16.42    4.72  107.00   16.42    4.72\n",
      "    3   5907  11A  11A N   CA     83   84  124.29  107.00   17.29    4.97  107.00   17.29    4.97\n",
      "    4   5914  12C  12C N   CA     88   89  126.42  107.00   19.42    5.58  107.00   19.42    5.58\n",
      "    5   5919  12C  13A C   N      92   94  145.55  120.00   25.55    5.81  120.00   25.55    5.81\n",
      "    6   5922  13A  13A N   CA     94   95  149.15  107.00   42.15   12.12  107.00   42.15   12.12\n",
      "    7   5924  13A  13A CA  C      95   97  136.20  116.50   19.70    5.66  116.50   19.70    5.66\n",
      "    8   5929  14H  14H N   CA     99  100  127.87  107.00   20.87    6.00  107.00   20.87    6.00\n",
      "    9   5944  15C  15C N   CA    109  110  126.05  107.00   19.05    5.48  107.00   19.05    5.48\n",
      "   10   5952  16W  16W N   CA    115  116  126.10  107.00   19.10    5.49  107.00   19.10    5.49\n",
      "   11   5975  17D  17D N   CA    129  130  129.78  107.00   22.78    6.55  107.00   22.78    6.55\n",
      "   12   5986  18E  18E N   CA    137  138  130.55  107.00   23.55    6.77  107.00   23.55    6.77\n",
      "   13   5998  19S  19S N   CA    146  147  131.26  107.00   24.26    6.98  107.00   24.26    6.98\n",
      "   14   6001  19S  19S CA  C     147  150  133.67  116.50   17.17    4.94  116.50   17.17    4.94\n",
      "   15   6006  20V  20V N   CA    152  153  133.02  107.00   26.02    7.48  107.00   26.02    7.48\n",
      "   16   6010  20V  20V CA  C     153  157  133.89  116.50   17.39    5.00  116.50   17.39    5.00\n",
      "   17   6016  21M  21M N   CA    159  160  133.93  107.00   26.93    7.74  107.00   26.93    7.74\n",
      "   18   6019  21M  21M CA  C     160  165  134.96  116.50   18.46    5.31  116.50   18.46    5.31\n",
      "   19   6026  22L  22L N   CA    167  168  134.25  107.00   27.25    7.84  107.00   27.25    7.84\n",
      "   20   6029  22L  22L CA  C     168  173  135.63  116.50   19.14    5.50  116.50   19.14    5.50\n",
      "   21   6034  22L  23V C   N     173  175  139.89  120.00   19.89    4.52  120.00   19.89    4.52\n",
      "   22   6037  23V  23V N   CA    175  176  137.83  107.00   30.83    8.87  107.00   30.83    8.87\n",
      "   23   6044  23V  24P C   N     180  182  146.98  117.00   29.98    7.46  117.00   29.98    7.46\n",
      "   24   6047  24P  24P N   CA    182  183  145.84  108.20   37.65    8.56  108.20   37.65    8.56\n",
      "   25   6051  24P  24P CA  C     183  187  136.93  116.50   20.44    5.88  116.50   20.44    5.88\n",
      "   26   6056  24P  25N C   N     187  189  141.68  120.00   21.68    4.93  120.00   21.68    4.93\n",
      "   27   6059  25N  25N N   CA    189  190  136.65  107.00   29.65    8.52  107.00   29.65    8.52\n",
      "   28   6062  25N  25N CA  C     190  195  137.54  116.50   21.04    6.05  116.50   21.04    6.05\n",
      "   29   6067  25N  26H C   N     195  197  141.84  120.00   21.84    4.96  120.00   21.84    4.96\n",
      "   30   6070  26H  26H N   CA    197  198  138.42  107.00   31.42    9.04  107.00   31.42    9.04\n",
      "   31   6072  26H  26H CA  C     198  205  138.46  116.50   21.96    6.31  116.50   21.96    6.31\n",
      "   32   6082  26H  27R C   N     205  207  142.45  120.00   22.45    5.10  120.00   22.45    5.10\n",
      "   33   6085  27R  27R N   CA    207  208  138.88  107.00   31.88    9.17  107.00   31.88    9.17\n",
      "   34   6088  27R  27R CA  C     208  216  139.48  116.50   22.98    6.61  116.50   22.98    6.61\n",
      "   35   6096  27R  28S C   N     216  218  143.65  120.00   23.65    5.37  120.00   23.65    5.37\n",
      "   36   6099  28S  28S N   CA    218  219  139.89  107.00   32.89    9.46  107.00   32.89    9.46\n",
      "   37   6102  28S  28S CA  C     219  222  139.43  116.50   22.93    6.59  116.50   22.93    6.59\n",
      "   38   6104  28S  29A C   N     222  224  144.64  120.00   24.64    5.60  120.00   24.64    5.60\n",
      "   39   6107  29A  29A N   CA    224  225  140.57  107.00   33.57    9.65  107.00   33.57    9.65\n",
      "   40   6109  29A  29A CA  C     225  227  139.94  116.50   23.44    6.74  116.50   23.44    6.74\n",
      "   41   6111  29A  30V C   N     227  229  144.94  120.00   24.94    5.67  120.00   24.94    5.67\n",
      "   42   6114  30V  30V N   CA    229  230  141.08  107.00   34.08    9.80  107.00   34.08    9.80\n",
      "   43   6118  30V  30V CA  C     230  234  140.65  116.50   24.15    6.94  116.50   24.15    6.94\n",
      "   44   6121  30V  31H C   N     234  236  145.12  120.00   25.12    5.71  120.00   25.12    5.71\n",
      "   45   6124  31H  31H N   CA    236  237  141.58  107.00   34.58    9.94  107.00   34.58    9.94\n",
      "   46   6126  31H  31H CA  C     237  244  141.54  116.50   25.04    7.20  116.50   25.04    7.20\n",
      "   47   6136  31H  32K C   N     244  246  146.13  120.00   26.13    5.94  120.00   26.13    5.94\n",
      "   48   6139  32K  32K N   CA    246  247  143.15  107.00   36.15   10.39  107.00   36.15   10.39\n",
      "   49   6142  32K  32K CA  C     247  253  141.63  116.50   25.13    7.23  116.50   25.13    7.23\n",
      "   50   6147  32K  33E C   N     253  255  146.98  120.00   26.98    6.13  120.00   26.98    6.13\n",
      "   51   6150  33E  33E N   CA    255  256  144.10  107.00   37.10   10.67  107.00   37.10   10.67\n",
      "   52   6153  33E  33E CA  C     256  262  141.99  116.50   25.49    7.33  116.50   25.49    7.33\n",
      "   53   6159  33E  34E C   N     262  264  147.98  120.00   27.98    6.36  120.00   27.98    6.36\n",
      "   54   6162  34E  34E N   CA    264  265  145.05  107.00   38.05   10.94  107.00   38.05   10.94\n",
      "   55   6165  34E  34E CA  C     265  271  142.49  116.50   25.99    7.47  116.50   25.99    7.47\n",
      "   56   6171  34E  35C C   N     271  273  148.78  120.00   28.78    6.54  120.00   28.78    6.54\n",
      "   57   6174  35C  35C N   CA    273  274  146.55  107.00   39.55   11.37  107.00   39.55   11.37\n",
      "   58   6177  35C  35C CA  C     274  277  142.64  116.50   26.14    7.52  116.50   26.14    7.52\n",
      "   59   6179  35C  36A C   N     277  279  148.67  120.00   28.67    6.52  120.00   28.67    6.52\n",
      "   60   6182  36A  36A N   CA    279  280  145.80  107.00   38.80   11.16  107.00   38.80   11.16\n",
      "   61   6184  36A  36A CA  C     280  282  142.97  116.50   26.47    7.61  116.50   26.47    7.61\n",
      "   62   6186  36A  37Y C   N     282  284  148.82  120.00   28.82    6.55  120.00   28.82    6.55\n",
      "   63   6189  37Y  37Y N   CA    284  285  145.92  107.00   38.92   11.19  107.00   38.92   11.19\n",
      "   64   6192  37Y  37Y CA  C     285  294  144.00  116.50   27.50    7.91  116.50   27.50    7.91\n",
      "   65   6204  37Y  38C C   N     294  296  149.64  120.00   29.64    6.74  120.00   29.64    6.74\n",
      "   66   6207  38C  38C N   CA    296  297  146.73  107.00   39.73   11.42  107.00   39.73   11.42\n",
      "   67   6210  38C  38C CA  C     297  300  144.14  116.50   27.64    7.95  116.50   27.64    7.95\n",
      "   68   6212  38C  39C C   N     300  302  149.46  120.00   29.46    6.69  120.00   29.46    6.69\n",
      "   69   6215  39C  39C N   CA    302  303  146.43  107.00   39.43   11.34  107.00   39.43   11.34\n",
      "   70   6218  39C  39C CA  C     303  306  145.08  116.50   28.58    8.22  116.50   28.58    8.22\n",
      "   71   6220  39C  40R C   N     306  308  149.03  120.00   29.03    6.60  120.00   29.03    6.60\n",
      "   72   6223  40R  40R N   CA    308  309  146.90  107.00   39.90   11.47  107.00   39.90   11.47\n",
      "   73   6226  40R  40R CA  C     309  317  145.60  116.50   29.11    8.37  116.50   29.11    8.37\n",
      "   74   6234  40R  41T C   N     317  319  149.87  120.00   29.87    6.79  120.00   29.87    6.79\n",
      "   75   6237  41T  41T N   CA    319  320  147.29  107.00   40.29   11.58  107.00   40.29   11.58\n",
      "   76   6241  41T  41T CA  C     320  324  146.05  116.50   29.56    8.50  116.50   29.56    8.50\n",
      "   77   6244  41T  42C C   N     324  326  150.37  120.00   30.37    6.90  120.00   30.37    6.90\n",
      "   78   6247  42C  42C N   CA    326  327  147.40  107.00   40.40   11.62  107.00   40.40   11.62\n",
      "   79   6250  42C  42C CA  C     327  330  146.96  116.50   30.46    8.76  116.50   30.46    8.76\n",
      "   80   6252  42C  43R C   N     330  332  150.55  120.00   30.54    6.94  120.00   30.54    6.94\n",
      "   81   6255  43R  43R N   CA    332  333  148.03  107.00   41.03   11.80  107.00   41.03   11.80\n",
      "   82   6258  43R  43R CA  C     333  341  147.31  116.50   30.81    8.86  116.50   30.81    8.86\n",
      "   83   6266  43R  44H C   N     341  343  151.70  120.00   31.70    7.20  120.00   31.70    7.20\n",
      "   84   6269  44H  44H N   CA    343  344  149.71  107.00   42.71   12.28  107.00   42.71   12.28\n",
      "   85   6271  44H  44H CA  C     344  351  147.60  116.50   31.10    8.94  116.50   31.10    8.94\n",
      "   86   6281  44H  45E C   N     351  353  151.96  120.00   31.96    7.26  120.00   31.96    7.26\n",
      "   87   6284  45E  45E N   CA    353  354  150.08  107.00   43.08   12.39  107.00   43.08   12.39\n",
      "   88   6287  45E  45E CA  C     354  360  147.99  116.50   31.49    9.05  116.50   31.49    9.05\n",
      "   89   6293  45E  46M C   N     360  362  152.80  120.00   32.80    7.45  120.00   32.80    7.45\n",
      "   90   6296  46M  46M N   CA    362  363  150.58  107.00   43.58   12.53  107.00   43.58   12.53\n",
      "   91   6299  46M  46M CA  C     363  368  148.71  116.50   32.21    9.26  116.50   32.21    9.26\n",
      "   92   6303  46M  47G C   N     368  370  152.40  120.00   32.40    7.36  120.00   32.40    7.36\n",
      "   93   6305  47G  47G N   CA    370  371  150.67  107.00   43.67   12.56  107.00   43.67   12.56\n",
      "   94   6307  47G  47G CA  C     371  372  135.26  116.50   18.76    5.40  116.50   18.76    5.40\n",
      "   95   6308  47G  48T C   N     372  374  146.42  120.00   26.42    6.00  120.00   26.42    6.00\n",
      "   96   6425  60V  60V N   CA    458  459  122.75  107.00   15.75    4.53  107.00   15.75    4.53\n",
      "   97   7426 154R 155L C   N    1175 1177  144.04  120.00   24.04    5.46  120.00   24.04    5.46\n",
      "   98   7437 155L 156Q C   N    1183 1185  141.71  120.00   21.71    4.93  120.00   21.71    4.93\n",
      "   99   7551 166L 166L N   CA   1264 1265  123.08  107.00   16.08    4.62  107.00   16.08    4.62\n",
      "  100   7582 169P 169P N   CA   1285 1286  130.43  108.20   22.23    5.05  108.20   22.23    5.05\n",
      "  101   8269 236A 236A N   CA   1777 1778  125.32  107.00   18.32    5.27  107.00   18.32    5.27\n",
      "  102   8284 237D 238F C   N    1788 1790  144.22  120.00   24.22    5.50  120.00   24.22    5.50\n",
      "  103   8396 246E 246E C   CA   1868 1862  131.28  108.00   23.27    5.39  108.00   23.27    5.39\n",
      "  104   8477 254N 254N N   CA   1924 1925  123.63  107.00   16.63    4.78  107.00   16.63    4.78\n",
      "  105   8555 260G 260G N   CA   1980 1981  132.54  107.00   25.54    7.34  107.00   25.54    7.34\n",
      "  106   8561 261I 261I N   CA   1984 1985  138.12  107.00   31.12    8.95  107.00   31.12    8.95\n",
      "  107   8565 261I 261I CA  C    1985 1990  134.98  116.50   18.49    5.32  116.50   18.49    5.32\n",
      "  108   8569 261I 262D C   N    1990 1992  142.16  120.00   22.16    5.04  120.00   22.16    5.04\n",
      "  109   8572 262D 262D N   CA   1992 1993  132.92  107.00   25.92    7.45  107.00   25.92    7.45\n",
      "  110   8580 262D 263I C   N    1998 2000  147.58  120.00   27.58    6.27  120.00   27.58    6.27\n",
      "  111   8583 263I 263I N   CA   2000 2001  139.18  107.00   32.18    9.25  107.00   32.18    9.25\n",
      "  112   8587 263I 263I CA  C    2001 2006  134.56  116.50   18.06    5.19  116.50   18.06    5.19\n",
      "  113   8591 263I 264Q C   N    2006 2008  140.59  120.00   20.59    4.68  120.00   20.59    4.68\n",
      "  114   8594 264Q 264Q N   CA   2008 2009  138.07  107.00   31.07    8.93  107.00   31.07    8.93\n",
      "  115   8597 264Q 264Q CA  C    2009 2015  132.50  116.50   16.00    4.60  116.50   16.00    4.60\n",
      "  116   8606 265T 265T N   CA   2017 2018  132.90  107.00   25.90    7.45  107.00   25.90    7.45\n",
      "  117   8610 265T 265T CA  C    2018 2022  133.06  116.50   16.56    4.76  116.50   16.56    4.76\n",
      "  118   8613 265T 266A C   N    2022 2024  139.86  120.00   19.86    4.51  120.00   19.86    4.51\n",
      "  119   8616 266A 266A N   CA   2024 2025  132.87  107.00   25.87    7.44  107.00   25.87    7.44\n",
      "  120   8623 267K 267K N   CA   2029 2030  131.57  107.00   24.57    7.06  107.00   24.57    7.06\n",
      "  121   8634 268K 268K N   CA   2038 2039  130.98  107.00   23.98    6.90  107.00   23.98    6.90\n",
      "  122   8637 268K 268K CA  C    2039 2045  132.67  116.50   16.17    4.65  116.50   16.17    4.65\n",
      "  123   8642 268K 269T C   N    2045 2047  140.17  120.00   20.17    4.58  120.00   20.17    4.58\n",
      "  124   8645 269T 269T N   CA   2047 2048  129.28  107.00   22.28    6.41  107.00   22.28    6.41\n",
      "  125   8649 269T 269T CA  C    2048 2052  132.42  116.50   15.92    4.58  116.50   15.92    4.58\n",
      "  126   8655 270A 270A N   CA   2054 2055  130.60  107.00   23.60    6.79  107.00   23.60    6.79\n",
      "  127   8662 271K 271K N   CA   2059 2060  129.95  107.00   22.95    6.60  107.00   22.95    6.60\n",
      "  128   8673 272T 272T N   CA   2068 2069  129.63  107.00   22.63    6.51  107.00   22.63    6.51\n",
      "  129   8683 273L 273L N   CA   2075 2076  128.81  107.00   21.81    6.27  107.00   21.81    6.27\n",
      "  130   8693 274G 274G N   CA   2083 2084  128.76  107.00   21.76    6.26  107.00   21.76    6.26\n",
      "  131   8699 275E 275E N   CA   2087 2088  127.37  107.00   20.37    5.86  107.00   20.37    5.86\n",
      "  132   8711 276M 276M N   CA   2096 2097  127.47  107.00   20.47    5.89  107.00   20.47    5.89\n",
      "  133   8721 277Q 277Q N   CA   2104 2105  125.65  107.00   18.65    5.36  107.00   18.65    5.36\n",
      "  134   8733 278Y 278Y N   CA   2113 2114  123.16  107.00   16.16    4.65  107.00   16.16    4.65\n",
      "  135   8751 279D 279D N   CA   2125 2126  123.33  107.00   16.33    4.69  107.00   16.33    4.69\n",
      "  136  11786 556F 556F N   CA   4320 4321  123.39  107.00   16.39    4.71  107.00   16.39    4.71\n",
      "  137  11802 557V 557V N   CA   4331 4332  123.35  107.00   16.35    4.70  107.00   16.35    4.70\n",
      "  138  11812 558D 558D N   CA   4338 4339  126.51  107.00   19.51    5.61  107.00   19.51    5.61\n",
      "  139  11823 559V 559V N   CA   4346 4347  129.94  107.00   22.94    6.59  107.00   22.94    6.59\n",
      "  140  11830 559V 560P C   N    4351 4353  142.90  117.00   25.90    6.45  117.00   25.90    6.45\n",
      "  141  11833 560P 560P N   CA   4353 4354  141.99  108.20   33.79    7.68  108.20   33.79    7.68\n",
      "  142  11837 560P 560P CA  C    4354 4358  132.65  116.50   16.15    4.64  116.50   16.15    4.64\n",
      "  143  11845 561L 561L N   CA   4360 4361  133.34  107.00   26.34    7.57  107.00   26.34    7.57\n",
      "  144  11848 561L 561L CA  C    4361 4366  134.11  116.50   17.62    5.06  116.50   17.62    5.06\n",
      "  145  11856 562E 562E N   CA   4368 4369  135.21  107.00   28.21    8.11  107.00   28.21    8.11\n",
      "  146  11859 562E 562E CA  C    4369 4375  136.57  116.50   20.07    5.77  116.50   20.07    5.77\n",
      "  147  11865 562E 563L C   N    4375 4377  141.50  120.00   21.50    4.89  120.00   21.50    4.89\n",
      "  148  11868 563L 563L N   CA   4377 4378  136.93  107.00   29.93    8.61  107.00   29.93    8.61\n",
      "  149  11871 563L 563L CA  C    4378 4383  138.31  116.50   21.81    6.27  116.50   21.81    6.27\n",
      "  150  11876 563L 564D C   N    4383 4385  142.01  120.00   22.01    5.00  120.00   22.01    5.00\n",
      "  151  11879 564D 564D N   CA   4385 4386  139.45  107.00   32.45    9.33  107.00   32.45    9.33\n",
      "  152  11882 564D 564D CA  C    4386 4391  137.81  116.50   21.31    6.13  116.50   21.31    6.13\n",
      "  153  11887 564D 565L C   N    4391 4393  143.94  120.00   23.94    5.44  120.00   23.94    5.44\n",
      "  154  11890 565L 565L N   CA   4393 4394  139.57  107.00   32.57    9.36  107.00   32.57    9.36\n",
      "  155  11893 565L 565L CA  C    4394 4399  140.39  116.50   23.89    6.87  116.50   23.89    6.87\n",
      "  156  11898 565L 566E C   N    4399 4401  143.79  120.00   23.79    5.41  120.00   23.79    5.41\n",
      "  157  11901 566E 566E N   CA   4401 4402  140.72  107.00   33.72    9.70  107.00   33.72    9.70\n",
      "  158  11904 566E 566E CA  C    4402 4408  140.90  116.50   24.40    7.02  116.50   24.40    7.02\n",
      "  159  11910 566E 567F C   N    4408 4410  145.20  120.00   25.20    5.73  120.00   25.20    5.73\n",
      "  160  11913 567F 567F N   CA   4410 4411  143.04  107.00   36.04   10.36  107.00   36.04   10.36\n",
      "  161  11916 567F 567F CA  C    4411 4419  142.26  116.50   25.76    7.41  116.50   25.76    7.41\n",
      "  162  11926 567F 568L C   N    4419 4421  148.80  120.00   28.80    6.54  120.00   28.80    6.54\n",
      "  163  11929 568L 568L N   CA   4421 4422  145.81  107.00   38.81   11.16  107.00   38.81   11.16\n",
      "  164  11932 568L 568L CA  C    4422 4427  141.39  116.50   24.89    7.16  116.50   24.89    7.16\n",
      "  165  11937 568L 569R C   N    4427 4429  149.77  120.00   29.77    6.76  120.00   29.77    6.76\n",
      "  166  11940 569R 569R N   CA   4429 4430  147.07  107.00   40.07   11.52  107.00   40.07   11.52\n",
      "  167  11943 569R 569R CA  C    4430 4438  143.78  116.50   27.28    7.84  116.50   27.28    7.84\n",
      "  168  11951 569R 570G C   N    4438 4440  150.64  120.00   30.64    6.96  120.00   30.64    6.96\n",
      "  169  11953 570G 570G N   CA   4440 4441  148.40  107.00   41.40   11.90  107.00   41.40   11.90\n",
      "  170  11955 570G 570G CA  C    4441 4442  145.08  116.50   28.58    8.22  116.50   28.58    8.22\n",
      "  171  11956 570G 571K C   N    4442 4444  148.92  120.00   28.92    6.57  120.00   28.92    6.57\n",
      "  172  11959 571K 571K N   CA   4444 4445  144.69  107.00   37.69   10.84  107.00   37.69   10.84\n",
      "  173  11962 571K 571K CA  C    4445 4451  145.10  116.50   28.60    8.22  116.50   28.60    8.22\n",
      "  174  11967 571K 572G C   N    4451 4453  150.50  120.00   30.50    6.93  120.00   30.50    6.93\n",
      "  175  11969 572G 572G N   CA   4453 4454  148.83  107.00   41.83   12.03  107.00   41.83   12.03\n",
      "  176  11971 572G 572G CA  C    4454 4455  144.14  116.50   27.64    7.95  116.50   27.64    7.95\n",
      "  177  11972 572G 573L C   N    4455 4457  149.97  120.00   29.97    6.81  120.00   29.97    6.81\n",
      "  178  11975 573L 573L N   CA   4457 4458  152.32  107.00   45.32   13.03  107.00   45.32   13.03\n",
      "  179  11978 573L 573L CA  C    4458 4463  143.10  116.50   26.60    7.65  116.50   26.60    7.65\n",
      "  180  11983 573L 574Q C   N    4463 4465  151.39  120.00   31.39    7.13  120.00   31.39    7.13\n",
      "  181  11986 574Q 574Q N   CA   4465 4466  145.62  107.00   38.62   11.11  107.00   38.62   11.11\n",
      "  182  11989 574Q 574Q CA  C    4466 4472  140.70  116.50   24.20    6.96  116.50   24.20    6.96\n",
      "  183  11995 574Q 575S C   N    4472 4474  147.53  120.00   27.53    6.26  120.00   27.53    6.26\n",
      "  184  11998 575S 575S N   CA   4474 4475  143.36  107.00   36.36   10.46  107.00   36.36   10.46\n",
      "  185  12001 575S 575S CA  C    4475 4478  144.60  116.50   28.10    8.08  116.50   28.10    8.08\n",
      "  186  12003 575S 576D C   N    4478 4480  155.94  120.00   35.94    8.17  120.00   35.94    8.17\n",
      "  187  12006 576D 576D N   CA   4480 4481  148.10  107.00   41.10   11.82  107.00   41.10   11.82\n",
      "  188  12009 576D 576D CA  C    4481 4486  148.99  116.50   32.49    9.34  116.50   32.49    9.34\n",
      "  189  12014 576D 577E C   N    4486 4488  152.59  120.00   32.59    7.41  120.00   32.59    7.41\n",
      "  190  12017 577E 577E N   CA   4488 4489  150.48  107.00   43.48   12.50  107.00   43.48   12.50\n",
      "  191  12020 577E 577E CA  C    4489 4495  149.69  116.50   33.19    9.54  116.50   33.19    9.54\n",
      "  192  12026 577E 578V C   N    4495 4497  156.99  120.00   36.99    8.41  120.00   36.99    8.41\n",
      "  193  12029 578V 578V N   CA   4497 4498  152.02  107.00   45.02   12.94  107.00   45.02   12.94\n",
      "  194  12033 578V 578V CA  C    4498 4502  151.02  116.50   34.52    9.93  116.50   34.52    9.93\n",
      "  195  12036 578V 579S C   N    4502 4504  156.06  120.00   36.06    8.20  120.00   36.06    8.20\n",
      "  196  12039 579S 579S N   CA   4504 4505  162.42  107.00   55.42   15.94  107.00   55.42   15.94\n",
      "  197  12042 579S 579S CA  C    4505 4508  157.98  116.50   41.48   11.93  116.50   41.48   11.93\n",
      "  198  12044 579S 580M C   N    4508 4510  159.51  120.00   39.51    8.98  120.00   39.51    8.98\n",
      "  199  12047 580M 580M N   CA   4510 4511  159.02  107.00   52.02   14.96  107.00   52.02   14.96\n",
      "  200  12050 580M 580M CA  C    4511 4516  164.93  112.50   52.43    7.53  112.50   52.43    7.53\n",
      "  201  12054 580M 581P C   N    4516 4518  166.03  117.00   49.03   12.21  117.00   49.03   12.21\n",
      "  202  12057 581P 581P N   CA   4518 4519  173.48  108.20   65.28   14.84  108.20   65.28   14.84\n",
      "  203  12061 581P 581P CA  C    4519 4523  163.14  116.50   46.65   13.41  116.50   46.65   13.41\n",
      "  204  12066 581P 582E C   N    4523 4525  171.94  120.00   51.94   11.80  120.00   51.94   11.80\n",
      "  205  12069 582E 582E N   CA   4525 4526  167.47  107.00   60.47   17.39  107.00   60.47   17.39\n",
      "  206  12072 582E 582E CA  C    4526 4532  162.05  116.50   45.55   13.10  116.50   45.55   13.10\n",
      "  207  12078 582E 583A C   N    4532 4534  164.88  120.00   44.88   10.20  120.00   44.88   10.20\n",
      "  208  12081 583A 583A N   CA   4534 4535  166.70  107.00   59.70   17.17  107.00   59.70   17.17\n",
      "  209  12083 583A 583A CA  C    4535 4537  164.05  116.50   47.55   13.67  116.50   47.55   13.67\n",
      "  210  12085 583A 584Q C   N    4537 4539  164.81  120.00   44.81   10.18  120.00   44.81   10.18\n",
      "  211  12088 584Q 584Q N   CA   4539 4540  170.19  107.00   63.19   18.17  107.00   63.19   18.17\n",
      "  212  12091 584Q 584Q CA  C    4540 4546  169.32  116.50   52.82   15.19  116.50   52.82   15.19\n",
      "  213  12097 584Q 585E C   N    4546 4548  170.70  120.00   50.70   11.52  120.00   50.70   11.52\n",
      "  214  12100 585E 585E N   CA   4548 4549  176.00  107.00   69.00   19.84  107.00   69.00   19.84\n",
      "  215  12103 585E 585E CA  C    4549 4555  167.53  116.50   51.03   14.67  116.50   51.03   14.67\n",
      "  216  12109 585E 586D C   N    4555 4557  170.62  120.00   50.62   11.50  120.00   50.62   11.50\n",
      "  217  12112 586D 586D N   CA   4557 4558  171.66  107.00   64.66   18.59  107.00   64.66   18.59\n",
      "  218  12115 586D 586D CA  C    4558 4563  170.98  116.50   54.48   15.66  116.50   54.48   15.66\n",
      "  219  12120 586D 587L C   N    4563 4565  172.52  120.00   52.52   11.94  120.00   52.52   11.94\n",
      "  220  12123 587L 587L N   CA   4565 4566  173.22  107.00   66.22   19.04  107.00   66.22   19.04\n",
      "  221  12126 587L 587L CA  C    4566 4571  174.17  112.50   61.67    8.86  112.50   61.67    8.86\n",
      "  222  12131 587L 588P C   N    4571 4573  175.29  117.00   58.29   14.51  117.00   58.29   14.51\n",
      "  223  12134 588P 588P N   CA   4573 4574  172.43  108.20   64.24   14.60  108.20   64.24   14.60\n",
      "  224  12138 588P 588P CA  C    4574 4578  174.29  116.50   57.79   16.62  116.50   57.79   16.62\n",
      "  225  12141 588P 588P CB  CG   4576 4577  128.20  108.50   19.69    5.30  108.50   19.69    5.30\n",
      "  226  12143 588P 589H C   N    4578 4580  175.83  120.00   55.83   12.69  120.00   55.83   12.69\n",
      "  227  12146 589H 589H N   CA   4580 4581  175.26  107.00   68.26   19.63  107.00   68.26   19.63\n",
      "  228  12148 589H 589H CA  C    4581 4588  172.62  116.50   56.12   16.14  116.50   56.12   16.14\n",
      "  229  12158 589H 590K C   N    4588 4590  176.67  120.00   56.67   12.88  120.00   56.67   12.88\n",
      "  230  12161 590K 590K N   CA   4590 4591  176.76  107.00   69.76   20.06  107.00   69.76   20.06\n",
      "  231  12164 590K 590K CA  C    4591 4597  176.01  112.50   63.51    9.12  112.50   63.51    9.12\n",
      "  232  12169 590K 591P C   N    4597 4599  177.55  117.00   60.55   15.07  117.00   60.55   15.07\n",
      "  233  12172 591P 591P N   CA   4599 4600  176.57  108.20   68.37   15.54  108.20   68.37   15.54\n",
      "  234  12176 591P 591P CA  C    4600 4604  174.59  116.50   58.09   16.70  116.50   58.09   16.70\n",
      "  235  12179 591P 591P CB  CG   4602 4603  137.68  108.50   29.18    7.85  108.50   29.18    7.85\n",
      "  236  12181 591P 592T C   N    4604 4606  174.62  120.00   54.62   12.41  120.00   54.62   12.41\n",
      "  237  12184 592T 592T N   CA   4606 4607  173.83  107.00   66.83   19.22  107.00   66.83   19.22\n",
      "  238  12186 592T 592T CA  CB   4607 4608  148.52  108.50   40.02    9.39  108.50   40.02    9.39\n",
      "  239  12188 592T 592T CA  C    4607 4611  176.06  116.50   59.56   17.13  116.50   59.56   17.13\n",
      "  240  12191 592T 593S C   N    4611 4613  174.99  120.00   54.99   12.50  120.00   54.99   12.50\n",
      "  241  12194 593S 593S N   CA   4613 4614  175.53  107.00   68.53   19.71  107.00   68.53   19.71\n",
      "  242  12197 593S 593S CA  C    4614 4617  175.31  116.50   58.81   16.91  116.50   58.81   16.91\n",
      "  243  12199 593S 594S C   N    4617 4619  176.01  120.00   56.01   12.73  120.00   56.01   12.73\n",
      "  244  12202 594S 594S N   CA   4619 4620  177.75  107.00   70.75   20.34  107.00   70.75   20.34\n",
      "  245  12205 594S 594S CA  C    4620 4623  176.89  116.50   60.39   17.36  116.50   60.39   17.36\n",
      "  246  12207 594S 595S C   N    4623 4625  176.81  120.00   56.81   12.91  120.00   56.81   12.91\n",
      "  247  12210 595S 595S N   CA   4625 4626  178.52  107.00   71.52   20.56  107.00   71.52   20.56\n",
      "  248  12213 595S 595S CA  C    4626 4629  176.20  116.50   59.70   17.17  116.50   59.70   17.17\n",
      "  249  12215 595S 596M C   N    4629 4631  177.90  120.00   57.90   13.16  120.00   57.90   13.16\n",
      "  250  12218 596M 596M N   CA   4631 4632  178.77  107.00   71.77   20.64  107.00   71.77   20.64\n",
      "  251  12221 596M 596M CA  C    4632 4637  176.19  116.50   59.69   17.16  116.50   59.69   17.16\n",
      "  252  12225 596M 597K C   N    4637 4639  177.94  120.00   57.94   13.17  120.00   57.94   13.17\n",
      "  253  12228 597K 597K N   CA   4639 4640  177.87  107.00   70.87   20.38  107.00   70.87   20.38\n",
      "  254  12231 597K 597K CA  C    4640 4646  176.65  116.50   60.15   17.30  116.50   60.15   17.30\n",
      "  255  12236 597K 598T C   N    4646 4648  176.41  120.00   56.41   12.82  120.00   56.41   12.82\n",
      "  256  12239 598T 598T N   CA   4648 4649  176.29  107.00   69.29   19.92  107.00   69.29   19.92\n",
      "  257  12243 598T 598T CA  C    4649 4653  177.07  116.50   60.57   17.42  116.50   60.57   17.42\n",
      "  258  12246 598T 599V C   N    4653 4655  177.64  120.00   57.64   13.10  120.00   57.64   13.10\n",
      "  259  12249 599V 599V N   CA   4655 4656  178.45  107.00   71.45   20.54  107.00   71.45   20.54\n",
      "  260  12253 599V 599V CA  C    4656 4660  177.26  116.50   60.76   17.47  116.50   60.76   17.47\n",
      "  261  12256 599V 600D C   N    4660 4662  178.13  120.00   58.13   13.21  120.00   58.13   13.21\n",
      "  262  12259 600D 600D N   CA   4662 4663  178.86  107.00   71.86   20.66  107.00   71.86   20.66\n",
      "  263  12262 600D 600D CA  C    4663 4668  177.26  116.50   60.76   17.47  116.50   60.76   17.47\n",
      "  264  12267 600D 601E C   N    4668 4670  177.69  120.00   57.69   13.11  120.00   57.69   13.11\n",
      "  265  12270 601E 601E N   CA   4670 4671  176.94  107.00   69.94   20.11  107.00   69.94   20.11\n",
      "  266  12271 601E 601E CA  CB   4671 4672  136.18  113.50   22.68    5.57  113.50   22.68    5.57\n",
      "  267  12273 601E 601E CA  C    4671 4677  176.62  116.50   60.12   17.29  116.50   60.12   17.29\n",
      "  268  12276 601E 601E CG  CD   4673 4674  143.27  118.00   25.27    5.13  118.00   25.27    5.13\n",
      "  269  12279 601E 602E C   N    4677 4679  178.40  120.00   58.40   13.27  120.00   58.40   13.27\n",
      "  270  12282 602E 602E N   CA   4679 4680  178.63  107.00   71.63   20.60  107.00   71.63   20.60\n",
      "  271  12285 602E 602E CA  C    4680 4686  176.74  116.50   60.24   17.32  116.50   60.24   17.32\n",
      "  272  12291 602E 603A C   N    4686 4688  177.79  120.00   57.79   13.13  120.00   57.79   13.13\n",
      "  273  12294 603A 603A N   CA   4688 4689  179.72  107.00   72.72   20.91  107.00   72.72   20.91\n",
      "  274  12296 603A 603A CA  C    4689 4691  177.80  116.50   61.30   17.63  116.50   61.30   17.63\n",
      "  275  12298 603A 604L C   N    4691 4693  179.51  120.00   59.51   13.52  120.00   59.51   13.52\n",
      "  276  12301 604L 604L N   CA   4693 4694  177.69  107.00   70.69   20.33  107.00   70.69   20.33\n",
      "  277  12304 604L 604L CA  C    4694 4699  176.91  116.50   60.41   17.37  116.50   60.41   17.37\n",
      "  278  12309 604L 605A C   N    4699 4701  177.29  120.00   57.29   13.02  120.00   57.29   13.02\n",
      "  279  12312 605A 605A N   CA   4701 4702  178.20  107.00   71.20   20.47  107.00   71.20   20.47\n",
      "  280  12314 605A 605A CA  C    4702 4704  177.19  116.50   60.69   17.45  116.50   60.69   17.45\n",
      "  281  12316 605A 606M C   N    4704 4706  178.19  120.00   58.19   13.22  120.00   58.19   13.22\n",
      "  282  12319 606M 606M N   CA   4706 4707  172.18  107.00   65.18   18.74  107.00   65.18   18.74\n",
      "  283  12321 606M 606M CA  C    4707 4712  148.96  121.00   27.96    8.04  121.00   27.96    8.04\n",
      "  284  12322 606M 606M CA  C    4707 4712  160.55  116.50   44.05   12.67  116.50   44.05   12.67\n",
      "  285  12326 606M 607L C   N    4712 4714  157.43  120.00   37.43    8.51  120.00   37.43    8.51\n",
      "  286  12329 607L 607L N   CA   4714 4715  154.80  107.00   47.80   13.74  107.00   47.80   13.74\n",
      "  287  12340 608L 608L N   CA   4722 4723  124.96  107.00   17.96    5.16  107.00   17.96    5.16\n",
      "  288  12348 608L 609S C   N    4728 4730  140.58  120.00   20.58    4.68  120.00   20.58    4.68\n",
      "  289  12351 609S 609S N   CA   4730 4731  130.25  107.00   23.25    6.68  107.00   23.25    6.68\n",
      "  290  12359 610M 610M N   CA   4736 4737  129.62  107.00   22.62    6.50  107.00   22.62    6.50\n",
      "  291  12368 611G 611G N   CA   4744 4745  131.05  107.00   24.05    6.92  107.00   24.05    6.92\n",
      "  292  12374 612I 612I N   CA   4748 4749  126.53  107.00   19.53    5.61  107.00   19.53    5.61\n",
      "  293  12385 613E 613E N   CA   4756 4757  126.94  107.00   19.94    5.73  107.00   19.94    5.73\n",
      "  294  12397 614E 614E N   CA   4765 4766  124.46  107.00   17.46    5.02  107.00   17.46    5.02\n",
      "  295  12409 615T 615T N   CA   4774 4775  124.34  107.00   17.34    4.99  107.00   17.34    4.99\n",
      "  296  12419 616V 616V N   CA   4781 4782  124.88  107.00   17.88    5.14  107.00   17.88    5.14\n",
      "  297  12429 617A 617A N   CA   4788 4789  124.99  107.00   17.99    5.17  107.00   17.99    5.17\n",
      "  298  12436 618R 618R N   CA   4793 4794  122.93  107.00   15.93    4.58  107.00   15.93    4.58\n",
      "  299  12450 619Y 619Y N   CA   4804 4805  123.22  107.00   16.22    4.66  107.00   16.22    4.66\n",
      "  300  13300 700L 700L N   CA   5414 5415  125.98  107.00   18.98    5.46  107.00   18.98    5.46\n",
      "  301  13311 701L 701L N   CA   5422 5423  125.75  107.00   18.76    5.39  107.00   18.76    5.39\n",
      "  302  13322 702F 702F N   CA   5430 5431  125.25  107.00   18.25    5.25  107.00   18.25    5.25\n",
      "  303  13338 703N 703N N   CA   5441 5442  129.51  107.00   22.51    6.47  107.00   22.51    6.47\n",
      "  304  13349 704D 704D N   CA   5449 5450  128.13  107.00   21.13    6.08  107.00   21.13    6.08\n",
      "  305  13360 705E 705E N   CA   5457 5458  128.21  107.00   21.21    6.10  107.00   21.21    6.10\n",
      "  306  13372 706K 706K N   CA   5466 5467  131.61  107.00   24.61    7.08  107.00   24.61    7.08\n",
      "  307  13383 707V 707V N   CA   5475 5476  131.24  107.00   24.24    6.97  107.00   24.24    6.97\n",
      "  308  13387 707V 707V CA  C    5476 5480  132.90  116.50   16.40    4.72  116.50   16.40    4.72\n",
      "  309  13393 708A 708A N   CA   5482 5483  130.09  107.00   23.09    6.64  107.00   23.09    6.64\n",
      "  310  13400 709E 709E N   CA   5487 5488  129.85  107.00   22.85    6.57  107.00   22.85    6.57\n",
      "  311  13403 709E 709E CA  C    5488 5494  133.98  116.50   17.48    5.03  116.50   17.48    5.03\n",
      "  312  13409 709E 710S C   N    5494 5496  140.59  120.00   20.59    4.68  120.00   20.59    4.68\n",
      "  313  13412 710S 710S N   CA   5496 5497  134.72  107.00   27.72    7.97  107.00   27.72    7.97\n",
      "  314  13415 710S 710S CA  C    5497 5500  133.17  116.50   16.67    4.79  116.50   16.67    4.79\n",
      "  315  13417 710S 711L C   N    5500 5502  141.12  120.00   21.12    4.80  120.00   21.12    4.80\n",
      "  316  13420 711L 711L N   CA   5502 5503  125.35  107.00   18.35    5.28  107.00   18.35    5.28\n",
      "  317  13423 711L 711L CA  C    5503 5508  140.01  116.50   23.51    6.76  116.50   23.51    6.76\n",
      "  318  13431 712N 712N N   CA   5510 5511  136.63  107.00   29.63    8.52  107.00   29.63    8.52\n",
      "  319  13434 712N 712N CA  C    5511 5516  148.81  112.50   36.31    5.22  112.50   36.31    5.22\n",
      "  320  13439 712N 713P C   N    5516 5518  156.49  117.00   39.49    9.83  117.00   39.49    9.83\n",
      "  321  13442 713P 713P N   CA   5518 5519  143.60  108.20   35.40    8.04  108.20   35.40    8.04\n",
      "  322  13446 713P 713P CA  C    5519 5523  152.68  112.50   40.18    5.77  112.50   40.18    5.77\n",
      "  323  13451 713P 714P C   N    5523 5525  150.22  117.00   33.22    8.27  117.00   33.22    8.27\n",
      "  324  13454 714P 714P N   CA   5525 5526  150.22  108.20   42.02    9.55  108.20   42.02    9.55\n",
      "  325  13458 714P 714P CA  C    5526 5530  145.54  116.50   29.05    8.35  116.50   29.05    8.35\n",
      "  326  13463 714P 715F C   N    5530 5532  140.49  120.00   20.49    4.66  120.00   20.49    4.66\n",
      "  327  13466 715F 715F N   CA   5532 5533  146.96  107.00   39.96   11.49  107.00   39.96   11.49\n",
      "  328  13469 715F 715F CA  C    5533 5541  140.84  116.50   24.34    7.00  116.50   24.34    7.00\n",
      "  329  13479 715F 716S C   N    5541 5543  153.90  120.00   33.90    7.70  120.00   33.90    7.70\n",
      "  330  13482 716S 716S N   CA   5543 5544  149.12  107.00   42.12   12.11  107.00   42.12   12.11\n",
      "  331  13485 716S 716S CA  C    5544 5547  142.11  116.50   25.61    7.37  116.50   25.61    7.37\n",
      "  332  13487 716S 717M C   N    5547 5549  149.09  120.00   29.09    6.61  120.00   29.09    6.61\n",
      "  333  13490 717M 717M N   CA   5549 5550  146.22  107.00   39.22   11.28  107.00   39.22   11.28\n",
      "  334  13493 717M 717M CA  C    5550 5555  148.19  116.50   31.69    9.11  116.50   31.69    9.11\n",
      "  335  13497 717M 718A C   N    5555 5557  163.13  120.00   43.13    9.80  120.00   43.13    9.80\n",
      "  336  13500 718A 718A N   CA   5557 5558  151.58  107.00   44.58   12.82  107.00   44.58   12.82\n",
      "  337  13502 718A 718A CA  C    5558 5560  137.87  116.50   21.37    6.14  116.50   21.37    6.14\n",
      "  338  13504 718A 719F C   N    5560 5562  171.38  120.00   51.38   11.68  120.00   51.38   11.68\n",
      "  339  13507 719F 719F N   CA   5562 5563  154.63  107.00   47.63   13.70  107.00   47.63   13.70\n",
      "  340  13510 719F 719F CA  C    5563 5571  148.20  116.50   31.70    9.12  116.50   31.70    9.12\n",
      "  341  13520 719F 720L C   N    5571 5573  156.83  120.00   36.83    8.37  120.00   36.83    8.37\n",
      "  342  13523 720L 720L N   CA   5573 5574  153.83  107.00   46.83   13.46  107.00   46.83   13.46\n",
      "  343  13526 720L 720L CA  C    5574 5579  148.93  116.50   32.43    9.32  116.50   32.43    9.32\n",
      "  344  13531 720L 721Y C   N    5579 5581  158.18  120.00   38.18    8.68  120.00   38.18    8.68\n",
      "  345  13534 721Y 721Y N   CA   5581 5582  158.19  107.00   51.19   14.72  107.00   51.19   14.72\n",
      "  346  13537 721Y 721Y CA  C    5582 5591  154.11  116.50   37.61   10.81  116.50   37.61   10.81\n",
      "  347  13549 721Y 722F C   N    5591 5593  156.65  120.00   36.65    8.33  120.00   36.65    8.33\n",
      "  348  13552 722F 722F N   CA   5593 5594  156.85  107.00   49.85   14.33  107.00   49.85   14.33\n",
      "  349  13555 722F 722F CA  C    5594 5602  163.29  116.50   46.80   13.46  116.50   46.80   13.46\n",
      "  350  13565 722F 723Y C   N    5602 5604  163.08  120.00   43.08    9.79  120.00   43.08    9.79\n",
      "  351  13568 723Y 723Y N   CA   5604 5605  164.46  107.00   57.46   16.52  107.00   57.46   16.52\n",
      "  352  13571 723Y 723Y CA  C    5605 5614  162.77  116.50   46.27   13.30  116.50   46.27   13.30\n",
      "  353  13583 723Y 724K C   N    5614 5616  163.79  120.00   43.79    9.95  120.00   43.79    9.95\n",
      "  354  13586 724K 724K N   CA   5616 5617  173.84  107.00   66.84   19.22  107.00   66.84   19.22\n",
      "  355  13589 724K 724K CA  C    5617 5623  169.69  116.50   53.19   15.29  116.50   53.19   15.29\n",
      "  356  13594 724K 725R C   N    5623 5625  165.25  120.00   45.25   10.28  120.00   45.25   10.28\n",
      "  357  13597 725R 725R N   CA   5625 5626  167.57  107.00   60.57   17.42  107.00   60.57   17.42\n",
      "  358  13600 725R 725R CA  C    5626 5634  170.80  116.50   54.30   15.61  116.50   54.30   15.61\n",
      "  359  13608 725R 726V C   N    5634 5636  163.53  120.00   43.53    9.89  120.00   43.53    9.89\n",
      "  360  13611 726V 726V N   CA   5636 5637  169.89  107.00   62.89   18.08  107.00   62.89   18.08\n",
      "  361  13615 726V 726V CA  C    5637 5641  164.71  116.50   48.21   13.86  116.50   48.21   13.86\n",
      "  362  13618 726V 727G C   N    5641 5643  171.37  120.00   51.37   11.67  120.00   51.37   11.67\n",
      "  363  13620 727G 727G N   CA   5643 5644  171.17  107.00   64.17   18.45  107.00   64.17   18.45\n",
      "  364  13622 727G 727G CA  C    5644 5645  166.98  116.50   50.48   14.52  116.50   50.48   14.52\n",
      "  365  13623 727G 728K C   N    5645 5647  168.11  120.00   48.11   10.93  120.00   48.11   10.93\n",
      "  366  13626 728K 728K N   CA   5647 5648  158.27  107.00   51.27   11.65  107.00   51.27   11.65\n",
      "  367  13628 728K 728K CA  C    5648 5654  147.24  118.00   29.24    5.94  118.00   29.24    5.94\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19292 602E 602E CA  N    4680 4679   55.87   35.27   20.61    4.91   35.27   20.61    4.91\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22527  47G  60V CA  CA    371  459   22.63    5.82   16.81    5.66    5.82   16.81    5.66\n",
      "    2  22528  47G  61E CA  CA    371  466   26.25    7.97   18.27    6.94    7.97   18.27    6.94\n",
      "    3  22529  47G  62H CA  CA    371  475   27.83    9.00   18.83    7.68    9.00   18.83    7.68\n",
      "    4  22530  47G  63V CA  CA    371  485   29.56   12.77   16.80    7.87   12.77   16.80    7.87\n",
      "    5  22531  47G 243C CA  CA    371 1840   19.84   13.26    6.57    9.54   13.26    6.57    9.54\n",
      "    6  22532  47G 244D CA  CA    371 1846   18.58   12.85    5.73    6.95   12.85    5.73    6.95\n",
      "    7  22533  47G 245D CA  CA    371 1854   20.89   10.17   10.72   14.47   10.17   10.72   14.47\n",
      "    8  22534  47G 246E CA  CA    371 1862   24.19   10.81   13.38   20.68   10.81   13.38   20.68\n",
      "    9  22535  47G 247V CA  CA    371 1871   26.93   11.44   15.48   25.21   11.44   15.48   25.21\n",
      "   10  22536  47G 248S CA  CA    371 1878   30.57   13.40   17.17   20.45   13.40   17.17   20.45\n",
      "   11  22537  48T  60V CA  CA    375  459   17.84    3.89   13.95    5.11    3.89   13.95    5.11\n",
      "   12  22538  48T  61E CA  CA    375  466   21.40    6.49   14.91    6.15    6.49   14.91    6.15\n",
      "   13  22539  48T  62H CA  CA    375  475   23.00    9.04   13.96    6.13    9.04   13.96    6.13\n",
      "   14  22540  48T  63V CA  CA    375  485   24.70   12.76   11.94    6.17   12.76   11.94    6.17\n",
      "   15  22541  48T 237D CA  CA    375 1783   29.00   13.73   15.26   25.19   13.73   15.26   25.19\n",
      "   16  22542  48T 239Y CA  CA    375 1802   21.88   13.74    8.14   15.02   13.74    8.14   15.02\n",
      "   17  22544  48T 245D CA  CA    375 1854   15.88   10.05    5.83    9.55   10.05    5.83    9.55\n",
      "   18  22545  48T 246E CA  CA    375 1862   19.18    9.43    9.75   19.73    9.43    9.75   19.73\n",
      "   19  22546  48T 247V CA  CA    375 1871   21.91   10.45   11.46   24.24   10.45   11.46   24.24\n",
      "   20  22547  48T 248S CA  CA    375 1878   25.56   12.09   13.47   20.86   12.09   13.47   20.86\n",
      "   21  22566  61E 235G CA  CA    466 1774   16.96   12.03    4.93    9.04   12.03    4.93    9.04\n",
      "   22  22567  61E 236A CA  CA    466 1778   14.79   11.52    3.27    6.86   11.52    3.27    6.86\n",
      "   23  22912  86D 254N CA  CA    676 1925   22.89   13.93    8.97    8.10   13.93    8.97    8.10\n",
      "   24  22913  86D 255H CA  CA    676 1933   19.11   12.25    6.86    8.52   12.25    6.86    8.52\n",
      "   25  22914  86D 256M CA  CA    676 1943   20.10   13.51    6.59    7.56   13.51    6.59    7.56\n",
      "   26  22915  86D 257Q CA  CA    676 1951   21.64   12.79    8.86    6.13   12.79    8.86    6.13\n",
      "   27  22916  86D 258H CA  CA    676 1960   18.33    8.96    9.38    6.22    8.96    9.38    6.22\n",
      "   28  22926  96N 198G CA  CA    744 1504   13.66   10.05    3.61    4.74   10.05    3.61    4.74\n",
      "   29  22928  96N 255H CA  CA    744 1933   22.90   13.60    9.30   10.66   13.60    9.30   10.66\n",
      "   30  22929  96N 256M CA  CA    744 1943   23.83   13.80   10.03   10.99   13.80   10.03   10.99\n",
      "   31  22930  96N 257Q CA  CA    744 1951   25.19   12.80   12.40    8.35   12.80   12.40    8.35\n",
      "   32  22931  96N 258H CA  CA    744 1960   21.77    9.26   12.51    8.08    9.26   12.51    8.08\n",
      "   33  22932  96N 259F CA  CA    744 1970   20.67    9.20   11.46    6.27    9.20   11.46    6.27\n",
      "   34  22933  96N 260G CA  CA    744 1981   23.27   11.36   11.91    4.78   11.36   11.91    4.78\n",
      "   35  22946 103P 254N CA  CA    793 1925   20.82   13.24    7.58    9.50   13.24    7.58    9.50\n",
      "   36  22947 103P 255H CA  CA    793 1933   17.39   11.11    6.28   11.24   11.11    6.28   11.24\n",
      "   37  22948 103P 256M CA  CA    793 1943   17.72   10.74    6.98   12.00   10.74    6.98   12.00\n",
      "   38  22949 103P 257Q CA  CA    793 1951   18.41   10.47    7.94    7.22   10.47    7.94    7.22\n",
      "   39  22950 103P 258H CA  CA    793 1960   14.89    7.11    7.78    6.21    7.11    7.78    6.21\n",
      "   40  22951 103P 259F CA  CA    793 1970   13.71    5.98    7.72    5.08    5.98    7.72    5.08\n",
      "   41  22997 117K 606M CA  CA    893 4707   36.98   13.64   23.34    9.41   13.64   23.34    9.41\n",
      "   42  23034 118S 606M CA  CA    902 4707   34.78   11.87   22.91    8.46   11.87   22.91    8.46\n",
      "   43  23061 122P 606M CA  CA    932 4707   36.50   11.70   24.80    8.74   11.70   24.80    8.74\n",
      "   44  23080 123P 232T CA  CA    939 1754   13.82   10.94    2.89    4.62   10.94    2.89    4.62\n",
      "   45  23081 123P 233P CA  CA    939 1761   16.67   13.32    3.35    4.90   13.32    3.35    4.90\n",
      "   46  23090 123P 606M CA  CA    939 4707   36.26    9.60   26.66    9.20    9.60   26.66    9.20\n",
      "   47  23091 123P 607L CA  CA    939 4715   29.62   12.23   17.39    5.37   12.23   17.39    5.37\n",
      "   48  23108 129C 229G CA  CA    986 1736   11.65    7.57    4.08    7.79    7.57    4.08    7.79\n",
      "   49  23109 129C 230T CA  CA    986 1740   15.26   11.11    4.15    7.29   11.11    4.15    7.29\n",
      "   50  23110 129C 231V CA  CA    986 1747   17.48   10.93    6.55   12.08   10.93    6.55   12.08\n",
      "   51  23111 129C 232T CA  CA    986 1754   20.66   12.90    7.76   11.64   12.90    7.76   11.64\n",
      "   52  23116 129C 606M CA  CA    986 4707   44.82   12.81   32.01   10.69   12.81   32.01   10.69\n",
      "   53  23134 140G 228L CA  CA   1076 1728   14.14    9.80    4.34    8.97    9.80    4.34    8.97\n",
      "   54  23135 140G 229G CA  CA   1076 1736   16.93    9.76    7.17   12.08    9.76    7.17   12.08\n",
      "   55  23136 140G 230T CA  CA   1076 1740   20.19   13.58    6.61   10.89   13.58    6.61   10.89\n",
      "   56  23160 155L 228L CA  CA   1178 1728   10.38    8.19    2.18    5.99    8.19    2.18    5.99\n",
      "   57  23161 155L 229G CA  CA   1178 1736   13.26    9.59    3.67    7.55    9.59    3.67    7.55\n",
      "   58  23162 155L 230T CA  CA   1178 1740   16.41   13.09    3.31    7.02   13.09    3.31    7.02\n",
      "   59  23163 155L 231V CA  CA   1178 1747   17.85   13.69    4.16    9.18   13.69    4.16    9.18\n",
      "   60  23191 156Q 229G CA  CA   1186 1736   14.67   12.03    2.64    4.61   12.03    2.64    4.61\n",
      "   61  23204 157C 198G CA  CA   1195 1504   14.17   11.71    2.46    4.87   11.71    2.46    4.87\n",
      "   62  23221 157C 225V CA  CA   1195 1705   15.17   13.49    1.68    4.57   13.49    1.68    4.57\n",
      "   63  23222 157C 226V CA  CA   1195 1712   15.71   13.60    2.12    5.77   13.60    2.12    5.77\n",
      "   64  23223 157C 227K CA  CA   1195 1719   14.18   11.98    2.20    5.09   11.98    2.20    5.09\n",
      "   65  23224 157C 228L CA  CA   1195 1728   13.42   10.94    2.48    6.94   10.94    2.48    6.94\n",
      "   66  23225 157C 229G CA  CA   1195 1736   16.24   13.47    2.77    5.45   13.47    2.77    5.45\n",
      "   67  23235 168S 193M CA  CA   1280 1469   17.30   10.88    6.42    6.12   10.88    6.42    6.12\n",
      "   68  23236 168S 194C CA  CA   1280 1477   16.93   12.24    4.69    6.25   12.24    4.69    6.25\n",
      "   69  23237 168S 197C CA  CA   1280 1498   17.33   13.00    4.33    5.96   13.00    4.33    5.96\n",
      "   70  23238 168S 198G CA  CA   1280 1504   17.47   11.08    6.39    9.53   11.08    6.39    9.53\n",
      "   71  23239 168S 199T CA  CA   1280 1508   13.82    7.73    6.09    8.89    7.73    6.09    8.89\n",
      "   72  23240 168S 200I CA  CA   1280 1515   13.44    6.23    7.21    7.59    6.23    7.21    7.59\n",
      "   73  23252 168S 228L CA  CA   1280 1728   21.40   13.67    7.73   14.87   13.67    7.73   14.87\n",
      "   74  23261 179K 198G CA  CA   1359 1504   18.76   13.88    4.87    7.39   13.88    4.87    7.39\n",
      "   75  23262 179K 199T CA  CA   1359 1508   15.06   10.15    4.91    7.00   10.15    4.91    7.00\n",
      "   76  23263 179K 200I CA  CA   1359 1515   13.42    8.82    4.60    4.70    8.82    4.60    4.70\n",
      "   77  23283 180C 199T CA  CA   1368 1508   16.21   12.88    3.33    5.02   12.88    3.33    5.02\n",
      "   78  23284 180C 200I CA  CA   1368 1515   13.93   10.55    3.38    5.45   10.55    3.38    5.45\n",
      "   79  23335 182L 228L CA  CA   1379 1728   14.62   12.33    2.29    5.56   12.33    2.29    5.56\n",
      "   80  23336 182L 229G CA  CA   1379 1736   17.18   13.56    3.62    6.50   13.56    3.62    6.50\n",
      "   81  23435 191N 231V CA  CA   1447 1747   14.37   12.10    2.27    5.27   12.10    2.27    5.27\n",
      "   82  23471 192W 231V CA  CA   1455 1747   13.48   11.32    2.16    4.78   11.32    2.16    4.78\n",
      "   83  23526 193M 606M CA  CA   1469 4707   38.30   13.92   24.38   25.85   13.92   24.38   25.85\n",
      "   84  23605 195M 252F CA  CA   1483 1909   10.74    9.18    1.56    4.60    9.18    1.56    4.60\n",
      "   85  23685 198G 252F CA  CA   1504 1909   15.58   13.63    1.95    5.04   13.63    1.95    5.04\n",
      "   86  23687 198G 255H CA  CA   1504 1933   11.82   10.19    1.63    4.62   10.19    1.63    4.62\n",
      "   87  23688 198G 256M CA  CA   1504 1943   11.24    9.59    1.65    4.72    9.59    1.65    4.72\n",
      "   88  23815 203P 230T CA  CA   1533 1740   15.76   13.52    2.24    4.80   13.52    2.24    4.80\n",
      "   89  23869 208G 227K CA  CA   1574 1719   15.36   12.89    2.47    4.88   12.89    2.47    4.88\n",
      "   90  24135 226V 606M CA  CA   1712 4707   34.90   11.91   22.99   61.45   11.91   22.99   61.45\n",
      "   91  24158 227K 606M CA  CA   1719 4707   35.91   12.70   23.21   50.43   12.70   23.21   50.43\n",
      "   92  24175 228L 606M CA  CA   1728 4707   35.74   11.25   24.50   64.59   11.25   24.50   64.59\n",
      "   93  24187 229G 606M CA  CA   1736 4707   33.54   10.88   22.67   45.95   10.88   22.67   45.95\n",
      "   94  24188 229G 607L CA  CA   1736 4715   26.38   13.87   12.52   18.12   13.87   12.52   18.12\n",
      "   95  24189 229G 728K CA  CA   1736 5648   19.93   12.91    7.01   12.27   12.91    7.01   12.27\n",
      "   96  24198 230T 246E CA  CA   1740 1862   16.45   12.94    3.51    6.68   12.94    3.51    6.68\n",
      "   97  24206 230T 606M CA  CA   1740 4707   30.65    9.84   20.81   49.35    9.84   20.81   49.35\n",
      "   98  24207 230T 607L CA  CA   1740 4715   23.38   12.62   10.76   18.46   12.62   10.76   18.46\n",
      "   99  24208 230T 728K CA  CA   1740 5648   23.05   12.71   10.34   20.84   12.71   10.34   20.84\n",
      "  100  24214 231V 238F CA  CA   1747 1791   11.68    9.55    2.13    4.94    9.55    2.13    4.94\n",
      "  101  24215 231V 239Y CA  CA   1747 1802   15.37   13.01    2.36    4.97   13.01    2.36    4.97\n",
      "  102  24229 231V 606M CA  CA   1747 4707   27.57    6.06   21.51   58.42    6.06   21.51   58.42\n",
      "  103  24230 231V 607L CA  CA   1747 4715   20.40    9.09   11.32   22.83    9.09   11.32   22.83\n",
      "  104  24231 231V 728K CA  CA   1747 5648   25.91    9.49   16.42   37.56    9.49   16.42   37.56\n",
      "  105  24244 232T 262D CA  CA   1754 1993   11.94    9.01    2.94    5.57    9.01    2.94    5.57\n",
      "  106  24245 232T 606M CA  CA   1754 4707   24.66    5.37   19.29   47.68    5.37   19.29   47.68\n",
      "  107  24246 232T 607L CA  CA   1754 4715   17.45    6.72   10.73   21.84    6.72   10.73   21.84\n",
      "  108  24247 232T 728K CA  CA   1754 5648   28.13    7.50   20.63   44.15    7.50   20.63   44.15\n",
      "  109  24261 233P 262D CA  CA   1761 1993   13.65    8.58    5.06    9.63    8.58    5.06    9.63\n",
      "  110  24262 233P 606M CA  CA   1761 4707   20.97    5.20   15.77   38.36    5.20   15.77   38.36\n",
      "  111  24263 233P 607L CA  CA   1761 4715   13.78    4.36    9.43   22.53    4.36    9.43   22.53\n",
      "  112  24264 233P 728K CA  CA   1761 5648   31.80    6.73   25.06   54.52    6.73   25.06   54.52\n",
      "  113  24268 234S 248S CA  CA   1768 1878   15.35   11.90    3.45    4.90   11.90    3.45    4.90\n",
      "  114  24277 234S 606M CA  CA   1768 4707   22.10    8.18   13.92   26.08    8.18   13.92   26.08\n",
      "  115  24278 234S 607L CA  CA   1768 4715   14.76    8.02    6.75   11.03    8.02    6.75   11.03\n",
      "  116  24279 234S 728K CA  CA   1768 5648   31.47   10.59   20.89   33.20   10.59   20.89   33.20\n",
      "  117  24283 235G 246E CA  CA   1774 1862   17.09   12.57    4.53    8.43   12.57    4.53    8.43\n",
      "  118  24284 235G 247V CA  CA   1774 1871   14.70   11.30    3.41    7.04   11.30    3.41    7.04\n",
      "  119  24285 235G 248S CA  CA   1774 1878   11.62    8.95    2.67    4.97    8.95    2.67    4.97\n",
      "  120  24297 235G 606M CA  CA   1774 4707   24.68    7.52   17.16   39.95    7.52   17.16   39.95\n",
      "  121  24298 235G 607L CA  CA   1774 4715   17.40    9.05    8.35   15.42    9.05    8.35   15.42\n",
      "  122  24299 235G 728K CA  CA   1774 5648   30.30   11.48   18.82   36.56   11.48   18.82   36.56\n",
      "  123  24303 236A 245D CA  CA   1778 1854   17.51   13.89    3.62    6.18   13.89    3.62    6.18\n",
      "  124  24304 236A 246E CA  CA   1778 1862   14.03   10.23    3.80    8.11   10.23    3.80    8.11\n",
      "  125  24305 236A 247V CA  CA   1778 1871   12.27    9.89    2.38    5.60    9.89    2.38    5.60\n",
      "  126  24318 236A 606M CA  CA   1778 4707   28.67    8.77   19.89   50.87    8.77   19.89   50.87\n",
      "  127  24319 236A 607L CA  CA   1778 4715   21.38   11.42    9.96   18.83   11.42    9.96   18.83\n",
      "  128  24320 236A 728K CA  CA   1778 5648   26.97   12.98   13.99   30.02   12.98   13.99   30.02\n",
      "  129  24325 237D 245D CA  CA   1783 1854   13.70   10.62    3.08    5.02   10.62    3.08    5.02\n",
      "  130  24326 237D 246E CA  CA   1783 1862   10.20    7.05    3.14    6.43    7.05    3.14    6.43\n",
      "  131  24336 237D 606M CA  CA   1783 4707   32.52   12.37   20.15   41.24   12.37   20.15   41.24\n",
      "  132  24354 238F 606M CA  CA   1791 4707   34.40   13.44   20.96   51.01   13.44   20.96   51.01\n",
      "  133  24424 249D 606M CA  CA   1884 4707   30.11   13.37   16.74   40.27   13.37   16.74   40.27\n",
      "  134  24431 250V 606M CA  CA   1892 4707   26.85   13.43   13.43   24.66   13.43   13.43   24.66\n",
      "  135  24441 251H 606M CA  CA   1899 4707   26.36   12.08   14.28   28.74   12.08   14.28   28.74\n",
      "  136  24442 251H 607L CA  CA   1899 4715   20.34   13.68    6.67    9.49   13.68    6.67    9.49\n",
      "  137  24452 252F 606M CA  CA   1909 4707   25.99    8.96   17.03   39.88    8.96   17.03   39.88\n",
      "  138  24453 252F 607L CA  CA   1909 4715   19.40   11.27    8.13   13.02   11.27    8.13   13.02\n",
      "  139  24454 252F 728K CA  CA   1909 5648   33.52   13.91   19.61   35.07   13.91   19.61   35.07\n",
      "  140  24462 253A 262D CA  CA   1920 1993   10.43    7.50    2.93    4.80    7.50    2.93    4.80\n",
      "  141  24463 253A 606M CA  CA   1920 4707   22.69    6.73   15.96   31.72    6.73   15.96   31.72\n",
      "  142  24464 253A 607L CA  CA   1920 4715   16.20    8.40    7.80   11.27    8.40    7.80   11.27\n",
      "  143  24465 253A 728K CA  CA   1920 5648   35.14   11.13   24.01   35.52   11.13   24.01   35.52\n",
      "  144  24473 254N 606M CA  CA   1925 4707   24.69    9.84   14.85   22.06    9.84   14.85   22.06\n",
      "  145  24474 254N 607L CA  CA   1925 4715   18.71   11.52    7.19    7.82   11.52    7.19    7.82\n",
      "  146  24475 254N 728K CA  CA   1925 5648   35.68   13.83   21.84   25.72   13.83   21.84   25.72\n",
      "  147  24482 255H 606M CA  CA   1933 4707   28.52   10.39   18.14   34.99   10.39   18.14   34.99\n",
      "  148  24483 255H 607L CA  CA   1933 4715   22.35   13.12    9.23   12.34   13.12    9.23   12.34\n",
      "  149  24489 256M 606M CA  CA   1943 4707   28.12    7.34   20.78   42.00    7.34   20.78   42.00\n",
      "  150  24490 256M 607L CA  CA   1943 4715   21.53   10.55   10.98   15.25   10.55   10.98   15.25\n",
      "  151  24491 256M 728K CA  CA   1943 5648   30.41   11.69   18.71   29.79   11.69   18.71   29.79\n",
      "  152  24494 257Q 261I CA  CA   1951 1985    8.09    5.03    3.06    5.46    5.03    3.06    5.46\n",
      "  153  24495 257Q 262D CA  CA   1951 1993   12.52    6.08    6.44    6.34    6.08    6.44    6.34\n",
      "  154  24496 257Q 606M CA  CA   1951 4707   26.61    8.14   18.46   18.45    8.14   18.46   18.45\n",
      "  155  24497 257Q 607L CA  CA   1951 4715   20.38   10.33   10.05    8.00   10.33   10.05    8.00\n",
      "  156  24498 257Q 728K CA  CA   1951 5648   32.00   11.32   20.69   17.88   11.32   20.69   17.88\n",
      "  157  24502 258H 606M CA  CA   1960 4707   29.94   11.57   18.37   13.85   11.57   18.37   13.85\n",
      "  158  24505 259F 606M CA  CA   1970 4707   31.82   10.95   20.87   13.10   10.95   20.87   13.10\n",
      "  159  24506 259F 728K CA  CA   1970 5648   28.11   13.89   14.22    8.31   13.89   14.22    8.31\n",
      "  160  24508 260G 606M CA  CA   1981 4707   29.77    9.52   20.25    9.69    9.52   20.25    9.69\n",
      "  161  24509 260G 607L CA  CA   1981 4715   22.97   11.87   11.10    4.66   11.87   11.10    4.66\n",
      "  162  24510 260G 728K CA  CA   1981 5648   27.49   11.23   16.26    7.36   11.23   16.26    7.36\n",
      "  163  24511 261I 606M CA  CA   1985 4707   29.85    5.99   23.85   10.65    5.99   23.85   10.65\n",
      "  164  24512 261I 607L CA  CA   1985 4715   22.82    8.92   13.90    5.59    8.92   13.90    5.59\n",
      "  165  24513 261I 728K CA  CA   1985 5648   27.68    8.38   19.30    8.20    8.38   19.30    8.20\n",
      "  166  24514 262D 606M CA  CA   1993 4707   31.27    3.83   27.44   10.50    3.83   27.44   10.50\n",
      "  167  24515 262D 607L CA  CA   1993 4715   24.23    5.40   18.83    6.71    5.40   18.83    6.71\n",
      "  168  24516 262D 728K CA  CA   1993 5648   28.39    5.28   23.11    8.52    5.28   23.11    8.52\n",
      "  169  24517 606M 728K CA  CA   4707 5648   52.50    5.10   47.41   18.10    5.10   47.41   18.10\n",
      "  170  24518 607L 728K CA  CA   4715 5648   45.43    3.83   41.60   15.50    3.83   41.60   15.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24519  47G  60V N   O     370  464   26.36    5.46   20.90    6.50    5.46   20.90    6.50\n",
      "    2  24520  47G  61E N   O     370  473   27.41    9.39   18.03    6.23    9.39   18.03    6.23\n",
      "    3  24521  47G  62H N   O     370  483   30.00   10.24   19.76    7.13   10.24   19.76    7.13\n",
      "    4  24522  47G 243C N   O     370 1844   19.93   10.34    9.58   14.14   10.34    9.58   14.14\n",
      "    5  24523  47G 244D N   O     370 1852   19.95    9.80   10.14   11.98    9.80   10.14   11.98\n",
      "    6  24524  47G 246E N   O     370 1869   26.36    8.86   17.49   26.28    8.86   17.49   26.28\n",
      "    7  24525  48T  60V N   O     374  464   21.44    4.55   16.89    5.73    4.55   16.89    5.73\n",
      "    8  24526  48T  61E N   O     374  473   22.23    8.51   13.72    5.19    8.51   13.72    5.19\n",
      "    9  24527  48T  62H N   O     374  483   24.77   10.49   14.27    5.69   10.49   14.27    5.69\n",
      "   10  24528  48T 246E N   O     374 1869   21.13    8.70   12.43   23.42    8.70   12.43   23.42\n",
      "   11  24529  60V  47G N   O     458  373   19.07    3.45   15.61    5.07    3.45   15.61    5.07\n",
      "   12  24530  60V  48T N   O     458  380   15.32    2.29   13.03    4.62    2.29   13.03    4.62\n",
      "   13  24534  61E  47G N   O     465  373   22.41    5.39   17.01    6.35    5.39   17.01    6.35\n",
      "   14  24535  61E  48T N   O     465  380   18.35    3.58   14.77    6.09    3.58   14.77    6.09\n",
      "   15  24537  61E 236A N   O     465 1781   13.16    9.83    3.32    6.76    9.83    3.32    6.76\n",
      "   16  24543  62H  47G N   O     474  373   25.24    6.52   18.73    7.33    6.52   18.73    7.33\n",
      "   17  24544  62H  48T N   O     474  380   21.13    6.51   14.62    6.30    6.51   14.62    6.30\n",
      "   18  24557  63V  47G N   O     484  373   26.89    9.89   17.00    7.80    9.89   17.00    7.80\n",
      "   19  24558  63V  48T N   O     484  380   22.66   10.09   12.57    6.36   10.09   12.57    6.36\n",
      "   20  24683  79I 251H N   O     620 1907   11.71    8.69    3.02    6.41    8.69    3.02    6.41\n",
      "   21  24746  86D 196T N   O     675 1496    9.38    6.06    3.31    5.08    6.06    3.31    5.08\n",
      "   22  24748  86D 255H N   O     675 1941   17.93   10.89    7.04    8.22   10.89    7.04    8.22\n",
      "   23  24749  86D 258H N   O     675 1968   18.00    8.43    9.56    5.93    8.43    9.56    5.93\n",
      "   24  24756  96N 195M N   O     743 1489   16.99   10.61    6.39    7.34   10.61    6.39    7.34\n",
      "   25  24757  96N 196T N   O     743 1496   14.07    8.31    5.75    7.62    8.31    5.75    7.62\n",
      "   26  24758  96N 197C N   O     743 1502   12.78    8.60    4.18    5.12    8.60    4.18    5.12\n",
      "   27  24759  96N 257Q N   O     743 1958   25.02   10.58   14.44    9.23   10.58   14.44    9.23\n",
      "   28  24760  96N 258H N   O     743 1968   22.28    7.30   14.98    9.37    7.30   14.98    9.37\n",
      "   29  24761  96N 259F N   O     743 1979   23.57   10.74   12.83    6.37   10.74   12.83    6.37\n",
      "   30  24774 103P 255H N   O     792 1941   15.44    9.83    5.61    9.42    9.83    5.61    9.42\n",
      "   31  24775 103P 256M N   O     792 1949   17.53   10.09    7.44   11.72   10.09    7.44   11.72\n",
      "   32  24776 103P 257Q N   O     792 1958   16.65    8.93    7.72    6.71    8.93    7.72    6.71\n",
      "   33  24777 103P 258H N   O     792 1968   13.79    5.45    8.34    6.51    5.45    8.34    6.51\n",
      "   34  24791 116A 255H N   O     887 1941   13.39    9.88    3.51    4.85    9.88    3.51    4.85\n",
      "   35  24792 116A 256M N   O     887 1949   14.29    9.39    4.90    6.57    9.39    4.90    6.57\n",
      "   36  24806 117K 256M N   O     892 1949   11.86    8.74    3.12    5.15    8.74    3.12    5.15\n",
      "   37  24840 122P 256M N   O     931 1949   12.60    9.12    3.48    5.08    9.12    3.48    5.08\n",
      "   38  24870 129C 229G N   O     985 1738   13.56    9.63    3.93    5.84    9.63    3.93    5.84\n",
      "   39  24871 129C 231V N   O     985 1752   18.33    9.74    8.58   15.19    9.74    8.58   15.19\n",
      "   40  24887 140G 228L N   O    1075 1734   15.22    8.93    6.30   12.26    8.93    6.30   12.26\n",
      "   41  24888 140G 229G N   O    1075 1738   19.95   10.45    9.50   13.91   10.45    9.50   13.91\n",
      "   42  24906 155L 228L N   O    1177 1734   11.89    8.66    3.23    8.19    8.66    3.23    8.19\n",
      "   43  24948 157C 228L N   O    1194 1734   12.61   10.65    1.96    5.36   10.65    1.96    5.36\n",
      "   44  24960 168S 191N N   O    1279 1453   19.00   10.13    8.86    4.79   10.13    8.86    4.79\n",
      "   45  24961 168S 192W N   O    1279 1467   15.93    8.00    7.92    5.46    8.00    7.92    5.46\n",
      "   46  24962 168S 198G N   O    1279 1506   16.74    8.70    8.04   11.70    8.70    8.04   11.70\n",
      "   47  24963 168S 199T N   O    1279 1513   14.90    7.75    7.15    9.17    7.75    7.15    9.17\n",
      "   48  24964 168S 200I N   O    1279 1521   12.49    3.06    9.43   10.22    3.06    9.43   10.22\n",
      "   49  24965 168S 201G N   O    1279 1525   14.17    6.91    7.25    5.21    6.91    7.25    5.21\n",
      "   50  24980 179K 198G N   O    1358 1506   18.48   10.77    7.71   11.43   10.77    7.71   11.43\n",
      "   51  24981 179K 199T N   O    1358 1513   15.91    9.61    6.30    8.15    9.61    6.30    8.15\n",
      "   52  24982 179K 200I N   O    1358 1521   12.73    5.15    7.58    8.00    5.15    7.58    8.00\n",
      "   53  24997 180C 200I N   O    1367 1521   11.35    7.05    4.31    7.10    7.05    4.31    7.10\n",
      "   54  25223 195M 251H N   O    1482 1907   12.52   10.45    2.07    4.96   10.45    2.07    4.96\n",
      "   55  25224 195M 252F N   O    1482 1918   12.17    9.69    2.48    6.81    9.69    2.48    6.81\n",
      "   56  25253 196T 251H N   O    1490 1907   12.62   10.67    1.95    4.61   10.67    1.95    4.61\n",
      "   57  25254 196T 252F N   O    1490 1918   12.94   10.58    2.35    6.23   10.58    2.35    6.23\n",
      "   58  25287 198G  96N N   O    1503  750   12.75    8.40    4.35    5.44    8.40    4.35    5.44\n",
      "   59  25304 198G 256M N   O    1503 1949   11.79    9.83    1.96    5.28    9.83    1.96    5.28\n",
      "   60  25316 199T 168S N   O    1507 1284   17.04   10.73    6.31    7.19   10.73    6.31    7.19\n",
      "   61  25341 200I 168S N   O    1514 1284   15.09    8.25    6.84    6.16    8.25    6.84    6.16\n",
      "   62  25892 228L 182L N   O    1727 1385   14.02   10.86    3.16    7.49   10.86    3.16    7.49\n",
      "   63  25909 228L 231V N   O    1727 1752   10.63    8.46    2.16    5.34    8.46    2.16    5.34\n",
      "   64  25918 229G 129C N   O    1735  990   12.18    9.03    3.15    4.77    9.03    3.15    4.77\n",
      "   65  25919 229G 140G N   O    1735 1078   16.88   10.74    6.14    8.91   10.74    6.14    8.91\n",
      "   66  25936 229G 606M N   O    1735 4713   29.39   10.86   18.53   34.95   10.86   18.53   34.95\n",
      "   67  25937 229G 728K N   O    1735 5655   16.80   10.98    5.82   10.08   10.98    5.82   10.08\n",
      "   68  25940 230T 188R N   O    1739 1429   12.91   10.35    2.55    4.61   10.35    2.55    4.61\n",
      "   69  25954 230T 606M N   O    1739 4713   27.02   10.05   16.96   37.36   10.05   16.96   37.36\n",
      "   70  25955 230T 728K N   O    1739 5655   18.68   10.90    7.78   15.33   10.90    7.78   15.33\n",
      "   71  25958 231V 191N N   O    1746 1453   10.91    8.88    2.03    4.79    8.88    2.03    4.79\n",
      "   72  25973 231V 606M N   O    1746 4713   24.23    7.57   16.66   40.70    7.57   16.66   40.70\n",
      "   73  25974 231V 728K N   O    1746 5655   21.30    9.06   12.25   26.28    9.06   12.25   26.28\n",
      "   74  25975 232T 123P N   O    1753  944   13.10    9.84    3.26    4.91    9.84    3.26    4.91\n",
      "   75  25988 232T 262D N   O    1753 1999   11.08    7.25    3.83    6.90    7.25    3.83    6.90\n",
      "   76  25989 232T 606M N   O    1753 4713   20.86    4.82   16.05   35.23    4.82   16.05   35.23\n",
      "   77  25990 232T 607L N   O    1753 4721   16.15    8.14    8.00   13.10    8.14    8.00   13.10\n",
      "   78  25991 232T 728K N   O    1753 5655   24.25    7.27   16.98   32.69    7.27   16.98   32.69\n",
      "   79  25998 233P 261I N   O    1760 1991   12.94   10.03    2.91    5.42   10.03    2.91    5.42\n",
      "   80  25999 233P 262D N   O    1760 1999   13.70    6.83    6.86   12.36    6.83    6.86   12.36\n",
      "   81  26000 233P 606M N   O    1760 4713   17.63    3.34   14.28   31.86    3.34   14.28   31.86\n",
      "   82  26001 233P 607L N   O    1760 4721   12.82    5.27    7.54   13.45    5.27    7.54   13.45\n",
      "   83  26002 233P 728K N   O    1760 5655   27.13    6.42   20.71   40.14    6.42   20.71   40.14\n",
      "   84  26010 234S 606M N   O    1767 4713   17.31    5.81   11.51   20.61    5.81   11.51   20.61\n",
      "   85  26011 234S 607L N   O    1767 4721   12.60    7.38    5.22    6.98    7.38    5.22    6.98\n",
      "   86  26012 234S 728K N   O    1767 5655   27.89    9.21   18.68   28.04    9.21   18.68   28.04\n",
      "   87  26020 235G 247V N   O    1773 1876   13.65   10.30    3.35    7.10   10.30    3.35    7.10\n",
      "   88  26026 235G 606M N   O    1773 4713   19.35    6.83   12.52   27.14    6.83   12.52   27.14\n",
      "   89  26027 235G 607L N   O    1773 4721   14.92    9.30    5.61    9.33    9.30    5.61    9.33\n",
      "   90  26028 235G 728K N   O    1773 5655   27.05   10.50   16.55   30.85   10.50   16.55   30.85\n",
      "   91  26039 236A 238F N   O    1777 1800   10.06    8.28    1.78    4.59    8.28    1.78    4.59\n",
      "   92  26040 236A 247V N   O    1777 1876   10.41    7.56    2.84    7.23    7.56    2.84    7.23\n",
      "   93  26048 236A 606M N   O    1777 4713   22.63    8.78   13.86   33.84    8.78   13.86   33.84\n",
      "   94  26063 237D 245D N   O    1782 1860   13.25   10.25    2.99    4.68   10.25    2.99    4.68\n",
      "   95  26064 237D 246E N   O    1782 1869   12.43    9.79    2.63    4.94    9.79    2.63    4.94\n",
      "   96  26088 238F 245D N   O    1790 1860    9.62    7.17    2.45    4.64    7.17    2.45    4.64\n",
      "   97  26106 239Y 230T N   O    1801 1745   13.07   10.87    2.20    4.55   10.87    2.20    4.55\n",
      "   98  26210 245D  47G N   O    1853  373   17.82    9.49    8.33    9.56    9.49    8.33    9.56\n",
      "   99  26225 246E  47G N   O    1861  373   20.54    8.04   12.51   19.52    8.04   12.51   19.52\n",
      "  100  26226 246E  48T N   O    1861  380   16.02    8.02    8.00   15.50    8.02    8.00   15.50\n",
      "  101  26233 246E 236A N   O    1861 1781   12.88    9.53    3.35    7.33    9.53    3.35    7.33\n",
      "  102  26234 246E 237D N   O    1861 1789   11.66    8.66    3.00    5.72    8.66    3.00    5.72\n",
      "  103  26243 247V  47G N   O    1870  373   23.83    9.09   14.74   24.05    9.09   14.74   24.05\n",
      "  104  26244 247V  48T N   O    1870  380   19.23    8.35   10.88   22.71    8.35   10.88   22.71\n",
      "  105  26253 247V 236A N   O    1870 1781    9.92    7.22    2.70    6.89    7.22    2.70    6.89\n",
      "  106  26264 248S  47G N   O    1877  373   26.90   10.21   16.69   19.94   10.21   16.69   19.94\n",
      "  107  26265 248S  48T N   O    1877  380   22.33    9.22   13.10   20.74    9.22   13.10   20.74\n",
      "  108  26276 248S 234S N   O    1877 1772   15.31   10.72    4.59    6.14   10.72    4.59    6.14\n",
      "  109  26379 253A 606M N   O    1919 4713   19.19    9.45    9.74   15.10    9.45    9.74   15.10\n",
      "  110  26395 254N 606M N   O    1924 4713   19.59   10.87    8.72   11.14   10.87    8.72   11.14\n",
      "  111  26446 256M 606M N   O    1942 4713   23.39   10.48   12.91   21.30   10.48   12.91   21.30\n",
      "  112  26461 257Q 261I N   O    1950 1991    8.17    3.50    4.67    7.90    3.50    4.67    7.90\n",
      "  113  26463 257Q 606M N   O    1950 4713   22.26    9.90   12.36   10.77    9.90   12.36   10.77\n",
      "  114  26468 258H  96N N   O    1959  750   23.25   10.98   12.27    7.00   10.98   12.27    7.00\n",
      "  115  26469 258H 103P N   O    1959  798   17.57    8.91    8.67    6.25    8.91    8.67    6.25\n",
      "  116  26489 259F  96N N   O    1969  750   20.98    9.14   11.84    6.06    9.14   11.84    6.06\n",
      "  117  26490 259F 103P N   O    1969  798   15.61    6.79    8.82    5.49    6.79    8.82    5.49\n",
      "  118  26511 260G  96N N   O    1980  750   22.13   10.13   12.00    4.55   10.13   12.00    4.55\n",
      "  119  26512 260G 103P N   O    1980  798   17.35    7.16   10.19    4.72    7.16   10.19    4.72\n",
      "  120  26540 261I 257Q N   O    1984 1958    7.18    4.22    2.96    4.79    4.22    2.96    4.79\n",
      "  121  26543 261I 606M N   O    1984 4713   25.36    9.30   16.06    6.63    9.30   16.06    6.63\n",
      "  122  26544 261I 728K N   O    1984 5655   24.64    9.61   15.02    5.99    9.61   15.02    5.99\n",
      "  123  26555 262D 253A N   O    1992 1923   10.15    6.80    3.35    4.95    6.80    3.35    4.95\n",
      "  124  26558 262D 256M N   O    1992 1949    9.15    5.39    3.76    6.27    5.39    3.76    6.27\n",
      "  125  26563 262D 606M N   O    1992 4713   26.09    6.20   19.89    7.05    6.20   19.89    7.05\n",
      "  126  26564 262D 607L N   O    1992 4721   22.12    8.01   14.11    4.53    8.01   14.11    4.53\n",
      "  127  26565 262D 728K N   O    1992 5655   24.77    6.56   18.22    6.28    6.56   18.22    6.28\n",
      "  128  26566 606M 118S N   O    4706  906   40.72    9.51   31.21   11.28    9.51   31.21   11.28\n",
      "  129  26567 606M 122P N   O    4706  937   40.76   10.15   30.61   10.42   10.15   30.61   10.42\n",
      "  130  26568 606M 123P N   O    4706  944   41.94   10.43   31.50   10.22   10.43   31.50   10.22\n",
      "  131  26569 606M 230T N   O    4706 1745   32.92    9.71   23.21   51.47    9.71   23.21   51.47\n",
      "  132  26570 606M 231V N   O    4706 1752   31.54    7.12   24.42   60.20    7.12   24.42   60.20\n",
      "  133  26571 606M 232T N   O    4706 1759   27.92    5.78   22.14   47.40    5.78   22.14   47.40\n",
      "  134  26572 606M 233P N   O    4706 1766   24.90    7.57   17.33   34.87    7.57   17.33   34.87\n",
      "  135  26573 606M 234S N   O    4706 1772   26.61   10.52   16.09   25.30   10.52   16.09   25.30\n",
      "  136  26574 606M 235G N   O    4706 1776   30.79    7.86   22.94   48.74    7.86   22.94   48.74\n",
      "  137  26575 606M 251H N   O    4706 1907   30.00   10.34   19.66   38.03   10.34   19.66   38.03\n",
      "  138  26576 606M 252F N   O    4706 1918   29.70    7.37   22.33   51.32    7.37   22.33   51.32\n",
      "  139  26577 606M 253A N   O    4706 1923   27.96    6.24   21.72   39.00    6.24   21.72   39.00\n",
      "  140  26578 606M 254N N   O    4706 1931   31.11    9.32   21.78   29.51    9.32   21.78   29.51\n",
      "  141  26579 606M 255H N   O    4706 1941   34.37    9.46   24.91   45.08    9.46   24.91   45.08\n",
      "  142  26580 606M 256M N   O    4706 1949   32.61    6.87   25.75   49.04    6.87   25.75   49.04\n",
      "  143  26581 606M 257Q N   O    4706 1958   33.13    8.68   24.45   22.03    8.68   24.45   22.03\n",
      "  144  26582 606M 259F N   O    4706 1979   35.94    9.56   26.38   16.05    9.56   26.38   16.05\n",
      "  145  26583 606M 260G N   O    4706 1983   35.70    7.12   28.59   13.46    7.12   28.59   13.46\n",
      "  146  26584 606M 261I N   O    4706 1991   35.49    3.73   31.75   14.16    3.73   31.75   14.16\n",
      "  147  26585 606M 262D N   O    4706 1999   36.79    2.24   34.55   13.02    2.24   34.55   13.02\n",
      "  148  26586 606M 728K N   O    4706 5655   53.90    5.74   48.16   17.57    5.74   48.16   17.57\n",
      "  149  26587 607L 230T N   O    4714 1745   22.82   10.06   12.76   21.97   10.06   12.76   21.97\n",
      "  150  26588 607L 231V N   O    4714 1752   21.63    7.47   14.16   27.26    7.47   14.16   27.26\n",
      "  151  26589 607L 232T N   O    4714 1759   17.90    5.05   12.84   23.48    5.05   12.84   23.48\n",
      "  152  26590 607L 233P N   O    4714 1766   14.78    5.75    9.03   15.81    5.75    9.03   15.81\n",
      "  153  26591 607L 234S N   O    4714 1772   16.49    9.53    6.96    8.65    9.53    6.96    8.65\n",
      "  154  26592 607L 235G N   O    4714 1776   20.67    7.92   12.75   22.00    7.92   12.75   22.00\n",
      "  155  26593 607L 252F N   O    4714 1918   20.14    8.95   11.19   17.71    8.95   11.19   17.71\n",
      "  156  26594 607L 253A N   O    4714 1923   18.65    7.87   10.79   13.87    7.87   10.79   13.87\n",
      "  157  26595 607L 256M N   O    4714 1949   23.17    9.47   13.70   17.63    9.47   13.70   17.63\n",
      "  158  26596 607L 257Q N   O    4714 1958   24.05   10.98   13.07    9.26   10.98   13.07    9.26\n",
      "  159  26597 607L 260G N   O    4714 1983   25.84    9.44   16.41    6.73    9.44   16.41    6.73\n",
      "  160  26598 607L 261I N   O    4714 1991   25.70    6.26   19.44    7.73    6.26   19.44    7.73\n",
      "  161  26599 607L 262D N   O    4714 1999   26.84    3.40   23.44    8.16    3.40   23.44    8.16\n",
      "  162  26600 607L 728K N   O    4714 5655   44.09    5.32   38.77   13.33    5.32   38.77   13.33\n",
      "  163  26601 728K 231V N   O    5647 1752   28.44    8.18   20.27   44.32    8.18   20.27   44.32\n",
      "  164  26602 728K 232T N   O    5647 1759   32.50    6.93   25.58   49.71    6.93   25.58   49.71\n",
      "  165  26603 728K 233P N   O    5647 1766   35.58    7.68   27.90   52.30    7.68   27.90   52.30\n",
      "  166  26604 728K 235G N   O    5647 1776   30.75    9.89   20.86   39.35    9.89   20.86   39.35\n",
      "  167  26605 728K 253A N   O    5647 1923   36.91   10.01   26.91   37.37   10.01   26.91   37.37\n",
      "  168  26606 728K 256M N   O    5647 1949   32.13   10.79   21.35   31.14   10.79   21.35   31.14\n",
      "  169  26607 728K 260G N   O    5647 1983   28.18    9.33   18.85    8.28    9.33   18.85    8.28\n",
      "  170  26608 728K 261I N   O    5647 1991   31.27    7.36   23.91    9.83    7.36   23.91    9.83\n",
      "  171  26609 728K 262D N   O    5647 1999   29.80    3.27   26.53    9.64    3.27   26.53    9.64\n",
      "  172  26610 728K 606M N   O    5647 4713   50.33    3.30   47.03   17.77    3.30   47.03   17.77\n",
      "  173  26611 728K 607L N   O    5647 4721   45.44    2.29   43.15   15.68    2.29   43.15   15.68\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20613 168S 168S CA  C    1280 1283 -152.37 -180.00   27.63    5.52 -180.00   27.63    5.52\n",
      "    2  21052 607L 607L CA  C    4715 4720 -155.64 -180.00   24.36    4.87 -180.00   24.36    4.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19720   1M   2K C   N       7    9 -160.95 -118.00   43.21    1.71  -62.90 -159.23   23.08\n",
      "    1          2K   2K N   CA      9   10  134.40  139.10                  -40.80\n",
      "    2  19722   3P   4P C   N      23   25  -83.22  -58.70   33.48    2.44  -64.50  156.03   12.65\n",
      "    2          4P   4P N   CA     25   26   -7.71  -30.50                  147.20\n",
      "    3  19723   4P   5F C   N      30   32  -24.08  -71.40   48.19    3.55  -63.20  170.47   21.20\n",
      "    3          5F   5F N   CA     32   33  149.78  140.70                  -44.30\n",
      "    4  19724   5F   6N C   N      41   43 -155.87 -119.90   54.55    1.62  -63.20  168.64   26.18\n",
      "    4          6N   6N N   CA     43   44  178.00  137.00                  -41.10\n",
      "    5  19726   7I   8T C   N      57   59 -147.36 -124.80   45.55    1.43  -63.20  158.93   25.73\n",
      "    5          8T   8T N   CA     59   60 -176.92  143.50                  -42.10\n",
      "    6  19727   8T   9D C   N      64   66 -148.97  -70.90   87.69    4.85  -63.30  155.52   24.64\n",
      "    6          9D   9D N   CA     66   67 -169.79  150.30                  -40.00\n",
      "    7  19728   9D  10E C   N      72   74 -147.42 -117.80   58.64    2.22  -63.60  156.61   25.47\n",
      "    7         10E  10E N   CA     74   75 -172.60  136.80                  -40.30\n",
      "    8  19729  10E  11A C   N      81   83 -157.11 -134.00   41.95    1.58  -62.50  166.58   32.30\n",
      "    8         11A  11A N   CA     83   84 -177.99  147.00                  -40.90\n",
      "    9  19732  13A  14H C   N      97   99  172.71 -125.60   75.21    1.87  -63.20 -175.97   30.62\n",
      "    9         14H  14H N   CA     99  100 -178.19  138.80                  -42.30\n",
      "   10  19734  15C  16W C   N     113  115 -166.59 -124.90   51.03    1.63  -63.00  176.56   28.98\n",
      "   10         16W  16W N   CA    115  116  172.82  143.40                  -44.20\n",
      "   11  19735  16W  17D C   N     127  129 -174.87  -96.50   99.22    4.13  -63.30 -177.08   29.57\n",
      "   11         17D  17D N   CA    129  130  175.05  114.20                  -40.00\n",
      "   12  19736  17D  18E C   N     135  137 -176.37 -117.80   67.57    1.98  -63.60 -172.97   30.98\n",
      "   12         18E  18E N   CA    137  138  170.50  136.80                  -40.30\n",
      "   13  19737  18E  19S C   N     144  146  171.71 -136.60   54.55    1.77  -64.10 -160.33   23.42\n",
      "   13         19S  19S N   CA    146  147  168.65  151.20                  -35.00\n",
      "   14  19739  20V  21M C   N     157  159  171.83 -125.60   68.78    2.11  -63.40 -164.54   38.40\n",
      "   14         21M  21M N   CA    159  160  169.04  140.50                  -40.50\n",
      "   15  19740  21M  22L C   N     165  167  177.97 -108.50   81.93    3.62  -63.50 -168.69   33.69\n",
      "   15         22L  22L N   CA    167  168  168.64  132.50                  -41.20\n",
      "   16  19743  24P  25N C   N     187  189 -178.41 -119.90   70.46    2.07  -63.20 -176.62   29.11\n",
      "   16         25N  25N N   CA    189  190  176.24  137.00                  -41.10\n",
      "   17  19744  25N  26H C   N     195  197  176.99 -125.60   65.01    1.69  -63.20 -169.27   31.33\n",
      "   17         26H  26H N   CA    197  198  169.30  138.80                  -42.30\n",
      "   18  19745  26H  27R C   N     205  207  173.80 -125.20   69.58    2.09  -63.00 -169.87   33.57\n",
      "   18         27R  27R N   CA    207  208  174.08  140.60                  -41.10\n",
      "   19  19746  27R  28S C   N     216  218 -176.72 -136.60   48.21    1.47  -64.10 -174.77   21.55\n",
      "   19         28S  28S N   CA    218  219  177.94  151.20                  -35.00\n",
      "   20  19747  28S  29A C   N     222  224 -171.52 -134.00   50.45    1.43  -62.50  176.17   34.57\n",
      "   20         29A  29A N   CA    224  225 -179.28  147.00                  -40.90\n",
      "   21  19749  30V  31H C   N     234  236 -169.71 -125.60   64.83    1.62  -63.20  169.14   27.79\n",
      "   21         31H  31H N   CA    236  237 -173.69  138.80                  -42.30\n",
      "   22  19750  31H  32K C   N     244  246 -161.09 -118.00   61.81    2.19  -62.90  167.58   27.55\n",
      "   22         32K  32K N   CA    246  247 -176.59  139.10                  -40.80\n",
      "   23  19751  32K  33E C   N     253  255 -157.05 -117.80   64.53    2.25  -63.60  161.47   26.58\n",
      "   23         33E  33E N   CA    255  256 -171.98  136.80                  -40.30\n",
      "   24  19752  33E  34E C   N     262  264 -154.24 -117.80   63.60    2.28  -63.60  159.12   26.13\n",
      "   24         34E  34E N   CA    264  265 -171.08  136.80                  -40.30\n",
      "   25  19754  35C  36A C   N     277  279 -155.07 -134.00   45.77    1.91  -62.50  160.80   31.24\n",
      "   25         36A  36A N   CA    279  280 -172.37  147.00                  -40.90\n",
      "   26  19755  36A  37Y C   N     282  284 -148.06 -124.30   57.61    2.29  -63.50  154.01   28.40\n",
      "   26         37Y  37Y N   CA    284  285 -172.12  135.40                  -43.40\n",
      "   27  19758  39C  40R C   N     306  308 -161.12 -125.20   54.73    1.72  -63.00  168.52   29.21\n",
      "   27         40R  40R N   CA    308  309 -178.11  140.60                  -41.10\n",
      "   28  19759  40R  41T C   N     317  319 -171.84 -124.80   59.05    2.06  -63.20  176.18   29.27\n",
      "   28         41T  41T N   CA    319  320  179.20  143.50                  -42.10\n",
      "   29  19761  42C  43R C   N     330  332 -173.14 -125.20   65.77    1.98  -63.00  172.90   30.44\n",
      "   29         43R  43R N   CA    332  333 -174.38  140.60                  -41.10\n",
      "   30  19762  43R  44H C   N     341  343 -158.60 -125.60   55.76    1.45  -63.20  164.45   26.60\n",
      "   30         44H  44H N   CA    343  344 -176.25  138.80                  -42.30\n",
      "   31  19763  44H  45E C   N     351  353 -158.07 -117.80   64.16    2.20  -63.60  163.10   26.85\n",
      "   31         45E  45E N   CA    353  354 -173.25  136.80                  -40.30\n",
      "   32  19764  45E  46M C   N     360  362 -153.74 -125.60   57.13    2.03  -63.40  157.72   30.39\n",
      "   32         46M  46M N   CA    362  363 -169.78  140.50                  -40.50\n",
      "   33  19767  48T  49L C   N     379  381 -166.59 -108.50   58.09    2.78  -63.50 -158.90   24.67\n",
      "   33         49L  49L N   CA    381  382  131.47  132.50                  -41.20\n",
      "   34  19773  54C  55H C   N     420  422 -149.50 -125.60   42.91    1.14  -63.20  167.25   26.46\n",
      "   34         55H  55H N   CA    422  423  174.44  138.80                  -42.30\n",
      "   35  19774  55H  56M C   N     430  432 -163.49 -125.60   52.45    1.59  -63.40  174.33   33.61\n",
      "   35         56M  56M N   CA    432  433  176.76  140.50                  -40.50\n",
      "   36  19775  56M  57G C   N     438  440 -156.24 -167.20   11.67    0.33   82.20 -150.94   15.37\n",
      "   36         57G  57G N   CA    440  441  178.59  174.60                    8.50\n",
      "   37  19776  57G  58L C   N     442  444 -144.60 -108.50   52.69    2.39  -63.50  168.68   28.42\n",
      "   37         58L  58L N   CA    444  445  170.89  132.50                  -41.20\n",
      "   38  19779  60V  61E C   N     463  465  129.98 -117.80  126.79    3.71  -69.30  169.33   10.91\n",
      "   38         61E  61E N   CA    465  466 -164.18  136.80                  142.50\n",
      "   39  19780  61E  62H C   N     472  474   81.72  -67.60  151.41   12.12  -67.60  151.41   12.12\n",
      "   39         62H  62H N   CA    474  475  165.07  140.00                  140.00\n",
      "   40  19787  68L  69N C   N     532  534  173.57 -119.90   67.51    2.51  -63.20 -149.65   33.01\n",
      "   40         69N  69N N   CA    534  535  148.43  137.00                  -41.10\n",
      "   41  19790  71P  72T C   N     553  555 -126.22 -124.80   30.75    1.28  -63.20  156.90   24.25\n",
      "   41         72T  72T N   CA    555  556  174.21  143.50                  -42.10\n",
      "   42  19792  73H  74V C   N     570  572  -65.26  -62.40   13.07    1.83 -125.40  172.37    9.68\n",
      "   42         74V  74V N   CA    572  573  -55.15  -42.40                  143.30\n",
      "   43  19793  74V  75M C   N     577  579  158.14 -125.60   90.77    4.70  -63.40 -168.87   24.94\n",
      "   43         75M  75M N   CA    579  580   91.26  140.50                  -40.50\n",
      "   44  19794  75M  76Y C   N     585  587 -153.38 -124.30   57.67    2.27  -63.50  159.20   29.44\n",
      "   44         76Y  76Y N   CA    587  588 -174.80  135.40                  -43.40\n",
      "   45  19796  77V  78W C   N     604  606 -135.78 -124.90   47.51    2.03  -63.00  145.64   23.10\n",
      "   45         78W  78W N   CA    606  607 -170.35  143.40                  -44.20\n",
      "   46  19800  81E  82L C   N     644  646  178.58 -108.50   76.71    3.43  -70.70  111.70    9.65\n",
      "   46         82L  82L N   CA    646  647  156.34  132.50                  141.60\n",
      "   47  19802  83P  84A C   N     659  661 -140.45 -134.00   59.75    3.24  -62.50  137.03   26.58\n",
      "   47         84A  84A N   CA    661  662 -153.60  147.00                  -40.90\n",
      "   48  19804  85K  86D C   N     673  675   67.40  -63.30  131.15   21.29  -70.90 -133.50    8.12\n",
      "   48         86D  86D N   CA    675  676  -29.08  -40.00                  150.30\n",
      "   49  19805  86D  87E C   N     681  683 -174.90 -117.80   63.45    1.86  -63.60 -168.98   31.48\n",
      "   49         87E  87E N   CA    683  684  164.46  136.80                  -40.30\n",
      "   50  19806  87E  88D C   N     690  692 -124.87  -70.90   67.67    3.15  -63.30  142.84   21.60\n",
      "   50         88D  88D N   CA    692  693 -168.89  150.30                  -40.00\n",
      "   51  19807  88D  89A C   N     698  700 -168.37 -134.00   48.04    1.43  -62.50  174.36   34.12\n",
      "   51         89A  89A N   CA    700  701 -179.44  147.00                  -40.90\n",
      "   52  19808  89A  90G C   N     703  705 -139.38 -167.20   29.35    0.42   82.20 -150.88   15.83\n",
      "   52         90G  90G N   CA    705  706  165.25  174.60                    8.50\n",
      "   53  19809  90G  91G C   N     707  709   80.91   78.70   30.23    0.95   82.20  144.46    7.08\n",
      "   53         91G  91G N   CA    709  710 -135.95 -166.10                    8.50\n",
      "   54  19811  92S  93K C   N     717  719 -134.70 -118.00   20.56    0.69  -62.90 -177.20   28.04\n",
      "   54         93K  93K N   CA    719  720  151.09  139.10                  -40.80\n",
      "   55  19812  93K  94D C   N     726  728 -141.31  -96.50   57.33    2.39  -63.30 -172.93   28.13\n",
      "   55         94D  94D N   CA    728  729  149.97  114.20                  -40.00\n",
      "   56  19814  95V  96N C   N     741  743  -47.78  -63.20   22.79    3.66   55.90  121.75   14.98\n",
      "   56         96N  96N N   CA    743  744  -24.31  -41.10                   39.50\n",
      "   57  19815  96N  97K C   N     749  751  -75.42  -70.20   39.20    2.72  -62.90  140.51   18.96\n",
      "   57         97K  97K N   CA    751  752  179.25  140.40                  -40.80\n",
      "   58  19816  97K  98L C   N     758  760   57.10  -70.70  132.34   10.78  -63.50 -168.76   33.78\n",
      "   58         98L  98L N   CA    760  761  107.22  141.60                  -41.20\n",
      "   59  19819 100V 101L C   N     777  779  155.27 -108.50   96.56    4.70  -63.50 -142.24   27.41\n",
      "   59        101L 101L N   CA    779  780  124.55  132.50                  -41.20\n",
      "   60  19820 101L 102A C   N     785  787  155.34 -134.00   76.11    1.74  -62.50 -157.76   40.37\n",
      "   60        102A 102A N   CA    787  788  175.27  147.00                  -40.90\n",
      "   61  19822 103P 104K C   N     797  799  128.88 -118.00  115.29    4.02  -62.90 -129.35   39.45\n",
      "   61        104K 104K N   CA    799  800  161.40  139.10                  -40.80\n",
      "   62  19823 104K 105E C   N     806  808 -169.57 -117.80   54.23    1.66  -63.60 -162.43   32.14\n",
      "   62        105E 105E N   CA    808  809  152.95  136.80                  -40.30\n",
      "   63  19824 105E 106Y C   N     815  817 -159.61 -124.30   44.21    2.18  -63.50 -177.97   33.43\n",
      "   63        106Y 106Y N   CA    817  818  162.01  135.40                  -43.40\n",
      "   64  19825 106Y 107E C   N     827  829 -163.71 -117.80   48.15    1.47  -63.60 -164.11   31.63\n",
      "   64        107E 107E N   CA    829  830  151.32  136.80                  -40.30\n",
      "   65  19826 107E 108N C   N     836  838  -97.44 -119.90   74.91    2.84  -63.20  112.00   12.44\n",
      "   65        108N 108N N   CA    838  839   65.54  137.00                  -41.10\n",
      "   66  19827 108N 109A C   N     844  846  164.07 -134.00   79.76    2.13  -62.50 -179.31   36.29\n",
      "   66        109A 109A N   CA    846  847 -162.74  147.00                  -40.90\n",
      "   67  19828 109A 110L C   N     849  851  -94.95 -108.50   15.63    0.69  -63.50  168.87   21.85\n",
      "   67        110L 110L N   CA    851  852  124.72  132.50                  -41.20\n",
      "   68  19831 112C 113A C   N     869  871 -147.84 -134.00   33.74    1.48  -62.50  165.10   31.60\n",
      "   68        113A 113A N   CA    871  872  177.77  147.00                  -40.90\n",
      "   69  19832 113A 114A C   N     874  876 -156.64 -134.00   36.53    1.26  -62.50  171.57   33.11\n",
      "   69        114A 114A N   CA    876  877  175.66  147.00                  -40.90\n",
      "   70  19834 115C 116A C   N     885  887 -151.55 -134.00   22.05    0.57  -68.20   84.70    6.65\n",
      "   70        116A 116A N   CA    887  888  160.34  147.00                  145.30\n",
      "   71  19836 117K 118S C   N     899  901  -66.63  -64.10    4.28    0.49  -72.40  169.24   10.16\n",
      "   71        118S 118S N   CA    901  902  -38.46  -35.00                  152.40\n",
      "   72  19837 118S 119F C   N     905  907 -144.83 -124.20   42.50    1.40  -63.20  157.97   27.05\n",
      "   72        119F 119F N   CA    907  908 -179.54  143.30                  -44.30\n",
      "   73  19838 119F 120V C   N     916  918  -60.69  -62.40    6.64    0.94 -125.40 -169.39    7.60\n",
      "   73        120V 120V N   CA    918  919  -35.99  -42.40                  143.30\n",
      "   74  19839 120V 121S C   N     923  925 -129.43 -136.60   31.11    1.41  -64.10  169.06   10.51\n",
      "   74        121S 121S N   CA    925  926  120.92  151.20                  -35.00\n",
      "   75  19840 121S 122P C   N     929  931  -39.55  -58.70   45.10    2.64  -64.50  143.65   12.07\n",
      "   75        122P 122P N   CA    931  932  -71.33  -30.50                  147.20\n",
      "   76  19842 123P 124E C   N     943  945  151.60 -117.80   90.61    3.36  -63.60 -132.48   26.59\n",
      "   76        124E 124E N   CA    945  946  135.19  136.80                  -40.30\n",
      "   77  19843 124E 125L C   N     952  954 -155.06 -108.50   46.57    2.20  -63.50 -162.64   24.18\n",
      "   77        125L 125L N   CA    954  955  133.63  132.50                  -41.20\n",
      "   78  19844 125L 126T C   N     960  962 -176.40 -124.80   80.21    2.51  -63.20  159.95   27.15\n",
      "   78        126T 126T N   CA    962  963 -155.10  143.50                  -42.10\n",
      "   79  19846 127I 128D C   N     975  977   72.02   54.50  127.89   10.82  -63.30 -156.18   22.53\n",
      "   79        128D 128D N   CA    977  978  167.58   40.90                  -40.00\n",
      "   80  19848 129C 130Y C   N     989  991 -172.10 -124.30   47.84    3.54  -63.50 -150.30   38.44\n",
      "   80        130Y 130Y N   CA    991  992  137.21  135.40                  -43.40\n",
      "   81  19849 130Y 131Q C   N    1001 1003 -174.36 -121.10   55.20    1.81  -63.80 -160.97   34.83\n",
      "   81        131Q 131Q N   CA   1003 1004  154.21  139.70                  -40.30\n",
      "   82  19850 131Q 132W C   N    1010 1012 -150.68 -124.90   28.07    0.97  -63.00 -176.39   28.86\n",
      "   82        132W 132W N   CA   1012 1013  154.48  143.40                  -44.20\n",
      "   83  19853 134I 135N C   N    1040 1042   90.33   55.90  161.31    8.94  -63.20  171.75   21.73\n",
      "   83        135N 135N N   CA   1042 1043 -118.09   39.50                  -41.10\n",
      "   84  19854 135N 136A C   N    1048 1050  -64.61  -68.20    3.60    0.30  -62.50  174.03   28.62\n",
      "   84        136A 136A N   CA   1050 1051  145.09  145.30                  -40.90\n",
      "   85  19855 136A 137T C   N    1053 1055 -147.42 -124.80   34.91    1.10  -63.20  170.12   27.22\n",
      "   85        137T 137T N   CA   1055 1056  170.09  143.50                  -42.10\n",
      "   86  19856 137T 138S C   N    1060 1062 -176.13 -136.60   39.63    1.50  -64.10 -155.64   22.89\n",
      "   86        138S 138S N   CA   1062 1063  154.08  151.20                  -35.00\n",
      "   87  19857 138S 139T C   N    1066 1068 -169.68 -124.80   84.53    2.62  -63.20  147.98   25.18\n",
      "   87        139T 139T N   CA   1068 1069 -144.87  143.50                  -42.10\n",
      "   88  19858 139T 140G C   N    1073 1075  -57.34  -62.40   12.85    2.19   82.20  144.60   10.82\n",
      "   88        140G 140G N   CA   1075 1076  -29.38  -41.20                    8.50\n",
      "   89  19859 140G 141A C   N    1077 1079  -90.01  -68.20   43.05    2.78  -62.50  139.42   24.29\n",
      "   89        141A 141A N   CA   1079 1080 -177.58  145.30                  -40.90\n",
      "   90  19860 141A 142Q C   N    1082 1084 -133.84 -121.10   40.40    1.67  -63.80  158.03   26.88\n",
      "   90        142Q 142Q N   CA   1084 1085  178.03  139.70                  -40.30\n",
      "   91  19861 142Q 143A C   N    1091 1093 -135.57 -134.00   21.09    1.16  -62.50  167.82   31.42\n",
      "   91        143A 143A N   CA   1093 1094  168.03  147.00                  -40.90\n",
      "   92  19862 143A 144A C   N    1096 1098 -157.64 -134.00   40.63    1.47  -62.50  168.50   32.64\n",
      "   92        144A 144A N   CA   1098 1099 -179.97  147.00                  -40.90\n",
      "   93  19864 145I 146E C   N    1109 1111   80.46  -69.30  150.41   10.89  -63.60 -138.07   37.17\n",
      "   93        146E 146E N   CA   1111 1112  128.52  142.50                  -40.30\n",
      "   94  19866 147P 148E C   N    1125 1127  147.44 -117.80  101.06    4.60  -63.60 -154.22   24.71\n",
      "   94        148E 148E N   CA   1127 1128  101.68  136.80                  -40.30\n",
      "   95  19869 150V 151A C   N    1145 1147 -136.46 -134.00    8.69    0.42  -62.50  179.70   33.41\n",
      "   95        151A 151A N   CA   1147 1148  155.33  147.00                  -40.90\n",
      "   96  19870 151A 152S C   N    1150 1152 -162.76 -136.60   31.84    0.97  -64.10 -175.73   20.47\n",
      "   96        152S 152S N   CA   1152 1153  169.36  151.20                  -35.00\n",
      "   97  19871 152S 153M C   N    1156 1158  175.15 -125.60   76.17    2.26  -63.40  178.73   35.48\n",
      "   97        153M 153M N   CA   1158 1159 -171.63  140.50                  -40.50\n",
      "   98  19872 153M 154R C   N    1164 1166  119.83   57.30  111.93   12.04  -63.00 -113.12   29.35\n",
      "   98        154R 154R N   CA   1166 1167  130.84   38.00                  -41.10\n",
      "   99  19873 154R 155L C   N    1175 1177  153.17 -108.50  162.53    9.23  -70.70 -165.79   19.31\n",
      "   99        155L 155L N   CA   1177 1178    3.09  132.50                  141.60\n",
      "  100  19877 158P 159H C   N    1205 1207  168.95 -125.60   69.13    1.96  -63.20 -157.79   33.25\n",
      "  100        159H 159H N   CA   1207 1208  161.04  138.80                  -42.30\n",
      "  101  19878 159H 160L C   N    1215 1217  169.61 -108.50   82.39    3.82  -63.50 -142.06   37.96\n",
      "  101        160L 160L N   CA   1217 1218  141.61  132.50                  -41.20\n",
      "  102  19882 163L 164E C   N    1244 1246 -141.00  -63.60   77.46   12.15  -63.60   77.46   12.15\n",
      "  102        164E 164E N   CA   1246 1247  -37.25  -40.30                  -40.30\n",
      "  103  19883 164E 165Q C   N    1253 1255 -161.51 -121.10   43.14    1.94  -63.80 -168.32   24.61\n",
      "  103        165Q 165Q N   CA   1255 1256  124.60  139.70                  -40.30\n",
      "  104  19884 165Q 166L C   N    1262 1264  114.75 -108.50  146.36    7.64  -63.50 -144.25   39.67\n",
      "  104        166L 166L N   CA   1264 1265   80.34  132.50                  -41.20\n",
      "  105  19886 167P 168S C   N    1277 1279  -53.68 -136.60   91.48    2.84  -64.10  147.92   11.37\n",
      "  105        168S 168S N   CA   1279 1280  112.55  151.20                  -35.00\n",
      "  106  19888 169P 170F C   N    1290 1292 -157.78 -124.20   39.63    1.04  -63.20  178.48   30.68\n",
      "  106        170F 170F N   CA   1292 1293  164.34  143.30                  -44.30\n",
      "  107  19889 170F 171Q C   N    1301 1303 -133.61 -121.10   15.71    0.49  -63.80 -175.76   30.77\n",
      "  107        171Q 171Q N   CA   1303 1304  149.20  139.70                  -40.30\n",
      "  108  19890 171Q 172T C   N    1310 1312  -87.77  -78.10   35.67    1.25  -63.20  136.00   19.08\n",
      "  108        172T 172T N   CA   1312 1313 -175.86  149.80                  -42.10\n",
      "  109  19891 172T 173A C   N    1317 1319   76.04  -68.20  150.79   14.09  -62.50 -170.14   29.56\n",
      "  109        173A 173A N   CA   1319 1320 -170.72  145.30                  -40.90\n",
      "  110  19892 173A 174P C   N    1322 1324  -65.36  -58.70   16.23    0.95  -64.50  162.91   12.20\n",
      "  110        174P 174P N   CA   1324 1325  -15.70  -30.50                  147.20\n",
      "  111  19893 174P 175T C   N    1329 1331  -85.63  -78.10   12.00    0.41  -63.20  160.33   22.05\n",
      "  111        175T 175T N   CA   1331 1332  159.14  149.80                  -42.10\n",
      "  112  19894 175T 176S C   N    1336 1338 -163.42 -136.60   33.57    1.03  -64.10 -177.08   20.42\n",
      "  112        176S 176S N   CA   1338 1339  171.39  151.20                  -35.00\n",
      "  113  19895 176S 177S C   N    1342 1344 -166.65 -136.60   32.40    1.02  -64.10 -168.55   21.27\n",
      "  113        177S 177S N   CA   1344 1345  163.32  151.20                  -35.00\n",
      "  114  19896 177S 178D C   N    1348 1350 -158.87  -96.50   67.90    2.80  -63.30 -157.12   31.19\n",
      "  114        178D 178D N   CA   1350 1351  141.04  114.20                  -40.00\n",
      "  115  19900 181A 182L C   N    1376 1378 -165.83 -108.50  117.05    6.67  -70.70  146.31   14.40\n",
      "  115        182L 182L N   CA   1378 1379   30.45  132.50                  141.60\n",
      "  116  19901 182L 183D C   N    1384 1386   24.60  -70.90   96.06    6.91  -63.30 -159.74   30.39\n",
      "  116        183D 183D N   CA   1386 1387  139.93  150.30                  -40.00\n",
      "  117  19904 185C 186E C   N    1402 1404 -140.06 -117.80   22.27    0.80  -63.60 -166.55   22.62\n",
      "  117        186E 186E N   CA   1404 1405  137.40  136.80                  -40.30\n",
      "  118  19906 187C 188R C   N    1417 1419 -147.59 -125.20   24.88    0.76  -63.00 -172.40   31.09\n",
      "  118        188R 188R N   CA   1419 1420  151.45  140.60                  -41.10\n",
      "  119  19921 202C 203P C   N    1530 1532  -54.45  -58.70   32.50    2.36  -64.50  150.42   11.76\n",
      "  119        203P 203P N   CA   1532 1533  -62.72  -30.50                  147.20\n",
      "  120  19922 203P 204R C   N    1537 1539 -175.83 -125.20   51.33    1.85  -72.10  103.97    7.78\n",
      "  120        204R 204R N   CA   1539 1540  149.11  140.60                  141.90\n",
      "  121  19924 205E 206E C   N    1557 1559   57.65   54.60    3.71    0.28  -63.60  145.59   25.04\n",
      "  121        206E 206E N   CA   1559 1560   40.28   42.40                  -40.30\n",
      "  122  19925 206E 207A C   N    1566 1568   60.67  -68.20  129.07   11.18  -68.20  129.07   11.18\n",
      "  122        207A 207A N   CA   1568 1569  152.50  145.30                  145.30\n",
      "  123  19928 209G 210N C   N    1579 1581 -137.00  -63.20   97.88   11.18  -63.20   97.88   11.18\n",
      "  123        210N 210N N   CA   1581 1582   23.21  -41.10                  -41.10\n",
      "  124  19977 258H 259F C   N    1967 1969 -115.76  -63.20   52.85    8.19  -63.20   52.85    8.19\n",
      "  124        259F 259F N   CA   1969 1970  -38.77  -44.30                  -44.30\n",
      "  125  19982 263I 264Q C   N    2006 2008 -139.55 -121.10   40.58    1.55  -63.80  162.58   27.83\n",
      "  125        264Q 264Q N   CA   2008 2009  175.85  139.70                  -40.30\n",
      "  126  19983 264Q 265T C   N    2015 2017 -174.79 -124.80   63.35    2.20  -63.20  175.53   29.31\n",
      "  126        265T 265T N   CA   2017 2018 -177.59  143.50                  -42.10\n",
      "  127  19984 265T 266A C   N    2022 2024 -179.06 -134.00   52.13    1.22  -62.50 -173.25   36.69\n",
      "  127        266A 266A N   CA   2024 2025  173.20  147.00                  -40.90\n",
      "  128  19985 266A 267K C   N    2027 2029 -169.87 -118.00   61.58    2.05  -62.90 -178.26   29.90\n",
      "  128        267K 267K N   CA   2029 2030  172.28  139.10                  -40.80\n",
      "  129  19986 267K 268K C   N    2036 2038 -174.64 -118.00   62.29    2.06  -62.90 -169.60   31.31\n",
      "  129        268K 268K N   CA   2038 2039  165.04  139.10                  -40.80\n",
      "  130  19987 268K 269T C   N    2045 2047 -161.08 -124.80   51.49    1.66  -63.20  169.08   27.80\n",
      "  130        269T 269T N   CA   2047 2048 -179.97  143.50                  -42.10\n",
      "  131  19988 269T 270A C   N    2052 2054  179.96 -134.00   53.89    1.27  -62.50 -174.04   36.60\n",
      "  131        270A 270A N   CA   2054 2055  175.01  147.00                  -40.90\n",
      "  132  19989 270A 271K C   N    2057 2059  175.58 -118.00   71.08    2.37  -62.90 -163.22   32.66\n",
      "  132        271K 271K N   CA   2059 2060  164.43  139.10                  -40.80\n",
      "  133  19990 271K 272T C   N    2066 2068 -171.71 -124.80   57.23    2.05  -63.20  178.41   29.58\n",
      "  133        272T 272T N   CA   2068 2069  176.28  143.50                  -42.10\n",
      "  134  19991 272T 273L C   N    2073 2075 -172.77 -108.50   75.29    3.32  -63.50 -176.78   32.05\n",
      "  134        273L 273L N   CA   2075 2076  171.73  132.50                  -41.20\n",
      "  135  19992 273L 274G C   N    2081 2083  175.08 -167.20   17.72    0.35   82.20 -170.03   13.38\n",
      "  135        274G 274G N   CA   2083 2084  174.22  174.60                    8.50\n",
      "  136  19993 274G 275E C   N    2085 2087  173.14 -117.80   76.04    2.23  -63.60 -165.03   32.53\n",
      "  136        275E 275E N   CA   2087 2088  168.63  136.80                  -40.30\n",
      "  137  19994 275E 276M C   N    2094 2096  169.66 -125.60   72.19    2.19  -63.40 -165.73   38.32\n",
      "  137        276M 276M N   CA   2096 2097  172.44  140.50                  -40.50\n",
      "  138  19995 276M 277Q C   N    2102 2104  174.91 -121.10   67.44    2.16  -63.80 -160.26   35.34\n",
      "  138        277Q 277Q N   CA   2104 2105  161.00  139.70                  -40.30\n",
      "  139  19996 277Q 278Y C   N    2111 2113 -169.53 -124.30   59.19    2.80  -63.50  178.03   33.10\n",
      "  139        278Y 278Y N   CA   2113 2114  173.58  135.40                  -43.40\n",
      "  140  19997 278Y 279D C   N    2123 2125 -175.73  -96.50   92.90    3.86  -63.30 -166.66   30.96\n",
      "  140        279D 279D N   CA   2125 2126  162.71  114.20                  -40.00\n",
      "  141  19998 279D 280Y C   N    2131 2133 -161.36 -124.30   57.26    2.40  -63.50  168.80   31.30\n",
      "  141        280Y 280Y N   CA   2133 2134  179.05  135.40                  -43.40\n",
      "  142  19999 280Y 281S C   N    2143 2145 -167.00 -136.60   39.99    1.25  -64.10 -179.90   20.48\n",
      "  142        281S 281S N   CA   2145 2146  177.18  151.20                  -35.00\n",
      "  143  20001 282C 283Q C   N    2155 2157 -167.91 -121.10   55.18    1.70  -63.80 -176.76   32.16\n",
      "  143        283Q 283Q N   CA   2157 2158  168.91  139.70                  -40.30\n",
      "  144  20002 283Q 284F C   N    2164 2166 -179.51 -124.20   61.43    1.69  -63.20 -173.60   32.82\n",
      "  144        284F 284F N   CA   2166 2167  170.04  143.30                  -44.30\n",
      "  145  20003 284F 285D C   N    2175 2177 -167.55  -96.50   85.39    3.55  -63.30 -170.35   30.03\n",
      "  145        285D 285D N   CA   2177 2178  161.57  114.20                  -40.00\n",
      "  146  20004 285D 286F C   N    2183 2185 -170.13 -124.20   57.02    1.48  -63.20  175.06   30.73\n",
      "  146        286F 286F N   CA   2185 2186  177.09  143.30                  -44.30\n",
      "  147  20005 286F 287K C   N    2194 2196 -161.18 -118.00   49.02    1.62  -62.90 -174.87   29.88\n",
      "  147        287K 287K N   CA   2196 2197  162.31  139.10                  -40.80\n",
      "  148  20006 287K 288R C   N    2203 2205 -160.65 -125.20   56.79    1.83  -63.00  165.75   28.79\n",
      "  148        288R 288R N   CA   2205 2206 -175.03  140.60                  -41.10\n",
      "  149  20008 289I 290T C   N    2222 2224 -150.02 -124.80   52.44    1.65  -63.20  155.01   25.34\n",
      "  149        290T 290T N   CA   2224 2225 -170.52  143.50                  -42.10\n",
      "  150  20009 290T 291E C   N    2229 2231 -138.16 -117.80   47.25    1.89  -63.60  158.85   25.32\n",
      "  150        291E 291E N   CA   2231 2232  179.44  136.80                  -40.30\n",
      "  151  20010 291E 292N C   N    2238 2240 -145.29 -119.90   43.31    1.34  -63.20  168.21   25.61\n",
      "  151        292N 292N N   CA   2240 2241  172.09  137.00                  -41.10\n",
      "  152  20011 292N 293G C   N    2246 2248 -165.85 -167.20   10.18    0.38   82.20 -167.98   14.12\n",
      "  152        293G 293G N   CA   2248 2249  164.51  174.60                    8.50\n",
      "  153  20012 293G 294D C   N    2250 2252 -158.85  -96.50   76.58    3.19  -63.30 -172.50   29.26\n",
      "  153        294D 294D N   CA   2252 2253  158.67  114.20                  -40.00\n",
      "  154  20013 294D 295S C   N    2258 2260 -163.20 -136.60   29.89    0.92  -64.10 -171.65   20.79\n",
      "  154        295S 295S N   CA   2260 2261  164.82  151.20                  -35.00\n",
      "  155  20014 295S 296L C   N    2264 2266 -157.21 -108.50   51.62    2.30  -63.50 -166.58   32.63\n",
      "  155        296L 296L N   CA   2266 2267  149.59  132.50                  -41.20\n",
      "  156  20017 298P 299A C   N    2286 2288 -160.29 -134.00   29.85    0.69  -62.50 -174.21   35.65\n",
      "  156        299A 299A N   CA   2288 2289  161.13  147.00                  -40.90\n",
      "  157  20018 299A 300F C   N    2291 2293 -161.73 -124.20   42.66    1.15  -63.20 -178.77   31.28\n",
      "  157        300F 300F N   CA   2293 2294  163.59  143.30                  -44.30\n",
      "  158  20019 300F 301G C   N    2302 2304 -172.27 -167.20   37.08    1.50   82.20  166.94   12.51\n",
      "  158        301G 301G N   CA   2304 2305  137.86  174.60                    8.50\n",
      "  159  20021 302P 303G C   N    2313 2315 -163.03 -167.20   15.83    0.55   82.20 -170.48   14.06\n",
      "  159        303G 303G N   CA   2315 2316  159.33  174.60                    8.50\n",
      "  160  20022 303G 304R C   N    2317 2319 -156.03 -125.20   37.89    1.12  -63.00 -178.13   30.80\n",
      "  160        304R 304R N   CA   2319 2320  162.63  140.60                  -41.10\n",
      "  161  20023 304R 305T C   N    2328 2330 -150.47 -124.80   42.39    1.32  -63.20  165.54   26.78\n",
      "  161        305T 305T N   CA   2330 2331  177.23  143.50                  -42.10\n",
      "  162  20024 305T 306G C   N    2335 2337 -178.15 -167.20   16.66    0.64   82.20 -176.97   13.24\n",
      "  162        306G 306G N   CA   2337 2338  162.03  174.60                    8.50\n",
      "  163  20025 306G 307M C   N    2339 2341 -161.74 -125.60   44.73    1.32  -63.40 -178.43   34.63\n",
      "  163        307M 307M N   CA   2341 2342  166.86  140.50                  -40.50\n",
      "  164  20026 307M 308H C   N    2347 2349 -158.32 -125.60   39.98    0.99  -63.20 -177.35   28.95\n",
      "  164        308H 308H N   CA   2349 2350  161.77  138.80                  -42.30\n",
      "  165  20027 308H 309N C   N    2357 2359 -155.91 -119.90   43.62    1.28  -63.20 -177.43   27.98\n",
      "  165        309N 309N N   CA   2359 2360  161.62  137.00                  -41.10\n",
      "  166  20028 309N 310F C   N    2365 2367 -163.89 -124.20   44.64    1.21  -63.20 -177.71   31.53\n",
      "  166        310F 310F N   CA   2367 2368  163.74  143.30                  -44.30\n",
      "  167  20029 310F 311G C   N    2376 2378  176.12 -167.20   23.33    0.87   82.20  176.79   12.71\n",
      "  167        311G 311G N   CA   2378 2379  158.28  174.60                    8.50\n",
      "  168  20030 311G 312N C   N    2380 2382 -159.05 -119.90   48.93    1.42  -63.20 -179.84   27.82\n",
      "  168        312N 312N N   CA   2382 2383  166.35  137.00                  -41.10\n",
      "  169  20031 312N 313S C   N    2388 2390 -160.17 -136.60   27.58    0.84  -64.10 -173.82   20.41\n",
      "  169        313S 313S N   CA   2390 2391  165.52  151.20                  -35.00\n",
      "  170  20033 314C 315Y C   N    2400 2402 -143.20 -124.30   28.50    1.21  -63.50  178.63   32.24\n",
      "  170        315Y 315Y N   CA   2402 2403  156.73  135.40                  -43.40\n",
      "  171  20034 315Y 316M C   N    2412 2414 -155.92 -125.60   31.80    1.04  -63.40 -166.97   36.01\n",
      "  171        316M 316M N   CA   2414 2415  150.09  140.50                  -40.50\n",
      "  172  20035 316M 317A C   N    2420 2422 -162.05 -134.00   30.91    0.70  -62.50 -172.31   36.05\n",
      "  172        317A 317A N   CA   2422 2423  159.98  147.00                  -40.90\n",
      "  173  20036 317A 318S C   N    2425 2427 -161.87 -136.60   27.64    0.86  -64.10 -170.28   20.79\n",
      "  173        318S 318S N   CA   2427 2428  162.41  151.20                  -35.00\n",
      "  174  20038 319V 320L C   N    2438 2440 -142.11 -108.50   40.93    1.81  -63.50 -179.10   29.98\n",
      "  174        320L 320L N   CA   2440 2441  155.87  132.50                  -41.20\n",
      "  175  20039 320L 321Q C   N    2446 2448 -152.87 -121.10   36.47    1.12  -63.80 -175.05   31.79\n",
      "  175        321Q 321Q N   CA   2448 2449  157.61  139.70                  -40.30\n",
      "  176  20041 322C 323L C   N    2461 2463 -142.08 -108.50   42.04    1.86  -63.50  179.17   29.74\n",
      "  176        323L 323L N   CA   2463 2464  157.78  132.50                  -41.20\n",
      "  177  20042 323L 324F C   N    2469 2471 -148.97 -124.20   27.79    0.76  -63.20 -178.65   30.65\n",
      "  177        324F 324F N   CA   2471 2472  155.91  143.30                  -44.30\n",
      "  178  20043 324F 325S C   N    2480 2482 -155.27 -136.60   19.41    0.65  -64.10 -168.40   20.44\n",
      "  178        325S 325S N   CA   2482 2483  156.48  151.20                  -35.00\n",
      "  179  20044 325S 326L C   N    2486 2488 -151.33 -108.50   43.54    1.99  -63.50 -161.07   33.05\n",
      "  179        326L 326L N   CA   2488 2489  140.31  132.50                  -41.20\n",
      "  180  20045 326L 327E C   N    2494 2496 -138.14 -117.80   20.63    0.84  -63.60 -171.00   22.10\n",
      "  180        327E 327E N   CA   2496 2497  133.37  136.80                  -40.30\n",
      "  181  20047 328P 329F C   N    2510 2512 -148.49 -124.20   27.50    0.74  -63.20 -179.14   30.55\n",
      "  181        329F 329F N   CA   2512 2513  156.21  143.30                  -44.30\n",
      "  182  20048 329F 330K C   N    2521 2523 -149.59 -118.00   34.74    1.15  -62.90 -173.05   29.46\n",
      "  182        330K 330K N   CA   2523 2524  153.56  139.10                  -40.80\n",
      "  183  20049 330K 331E C   N    2530 2532 -159.03 -117.80   45.61    1.34  -63.60 -170.77   30.50\n",
      "  183        331E 331E N   CA   2532 2533  156.30  136.80                  -40.30\n",
      "  184  20050 331E 332A C   N    2539 2541 -159.01 -134.00   26.47    0.62  -62.50 -170.21   36.25\n",
      "  184        332A 332A N   CA   2541 2542  155.68  147.00                  -40.90\n",
      "  185  20051 332A 333F C   N    2544 2546 -153.23 -124.20   30.63    0.92  -63.20 -174.11   31.53\n",
      "  185        333F 333F N   CA   2546 2547  153.06  143.30                  -44.30\n",
      "  186  20052 333F 334Y C   N    2555 2557 -138.76 -124.30   18.45    0.89  -63.50 -174.32   33.19\n",
      "  186        334Y 334Y N   CA   2557 2558  146.85  135.40                  -43.40\n",
      "  187  20053 334Y 335Y C   N    2567 2569 -140.86 -124.30   26.28    1.09  -63.50  178.43   32.11\n",
      "  187        335Y 335Y N   CA   2569 2570  155.81  135.40                  -43.40\n",
      "  188  20054 335Y 336N C   N    2579 2581 -144.00 -119.90   32.20    0.93  -63.20  179.73   27.01\n",
      "  188        336N 336N N   CA   2581 2582  158.35  137.00                  -41.10\n",
      "  189  20056 337C 338E C   N    2593 2595 -153.97 -117.80   37.75    1.16  -63.60 -165.65   30.94\n",
      "  189        338E 338E N   CA   2595 2596  147.64  136.80                  -40.30\n",
      "  190  20057 338E 339T C   N    2602 2604 -146.75 -124.80   25.11    0.97  -63.20 -177.55   28.81\n",
      "  190        339T 339T N   CA   2604 2605  155.70  143.50                  -42.10\n",
      "  191  20058 339T 340R C   N    2609 2611 -156.57 -125.20   34.61    1.06  -63.00 -171.46   31.76\n",
      "  191        340R 340R N   CA   2611 2612  155.22  140.60                  -41.10\n",
      "  192  20059 340R 341H C   N    2620 2622 -160.54 -125.60   37.69    1.03  -63.20 -168.63   30.19\n",
      "  192        341H 341H N   CA   2622 2623  152.93  138.80                  -42.30\n",
      "  193  20060 341H 342Q C   N    2630 2632 -158.65 -121.10   39.60    1.27  -63.80 -167.56   33.17\n",
      "  193        342Q 342Q N   CA   2632 2633  152.26  139.70                  -40.30\n",
      "  194  20061 342Q 343N C   N    2639 2641 -152.46 -119.90   37.01    1.13  -63.20 -173.02   28.37\n",
      "  194        343N 343N N   CA   2641 2642  154.60  137.00                  -41.10\n",
      "  195  20062 343N 344A C   N    2647 2649 -161.55 -134.00   29.13    0.68  -62.50 -169.57   36.48\n",
      "  195        344A 344A N   CA   2649 2650  156.46  147.00                  -40.90\n",
      "  196  20064 345C 346Q C   N    2658 2660 -156.80 -121.10   38.58    1.21  -63.80 -170.26   32.68\n",
      "  196        346Q 346Q N   CA   2660 2661  154.32  139.70                  -40.30\n",
      "  197  20065 346Q 347E C   N    2667 2669 -151.01 -117.80   35.96    1.06  -63.60 -169.63   30.25\n",
      "  197        347E 347E N   CA   2669 2670  150.58  136.80                  -40.30\n",
      "  198  20066 347E 348N C   N    2676 2678 -139.97 -119.90   20.09    0.81  -63.20 -165.18   21.19\n",
      "  198        348N 348N N   CA   2678 2679  137.96  137.00                  -41.10\n",
      "  199  20068 349P 350Y C   N    2691 2693 -138.94 -124.30   24.14    0.98  -63.50  178.71   32.07\n",
      "  199        350Y 350Y N   CA   2693 2694  154.59  135.40                  -43.40\n",
      "  200  20069 350Y 351K C   N    2703 2705 -137.41 -118.00   24.93    0.85  -62.90 -179.46   27.91\n",
      "  200        351K 351K N   CA   2705 2706  154.75  139.10                  -40.80\n",
      "  201  20071 352C 353H C   N    2718 2720 -152.03 -125.60   30.32    0.78  -63.20 -173.45   29.07\n",
      "  201        353H 353H N   CA   2720 2721  153.65  138.80                  -42.30\n",
      "  202  20074 355C 356Q C   N    2740 2742 -151.80 -121.10   33.79    1.05  -63.80 -172.22   32.16\n",
      "  202        356Q 356Q N   CA   2742 2743  153.82  139.70                  -40.30\n",
      "  203  20075 356Q 357T C   N    2749 2751 -141.93 -124.80   21.26    0.75  -63.20  179.94   28.20\n",
      "  203        357T 357T N   CA   2751 2752  156.09  143.50                  -42.10\n",
      "  204  20076 357T 358E C   N    2756 2758 -147.70 -117.80   33.54    0.98  -63.60 -172.39   29.71\n",
      "  204        358E 358E N   CA   2758 2759  151.99  136.80                  -40.30\n",
      "  205  20077 358E 359R C   N    2765 2767 -148.12 -125.20   23.08    0.86  -63.00 -164.93   32.15\n",
      "  205        359R 359R N   CA   2767 2768  143.38  140.60                  -41.10\n",
      "  206  20079 360V 361A C   N    2783 2785 -158.57 -134.00   26.80    0.61  -62.50 -172.18   35.90\n",
      "  206        361A 361A N   CA   2785 2786  157.71  147.00                  -40.90\n",
      "  207  20080 361A 362S C   N    2788 2790 -164.29 -136.60   33.14    1.01  -64.10 -174.94   20.64\n",
      "  207        362S 362S N   CA   2790 2791  169.40  151.20                  -35.00\n",
      "  208  20081 362S 363G C   N    2794 2796  176.83 -167.20   30.74    1.24   82.20  168.83   12.30\n",
      "  208        363G 363G N   CA   2796 2797  148.33  174.60                    8.50\n",
      "  209  20082 363G 364L C   N    2798 2800 -145.20 -108.50   37.04    1.71  -63.50 -163.51   24.13\n",
      "  209        364L 364L N   CA   2800 2801  137.50  132.50                  -41.20\n",
      "  210  20083 364L 365L C   N    2806 2808 -129.41 -108.50   26.74    1.19  -63.50 -178.01   29.35\n",
      "  210        365L 365L N   CA   2808 2809  149.17  132.50                  -41.20\n",
      "  211  20084 365L 366S C   N    2814 2816 -157.99 -136.60   27.12    0.84  -64.10 -176.95   20.03\n",
      "  211        366S 366S N   CA   2816 2817  167.86  151.20                  -35.00\n",
      "  212  20085 366S 367G C   N    2820 2822  173.01 -167.20   27.44    1.02   82.20  172.86   12.39\n",
      "  212        367G 367G N   CA   2822 2823  155.59  174.60                    8.50\n",
      "  213  20086 367G 368E C   N    2824 2826 -154.20 -117.80   41.39    1.21  -63.60 -173.34   29.91\n",
      "  213        368E 368E N   CA   2826 2827  156.50  136.80                  -40.30\n",
      "  214  20087 368E 369F C   N    2833 2835 -149.06 -124.20   29.35    0.77  -63.20  178.77   30.28\n",
      "  214        369F 369F N   CA   2835 2836  158.90  143.30                  -44.30\n",
      "  215  20088 369F 370S C   N    2844 2846 -151.87 -136.60   17.98    0.55  -64.10 -173.73   19.80\n",
      "  215        370S 370S N   CA   2846 2847  160.70  151.20                  -35.00\n",
      "  216  20090 371V 372E C   N    2857 2859 -155.94 -117.80   44.96    1.32  -63.60 -176.05   29.62\n",
      "  216        372E 372E N   CA   2859 2860  160.60  136.80                  -40.30\n",
      "  217  20091 372E 373G C   N    2866 2868  173.86 -167.20   28.23    1.08   82.20  171.68   12.36\n",
      "  217        373G 373G N   CA   2868 2869  153.67  174.60                    8.50\n",
      "  218  20092 373G 374Q C   N    2870 2872 -157.87 -121.10   40.89    1.27  -63.80 -172.56   32.38\n",
      "  218        374Q 374Q N   CA   2872 2873  157.57  139.70                  -40.30\n",
      "  219  20093 374Q 375E C   N    2879 2881 -147.01 -117.80   31.63    0.94  -63.60 -169.96   30.00\n",
      "  219        375E 375E N   CA   2881 2882  148.94  136.80                  -40.30\n",
      "  220  20094 375E 376L C   N    2888 2890 -138.63 -108.50   33.98    1.50  -63.50 -173.59   30.54\n",
      "  220        376L 376L N   CA   2890 2891  148.20  132.50                  -41.20\n",
      "  221  20095 376L 377T C   N    2896 2898 -138.51 -124.80   17.62    0.60  -63.20  179.87   27.99\n",
      "  221        377T 377T N   CA   2898 2899  154.56  143.50                  -42.10\n",
      "  222  20096 377T 378N C   N    2903 2905 -154.83 -119.90   44.79    1.30  -63.20  179.08   27.48\n",
      "  222        378N 378N N   CA   2905 2906  165.04  137.00                  -41.10\n",
      "  223  20097 378N 379G C   N    2911 2913  177.77 -167.20   34.07    1.39   82.20  165.83   12.16\n",
      "  223        379G 379G N   CA   2913 2914  144.02  174.60                    8.50\n",
      "  224  20099 380I 381T C   N    2923 2925 -139.78 -124.80   18.99    0.66  -63.20  179.85   28.06\n",
      "  224        381T 381T N   CA   2925 2926  155.17  143.50                  -42.10\n",
      "  225  20100 381T 382A C   N    2930 2932 -158.13 -134.00   25.49    0.60  -62.50 -170.24   36.20\n",
      "  225        382A 382A N   CA   2932 2933  155.20  147.00                  -40.90\n",
      "  226  20101 382A 383R C   N    2935 2937 -152.70 -125.20   30.80    0.93  -63.00 -172.70   31.36\n",
      "  226        383R 383R N   CA   2937 2938  154.48  140.60                  -41.10\n",
      "  227  20102 383R 384E C   N    2946 2948 -150.43 -117.80   34.63    1.04  -63.60 -167.95   30.45\n",
      "  227        384E 384E N   CA   2948 2949  148.40  136.80                  -40.30\n",
      "  228  20103 384E 385F C   N    2955 2957 -150.43 -124.20   27.65    0.83  -63.20 -174.55   31.31\n",
      "  228        385F 385F N   CA   2957 2958  152.05  143.30                  -44.30\n",
      "  229  20104 385F 386K C   N    2966 2968 -157.87 -118.00   41.26    1.41  -62.90 -165.73   30.90\n",
      "  229        386K 386K N   CA   2968 2969  149.73  139.10                  -40.80\n",
      "  230  20105 386K 387R C   N    2975 2977 -154.41 -125.20   32.03    0.99  -63.00 -171.22   31.67\n",
      "  230        387R 387R N   CA   2977 2978  153.73  140.60                  -41.10\n",
      "  231  20107 388V 389F C   N    2993 2995 -152.16 -124.20   31.08    0.85  -63.20 -177.95   30.92\n",
      "  231        389F 389F N   CA   2995 2996  156.87  143.30                  -44.30\n",
      "  232  20108 389F 390A C   N    3004 3006 -154.35 -134.00   21.65    0.50  -62.50 -171.40   35.81\n",
      "  232        390A 390A N   CA   3006 3007  154.38  147.00                  -40.90\n",
      "  233  20109 390A 391Q C   N    3009 3011 -149.23 -121.10   31.87    0.98  -63.80 -174.17   31.75\n",
      "  233        391Q 391Q N   CA   3011 3012  154.67  139.70                  -40.30\n",
      "  234  20110 391Q 392K C   N    3018 3020 -142.23 -118.00   26.44    0.88  -62.90 -172.84   29.05\n",
      "  234        392K 392K N   CA   3020 3021  149.68  139.10                  -40.80\n",
      "  235  20111 392K 393H C   N    3027 3029 -142.39 -125.60   16.79    0.59  -63.20 -164.11   29.66\n",
      "  235        393H 393H N   CA   3029 3030  138.53  138.80                  -42.30\n",
      "  236  20113 394P 395D C   N    3044 3046 -136.98  -96.50   43.82    1.81  -63.30 -173.82   19.86\n",
      "  236        395D 395D N   CA   3046 3047  130.99  114.20                  -40.00\n",
      "  237  20114 395D 396F C   N    3052 3054 -132.48 -124.20    9.10    0.25  -63.20 -177.69   29.85\n",
      "  237        396F 396F N   CA   3054 3055  147.07  143.30                  -44.30\n",
      "  238  20115 396F 397S C   N    3063 3065 -154.96 -136.60   18.98    0.64  -64.10 -168.13   20.43\n",
      "  238        397S 397S N   CA   3065 3066  156.01  151.20                  -35.00\n",
      "  239  20116 397S 398T C   N    3069 3071 -142.06 -124.80   21.17    0.76  -63.20 -179.70   28.26\n",
      "  239        398T 398T N   CA   3071 3072  155.77  143.50                  -42.10\n",
      "  240  20117 398T 399A C   N    3076 3078 -155.07 -134.00   22.24    0.52  -62.50 -170.81   35.95\n",
      "  240        399A 399A N   CA   3078 3079  154.11  147.00                  -40.90\n",
      "  241  20118 399A 400E C   N    3081 3083 -150.61 -117.80   34.94    1.05  -63.60 -168.22   30.42\n",
      "  241        400E 400E N   CA   3083 3084  148.80  136.80                  -40.30\n",
      "  242  20119 400E 401Q C   N    3090 3092 -152.30 -121.10   33.10    1.06  -63.80 -169.27   32.63\n",
      "  242        401Q 401Q N   CA   3092 3093  150.74  139.70                  -40.30\n",
      "  243  20120 401Q 402Q C   N    3099 3101 -144.21 -121.10   23.43    0.81  -63.80 -166.36   32.69\n",
      "  243        402Q 402Q N   CA   3101 3102  143.54  139.70                  -40.30\n",
      "  244  20121 402Q 403D C   N    3108 3110 -142.15  -96.50   52.24    2.17  -63.30 -163.86   20.91\n",
      "  244        403D 403D N   CA   3110 3111  139.59  114.20                  -40.00\n",
      "  245  20122 403D 404A C   N    3116 3118 -148.30 -134.00   14.66    0.38  -62.50 -170.58   35.63\n",
      "  245        404A 404A N   CA   3118 3119  150.23  147.00                  -40.90\n",
      "  246  20123 404A 405Q C   N    3121 3123 -155.79 -121.10   36.44    1.17  -63.80 -167.74   33.02\n",
      "  246        405Q 405Q N   CA   3123 3124  150.87  139.70                  -40.30\n",
      "  247  20124 405Q 406E C   N    3130 3132 -148.16 -117.80   31.46    0.98  -63.60 -165.95   30.60\n",
      "  247        406E 406E N   CA   3132 3133  145.05  136.80                  -40.30\n",
      "  248  20125 406E 407Y C   N    3139 3141 -138.24 -124.30   22.00    0.91  -63.50 -179.61   32.31\n",
      "  248        407Y 407Y N   CA   3141 3142  152.42  135.40                  -43.40\n",
      "  249  20126 407Y 408F C   N    3151 3153 -137.83 -124.20   15.34    0.42  -63.20 -178.59   30.04\n",
      "  249        408F 408F N   CA   3153 3154  150.35  143.30                  -44.30\n",
      "  250  20127 408F 409L C   N    3162 3164 -129.62 -108.50   21.52    0.98  -63.50 -170.27   23.52\n",
      "  250        409L 409L N   CA   3164 3165  136.64  132.50                  -41.20\n",
      "  251  20128 409L 410Y C   N    3170 3172 -130.86 -124.30    8.28    0.68  -63.50 -173.65   26.55\n",
      "  251        410Y 410Y N   CA   3172 3173  130.35  135.40                  -43.40\n",
      "  252  20129 410Y 411L C   N    3182 3184 -146.02 -108.50   37.61    1.76  -63.50 -165.35   23.89\n",
      "  252        411L 411L N   CA   3184 3185  135.09  132.50                  -41.20\n",
      "  253  20131 412V 413E C   N    3197 3199 -154.80 -117.80   38.74    1.18  -63.60 -165.82   30.96\n",
      "  253        413E 413E N   CA   3199 3200  148.27  136.80                  -40.30\n",
      "  254  20132 413E 414Q C   N    3206 3208 -151.58 -121.10   31.90    1.03  -63.80 -168.15   32.76\n",
      "  254        414Q 414Q N   CA   3208 3209  149.11  139.70                  -40.30\n",
      "  255  20133 414Q 415M C   N    3215 3217 -151.18 -125.60   27.53    0.86  -63.40 -169.72   35.26\n",
      "  255        415M 415M N   CA   3217 3218  150.67  140.50                  -40.50\n",
      "  256  20134 415M 416R C   N    3223 3225 -150.62 -125.20   27.86    0.86  -63.00 -171.51   31.40\n",
      "  256        416R 416R N   CA   3225 3226  152.01  140.60                  -41.10\n",
      "  257  20135 416R 417R C   N    3234 3236 -143.68 -125.20   18.98    0.65  -63.00 -168.22   31.42\n",
      "  257        417R 417R N   CA   3236 3237  144.92  140.60                  -41.10\n",
      "  258  20136 417R 418Y C   N    3245 3247 -138.27 -124.30   20.67    0.89  -63.50 -177.98   32.58\n",
      "  258        418Y 418Y N   CA   3247 3248  150.64  135.40                  -43.40\n",
      "  259  20138 419V 420K C   N    3264 3266 -139.65 -118.00   21.70    0.84  -62.90 -165.68   22.40\n",
      "  259        420K 420K N   CA   3266 3267  137.72  139.10                  -40.80\n",
      "  260  20140 421P 422S C   N    3280 3282 -141.42 -136.60    5.58    0.17  -64.10 -172.34   19.11\n",
      "  260        422S 422S N   CA   3282 3283  154.00  151.20                  -35.00\n",
      "  261  20141 422S 423N C   N    3286 3288 -138.07 -119.90   20.10    0.63  -63.20 -171.21   27.84\n",
      "  261        423N 423N N   CA   3288 3289  145.59  137.00                  -41.10\n",
      "  262  20144 425V 426G C   N    3309 3311  169.03 -167.20   29.40    1.02   82.20  172.27   12.23\n",
      "  262        426G 426G N   CA   3311 3312  157.29  174.60                    8.50\n",
      "  263  20145 426G 427A C   N    3313 3315 -156.91 -134.00   24.74    0.57  -62.50 -171.83   35.87\n",
      "  263        427A 427A N   CA   3315 3316  156.34  147.00                  -40.90\n",
      "  264  20146 427A 428N C   N    3318 3320 -138.04 -119.90   20.16    0.63  -63.20 -171.41   27.82\n",
      "  264        428N 428N N   CA   3320 3321  145.80  137.00                  -41.10\n",
      "  265  20148 429I 430S C   N    3334 3336 -143.85 -136.60    7.52    0.25  -64.10 -170.60   19.42\n",
      "  265        430S 430S N   CA   3336 3337  153.20  151.20                  -35.00\n",
      "  266  20149 430S 431H C   N    3340 3342 -140.87 -125.60   15.78    0.63  -63.20 -166.60   20.71\n",
      "  266        431H 431H N   CA   3342 3343  134.82  138.80                  -42.30\n",
      "  267  20152 433V 434D C   N    3364 3366 -128.59  -96.50   34.51    1.42  -63.30  179.20   19.22\n",
      "  267        434D 434D N   CA   3366 3367  126.89  114.20                  -40.00\n",
      "  268  20154 435I 436F C   N    3380 3382 -137.03 -124.20   13.79    0.40  -63.20 -177.08   30.20\n",
      "  268        436F 436F N   CA   3382 3383  148.34  143.30                  -44.30\n",
      "  269  20155 436F 437K C   N    3391 3393 -148.50 -118.00   31.96    1.08  -62.90 -169.16   29.90\n",
      "  269        437K 437K N   CA   3393 3394  148.63  139.10                  -40.80\n",
      "  270  20156 437K 438M C   N    3400 3402 -144.83 -125.60   19.67    0.68  -63.40 -167.10   35.21\n",
      "  270        438M 438M N   CA   3402 3403  144.63  140.50                  -40.50\n",
      "  271  20157 438M 439T C   N    3408 3410 -142.44 -124.80   20.30    0.78  -63.20 -177.54   28.56\n",
      "  271        439T 439T N   CA   3410 3411  153.54  143.50                  -42.10\n",
      "  272  20159 440V 441E C   N    3422 3424 -152.47 -117.80   36.70    1.11  -63.60 -167.39   30.63\n",
      "  272        441E 441E N   CA   3424 3425  148.81  136.80                  -40.30\n",
      "  273  20160 441E 442H C   N    3431 3433 -152.62 -125.60   28.21    0.82  -63.20 -167.24   29.90\n",
      "  273        442H 442H N   CA   3433 3434  146.93  138.80                  -42.30\n",
      "  274  20161 442H 443R C   N    3441 3443 -150.37 -125.20   27.50    0.85  -63.00 -171.33   31.41\n",
      "  274        443R 443R N   CA   3443 3444  151.68  140.60                  -41.10\n",
      "  275  20163 444V 445Q C   N    3459 3461 -146.26 -121.10   26.63    0.85  -63.80 -169.91   32.25\n",
      "  275        445Q 445Q N   CA   3461 3462  148.43  139.70                  -40.30\n",
      "  276  20165 446C 447G C   N    3474 3476  163.88 -167.20   35.65    1.23   82.20  166.64   11.74\n",
      "  276        447G 447G N   CA   3476 3477  153.75  174.60                    8.50\n",
      "  277  20166 447G 448S C   N    3478 3480 -159.04 -136.60   23.90    0.77  -64.10 -169.15   20.66\n",
      "  277        448S 448S N   CA   3480 3481  159.43  151.20                  -35.00\n",
      "  278  20168 449C 450H C   N    3490 3492 -147.23 -125.60   23.49    0.64  -63.20 -170.60   29.14\n",
      "  278        450H 450H N   CA   3492 3493  147.96  138.80                  -42.30\n",
      "  279  20169 450H 451K C   N    3500 3502 -144.74 -118.00   27.89    0.95  -62.90 -169.37   29.65\n",
      "  279        451K 451K N   CA   3502 3503  147.03  139.10                  -40.80\n",
      "  280  20171 452V 453R C   N    3516 3518 -143.62 -125.20   18.78    0.66  -63.00 -167.66   31.49\n",
      "  280        453R 453R N   CA   3518 3519  144.28  140.60                  -41.10\n",
      "  281  20172 453R 454Y C   N    3527 3529 -133.02 -124.30   10.37    0.54  -63.50 -171.16   33.44\n",
      "  281        454Y 454Y N   CA   3529 3530  141.02  135.40                  -43.40\n",
      "  282  20173 454Y 455T C   N    3539 3541 -136.68 -124.80   12.52    0.55  -63.20 -174.38   28.63\n",
      "  282        455T 455T N   CA   3541 3542  147.45  143.50                  -42.10\n",
      "  283  20174 455T 456R C   N    3546 3548 -145.55 -125.20   21.22    0.70  -63.00 -168.97   31.43\n",
      "  283        456R 456R N   CA   3548 3549  146.63  140.60                  -41.10\n",
      "  284  20175 456R 457E C   N    3557 3559 -144.93 -117.80   28.43    0.87  -63.60 -167.57   30.21\n",
      "  284        457E 457E N   CA   3559 3560  145.30  136.80                  -40.30\n",
      "  285  20176 457E 458T C   N    3566 3568 -131.28 -124.80    6.55    0.37  -63.20 -171.90   28.62\n",
      "  285        458T 458T N   CA   3568 3569  142.55  143.50                  -42.10\n",
      "  286  20177 458T 459D C   N    3573 3575 -130.45  -96.50   35.00    1.44  -63.30  176.01   18.82\n",
      "  286        459D 459D N   CA   3575 3576  122.71  114.20                  -40.00\n",
      "  287  20180 461C 462L C   N    3593 3595 -131.56 -108.50   23.28    1.07  -63.50 -170.45   23.45\n",
      "  287        462L 462L N   CA   3595 3596  135.71  132.50                  -41.20\n",
      "  288  20181 462L 463S C   N    3601 3603 -125.74 -136.60   15.06    0.48  -64.10 -173.74   11.47\n",
      "  288        463S 463S N   CA   3603 3604  140.76  151.20                  -35.00\n",
      "  289  20182 463S 464L C   N    3607 3609 -131.62 -108.50   25.10    1.11  -63.50 -170.78   30.49\n",
      "  289        464L 464L N   CA   3609 3610  142.27  132.50                  -41.20\n",
      "  290  20185 466I 467P C   N    3630 3632  -72.41  -58.70   22.14    1.40  -64.50  160.51   12.40\n",
      "  290        467P 467P N   CA   3632 3633  -13.12  -30.50                  147.20\n",
      "  291  20186 467P 468L C   N    3637 3639  -64.82  -70.70    5.88    0.52  -63.50  177.54   24.73\n",
      "  291        468L 468L N   CA   3639 3640  141.27  141.60                  -41.20\n",
      "  292  20187 468L 469D C   N    3645 3647 -107.30  -96.50   22.70    0.96  -63.30  179.64   19.92\n",
      "  292        469D 469D N   CA   3647 3648  134.17  114.20                  -40.00\n",
      "  293  20189 470P 471N C   N    3660 3662 -127.09 -119.90   23.77    0.90  -63.20  171.59   25.08\n",
      "  293        471N 471N N   CA   3662 3663  159.66  137.00                  -41.10\n",
      "  294  20190 471N 472L C   N    3668 3670 -100.83 -108.50    9.32    0.51  -63.50 -177.15   23.52\n",
      "  294        472L 472L N   CA   3670 3671  137.79  132.50                  -41.20\n",
      "  295  20191 472L 473K C   N    3676 3678 -123.40 -118.00    8.60    0.32  -62.90 -176.35   27.46\n",
      "  295        473K 473K N   CA   3678 3679  145.80  139.10                  -40.80\n",
      "  296  20192 473K 474N C   N    3685 3687 -131.37 -119.90   15.48    0.45  -63.20 -175.44   26.95\n",
      "  296        474N 474N N   CA   3687 3688  147.39  137.00                  -41.10\n",
      "  297  20193 474N 475S C   N    3693 3695 -132.01 -136.60   10.71    0.43  -64.10 -170.87   11.68\n",
      "  297        475S 475S N   CA   3695 3696  141.52  151.20                  -35.00\n",
      "  298  20194 475S 476D C   N    3699 3701 -135.24  -96.50   43.46    1.80  -63.30 -171.82   20.12\n",
      "  298        476D 476D N   CA   3701 3702  133.88  114.20                  -40.00\n",
      "  299  20195 476D 477G C   N    3707 3709 -162.62 -167.20   26.59    0.98   82.20 -178.78   13.60\n",
      "  299        477G 477G N   CA   3709 3710  148.41  174.60                    8.50\n",
      "  300  20196 477G 478N C   N    3711 3713 -134.21 -119.90   15.20    0.50  -63.20 -169.49   27.85\n",
      "  300        478N 478N N   CA   3713 3714  142.12  137.00                  -41.10\n",
      "  301  20198 479P 480K C   N    3726 3728 -136.72 -118.00   21.37    0.71  -62.90 -174.85   28.46\n",
      "  301        480K 480K N   CA   3728 3729  149.40  139.10                  -40.80\n",
      "  302  20199 480K 481Q C   N    3735 3737 -130.50 -121.10    9.69    0.32  -63.80 -170.25   31.43\n",
      "  302        481Q 481Q N   CA   3737 3738  142.05  139.70                  -40.30\n",
      "  303  20200 481Q 482T C   N    3744 3746 -136.10 -124.80   13.89    0.49  -63.20 -178.40   28.07\n",
      "  303        482T 482T N   CA   3746 3747  151.58  143.50                  -42.10\n",
      "  304  20201 482T 483E C   N    3751 3753 -137.54 -117.80   20.21    0.64  -63.60 -166.73   29.93\n",
      "  304        483E 483E N   CA   3753 3754  141.14  136.80                  -40.30\n",
      "  305  20202 483E 484E C   N    3760 3762 -145.02 -117.80   28.80    0.87  -63.60 -168.36   30.11\n",
      "  305        484E 484E N   CA   3762 3763  146.21  136.80                  -40.30\n",
      "  306  20203 484E 485E C   N    3769 3771 -135.61 -117.80   17.82    0.64  -63.60 -168.49   22.48\n",
      "  306        485E 485E N   CA   3771 3772  137.16  136.80                  -40.30\n",
      "  307  20205 486I 487F C   N    3786 3788 -138.29 -124.20   14.25    0.49  -63.20 -173.92   30.72\n",
      "  307        487F 487F N   CA   3788 3789  145.44  143.30                  -44.30\n",
      "  308  20206 487F 488A C   N    3797 3799 -161.77 -134.00   28.61    0.72  -62.50 -167.25   36.88\n",
      "  308        488A 488A N   CA   3799 3800  153.89  147.00                  -40.90\n",
      "  309  20207 488A 489S C   N    3802 3804 -157.79 -136.60   21.33    0.78  -64.10 -164.74   20.88\n",
      "  309        489S 489S N   CA   3804 3805  153.68  151.20                  -35.00\n",
      "  310  20208 489S 490R C   N    3808 3810 -141.91 -125.20   16.86    0.73  -63.00 -163.95   22.47\n",
      "  310        490R 490R N   CA   3810 3811  138.37  140.60                  -41.10\n",
      "  311  20210 491P 492H C   N    3826 3828 -139.11 -125.60   14.63    0.40  -63.20 -170.82   28.61\n",
      "  311        492H 492H N   CA   3828 3829  144.41  138.80                  -42.30\n",
      "  312  20212 493I 494S C   N    3844 3846 -142.10 -136.60    5.56    0.24  -64.10 -168.79   19.41\n",
      "  312        494S 494S N   CA   3846 3847  150.42  151.20                  -35.00\n",
      "  313  20213 494S 495L C   N    3850 3852 -131.01 -108.50   23.18    1.05  -63.50 -168.44   23.73\n",
      "  313        495L 495L N   CA   3852 3853  138.07  132.50                  -41.20\n",
      "  314  20214 495L 496D C   N    3858 3860 -132.04  -96.50   41.26    1.71  -63.30 -171.85   20.18\n",
      "  314        496D 496D N   CA   3860 3861  135.15  114.20                  -40.00\n",
      "  315  20215 496D 497A C   N    3866 3868 -144.28 -134.00   10.76    0.26  -62.50 -172.31   35.14\n",
      "  315        497A 497A N   CA   3868 3869  150.17  147.00                  -40.90\n",
      "  316  20217 498C 499L C   N    3877 3879 -140.58 -108.50   38.92    1.72  -63.50 -178.55   29.96\n",
      "  316        499L 499L N   CA   3879 3880  154.54  132.50                  -41.20\n",
      "  317  20218 499L 500G C   N    3885 3887  174.67 -167.20   30.29    1.19   82.20  169.31   12.26\n",
      "  317        500G 500G N   CA   3887 3888  150.33  174.60                    8.50\n",
      "  318  20219 500G 501S C   N    3889 3891 -155.48 -136.60   19.15    0.68  -64.10 -166.44   20.59\n",
      "  318        501S 501S N   CA   3891 3892  154.37  151.20                  -35.00\n",
      "  319  20220 501S 502L C   N    3895 3897 -136.35 -108.50   28.02    1.30  -63.50 -168.81   23.58\n",
      "  319        502L 502L N   CA   3897 3898  135.56  132.50                  -41.20\n",
      "  320  20221 502L 503M C   N    3903 3905 -138.50 -125.60   12.90    0.50  -63.40 -166.10   34.89\n",
      "  320        503M 503M N   CA   3905 3906  140.73  140.50                  -40.50\n",
      "  321  20222 503M 504Q C   N    3911 3913 -147.00 -121.10   28.75    0.89  -63.80 -172.96   31.83\n",
      "  321        504Q 504Q N   CA   3913 3914  152.19  139.70                  -40.30\n",
      "  322  20223 504Q 505A C   N    3920 3922 -152.90 -134.00   20.04    0.47  -62.50 -171.48   35.73\n",
      "  322        505A 505A N   CA   3922 3923  153.67  147.00                  -40.90\n",
      "  323  20224 505A 506T C   N    3925 3927 -137.01 -124.80   14.00    0.54  -63.20 -176.91   28.32\n",
      "  323        506T 506T N   CA   3927 3928  150.34  143.50                  -42.10\n",
      "  324  20225 506T 507D C   N    3932 3934 -135.27  -96.50   41.91    1.73  -63.30 -175.30   19.72\n",
      "  324        507D 507D N   CA   3934 3935  130.10  114.20                  -40.00\n",
      "  325  20227 508I 509D C   N    3948 3950 -127.83  -96.50   38.44    1.60  -63.30 -172.10   20.24\n",
      "  325        509D 509D N   CA   3950 3951  136.47  114.20                  -40.00\n",
      "  326  20229 510C 511R C   N    3962 3964 -143.64 -125.20   32.74    1.10  -63.00  171.40   28.61\n",
      "  326        511R 511R N   CA   3964 3965  167.65  140.60                  -41.10\n",
      "  327  20231 512C 513S C   N    3979 3981 -128.77 -136.60    8.97    0.47  -64.10 -178.66   17.71\n",
      "  327        513S 513S N   CA   3981 3982  155.58  151.20                  -35.00\n",
      "  328  20232 513S 514A C   N    3985 3987  -84.29  -68.20   25.25    1.59  -62.50  155.88   26.67\n",
      "  328        514A 514A N   CA   3987 3988  164.76  145.30                  -40.90\n",
      "  329  20234 515C 516G C   N    3996 3998 -153.71 -167.20   13.53    0.28   82.20 -151.84   15.39\n",
      "  329        516G 516G N   CA   3998 3999  175.63  174.60                    8.50\n",
      "  330  20236 517I 518P C   N    4008 4010  -67.86  -58.70    9.47    1.06  -64.50  175.33   13.26\n",
      "  330        518P 518P N   CA   4010 4011  -28.10  -30.50                  147.20\n",
      "  331  20238 519V 520T C   N    4022 4024 -133.15 -124.80   15.09    0.47  -63.20  176.29   27.20\n",
      "  331        520T 520T N   CA   4024 4025  156.08  143.50                  -42.10\n",
      "  332  20239 520T 521Y C   N    4029 4031 -137.96 -124.30   23.14    0.94  -63.50  178.76   32.04\n",
      "  332        521Y 521Y N   CA   4031 4032  154.08  135.40                  -43.40\n",
      "  333  20241 522C 523K C   N    4047 4049 -143.30 -118.00   31.61    1.07  -62.90 -179.90   28.20\n",
      "  333        523K 523K N   CA   4049 4050  158.04  139.10                  -40.80\n",
      "  334  20242 523K 524T C   N    4056 4058 -139.28 -124.80   18.31    0.64  -63.20 -179.95   28.06\n",
      "  334        524T 524T N   CA   4058 4059  154.71  143.50                  -42.10\n",
      "  335  20243 524T 525T C   N    4063 4065 -147.64 -124.80   27.46    1.00  -63.20 -179.82   28.56\n",
      "  335        525T 525T N   CA   4065 4066  158.74  143.50                  -42.10\n",
      "  336  20244 525T 526R C   N    4070 4072 -147.38 -125.20   24.20    0.75  -63.00 -171.46   31.20\n",
      "  336        526R 526R N   CA   4072 4073  150.30  140.60                  -41.10\n",
      "  337  20245 526R 527L C   N    4081 4083 -148.36 -108.50   42.04    1.88  -63.50 -167.38   31.99\n",
      "  337        527L 527L N   CA   4083 4084  145.87  132.50                  -41.20\n",
      "  338  20246 527L 528R C   N    4089 4091 -149.08 -125.20   26.02    0.81  -63.00 -171.25   31.34\n",
      "  338        528R 528R N   CA   4091 4092  150.92  140.60                  -41.10\n",
      "  339  20247 528R 529T C   N    4100 4102 -149.97 -124.80   36.34    1.16  -63.20  171.72   27.58\n",
      "  339        529T 529T N   CA   4102 4103  169.72  143.50                  -42.10\n",
      "  340  20248 529T 530F C   N    4107 4109 -147.41 -124.20   23.95    0.76  -63.20 -173.42   31.31\n",
      "  340        530F 530F N   CA   4109 4110  149.21  143.30                  -44.30\n",
      "  341  20250 531P 532D C   N    4125 4127 -147.96  -96.50   71.58    2.99  -63.30  177.54   27.35\n",
      "  341        532D 532D N   CA   4127 4128  163.95  114.20                  -40.00\n",
      "  342  20252 533V 534L C   N    4140 4142 -151.36 -108.50   45.60    2.03  -63.50 -167.99   32.09\n",
      "  342        534L 534L N   CA   4142 4143  148.07  132.50                  -41.20\n",
      "  343  20255 536I 537F C   N    4163 4165 -153.49 -124.20   35.78    0.93  -63.20  176.67   30.21\n",
      "  343        537F 537F N   CA   4165 4166  163.85  143.30                  -44.30\n",
      "  344  20256 537F 538L C   N    4174 4176 -154.12 -108.50   56.52    2.50  -63.50  177.76   30.25\n",
      "  344        538L 538L N   CA   4176 4177  165.87  132.50                  -41.20\n",
      "  345  20257 538L 539R C   N    4182 4184 -159.68 -125.20   43.68    1.29  -63.00  179.71   30.71\n",
      "  345        539R 539R N   CA   4184 4185  167.42  140.60                  -41.10\n",
      "  346  20258 539R 540R C   N    4193 4195 -167.93 -125.20   50.88    1.51  -63.00 -176.39   31.72\n",
      "  346        540R 540R N   CA   4195 4196  168.23  140.60                  -41.10\n",
      "  347  20259 540R 541A C   N    4204 4206 -170.61 -134.00   41.25    0.94  -62.50 -172.56   36.42\n",
      "  347        541A 541A N   CA   4206 4207  165.99  147.00                  -40.90\n",
      "  348  20260 541A 542Q C   N    4209 4211 -163.75 -121.10   52.35    1.62  -63.80  179.94   31.49\n",
      "  348        542Q 542Q N   CA   4211 4212  170.07  139.70                  -40.30\n",
      "  349  20261 542Q 543F C   N    4218 4220 -170.88 -124.20   51.73    1.43  -63.20 -175.27   32.21\n",
      "  349        543F 543F N   CA   4220 4221  165.59  143.30                  -44.30\n",
      "  350  20262 543F 544D C   N    4229 4231 -170.90  -96.50   92.48    3.85  -63.30 -174.69   29.67\n",
      "  350        544D 544D N   CA   4231 4232  169.13  114.20                  -40.00\n",
      "  351  20263 544D 545M C   N    4237 4239 -166.34 -125.60   50.23    1.48  -63.40 -178.39   34.93\n",
      "  351        545M 545M N   CA   4239 4240  169.88  140.50                  -40.50\n",
      "  352  20264 545M 546G C   N    4245 4247  169.13 -167.20   25.12    0.70   82.20 -179.95   12.64\n",
      "  352        546G 546G N   CA   4247 4248  166.18  174.60                    8.50\n",
      "  353  20265 546G 547T C   N    4249 4251 -165.17 -124.80   41.52    1.93  -63.20 -166.30   31.32\n",
      "  353        547T 547T N   CA   4251 4252  153.21  143.50                  -42.10\n",
      "  354  20266 547T 548M C   N    4256 4258 -169.49 -125.60   61.81    1.89  -63.40  172.07   33.60\n",
      "  354        548M 548M N   CA   4258 4259 -175.98  140.50                  -40.50\n",
      "  355  20267 548M 549S C   N    4264 4266 -158.14 -136.60   23.38    0.73  -64.10 -170.34   20.51\n",
      "  355        549S 549S N   CA   4266 4267  160.29  151.20                  -35.00\n",
      "  356  20269 550V 551K C   N    4277 4279 -157.75 -118.00   40.50    1.41  -62.90 -163.28   31.21\n",
      "  356        551K 551K N   CA   4279 4280  146.86  139.10                  -40.80\n",
      "  357  20270 551K 552K C   N    4286 4288 -175.03 -118.00   72.32    2.45  -62.90  175.99   29.38\n",
      "  357        552K 552K N   CA   4288 4289 -176.44  139.10                  -40.80\n",
      "  358  20271 552K 553L C   N    4295 4297 -133.28 -108.50   57.30    2.78  -63.50  151.64   25.36\n",
      "  358        553L 553L N   CA   4297 4298 -175.83  132.50                  -41.20\n",
      "  359  20272 553L 554D C   N    4303 4305 -149.72  -96.50   80.23    3.36  -63.30  169.45   26.44\n",
      "  359        554D 554D N   CA   4305 4306  174.24  114.20                  -40.00\n",
      "  360  20274 555V 556F C   N    4318 4320  179.41 -124.20   62.28    1.72  -63.20 -172.67   33.00\n",
      "  360        556F 556F N   CA   4320 4321  169.72  143.30                  -44.30\n",
      "  361  20276 557V 558D C   N    4336 4338 -177.49  -96.50  101.42    4.22  -63.30 -175.63   29.89\n",
      "  361        558D 558D N   CA   4338 4339  175.25  114.20                  -40.00\n",
      "  362  20279 560P 561L C   N    4358 4360 -177.17 -108.50   83.16    3.67  -63.50  179.86   31.79\n",
      "  362        561L 561L N   CA   4360 4361  179.41  132.50                  -41.20\n",
      "  363  20280 561L 562E C   N    4366 4368 -164.89 -117.80   61.57    1.89  -63.60  175.43   28.87\n",
      "  363        562E 562E N   CA   4368 4369  176.47  136.80                  -40.30\n",
      "  364  20281 562E 563L C   N    4375 4377  165.91 -108.50   95.25    4.21  -63.50 -165.25   34.77\n",
      "  364        563L 563L N   CA   4377 4378  174.31  132.50                  -41.20\n",
      "  365  20282 563L 564D C   N    4383 4385 -156.54  -96.50   86.31    3.61  -63.30  171.38   27.08\n",
      "  365        564D 564D N   CA   4385 4386  176.20  114.20                  -40.00\n",
      "  366  20283 564D 565L C   N    4391 4393 -170.86 -108.50   82.25    3.67  -63.50  170.66   30.14\n",
      "  366        565L 565L N   CA   4393 4394 -173.86  132.50                  -41.20\n",
      "  367  20284 565L 566E C   N    4399 4401 -164.83 -117.80   64.27    2.02  -63.60  172.04   28.39\n",
      "  367        566E 566E N   CA   4401 4402 -179.40  136.80                  -40.30\n",
      "  368  20285 566E 567F C   N    4408 4410 -150.12 -124.20   51.27    1.65  -63.20  154.86   26.85\n",
      "  368        567F 567F N   CA   4410 4411 -172.47  143.30                  -44.30\n",
      "  369  20286 567F 568L C   N    4419 4421 -141.03 -108.50   65.12    3.10  -63.50  151.26   25.76\n",
      "  369        568L 568L N   CA   4421 4422 -171.08  132.50                  -41.20\n",
      "  370  20287 568L 569R C   N    4427 4429 -146.04 -125.20   56.78    2.22  -63.00  150.47   25.83\n",
      "  370        569R 569R N   CA   4429 4430 -166.58  140.60                  -41.10\n",
      "  371  20289 570G 571K C   N    4442 4444 -172.77 -118.00   64.54    2.15  -62.90 -177.32   30.18\n",
      "  371        571K 571K N   CA   4444 4445  173.25  139.10                  -40.80\n",
      "  372  20290 571K 572G C   N    4451 4453 -153.76 -167.20   17.65    0.64   82.20 -143.43   15.86\n",
      "  372        572G 572G N   CA   4453 4454 -173.96  174.60                    8.50\n",
      "  373  20291 572G 573L C   N    4455 4457 -139.40 -108.50   82.51    4.08  -63.50  133.47   23.15\n",
      "  373        573L 573L N   CA   4457 4458 -150.99  132.50                  -41.20\n",
      "  374  20292 573L 574Q C   N    4463 4465 -144.76 -121.10   48.02    1.79  -63.80  160.19   27.70\n",
      "  374        574Q 574Q N   CA   4465 4466 -178.52  139.70                  -40.30\n",
      "  375  20293 574Q 575S C   N    4472 4474 -138.10 -136.60   68.68    3.46  -64.10  128.56   14.66\n",
      "  375        575S 575S N   CA   4474 4475 -140.14  151.20                  -35.00\n",
      "  376  20294 575S 576D C   N    4478 4480  122.66   54.50  135.52   15.25  -63.30 -122.25   39.78\n",
      "  376        576D 576D N   CA   4480 4481  158.03   40.90                  -40.00\n",
      "  377  20295 576D 577E C   N    4486 4488 -166.30 -117.80   59.50    1.77  -63.60 -179.50   29.64\n",
      "  377        577E 577E N   CA   4488 4489  171.26  136.80                  -40.30\n",
      "  378  20297 578V 579S C   N    4502 4504  109.34 -136.60  116.63    5.27  -64.10 -122.75   29.55\n",
      "  378        579S 579S N   CA   4504 4505  126.89  151.20                  -35.00\n",
      "  379  20298 579S 580M C   N    4508 4510 -145.92 -125.60   22.27    0.69  -63.40 -171.12   34.68\n",
      "  379        580M 580M N   CA   4510 4511  149.60  140.50                  -40.50\n",
      "  380  20299 580M 581P C   N    4516 4518 -128.25  -58.70   85.53   11.95  -64.50  147.05    8.58\n",
      "  380        581P 581P N   CA   4518 4519  -80.28  -30.50                  147.20\n",
      "  381  20300 581P 582E C   N    4523 4525  -44.17  -69.30   25.90    2.17  -63.60  172.02   21.93\n",
      "  381        582E 582E N   CA   4525 4526  148.78  142.50                  -40.30\n",
      "  382  20301 582E 583A C   N    4532 4534 -167.98 -134.00   37.26    0.85  -62.50 -171.01   36.56\n",
      "  382        583A 583A N   CA   4534 4535  162.29  147.00                  -40.90\n",
      "  383  20302 583A 584Q C   N    4537 4539  154.40 -121.10   85.00    3.39  -63.80 -137.99   28.59\n",
      "  383        584Q 584Q N   CA   4539 4540  130.52  139.70                  -40.30\n",
      "  384  20303 584Q 585E C   N    4546 4548   91.19 -117.80  169.61    4.96  -63.60 -172.58   23.76\n",
      "  384        585E 585E N   CA   4548 4549 -145.97  136.80                  -40.30\n",
      "  385  20304 585E 586D C   N    4555 4557  179.03  -96.50  115.07    4.81  -63.30  173.61   28.64\n",
      "  385        586D 586D N   CA   4557 4558 -167.66  114.20                  -40.00\n",
      "  386  20305 586D 587L C   N    4563 4565 -136.79 -108.50   49.50    2.31  -63.50  163.08   27.17\n",
      "  386        587L 587L N   CA   4565 4566  173.12  132.50                  -41.20\n",
      "  387  20306 587L 588P C   N    4571 4573 -176.33  -64.50  113.06   12.10  -64.50  113.06   12.10\n",
      "  387        588P 588P N   CA   4573 4574  130.60  147.20                  147.20\n",
      "  388  20307 588P 589H C   N    4578 4580  -88.60  -67.60   41.59    2.59  -63.20  144.06   19.66\n",
      "  388        589H 589H N   CA   4580 4581  175.90  140.00                  -42.30\n",
      "  389  20308 589H 590K C   N    4588 4590 -103.84 -118.00   21.44    1.09  -62.90  169.03   24.37\n",
      "  389        590K 590K N   CA   4590 4591  155.20  139.10                  -40.80\n",
      "  390  20310 591P 592T C   N    4604 4606 -128.51 -124.80   61.82    2.54  -63.20  130.26   20.89\n",
      "  390        592T 592T N   CA   4606 4607 -154.80  143.50                  -42.10\n",
      "  391  20311 592T 593S C   N    4611 4613 -167.25 -136.60   63.66    2.44  -64.10  156.74   18.78\n",
      "  391        593S 593S N   CA   4613 4614 -153.01  151.20                  -35.00\n",
      "  392  20312 593S 594S C   N    4617 4619 -177.51 -136.60   99.14    4.00  -64.10  140.83   18.15\n",
      "  392        594S 594S N   CA   4619 4620 -118.50  151.20                  -35.00\n",
      "  393  20313 594S 595S C   N    4623 4625  -33.99  -72.40   51.13    4.09  -64.10  142.08    8.96\n",
      "  393        595S 595S N   CA   4625 4626 -173.85  152.40                  -35.00\n",
      "  394  20314 595S 596M C   N    4629 4631  -95.22  -73.00   37.78    2.36  -63.40  149.38   24.85\n",
      "  394        596M 596M N   CA   4631 4632  173.55  143.00                  -40.50\n",
      "  395  20315 596M 597K C   N    4637 4639  -55.40  -70.20   49.45    3.23  -62.90  134.22   17.85\n",
      "  395        597K 597K N   CA   4639 4640   93.21  140.40                  -40.80\n",
      "  396  20316 597K 598T C   N    4646 4648 -166.33 -124.80   63.82    2.01  -63.20  162.78   27.17\n",
      "  396        598T 598T N   CA   4648 4649 -168.04  143.50                  -42.10\n",
      "  397  20318 599V 600D C   N    4660 4662   55.83  -70.90  130.50   10.67  -63.30 -177.27   20.10\n",
      "  397        600D 600D N   CA   4662 4663 -178.56  150.30                  -40.00\n",
      "  398  20319 600D 601E C   N    4668 4670  166.83 -117.80   90.91    2.69  -63.60 -174.98   31.32\n",
      "  398        601E 601E N   CA   4670 4671 -172.37  136.80                  -40.30\n",
      "  399  20320 601E 602E C   N    4677 4679  -92.98  -69.30   25.77    1.59  -63.60  169.58   24.26\n",
      "  399        602E 602E N   CA   4679 4680  152.68  142.50                  -40.30\n",
      "  400  20321 602E 603A C   N    4686 4688   70.52  -68.20  139.42   11.33  -62.50 -142.39   42.63\n",
      "  400        603A 603A N   CA   4688 4689  131.32  145.30                  -40.90\n",
      "  401  20322 603A 604L C   N    4691 4693  -49.18  -70.70   76.00    6.59  -63.50  105.29   13.86\n",
      "  401        604L 604L N   CA   4693 4694 -145.51  141.60                  -41.20\n",
      "  402  20323 604L 605A C   N    4699 4701  168.70 -134.00   58.18    2.32  -62.50 -140.40   33.10\n",
      "  402        605A 605A N   CA   4701 4702  136.96  147.00                  -40.90\n",
      "  403  20324 605A 606M C   N    4704 4706  -92.49  -63.40   32.43    6.58  -63.40   32.43    6.58\n",
      "  403        606M 606M N   CA   4706 4707  -54.83  -40.50                  -40.50\n",
      "  404  20325 606M 607L C   N    4712 4714  -82.07  -63.50  114.08   16.96  -63.50  114.08   16.96\n",
      "  404        607L 607L N   CA   4714 4715 -153.76  -41.20                  -41.20\n",
      "  405  20326 607L 608L C   N    4720 4722  178.49 -108.50   74.48    3.73  -63.50 -162.01   24.57\n",
      "  405        608L 608L N   CA   4722 4723  117.78  132.50                  -41.20\n",
      "  406  20327 608L 609S C   N    4728 4730  -93.27  -72.40   45.81    2.12  -64.10  135.01   11.76\n",
      "  406        609S 609S N   CA   4730 4731 -166.83  152.40                  -35.00\n",
      "  407  20328 609S 610M C   N    4734 4736 -167.77 -125.60   54.93    1.64  -63.40  177.69   34.39\n",
      "  407        610M 610M N   CA   4736 4737  175.70  140.50                  -40.50\n",
      "  408  20331 612I 613E C   N    4754 4756 -147.55 -117.80   62.94    2.45  -63.60  152.60   24.92\n",
      "  408        613E 613E N   CA   4756 4757 -167.73  136.80                  -40.30\n",
      "  409  20332 613E 614E C   N    4763 4765  176.57 -117.80   77.59    2.28  -63.60 -174.58   31.04\n",
      "  409        614E 614E N   CA   4765 4766  178.20  136.80                  -40.30\n",
      "  410  20333 614E 615T C   N    4772 4774 -173.58 -124.80   52.58    2.22  -63.20 -169.89   31.27\n",
      "  410        615T 615T N   CA   4774 4775  163.12  143.50                  -42.10\n",
      "  411  20335 616V 617A C   N    4786 4788  172.32 -134.00   66.61    1.69  -62.50 -177.59   36.30\n",
      "  411        617A 617A N   CA   4788 4789 -173.58  147.00                  -40.90\n",
      "  412  20336 617A 618R C   N    4791 4793 -145.49 -125.20   45.54    1.68  -63.00  160.37   27.19\n",
      "  412        618R 618R N   CA   4793 4794 -178.63  140.60                  -41.10\n",
      "  413  20337 618R 619Y C   N    4802 4804 -136.45 -124.30   48.31    2.09  -63.50  152.96   27.80\n",
      "  413        619Y 619Y N   CA   4804 4805 -177.84  135.40                  -43.40\n",
      "  414  20338 619Y 620A C   N    4814 4816 -156.13 -134.00   33.01    1.06  -62.50  174.80   33.62\n",
      "  414        620A 620A N   CA   4816 4817  171.49  147.00                  -40.90\n",
      "  415  20339 620A 621L C   N    4819 4821 -149.85 -108.50   60.43    2.74  -63.50  166.39   28.40\n",
      "  415        621L 621L N   CA   4821 4822  176.57  132.50                  -41.20\n",
      "  416  20340 621L 622L C   N    4827 4829 -165.96 -108.50   69.18    3.06  -63.50  179.82   31.20\n",
      "  416        622L 622L N   CA   4829 4830  171.03  132.50                  -41.20\n",
      "  417  20341 622L 623Q C   N    4835 4837 -153.64 -121.10   51.12    1.74  -63.80  166.83   29.08\n",
      "  417        623Q 623Q N   CA   4837 4838  179.13  139.70                  -40.30\n",
      "  418  20342 623Q 624T C   N    4844 4846 -152.18 -124.80   50.74    1.57  -63.20  158.92   25.98\n",
      "  418        624T 624T N   CA   4846 4847 -173.78  143.50                  -42.10\n",
      "  419  20343 624T 625G C   N    4851 4853 -176.19 -167.20    9.26    0.23   82.20 -167.19   13.83\n",
      "  419        625G 625G N   CA   4853 4854  172.36  174.60                    8.50\n",
      "  420  20344 625G 626M C   N    4855 4857 -173.95 -125.60   54.01    1.64  -63.40 -169.67   36.77\n",
      "  420        626M 626M N   CA   4857 4858  164.56  140.50                  -40.50\n",
      "  421  20345 626M 627N C   N    4863 4865 -168.32 -119.90   62.03    1.80  -63.20  177.59   27.91\n",
      "  421        627N 627N N   CA   4865 4866  175.77  137.00                  -41.10\n",
      "  422  20346 627N 628A C   N    4871 4873 -164.98 -134.00   39.77    1.06  -62.50  179.34   34.80\n",
      "  422        628A 628A N   CA   4873 4874  171.92  147.00                  -40.90\n",
      "  423  20347 628A 629E C   N    4876 4878 -171.14 -117.80   64.70    1.92  -63.60 -178.44   29.99\n",
      "  423        629E 629E N   CA   4878 4879  173.42  136.80                  -40.30\n",
      "  424  20348 629E 630R C   N    4885 4887 -146.36 -125.20   34.51    1.12  -63.00  172.51   28.93\n",
      "  424        630R 630R N   CA   4887 4888  167.86  140.60                  -41.10\n",
      "  425  20349 630R 631A C   N    4896 4898 -166.82 -134.00   38.91    0.93  -62.50 -176.30   35.62\n",
      "  425        631A 631A N   CA   4898 4899  167.90  147.00                  -40.90\n",
      "  426  20351 632V 633D C   N    4908 4910 -162.30  -96.50   83.07    3.46  -63.30 -175.99   29.02\n",
      "  426        633D 633D N   CA   4910 4911  164.90  114.20                  -40.00\n",
      "  427  20352 633D 634Y C   N    4916 4918 -133.25 -124.30   17.01    0.67  -63.50 -179.27   32.15\n",
      "  427        634Y 634Y N   CA   4918 4919  149.86  135.40                  -43.40\n",
      "  428  20354 635V 636F C   N    4935 4937 -140.26 -124.20   16.95    0.51  -63.20 -176.11   30.53\n",
      "  428        636F 636F N   CA   4937 4938  148.73  143.30                  -44.30\n",
      "  429  20355 636F 637S C   N    4946 4948 -167.63 -136.60   36.77    1.12  -64.10 -174.37   20.92\n",
      "  429        637S 637S N   CA   4948 4949  170.92  151.20                  -35.00\n",
      "  430  20356 637S 638H C   N    4952 4954 -143.95 -125.60   25.54    0.63  -63.20 -179.76   27.78\n",
      "  430        638H 638H N   CA   4954 4955  156.56  138.80                  -42.30\n",
      "  431  20357 638H 639E C   N    4962 4964 -165.85 -117.80   55.49    1.62  -63.60 -174.19   30.35\n",
      "  431        639E 639E N   CA   4964 4965  164.56  136.80                  -40.30\n",
      "  432  20358 639E 640N C   N    4971 4973 -138.01 -119.90   22.85    0.66  -63.20 -176.11   27.22\n",
      "  432        640N 640N N   CA   4973 4974  150.92  137.00                  -41.10\n",
      "  433  20360 641I 642A C   N    4987 4989 -147.47 -134.00   15.26    0.35  -62.50 -174.46   34.95\n",
      "  433        642A 642A N   CA   4989 4990  154.17  147.00                  -40.90\n",
      "  434  20361 642A 643E C   N    4992 4994 -161.84 -117.80   49.34    1.44  -63.60 -171.69   30.51\n",
      "  434        643E 643E N   CA   4994 4995  159.04  136.80                  -40.30\n",
      "  435  20362 643E 644E C   N    5001 5003 -153.97 -117.80   41.33    1.21  -63.60 -173.71   29.85\n",
      "  435        644E 644E N   CA   5003 5004  156.80  136.80                  -40.30\n",
      "  436  20363 644E 645A C   N    5010 5012 -167.86 -134.00   38.72    0.89  -62.50 -173.96   36.06\n",
      "  436        645A 645A N   CA   5012 5013  165.78  147.00                  -40.90\n",
      "  437  20364 645A 646G C   N    5015 5017  175.63 -167.20   32.02    1.28   82.20  167.54   12.19\n",
      "  437        646G 646G N   CA   5017 5018  147.57  174.60                    8.50\n",
      "  438  20365 646G 647L C   N    5019 5021 -152.35 -108.50   45.30    2.04  -63.50 -163.81   32.73\n",
      "  438        647L 647L N   CA   5021 5022  143.88  132.50                  -41.20\n",
      "  439  20366 647L 648A C   N    5027 5029 -149.31 -134.00   17.35    0.40  -62.50 -174.49   35.04\n",
      "  439        648A 648A N   CA   5029 5030  155.16  147.00                  -40.90\n",
      "  440  20367 648A 649E C   N    5032 5034 -150.52 -117.80   35.67    1.05  -63.60 -170.22   30.15\n",
      "  440        649E 649E N   CA   5034 5035  150.99  136.80                  -40.30\n",
      "  441  20369 650I 651S C   N    5049 5051 -153.16 -136.60   17.80    0.56  -64.10 -170.51   20.13\n",
      "  441        651S 651S N   CA   5051 5052  157.74  151.20                  -35.00\n",
      "  442  20370 651S 652A C   N    5055 5057 -162.81 -134.00   30.66    0.71  -62.50 -169.77   36.51\n",
      "  442        652A 652A N   CA   5057 5058  157.47  147.00                  -40.90\n",
      "  443  20371 652A 653T C   N    5060 5062 -147.40 -124.80   29.51    1.00  -63.20  176.77   28.10\n",
      "  443        653T 653T N   CA   5062 5063  162.47  143.50                  -42.10\n",
      "  444  20372 653T 654A C   N    5067 5069 -163.31 -134.00   31.28    0.72  -62.50 -169.90   36.51\n",
      "  444        654A 654A N   CA   5069 5070  157.93  147.00                  -40.90\n",
      "  445  20373 654A 655S C   N    5072 5074 -156.74 -136.60   20.67    0.71  -64.10 -167.17   20.63\n",
      "  445        655S 655S N   CA   5074 5075  155.87  151.20                  -35.00\n",
      "  446  20374 655S 656E C   N    5078 5080 -158.61 -117.80   44.19    1.31  -63.60 -168.78   30.75\n",
      "  446        656E 656E N   CA   5080 5081  153.75  136.80                  -40.30\n",
      "  447  20375 656E 657S C   N    5087 5089 -145.70 -136.60    9.37    0.32  -64.10 -170.03   19.60\n",
      "  447        657S 657S N   CA   5089 5090  153.44  151.20                  -35.00\n",
      "  448  20376 657S 658Q C   N    5093 5095 -149.66 -121.10   30.04    1.33  -63.80 -168.92   24.69\n",
      "  448        658Q 658Q N   CA   5095 5096  130.40  139.70                  -40.30\n",
      "  449  20378 659P 660A C   N    5109 5111 -146.10 -134.00   14.75    0.37  -62.50 -176.21   34.59\n",
      "  449        660A 660A N   CA   5111 5112  155.43  147.00                  -40.90\n",
      "  450  20379 660A 661H C   N    5114 5116 -142.70 -125.60   23.51    0.58  -63.20 -178.86   27.82\n",
      "  450        661H 661H N   CA   5116 5117  154.93  138.80                  -42.30\n",
      "  451  20381 662V 663L C   N    5131 5133 -126.76 -108.50   19.41    0.86  -63.50 -169.48   30.37\n",
      "  451        663L 663L N   CA   5133 5134  139.09  132.50                  -41.20\n",
      "  452  20382 663L 664D C   N    5139 5141  -91.45  -70.90   33.17    1.20  -63.30  146.39   19.86\n",
      "  452        664D 664D N   CA   5141 5142  176.34  150.30                  -40.00\n",
      "  453  20383 664D 665G C   N    5147 5149 -167.22 -167.20   41.06    1.61   82.20  166.92   12.64\n",
      "  453        665G 665G N   CA   5149 5150  133.54  174.60                    8.50\n",
      "  454  20385 666P 667A C   N    5158 5160 -151.72 -134.00   18.17    0.47  -62.50 -169.71   35.96\n",
      "  454        667A 667A N   CA   5160 5161  151.02  147.00                  -40.90\n",
      "  455  20386 667A 668K C   N    5163 5165 -140.25 -118.00   22.61    0.93  -62.90 -167.87   22.13\n",
      "  455        668K 668K N   CA   5165 5166  135.08  139.10                  -40.80\n",
      "  456  20387 668K 669Y C   N    5172 5174 -134.94 -124.30   11.44    0.94  -63.50 -171.35   26.77\n",
      "  456        669Y 669Y N   CA   5174 5175  131.20  135.40                  -43.40\n",
      "  457  20388 669Y 670R C   N    5184 5186 -156.89 -125.20   36.89    1.92  -63.00 -172.07   21.27\n",
      "  457        670R 670R N   CA   5186 5187  121.70  140.60                  -41.10\n",
      "  458  20389 670R 671L C   N    5195 5197 -150.04 -108.50   44.82    1.99  -63.50 -169.71   31.77\n",
      "  458        671L 671L N   CA   5197 5198  149.33  132.50                  -41.20\n",
      "  459  20390 671L 672H C   N    5203 5205 -147.10 -125.60   29.47    0.73  -63.20  179.56   27.89\n",
      "  459        672H 672H N   CA   5205 5206  158.95  138.80                  -42.30\n",
      "  460  20391 672H 673A C   N    5213 5215 -152.99 -134.00   22.97    0.56  -62.50 -176.88   34.84\n",
      "  460        673A 673A N   CA   5215 5216  159.91  147.00                  -40.90\n",
      "  461  20392 673A 674M C   N    5218 5220 -137.64 -125.60   14.94    0.44  -63.40 -174.35   33.58\n",
      "  461        674M 674M N   CA   5220 5221  149.34  140.50                  -40.50\n",
      "  462  20394 675I 676S C   N    5234 5236 -151.94 -136.60   16.60    0.52  -64.10 -170.91   20.01\n",
      "  462        676S 676S N   CA   5236 5237  157.54  151.20                  -35.00\n",
      "  463  20395 676S 677H C   N    5240 5242 -156.72 -125.60   36.26    0.92  -63.20 -174.41   29.23\n",
      "  463        677H 677H N   CA   5242 5243  157.40  138.80                  -42.30\n",
      "  464  20397 678V 679G C   N    5257 5259  172.28 -167.20   28.44    1.06   82.20  171.89   12.32\n",
      "  464        679G 679G N   CA   5259 5260  154.90  174.60                    8.50\n",
      "  465  20398 679G 680A C   N    5261 5263 -166.09 -134.00   34.74    0.79  -62.50 -170.40   36.57\n",
      "  465        680A 680A N   CA   5263 5264  160.31  147.00                  -40.90\n",
      "  466  20399 680A 681S C   N    5266 5268 -155.83 -136.60   21.27    0.66  -64.10 -171.46   20.26\n",
      "  466        681S 681S N   CA   5268 5269  160.28  151.20                  -35.00\n",
      "  467  20400 681S 682A C   N    5272 5274 -157.85 -134.00   25.72    0.59  -62.50 -171.62   35.96\n",
      "  467        682A 682A N   CA   5274 5275  156.63  147.00                  -40.90\n",
      "  468  20401 682A 683K C   N    5277 5279 -148.47 -118.00   33.18    1.10  -62.90 -172.39   29.48\n",
      "  468        683K 683K N   CA   5279 5280  152.24  139.10                  -40.80\n",
      "  469  20402 683K 684T C   N    5286 5288 -145.18 -124.80   34.52    1.07  -63.20  167.91   26.81\n",
      "  469        684T 684T N   CA   5288 5289  171.36  143.50                  -42.10\n",
      "  470  20403 684T 685G C   N    5293 5295  175.53 -167.20   26.81    1.04   82.20  172.93   12.48\n",
      "  470        685G 685G N   CA   5295 5296  154.09  174.60                    8.50\n",
      "  471  20404 685G 686H C   N    5297 5299 -153.81 -125.60   29.50    0.85  -63.20 -167.12   29.98\n",
      "  471        686H 686H N   CA   5299 5300  147.43  138.80                  -42.30\n",
      "  472  20405 686H 687Y C   N    5307 5309 -142.06 -124.30   23.86    1.10  -63.50 -177.01   32.89\n",
      "  472        687Y 687Y N   CA   5309 5310  151.33  135.40                  -43.40\n",
      "  473  20408 689C 690H C   N    5332 5334 -125.95 -125.60  100.74    3.84  -63.20  101.96   11.04\n",
      "  473        690H 690H N   CA   5334 5335   38.06  138.80                  -42.30\n",
      "  474  20411 692C 693D C   N    5356 5358 -158.73  -96.50   78.75    3.28  -63.30 -175.80   28.83\n",
      "  474        693D 693D N   CA   5358 5359  162.45  114.20                  -40.00\n",
      "  475  20412 693D 694A C   N    5364 5366  176.81 -134.00   51.97    1.22  -62.50 -163.28   38.55\n",
      "  475        694A 694A N   CA   5366 5367  163.76  147.00                  -40.90\n",
      "  476  20413 694A 695Q C   N    5369 5371 -168.80 -121.10   54.02    1.67  -63.80 -173.09   32.76\n",
      "  476        695Q 695Q N   CA   5371 5372  165.07  139.70                  -40.30\n",
      "  477  20414 695Q 696T C   N    5378 5380 -156.29 -124.80   48.90    1.53  -63.20  165.62   27.09\n",
      "  477        696T 696T N   CA   5380 5381 -179.08  143.50                  -42.10\n",
      "  478  20415 696T 697G C   N    5385 5387 -161.27 -167.20   12.76    0.52   82.20 -147.76   15.38\n",
      "  478        697G 697G N   CA   5387 5388 -174.11  174.60                    8.50\n",
      "  479  20416 697G 698K C   N    5389 5391  178.19 -118.00   65.61    2.26  -62.90 -156.77   33.40\n",
      "  479        698K 698K N   CA   5391 5392  154.39  139.10                  -40.80\n",
      "  480  20417 698K 699W C   N    5398 5400 -159.61 -124.90   47.26    1.50  -63.00  170.37   27.76\n",
      "  480        699W 699W N   CA   5400 5401  175.47  143.40                  -44.20\n",
      "  481  20418 699W 700L C   N    5412 5414 -157.45 -108.50   67.31    3.02  -63.50  168.67   29.15\n",
      "  481        700L 700L N   CA   5414 5415  178.71  132.50                  -41.20\n",
      "  482  20419 700L 701L C   N    5420 5422 -162.16 -108.50   72.44    3.24  -63.50  169.34   29.50\n",
      "  482        701L 701L N   CA   5422 5423 -178.84  132.50                  -41.20\n",
      "  483  20420 701L 702F C   N    5428 5430 -157.92 -124.20   57.76    1.73  -63.20  157.23   27.55\n",
      "  483        702F 702F N   CA   5430 5431 -169.80  143.30                  -44.30\n",
      "  484  20421 702F 703N C   N    5439 5441  170.74 -119.90   75.37    2.41  -63.20 -162.21   31.46\n",
      "  484        703N 703N N   CA   5441 5442  166.49  137.00                  -41.10\n",
      "  485  20422 703N 704D C   N    5447 5449 -166.92  -96.50   86.66    3.61  -63.30 -173.30   29.62\n",
      "  485        704D 704D N   CA   5449 5450  164.70  114.20                  -40.00\n",
      "  486  20423 704D 705E C   N    5455 5457 -178.56 -117.80   70.09    2.05  -63.60 -172.61   31.12\n",
      "  486        705E 705E N   CA   5457 5458  171.72  136.80                  -40.30\n",
      "  487  20424 705E 706K C   N    5464 5466 -175.06 -118.00   61.27    2.04  -62.90 -166.42   31.75\n",
      "  487        706K 706K N   CA   5466 5467  161.42  139.10                  -40.80\n",
      "  488  20426 707V 708A C   N    5480 5482  156.49 -134.00   72.28    1.76  -62.50 -152.43   41.25\n",
      "  488        708A 708A N   CA   5482 5483  166.78  147.00                  -40.90\n",
      "  489  20427 708A 709E C   N    5485 5487 -178.91 -117.80   73.48    2.17  -63.60 -176.99   30.52\n",
      "  489        709E 709E N   CA   5487 5488  177.59  136.80                  -40.30\n",
      "  490  20428 709E 710S C   N    5494 5496 -152.70 -136.60   74.36    3.40  -64.10  134.51   16.12\n",
      "  490        710S 710S N   CA   5496 5497 -136.21  151.20                  -35.00\n",
      "  491  20429 710S 711L C   N    5500 5502  131.38 -108.50  124.91    5.61  -63.50 -135.54   40.68\n",
      "  491        711L 711L N   CA   5502 5503  166.76  132.50                  -41.20\n",
      "  492  20430 711L 712N C   N    5508 5510  174.83 -119.90   67.85    2.35  -63.20 -156.13   32.10\n",
      "  492        712N 712N N   CA   5510 5511  155.54  137.00                  -41.10\n",
      "  493  20431 712N 713P C   N    5516 5518  166.77  -64.50  133.25   12.01  -64.50  133.25   12.01\n",
      "  493        713P 713P N   CA   5518 5519 -178.41  147.20                  147.20\n",
      "  494  20432 713P 714P C   N    5523 5525  156.45  -64.50  140.32   14.97  -64.50  140.32   14.97\n",
      "  494        714P 714P N   CA   5525 5526  128.39  147.20                  147.20\n",
      "  495  20433 714P 715F C   N    5530 5532 -179.14 -124.20   58.89    1.71  -63.20 -169.46   33.43\n",
      "  495        715F 715F N   CA   5532 5533  164.50  143.30                  -44.30\n",
      "  496  20434 715F 716S C   N    5541 5543 -117.46 -136.60   39.03    2.21  -64.10  149.62   14.61\n",
      "  496        716S 716S N   CA   5543 5544 -174.79  151.20                  -35.00\n",
      "  497  20435 716S 717M C   N    5547 5549 -159.38 -125.60   95.56    3.76  -63.40  131.31   26.36\n",
      "  497        717M 717M N   CA   5549 5550 -130.11  140.50                  -40.50\n",
      "  498  20436 717M 718A C   N    5555 5557   22.95  -68.20   91.19    7.82  -62.50 -168.79   28.66\n",
      "  498        718A 718A N   CA   5557 5558  148.05  145.30                  -40.90\n",
      "  499  20437 718A 719F C   N    5560 5562  -42.40  -71.40   31.40    2.51  -63.20  164.29   21.45\n",
      "  499        719F 719F N   CA   5562 5563  152.73  140.70                  -44.30\n",
      "  500  20438 719F 720L C   N    5571 5573 -127.73 -108.50   33.83    1.58  -63.50  170.98   27.72\n",
      "  500        720L 720L N   CA   5573 5574  160.34  132.50                  -41.20\n",
      "  501  20439 720L 721Y C   N    5579 5581  153.06 -124.30   83.75    6.65  -63.50 -141.16   30.24\n",
      "  501        721Y 721Y N   CA   5581 5582  121.87  135.40                  -43.40\n",
      "  502  20440 721Y 722F C   N    5591 5593  172.36 -124.20   72.43    1.94  -63.20 -174.59   32.96\n",
      "  502        722F 722F N   CA   5593 5594  178.25  143.30                  -44.30\n",
      "  503  20441 722F 723Y C   N    5602 5604 -155.35 -124.30   57.12    2.27  -63.50  161.84   29.95\n",
      "  503        723Y 723Y N   CA   5604 5605 -176.66  135.40                  -43.40\n",
      "  504  20442 723Y 724K C   N    5614 5616  168.65   56.60  125.27   11.04  -62.90  130.57   20.24\n",
      "  504        724K 724K N   CA   5616 5617  -17.42   38.60                  -40.80\n",
      "  505  20443 724K 725R C   N    5623 5625  166.40 -125.20   74.00    3.66  -63.00 -158.49   23.27\n",
      "  505        725R 725R N   CA   5625 5626  112.36  140.60                  -41.10\n",
      "  506  20445 726V 727G C   N    5641 5643   95.69   78.70   43.28    0.66   82.20  146.21    7.77\n",
      "  506        727G 727G N   CA   5643 5644  154.09 -166.10                    8.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  27813  82L 197C CD1 CB    650 1499   10.74    3.80    6.94    4.58    3.80    6.94    4.58\n",
      "    2  28118 233P 607L CB  CD2  1763 4719   13.21    4.45    8.76    5.52    4.45    8.76    5.52\n",
      "    3  28184 252F 606M CB  SD   1910 4710   22.15    4.79   17.36   13.73    4.79   17.36   13.73\n",
      "    4  28185 253A 606M CB  CG   1921 4709   19.58    4.50   15.08    9.99    4.50   15.08    9.99\n",
      "    5  28186 253A 606M CB  SD   1921 4710   17.78    3.87   13.90    9.45    3.87   13.90    9.45\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    4   59  108  294  407  485  463  475  520  625  628\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00192.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      728     5656     5656\n",
      "openf___224_> Open           EAO00192.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18229916   17802.652    17.385\n",
      "read_mo_297_> Segments, residues, atoms:        1      728     5656\n",
      "read_mo_298_> Segment:        1  1:A  728:A     5656\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5656     5656        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    23948\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24395\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26848\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27084\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27270\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28042\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28044\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28188\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28194\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.2300        9  9\n",
      "iupac_m_485_> OD1/2 will be swapped:      166.6587       17  17\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.7028       37  37\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2953       40  40\n",
      "iupac_m_486_> OE1/2 will be swapped:     -162.9444       45  45\n",
      "iupac_m_485_> OD1/2 will be swapped:      106.5896       86  86\n",
      "iupac_m_485_> OD1/2 will be swapped:     -175.8330       94  94\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.0176      107  107\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.8077      119  119\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      -99.9520      124  124\n",
      "iupac_m_485_> OD1/2 will be swapped:       96.7255      128  128\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.4811      146  146\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.5756      170  170\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -144.9866      183  183\n",
      "iupac_m_487_> NH1/2 swapped:     -112.6117      188  188\n",
      "iupac_m_486_> OE1/2 will be swapped:      100.3221      205  205\n",
      "iupac_m_486_> OE1/2 will be swapped:     -106.4293      246  246\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      135.3018      252  252\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -156.9341      275  275\n",
      "iupac_m_485_> OD1/2 will be swapped:     -134.7758      279  279\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.6413      291  291\n",
      "iupac_m_485_> OD1/2 will be swapped:     -151.9265      294  294\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.9247      331  331\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.0242      338  338\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      358  358\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4503      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.7928      372  372\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9604      383  383\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.0187      384  384\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8847      387  387\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      400  400\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9802      406  406\n",
      "iupac_m_487_> NH1/2 swapped:      179.7295      416  416\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7444      417  417\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.6887      434  434\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.9875      441  441\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1815      443  443\n",
      "iupac_m_487_> NH1/2 swapped:      179.9441      456  456\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.7497      469  469\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.6311      476  476\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9720      484  484\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.9974      485  485\n",
      "iupac_m_485_> OD1/2 will be swapped:      177.9335      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      165.6377      509  509\n",
      "iupac_m_487_> NH1/2 swapped:      179.6289      511  511\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.6234      532  532\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.8730      537  537\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.7656      539  539\n",
      "iupac_m_485_> OD1/2 will be swapped:      126.6038      544  544\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.3074      556  556\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.8094      566  566\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      110.6956      567  567\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:       90.4447      577  577\n",
      "iupac_m_484W> Dihedral still outside +-90:      -94.6246\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.6776      582  582\n",
      "iupac_m_486_> OE1/2 will be swapped:      122.6350      585  585\n",
      "iupac_m_485_> OD1/2 will be swapped:     -142.0980      600  600\n",
      "iupac_m_486_> OE1/2 will be swapped:      141.3353      601  601\n",
      "iupac_m_486_> OE1/2 will be swapped:      132.2200      614  614\n",
      "iupac_m_487_> NH1/2 swapped:     -121.9121      618  618\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -113.7366      619  619\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.3354      629  629\n",
      "iupac_m_485_> OD1/2 will be swapped:     -168.5675      633  633\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.9937      639  639\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.2190      649  649\n",
      "iupac_m_487_> NH1/2 swapped:      179.6835      670  670\n",
      "iupac_m_485_> OD1/2 will be swapped:      128.9090      704  704\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.0589      705  705\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.5798      715  715\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.7522      719  719\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.9037\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      127.7357      721  721\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.2199      723  723\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.7645\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   476498\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -18842.2461\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -18842.246094\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    53.912998\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sequence length          :          728\n",
      "Compactness              :     0.000337\n",
      "Native energy (pair)     :   981.004451\n",
      "Native energy (surface)  :   107.958218\n",
      "Native energy (combined) :    30.087999\n",
      "Z score (pair)           :    -2.012878\n",
      "Z score (surface)        :    -1.431667\n",
      "Z score (combined)       :    -2.380833\n",
      "GA341 score              :     0.590323\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     8756\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      382246.9375\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5789     415    529   0.429   0.429     0.31497E+06   1.000\n",
      " 2 Bond angle potential               :    7845     486    816   9.952   9.952      16470.       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3676       0    268  55.910  55.910      2009.7       1.000\n",
      " 4 Stereochemical improper torsion pot:    2409       4     25   2.435   2.435      279.68       1.000\n",
      " 5 Soft-sphere overlap restraints     :    8756       0      6   0.007   0.007      56.349       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    1992     207    383   4.456   4.456      23740.       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2093     175    367   4.920   4.920      21625.       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     727       2     31   5.462   5.462      255.77       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     627       0     54  91.461  91.461      442.75       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     449       0     20  77.404  77.404      280.66       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     214       0      5 106.496 106.496      172.46       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0  91.802  91.802      39.031       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1112       0      0   0.746   0.746      62.273       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     726     500    222  51.359 152.299      1551.8       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     471       5     14   2.788   2.788      289.24       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00192.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00192.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  229939.3281\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     84  10E  10E C   CA     81   75    1.66    1.49    0.17    5.00    1.49    0.17    5.00\n",
      "    2     87  11A  11A N   CA     83   84    1.58    1.43    0.15    4.79    1.43    0.15    4.79\n",
      "    3     89  11A  11A C   CA     86   84    1.67    1.49    0.18    5.24    1.49    0.18    5.24\n",
      "    4     90  11A  12C C   N      86   88    1.48    1.35    0.13    4.77    1.35    0.13    4.77\n",
      "    5     93  12C  12C N   CA     88   89    1.59    1.43    0.16    5.10    1.43    0.16    5.10\n",
      "    6     95  12C  12C C   CA     92   89    1.68    1.49    0.19    5.66    1.49    0.19    5.66\n",
      "    7     96  12C  13A C   N      92   94    1.49    1.35    0.14    5.09    1.35    0.14    5.09\n",
      "    8     98  13A  13A N   CA     94   95    1.60    1.43    0.17    5.46    1.43    0.17    5.46\n",
      "    9    100  13A  13A C   CA     97   95    1.69    1.49    0.20    5.95    1.49    0.20    5.95\n",
      "   10    101  13A  14H C   N      97   99    1.50    1.35    0.15    5.48    1.35    0.15    5.48\n",
      "   11    108  14H  14H N   CA     99  100    1.61    1.43    0.18    5.83    1.43    0.18    5.83\n",
      "   12    110  14H  14H C   CA    107  100    1.72    1.49    0.23    6.79    1.49    0.23    6.79\n",
      "   13    111  14H  15C C   N     107  109    1.51    1.35    0.17    5.94    1.35    0.17    5.94\n",
      "   14    115  15C  15C N   CA    109  110    1.62    1.43    0.19    6.39    1.43    0.19    6.39\n",
      "   15    117  15C  15C C   CA    113  110    1.74    1.49    0.25    7.14    1.49    0.25    7.14\n",
      "   16    118  15C  16W C   N     113  115    1.53    1.35    0.18    6.40    1.35    0.18    6.40\n",
      "   17    128  16W  16W N   CA    115  116    1.64    1.43    0.21    6.79    1.43    0.21    6.79\n",
      "   18    130  16W  16W C   CA    127  116    1.77    1.49    0.28    8.05    1.49    0.28    8.05\n",
      "   19    131  16W  17D C   N     127  129    1.55    1.35    0.20    7.24    1.35    0.20    7.24\n",
      "   20    139  17D  17D N   CA    129  130    1.67    1.43    0.24    7.83    1.43    0.24    7.83\n",
      "   21    141  17D  17D C   CA    135  130    1.78    1.49    0.29    8.54    1.49    0.29    8.54\n",
      "   22    142  17D  18E C   N     135  137    1.56    1.35    0.21    7.48    1.35    0.21    7.48\n",
      "   23    148  18E  18E N   CA    137  138    1.67    1.43    0.24    8.05    1.43    0.24    8.05\n",
      "   24    150  18E  18E C   CA    144  138    1.81    1.49    0.32    9.18    1.49    0.32    9.18\n",
      "   25    151  18E  19S C   N     144  146    1.57    1.35    0.22    7.97    1.35    0.22    7.97\n",
      "   26    154  19S  19S N   CA    146  147    1.69    1.43    0.26    8.62    1.43    0.26    8.62\n",
      "   27    156  19S  19S C   CA    150  147    1.82    1.49    0.33    9.55    1.49    0.33    9.55\n",
      "   28    157  19S  20V C   N     150  152    1.58    1.35    0.24    8.46    1.35    0.24    8.46\n",
      "   29    161  20V  20V N   CA    152  153    1.71    1.43    0.28    9.16    1.43    0.28    9.16\n",
      "   30    163  20V  20V C   CA    157  153    1.84    1.49    0.35   10.17    1.49    0.35   10.17\n",
      "   31    164  20V  21M C   N     157  159    1.60    1.35    0.25    8.93    1.35    0.25    8.93\n",
      "   32    169  21M  21M N   CA    159  160    1.72    1.43    0.29    9.54    1.43    0.29    9.54\n",
      "   33    171  21M  21M C   CA    165  160    1.85    1.49    0.36   10.56    1.49    0.36   10.56\n",
      "   34    172  21M  22L C   N     165  167    1.60    1.35    0.26    9.14    1.35    0.26    9.14\n",
      "   35    177  22L  22L N   CA    167  168    1.73    1.43    0.30    9.94    1.43    0.30    9.94\n",
      "   36    179  22L  22L C   CA    173  168    1.87    1.49    0.38   10.98    1.49    0.38   10.98\n",
      "   37    180  22L  23V C   N     173  175    1.61    1.35    0.27    9.50    1.35    0.27    9.50\n",
      "   38    184  23V  23V N   CA    175  176    1.74    1.43    0.31   10.16    1.43    0.31   10.16\n",
      "   39    186  23V  23V C   CA    180  176    1.90    1.49    0.41   11.92    1.49    0.41   11.92\n",
      "   40    187  23V  24P C   N     180  182    1.69    1.30    0.39   11.59    1.30    0.39   11.59\n",
      "   41    189  24P  24P C   CA    187  183    1.90    1.49    0.41   12.02    1.49    0.41   12.02\n",
      "   42    190  24P  25N C   N     187  189    1.64    1.35    0.29   10.32    1.35    0.29   10.32\n",
      "   43    191  24P  24P N   CA    182  183    1.78    1.43    0.34   11.25    1.43    0.34   11.25\n",
      "   44    200  25N  25N N   CA    189  190    1.77    1.43    0.34   11.17    1.43    0.34   11.17\n",
      "   45    202  25N  25N C   CA    195  190    1.93    1.49    0.44   12.68    1.49    0.44   12.68\n",
      "   46    203  25N  26H C   N     195  197    1.65    1.35    0.31   10.83    1.35    0.31   10.83\n",
      "   47    210  26H  26H N   CA    197  198    1.79    1.43    0.36   11.73    1.43    0.36   11.73\n",
      "   48    212  26H  26H C   CA    205  198    1.94    1.49    0.45   13.18    1.49    0.45   13.18\n",
      "   49    213  26H  27R C   N     205  207    1.66    1.35    0.32   11.26    1.35    0.32   11.26\n",
      "   50    222  27R  27R N   CA    207  208    1.80    1.43    0.37   12.17    1.43    0.37   12.17\n",
      "   51    224  27R  27R C   CA    216  208    1.97    1.49    0.48   14.02    1.49    0.48   14.02\n",
      "   52    225  27R  28S C   N     216  218    1.68    1.35    0.33   11.88    1.35    0.33   11.88\n",
      "   53    228  28S  28S N   CA    218  219    1.82    1.43    0.39   12.89    1.43    0.39   12.89\n",
      "   54    230  28S  28S C   CA    222  219    1.98    1.49    0.49   14.35    1.49    0.49   14.35\n",
      "   55    231  28S  29A C   N     222  224    1.69    1.35    0.34   12.17    1.35    0.34   12.17\n",
      "   56    233  29A  29A N   CA    224  225    1.83    1.43    0.40   13.22    1.43    0.40   13.22\n",
      "   57    235  29A  29A C   CA    227  225    1.99    1.49    0.50   14.64    1.49    0.50   14.64\n",
      "   58    236  29A  30V C   N     227  229    1.70    1.35    0.35   12.44    1.35    0.35   12.44\n",
      "   59    240  30V  30V N   CA    229  230    1.84    1.43    0.41   13.49    1.43    0.41   13.49\n",
      "   60    242  30V  30V C   CA    234  230    2.01    1.49    0.52   15.11    1.49    0.52   15.11\n",
      "   61    243  30V  31H C   N     234  236    1.71    1.35    0.36   12.87    1.35    0.36   12.87\n",
      "   62    250  31H  31H N   CA    236  237    1.85    1.43    0.42   13.95    1.43    0.42   13.95\n",
      "   63    252  31H  31H C   CA    244  237    2.03    1.49    0.54   15.74    1.49    0.54   15.74\n",
      "   64    253  31H  32K C   N     244  246    1.72    1.35    0.37   13.29    1.35    0.37   13.29\n",
      "   65    260  32K  32K N   CA    246  247    1.87    1.43    0.44   14.40    1.43    0.44   14.40\n",
      "   66    262  32K  32K C   CA    253  247    2.05    1.49    0.56   16.36    1.49    0.56   16.36\n",
      "   67    263  32K  33E C   N     253  255    1.73    1.35    0.39   13.83    1.35    0.39   13.83\n",
      "   68    269  33E  33E N   CA    255  256    1.89    1.43    0.46   15.02    1.43    0.46   15.02\n",
      "   69    271  33E  33E C   CA    262  256    2.07    1.49    0.58   16.95    1.49    0.58   16.95\n",
      "   70    272  33E  34E C   N     262  264    1.75    1.35    0.40   14.35    1.35    0.40   14.35\n",
      "   71    278  34E  34E N   CA    264  265    1.90    1.43    0.47   15.56    1.43    0.47   15.56\n",
      "   72    280  34E  34E C   CA    271  265    2.10    1.49    0.61   17.71    1.49    0.61   17.71\n",
      "   73    281  34E  35C C   N     271  273    1.76    1.35    0.42   14.86    1.35    0.42   14.86\n",
      "   74    284  35C  35C N   CA    273  274    1.92    1.43    0.49   16.14    1.43    0.49   16.14\n",
      "   75    286  35C  35C C   CA    277  274    2.11    1.49    0.62   18.03    1.49    0.62   18.03\n",
      "   76    287  35C  36A C   N     277  279    1.77    1.35    0.43   15.13    1.35    0.43   15.13\n",
      "   77    289  36A  36A N   CA    279  280    1.93    1.43    0.50   16.43    1.43    0.50   16.43\n",
      "   78    291  36A  36A C   CA    282  280    2.12    1.49    0.63   18.23    1.49    0.63   18.23\n",
      "   79    292  36A  37Y C   N     282  284    1.78    1.35    0.43   15.30    1.35    0.43   15.30\n",
      "   80    302  37Y  37Y N   CA    284  285    1.94    1.43    0.51   16.62    1.43    0.51   16.62\n",
      "   81    304  37Y  37Y C   CA    294  285    2.14    1.49    0.65   18.96    1.49    0.65   18.96\n",
      "   82    305  37Y  38C C   N     294  296    1.79    1.35    0.45   15.85    1.35    0.45   15.85\n",
      "   83    308  38C  38C N   CA    296  297    1.95    1.43    0.52   17.22    1.43    0.52   17.22\n",
      "   84    310  38C  38C C   CA    300  297    2.15    1.49    0.66   19.21    1.49    0.66   19.21\n",
      "   85    311  38C  39C C   N     300  302    1.80    1.35    0.45   16.09    1.35    0.45   16.09\n",
      "   86    314  39C  39C N   CA    302  303    1.96    1.43    0.53   17.49    1.43    0.53   17.49\n",
      "   87    316  39C  39C C   CA    306  303    2.16    1.49    0.67   19.48    1.49    0.67   19.48\n",
      "   88    317  39C  40R C   N     306  308    1.81    1.35    0.46   16.36    1.35    0.46   16.36\n",
      "   89    325  40R  40R N   CA    308  309    1.97    1.43    0.54   17.75    1.43    0.54   17.75\n",
      "   90    327  40R  40R C   CA    317  309    2.18    1.49    0.69   20.00    1.49    0.69   20.00\n",
      "   91    328  40R  41T C   N     317  319    1.82    1.35    0.47   16.73    1.35    0.47   16.73\n",
      "   92    332  41T  41T N   CA    319  320    1.98    1.43    0.55   18.18    1.43    0.55   18.18\n",
      "   93    334  41T  41T C   CA    324  320    2.20    1.49    0.71   20.51    1.49    0.71   20.51\n",
      "   94    335  41T  42C C   N     324  326    1.83    1.35    0.48   17.17    1.35    0.48   17.17\n",
      "   95    338  42C  42C N   CA    326  327    2.00    1.43    0.57   18.67    1.43    0.57   18.67\n",
      "   96    340  42C  42C C   CA    330  327    2.21    1.49    0.72   20.90    1.49    0.72   20.90\n",
      "   97    341  42C  43R C   N     330  332    1.84    1.35    0.49   17.51    1.35    0.49   17.51\n",
      "   98    349  43R  43R N   CA    332  333    2.01    1.43    0.58   18.99    1.43    0.58   18.99\n",
      "   99    351  43R  43R C   CA    341  333    2.23    1.49    0.74   21.62    1.49    0.74   21.62\n",
      "  100    352  43R  44H C   N     341  343    1.85    1.35    0.51   18.03    1.35    0.51   18.03\n",
      "  101    359  44H  44H N   CA    343  344    2.03    1.43    0.60   19.64    1.43    0.60   19.64\n",
      "  102    361  44H  44H C   CA    351  344    2.26    1.49    0.77   22.29    1.49    0.77   22.29\n",
      "  103    362  44H  45E C   N     351  353    1.87    1.35    0.53   18.63    1.35    0.53   18.63\n",
      "  104    369  45E  45E N   CA    353  354    2.05    1.43    0.62   20.27    1.43    0.62   20.27\n",
      "  105    371  45E  45E C   CA    360  354    2.28    1.49    0.79   22.95    1.49    0.79   22.95\n",
      "  106    372  45E  46M C   N     360  362    1.88    1.35    0.54   19.11    1.35    0.54   19.11\n",
      "  107    377  46M  46M N   CA    362  363    2.06    1.43    0.63   20.80    1.43    0.63   20.80\n",
      "  108    379  46M  46M C   CA    368  363    2.29    1.49    0.80   23.35    1.49    0.80   23.35\n",
      "  109    380  46M  47G C   N     368  370    1.89    1.35    0.55   19.40    1.35    0.55   19.40\n",
      "  110    381  47G  47G N   CA    370  371    2.00    1.43    0.57   18.62    1.43    0.57   18.62\n",
      "  111    383  47G  47G C   CA    372  371    2.03    1.49    0.54   15.81    1.49    0.54   15.81\n",
      "  112    384  47G  48T C   N     372  374    1.60    1.35    0.25    8.91    1.35    0.25    8.91\n",
      "  113    388  48T  48T N   CA    374  375    1.72    1.43    0.29    9.55    1.43    0.29    9.55\n",
      "  114   2043 261I 261I C   CA   1990 1985    1.75    1.49    0.26    7.70    1.49    0.26    7.70\n",
      "  115   2044 261I 262D C   N    1990 1992    1.53    1.35    0.19    6.65    1.35    0.19    6.65\n",
      "  116   2049 262D 262D N   CA   1992 1993    1.69    1.43    0.26    8.45    1.43    0.26    8.45\n",
      "  117   2051 262D 262D C   CA   1998 1993    1.87    1.49    0.38   10.93    1.49    0.38   10.93\n",
      "  118   2052 262D 263I C   N    1998 2000    1.64    1.35    0.30   10.48    1.35    0.30   10.48\n",
      "  119   2057 263I 263I N   CA   2000 2001    1.77    1.43    0.34   11.04    1.43    0.34   11.04\n",
      "  120   2059 263I 263I C   CA   2006 2001    1.90    1.49    0.41   11.84    1.49    0.41   11.84\n",
      "  121   2060 263I 264Q C   N    2006 2008    1.62    1.35    0.27    9.63    1.35    0.27    9.63\n",
      "  122   2066 264Q 264Q N   CA   2008 2009    1.75    1.43    0.32   10.52    1.43    0.32   10.52\n",
      "  123   2068 264Q 264Q C   CA   2015 2009    1.88    1.49    0.39   11.28    1.49    0.39   11.28\n",
      "  124   2069 264Q 265T C   N    2015 2017    1.60    1.35    0.25    9.00    1.35    0.25    9.00\n",
      "  125   2073 265T 265T N   CA   2017 2018    1.72    1.43    0.29    9.69    1.43    0.29    9.69\n",
      "  126   2075 265T 265T C   CA   2022 2018    1.83    1.49    0.34   10.03    1.49    0.34   10.03\n",
      "  127   2076 265T 266A C   N    2022 2024    1.58    1.35    0.24    8.40    1.35    0.24    8.40\n",
      "  128   2078 266A 266A N   CA   2024 2025    1.70    1.43    0.27    9.00    1.43    0.27    9.00\n",
      "  129   2080 266A 266A C   CA   2027 2025    1.82    1.49    0.33    9.65    1.49    0.33    9.65\n",
      "  130   2081 266A 267K C   N    2027 2029    1.57    1.35    0.23    8.11    1.35    0.23    8.11\n",
      "  131   2087 267K 267K N   CA   2029 2030    1.69    1.43    0.26    8.57    1.43    0.26    8.57\n",
      "  132   2089 267K 267K C   CA   2036 2030    1.81    1.49    0.32    9.42    1.49    0.32    9.42\n",
      "  133   2090 267K 268K C   N    2036 2038    1.56    1.35    0.22    7.78    1.35    0.22    7.78\n",
      "  134   2096 268K 268K N   CA   2038 2039    1.68    1.43    0.25    8.26    1.43    0.25    8.26\n",
      "  135   2098 268K 268K C   CA   2045 2039    1.79    1.49    0.30    8.83    1.49    0.30    8.83\n",
      "  136   2099 268K 269T C   N    2045 2047    1.56    1.35    0.21    7.45    1.35    0.21    7.45\n",
      "  137   2103 269T 269T N   CA   2047 2048    1.67    1.43    0.24    8.02    1.43    0.24    8.02\n",
      "  138   2105 269T 269T C   CA   2052 2048    1.77    1.49    0.28    8.19    1.49    0.28    8.19\n",
      "  139   2106 269T 270A C   N    2052 2054    1.54    1.35    0.20    7.07    1.35    0.20    7.07\n",
      "  140   2108 270A 270A N   CA   2054 2055    1.66    1.43    0.23    7.53    1.43    0.23    7.53\n",
      "  141   2110 270A 270A C   CA   2057 2055    1.77    1.49    0.28    8.12    1.49    0.28    8.12\n",
      "  142   2111 270A 271K C   N    2057 2059    1.54    1.35    0.20    6.96    1.35    0.20    6.96\n",
      "  143   2117 271K 271K N   CA   2059 2060    1.66    1.43    0.23    7.48    1.43    0.23    7.48\n",
      "  144   2119 271K 271K C   CA   2066 2060    1.76    1.49    0.27    7.89    1.49    0.27    7.89\n",
      "  145   2120 271K 272T C   N    2066 2068    1.54    1.35    0.19    6.81    1.35    0.19    6.81\n",
      "  146   2124 272T 272T N   CA   2068 2069    1.65    1.43    0.22    7.30    1.43    0.22    7.30\n",
      "  147   2126 272T 272T C   CA   2073 2069    1.75    1.49    0.26    7.69    1.49    0.26    7.69\n",
      "  148   2127 272T 273L C   N    2073 2075    1.53    1.35    0.19    6.65    1.35    0.19    6.65\n",
      "  149   2132 273L 273L N   CA   2075 2076    1.65    1.43    0.22    7.07    1.43    0.22    7.07\n",
      "  150   2134 273L 273L C   CA   2081 2076    1.75    1.49    0.26    7.57    1.49    0.26    7.57\n",
      "  151   2135 273L 274G C   N    2081 2083    1.53    1.35    0.18    6.38    1.35    0.18    6.38\n",
      "  152   2136 274G 274G N   CA   2083 2084    1.64    1.43    0.21    6.83    1.43    0.21    6.83\n",
      "  153   2138 274G 274G C   CA   2085 2084    1.74    1.49    0.25    7.30    1.49    0.25    7.30\n",
      "  154   2139 274G 275E C   N    2085 2087    1.52    1.35    0.17    6.16    1.35    0.17    6.16\n",
      "  155   2145 275E 275E N   CA   2087 2088    1.63    1.43    0.20    6.54    1.43    0.20    6.54\n",
      "  156   2147 275E 275E C   CA   2094 2088    1.72    1.49    0.23    6.63    1.49    0.23    6.63\n",
      "  157   2148 275E 276M C   N    2094 2096    1.51    1.35    0.16    5.67    1.35    0.16    5.67\n",
      "  158   2153 276M 276M N   CA   2096 2097    1.61    1.43    0.18    6.01    1.43    0.18    6.01\n",
      "  159   2155 276M 276M C   CA   2102 2097    1.71    1.49    0.22    6.25    1.49    0.22    6.25\n",
      "  160   2156 276M 277Q C   N    2102 2104    1.50    1.35    0.15    5.33    1.35    0.15    5.33\n",
      "  161   2162 277Q 277Q N   CA   2104 2105    1.60    1.43    0.17    5.74    1.43    0.17    5.74\n",
      "  162   2164 277Q 277Q C   CA   2111 2105    1.69    1.49    0.20    5.67    1.49    0.20    5.67\n",
      "  163   2165 277Q 278Y C   N    2111 2113    1.48    1.35    0.14    4.89    1.35    0.14    4.89\n",
      "  164   2175 278Y 278Y N   CA   2113 2114    1.59    1.43    0.16    5.19    1.43    0.16    5.19\n",
      "  165   2177 278Y 278Y C   CA   2123 2114    1.68    1.49    0.19    5.40    1.49    0.19    5.40\n",
      "  166   2178 278Y 279D C   N    2123 2125    1.47    1.35    0.13    4.53    1.35    0.13    4.53\n",
      "  167   2183 279D 279D N   CA   2125 2126    1.57    1.43    0.14    4.67    1.43    0.14    4.67\n",
      "  168   2185 279D 279D C   CA   2131 2126    1.66    1.49    0.17    4.95    1.49    0.17    4.95\n",
      "  169   2198 280Y 280Y C   CA   2143 2134    1.65    1.49    0.16    4.59    1.49    0.16    4.59\n",
      "  170   4434 556F 556F C   CA   4329 4321    1.67    1.49    0.18    5.10    1.49    0.18    5.10\n",
      "  171   4439 557V 557V N   CA   4331 4332    1.57    1.43    0.14    4.77    1.43    0.14    4.77\n",
      "  172   4441 557V 557V C   CA   4336 4332    1.67    1.49    0.18    5.34    1.49    0.18    5.34\n",
      "  173   4442 557V 558D C   N    4336 4338    1.48    1.35    0.14    4.81    1.35    0.14    4.81\n",
      "  174   4447 558D 558D N   CA   4338 4339    1.59    1.43    0.16    5.39    1.43    0.16    5.39\n",
      "  175   4449 558D 558D C   CA   4344 4339    1.70    1.49    0.21    6.00    1.49    0.21    6.00\n",
      "  176   4450 558D 559V C   N    4344 4346    1.51    1.35    0.16    5.75    1.35    0.16    5.75\n",
      "  177   4454 559V 559V N   CA   4346 4347    1.61    1.43    0.18    6.05    1.43    0.18    6.05\n",
      "  178   4456 559V 559V C   CA   4351 4347    1.74    1.49    0.25    7.24    1.49    0.25    7.24\n",
      "  179   4457 559V 560P C   N    4351 4353    1.54    1.30    0.24    7.03    1.30    0.24    7.03\n",
      "  180   4459 560P 560P C   CA   4358 4354    1.76    1.49    0.27    7.90    1.49    0.27    7.90\n",
      "  181   4460 560P 561L C   N    4358 4360    1.55    1.35    0.20    7.18    1.35    0.20    7.18\n",
      "  182   4461 560P 560P N   CA   4353 4354    1.66    1.43    0.22    7.30    1.43    0.22    7.30\n",
      "  183   4470 561L 561L N   CA   4360 4361    1.67    1.43    0.24    7.80    1.43    0.24    7.80\n",
      "  184   4472 561L 561L C   CA   4366 4361    1.80    1.49    0.31    9.12    1.49    0.31    9.12\n",
      "  185   4473 561L 562E C   N    4366 4368    1.57    1.35    0.23    8.08    1.35    0.23    8.08\n",
      "  186   4479 562E 562E N   CA   4368 4369    1.70    1.43    0.27    8.77    1.43    0.27    8.77\n",
      "  187   4481 562E 562E C   CA   4375 4369    1.87    1.49    0.38   10.98    1.49    0.38   10.98\n",
      "  188   4482 562E 563L C   N    4375 4377    1.62    1.35    0.28    9.76    1.35    0.28    9.76\n",
      "  189   4487 563L 563L N   CA   4377 4378    1.75    1.43    0.32   10.66    1.43    0.32   10.66\n",
      "  190   4489 563L 563L C   CA   4383 4378    1.91    1.49    0.42   12.27    1.49    0.42   12.27\n",
      "  191   4490 563L 564D C   N    4383 4385    1.66    1.35    0.31   11.04    1.35    0.31   11.04\n",
      "  192   4495 564D 564D N   CA   4385 4386    1.80    1.43    0.37   12.01    1.43    0.37   12.01\n",
      "  193   4497 564D 564D C   CA   4391 4386    1.96    1.49    0.47   13.67    1.49    0.47   13.67\n",
      "  194   4498 564D 565L C   N    4391 4393    1.68    1.35    0.33   11.81    1.35    0.33   11.81\n",
      "  195   4503 565L 565L N   CA   4393 4394    1.82    1.43    0.39   12.90    1.43    0.39   12.90\n",
      "  196   4505 565L 565L C   CA   4399 4394    1.99    1.49    0.50   14.56    1.49    0.50   14.56\n",
      "  197   4506 565L 566E C   N    4399 4401    1.70    1.35    0.36   12.68    1.35    0.36   12.68\n",
      "  198   4512 566E 566E N   CA   4401 4402    1.85    1.43    0.42   13.83    1.43    0.42   13.83\n",
      "  199   4514 566E 566E C   CA   4408 4402    2.04    1.49    0.55   15.93    1.49    0.55   15.93\n",
      "  200   4515 566E 567F C   N    4408 4410    1.73    1.35    0.38   13.64    1.35    0.38   13.64\n",
      "  201   4524 567F 567F N   CA   4410 4411    1.88    1.43    0.45   14.89    1.43    0.45   14.89\n",
      "  202   4526 567F 567F C   CA   4419 4411    2.08    1.49    0.59   17.13    1.49    0.59   17.13\n",
      "  203   4527 567F 568L C   N    4419 4421    1.76    1.35    0.41   14.58    1.35    0.41   14.58\n",
      "  204   4532 568L 568L N   CA   4421 4422    1.91    1.43    0.48   15.78    1.43    0.48   15.78\n",
      "  205   4534 568L 568L C   CA   4427 4422    2.12    1.49    0.63   18.32    1.49    0.63   18.32\n",
      "  206   4535 568L 569R C   N    4427 4429    1.78    1.35    0.44   15.57    1.35    0.44   15.57\n",
      "  207   4543 569R 569R N   CA   4429 4430    1.94    1.43    0.51   16.86    1.43    0.51   16.86\n",
      "  208   4545 569R 569R C   CA   4438 4430    2.16    1.49    0.67   19.40    1.49    0.67   19.40\n",
      "  209   4546 569R 570G C   N    4438 4440    1.80    1.35    0.45   16.05    1.35    0.45   16.05\n",
      "  210   4547 570G 570G N   CA   4440 4441    1.96    1.43    0.53   17.44    1.43    0.53   17.44\n",
      "  211   4549 570G 570G C   CA   4442 4441    2.15    1.49    0.66   19.23    1.49    0.66   19.23\n",
      "  212   4550 570G 571K C   N    4442 4444    1.79    1.35    0.44   15.73    1.35    0.44   15.73\n",
      "  213   4556 571K 571K N   CA   4444 4445    1.95    1.43    0.52   16.96    1.43    0.52   16.96\n",
      "  214   4558 571K 571K C   CA   4451 4445    2.14    1.49    0.65   18.79    1.49    0.65   18.79\n",
      "  215   4559 571K 572G C   N    4451 4453    1.78    1.35    0.43   15.39    1.35    0.43   15.39\n",
      "  216   4560 572G 572G N   CA   4453 4454    1.94    1.43    0.51   16.66    1.43    0.51   16.66\n",
      "  217   4562 572G 572G C   CA   4455 4454    2.12    1.49    0.63   18.35    1.49    0.63   18.35\n",
      "  218   4563 572G 573L C   N    4455 4457    1.77    1.35    0.43   15.19    1.35    0.43   15.19\n",
      "  219   4568 573L 573L N   CA   4457 4458    1.94    1.43    0.51   16.62    1.43    0.51   16.62\n",
      "  220   4570 573L 573L C   CA   4463 4458    2.11    1.49    0.62   17.94    1.49    0.62   17.94\n",
      "  221   4571 573L 574Q C   N    4463 4465    1.78    1.35    0.43   15.29    1.35    0.43   15.29\n",
      "  222   4577 574Q 574Q N   CA   4465 4466    1.94    1.43    0.51   16.90    1.43    0.51   16.90\n",
      "  223   4579 574Q 574Q C   CA   4472 4466    2.15    1.49    0.66   19.10    1.49    0.66   19.10\n",
      "  224   4580 574Q 575S C   N    4472 4474    1.81    1.35    0.46   16.40    1.35    0.46   16.40\n",
      "  225   4583 575S 575S N   CA   4474 4475    1.98    1.43    0.55   17.95    1.43    0.55   17.95\n",
      "  226   4585 575S 575S C   CA   4478 4475    2.21    1.49    0.72   20.87    1.49    0.72   20.87\n",
      "  227   4586 575S 576D C   N    4478 4480    1.84    1.35    0.50   17.55    1.35    0.50   17.55\n",
      "  228   4591 576D 576D N   CA   4480 4481    2.01    1.43    0.58   19.10    1.43    0.58   19.10\n",
      "  229   4593 576D 576D C   CA   4486 4481    2.23    1.49    0.74   21.58    1.49    0.74   21.58\n",
      "  230   4594 576D 577E C   N    4486 4488    1.85    1.35    0.50   17.78    1.35    0.50   17.78\n",
      "  231   4600 577E 577E N   CA   4488 4489    2.02    1.43    0.59   19.55    1.43    0.59   19.55\n",
      "  232   4602 577E 577E C   CA   4495 4489    2.27    1.49    0.78   22.74    1.49    0.78   22.74\n",
      "  233   4603 577E 578V C   N    4495 4497    1.88    1.35    0.54   19.02    1.35    0.54   19.02\n",
      "  234   4607 578V 578V N   CA   4497 4498    2.06    1.43    0.63   20.80    1.43    0.63   20.80\n",
      "  235   4609 578V 578V C   CA   4502 4498    2.33    1.49    0.84   24.49    1.49    0.84   24.49\n",
      "  236   4610 578V 579S C   N    4502 4504    1.92    1.35    0.57   20.26    1.35    0.57   20.26\n",
      "  237   4613 579S 579S N   CA   4504 4505    2.10    1.43    0.67   22.16    1.43    0.67   22.16\n",
      "  238   4615 579S 579S C   CA   4508 4505    2.42    1.49    0.93   26.95    1.49    0.93   26.95\n",
      "  239   4616 579S 580M C   N    4508 4510    1.98    1.35    0.64   22.66    1.35    0.64   22.66\n",
      "  240   4621 580M 580M N   CA   4510 4511    2.19    1.43    0.76   25.14    1.43    0.76   25.14\n",
      "  241   4623 580M 580M C   CA   4516 4511    2.56    1.49    1.07   31.09    1.49    1.07   31.09\n",
      "  242   4624 580M 581P C   N    4516 4518    2.38    1.30    1.08   32.06    1.30    1.08   32.06\n",
      "  243   4626 581P 581P C   CA   4523 4519    2.81    1.49    1.32   38.34    1.49    1.32   38.34\n",
      "  244   4627 581P 582E C   N    4523 4525    2.29    1.35    0.94   33.51    1.35    0.94   33.51\n",
      "  245   4628 581P 581P N   CA   4518 4519    2.36    1.43    0.92   30.31    1.43    0.92   30.31\n",
      "  246   4638 582E 582E N   CA   4525 4526    2.57    1.43    1.14   37.49    1.43    1.14   37.49\n",
      "  247   4640 582E 582E C   CA   4532 4526    3.04    1.49    1.55   45.17    1.49    1.55   45.17\n",
      "  248   4641 582E 583A C   N    4532 4534    2.45    1.35    1.10   39.01    1.35    1.10   39.01\n",
      "  249   4643 583A 583A N   CA   4534 4535    2.75    1.43    1.32   43.36    1.43    1.32   43.36\n",
      "  250   4645 583A 583A C   CA   4537 4535    3.25    1.49    1.76   51.08    1.49    1.76   51.08\n",
      "  251   4646 583A 584Q C   N    4537 4539    2.59    1.35    1.24   44.06    1.35    1.24   44.06\n",
      "  252   4652 584Q 584Q N   CA   4539 4540    2.92    1.43    1.49   48.85    1.43    1.49   48.85\n",
      "  253   4654 584Q 584Q C   CA   4546 4540    3.46    1.49    1.97   57.28    1.49    1.97   57.28\n",
      "  254   4655 584Q 585E C   N    4546 4548    2.73    1.35    1.38   49.10    1.35    1.38   49.10\n",
      "  255   4661 585E 585E N   CA   4548 4549    3.07    1.43    1.64   54.08    1.43    1.64   54.08\n",
      "  256   4663 585E 585E C   CA   4555 4549    3.66    1.49    2.17   63.04    1.49    2.17   63.04\n",
      "  257   4664 585E 586D C   N    4555 4557    2.87    1.35    1.52   53.98    1.35    1.52   53.98\n",
      "  258   4669 586D 586D N   CA   4557 4558    3.25    1.43    1.82   59.98    1.43    1.82   59.98\n",
      "  259   4671 586D 586D C   CA   4563 4558    3.99    1.49    2.50   72.60    1.49    2.50   72.60\n",
      "  260   4672 586D 587L C   N    4563 4565    3.07    1.35    1.73   61.28    1.35    1.73   61.28\n",
      "  261   4677 587L 587L N   CA   4565 4566    3.48    1.43    2.05   67.30    1.43    2.05   67.30\n",
      "  262   4679 587L 587L C   CA   4571 4566    4.30    1.49    2.81   81.69    1.49    2.81   81.69\n",
      "  263   4680 587L 588P C   N    4571 4573    4.07    1.30    2.77   82.06    1.30    2.77   82.06\n",
      "  264   4682 588P 588P C   CA   4578 4574    4.57    1.49    3.08   89.50    1.49    3.08   89.50\n",
      "  265   4683 588P 589H C   N    4578 4580    3.45    1.35    2.11   74.66    1.35    2.11   74.66\n",
      "  266   4684 588P 588P N   CA   4573 4574    3.66    1.43    2.23   73.19    1.43    2.23   73.19\n",
      "  267   4686 588P 588P CB  CG   4576 4577    1.72    1.54    0.18    5.02    1.54    0.18    5.02\n",
      "  268   4695 589H 589H N   CA   4580 4581    3.91    1.43    2.48   81.57    1.43    2.48   81.57\n",
      "  269   4697 589H 589H C   CA   4588 4581    4.83    1.49    3.34   97.20    1.49    3.34   97.20\n",
      "  270   4698 589H 590K C   N    4588 4590    3.63    1.35    2.29   81.14    1.35    2.29   81.14\n",
      "  271   4705 590K 590K N   CA   4590 4591    4.11    1.43    2.68   88.06    1.43    2.68   88.06\n",
      "  272   4707 590K 590K C   CA   4597 4591    4.96    1.49    3.47  100.85    1.49    3.47  100.85\n",
      "  273   4708 590K 591P C   N    4597 4599    4.68    1.30    3.38  100.34    1.30    3.38  100.34\n",
      "  274   4710 591P 591P C   CA   4604 4600    5.17    1.49    3.68  106.92    1.49    3.68  106.92\n",
      "  275   4711 591P 592T C   N    4604 4606    3.88    1.35    2.53   89.77    1.35    2.53   89.77\n",
      "  276   4712 591P 591P N   CA   4599 4600    4.11    1.43    2.68   88.03    1.43    2.68   88.03\n",
      "  277   4714 591P 591P CB  CG   4602 4603    1.71    1.54    0.18    4.83    1.54    0.18    4.83\n",
      "  278   4720 592T 592T N   CA   4606 4607    4.40    1.43    2.97   97.81    1.43    2.97   97.81\n",
      "  279   4722 592T 592T C   CA   4611 4607    5.41    1.49    3.92  113.99    1.49    3.92  113.99\n",
      "  280   4723 592T 593S C   N    4611 4613    4.04    1.35    2.70   95.57    1.35    2.70   95.57\n",
      "  281   4726 593S 593S N   CA   4613 4614    4.60    1.43    3.17  104.27    1.43    3.17  104.27\n",
      "  282   4728 593S 593S C   CA   4617 4614    5.64    1.49    4.15  120.78    1.49    4.15  120.78\n",
      "  283   4729 593S 594S C   N    4617 4619    4.22    1.35    2.87  101.92    1.35    2.87  101.92\n",
      "  284   4732 594S 594S N   CA   4619 4620    4.81    1.43    3.38  111.27    1.43    3.38  111.27\n",
      "  285   4734 594S 594S C   CA   4623 4620    5.98    1.49    4.49  130.59    1.49    4.49  130.59\n",
      "  286   4735 594S 595S C   N    4623 4625    4.43    1.35    3.08  109.26    1.35    3.08  109.26\n",
      "  287   4738 595S 595S N   CA   4625 4626    5.05    1.43    3.62  119.10    1.43    3.62  119.10\n",
      "  288   4740 595S 595S C   CA   4629 4626    6.22    1.49    4.73  137.64    1.49    4.73  137.64\n",
      "  289   4741 595S 596M C   N    4629 4631    4.59    1.35    3.24  114.96    1.35    3.24  114.96\n",
      "  290   4746 596M 596M N   CA   4631 4632    5.24    1.43    3.81  125.20    1.43    3.81  125.20\n",
      "  291   4748 596M 596M C   CA   4637 4632    6.50    1.49    5.01  145.75    1.49    5.01  145.75\n",
      "  292   4749 596M 597K C   N    4637 4639    4.77    1.35    3.43  121.61    1.35    3.43  121.61\n",
      "  293   4755 597K 597K N   CA   4639 4640    5.44    1.43    4.01  131.99    1.43    4.01  131.99\n",
      "  294   4757 597K 597K C   CA   4646 4640    6.70    1.49    5.21  151.40    1.49    5.21  151.40\n",
      "  295   4758 597K 598T C   N    4646 4648    4.89    1.35    3.55  125.75    1.35    3.55  125.75\n",
      "  296   4762 598T 598T N   CA   4648 4649    5.58    1.43    4.15  136.56    1.43    4.15  136.56\n",
      "  297   4764 598T 598T C   CA   4653 4649    6.96    1.49    5.47  159.03    1.49    5.47  159.03\n",
      "  298   4765 598T 599V C   N    4653 4655    5.05    1.35    3.71  131.40    1.35    3.71  131.40\n",
      "  299   4769 599V 599V N   CA   4655 4656    5.76    1.43    4.33  142.39    1.43    4.33  142.39\n",
      "  300   4771 599V 599V C   CA   4660 4656    7.07    1.49    5.58  162.13    1.49    5.58  162.13\n",
      "  301   4772 599V 600D C   N    4660 4662    5.11    1.35    3.77  133.66    1.35    3.77  133.66\n",
      "  302   4777 600D 600D N   CA   4662 4663    5.82    1.43    4.39  144.35    1.43    4.39  144.35\n",
      "  303   4779 600D 600D C   CA   4668 4663    7.20    1.49    5.71  166.06    1.49    5.71  166.06\n",
      "  304   4780 600D 601E C   N    4668 4670    5.19    1.35    3.85  136.51    1.35    3.85  136.51\n",
      "  305   4786 601E 601E N   CA   4670 4671    5.91    1.43    4.48  147.28    1.43    4.48  147.28\n",
      "  306   4788 601E 601E C   CA   4677 4671    7.24    1.49    5.75  167.11    1.49    5.75  167.11\n",
      "  307   4789 601E 602E C   N    4677 4679    5.22    1.35    3.87  137.27    1.35    3.87  137.27\n",
      "  308   4795 602E 602E N   CA   4679 4680    5.94    1.43    4.51  148.42    1.43    4.51  148.42\n",
      "  309   4797 602E 602E C   CA   4686 4680    7.29    1.49    5.80  168.74    1.49    5.80  168.74\n",
      "  310   4798 602E 603A C   N    4686 4688    5.25    1.35    3.90  138.39    1.35    3.90  138.39\n",
      "  311   4800 603A 603A N   CA   4688 4689    5.96    1.43    4.53  149.17    1.43    4.53  149.17\n",
      "  312   4802 603A 603A C   CA   4691 4689    7.31    1.49    5.82  169.22    1.49    5.82  169.22\n",
      "  313   4803 603A 604L C   N    4691 4693    5.26    1.35    3.91  138.82    1.35    3.91  138.82\n",
      "  314   4808 604L 604L N   CA   4693 4694    5.98    1.43    4.55  149.63    1.43    4.55  149.63\n",
      "  315   4810 604L 604L C   CA   4699 4694    7.22    1.49    5.73  166.53    1.49    5.73  166.53\n",
      "  316   4811 604L 605A C   N    4699 4701    5.19    1.35    3.84  136.22    1.35    3.84  136.22\n",
      "  317   4813 605A 605A N   CA   4701 4702    5.89    1.43    4.46  146.78    1.43    4.46  146.78\n",
      "  318   4815 605A 605A C   CA   4704 4702    7.19    1.49    5.70  165.57    1.49    5.70  165.57\n",
      "  319   4816 605A 606M C   N    4704 4706    5.16    1.35    3.82  135.43    1.35    3.82  135.43\n",
      "  320   4821 606M 606M N   CA   4706 4707    4.84    1.43    3.41  112.20    1.43    3.41  112.20\n",
      "  321   4822 606M 606M O   C    4713 4712    1.48    1.23    0.25   11.58    1.23    0.25   11.58\n",
      "  322   4823 606M 606M C   CA   4712 4707    3.32    1.49    1.83   53.33    1.49    1.83   53.33\n",
      "  323   4824 606M 607L C   N    4712 4714    2.02    1.35    0.67   23.93    1.35    0.67   23.93\n",
      "  324   4829 607L 607L N   CA   4714 4715    1.96    1.43    0.53   17.56    1.43    0.53   17.56\n",
      "  325   4831 607L 607L C   CA   4720 4715    1.66    1.49    0.17    4.80    1.49    0.17    4.80\n",
      "  326   5535 699W 699W C   CA   5412 5401    1.65    1.49    0.16    4.77    1.49    0.16    4.77\n",
      "  327   5544 700L 700L N   CA   5414 5415    1.57    1.43    0.14    4.64    1.43    0.14    4.64\n",
      "  328   5546 700L 700L C   CA   5420 5415    1.66    1.49    0.17    4.92    1.49    0.17    4.92\n",
      "  329   5547 700L 701L C   N    5420 5422    1.48    1.35    0.13    4.68    1.35    0.13    4.68\n",
      "  330   5552 701L 701L N   CA   5422 5423    1.58    1.43    0.15    4.85    1.43    0.15    4.85\n",
      "  331   5554 701L 701L C   CA   5428 5423    1.69    1.49    0.20    5.93    1.49    0.20    5.93\n",
      "  332   5555 701L 702F C   N    5428 5430    1.49    1.35    0.15    5.21    1.35    0.15    5.21\n",
      "  333   5564 702F 702F N   CA   5430 5431    1.60    1.43    0.17    5.60    1.43    0.17    5.60\n",
      "  334   5566 702F 702F C   CA   5439 5431    1.71    1.49    0.22    6.43    1.49    0.22    6.43\n",
      "  335   5567 702F 703N C   N    5439 5441    1.51    1.35    0.16    5.76    1.35    0.16    5.76\n",
      "  336   5572 703N 703N N   CA   5441 5442    1.62    1.43    0.19    6.25    1.43    0.19    6.25\n",
      "  337   5574 703N 703N C   CA   5447 5442    1.72    1.49    0.23    6.74    1.49    0.23    6.74\n",
      "  338   5575 703N 704D C   N    5447 5449    1.51    1.35    0.17    5.98    1.35    0.17    5.98\n",
      "  339   5580 704D 704D N   CA   5449 5450    1.63    1.43    0.20    6.52    1.43    0.20    6.52\n",
      "  340   5582 704D 704D C   CA   5455 5450    1.74    1.49    0.25    7.26    1.49    0.25    7.26\n",
      "  341   5583 704D 705E C   N    5455 5457    1.53    1.35    0.18    6.46    1.35    0.18    6.46\n",
      "  342   5589 705E 705E N   CA   5457 5458    1.64    1.43    0.21    7.00    1.43    0.21    7.00\n",
      "  343   5591 705E 705E C   CA   5464 5458    1.78    1.49    0.29    8.32    1.49    0.29    8.32\n",
      "  344   5592 705E 706K C   N    5464 5466    1.55    1.35    0.21    7.42    1.35    0.21    7.42\n",
      "  345   5598 706K 706K N   CA   5466 5467    1.68    1.43    0.25    8.08    1.43    0.25    8.08\n",
      "  346   5600 706K 706K C   CA   5473 5467    1.83    1.49    0.34    9.82    1.49    0.34    9.82\n",
      "  347   5601 706K 707V C   N    5473 5475    1.59    1.35    0.24    8.56    1.35    0.24    8.56\n",
      "  348   5605 707V 707V N   CA   5475 5476    1.71    1.43    0.28    9.28    1.43    0.28    9.28\n",
      "  349   5607 707V 707V C   CA   5480 5476    1.84    1.49    0.35   10.16    1.49    0.35   10.16\n",
      "  350   5608 707V 708A C   N    5480 5482    1.59    1.35    0.25    8.71    1.35    0.25    8.71\n",
      "  351   5610 708A 708A N   CA   5482 5483    1.72    1.43    0.29    9.42    1.43    0.29    9.42\n",
      "  352   5612 708A 708A C   CA   5485 5483    1.83    1.49    0.34    9.98    1.49    0.34    9.98\n",
      "  353   5613 708A 709E C   N    5485 5487    1.58    1.35    0.24    8.50    1.35    0.24    8.50\n",
      "  354   5619 709E 709E N   CA   5487 5488    1.70    1.43    0.27    8.86    1.43    0.27    8.86\n",
      "  355   5621 709E 709E C   CA   5494 5488    1.83    1.49    0.34    9.94    1.49    0.34    9.94\n",
      "  356   5622 709E 710S C   N    5494 5496    1.57    1.35    0.23    8.15    1.35    0.23    8.15\n",
      "  357   5625 710S 710S N   CA   5496 5497    1.71    1.43    0.28    9.06    1.43    0.28    9.06\n",
      "  358   5627 710S 710S C   CA   5500 5497    1.80    1.49    0.31    9.08    1.49    0.31    9.08\n",
      "  359   5628 710S 711L C   N    5500 5502    1.56    1.35    0.21    7.61    1.35    0.21    7.61\n",
      "  360   5633 711L 711L N   CA   5502 5503    1.68    1.43    0.25    8.36    1.43    0.25    8.36\n",
      "  361   5635 711L 711L C   CA   5508 5503    1.80    1.49    0.31    8.93    1.49    0.31    8.93\n",
      "  362   5636 711L 712N C   N    5508 5510    1.57    1.35    0.22    7.82    1.35    0.22    7.82\n",
      "  363   5641 712N 712N N   CA   5510 5511    1.70    1.43    0.27    8.79    1.43    0.27    8.79\n",
      "  364   5643 712N 712N C   CA   5516 5511    1.79    1.49    0.30    8.79    1.49    0.30    8.79\n",
      "  365   5644 712N 713P C   N    5516 5518    1.65    1.30    0.35   10.31    1.30    0.35   10.31\n",
      "  366   5646 713P 713P C   CA   5523 5519    1.84    1.49    0.35   10.25    1.49    0.35   10.25\n",
      "  367   5647 713P 714P C   N    5523 5525    1.62    1.30    0.32    9.60    1.30    0.32    9.60\n",
      "  368   5648 713P 713P N   CA   5518 5519    1.72    1.43    0.29    9.51    1.43    0.29    9.51\n",
      "  369   5654 714P 714P C   CA   5530 5526    1.95    1.49    0.46   13.40    1.49    0.46   13.40\n",
      "  370   5655 714P 715F C   N    5530 5532    1.69    1.35    0.34   12.12    1.35    0.34   12.12\n",
      "  371   5656 714P 714P N   CA   5525 5526    1.77    1.43    0.33   10.98    1.43    0.33   10.98\n",
      "  372   5669 715F 715F N   CA   5532 5533    1.84    1.43    0.41   13.48    1.43    0.41   13.48\n",
      "  373   5671 715F 715F C   CA   5541 5533    2.04    1.49    0.55   15.98    1.49    0.55   15.98\n",
      "  374   5672 715F 716S C   N    5541 5543    1.73    1.35    0.39   13.66    1.35    0.39   13.66\n",
      "  375   5675 716S 716S N   CA   5543 5544    1.89    1.43    0.46   15.28    1.43    0.46   15.28\n",
      "  376   5677 716S 716S C   CA   5547 5544    2.09    1.49    0.60   17.53    1.49    0.60   17.53\n",
      "  377   5678 716S 717M C   N    5547 5549    1.77    1.35    0.42   15.04    1.35    0.42   15.04\n",
      "  378   5683 717M 717M N   CA   5549 5550    1.94    1.43    0.51   16.61    1.43    0.51   16.61\n",
      "  379   5685 717M 717M C   CA   5555 5550    2.12    1.49    0.63   18.43    1.49    0.63   18.43\n",
      "  380   5686 717M 718A C   N    5555 5557    1.78    1.35    0.44   15.51    1.35    0.44   15.51\n",
      "  381   5688 718A 718A N   CA   5557 5558    1.96    1.43    0.53   17.38    1.43    0.53   17.38\n",
      "  382   5690 718A 718A C   CA   5560 5558    2.14    1.49    0.65   18.97    1.49    0.65   18.97\n",
      "  383   5691 718A 719F C   N    5560 5562    1.79    1.35    0.45   15.89    1.35    0.45   15.89\n",
      "  384   5700 719F 719F N   CA   5562 5563    1.98    1.43    0.55   17.93    1.43    0.55   17.93\n",
      "  385   5702 719F 719F C   CA   5571 5563    2.25    1.49    0.76   22.04    1.49    0.76   22.04\n",
      "  386   5703 719F 720L C   N    5571 5573    1.87    1.35    0.53   18.65    1.35    0.53   18.65\n",
      "  387   5708 720L 720L N   CA   5573 5574    2.05    1.43    0.62   20.50    1.43    0.62   20.50\n",
      "  388   5710 720L 720L C   CA   5579 5574    2.33    1.49    0.84   24.43    1.49    0.84   24.43\n",
      "  389   5711 720L 721Y C   N    5579 5581    1.94    1.35    0.59   20.92    1.35    0.59   20.92\n",
      "  390   5721 721Y 721Y N   CA   5581 5582    2.14    1.43    0.71   23.42    1.43    0.71   23.42\n",
      "  391   5723 721Y 721Y C   CA   5591 5582    2.54    1.49    1.05   30.59    1.49    1.05   30.59\n",
      "  392   5724 721Y 722F C   N    5591 5593    2.08    1.35    0.74   26.19    1.35    0.74   26.19\n",
      "  393   5733 722F 722F N   CA   5593 5594    2.32    1.43    0.89   29.27    1.43    0.89   29.27\n",
      "  394   5735 722F 722F C   CA   5602 5594    2.65    1.49    1.16   33.59    1.49    1.16   33.59\n",
      "  395   5736 722F 723Y C   N    5602 5604    2.15    1.35    0.81   28.71    1.35    0.81   28.71\n",
      "  396   5737 723Y 723Y CB  CA   5606 5605    1.73    1.54    0.20    5.41    1.54    0.20    5.41\n",
      "  397   5738 723Y 723Y CG  CB   5607 5606    1.71    1.49    0.22    6.02    1.49    0.22    6.02\n",
      "  398   5746 723Y 723Y N   CA   5604 5605    2.40    1.43    0.97   32.02    1.43    0.97   32.02\n",
      "  399   5748 723Y 723Y C   CA   5614 5605    2.93    1.49    1.44   41.85    1.49    1.44   41.85\n",
      "  400   5749 723Y 724K C   N    5614 5616    2.34    1.35    0.99   35.23    1.35    0.99   35.23\n",
      "  401   5755 724K 724K N   CA   5616 5617    2.60    1.43    1.17   38.59    1.43    1.17   38.59\n",
      "  402   5757 724K 724K C   CA   5623 5617    3.01    1.49    1.52   44.21    1.49    1.52   44.21\n",
      "  403   5758 724K 725R C   N    5623 5625    2.41    1.35    1.06   37.70    1.35    1.06   37.70\n",
      "  404   5766 725R 725R N   CA   5625 5626    2.70    1.43    1.27   41.62    1.43    1.27   41.62\n",
      "  405   5768 725R 725R C   CA   5634 5626    3.13    1.49    1.64   47.79    1.49    1.64   47.79\n",
      "  406   5769 725R 726V C   N    5634 5636    2.48    1.35    1.14   40.33    1.35    1.14   40.33\n",
      "  407   5773 726V 726V N   CA   5636 5637    2.77    1.43    1.34   44.20    1.43    1.34   44.20\n",
      "  408   5775 726V 726V C   CA   5641 5637    3.25    1.49    1.76   51.15    1.49    1.76   51.15\n",
      "  409   5776 726V 727G C   N    5641 5643    2.57    1.35    1.22   43.40    1.35    1.22   43.40\n",
      "  410   5777 727G 727G N   CA   5643 5644    2.87    1.43    1.44   47.43    1.43    1.44   47.43\n",
      "  411   5779 727G 727G C   CA   5645 5644    3.36    1.49    1.87   54.28    1.49    1.87   54.28\n",
      "  412   5780 727G 728K C   N    5645 5647    2.63    1.35    1.29   45.66    1.35    1.29   45.66\n",
      "  413   5786 728K 728K N   CA   5647 5648    2.49    1.43    1.06   34.84    1.43    1.06   34.84\n",
      "  414   5787 728K 728K O   C    5655 5654    1.46    1.26    0.20    8.30    1.26    0.20    8.30\n",
      "  415   5788 728K 728K C   CA   5654 5648    1.98    1.52    0.46   11.87    1.52    0.46   11.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5824   4P   4P N   CA     25   26  131.16  108.20   22.96    5.22  108.20   22.96    5.22\n",
      "    2   5895  10E  10E N   CA     74   75  122.92  107.00   15.93    4.58  107.00   15.93    4.58\n",
      "    3   5907  11A  11A N   CA     83   84  123.73  107.00   16.73    4.81  107.00   16.73    4.81\n",
      "    4   5914  12C  12C N   CA     88   89  124.54  107.00   17.54    5.04  107.00   17.54    5.04\n",
      "    5   5922  13A  13A N   CA     94   95  125.08  107.00   18.08    5.20  107.00   18.08    5.20\n",
      "    6   5929  14H  14H N   CA     99  100  125.31  107.00   18.31    5.26  107.00   18.31    5.26\n",
      "    7   5944  15C  15C N   CA    109  110  127.26  107.00   20.26    5.83  107.00   20.26    5.83\n",
      "    8   5952  16W  16W N   CA    115  116  127.36  107.00   20.36    5.85  107.00   20.36    5.85\n",
      "    9   5975  17D  17D N   CA    129  130  131.23  107.00   24.23    6.97  107.00   24.23    6.97\n",
      "   10   5986  18E  18E N   CA    137  138  131.75  107.00   24.75    7.12  107.00   24.75    7.12\n",
      "   11   5989  18E  18E CA  C     138  144  133.00  116.50   16.50    4.74  116.50   16.50    4.74\n",
      "   12   5998  19S  19S N   CA    146  147  132.43  107.00   25.43    7.31  107.00   25.43    7.31\n",
      "   13   6001  19S  19S CA  C     147  150  133.72  116.50   17.22    4.95  116.50   17.22    4.95\n",
      "   14   6006  20V  20V N   CA    152  153  132.94  107.00   25.94    7.46  107.00   25.94    7.46\n",
      "   15   6010  20V  20V CA  C     153  157  134.61  116.50   18.11    5.21  116.50   18.11    5.21\n",
      "   16   6013  20V  21M C   N     157  159  140.05  120.00   20.05    4.56  120.00   20.05    4.56\n",
      "   17   6016  21M  21M N   CA    159  160  134.33  107.00   27.33    7.86  107.00   27.33    7.86\n",
      "   18   6019  21M  21M CA  C     160  165  135.28  116.50   18.78    5.40  116.50   18.78    5.40\n",
      "   19   6023  21M  22L C   N     165  167  139.93  120.00   19.93    4.53  120.00   19.93    4.53\n",
      "   20   6026  22L  22L N   CA    167  168  135.03  107.00   28.03    8.06  107.00   28.03    8.06\n",
      "   21   6029  22L  22L CA  C     168  173  135.23  116.50   18.73    5.38  116.50   18.73    5.38\n",
      "   22   6037  23V  23V N   CA    175  176  138.03  107.00   31.03    8.92  107.00   31.03    8.92\n",
      "   23   6044  23V  24P C   N     180  182  148.19  117.00   31.19    7.77  117.00   31.19    7.77\n",
      "   24   6047  24P  24P N   CA    182  183  148.00  108.20   39.80    9.04  108.20   39.80    9.04\n",
      "   25   6051  24P  24P CA  C     183  187  137.32  116.50   20.82    5.99  116.50   20.82    5.99\n",
      "   26   6056  24P  25N C   N     187  189  142.15  120.00   22.15    5.03  120.00   22.15    5.03\n",
      "   27   6059  25N  25N N   CA    189  190  138.17  107.00   31.17    8.96  107.00   31.17    8.96\n",
      "   28   6062  25N  25N CA  C     190  195  137.89  116.50   21.39    6.15  116.50   21.39    6.15\n",
      "   29   6067  25N  26H C   N     195  197  142.44  120.00   22.44    5.10  120.00   22.44    5.10\n",
      "   30   6070  26H  26H N   CA    197  198  138.02  107.00   31.02    8.92  107.00   31.02    8.92\n",
      "   31   6072  26H  26H CA  C     198  205  138.54  116.50   22.04    6.34  116.50   22.04    6.34\n",
      "   32   6082  26H  27R C   N     205  207  142.73  120.00   22.73    5.16  120.00   22.73    5.16\n",
      "   33   6085  27R  27R N   CA    207  208  140.02  107.00   33.02    9.50  107.00   33.02    9.50\n",
      "   34   6088  27R  27R CA  C     208  216  139.62  116.50   23.12    6.65  116.50   23.12    6.65\n",
      "   35   6096  27R  28S C   N     216  218  144.06  120.00   24.06    5.47  120.00   24.06    5.47\n",
      "   36   6099  28S  28S N   CA    218  219  140.37  107.00   33.37    9.59  107.00   33.37    9.59\n",
      "   37   6102  28S  28S CA  C     219  222  139.96  116.50   23.46    6.74  116.50   23.46    6.74\n",
      "   38   6104  28S  29A C   N     222  224  144.62  120.00   24.62    5.60  120.00   24.62    5.60\n",
      "   39   6107  29A  29A N   CA    224  225  140.93  107.00   33.93    9.76  107.00   33.93    9.76\n",
      "   40   6109  29A  29A CA  C     225  227  140.32  116.50   23.83    6.85  116.50   23.83    6.85\n",
      "   41   6111  29A  30V C   N     227  229  144.76  120.00   24.76    5.63  120.00   24.76    5.63\n",
      "   42   6114  30V  30V N   CA    229  230  141.15  107.00   34.15    9.82  107.00   34.15    9.82\n",
      "   43   6118  30V  30V CA  C     230  234  141.00  116.50   24.50    7.04  116.50   24.50    7.04\n",
      "   44   6121  30V  31H C   N     234  236  145.43  120.00   25.43    5.78  120.00   25.43    5.78\n",
      "   45   6124  31H  31H N   CA    236  237  141.99  107.00   34.99   10.06  107.00   34.99   10.06\n",
      "   46   6126  31H  31H CA  C     237  244  141.86  116.50   25.36    7.29  116.50   25.36    7.29\n",
      "   47   6136  31H  32K C   N     244  246  145.53  120.00   25.53    5.80  120.00   25.53    5.80\n",
      "   48   6139  32K  32K N   CA    246  247  143.04  107.00   36.04   10.36  107.00   36.04   10.36\n",
      "   49   6142  32K  32K CA  C     247  253  142.50  116.50   26.00    7.47  116.50   26.00    7.47\n",
      "   50   6147  32K  33E C   N     253  255  146.91  120.00   26.91    6.12  120.00   26.91    6.12\n",
      "   51   6150  33E  33E N   CA    255  256  143.46  107.00   36.46   10.48  107.00   36.46   10.48\n",
      "   52   6153  33E  33E CA  C     256  262  143.29  116.50   26.79    7.70  116.50   26.79    7.70\n",
      "   53   6159  33E  34E C   N     262  264  146.93  120.00   26.93    6.12  120.00   26.93    6.12\n",
      "   54   6162  34E  34E N   CA    264  265  144.26  107.00   37.26   10.71  107.00   37.26   10.71\n",
      "   55   6165  34E  34E CA  C     265  271  143.81  116.50   27.31    7.85  116.50   27.31    7.85\n",
      "   56   6171  34E  35C C   N     271  273  147.96  120.00   27.96    6.35  120.00   27.96    6.35\n",
      "   57   6174  35C  35C N   CA    273  274  145.04  107.00   38.04   10.94  107.00   38.04   10.94\n",
      "   58   6177  35C  35C CA  C     274  277  144.22  116.50   27.72    7.97  116.50   27.72    7.97\n",
      "   59   6179  35C  36A C   N     277  279  148.34  120.00   28.34    6.44  120.00   28.34    6.44\n",
      "   60   6182  36A  36A N   CA    279  280  145.43  107.00   38.43   11.05  107.00   38.43   11.05\n",
      "   61   6184  36A  36A CA  C     280  282  144.32  116.50   27.83    8.00  116.50   27.83    8.00\n",
      "   62   6186  36A  37Y C   N     282  284  148.79  120.00   28.79    6.54  120.00   28.79    6.54\n",
      "   63   6189  37Y  37Y N   CA    284  285  145.55  107.00   38.55   11.09  107.00   38.55   11.09\n",
      "   64   6192  37Y  37Y CA  C     285  294  145.26  116.50   28.76    8.27  116.50   28.76    8.27\n",
      "   65   6204  37Y  38C C   N     294  296  149.26  120.00   29.26    6.65  120.00   29.26    6.65\n",
      "   66   6207  38C  38C N   CA    296  297  146.55  107.00   39.55   11.37  107.00   39.55   11.37\n",
      "   67   6210  38C  38C CA  C     297  300  145.19  116.50   28.69    8.25  116.50   28.69    8.25\n",
      "   68   6212  38C  39C C   N     300  302  149.80  120.00   29.80    6.77  120.00   29.80    6.77\n",
      "   69   6215  39C  39C N   CA    302  303  146.93  107.00   39.93   11.48  107.00   39.93   11.48\n",
      "   70   6218  39C  39C CA  C     303  306  145.65  116.50   29.15    8.38  116.50   29.15    8.38\n",
      "   71   6220  39C  40R C   N     306  308  149.52  120.00   29.52    6.71  120.00   29.52    6.71\n",
      "   72   6223  40R  40R N   CA    308  309  146.77  107.00   39.77   11.44  107.00   39.77   11.44\n",
      "   73   6226  40R  40R CA  C     309  317  146.19  116.50   29.69    8.54  116.50   29.69    8.54\n",
      "   74   6234  40R  41T C   N     317  319  150.25  120.00   30.25    6.88  120.00   30.25    6.88\n",
      "   75   6237  41T  41T N   CA    319  320  147.69  107.00   40.69   11.70  107.00   40.69   11.70\n",
      "   76   6241  41T  41T CA  C     320  324  146.79  116.50   30.29    8.71  116.50   30.29    8.71\n",
      "   77   6244  41T  42C C   N     324  326  150.65  120.00   30.65    6.97  120.00   30.65    6.97\n",
      "   78   6247  42C  42C N   CA    326  327  148.02  107.00   41.02   11.79  107.00   41.02   11.79\n",
      "   79   6250  42C  42C CA  C     327  330  147.18  116.50   30.68    8.82  116.50   30.68    8.82\n",
      "   80   6252  42C  43R C   N     330  332  150.71  120.00   30.71    6.98  120.00   30.71    6.98\n",
      "   81   6255  43R  43R N   CA    332  333  148.50  107.00   41.50   11.93  107.00   41.50   11.93\n",
      "   82   6258  43R  43R CA  C     333  341  147.86  116.50   31.36    9.02  116.50   31.36    9.02\n",
      "   83   6266  43R  44H C   N     341  343  151.91  120.00   31.91    7.25  120.00   31.91    7.25\n",
      "   84   6269  44H  44H N   CA    343  344  149.67  107.00   42.67   12.27  107.00   42.67   12.27\n",
      "   85   6271  44H  44H CA  C     344  351  148.47  116.50   31.97    9.19  116.50   31.97    9.19\n",
      "   86   6281  44H  45E C   N     351  353  152.52  120.00   32.52    7.39  120.00   32.52    7.39\n",
      "   87   6284  45E  45E N   CA    353  354  150.67  107.00   43.67   12.56  107.00   43.67   12.56\n",
      "   88   6287  45E  45E CA  C     354  360  148.83  116.50   32.33    9.30  116.50   32.33    9.30\n",
      "   89   6293  45E  46M C   N     360  362  153.25  120.00   33.25    7.56  120.00   33.25    7.56\n",
      "   90   6296  46M  46M N   CA    362  363  151.31  107.00   44.31   12.74  107.00   44.31   12.74\n",
      "   91   6299  46M  46M CA  C     363  368  149.22  116.50   32.72    9.41  116.50   32.72    9.41\n",
      "   92   6303  46M  47G C   N     368  370  153.06  120.00   33.06    7.51  120.00   33.06    7.51\n",
      "   93   6305  47G  47G N   CA    370  371  151.09  107.00   44.09   12.68  107.00   44.09   12.68\n",
      "   94   6307  47G  47G CA  C     371  372  135.83  116.50   19.33    5.56  116.50   19.33    5.56\n",
      "   95   6308  47G  48T C   N     372  374  146.96  120.00   26.96    6.13  120.00   26.96    6.13\n",
      "   96   6318  48T  49L C   N     379  381  141.24  120.00   21.24    4.83  120.00   21.24    4.83\n",
      "   97   6902 104K 104K N   CA    799  800  124.86  107.00   17.86    5.14  107.00   17.86    5.14\n",
      "   98   7184 130Y 131Q C   N    1001 1003  143.91  120.00   23.91    5.43  120.00   23.91    5.43\n",
      "   99   7187 131Q 131Q N   CA   1003 1004  128.23  107.00   21.23    6.10  107.00   21.23    6.10\n",
      "  100   8269 236A 236A N   CA   1777 1778  122.79  107.00   15.79    4.54  107.00   15.79    4.54\n",
      "  101   8561 261I 261I N   CA   1984 1985  131.46  107.00   24.46    7.03  107.00   24.46    7.03\n",
      "  102   8569 261I 262D C   N    1990 1992  150.13  120.00   30.13    6.85  120.00   30.13    6.85\n",
      "  103   8572 262D 262D N   CA   1992 1993  130.00  107.00   23.00    6.61  107.00   23.00    6.61\n",
      "  104   8575 262D 262D CA  C    1993 1998  141.30  116.50   24.80    7.13  116.50   24.80    7.13\n",
      "  105   8580 262D 263I C   N    1998 2000  145.35  120.00   25.35    5.76  120.00   25.35    5.76\n",
      "  106   8583 263I 263I N   CA   2000 2001  138.27  107.00   31.27    8.99  107.00   31.27    8.99\n",
      "  107   8587 263I 263I CA  C    2001 2006  135.25  116.50   18.75    5.39  116.50   18.75    5.39\n",
      "  108   8591 263I 264Q C   N    2006 2008  142.20  120.00   22.20    5.04  120.00   22.20    5.04\n",
      "  109   8594 264Q 264Q N   CA   2008 2009  137.41  107.00   30.41    8.74  107.00   30.41    8.74\n",
      "  110   8597 264Q 264Q CA  C    2009 2015  135.73  116.50   19.23    5.53  116.50   19.23    5.53\n",
      "  111   8603 264Q 265T C   N    2015 2017  139.82  120.00   19.82    4.50  120.00   19.82    4.50\n",
      "  112   8606 265T 265T N   CA   2017 2018  135.49  107.00   28.49    8.19  107.00   28.49    8.19\n",
      "  113   8610 265T 265T CA  C    2018 2022  132.94  116.50   16.44    4.73  116.50   16.44    4.73\n",
      "  114   8613 265T 266A C   N    2022 2024  140.51  120.00   20.51    4.66  120.00   20.51    4.66\n",
      "  115   8616 266A 266A N   CA   2024 2025  134.09  107.00   27.09    7.79  107.00   27.09    7.79\n",
      "  116   8618 266A 266A CA  C    2025 2027  133.17  116.50   16.67    4.79  116.50   16.67    4.79\n",
      "  117   8623 267K 267K N   CA   2029 2030  131.85  107.00   24.85    7.14  107.00   24.85    7.14\n",
      "  118   8626 267K 267K CA  C    2030 2036  133.03  116.50   16.53    4.75  116.50   16.53    4.75\n",
      "  119   8634 268K 268K N   CA   2038 2039  131.56  107.00   24.56    7.06  107.00   24.56    7.06\n",
      "  120   8645 269T 269T N   CA   2047 2048  130.57  107.00   23.57    6.78  107.00   23.57    6.78\n",
      "  121   8655 270A 270A N   CA   2054 2055  130.72  107.00   23.72    6.82  107.00   23.72    6.82\n",
      "  122   8662 271K 271K N   CA   2059 2060  129.60  107.00   22.60    6.50  107.00   22.60    6.50\n",
      "  123   8673 272T 272T N   CA   2068 2069  129.40  107.00   22.40    6.44  107.00   22.40    6.44\n",
      "  124   8683 273L 273L N   CA   2075 2076  128.93  107.00   21.93    6.30  107.00   21.93    6.30\n",
      "  125   8693 274G 274G N   CA   2083 2084  130.48  107.00   23.48    6.75  107.00   23.48    6.75\n",
      "  126   8699 275E 275E N   CA   2087 2088  128.13  107.00   21.13    6.08  107.00   21.13    6.08\n",
      "  127   8711 276M 276M N   CA   2096 2097  126.29  107.00   19.29    5.55  107.00   19.29    5.55\n",
      "  128   8721 277Q 277Q N   CA   2104 2105  125.66  107.00   18.66    5.37  107.00   18.66    5.37\n",
      "  129   8733 278Y 278Y N   CA   2113 2114  123.40  107.00   16.40    4.72  107.00   16.40    4.72\n",
      "  130   8751 279D 279D N   CA   2125 2126  123.21  107.00   16.21    4.66  107.00   16.21    4.66\n",
      "  131  11802 557V 557V N   CA   4331 4332  126.39  107.00   19.39    5.57  107.00   19.39    5.57\n",
      "  132  11812 558D 558D N   CA   4338 4339  127.88  107.00   20.88    6.00  107.00   20.88    6.00\n",
      "  133  11823 559V 559V N   CA   4346 4347  129.20  107.00   22.20    6.38  107.00   22.20    6.38\n",
      "  134  11830 559V 560P C   N    4351 4353  139.27  117.00   22.27    5.55  117.00   22.27    5.55\n",
      "  135  11833 560P 560P N   CA   4353 4354  138.37  108.20   30.18    6.86  108.20   30.18    6.86\n",
      "  136  11845 561L 561L N   CA   4360 4361  130.59  107.00   23.59    6.78  107.00   23.59    6.78\n",
      "  137  11848 561L 561L CA  C    4361 4366  133.12  116.50   16.62    4.78  116.50   16.62    4.78\n",
      "  138  11856 562E 562E N   CA   4368 4369  134.24  107.00   27.24    7.83  107.00   27.24    7.83\n",
      "  139  11859 562E 562E CA  C    4369 4375  136.61  116.50   20.11    5.78  116.50   20.11    5.78\n",
      "  140  11865 562E 563L C   N    4375 4377  141.27  120.00   21.27    4.83  120.00   21.27    4.83\n",
      "  141  11868 563L 563L N   CA   4377 4378  135.23  107.00   28.23    8.12  107.00   28.23    8.12\n",
      "  142  11871 563L 563L CA  C    4378 4383  138.77  116.50   22.27    6.40  116.50   22.27    6.40\n",
      "  143  11876 563L 564D C   N    4383 4385  142.24  120.00   22.24    5.05  120.00   22.24    5.05\n",
      "  144  11879 564D 564D N   CA   4385 4386  139.15  107.00   32.15    9.25  107.00   32.15    9.25\n",
      "  145  11882 564D 564D CA  C    4386 4391  139.40  116.50   22.90    6.58  116.50   22.90    6.58\n",
      "  146  11887 564D 565L C   N    4391 4393  144.62  120.00   24.62    5.59  120.00   24.62    5.59\n",
      "  147  11890 565L 565L N   CA   4393 4394  139.98  107.00   32.98    9.48  107.00   32.98    9.48\n",
      "  148  11893 565L 565L CA  C    4394 4399  141.18  116.50   24.68    7.10  116.50   24.68    7.10\n",
      "  149  11898 565L 566E C   N    4399 4401  144.87  120.00   24.87    5.65  120.00   24.87    5.65\n",
      "  150  11901 566E 566E N   CA   4401 4402  141.10  107.00   34.10    9.80  107.00   34.10    9.80\n",
      "  151  11904 566E 566E CA  C    4402 4408  142.43  116.50   25.93    7.46  116.50   25.93    7.46\n",
      "  152  11910 566E 567F C   N    4408 4410  146.57  120.00   26.57    6.04  120.00   26.57    6.04\n",
      "  153  11913 567F 567F N   CA   4410 4411  142.61  107.00   35.61   10.24  107.00   35.61   10.24\n",
      "  154  11916 567F 567F CA  C    4411 4419  143.57  116.50   27.07    7.78  116.50   27.07    7.78\n",
      "  155  11926 567F 568L C   N    4419 4421  146.52  120.00   26.52    6.03  120.00   26.52    6.03\n",
      "  156  11929 568L 568L N   CA   4421 4422  144.28  107.00   37.28   10.72  107.00   37.28   10.72\n",
      "  157  11932 568L 568L CA  C    4422 4427  145.18  116.50   28.68    8.25  116.50   28.68    8.25\n",
      "  158  11937 568L 569R C   N    4427 4429  148.43  120.00   28.43    6.46  120.00   28.43    6.46\n",
      "  159  11940 569R 569R N   CA   4429 4430  146.03  107.00   39.03   11.22  107.00   39.03   11.22\n",
      "  160  11943 569R 569R CA  C    4430 4438  146.11  116.50   29.61    8.51  116.50   29.61    8.51\n",
      "  161  11951 569R 570G C   N    4438 4440  149.54  120.00   29.54    6.71  120.00   29.54    6.71\n",
      "  162  11953 570G 570G N   CA   4440 4441  148.00  107.00   41.00   11.79  107.00   41.00   11.79\n",
      "  163  11955 570G 570G CA  C    4441 4442  145.32  116.50   28.82    8.29  116.50   28.82    8.29\n",
      "  164  11956 570G 571K C   N    4442 4444  148.85  120.00   28.85    6.56  120.00   28.85    6.56\n",
      "  165  11959 571K 571K N   CA   4444 4445  145.69  107.00   38.69   11.12  107.00   38.69   11.12\n",
      "  166  11962 571K 571K CA  C    4445 4451  144.77  116.50   28.27    8.13  116.50   28.27    8.13\n",
      "  167  11967 571K 572G C   N    4451 4453  148.94  120.00   28.94    6.58  120.00   28.94    6.58\n",
      "  168  11969 572G 572G N   CA   4453 4454  146.68  107.00   39.68   11.41  107.00   39.68   11.41\n",
      "  169  11971 572G 572G CA  C    4454 4455  144.68  116.50   28.18    8.10  116.50   28.18    8.10\n",
      "  170  11972 572G 573L C   N    4455 4457  149.07  120.00   29.07    6.61  120.00   29.07    6.61\n",
      "  171  11975 573L 573L N   CA   4457 4458  145.41  107.00   38.41   11.04  107.00   38.41   11.04\n",
      "  172  11978 573L 573L CA  C    4458 4463  142.60  116.50   26.10    7.50  116.50   26.10    7.50\n",
      "  173  11983 573L 574Q C   N    4463 4465  152.97  120.00   32.97    7.49  120.00   32.97    7.49\n",
      "  174  11986 574Q 574Q N   CA   4465 4466  147.36  107.00   40.36   11.61  107.00   40.36   11.61\n",
      "  175  11989 574Q 574Q CA  C    4466 4472  146.42  116.50   29.93    8.60  116.50   29.93    8.60\n",
      "  176  11995 574Q 575S C   N    4472 4474  149.50  120.00   29.50    6.70  120.00   29.50    6.70\n",
      "  177  11998 575S 575S N   CA   4474 4475  147.75  107.00   40.75   11.72  107.00   40.75   11.72\n",
      "  178  12001 575S 575S CA  C    4475 4478  148.06  116.50   31.56    9.07  116.50   31.56    9.07\n",
      "  179  12003 575S 576D C   N    4478 4480  150.03  120.00   30.03    6.82  120.00   30.03    6.82\n",
      "  180  12006 576D 576D N   CA   4480 4481  149.10  107.00   42.10   12.11  107.00   42.10   12.11\n",
      "  181  12009 576D 576D CA  C    4481 4486  147.28  116.50   30.78    8.85  116.50   30.78    8.85\n",
      "  182  12014 576D 577E C   N    4486 4488  152.64  120.00   32.64    7.42  120.00   32.64    7.42\n",
      "  183  12017 577E 577E N   CA   4488 4489  152.05  107.00   45.05   12.95  107.00   45.05   12.95\n",
      "  184  12020 577E 577E CA  C    4489 4495  147.79  116.50   31.29    9.00  116.50   31.29    9.00\n",
      "  185  12026 577E 578V C   N    4495 4497  153.78  120.00   33.78    7.68  120.00   33.78    7.68\n",
      "  186  12029 578V 578V N   CA   4497 4498  151.90  107.00   44.90   12.91  107.00   44.90   12.91\n",
      "  187  12033 578V 578V CA  C    4498 4502  150.95  116.50   34.45    9.91  116.50   34.45    9.91\n",
      "  188  12036 578V 579S C   N    4502 4504  151.86  120.00   31.86    7.24  120.00   31.86    7.24\n",
      "  189  12039 579S 579S N   CA   4504 4505  153.99  107.00   46.99   13.51  107.00   46.99   13.51\n",
      "  190  12042 579S 579S CA  C    4505 4508  154.69  116.50   38.19   10.98  116.50   38.19   10.98\n",
      "  191  12044 579S 580M C   N    4508 4510  155.19  120.00   35.19    8.00  120.00   35.19    8.00\n",
      "  192  12047 580M 580M N   CA   4510 4511  160.37  107.00   53.37   15.35  107.00   53.37   15.35\n",
      "  193  12050 580M 580M CA  C    4511 4516  161.92  112.50   49.42    7.10  112.50   49.42    7.10\n",
      "  194  12054 580M 581P C   N    4516 4518  162.38  117.00   45.38   11.30  117.00   45.38   11.30\n",
      "  195  12057 581P 581P N   CA   4518 4519  167.90  108.20   59.70   13.57  108.20   59.70   13.57\n",
      "  196  12061 581P 581P CA  C    4519 4523  161.01  116.50   44.51   12.80  116.50   44.51   12.80\n",
      "  197  12066 581P 582E C   N    4523 4525  165.27  120.00   45.27   10.29  120.00   45.27   10.29\n",
      "  198  12069 582E 582E N   CA   4525 4526  163.95  107.00   56.95   16.37  107.00   56.95   16.37\n",
      "  199  12072 582E 582E CA  C    4526 4532  163.38  116.50   46.88   13.48  116.50   46.88   13.48\n",
      "  200  12078 582E 583A C   N    4532 4534  167.74  120.00   47.74   10.85  120.00   47.74   10.85\n",
      "  201  12081 583A 583A N   CA   4534 4535  168.42  107.00   61.42   17.66  107.00   61.42   17.66\n",
      "  202  12083 583A 583A CA  C    4535 4537  164.41  116.50   47.91   13.78  116.50   47.91   13.78\n",
      "  203  12085 583A 584Q C   N    4537 4539  170.76  120.00   50.76   11.53  120.00   50.76   11.53\n",
      "  204  12088 584Q 584Q N   CA   4539 4540  168.79  107.00   61.79   17.77  107.00   61.79   17.77\n",
      "  205  12091 584Q 584Q CA  C    4540 4546  167.18  116.50   50.68   14.57  116.50   50.68   14.57\n",
      "  206  12097 584Q 585E C   N    4546 4548  168.21  120.00   48.20   10.95  120.00   48.20   10.95\n",
      "  207  12100 585E 585E N   CA   4548 4549  170.76  107.00   63.76   18.33  107.00   63.76   18.33\n",
      "  208  12103 585E 585E CA  C    4549 4555  169.54  116.50   53.04   15.25  116.50   53.04   15.25\n",
      "  209  12109 585E 586D C   N    4555 4557  177.42  120.00   57.42   13.05  120.00   57.42   13.05\n",
      "  210  12112 586D 586D N   CA   4557 4558  171.57  107.00   64.57   18.57  107.00   64.57   18.57\n",
      "  211  12113 586D 586D CA  CB   4558 4559  130.21  108.00   22.21    5.15  108.00   22.21    5.15\n",
      "  212  12115 586D 586D CA  C    4558 4563  170.99  116.50   54.49   15.67  116.50   54.49   15.67\n",
      "  213  12120 586D 587L C   N    4563 4565  172.47  120.00   52.47   11.92  120.00   52.47   11.92\n",
      "  214  12123 587L 587L N   CA   4565 4566  173.10  107.00   66.10   19.01  107.00   66.10   19.01\n",
      "  215  12126 587L 587L CA  C    4566 4571  175.43  112.50   62.93    9.04  112.50   62.93    9.04\n",
      "  216  12131 587L 588P C   N    4571 4573  178.36  117.00   61.36   15.28  117.00   61.36   15.28\n",
      "  217  12134 588P 588P N   CA   4573 4574  178.11  108.20   69.91   15.89  108.20   69.91   15.89\n",
      "  218  12138 588P 588P CA  C    4574 4578  175.53  116.50   59.03   16.97  116.50   59.03   16.97\n",
      "  219  12141 588P 588P CB  CG   4576 4577  133.45  108.50   24.95    6.71  108.50   24.95    6.71\n",
      "  220  12143 588P 589H C   N    4578 4580  173.82  120.00   53.82   12.23  120.00   53.82   12.23\n",
      "  221  12146 589H 589H N   CA   4580 4581  173.08  107.00   66.08   19.00  107.00   66.08   19.00\n",
      "  222  12148 589H 589H CA  C    4581 4588  173.75  116.50   57.26   16.46  116.50   57.26   16.46\n",
      "  223  12158 589H 590K C   N    4588 4590  174.04  120.00   54.04   12.28  120.00   54.04   12.28\n",
      "  224  12161 590K 590K N   CA   4590 4591  175.22  107.00   68.22   19.61  107.00   68.22   19.61\n",
      "  225  12164 590K 590K CA  C    4591 4597  175.61  112.50   63.11    9.07  112.50   63.11    9.07\n",
      "  226  12169 590K 591P C   N    4597 4599  176.77  117.00   59.77   14.88  117.00   59.77   14.88\n",
      "  227  12172 591P 591P N   CA   4599 4600  179.11  108.20   70.91   16.12  108.20   70.91   16.12\n",
      "  228  12176 591P 591P CA  C    4600 4604  178.26  116.50   61.76   17.76  116.50   61.76   17.76\n",
      "  229  12179 591P 591P CB  CG   4602 4603  135.17  108.50   26.67    7.17  108.50   26.67    7.17\n",
      "  230  12181 591P 592T C   N    4604 4606  177.62  120.00   57.62   13.10  120.00   57.62   13.10\n",
      "  231  12184 592T 592T N   CA   4606 4607  176.36  107.00   69.36   19.94  107.00   69.36   19.94\n",
      "  232  12188 592T 592T CA  C    4607 4611  175.27  116.50   58.77   16.90  116.50   58.77   16.90\n",
      "  233  12191 592T 593S C   N    4611 4613  176.67  120.00   56.67   12.88  120.00   56.67   12.88\n",
      "  234  12194 593S 593S N   CA   4613 4614  178.54  107.00   71.54   20.57  107.00   71.54   20.57\n",
      "  235  12197 593S 593S CA  C    4614 4617  176.96  116.50   60.46   17.39  116.50   60.46   17.39\n",
      "  236  12199 593S 594S C   N    4617 4619  176.74  120.00   56.74   12.89  120.00   56.74   12.89\n",
      "  237  12202 594S 594S N   CA   4619 4620  176.99  107.00   69.99   20.12  107.00   69.99   20.12\n",
      "  238  12205 594S 594S CA  C    4620 4623  176.31  116.50   59.81   17.20  116.50   59.81   17.20\n",
      "  239  12207 594S 595S C   N    4623 4625  178.14  120.00   58.13   13.21  120.00   58.13   13.21\n",
      "  240  12210 595S 595S N   CA   4625 4626  178.14  107.00   71.14   20.46  107.00   71.14   20.46\n",
      "  241  12213 595S 595S CA  C    4626 4629  176.84  116.50   60.34   17.35  116.50   60.34   17.35\n",
      "  242  12215 595S 596M C   N    4629 4631  179.32  120.00   59.32   13.48  120.00   59.32   13.48\n",
      "  243  12218 596M 596M N   CA   4631 4632  177.39  107.00   70.39   20.24  107.00   70.39   20.24\n",
      "  244  12221 596M 596M CA  C    4632 4637  175.76  116.50   59.26   17.04  116.50   59.26   17.04\n",
      "  245  12225 596M 597K C   N    4637 4639  176.59  120.00   56.59   12.86  120.00   56.59   12.86\n",
      "  246  12228 597K 597K N   CA   4639 4640  177.18  107.00   70.18   20.18  107.00   70.18   20.18\n",
      "  247  12231 597K 597K CA  C    4640 4646  176.79  116.50   60.29   17.34  116.50   60.29   17.34\n",
      "  248  12236 597K 598T C   N    4646 4648  177.68  120.00   57.68   13.11  120.00   57.68   13.11\n",
      "  249  12239 598T 598T N   CA   4648 4649  178.39  107.00   71.39   20.53  107.00   71.39   20.53\n",
      "  250  12243 598T 598T CA  C    4649 4653  178.22  116.50   61.72   17.75  116.50   61.72   17.75\n",
      "  251  12246 598T 599V C   N    4653 4655  178.82  120.00   58.82   13.37  120.00   58.82   13.37\n",
      "  252  12249 599V 599V N   CA   4655 4656  178.36  107.00   71.36   20.52  107.00   71.36   20.52\n",
      "  253  12253 599V 599V CA  C    4656 4660  177.61  116.50   61.11   17.57  116.50   61.11   17.57\n",
      "  254  12256 599V 600D C   N    4660 4662  179.30  120.00   59.30   13.48  120.00   59.30   13.48\n",
      "  255  12259 600D 600D N   CA   4662 4663  178.81  107.00   71.81   20.65  107.00   71.81   20.65\n",
      "  256  12262 600D 600D CA  C    4663 4668  178.13  116.50   61.63   17.72  116.50   61.63   17.72\n",
      "  257  12265 600D 600D OD2 CG   4667 4665  142.24  124.00   18.24    5.86  124.00   18.24    5.86\n",
      "  258  12267 600D 601E C   N    4668 4670  177.28  120.00   57.28   13.02  120.00   57.28   13.02\n",
      "  259  12270 601E 601E N   CA   4670 4671  176.42  107.00   69.42   19.96  107.00   69.42   19.96\n",
      "  260  12273 601E 601E CA  C    4671 4677  176.67  116.50   60.17   17.30  116.50   60.17   17.30\n",
      "  261  12279 601E 602E C   N    4677 4679  178.25  120.00   58.25   13.24  120.00   58.25   13.24\n",
      "  262  12282 602E 602E N   CA   4679 4680  178.32  107.00   71.32   20.51  107.00   71.32   20.51\n",
      "  263  12285 602E 602E CA  C    4680 4686  177.13  116.50   60.63   17.43  116.50   60.63   17.43\n",
      "  264  12291 602E 603A C   N    4686 4688  178.34  120.00   58.34   13.26  120.00   58.34   13.26\n",
      "  265  12294 603A 603A N   CA   4688 4689  178.79  107.00   71.79   20.64  107.00   71.79   20.64\n",
      "  266  12296 603A 603A CA  C    4689 4691  176.93  116.50   60.43   17.37  116.50   60.43   17.37\n",
      "  267  12298 603A 604L C   N    4691 4693  177.63  120.00   57.63   13.10  120.00   57.63   13.10\n",
      "  268  12301 604L 604L N   CA   4693 4694  179.63  107.00   72.63   20.88  107.00   72.63   20.88\n",
      "  269  12304 604L 604L CA  C    4694 4699  177.72  116.50   61.22   17.60  116.50   61.22   17.60\n",
      "  270  12309 604L 605A C   N    4699 4701  177.64  120.00   57.64   13.10  120.00   57.64   13.10\n",
      "  271  12312 605A 605A N   CA   4701 4702  177.57  107.00   70.57   20.29  107.00   70.57   20.29\n",
      "  272  12314 605A 605A CA  C    4702 4704  178.36  116.50   61.86   17.79  116.50   61.86   17.79\n",
      "  273  12316 605A 606M C   N    4704 4706  179.04  120.00   59.04   13.42  120.00   59.04   13.42\n",
      "  274  12319 606M 606M N   CA   4706 4707  173.54  107.00   66.54   19.13  107.00   66.54   19.13\n",
      "  275  12321 606M 606M CA  C    4707 4712  149.53  121.00   28.53    8.20  121.00   28.53    8.20\n",
      "  276  12322 606M 606M CA  C    4707 4712  162.28  116.50   45.78   13.16  116.50   45.78   13.16\n",
      "  277  12326 606M 607L C   N    4712 4714  155.59  120.00   35.59    8.09  120.00   35.59    8.09\n",
      "  278  12329 607L 607L N   CA   4714 4715  133.53  107.00   26.53    7.63  107.00   26.53    7.63\n",
      "  279  12409 615T 615T N   CA   4774 4775  126.29  107.00   19.29    5.55  107.00   19.29    5.55\n",
      "  280  12450 619Y 619Y N   CA   4804 4805  125.78  107.00   18.78    5.40  107.00   18.78    5.40\n",
      "  281  12468 620A 620A N   CA   4816 4817  123.08  107.00   16.08    4.62  107.00   16.08    4.62\n",
      "  282  13300 700L 700L N   CA   5414 5415  123.96  107.00   16.96    4.88  107.00   16.96    4.88\n",
      "  283  13322 702F 702F N   CA   5430 5431  124.33  107.00   17.33    4.98  107.00   17.33    4.98\n",
      "  284  13338 703N 703N N   CA   5441 5442  126.77  107.00   19.77    5.68  107.00   19.77    5.68\n",
      "  285  13349 704D 704D N   CA   5449 5450  128.66  107.00   21.66    6.23  107.00   21.66    6.23\n",
      "  286  13360 705E 705E N   CA   5457 5458  129.25  107.00   22.25    6.40  107.00   22.25    6.40\n",
      "  287  13372 706K 706K N   CA   5466 5467  129.45  107.00   22.45    6.45  107.00   22.45    6.45\n",
      "  288  13375 706K 706K CA  C    5467 5473  135.36  116.50   18.86    5.42  116.50   18.86    5.42\n",
      "  289  13383 707V 707V N   CA   5475 5476  134.51  107.00   27.51    7.91  107.00   27.51    7.91\n",
      "  290  13387 707V 707V CA  C    5476 5480  133.75  116.50   17.25    4.96  116.50   17.25    4.96\n",
      "  291  13390 707V 708A C   N    5480 5482  139.82  120.00   19.82    4.50  120.00   19.82    4.50\n",
      "  292  13393 708A 708A N   CA   5482 5483  133.66  107.00   26.66    7.67  107.00   26.66    7.67\n",
      "  293  13395 708A 708A CA  C    5483 5485  133.47  116.50   16.97    4.88  116.50   16.97    4.88\n",
      "  294  13400 709E 709E N   CA   5487 5488  130.78  107.00   23.78    6.84  107.00   23.78    6.84\n",
      "  295  13403 709E 709E CA  C    5488 5494  135.16  116.50   18.66    5.36  116.50   18.66    5.36\n",
      "  296  13412 710S 710S N   CA   5496 5497  132.80  107.00   25.80    7.42  107.00   25.80    7.42\n",
      "  297  13415 710S 710S CA  C    5497 5500  132.40  116.50   15.90    4.57  116.50   15.90    4.57\n",
      "  298  13420 711L 711L N   CA   5502 5503  131.02  107.00   24.02    6.91  107.00   24.02    6.91\n",
      "  299  13428 711L 712N C   N    5508 5510  140.40  120.00   20.40    4.64  120.00   20.40    4.64\n",
      "  300  13431 712N 712N N   CA   5510 5511  140.11  107.00   33.11    9.52  107.00   33.11    9.52\n",
      "  301  13439 712N 713P C   N    5516 5518  153.45  117.00   36.45    9.08  117.00   36.45    9.08\n",
      "  302  13442 713P 713P N   CA   5518 5519  150.52  108.20   42.32    9.62  108.20   42.32    9.62\n",
      "  303  13451 713P 714P C   N    5523 5525  144.73  117.00   27.73    6.90  117.00   27.73    6.90\n",
      "  304  13454 714P 714P N   CA   5525 5526  143.66  108.20   35.46    8.06  108.20   35.46    8.06\n",
      "  305  13458 714P 714P CA  C    5526 5530  143.68  116.50   27.18    7.81  116.50   27.18    7.81\n",
      "  306  13463 714P 715F C   N    5530 5532  143.42  120.00   23.42    5.32  120.00   23.42    5.32\n",
      "  307  13466 715F 715F N   CA   5532 5533  141.49  107.00   34.49    9.92  107.00   34.49    9.92\n",
      "  308  13469 715F 715F CA  C    5533 5541  141.92  116.50   25.42    7.31  116.50   25.42    7.31\n",
      "  309  13479 715F 716S C   N    5541 5543  150.58  120.00   30.58    6.95  120.00   30.58    6.95\n",
      "  310  13482 716S 716S N   CA   5543 5544  146.43  107.00   39.43   11.34  107.00   39.43   11.34\n",
      "  311  13485 716S 716S CA  C    5544 5547  143.82  116.50   27.32    7.86  116.50   27.32    7.86\n",
      "  312  13487 716S 717M C   N    5547 5549  148.50  120.00   28.50    6.48  120.00   28.50    6.48\n",
      "  313  13490 717M 717M N   CA   5549 5550  149.42  107.00   42.42   12.20  107.00   42.42   12.20\n",
      "  314  13493 717M 717M CA  C    5550 5555  140.71  116.50   24.21    6.96  116.50   24.21    6.96\n",
      "  315  13497 717M 718A C   N    5555 5557  169.14  120.00   49.14   11.17  120.00   49.14   11.17\n",
      "  316  13500 718A 718A N   CA   5557 5558  156.56  107.00   49.56   14.25  107.00   49.56   14.25\n",
      "  317  13502 718A 718A CA  C    5558 5560  148.46  116.50   31.96    9.19  116.50   31.96    9.19\n",
      "  318  13504 718A 719F C   N    5560 5562  148.98  120.00   28.98    6.59  120.00   28.98    6.59\n",
      "  319  13507 719F 719F N   CA   5562 5563  146.76  107.00   39.76   11.43  107.00   39.76   11.43\n",
      "  320  13510 719F 719F CA  C    5563 5571  148.80  116.50   32.30    9.29  116.50   32.30    9.29\n",
      "  321  13520 719F 720L C   N    5571 5573  151.51  120.00   31.51    7.16  120.00   31.51    7.16\n",
      "  322  13523 720L 720L N   CA   5573 5574  162.75  107.00   55.75   16.03  107.00   55.75   16.03\n",
      "  323  13526 720L 720L CA  C    5574 5579  161.99  116.50   45.49   13.08  116.50   45.49   13.08\n",
      "  324  13531 720L 721Y C   N    5579 5581  156.89  120.00   36.89    8.38  120.00   36.89    8.38\n",
      "  325  13534 721Y 721Y N   CA   5581 5582  156.56  107.00   49.56   14.25  107.00   49.56   14.25\n",
      "  326  13535 721Y 721Y CA  CB   5582 5583  133.97  107.50   26.47    6.12  107.50   26.47    6.12\n",
      "  327  13537 721Y 721Y CA  C    5582 5591  155.10  116.50   38.60   11.10  116.50   38.60   11.10\n",
      "  328  13549 721Y 722F C   N    5591 5593  172.40  120.00   52.40   11.91  120.00   52.40   11.91\n",
      "  329  13552 722F 722F N   CA   5593 5594  164.70  107.00   57.70   16.59  107.00   57.70   16.59\n",
      "  330  13555 722F 722F CA  C    5594 5602  155.27  116.50   38.78   11.15  116.50   38.78   11.15\n",
      "  331  13565 722F 723Y C   N    5602 5604  162.49  120.00   42.49    9.66  120.00   42.49    9.66\n",
      "  332  13568 723Y 723Y N   CA   5604 5605  160.10  107.00   53.10   15.27  107.00   53.10   15.27\n",
      "  333  13569 723Y 723Y CA  CB   5605 5606  128.94  107.50   21.45    4.96  107.50   21.45    4.96\n",
      "  334  13571 723Y 723Y CA  C    5605 5614  162.60  116.50   46.10   13.26  116.50   46.10   13.26\n",
      "  335  13583 723Y 724K C   N    5614 5616  164.08  120.00   44.08   10.02  120.00   44.08   10.02\n",
      "  336  13586 724K 724K N   CA   5616 5617  164.08  107.00   57.08   16.41  107.00   57.08   16.41\n",
      "  337  13589 724K 724K CA  C    5617 5623  161.62  116.50   45.12   12.97  116.50   45.12   12.97\n",
      "  338  13594 724K 725R C   N    5623 5625  168.03  120.00   48.03   10.92  120.00   48.03   10.92\n",
      "  339  13597 725R 725R N   CA   5625 5626  167.93  107.00   60.93   17.52  107.00   60.93   17.52\n",
      "  340  13600 725R 725R CA  C    5626 5634  163.64  116.50   47.14   13.55  116.50   47.14   13.55\n",
      "  341  13608 725R 726V C   N    5634 5636  166.57  120.00   46.57   10.58  120.00   46.57   10.58\n",
      "  342  13611 726V 726V N   CA   5636 5637  166.70  107.00   59.70   17.17  107.00   59.70   17.17\n",
      "  343  13615 726V 726V CA  C    5637 5641  165.07  116.50   48.57   13.97  116.50   48.57   13.97\n",
      "  344  13618 726V 727G C   N    5641 5643  167.80  120.00   47.80   10.86  120.00   47.80   10.86\n",
      "  345  13620 727G 727G N   CA   5643 5644  168.74  107.00   61.74   17.75  107.00   61.74   17.75\n",
      "  346  13622 727G 727G CA  C    5644 5645  166.28  116.50   49.78   14.31  116.50   49.78   14.31\n",
      "  347  13623 727G 728K C   N    5645 5647  166.89  120.00   46.89   10.66  120.00   46.89   10.66\n",
      "  348  13626 728K 728K N   CA   5647 5648  150.73  107.00   43.73    9.94  107.00   43.73    9.94\n",
      "  349  13628 728K 728K CA  C    5648 5654  146.60  118.00   28.60    5.81  118.00   28.60    5.81\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22527  47G  60V CA  CA    371  459   22.17    5.82   16.36    5.51    5.82   16.36    5.51\n",
      "    2  22528  47G  61E CA  CA    371  466   25.94    7.97   17.96    6.83    7.97   17.96    6.83\n",
      "    3  22529  47G  62H CA  CA    371  475   28.63    9.00   19.63    8.01    9.00   19.63    8.01\n",
      "    4  22530  47G  63V CA  CA    371  485   30.05   12.77   17.29    8.10   12.77   17.29    8.10\n",
      "    5  22531  47G 243C CA  CA    371 1840   20.06   13.26    6.79    9.86   13.26    6.79    9.86\n",
      "    6  22532  47G 244D CA  CA    371 1846   18.27   12.85    5.41    6.57   12.85    5.41    6.57\n",
      "    7  22533  47G 245D CA  CA    371 1854   21.08   10.17   10.91   14.73   10.17   10.91   14.73\n",
      "    8  22534  47G 246E CA  CA    371 1862   24.75   10.81   13.94   21.55   10.81   13.94   21.55\n",
      "    9  22535  47G 247V CA  CA    371 1871   27.03   11.44   15.59   25.39   11.44   15.59   25.39\n",
      "   10  22536  47G 248S CA  CA    371 1878   30.78   13.40   17.38   20.69   13.40   17.38   20.69\n",
      "   11  22537  48T  60V CA  CA    375  459   17.34    3.89   13.45    4.93    3.89   13.45    4.93\n",
      "   12  22538  48T  61E CA  CA    375  466   21.00    6.49   14.51    5.99    6.49   14.51    5.99\n",
      "   13  22539  48T  62H CA  CA    375  475   23.73    9.04   14.70    6.45    9.04   14.70    6.45\n",
      "   14  22540  48T  63V CA  CA    375  485   25.13   12.76   12.37    6.39   12.76   12.37    6.39\n",
      "   15  22541  48T 237D CA  CA    375 1783   29.02   13.73   15.28   25.21   13.73   15.28   25.21\n",
      "   16  22542  48T 239Y CA  CA    375 1802   21.80   13.74    8.07   14.88   13.74    8.07   14.88\n",
      "   17  22544  48T 245D CA  CA    375 1854   16.02   10.05    5.97    9.78   10.05    5.97    9.78\n",
      "   18  22545  48T 246E CA  CA    375 1862   19.70    9.43   10.28   20.79    9.43   10.28   20.79\n",
      "   19  22546  48T 247V CA  CA    375 1871   21.97   10.45   11.52   24.36   10.45   11.52   24.36\n",
      "   20  22547  48T 248S CA  CA    375 1878   25.72   12.09   13.63   21.11   12.09   13.63   21.11\n",
      "   21  22566  61E 235G CA  CA    466 1774   17.51   12.03    5.48   10.04   12.03    5.48   10.04\n",
      "   22  22567  61E 236A CA  CA    466 1778   15.29   11.52    3.77    7.90   11.52    3.77    7.90\n",
      "   23  22580  61E 252F CA  CA    466 1909   16.32   13.72    2.60    4.65   13.72    2.60    4.65\n",
      "   24  22912  86D 254N CA  CA    676 1925   23.24   13.93    9.32    8.41   13.93    9.32    8.41\n",
      "   25  22913  86D 255H CA  CA    676 1933   19.57   12.25    7.31    9.09   12.25    7.31    9.09\n",
      "   26  22914  86D 256M CA  CA    676 1943   20.69   13.51    7.18    8.24   13.51    7.18    8.24\n",
      "   27  22915  86D 257Q CA  CA    676 1951   22.21   12.79    9.42    6.52   12.79    9.42    6.52\n",
      "   28  22916  86D 258H CA  CA    676 1960   18.97    8.96   10.01    6.65    8.96   10.01    6.65\n",
      "   29  22928  96N 255H CA  CA    744 1933   22.64   13.60    9.04   10.36   13.60    9.04   10.36\n",
      "   30  22929  96N 256M CA  CA    744 1943   23.51   13.80    9.71   10.64   13.80    9.71   10.64\n",
      "   31  22930  96N 257Q CA  CA    744 1951   24.63   12.80   11.83    7.97   12.80   11.83    7.97\n",
      "   32  22931  96N 258H CA  CA    744 1960   21.21    9.26   11.95    7.72    9.26   11.95    7.72\n",
      "   33  22932  96N 259F CA  CA    744 1970   19.91    9.20   10.70    5.85    9.20   10.70    5.85\n",
      "   34  22933  96N 260G CA  CA    744 1981   22.69   11.36   11.34    4.55   11.36   11.34    4.55\n",
      "   35  22946 103P 254N CA  CA    793 1925   20.46   13.24    7.21    9.04   13.24    7.21    9.04\n",
      "   36  22947 103P 255H CA  CA    793 1933   17.08   11.11    5.97   10.68   11.11    5.97   10.68\n",
      "   37  22948 103P 256M CA  CA    793 1943   17.07   10.74    6.33   10.89   10.74    6.33   10.89\n",
      "   38  22949 103P 257Q CA  CA    793 1951   17.64   10.47    7.17    6.52   10.47    7.17    6.52\n",
      "   39  22950 103P 258H CA  CA    793 1960   14.30    7.11    7.19    5.74    7.11    7.19    5.74\n",
      "   40  22997 117K 606M CA  CA    893 4707   38.46   13.64   24.81   10.00   13.64   24.81   10.00\n",
      "   41  23034 118S 606M CA  CA    902 4707   36.00   11.87   24.13    8.92   11.87   24.13    8.92\n",
      "   42  23061 122P 606M CA  CA    932 4707   37.36   11.70   25.66    9.04   11.70   25.66    9.04\n",
      "   43  23080 123P 232T CA  CA    939 1754   14.52   10.94    3.59    5.74   10.94    3.59    5.74\n",
      "   44  23081 123P 233P CA  CA    939 1761   17.50   13.32    4.18    6.11   13.32    4.18    6.11\n",
      "   45  23090 123P 606M CA  CA    939 4707   37.51    9.60   27.91    9.63    9.60   27.91    9.63\n",
      "   46  23091 123P 607L CA  CA    939 4715   30.74   12.23   18.51    5.72   12.23   18.51    5.72\n",
      "   47  23107 129C 228L CA  CA    986 1728   10.83    8.21    2.62    5.75    8.21    2.62    5.75\n",
      "   48  23108 129C 229G CA  CA    986 1736   13.02    7.57    5.45   10.40    7.57    5.45   10.40\n",
      "   49  23109 129C 230T CA  CA    986 1740   16.75   11.11    5.64    9.93   11.11    5.64    9.93\n",
      "   50  23110 129C 231V CA  CA    986 1747   18.58   10.93    7.65   14.11   10.93    7.65   14.11\n",
      "   51  23111 129C 232T CA  CA    986 1754   22.23   12.90    9.33   14.00   12.90    9.33   14.00\n",
      "   52  23116 129C 606M CA  CA    986 4707   46.46   12.81   33.65   11.24   12.81   33.65   11.24\n",
      "   53  23133 140G 227K CA  CA   1076 1719   17.07   13.32    3.75    6.01   13.32    3.75    6.01\n",
      "   54  23134 140G 228L CA  CA   1076 1728   15.93    9.80    6.14   12.69    9.80    6.14   12.69\n",
      "   55  23135 140G 229G CA  CA   1076 1736   18.75    9.76    8.98   15.14    9.76    8.98   15.14\n",
      "   56  23136 140G 230T CA  CA   1076 1740   22.21   13.58    8.63   14.22   13.58    8.63   14.22\n",
      "   57  23160 155L 228L CA  CA   1178 1728   10.74    8.19    2.54    6.98    8.19    2.54    6.98\n",
      "   58  23161 155L 229G CA  CA   1178 1736   13.86    9.59    4.27    8.79    9.59    4.27    8.79\n",
      "   59  23162 155L 230T CA  CA   1178 1740   17.03   13.09    3.94    8.35   13.09    3.94    8.35\n",
      "   60  23163 155L 231V CA  CA   1178 1747   18.38   13.69    4.69   10.35   13.69    4.69   10.35\n",
      "   61  23191 156Q 229G CA  CA   1186 1736   14.78   12.03    2.76    4.81   12.03    2.76    4.81\n",
      "   62  23224 157C 228L CA  CA   1195 1728   13.11   10.94    2.17    6.06   10.94    2.17    6.06\n",
      "   63  23225 157C 229G CA  CA   1195 1736   16.19   13.47    2.72    5.35   13.47    2.72    5.35\n",
      "   64  23235 168S 193M CA  CA   1280 1469   16.04   10.88    5.16    4.92   10.88    5.16    4.92\n",
      "   65  23238 168S 198G CA  CA   1280 1504   15.63   11.08    4.55    6.78   11.08    4.55    6.78\n",
      "   66  23239 168S 199T CA  CA   1280 1508   11.93    7.73    4.20    6.13    7.73    4.20    6.13\n",
      "   67  23240 168S 200I CA  CA   1280 1515   12.28    6.23    6.05    6.37    6.23    6.05    6.37\n",
      "   68  23252 168S 228L CA  CA   1280 1728   20.60   13.67    6.93   13.33   13.67    6.93   13.33\n",
      "   69  23261 179K 198G CA  CA   1359 1504   18.75   13.88    4.86    7.38   13.88    4.86    7.38\n",
      "   70  23262 179K 199T CA  CA   1359 1508   14.97   10.15    4.82    6.87   10.15    4.82    6.87\n",
      "   71  23263 179K 200I CA  CA   1359 1515   13.95    8.82    5.12    5.24    8.82    5.12    5.24\n",
      "   72  23284 180C 200I CA  CA   1368 1515   13.85   10.55    3.30    5.32   10.55    3.30    5.32\n",
      "   73  23334 182L 227K CA  CA   1379 1719   16.24   13.34    2.90    5.89   13.34    2.90    5.89\n",
      "   74  23335 182L 228L CA  CA   1379 1728   16.12   12.33    3.79    9.21   12.33    3.79    9.21\n",
      "   75  23336 182L 229G CA  CA   1379 1736   18.51   13.56    4.95    8.90   13.56    4.95    8.90\n",
      "   76  23353 188R 228L CA  CA   1420 1728   12.19    9.97    2.22    4.90    9.97    2.22    4.90\n",
      "   77  23354 188R 229G CA  CA   1420 1736   13.45   10.56    2.89    4.99   10.56    2.89    4.99\n",
      "   78  23435 191N 231V CA  CA   1447 1747   14.08   12.10    1.98    4.61   12.10    1.98    4.61\n",
      "   79  23471 192W 231V CA  CA   1455 1747   13.71   11.32    2.39    5.27   11.32    2.39    5.27\n",
      "   80  23509 193M 231V CA  CA   1469 1747   13.06   11.00    2.06    4.75   11.00    2.06    4.75\n",
      "   81  23526 193M 606M CA  CA   1469 4707   38.70   13.92   24.78   26.28   13.92   24.78   26.28\n",
      "   82  23685 198G 252F CA  CA   1504 1909   15.39   13.63    1.76    4.54   13.63    1.76    4.54\n",
      "   83  23688 198G 256M CA  CA   1504 1943   11.20    9.59    1.61    4.62    9.59    1.61    4.62\n",
      "   84  23815 203P 230T CA  CA   1533 1740   15.62   13.52    2.10    4.51   13.52    2.10    4.51\n",
      "   85  24079 225V 231V CA  CA   1705 1747   14.18   11.82    2.36    5.36   11.82    2.36    5.36\n",
      "   86  24107 226V 231V CA  CA   1712 1747   10.41    8.04    2.37    5.62    8.04    2.37    5.62\n",
      "   87  24135 226V 606M CA  CA   1712 4707   35.14   11.91   23.23   62.10   11.91   23.23   62.10\n",
      "   88  24138 227K 231V CA  CA   1719 1747    9.66    7.49    2.17    4.65    7.49    2.17    4.65\n",
      "   89  24158 227K 606M CA  CA   1719 4707   36.27   12.70   23.57   51.22   12.70   23.57   51.22\n",
      "   90  24175 228L 606M CA  CA   1728 4707   36.13   11.25   24.88   65.59   11.25   24.88   65.59\n",
      "   91  24187 229G 606M CA  CA   1736 4707   33.98   10.88   23.10   46.83   10.88   23.10   46.83\n",
      "   92  24188 229G 607L CA  CA   1736 4715   26.83   13.87   12.96   18.76   13.87   12.96   18.76\n",
      "   93  24189 229G 728K CA  CA   1736 5648   20.43   12.91    7.51   13.15   12.91    7.51   13.15\n",
      "   94  24198 230T 246E CA  CA   1740 1862   15.52   12.94    2.57    4.90   12.94    2.57    4.90\n",
      "   95  24206 230T 606M CA  CA   1740 4707   30.68    9.84   20.84   49.42    9.84   20.84   49.42\n",
      "   96  24207 230T 607L CA  CA   1740 4715   23.49   12.62   10.87   18.64   12.62   10.87   18.64\n",
      "   97  24208 230T 728K CA  CA   1740 5648   24.12   12.71   11.41   23.01   12.71   11.41   23.01\n",
      "   98  24214 231V 238F CA  CA   1747 1791   12.18    9.55    2.63    6.10    9.55    2.63    6.10\n",
      "   99  24215 231V 239Y CA  CA   1747 1802   15.67   13.01    2.65    5.59   13.01    2.65    5.59\n",
      "  100  24216 231V 248S CA  CA   1747 1878   15.58   12.98    2.59    4.73   12.98    2.59    4.73\n",
      "  101  24229 231V 606M CA  CA   1747 4707   28.04    6.06   21.98   59.69    6.06   21.98   59.69\n",
      "  102  24230 231V 607L CA  CA   1747 4715   20.92    9.09   11.83   23.86    9.09   11.83   23.86\n",
      "  103  24231 231V 728K CA  CA   1747 5648   26.10    9.49   16.60   37.98    9.49   16.60   37.98\n",
      "  104  24245 232T 606M CA  CA   1754 4707   24.69    5.37   19.32   47.76    5.37   19.32   47.76\n",
      "  105  24246 232T 607L CA  CA   1754 4715   17.54    6.72   10.82   22.01    6.72   10.82   22.01\n",
      "  106  24247 232T 728K CA  CA   1754 5648   28.96    7.50   21.46   45.92    7.50   21.46   45.92\n",
      "  107  24261 233P 262D CA  CA   1761 1993   13.69    8.58    5.11    9.71    8.58    5.11    9.71\n",
      "  108  24262 233P 606M CA  CA   1761 4707   21.05    5.20   15.85   38.55    5.20   15.85   38.55\n",
      "  109  24263 233P 607L CA  CA   1761 4715   13.91    4.36    9.56   22.84    4.36    9.56   22.84\n",
      "  110  24264 233P 728K CA  CA   1761 5648   32.58    6.73   25.84   56.21    6.73   25.84   56.21\n",
      "  111  24268 234S 248S CA  CA   1768 1878   15.13   11.90    3.23    4.58   11.90    3.23    4.58\n",
      "  112  24277 234S 606M CA  CA   1768 4707   21.70    8.18   13.52   25.33    8.18   13.52   25.33\n",
      "  113  24278 234S 607L CA  CA   1768 4715   14.52    8.02    6.50   10.63    8.02    6.50   10.63\n",
      "  114  24279 234S 728K CA  CA   1768 5648   32.84   10.59   22.26   35.38   10.59   22.26   35.38\n",
      "  115  24283 235G 246E CA  CA   1774 1862   16.71   12.57    4.14    7.71   12.57    4.14    7.71\n",
      "  116  24284 235G 247V CA  CA   1774 1871   14.68   11.30    3.39    7.00   11.30    3.39    7.00\n",
      "  117  24285 235G 248S CA  CA   1774 1878   11.60    8.95    2.64    4.92    8.95    2.64    4.92\n",
      "  118  24297 235G 606M CA  CA   1774 4707   24.47    7.52   16.95   39.46    7.52   16.95   39.46\n",
      "  119  24298 235G 607L CA  CA   1774 4715   17.35    9.05    8.30   15.32    9.05    8.30   15.32\n",
      "  120  24299 235G 728K CA  CA   1774 5648   31.43   11.48   19.95   38.75   11.48   19.95   38.75\n",
      "  121  24303 236A 245D CA  CA   1778 1854   17.10   13.89    3.21    5.49   13.89    3.21    5.49\n",
      "  122  24304 236A 246E CA  CA   1778 1862   13.51   10.23    3.28    7.01   10.23    3.28    7.01\n",
      "  123  24305 236A 247V CA  CA   1778 1871   12.08    9.89    2.18    5.14    9.89    2.18    5.14\n",
      "  124  24318 236A 606M CA  CA   1778 4707   28.39    8.77   19.62   50.17    8.77   19.62   50.17\n",
      "  125  24319 236A 607L CA  CA   1778 4715   21.26   11.42    9.84   18.61   11.42    9.84   18.61\n",
      "  126  24320 236A 728K CA  CA   1778 5648   28.36   12.98   15.38   33.00   12.98   15.38   33.00\n",
      "  127  24326 237D 246E CA  CA   1783 1862    9.60    7.05    2.55    5.21    7.05    2.55    5.21\n",
      "  128  24336 237D 606M CA  CA   1783 4707   31.85   12.37   19.48   39.86   12.37   19.48   39.86\n",
      "  129  24354 238F 606M CA  CA   1791 4707   33.90   13.44   20.46   49.79   13.44   20.46   49.79\n",
      "  130  24424 249D 606M CA  CA   1884 4707   29.98   13.37   16.61   39.96   13.37   16.61   39.96\n",
      "  131  24431 250V 606M CA  CA   1892 4707   26.82   13.43   13.39   24.60   13.43   13.39   24.60\n",
      "  132  24441 251H 606M CA  CA   1899 4707   26.08   12.08   14.00   28.17   12.08   14.00   28.17\n",
      "  133  24442 251H 607L CA  CA   1899 4715   20.12   13.68    6.44    9.17   13.68    6.44    9.17\n",
      "  134  24452 252F 606M CA  CA   1909 4707   26.32    8.96   17.36   40.65    8.96   17.36   40.65\n",
      "  135  24453 252F 607L CA  CA   1909 4715   19.79   11.27    8.52   13.66   11.27    8.52   13.66\n",
      "  136  24454 252F 728K CA  CA   1909 5648   34.04   13.91   20.13   36.00   13.91   20.13   36.00\n",
      "  137  24462 253A 262D CA  CA   1920 1993   11.49    7.50    4.00    6.55    7.50    4.00    6.55\n",
      "  138  24463 253A 606M CA  CA   1920 4707   23.29    6.73   16.55   32.90    6.73   16.55   32.90\n",
      "  139  24464 253A 607L CA  CA   1920 4715   16.76    8.40    8.36   12.09    8.40    8.36   12.09\n",
      "  140  24465 253A 728K CA  CA   1920 5648   35.13   11.13   24.00   35.50   11.13   24.00   35.50\n",
      "  141  24473 254N 606M CA  CA   1925 4707   25.36    9.84   15.53   23.07    9.84   15.53   23.07\n",
      "  142  24474 254N 607L CA  CA   1925 4715   19.32   11.52    7.80    8.48   11.52    7.80    8.48\n",
      "  143  24475 254N 728K CA  CA   1925 5648   35.93   13.83   22.10   26.01   13.83   22.10   26.01\n",
      "  144  24482 255H 606M CA  CA   1933 4707   29.03   10.39   18.65   35.97   10.39   18.65   35.97\n",
      "  145  24483 255H 607L CA  CA   1933 4715   22.79   13.12    9.67   12.93   13.12    9.67   12.93\n",
      "  146  24489 256M 606M CA  CA   1943 4707   28.70    7.34   21.36   43.17    7.34   21.36   43.17\n",
      "  147  24490 256M 607L CA  CA   1943 4715   22.09   10.55   11.54   16.03   10.55   11.54   16.03\n",
      "  148  24491 256M 728K CA  CA   1943 5648   30.59   11.69   18.90   30.08   11.69   18.90   30.08\n",
      "  149  24494 257Q 261I CA  CA   1951 1985    7.64    5.03    2.61    4.66    5.03    2.61    4.66\n",
      "  150  24495 257Q 262D CA  CA   1951 1993   11.95    6.08    5.87    5.78    6.08    5.87    5.78\n",
      "  151  24496 257Q 606M CA  CA   1951 4707   27.60    8.14   19.46   19.44    8.14   19.46   19.44\n",
      "  152  24497 257Q 607L CA  CA   1951 4715   21.34   10.33   11.01    8.77   10.33   11.01    8.77\n",
      "  153  24498 257Q 728K CA  CA   1951 5648   32.22   11.32   20.91   18.07   11.32   20.91   18.07\n",
      "  154  24502 258H 606M CA  CA   1960 4707   30.95   11.57   19.38   14.62   11.57   19.38   14.62\n",
      "  155  24505 259F 606M CA  CA   1970 4707   33.06   10.95   22.11   13.87   10.95   22.11   13.87\n",
      "  156  24506 259F 728K CA  CA   1970 5648   28.28   13.89   14.40    8.41   13.89   14.40    8.41\n",
      "  157  24508 260G 606M CA  CA   1981 4707   30.92    9.52   21.40   10.24    9.52   21.40   10.24\n",
      "  158  24509 260G 607L CA  CA   1981 4715   24.18   11.87   12.31    5.17   11.87   12.31    5.17\n",
      "  159  24510 260G 728K CA  CA   1981 5648   27.17   11.23   15.94    7.21   11.23   15.94    7.21\n",
      "  160  24511 261I 606M CA  CA   1985 4707   31.92    5.99   25.92   11.57    5.99   25.92   11.57\n",
      "  161  24512 261I 607L CA  CA   1985 4715   25.01    8.92   16.09    6.47    8.92   16.09    6.47\n",
      "  162  24513 261I 728K CA  CA   1985 5648   26.79    8.38   18.41    7.82    8.38   18.41    7.82\n",
      "  163  24514 262D 606M CA  CA   1993 4707   33.24    3.83   29.41   11.25    3.83   29.41   11.25\n",
      "  164  24515 262D 607L CA  CA   1993 4715   26.18    5.40   20.78    7.40    5.40   20.78    7.40\n",
      "  165  24516 262D 728K CA  CA   1993 5648   25.62    5.28   20.34    7.50    5.28   20.34    7.50\n",
      "  166  24517 606M 728K CA  CA   4707 5648   53.26    5.10   48.16   18.39    5.10   48.16   18.39\n",
      "  167  24518 607L 728K CA  CA   4715 5648   46.27    3.83   42.44   15.81    3.83   42.44   15.81\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24519  47G  60V N   O     370  464   25.53    5.46   20.07    6.24    5.46   20.07    6.24\n",
      "    2  24520  47G  61E N   O     370  473   27.98    9.39   18.60    6.43    9.39   18.60    6.43\n",
      "    3  24521  47G  62H N   O     370  483   30.42   10.24   20.18    7.28   10.24   20.18    7.28\n",
      "    4  24522  47G 243C N   O     370 1844   20.37   10.34   10.03   14.80   10.34   10.03   14.80\n",
      "    5  24523  47G 244D N   O     370 1852   20.24    9.80   10.43   12.32    9.80   10.43   12.32\n",
      "    6  24524  47G 246E N   O     370 1869   26.37    8.86   17.51   26.31    8.86   17.51   26.31\n",
      "    7  24525  48T  60V N   O     374  464   20.35    4.55   15.80    5.36    4.55   15.80    5.36\n",
      "    8  24526  48T  61E N   O     374  473   22.71    8.51   14.20    5.37    8.51   14.20    5.37\n",
      "    9  24527  48T  62H N   O     374  483   25.11   10.49   14.62    5.83   10.49   14.62    5.83\n",
      "   10  24528  48T 246E N   O     374 1869   21.08    8.70   12.38   23.32    8.70   12.38   23.32\n",
      "   11  24529  60V  47G N   O     458  373   18.74    3.45   15.29    4.96    3.45   15.29    4.96\n",
      "   12  24534  61E  47G N   O     465  373   22.47    5.39   17.08    6.37    5.39   17.08    6.37\n",
      "   13  24535  61E  48T N   O     465  380   18.23    3.58   14.65    6.04    3.58   14.65    6.04\n",
      "   14  24537  61E 236A N   O     465 1781   14.18    9.83    4.35    8.86    9.83    4.35    8.86\n",
      "   15  24543  62H  47G N   O     474  373   25.84    6.52   19.33    7.56    6.52   19.33    7.56\n",
      "   16  24544  62H  48T N   O     474  380   21.48    6.51   14.97    6.45    6.51   14.97    6.45\n",
      "   17  24557  63V  47G N   O     484  373   27.74    9.89   17.85    8.18    9.89   17.85    8.18\n",
      "   18  24558  63V  48T N   O     484  380   23.23   10.09   13.13    6.65   10.09   13.13    6.65\n",
      "   19  24681  79I 249D N   O     620 1890   13.06   10.58    2.48    6.28   10.58    2.48    6.28\n",
      "   20  24682  79I 250V N   O     620 1897   13.99   10.95    3.05    5.64   10.95    3.05    5.64\n",
      "   21  24683  79I 251H N   O     620 1907   12.25    8.69    3.56    7.55    8.69    3.56    7.55\n",
      "   22  24746  86D 196T N   O     675 1496    9.50    6.06    3.44    5.28    6.06    3.44    5.28\n",
      "   23  24748  86D 255H N   O     675 1941   18.01   10.89    7.11    8.31   10.89    7.11    8.31\n",
      "   24  24749  86D 258H N   O     675 1968   17.75    8.43    9.32    5.78    8.43    9.32    5.78\n",
      "   25  24756  96N 195M N   O     743 1489   16.94   10.61    6.33    7.28   10.61    6.33    7.28\n",
      "   26  24757  96N 196T N   O     743 1496   14.18    8.31    5.87    7.77    8.31    5.87    7.77\n",
      "   27  24758  96N 197C N   O     743 1502   12.72    8.60    4.12    5.04    8.60    4.12    5.04\n",
      "   28  24759  96N 257Q N   O     743 1958   24.65   10.58   14.07    9.00   10.58   14.07    9.00\n",
      "   29  24760  96N 258H N   O     743 1968   21.82    7.30   14.52    9.08    7.30   14.52    9.08\n",
      "   30  24761  96N 259F N   O     743 1979   22.71   10.74   11.96    5.94   10.74   11.96    5.94\n",
      "   31  24774 103P 255H N   O     792 1941   14.89    9.83    5.06    8.49    9.83    5.06    8.49\n",
      "   32  24775 103P 256M N   O     792 1949   16.65   10.09    6.56   10.33   10.09    6.56   10.33\n",
      "   33  24776 103P 257Q N   O     792 1958   15.90    8.93    6.98    6.07    8.93    6.98    6.07\n",
      "   34  24777 103P 258H N   O     792 1968   13.03    5.45    7.58    5.92    5.45    7.58    5.92\n",
      "   35  24791 116A 255H N   O     887 1941   14.00    9.88    4.12    5.71    9.88    4.12    5.71\n",
      "   36  24792 116A 256M N   O     887 1949   14.61    9.39    5.22    6.99    9.39    5.22    6.99\n",
      "   37  24806 117K 256M N   O     892 1949   11.86    8.74    3.12    5.15    8.74    3.12    5.15\n",
      "   38  24854 123P 231V N   O     938 1752   12.99    9.86    3.13    5.50    9.86    3.13    5.50\n",
      "   39  24868 129C 227K N   O     985 1726   13.54   10.56    2.99    4.80   10.56    2.99    4.80\n",
      "   40  24870 129C 229G N   O     985 1738   14.77    9.63    5.14    7.62    9.63    5.14    7.62\n",
      "   41  24871 129C 231V N   O     985 1752   19.62    9.74    9.88   17.49    9.74    9.88   17.49\n",
      "   42  24887 140G 228L N   O    1075 1734   16.76    8.93    7.83   15.25    8.93    7.83   15.25\n",
      "   43  24888 140G 229G N   O    1075 1738   21.50   10.45   11.05   16.18   10.45   11.05   16.18\n",
      "   44  24906 155L 228L N   O    1177 1734   12.48    8.66    3.82    9.68    8.66    3.82    9.68\n",
      "   45  24926 156Q 228L N   O    1185 1734   12.17    9.87    2.30    5.06    9.87    2.30    5.06\n",
      "   46  24948 157C 228L N   O    1194 1734   12.58   10.65    1.93    5.28   10.65    1.93    5.28\n",
      "   47  24961 168S 192W N   O    1279 1467   14.85    8.00    6.85    4.72    8.00    6.85    4.72\n",
      "   48  24962 168S 198G N   O    1279 1506   15.01    8.70    6.30    9.17    8.70    6.30    9.17\n",
      "   49  24963 168S 199T N   O    1279 1513   13.91    7.75    6.16    7.90    7.75    6.16    7.90\n",
      "   50  24964 168S 200I N   O    1279 1521   10.89    3.06    7.82    8.48    3.06    7.82    8.48\n",
      "   51  24980 179K 198G N   O    1358 1506   18.26   10.77    7.49   11.11   10.77    7.49   11.11\n",
      "   52  24981 179K 199T N   O    1358 1513   16.24    9.61    6.63    8.58    9.61    6.63    8.58\n",
      "   53  24982 179K 200I N   O    1358 1521   12.94    5.15    7.79    8.22    5.15    7.79    8.22\n",
      "   54  24997 180C 200I N   O    1367 1521   11.24    7.05    4.19    6.90    7.05    4.19    6.90\n",
      "   55  25224 195M 252F N   O    1482 1918   11.62    9.69    1.93    5.30    9.69    1.93    5.30\n",
      "   56  25254 196T 252F N   O    1490 1918   12.49   10.58    1.91    5.05   10.58    1.91    5.05\n",
      "   57  25286 198G  86D N   O    1503  682   11.93    8.42    3.51    4.56    8.42    3.51    4.56\n",
      "   58  25304 198G 256M N   O    1503 1949   11.64    9.83    1.81    4.87    9.83    1.81    4.87\n",
      "   59  25892 228L 182L N   O    1727 1385   14.43   10.86    3.57    8.45   10.86    3.57    8.45\n",
      "   60  25909 228L 231V N   O    1727 1752   10.78    8.46    2.32    5.73    8.46    2.32    5.73\n",
      "   61  25918 229G 129C N   O    1735  990   13.44    9.03    4.41    6.68    9.03    4.41    6.68\n",
      "   62  25919 229G 140G N   O    1735 1078   18.65   10.74    7.91   11.49   10.74    7.91   11.49\n",
      "   63  25921 229G 188R N   O    1735 1429   11.48    8.26    3.22    5.48    8.26    3.22    5.48\n",
      "   64  25936 229G 606M N   O    1735 4713   30.01   10.86   19.15   36.11   10.86   19.15   36.11\n",
      "   65  25937 229G 728K N   O    1735 5655   17.23   10.98    6.25   10.82   10.98    6.25   10.82\n",
      "   66  25940 230T 188R N   O    1739 1429   13.98   10.35    3.63    6.56   10.35    3.63    6.56\n",
      "   67  25954 230T 606M N   O    1739 4713   27.28   10.05   17.23   37.94   10.05   17.23   37.94\n",
      "   68  25955 230T 728K N   O    1739 5655   19.62   10.90    8.72   17.18   10.90    8.72   17.18\n",
      "   69  25957 231V 123P N   O    1746  944   11.37    8.38    2.99    5.47    8.38    2.99    5.47\n",
      "   70  25958 231V 191N N   O    1746 1453   10.98    8.88    2.10    4.96    8.88    2.10    4.96\n",
      "   71  25973 231V 606M N   O    1746 4713   24.58    7.57   17.01   41.56    7.57   17.01   41.56\n",
      "   72  25974 231V 728K N   O    1746 5655   21.83    9.06   12.77   27.40    9.06   12.77   27.40\n",
      "   73  25975 232T 123P N   O    1753  944   14.35    9.84    4.51    6.81    9.84    4.51    6.81\n",
      "   74  25988 232T 262D N   O    1753 1999   11.98    7.25    4.73    8.54    7.25    4.73    8.54\n",
      "   75  25989 232T 606M N   O    1753 4713   21.11    4.82   16.30   35.78    4.82   16.30   35.78\n",
      "   76  25990 232T 607L N   O    1753 4721   17.50    8.14    9.35   15.31    8.14    9.35   15.31\n",
      "   77  25991 232T 728K N   O    1753 5655   24.94    7.27   17.66   34.00    7.27   17.66   34.00\n",
      "   78  25999 233P 262D N   O    1760 1999   15.03    6.83    8.19   14.75    6.83    8.19   14.75\n",
      "   79  26000 233P 606M N   O    1760 4713   17.75    3.34   14.41   32.13    3.34   14.41   32.13\n",
      "   80  26001 233P 607L N   O    1760 4721   14.15    5.27    8.88   15.83    5.27    8.88   15.83\n",
      "   81  26002 233P 728K N   O    1760 5655   27.94    6.42   21.52   41.71    6.42   21.52   41.71\n",
      "   82  26009 234S 262D N   O    1767 1999   14.53    9.20    5.33    7.35    9.20    5.33    7.35\n",
      "   83  26010 234S 606M N   O    1767 4713   17.25    5.81   11.44   20.49    5.81   11.44   20.49\n",
      "   84  26011 234S 607L N   O    1767 4721   13.90    7.38    6.52    8.71    7.38    6.52    8.71\n",
      "   85  26012 234S 728K N   O    1767 5655   29.02    9.21   19.81   29.74    9.21   19.81   29.74\n",
      "   86  26020 235G 247V N   O    1773 1876   13.57   10.30    3.28    6.94   10.30    3.28    6.94\n",
      "   87  26026 235G 606M N   O    1773 4713   19.13    6.83   12.30   26.67    6.83   12.30   26.67\n",
      "   88  26027 235G 607L N   O    1773 4721   15.93    9.30    6.62   11.01    9.30    6.62   11.01\n",
      "   89  26028 235G 728K N   O    1773 5655   28.31   10.50   17.81   33.19   10.50   17.81   33.19\n",
      "   90  26039 236A 238F N   O    1777 1800   10.17    8.28    1.88    4.87    8.28    1.88    4.87\n",
      "   91  26040 236A 247V N   O    1777 1876   10.26    7.56    2.70    6.87    7.56    2.70    6.87\n",
      "   92  26048 236A 606M N   O    1777 4713   22.43    8.78   13.65   33.34    8.78   13.65   33.34\n",
      "   93  26210 245D  47G N   O    1853  373   17.93    9.49    8.44    9.68    9.49    8.44    9.68\n",
      "   94  26225 246E  47G N   O    1861  373   20.87    8.04   12.84   20.02    8.04   12.84   20.02\n",
      "   95  26226 246E  48T N   O    1861  380   16.28    8.02    8.25   15.99    8.02    8.25   15.99\n",
      "   96  26233 246E 236A N   O    1861 1781   12.76    9.53    3.22    7.06    9.53    3.22    7.06\n",
      "   97  26243 247V  47G N   O    1870  373   24.01    9.09   14.92   24.34    9.09   14.92   24.34\n",
      "   98  26244 247V  48T N   O    1870  380   19.31    8.35   10.95   22.86    8.35   10.95   22.86\n",
      "   99  26253 247V 236A N   O    1870 1781    9.84    7.22    2.62    6.69    7.22    2.62    6.69\n",
      "  100  26264 248S  47G N   O    1877  373   27.12   10.21   16.91   20.21   10.21   16.91   20.21\n",
      "  101  26265 248S  48T N   O    1877  380   22.37    9.22   13.15   20.81    9.22   13.15   20.81\n",
      "  102  26276 248S 234S N   O    1877 1772   14.82   10.72    4.09    5.47   10.72    4.09    5.47\n",
      "  103  26379 253A 606M N   O    1919 4713   19.52    9.45   10.06   15.61    9.45   10.06   15.61\n",
      "  104  26395 254N 606M N   O    1924 4713   20.09   10.87    9.22   11.78   10.87    9.22   11.78\n",
      "  105  26424 256M 116A N   O    1942  891   13.54    9.86    3.68    4.84    9.86    3.68    4.84\n",
      "  106  26446 256M 606M N   O    1942 4713   23.90   10.48   13.42   22.13   10.48   13.42   22.13\n",
      "  107  26461 257Q 261I N   O    1950 1991    7.34    3.50    3.84    6.50    3.50    3.84    6.50\n",
      "  108  26463 257Q 606M N   O    1950 4713   23.05    9.90   13.16   11.47    9.90   13.16   11.47\n",
      "  109  26468 258H  96N N   O    1959  750   21.60   10.98   10.62    6.06   10.98   10.62    6.06\n",
      "  110  26469 258H 103P N   O    1959  798   17.33    8.91    8.42    6.07    8.91    8.42    6.07\n",
      "  111  26488 259F  86D N   O    1969  682   19.74    9.86    9.87    5.07    9.86    9.87    5.07\n",
      "  112  26489 259F  96N N   O    1969  750   19.33    9.14   10.18    5.21    9.14   10.18    5.21\n",
      "  113  26490 259F 103P N   O    1969  798   15.03    6.79    8.24    5.13    6.79    8.24    5.13\n",
      "  114  26543 261I 606M N   O    1984 4713   26.41    9.30   17.11    7.07    9.30   17.11    7.07\n",
      "  115  26544 261I 728K N   O    1984 5655   24.07    9.61   14.45    5.77    9.61   14.45    5.77\n",
      "  116  26555 262D 253A N   O    1992 1923   10.59    6.80    3.78    5.59    6.80    3.78    5.59\n",
      "  117  26558 262D 256M N   O    1992 1949    8.10    5.39    2.70    4.51    5.39    2.70    4.51\n",
      "  118  26563 262D 606M N   O    1992 4713   28.10    6.20   21.90    7.77    6.20   21.90    7.77\n",
      "  119  26564 262D 607L N   O    1992 4721   24.72    8.01   16.71    5.37    8.01   16.71    5.37\n",
      "  120  26565 262D 728K N   O    1992 5655   22.86    6.56   16.31    5.62    6.56   16.31    5.62\n",
      "  121  26566 606M 118S N   O    4706  906   41.58    9.51   32.07   11.59    9.51   32.07   11.59\n",
      "  122  26567 606M 122P N   O    4706  937   40.97   10.15   30.82   10.49   10.15   30.82   10.49\n",
      "  123  26568 606M 123P N   O    4706  944   43.75   10.43   33.32   10.80   10.43   33.32   10.80\n",
      "  124  26569 606M 230T N   O    4706 1745   32.89    9.71   23.18   51.40    9.71   23.18   51.40\n",
      "  125  26570 606M 231V N   O    4706 1752   31.82    7.12   24.70   60.89    7.12   24.70   60.89\n",
      "  126  26571 606M 232T N   O    4706 1759   27.97    5.78   22.19   47.49    5.78   22.19   47.49\n",
      "  127  26572 606M 233P N   O    4706 1766   24.87    7.57   17.30   34.81    7.57   17.30   34.81\n",
      "  128  26573 606M 234S N   O    4706 1772   26.45   10.52   15.94   25.06   10.52   15.94   25.06\n",
      "  129  26574 606M 235G N   O    4706 1776   30.68    7.86   22.83   48.51    7.86   22.83   48.51\n",
      "  130  26575 606M 251H N   O    4706 1907   30.24   10.34   19.90   38.50   10.34   19.90   38.50\n",
      "  131  26576 606M 252F N   O    4706 1918   30.41    7.37   23.04   52.96    7.37   23.04   52.96\n",
      "  132  26577 606M 253A N   O    4706 1923   28.80    6.24   22.56   40.50    6.24   22.56   40.50\n",
      "  133  26578 606M 254N N   O    4706 1931   31.70    9.32   22.38   30.31    9.32   22.38   30.31\n",
      "  134  26579 606M 255H N   O    4706 1941   34.89    9.46   25.43   46.02    9.46   25.43   46.02\n",
      "  135  26580 606M 256M N   O    4706 1949   33.51    6.87   26.64   50.75    6.87   26.64   50.75\n",
      "  136  26581 606M 257Q N   O    4706 1958   34.11    8.68   25.43   22.92    8.68   25.43   22.92\n",
      "  137  26582 606M 259F N   O    4706 1979   37.53    9.56   27.97   17.02    9.56   27.97   17.02\n",
      "  138  26583 606M 260G N   O    4706 1983   37.62    7.12   30.51   14.36    7.12   30.51   14.36\n",
      "  139  26584 606M 261I N   O    4706 1991   35.56    3.73   31.83   14.19    3.73   31.83   14.19\n",
      "  140  26585 606M 262D N   O    4706 1999   39.11    2.24   36.87   13.89    2.24   36.87   13.89\n",
      "  141  26586 606M 728K N   O    4706 5655   54.60    5.74   48.86   17.83    5.74   48.86   17.83\n",
      "  142  26587 607L 230T N   O    4714 1745   22.82   10.06   12.76   21.98   10.06   12.76   21.98\n",
      "  143  26588 607L 231V N   O    4714 1752   21.85    7.47   14.38   27.69    7.47   14.38   27.69\n",
      "  144  26589 607L 232T N   O    4714 1759   17.94    5.05   12.89   23.56    5.05   12.89   23.56\n",
      "  145  26590 607L 233P N   O    4714 1766   14.81    5.75    9.05   15.86    5.75    9.05   15.86\n",
      "  146  26591 607L 234S N   O    4714 1772   16.50    9.53    6.97    8.67    9.53    6.97    8.67\n",
      "  147  26592 607L 235G N   O    4714 1776   20.63    7.92   12.72   21.95    7.92   12.72   21.95\n",
      "  148  26593 607L 252F N   O    4714 1918   20.92    8.95   11.97   18.95    8.95   11.97   18.95\n",
      "  149  26594 607L 253A N   O    4714 1923   19.48    7.87   11.61   14.93    7.87   11.61   14.93\n",
      "  150  26595 607L 256M N   O    4714 1949   24.05    9.47   14.59   18.77    9.47   14.59   18.77\n",
      "  151  26596 607L 257Q N   O    4714 1958   25.05   10.98   14.07    9.96   10.98   14.07    9.96\n",
      "  152  26597 607L 260G N   O    4714 1983   27.88    9.44   18.45    7.57    9.44   18.45    7.57\n",
      "  153  26598 607L 261I N   O    4714 1991   25.75    6.26   19.48    7.75    6.26   19.48    7.75\n",
      "  154  26599 607L 262D N   O    4714 1999   29.30    3.40   25.90    9.02    3.40   25.90    9.02\n",
      "  155  26600 607L 728K N   O    4714 5655   44.80    5.32   39.48   13.58    5.32   39.48   13.58\n",
      "  156  26601 728K 231V N   O    5647 1752   28.86    8.18   20.69   45.24    8.18   20.69   45.24\n",
      "  157  26602 728K 232T N   O    5647 1759   33.33    6.93   26.40   51.31    6.93   26.40   51.31\n",
      "  158  26603 728K 233P N   O    5647 1766   36.53    7.68   28.85   54.08    7.68   28.85   54.08\n",
      "  159  26604 728K 235G N   O    5647 1776   31.76    9.89   21.87   41.26    9.89   21.87   41.26\n",
      "  160  26605 728K 253A N   O    5647 1923   36.87   10.01   26.87   37.32   10.01   26.87   37.32\n",
      "  161  26606 728K 256M N   O    5647 1949   32.27   10.79   21.48   31.33   10.79   21.48   31.33\n",
      "  162  26607 728K 260G N   O    5647 1983   27.86    9.33   18.53    8.14    9.33   18.53    8.14\n",
      "  163  26608 728K 261I N   O    5647 1991   30.46    7.36   23.10    9.49    7.36   23.10    9.49\n",
      "  164  26609 728K 262D N   O    5647 1999   29.14    3.27   25.87    9.40    3.27   25.87    9.40\n",
      "  165  26610 728K 606M N   O    5647 4713   51.19    3.30   47.89   18.09    3.30   47.89   18.09\n",
      "  166  26611 728K 607L N   O    5647 4721   47.50    2.29   45.21   16.42    2.29   45.21   16.42\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19720   1M   2K C   N       7    9 -157.90 -118.00   44.40    1.47  -62.90 -173.40   29.89\n",
      "    1          2K   2K N   CA      9   10  158.59  139.10                  -40.80\n",
      "    2  19722   3P   4P C   N      23   25  -89.10  -58.70   41.61    3.03  -64.50  151.30   12.62\n",
      "    2          4P   4P N   CA     25   26   -2.09  -30.50                  147.20\n",
      "    3  19723   4P   5F C   N      30   32  -40.28  -71.40   33.61    2.68  -63.20  163.91   21.26\n",
      "    3          5F   5F N   CA     32   33  153.40  140.70                  -44.30\n",
      "    4  19724   5F   6N C   N      41   43 -157.17 -119.90   56.95    1.70  -63.20  167.66   26.12\n",
      "    4          6N   6N N   CA     43   44 -179.94  137.00                  -41.10\n",
      "    5  19726   7I   8T C   N      57   59 -155.48 -124.80   48.90    1.53  -63.20  164.62   26.91\n",
      "    5          8T   8T N   CA     59   60 -178.42  143.50                  -42.10\n",
      "    6  19727   8T   9D C   N      64   66 -157.38  -70.90   93.63    5.55  -63.30  163.58   26.14\n",
      "    6          9D   9D N   CA     66   67 -173.82  150.30                  -40.00\n",
      "    7  19728   9D  10E C   N      72   74 -168.49 -117.80   63.00    1.89  -63.60  179.36   29.57\n",
      "    7         10E  10E N   CA     74   75  174.21  136.80                  -40.30\n",
      "    8  19729  10E  11A C   N      81   83 -170.01 -134.00   47.69    1.32  -62.50  177.18   34.67\n",
      "    8         11A  11A N   CA     83   84  178.27  147.00                  -40.90\n",
      "    9  19731  12C  13A C   N      92   94 -176.52 -134.00   50.31    1.20  -62.50 -175.37   36.22\n",
      "    9         13A  13A N   CA     94   95  173.89  147.00                  -40.90\n",
      "   10  19732  13A  14H C   N      97   99 -166.54 -125.60   54.68    1.34  -63.20  176.16   28.56\n",
      "   10         14H  14H N   CA     99  100  175.04  138.80                  -42.30\n",
      "   11  19734  15C  16W C   N     113  115 -169.44 -124.90   53.56    1.72  -63.00  177.99   29.34\n",
      "   11         16W  16W N   CA    115  116  173.14  143.40                  -44.20\n",
      "   12  19735  16W  17D C   N     127  129  170.48  -96.50  107.07    4.44  -63.30 -161.81   32.31\n",
      "   12         17D  17D N   CA    129  130  167.21  114.20                  -40.00\n",
      "   13  19736  17D  18E C   N     135  137  179.20 -117.80   71.04    2.08  -63.60 -169.60   31.64\n",
      "   13         18E  18E N   CA    137  138  169.65  136.80                  -40.30\n",
      "   14  19737  18E  19S C   N     144  146  178.96 -136.60   50.39    1.54  -64.10 -169.76   22.22\n",
      "   14         19S  19S N   CA    146  147  174.95  151.20                  -35.00\n",
      "   15  19739  20V  21M C   N     157  159  178.54 -125.60   64.37    1.92  -63.40 -171.46   36.91\n",
      "   15         21M  21M N   CA    159  160  172.50  140.50                  -40.50\n",
      "   16  19740  21M  22L C   N     165  167 -168.00 -108.50   72.54    3.21  -63.50  178.57   31.13\n",
      "   16         22L  22L N   CA    167  168  173.99  132.50                  -41.20\n",
      "   17  19743  24P  25N C   N     187  189 -175.34 -119.90   67.16    1.97  -63.20 -177.49   28.86\n",
      "   17         25N  25N N   CA    189  190  174.90  137.00                  -41.10\n",
      "   18  19744  25N  26H C   N     195  197 -173.24 -125.60   59.84    1.48  -63.20 -179.80   29.44\n",
      "   18         26H  26H N   CA    197  198  175.00  138.80                  -42.30\n",
      "   19  19745  26H  27R C   N     205  207  175.86 -125.20   65.66    2.00  -63.00 -167.70   33.79\n",
      "   19         27R  27R N   CA    207  208  169.55  140.60                  -41.10\n",
      "   20  19746  27R  28S C   N     216  218  171.72 -136.60   55.62    1.76  -64.10 -162.75   23.24\n",
      "   20         28S  28S N   CA    218  219  171.74  151.20                  -35.00\n",
      "   21  19747  28S  29A C   N     222  224  174.32 -134.00   58.23    1.33  -62.50 -169.53   37.60\n",
      "   21         29A  29A N   CA    224  225  173.82  147.00                  -40.90\n",
      "   22  19749  30V  31H C   N     234  236  171.76 -125.60   72.60    1.85  -63.20 -170.65   31.39\n",
      "   22         31H  31H N   CA    236  237  175.51  138.80                  -42.30\n",
      "   23  19750  31H  32K C   N     244  246  177.64 -118.00   71.33    2.36  -62.90 -168.75   31.81\n",
      "   23         32K  32K N   CA    246  247  169.85  139.10                  -40.80\n",
      "   24  19751  32K  33E C   N     253  255 -179.77 -117.80   72.39    2.12  -63.60 -173.82   31.00\n",
      "   24         33E  33E N   CA    255  256  174.22  136.80                  -40.30\n",
      "   25  19752  33E  34E C   N     262  264 -172.61 -117.80   65.09    1.92  -63.60 -176.35   30.35\n",
      "   25         34E  34E N   CA    264  265  171.90  136.80                  -40.30\n",
      "   26  19754  35C  36A C   N     277  279  178.58 -134.00   53.65    1.23  -62.50 -170.91   37.19\n",
      "   26         36A  36A N   CA    279  280  172.09  147.00                  -40.90\n",
      "   27  19755  36A  37Y C   N     282  284 -161.94 -124.30   61.04    2.50  -63.50  165.57   30.78\n",
      "   27         37Y  37Y N   CA    284  285 -176.54  135.40                  -43.40\n",
      "   28  19758  39C  40R C   N     306  308 -170.93 -125.20   58.27    1.72  -63.00  178.51   31.15\n",
      "   28         40R  40R N   CA    308  309  176.71  140.60                  -41.10\n",
      "   29  19759  40R  41T C   N     317  319 -173.59 -124.80   61.15    2.14  -63.20  176.36   29.38\n",
      "   29         41T  41T N   CA    319  320 -179.64  143.50                  -42.10\n",
      "   30  19761  42C  43R C   N     330  332 -178.64 -125.20   67.73    2.00  -63.00  179.04   31.59\n",
      "   30         43R  43R N   CA    332  333 -177.79  140.60                  -41.10\n",
      "   31  19762  43R  44H C   N     341  343 -165.17 -125.60   57.33    1.43  -63.20  171.12   27.82\n",
      "   31         44H  44H N   CA    343  344 -179.72  138.80                  -42.30\n",
      "   32  19763  44H  45E C   N     351  353 -158.68 -117.80   65.16    2.24  -63.60  162.80   26.84\n",
      "   32         45E  45E N   CA    353  354 -172.46  136.80                  -40.30\n",
      "   33  19764  45E  46M C   N     360  362 -157.71 -125.60   58.60    2.00  -63.40  160.59   31.08\n",
      "   33         46M  46M N   CA    362  363 -170.48  140.50                  -40.50\n",
      "   34  19767  48T  49L C   N     379  381 -120.69 -108.50   66.80    3.47  -63.50  133.49   22.07\n",
      "   34         49L  49L N   CA    381  382 -161.82  132.50                  -41.20\n",
      "   35  19773  54C  55H C   N     420  422 -160.03 -125.60   37.80    1.01  -63.20 -170.13   29.97\n",
      "   35         55H  55H N   CA    422  423  154.38  138.80                  -42.30\n",
      "   36  19774  55H  56M C   N     430  432 -158.59 -125.60   45.57    1.38  -63.40  175.60   33.50\n",
      "   36         56M  56M N   CA    432  433  171.93  140.50                  -40.50\n",
      "   37  19775  56M  57G C   N     438  440  168.23 -167.20   25.93    0.38   82.20 -165.58   13.36\n",
      "   37         57G  57G N   CA    440  441 -177.14  174.60                    8.50\n",
      "   38  19776  57G  58L C   N     442  444  123.43 -108.50  128.50    6.00  -63.50 -113.30   44.38\n",
      "   38         58L  58L N   CA    444  445  143.00  132.50                  -41.20\n",
      "   39  19778  59C  60V C   N     456  458 -177.34 -125.40   80.82    2.40  -73.50  123.05    8.41\n",
      "   39         60V  60V N   CA    458  459 -154.78  143.30                  139.20\n",
      "   40  19779  60V  61E C   N     463  465 -164.21 -117.80   90.29    3.40  -69.30  118.98    6.73\n",
      "   40         61E  61E N   CA    465  466 -145.74  136.80                  142.50\n",
      "   41  19780  61E  62H C   N     472  474   78.94  -67.60  151.35   12.46  -67.60  151.35   12.46\n",
      "   41         62H  62H N   CA    474  475  177.87  140.00                  140.00\n",
      "   42  19787  68L  69N C   N     532  534 -152.14 -119.90   41.51    1.20  -63.20  179.36   27.38\n",
      "   42         69N  69N N   CA    534  535  163.15  137.00                  -41.10\n",
      "   43  19790  71P  72T C   N     553  555 -143.41 -124.80   39.50    1.25  -63.20  160.97   25.79\n",
      "   43         72T  72T N   CA    555  556  178.34  143.50                  -42.10\n",
      "   44  19791  72T  73H C   N     560  562 -126.69 -125.60   83.80    3.17  -63.20  114.36   18.34\n",
      "   44         73H  73H N   CA    562  563 -137.41  138.80                  -42.30\n",
      "   45  19793  74V  75M C   N     577  579   41.88  -73.00  124.37    8.58  -63.40  171.85   33.52\n",
      "   45         75M  75M N   CA    579  580   95.33  143.00                  -40.50\n",
      "   46  19794  75M  76Y C   N     585  587 -147.07 -124.30   63.72    2.59  -63.50  147.61   27.31\n",
      "   46         76Y  76Y N   CA    587  588 -165.08  135.40                  -43.40\n",
      "   47  19796  77V  78W C   N     604  606 -148.49 -124.90   58.37    2.23  -63.00  146.54   24.03\n",
      "   47         78W  78W N   CA    606  607 -163.21  143.40                  -44.20\n",
      "   48  19800  81E  82L C   N     644  646 -151.04 -108.50   43.40    1.98  -70.70   80.35    7.32\n",
      "   48         82L  82L N   CA    646  647  141.08  132.50                  141.60\n",
      "   49  19802  83P  84A C   N     659  661  -72.05  -68.20    3.91    0.31  -62.50  173.42   28.90\n",
      "   49         84A  84A N   CA    661  662  145.95  145.30                  -40.90\n",
      "   50  19803  84A  85K C   N     664  666  -82.32  -70.20   46.70    3.65  -62.90  137.48   16.88\n",
      "   50         85K  85K N   CA    666  667   95.30  140.40                  -40.80\n",
      "   51  19805  86D  87E C   N     681  683   -0.56  -69.30   75.74    6.68  -63.60  158.48   18.52\n",
      "   51         87E  87E N   CA    683  684  174.30  142.50                  -40.30\n",
      "   52  19806  87E  88D C   N     690  692 -111.47  -70.90   82.49    2.77  -63.30  109.10   16.57\n",
      "   52         88D  88D N   CA    692  693 -137.88  150.30                  -40.00\n",
      "   53  19807  88D  89A C   N     698  700 -148.87 -134.00   23.89    0.82  -62.50  176.04   33.46\n",
      "   53         89A  89A N   CA    700  701  165.70  147.00                  -40.90\n",
      "   54  19808  89A  90G C   N     703  705 -119.11  -80.20   38.92    2.58   82.20 -130.20   17.54\n",
      "   54         90G  90G N   CA    705  706  174.71  174.10                    8.50\n",
      "   55  19809  90G  91G C   N     707  709   62.77   78.70   52.19    0.89   82.20  126.40    7.10\n",
      "   55         91G  91G N   CA    709  710 -116.40 -166.10                    8.50\n",
      "   56  19810  91G  92S C   N     711  713 -141.85 -136.60   18.12    1.00  -64.10 -174.11   11.68\n",
      "   56         92S  92S N   CA    713  714  133.85  151.20                  -35.00\n",
      "   57  19811  92S  93K C   N     717  719   78.89   56.60  103.44    9.12  -62.90 -131.19   26.89\n",
      "   57         93K  93K N   CA    719  720  139.61   38.60                  -40.80\n",
      "   58  19812  93K  94D C   N     726  728  127.41   54.50  125.68   14.77  -63.30 -115.27   40.53\n",
      "   58         94D  94D N   CA    728  729  143.27   40.90                  -40.00\n",
      "   59  19815  96N  97K C   N     749  751  138.40 -118.00  106.52    3.66  -62.90 -137.95   37.85\n",
      "   59         97K  97K N   CA    751  752  163.89  139.10                  -40.80\n",
      "   60  19816  97K  98L C   N     758  760  -74.89  -70.70   62.29    4.88  -63.50  121.19   16.21\n",
      "   60         98L  98L N   CA    760  761   79.45  141.60                  -41.20\n",
      "   61  19819 100V 101L C   N     777  779  -71.02  -70.70   51.04    3.85  -63.50  126.38   17.99\n",
      "   61        101L 101L N   CA    779  780 -167.36  141.60                  -41.20\n",
      "   62  19820 101L 102A C   N     785  787  -75.15  -62.50   79.55   13.69  -62.50   79.55   13.69\n",
      "   62        102A 102A N   CA    787  788 -119.44  -40.90                  -40.90\n",
      "   63  19822 103P 104K C   N     797  799  -86.81  -70.20   33.10    2.04  -62.90  152.06   21.13\n",
      "   63        104K 104K N   CA    799  800  169.03  140.40                  -40.80\n",
      "   64  19823 104K 105E C   N     806  808 -178.67 -117.80   61.02    2.13  -63.60 -147.46   34.63\n",
      "   64        105E 105E N   CA    808  809  141.01  136.80                  -40.30\n",
      "   65  19824 105E 106Y C   N     815  817 -141.90 -124.30   27.04    1.14  -63.50  178.78   32.21\n",
      "   65        106Y 106Y N   CA    817  818  155.92  135.40                  -43.40\n",
      "   66  19825 106Y 107E C   N     827  829 -144.65 -117.80   27.80    0.87  -63.60 -166.53   30.34\n",
      "   66        107E 107E N   CA    829  830  144.02  136.80                  -40.30\n",
      "   67  19826 107E 108N C   N     836  838  153.87 -119.90  138.32    4.19  -63.20  160.84   26.42\n",
      "   67        108N 108N N   CA    838  839 -114.85  137.00                  -41.10\n",
      "   68  19827 108N 109A C   N     844  846  -70.91  -68.20   41.05    3.21  -62.50  133.11   22.24\n",
      "   68        109A 109A N   CA    846  847 -173.74  145.30                  -40.90\n",
      "   69  19828 109A 110L C   N     849  851  -52.23  -70.70   23.55    1.57  -63.50  168.58   24.07\n",
      "   69        110L 110L N   CA    851  852  127.00  141.60                  -41.20\n",
      "   70  19831 112C 113A C   N     869  871 -158.69 -134.00   33.34    0.95  -62.50  177.94   34.27\n",
      "   70        113A 113A N   CA    871  872  169.41  147.00                  -40.90\n",
      "   71  19832 113A 114A C   N     874  876 -172.26 -134.00   47.83    1.23  -62.50 -179.41   35.35\n",
      "   71        114A 114A N   CA    876  877  175.70  147.00                  -40.90\n",
      "   72  19834 115C 116A C   N     885  887 -157.94 -134.00   23.95    0.78  -68.20   89.77    7.54\n",
      "   72        116A 116A N   CA    887  888  147.37  147.00                  145.30\n",
      "   73  19836 117K 118S C   N     899  901  -60.15  -64.10   12.24    1.16  -72.40  176.24    9.95\n",
      "   73        118S 118S N   CA    901  902  -23.42  -35.00                  152.40\n",
      "   74  19838 119F 120V C   N     916  918  -59.87  -62.40    3.30    0.56 -125.40 -171.81   10.57\n",
      "   74        120V 120V N   CA    918  919  -40.28  -42.40                  143.30\n",
      "   75  19839 120V 121S C   N     923  925 -145.66 -136.60   89.31    4.70  -64.10  127.00    9.54\n",
      "   75        121S 121S N   CA    925  926   62.35  151.20                  -35.00\n",
      "   76  19840 121S 122P C   N     929  931  -47.66  -58.70   42.55    2.73  -64.50  142.21   11.52\n",
      "   76        122P 122P N   CA    931  932  -71.59  -30.50                  147.20\n",
      "   77  19842 123P 124E C   N     943  945  169.26 -117.80   80.05    2.35  -63.60 -163.42   32.91\n",
      "   77        124E 124E N   CA    945  946  169.77  136.80                  -40.30\n",
      "   78  19843 124E 125L C   N     952  954  -74.15  -70.70   31.58    2.26  -63.50  146.20   20.93\n",
      "   78        125L 125L N   CA    954  955  172.99  141.60                  -41.20\n",
      "   79  19846 127I 128D C   N     975  977  -69.27  -70.90   13.64    0.75  -63.30  156.27   19.65\n",
      "   79        128D 128D N   CA    977  978  163.84  150.30                  -40.00\n",
      "   80  19848 129C 130Y C   N     989  991 -110.40 -124.30   41.91    2.82  -63.50  149.22   26.05\n",
      "   80        130Y 130Y N   CA    991  992  174.94  135.40                  -43.40\n",
      "   81  19849 130Y 131Q C   N    1001 1003  -82.46  -73.00   39.43    2.40  -63.80  141.95   21.87\n",
      "   81        131Q 131Q N   CA   1003 1004  178.98  140.70                  -40.30\n",
      "   82  19850 131Q 132W C   N    1010 1012 -115.27 -124.90   13.09    0.75  -63.00  171.69   24.94\n",
      "   82        132W 132W N   CA   1012 1013  152.26  143.40                  -44.20\n",
      "   83  19853 134I 135N C   N    1040 1042   60.06   55.90    6.74    0.36  -63.20  144.43   23.75\n",
      "   83        135N 135N N   CA   1042 1043   34.19   39.50                  -41.10\n",
      "   84  19854 135N 136A C   N    1048 1050 -177.50 -134.00   64.80    2.07  -62.50  169.18   33.62\n",
      "   84        136A 136A N   CA   1050 1051 -164.98  147.00                  -40.90\n",
      "   85  19855 136A 137T C   N    1053 1055 -113.19 -124.80   15.41    0.93  -63.20  171.72   25.35\n",
      "   85        137T 137T N   CA   1055 1056  153.62  143.50                  -42.10\n",
      "   86  19856 137T 138S C   N    1060 1062  160.29 -136.60   63.74    2.31  -64.10 -146.51   25.24\n",
      "   86        138S 138S N   CA   1062 1063  160.11  151.20                  -35.00\n",
      "   87  19857 138S 139T C   N    1066 1068  157.73 -124.80   99.91    3.42  -63.20  178.14   30.58\n",
      "   87        139T 139T N   CA   1068 1069 -153.42  143.50                  -42.10\n",
      "   88  19858 139T 140G C   N    1073 1075  -52.85  -62.40   12.00    2.26   82.20  141.57   10.70\n",
      "   88        140G 140G N   CA   1075 1076  -33.94  -41.20                    8.50\n",
      "   89  19859 140G 141A C   N    1077 1079 -139.68 -134.00   81.71    4.51  -62.50  119.01   23.50\n",
      "   89        141A 141A N   CA   1079 1080 -131.49  147.00                  -40.90\n",
      "   90  19860 141A 142Q C   N    1082 1084 -113.59 -121.10   45.23    2.26  -63.80  144.26   23.85\n",
      "   90        142Q 142Q N   CA   1084 1085 -175.70  139.70                  -40.30\n",
      "   91  19861 142Q 143A C   N    1091 1093 -146.57 -134.00   47.29    2.33  -62.50  151.90   29.35\n",
      "   91        143A 143A N   CA   1093 1094 -167.41  147.00                  -40.90\n",
      "   92  19862 143A 144A C   N    1096 1098 -137.55 -134.00   33.38    1.81  -62.50  157.89   29.89\n",
      "   92        144A 144A N   CA   1098 1099 -179.81  147.00                  -40.90\n",
      "   93  19864 145I 146E C   N    1109 1111  -51.51  -69.30   24.22    2.23  -63.60  161.22   20.86\n",
      "   93        146E 146E N   CA   1111 1112  158.93  142.50                  -40.30\n",
      "   94  19866 147P 148E C   N    1125 1127  111.27 -117.80  130.93    4.78  -63.60 -110.57   42.23\n",
      "   94        148E 148E N   CA   1127 1128  137.56  136.80                  -40.30\n",
      "   95  19867 148E 149G C   N    1134 1136  137.54 -167.20   60.19    1.78   82.20  152.61   10.00\n",
      "   95        149G 149G N   CA   1136 1137  150.73  174.60                    8.50\n",
      "   96  19869 150V 151A C   N    1145 1147 -177.66 -134.00   47.37    1.08  -62.50 -167.90   37.53\n",
      "   96        151A 151A N   CA   1147 1148  165.35  147.00                  -40.90\n",
      "   97  19870 151A 152S C   N    1150 1152 -155.58 -136.60   19.42    0.67  -64.10 -167.20   20.54\n",
      "   97        152S 152S N   CA   1152 1153  155.29  151.20                  -35.00\n",
      "   98  19871 152S 153M C   N    1156 1158 -171.49 -125.60   55.29    1.63  -63.40 -176.60   35.52\n",
      "   98        153M 153M N   CA   1158 1159  171.34  140.50                  -40.50\n",
      "   99  19872 153M 154R C   N    1164 1166 -163.04 -125.20   55.82    1.73  -63.00  169.84   29.50\n",
      "   99        154R 154R N   CA   1166 1167 -178.36  140.60                  -41.10\n",
      "  100  19873 154R 155L C   N    1175 1177  102.46 -108.50 -127.54   10.65  -70.70 -117.83   24.14\n",
      "  100        155L 155L N   CA   1177 1178  -49.10  132.50                  141.60\n",
      "  101  19877 158P 159H C   N    1205 1207 -164.42 -125.60   58.37    1.47  -63.20  168.98   27.50\n",
      "  101        159H 159H N   CA   1207 1208 -177.61  138.80                  -42.30\n",
      "  102  19878 159H 160L C   N    1215 1217 -142.36 -108.50   46.98    2.11  -63.50  172.77   28.86\n",
      "  102        160L 160L N   CA   1217 1218  165.08  132.50                  -41.20\n",
      "  103  19881 162C 163L C   N    1236 1238  -80.71  -70.70   40.92    3.49  -63.50  144.15   19.10\n",
      "  103        163L 163L N   CA   1238 1239  101.92  141.60                  -41.20\n",
      "  104  19882 163L 164E C   N    1244 1246   72.01  -69.30  141.77   10.33  -63.60 -141.54   36.35\n",
      "  104        164E 164E N   CA   1246 1247  130.97  142.50                  -40.30\n",
      "  105  19883 164E 165Q C   N    1253 1255  110.43 -121.10  129.29    4.57  -63.80 -119.68   31.50\n",
      "  105        165Q 165Q N   CA   1255 1256  154.18  139.70                  -40.30\n",
      "  106  19884 165Q 166L C   N    1262 1264 -138.18 -108.50   51.14    2.38  -63.50  162.79   27.22\n",
      "  106        166L 166L N   CA   1264 1265  174.15  132.50                  -41.20\n",
      "  107  19888 169P 170F C   N    1290 1292 -138.30 -124.20   19.49    0.52  -63.20  175.80   29.27\n",
      "  107        170F 170F N   CA   1292 1293  156.76  143.30                  -44.30\n",
      "  108  19889 170F 171Q C   N    1301 1303 -140.44 -121.10   27.62    0.90  -63.80  177.67   30.12\n",
      "  108        171Q 171Q N   CA   1303 1304  159.41  139.70                  -40.30\n",
      "  109  19890 171Q 172T C   N    1310 1312 -106.67  -78.10   86.81    2.95  -63.20   96.47   15.19\n",
      "  109        172T 172T N   CA   1312 1313 -128.22  149.80                  -42.10\n",
      "  110  19891 172T 173A C   N    1317 1319  -56.60  -68.20   15.12    1.49  -62.50  164.20   26.63\n",
      "  110        173A 173A N   CA   1319 1320  155.00  145.30                  -40.90\n",
      "  111  19893 174P 175T C   N    1329 1331 -160.69 -124.80   80.96    2.58  -63.20  140.98   23.85\n",
      "  111        175T 175T N   CA   1331 1332 -143.94  143.50                  -42.10\n",
      "  112  19894 175T 176S C   N    1336 1338  -95.00  -72.40   48.98    2.27  -64.10  132.79   11.73\n",
      "  112        176S 176S N   CA   1338 1339 -164.14  152.40                  -35.00\n",
      "  113  19895 176S 177S C   N    1342 1344 -124.30 -136.60   34.21    1.92  -64.10  154.12   15.45\n",
      "  113        177S 177S N   CA   1344 1345 -176.88  151.20                  -35.00\n",
      "  114  19896 177S 178D C   N    1348 1350 -135.07  -63.30   89.34   15.15  -63.30   89.34   15.15\n",
      "  114        178D 178D N   CA   1350 1351  -93.20  -40.00                  -40.00\n",
      "  115  19900 181A 182L C   N    1376 1378 -158.26 -108.50 -173.55    9.48  -70.70 -168.25   11.98\n",
      "  115        182L 182L N   CA   1378 1379  -47.81  132.50                  141.60\n",
      "  116  19901 182L 183D C   N    1384 1386 -170.83  -70.90  133.10    5.74  -63.30  135.09   22.89\n",
      "  116        183D 183D N   CA   1386 1387 -121.78  150.30                  -40.00\n",
      "  117  19902 183D 184G C   N    1392 1394 -135.86  -62.40   74.35   12.77   82.20  147.00    8.22\n",
      "  117        184G 184G N   CA   1394 1395  -29.73  -41.20                    8.50\n",
      "  118  19904 185C 186E C   N    1402 1404 -155.39 -117.80   37.94    1.27  -63.60 -159.91   31.79\n",
      "  118        186E 186E N   CA   1404 1405  141.91  136.80                  -40.30\n",
      "  119  19906 187C 188R C   N    1417 1419 -148.34 -125.20   39.46    1.31  -63.00  169.40   28.62\n",
      "  119        188R 188R N   CA   1419 1420  172.56  140.60                  -41.10\n",
      "  120  19977 258H 259F C   N    1967 1969 -110.74  -63.20   50.78    7.01  -63.20   50.78    7.01\n",
      "  120        259F 259F N   CA   1969 1970  -26.46  -44.30                  -44.30\n",
      "  121  19982 263I 264Q C   N    2006 2008  143.95 -121.10   99.71    3.21  -63.80 -146.59   38.44\n",
      "  121        264Q 264Q N   CA   2008 2009  170.15  139.70                  -40.30\n",
      "  122  19983 264Q 265T C   N    2015 2017  176.30 -124.80   64.69    2.65  -63.20 -169.41   31.80\n",
      "  122        265T 265T N   CA   2017 2018  170.24  143.50                  -42.10\n",
      "  123  19984 265T 266A C   N    2022 2024  164.85 -134.00   66.27    1.51  -62.50 -162.32   39.22\n",
      "  123        266A 266A N   CA   2024 2025  172.54  147.00                  -40.90\n",
      "  124  19985 266A 267K C   N    2027 2029  176.35 -118.00   71.94    2.39  -62.90 -166.90   32.13\n",
      "  124        267K 267K N   CA   2029 2030  168.51  139.10                  -40.80\n",
      "  125  19986 267K 268K C   N    2036 2038 -176.63 -118.00   64.59    2.14  -62.90 -169.36   31.44\n",
      "  125        268K 268K N   CA   2038 2039  166.20  139.10                  -40.80\n",
      "  126  19987 268K 269T C   N    2045 2047 -174.98 -124.80   59.19    2.20  -63.20 -178.48   30.15\n",
      "  126        269T 269T N   CA   2047 2048  174.88  143.50                  -42.10\n",
      "  127  19988 269T 270A C   N    2052 2054  177.30 -134.00   54.31    1.24  -62.50 -169.28   37.52\n",
      "  127        270A 270A N   CA   2054 2055  171.03  147.00                  -40.90\n",
      "  128  19989 270A 271K C   N    2057 2059  178.18 -118.00   70.49    2.33  -62.90 -168.44   31.83\n",
      "  128        271K 271K N   CA   2059 2060  169.03  139.10                  -40.80\n",
      "  129  19990 271K 272T C   N    2066 2068 -173.31 -124.80   61.26    2.13  -63.20  175.77   29.28\n",
      "  129        272T 272T N   CA   2068 2069 -179.10  143.50                  -42.10\n",
      "  130  19991 272T 273L C   N    2073 2075  176.76 -108.50   84.20    3.72  -63.50 -170.00   33.56\n",
      "  130        273L 273L N   CA   2075 2076  171.28  132.50                  -41.20\n",
      "  131  19992 273L 274G C   N    2081 2083  164.66 -167.20   28.76    0.70   82.20 -179.86   12.48\n",
      "  131        274G 274G N   CA   2083 2084  168.66  174.60                    8.50\n",
      "  132  19993 274G 275E C   N    2085 2087  172.18 -117.80   75.65    2.24  -63.60 -161.95   33.00\n",
      "  132        275E 275E N   CA   2087 2088  165.45  136.80                  -40.30\n",
      "  133  19994 275E 276M C   N    2094 2096  177.95 -125.60   63.97    1.92  -63.40 -169.60   37.25\n",
      "  133        276M 276M N   CA   2096 2097  170.59  140.50                  -40.50\n",
      "  134  19995 276M 277Q C   N    2102 2104 -177.28 -121.10   60.76    1.91  -63.80 -166.40   34.11\n",
      "  134        277Q 277Q N   CA   2104 2105  162.85  139.70                  -40.30\n",
      "  135  19996 277Q 278Y C   N    2111 2113 -172.94 -124.30   59.60    3.02  -63.50 -176.92   34.05\n",
      "  135        278Y 278Y N   CA   2113 2114  169.83  135.40                  -43.40\n",
      "  136  19997 278Y 279D C   N    2123 2125 -172.93  -96.50   90.28    3.75  -63.30 -167.90   30.65\n",
      "  136        279D 279D N   CA   2125 2126  162.25  114.20                  -40.00\n",
      "  137  19998 279D 280Y C   N    2131 2133 -164.99 -124.30   60.43    2.59  -63.50  170.12   31.63\n",
      "  137        280Y 280Y N   CA   2133 2134 -179.93  135.40                  -43.40\n",
      "  138  19999 280Y 281S C   N    2143 2145 -168.35 -136.60   40.54    1.25  -64.10 -178.48   20.68\n",
      "  138        281S 281S N   CA   2145 2146  176.40  151.20                  -35.00\n",
      "  139  20001 282C 283Q C   N    2155 2157 -170.40 -121.10   55.85    1.72  -63.80 -172.90   32.85\n",
      "  139        283Q 283Q N   CA   2157 2158  165.94  139.70                  -40.30\n",
      "  140  20002 283Q 284F C   N    2164 2166 -174.73 -124.20   59.04    1.55  -63.20 -179.55   31.73\n",
      "  140        284F 284F N   CA   2166 2167  173.84  143.30                  -44.30\n",
      "  141  20003 284F 285D C   N    2175 2177 -158.26  -96.50   84.14    3.52  -63.30  176.40   27.82\n",
      "  141        285D 285D N   CA   2177 2178  171.35  114.20                  -40.00\n",
      "  142  20004 285D 286F C   N    2183 2185 -165.49 -124.20   55.66    1.47  -63.20  169.44   29.69\n",
      "  142        286F 286F N   CA   2185 2186 -179.38  143.30                  -44.30\n",
      "  143  20005 286F 287K C   N    2194 2196 -127.32 -118.00   38.11    1.70  -62.90  156.97   24.24\n",
      "  143        287K 287K N   CA   2196 2197  176.06  139.10                  -40.80\n",
      "  144  20006 287K 288R C   N    2203 2205 -156.78 -125.20   38.37    1.13  -63.00 -177.55   30.93\n",
      "  144        288R 288R N   CA   2205 2206  162.39  140.60                  -41.10\n",
      "  145  20008 289I 290T C   N    2222 2224 -156.34 -124.80   49.02    1.54  -63.20  165.57   27.08\n",
      "  145        290T 290T N   CA   2224 2225 -178.98  143.50                  -42.10\n",
      "  146  20009 290T 291E C   N    2229 2231 -154.40 -117.80   55.31    1.84  -63.60  168.07   27.37\n",
      "  146        291E 291E N   CA   2231 2232  178.27  136.80                  -40.30\n",
      "  147  20010 291E 292N C   N    2238 2240 -147.34 -119.90   49.51    1.57  -63.20  163.93   25.17\n",
      "  147        292N 292N N   CA   2240 2241  178.21  137.00                  -41.10\n",
      "  148  20011 292N 293G C   N    2246 2248 -175.93 -167.20   12.43    0.47   82.20 -172.64   13.55\n",
      "  148        293G 293G N   CA   2248 2249  165.75  174.60                    8.50\n",
      "  149  20012 293G 294D C   N    2250 2252 -162.09  -96.50   79.77    3.32  -63.30 -171.61   29.56\n",
      "  149        294D 294D N   CA   2252 2253  159.59  114.20                  -40.00\n",
      "  150  20013 294D 295S C   N    2258 2260 -164.56 -136.60   31.38    0.96  -64.10 -171.45   20.91\n",
      "  150        295S 295S N   CA   2260 2261  165.44  151.20                  -35.00\n",
      "  151  20014 295S 296L C   N    2264 2266 -161.63 -108.50   55.27    2.48  -63.50 -162.80   33.42\n",
      "  151        296L 296L N   CA   2266 2267  147.75  132.50                  -41.20\n",
      "  152  20017 298P 299A C   N    2286 2288 -161.21 -134.00   30.65    0.70  -62.50 -173.70   35.78\n",
      "  152        299A 299A N   CA   2288 2289  161.11  147.00                  -40.90\n",
      "  153  20018 299A 300F C   N    2291 2293 -164.45 -124.20   44.89    1.23  -63.20 -176.92   31.68\n",
      "  153        300F 300F N   CA   2293 2294  163.17  143.30                  -44.30\n",
      "  154  20019 300F 301G C   N    2302 2304 -175.71 -167.20   38.76    1.59   82.20  163.94   12.24\n",
      "  154        301G 301G N   CA   2304 2305  136.78  174.60                    8.50\n",
      "  155  20021 302P 303G C   N    2313 2315 -168.21 -167.20   13.54    0.54   82.20 -172.13   13.82\n",
      "  155        303G 303G N   CA   2315 2316  161.09  174.60                    8.50\n",
      "  156  20022 303G 304R C   N    2317 2319 -154.87 -125.20   40.83    1.23  -63.00  176.12   29.93\n",
      "  156        304R 304R N   CA   2319 2320  168.65  140.60                  -41.10\n",
      "  157  20023 304R 305T C   N    2328 2330 -149.10 -124.80   43.55    1.35  -63.20  162.78   26.34\n",
      "  157        305T 305T N   CA   2330 2331  179.64  143.50                  -42.10\n",
      "  158  20024 305T 306G C   N    2335 2337 -175.92 -167.20   14.35    0.56   82.20 -174.77   13.43\n",
      "  158        306G 306G N   CA   2337 2338  163.21  174.60                    8.50\n",
      "  159  20025 306G 307M C   N    2339 2341 -161.32 -125.60   43.35    1.28  -63.40 -177.15   34.81\n",
      "  159        307M 307M N   CA   2341 2342  165.07  140.50                  -40.50\n",
      "  160  20026 307M 308H C   N    2347 2349 -154.30 -125.60   38.80    0.95  -63.20  177.89   28.10\n",
      "  160        308H 308H N   CA   2349 2350  164.91  138.80                  -42.30\n",
      "  161  20027 308H 309N C   N    2357 2359 -153.65 -119.90   42.37    1.23  -63.20 -179.42   27.61\n",
      "  161        309N 309N N   CA   2359 2360  162.61  137.00                  -41.10\n",
      "  162  20028 309N 310F C   N    2365 2367 -164.10 -124.20   44.97    1.22  -63.20 -177.84   31.52\n",
      "  162        310F 310F N   CA   2367 2368  164.04  143.30                  -44.30\n",
      "  163  20029 310F 311G C   N    2376 2378  176.76 -167.20   24.16    0.93   82.20  175.65   12.67\n",
      "  163        311G 311G N   CA   2378 2379  156.52  174.60                    8.50\n",
      "  164  20030 311G 312N C   N    2380 2382 -162.06 -119.90   47.96    1.46  -63.20 -172.74   28.88\n",
      "  164        312N 312N N   CA   2382 2383  159.86  137.00                  -41.10\n",
      "  165  20031 312N 313S C   N    2388 2390 -161.22 -136.60   27.86    0.85  -64.10 -172.18   20.61\n",
      "  165        313S 313S N   CA   2390 2391  164.24  151.20                  -35.00\n",
      "  166  20033 314C 315Y C   N    2400 2402 -144.26 -124.30   29.20    1.26  -63.50  179.12   32.36\n",
      "  166        315Y 315Y N   CA   2402 2403  156.71  135.40                  -43.40\n",
      "  167  20034 315Y 316M C   N    2412 2414 -154.36 -125.60   34.02    1.01  -63.40 -175.23   34.64\n",
      "  167        316M 316M N   CA   2414 2415  158.67  140.50                  -40.50\n",
      "  168  20035 316M 317A C   N    2420 2422 -166.87 -134.00   35.58    0.81  -62.50 -170.24   36.63\n",
      "  168        317A 317A N   CA   2422 2423  160.62  147.00                  -40.90\n",
      "  169  20036 317A 318S C   N    2425 2427 -162.09 -136.60   28.05    0.87  -64.10 -170.59   20.79\n",
      "  169        318S 318S N   CA   2427 2428  162.91  151.20                  -35.00\n",
      "  170  20038 319V 320L C   N    2438 2440 -154.19 -108.50   46.79    2.12  -63.50 -161.85   33.12\n",
      "  170        320L 320L N   CA   2440 2441  142.62  132.50                  -41.20\n",
      "  171  20039 320L 321Q C   N    2446 2448 -147.70 -121.10   33.00    1.03  -63.80 -178.93   30.97\n",
      "  171        321Q 321Q N   CA   2448 2449  159.24  139.70                  -40.30\n",
      "  172  20041 322C 323L C   N    2461 2463 -146.92 -108.50   41.15    1.83  -63.50 -169.24   31.65\n",
      "  172        323L 323L N   CA   2463 2464  147.24  132.50                  -41.20\n",
      "  173  20042 323L 324F C   N    2469 2471 -143.06 -124.20   22.72    0.59  -63.20  178.57   29.93\n",
      "  173        324F 324F N   CA   2471 2472  155.98  143.30                  -44.30\n",
      "  174  20043 324F 325S C   N    2480 2482 -151.79 -136.60   16.52    0.52  -64.10 -171.10   19.98\n",
      "  174        325S 325S N   CA   2482 2483  157.69  151.20                  -35.00\n",
      "  175  20044 325S 326L C   N    2486 2488 -147.02 -108.50   39.79    1.80  -63.50 -164.91   32.26\n",
      "  175        326L 326L N   CA   2488 2489  142.49  132.50                  -41.20\n",
      "  176  20045 326L 327E C   N    2494 2496 -135.18 -117.80   17.38    0.64  -63.60 -169.04   22.42\n",
      "  176        327E 327E N   CA   2496 2497  136.73  136.80                  -40.30\n",
      "  177  20047 328P 329F C   N    2510 2512 -146.56 -124.20   26.43    0.69  -63.20  178.91   30.17\n",
      "  177        329F 329F N   CA   2512 2513  157.40  143.30                  -44.30\n",
      "  178  20048 329F 330K C   N    2521 2523 -147.68 -118.00   32.68    1.08  -62.90 -173.24   29.32\n",
      "  178        330K 330K N   CA   2523 2524  152.79  139.10                  -40.80\n",
      "  179  20049 330K 331E C   N    2530 2532 -155.63 -117.80   41.53    1.22  -63.60 -170.41   30.38\n",
      "  179        331E 331E N   CA   2532 2533  153.94  136.80                  -40.30\n",
      "  180  20050 331E 332A C   N    2539 2541 -153.93 -134.00   21.63    0.49  -62.50 -172.49   35.61\n",
      "  180        332A 332A N   CA   2541 2542  155.39  147.00                  -40.90\n",
      "  181  20051 332A 333F C   N    2544 2546 -150.34 -124.20   28.01    0.81  -63.20 -175.76   31.14\n",
      "  181        333F 333F N   CA   2546 2547  153.36  143.30                  -44.30\n",
      "  182  20052 333F 334Y C   N    2555 2557 -138.04 -124.30   17.92    0.85  -63.50 -174.65   33.10\n",
      "  182        334Y 334Y N   CA   2557 2558  146.90  135.40                  -43.40\n",
      "  183  20053 334Y 335Y C   N    2567 2569 -138.33 -124.30   22.24    0.92  -63.50 -179.78   32.29\n",
      "  183        335Y 335Y N   CA   2569 2570  152.65  135.40                  -43.40\n",
      "  184  20054 335Y 336N C   N    2579 2581 -147.43 -119.90   32.03    0.96  -63.20 -174.27   27.95\n",
      "  184        336N 336N N   CA   2581 2582  153.37  137.00                  -41.10\n",
      "  185  20056 337C 338E C   N    2593 2595 -155.24 -117.80   40.02    1.20  -63.60 -167.97   30.69\n",
      "  185        338E 338E N   CA   2595 2596  150.95  136.80                  -40.30\n",
      "  186  20057 338E 339T C   N    2602 2604 -145.02 -124.80   25.71    0.89  -63.20  178.39   28.18\n",
      "  186        339T 339T N   CA   2604 2605  159.38  143.50                  -42.10\n",
      "  187  20058 339T 340R C   N    2609 2611 -155.86 -125.20   34.43    1.04  -63.00 -172.73   31.55\n",
      "  187        340R 340R N   CA   2611 2612  156.28  140.60                  -41.10\n",
      "  188  20059 340R 341H C   N    2620 2622 -158.63 -125.60   36.09    0.97  -63.20 -169.95   29.91\n",
      "  188        341H 341H N   CA   2622 2623  153.34  138.80                  -42.30\n",
      "  189  20060 341H 342Q C   N    2630 2632 -156.29 -121.10   37.59    1.19  -63.80 -169.29   32.81\n",
      "  189        342Q 342Q N   CA   2632 2633  152.92  139.70                  -40.30\n",
      "  190  20061 342Q 343N C   N    2639 2641 -150.95 -119.90   35.60    1.08  -63.20 -173.56   28.23\n",
      "  190        343N 343N N   CA   2641 2642  154.41  137.00                  -41.10\n",
      "  191  20062 343N 344A C   N    2647 2649 -160.20 -134.00   27.75    0.65  -62.50 -170.00   36.34\n",
      "  191        344A 344A N   CA   2649 2650  156.15  147.00                  -40.90\n",
      "  192  20064 345C 346Q C   N    2658 2660 -155.23 -121.10   37.39    1.17  -63.80 -171.59   32.41\n",
      "  192        346Q 346Q N   CA   2660 2661  154.96  139.70                  -40.30\n",
      "  193  20065 346Q 347E C   N    2667 2669 -149.05 -117.80   34.23    1.01  -63.60 -170.70   30.01\n",
      "  193        347E 347E N   CA   2669 2670  150.78  136.80                  -40.30\n",
      "  194  20066 347E 348N C   N    2676 2678 -140.73 -119.90   20.84    0.86  -63.20 -165.46   21.14\n",
      "  194        348N 348N N   CA   2678 2679  137.32  137.00                  -41.10\n",
      "  195  20068 349P 350Y C   N    2691 2693 -138.77 -124.30   24.33    0.99  -63.50  178.30   32.00\n",
      "  195        350Y 350Y N   CA   2693 2694  154.96  135.40                  -43.40\n",
      "  196  20069 350Y 351K C   N    2703 2705 -138.54 -118.00   25.06    0.84  -62.90 -177.82   28.19\n",
      "  196        351K 351K N   CA   2705 2706  153.46  139.10                  -40.80\n",
      "  197  20071 352C 353H C   N    2718 2720 -156.25 -125.60   32.78    0.91  -63.20 -168.58   29.95\n",
      "  197        353H 353H N   CA   2720 2721  150.42  138.80                  -42.30\n",
      "  198  20074 355C 356Q C   N    2740 2742 -156.71 -121.10   37.46    1.20  -63.80 -167.71   33.06\n",
      "  198        356Q 356Q N   CA   2742 2743  151.34  139.70                  -40.30\n",
      "  199  20075 356Q 357T C   N    2749 2751 -144.46 -124.80   24.48    0.86  -63.20  179.29   28.26\n",
      "  199        357T 357T N   CA   2751 2752  158.08  143.50                  -42.10\n",
      "  200  20076 357T 358E C   N    2756 2758 -154.32 -117.80   40.20    1.18  -63.60 -170.74   30.27\n",
      "  200        358E 358E N   CA   2758 2759  153.60  136.80                  -40.30\n",
      "  201  20077 358E 359R C   N    2765 2767 -150.84 -125.20   29.90    0.89  -63.00 -174.90   30.94\n",
      "  201        359R 359R N   CA   2767 2768  155.97  140.60                  -41.10\n",
      "  202  20079 360V 361A C   N    2783 2785 -156.92 -134.00   25.26    0.57  -62.50 -172.94   35.69\n",
      "  202        361A 361A N   CA   2785 2786  157.62  147.00                  -40.90\n",
      "  203  20080 361A 362S C   N    2788 2790 -162.53 -136.60   31.78    0.97  -64.10 -176.02   20.43\n",
      "  203        362S 362S N   CA   2790 2791  169.56  151.20                  -35.00\n",
      "  204  20081 362S 363G C   N    2794 2796  177.54 -167.20   27.70    1.11   82.20  171.85   12.49\n",
      "  204        363G 363G N   CA   2796 2797  151.48  174.60                    8.50\n",
      "  205  20082 363G 364L C   N    2798 2800 -140.40 -108.50   36.56    1.61  -63.50 -174.85   30.47\n",
      "  205        364L 364L N   CA   2800 2801  150.38  132.50                  -41.20\n",
      "  206  20083 364L 365L C   N    2806 2808 -133.70 -108.50   28.42    1.25  -63.50 -173.14   30.29\n",
      "  206        365L 365L N   CA   2808 2809  145.63  132.50                  -41.20\n",
      "  207  20084 365L 366S C   N    2814 2816 -154.79 -136.60   23.67    0.74  -64.10 -177.26   19.77\n",
      "  207        366S 366S N   CA   2816 2817  166.35  151.20                  -35.00\n",
      "  208  20085 366S 367G C   N    2820 2822  174.78 -167.20   24.86    0.92   82.20  175.39   12.59\n",
      "  208        367G 367G N   CA   2822 2823  157.46  174.60                    8.50\n",
      "  209  20086 367G 368E C   N    2824 2826 -151.97 -117.80   38.87    1.14  -63.60 -173.38   29.79\n",
      "  209        368E 368E N   CA   2826 2827  155.33  136.80                  -40.30\n",
      "  210  20087 368E 369F C   N    2833 2835 -149.59 -124.20   27.28    0.79  -63.20 -176.04   31.06\n",
      "  210        369F 369F N   CA   2835 2836  153.29  143.30                  -44.30\n",
      "  211  20088 369F 370S C   N    2844 2846 -162.51 -136.60   27.49    0.88  -64.10 -168.22   20.99\n",
      "  211        370S 370S N   CA   2846 2847  160.39  151.20                  -35.00\n",
      "  212  20090 371V 372E C   N    2857 2859 -153.45 -117.80   43.65    1.30  -63.60 -178.49   29.17\n",
      "  212        372E 372E N   CA   2859 2860  161.98  136.80                  -40.30\n",
      "  213  20091 372E 373G C   N    2866 2868  176.82 -167.20   26.00    1.02   82.20  173.63   12.56\n",
      "  213        373G 373G N   CA   2868 2869  154.09  174.60                    8.50\n",
      "  214  20092 373G 374Q C   N    2870 2872 -155.00 -121.10   38.44    1.19  -63.80 -174.19   32.01\n",
      "  214        374Q 374Q N   CA   2872 2873  157.81  139.70                  -40.30\n",
      "  215  20093 374Q 375E C   N    2879 2881 -144.88 -117.80   29.39    0.87  -63.60 -170.22   29.85\n",
      "  215        375E 375E N   CA   2881 2882  148.21  136.80                  -40.30\n",
      "  216  20094 375E 376L C   N    2888 2890 -138.85 -108.50   33.77    1.49  -63.50 -172.69   30.67\n",
      "  216        376L 376L N   CA   2890 2891  147.31  132.50                  -41.20\n",
      "  217  20095 376L 377T C   N    2896 2898 -138.16 -124.80   16.80    0.59  -63.20 -179.48   28.05\n",
      "  217        377T 377T N   CA   2898 2899  153.68  143.50                  -42.10\n",
      "  218  20096 377T 378N C   N    2903 2905 -156.01 -119.90   44.93    1.31  -63.20 -179.18   27.76\n",
      "  218        378N 378N N   CA   2905 2906  163.73  137.00                  -41.10\n",
      "  219  20097 378N 379G C   N    2911 2913  174.54 -167.20   37.08    1.50   82.20  162.59   11.88\n",
      "  219        379G 379G N   CA   2913 2914  142.33  174.60                    8.50\n",
      "  220  20099 380I 381T C   N    2923 2925 -141.59 -124.80   20.40    0.73  -63.20 -179.30   28.28\n",
      "  220        381T 381T N   CA   2925 2926  155.09  143.50                  -42.10\n",
      "  221  20100 381T 382A C   N    2930 2932 -159.83 -134.00   26.80    0.66  -62.50 -168.45   36.58\n",
      "  221        382A 382A N   CA   2932 2933  154.12  147.00                  -40.90\n",
      "  222  20101 382A 383R C   N    2935 2937 -154.31 -125.20   31.36    0.99  -63.00 -170.00   31.83\n",
      "  222        383R 383R N   CA   2937 2938  152.28  140.60                  -41.10\n",
      "  223  20102 383R 384E C   N    2946 2948 -154.52 -117.80   38.50    1.17  -63.60 -166.06   30.91\n",
      "  223        384E 384E N   CA   2948 2949  148.39  136.80                  -40.30\n",
      "  224  20103 384E 385F C   N    2955 2957 -152.15 -124.20   29.72    0.88  -63.20 -174.92   31.35\n",
      "  224        385F 385F N   CA   2957 2958  153.40  143.30                  -44.30\n",
      "  225  20104 385F 386K C   N    2966 2968 -158.31 -118.00   41.69    1.42  -62.90 -165.51   30.95\n",
      "  225        386K 386K N   CA   2968 2969  149.73  139.10                  -40.80\n",
      "  226  20105 386K 387R C   N    2975 2977 -151.66 -125.20   29.10    0.90  -63.00 -171.65   31.44\n",
      "  226        387R 387R N   CA   2977 2978  152.72  140.60                  -41.10\n",
      "  227  20107 388V 389F C   N    2993 2995 -148.24 -124.20   28.04    0.74  -63.20  179.40   30.33\n",
      "  227        389F 389F N   CA   2995 2996  157.74  143.30                  -44.30\n",
      "  228  20108 389F 390A C   N    3004 3006 -151.88 -134.00   19.79    0.45  -62.50 -173.57   35.33\n",
      "  228        390A 390A N   CA   3006 3007  155.50  147.00                  -40.90\n",
      "  229  20109 390A 391Q C   N    3009 3011 -144.78 -121.10   28.55    0.88  -63.80 -177.05   31.12\n",
      "  229        391Q 391Q N   CA   3011 3012  155.65  139.70                  -40.30\n",
      "  230  20110 391Q 392K C   N    3018 3020 -137.99 -118.00   23.59    0.79  -62.90 -176.38   28.34\n",
      "  230        392K 392K N   CA   3020 3021  151.64  139.10                  -40.80\n",
      "  231  20111 392K 393H C   N    3027 3029 -140.03 -125.60   14.46    0.49  -63.20 -166.10   29.26\n",
      "  231        393H 393H N   CA   3029 3030  139.67  138.80                  -42.30\n",
      "  232  20113 394P 395D C   N    3044 3046 -134.35  -96.50   45.27    1.88  -63.30 -167.39   20.64\n",
      "  232        395D 395D N   CA   3046 3047  139.02  114.20                  -40.00\n",
      "  233  20114 395D 396F C   N    3052 3054 -138.00 -124.20   14.28    0.45  -63.20 -175.46   30.49\n",
      "  233        396F 396F N   CA   3054 3055  147.00  143.30                  -44.30\n",
      "  234  20115 396F 397S C   N    3063 3065 -156.85 -136.60   20.57    0.73  -64.10 -166.14   20.71\n",
      "  234        397S 397S N   CA   3065 3066  154.77  151.20                  -35.00\n",
      "  235  20116 397S 398T C   N    3069 3071 -143.76 -124.80   22.97    0.83  -63.20 -179.59   28.37\n",
      "  235        398T 398T N   CA   3071 3072  156.47  143.50                  -42.10\n",
      "  236  20117 398T 399A C   N    3076 3078 -157.93 -134.00   24.86    0.61  -62.50 -169.06   36.38\n",
      "  236        399A 399A N   CA   3078 3079  153.73  147.00                  -40.90\n",
      "  237  20118 399A 400E C   N    3081 3083 -152.65 -117.80   36.59    1.11  -63.60 -166.52   30.76\n",
      "  237        400E 400E N   CA   3083 3084  147.94  136.80                  -40.30\n",
      "  238  20119 400E 401Q C   N    3090 3092 -153.64 -121.10   34.58    1.10  -63.80 -169.22   32.70\n",
      "  238        401Q 401Q N   CA   3092 3093  151.40  139.70                  -40.30\n",
      "  239  20120 401Q 402Q C   N    3099 3101 -144.96 -121.10   24.29    0.83  -63.80 -166.70   32.67\n",
      "  239        402Q 402Q N   CA   3101 3102  144.26  139.70                  -40.30\n",
      "  240  20121 402Q 403D C   N    3108 3110 -145.25  -96.50   51.64    2.13  -63.30 -170.17   20.17\n",
      "  240        403D 403D N   CA   3110 3111  131.23  114.20                  -40.00\n",
      "  241  20122 403D 404A C   N    3116 3118 -143.07 -134.00    9.19    0.25  -62.50 -171.34   35.23\n",
      "  241        404A 404A N   CA   3118 3119  148.51  147.00                  -40.90\n",
      "  242  20123 404A 405Q C   N    3121 3123 -153.14 -121.10   34.40    1.09  -63.80 -170.21   32.53\n",
      "  242        405Q 405Q N   CA   3123 3124  152.24  139.70                  -40.30\n",
      "  243  20124 405Q 406E C   N    3130 3132 -146.12 -117.80   29.33    0.91  -63.60 -166.27   30.45\n",
      "  243        406E 406E N   CA   3132 3133  144.42  136.80                  -40.30\n",
      "  244  20125 406E 407Y C   N    3139 3141 -136.55 -124.30   17.35    0.77  -63.50 -175.97   32.82\n",
      "  244        407Y 407Y N   CA   3141 3142  147.68  135.40                  -43.40\n",
      "  245  20126 407Y 408F C   N    3151 3153 -143.46 -124.20   20.03    0.62  -63.20 -174.83   30.89\n",
      "  245        408F 408F N   CA   3153 3154  148.83  143.30                  -44.30\n",
      "  246  20127 408F 409L C   N    3162 3164 -138.27 -108.50   30.25    1.38  -63.50 -165.97   23.91\n",
      "  246        409L 409L N   CA   3164 3165  137.85  132.50                  -41.20\n",
      "  247  20128 409L 410Y C   N    3170 3172 -132.65 -124.30    8.53    0.68  -63.50 -169.89   27.07\n",
      "  247        410Y 410Y N   CA   3172 3173  133.69  135.40                  -43.40\n",
      "  248  20129 410Y 411L C   N    3182 3184 -143.90 -108.50   37.10    1.66  -63.50 -167.25   31.74\n",
      "  248        411L 411L N   CA   3184 3185  143.62  132.50                  -41.20\n",
      "  249  20131 412V 413E C   N    3197 3199 -153.33 -117.80   36.97    1.14  -63.60 -165.40   30.94\n",
      "  249        413E 413E N   CA   3199 3200  147.02  136.80                  -40.30\n",
      "  250  20132 413E 414Q C   N    3206 3208 -150.67 -121.10   31.49    1.00  -63.80 -169.83   32.47\n",
      "  250        414Q 414Q N   CA   3208 3209  150.53  139.70                  -40.30\n",
      "  251  20133 414Q 415M C   N    3215 3217 -151.60 -125.60   28.86    0.88  -63.40 -171.60   35.01\n",
      "  251        415M 415M N   CA   3217 3218  153.02  140.50                  -40.50\n",
      "  252  20134 415M 416R C   N    3223 3225 -150.81 -125.20   28.54    0.87  -63.00 -172.49   31.27\n",
      "  252        416R 416R N   CA   3225 3226  153.22  140.60                  -41.10\n",
      "  253  20135 416R 417R C   N    3234 3236 -143.26 -125.20   18.33    0.66  -63.00 -167.31   31.51\n",
      "  253        417R 417R N   CA   3236 3237  143.73  140.60                  -41.10\n",
      "  254  20136 417R 418Y C   N    3245 3247 -137.49 -124.30   20.50    0.86  -63.50 -178.70   32.43\n",
      "  254        418Y 418Y N   CA   3247 3248  151.09  135.40                  -43.40\n",
      "  255  20138 419V 420K C   N    3264 3266 -134.39 -118.00   16.56    0.59  -62.90 -168.47   29.15\n",
      "  255        420K 420K N   CA   3266 3267  141.51  139.10                  -40.80\n",
      "  256  20140 421P 422S C   N    3280 3282 -144.91 -136.60    9.37    0.29  -64.10 -172.25   19.38\n",
      "  256        422S 422S N   CA   3282 3283  155.53  151.20                  -35.00\n",
      "  257  20141 422S 423N C   N    3286 3288 -138.81 -119.90   21.55    0.66  -63.20 -172.51   27.72\n",
      "  257        423N 423N N   CA   3288 3289  147.33  137.00                  -41.10\n",
      "  258  20144 425V 426G C   N    3309 3311  169.32 -167.20   29.00    1.00   82.20  172.66   12.26\n",
      "  258        426G 426G N   CA   3311 3312  157.57  174.60                    8.50\n",
      "  259  20145 426G 427A C   N    3313 3315 -155.43 -134.00   23.10    0.53  -62.50 -171.94   35.78\n",
      "  259        427A 427A N   CA   3315 3316  155.62  147.00                  -40.90\n",
      "  260  20146 427A 428N C   N    3318 3320 -137.23 -119.90   17.98    0.63  -63.20 -168.01   28.20\n",
      "  260        428N 428N N   CA   3320 3321  141.77  137.00                  -41.10\n",
      "  261  20148 429I 430S C   N    3334 3336 -143.72 -136.60    7.59    0.24  -64.10 -171.23   19.36\n",
      "  261        430S 430S N   CA   3336 3337  153.83  151.20                  -35.00\n",
      "  262  20149 430S 431H C   N    3340 3342 -141.42 -125.60   16.23    0.64  -63.20 -166.05   20.76\n",
      "  262        431H 431H N   CA   3342 3343  135.17  138.80                  -42.30\n",
      "  263  20152 433V 434D C   N    3364 3366 -125.33  -96.50   33.42    1.39  -63.30 -178.00   19.62\n",
      "  263        434D 434D N   CA   3366 3367  131.10  114.20                  -40.00\n",
      "  264  20154 435I 436F C   N    3380 3382 -136.32 -124.20   12.60    0.39  -63.20 -175.90   30.33\n",
      "  264        436F 436F N   CA   3382 3383  146.74  143.30                  -44.30\n",
      "  265  20155 436F 437K C   N    3391 3393 -147.50 -118.00   32.50    1.08  -62.90 -173.29   29.31\n",
      "  265        437K 437K N   CA   3393 3394  152.76  139.10                  -40.80\n",
      "  266  20156 437K 438M C   N    3400 3402 -142.61 -125.60   19.59    0.58  -63.40 -173.10   34.14\n",
      "  266        438M 438M N   CA   3402 3403  150.21  140.50                  -40.50\n",
      "  267  20157 438M 439T C   N    3408 3410 -138.25 -124.80   18.85    0.61  -63.20  177.81   27.70\n",
      "  267        439T 439T N   CA   3410 3411  156.71  143.50                  -42.10\n",
      "  268  20159 440V 441E C   N    3422 3424 -140.87 -117.80   28.14    0.84  -63.60 -176.18   28.84\n",
      "  268        441E 441E N   CA   3424 3425  152.91  136.80                  -40.30\n",
      "  269  20160 441E 442H C   N    3431 3433 -142.52 -125.60   26.63    0.68  -63.20  177.10   27.29\n",
      "  269        442H 442H N   CA   3433 3434  159.36  138.80                  -42.30\n",
      "  270  20161 442H 443R C   N    3441 3443 -138.01 -125.20   23.15    0.79  -63.00  175.81   28.87\n",
      "  270        443R 443R N   CA   3443 3444  159.89  140.60                  -41.10\n",
      "  271  20163 444V 445Q C   N    3459 3461 -125.19 -121.10   18.75    0.82  -63.80  172.96   28.68\n",
      "  271        445Q 445Q N   CA   3461 3462  158.00  139.70                  -40.30\n",
      "  272  20165 446C 447G C   N    3474 3476  -94.82  -80.20   21.07    0.65   82.20 -108.09   19.30\n",
      "  272        447G 447G N   CA   3476 3477 -170.72  174.10                    8.50\n",
      "  273  20166 447G 448S C   N    3478 3480 -138.42 -136.60   13.12    0.63  -64.10  177.15   18.14\n",
      "  273        448S 448S N   CA   3480 3481  164.19  151.20                  -35.00\n",
      "  274  20168 449C 450H C   N    3490 3492 -130.16 -125.60   17.16    0.56  -63.20  175.62   26.33\n",
      "  274        450H 450H N   CA   3492 3493  155.34  138.80                  -42.30\n",
      "  275  20169 450H 451K C   N    3500 3502 -129.81 -118.00   18.68    0.69  -62.90  178.63   27.20\n",
      "  275        451K 451K N   CA   3502 3503  153.57  139.10                  -40.80\n",
      "  276  20171 452V 453R C   N    3516 3518 -133.82 -125.20    9.81    0.29  -63.00 -172.49   30.19\n",
      "  276        453R 453R N   CA   3518 3519  145.28  140.60                  -41.10\n",
      "  277  20172 453R 454Y C   N    3527 3529 -131.06 -124.30    8.68    0.42  -63.50 -171.70   33.26\n",
      "  277        454Y 454Y N   CA   3529 3530  140.84  135.40                  -43.40\n",
      "  278  20173 454Y 455T C   N    3539 3541 -133.70 -124.80   10.17    0.39  -63.20 -176.46   28.17\n",
      "  278        455T 455T N   CA   3541 3542  148.44  143.50                  -42.10\n",
      "  279  20174 455T 456R C   N    3546 3548 -142.59 -125.20   18.06    0.61  -63.00 -169.19   31.21\n",
      "  279        456R 456R N   CA   3548 3549  145.49  140.60                  -41.10\n",
      "  280  20175 456R 457E C   N    3557 3559 -141.47 -117.80   24.77    0.76  -63.60 -167.92   29.98\n",
      "  280        457E 457E N   CA   3559 3560  144.11  136.80                  -40.30\n",
      "  281  20176 457E 458T C   N    3566 3568 -131.53 -124.80    6.96    0.40  -63.20 -171.01   28.75\n",
      "  281        458T 458T N   CA   3568 3569  141.69  143.50                  -42.10\n",
      "  282  20177 458T 459D C   N    3573 3575 -129.35  -96.50   34.59    1.42  -63.30  177.75   19.04\n",
      "  282        459D 459D N   CA   3575 3576  125.02  114.20                  -40.00\n",
      "  283  20180 461C 462L C   N    3593 3595 -131.88 -108.50   24.57    1.10  -63.50 -168.61   30.81\n",
      "  283        462L 462L N   CA   3595 3596  140.04  132.50                  -41.20\n",
      "  284  20181 462L 463S C   N    3601 3603 -135.86 -136.60    8.07    0.39  -64.10 -167.93   11.90\n",
      "  284        463S 463S N   CA   3603 3604  143.16  151.20                  -35.00\n",
      "  285  20182 463S 464L C   N    3607 3609 -128.15 -108.50   20.86    0.93  -63.50 -169.42   30.46\n",
      "  285        464L 464L N   CA   3609 3610  139.52  132.50                  -41.20\n",
      "  286  20186 467P 468L C   N    3637 3639 -117.82 -108.50    9.32    0.44  -63.50 -177.81   22.82\n",
      "  286        468L 468L N   CA   3639 3640  132.71  132.50                  -41.20\n",
      "  287  20187 468L 469D C   N    3645 3647 -116.28  -96.50   19.79    0.81  -63.30  162.94   17.63\n",
      "  287        469D 469D N   CA   3647 3648  114.09  114.20                  -40.00\n",
      "  288  20189 470P 471N C   N    3660 3662 -139.96 -119.90   21.64    0.70  -63.20 -170.00   28.10\n",
      "  288        471N 471N N   CA   3662 3663  145.10  137.00                  -41.10\n",
      "  289  20190 471N 472L C   N    3668 3670 -120.33 -108.50   12.12    0.61  -63.50 -179.72   22.50\n",
      "  289        472L 472L N   CA   3670 3671  129.88  132.50                  -41.20\n",
      "  290  20191 472L 473K C   N    3676 3678 -137.05 -118.00   19.05    0.73  -62.90 -165.79   22.42\n",
      "  290        473K 473K N   CA   3678 3679  138.70  139.10                  -40.80\n",
      "  291  20192 473K 474N C   N    3685 3687 -140.11 -119.90   21.83    0.71  -63.20 -170.07   28.10\n",
      "  291        474N 474N N   CA   3687 3688  145.25  137.00                  -41.10\n",
      "  292  20193 474N 475S C   N    3693 3695 -134.90 -136.60   13.87    0.67  -64.10 -173.60   11.56\n",
      "  292        475S 475S N   CA   3695 3696  137.43  151.20                  -35.00\n",
      "  293  20194 475S 476D C   N    3699 3701 -133.15  -96.50   44.31    1.84  -63.30 -167.77   20.62\n",
      "  293        476D 476D N   CA   3701 3702  139.09  114.20                  -40.00\n",
      "  294  20195 476D 477G C   N    3707 3709  178.98 -167.20   23.37    0.92   82.20  176.20   12.77\n",
      "  294        477G 477G N   CA   3709 3710  155.75  174.60                    8.50\n",
      "  295  20196 477G 478N C   N    3711 3713 -125.51 -119.90    5.80    0.20  -63.20 -169.95   20.98\n",
      "  295        478N 478N N   CA   3713 3714  138.45  137.00                  -41.10\n",
      "  296  20197 478N 479P C   N    3719 3721  -52.62  -58.70   14.25    0.83  -64.50  169.83   13.31\n",
      "  296        479P 479P N   CA   3721 3722  -43.38  -30.50                  147.20\n",
      "  297  20198 479P 480K C   N    3726 3728   65.50   56.60  105.54    8.22  -62.90 -142.59   25.35\n",
      "  297        480K 480K N   CA   3728 3729  143.76   38.60                  -40.80\n",
      "  298  20199 480K 481Q C   N    3735 3737 -138.83 -121.10   17.86    0.63  -63.80 -167.02   32.33\n",
      "  298        481Q 481Q N   CA   3737 3738  141.90  139.70                  -40.30\n",
      "  299  20200 481Q 482T C   N    3744 3746 -136.26 -124.80   13.15    0.51  -63.20 -176.85   28.28\n",
      "  299        482T 482T N   CA   3746 3747  149.95  143.50                  -42.10\n",
      "  300  20201 482T 483E C   N    3751 3753 -137.09 -117.80   19.66    0.63  -63.60 -166.42   29.95\n",
      "  300        483E 483E N   CA   3753 3754  140.61  136.80                  -40.30\n",
      "  301  20202 483E 484E C   N    3760 3762 -143.20 -117.80   27.13    0.81  -63.60 -169.23   29.90\n",
      "  301        484E 484E N   CA   3762 3763  146.32  136.80                  -40.30\n",
      "  302  20203 484E 485E C   N    3769 3771 -134.61 -117.80   17.11    0.56  -63.60 -166.76   29.77\n",
      "  302        485E 485E N   CA   3771 3772  139.97  136.80                  -40.30\n",
      "  303  20205 486I 487F C   N    3786 3788 -136.20 -124.20   12.45    0.39  -63.20 -175.83   30.33\n",
      "  303        487F 487F N   CA   3788 3789  146.62  143.30                  -44.30\n",
      "  304  20206 487F 488A C   N    3797 3799 -156.16 -134.00   23.18    0.56  -62.50 -170.00   36.14\n",
      "  304        488A 488A N   CA   3799 3800  153.79  147.00                  -40.90\n",
      "  305  20207 488A 489S C   N    3802 3804 -151.53 -136.60   15.35    0.52  -64.10 -168.65   20.14\n",
      "  305        489S 489S N   CA   3804 3805  154.79  151.20                  -35.00\n",
      "  306  20208 489S 490R C   N    3808 3810 -138.62 -125.20   13.44    0.56  -63.00 -165.59   31.45\n",
      "  306        490R 490R N   CA   3810 3811  139.80  140.60                  -41.10\n",
      "  307  20210 491P 492H C   N    3826 3828 -136.97 -125.60   13.13    0.34  -63.20 -172.54   28.26\n",
      "  307        492H 492H N   CA   3828 3829  145.36  138.80                  -42.30\n",
      "  308  20212 493I 494S C   N    3844 3846 -139.88 -136.60    3.29    0.13  -64.10 -170.27   19.14\n",
      "  308        494S 494S N   CA   3846 3847  151.06  151.20                  -35.00\n",
      "  309  20213 494S 495L C   N    3850 3852 -127.42 -108.50   19.90    0.89  -63.50 -169.10   23.72\n",
      "  309        495L 495L N   CA   3852 3853  138.68  132.50                  -41.20\n",
      "  310  20214 495L 496D C   N    3858 3860 -130.32  -96.50   36.54    1.51  -63.30 -179.09   19.38\n",
      "  310        496D 496D N   CA   3860 3861  128.04  114.20                  -40.00\n",
      "  311  20215 496D 497A C   N    3866 3868 -139.90 -134.00    6.03    0.16  -62.50 -172.39   34.89\n",
      "  311        497A 497A N   CA   3868 3869  148.21  147.00                  -40.90\n",
      "  312  20217 498C 499L C   N    3877 3879 -139.86 -108.50   37.29    1.65  -63.50 -177.18   30.11\n",
      "  312        499L 499L N   CA   3879 3880  152.69  132.50                  -41.20\n",
      "  313  20218 499L 500G C   N    3885 3887  171.04 -167.20   33.70    1.30   82.20  166.11   11.96\n",
      "  313        500G 500G N   CA   3887 3888  148.86  174.60                    8.50\n",
      "  314  20219 500G 501S C   N    3889 3891 -157.44 -136.60   21.04    0.76  -64.10 -165.21   20.83\n",
      "  314        501S 501S N   CA   3891 3892  154.02  151.20                  -35.00\n",
      "  315  20220 501S 502L C   N    3895 3897 -136.84 -108.50   29.84    1.33  -63.50 -168.46   31.14\n",
      "  315        502L 502L N   CA   3897 3898  141.86  132.50                  -41.20\n",
      "  316  20221 502L 503M C   N    3903 3905 -140.74 -125.60   15.47    0.53  -63.40 -167.92   34.78\n",
      "  316        503M 503M N   CA   3905 3906  143.68  140.50                  -40.50\n",
      "  317  20222 503M 504Q C   N    3911 3913 -151.66 -121.10   31.97    1.03  -63.80 -168.10   32.77\n",
      "  317        504Q 504Q N   CA   3913 3914  149.10  139.70                  -40.30\n",
      "  318  20223 504Q 505A C   N    3920 3922 -154.90 -134.00   21.42    0.55  -62.50 -168.78   36.27\n",
      "  318        505A 505A N   CA   3922 3923  151.69  147.00                  -40.90\n",
      "  319  20224 505A 506T C   N    3925 3927 -137.67 -124.80   13.07    0.63  -63.20 -172.48   28.93\n",
      "  319        506T 506T N   CA   3927 3928  145.80  143.50                  -42.10\n",
      "  320  20225 506T 507D C   N    3932 3934 -131.58  -96.50   37.10    1.53  -63.30  179.73   19.22\n",
      "  320        507D 507D N   CA   3934 3935  126.26  114.20                  -40.00\n",
      "  321  20227 508I 509D C   N    3948 3950 -132.32  -96.50   39.04    1.61  -63.30 -176.77   19.61\n",
      "  321        509D 509D N   CA   3950 3951  129.73  114.20                  -40.00\n",
      "  322  20229 510C 511R C   N    3962 3964 -152.32 -125.20   29.17    0.92  -63.00 -170.14   31.69\n",
      "  322        511R 511R N   CA   3964 3965  151.36  140.60                  -41.10\n",
      "  323  20231 512C 513S C   N    3979 3981 -149.96 -136.60   14.11    0.46  -64.10 -170.21   19.91\n",
      "  323        513S 513S N   CA   3981 3982  155.74  151.20                  -35.00\n",
      "  324  20232 513S 514A C   N    3985 3987 -148.40 -134.00   14.90    0.37  -62.50 -171.09   35.56\n",
      "  324        514A 514A N   CA   3987 3988  150.85  147.00                  -40.90\n",
      "  325  20234 515C 516G C   N    3996 3998  172.27 -167.20   33.45    1.31   82.20  166.21   12.01\n",
      "  325        516G 516G N   CA   3998 3999  148.20  174.60                    8.50\n",
      "  326  20238 519V 520T C   N    4022 4024 -136.53 -124.80   12.53    0.54  -63.20 -174.88   28.55\n",
      "  326        520T 520T N   CA   4024 4025  147.93  143.50                  -42.10\n",
      "  327  20239 520T 521Y C   N    4029 4031 -140.16 -124.30   23.80    1.01  -63.50 -179.45   32.42\n",
      "  327        521Y 521Y N   CA   4031 4032  153.14  135.40                  -43.40\n",
      "  328  20241 522C 523K C   N    4047 4049 -154.65 -118.00   38.43    1.30  -62.90 -168.10   30.40\n",
      "  328        523K 523K N   CA   4049 4050  150.66  139.10                  -40.80\n",
      "  329  20242 523K 524T C   N    4056 4058 -139.86 -124.80   17.77    0.66  -63.20 -178.09   28.34\n",
      "  329        524T 524T N   CA   4058 4059  152.94  143.50                  -42.10\n",
      "  330  20243 524T 525T C   N    4063 4065 -147.56 -124.80   29.58    1.01  -63.20  176.91   28.13\n",
      "  330        525T 525T N   CA   4065 4066  162.40  143.50                  -42.10\n",
      "  331  20244 525T 526R C   N    4070 4072 -150.14 -125.20   27.13    0.85  -63.00 -171.09   31.43\n",
      "  331        526R 526R N   CA   4072 4073  151.29  140.60                  -41.10\n",
      "  332  20245 526R 527L C   N    4081 4083 -143.03 -108.50   41.30    1.82  -63.50 -178.06   30.18\n",
      "  332        527L 527L N   CA   4083 4084  155.16  132.50                  -41.20\n",
      "  333  20246 527L 528R C   N    4089 4091 -148.09 -125.20   27.11    0.80  -63.00 -175.45   30.70\n",
      "  333        528R 528R N   CA   4091 4092  155.13  140.60                  -41.10\n",
      "  334  20247 528R 529T C   N    4100 4102 -147.62 -124.80   32.25    1.04  -63.20  173.53   27.68\n",
      "  334        529T 529T N   CA   4102 4103  166.29  143.50                  -42.10\n",
      "  335  20248 529T 530F C   N    4107 4109 -147.22 -124.20   24.07    0.74  -63.20 -174.50   31.15\n",
      "  335        530F 530F N   CA   4109 4110  150.32  143.30                  -44.30\n",
      "  336  20250 531P 532D C   N    4125 4127 -148.87  -96.50   68.21    2.85  -63.30 -176.70   28.13\n",
      "  336        532D 532D N   CA   4127 4128  157.90  114.20                  -40.00\n",
      "  337  20252 533V 534L C   N    4140 4142 -140.40 -108.50   34.82    1.54  -63.50 -171.29   30.96\n",
      "  337        534L 534L N   CA   4142 4143  146.47  132.50                  -41.20\n",
      "  338  20255 536I 537F C   N    4163 4165 -141.82 -124.20   22.22    0.58  -63.20  177.24   29.67\n",
      "  338        537F 537F N   CA   4165 4166  156.85  143.30                  -44.30\n",
      "  339  20256 537F 538L C   N    4174 4176 -151.69 -108.50   48.00    2.12  -63.50 -172.62   31.46\n",
      "  339        538L 538L N   CA   4176 4177  153.46  132.50                  -41.20\n",
      "  340  20257 538L 539R C   N    4182 4184 -150.16 -125.20   33.02    0.99  -63.00  179.28   30.10\n",
      "  340        539R 539R N   CA   4184 4185  162.23  140.60                  -41.10\n",
      "  341  20258 539R 540R C   N    4193 4195 -161.52 -125.20   46.60    1.38  -63.00  178.72   30.68\n",
      "  341        540R 540R N   CA   4195 4196  169.79  140.60                  -41.10\n",
      "  342  20259 540R 541A C   N    4204 4206 -171.89 -134.00   43.24    1.00  -62.50 -173.32   36.35\n",
      "  342        541A 541A N   CA   4206 4207  167.83  147.00                  -40.90\n",
      "  343  20260 541A 542Q C   N    4209 4211 -173.59 -121.10   57.57    1.79  -63.80 -168.94   33.58\n",
      "  343        542Q 542Q N   CA   4211 4212  163.34  139.70                  -40.30\n",
      "  344  20261 542Q 543F C   N    4218 4220 -167.75 -124.20   47.67    1.34  -63.20 -174.68   32.16\n",
      "  344        543F 543F N   CA   4220 4221  162.69  143.30                  -44.30\n",
      "  345  20262 543F 544D C   N    4229 4231 -161.10  -96.50   85.26    3.56  -63.30  179.20   28.34\n",
      "  345        544D 544D N   CA   4231 4232  169.84  114.20                  -40.00\n",
      "  346  20263 544D 545M C   N    4237 4239 -176.64 -125.60   59.04    1.76  -63.40 -172.58   36.46\n",
      "  346        545M 545M N   CA   4239 4240  170.16  140.50                  -40.50\n",
      "  347  20264 545M 546G C   N    4245 4247  175.55 -167.20   18.41    0.52   82.20 -175.05   13.13\n",
      "  347        546G 546G N   CA   4247 4248  168.17  174.60                    8.50\n",
      "  348  20265 546G 547T C   N    4249 4251 -161.25 -124.80   42.84    1.60  -63.20 -179.21   29.39\n",
      "  348        547T 547T N   CA   4251 4252  166.01  143.50                  -42.10\n",
      "  349  20266 547T 548M C   N    4256 4258 -176.00 -125.60   62.65    1.85  -63.40 -178.94   35.41\n",
      "  349        548M 548M N   CA   4258 4259  177.71  140.50                  -40.50\n",
      "  350  20267 548M 549S C   N    4264 4266 -173.33 -136.60   37.63    1.71  -64.10 -151.12   13.99\n",
      "  350        549S 549S N   CA   4266 4267  143.05  151.20                  -35.00\n",
      "  351  20269 550V 551K C   N    4277 4279 -165.24 -118.00   48.51    1.67  -62.90 -162.35   31.76\n",
      "  351        551K 551K N   CA   4279 4280  150.11  139.10                  -40.80\n",
      "  352  20270 551K 552K C   N    4286 4288  177.72 -118.00   75.81    2.52  -62.90 -176.08   30.81\n",
      "  352        552K 552K N   CA   4288 4289  179.29  139.10                  -40.80\n",
      "  353  20271 552K 553L C   N    4295 4297 -149.85 -108.50   51.53    2.28  -63.50  177.90   30.02\n",
      "  353        553L 553L N   CA   4297 4298  163.26  132.50                  -41.20\n",
      "  354  20272 553L 554D C   N    4303 4305  166.10  -96.50  107.70    4.45  -63.30 -153.60   33.61\n",
      "  354        554D 554D N   CA   4305 4306  160.17  114.20                  -40.00\n",
      "  355  20274 555V 556F C   N    4318 4320 -150.27 -124.20   62.91    2.21  -63.20  144.36   25.30\n",
      "  355        556F 556F N   CA   4320 4321 -159.45  143.30                  -44.30\n",
      "  356  20276 557V 558D C   N    4336 4338 -142.77  -70.90   77.66    4.62  -63.30  161.23   25.00\n",
      "  356        558D 558D N   CA   4338 4339  179.72  150.30                  -40.00\n",
      "  357  20279 560P 561L C   N    4358 4360 -154.96 -108.50   68.77    3.12  -63.50  163.55   28.28\n",
      "  357        561L 561L N   CA   4360 4361 -176.79  132.50                  -41.20\n",
      "  358  20280 561L 562E C   N    4366 4368 -171.17 -117.80   57.39    1.71  -63.60 -165.70   31.77\n",
      "  358        562E 562E N   CA   4368 4369  157.89  136.80                  -40.30\n",
      "  359  20281 562E 563L C   N    4375 4377 -175.45 -108.50   83.39    3.69  -63.50  176.60   31.23\n",
      "  359        563L 563L N   CA   4377 4378 -177.78  132.50                  -41.20\n",
      "  360  20282 563L 564D C   N    4383 4385  178.60  -96.50   97.27    4.03  -63.30 -162.47   31.81\n",
      "  360        564D 564D N   CA   4385 4386  161.66  114.20                  -40.00\n",
      "  361  20283 564D 565L C   N    4391 4393  175.58 -108.50   84.34    3.73  -63.50 -167.67   33.96\n",
      "  361        565L 565L N   CA   4393 4394  169.23  132.50                  -41.20\n",
      "  362  20284 565L 566E C   N    4399 4401 -171.51 -117.80   66.50    1.99  -63.60  179.69   29.75\n",
      "  362        566E 566E N   CA   4401 4402  176.01  136.80                  -40.30\n",
      "  363  20285 566E 567F C   N    4408 4410 -175.09 -124.20   59.70    1.57  -63.20 -179.86   31.70\n",
      "  363        567F 567F N   CA   4410 4411  174.52  143.30                  -44.30\n",
      "  364  20286 567F 568L C   N    4419 4421 -177.39 -108.50   77.71    3.43  -63.50 -171.40   33.06\n",
      "  364        568L 568L N   CA   4421 4422  168.46  132.50                  -41.20\n",
      "  365  20287 568L 569R C   N    4427 4429 -177.27 -125.20   67.49    2.00  -63.00  177.14   31.25\n",
      "  365        569R 569R N   CA   4429 4430 -176.46  140.60                  -41.10\n",
      "  366  20288 569R 570G C   N    4438 4440  178.65 -167.20   14.66    0.22   82.20 -164.61   13.79\n",
      "  366        570G 570G N   CA   4440 4441  178.43  174.60                    8.50\n",
      "  367  20289 570G 571K C   N    4442 4444  174.52 -118.00   76.84    2.54  -62.90 -171.39   31.61\n",
      "  367        571K 571K N   CA   4444 4445  175.86  139.10                  -40.80\n",
      "  368  20290 571K 572G C   N    4451 4453 -175.32 -167.20   11.26    0.25   82.20 -158.15   14.35\n",
      "  368        572G 572G N   CA   4453 4454 -177.60  174.60                    8.50\n",
      "  369  20291 572G 573L C   N    4455 4457 -155.44 -108.50   68.40    3.10  -63.50  164.61   28.46\n",
      "  369        573L 573L N   CA   4457 4458 -177.74  132.50                  -41.20\n",
      "  370  20292 573L 574Q C   N    4463 4465 -126.04 -121.10   62.17    2.87  -63.80  133.43   22.84\n",
      "  370        574Q 574Q N   CA   4465 4466 -158.33  139.70                  -40.30\n",
      "  371  20293 574Q 575S C   N    4472 4474  171.68 -136.60   64.12    1.97  -64.10 -175.89   22.26\n",
      "  371        575S 575S N   CA   4474 4475 -170.89  151.20                  -35.00\n",
      "  372  20294 575S 576D C   N    4478 4480  178.06  -96.50   97.60    4.05  -63.30 -161.93   31.90\n",
      "  372        576D 576D N   CA   4480 4481  161.39  114.20                  -40.00\n",
      "  373  20295 576D 577E C   N    4486 4488 -145.63 -117.80   42.67    1.43  -63.60  171.45   27.41\n",
      "  373        577E 577E N   CA   4488 4489  169.14  136.80                  -40.30\n",
      "  374  20297 578V 579S C   N    4502 4504  179.16 -136.60   76.53    2.72  -64.10  161.33   20.02\n",
      "  374        579S 579S N   CA   4504 4505 -146.36  151.20                  -35.00\n",
      "  375  20298 579S 580M C   N    4508 4510  156.00 -125.60   78.40    3.10  -63.40 -132.23   44.48\n",
      "  375        580M 580M N   CA   4510 4511  140.30  140.50                  -40.50\n",
      "  376  20299 580M 581P C   N    4516 4518  122.10  -64.50  174.03   18.29  -64.50  174.03   18.29\n",
      "  376        581P 581P N   CA   4518 4519  132.53  147.20                  147.20\n",
      "  377  20300 581P 582E C   N    4523 4525 -137.87 -117.80   57.32    2.42  -63.60  149.03   23.97\n",
      "  377        582E 582E N   CA   4525 4526 -169.51  136.80                  -40.30\n",
      "  378  20301 582E 583A C   N    4532 4534 -124.04 -134.00   48.93    2.95  -62.50  138.61   26.01\n",
      "  378        583A 583A N   CA   4534 4535 -165.09  147.00                  -40.90\n",
      "  379  20302 583A 584Q C   N    4537 4539 -116.09 -121.10   67.54    3.29  -63.80  124.19   21.00\n",
      "  379        584Q 584Q N   CA   4539 4540 -152.95  139.70                  -40.30\n",
      "  380  20303 584Q 585E C   N    4546 4548 -173.54 -117.80   59.22    1.78  -63.60 -163.49   32.18\n",
      "  380        585E 585E N   CA   4548 4549  156.82  136.80                  -40.30\n",
      "  381  20304 585E 586D C   N    4555 4557   17.68  -70.90  102.26    8.58  -63.30  143.61   15.37\n",
      "  381        586D 586D N   CA   4557 4558 -158.60  150.30                  -40.00\n",
      "  382  20305 586D 587L C   N    4563 4565  123.30 -108.50  128.54    6.23  -63.50 -121.19   31.01\n",
      "  382        587L 587L N   CA   4565 4566  123.22  132.50                  -41.20\n",
      "  383  20307 588P 589H C   N    4578 4580  153.79 -125.60   87.11    2.38  -63.20 -155.70   34.22\n",
      "  383        589H 589H N   CA   4580 4581  171.80  138.80                  -42.30\n",
      "  384  20308 589H 590K C   N    4588 4590 -145.92 -118.00   51.67    2.00  -62.90  159.86   25.70\n",
      "  384        590K 590K N   CA   4590 4591 -177.42  139.10                  -40.80\n",
      "  385  20309 590K 591P C   N    4597 4599 -102.68  -58.70   71.97    4.53  -64.50  126.63   11.52\n",
      "  385        591P 591P N   CA   4599 4600   26.46  -30.50                  147.20\n",
      "  386  20310 591P 592T C   N    4604 4606   82.31 -124.80  156.12    7.41  -63.20 -156.12   22.45\n",
      "  386        592T 592T N   CA   4606 4607  175.09  143.50                  -42.10\n",
      "  387  20311 592T 593S C   N    4611 4613  135.64 -136.60  119.31    6.60  -64.10 -168.20   18.32\n",
      "  387        593S 593S N   CA   4613 4614   70.37  151.20                  -35.00\n",
      "  388  20312 593S 594S C   N    4617 4619  121.17 -136.60  114.18    3.52  -64.10 -146.36   27.67\n",
      "  388        594S 594S N   CA   4619 4620 -157.94  151.20                  -35.00\n",
      "  389  20313 594S 595S C   N    4623 4625   77.37 -136.60  146.05    5.86  -64.10 -133.91   16.68\n",
      "  389        595S 595S N   CA   4625 4626  148.64  151.20                  -35.00\n",
      "  390  20314 595S 596M C   N    4629 4631  -87.22  -73.00   40.16    2.53  -63.40  140.97   22.95\n",
      "  390        596M 596M N   CA   4631 4632 -179.45  143.00                  -40.50\n",
      "  391  20315 596M 597K C   N    4637 4639 -145.44 -118.00   57.53    2.30  -62.90  153.60   24.85\n",
      "  391        597K 597K N   CA   4639 4640 -170.33  139.10                  -40.80\n",
      "  392  20316 597K 598T C   N    4646 4648  -79.49  -63.20   59.29    8.69  -63.20   59.29    8.69\n",
      "  392        598T 598T N   CA   4648 4649  -99.10  -42.10                  -42.10\n",
      "  393  20317 598T 599V C   N    4653 4655   41.59  -73.50  115.22    8.26 -125.40  167.00    6.56\n",
      "  393        599V 599V N   CA   4655 4656  144.51  139.20                  143.30\n",
      "  394  20318 599V 600D C   N    4660 4662   13.73  -63.30   80.01   11.48  -63.30   80.01   11.48\n",
      "  394        600D 600D N   CA   4662 4663  -61.62  -40.00                  -40.00\n",
      "  395  20319 600D 601E C   N    4668 4670 -173.08 -117.80   61.84    1.81  -63.60 -170.08   31.24\n",
      "  395        601E 601E N   CA   4670 4671  164.52  136.80                  -40.30\n",
      "  396  20320 601E 602E C   N    4677 4679 -113.50 -117.80   20.13    1.07  -63.60  170.69   25.58\n",
      "  396        602E 602E N   CA   4679 4680  156.46  136.80                  -40.30\n",
      "  397  20321 602E 603A C   N    4686 4688   49.61  -68.20  119.29   10.71  -62.50 -168.68   28.84\n",
      "  397        603A 603A N   CA   4688 4689  164.07  145.30                  -40.90\n",
      "  398  20322 603A 604L C   N    4691 4693  -96.29  -63.50   56.89    9.89  -63.50   56.89    9.89\n",
      "  398        604L 604L N   CA   4693 4694  -87.68  -41.20                  -41.20\n",
      "  399  20323 604L 605A C   N    4699 4701 -167.12 -134.00   63.50    2.48  -62.50  157.65   31.23\n",
      "  399        605A 605A N   CA   4701 4702 -158.82  147.00                  -40.90\n",
      "  400  20325 606M 607L C   N    4712 4714  -49.24  -63.50  151.37   20.24  -63.50  151.37   20.24\n",
      "  400        607L 607L N   CA   4714 4715  168.11  -41.20                  -41.20\n",
      "  401  20326 607L 608L C   N    4720 4722 -163.58 -108.50   55.51    2.73  -63.50 -165.44   23.87\n",
      "  401        608L 608L N   CA   4722 4723  125.64  132.50                  -41.20\n",
      "  402  20327 608L 609S C   N    4728 4730  -94.20  -72.40   32.44    1.51  -64.10  151.60   13.01\n",
      "  402        609S 609S N   CA   4730 4731  176.42  152.40                  -35.00\n",
      "  403  20328 609S 610M C   N    4734 4736 -153.03 -125.60   52.44    1.82  -63.40  161.46   30.94\n",
      "  403        610M 610M N   CA   4736 4737 -174.81  140.50                  -40.50\n",
      "  404  20329 610M 611G C   N    4742 4744 -144.42 -167.20   26.14    0.42   82.20 -156.82   15.35\n",
      "  404        611G 611G N   CA   4744 4745  161.78  174.60                    8.50\n",
      "  405  20331 612I 613E C   N    4754 4756  167.57 -117.80   76.11    2.45  -63.60 -148.31   35.12\n",
      "  405        613E 613E N   CA   4756 4757  151.72  136.80                  -40.30\n",
      "  406  20332 613E 614E C   N    4763 4765 -179.97 -117.80   72.10    2.11  -63.60 -173.00   31.12\n",
      "  406        614E 614E N   CA   4765 4766  173.32  136.80                  -40.30\n",
      "  407  20333 614E 615T C   N    4772 4774  -91.74  -78.10   57.31    2.05  -63.20  116.00   16.78\n",
      "  407        615T 615T N   CA   4774 4775 -154.54  149.80                  -42.10\n",
      "  408  20335 616V 617A C   N    4786 4788  145.11 -134.00   89.18    2.03  -62.50 -156.71   40.90\n",
      "  408        617A 617A N   CA   4788 4789 -175.46  147.00                  -40.90\n",
      "  409  20336 617A 618R C   N    4791 4793 -158.04 -125.20   62.94    2.19  -63.00  156.71   27.36\n",
      "  409        618R 618R N   CA   4793 4794 -165.70  140.60                  -41.10\n",
      "  410  20337 618R 619Y C   N    4802 4804 -165.99 -124.30   51.63    2.58  -63.50 -177.70   33.70\n",
      "  410        619Y 619Y N   CA   4804 4805  165.84  135.40                  -43.40\n",
      "  411  20338 619Y 620A C   N    4814 4816  -87.29  -68.20   20.05    1.48  -62.50  169.50   29.06\n",
      "  411        620A 620A N   CA   4816 4817  151.42  145.30                  -40.90\n",
      "  412  20339 620A 621L C   N    4819 4821 -110.47 -108.50   40.97    2.19  -63.50  152.77   24.11\n",
      "  412        621L 621L N   CA   4821 4822  173.43  132.50                  -41.20\n",
      "  413  20340 621L 622L C   N    4827 4829 -169.30 -108.50   76.94    3.41  -63.50  174.80   30.66\n",
      "  413        622L 622L N   CA   4829 4830  179.66  132.50                  -41.20\n",
      "  414  20341 622L 623Q C   N    4835 4837  161.15 -121.10   80.68    2.64  -63.80 -151.78   37.13\n",
      "  414        623Q 623Q N   CA   4837 4838  161.22  139.70                  -40.30\n",
      "  415  20342 623Q 624T C   N    4844 4846 -157.07 -124.80   48.66    1.54  -63.20  166.88   27.30\n",
      "  415        624T 624T N   CA   4846 4847  179.93  143.50                  -42.10\n",
      "  416  20343 624T 625G C   N    4851 4853 -177.56 -167.20   10.36    0.20   82.20 -166.09   13.85\n",
      "  416        625G 625G N   CA   4853 4854  174.49  174.60                    8.50\n",
      "  417  20344 625G 626M C   N    4855 4857 -160.74 -125.60   45.54    1.36  -63.40  178.85   34.14\n",
      "  417        626M 626M N   CA   4857 4858  169.46  140.50                  -40.50\n",
      "  418  20345 626M 627N C   N    4863 4865 -169.42 -119.90   59.84    1.76  -63.20 -177.57   28.60\n",
      "  418        627N 627N N   CA   4865 4866  170.59  137.00                  -41.10\n",
      "  419  20346 627N 628A C   N    4871 4873 -170.70 -134.00   42.64    1.00  -62.50 -174.72   36.06\n",
      "  419        628A 628A N   CA   4873 4874  168.70  147.00                  -40.90\n",
      "  420  20347 628A 629E C   N    4876 4878 -153.34 -117.80   56.60    1.94  -63.60  165.32   26.95\n",
      "  420        629E 629E N   CA   4878 4879 -179.15  136.80                  -40.30\n",
      "  421  20348 629E 630R C   N    4885 4887 -148.42 -125.20   34.97    1.10  -63.00  174.50   29.33\n",
      "  421        630R 630R N   CA   4887 4888  166.74  140.60                  -41.10\n",
      "  422  20349 630R 631A C   N    4896 4898 -163.83 -134.00   35.67    0.86  -62.50 -176.87   35.38\n",
      "  422        631A 631A N   CA   4898 4899  166.56  147.00                  -40.90\n",
      "  423  20351 632V 633D C   N    4908 4910 -156.77  -96.50   81.34    3.40  -63.30  177.74   27.90\n",
      "  423        633D 633D N   CA   4910 4911  168.82  114.20                  -40.00\n",
      "  424  20352 633D 634Y C   N    4916 4918 -145.10 -124.30   28.19    1.29  -63.50 -178.46   32.79\n",
      "  424        634Y 634Y N   CA   4918 4919  154.43  135.40                  -43.40\n",
      "  425  20354 635V 636F C   N    4935 4937 -147.31 -124.20   25.53    0.71  -63.20 -177.86   30.67\n",
      "  425        636F 636F N   CA   4937 4938  154.14  143.30                  -44.30\n",
      "  426  20355 636F 637S C   N    4946 4948 -168.78 -136.60   37.32    1.14  -64.10 -173.05   21.10\n",
      "  426        637S 637S N   CA   4948 4949  170.10  151.20                  -35.00\n",
      "  427  20356 637S 638H C   N    4952 4954 -152.27 -125.60   27.86    0.81  -63.20 -167.32   29.87\n",
      "  427        638H 638H N   CA   4954 4955  146.84  138.80                  -42.30\n",
      "  428  20357 638H 639E C   N    4962 4964 -173.02 -117.80   61.28    1.80  -63.60 -169.19   31.36\n",
      "  428        639E 639E N   CA   4964 4965  163.38  136.80                  -40.30\n",
      "  429  20358 639E 640N C   N    4971 4973 -131.68 -119.90   13.30    0.41  -63.20 -171.39   27.47\n",
      "  429        640N 640N N   CA   4973 4974  143.17  137.00                  -41.10\n",
      "  430  20360 641I 642A C   N    4987 4989 -146.77 -134.00   14.24    0.32  -62.50 -174.01   34.99\n",
      "  430        642A 642A N   CA   4989 4990  153.30  147.00                  -40.90\n",
      "  431  20361 642A 643E C   N    4992 4994 -160.88 -117.80   48.16    1.41  -63.60 -171.56   30.48\n",
      "  431        643E 643E N   CA   4994 4995  158.31  136.80                  -40.30\n",
      "  432  20362 643E 644E C   N    5001 5003 -151.32 -117.80   41.95    1.26  -63.60 -179.56   28.92\n",
      "  432        644E 644E N   CA   5003 5004  162.02  136.80                  -40.30\n",
      "  433  20363 644E 645A C   N    5010 5012 -178.92 -134.00   47.61    1.11  -62.50 -165.08   38.06\n",
      "  433        645A 645A N   CA   5012 5013  162.77  147.00                  -40.90\n",
      "  434  20364 645A 646G C   N    5015 5017 -179.69 -167.20   28.82    1.18   82.20  171.05   12.53\n",
      "  434        646G 646G N   CA   5017 5018  148.62  174.60                    8.50\n",
      "  435  20365 646G 647L C   N    5019 5021 -150.12 -108.50   46.35    2.05  -63.50 -172.85   31.33\n",
      "  435        647L 647L N   CA   5021 5022  152.90  132.50                  -41.20\n",
      "  436  20366 647L 648A C   N    5027 5029 -155.34 -134.00   22.61    0.53  -62.50 -171.00   35.93\n",
      "  436        648A 648A N   CA   5029 5030  154.48  147.00                  -40.90\n",
      "  437  20367 648A 649E C   N    5032 5034 -151.61 -117.80   35.93    1.08  -63.60 -167.92   30.51\n",
      "  437        649E 649E N   CA   5034 5035  148.97  136.80                  -40.30\n",
      "  438  20369 650I 651S C   N    5049 5051 -153.05 -136.60   17.02    0.57  -64.10 -168.62   20.25\n",
      "  438        651S 651S N   CA   5051 5052  155.54  151.20                  -35.00\n",
      "  439  20370 651S 652A C   N    5055 5057 -162.53 -134.00   29.83    0.72  -62.50 -168.42   36.72\n",
      "  439        652A 652A N   CA   5057 5058  155.71  147.00                  -40.90\n",
      "  440  20371 652A 653T C   N    5060 5062 -146.34 -124.80   28.38    0.96  -63.20  176.70   28.03\n",
      "  440        653T 653T N   CA   5062 5063  161.98  143.50                  -42.10\n",
      "  441  20372 653T 654A C   N    5067 5069 -163.76 -134.00   31.35    0.74  -62.50 -168.74   36.73\n",
      "  441        654A 654A N   CA   5069 5070  156.84  147.00                  -40.90\n",
      "  442  20373 654A 655S C   N    5072 5074 -157.39 -136.60   21.24    0.74  -64.10 -166.56   20.73\n",
      "  442        655S 655S N   CA   5074 5075  155.54  151.20                  -35.00\n",
      "  443  20374 655S 656E C   N    5078 5080 -158.57 -117.80   44.94    1.32  -63.60 -170.49   30.51\n",
      "  443        656E 656E N   CA   5080 5081  155.71  136.80                  -40.30\n",
      "  444  20375 656E 657S C   N    5087 5089 -151.11 -136.60   15.48    0.49  -64.10 -170.43   19.98\n",
      "  444        657S 657S N   CA   5089 5090  156.58  151.20                  -35.00\n",
      "  445  20376 657S 658Q C   N    5093 5095 -147.87 -121.10   27.39    0.92  -63.80 -166.59   32.83\n",
      "  445        658Q 658Q N   CA   5095 5096  145.51  139.70                  -40.30\n",
      "  446  20378 659P 660A C   N    5109 5111 -143.68 -134.00   12.77    0.35  -62.50 -177.20   34.31\n",
      "  446        660A 660A N   CA   5111 5112  155.32  147.00                  -40.90\n",
      "  447  20379 660A 661H C   N    5114 5116 -141.63 -125.60   23.73    0.59  -63.20  179.46   27.54\n",
      "  447        661H 661H N   CA   5116 5117  156.29  138.80                  -42.30\n",
      "  448  20381 662V 663L C   N    5131 5133 -127.93 -108.50   23.96    1.06  -63.50 -176.08   29.53\n",
      "  448        663L 663L N   CA   5133 5134  146.53  132.50                  -41.20\n",
      "  449  20382 663L 664D C   N    5139 5141 -122.19  -96.50   43.00    1.81  -63.30 -178.85   26.17\n",
      "  449        664D 664D N   CA   5141 5142  148.69  114.20                  -40.00\n",
      "  450  20383 664D 665G C   N    5147 5149 -152.94 -167.20   46.37    1.58   82.20  174.55   13.41\n",
      "  450        665G 665G N   CA   5149 5150  130.48  174.60                    8.50\n",
      "  451  20385 666P 667A C   N    5158 5160 -152.43 -134.00   18.98    0.48  -62.50 -169.81   35.98\n",
      "  451        667A 667A N   CA   5160 5161  151.51  147.00                  -40.90\n",
      "  452  20386 667A 668K C   N    5163 5165 -151.35 -118.00   33.36    1.28  -62.90 -160.45   22.93\n",
      "  452        668K 668K N   CA   5165 5166  138.08  139.10                  -40.80\n",
      "  453  20387 668K 669Y C   N    5172 5174 -140.24 -124.30   24.31    1.03  -63.50 -179.99   32.34\n",
      "  453        669Y 669Y N   CA   5174 5175  153.76  135.40                  -43.40\n",
      "  454  20388 669Y 670R C   N    5184 5186 -141.23 -125.20   16.22    0.59  -63.00 -167.58   31.35\n",
      "  454        670R 670R N   CA   5186 5187  143.10  140.60                  -41.10\n",
      "  455  20389 670R 671L C   N    5195 5197 -142.79 -108.50   38.54    1.70  -63.50 -173.59   30.79\n",
      "  455        671L 671L N   CA   5197 5198  150.10  132.50                  -41.20\n",
      "  456  20390 671L 672H C   N    5203 5205 -148.43 -125.60   26.73    0.68  -63.20 -174.29   28.75\n",
      "  456        672H 672H N   CA   5205 5206  152.71  138.80                  -42.30\n",
      "  457  20391 672H 673A C   N    5213 5215 -159.83 -134.00   28.05    0.64  -62.50 -171.71   36.04\n",
      "  457        673A 673A N   CA   5215 5216  157.92  147.00                  -40.90\n",
      "  458  20392 673A 674M C   N    5218 5220 -148.78 -125.60   25.28    0.78  -63.40 -170.75   34.94\n",
      "  458        674M 674M N   CA   5220 5221  150.60  140.50                  -40.50\n",
      "  459  20394 675I 676S C   N    5234 5236 -149.57 -136.60   14.64    0.45  -64.10 -172.39   19.72\n",
      "  459        676S 676S N   CA   5236 5237  157.98  151.20                  -35.00\n",
      "  460  20395 676S 677H C   N    5240 5242 -155.21 -125.60   35.16    0.88  -63.20 -175.48   29.00\n",
      "  460        677H 677H N   CA   5242 5243  157.76  138.80                  -42.30\n",
      "  461  20397 678V 679G C   N    5257 5259  173.11 -167.20   27.77    1.04   82.20  172.42   12.37\n",
      "  461        679G 679G N   CA   5259 5260  155.01  174.60                    8.50\n",
      "  462  20398 679G 680A C   N    5261 5263 -166.10 -134.00   34.79    0.79  -62.50 -170.49   36.55\n",
      "  462        680A 680A N   CA   5263 5264  160.42  147.00                  -40.90\n",
      "  463  20399 680A 681S C   N    5266 5268 -154.82 -136.60   20.45    0.63  -64.10 -172.13   20.14\n",
      "  463        681S 681S N   CA   5268 5269  160.49  151.20                  -35.00\n",
      "  464  20400 681S 682A C   N    5272 5274 -157.35 -134.00   24.73    0.58  -62.50 -170.58   36.10\n",
      "  464        682A 682A N   CA   5274 5275  155.15  147.00                  -40.90\n",
      "  465  20401 682A 683K C   N    5277 5279 -146.49 -118.00   31.21    1.04  -62.90 -172.93   29.29\n",
      "  465        683K 683K N   CA   5279 5280  151.85  139.10                  -40.80\n",
      "  466  20402 683K 684T C   N    5286 5288 -144.82 -124.80   32.93    1.02  -63.20  169.24   26.96\n",
      "  466        684T 684T N   CA   5288 5289  169.65  143.50                  -42.10\n",
      "  467  20403 684T 685G C   N    5293 5295  170.39 -167.20   33.43    1.28   82.20  166.55   11.96\n",
      "  467        685G 685G N   CA   5295 5296  149.79  174.60                    8.50\n",
      "  468  20404 685G 686H C   N    5297 5299 -151.54 -125.60   26.25    1.00  -63.20 -162.10   21.07\n",
      "  468        686H 686H N   CA   5299 5300  134.79  138.80                  -42.30\n",
      "  469  20405 686H 687Y C   N    5307 5309 -140.20 -124.30   27.65    1.11  -63.50  176.15   31.71\n",
      "  469        687Y 687Y N   CA   5309 5310  158.02  135.40                  -43.40\n",
      "  470  20408 689C 690H C   N    5332 5334 -126.71  -63.20   87.25    9.93  -63.20   87.25    9.93\n",
      "  470        690H 690H N   CA   5334 5335   17.51  -42.30                  -42.30\n",
      "  471  20411 692C 693D C   N    5356 5358 -165.64  -96.50   84.35    3.51  -63.30 -172.18   29.69\n",
      "  471        693D 693D N   CA   5358 5359  162.51  114.20                  -40.00\n",
      "  472  20412 693D 694A C   N    5364 5366 -169.73 -134.00   43.40    1.07  -62.50 -177.66   35.53\n",
      "  472        694A 694A N   CA   5366 5367  171.63  147.00                  -40.90\n",
      "  473  20413 694A 695Q C   N    5369 5371 -171.16 -121.10   56.97    1.76  -63.80 -173.27   32.82\n",
      "  473        695Q 695Q N   CA   5371 5372  166.91  139.70                  -40.30\n",
      "  474  20414 695Q 696T C   N    5378 5380 -147.10 -124.80   46.61    1.47  -63.20  157.65   25.54\n",
      "  474        696T 696T N   CA   5380 5381 -175.57  143.50                  -42.10\n",
      "  475  20415 696T 697G C   N    5385 5387 -164.83 -167.20    8.01    0.33   82.20 -152.76   14.99\n",
      "  475        697G 697G N   CA   5387 5388 -177.76  174.60                    8.50\n",
      "  476  20416 697G 698K C   N    5389 5391 -166.32 -118.00   61.83    2.11  -62.90  175.27   28.85\n",
      "  476        698K 698K N   CA   5391 5392  177.69  139.10                  -40.80\n",
      "  477  20417 698K 699W C   N    5398 5400  147.75 -124.90   89.31    3.46  -63.00 -145.67   36.52\n",
      "  477        699W 699W N   CA   5400 5401  161.98  143.40                  -44.20\n",
      "  478  20418 699W 700L C   N    5412 5414 -162.66 -108.50   73.61    3.30  -63.50  168.66   29.43\n",
      "  478        700L 700L N   CA   5414 5415 -177.64  132.50                  -41.20\n",
      "  479  20419 700L 701L C   N    5420 5422  179.53 -108.50   79.44    3.51  -63.50 -167.68   33.76\n",
      "  479        701L 701L N   CA   5422 5423  166.14  132.50                  -41.20\n",
      "  480  20420 701L 702F C   N    5428 5430 -158.56 -124.20   57.62    1.70  -63.20  158.13   27.71\n",
      "  480        702F 702F N   CA   5430 5431 -170.44  143.30                  -44.30\n",
      "  481  20421 702F 703N C   N    5439 5441 -176.46 -119.90   72.54    2.10  -63.20  177.36   28.22\n",
      "  481        703N 703N N   CA   5441 5442 -177.59  137.00                  -41.10\n",
      "  482  20422 703N 704D C   N    5447 5449 -176.79  -96.50   96.11    3.99  -63.30 -169.51   30.65\n",
      "  482        704D 704D N   CA   5449 5450  167.01  114.20                  -40.00\n",
      "  483  20423 704D 705E C   N    5455 5457 -170.30 -117.80   61.14    1.79  -63.60 -174.66   30.48\n",
      "  483        705E 705E N   CA   5457 5458  168.15  136.80                  -40.30\n",
      "  484  20424 705E 706K C   N    5464 5466 -164.13 -118.00   54.00    1.79  -62.90 -177.35   29.72\n",
      "  484        706K 706K N   CA   5466 5467  167.18  139.10                  -40.80\n",
      "  485  20426 707V 708A C   N    5480 5482 -179.78 -134.00   56.23    1.41  -62.50 -177.78   35.95\n",
      "  485        708A 708A N   CA   5482 5483  179.64  147.00                  -40.90\n",
      "  486  20427 708A 709E C   N    5485 5487 -168.86 -117.80   67.94    2.11  -63.60  173.62   28.78\n",
      "  486        709E 709E N   CA   5487 5488 -178.38  136.80                  -40.30\n",
      "  487  20428 709E 710S C   N    5494 5496 -151.40 -136.60   26.65    0.97  -64.10  174.97   18.96\n",
      "  487        710S 710S N   CA   5496 5497  173.36  151.20                  -35.00\n",
      "  488  20429 710S 711L C   N    5500 5502 -138.65 -108.50   63.92    3.07  -63.50  150.10   25.47\n",
      "  488        711L 711L N   CA   5502 5503 -171.14  132.50                  -41.20\n",
      "  489  20430 711L 712N C   N    5508 5510  171.08 -119.90   71.36    2.51  -63.20 -153.50   32.61\n",
      "  489        712N 712N N   CA   5510 5511  155.09  137.00                  -41.10\n",
      "  490  20433 714P 715F C   N    5530 5532  177.53 -124.20   74.94    1.95  -63.20  172.97   30.84\n",
      "  490        715F 715F N   CA   5532 5533 -169.57  143.30                  -44.30\n",
      "  491  20434 715F 716S C   N    5541 5543  126.18 -136.60  100.23    3.39  -64.10 -133.87   28.44\n",
      "  491        716S 716S N   CA   5543 5544  175.57  151.20                  -35.00\n",
      "  492  20435 716S 717M C   N    5547 5549  148.32 -125.60   94.34    2.90  -63.40 -155.80   40.95\n",
      "  492        717M 717M N   CA   5549 5550  179.11  140.50                  -40.50\n",
      "  493  20436 717M 718A C   N    5555 5557  -52.77  -68.20   15.43    1.29  -62.50  174.47   28.13\n",
      "  493        718A 718A N   CA   5557 5558  144.91  145.30                  -40.90\n",
      "  494  20437 718A 719F C   N    5560 5562 -173.77 -124.20   50.93    1.65  -63.20 -164.91   33.88\n",
      "  494        719F 719F N   CA   5562 5563  154.97  143.30                  -44.30\n",
      "  495  20438 719F 720L C   N    5571 5573 -143.71  -63.50   80.30   13.67  -63.50   80.30   13.67\n",
      "  495        720L 720L N   CA   5573 5574  -37.39  -41.20                  -41.20\n",
      "  496  20439 720L 721Y C   N    5579 5581  105.68 -124.30  131.79    9.18  -63.50 -127.34   32.47\n",
      "  496        721Y 721Y N   CA   5581 5582  156.88  135.40                  -43.40\n",
      "  497  20440 721Y 722F C   N    5591 5593  -30.29  -71.40   44.68    3.58  -63.20  160.90   20.24\n",
      "  497        722F 722F N   CA   5593 5594  158.20  140.70                  -44.30\n",
      "  498  20441 722F 723Y C   N    5602 5604  118.25 -124.30  117.46    8.84  -63.50 -107.61   35.09\n",
      "  498        723Y 723Y N   CA   5604 5605  135.28  135.40                  -43.40\n",
      "  499  20442 723Y 724K C   N    5614 5616 -165.04 -118.00   61.83    2.13  -62.90  173.27   28.51\n",
      "  499        724K 724K N   CA   5616 5617  179.24  139.10                  -40.80\n",
      "  500  20443 724K 725R C   N    5623 5625 -130.43 -125.20   50.17    2.30  -63.00  145.04   24.16\n",
      "  500        725R 725R N   CA   5625 5626 -169.51  140.60                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  27959 192W 237D CH2 CB   1465 1784   11.02    4.49    6.53    4.52    4.49    6.53    4.52\n",
      "    2  28118 233P 607L CB  CD2  1763 4719   11.77    4.45    7.32    4.62    4.45    7.32    4.62\n",
      "    3  28184 252F 606M CB  SD   1910 4710   23.17    4.79   18.38   14.54    4.79   18.38   14.54\n",
      "    4  28185 253A 606M CB  CG   1921 4709   20.19    4.50   15.69   10.39    4.50   15.69   10.39\n",
      "    5  28186 253A 606M CB  SD   1921 4710   18.57    3.87   14.70    9.98    3.87   14.70    9.98\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   51   92  322  393  493  475  449  542  573  641\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00192.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      728     5656     5656\n",
      "openf___224_> Open           EAO00192.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18229916   17802.652    17.385\n",
      "read_mo_297_> Segments, residues, atoms:        1      728     5656\n",
      "read_mo_298_> Segment:        1  1:A  728:A     5656\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5656     5656        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    23948\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24395\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26848\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27084\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27270\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28042\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28044\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28188\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28194\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      116.9736        5  5\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      115.9874        9  9\n",
      "iupac_m_486_> OE1/2 will be swapped:      158.9947       10  10\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.3799       34  34\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.0395       37  37\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.9152       43  43\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.4231       45  45\n",
      "iupac_m_486_> OE1/2 will be swapped:      112.1900       61  61\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.3295       76  76\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      104.5329       81  81\n",
      "iupac_m_485_> OD1/2 will be swapped:      140.8456       86  86\n",
      "iupac_m_486_> OE1/2 will be swapped:      128.3292      105  105\n",
      "iupac_m_486_> OE1/2 will be swapped:      131.4658      124  124\n",
      "iupac_m_485_> OD1/2 will be swapped:      112.9856      128  128\n",
      "iupac_m_486_> OE1/2 will be swapped:     -122.9676      146  146\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.3909      148  148\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0608      170  170\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -135.3276      183  183\n",
      "iupac_m_486_> OE1/2 will be swapped:     -157.3253      186  186\n",
      "iupac_m_487_> NH1/2 swapped:      147.6196      188  188\n",
      "iupac_m_486_> OE1/2 will be swapped:     -123.2725      206  206\n",
      "iupac_m_485_> OD1/2 will be swapped:      -94.5077      237  237\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -103.7905      238  238\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      117.0659      239  239\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.8300      246  246\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.0067      252  252\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.0447\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.2199      262  262\n",
      "iupac_m_486_> OE1/2 will be swapped:      158.6771      275  275\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.0662      285  285\n",
      "iupac_m_487_> NH1/2 swapped:      180.0000      304  304\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.3927      327  327\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9558      338  338\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.3303      350  350\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.9441      358  358\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      368  368\n",
      "iupac_m_486_> OE1/2 will be swapped:      180.0000      372  372\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      375  375\n",
      "iupac_m_487_> NH1/2 swapped:      179.9515      383  383\n",
      "iupac_m_487_> NH1/2 swapped:      179.9287      387  387\n",
      "iupac_m_485_> OD1/2 will be swapped:      149.6381      395  395\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.7897      400  400\n",
      "iupac_m_485_> OD1/2 will be swapped:      177.6980      403  403\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.7916      406  406\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.8994      413  413\n",
      "iupac_m_487_> NH1/2 swapped:      179.8881      416  416\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.0894      434  434\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9802      441  441\n",
      "iupac_m_487_> NH1/2 swapped:      179.8587      453  453\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.7624      457  457\n",
      "iupac_m_485_> OD1/2 will be swapped:     -149.1830      469  469\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.8455      476  476\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9604      485  485\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.6974      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      163.4026      509  509\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8481      511  511\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9184      526  526\n",
      "iupac_m_487_> NH1/2 swapped:      179.9604      528  528\n",
      "iupac_m_485_> OD1/2 will be swapped:      110.9897      532  532\n",
      "iupac_m_487_> NH1/2 swapped:      179.7888      540  540\n",
      "iupac_m_485_> OD1/2 will be swapped:     -164.7553      544  544\n",
      "iupac_m_485_> OD1/2 will be swapped:      161.9784      554  554\n",
      "iupac_m_486_> OE1/2 will be swapped:     -150.4203      562  562\n",
      "iupac_m_485_> OD1/2 will be swapped:      100.4145      564  564\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.3686      566  566\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      137.3813      567  567\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      138.5201      576  576\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.0195      582  582\n",
      "iupac_m_486_> OE1/2 will be swapped:     -153.0737      585  585\n",
      "iupac_m_485_> OD1/2 will be swapped:     -158.1960      600  600\n",
      "iupac_m_486_> OE1/2 will be swapped:     -138.3613      601  601\n",
      "iupac_m_486_> OE1/2 will be swapped:      105.7864      602  602\n",
      "iupac_m_484W> Dihedral still outside +-90:     -112.9351\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.2287      629  629\n",
      "iupac_m_487_> NH1/2 swapped:      177.5602      630  630\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.5587      639  639\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.8104      649  649\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.9330      656  656\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2734      670  670\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.5997      693  693\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.9047      709  709\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -93.8370      719  719\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      142.5430      721  721\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -113.6363      723  723\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -119.8680      725  725\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   464562\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -17959.5645\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -17959.564453\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    53.912998\n",
      "Sequence length          :          728\n",
      "Compactness              :     0.000340\n",
      "Native energy (pair)     :  1005.864381\n",
      "Native energy (surface)  :   110.126133\n",
      "Native energy (combined) :    30.712683\n",
      "Z score (pair)           :    -2.121711\n",
      "Z score (surface)        :    -0.791739\n",
      "Z score (combined)       :    -1.981667\n",
      "GA341 score              :     0.450467\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     8652\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      375530.3750\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5789     448    555   0.423   0.423     0.30690E+06   1.000\n",
      " 2 Bond angle potential               :    7845     507    858  10.063  10.063      16790.       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3676       0    299  56.641  56.641      2035.9       1.000\n",
      " 4 Stereochemical improper torsion pot:    2409       9     36   2.848   2.848      382.86       1.000\n",
      " 5 Soft-sphere overlap restraints     :    8652       3      7   0.009   0.009      75.444       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    1992     218    423   4.497   4.497      24049.       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2093     190    399   4.972   4.972      21693.       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     727       9     41   6.100   6.100      319.06       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     627       0     57  94.142  94.142      505.60       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     449       0     18  81.993  81.993      302.54       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     214       0      7 106.667 106.667      174.14       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      1  92.623  92.623      47.594       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1112       0      0   0.863   0.863      84.231       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     726     520    243  55.674 155.948      1795.8       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     471       4     16   3.122   3.122      377.82       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00192.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00192.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  236710.0312\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     84  10E  10E C   CA     81   75    1.65    1.49    0.16    4.73    1.49    0.16    4.73\n",
      "    2     87  11A  11A N   CA     83   84    1.57    1.43    0.14    4.54    1.43    0.14    4.54\n",
      "    3     89  11A  11A C   CA     86   84    1.66    1.49    0.17    5.01    1.49    0.17    5.01\n",
      "    4     90  11A  12C C   N      86   88    1.47    1.35    0.13    4.57    1.35    0.13    4.57\n",
      "    5     93  12C  12C N   CA     88   89    1.58    1.43    0.15    4.85    1.43    0.15    4.85\n",
      "    6     95  12C  12C C   CA     92   89    1.68    1.49    0.19    5.50    1.49    0.19    5.50\n",
      "    7     96  12C  13A C   N      92   94    1.48    1.35    0.14    4.90    1.35    0.14    4.90\n",
      "    8     98  13A  13A N   CA     94   95    1.59    1.43    0.16    5.25    1.43    0.16    5.25\n",
      "    9    100  13A  13A C   CA     97   95    1.69    1.49    0.20    5.72    1.49    0.20    5.72\n",
      "   10    101  13A  14H C   N      97   99    1.50    1.35    0.15    5.36    1.35    0.15    5.36\n",
      "   11    108  14H  14H N   CA     99  100    1.60    1.43    0.17    5.73    1.43    0.17    5.73\n",
      "   12    110  14H  14H C   CA    107  100    1.72    1.49    0.23    6.79    1.49    0.23    6.79\n",
      "   13    111  14H  15C C   N     107  109    1.51    1.35    0.17    5.93    1.35    0.17    5.93\n",
      "   14    115  15C  15C N   CA    109  110    1.62    1.43    0.19    6.40    1.43    0.19    6.40\n",
      "   15    117  15C  15C C   CA    113  110    1.74    1.49    0.25    7.18    1.49    0.25    7.18\n",
      "   16    118  15C  16W C   N     113  115    1.53    1.35    0.18    6.47    1.35    0.18    6.47\n",
      "   17    128  16W  16W N   CA    115  116    1.64    1.43    0.21    6.88    1.43    0.21    6.88\n",
      "   18    130  16W  16W C   CA    127  116    1.77    1.49    0.28    8.08    1.49    0.28    8.08\n",
      "   19    131  16W  17D C   N     127  129    1.55    1.35    0.21    7.32    1.35    0.21    7.32\n",
      "   20    139  17D  17D N   CA    129  130    1.67    1.43    0.24    7.92    1.43    0.24    7.92\n",
      "   21    141  17D  17D C   CA    135  130    1.79    1.49    0.30    8.70    1.49    0.30    8.70\n",
      "   22    142  17D  18E C   N     135  137    1.56    1.35    0.22    7.70    1.35    0.22    7.70\n",
      "   23    148  18E  18E N   CA    137  138    1.68    1.43    0.25    8.29    1.43    0.25    8.29\n",
      "   24    150  18E  18E C   CA    144  138    1.81    1.49    0.32    9.28    1.49    0.32    9.28\n",
      "   25    151  18E  19S C   N     144  146    1.57    1.35    0.23    8.08    1.35    0.23    8.08\n",
      "   26    154  19S  19S N   CA    146  147    1.70    1.43    0.27    8.74    1.43    0.27    8.74\n",
      "   27    156  19S  19S C   CA    150  147    1.82    1.49    0.33    9.62    1.49    0.33    9.62\n",
      "   28    157  19S  20V C   N     150  152    1.58    1.35    0.24    8.48    1.35    0.24    8.48\n",
      "   29    161  20V  20V N   CA    152  153    1.71    1.43    0.28    9.18    1.43    0.28    9.18\n",
      "   30    163  20V  20V C   CA    157  153    1.83    1.49    0.34   10.00    1.49    0.34   10.00\n",
      "   31    164  20V  21M C   N     157  159    1.59    1.35    0.25    8.84    1.35    0.25    8.84\n",
      "   32    169  21M  21M N   CA    159  160    1.72    1.43    0.29    9.53    1.43    0.29    9.53\n",
      "   33    171  21M  21M C   CA    165  160    1.85    1.49    0.36   10.55    1.49    0.36   10.55\n",
      "   34    172  21M  22L C   N     165  167    1.60    1.35    0.26    9.10    1.35    0.26    9.10\n",
      "   35    177  22L  22L N   CA    167  168    1.73    1.43    0.30    9.79    1.43    0.30    9.79\n",
      "   36    179  22L  22L C   CA    173  168    1.87    1.49    0.38   11.05    1.49    0.38   11.05\n",
      "   37    180  22L  23V C   N     173  175    1.62    1.35    0.27    9.67    1.35    0.27    9.67\n",
      "   38    184  23V  23V N   CA    175  176    1.75    1.43    0.32   10.41    1.43    0.32   10.41\n",
      "   39    186  23V  23V C   CA    180  176    1.90    1.49    0.41   11.84    1.49    0.41   11.84\n",
      "   40    187  23V  24P C   N     180  182    1.69    1.30    0.39   11.57    1.30    0.39   11.57\n",
      "   41    189  24P  24P C   CA    187  183    1.91    1.49    0.42   12.07    1.49    0.42   12.07\n",
      "   42    190  24P  25N C   N     187  189    1.64    1.35    0.29   10.32    1.35    0.29   10.32\n",
      "   43    191  24P  24P N   CA    182  183    1.77    1.43    0.34   11.09    1.43    0.34   11.09\n",
      "   44    200  25N  25N N   CA    189  190    1.77    1.43    0.34   11.18    1.43    0.34   11.18\n",
      "   45    202  25N  25N C   CA    195  190    1.92    1.49    0.43   12.61    1.49    0.43   12.61\n",
      "   46    203  25N  26H C   N     195  197    1.65    1.35    0.30   10.79    1.35    0.30   10.79\n",
      "   47    210  26H  26H N   CA    197  198    1.79    1.43    0.36   11.70    1.43    0.36   11.70\n",
      "   48    212  26H  26H C   CA    205  198    1.95    1.49    0.46   13.24    1.49    0.46   13.24\n",
      "   49    213  26H  27R C   N     205  207    1.67    1.35    0.33   11.59    1.35    0.33   11.59\n",
      "   50    222  27R  27R N   CA    207  208    1.81    1.43    0.38   12.57    1.43    0.38   12.57\n",
      "   51    224  27R  27R C   CA    216  208    1.97    1.49    0.48   14.08    1.49    0.48   14.08\n",
      "   52    225  27R  28S C   N     216  218    1.68    1.35    0.34   11.94    1.35    0.34   11.94\n",
      "   53    228  28S  28S N   CA    218  219    1.82    1.43    0.39   12.95    1.43    0.39   12.95\n",
      "   54    230  28S  28S C   CA    222  219    1.99    1.49    0.50   14.45    1.49    0.50   14.45\n",
      "   55    231  28S  29A C   N     222  224    1.69    1.35    0.35   12.25    1.35    0.35   12.25\n",
      "   56    233  29A  29A N   CA    224  225    1.83    1.43    0.40   13.30    1.43    0.40   13.30\n",
      "   57    235  29A  29A C   CA    227  225    2.00    1.49    0.51   14.73    1.49    0.51   14.73\n",
      "   58    236  29A  30V C   N     227  229    1.70    1.35    0.35   12.48    1.35    0.35   12.48\n",
      "   59    240  30V  30V N   CA    229  230    1.84    1.43    0.41   13.55    1.43    0.41   13.55\n",
      "   60    242  30V  30V C   CA    234  230    2.01    1.49    0.52   15.18    1.49    0.52   15.18\n",
      "   61    243  30V  31H C   N     234  236    1.71    1.35    0.36   12.89    1.35    0.36   12.89\n",
      "   62    250  31H  31H N   CA    236  237    1.86    1.43    0.43   13.99    1.43    0.43   13.99\n",
      "   63    252  31H  31H C   CA    244  237    2.04    1.49    0.55   15.86    1.49    0.55   15.86\n",
      "   64    253  31H  32K C   N     244  246    1.72    1.35    0.38   13.39    1.35    0.38   13.39\n",
      "   65    260  32K  32K N   CA    246  247    1.87    1.43    0.44   14.54    1.43    0.44   14.54\n",
      "   66    262  32K  32K C   CA    253  247    2.05    1.49    0.56   16.38    1.49    0.56   16.38\n",
      "   67    263  32K  33E C   N     253  255    1.74    1.35    0.39   13.94    1.35    0.39   13.94\n",
      "   68    269  33E  33E N   CA    255  256    1.89    1.43    0.46   15.16    1.43    0.46   15.16\n",
      "   69    271  33E  33E C   CA    262  256    2.07    1.49    0.58   16.90    1.49    0.58   16.90\n",
      "   70    272  33E  34E C   N     262  264    1.75    1.35    0.40   14.28    1.35    0.40   14.28\n",
      "   71    278  34E  34E N   CA    264  265    1.90    1.43    0.47   15.49    1.43    0.47   15.49\n",
      "   72    280  34E  34E C   CA    271  265    2.09    1.49    0.60   17.41    1.49    0.60   17.41\n",
      "   73    281  34E  35C C   N     271  273    1.76    1.35    0.41   14.63    1.35    0.41   14.63\n",
      "   74    284  35C  35C N   CA    273  274    1.91    1.43    0.48   15.87    1.43    0.48   15.87\n",
      "   75    286  35C  35C C   CA    277  274    2.10    1.49    0.61   17.75    1.49    0.61   17.75\n",
      "   76    287  35C  36A C   N     277  279    1.77    1.35    0.42   14.92    1.35    0.42   14.92\n",
      "   77    289  36A  36A N   CA    279  280    1.92    1.43    0.49   16.21    1.43    0.49   16.21\n",
      "   78    291  36A  36A C   CA    282  280    2.11    1.49    0.62   18.08    1.49    0.62   18.08\n",
      "   79    292  36A  37Y C   N     282  284    1.77    1.35    0.43   15.21    1.35    0.43   15.21\n",
      "   80    302  37Y  37Y N   CA    284  285    1.93    1.43    0.50   16.54    1.43    0.50   16.54\n",
      "   81    304  37Y  37Y C   CA    294  285    2.14    1.49    0.65   18.94    1.49    0.65   18.94\n",
      "   82    305  37Y  38C C   N     294  296    1.79    1.35    0.45   15.90    1.35    0.45   15.90\n",
      "   83    308  38C  38C N   CA    296  297    1.96    1.43    0.53   17.30    1.43    0.53   17.30\n",
      "   84    310  38C  38C C   CA    300  297    2.15    1.49    0.66   19.31    1.49    0.66   19.31\n",
      "   85    311  38C  39C C   N     300  302    1.80    1.35    0.46   16.22    1.35    0.46   16.22\n",
      "   86    314  39C  39C N   CA    302  303    1.97    1.43    0.54   17.63    1.43    0.54   17.63\n",
      "   87    316  39C  39C C   CA    306  303    2.16    1.49    0.67   19.60    1.49    0.67   19.60\n",
      "   88    317  39C  40R C   N     306  308    1.81    1.35    0.46   16.48    1.35    0.46   16.48\n",
      "   89    325  40R  40R N   CA    308  309    1.97    1.43    0.54   17.87    1.43    0.54   17.87\n",
      "   90    327  40R  40R C   CA    317  309    2.19    1.49    0.70   20.40    1.49    0.70   20.40\n",
      "   91    328  40R  41T C   N     317  319    1.83    1.35    0.48   17.05    1.35    0.48   17.05\n",
      "   92    332  41T  41T N   CA    319  320    1.99    1.43    0.56   18.52    1.43    0.56   18.52\n",
      "   93    334  41T  41T C   CA    324  320    2.20    1.49    0.71   20.65    1.49    0.71   20.65\n",
      "   94    335  41T  42C C   N     324  326    1.83    1.35    0.49   17.25    1.35    0.49   17.25\n",
      "   95    338  42C  42C N   CA    326  327    2.00    1.43    0.57   18.71    1.43    0.57   18.71\n",
      "   96    340  42C  42C C   CA    330  327    2.21    1.49    0.72   21.01    1.49    0.72   21.01\n",
      "   97    341  42C  43R C   N     330  332    1.84    1.35    0.49   17.53    1.35    0.49   17.53\n",
      "   98    349  43R  43R N   CA    332  333    2.01    1.43    0.58   18.99    1.43    0.58   18.99\n",
      "   99    351  43R  43R C   CA    341  333    2.23    1.49    0.74   21.53    1.49    0.74   21.53\n",
      "  100    352  43R  44H C   N     341  343    1.85    1.35    0.51   17.93    1.35    0.51   17.93\n",
      "  101    359  44H  44H N   CA    343  344    2.02    1.43    0.59   19.47    1.43    0.59   19.47\n",
      "  102    361  44H  44H C   CA    351  344    2.24    1.49    0.75   21.88    1.49    0.75   21.88\n",
      "  103    362  44H  45E C   N     351  353    1.86    1.35    0.52   18.42    1.35    0.52   18.42\n",
      "  104    369  45E  45E N   CA    353  354    2.04    1.43    0.61   20.02    1.43    0.61   20.02\n",
      "  105    371  45E  45E C   CA    360  354    2.26    1.49    0.77   22.46    1.49    0.77   22.46\n",
      "  106    372  45E  46M C   N     360  362    1.87    1.35    0.53   18.78    1.35    0.53   18.78\n",
      "  107    377  46M  46M N   CA    362  363    2.05    1.43    0.62   20.36    1.43    0.62   20.36\n",
      "  108    379  46M  46M C   CA    368  363    2.28    1.49    0.79   23.08    1.49    0.79   23.08\n",
      "  109    380  46M  47G C   N     368  370    1.89    1.35    0.54   19.24    1.35    0.54   19.24\n",
      "  110    381  47G  47G N   CA    370  371    1.99    1.43    0.56   18.44    1.43    0.56   18.44\n",
      "  111    383  47G  47G C   CA    372  371    2.03    1.49    0.54   15.58    1.49    0.54   15.58\n",
      "  112    384  47G  48T C   N     372  374    1.59    1.35    0.25    8.77    1.35    0.25    8.77\n",
      "  113    388  48T  48T N   CA    374  375    1.72    1.43    0.29    9.38    1.43    0.29    9.38\n",
      "  114   1825 236A 236A C   CA   1780 1778    1.70    1.49    0.21    6.18    1.49    0.21    6.18\n",
      "  115   1845 238F 238F C   CA   1799 1791    1.69    1.49    0.20    5.86    1.49    0.20    5.86\n",
      "  116   2035 260G 260G C   CA   1982 1981    1.65    1.49    0.16    4.59    1.49    0.16    4.59\n",
      "  117   2041 261I 261I N   CA   1984 1985    1.59    1.43    0.16    5.10    1.43    0.16    5.10\n",
      "  118   2043 261I 261I C   CA   1990 1985    1.74    1.49    0.25    7.37    1.49    0.25    7.37\n",
      "  119   2044 261I 262D C   N    1990 1992    1.53    1.35    0.18    6.50    1.35    0.18    6.50\n",
      "  120   2049 262D 262D N   CA   1992 1993    1.72    1.43    0.29    9.65    1.43    0.29    9.65\n",
      "  121   2051 262D 262D C   CA   1998 1993    1.85    1.49    0.36   10.59    1.49    0.36   10.59\n",
      "  122   2052 262D 263I C   N    1998 2000    1.61    1.35    0.27    9.52    1.35    0.27    9.52\n",
      "  123   2057 263I 263I N   CA   2000 2001    1.73    1.43    0.30    9.95    1.43    0.30    9.95\n",
      "  124   2059 263I 263I C   CA   2006 2001    1.84    1.49    0.35   10.24    1.49    0.35   10.24\n",
      "  125   2060 263I 264Q C   N    2006 2008    1.57    1.35    0.22    7.86    1.35    0.22    7.86\n",
      "  126   2066 264Q 264Q N   CA   2008 2009    1.69    1.43    0.26    8.52    1.43    0.26    8.52\n",
      "  127   2068 264Q 264Q C   CA   2015 2009    1.84    1.49    0.35   10.20    1.49    0.35   10.20\n",
      "  128   2069 264Q 265T C   N    2015 2017    1.56    1.35    0.21    7.59    1.35    0.21    7.59\n",
      "  129   2073 265T 265T N   CA   2017 2018    1.68    1.43    0.25    8.28    1.43    0.25    8.28\n",
      "  130   2075 265T 265T C   CA   2022 2018    1.77    1.49    0.28    8.05    1.49    0.28    8.05\n",
      "  131   2076 265T 266A C   N    2022 2024    1.53    1.35    0.19    6.71    1.35    0.19    6.71\n",
      "  132   2078 266A 266A N   CA   2024 2025    1.65    1.43    0.22    7.09    1.43    0.22    7.09\n",
      "  133   2080 266A 266A C   CA   2027 2025    1.75    1.49    0.26    7.67    1.49    0.26    7.67\n",
      "  134   2081 266A 267K C   N    2027 2029    1.54    1.35    0.19    6.74    1.35    0.19    6.74\n",
      "  135   2087 267K 267K N   CA   2029 2030    1.65    1.43    0.22    7.31    1.43    0.22    7.31\n",
      "  136   2089 267K 267K C   CA   2036 2030    1.77    1.49    0.28    8.00    1.49    0.28    8.00\n",
      "  137   2090 267K 268K C   N    2036 2038    1.55    1.35    0.20    7.20    1.35    0.20    7.20\n",
      "  138   2096 268K 268K N   CA   2038 2039    1.66    1.43    0.23    7.63    1.43    0.23    7.63\n",
      "  139   2098 268K 268K C   CA   2045 2039    1.79    1.49    0.30    8.73    1.49    0.30    8.73\n",
      "  140   2099 268K 269T C   N    2045 2047    1.55    1.35    0.20    7.24    1.35    0.20    7.24\n",
      "  141   2103 269T 269T N   CA   2047 2048    1.67    1.43    0.24    7.77    1.43    0.24    7.77\n",
      "  142   2105 269T 269T C   CA   2052 2048    1.76    1.49    0.27    7.88    1.49    0.27    7.88\n",
      "  143   2106 269T 270A C   N    2052 2054    1.54    1.35    0.20    6.97    1.35    0.20    6.97\n",
      "  144   2108 270A 270A N   CA   2054 2055    1.65    1.43    0.22    7.29    1.43    0.22    7.29\n",
      "  145   2110 270A 270A C   CA   2057 2055    1.76    1.49    0.27    7.83    1.49    0.27    7.83\n",
      "  146   2111 270A 271K C   N    2057 2059    1.53    1.35    0.19    6.70    1.35    0.19    6.70\n",
      "  147   2117 271K 271K N   CA   2059 2060    1.65    1.43    0.22    7.23    1.43    0.22    7.23\n",
      "  148   2119 271K 271K C   CA   2066 2060    1.76    1.49    0.27    7.77    1.49    0.27    7.77\n",
      "  149   2120 271K 272T C   N    2066 2068    1.54    1.35    0.19    6.76    1.35    0.19    6.76\n",
      "  150   2124 272T 272T N   CA   2068 2069    1.65    1.43    0.22    7.20    1.43    0.22    7.20\n",
      "  151   2126 272T 272T C   CA   2073 2069    1.77    1.49    0.28    8.19    1.49    0.28    8.19\n",
      "  152   2127 272T 273L C   N    2073 2075    1.54    1.35    0.20    6.98    1.35    0.20    6.98\n",
      "  153   2132 273L 273L N   CA   2075 2076    1.65    1.43    0.22    7.39    1.43    0.22    7.39\n",
      "  154   2134 273L 273L C   CA   2081 2076    1.76    1.49    0.27    7.83    1.49    0.27    7.83\n",
      "  155   2135 273L 274G C   N    2081 2083    1.53    1.35    0.19    6.62    1.35    0.19    6.62\n",
      "  156   2136 274G 274G N   CA   2083 2084    1.65    1.43    0.22    7.12    1.43    0.22    7.12\n",
      "  157   2138 274G 274G C   CA   2085 2084    1.75    1.49    0.26    7.62    1.49    0.26    7.62\n",
      "  158   2139 274G 275E C   N    2085 2087    1.53    1.35    0.18    6.47    1.35    0.18    6.47\n",
      "  159   2145 275E 275E N   CA   2087 2088    1.64    1.43    0.21    6.93    1.43    0.21    6.93\n",
      "  160   2147 275E 275E C   CA   2094 2088    1.73    1.49    0.24    6.89    1.49    0.24    6.89\n",
      "  161   2148 275E 276M C   N    2094 2096    1.51    1.35    0.17    5.90    1.35    0.17    5.90\n",
      "  162   2153 276M 276M N   CA   2096 2097    1.62    1.43    0.19    6.24    1.43    0.19    6.24\n",
      "  163   2155 276M 276M C   CA   2102 2097    1.72    1.49    0.23    6.62    1.49    0.23    6.62\n",
      "  164   2156 276M 277Q C   N    2102 2104    1.50    1.35    0.16    5.59    1.35    0.16    5.59\n",
      "  165   2162 277Q 277Q N   CA   2104 2105    1.61    1.43    0.18    5.92    1.43    0.18    5.92\n",
      "  166   2164 277Q 277Q C   CA   2111 2105    1.70    1.49    0.21    6.05    1.49    0.21    6.05\n",
      "  167   2165 277Q 278Y C   N    2111 2113    1.49    1.35    0.15    5.16    1.35    0.15    5.16\n",
      "  168   2175 278Y 278Y N   CA   2113 2114    1.60    1.43    0.17    5.48    1.43    0.17    5.48\n",
      "  169   2177 278Y 278Y C   CA   2123 2114    1.68    1.49    0.19    5.63    1.49    0.19    5.63\n",
      "  170   2178 278Y 279D C   N    2123 2125    1.48    1.35    0.13    4.72    1.35    0.13    4.72\n",
      "  171   2183 279D 279D N   CA   2125 2126    1.58    1.43    0.15    4.83    1.43    0.15    4.83\n",
      "  172   2185 279D 279D C   CA   2131 2126    1.67    1.49    0.18    5.09    1.49    0.18    5.09\n",
      "  173   2198 280Y 280Y C   CA   2143 2134    1.65    1.49    0.16    4.67    1.49    0.16    4.67\n",
      "  174   4422 555V 555V C   CA   4318 4314    1.65    1.49    0.16    4.65    1.49    0.16    4.65\n",
      "  175   4434 556F 556F C   CA   4329 4321    1.65    1.49    0.16    4.64    1.49    0.16    4.64\n",
      "  176   4439 557V 557V N   CA   4331 4332    1.57    1.43    0.14    4.51    1.43    0.14    4.51\n",
      "  177   4441 557V 557V C   CA   4336 4332    1.67    1.49    0.18    5.34    1.49    0.18    5.34\n",
      "  178   4442 557V 558D C   N    4336 4338    1.48    1.35    0.13    4.62    1.35    0.13    4.62\n",
      "  179   4447 558D 558D N   CA   4338 4339    1.58    1.43    0.15    5.08    1.43    0.15    5.08\n",
      "  180   4449 558D 558D C   CA   4344 4339    1.69    1.49    0.20    5.86    1.49    0.20    5.86\n",
      "  181   4450 558D 559V C   N    4344 4346    1.50    1.35    0.16    5.60    1.35    0.16    5.60\n",
      "  182   4454 559V 559V N   CA   4346 4347    1.61    1.43    0.18    5.93    1.43    0.18    5.93\n",
      "  183   4456 559V 559V C   CA   4351 4347    1.73    1.49    0.24    7.04    1.49    0.24    7.04\n",
      "  184   4457 559V 560P C   N    4351 4353    1.53    1.30    0.23    6.85    1.30    0.23    6.85\n",
      "  185   4459 560P 560P C   CA   4358 4354    1.75    1.49    0.26    7.45    1.49    0.26    7.45\n",
      "  186   4460 560P 561L C   N    4358 4360    1.53    1.35    0.19    6.69    1.35    0.19    6.69\n",
      "  187   4461 560P 560P N   CA   4353 4354    1.65    1.43    0.21    7.07    1.43    0.21    7.07\n",
      "  188   4470 561L 561L N   CA   4360 4361    1.65    1.43    0.22    7.28    1.43    0.22    7.28\n",
      "  189   4472 561L 561L C   CA   4366 4361    1.78    1.49    0.29    8.56    1.49    0.29    8.56\n",
      "  190   4473 561L 562E C   N    4366 4368    1.56    1.35    0.21    7.48    1.35    0.21    7.48\n",
      "  191   4479 562E 562E N   CA   4368 4369    1.68    1.43    0.25    8.10    1.43    0.25    8.10\n",
      "  192   4481 562E 562E C   CA   4375 4369    1.82    1.49    0.33    9.64    1.49    0.33    9.64\n",
      "  193   4482 562E 563L C   N    4375 4377    1.59    1.35    0.25    8.74    1.35    0.25    8.74\n",
      "  194   4487 563L 563L N   CA   4377 4378    1.72    1.43    0.29    9.61    1.43    0.29    9.61\n",
      "  195   4489 563L 563L C   CA   4383 4378    1.86    1.49    0.37   10.89    1.49    0.37   10.89\n",
      "  196   4490 563L 564D C   N    4383 4385    1.62    1.35    0.27    9.60    1.35    0.27    9.60\n",
      "  197   4495 564D 564D N   CA   4385 4386    1.75    1.43    0.32   10.49    1.43    0.32   10.49\n",
      "  198   4497 564D 564D C   CA   4391 4386    1.90    1.49    0.41   11.84    1.49    0.41   11.84\n",
      "  199   4498 564D 565L C   N    4391 4393    1.64    1.35    0.30   10.48    1.35    0.30   10.48\n",
      "  200   4503 565L 565L N   CA   4393 4394    1.78    1.43    0.35   11.42    1.43    0.35   11.42\n",
      "  201   4505 565L 565L C   CA   4399 4394    1.94    1.49    0.45   13.13    1.49    0.45   13.13\n",
      "  202   4506 565L 566E C   N    4399 4401    1.66    1.35    0.32   11.31    1.35    0.32   11.31\n",
      "  203   4512 566E 566E N   CA   4401 4402    1.80    1.43    0.37   12.30    1.43    0.37   12.30\n",
      "  204   4514 566E 566E C   CA   4408 4402    1.97    1.49    0.48   13.92    1.49    0.48   13.92\n",
      "  205   4515 566E 567F C   N    4408 4410    1.69    1.35    0.35   12.32    1.35    0.35   12.32\n",
      "  206   4524 567F 567F N   CA   4410 4411    1.84    1.43    0.41   13.42    1.43    0.41   13.42\n",
      "  207   4526 567F 567F C   CA   4419 4411    2.03    1.49    0.54   15.83    1.49    0.54   15.83\n",
      "  208   4527 567F 568L C   N    4419 4421    1.74    1.35    0.39   13.91    1.35    0.39   13.91\n",
      "  209   4532 568L 568L N   CA   4421 4422    1.89    1.43    0.46   15.06    1.43    0.46   15.06\n",
      "  210   4534 568L 568L C   CA   4427 4422    2.07    1.49    0.58   16.97    1.49    0.58   16.97\n",
      "  211   4535 568L 569R C   N    4427 4429    1.75    1.35    0.41   14.53    1.35    0.41   14.53\n",
      "  212   4543 569R 569R N   CA   4429 4430    1.91    1.43    0.48   15.73    1.43    0.48   15.73\n",
      "  213   4545 569R 569R C   CA   4438 4430    2.11    1.49    0.62   18.15    1.49    0.62   18.15\n",
      "  214   4546 569R 570G C   N    4438 4440    1.77    1.35    0.42   14.92    1.35    0.42   14.92\n",
      "  215   4547 570G 570G N   CA   4440 4441    1.92    1.43    0.49   16.23    1.43    0.49   16.23\n",
      "  216   4549 570G 570G C   CA   4442 4441    2.11    1.49    0.62   17.94    1.49    0.62   17.94\n",
      "  217   4550 570G 571K C   N    4442 4444    1.76    1.35    0.41   14.55    1.35    0.41   14.55\n",
      "  218   4556 571K 571K N   CA   4444 4445    1.91    1.43    0.48   15.85    1.43    0.48   15.85\n",
      "  219   4558 571K 571K C   CA   4451 4445    2.09    1.49    0.60   17.36    1.49    0.60   17.36\n",
      "  220   4559 571K 572G C   N    4451 4453    1.76    1.35    0.42   14.80    1.35    0.42   14.80\n",
      "  221   4560 572G 572G N   CA   4453 4454    1.92    1.43    0.49   16.26    1.43    0.49   16.26\n",
      "  222   4562 572G 572G C   CA   4455 4454    2.11    1.49    0.62   18.16    1.49    0.62   18.16\n",
      "  223   4563 572G 573L C   N    4455 4457    1.79    1.35    0.44   15.75    1.35    0.44   15.75\n",
      "  224   4568 573L 573L N   CA   4457 4458    1.95    1.43    0.52   17.24    1.43    0.52   17.24\n",
      "  225   4570 573L 573L C   CA   4463 4458    2.17    1.49    0.68   19.66    1.49    0.68   19.66\n",
      "  226   4571 573L 574Q C   N    4463 4465    1.80    1.35    0.46   16.20    1.35    0.46   16.20\n",
      "  227   4577 574Q 574Q N   CA   4465 4466    1.98    1.43    0.55   18.21    1.43    0.55   18.21\n",
      "  228   4579 574Q 574Q C   CA   4472 4466    2.20    1.49    0.71   20.73    1.49    0.71   20.73\n",
      "  229   4580 574Q 575S C   N    4472 4474    1.83    1.35    0.48   17.08    1.35    0.48   17.08\n",
      "  230   4583 575S 575S N   CA   4474 4475    2.00    1.43    0.57   18.61    1.43    0.57   18.61\n",
      "  231   4585 575S 575S C   CA   4478 4475    2.20    1.49    0.71   20.55    1.49    0.71   20.55\n",
      "  232   4586 575S 576D C   N    4478 4480    1.84    1.35    0.49   17.48    1.35    0.49   17.48\n",
      "  233   4591 576D 576D N   CA   4480 4481    2.01    1.43    0.58   19.02    1.43    0.58   19.02\n",
      "  234   4593 576D 576D C   CA   4486 4481    2.24    1.49    0.75   21.92    1.49    0.75   21.92\n",
      "  235   4594 576D 577E C   N    4486 4488    1.86    1.35    0.52   18.33    1.35    0.52   18.33\n",
      "  236   4600 577E 577E N   CA   4488 4489    2.04    1.43    0.61   20.03    1.43    0.61   20.03\n",
      "  237   4602 577E 577E C   CA   4495 4489    2.29    1.49    0.80   23.14    1.49    0.80   23.14\n",
      "  238   4603 577E 578V C   N    4495 4497    1.88    1.35    0.54   19.09    1.35    0.54   19.09\n",
      "  239   4607 578V 578V N   CA   4497 4498    2.07    1.43    0.64   21.12    1.43    0.64   21.12\n",
      "  240   4609 578V 578V C   CA   4502 4498    2.35    1.49    0.86   25.05    1.49    0.86   25.05\n",
      "  241   4610 578V 579S C   N    4502 4504    1.97    1.35    0.62   21.99    1.35    0.62   21.99\n",
      "  242   4613 579S 579S N   CA   4504 4505    2.17    1.43    0.74   24.34    1.43    0.74   24.34\n",
      "  243   4615 579S 579S C   CA   4508 4505    2.53    1.49    1.04   30.27    1.49    1.04   30.27\n",
      "  244   4616 579S 580M C   N    4508 4510    2.08    1.35    0.74   26.12    1.35    0.74   26.12\n",
      "  245   4621 580M 580M N   CA   4510 4511    2.32    1.43    0.89   29.43    1.43    0.89   29.43\n",
      "  246   4623 580M 580M C   CA   4516 4511    2.64    1.49    1.15   33.53    1.49    1.15   33.53\n",
      "  247   4624 580M 581P C   N    4516 4518    2.43    1.30    1.13   33.39    1.30    1.13   33.39\n",
      "  248   4626 581P 581P C   CA   4523 4519    2.80    1.49    1.31   38.01    1.49    1.31   38.01\n",
      "  249   4627 581P 582E C   N    4523 4525    2.25    1.35    0.91   32.15    1.35    0.91   32.15\n",
      "  250   4628 581P 581P N   CA   4518 4519    2.36    1.43    0.93   30.59    1.43    0.93   30.59\n",
      "  251   4638 582E 582E N   CA   4525 4526    2.52    1.43    1.09   35.77    1.43    1.09   35.77\n",
      "  252   4640 582E 582E C   CA   4532 4526    2.95    1.49    1.46   42.48    1.49    1.46   42.48\n",
      "  253   4641 582E 583A C   N    4532 4534    2.38    1.35    1.03   36.61    1.35    1.03   36.61\n",
      "  254   4643 583A 583A N   CA   4534 4535    2.67    1.43    1.24   40.94    1.43    1.24   40.94\n",
      "  255   4645 583A 583A C   CA   4537 4535    3.17    1.49    1.68   48.95    1.49    1.68   48.95\n",
      "  256   4646 583A 584Q C   N    4537 4539    2.53    1.35    1.19   42.14    1.35    1.19   42.14\n",
      "  257   4652 584Q 584Q N   CA   4539 4540    2.84    1.43    1.41   46.45    1.43    1.41   46.45\n",
      "  258   4654 584Q 584Q C   CA   4546 4540    3.40    1.49    1.91   55.62    1.49    1.91   55.62\n",
      "  259   4655 584Q 585E C   N    4546 4548    2.70    1.35    1.35   47.96    1.35    1.35   47.96\n",
      "  260   4661 585E 585E N   CA   4548 4549    3.04    1.43    1.61   53.00    1.43    1.61   53.00\n",
      "  261   4663 585E 585E C   CA   4555 4549    3.68    1.49    2.19   63.62    1.49    2.19   63.62\n",
      "  262   4664 585E 586D C   N    4555 4557    2.87    1.35    1.53   54.16    1.35    1.53   54.16\n",
      "  263   4669 586D 586D N   CA   4557 4558    3.25    1.43    1.82   59.85    1.43    1.82   59.85\n",
      "  264   4671 586D 586D C   CA   4563 4558    3.85    1.49    2.36   68.46    1.49    2.36   68.46\n",
      "  265   4672 586D 587L C   N    4563 4565    2.99    1.35    1.65   58.35    1.35    1.65   58.35\n",
      "  266   4677 587L 587L N   CA   4565 4566    3.38    1.43    1.95   64.02    1.43    1.95   64.02\n",
      "  267   4679 587L 587L C   CA   4571 4566    3.96    1.49    2.47   71.83    1.49    2.47   71.83\n",
      "  268   4680 587L 588P C   N    4571 4573    3.74    1.30    2.44   72.54    1.30    2.44   72.54\n",
      "  269   4682 588P 588P C   CA   4578 4574    4.19    1.49    2.70   78.55    1.49    2.70   78.55\n",
      "  270   4683 588P 589H C   N    4578 4580    3.22    1.35    1.88   66.58    1.35    1.88   66.58\n",
      "  271   4684 588P 588P N   CA   4573 4574    3.40    1.43    1.96   64.60    1.43    1.96   64.60\n",
      "  272   4695 589H 589H N   CA   4580 4581    3.65    1.43    2.22   73.05    1.43    2.22   73.05\n",
      "  273   4697 589H 589H C   CA   4588 4581    4.49    1.49    3.00   87.10    1.49    3.00   87.10\n",
      "  274   4698 589H 590K C   N    4588 4590    3.44    1.35    2.10   74.40    1.35    2.10   74.40\n",
      "  275   4705 590K 590K N   CA   4590 4591    3.92    1.43    2.49   81.88    1.43    2.49   81.88\n",
      "  276   4707 590K 590K C   CA   4597 4591    4.80    1.49    3.31   96.15    1.49    3.31   96.15\n",
      "  277   4708 590K 591P C   N    4597 4599    4.58    1.30    3.28   97.48    1.30    3.28   97.48\n",
      "  278   4710 591P 591P C   CA   4604 4600    5.23    1.49    3.74  108.79    1.49    3.74  108.79\n",
      "  279   4711 591P 592T C   N    4604 4606    3.92    1.35    2.57   91.16    1.35    2.57   91.16\n",
      "  280   4712 591P 591P N   CA   4599 4600    4.10    1.43    2.67   87.74    1.43    2.67   87.74\n",
      "  281   4714 591P 591P CB  CG   4602 4603    1.76    1.54    0.23    6.21    1.54    0.23    6.21\n",
      "  282   4720 592T 592T N   CA   4606 4607    4.45    1.43    3.02   99.45    1.43    3.02   99.45\n",
      "  283   4722 592T 592T C   CA   4611 4607    5.48    1.49    3.99  115.88    1.49    3.99  115.88\n",
      "  284   4723 592T 593S C   N    4611 4613    4.05    1.35    2.71   96.05    1.35    2.71   96.05\n",
      "  285   4726 593S 593S N   CA   4613 4614    4.61    1.43    3.18  104.45    1.43    3.18  104.45\n",
      "  286   4728 593S 593S C   CA   4617 4614    5.65    1.49    4.16  120.85    1.49    4.16  120.85\n",
      "  287   4729 593S 594S C   N    4617 4619    4.19    1.35    2.84  100.85    1.35    2.84  100.85\n",
      "  288   4732 594S 594S N   CA   4619 4620    4.76    1.43    3.33  109.69    1.43    3.33  109.69\n",
      "  289   4734 594S 594S C   CA   4623 4620    5.82    1.49    4.33  125.80    1.49    4.33  125.80\n",
      "  290   4735 594S 595S C   N    4623 4625    4.31    1.35    2.97  105.26    1.35    2.97  105.26\n",
      "  291   4738 595S 595S N   CA   4625 4626    4.93    1.43    3.50  114.97    1.43    3.50  114.97\n",
      "  292   4740 595S 595S C   CA   4629 4626    6.09    1.49    4.60  133.72    1.49    4.60  133.72\n",
      "  293   4741 595S 596M C   N    4629 4631    4.52    1.35    3.17  112.45    1.35    3.17  112.45\n",
      "  294   4746 596M 596M N   CA   4631 4632    5.17    1.43    3.74  122.88    1.43    3.74  122.88\n",
      "  295   4748 596M 596M C   CA   4637 4632    6.37    1.49    4.88  141.88    1.49    4.88  141.88\n",
      "  296   4749 596M 597K C   N    4637 4639    4.70    1.35    3.36  119.03    1.35    3.36  119.03\n",
      "  297   4755 597K 597K N   CA   4639 4640    5.38    1.43    3.95  129.93    1.43    3.95  129.93\n",
      "  298   4757 597K 597K C   CA   4646 4640    6.62    1.49    5.13  148.99    1.49    5.13  148.99\n",
      "  299   4758 597K 598T C   N    4646 4648    4.84    1.35    3.50  124.04    1.35    3.50  124.04\n",
      "  300   4762 598T 598T N   CA   4648 4649    5.52    1.43    4.09  134.63    1.43    4.09  134.63\n",
      "  301   4764 598T 598T C   CA   4653 4649    6.78    1.49    5.29  153.91    1.49    5.29  153.91\n",
      "  302   4765 598T 599V C   N    4653 4655    4.92    1.35    3.58  126.83    1.35    3.58  126.83\n",
      "  303   4769 599V 599V N   CA   4655 4656    5.61    1.43    4.18  137.46    1.43    4.18  137.46\n",
      "  304   4771 599V 599V C   CA   4660 4656    6.99    1.49    5.50  160.02    1.49    5.50  160.02\n",
      "  305   4772 599V 600D C   N    4660 4662    5.05    1.35    3.71  131.53    1.35    3.71  131.53\n",
      "  306   4777 600D 600D N   CA   4662 4663    5.76    1.43    4.33  142.46    1.43    4.33  142.46\n",
      "  307   4779 600D 600D C   CA   4668 4663    7.09    1.49    5.60  162.77    1.49    5.60  162.77\n",
      "  308   4780 600D 601E C   N    4668 4670    5.10    1.35    3.75  133.14    1.35    3.75  133.14\n",
      "  309   4786 601E 601E N   CA   4670 4671    5.82    1.43    4.39  144.30    1.43    4.39  144.30\n",
      "  310   4788 601E 601E C   CA   4677 4671    7.15    1.49    5.66  164.51    1.49    5.66  164.51\n",
      "  311   4789 601E 602E C   N    4677 4679    5.15    1.35    3.81  134.98    1.35    3.81  134.98\n",
      "  312   4795 602E 602E N   CA   4679 4680    5.86    1.43    4.43  145.72    1.43    4.43  145.72\n",
      "  313   4797 602E 602E C   CA   4686 4680    7.21    1.49    5.72  166.38    1.49    5.72  166.38\n",
      "  314   4798 602E 603A C   N    4686 4688    5.20    1.35    3.86  136.82    1.35    3.86  136.82\n",
      "  315   4800 603A 603A N   CA   4688 4689    5.93    1.43    4.50  147.98    1.43    4.50  147.98\n",
      "  316   4802 603A 603A C   CA   4691 4689    7.30    1.49    5.81  168.82    1.49    5.81  168.82\n",
      "  317   4803 603A 604L C   N    4691 4693    5.25    1.35    3.91  138.62    1.35    3.91  138.62\n",
      "  318   4808 604L 604L N   CA   4693 4694    5.97    1.43    4.54  149.40    1.43    4.54  149.40\n",
      "  319   4810 604L 604L C   CA   4699 4694    7.31    1.49    5.82  169.15    1.49    5.82  169.15\n",
      "  320   4811 604L 605A C   N    4699 4701    5.23    1.35    3.89  137.90    1.35    3.89  137.90\n",
      "  321   4813 605A 605A N   CA   4701 4702    5.94    1.43    4.51  148.41    1.43    4.51  148.41\n",
      "  322   4815 605A 605A C   CA   4704 4702    7.23    1.49    5.74  166.99    1.49    5.74  166.99\n",
      "  323   4816 605A 606M C   N    4704 4706    5.19    1.35    3.84  136.22    1.35    3.84  136.22\n",
      "  324   4821 606M 606M N   CA   4706 4707    4.88    1.43    3.45  113.64    1.43    3.45  113.64\n",
      "  325   4822 606M 606M O   C    4713 4712    1.49    1.23    0.26   11.72    1.23    0.26   11.72\n",
      "  326   4823 606M 606M C   CA   4712 4707    3.45    1.49    1.96   56.96    1.49    1.96   56.96\n",
      "  327   4824 606M 607L C   N    4712 4714    2.11    1.35    0.76   27.01    1.35    0.76   27.01\n",
      "  328   4829 607L 607L N   CA   4714 4715    2.08    1.43    0.65   21.48    1.43    0.65   21.48\n",
      "  329   4831 607L 607L C   CA   4720 4715    1.85    1.49    0.36   10.53    1.49    0.36   10.53\n",
      "  330   4832 607L 608L C   N    4720 4722    1.54    1.35    0.19    6.88    1.35    0.19    6.88\n",
      "  331   4837 608L 608L N   CA   4722 4723    1.70    1.43    0.27    8.72    1.43    0.27    8.72\n",
      "  332   4839 608L 608L C   CA   4728 4723    1.76    1.49    0.27    7.87    1.49    0.27    7.87\n",
      "  333   4840 608L 609S C   N    4728 4730    1.56    1.35    0.21    7.50    1.35    0.21    7.50\n",
      "  334   4843 609S 609S N   CA   4730 4731    1.67    1.43    0.24    7.94    1.43    0.24    7.94\n",
      "  335   4845 609S 609S C   CA   4734 4731    1.79    1.49    0.30    8.83    1.49    0.30    8.83\n",
      "  336   4846 609S 610M C   N    4734 4736    1.53    1.35    0.19    6.69    1.35    0.19    6.69\n",
      "  337   4851 610M 610M N   CA   4736 4737    1.66    1.43    0.23    7.44    1.43    0.23    7.44\n",
      "  338   4853 610M 610M C   CA   4742 4737    1.76    1.49    0.27    7.78    1.49    0.27    7.78\n",
      "  339   4854 610M 611G C   N    4742 4744    1.51    1.35    0.17    6.02    1.35    0.17    6.02\n",
      "  340   4855 611G 611G N   CA   4744 4745    1.63    1.43    0.20    6.57    1.43    0.20    6.57\n",
      "  341   4857 611G 611G C   CA   4746 4745    1.72    1.49    0.23    6.78    1.49    0.23    6.78\n",
      "  342   4858 611G 612I C   N    4746 4748    1.50    1.35    0.15    5.43    1.35    0.15    5.43\n",
      "  343   4863 612I 612I N   CA   4748 4749    1.60    1.43    0.17    5.55    1.43    0.17    5.55\n",
      "  344   4865 612I 612I C   CA   4754 4749    1.73    1.49    0.24    6.88    1.49    0.24    6.88\n",
      "  345   4866 612I 613E C   N    4754 4756    1.51    1.35    0.16    5.69    1.35    0.16    5.69\n",
      "  346   4872 613E 613E N   CA   4756 4757    1.61    1.43    0.18    5.98    1.43    0.18    5.98\n",
      "  347   4874 613E 613E C   CA   4763 4757    1.73    1.49    0.24    6.87    1.49    0.24    6.87\n",
      "  348   4875 613E 614E C   N    4763 4765    1.52    1.35    0.17    6.10    1.35    0.17    6.10\n",
      "  349   4881 614E 614E N   CA   4765 4766    1.63    1.43    0.20    6.48    1.43    0.20    6.48\n",
      "  350   4883 614E 614E C   CA   4772 4766    1.74    1.49    0.25    7.40    1.49    0.25    7.40\n",
      "  351   4884 614E 615T C   N    4772 4774    1.52    1.35    0.18    6.30    1.35    0.18    6.30\n",
      "  352   4888 615T 615T N   CA   4774 4775    1.64    1.43    0.21    6.79    1.43    0.21    6.79\n",
      "  353   4890 615T 615T C   CA   4779 4775    1.74    1.49    0.25    7.18    1.49    0.25    7.18\n",
      "  354   4891 615T 616V C   N    4779 4781    1.52    1.35    0.17    6.19    1.35    0.17    6.19\n",
      "  355   4895 616V 616V N   CA   4781 4782    1.63    1.43    0.20    6.56    1.43    0.20    6.56\n",
      "  356   4897 616V 616V C   CA   4786 4782    1.71    1.49    0.22    6.47    1.49    0.22    6.47\n",
      "  357   4898 616V 617A C   N    4786 4788    1.49    1.35    0.15    5.28    1.35    0.15    5.28\n",
      "  358   4900 617A 617A N   CA   4788 4789    1.60    1.43    0.17    5.57    1.43    0.17    5.57\n",
      "  359   4902 617A 617A C   CA   4791 4789    1.66    1.49    0.17    4.85    1.49    0.17    4.85\n",
      "  360   4926 619Y 619Y C   CA   4814 4805    1.65    1.49    0.16    4.55    1.49    0.16    4.55\n",
      "  361   4929 620A 620A N   CA   4816 4817    1.59    1.43    0.16    5.10    1.43    0.16    5.10\n",
      "  362   4931 620A 620A C   CA   4819 4817    1.66    1.49    0.17    4.91    1.49    0.17    4.91\n",
      "  363   5546 700L 700L C   CA   5420 5415    1.67    1.49    0.18    5.28    1.49    0.18    5.28\n",
      "  364   5547 700L 701L C   N    5420 5422    1.49    1.35    0.14    4.98    1.35    0.14    4.98\n",
      "  365   5552 701L 701L N   CA   5422 5423    1.60    1.43    0.17    5.44    1.43    0.17    5.44\n",
      "  366   5554 701L 701L C   CA   5428 5423    1.69    1.49    0.20    5.88    1.49    0.20    5.88\n",
      "  367   5555 701L 702F C   N    5428 5430    1.50    1.35    0.15    5.43    1.35    0.15    5.43\n",
      "  368   5564 702F 702F N   CA   5430 5431    1.61    1.43    0.18    5.89    1.43    0.18    5.89\n",
      "  369   5566 702F 702F C   CA   5439 5431    1.73    1.49    0.24    6.91    1.49    0.24    6.91\n",
      "  370   5567 702F 703N C   N    5439 5441    1.52    1.35    0.17    6.14    1.35    0.17    6.14\n",
      "  371   5572 703N 703N N   CA   5441 5442    1.63    1.43    0.20    6.64    1.43    0.20    6.64\n",
      "  372   5574 703N 703N C   CA   5447 5442    1.73    1.49    0.24    6.98    1.49    0.24    6.98\n",
      "  373   5575 703N 704D C   N    5447 5449    1.52    1.35    0.17    6.15    1.35    0.17    6.15\n",
      "  374   5580 704D 704D N   CA   5449 5450    1.63    1.43    0.20    6.66    1.43    0.20    6.66\n",
      "  375   5582 704D 704D C   CA   5455 5450    1.76    1.49    0.27    7.85    1.49    0.27    7.85\n",
      "  376   5583 704D 705E C   N    5455 5457    1.53    1.35    0.19    6.58    1.35    0.19    6.58\n",
      "  377   5589 705E 705E N   CA   5457 5458    1.65    1.43    0.22    7.31    1.43    0.22    7.31\n",
      "  378   5591 705E 705E C   CA   5464 5458    1.76    1.49    0.27    7.98    1.49    0.27    7.98\n",
      "  379   5592 705E 706K C   N    5464 5466    1.55    1.35    0.21    7.35    1.35    0.21    7.35\n",
      "  380   5598 706K 706K N   CA   5466 5467    1.67    1.43    0.24    7.99    1.43    0.24    7.99\n",
      "  381   5600 706K 706K C   CA   5473 5467    1.79    1.49    0.30    8.86    1.49    0.30    8.86\n",
      "  382   5601 706K 707V C   N    5473 5475    1.57    1.35    0.22    7.94    1.35    0.22    7.94\n",
      "  383   5605 707V 707V N   CA   5475 5476    1.68    1.43    0.25    8.36    1.43    0.25    8.36\n",
      "  384   5607 707V 707V C   CA   5480 5476    1.86    1.49    0.37   10.84    1.49    0.37   10.84\n",
      "  385   5608 707V 708A C   N    5480 5482    1.59    1.35    0.24    8.67    1.35    0.24    8.67\n",
      "  386   5610 708A 708A N   CA   5482 5483    1.72    1.43    0.29    9.39    1.43    0.29    9.39\n",
      "  387   5612 708A 708A C   CA   5485 5483    1.82    1.49    0.33    9.64    1.49    0.33    9.64\n",
      "  388   5613 708A 709E C   N    5485 5487    1.57    1.35    0.23    8.05    1.35    0.23    8.05\n",
      "  389   5619 709E 709E N   CA   5487 5488    1.69    1.43    0.26    8.54    1.43    0.26    8.54\n",
      "  390   5621 709E 709E C   CA   5494 5488    1.79    1.49    0.30    8.68    1.49    0.30    8.68\n",
      "  391   5622 709E 710S C   N    5494 5496    1.54    1.35    0.20    7.06    1.35    0.20    7.06\n",
      "  392   5625 710S 710S N   CA   5496 5497    1.66    1.43    0.23    7.56    1.43    0.23    7.56\n",
      "  393   5627 710S 710S C   CA   5500 5497    1.78    1.49    0.29    8.31    1.49    0.29    8.31\n",
      "  394   5628 710S 711L C   N    5500 5502    1.54    1.35    0.19    6.91    1.35    0.19    6.91\n",
      "  395   5633 711L 711L N   CA   5502 5503    1.67    1.43    0.24    7.75    1.43    0.24    7.75\n",
      "  396   5635 711L 711L C   CA   5508 5503    1.74    1.49    0.25    7.13    1.49    0.25    7.13\n",
      "  397   5636 711L 712N C   N    5508 5510    1.56    1.35    0.21    7.54    1.35    0.21    7.54\n",
      "  398   5641 712N 712N N   CA   5510 5511    1.68    1.43    0.25    8.29    1.43    0.25    8.29\n",
      "  399   5643 712N 712N C   CA   5516 5511    1.81    1.49    0.32    9.17    1.49    0.32    9.17\n",
      "  400   5644 712N 713P C   N    5516 5518    1.63    1.30    0.33    9.89    1.30    0.33    9.89\n",
      "  401   5646 713P 713P C   CA   5523 5519    1.90    1.49    0.41   11.96    1.49    0.41   11.96\n",
      "  402   5647 713P 714P C   N    5523 5525    1.71    1.30    0.41   12.04    1.30    0.41   12.04\n",
      "  403   5648 713P 713P N   CA   5518 5519    1.76    1.43    0.33   10.88    1.43    0.33   10.88\n",
      "  404   5654 714P 714P C   CA   5530 5526    1.99    1.49    0.50   14.55    1.49    0.50   14.55\n",
      "  405   5655 714P 715F C   N    5530 5532    1.68    1.35    0.34   12.02    1.35    0.34   12.02\n",
      "  406   5656 714P 714P N   CA   5525 5526    1.76    1.43    0.32   10.57    1.43    0.32   10.57\n",
      "  407   5669 715F 715F N   CA   5532 5533    1.86    1.43    0.43   14.29    1.43    0.43   14.29\n",
      "  408   5671 715F 715F C   CA   5541 5533    2.06    1.49    0.57   16.68    1.49    0.57   16.68\n",
      "  409   5672 715F 716S C   N    5541 5543    1.76    1.35    0.42   14.75    1.35    0.42   14.75\n",
      "  410   5675 716S 716S N   CA   5543 5544    1.94    1.43    0.51   16.68    1.43    0.51   16.68\n",
      "  411   5677 716S 716S C   CA   5547 5544    2.15    1.49    0.66   19.18    1.49    0.66   19.18\n",
      "  412   5678 716S 717M C   N    5547 5549    1.77    1.35    0.42   15.03    1.35    0.42   15.03\n",
      "  413   5683 717M 717M N   CA   5549 5550    1.94    1.43    0.51   16.93    1.43    0.51   16.93\n",
      "  414   5685 717M 717M C   CA   5555 5550    2.13    1.49    0.64   18.55    1.49    0.64   18.55\n",
      "  415   5686 717M 718A C   N    5555 5557    1.83    1.35    0.48   17.13    1.35    0.48   17.13\n",
      "  416   5688 718A 718A N   CA   5557 5558    2.02    1.43    0.59   19.43    1.43    0.59   19.43\n",
      "  417   5690 718A 718A C   CA   5560 5558    2.33    1.49    0.84   24.55    1.49    0.84   24.55\n",
      "  418   5691 718A 719F C   N    5560 5562    1.94    1.35    0.59   21.06    1.35    0.59   21.06\n",
      "  419   5700 719F 719F N   CA   5562 5563    2.15    1.43    0.72   23.60    1.43    0.72   23.60\n",
      "  420   5702 719F 719F C   CA   5571 5563    2.45    1.49    0.96   27.88    1.49    0.96   27.88\n",
      "  421   5703 719F 720L C   N    5571 5573    1.98    1.35    0.64   22.63    1.35    0.64   22.63\n",
      "  422   5708 720L 720L N   CA   5573 5574    2.19    1.43    0.76   24.93    1.43    0.76   24.93\n",
      "  423   5710 720L 720L C   CA   5579 5574    2.45    1.49    0.96   27.83    1.49    0.96   27.83\n",
      "  424   5711 720L 721Y C   N    5579 5581    2.04    1.35    0.70   24.73    1.35    0.70   24.73\n",
      "  425   5721 721Y 721Y N   CA   5581 5582    2.25    1.43    0.82   27.12    1.43    0.82   27.12\n",
      "  426   5723 721Y 721Y C   CA   5591 5582    2.57    1.49    1.08   31.48    1.49    1.08   31.48\n",
      "  427   5724 721Y 722F C   N    5591 5593    2.12    1.35    0.78   27.52    1.35    0.78   27.52\n",
      "  428   5733 722F 722F N   CA   5593 5594    2.35    1.43    0.92   30.40    1.43    0.92   30.40\n",
      "  429   5735 722F 722F C   CA   5602 5594    2.82    1.49    1.33   38.70    1.49    1.33   38.70\n",
      "  430   5736 722F 723Y C   N    5602 5604    2.28    1.35    0.93   32.98    1.35    0.93   32.98\n",
      "  431   5746 723Y 723Y N   CA   5604 5605    2.53    1.43    1.10   36.29    1.43    1.10   36.29\n",
      "  432   5748 723Y 723Y C   CA   5614 5605    2.99    1.49    1.50   43.70    1.49    1.50   43.70\n",
      "  433   5749 723Y 724K C   N    5614 5616    2.39    1.35    1.04   36.96    1.35    1.04   36.96\n",
      "  434   5755 724K 724K N   CA   5616 5617    2.66    1.43    1.23   40.60    1.43    1.23   40.60\n",
      "  435   5757 724K 724K C   CA   5623 5617    3.10    1.49    1.61   46.89    1.49    1.61   46.89\n",
      "  436   5758 724K 725R C   N    5623 5625    2.46    1.35    1.12   39.64    1.35    1.12   39.64\n",
      "  437   5766 725R 725R N   CA   5625 5626    2.76    1.43    1.33   43.80    1.43    1.33   43.80\n",
      "  438   5768 725R 725R C   CA   5634 5626    3.19    1.49    1.70   49.38    1.49    1.70   49.38\n",
      "  439   5769 725R 726V C   N    5634 5636    2.52    1.35    1.18   41.80    1.35    1.18   41.80\n",
      "  440   5773 726V 726V N   CA   5636 5637    2.82    1.43    1.39   45.78    1.43    1.39   45.78\n",
      "  441   5775 726V 726V C   CA   5641 5637    3.37    1.49    1.88   54.53    1.49    1.88   54.53\n",
      "  442   5776 726V 727G C   N    5641 5643    2.63    1.35    1.28   45.44    1.35    1.28   45.44\n",
      "  443   5777 727G 727G N   CA   5643 5644    2.94    1.43    1.51   49.61    1.43    1.51   49.61\n",
      "  444   5779 727G 727G C   CA   5645 5644    3.43    1.49    1.94   56.28    1.49    1.94   56.28\n",
      "  445   5780 727G 728K C   N    5645 5647    2.67    1.35    1.32   46.82    1.35    1.32   46.82\n",
      "  446   5786 728K 728K N   CA   5647 5648    2.51    1.43    1.08   35.66    1.43    1.08   35.66\n",
      "  447   5787 728K 728K O   C    5655 5654    1.45    1.26    0.19    8.21    1.26    0.19    8.21\n",
      "  448   5788 728K 728K C   CA   5654 5648    2.08    1.52    0.55   14.42    1.52    0.55   14.42\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5824   4P   4P N   CA     25   26  132.06  108.20   23.86    5.42  108.20   23.86    5.42\n",
      "    2   5907  11A  11A N   CA     83   84  123.44  107.00   16.44    4.73  107.00   16.44    4.73\n",
      "    3   5914  12C  12C N   CA     88   89  123.56  107.00   16.56    4.76  107.00   16.56    4.76\n",
      "    4   5922  13A  13A N   CA     94   95  124.23  107.00   17.23    4.95  107.00   17.23    4.95\n",
      "    5   5929  14H  14H N   CA     99  100  124.92  107.00   17.92    5.15  107.00   17.92    5.15\n",
      "    6   5944  15C  15C N   CA    109  110  127.61  107.00   20.61    5.93  107.00   20.61    5.93\n",
      "    7   5952  16W  16W N   CA    115  116  127.56  107.00   20.56    5.91  107.00   20.56    5.91\n",
      "    8   5955  16W  16W CA  C     116  127  132.21  116.50   15.71    4.52  116.50   15.71    4.52\n",
      "    9   5975  17D  17D N   CA    129  130  130.96  107.00   23.96    6.89  107.00   23.96    6.89\n",
      "   10   5978  17D  17D CA  C     130  135  132.33  116.50   15.83    4.55  116.50   15.83    4.55\n",
      "   11   5986  18E  18E N   CA    137  138  131.49  107.00   24.49    7.04  107.00   24.49    7.04\n",
      "   12   5989  18E  18E CA  C     138  144  133.17  116.50   16.67    4.79  116.50   16.67    4.79\n",
      "   13   5998  19S  19S N   CA    146  147  132.66  107.00   25.66    7.38  107.00   25.66    7.38\n",
      "   14   6001  19S  19S CA  C     147  150  133.81  116.50   17.31    4.98  116.50   17.31    4.98\n",
      "   15   6006  20V  20V N   CA    152  153  133.59  107.00   26.59    7.65  107.00   26.59    7.65\n",
      "   16   6010  20V  20V CA  C     153  157  134.13  116.50   17.64    5.07  116.50   17.64    5.07\n",
      "   17   6013  20V  21M C   N     157  159  139.93  120.00   19.93    4.53  120.00   19.93    4.53\n",
      "   18   6016  21M  21M N   CA    159  160  133.92  107.00   26.92    7.74  107.00   26.92    7.74\n",
      "   19   6019  21M  21M CA  C     160  165  135.02  116.50   18.52    5.32  116.50   18.52    5.32\n",
      "   20   6026  22L  22L N   CA    167  168  134.75  107.00   27.75    7.98  107.00   27.75    7.98\n",
      "   21   6029  22L  22L CA  C     168  173  136.00  116.50   19.50    5.61  116.50   19.50    5.61\n",
      "   22   6034  22L  23V C   N     173  175  140.53  120.00   20.53    4.66  120.00   20.53    4.66\n",
      "   23   6037  23V  23V N   CA    175  176  138.54  107.00   31.54    9.07  107.00   31.54    9.07\n",
      "   24   6044  23V  24P C   N     180  182  147.42  117.00   30.42    7.57  117.00   30.42    7.57\n",
      "   25   6047  24P  24P N   CA    182  183  146.21  108.20   38.02    8.64  108.20   38.02    8.64\n",
      "   26   6051  24P  24P CA  C     183  187  137.54  116.50   21.04    6.05  116.50   21.04    6.05\n",
      "   27   6056  24P  25N C   N     187  189  142.09  120.00   22.09    5.02  120.00   22.09    5.02\n",
      "   28   6059  25N  25N N   CA    189  190  138.05  107.00   31.05    8.93  107.00   31.05    8.93\n",
      "   29   6062  25N  25N CA  C     190  195  137.85  116.50   21.35    6.14  116.50   21.35    6.14\n",
      "   30   6067  25N  26H C   N     195  197  142.48  120.00   22.48    5.11  120.00   22.48    5.11\n",
      "   31   6070  26H  26H N   CA    197  198  138.39  107.00   31.39    9.03  107.00   31.39    9.03\n",
      "   32   6072  26H  26H CA  C     198  205  139.23  116.50   22.73    6.54  116.50   22.73    6.54\n",
      "   33   6082  26H  27R C   N     205  207  144.23  120.00   24.23    5.51  120.00   24.23    5.51\n",
      "   34   6085  27R  27R N   CA    207  208  139.50  107.00   32.50    9.35  107.00   32.50    9.35\n",
      "   35   6088  27R  27R CA  C     208  216  139.74  116.50   23.24    6.68  116.50   23.24    6.68\n",
      "   36   6096  27R  28S C   N     216  218  144.05  120.00   24.05    5.47  120.00   24.05    5.47\n",
      "   37   6099  28S  28S N   CA    218  219  140.39  107.00   33.39    9.60  107.00   33.39    9.60\n",
      "   38   6102  28S  28S CA  C     219  222  140.03  116.50   23.53    6.77  116.50   23.53    6.77\n",
      "   39   6104  28S  29A C   N     222  224  144.77  120.00   24.77    5.63  120.00   24.77    5.63\n",
      "   40   6107  29A  29A N   CA    224  225  140.94  107.00   33.94    9.76  107.00   33.94    9.76\n",
      "   41   6109  29A  29A CA  C     225  227  140.38  116.50   23.88    6.87  116.50   23.88    6.87\n",
      "   42   6111  29A  30V C   N     227  229  144.86  120.00   24.86    5.65  120.00   24.86    5.65\n",
      "   43   6114  30V  30V N   CA    229  230  141.65  107.00   34.65    9.96  107.00   34.65    9.96\n",
      "   44   6118  30V  30V CA  C     230  234  140.90  116.50   24.40    7.01  116.50   24.40    7.01\n",
      "   45   6121  30V  31H C   N     234  236  145.58  120.00   25.58    5.81  120.00   25.58    5.81\n",
      "   46   6124  31H  31H N   CA    236  237  141.46  107.00   34.46    9.91  107.00   34.46    9.91\n",
      "   47   6126  31H  31H CA  C     237  244  141.99  116.50   25.49    7.33  116.50   25.49    7.33\n",
      "   48   6136  31H  32K C   N     244  246  145.78  120.00   25.78    5.86  120.00   25.78    5.86\n",
      "   49   6139  32K  32K N   CA    246  247  142.90  107.00   35.90   10.32  107.00   35.90   10.32\n",
      "   50   6142  32K  32K CA  C     247  253  142.54  116.50   26.04    7.49  116.50   26.04    7.49\n",
      "   51   6147  32K  33E C   N     253  255  147.43  120.00   27.43    6.23  120.00   27.43    6.23\n",
      "   52   6150  33E  33E N   CA    255  256  143.12  107.00   36.12   10.39  107.00   36.12   10.39\n",
      "   53   6153  33E  33E CA  C     256  262  143.04  116.50   26.54    7.63  116.50   26.54    7.63\n",
      "   54   6159  33E  34E C   N     262  264  147.11  120.00   27.11    6.16  120.00   27.11    6.16\n",
      "   55   6162  34E  34E N   CA    264  265  144.02  107.00   37.02   10.64  107.00   37.02   10.64\n",
      "   56   6165  34E  34E CA  C     265  271  143.37  116.50   26.87    7.73  116.50   26.87    7.73\n",
      "   57   6171  34E  35C C   N     271  273  147.74  120.00   27.74    6.30  120.00   27.74    6.30\n",
      "   58   6174  35C  35C N   CA    273  274  144.49  107.00   37.49   10.78  107.00   37.49   10.78\n",
      "   59   6177  35C  35C CA  C     274  277  144.00  116.50   27.50    7.91  116.50   27.50    7.91\n",
      "   60   6179  35C  36A C   N     277  279  147.99  120.00   27.99    6.36  120.00   27.99    6.36\n",
      "   61   6182  36A  36A N   CA    279  280  145.22  107.00   38.22   10.99  107.00   38.22   10.99\n",
      "   62   6184  36A  36A CA  C     280  282  144.32  116.50   27.82    8.00  116.50   27.82    8.00\n",
      "   63   6186  36A  37Y C   N     282  284  148.59  120.00   28.59    6.50  120.00   28.59    6.50\n",
      "   64   6189  37Y  37Y N   CA    284  285  145.28  107.00   38.28   11.01  107.00   38.28   11.01\n",
      "   65   6192  37Y  37Y CA  C     285  294  145.27  116.50   28.77    8.27  116.50   28.77    8.27\n",
      "   66   6204  37Y  38C C   N     294  296  149.43  120.00   29.42    6.69  120.00   29.42    6.69\n",
      "   67   6207  38C  38C N   CA    296  297  146.54  107.00   39.54   11.37  107.00   39.54   11.37\n",
      "   68   6210  38C  38C CA  C     297  300  145.43  116.50   28.93    8.32  116.50   28.93    8.32\n",
      "   69   6212  38C  39C C   N     300  302  149.74  120.00   29.74    6.76  120.00   29.74    6.76\n",
      "   70   6215  39C  39C N   CA    302  303  146.88  107.00   39.88   11.47  107.00   39.88   11.47\n",
      "   71   6218  39C  39C CA  C     303  306  145.67  116.50   29.17    8.39  116.50   29.17    8.39\n",
      "   72   6220  39C  40R C   N     306  308  149.55  120.00   29.55    6.72  120.00   29.55    6.72\n",
      "   73   6223  40R  40R N   CA    308  309  147.01  107.00   40.01   11.51  107.00   40.01   11.51\n",
      "   74   6226  40R  40R CA  C     309  317  146.79  116.50   30.30    8.71  116.50   30.30    8.71\n",
      "   75   6234  40R  41T C   N     317  319  150.42  120.00   30.42    6.91  120.00   30.42    6.91\n",
      "   76   6237  41T  41T N   CA    319  320  148.04  107.00   41.04   11.80  107.00   41.04   11.80\n",
      "   77   6241  41T  41T CA  C     320  324  146.84  116.50   30.34    8.72  116.50   30.34    8.72\n",
      "   78   6244  41T  42C C   N     324  326  150.47  120.00   30.47    6.92  120.00   30.47    6.92\n",
      "   79   6247  42C  42C N   CA    326  327  148.61  107.00   41.61   11.96  107.00   41.61   11.96\n",
      "   80   6250  42C  42C CA  C     327  330  147.16  116.50   30.66    8.82  116.50   30.66    8.82\n",
      "   81   6252  42C  43R C   N     330  332  150.72  120.00   30.72    6.98  120.00   30.72    6.98\n",
      "   82   6255  43R  43R N   CA    332  333  148.16  107.00   41.16   11.83  107.00   41.16   11.83\n",
      "   83   6258  43R  43R CA  C     333  341  148.08  116.50   31.58    9.08  116.50   31.58    9.08\n",
      "   84   6266  43R  44H C   N     341  343  151.09  120.00   31.09    7.06  120.00   31.09    7.06\n",
      "   85   6269  44H  44H N   CA    343  344  149.55  107.00   42.55   12.24  107.00   42.55   12.24\n",
      "   86   6271  44H  44H CA  C     344  351  148.17  116.50   31.67    9.11  116.50   31.67    9.11\n",
      "   87   6281  44H  45E C   N     351  353  152.29  120.00   32.29    7.34  120.00   32.29    7.34\n",
      "   88   6284  45E  45E N   CA    353  354  149.49  107.00   42.49   12.22  107.00   42.49   12.22\n",
      "   89   6287  45E  45E CA  C     354  360  148.71  116.50   32.21    9.26  116.50   32.21    9.26\n",
      "   90   6293  45E  46M C   N     360  362  151.82  120.00   31.82    7.23  120.00   31.82    7.23\n",
      "   91   6296  46M  46M N   CA    362  363  150.14  107.00   43.14   12.41  107.00   43.14   12.41\n",
      "   92   6299  46M  46M CA  C     363  368  149.22  116.50   32.73    9.41  116.50   32.73    9.41\n",
      "   93   6303  46M  47G C   N     368  370  152.17  120.00   32.17    7.31  120.00   32.17    7.31\n",
      "   94   6305  47G  47G N   CA    370  371  150.40  107.00   43.40   12.48  107.00   43.40   12.48\n",
      "   95   6307  47G  47G CA  C     371  372  135.48  116.50   18.98    5.46  116.50   18.98    5.46\n",
      "   96   6308  47G  48T C   N     372  374  146.66  120.00   26.66    6.06  120.00   26.66    6.06\n",
      "   97   6318  48T  49L C   N     379  381  144.41  120.00   24.41    5.55  120.00   24.41    5.55\n",
      "   98   6902 104K 104K N   CA    799  800  126.17  107.00   19.17    5.51  107.00   19.17    5.51\n",
      "   99   7460 158P 158P N   CA   1200 1201  128.46  108.20   20.26    4.60  108.20   20.26    4.60\n",
      "  100   7582 169P 169P N   CA   1285 1286  129.29  108.20   21.09    4.79  108.20   21.09    4.79\n",
      "  101   7947 204R 205E C   N    1548 1550  143.39  120.00   23.39    5.32  120.00   23.39    5.32\n",
      "  102   8269 236A 236A N   CA   1777 1778  126.55  107.00   19.55    5.62  107.00   19.55    5.62\n",
      "  103   8284 237D 238F C   N    1788 1790  140.54  120.00   20.54    4.67  120.00   20.54    4.67\n",
      "  104   8287 238F 238F N   CA   1790 1791  123.69  107.00   16.69    4.80  107.00   16.69    4.80\n",
      "  105   8304 239Y 239Y CA  CB   1802 1803  127.09  107.50   19.59    4.53  107.50   19.59    4.53\n",
      "  106   8400 247V 247V N   CA   1870 1871  123.07  107.00   16.07    4.62  107.00   16.07    4.62\n",
      "  107   8555 260G 260G N   CA   1980 1981  130.80  107.00   23.80    6.84  107.00   23.80    6.84\n",
      "  108   8558 260G 261I C   N    1982 1984  145.44  120.00   25.44    5.78  120.00   25.44    5.78\n",
      "  109   8561 261I 261I N   CA   1984 1985  126.70  107.00   19.70    5.66  107.00   19.70    5.66\n",
      "  110   8569 261I 262D C   N    1990 1992  167.95  120.00   47.95   10.90  120.00   47.95   10.90\n",
      "  111   8572 262D 262D N   CA   1992 1993  131.16  107.00   24.16    6.95  107.00   24.16    6.95\n",
      "  112   8580 262D 263I C   N    1998 2000  149.54  120.00   29.54    6.71  120.00   29.54    6.71\n",
      "  113   8583 263I 263I N   CA   2000 2001  137.79  107.00   30.79    8.85  107.00   30.79    8.85\n",
      "  114   8587 263I 263I CA  C    2001 2006  132.72  116.50   16.22    4.67  116.50   16.22    4.67\n",
      "  115   8594 264Q 264Q N   CA   2008 2009  135.55  107.00   28.55    8.21  107.00   28.55    8.21\n",
      "  116   8597 264Q 264Q CA  C    2009 2015  134.10  116.50   17.60    5.06  116.50   17.60    5.06\n",
      "  117   8606 265T 265T N   CA   2017 2018  132.29  107.00   25.29    7.27  107.00   25.29    7.27\n",
      "  118   8616 266A 266A N   CA   2024 2025  131.53  107.00   24.53    7.05  107.00   24.53    7.05\n",
      "  119   8623 267K 267K N   CA   2029 2030  132.22  107.00   25.22    7.25  107.00   25.22    7.25\n",
      "  120   8634 268K 268K N   CA   2038 2039  130.76  107.00   23.76    6.83  107.00   23.76    6.83\n",
      "  121   8645 269T 269T N   CA   2047 2048  127.99  107.00   20.99    6.03  107.00   20.99    6.03\n",
      "  122   8655 270A 270A N   CA   2054 2055  129.31  107.00   22.31    6.42  107.00   22.31    6.42\n",
      "  123   8662 271K 271K N   CA   2059 2060  129.83  107.00   22.83    6.57  107.00   22.83    6.57\n",
      "  124   8673 272T 272T N   CA   2068 2069  129.68  107.00   22.68    6.52  107.00   22.68    6.52\n",
      "  125   8683 273L 273L N   CA   2075 2076  129.45  107.00   22.45    6.46  107.00   22.45    6.46\n",
      "  126   8693 274G 274G N   CA   2083 2084  130.66  107.00   23.66    6.80  107.00   23.66    6.80\n",
      "  127   8699 275E 275E N   CA   2087 2088  127.83  107.00   20.83    5.99  107.00   20.83    5.99\n",
      "  128   8711 276M 276M N   CA   2096 2097  127.64  107.00   20.64    5.93  107.00   20.64    5.93\n",
      "  129   8721 277Q 277Q N   CA   2104 2105  126.48  107.00   19.48    5.60  107.00   19.48    5.60\n",
      "  130   8733 278Y 278Y N   CA   2113 2114  123.55  107.00   16.55    4.76  107.00   16.55    4.76\n",
      "  131   8751 279D 279D N   CA   2125 2126  124.05  107.00   17.05    4.90  107.00   17.05    4.90\n",
      "  132  11786 556F 556F N   CA   4320 4321  123.78  107.00   16.78    4.82  107.00   16.78    4.82\n",
      "  133  11812 558D 558D N   CA   4338 4339  126.78  107.00   19.78    5.69  107.00   19.78    5.69\n",
      "  134  11823 559V 559V N   CA   4346 4347  128.26  107.00   21.26    6.11  107.00   21.26    6.11\n",
      "  135  11830 559V 560P C   N    4351 4353  139.74  117.00   22.74    5.66  117.00   22.74    5.66\n",
      "  136  11833 560P 560P N   CA   4353 4354  138.34  108.20   30.14    6.85  108.20   30.14    6.85\n",
      "  137  11845 561L 561L N   CA   4360 4361  129.11  107.00   22.11    6.36  107.00   22.11    6.36\n",
      "  138  11856 562E 562E N   CA   4368 4369  130.68  107.00   23.68    6.81  107.00   23.68    6.81\n",
      "  139  11859 562E 562E CA  C    4369 4375  134.02  116.50   17.52    5.04  116.50   17.52    5.04\n",
      "  140  11865 562E 563L C   N    4375 4377  141.43  120.00   21.43    4.87  120.00   21.43    4.87\n",
      "  141  11868 563L 563L N   CA   4377 4378  135.53  107.00   28.53    8.20  107.00   28.53    8.20\n",
      "  142  11871 563L 563L CA  C    4378 4383  136.03  116.50   19.53    5.62  116.50   19.53    5.62\n",
      "  143  11876 563L 564D C   N    4383 4385  140.64  120.00   20.64    4.69  120.00   20.64    4.69\n",
      "  144  11879 564D 564D N   CA   4385 4386  137.09  107.00   30.09    8.65  107.00   30.09    8.65\n",
      "  145  11882 564D 564D CA  C    4386 4391  136.73  116.50   20.23    5.82  116.50   20.23    5.82\n",
      "  146  11887 564D 565L C   N    4391 4393  142.88  120.00   22.88    5.20  120.00   22.88    5.20\n",
      "  147  11890 565L 565L N   CA   4393 4394  137.19  107.00   30.19    8.68  107.00   30.19    8.68\n",
      "  148  11893 565L 565L CA  C    4394 4399  138.68  116.50   22.18    6.38  116.50   22.18    6.38\n",
      "  149  11898 565L 566E C   N    4399 4401  142.93  120.00   22.93    5.21  120.00   22.93    5.21\n",
      "  150  11901 566E 566E N   CA   4401 4402  138.26  107.00   31.26    8.99  107.00   31.26    8.99\n",
      "  151  11904 566E 566E CA  C    4402 4408  140.10  116.50   23.60    6.79  116.50   23.60    6.79\n",
      "  152  11910 566E 567F C   N    4408 4410  145.40  120.00   25.40    5.77  120.00   25.40    5.77\n",
      "  153  11913 567F 567F N   CA   4410 4411  139.55  107.00   32.55    9.36  107.00   32.55    9.36\n",
      "  154  11916 567F 567F CA  C    4411 4419  143.04  116.50   26.54    7.63  116.50   26.54    7.63\n",
      "  155  11926 567F 568L C   N    4419 4421  146.71  120.00   26.71    6.07  120.00   26.71    6.07\n",
      "  156  11929 568L 568L N   CA   4421 4422  143.34  107.00   36.34   10.45  107.00   36.34   10.45\n",
      "  157  11932 568L 568L CA  C    4422 4427  143.81  116.50   27.31    7.85  116.50   27.31    7.85\n",
      "  158  11937 568L 569R C   N    4427 4429  147.88  120.00   27.88    6.34  120.00   27.88    6.34\n",
      "  159  11940 569R 569R N   CA   4429 4430  144.72  107.00   37.72   10.85  107.00   37.72   10.85\n",
      "  160  11943 569R 569R CA  C    4430 4438  145.41  116.50   28.91    8.31  116.50   28.91    8.31\n",
      "  161  11951 569R 570G C   N    4438 4440  147.47  120.00   27.47    6.24  120.00   27.47    6.24\n",
      "  162  11953 570G 570G N   CA   4440 4441  146.98  107.00   39.98   11.50  107.00   39.98   11.50\n",
      "  163  11955 570G 570G CA  C    4441 4442  143.63  116.50   27.13    7.80  116.50   27.13    7.80\n",
      "  164  11956 570G 571K C   N    4442 4444  147.58  120.00   27.58    6.27  120.00   27.58    6.27\n",
      "  165  11959 571K 571K N   CA   4444 4445  144.77  107.00   37.77   10.86  107.00   37.77   10.86\n",
      "  166  11962 571K 571K CA  C    4445 4451  142.78  116.50   26.28    7.56  116.50   26.28    7.56\n",
      "  167  11967 571K 572G C   N    4451 4453  149.29  120.00   29.29    6.66  120.00   29.29    6.66\n",
      "  168  11969 572G 572G N   CA   4453 4454  145.50  107.00   38.50   11.07  107.00   38.50   11.07\n",
      "  169  11971 572G 572G CA  C    4454 4455  145.50  116.50   29.00    8.34  116.50   29.00    8.34\n",
      "  170  11972 572G 573L C   N    4455 4457  148.61  120.00   28.61    6.50  120.00   28.61    6.50\n",
      "  171  11975 573L 573L N   CA   4457 4458  148.62  107.00   41.62   11.97  107.00   41.62   11.97\n",
      "  172  11978 573L 573L CA  C    4458 4463  143.93  116.50   27.43    7.89  116.50   27.43    7.89\n",
      "  173  11983 573L 574Q C   N    4463 4465  154.41  120.00   34.41    7.82  120.00   34.41    7.82\n",
      "  174  11986 574Q 574Q N   CA   4465 4466  154.67  107.00   47.67   13.71  107.00   47.67   13.71\n",
      "  175  11989 574Q 574Q CA  C    4466 4472  147.35  116.50   30.86    8.87  116.50   30.86    8.87\n",
      "  176  11995 574Q 575S C   N    4472 4474  149.86  120.00   29.86    6.79  120.00   29.86    6.79\n",
      "  177  11998 575S 575S N   CA   4474 4475  148.63  107.00   41.63   11.97  107.00   41.63   11.97\n",
      "  178  12001 575S 575S CA  C    4475 4478  146.66  116.50   30.16    8.67  116.50   30.16    8.67\n",
      "  179  12003 575S 576D C   N    4478 4480  151.23  120.00   31.23    7.10  120.00   31.23    7.10\n",
      "  180  12006 576D 576D N   CA   4480 4481  148.09  107.00   41.09   11.82  107.00   41.09   11.82\n",
      "  181  12009 576D 576D CA  C    4481 4486  148.96  116.50   32.46    9.33  116.50   32.46    9.33\n",
      "  182  12014 576D 577E C   N    4486 4488  150.16  120.00   30.16    6.85  120.00   30.16    6.85\n",
      "  183  12017 577E 577E N   CA   4488 4489  153.16  107.00   46.16   13.27  107.00   46.16   13.27\n",
      "  184  12020 577E 577E CA  C    4489 4495  150.54  116.50   34.04    9.79  116.50   34.04    9.79\n",
      "  185  12026 577E 578V C   N    4495 4497  154.43  120.00   34.43    7.82  120.00   34.43    7.82\n",
      "  186  12029 578V 578V N   CA   4497 4498  149.90  107.00   42.90   12.34  107.00   42.90   12.34\n",
      "  187  12033 578V 578V CA  C    4498 4502  153.43  116.50   36.93   10.62  116.50   36.93   10.62\n",
      "  188  12036 578V 579S C   N    4502 4504  154.31  120.00   34.31    7.80  120.00   34.31    7.80\n",
      "  189  12039 579S 579S N   CA   4504 4505  153.04  107.00   46.04   13.24  107.00   46.04   13.24\n",
      "  190  12042 579S 579S CA  C    4505 4508  155.52  116.50   39.02   11.22  116.50   39.02   11.22\n",
      "  191  12044 579S 580M C   N    4508 4510  166.61  120.00   46.61   10.59  120.00   46.61   10.59\n",
      "  192  12047 580M 580M N   CA   4510 4511  164.71  107.00   57.71   16.59  107.00   57.71   16.59\n",
      "  193  12050 580M 580M CA  C    4511 4516  160.18  112.50   47.68    6.85  112.50   47.68    6.85\n",
      "  194  12054 580M 581P C   N    4516 4518  161.07  117.00   44.07   10.97  117.00   44.07   10.97\n",
      "  195  12057 581P 581P N   CA   4518 4519  168.88  108.20   60.68   13.79  108.20   60.68   13.79\n",
      "  196  12061 581P 581P CA  C    4519 4523  160.50  116.50   44.00   12.65  116.50   44.00   12.65\n",
      "  197  12066 581P 582E C   N    4523 4525  166.23  120.00   46.23   10.51  120.00   46.23   10.51\n",
      "  198  12069 582E 582E N   CA   4525 4526  164.23  107.00   57.23   16.45  107.00   57.23   16.45\n",
      "  199  12072 582E 582E CA  C    4526 4532  160.58  116.50   44.09   12.68  116.50   44.09   12.68\n",
      "  200  12078 582E 583A C   N    4532 4534  168.95  120.00   48.95   11.12  120.00   48.95   11.12\n",
      "  201  12081 583A 583A N   CA   4534 4535  168.40  107.00   61.40   17.65  107.00   61.40   17.65\n",
      "  202  12083 583A 583A CA  C    4535 4537  164.87  116.50   48.37   13.91  116.50   48.37   13.91\n",
      "  203  12085 583A 584Q C   N    4537 4539  165.82  120.00   45.82   10.41  120.00   45.82   10.41\n",
      "  204  12088 584Q 584Q N   CA   4539 4540  165.95  107.00   58.95   16.95  107.00   58.95   16.95\n",
      "  205  12091 584Q 584Q CA  C    4540 4546  166.20  116.50   49.70   14.29  116.50   49.70   14.29\n",
      "  206  12097 584Q 585E C   N    4546 4548  168.33  120.00   48.33   10.98  120.00   48.33   10.98\n",
      "  207  12100 585E 585E N   CA   4548 4549  169.78  107.00   62.78   18.05  107.00   62.78   18.05\n",
      "  208  12103 585E 585E CA  C    4549 4555  170.59  116.50   54.09   15.55  116.50   54.09   15.55\n",
      "  209  12109 585E 586D C   N    4555 4557  171.14  120.00   51.14   11.62  120.00   51.14   11.62\n",
      "  210  12112 586D 586D N   CA   4557 4558  172.25  107.00   65.25   18.76  107.00   65.25   18.76\n",
      "  211  12115 586D 586D CA  C    4558 4563  170.13  116.50   53.63   15.42  116.50   53.63   15.42\n",
      "  212  12120 586D 587L C   N    4563 4565  172.43  120.00   52.43   11.92  120.00   52.43   11.92\n",
      "  213  12123 587L 587L N   CA   4565 4566  174.27  107.00   67.27   19.34  107.00   67.27   19.34\n",
      "  214  12126 587L 587L CA  C    4566 4571  174.05  112.50   61.55    8.84  112.50   61.55    8.84\n",
      "  215  12131 587L 588P C   N    4571 4573  176.33  117.00   59.33   14.77  117.00   59.33   14.77\n",
      "  216  12134 588P 588P N   CA   4573 4574  175.11  108.20   66.91   15.21  108.20   66.91   15.21\n",
      "  217  12138 588P 588P CA  C    4574 4578  171.77  116.50   55.27   15.89  116.50   55.27   15.89\n",
      "  218  12141 588P 588P CB  CG   4576 4577  130.22  108.50   21.72    5.84  108.50   21.72    5.84\n",
      "  219  12143 588P 589H C   N    4578 4580  172.54  120.00   52.54   11.94  120.00   52.54   11.94\n",
      "  220  12146 589H 589H N   CA   4580 4581  171.92  107.00   64.92   18.67  107.00   64.92   18.67\n",
      "  221  12148 589H 589H CA  C    4581 4588  172.82  116.50   56.32   16.19  116.50   56.32   16.19\n",
      "  222  12158 589H 590K C   N    4588 4590  174.91  120.00   54.91   12.48  120.00   54.91   12.48\n",
      "  223  12161 590K 590K N   CA   4590 4591  175.50  107.00   68.50   19.70  107.00   68.50   19.70\n",
      "  224  12164 590K 590K CA  C    4591 4597  175.84  112.50   63.34    9.10  112.50   63.34    9.10\n",
      "  225  12169 590K 591P C   N    4597 4599  178.06  117.00   61.06   15.20  117.00   61.06   15.20\n",
      "  226  12172 591P 591P N   CA   4599 4600  176.86  108.20   68.66   15.60  108.20   68.66   15.60\n",
      "  227  12176 591P 591P CA  C    4600 4604  175.33  116.50   58.83   16.92  116.50   58.83   16.92\n",
      "  228  12177 591P 591P CA  CB   4600 4602  127.89  108.50   19.38    5.21  108.50   19.38    5.21\n",
      "  229  12179 591P 591P CB  CG   4602 4603  138.07  108.50   29.57    7.95  108.50   29.57    7.95\n",
      "  230  12181 591P 592T C   N    4604 4606  178.93  120.00   58.93   13.39  120.00   58.93   13.39\n",
      "  231  12184 592T 592T N   CA   4606 4607  176.52  107.00   69.52   19.99  107.00   69.52   19.99\n",
      "  232  12188 592T 592T CA  C    4607 4611  177.83  116.50   61.33   17.64  116.50   61.33   17.64\n",
      "  233  12191 592T 593S C   N    4611 4613  178.59  120.00   58.59   13.31  120.00   58.59   13.31\n",
      "  234  12194 593S 593S N   CA   4613 4614  175.59  107.00   68.59   19.72  107.00   68.59   19.72\n",
      "  235  12197 593S 593S CA  C    4614 4617  175.12  116.50   58.62   16.86  116.50   58.62   16.86\n",
      "  236  12199 593S 594S C   N    4617 4619  175.91  120.00   55.91   12.71  120.00   55.91   12.71\n",
      "  237  12202 594S 594S N   CA   4619 4620  176.38  107.00   69.38   19.95  107.00   69.38   19.95\n",
      "  238  12205 594S 594S CA  C    4620 4623  177.29  116.50   60.79   17.48  116.50   60.79   17.48\n",
      "  239  12207 594S 595S C   N    4623 4625  175.95  120.00   55.95   12.72  120.00   55.95   12.72\n",
      "  240  12210 595S 595S N   CA   4625 4626  178.22  107.00   71.22   20.48  107.00   71.22   20.48\n",
      "  241  12213 595S 595S CA  C    4626 4629  178.14  116.50   61.64   17.72  116.50   61.64   17.72\n",
      "  242  12215 595S 596M C   N    4629 4631  178.76  120.00   58.76   13.35  120.00   58.76   13.35\n",
      "  243  12218 596M 596M N   CA   4631 4632  177.72  107.00   70.72   20.34  107.00   70.72   20.34\n",
      "  244  12221 596M 596M CA  C    4632 4637  177.20  116.50   60.70   17.45  116.50   60.70   17.45\n",
      "  245  12225 596M 597K C   N    4637 4639  177.51  120.00   57.51   13.07  120.00   57.51   13.07\n",
      "  246  12228 597K 597K N   CA   4639 4640  176.87  107.00   69.87   20.09  107.00   69.87   20.09\n",
      "  247  12231 597K 597K CA  C    4640 4646  176.47  116.50   59.97   17.24  116.50   59.97   17.24\n",
      "  248  12236 597K 598T C   N    4646 4648  177.04  120.00   57.04   12.96  120.00   57.04   12.96\n",
      "  249  12239 598T 598T N   CA   4648 4649  176.75  107.00   69.75   20.05  107.00   69.75   20.05\n",
      "  250  12243 598T 598T CA  C    4649 4653  177.19  116.50   60.69   17.45  116.50   60.69   17.45\n",
      "  251  12246 598T 599V C   N    4653 4655  177.87  120.00   57.87   13.15  120.00   57.87   13.15\n",
      "  252  12249 599V 599V N   CA   4655 4656  177.85  107.00   70.85   20.37  107.00   70.85   20.37\n",
      "  253  12250 599V 599V CA  CB   4656 4657  146.53  108.50   38.03    8.92  108.50   38.03    8.92\n",
      "  254  12253 599V 599V CA  C    4656 4660  177.79  116.50   61.29   17.62  116.50   61.29   17.62\n",
      "  255  12256 599V 600D C   N    4660 4662  176.76  120.00   56.76   12.90  120.00   56.76   12.90\n",
      "  256  12259 600D 600D N   CA   4662 4663  177.84  107.00   70.84   20.37  107.00   70.84   20.37\n",
      "  257  12262 600D 600D CA  C    4663 4668  177.73  116.50   61.23   17.61  116.50   61.23   17.61\n",
      "  258  12267 600D 601E C   N    4668 4670  178.36  120.00   58.36   13.26  120.00   58.36   13.26\n",
      "  259  12270 601E 601E N   CA   4670 4671  178.81  107.00   71.81   20.65  107.00   71.81   20.65\n",
      "  260  12273 601E 601E CA  C    4671 4677  178.09  116.50   61.60   17.71  116.50   61.60   17.71\n",
      "  261  12279 601E 602E C   N    4677 4679  177.46  120.00   57.46   13.06  120.00   57.46   13.06\n",
      "  262  12282 602E 602E N   CA   4679 4680  179.21  107.00   72.21   20.76  107.00   72.21   20.76\n",
      "  263  12285 602E 602E CA  C    4680 4686  177.71  116.50   61.21   17.60  116.50   61.21   17.60\n",
      "  264  12289 602E 602E OE2 CD   4685 4683  141.32  124.00   17.32    5.57  124.00   17.32    5.57\n",
      "  265  12291 602E 603A C   N    4686 4688  177.94  120.00   57.94   13.17  120.00   57.94   13.17\n",
      "  266  12294 603A 603A N   CA   4688 4689  179.86  107.00   72.86   20.95  107.00   72.86   20.95\n",
      "  267  12296 603A 603A CA  C    4689 4691  178.14  116.50   61.64   17.72  116.50   61.64   17.72\n",
      "  268  12298 603A 604L C   N    4691 4693  178.95  120.00   58.95   13.40  120.00   58.95   13.40\n",
      "  269  12301 604L 604L N   CA   4693 4694  177.23  107.00   70.24   20.19  107.00   70.24   20.19\n",
      "  270  12304 604L 604L CA  C    4694 4699  177.29  116.50   60.79   17.48  116.50   60.79   17.48\n",
      "  271  12309 604L 605A C   N    4699 4701  177.94  120.00   57.94   13.17  120.00   57.94   13.17\n",
      "  272  12312 605A 605A N   CA   4701 4702  177.59  107.00   70.59   20.30  107.00   70.59   20.30\n",
      "  273  12314 605A 605A CA  C    4702 4704  177.94  116.50   61.44   17.67  116.50   61.44   17.67\n",
      "  274  12316 605A 606M C   N    4704 4706  178.71  120.00   58.71   13.34  120.00   58.71   13.34\n",
      "  275  12319 606M 606M N   CA   4706 4707  171.67  107.00   64.67   18.59  107.00   64.67   18.59\n",
      "  276  12321 606M 606M CA  C    4707 4712  155.83  121.00   34.84   10.02  121.00   34.84   10.02\n",
      "  277  12322 606M 606M CA  C    4707 4712  155.09  116.50   38.59   11.10  116.50   38.59   11.10\n",
      "  278  12326 606M 607L C   N    4712 4714  155.80  120.00   35.80    8.14  120.00   35.80    8.14\n",
      "  279  12329 607L 607L N   CA   4714 4715  143.24  107.00   36.24   10.42  107.00   36.24   10.42\n",
      "  280  12337 607L 608L C   N    4720 4722  150.21  120.00   30.21    6.87  120.00   30.21    6.87\n",
      "  281  12340 608L 608L N   CA   4722 4723  138.26  107.00   31.26    8.99  107.00   31.26    8.99\n",
      "  282  12348 608L 609S C   N    4728 4730  139.93  120.00   19.93    4.53  120.00   19.93    4.53\n",
      "  283  12351 609S 609S N   CA   4730 4731  134.35  107.00   27.35    7.86  107.00   27.35    7.86\n",
      "  284  12354 609S 609S CA  C    4731 4734  133.48  116.50   16.98    4.88  116.50   16.98    4.88\n",
      "  285  12359 610M 610M N   CA   4736 4737  126.96  107.00   19.96    5.74  107.00   19.96    5.74\n",
      "  286  12368 611G 611G N   CA   4744 4745  134.72  107.00   27.72    7.97  107.00   27.72    7.97\n",
      "  287  12374 612I 612I N   CA   4748 4749  131.91  107.00   24.91    7.16  107.00   24.91    7.16\n",
      "  288  12385 613E 613E N   CA   4756 4757  131.78  107.00   24.78    7.12  107.00   24.78    7.12\n",
      "  289  12397 614E 614E N   CA   4765 4766  127.73  107.00   20.73    5.96  107.00   20.73    5.96\n",
      "  290  12409 615T 615T N   CA   4774 4775  125.61  107.00   18.61    5.35  107.00   18.61    5.35\n",
      "  291  12419 616V 616V N   CA   4781 4782  127.28  107.00   20.28    5.83  107.00   20.28    5.83\n",
      "  292  12429 617A 617A N   CA   4788 4789  125.64  107.00   18.64    5.36  107.00   18.64    5.36\n",
      "  293  12465 619Y 620A C   N    4814 4816  143.27  120.00   23.27    5.29  120.00   23.27    5.29\n",
      "  294  12468 620A 620A N   CA   4816 4817  127.93  107.00   20.93    6.02  107.00   20.93    6.02\n",
      "  295  12475 621L 621L N   CA   4821 4822  122.81  107.00   15.81    4.55  107.00   15.81    4.55\n",
      "  296  13311 701L 701L N   CA   5422 5423  125.00  107.00   18.00    5.18  107.00   18.00    5.18\n",
      "  297  13322 702F 702F N   CA   5430 5431  126.68  107.00   19.68    5.66  107.00   19.68    5.66\n",
      "  298  13338 703N 703N N   CA   5441 5442  127.60  107.00   20.60    5.92  107.00   20.60    5.92\n",
      "  299  13349 704D 704D N   CA   5449 5450  129.67  107.00   22.67    6.52  107.00   22.67    6.52\n",
      "  300  13360 705E 705E N   CA   5457 5458  132.23  107.00   25.23    7.25  107.00   25.23    7.25\n",
      "  301  13372 706K 706K N   CA   5466 5467  132.40  107.00   25.40    7.30  107.00   25.40    7.30\n",
      "  302  13383 707V 707V N   CA   5475 5476  132.18  107.00   25.18    7.24  107.00   25.18    7.24\n",
      "  303  13387 707V 707V CA  C    5476 5480  135.75  116.50   19.25    5.53  116.50   19.25    5.53\n",
      "  304  13393 708A 708A N   CA   5482 5483  132.56  107.00   25.56    7.35  107.00   25.56    7.35\n",
      "  305  13395 708A 708A CA  C    5483 5485  134.19  116.50   17.69    5.09  116.50   17.69    5.09\n",
      "  306  13400 709E 709E N   CA   5487 5488  136.91  107.00   29.91    8.60  107.00   29.91    8.60\n",
      "  307  13412 710S 710S N   CA   5496 5497  134.21  107.00   27.21    7.82  107.00   27.21    7.82\n",
      "  308  13417 710S 711L C   N    5500 5502  140.84  120.00   20.84    4.74  120.00   20.84    4.74\n",
      "  309  13420 711L 711L N   CA   5502 5503  139.31  107.00   32.31    9.29  107.00   32.31    9.29\n",
      "  310  13428 711L 712N C   N    5508 5510  144.72  120.00   24.72    5.62  120.00   24.72    5.62\n",
      "  311  13431 712N 712N N   CA   5510 5511  134.73  107.00   27.73    7.97  107.00   27.73    7.97\n",
      "  312  13439 712N 713P C   N    5516 5518  155.24  117.00   38.24    9.52  117.00   38.24    9.52\n",
      "  313  13442 713P 713P N   CA   5518 5519  154.90  108.20   46.71   10.61  108.20   46.71   10.61\n",
      "  314  13451 713P 714P C   N    5523 5525  135.76  117.00   18.76    4.67  117.00   18.76    4.67\n",
      "  315  13454 714P 714P N   CA   5525 5526  146.58  108.20   38.38    8.72  108.20   38.38    8.72\n",
      "  316  13458 714P 714P CA  C    5526 5530  142.53  116.50   26.03    7.48  116.50   26.03    7.48\n",
      "  317  13463 714P 715F C   N    5530 5532  151.75  120.00   31.75    7.21  120.00   31.75    7.21\n",
      "  318  13466 715F 715F N   CA   5532 5533  148.99  107.00   41.99   12.07  107.00   41.99   12.07\n",
      "  319  13467 715F 715F CA  CB   5533 5534  132.60  107.50   25.10    5.80  107.50   25.10    5.80\n",
      "  320  13469 715F 715F CA  C    5533 5541  142.08  116.50   25.58    7.36  116.50   25.58    7.36\n",
      "  321  13479 715F 716S C   N    5541 5543  151.92  120.00   31.92    7.25  120.00   31.92    7.25\n",
      "  322  13482 716S 716S N   CA   5543 5544  152.08  107.00   45.08   12.96  107.00   45.08   12.96\n",
      "  323  13485 716S 716S CA  C    5544 5547  145.86  116.50   29.36    8.44  116.50   29.36    8.44\n",
      "  324  13487 716S 717M C   N    5547 5549  148.30  120.00   28.30    6.43  120.00   28.30    6.43\n",
      "  325  13490 717M 717M N   CA   5549 5550  146.51  107.00   39.51   11.36  107.00   39.51   11.36\n",
      "  326  13493 717M 717M CA  C    5550 5555  142.75  116.50   26.25    7.55  116.50   26.25    7.55\n",
      "  327  13497 717M 718A C   N    5555 5557  153.99  120.00   33.99    7.73  120.00   33.99    7.73\n",
      "  328  13500 718A 718A N   CA   5557 5558  151.53  107.00   44.53   12.80  107.00   44.53   12.80\n",
      "  329  13502 718A 718A CA  C    5558 5560  152.11  116.50   35.61   10.24  116.50   35.61   10.24\n",
      "  330  13504 718A 719F C   N    5560 5562  158.15  120.00   38.15    8.67  120.00   38.15    8.67\n",
      "  331  13507 719F 719F N   CA   5562 5563  154.54  107.00   47.54   13.67  107.00   47.54   13.67\n",
      "  332  13510 719F 719F CA  C    5563 5571  154.66  116.50   38.16   10.97  116.50   38.16   10.97\n",
      "  333  13520 719F 720L C   N    5571 5573  155.15  120.00   35.15    7.99  120.00   35.15    7.99\n",
      "  334  13523 720L 720L N   CA   5573 5574  157.91  107.00   50.91   14.64  107.00   50.91   14.64\n",
      "  335  13526 720L 720L CA  C    5574 5579  154.07  116.50   37.57   10.80  116.50   37.57   10.80\n",
      "  336  13531 720L 721Y C   N    5579 5581  152.64  120.00   32.64    7.42  120.00   32.64    7.42\n",
      "  337  13534 721Y 721Y N   CA   5581 5582  161.33  107.00   54.33   15.62  107.00   54.33   15.62\n",
      "  338  13537 721Y 721Y CA  C    5582 5591  157.85  116.50   41.35   11.89  116.50   41.35   11.89\n",
      "  339  13549 721Y 722F C   N    5591 5593  157.81  120.00   37.81    8.59  120.00   37.81    8.59\n",
      "  340  13552 722F 722F N   CA   5593 5594  158.32  107.00   51.32   14.76  107.00   51.32   14.76\n",
      "  341  13555 722F 722F CA  C    5594 5602  164.52  116.50   48.02   13.81  116.50   48.02   13.81\n",
      "  342  13565 722F 723Y C   N    5602 5604  162.80  120.00   42.80    9.73  120.00   42.80    9.73\n",
      "  343  13568 723Y 723Y N   CA   5604 5605  157.64  107.00   50.64   14.56  107.00   50.64   14.56\n",
      "  344  13571 723Y 723Y CA  C    5605 5614  163.61  116.50   47.11   13.54  116.50   47.11   13.54\n",
      "  345  13583 723Y 724K C   N    5614 5616  164.30  120.00   44.30   10.07  120.00   44.30   10.07\n",
      "  346  13586 724K 724K N   CA   5616 5617  162.36  107.00   55.36   15.92  107.00   55.36   15.92\n",
      "  347  13589 724K 724K CA  C    5617 5623  162.61  116.50   46.11   13.26  116.50   46.11   13.26\n",
      "  348  13594 724K 725R C   N    5623 5625  172.27  120.00   52.27   11.88  120.00   52.27   11.88\n",
      "  349  13597 725R 725R N   CA   5625 5626  171.23  107.00   64.23   18.47  107.00   64.23   18.47\n",
      "  350  13600 725R 725R CA  C    5626 5634  167.59  116.50   51.09   14.69  116.50   51.09   14.69\n",
      "  351  13608 725R 726V C   N    5634 5636  166.46  120.00   46.46   10.56  120.00   46.46   10.56\n",
      "  352  13611 726V 726V N   CA   5636 5637  168.63  107.00   61.63   17.72  107.00   61.63   17.72\n",
      "  353  13615 726V 726V CA  C    5637 5641  165.63  116.50   49.13   14.13  116.50   49.13   14.13\n",
      "  354  13618 726V 727G C   N    5641 5643  171.56  120.00   51.56   11.72  120.00   51.56   11.72\n",
      "  355  13620 727G 727G N   CA   5643 5644  170.86  107.00   63.86   18.36  107.00   63.86   18.36\n",
      "  356  13622 727G 727G CA  C    5644 5645  166.52  116.50   50.02   14.38  116.50   50.02   14.38\n",
      "  357  13623 727G 728K C   N    5645 5647  174.25  120.00   54.25   12.33  120.00   54.25   12.33\n",
      "  358  13626 728K 728K N   CA   5647 5648  167.34  107.00   60.34   13.71  107.00   60.34   13.71\n",
      "  359  13628 728K 728K CA  C    5648 5654  164.59  118.00   46.59    9.47  118.00   46.59    9.47\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19291 602E 602E OE1 CG   4684 4682   26.15    0.00   26.15    8.24    0.00   26.15    8.24\n",
      "    2  19301 606M 606M CA  N    4707 4706   62.51   35.27   27.25    6.50   35.27   27.25    6.50\n",
      "    3  19655 714P 714P CA  N    5526 5525   54.64   35.27   19.37    4.62   35.27   19.37    4.62\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22527  47G  60V CA  CA    371  459   23.08    5.82   17.26    5.81    5.82   17.26    5.81\n",
      "    2  22528  47G  61E CA  CA    371  466   26.50    7.97   18.53    7.04    7.97   18.53    7.04\n",
      "    3  22529  47G  62H CA  CA    371  475   27.73    9.00   18.73    7.64    9.00   18.73    7.64\n",
      "    4  22530  47G  63V CA  CA    371  485   29.99   12.77   17.23    8.07   12.77   17.23    8.07\n",
      "    5  22531  47G 243C CA  CA    371 1840   20.12   13.26    6.86    9.95   13.26    6.86    9.95\n",
      "    6  22532  47G 244D CA  CA    371 1846   18.82   12.85    5.97    7.24   12.85    5.97    7.24\n",
      "    7  22533  47G 245D CA  CA    371 1854   21.09   10.17   10.93   14.75   10.17   10.93   14.75\n",
      "    8  22534  47G 246E CA  CA    371 1862   24.68   10.81   13.87   21.44   10.81   13.87   21.44\n",
      "    9  22535  47G 247V CA  CA    371 1871   27.11   11.44   15.67   25.51   11.44   15.67   25.51\n",
      "   10  22536  47G 248S CA  CA    371 1878   30.97   13.40   17.57   20.92   13.40   17.57   20.92\n",
      "   11  22537  48T  60V CA  CA    375  459   18.23    3.89   14.34    5.25    3.89   14.34    5.25\n",
      "   12  22538  48T  61E CA  CA    375  466   21.54    6.49   15.05    6.21    6.49   15.05    6.21\n",
      "   13  22539  48T  62H CA  CA    375  475   22.79    9.04   13.75    6.03    9.04   13.75    6.03\n",
      "   14  22540  48T  63V CA  CA    375  485   25.04   12.76   12.28    6.34   12.76   12.28    6.34\n",
      "   15  22541  48T 237D CA  CA    375 1783   29.31   13.73   15.57   25.70   13.73   15.57   25.70\n",
      "   16  22542  48T 239Y CA  CA    375 1802   21.98   13.74    8.24   15.20   13.74    8.24   15.20\n",
      "   17  22544  48T 245D CA  CA    375 1854   16.07   10.05    6.02    9.87   10.05    6.02    9.87\n",
      "   18  22545  48T 246E CA  CA    375 1862   19.67    9.43   10.24   20.72    9.43   10.24   20.72\n",
      "   19  22546  48T 247V CA  CA    375 1871   22.05   10.45   11.61   24.54   10.45   11.61   24.54\n",
      "   20  22547  48T 248S CA  CA    375 1878   25.92   12.09   13.83   21.41   12.09   13.83   21.41\n",
      "   21  22566  61E 235G CA  CA    466 1774   16.78   12.03    4.76    8.71   12.03    4.76    8.71\n",
      "   22  22567  61E 236A CA  CA    466 1778   14.64   11.52    3.13    6.54   11.52    3.13    6.54\n",
      "   23  22912  86D 254N CA  CA    676 1925   21.76   13.93    7.84    7.08   13.93    7.84    7.08\n",
      "   24  22913  86D 255H CA  CA    676 1933   18.23   12.25    5.97    7.42   12.25    5.97    7.42\n",
      "   25  22914  86D 256M CA  CA    676 1943   19.77   13.51    6.26    7.18   13.51    6.26    7.18\n",
      "   26  22915  86D 257Q CA  CA    676 1951   21.16   12.79    8.38    5.80   12.79    8.38    5.80\n",
      "   27  22916  86D 258H CA  CA    676 1960   17.86    8.96    8.90    5.91    8.96    8.90    5.91\n",
      "   28  22926  96N 198G CA  CA    744 1504   13.55   10.05    3.50    4.61   10.05    3.50    4.61\n",
      "   29  22928  96N 255H CA  CA    744 1933   22.79   13.60    9.19   10.53   13.60    9.19   10.53\n",
      "   30  22929  96N 256M CA  CA    744 1943   23.87   13.80   10.07   11.03   13.80   10.07   11.03\n",
      "   31  22930  96N 257Q CA  CA    744 1951   25.36   12.80   12.56    8.46   12.80   12.56    8.46\n",
      "   32  22931  96N 258H CA  CA    744 1960   22.01    9.26   12.75    8.23    9.26   12.75    8.23\n",
      "   33  22932  96N 259F CA  CA    744 1970   20.80    9.20   11.60    6.34    9.20   11.60    6.34\n",
      "   34  22933  96N 260G CA  CA    744 1981   23.06   11.36   11.70    4.69   11.36   11.70    4.69\n",
      "   35  22946 103P 254N CA  CA    793 1925   20.89   13.24    7.65    9.58   13.24    7.65    9.58\n",
      "   36  22947 103P 255H CA  CA    793 1933   17.36   11.11    6.25   11.19   11.11    6.25   11.19\n",
      "   37  22948 103P 256M CA  CA    793 1943   17.55   10.74    6.81   11.70   10.74    6.81   11.70\n",
      "   38  22949 103P 257Q CA  CA    793 1951   18.45   10.47    7.98    7.26   10.47    7.98    7.26\n",
      "   39  22950 103P 258H CA  CA    793 1960   15.01    7.11    7.90    6.31    7.11    7.90    6.31\n",
      "   40  22951 103P 259F CA  CA    793 1970   13.64    5.98    7.66    5.03    5.98    7.66    5.03\n",
      "   41  22997 117K 606M CA  CA    893 4707   35.99   13.64   22.34    9.01   13.64   22.34    9.01\n",
      "   42  23034 118S 606M CA  CA    902 4707   35.32   11.87   23.45    8.67   11.87   23.45    8.67\n",
      "   43  23061 122P 606M CA  CA    932 4707   37.21   11.70   25.51    8.98   11.70   25.51    8.98\n",
      "   44  23081 123P 233P CA  CA    939 1761   16.57   13.32    3.25    4.75   13.32    3.25    4.75\n",
      "   45  23090 123P 606M CA  CA    939 4707   36.44    9.60   26.84    9.26    9.60   26.84    9.26\n",
      "   46  23091 123P 607L CA  CA    939 4715   29.39   12.23   17.17    5.30   12.23   17.17    5.30\n",
      "   47  23108 129C 229G CA  CA    986 1736   11.08    7.57    3.51    6.70    7.57    3.51    6.70\n",
      "   48  23109 129C 230T CA  CA    986 1740   14.70   11.11    3.59    6.31   11.11    3.59    6.31\n",
      "   49  23110 129C 231V CA  CA    986 1747   17.03   10.93    6.09   11.24   10.93    6.09   11.24\n",
      "   50  23111 129C 232T CA  CA    986 1754   20.04   12.90    7.14   10.71   12.90    7.14   10.71\n",
      "   51  23116 129C 606M CA  CA    986 4707   44.51   12.81   31.71   10.59   12.81   31.71   10.59\n",
      "   52  23133 140G 227K CA  CA   1076 1719   17.32   13.32    4.00    6.41   13.32    4.00    6.41\n",
      "   53  23134 140G 228L CA  CA   1076 1728   16.46    9.80    6.66   13.77    9.80    6.66   13.77\n",
      "   54  23135 140G 229G CA  CA   1076 1736   19.03    9.76    9.27   15.62    9.76    9.27   15.62\n",
      "   55  23136 140G 230T CA  CA   1076 1740   22.35   13.58    8.77   14.45   13.58    8.77   14.45\n",
      "   56  23160 155L 228L CA  CA   1178 1728   11.18    8.19    2.98    8.19    8.19    2.98    8.19\n",
      "   57  23161 155L 229G CA  CA   1178 1736   14.13    9.59    4.53    9.34    9.59    4.53    9.34\n",
      "   58  23162 155L 230T CA  CA   1178 1740   17.40   13.09    4.31    9.13   13.09    4.31    9.13\n",
      "   59  23163 155L 231V CA  CA   1178 1747   18.76   13.69    5.06   11.17   13.69    5.06   11.17\n",
      "   60  23222 157C 226V CA  CA   1195 1712   15.25   13.60    1.65    4.51   13.60    1.65    4.51\n",
      "   61  23223 157C 227K CA  CA   1195 1719   14.38   11.98    2.40    5.56   11.98    2.40    5.56\n",
      "   62  23224 157C 228L CA  CA   1195 1728   13.45   10.94    2.51    7.02   10.94    2.51    7.02\n",
      "   63  23225 157C 229G CA  CA   1195 1736   16.45   13.47    2.98    5.86   13.47    2.98    5.86\n",
      "   64  23235 168S 193M CA  CA   1280 1469   16.55   10.88    5.67    5.41   10.88    5.67    5.41\n",
      "   65  23236 168S 194C CA  CA   1280 1477   16.78   12.24    4.53    6.04   12.24    4.53    6.04\n",
      "   66  23237 168S 197C CA  CA   1280 1498   17.35   13.00    4.36    5.99   13.00    4.36    5.99\n",
      "   67  23238 168S 198G CA  CA   1280 1504   17.08   11.08    6.01    8.95   11.08    6.01    8.95\n",
      "   68  23239 168S 199T CA  CA   1280 1508   13.26    7.73    5.53    8.08    7.73    5.53    8.08\n",
      "   69  23240 168S 200I CA  CA   1280 1515   12.65    6.23    6.42    6.76    6.23    6.42    6.76\n",
      "   70  23252 168S 228L CA  CA   1280 1728   20.65   13.67    6.98   13.43   13.67    6.98   13.43\n",
      "   71  23261 179K 198G CA  CA   1359 1504   19.06   13.88    5.17    7.85   13.88    5.17    7.85\n",
      "   72  23262 179K 199T CA  CA   1359 1508   15.23   10.15    5.09    7.25   10.15    5.09    7.25\n",
      "   73  23263 179K 200I CA  CA   1359 1515   13.60    8.82    4.78    4.89    8.82    4.78    4.89\n",
      "   74  23283 180C 199T CA  CA   1368 1508   16.00   12.88    3.12    4.71   12.88    3.12    4.71\n",
      "   75  23284 180C 200I CA  CA   1368 1515   13.64   10.55    3.09    4.98   10.55    3.09    4.98\n",
      "   76  23335 182L 228L CA  CA   1379 1728   14.20   12.33    1.87    4.54   12.33    1.87    4.54\n",
      "   77  23336 182L 229G CA  CA   1379 1736   16.79   13.56    3.23    5.80   13.56    3.23    5.80\n",
      "   78  23354 188R 229G CA  CA   1420 1736   13.33   10.56    2.77    4.78   10.56    2.77    4.78\n",
      "   79  23435 191N 231V CA  CA   1447 1747   14.23   12.10    2.13    4.95   12.10    2.13    4.95\n",
      "   80  23471 192W 231V CA  CA   1455 1747   13.57   11.32    2.25    4.97   11.32    2.25    4.97\n",
      "   81  23517 193M 252F CA  CA   1469 1909   14.53   12.33    2.20    5.77   12.33    2.20    5.77\n",
      "   82  23518 193M 253A CA  CA   1469 1920   16.82   14.00    2.82    5.23   14.00    2.82    5.23\n",
      "   83  23526 193M 606M CA  CA   1469 4707   38.75   13.92   24.84   26.34   13.92   24.84   26.34\n",
      "   84  23567 194C 252F CA  CA   1477 1909   14.06   12.17    1.89    4.89   12.17    1.89    4.89\n",
      "   85  23605 195M 252F CA  CA   1483 1909   10.96    9.18    1.78    5.24    9.18    1.78    5.24\n",
      "   86  23685 198G 252F CA  CA   1504 1909   15.77   13.63    2.13    5.51   13.63    2.13    5.51\n",
      "   87  23688 198G 256M CA  CA   1504 1943   11.43    9.59    1.84    5.26    9.59    1.84    5.26\n",
      "   88  23815 203P 230T CA  CA   1533 1740   16.19   13.52    2.67    5.73   13.52    2.67    5.73\n",
      "   89  24135 226V 606M CA  CA   1712 4707   35.13   11.91   23.22   62.08   11.91   23.22   62.08\n",
      "   90  24158 227K 606M CA  CA   1719 4707   36.15   12.70   23.46   50.96   12.70   23.46   50.96\n",
      "   91  24175 228L 606M CA  CA   1728 4707   35.88   11.25   24.63   64.93   11.25   24.63   64.93\n",
      "   92  24187 229G 606M CA  CA   1736 4707   33.62   10.88   22.74   46.10   10.88   22.74   46.10\n",
      "   93  24188 229G 607L CA  CA   1736 4715   26.25   13.87   12.38   17.92   13.87   12.38   17.92\n",
      "   94  24189 229G 728K CA  CA   1736 5648   21.19   12.91    8.28   14.49   12.91    8.28   14.49\n",
      "   95  24198 230T 246E CA  CA   1740 1862   16.98   12.94    4.04    7.69   12.94    4.04    7.69\n",
      "   96  24206 230T 606M CA  CA   1740 4707   30.67    9.84   20.82   49.37    9.84   20.82   49.37\n",
      "   97  24207 230T 607L CA  CA   1740 4715   23.24   12.62   10.62   18.22   12.62   10.62   18.22\n",
      "   98  24208 230T 728K CA  CA   1740 5648   24.17   12.71   11.46   23.11   12.71   11.46   23.11\n",
      "   99  24214 231V 238F CA  CA   1747 1791   11.65    9.55    2.11    4.89    9.55    2.11    4.89\n",
      "  100  24215 231V 239Y CA  CA   1747 1802   15.55   13.01    2.54    5.34   13.01    2.54    5.34\n",
      "  101  24229 231V 606M CA  CA   1747 4707   27.68    6.06   21.62   58.71    6.06   21.62   58.71\n",
      "  102  24230 231V 607L CA  CA   1747 4715   20.26    9.09   11.17   22.53    9.09   11.17   22.53\n",
      "  103  24231 231V 728K CA  CA   1747 5648   27.16    9.49   17.67   40.41    9.49   17.67   40.41\n",
      "  104  24244 232T 262D CA  CA   1754 1993   13.15    9.01    4.15    7.85    9.01    4.15    7.85\n",
      "  105  24245 232T 606M CA  CA   1754 4707   24.80    5.37   19.43   48.02    5.37   19.43   48.02\n",
      "  106  24246 232T 607L CA  CA   1754 4715   17.46    6.72   10.74   21.86    6.72   10.74   21.86\n",
      "  107  24247 232T 728K CA  CA   1754 5648   29.09    7.50   21.59   46.21    7.50   21.59   46.21\n",
      "  108  24261 233P 262D CA  CA   1761 1993   14.92    8.58    6.34   12.06    8.58    6.34   12.06\n",
      "  109  24262 233P 606M CA  CA   1761 4707   21.16    5.20   15.96   38.81    5.20   15.96   38.81\n",
      "  110  24263 233P 607L CA  CA   1761 4715   13.83    4.36    9.47   22.64    4.36    9.47   22.64\n",
      "  111  24264 233P 728K CA  CA   1761 5648   32.64    6.73   25.91   56.36    6.73   25.91   56.36\n",
      "  112  24268 234S 248S CA  CA   1768 1878   15.21   11.90    3.31    4.70   11.90    3.31    4.70\n",
      "  113  24277 234S 606M CA  CA   1768 4707   22.23    8.18   14.05   26.32    8.18   14.05   26.32\n",
      "  114  24278 234S 607L CA  CA   1768 4715   14.88    8.02    6.86   11.22    8.02    6.86   11.22\n",
      "  115  24279 234S 728K CA  CA   1768 5648   32.53   10.59   21.94   34.88   10.59   21.94   34.88\n",
      "  116  24282 235G 239Y CA  CA   1774 1802   15.01   12.66    2.34    4.56   12.66    2.34    4.56\n",
      "  117  24283 235G 246E CA  CA   1774 1862   17.11   12.57    4.54    8.46   12.57    4.54    8.46\n",
      "  118  24284 235G 247V CA  CA   1774 1871   14.81   11.30    3.52    7.27   11.30    3.52    7.27\n",
      "  119  24285 235G 248S CA  CA   1774 1878   11.56    8.95    2.61    4.85    8.95    2.61    4.85\n",
      "  120  24297 235G 606M CA  CA   1774 4707   24.93    7.52   17.41   40.52    7.52   17.41   40.52\n",
      "  121  24298 235G 607L CA  CA   1774 4715   17.50    9.05    8.45   15.60    9.05    8.45   15.60\n",
      "  122  24299 235G 728K CA  CA   1774 5648   31.37   11.48   19.89   38.64   11.48   19.89   38.64\n",
      "  123  24303 236A 245D CA  CA   1778 1854   17.36   13.89    3.47    5.92   13.89    3.47    5.92\n",
      "  124  24304 236A 246E CA  CA   1778 1862   14.11   10.23    3.88    8.28   10.23    3.88    8.28\n",
      "  125  24305 236A 247V CA  CA   1778 1871   12.29    9.89    2.39    5.63    9.89    2.39    5.63\n",
      "  126  24318 236A 606M CA  CA   1778 4707   28.89    8.77   20.12   51.45    8.77   20.12   51.45\n",
      "  127  24319 236A 607L CA  CA   1778 4715   21.42   11.42   10.00   18.91   11.42   10.00   18.91\n",
      "  128  24320 236A 728K CA  CA   1778 5648   28.21   12.98   15.23   32.68   12.98   15.23   32.68\n",
      "  129  24325 237D 245D CA  CA   1783 1854   13.49   10.62    2.87    4.69   10.62    2.87    4.69\n",
      "  130  24326 237D 246E CA  CA   1783 1862   10.43    7.05    3.38    6.91    7.05    3.38    6.91\n",
      "  131  24336 237D 606M CA  CA   1783 4707   32.80   12.37   20.42   41.80   12.37   20.42   41.80\n",
      "  132  24343 238F 246E CA  CA   1791 1862    7.93    5.73    2.20    5.04    5.73    2.20    5.04\n",
      "  133  24354 238F 606M CA  CA   1791 4707   34.83   13.44   21.39   52.06   13.44   21.39   52.06\n",
      "  134  24405 246E 252F CA  CA   1862 1909   16.35   13.43    2.92    6.09   13.43    2.92    6.09\n",
      "  135  24409 247V 252F CA  CA   1871 1909   13.07   10.88    2.19    5.45   10.88    2.19    5.45\n",
      "  136  24424 249D 606M CA  CA   1884 4707   30.39   13.37   17.02   40.95   13.37   17.02   40.95\n",
      "  137  24431 250V 606M CA  CA   1892 4707   27.13   13.43   13.70   25.16   13.43   13.70   25.16\n",
      "  138  24441 251H 606M CA  CA   1899 4707   26.40   12.08   14.32   28.82   12.08   14.32   28.82\n",
      "  139  24442 251H 607L CA  CA   1899 4715   19.74   13.68    6.06    8.62   13.68    6.06    8.62\n",
      "  140  24452 252F 606M CA  CA   1909 4707   25.84    8.96   16.87   39.52    8.96   16.87   39.52\n",
      "  141  24453 252F 607L CA  CA   1909 4715   18.70   11.27    7.43   11.91   11.27    7.43   11.91\n",
      "  142  24454 252F 728K CA  CA   1909 5648   34.82   13.91   20.91   37.40   13.91   20.91   37.40\n",
      "  143  24462 253A 262D CA  CA   1920 1993   11.93    7.50    4.44    7.28    7.50    4.44    7.28\n",
      "  144  24463 253A 606M CA  CA   1920 4707   22.66    6.73   15.93   31.65    6.73   15.93   31.65\n",
      "  145  24464 253A 607L CA  CA   1920 4715   15.56    8.40    7.16   10.34    8.40    7.16   10.34\n",
      "  146  24465 253A 728K CA  CA   1920 5648   36.46   11.13   25.34   37.47   11.13   25.34   37.47\n",
      "  147  24472 254N 262D CA  CA   1925 1993   12.97    9.29    3.68    4.79    9.29    3.68    4.79\n",
      "  148  24473 254N 606M CA  CA   1925 4707   24.97    9.84   15.14   22.49    9.84   15.14   22.49\n",
      "  149  24474 254N 607L CA  CA   1925 4715   18.25   11.52    6.72    7.31   11.52    6.72    7.31\n",
      "  150  24475 254N 728K CA  CA   1925 5648   37.06   13.83   23.22   27.34   13.83   23.22   27.34\n",
      "  151  24482 255H 606M CA  CA   1933 4707   28.64   10.39   18.25   35.22   10.39   18.25   35.22\n",
      "  152  24483 255H 607L CA  CA   1933 4715   21.75   13.12    8.63   11.55   13.12    8.63   11.55\n",
      "  153  24488 256M 262D CA  CA   1943 1993   10.07    7.00    3.08    5.34    7.00    3.08    5.34\n",
      "  154  24489 256M 606M CA  CA   1943 4707   28.13    7.34   20.79   42.02    7.34   20.79   42.02\n",
      "  155  24490 256M 607L CA  CA   1943 4715   20.91   10.55   10.36   14.39   10.55   10.36   14.39\n",
      "  156  24491 256M 728K CA  CA   1943 5648   31.66   11.69   19.97   31.78   11.69   19.97   31.78\n",
      "  157  24494 257Q 261I CA  CA   1951 1985    9.35    5.03    4.32    7.70    5.03    4.32    7.70\n",
      "  158  24495 257Q 262D CA  CA   1951 1993   13.63    6.08    7.56    7.44    6.08    7.56    7.44\n",
      "  159  24496 257Q 606M CA  CA   1951 4707   26.53    8.14   18.39   18.37    8.14   18.39   18.37\n",
      "  160  24497 257Q 607L CA  CA   1951 4715   19.57   10.33    9.24    7.36   10.33    9.24    7.36\n",
      "  161  24498 257Q 728K CA  CA   1951 5648   33.23   11.32   21.91   18.94   11.32   21.91   18.94\n",
      "  162  24502 258H 606M CA  CA   1960 4707   29.95   11.57   18.38   13.86   11.57   18.38   13.86\n",
      "  163  24505 259F 606M CA  CA   1970 4707   31.75   10.95   20.79   13.05   10.95   20.79   13.05\n",
      "  164  24506 259F 728K CA  CA   1970 5648   28.99   13.89   15.10    8.82   13.89   15.10    8.82\n",
      "  165  24508 260G 606M CA  CA   1981 4707   30.29    9.52   20.77    9.93    9.52   20.77    9.93\n",
      "  166  24509 260G 607L CA  CA   1981 4715   22.90   11.87   11.02    4.63   11.87   11.02    4.63\n",
      "  167  24510 260G 728K CA  CA   1981 5648   28.33   11.23   17.10    7.74   11.23   17.10    7.74\n",
      "  168  24511 261I 606M CA  CA   1985 4707   31.30    5.99   25.31   11.30    5.99   25.31   11.30\n",
      "  169  24512 261I 607L CA  CA   1985 4715   23.94    8.92   15.02    6.04    8.92   15.02    6.04\n",
      "  170  24513 261I 728K CA  CA   1985 5648   29.57    8.38   21.19    9.00    8.38   21.19    9.00\n",
      "  171  24514 262D 606M CA  CA   1993 4707   32.58    3.83   28.75   11.00    3.83   28.75   11.00\n",
      "  172  24515 262D 607L CA  CA   1993 4715   25.29    5.40   19.90    7.09    5.40   19.90    7.09\n",
      "  173  24516 262D 728K CA  CA   1993 5648   29.66    5.28   24.38    8.99    5.28   24.38    8.99\n",
      "  174  24517 606M 728K CA  CA   4707 5648   53.41    5.10   48.31   18.44    5.10   48.31   18.44\n",
      "  175  24518 607L 728K CA  CA   4715 5648   46.35    3.83   42.53   15.84    3.83   42.53   15.84\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24519  47G  60V N   O     370  464   25.75    5.46   20.29    6.30    5.46   20.29    6.30\n",
      "    2  24520  47G  61E N   O     370  473   29.97    9.39   20.58    7.12    9.39   20.58    7.12\n",
      "    3  24521  47G  62H N   O     370  483   30.48   10.24   20.24    7.30   10.24   20.24    7.30\n",
      "    4  24522  47G 243C N   O     370 1844   20.22   10.34    9.87   14.57   10.34    9.87   14.57\n",
      "    5  24523  47G 244D N   O     370 1852   20.23    9.80   10.43   12.31    9.80   10.43   12.31\n",
      "    6  24524  47G 246E N   O     370 1869   26.17    8.86   17.31   26.00    8.86   17.31   26.00\n",
      "    7  24525  48T  60V N   O     374  464   20.69    4.55   16.14    5.47    4.55   16.14    5.47\n",
      "    8  24526  48T  61E N   O     374  473   24.93    8.51   16.42    6.21    8.51   16.42    6.21\n",
      "    9  24527  48T  62H N   O     374  483   25.27   10.49   14.78    5.90   10.49   14.78    5.90\n",
      "   10  24528  48T 246E N   O     374 1869   20.92    8.70   12.22   23.03    8.70   12.22   23.03\n",
      "   11  24529  60V  47G N   O     458  373   19.06    3.45   15.61    5.07    3.45   15.61    5.07\n",
      "   12  24530  60V  48T N   O     458  380   15.23    2.29   12.94    4.59    2.29   12.94    4.59\n",
      "   13  24534  61E  47G N   O     465  373   22.75    5.39   17.35    6.47    5.39   17.35    6.47\n",
      "   14  24535  61E  48T N   O     465  380   18.67    3.58   15.09    6.23    3.58   15.09    6.23\n",
      "   15  24537  61E 236A N   O     465 1781   13.33    9.83    3.50    7.13    9.83    3.50    7.13\n",
      "   16  24543  62H  47G N   O     474  373   24.66    6.52   18.14    7.10    6.52   18.14    7.10\n",
      "   17  24544  62H  48T N   O     474  380   20.47    6.51   13.96    6.02    6.51   13.96    6.02\n",
      "   18  24557  63V  47G N   O     484  373   27.01    9.89   17.12    7.85    9.89   17.12    7.85\n",
      "   19  24558  63V  48T N   O     484  380   22.83   10.09   12.74    6.45   10.09   12.74    6.45\n",
      "   20  24681  79I 249D N   O     620 1890   12.90   10.58    2.33    5.89   10.58    2.33    5.89\n",
      "   21  24682  79I 250V N   O     620 1897   13.66   10.95    2.72    5.03   10.95    2.72    5.03\n",
      "   22  24683  79I 251H N   O     620 1907   12.01    8.69    3.32    7.04    8.69    3.32    7.04\n",
      "   23  24746  86D 196T N   O     675 1496    9.23    6.06    3.17    4.87    6.06    3.17    4.87\n",
      "   24  24748  86D 255H N   O     675 1941   17.90   10.89    7.01    8.18   10.89    7.01    8.18\n",
      "   25  24749  86D 258H N   O     675 1968   17.98    8.43    9.55    5.92    8.43    9.55    5.92\n",
      "   26  24756  96N 195M N   O     743 1489   17.28   10.61    6.68    7.67   10.61    6.68    7.67\n",
      "   27  24757  96N 196T N   O     743 1496   14.58    8.31    6.27    8.30    8.31    6.27    8.30\n",
      "   28  24758  96N 197C N   O     743 1502   12.83    8.60    4.22    5.17    8.60    4.22    5.17\n",
      "   29  24759  96N 257Q N   O     743 1958   25.49   10.58   14.91    9.53   10.58   14.91    9.53\n",
      "   30  24760  96N 258H N   O     743 1968   22.69    7.30   15.39    9.62    7.30   15.39    9.62\n",
      "   31  24761  96N 259F N   O     743 1979   23.39   10.74   12.64    6.28   10.74   12.64    6.28\n",
      "   32  24774 103P 255H N   O     792 1941   15.98    9.83    6.15   10.33    9.83    6.15   10.33\n",
      "   33  24775 103P 256M N   O     792 1949   18.14   10.09    8.04   12.67   10.09    8.04   12.67\n",
      "   34  24776 103P 257Q N   O     792 1958   17.46    8.93    8.53    7.42    8.93    8.53    7.42\n",
      "   35  24777 103P 258H N   O     792 1968   14.54    5.45    9.09    7.10    5.45    9.09    7.10\n",
      "   36  24792 116A 256M N   O     887 1949   14.07    9.39    4.69    6.28    9.39    4.69    6.28\n",
      "   37  24840 122P 256M N   O     931 1949   13.06    9.12    3.93    5.75    9.12    3.93    5.75\n",
      "   38  24871 129C 231V N   O     985 1752   17.73    9.74    7.99   14.14    9.74    7.99   14.14\n",
      "   39  24887 140G 228L N   O    1075 1734   16.92    8.93    8.00   15.56    8.93    8.00   15.56\n",
      "   40  24888 140G 229G N   O    1075 1738   21.66   10.45   11.21   16.42   10.45   11.21   16.42\n",
      "   41  24906 155L 228L N   O    1177 1734   12.52    8.66    3.86    9.78    8.66    3.86    9.78\n",
      "   42  24948 157C 228L N   O    1194 1734   12.72   10.65    2.07    5.65   10.65    2.07    5.65\n",
      "   43  24961 168S 192W N   O    1279 1467   15.37    8.00    7.37    5.08    8.00    7.37    5.08\n",
      "   44  24962 168S 198G N   O    1279 1506   16.25    8.70    7.55   10.98    8.70    7.55   10.98\n",
      "   45  24963 168S 199T N   O    1279 1513   14.86    7.75    7.11    9.12    7.75    7.11    9.12\n",
      "   46  24964 168S 200I N   O    1279 1521   11.37    3.06    8.30    9.01    3.06    8.30    9.01\n",
      "   47  24965 168S 201G N   O    1279 1525   13.67    6.91    6.76    4.85    6.91    6.76    4.85\n",
      "   48  24980 179K 198G N   O    1358 1506   18.76   10.77    8.00   11.85   10.77    8.00   11.85\n",
      "   49  24981 179K 199T N   O    1358 1513   16.38    9.61    6.77    8.76    9.61    6.77    8.76\n",
      "   50  24982 179K 200I N   O    1358 1521   12.78    5.15    7.63    8.05    5.15    7.63    8.05\n",
      "   51  24997 180C 200I N   O    1367 1521   11.50    7.05    4.45    7.34    7.05    4.45    7.34\n",
      "   52  25223 195M 251H N   O    1482 1907   12.65   10.45    2.21    5.28   10.45    2.21    5.28\n",
      "   53  25224 195M 252F N   O    1482 1918   12.30    9.69    2.61    7.16    9.69    2.61    7.16\n",
      "   54  25253 196T 251H N   O    1490 1907   12.67   10.67    1.99    4.71   10.67    1.99    4.71\n",
      "   55  25254 196T 252F N   O    1490 1918   12.94   10.58    2.36    6.26   10.58    2.36    6.26\n",
      "   56  25287 198G  96N N   O    1503  750   12.30    8.40    3.90    4.87    8.40    3.90    4.87\n",
      "   57  25304 198G 256M N   O    1503 1949   11.87    9.83    2.04    5.50    9.83    2.04    5.50\n",
      "   58  25316 199T 168S N   O    1507 1284   16.76   10.73    6.03    6.88   10.73    6.03    6.88\n",
      "   59  25341 200I 168S N   O    1514 1284   14.46    8.25    6.22    5.60    8.25    6.22    5.60\n",
      "   60  25832 225V 252F N   O    1704 1918   13.23   10.90    2.32    5.28   10.90    2.32    5.28\n",
      "   61  25860 226V 252F N   O    1711 1918   11.69    9.59    2.11    4.86    9.59    2.11    4.86\n",
      "   62  25892 228L 182L N   O    1727 1385   14.41   10.86    3.56    8.42   10.86    3.56    8.42\n",
      "   63  25909 228L 231V N   O    1727 1752   10.54    8.46    2.08    5.13    8.46    2.08    5.13\n",
      "   64  25918 229G 129C N   O    1735  990   12.31    9.03    3.28    4.97    9.03    3.28    4.97\n",
      "   65  25919 229G 140G N   O    1735 1078   19.22   10.74    8.48   12.31   10.74    8.48   12.31\n",
      "   66  25936 229G 606M N   O    1735 4713   29.47   10.86   18.61   35.09   10.86   18.61   35.09\n",
      "   67  25937 229G 728K N   O    1735 5655   18.18   10.98    7.20   12.45   10.98    7.20   12.45\n",
      "   68  25940 230T 188R N   O    1739 1429   13.18   10.35    2.83    5.11   10.35    2.83    5.11\n",
      "   69  25954 230T 606M N   O    1739 4713   26.96   10.05   16.91   37.24   10.05   16.91   37.24\n",
      "   70  25955 230T 728K N   O    1739 5655   19.90   10.90    9.00   17.73   10.90    9.00   17.73\n",
      "   71  25958 231V 191N N   O    1746 1453   10.82    8.88    1.94    4.59    8.88    1.94    4.59\n",
      "   72  25973 231V 606M N   O    1746 4713   24.23    7.57   16.66   40.70    7.57   16.66   40.70\n",
      "   73  25974 231V 728K N   O    1746 5655   22.52    9.06   13.47   28.89    9.06   13.47   28.89\n",
      "   74  25988 232T 262D N   O    1753 1999   11.73    7.25    4.48    8.08    7.25    4.48    8.08\n",
      "   75  25989 232T 606M N   O    1753 4713   20.86    4.82   16.04   35.22    4.82   16.04   35.22\n",
      "   76  25990 232T 607L N   O    1753 4721   16.92    8.14    8.78   14.37    8.14    8.78   14.37\n",
      "   77  25991 232T 728K N   O    1753 5655   25.23    7.27   17.96   34.58    7.27   17.96   34.58\n",
      "   78  25999 233P 262D N   O    1760 1999   14.37    6.83    7.54   13.58    6.83    7.54   13.58\n",
      "   79  26000 233P 606M N   O    1760 4713   17.64    3.34   14.30   31.88    3.34   14.30   31.88\n",
      "   80  26001 233P 607L N   O    1760 4721   13.70    5.27    8.43   15.02    5.27    8.43   15.02\n",
      "   81  26002 233P 728K N   O    1760 5655   27.81    6.42   21.39   41.45    6.42   21.39   41.45\n",
      "   82  26009 234S 262D N   O    1767 1999   13.18    9.20    3.98    5.48    9.20    3.98    5.48\n",
      "   83  26010 234S 606M N   O    1767 4713   17.28    5.81   11.47   20.54    5.81   11.47   20.54\n",
      "   84  26011 234S 607L N   O    1767 4721   13.06    7.38    5.68    7.59    7.38    5.68    7.59\n",
      "   85  26012 234S 728K N   O    1767 5655   28.75    9.21   19.54   29.32    9.21   19.54   29.32\n",
      "   86  26020 235G 247V N   O    1773 1876   13.68   10.30    3.38    7.16   10.30    3.38    7.16\n",
      "   87  26026 235G 606M N   O    1773 4713   19.45    6.83   12.61   27.35    6.83   12.61   27.35\n",
      "   88  26027 235G 607L N   O    1773 4721   15.17    9.30    5.87    9.75    9.30    5.87    9.75\n",
      "   89  26028 235G 728K N   O    1773 5655   28.04   10.50   17.54   32.70   10.50   17.54   32.70\n",
      "   90  26039 236A 238F N   O    1777 1800   10.37    8.28    2.08    5.39    8.28    2.08    5.39\n",
      "   91  26040 236A 247V N   O    1777 1876   10.37    7.56    2.81    7.14    7.56    2.81    7.14\n",
      "   92  26048 236A 606M N   O    1777 4713   22.81    8.78   14.03   34.27    8.78   14.03   34.27\n",
      "   93  26064 237D 246E N   O    1782 1869   12.87    9.79    3.07    5.76    9.79    3.07    5.76\n",
      "   94  26081 238F 230T N   O    1790 1745    9.69    7.72    1.96    4.51    7.72    1.96    4.51\n",
      "   95  26083 238F 232T N   O    1790 1759   13.39   10.92    2.48    4.93   10.92    2.48    4.93\n",
      "   96  26084 238F 234S N   O    1790 1772   12.65    9.95    2.70    4.56    9.95    2.70    4.56\n",
      "   97  26089 238F 246E N   O    1790 1869    9.53    7.22    2.31    5.21    7.22    2.31    5.21\n",
      "   98  26106 239Y 230T N   O    1801 1745   13.41   10.87    2.54    5.25   10.87    2.54    5.25\n",
      "   99  26210 245D  47G N   O    1853  373   18.20    9.49    8.70    9.99    9.49    8.70    9.99\n",
      "  100  26225 246E  47G N   O    1861  373   20.80    8.04   12.77   19.92    8.04   12.77   19.92\n",
      "  101  26226 246E  48T N   O    1861  380   16.18    8.02    8.16   15.81    8.02    8.16   15.81\n",
      "  102  26233 246E 236A N   O    1861 1781   12.97    9.53    3.44    7.53    9.53    3.44    7.53\n",
      "  103  26234 246E 237D N   O    1861 1789   11.78    8.66    3.12    5.96    8.66    3.12    5.96\n",
      "  104  26243 247V  47G N   O    1870  373   23.94    9.09   14.85   24.23    9.09   14.85   24.23\n",
      "  105  26244 247V  48T N   O    1870  380   19.28    8.35   10.93   22.81    8.35   10.93   22.81\n",
      "  106  26253 247V 236A N   O    1870 1781   10.09    7.22    2.88    7.34    7.22    2.88    7.34\n",
      "  107  26264 248S  47G N   O    1877  373   27.20   10.21   16.99   20.29   10.21   16.99   20.29\n",
      "  108  26265 248S  48T N   O    1877  380   22.60    9.22   13.38   21.17    9.22   13.38   21.17\n",
      "  109  26276 248S 234S N   O    1877 1772   15.05   10.72    4.33    5.79   10.72    4.33    5.79\n",
      "  110  26379 253A 606M N   O    1919 4713   19.46    9.45   10.01   15.53    9.45   10.01   15.53\n",
      "  111  26395 254N 606M N   O    1924 4713   20.15   10.87    9.29   11.87   10.87    9.29   11.87\n",
      "  112  26446 256M 606M N   O    1942 4713   23.79   10.48   13.31   21.96   10.48   13.31   21.96\n",
      "  113  26461 257Q 261I N   O    1950 1991    8.65    3.50    5.15    8.71    3.50    5.15    8.71\n",
      "  114  26462 257Q 262D N   O    1950 1999   13.72    8.00    5.73    4.71    8.00    5.73    4.71\n",
      "  115  26463 257Q 606M N   O    1950 4713   22.61    9.90   12.71   11.08    9.90   12.71   11.08\n",
      "  116  26468 258H  96N N   O    1959  750   22.74   10.98   11.76    6.71   10.98   11.76    6.71\n",
      "  117  26469 258H 103P N   O    1959  798   17.63    8.91    8.73    6.29    8.91    8.73    6.29\n",
      "  118  26483 258H 261I N   O    1959 1991   10.31    5.68    4.63    5.04    5.68    4.63    5.04\n",
      "  119  26489 259F  96N N   O    1969  750   20.55    9.14   11.41    5.84    9.14   11.41    5.84\n",
      "  120  26490 259F 103P N   O    1969  798   15.32    6.79    8.54    5.32    6.79    8.54    5.32\n",
      "  121  26539 261I 256M N   O    1984 1949    5.94    3.07    2.87    7.05    3.07    2.87    7.05\n",
      "  122  26540 261I 257Q N   O    1984 1958    8.54    4.22    4.32    6.99    4.22    4.32    6.99\n",
      "  123  26543 261I 606M N   O    1984 4713   26.52    9.30   17.22    7.11    9.30   17.22    7.11\n",
      "  124  26544 261I 728K N   O    1984 5655   26.15    9.61   16.54    6.60    9.61   16.54    6.60\n",
      "  125  26550 262D 231V N   O    1992 1752   12.08    8.91    3.17    6.30    8.91    3.17    6.30\n",
      "  126  26551 262D 232T N   O    1992 1759   11.87    8.79    3.08    5.31    8.79    3.08    5.31\n",
      "  127  26555 262D 253A N   O    1992 1923   11.37    6.80    4.57    6.75    6.80    4.57    6.75\n",
      "  128  26558 262D 256M N   O    1992 1949    9.95    5.39    4.56    7.61    5.39    4.56    7.61\n",
      "  129  26559 262D 257Q N   O    1992 1958   12.72    6.62    6.09    5.25    6.62    6.09    5.25\n",
      "  130  26563 262D 606M N   O    1992 4713   27.42    6.20   21.22    7.52    6.20   21.22    7.52\n",
      "  131  26564 262D 607L N   O    1992 4721   23.42    8.01   15.41    4.95    8.01   15.41    4.95\n",
      "  132  26565 262D 728K N   O    1992 5655   26.70    6.56   20.14    6.94    6.56   20.14    6.94\n",
      "  133  26566 606M 118S N   O    4706  906   41.54    9.51   32.03   11.58    9.51   32.03   11.58\n",
      "  134  26567 606M 122P N   O    4706  937   41.29   10.15   31.14   10.60   10.15   31.14   10.60\n",
      "  135  26568 606M 123P N   O    4706  944   42.04   10.43   31.60   10.25   10.43   31.60   10.25\n",
      "  136  26569 606M 230T N   O    4706 1745   32.96    9.71   23.25   51.56    9.71   23.25   51.56\n",
      "  137  26570 606M 231V N   O    4706 1752   31.66    7.12   24.53   60.49    7.12   24.53   60.49\n",
      "  138  26571 606M 232T N   O    4706 1759   27.99    5.78   22.21   47.53    5.78   22.21   47.53\n",
      "  139  26572 606M 233P N   O    4706 1766   25.17    7.57   17.60   35.41    7.57   17.60   35.41\n",
      "  140  26573 606M 234S N   O    4706 1772   26.89   10.52   16.37   25.74   10.52   16.37   25.74\n",
      "  141  26574 606M 235G N   O    4706 1776   30.93    7.86   23.08   49.04    7.86   23.08   49.04\n",
      "  142  26575 606M 251H N   O    4706 1907   30.01   10.34   19.68   38.06   10.34   19.68   38.06\n",
      "  143  26576 606M 252F N   O    4706 1918   29.66    7.37   22.29   51.24    7.37   22.29   51.24\n",
      "  144  26577 606M 253A N   O    4706 1923   28.06    6.24   21.82   39.17    6.24   21.82   39.17\n",
      "  145  26578 606M 254N N   O    4706 1931   31.09    9.32   21.76   29.47    9.32   21.76   29.47\n",
      "  146  26579 606M 255H N   O    4706 1941   34.44    9.46   24.98   45.20    9.46   24.98   45.20\n",
      "  147  26580 606M 256M N   O    4706 1949   32.70    6.87   25.83   49.20    6.87   25.83   49.20\n",
      "  148  26581 606M 257Q N   O    4706 1958   32.98    8.68   24.30   21.90    8.68   24.30   21.90\n",
      "  149  26582 606M 259F N   O    4706 1979   36.40    9.56   26.84   16.33    9.56   26.84   16.33\n",
      "  150  26583 606M 260G N   O    4706 1983   34.34    7.12   27.22   12.81    7.12   27.22   12.81\n",
      "  151  26584 606M 261I N   O    4706 1991   34.60    3.73   30.87   13.77    3.73   30.87   13.77\n",
      "  152  26585 606M 262D N   O    4706 1999   38.16    2.24   35.92   13.54    2.24   35.92   13.54\n",
      "  153  26586 606M 728K N   O    4706 5655   54.57    5.74   48.82   17.82    5.74   48.82   17.82\n",
      "  154  26587 607L 230T N   O    4714 1745   22.75   10.06   12.69   21.85   10.06   12.69   21.85\n",
      "  155  26588 607L 231V N   O    4714 1752   21.63    7.47   14.16   27.27    7.47   14.16   27.27\n",
      "  156  26589 607L 232T N   O    4714 1759   17.80    5.05   12.75   23.31    5.05   12.75   23.31\n",
      "  157  26590 607L 233P N   O    4714 1766   14.93    5.75    9.18   16.07    5.75    9.18   16.07\n",
      "  158  26591 607L 234S N   O    4714 1772   16.65    9.53    7.13    8.86    9.53    7.13    8.86\n",
      "  159  26592 607L 235G N   O    4714 1776   20.65    7.92   12.73   21.98    7.92   12.73   21.98\n",
      "  160  26593 607L 252F N   O    4714 1918   19.58    8.95   10.63   16.82    8.95   10.63   16.82\n",
      "  161  26594 607L 253A N   O    4714 1923   18.14    7.87   10.27   13.21    7.87   10.27   13.21\n",
      "  162  26595 607L 256M N   O    4714 1949   22.68    9.47   13.21   17.01    9.47   13.21   17.01\n",
      "  163  26596 607L 257Q N   O    4714 1958   23.29   10.98   12.31    8.71   10.98   12.31    8.71\n",
      "  164  26597 607L 260G N   O    4714 1983   24.05    9.44   14.61    5.99    9.44   14.61    5.99\n",
      "  165  26598 607L 261I N   O    4714 1991   24.36    6.26   18.09    7.19    6.26   18.09    7.19\n",
      "  166  26599 607L 262D N   O    4714 1999   28.00    3.40   24.60    8.57    3.40   24.60    8.57\n",
      "  167  26600 607L 728K N   O    4714 5655   44.90    5.32   39.59   13.61    5.32   39.59   13.61\n",
      "  168  26601 728K 231V N   O    5647 1752   29.65    8.18   21.47   46.96    8.18   21.47   46.96\n",
      "  169  26602 728K 232T N   O    5647 1759   33.67    6.93   26.74   51.98    6.93   26.74   51.98\n",
      "  170  26603 728K 233P N   O    5647 1766   36.57    7.68   28.89   54.16    7.68   28.89   54.16\n",
      "  171  26604 728K 235G N   O    5647 1776   32.01    9.89   22.12   41.74    9.89   22.12   41.74\n",
      "  172  26605 728K 253A N   O    5647 1923   38.16   10.01   28.15   39.11   10.01   28.15   39.11\n",
      "  173  26606 728K 256M N   O    5647 1949   33.19   10.79   22.40   32.68   10.79   22.40   32.68\n",
      "  174  26607 728K 260G N   O    5647 1983   31.16    9.33   21.83    9.59    9.33   21.83    9.59\n",
      "  175  26608 728K 261I N   O    5647 1991   32.38    7.36   25.02   10.29    7.36   25.02   10.29\n",
      "  176  26609 728K 262D N   O    5647 1999   30.22    3.27   26.95    9.79    3.27   26.95    9.79\n",
      "  177  26610 728K 606M N   O    5647 4713   51.03    3.30   47.73   18.03    3.30   47.73   18.03\n",
      "  178  26611 728K 607L N   O    5647 4721   47.14    2.29   44.85   16.29    2.29   44.85   16.29\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20613 168S 168S CA  C    1280 1283 -154.35 -180.00   25.65    5.13 -180.00   25.65    5.13\n",
      "    2  20708 263I 263I CA  C    2001 2006 -154.41 -180.00   25.59    5.12 -180.00   25.59    5.12\n",
      "    3  21158 713P 713P CA  C    5519 5523 -155.22 -180.00   24.78    4.95 -180.00   24.78    4.95\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19720   1M   2K C   N       7    9 -143.59 -118.00   25.71    0.93  -62.90 -164.88   30.17\n",
      "    1          2K   2K N   CA      9   10  141.55  139.10                  -40.80\n",
      "    2  19722   3P   4P C   N      23   25  -58.55  -58.70   53.85    4.44  -64.50  128.59    9.91\n",
      "    2          4P   4P N   CA     25   26  -84.35  -30.50                  147.20\n",
      "    3  19723   4P   5F C   N      30   32  109.41 -124.20  128.62    4.32  -63.20 -132.26   26.88\n",
      "    3          5F   5F N   CA     32   33  167.13  143.30                  -44.30\n",
      "    4  19724   5F   6N C   N      41   43 -171.07 -119.90   61.32    1.81  -63.20 -176.77   28.78\n",
      "    4          6N   6N N   CA     43   44  170.79  137.00                  -41.10\n",
      "    5  19726   7I   8T C   N      57   59 -154.50 -124.80   46.16    1.45  -63.20  166.36   27.10\n",
      "    5          8T   8T N   CA     59   60  178.84  143.50                  -42.10\n",
      "    6  19727   8T   9D C   N      64   66 -163.41  -96.50   91.78    3.84  -63.30  174.55   27.87\n",
      "    6          9D   9D N   CA     66   67  177.02  114.20                  -40.00\n",
      "    7  19728   9D  10E C   N      72   74 -162.37 -117.80   58.62    1.81  -63.60  175.30   28.74\n",
      "    7         10E  10E N   CA     74   75  174.87  136.80                  -40.30\n",
      "    8  19729  10E  11A C   N      81   83 -168.04 -134.00   47.84    1.43  -62.50  174.13   34.07\n",
      "    8         11A  11A N   CA     83   84 -179.39  147.00                  -40.90\n",
      "    9  19731  12C  13A C   N      92   94 -176.48 -134.00   48.46    1.12  -62.50 -172.58   36.69\n",
      "    9         13A  13A N   CA     94   95  170.33  147.00                  -40.90\n",
      "   10  19732  13A  14H C   N      97   99 -165.95 -125.60   52.69    1.30  -63.20  177.74   28.73\n",
      "   10         14H  14H N   CA     99  100  172.67  138.80                  -42.30\n",
      "   11  19734  15C  16W C   N     113  115 -169.90 -124.90   53.98    1.73  -63.00  178.22   29.40\n",
      "   11         16W  16W N   CA    115  116  173.20  143.40                  -44.20\n",
      "   12  19735  16W  17D C   N     127  129  177.75  -96.50  104.73    4.36  -63.30 -171.94   30.62\n",
      "   12         17D  17D N   CA    129  130  174.33  114.20                  -40.00\n",
      "   13  19736  17D  18E C   N     135  137 -178.81 -117.80   69.36    2.03  -63.60 -170.94   31.37\n",
      "   13         18E  18E N   CA    137  138  169.79  136.80                  -40.30\n",
      "   14  19737  18E  19S C   N     144  146  173.09 -136.60   54.34    1.71  -64.10 -163.61   23.08\n",
      "   14         19S  19S N   CA    146  147  171.74  151.20                  -35.00\n",
      "   15  19739  20V  21M C   N     157  159 -179.87 -125.60   64.66    1.91  -63.40 -174.92   36.27\n",
      "   15         21M  21M N   CA    159  160  175.66  140.50                  -40.50\n",
      "   16  19740  21M  22L C   N     165  167  177.65 -108.50   81.92    3.62  -63.50 -167.97   33.81\n",
      "   16         22L  22L N   CA    167  168  167.97  132.50                  -41.20\n",
      "   17  19743  24P  25N C   N     187  189  178.14 -119.90   71.90    2.16  -63.20 -172.29   29.82\n",
      "   17         25N  25N N   CA    189  190  173.46  137.00                  -41.10\n",
      "   18  19744  25N  26H C   N     195  197  176.24 -125.60   66.33    1.71  -63.20 -169.87   31.28\n",
      "   18         26H  26H N   CA    197  198  170.68  138.80                  -42.30\n",
      "   19  19745  26H  27R C   N     205  207  166.90 -125.20   75.06    2.30  -63.00 -164.21   34.71\n",
      "   19         27R  27R N   CA    207  208  172.58  140.60                  -41.10\n",
      "   20  19746  27R  28S C   N     216  218  177.40 -136.60   51.87    1.59  -64.10 -168.96   22.39\n",
      "   20         28S  28S N   CA    218  219  175.15  151.20                  -35.00\n",
      "   21  19747  28S  29A C   N     222  224 -179.16 -134.00   53.94    1.30  -62.50 -175.75   36.27\n",
      "   21         29A  29A N   CA    224  225  176.49  147.00                  -40.90\n",
      "   22  19749  30V  31H C   N     234  236 -178.04 -125.60   65.68    1.62  -63.20 -179.42   29.73\n",
      "   22         31H  31H N   CA    236  237  178.35  138.80                  -42.30\n",
      "   23  19750  31H  32K C   N     244  246  179.37 -118.00   70.29    2.33  -62.90 -170.74   31.46\n",
      "   23         32K  32K N   CA    246  247  171.01  139.10                  -40.80\n",
      "   24  19751  32K  33E C   N     253  255 -172.14 -117.80   67.10    2.00  -63.60  179.96   29.81\n",
      "   24         33E  33E N   CA    255  256  176.16  136.80                  -40.30\n",
      "   25  19752  33E  34E C   N     262  264 -171.10 -117.80   65.86    1.97  -63.60  179.88   29.76\n",
      "   25         34E  34E N   CA    264  265  175.47  136.80                  -40.30\n",
      "   26  19754  35C  36A C   N     277  279  175.94 -134.00   55.04    1.25  -62.50 -167.50   37.88\n",
      "   26         36A  36A N   CA    279  280  169.85  147.00                  -40.90\n",
      "   27  19755  36A  37Y C   N     282  284 -163.17 -124.30   61.19    2.54  -63.50  166.96   31.05\n",
      "   27         37Y  37Y N   CA    284  285 -177.34  135.40                  -43.40\n",
      "   28  19758  39C  40R C   N     306  308 -172.72 -125.20   57.49    1.70  -63.00 -177.42   31.82\n",
      "   28         40R  40R N   CA    308  309  172.96  140.60                  -41.10\n",
      "   29  19759  40R  41T C   N     317  319  178.38 -124.80   65.64    2.50  -63.20 -175.46   30.87\n",
      "   29         41T  41T N   CA    319  320  176.37  143.50                  -42.10\n",
      "   30  19761  42C  43R C   N     330  332  176.72 -125.20   67.41    2.01  -63.00 -172.31   33.08\n",
      "   30         43R  43R N   CA    332  333  174.81  140.60                  -41.10\n",
      "   31  19762  43R  44H C   N     341  343  177.06 -125.60   66.61    1.70  -63.20 -171.93   30.97\n",
      "   31         44H  44H N   CA    343  344  172.68  138.80                  -42.30\n",
      "   32  19763  44H  45E C   N     351  353  172.78 -117.80   77.51    2.27  -63.60 -166.84   32.29\n",
      "   32         45E  45E N   CA    353  354  171.28  136.80                  -40.30\n",
      "   33  19764  45E  46M C   N     360  362  171.37 -125.60   70.64    2.14  -63.40 -166.80   38.05\n",
      "   33         46M  46M N   CA    362  363  172.39  140.50                  -40.50\n",
      "   34  19765  46M  47G C   N     368  370  163.94 -167.20   31.15    0.90  -80.20  116.40    8.06\n",
      "   34         47G  47G N   CA    370  371  162.87  174.60                  174.10\n",
      "   35  19767  48T  49L C   N     379  381 -167.00 -108.50   60.02    2.72  -63.50 -158.53   34.33\n",
      "   35         49L  49L N   CA    381  382  145.95  132.50                  -41.20\n",
      "   36  19770  51C  52M C   N     400  402  -20.66  -73.00   80.64    5.06  -63.40  129.41   22.72\n",
      "   36         52M  52M N   CA    402  403   81.65  143.00                  -40.50\n",
      "   37  19773  54C  55H C   N     420  422 -156.59 -125.60   35.15    0.91  -63.20 -172.73   29.44\n",
      "   37         55H  55H N   CA    422  423  155.38  138.80                  -42.30\n",
      "   38  19774  55H  56M C   N     430  432 -179.32 -125.60   59.99    1.82  -63.40 -168.60   37.25\n",
      "   38         56M  56M N   CA    432  433  167.19  140.50                  -40.50\n",
      "   39  19775  56M  57G C   N     438  440 -164.34 -167.20   11.25    0.40   82.20 -167.74   14.18\n",
      "   39         57G  57G N   CA    440  441  163.72  174.60                    8.50\n",
      "   40  19776  57G  58L C   N     442  444 -139.54 -108.50   60.70    2.88  -63.50  154.19   26.09\n",
      "   40         58L  58L N   CA    444  445 -175.34  132.50                  -41.20\n",
      "   41  19787  68L  69N C   N     532  534 -177.47 -119.90   65.61    2.00  -63.20 -171.08   29.80\n",
      "   41         69N  69N N   CA    534  535  168.46  137.00                  -41.10\n",
      "   42  19790  71P  72T C   N     553  555 -123.30 -124.80   28.16    1.25  -63.20  158.15   24.24\n",
      "   42         72T  72T N   CA    555  556  171.62  143.50                  -42.10\n",
      "   43  19791  72T  73H C   N     560  562 -125.28 -125.60   99.10    3.77  -63.20  101.11   16.53\n",
      "   43         73H  73H N   CA    562  563 -122.11  138.80                  -42.30\n",
      "   44  19792  73H  74V C   N     570  572  -60.88  -62.40    5.28    0.60 -125.40 -178.88   10.19\n",
      "   44         74V  74V N   CA    572  573  -47.46  -42.40                  143.30\n",
      "   45  19793  74V  75M C   N     577  579 -171.62 -125.60   53.17    2.71  -63.40 -171.47   23.36\n",
      "   45         75M  75M N   CA    579  580  113.88  140.50                  -40.50\n",
      "   46  19794  75M  76Y C   N     585  587 -147.78 -124.30   68.06    2.78  -63.50  144.46   26.80\n",
      "   46         76Y  76Y N   CA    587  588 -160.72  135.40                  -43.40\n",
      "   47  19796  77V  78W C   N     604  606 -149.61 -124.90   64.31    2.49  -63.00  142.39   23.56\n",
      "   47         78W  78W N   CA    606  607 -157.22  143.40                  -44.20\n",
      "   48  19800  81E  82L C   N     644  646  159.02 -108.50   92.50    4.42  -70.70  130.73   12.21\n",
      "   48         82L  82L N   CA    646  647  130.88  132.50                  141.60\n",
      "   49  19802  83P  84A C   N     659  661  -81.54  -68.20   70.18    5.18  -62.50  106.61   18.46\n",
      "   49         84A  84A N   CA    661  662 -145.80  145.30                  -40.90\n",
      "   50  19804  85K  86D C   N     673  675  125.50  -63.30  171.90   27.94  -70.90 -135.13    9.35\n",
      "   50         86D  86D N   CA    675  676  -55.42  -40.00                  150.30\n",
      "   51  19805  86D  87E C   N     681  683 -159.12 -117.80   46.10    1.35  -63.60 -171.55   30.39\n",
      "   51         87E  87E N   CA    683  684  157.25  136.80                  -40.30\n",
      "   52  19806  87E  88D C   N     690  692 -118.25  -70.90   60.89    2.74  -63.30  142.44   21.13\n",
      "   52         88D  88D N   CA    692  693 -171.41  150.30                  -40.00\n",
      "   53  19807  88D  89A C   N     698  700 -167.07 -134.00   43.73    1.21  -62.50  177.56   34.60\n",
      "   53         89A  89A N   CA    700  701  175.60  147.00                  -40.90\n",
      "   54  19808  89A  90G C   N     703  705 -128.10 -167.20   45.14    0.74   82.20 -152.59   15.96\n",
      "   54         90G  90G N   CA    705  706  152.05  174.60                    8.50\n",
      "   55  19811  92S  93K C   N     717  719 -135.23 -118.00   17.65    0.61  -62.90 -169.49   29.07\n",
      "   55         93K  93K N   CA    719  720  142.95  139.10                  -40.80\n",
      "   56  19812  93K  94D C   N     726  728 -160.04  -96.50   70.39    2.91  -63.30 -159.58   30.95\n",
      "   56         94D  94D N   CA    728  729  144.47  114.20                  -40.00\n",
      "   57  19814  95V  96N C   N     741  743  -51.83  -63.20   19.42    3.05   55.90  125.74   15.48\n",
      "   57         96N  96N N   CA    743  744  -25.36  -41.10                   39.50\n",
      "   58  19815  96N  97K C   N     749  751 -145.79 -118.00   43.04    1.57  -62.90  168.96   26.89\n",
      "   58         97K  97K N   CA    751  752  171.97  139.10                  -40.80\n",
      "   59  19816  97K  98L C   N     758  760  -89.04 -108.50   85.99    4.42  -63.50   93.50   11.79\n",
      "   59         98L  98L N   CA    760  761   48.74  132.50                  -41.20\n",
      "   60  19819 100V 101L C   N     777  779 -130.48 -108.50   71.56    3.60  -63.50  135.86   22.99\n",
      "   60        101L 101L N   CA    779  780 -159.40  132.50                  -41.20\n",
      "   61  19820 101L 102A C   N     785  787  101.54 -134.00  135.30    3.08  -62.50 -157.32   32.59\n",
      "   61        102A 102A N   CA    787  788 -159.93  147.00                  -40.90\n",
      "   62  19821 102A 103P C   N     790  792  -51.60  -58.70    7.23    0.99  -64.50  176.80   12.53\n",
      "   62        103P 103P N   CA    792  793  -29.13  -30.50                  147.20\n",
      "   63  19822 103P 104K C   N     797  799  111.98   56.60  115.44   12.46  -62.90 -109.53   30.32\n",
      "   63        104K 104K N   CA    799  800  139.89   38.60                  -40.80\n",
      "   64  19823 104K 105E C   N     806  808  152.28 -117.80   90.66    3.05  -63.60 -136.14   37.45\n",
      "   64        105E 105E N   CA    808  809  148.41  136.80                  -40.30\n",
      "   65  19824 105E 106Y C   N     815  817  147.05 -124.30   88.70    6.58  -63.50 -127.36   43.56\n",
      "   65        106Y 106Y N   CA    817  818  138.31  135.40                  -43.40\n",
      "   66  19825 106Y 107E C   N     827  829 -119.33 -117.80   39.74    1.97  -63.60  153.65   23.63\n",
      "   66        107E 107E N   CA    829  830  176.51  136.80                  -40.30\n",
      "   67  19826 107E 108N C   N     836  838  -52.68  -71.20   21.02    1.25  -63.20  174.26   22.37\n",
      "   67        108N 108N N   CA    838  839  132.85  142.80                  -41.10\n",
      "   68  19827 108N 109A C   N     844  846   18.16  -68.20  100.77    6.74  -62.50  156.63   29.96\n",
      "   68        109A 109A N   CA    846  847   93.37  145.30                  -40.90\n",
      "   69  19828 109A 110L C   N     849  851 -165.05 -108.50   92.52    4.27  -63.50  151.98   27.12\n",
      "   69        110L 110L N   CA    851  852 -154.28  132.50                  -41.20\n",
      "   70  19831 112C 113A C   N     869  871 -158.68 -134.00   38.85    1.31  -62.50  171.59   33.21\n",
      "   70        113A 113A N   CA    871  872  177.00  147.00                  -40.90\n",
      "   71  19832 113A 114A C   N     874  876 -154.78 -134.00   32.57    1.09  -62.50  173.58   33.35\n",
      "   71        114A 114A N   CA    876  877  172.08  147.00                  -40.90\n",
      "   72  19834 115C 116A C   N     885  887 -177.13 -134.00   77.19    2.89  -68.20  127.22    8.50\n",
      "   72        116A 116A N   CA    887  888 -148.98  147.00                  145.30\n",
      "   73  19836 117K 118S C   N     899  901   53.72  -72.40  148.53   11.95  -72.40  148.53   11.95\n",
      "   73        118S 118S N   CA    901  902 -129.15  152.40                  152.40\n",
      "   74  19837 118S 119F C   N     905  907 -126.59 -124.20   36.72    1.67  -63.20  149.83   24.91\n",
      "   74        119F 119F N   CA    907  908  179.94  143.30                  -44.30\n",
      "   75  19838 119F 120V C   N     916  918  -61.00  -62.40    7.94    1.09 -125.40 -170.82    7.53\n",
      "   75        120V 120V N   CA    918  919  -34.58  -42.40                  143.30\n",
      "   76  19839 120V 121S C   N     923  925 -111.00 -136.60   64.95    2.66  -64.10  134.92    8.32\n",
      "   76        121S 121S N   CA    925  926   91.50  151.20                  -35.00\n",
      "   77  19840 121S 122P C   N     929  931  -46.90  -58.70   44.87    2.87  -64.50  140.12   11.41\n",
      "   77        122P 122P N   CA    931  932  -73.79  -30.50                  147.20\n",
      "   78  19842 123P 124E C   N     943  945  169.55 -117.80   77.84    2.32  -63.60 -159.74   33.42\n",
      "   78        124E 124E N   CA    945  946  164.74  136.80                  -40.30\n",
      "   79  19843 124E 125L C   N     952  954  -74.91  -70.70   45.23    3.27  -63.50  132.66   19.09\n",
      "   79        125L 125L N   CA    954  955 -173.37  141.60                  -41.20\n",
      "   80  19844 125L 126T C   N     960  962  -59.91  -63.20   83.61   10.51  -63.20   83.61   10.51\n",
      "   80        126T 126T N   CA    962  963 -125.65  -42.10                  -42.10\n",
      "   81  19846 127I 128D C   N     975  977   44.54  -70.90  124.68   10.42  -63.30  163.27   18.02\n",
      "   81        128D 128D N   CA    977  978 -162.59  150.30                  -40.00\n",
      "   82  19847 128D 129C C   N     983  985  -49.77  -69.10  162.11   10.71 -117.90  175.42    7.68\n",
      "   82        129C 129C N   CA    985  986  -57.24  141.80                  141.10\n",
      "   83  19848 129C 130Y C   N     989  991 -167.55 -124.30   68.11    2.83  -63.50  165.41   30.91\n",
      "   83        130Y 130Y N   CA    991  992 -171.99  135.40                  -43.40\n",
      "   84  19849 130Y 131Q C   N    1001 1003 -132.05 -121.10   47.94    2.08  -63.80  149.78   25.57\n",
      "   84        131Q 131Q N   CA   1003 1004 -173.62  139.70                  -40.30\n",
      "   85  19850 131Q 132W C   N    1010 1012 -134.22 -124.90   16.26    0.55  -63.00  174.30   26.58\n",
      "   85        132W 132W N   CA   1012 1013  156.72  143.40                  -44.20\n",
      "   86  19853 134I 135N C   N    1040 1042   52.40   55.90   37.37    2.36  -63.20  165.05   26.64\n",
      "   86        135N 135N N   CA   1042 1043   76.70   39.50                  -41.10\n",
      "   87  19854 135N 136A C   N    1048 1050    8.54  -68.20  103.59    6.57  -62.50  136.54   26.16\n",
      "   87        136A 136A N   CA   1050 1051   75.71  145.30                  -40.90\n",
      "   88  19855 136A 137T C   N    1053 1055 -172.68 -124.80   48.13    2.66  -63.20 -149.85   33.92\n",
      "   88        137T 137T N   CA   1055 1056  138.52  143.50                  -42.10\n",
      "   89  19856 137T 138S C   N    1060 1062 -164.79 -136.60   38.60    1.23  -64.10  178.53   20.20\n",
      "   89        138S 138S N   CA   1062 1063  177.57  151.20                  -35.00\n",
      "   90  19857 138S 139T C   N    1066 1068 -161.36 -124.80   37.29    1.78  -63.20 -166.22   31.13\n",
      "   90        139T 139T N   CA   1068 1069  150.82  143.50                  -42.10\n",
      "   91  19859 140G 141A C   N    1077 1079  153.82 -134.00   79.53    1.81  -62.50 -160.28   39.98\n",
      "   91        141A 141A N   CA   1079 1080 -179.62  147.00                  -40.90\n",
      "   92  19860 141A 142Q C   N    1082 1084  162.81 -121.10   76.36    3.00  -63.80 -141.02   28.11\n",
      "   92        142Q 142Q N   CA   1084 1085  133.35  139.70                  -40.30\n",
      "   93  19861 142Q 143A C   N    1091 1093 -134.11 -134.00   21.02    1.19  -62.50  167.19   31.24\n",
      "   93        143A 143A N   CA   1093 1094  168.02  147.00                  -40.90\n",
      "   94  19862 143A 144A C   N    1096 1098 -158.57 -134.00   46.54    1.81  -62.50  163.73   31.89\n",
      "   94        144A 144A N   CA   1098 1099 -173.48  147.00                  -40.90\n",
      "   95  19864 145I 146E C   N    1109 1111  148.06 -117.80  102.60    3.03  -63.60 -154.58   34.91\n",
      "   95        146E 146E N   CA   1111 1112  177.60  136.80                  -40.30\n",
      "   96  19865 146E 147P C   N    1118 1120  -61.16  -58.70    6.42    0.73  -64.50  176.40   13.34\n",
      "   96        147P 147P N   CA   1120 1121  -36.43  -30.50                  147.20\n",
      "   97  19866 147P 148E C   N    1125 1127 -162.71 -117.80   47.95    1.43  -63.60 -166.59   31.25\n",
      "   97        148E 148E N   CA   1127 1128  153.61  136.80                  -40.30\n",
      "   98  19867 148E 149G C   N    1134 1136  168.05 -167.20   26.71    0.78   82.20  178.11   12.50\n",
      "   98        149G 149G N   CA   1136 1137  164.56  174.60                    8.50\n",
      "   99  19869 150V 151A C   N    1145 1147 -176.11 -134.00   68.97    2.41  -62.50  163.44   32.58\n",
      "   99        151A 151A N   CA   1147 1148 -158.39  147.00                  -40.90\n",
      "  100  19870 151A 152S C   N    1150 1152 -161.37 -136.60   36.70    1.21  -64.10  176.03   19.78\n",
      "  100        152S 152S N   CA   1152 1153  178.28  151.20                  -35.00\n",
      "  101  19871 152S 153M C   N    1156 1158 -161.01 -125.60   61.66    2.06  -63.40  161.39   31.40\n",
      "  101        153M 153M N   CA   1158 1159 -169.02  140.50                  -40.50\n",
      "  102  19872 153M 154R C   N    1164 1166  142.61 -125.20   93.22    4.10  -63.00 -131.94   26.61\n",
      "  102        154R 154R N   CA   1166 1167  126.76  140.60                  -41.10\n",
      "  103  19873 154R 155L C   N    1175 1177 -170.31 -108.50  133.93    6.45  -70.70  148.19    9.35\n",
      "  103        155L 155L N   CA   1177 1178 -108.69  132.50                  141.60\n",
      "  104  19874 155L 156Q C   N    1183 1185  -49.67  -73.00  168.99   11.53 -121.10 -177.10    9.50\n",
      "  104        156Q 156Q N   CA   1185 1186  -51.92  140.70                  139.70\n",
      "  105  19877 158P 159H C   N    1205 1207 -135.45 -125.60   22.20    0.64  -63.20  174.66   26.55\n",
      "  105        159H 159H N   CA   1207 1208  158.69  138.80                  -42.30\n",
      "  106  19878 159H 160L C   N    1215 1217 -164.13 -108.50   58.97    2.63  -63.50 -165.27   33.22\n",
      "  106        160L 160L N   CA   1217 1218  152.08  132.50                  -41.20\n",
      "  107  19881 162C 163L C   N    1236 1238  -61.73  -70.70   19.49    1.22  -63.50  165.51   23.08\n",
      "  107        163L 163L N   CA   1238 1239  124.30  141.60                  -41.20\n",
      "  108  19882 163L 164E C   N    1244 1246   71.11  -69.30  141.41   10.11  -63.60 -146.20   35.65\n",
      "  108        164E 164E N   CA   1246 1247  125.72  142.50                  -40.30\n",
      "  109  19883 164E 165Q C   N    1253 1255  -76.13  -73.00    3.75    0.29  -63.80  179.36   25.74\n",
      "  109        165Q 165Q N   CA   1255 1256  138.63  140.70                  -40.30\n",
      "  110  19884 165Q 166L C   N    1262 1264 -156.13 -108.50   67.34    3.78  -63.50  156.47   19.40\n",
      "  110        166L 166L N   CA   1264 1265   84.90  132.50                  -41.20\n",
      "  111  19886 167P 168S C   N    1277 1279  -42.19 -136.60   99.74    3.23  -64.10  155.58   12.70\n",
      "  111        168S 168S N   CA   1279 1280  119.03  151.20                  -35.00\n",
      "  112  19888 169P 170F C   N    1290 1292 -156.36 -124.20   33.16    1.06  -63.20 -171.13   32.12\n",
      "  112        170F 170F N   CA   1292 1293  151.41  143.30                  -44.30\n",
      "  113  19889 170F 171Q C   N    1301 1303 -148.32 -121.10   28.13    0.93  -63.80 -167.56   32.70\n",
      "  113        171Q 171Q N   CA   1303 1304  146.81  139.70                  -40.30\n",
      "  114  19890 171Q 172T C   N    1310 1312 -127.68 -124.80    9.35    0.32  -63.20  177.63   27.03\n",
      "  114        172T 172T N   CA   1312 1313  152.39  143.50                  -42.10\n",
      "  115  19891 172T 173A C   N    1317 1319  165.48 -134.00   84.89    2.53  -62.50  173.49   34.96\n",
      "  115        173A 173A N   CA   1319 1320 -153.47  147.00                  -40.90\n",
      "  116  19892 173A 174P C   N    1322 1324  -64.64  -58.70   17.47    1.06  -64.50  161.27   12.04\n",
      "  116        174P 174P N   CA   1324 1325  -14.07  -30.50                  147.20\n",
      "  117  19893 174P 175T C   N    1329 1331 -128.12 -124.80   20.77    0.79  -63.20  167.04   25.69\n",
      "  117        175T 175T N   CA   1331 1332  164.00  143.50                  -42.10\n",
      "  118  19894 175T 176S C   N    1336 1338 -157.10 -136.60   31.74    1.07  -64.10  176.12   19.47\n",
      "  118        176S 176S N   CA   1338 1339  175.43  151.20                  -35.00\n",
      "  119  19895 176S 177S C   N    1342 1344 -160.61 -136.60   26.05    0.82  -64.10 -169.99   20.72\n",
      "  119        177S 177S N   CA   1344 1345  161.32  151.20                  -35.00\n",
      "  120  19896 177S 178D C   N    1348 1350 -153.85  -96.50   62.16    2.57  -63.30 -160.15   21.21\n",
      "  120        178D 178D N   CA   1350 1351  138.17  114.20                  -40.00\n",
      "  121  19900 181A 182L C   N    1376 1378 -131.11 -108.50  109.87    6.14  -70.70  131.34   12.22\n",
      "  121        182L 182L N   CA   1378 1379   24.98  132.50                  141.60\n",
      "  122  19901 182L 183D C   N    1384 1386 -102.90  -70.90   50.20    1.85  -63.30  136.89   19.45\n",
      "  122        183D 183D N   CA   1386 1387 -171.03  150.30                  -40.00\n",
      "  123  19904 185C 186E C   N    1402 1404  -71.91  -69.30    8.84    0.76  -63.60  174.55   22.83\n",
      "  123        186E 186E N   CA   1404 1405  134.05  142.50                  -40.30\n",
      "  124  19906 187C 188R C   N    1417 1419 -153.27 -125.20   28.30    1.23  -63.00 -160.37   22.68\n",
      "  124        188R 188R N   CA   1419 1420  136.96  140.60                  -41.10\n",
      "  125  19924 205E 206E C   N    1557 1559   33.12   54.60   34.92    1.90  -63.60  146.66   24.62\n",
      "  125        206E 206E N   CA   1559 1560   69.94   42.40                  -40.30\n",
      "  126  19925 206E 207A C   N    1566 1568  172.08 -134.00   87.75    3.05  -68.20  139.15    9.32\n",
      "  126        207A 207A N   CA   1568 1569 -143.77  147.00                  145.30\n",
      "  127  19926 207A 208G C   N    1571 1573  -75.39  -80.20   44.74    1.52   78.70  166.96    9.34\n",
      "  127        208G 208G N   CA   1573 1574  129.62  174.10                 -166.10\n",
      "  128  19927 208G 209G C   N    1575 1577   83.86   78.70   73.08    1.87  -80.20  172.44    9.46\n",
      "  128        209G 209G N   CA   1577 1578  121.00 -166.10                  174.10\n",
      "  129  19928 209G 210N C   N    1579 1581 -129.05  -63.20   90.31   10.18  -63.20   90.31   10.18\n",
      "  129        210N 210N N   CA   1581 1582   20.69  -41.10                  -41.10\n",
      "  130  19977 258H 259F C   N    1967 1969 -123.89  -63.20   61.84    9.23  -63.20   61.84    9.23\n",
      "  130        259F 259F N   CA   1969 1970  -32.44  -44.30                  -44.30\n",
      "  131  19979 260G 261I C   N    1982 1984    8.83  -97.30  107.68    4.59 -120.60  130.31    9.50\n",
      "  131        261I 261I N   CA   1984 1985  145.40  127.20                  130.30\n",
      "  132  19980 261I 262D C   N    1990 1992  101.93  -96.50  161.74    6.60  -96.50  161.74    6.60\n",
      "  132        262D 262D N   CA   1992 1993  106.88  114.20                  114.20\n",
      "  133  19982 263I 264Q C   N    2006 2008 -147.99 -121.10   31.46    0.97  -63.80 -175.94   31.43\n",
      "  133        264Q 264Q N   CA   2008 2009  156.03  139.70                  -40.30\n",
      "  134  19983 264Q 265T C   N    2015 2017 -131.85 -124.80   50.97    1.98  -63.20  141.66   22.59\n",
      "  134        265T 265T N   CA   2017 2018 -166.02  143.50                  -42.10\n",
      "  135  19984 265T 266A C   N    2022 2024 -164.92 -134.00   38.35    0.97  -62.50 -178.85   35.10\n",
      "  135        266A 266A N   CA   2024 2025  169.68  147.00                  -40.90\n",
      "  136  19985 266A 267K C   N    2027 2029  152.12 -118.00   94.82    3.19  -62.90 -151.48   35.37\n",
      "  136        267K 267K N   CA   2029 2030  169.32  139.10                  -40.80\n",
      "  137  19986 267K 268K C   N    2036 2038  177.55 -118.00   69.82    2.32  -62.90 -165.63   32.24\n",
      "  137        268K 268K N   CA   2038 2039  165.94  139.10                  -40.80\n",
      "  138  19987 268K 269T C   N    2045 2047 -173.80 -124.80   56.31    2.16  -63.20 -176.32   30.40\n",
      "  138        269T 269T N   CA   2047 2048  171.25  143.50                  -42.10\n",
      "  139  19988 269T 270A C   N    2052 2054 -179.14 -134.00   50.64    1.15  -62.50 -170.64   37.13\n",
      "  139        270A 270A N   CA   2054 2055  169.93  147.00                  -40.90\n",
      "  140  19989 270A 271K C   N    2057 2059  178.88 -118.00   69.25    2.30  -62.90 -167.76   31.89\n",
      "  140        271K 271K N   CA   2059 2060  167.61  139.10                  -40.80\n",
      "  141  19990 271K 272T C   N    2066 2068 -172.29 -124.80   55.89    2.08  -63.20 -178.60   30.01\n",
      "  141        272T 272T N   CA   2068 2069  172.97  143.50                  -42.10\n",
      "  142  19991 272T 273L C   N    2073 2075  176.64 -108.50   81.43    3.61  -63.50 -164.67   34.34\n",
      "  142        273L 273L N   CA   2075 2076  164.56  132.50                  -41.20\n",
      "  143  19992 273L 274G C   N    2081 2083  167.55 -167.20   25.96    0.65   82.20 -178.62   12.65\n",
      "  143        274G 274G N   CA   2083 2084  168.54  174.60                    8.50\n",
      "  144  19993 274G 275E C   N    2085 2087 -177.00 -117.80   65.82    1.93  -63.60 -168.65   31.61\n",
      "  144        275E 275E N   CA   2087 2088  165.57  136.80                  -40.30\n",
      "  145  19994 275E 276M C   N    2094 2096  172.99 -125.60   67.66    2.07  -63.40 -165.17   38.23\n",
      "  145        276M 276M N   CA   2096 2097  168.91  140.50                  -40.50\n",
      "  146  19995 276M 277Q C   N    2102 2104  175.62 -121.10   68.50    2.15  -63.80 -164.59   34.65\n",
      "  146        277Q 277Q N   CA   2104 2105  165.93  139.70                  -40.30\n",
      "  147  19996 277Q 278Y C   N    2111 2113 -170.85 -124.30   60.84    2.88  -63.50  178.02   33.14\n",
      "  147        278Y 278Y N   CA   2113 2114  174.58  135.40                  -43.40\n",
      "  148  19997 278Y 279D C   N    2123 2125 -176.52  -96.50   91.55    3.80  -63.30 -162.92   31.48\n",
      "  148        279D 279D N   CA   2125 2126  158.68  114.20                  -40.00\n",
      "  149  19998 279D 280Y C   N    2131 2133 -162.16 -124.30   56.19    2.41  -63.50  171.01   31.69\n",
      "  149        280Y 280Y N   CA   2133 2134  176.92  135.40                  -43.40\n",
      "  150  19999 280Y 281S C   N    2143 2145 -166.87 -136.60   39.92    1.25  -64.10  179.99   20.46\n",
      "  150        281S 281S N   CA   2145 2146  177.23  151.20                  -35.00\n",
      "  151  20001 282C 283Q C   N    2155 2157 -170.44 -121.10   55.82    1.72  -63.80 -172.77   32.87\n",
      "  151        283Q 283Q N   CA   2157 2158  165.81  139.70                  -40.30\n",
      "  152  20002 283Q 284F C   N    2164 2166 -176.49 -124.20   59.70    1.60  -63.20 -177.09   32.17\n",
      "  152        284F 284F N   CA   2166 2167  172.10  143.30                  -44.30\n",
      "  153  20003 284F 285D C   N    2175 2177 -163.21  -96.50   86.35    3.60  -63.30 -178.96   28.69\n",
      "  153        285D 285D N   CA   2177 2178  169.03  114.20                  -40.00\n",
      "  154  20004 285D 286F C   N    2183 2185 -168.83 -124.20   58.56    1.53  -63.20  171.01   30.07\n",
      "  154        286F 286F N   CA   2185 2186 -178.79  143.30                  -44.30\n",
      "  155  20005 286F 287K C   N    2194 2196 -131.82 -118.00   51.36    2.27  -62.90  147.70   23.30\n",
      "  155        287K 287K N   CA   2196 2197 -171.43  139.10                  -40.80\n",
      "  156  20006 287K 288R C   N    2203 2205 -148.86 -125.20   45.04    1.56  -63.00  164.22   27.92\n",
      "  156        288R 288R N   CA   2205 2206  178.92  140.60                  -41.10\n",
      "  157  20008 289I 290T C   N    2222 2224 -147.34 -124.80   39.35    1.22  -63.20  165.18   26.56\n",
      "  157        290T 290T N   CA   2224 2225  175.76  143.50                  -42.10\n",
      "  158  20009 290T 291E C   N    2229 2231 -153.81 -117.80   55.19    1.85  -63.60  167.45   27.26\n",
      "  158        291E 291E N   CA   2231 2232  178.62  136.80                  -40.30\n",
      "  159  20010 291E 292N C   N    2238 2240 -146.48 -119.90   43.48    1.33  -63.20  169.39   25.82\n",
      "  159        292N 292N N   CA   2240 2241  171.41  137.00                  -41.10\n",
      "  160  20011 292N 293G C   N    2246 2248  179.83 -167.20   16.75    0.60   82.20 -176.40   13.20\n",
      "  160        293G 293G N   CA   2248 2249  163.99  174.60                    8.50\n",
      "  161  20012 293G 294D C   N    2250 2252 -166.08  -96.50   79.87    3.31  -63.30 -164.25   30.72\n",
      "  161        294D 294D N   CA   2252 2253  153.41  114.20                  -40.00\n",
      "  162  20013 294D 295S C   N    2258 2260 -163.89 -136.60   30.15    0.93  -64.10 -170.61   20.92\n",
      "  162        295S 295S N   CA   2260 2261  164.03  151.20                  -35.00\n",
      "  163  20014 295S 296L C   N    2264 2266 -160.39 -108.50   54.22    2.43  -63.50 -163.84   33.20\n",
      "  163        296L 296L N   CA   2266 2267  148.24  132.50                  -41.20\n",
      "  164  20017 298P 299A C   N    2286 2288 -164.26 -134.00   33.75    0.77  -62.50 -172.76   36.08\n",
      "  164        299A 299A N   CA   2288 2289  161.93  147.00                  -40.90\n",
      "  165  20018 299A 300F C   N    2291 2293 -165.79 -124.20   45.57    1.28  -63.20 -175.15   32.00\n",
      "  165        300F 300F N   CA   2293 2294  161.93  143.30                  -44.30\n",
      "  166  20019 300F 301G C   N    2302 2304 -176.22 -167.20   40.33    1.65   82.20  162.46   12.14\n",
      "  166        301G 301G N   CA   2304 2305  135.29  174.60                    8.50\n",
      "  167  20021 302P 303G C   N    2313 2315 -167.74 -167.20   18.38    0.73   82.20 -175.79   13.63\n",
      "  167        303G 303G N   CA   2315 2316  156.23  174.60                    8.50\n",
      "  168  20022 303G 304R C   N    2317 2319 -156.76 -125.20   39.70    1.17  -63.00 -179.52   30.65\n",
      "  168        304R 304R N   CA   2319 2320  164.69  140.60                  -41.10\n",
      "  169  20023 304R 305T C   N    2328 2330 -149.15 -124.80   43.21    1.34  -63.20  163.18   26.39\n",
      "  169        305T 305T N   CA   2330 2331  179.19  143.50                  -42.10\n",
      "  170  20024 305T 306G C   N    2335 2337 -171.98 -167.20   11.69    0.48   82.20 -171.97   13.71\n",
      "  170        306G 306G N   CA   2337 2338  163.93  174.60                    8.50\n",
      "  171  20025 306G 307M C   N    2339 2341 -159.43 -125.60   43.72    1.30  -63.40  179.20   34.11\n",
      "  171        307M 307M N   CA   2341 2342  168.20  140.50                  -40.50\n",
      "  172  20026 307M 308H C   N    2347 2349 -156.34 -125.60   39.46    0.97  -63.20 -179.89   28.50\n",
      "  172        308H 308H N   CA   2349 2350  163.54  138.80                  -42.30\n",
      "  173  20027 308H 309N C   N    2357 2359 -155.93 -119.90   43.09    1.27  -63.20 -176.56   28.09\n",
      "  173        309N 309N N   CA   2359 2360  160.62  137.00                  -41.10\n",
      "  174  20028 309N 310F C   N    2365 2367 -161.29 -124.20   42.91    1.14  -63.20  179.91   31.06\n",
      "  174        310F 310F N   CA   2367 2368  164.89  143.30                  -44.30\n",
      "  175  20029 310F 311G C   N    2376 2378  180.00 -167.20   19.17    0.73   82.20 -179.41   13.05\n",
      "  175        311G 311G N   CA   2378 2379  160.32  174.60                    8.50\n",
      "  176  20030 311G 312N C   N    2380 2382 -159.21 -119.90   49.22    1.43  -63.20 -179.98   27.81\n",
      "  176        312N 312N N   CA   2382 2383  166.62  137.00                  -41.10\n",
      "  177  20031 312N 313S C   N    2388 2390 -161.05 -136.60   28.68    0.87  -64.10 -173.95   20.47\n",
      "  177        313S 313S N   CA   2390 2391  166.20  151.20                  -35.00\n",
      "  178  20033 314C 315Y C   N    2400 2402 -144.62 -124.30   30.62    1.30  -63.50  177.86   32.17\n",
      "  178        315Y 315Y N   CA   2402 2403  158.31  135.40                  -43.40\n",
      "  179  20034 315Y 316M C   N    2412 2414 -154.54 -125.60   35.16    1.04  -63.40 -176.71   34.42\n",
      "  179        316M 316M N   CA   2414 2415  160.47  140.50                  -40.50\n",
      "  180  20035 316M 317A C   N    2420 2422 -167.68 -134.00   36.98    0.84  -62.50 -171.17   36.51\n",
      "  180        317A 317A N   CA   2422 2423  162.28  147.00                  -40.90\n",
      "  181  20036 317A 318S C   N    2425 2427 -162.75 -136.60   28.48    0.89  -64.10 -169.89   20.89\n",
      "  181        318S 318S N   CA   2427 2428  162.48  151.20                  -35.00\n",
      "  182  20038 319V 320L C   N    2438 2440 -152.16 -108.50   44.95    2.03  -63.50 -163.29   32.79\n",
      "  182        320L 320L N   CA   2440 2441  143.20  132.50                  -41.20\n",
      "  183  20039 320L 321Q C   N    2446 2448 -150.74 -121.10   35.04    1.08  -63.80 -176.76   31.44\n",
      "  183        321Q 321Q N   CA   2448 2449  158.39  139.70                  -40.30\n",
      "  184  20041 322C 323L C   N    2461 2463 -150.17 -108.50   44.20    1.97  -63.50 -167.79   32.04\n",
      "  184        323L 323L N   CA   2463 2464  147.24  132.50                  -41.20\n",
      "  185  20042 323L 324F C   N    2469 2471 -146.00 -124.20   25.31    0.67  -63.20  179.75   30.26\n",
      "  185        324F 324F N   CA   2471 2472  156.15  143.30                  -44.30\n",
      "  186  20043 324F 325S C   N    2480 2482 -155.37 -136.60   19.88    0.64  -64.10 -169.47   20.37\n",
      "  186        325S 325S N   CA   2482 2483  157.75  151.20                  -35.00\n",
      "  187  20044 325S 326L C   N    2486 2488 -151.65 -108.50   44.33    2.01  -63.50 -163.04   32.80\n",
      "  187        326L 326L N   CA   2488 2489  142.67  132.50                  -41.20\n",
      "  188  20045 326L 327E C   N    2494 2496 -137.48 -117.80   20.13    0.85  -63.60 -172.01   21.99\n",
      "  188        327E 327E N   CA   2496 2497  132.56  136.80                  -40.30\n",
      "  189  20047 328P 329F C   N    2510 2512 -149.42 -124.20   30.21    0.79  -63.20  178.04   30.19\n",
      "  189        329F 329F N   CA   2512 2513  159.93  143.30                  -44.30\n",
      "  190  20048 329F 330K C   N    2521 2523 -150.77 -118.00   37.33    1.24  -62.90 -175.52   29.21\n",
      "  190        330K 330K N   CA   2523 2524  156.99  139.10                  -40.80\n",
      "  191  20049 330K 331E C   N    2530 2532 -155.01 -117.80   43.77    1.29  -63.60 -175.86   29.60\n",
      "  191        331E 331E N   CA   2532 2533  159.85  136.80                  -40.30\n",
      "  192  20050 331E 332A C   N    2539 2541 -158.74 -134.00   26.81    0.61  -62.50 -171.76   35.98\n",
      "  192        332A 332A N   CA   2541 2542  157.33  147.00                  -40.90\n",
      "  193  20051 332A 333F C   N    2544 2546 -154.59 -124.20   32.48    0.95  -63.20 -174.94   31.48\n",
      "  193        333F 333F N   CA   2546 2547  154.78  143.30                  -44.30\n",
      "  194  20052 333F 334Y C   N    2555 2557 -140.80 -124.30   21.70    1.02  -63.50 -175.89   33.02\n",
      "  194        334Y 334Y N   CA   2557 2558  149.50  135.40                  -43.40\n",
      "  195  20053 334Y 335Y C   N    2567 2569 -141.16 -124.30   25.75    1.09  -63.50  179.41   32.28\n",
      "  195        335Y 335Y N   CA   2569 2570  154.87  135.40                  -43.40\n",
      "  196  20054 335Y 336N C   N    2579 2581 -150.85 -119.90   36.59    1.09  -63.20 -175.47   27.98\n",
      "  196        336N 336N N   CA   2581 2582  156.52  137.00                  -41.10\n",
      "  197  20056 337C 338E C   N    2593 2595 -156.21 -117.80   41.33    1.23  -63.60 -168.48   30.67\n",
      "  197        338E 338E N   CA   2595 2596  152.05  136.80                  -40.30\n",
      "  198  20057 338E 339T C   N    2602 2604 -147.47 -124.80   26.40    1.00  -63.20 -178.41   28.74\n",
      "  198        339T 339T N   CA   2604 2605  157.04  143.50                  -42.10\n",
      "  199  20058 339T 340R C   N    2609 2611 -156.49 -125.20   35.72    1.07  -63.00 -173.77   31.44\n",
      "  199        340R 340R N   CA   2611 2612  157.84  140.60                  -41.10\n",
      "  200  20059 340R 341H C   N    2620 2622 -159.28 -125.60   38.03    0.99  -63.20 -172.30   29.65\n",
      "  200        341H 341H N   CA   2622 2623  156.45  138.80                  -42.30\n",
      "  201  20060 341H 342Q C   N    2630 2632 -158.16 -121.10   39.86    1.26  -63.80 -169.64   32.84\n",
      "  201        342Q 342Q N   CA   2632 2633  154.38  139.70                  -40.30\n",
      "  202  20061 342Q 343N C   N    2639 2641 -153.56 -119.90   36.59    1.17  -63.20 -169.64   28.86\n",
      "  202        343N 343N N   CA   2641 2642  151.35  137.00                  -41.10\n",
      "  203  20062 343N 344A C   N    2647 2649 -160.15 -134.00   27.83    0.65  -62.50 -170.35   36.28\n",
      "  203        344A 344A N   CA   2649 2650  156.53  147.00                  -40.90\n",
      "  204  20064 345C 346Q C   N    2658 2660 -158.54 -121.10   41.73    1.29  -63.80 -172.71   32.39\n",
      "  204        346Q 346Q N   CA   2660 2661  158.13  139.70                  -40.30\n",
      "  205  20065 346Q 347E C   N    2667 2669 -152.00 -117.80   38.71    1.13  -63.60 -173.01   29.84\n",
      "  205        347E 347E N   CA   2669 2670  154.92  136.80                  -40.30\n",
      "  206  20066 347E 348N C   N    2676 2678 -139.92 -119.90   20.11    0.88  -63.20 -167.75   20.89\n",
      "  206        348N 348N N   CA   2678 2679  135.18  137.00                  -41.10\n",
      "  207  20068 349P 350Y C   N    2691 2693 -126.08 -124.30    6.41    0.42  -63.50 -176.36   26.29\n",
      "  207        350Y 350Y N   CA   2693 2694  129.24  135.40                  -43.40\n",
      "  208  20071 352C 353H C   N    2718 2720 -161.10 -125.60   40.67    1.05  -63.20 -173.23   29.63\n",
      "  208        353H 353H N   CA   2720 2721  158.65  138.80                  -42.30\n",
      "  209  20074 355C 356Q C   N    2740 2742 -157.35 -121.10   40.40    1.25  -63.80 -172.79   32.33\n",
      "  209        356Q 356Q N   CA   2742 2743  157.54  139.70                  -40.30\n",
      "  210  20075 356Q 357T C   N    2749 2751 -146.39 -124.80   30.02    0.98  -63.20  174.63   27.76\n",
      "  210        357T 357T N   CA   2751 2752  164.36  143.50                  -42.10\n",
      "  211  20076 357T 358E C   N    2756 2758 -158.10 -117.80   44.63    1.31  -63.60 -170.97   30.42\n",
      "  211        358E 358E N   CA   2758 2759  155.98  136.80                  -40.30\n",
      "  212  20077 358E 359R C   N    2765 2767 -152.16 -125.20   30.79    0.92  -63.00 -173.82   31.17\n",
      "  212        359R 359R N   CA   2767 2768  155.46  140.60                  -41.10\n",
      "  213  20079 360V 361A C   N    2783 2785 -161.71 -134.00   29.92    0.68  -62.50 -171.05   36.24\n",
      "  213        361A 361A N   CA   2785 2786  158.29  147.00                  -40.90\n",
      "  214  20080 361A 362S C   N    2788 2790 -167.25 -136.60   36.11    1.10  -64.10 -174.06   20.92\n",
      "  214        362S 362S N   CA   2790 2791  170.28  151.20                  -35.00\n",
      "  215  20081 362S 363G C   N    2794 2796  173.17 -167.20   30.69    1.19   82.20  169.07   12.19\n",
      "  215        363G 363G N   CA   2796 2797  151.01  174.60                    8.50\n",
      "  216  20082 363G 364L C   N    2798 2800 -146.19 -108.50   42.22    1.86  -63.50 -173.39   31.02\n",
      "  216        364L 364L N   CA   2800 2801  151.51  132.50                  -41.20\n",
      "  217  20083 364L 365L C   N    2806 2808 -141.52 -108.50   34.07    1.54  -63.50 -165.74   31.80\n",
      "  217        365L 365L N   CA   2808 2809  140.89  132.50                  -41.20\n",
      "  218  20084 365L 366S C   N    2814 2816 -155.18 -136.60   24.15    0.75  -64.10 -177.30   19.80\n",
      "  218        366S 366S N   CA   2816 2817  166.62  151.20                  -35.00\n",
      "  219  20085 366S 367G C   N    2820 2822  172.21 -167.20   26.29    0.93   82.20  174.72   12.47\n",
      "  219        367G 367G N   CA   2822 2823  158.26  174.60                    8.50\n",
      "  220  20086 367G 368E C   N    2824 2826 -158.38 -117.80   45.89    1.34  -63.60 -172.77   30.19\n",
      "  220        368E 368E N   CA   2826 2827  158.23  136.80                  -40.30\n",
      "  221  20087 368E 369F C   N    2833 2835 -154.91 -124.20   33.78    0.94  -63.20 -177.02   31.19\n",
      "  221        369F 369F N   CA   2835 2836  157.37  143.30                  -44.30\n",
      "  222  20088 369F 370S C   N    2844 2846 -163.96 -136.60   29.62    0.93  -64.10 -169.32   21.02\n",
      "  222        370S 370S N   CA   2846 2847  162.55  151.20                  -35.00\n",
      "  223  20090 371V 372E C   N    2857 2859 -159.53 -117.80   50.64    1.50  -63.60 -178.39   29.48\n",
      "  223        372E 372E N   CA   2859 2860  165.49  136.80                  -40.30\n",
      "  224  20091 372E 373G C   N    2866 2868  178.21 -167.20   22.98    0.89   82.20  176.70   12.78\n",
      "  224        373G 373G N   CA   2868 2869  156.85  174.60                    8.50\n",
      "  225  20092 373G 374Q C   N    2870 2872 -157.49 -121.10   42.40    1.31  -63.80 -176.11   31.83\n",
      "  225        374Q 374Q N   CA   2872 2873  161.47  139.70                  -40.30\n",
      "  226  20093 374Q 375E C   N    2879 2881 -146.11 -117.80   31.90    0.93  -63.60 -172.67   29.59\n",
      "  226        375E 375E N   CA   2881 2882  151.52  136.80                  -40.30\n",
      "  227  20094 375E 376L C   N    2888 2890 -142.98 -108.50   35.45    1.60  -63.50 -165.01   32.00\n",
      "  227        376L 376L N   CA   2890 2891  140.74  132.50                  -41.20\n",
      "  228  20095 376L 377T C   N    2896 2898 -136.50 -124.80   18.30    0.57  -63.20  176.30   27.40\n",
      "  228        377T 377T N   CA   2898 2899  157.57  143.50                  -42.10\n",
      "  229  20096 377T 378N C   N    2903 2905 -155.03 -119.90   46.05    1.33  -63.20  177.71   27.31\n",
      "  229        378N 378N N   CA   2905 2906  166.76  137.00                  -41.10\n",
      "  230  20097 378N 379G C   N    2911 2913 -147.23 -167.20   72.78    2.53   82.20  162.13   12.65\n",
      "  230        379G 379G N   CA   2913 2914  104.61  174.60                    8.50\n",
      "  231  20098 379G 380I C   N    2915 2917  -71.66  -63.40    8.50    1.45 -120.60  178.73    8.41\n",
      "  231        380I 380I N   CA   2917 2918  -41.60  -43.60                  130.30\n",
      "  232  20099 380I 381T C   N    2923 2925  -46.40  -78.10   34.16    1.69  -63.20  179.96   24.19\n",
      "  232        381T 381T N   CA   2925 2926  137.08  149.80                  -42.10\n",
      "  233  20100 381T 382A C   N    2930 2932 -161.75 -134.00   29.66    0.68  -62.50 -170.32   36.37\n",
      "  233        382A 382A N   CA   2932 2933  157.46  147.00                  -40.90\n",
      "  234  20101 382A 383R C   N    2935 2937 -151.40 -125.20   32.19    0.95  -63.00 -177.56   30.61\n",
      "  234        383R 383R N   CA   2937 2938  159.30  140.60                  -41.10\n",
      "  235  20102 383R 384E C   N    2946 2948 -153.18 -117.80   39.66    1.16  -63.60 -172.27   30.00\n",
      "  235        384E 384E N   CA   2948 2949  154.72  136.80                  -40.30\n",
      "  236  20103 384E 385F C   N    2955 2957 -148.24 -124.20   27.16    0.73  -63.20 -179.01   30.56\n",
      "  236        385F 385F N   CA   2957 2958  155.93  143.30                  -44.30\n",
      "  237  20104 385F 386K C   N    2966 2968 -153.29 -118.00   38.46    1.28  -62.90 -172.04   29.81\n",
      "  237        386K 386K N   CA   2968 2969  154.40  139.10                  -40.80\n",
      "  238  20105 386K 387R C   N    2975 2977 -152.58 -125.20   31.26    0.94  -63.00 -173.82   31.20\n",
      "  238        387R 387R N   CA   2977 2978  155.69  140.60                  -41.10\n",
      "  239  20107 388V 389F C   N    2993 2995 -150.78 -124.20   30.71    0.81  -63.20  179.80   30.52\n",
      "  239        389F 389F N   CA   2995 2996  158.67  143.30                  -44.30\n",
      "  240  20108 389F 390A C   N    3004 3006 -152.43 -134.00   20.83    0.48  -62.50 -174.35   35.23\n",
      "  240        390A 390A N   CA   3006 3007  156.69  147.00                  -40.90\n",
      "  241  20109 390A 391Q C   N    3009 3011 -147.74 -121.10   32.60    1.01  -63.80 -178.25   31.07\n",
      "  241        391Q 391Q N   CA   3011 3012  158.49  139.70                  -40.30\n",
      "  242  20110 391Q 392K C   N    3018 3020 -142.10 -118.00   28.67    0.96  -62.90 -177.37   28.46\n",
      "  242        392K 392K N   CA   3020 3021  154.64  139.10                  -40.80\n",
      "  243  20111 392K 393H C   N    3027 3029 -144.65 -125.60   19.83    0.58  -63.20 -168.41   29.26\n",
      "  243        393H 393H N   CA   3029 3030  144.28  138.80                  -42.30\n",
      "  244  20113 394P 395D C   N    3044 3046 -135.68  -96.50   49.30    2.05  -63.30 -169.81   28.16\n",
      "  244        395D 395D N   CA   3046 3047  144.13  114.20                  -40.00\n",
      "  245  20114 395D 396F C   N    3052 3054 -139.87 -124.20   17.67    0.48  -63.20 -178.76   30.13\n",
      "  245        396F 396F N   CA   3054 3055  151.48  143.30                  -44.30\n",
      "  246  20115 396F 397S C   N    3063 3065 -157.60 -136.60   22.00    0.72  -64.10 -168.41   20.61\n",
      "  246        397S 397S N   CA   3065 3066  157.77  151.20                  -35.00\n",
      "  247  20116 397S 398T C   N    3069 3071 -145.37 -124.80   25.20    0.90  -63.20  179.72   28.37\n",
      "  247        398T 398T N   CA   3071 3072  158.06  143.50                  -42.10\n",
      "  248  20117 398T 399A C   N    3076 3078 -157.80 -134.00   25.18    0.59  -62.50 -170.39   36.16\n",
      "  248        399A 399A N   CA   3078 3079  155.19  147.00                  -40.90\n",
      "  249  20118 399A 400E C   N    3081 3083 -153.79 -117.80   38.47    1.15  -63.60 -168.17   30.59\n",
      "  249        400E 400E N   CA   3083 3084  150.39  136.80                  -40.30\n",
      "  250  20119 400E 401Q C   N    3090 3092 -154.62 -121.10   36.37    1.14  -63.80 -170.86   32.50\n",
      "  250        401Q 401Q N   CA   3092 3093  153.79  139.70                  -40.30\n",
      "  251  20120 401Q 402Q C   N    3099 3101 -151.01 -121.10   30.39    1.04  -63.80 -164.83   33.24\n",
      "  251        402Q 402Q N   CA   3101 3102  145.09  139.70                  -40.30\n",
      "  252  20121 402Q 403D C   N    3108 3110 -151.36  -96.50   58.02    2.39  -63.30 -165.80   20.61\n",
      "  252        403D 403D N   CA   3110 3111  133.09  114.20                  -40.00\n",
      "  253  20122 403D 404A C   N    3116 3118 -148.74 -134.00   15.11    0.39  -62.50 -170.43   35.68\n",
      "  253        404A 404A N   CA   3118 3119  150.29  147.00                  -40.90\n",
      "  254  20123 404A 405Q C   N    3121 3123 -161.23 -121.10   42.44    1.36  -63.80 -167.35   33.31\n",
      "  254        405Q 405Q N   CA   3123 3124  153.50  139.70                  -40.30\n",
      "  255  20124 405Q 406E C   N    3130 3132 -140.58 -117.80   22.81    0.81  -63.60 -165.81   22.70\n",
      "  255        406E 406E N   CA   3132 3133  137.98  136.80                  -40.30\n",
      "  256  20125 406E 407Y C   N    3139 3141 -138.79 -124.30   14.58    1.14  -63.50 -167.46   27.25\n",
      "  256        407Y 407Y N   CA   3141 3142  133.81  135.40                  -43.40\n",
      "  257  20126 407Y 408F C   N    3151 3153 -159.82 -124.20   36.82    1.17  -63.20 -170.45   32.40\n",
      "  257        408F 408F N   CA   3153 3154  152.62  143.30                  -44.30\n",
      "  258  20127 408F 409L C   N    3162 3164 -143.76 -108.50   41.83    1.85  -63.50 -177.60   30.29\n",
      "  258        409L 409L N   CA   3164 3165  155.01  132.50                  -41.20\n",
      "  259  20128 409L 410Y C   N    3170 3172 -131.18 -124.30   10.37    0.44  -63.50 -173.82   32.93\n",
      "  259        410Y 410Y N   CA   3172 3173  143.15  135.40                  -43.40\n",
      "  260  20129 410Y 411L C   N    3182 3184 -137.58 -108.50   29.38    1.35  -63.50 -167.31   23.75\n",
      "  260        411L 411L N   CA   3184 3185  136.67  132.50                  -41.20\n",
      "  261  20131 412V 413E C   N    3197 3199 -150.35 -117.80   36.13    1.06  -63.60 -171.61   29.95\n",
      "  261        413E 413E N   CA   3199 3200  152.47  136.80                  -40.30\n",
      "  262  20132 413E 414Q C   N    3206 3208 -150.07 -121.10   32.12    1.00  -63.80 -172.81   32.00\n",
      "  262        414Q 414Q N   CA   3208 3209  153.57  139.70                  -40.30\n",
      "  263  20133 414Q 415M C   N    3215 3217 -150.65 -125.60   28.30    0.85  -63.40 -172.61   34.78\n",
      "  263        415M 415M N   CA   3217 3218  153.66  140.50                  -40.50\n",
      "  264  20134 415M 416R C   N    3223 3225 -150.15 -125.20   29.08    0.86  -63.00 -174.86   30.91\n",
      "  264        416R 416R N   CA   3225 3226  155.55  140.60                  -41.10\n",
      "  265  20135 416R 417R C   N    3234 3236 -140.70 -125.20   15.97    0.55  -63.00 -169.02   31.11\n",
      "  265        417R 417R N   CA   3236 3237  144.45  140.60                  -41.10\n",
      "  266  20136 417R 418Y C   N    3245 3247 -139.21 -124.30   22.54    0.96  -63.50 -179.10   32.44\n",
      "  266        418Y 418Y N   CA   3247 3248  152.30  135.40                  -43.40\n",
      "  267  20138 419V 420K C   N    3264 3266 -129.18 -118.00   15.41    0.54  -62.90 -178.02   27.60\n",
      "  267        420K 420K N   CA   3266 3267  149.72  139.10                  -40.80\n",
      "  268  20140 421P 422S C   N    3280 3282 -129.26 -136.60   11.96    0.68  -64.10  176.80   17.42\n",
      "  268        422S 422S N   CA   3282 3283  160.64  151.20                  -35.00\n",
      "  269  20141 422S 423N C   N    3286 3288 -130.56 -119.90   26.86    0.95  -63.20  171.07   25.20\n",
      "  269        423N 423N N   CA   3288 3289  161.65  137.00                  -41.10\n",
      "  270  20144 425V 426G C   N    3309 3311  -88.90  -80.20   12.08    0.86   82.20 -127.63    7.63\n",
      "  270        426G 426G N   CA   3311 3312  165.72  174.10                    8.50\n",
      "  271  20145 426G 427A C   N    3313 3315  -90.95  -68.20   30.72    1.95  -62.50  155.79   27.01\n",
      "  271        427A 427A N   CA   3315 3316  165.94  145.30                  -40.90\n",
      "  272  20146 427A 428N C   N    3318 3320 -125.90 -119.90   19.94    0.76  -63.20  174.54   25.38\n",
      "  272        428N 428N N   CA   3320 3321  156.01  137.00                  -41.10\n",
      "  273  20148 429I 430S C   N    3334 3336 -135.26 -136.60    9.03    0.48  -64.10  179.57   18.07\n",
      "  273        430S 430S N   CA   3336 3337  160.13  151.20                  -35.00\n",
      "  274  20149 430S 431H C   N    3340 3342 -127.40 -125.60    5.16    0.22  -63.20 -172.41   20.35\n",
      "  274        431H 431H N   CA   3342 3343  133.96  138.80                  -42.30\n",
      "  275  20152 433V 434D C   N    3364 3366 -122.90  -96.50   33.38    1.39  -63.30 -175.49   19.98\n",
      "  275        434D 434D N   CA   3366 3367  134.62  114.20                  -40.00\n",
      "  276  20154 435I 436F C   N    3380 3382 -137.37 -124.20   15.26    0.40  -63.20 -179.38   29.90\n",
      "  276        436F 436F N   CA   3382 3383  151.01  143.30                  -44.30\n",
      "  277  20155 436F 437K C   N    3391 3393 -149.29 -118.00   34.38    1.14  -62.90 -173.01   29.45\n",
      "  277        437K 437K N   CA   3393 3394  153.37  139.10                  -40.80\n",
      "  278  20156 437K 438M C   N    3400 3402 -142.94 -125.60   18.66    0.59  -63.40 -170.42   34.57\n",
      "  278        438M 438M N   CA   3402 3403  147.41  140.50                  -40.50\n",
      "  279  20157 438M 439T C   N    3408 3410 -140.42 -124.80   19.26    0.68  -63.20 -179.51   28.18\n",
      "  279        439T 439T N   CA   3410 3411  154.77  143.50                  -42.10\n",
      "  280  20159 440V 441E C   N    3422 3424 -147.57 -117.80   32.99    0.97  -63.60 -171.58   29.81\n",
      "  280        441E 441E N   CA   3424 3425  151.03  136.80                  -40.30\n",
      "  281  20160 441E 442H C   N    3431 3433 -146.31 -125.60   23.89    0.61  -63.20 -173.45   28.72\n",
      "  281        442H 442H N   CA   3433 3434  150.69  138.80                  -42.30\n",
      "  282  20161 442H 443R C   N    3441 3443 -146.52 -125.20   24.35    0.73  -63.00 -173.71   30.84\n",
      "  282        443R 443R N   CA   3443 3444  152.38  140.60                  -41.10\n",
      "  283  20163 444V 445Q C   N    3459 3461 -146.38 -121.10   26.93    0.86  -63.80 -170.38   32.19\n",
      "  283        445Q 445Q N   CA   3461 3462  149.00  139.70                  -40.30\n",
      "  284  20165 446C 447G C   N    3474 3476  164.80 -167.20   35.46    1.25   82.20  166.29   11.76\n",
      "  284        447G 447G N   CA   3476 3477  152.83  174.60                    8.50\n",
      "  285  20166 447G 448S C   N    3478 3480 -158.66 -136.60   23.50    0.75  -64.10 -169.21   20.63\n",
      "  285        448S 448S N   CA   3480 3481  159.29  151.20                  -35.00\n",
      "  286  20168 449C 450H C   N    3490 3492 -149.41 -125.60   25.11    0.72  -63.20 -168.55   29.54\n",
      "  286        450H 450H N   CA   3492 3493  146.76  138.80                  -42.30\n",
      "  287  20169 450H 451K C   N    3500 3502 -146.75 -118.00   29.83    1.02  -62.90 -168.54   29.88\n",
      "  287        451K 451K N   CA   3502 3503  147.08  139.10                  -40.80\n",
      "  288  20171 452V 453R C   N    3516 3518 -145.59 -125.20   20.40    0.80  -63.00 -164.21   32.09\n",
      "  288        453R 453R N   CA   3518 3519  141.38  140.60                  -41.10\n",
      "  289  20172 453R 454Y C   N    3527 3529 -133.86 -124.30   10.12    0.64  -63.50 -168.72   33.87\n",
      "  289        454Y 454Y N   CA   3529 3530  138.73  135.40                  -43.40\n",
      "  290  20173 454Y 455T C   N    3539 3541 -138.27 -124.80   13.57    0.68  -63.20 -171.62   29.08\n",
      "  290        455T 455T N   CA   3541 3542  145.13  143.50                  -42.10\n",
      "  291  20174 455T 456R C   N    3546 3548 -149.34 -125.20   24.71    0.86  -63.00 -166.64   31.99\n",
      "  291        456R 456R N   CA   3548 3549  145.88  140.60                  -41.10\n",
      "  292  20175 456R 457E C   N    3557 3559 -147.13 -117.80   30.37    0.95  -63.60 -166.07   30.53\n",
      "  292        457E 457E N   CA   3559 3560  144.69  136.80                  -40.30\n",
      "  293  20176 457E 458T C   N    3566 3568 -133.90 -124.80    9.79    0.58  -63.20 -168.48   29.22\n",
      "  293        458T 458T N   CA   3568 3569  139.90  143.50                  -42.10\n",
      "  294  20177 458T 459D C   N    3573 3575 -130.80  -96.50   36.52    1.51  -63.30  179.87   19.25\n",
      "  294        459D 459D N   CA   3575 3576  126.73  114.20                  -40.00\n",
      "  295  20180 461C 462L C   N    3593 3595 -129.85 -108.50   21.44    1.00  -63.50 -172.18   23.26\n",
      "  295        462L 462L N   CA   3595 3596  134.51  132.50                  -41.20\n",
      "  296  20181 462L 463S C   N    3601 3603 -134.49 -136.60    4.45    0.17  -64.10 -168.84   18.83\n",
      "  296        463S 463S N   CA   3603 3604  147.28  151.20                  -35.00\n",
      "  297  20182 463S 464L C   N    3607 3609 -121.68 -108.50   16.22    0.72  -63.50 -173.83   29.44\n",
      "  297        464L 464L N   CA   3609 3610  141.96  132.50                  -41.20\n",
      "  298  20186 467P 468L C   N    3637 3639 -122.91 -108.50   16.70    0.74  -63.50 -172.48   29.71\n",
      "  298        468L 468L N   CA   3639 3640  140.94  132.50                  -41.20\n",
      "  299  20187 468L 469D C   N    3645 3647 -102.77  -96.50    6.31    0.26  -63.30  158.48   17.55\n",
      "  299        469D 469D N   CA   3647 3648  113.49  114.20                  -40.00\n",
      "  300  20188 469D 470P C   N    3653 3655  -51.47  -58.70   14.86    0.87  -64.50  169.82   13.37\n",
      "  300        470P 470P N   CA   3655 3656  -43.48  -30.50                  147.20\n",
      "  301  20189 470P 471N C   N    3660 3662   54.70   55.90  100.18    6.92  -63.20 -145.47   23.08\n",
      "  301        471N 471N N   CA   3662 3663  139.67   39.50                  -41.10\n",
      "  302  20190 471N 472L C   N    3668 3670 -116.23 -108.50    7.91    0.36  -63.50 -176.85   23.00\n",
      "  302        472L 472L N   CA   3670 3671  134.19  132.50                  -41.20\n",
      "  303  20191 472L 473K C   N    3676 3678 -133.08 -118.00   15.29    0.54  -62.90 -169.06   28.99\n",
      "  303        473K 473K N   CA   3678 3679  141.62  139.10                  -40.80\n",
      "  304  20192 473K 474N C   N    3685 3687 -138.36 -119.90   19.94    0.64  -63.20 -170.11   28.00\n",
      "  304        474N 474N N   CA   3687 3688  144.52  137.00                  -41.10\n",
      "  305  20193 474N 475S C   N    3693 3695 -133.40 -136.60   13.36    0.60  -64.10 -173.43   11.55\n",
      "  305        475S 475S N   CA   3695 3696  138.22  151.20                  -35.00\n",
      "  306  20194 475S 476D C   N    3699 3701 -142.67  -96.50   50.54    2.09  -63.30 -168.07   20.43\n",
      "  306        476D 476D N   CA   3701 3702  134.75  114.20                  -40.00\n",
      "  307  20195 476D 477G C   N    3707 3709 -169.49 -167.20   20.99    0.85   82.20 -178.83   13.40\n",
      "  307        477G 477G N   CA   3709 3710  153.73  174.60                    8.50\n",
      "  308  20196 477G 478N C   N    3711 3713 -127.15 -119.90    8.42    0.25  -63.20 -171.21   27.24\n",
      "  308        478N 478N N   CA   3713 3714  141.27  137.00                  -41.10\n",
      "  309  20197 478N 479P C   N    3719 3721  -65.08  -58.70    7.31    1.03  -64.50  178.74   13.31\n",
      "  309        479P 479P N   CA   3721 3722  -34.06  -30.50                  147.20\n",
      "  310  20198 479P 480K C   N    3726 3728   64.51   56.60  107.63    8.30  -62.90 -144.94   25.09\n",
      "  310        480K 480K N   CA   3728 3729  145.94   38.60                  -40.80\n",
      "  311  20199 480K 481Q C   N    3735 3737 -137.20 -121.10   16.10    0.61  -63.80 -165.63   25.44\n",
      "  311        481Q 481Q N   CA   3737 3738  139.68  139.70                  -40.30\n",
      "  312  20200 481Q 482T C   N    3744 3746 -139.34 -124.80   16.20    0.65  -63.20 -176.22   28.55\n",
      "  312        482T 482T N   CA   3746 3747  150.64  143.50                  -42.10\n",
      "  313  20201 482T 483E C   N    3751 3753 -144.13 -117.80   26.72    0.87  -63.60 -164.34   30.61\n",
      "  313        483E 483E N   CA   3753 3754  141.37  136.80                  -40.30\n",
      "  314  20202 483E 484E C   N    3760 3762 -148.75 -117.80   32.55    0.99  -63.60 -167.35   30.44\n",
      "  314        484E 484E N   CA   3762 3763  146.89  136.80                  -40.30\n",
      "  315  20203 484E 485E C   N    3769 3771 -139.10 -117.80   21.55    0.71  -63.60 -165.17   30.23\n",
      "  315        485E 485E N   CA   3771 3772  140.09  136.80                  -40.30\n",
      "  316  20205 486I 487F C   N    3786 3788 -140.69 -124.20   17.21    0.53  -63.20 -175.48   30.64\n",
      "  316        487F 487F N   CA   3788 3789  148.24  143.30                  -44.30\n",
      "  317  20206 487F 488A C   N    3797 3799 -160.71 -134.00   27.82    0.68  -62.50 -168.58   36.60\n",
      "  317        488A 488A N   CA   3799 3800  154.80  147.00                  -40.90\n",
      "  318  20207 488A 489S C   N    3802 3804 -157.38 -136.60   21.36    0.73  -64.10 -167.06   20.69\n",
      "  318        489S 489S N   CA   3804 3805  156.11  151.20                  -35.00\n",
      "  319  20208 489S 490R C   N    3808 3810 -131.98 -125.20   17.59    0.96  -63.00  179.27   20.63\n",
      "  319        490R 490R N   CA   3810 3811  124.37  140.60                  -41.10\n",
      "  320  20209 490R 491P C   N    3819 3821  -53.73  -58.70   14.78    0.90  -64.50  168.72   13.17\n",
      "  320        491P 491P N   CA   3821 3822  -44.42  -30.50                  147.20\n",
      "  321  20210 491P 492H C   N    3826 3828   66.31   56.30   97.48    9.49  -63.20 -138.30   23.81\n",
      "  321        492H 492H N   CA   3828 3829  137.76   40.80                  -42.30\n",
      "  322  20212 493I 494S C   N    3844 3846 -142.43 -136.60    5.87    0.25  -64.10 -168.74   19.44\n",
      "  322        494S 494S N   CA   3846 3847  150.51  151.20                  -35.00\n",
      "  323  20213 494S 495L C   N    3850 3852 -132.23 -108.50   24.91    1.11  -63.50 -168.51   30.84\n",
      "  323        495L 495L N   CA   3852 3853  140.07  132.50                  -41.20\n",
      "  324  20214 495L 496D C   N    3858 3860 -134.72  -96.50   41.65    1.72  -63.30 -174.92   19.77\n",
      "  324        496D 496D N   CA   3860 3861  130.75  114.20                  -40.00\n",
      "  325  20215 496D 497A C   N    3866 3868 -142.90 -134.00    9.25    0.23  -62.50 -172.34   35.06\n",
      "  325        497A 497A N   CA   3868 3869  149.54  147.00                  -40.90\n",
      "  326  20217 498C 499L C   N    3877 3879 -142.10 -108.50   40.27    1.78  -63.50 -178.04   30.13\n",
      "  326        499L 499L N   CA   3879 3880  154.69  132.50                  -41.20\n",
      "  327  20218 499L 500G C   N    3885 3887  173.83 -167.20   30.84    1.21   82.20  168.81   12.20\n",
      "  327        500G 500G N   CA   3887 3888  150.28  174.60                    8.50\n",
      "  328  20219 500G 501S C   N    3889 3891 -157.64 -136.60   21.29    0.76  -64.10 -165.48   20.82\n",
      "  328        501S 501S N   CA   3891 3892  154.44  151.20                  -35.00\n",
      "  329  20220 501S 502L C   N    3895 3897 -137.55 -108.50   31.46    1.39  -63.50 -170.70   30.87\n",
      "  329        502L 502L N   CA   3897 3898  144.58  132.50                  -41.20\n",
      "  330  20221 502L 503M C   N    3903 3905 -144.19 -125.60   19.72    0.63  -63.40 -169.58   34.78\n",
      "  330        503M 503M N   CA   3905 3906  147.07  140.50                  -40.50\n",
      "  331  20222 503M 504Q C   N    3911 3913 -154.70 -121.10   35.74    1.14  -63.80 -169.15   32.76\n",
      "  331        504Q 504Q N   CA   3913 3914  151.89  139.70                  -40.30\n",
      "  332  20223 504Q 505A C   N    3920 3922 -156.47 -134.00   23.22    0.58  -62.50 -169.00   36.32\n",
      "  332        505A 505A N   CA   3922 3923  152.81  147.00                  -40.90\n",
      "  333  20224 505A 506T C   N    3925 3927 -141.36 -124.80   17.95    0.75  -63.20 -175.19   28.80\n",
      "  333        506T 506T N   CA   3927 3928  150.43  143.50                  -42.10\n",
      "  334  20225 506T 507D C   N    3932 3934 -141.24  -96.50   47.62    1.96  -63.30 -172.52   19.95\n",
      "  334        507D 507D N   CA   3934 3935  130.51  114.20                  -40.00\n",
      "  335  20227 508I 509D C   N    3948 3950 -133.30  -96.50   42.10    1.75  -63.30 -171.86   20.15\n",
      "  335        509D 509D N   CA   3950 3951  134.64  114.20                  -40.00\n",
      "  336  20229 510C 511R C   N    3962 3964 -150.30 -125.20   28.24    0.85  -63.00 -173.03   31.17\n",
      "  336        511R 511R N   CA   3964 3965  153.56  140.60                  -41.10\n",
      "  337  20231 512C 513S C   N    3979 3981 -153.97 -136.60   19.71    0.60  -64.10 -172.56   20.04\n",
      "  337        513S 513S N   CA   3981 3982  160.51  151.20                  -35.00\n",
      "  338  20232 513S 514A C   N    3985 3987 -154.85 -134.00   22.04    0.52  -62.50 -170.94   35.92\n",
      "  338        514A 514A N   CA   3987 3988  154.13  147.00                  -40.90\n",
      "  339  20234 515C 516G C   N    3996 3998  173.40 -167.20   26.82    0.99   82.20  173.48   12.44\n",
      "  339        516G 516G N   CA   3998 3999  156.07  174.60                    8.50\n",
      "  340  20236 517I 518P C   N    4008 4010  -63.86  -58.70   13.04    1.50  -64.50  170.33   12.74\n",
      "  340        518P 518P N   CA   4010 4011  -42.47  -30.50                  147.20\n",
      "  341  20237 518P 519V C   N    4015 4017   52.45  -73.50  125.97    8.97 -125.40  177.86    6.89\n",
      "  341        519V 519V N   CA   4017 4018  141.35  139.20                  143.30\n",
      "  342  20238 519V 520T C   N    4022 4024 -138.81 -124.80   14.56    0.66  -63.20 -173.57   28.86\n",
      "  342        520T 520T N   CA   4024 4025  147.49  143.50                  -42.10\n",
      "  343  20239 520T 521Y C   N    4029 4031 -142.16 -124.30   25.43    1.12  -63.50 -178.92   32.59\n",
      "  343        521Y 521Y N   CA   4031 4032  153.50  135.40                  -43.40\n",
      "  344  20241 522C 523K C   N    4047 4049 -160.88 -118.00   44.99    1.52  -62.90 -166.81   30.93\n",
      "  344        523K 523K N   CA   4049 4050  152.70  139.10                  -40.80\n",
      "  345  20242 523K 524T C   N    4056 4058 -144.60 -124.80   22.22    0.88  -63.20 -176.61   28.81\n",
      "  345        524T 524T N   CA   4058 4059  153.57  143.50                  -42.10\n",
      "  346  20243 524T 525T C   N    4063 4065 -151.06 -124.80   30.25    1.16  -63.20 -178.00   29.00\n",
      "  346        525T 525T N   CA   4065 4066  158.51  143.50                  -42.10\n",
      "  347  20244 525T 526R C   N    4070 4072 -150.50 -125.20   26.49    0.87  -63.00 -168.42   31.82\n",
      "  347        526R 526R N   CA   4072 4073  148.47  140.60                  -41.10\n",
      "  348  20245 526R 527L C   N    4081 4083 -152.65 -108.50   45.95    2.06  -63.50 -164.86   32.60\n",
      "  348        527L 527L N   CA   4083 4084  145.21  132.50                  -41.20\n",
      "  349  20246 527L 528R C   N    4089 4091 -144.45 -125.20   23.46    0.69  -63.00 -176.10   30.38\n",
      "  349        528R 528R N   CA   4091 4092  154.02  140.60                  -41.10\n",
      "  350  20247 528R 529T C   N    4100 4102 -144.81 -124.80   30.36    0.96  -63.20  172.14   27.34\n",
      "  350        529T 529T N   CA   4102 4103  166.34  143.50                  -42.10\n",
      "  351  20248 529T 530F C   N    4107 4109 -137.29 -124.20   15.74    0.41  -63.20  179.64   29.75\n",
      "  351        530F 530F N   CA   4109 4110  152.05  143.30                  -44.30\n",
      "  352  20250 531P 532D C   N    4125 4127 -145.08  -96.50   63.97    2.67  -63.30 -176.57   27.91\n",
      "  352        532D 532D N   CA   4127 4128  155.81  114.20                  -40.00\n",
      "  353  20252 533V 534L C   N    4140 4142 -133.22 -108.50   26.12    1.17  -63.50 -168.96   30.84\n",
      "  353        534L 534L N   CA   4142 4143  140.93  132.50                  -41.20\n",
      "  354  20253 534L 535P C   N    4148 4150  -65.17  -58.70   17.47    1.97  -64.50  166.07   12.36\n",
      "  354        535P 535P N   CA   4150 4151  -46.73  -30.50                  147.20\n",
      "  355  20254 535P 536I C   N    4155 4157   52.57  -97.30  150.21    6.20 -120.60  173.32   12.26\n",
      "  355        536I 536I N   CA   4157 4158  137.31  127.20                  130.30\n",
      "  356  20255 536I 537F C   N    4163 4165 -148.15 -124.20   30.52    0.79  -63.20  175.41   29.75\n",
      "  356        537F 537F N   CA   4165 4166  162.23  143.30                  -44.30\n",
      "  357  20256 537F 538L C   N    4174 4176 -152.84 -108.50   54.93    2.43  -63.50  177.93   30.20\n",
      "  357        538L 538L N   CA   4176 4177  164.93  132.50                  -41.20\n",
      "  358  20257 538L 539R C   N    4182 4184 -158.67 -125.20   40.98    1.21  -63.00 -178.15   30.95\n",
      "  358        539R 539R N   CA   4184 4185  164.25  140.60                  -41.10\n",
      "  359  20258 539R 540R C   N    4193 4195 -161.40 -125.20   47.35    1.41  -63.00  177.53   30.50\n",
      "  359        540R 540R N   CA   4195 4196  171.13  140.60                  -41.10\n",
      "  360  20259 540R 541A C   N    4204 4206 -169.98 -134.00   40.98    0.94  -62.50 -173.43   36.25\n",
      "  360        541A 541A N   CA   4206 4207  166.60  147.00                  -40.90\n",
      "  361  20260 541A 542Q C   N    4209 4211 -167.02 -121.10   54.34    1.68  -63.80 -177.13   32.07\n",
      "  361        542Q 542Q N   CA   4211 4212  168.75  139.70                  -40.30\n",
      "  362  20261 542Q 543F C   N    4218 4220 -169.71 -124.20   51.08    1.39  -63.20 -176.68   31.95\n",
      "  362        543F 543F N   CA   4220 4221  166.50  143.30                  -44.30\n",
      "  363  20262 543F 544D C   N    4229 4231 -171.10  -96.50   88.70    3.69  -63.30 -168.89   30.42\n",
      "  363        544D 544D N   CA   4231 4232  162.19  114.20                  -40.00\n",
      "  364  20263 544D 545M C   N    4237 4239 -163.59 -125.60   50.05    1.50  -63.40  177.42   34.10\n",
      "  364        545M 545M N   CA   4239 4240  173.08  140.50                  -40.50\n",
      "  365  20264 545M 546G C   N    4245 4247  173.21 -167.20   20.32    0.53   82.20 -175.33   13.03\n",
      "  365        546G 546G N   CA   4247 4248  169.19  174.60                    8.50\n",
      "  366  20265 546G 547T C   N    4249 4251 -157.80 -124.80   40.87    1.45  -63.20  177.58   28.78\n",
      "  366        547T 547T N   CA   4251 4252  167.61  143.50                  -42.10\n",
      "  367  20266 547T 548M C   N    4256 4258  173.43 -125.60   65.84    2.06  -63.40 -162.68   38.61\n",
      "  367        548M 548M N   CA   4258 4259  165.34  140.50                  -40.50\n",
      "  368  20267 548M 549S C   N    4264 4266 -159.38 -136.60   22.99    0.99  -64.10 -159.11   21.40\n",
      "  368        549S 549S N   CA   4266 4267  148.13  151.20                  -35.00\n",
      "  369  20269 550V 551K C   N    4277 4279 -154.52 -118.00   45.15    1.52  -62.90  178.80   28.68\n",
      "  369        551K 551K N   CA   4279 4280  165.66  139.10                  -40.80\n",
      "  370  20270 551K 552K C   N    4286 4288  149.56 -118.00   99.62    3.32  -62.90 -154.56   35.04\n",
      "  370        552K 552K N   CA   4288 4289  176.24  139.10                  -40.80\n",
      "  371  20271 552K 553L C   N    4295 4297 -146.13 -108.50   47.37    2.10  -63.50  177.87   29.80\n",
      "  371        553L 553L N   CA   4297 4298  161.28  132.50                  -41.20\n",
      "  372  20272 553L 554D C   N    4303 4305 -173.08  -96.50  105.31    4.40  -63.30  172.85   28.17\n",
      "  372        554D 554D N   CA   4305 4306 -173.51  114.20                  -40.00\n",
      "  373  20274 555V 556F C   N    4318 4320 -159.24 -124.20   55.65    1.59  -63.20  160.96   28.16\n",
      "  373        556F 556F N   CA   4320 4321 -173.47  143.30                  -44.30\n",
      "  374  20276 557V 558D C   N    4336 4338 -157.97  -96.50   83.98    3.51  -63.30  176.18   27.78\n",
      "  374        558D 558D N   CA   4338 4339  171.42  114.20                  -40.00\n",
      "  375  20279 560P 561L C   N    4358 4360 -157.60 -108.50   69.45    3.13  -63.50  166.35   28.83\n",
      "  375        561L 561L N   CA   4360 4361 -178.38  132.50                  -41.20\n",
      "  376  20280 561L 562E C   N    4366 4368 -170.16 -117.80   64.53    1.93  -63.60 -179.92   29.74\n",
      "  376        562E 562E N   CA   4368 4369  174.53  136.80                  -40.30\n",
      "  377  20281 562E 563L C   N    4375 4377  152.23 -108.50  105.32    4.69  -63.50 -151.06   37.49\n",
      "  377        563L 563L N   CA   4377 4378  167.67  132.50                  -41.20\n",
      "  378  20282 563L 564D C   N    4383 4385 -179.67  -96.50   98.37    4.09  -63.30 -167.56   31.06\n",
      "  378        564D 564D N   CA   4385 4386  166.73  114.20                  -40.00\n",
      "  379  20283 564D 565L C   N    4391 4393 -173.08 -108.50   78.68    3.48  -63.50  178.86   31.44\n",
      "  379        565L 565L N   CA   4393 4394  177.44  132.50                  -41.20\n",
      "  380  20284 565L 566E C   N    4399 4401 -175.61 -117.80   69.66    2.06  -63.60 -177.53   30.31\n",
      "  380        566E 566E N   CA   4401 4402  175.66  136.80                  -40.30\n",
      "  381  20285 566E 567F C   N    4408 4410  169.89 -124.20   73.10    2.01  -63.20 -170.47   33.68\n",
      "  381        567F 567F N   CA   4410 4411  174.93  143.30                  -44.30\n",
      "  382  20286 567F 568L C   N    4419 4421  168.52 -108.50   89.44    3.97  -63.50 -160.58   35.34\n",
      "  382        568L 568L N   CA   4421 4422  165.86  132.50                  -41.20\n",
      "  383  20287 568L 569R C   N    4427 4429  175.27 -125.20   66.48    2.02  -63.00 -167.81   33.80\n",
      "  383        569R 569R N   CA   4429 4430  170.18  140.60                  -41.10\n",
      "  384  20288 569R 570G C   N    4438 4440  172.17 -167.20   20.64    0.39   82.20 -170.83   13.23\n",
      "  384        570G 570G N   CA   4440 4441  174.91  174.60                    8.50\n",
      "  385  20289 570G 571K C   N    4442 4444 -165.41 -118.00   58.16    1.95  -62.90  178.74   29.26\n",
      "  385        571K 571K N   CA   4444 4445  172.78  139.10                  -40.80\n",
      "  386  20290 571K 572G C   N    4451 4453  176.96 -167.20   15.95    0.35   82.20 -170.40   13.43\n",
      "  386        572G 572G N   CA   4453 4454  172.72  174.60                    8.50\n",
      "  387  20291 572G 573L C   N    4455 4457  172.66 -108.50   86.90    3.84  -63.50 -165.69   34.39\n",
      "  387        573L 573L N   CA   4457 4458  169.07  132.50                  -41.20\n",
      "  388  20292 573L 574Q C   N    4463 4465  101.17 -121.10  138.01    5.01  -63.80 -122.20   30.92\n",
      "  388        574Q 574Q N   CA   4465 4466  148.43  139.70                  -40.30\n",
      "  389  20293 574Q 575S C   N    4472 4474  172.91 -136.60   50.88    1.86  -64.10 -152.15   23.94\n",
      "  389        575S 575S N   CA   4474 4475  157.44  151.20                  -35.00\n",
      "  390  20294 575S 576D C   N    4478 4480 -176.25  -96.50  111.23    4.65  -63.30  170.92   28.06\n",
      "  390        576D 576D N   CA   4480 4481 -168.27  114.20                  -40.00\n",
      "  391  20295 576D 577E C   N    4486 4488 -177.67 -117.80  109.40    4.01  -63.60  146.13   25.04\n",
      "  391        577E 577E N   CA   4488 4489 -131.64  136.80                  -40.30\n",
      "  392  20297 578V 579S C   N    4502 4504 -162.71 -136.60   49.89    1.85  -64.10  164.19   19.01\n",
      "  392        579S 579S N   CA   4504 4505 -166.28  151.20                  -35.00\n",
      "  393  20298 579S 580M C   N    4508 4510  -94.28  -73.00   48.49    3.03  -63.40  136.47   22.84\n",
      "  393        580M 580M N   CA   4510 4511 -173.43  143.00                  -40.50\n",
      "  394  20299 580M 581P C   N    4516 4518  125.77  -64.50  173.14   16.15  -64.50  173.14   16.15\n",
      "  394        581P 581P N   CA   4518 4519 -178.61  147.20                  147.20\n",
      "  395  20300 581P 582E C   N    4523 4525  113.61 -117.80  131.51    4.21  -63.60 -124.35   28.64\n",
      "  395        582E 582E N   CA   4525 4526  164.36  136.80                  -40.30\n",
      "  396  20301 582E 583A C   N    4532 4534 -129.29 -134.00   16.42    1.00  -62.50  170.04   31.44\n",
      "  396        583A 583A N   CA   4534 4535  162.72  147.00                  -40.90\n",
      "  397  20302 583A 584Q C   N    4537 4539 -173.33 -121.10   67.01    2.10  -63.80  176.20   31.27\n",
      "  397        584Q 584Q N   CA   4539 4540 -178.32  139.70                  -40.30\n",
      "  398  20303 584Q 585E C   N    4546 4548 -176.51 -117.80   60.72    1.89  -63.60 -158.08   33.06\n",
      "  398        585E 585E N   CA   4548 4549  152.30  136.80                  -40.30\n",
      "  399  20304 585E 586D C   N    4555 4557 -116.18  -70.90   73.92    2.66  -63.30  123.21   18.62\n",
      "  399        586D 586D N   CA   4557 4558 -151.28  150.30                  -40.00\n",
      "  400  20305 586D 587L C   N    4563 4565 -100.37  -70.70   58.09    3.60  -63.50  132.50   20.67\n",
      "  400        587L 587L N   CA   4565 4566 -168.46  141.60                  -41.20\n",
      "  401  20306 587L 588P C   N    4571 4573 -141.44  -64.50   95.26    6.72  -64.50   95.26    6.72\n",
      "  401        588P 588P N   CA   4573 4574 -156.64  147.20                  147.20\n",
      "  402  20307 588P 589H C   N    4578 4580  173.46 -125.60   80.55    1.98  -63.20  176.48   29.55\n",
      "  402        589H 589H N   CA   4580 4581 -168.53  138.80                  -42.30\n",
      "  403  20308 589H 590K C   N    4588 4590  121.48   56.60  116.11   13.15  -62.90 -111.58   30.20\n",
      "  403        590K 590K N   CA   4590 4591  134.90   38.60                  -40.80\n",
      "  404  20310 591P 592T C   N    4604 4606   17.36  -63.20  102.32   11.78  -63.20  102.32   11.78\n",
      "  404        592T 592T N   CA   4606 4607 -105.19  -42.10                  -42.10\n",
      "  405  20311 592T 593S C   N    4611 4613   46.27  -72.40  120.11    9.06  -64.10 -170.48   13.38\n",
      "  405        593S 593S N   CA   4613 4614  170.93  152.40                  -35.00\n",
      "  406  20312 593S 594S C   N    4617 4619 -171.45 -136.60   42.56    2.28  -64.10 -165.84   13.34\n",
      "  406        594S 594S N   CA   4619 4620  126.78  151.20                  -35.00\n",
      "  407  20313 594S 595S C   N    4623 4625  136.57 -136.60  103.71    5.51  -64.10 -154.69   18.12\n",
      "  407        595S 595S N   CA   4625 4626   94.49  151.20                  -35.00\n",
      "  408  20314 595S 596M C   N    4629 4631   33.89  -73.00  124.38    9.97  -63.40  149.03   18.82\n",
      "  408        596M 596M N   CA   4631 4632 -153.39  143.00                  -40.50\n",
      "  409  20315 596M 597K C   N    4637 4639 -173.67 -118.00   71.16    2.42  -62.90  175.22   29.21\n",
      "  409        597K 597K N   CA   4639 4640 -176.57  139.10                  -40.80\n",
      "  410  20316 597K 598T C   N    4646 4648 -174.75 -124.80   89.61    2.78  -63.20  149.81   25.60\n",
      "  410        598T 598T N   CA   4648 4649 -142.10  143.50                  -42.10\n",
      "  411  20318 599V 600D C   N    4660 4662  155.85  -96.50  107.88    4.40  -63.30 -156.36   22.81\n",
      "  411        600D 600D N   CA   4662 4663  107.08  114.20                  -40.00\n",
      "  412  20319 600D 601E C   N    4668 4670  117.75  -63.60 -176.23   30.79  -63.60 -176.23   30.79\n",
      "  412        601E 601E N   CA   4670 4671  -83.38  -40.30                  -40.30\n",
      "  413  20320 601E 602E C   N    4677 4679   81.97  -69.30  154.14   10.61  -63.60 -148.66   35.69\n",
      "  413        602E 602E N   CA   4679 4680  112.91  142.50                  -40.30\n",
      "  414  20321 602E 603A C   N    4686 4688   70.96  -62.50  141.67   24.84  -62.50  141.67   24.84\n",
      "  414        603A 603A N   CA   4688 4689  -88.45  -40.90                  -40.90\n",
      "  415  20322 603A 604L C   N    4691 4693  -40.21  -70.70   39.27    3.95  -63.50  154.22   20.20\n",
      "  415        604L 604L N   CA   4693 4694  166.35  141.60                  -41.20\n",
      "  416  20323 604L 605A C   N    4699 4701  100.66 -134.00  125.48    3.93  -62.50 -127.13   35.93\n",
      "  416        605A 605A N   CA   4701 4702  152.95  147.00                  -40.90\n",
      "  417  20324 605A 606M C   N    4704 4706    3.87  -63.40  141.10   17.46  -63.40  141.10   17.46\n",
      "  417        606M 606M N   CA   4706 4707 -164.54  -40.50                  -40.50\n",
      "  418  20325 606M 607L C   N    4712 4714  -75.14  -63.50  122.24   17.66  -63.50  122.24   17.66\n",
      "  418        607L 607L N   CA   4714 4715 -162.88  -41.20                  -41.20\n",
      "  419  20326 607L 608L C   N    4720 4722  -53.04  -70.70   73.34    6.24  -63.50  106.53   14.22\n",
      "  419        608L 608L N   CA   4722 4723 -147.22  141.60                  -41.20\n",
      "  420  20327 608L 609S C   N    4728 4730 -146.36 -136.60   65.14    3.09  -64.10  136.87   15.85\n",
      "  420        609S 609S N   CA   4730 4731 -144.39  151.20                  -35.00\n",
      "  421  20328 609S 610M C   N    4734 4736 -166.32 -125.60   43.27    1.38  -63.40 -166.05   36.86\n",
      "  421        610M 610M N   CA   4736 4737  155.11  140.50                  -40.50\n",
      "  422  20329 610M 611G C   N    4742 4744 -126.18 -167.20   44.11    1.27   82.20 -126.42    6.77\n",
      "  422        611G 611G N   CA   4744 4745 -169.18  174.60                    8.50\n",
      "  423  20331 612I 613E C   N    4754 4756 -157.08 -117.80   69.35    2.50  -63.60  156.69   25.91\n",
      "  423        613E 613E N   CA   4756 4757 -166.05  136.80                  -40.30\n",
      "  424  20332 613E 614E C   N    4763 4765 -172.40 -117.80   71.64    2.21  -63.60  174.56   29.06\n",
      "  424        614E 614E N   CA   4765 4766 -176.81  136.80                  -40.30\n",
      "  425  20333 614E 615T C   N    4772 4774 -145.45 -124.80   38.30    1.19  -63.20  164.23   26.33\n",
      "  425        615T 615T N   CA   4774 4775  175.76  143.50                  -42.10\n",
      "  426  20335 616V 617A C   N    4786 4788  153.28 -134.00   74.06    1.97  -62.50 -146.01   42.48\n",
      "  426        617A 617A N   CA   4788 4789  161.01  147.00                  -40.90\n",
      "  427  20336 617A 618R C   N    4791 4793  150.40 -125.20   86.26    3.02  -63.00 -142.67   38.63\n",
      "  427        618R 618R N   CA   4793 4794  158.45  140.60                  -41.10\n",
      "  428  20337 618R 619Y C   N    4802 4804 -117.39 -124.30   49.65    2.93  -63.50  142.61   25.31\n",
      "  428        619Y 619Y N   CA   4804 4805 -175.43  135.40                  -43.40\n",
      "  429  20338 619Y 620A C   N    4814 4816 -123.04 -134.00   29.24    1.80  -62.50  157.12   28.99\n",
      "  429        620A 620A N   CA   4816 4817  174.11  147.00                  -40.90\n",
      "  430  20339 620A 621L C   N    4819 4821  173.14 -108.50   78.46    3.78  -63.50 -150.08   26.01\n",
      "  430        621L 621L N   CA   4821 4822  128.65  132.50                  -41.20\n",
      "  431  20340 621L 622L C   N    4827 4829 -112.29 -108.50   37.91    2.01  -63.50  156.38   24.73\n",
      "  431        622L 622L N   CA   4829 4830  170.22  132.50                  -41.20\n",
      "  432  20341 622L 623Q C   N    4835 4837 -168.14 -121.10   54.54    1.68  -63.80 -175.32   32.39\n",
      "  432        623Q 623Q N   CA   4837 4838  167.31  139.70                  -40.30\n",
      "  433  20342 623Q 624T C   N    4844 4846 -148.08 -124.80   62.08    2.05  -63.20  144.43   23.81\n",
      "  433        624T 624T N   CA   4846 4847 -158.96  143.50                  -42.10\n",
      "  434  20343 624T 625G C   N    4851 4853  170.86 -167.20   22.42    0.55   82.20 -175.80   12.92\n",
      "  434        625G 625G N   CA   4853 4854  169.96  174.60                    8.50\n",
      "  435  20344 625G 626M C   N    4855 4857 -168.60 -125.60   54.63    1.62  -63.40  179.39   34.71\n",
      "  435        626M 626M N   CA   4857 4858  174.20  140.50                  -40.50\n",
      "  436  20345 626M 627N C   N    4863 4865 -174.08 -119.90   69.31    2.01  -63.20  177.56   28.15\n",
      "  436        627N 627N N   CA   4865 4866 -179.78  137.00                  -41.10\n",
      "  437  20346 627N 628A C   N    4871 4873  179.67 -134.00   48.05    1.18  -62.50 -161.83   38.67\n",
      "  437        628A 628A N   CA   4873 4874  159.76  147.00                  -40.90\n",
      "  438  20347 628A 629E C   N    4876 4878 -161.52 -117.80   62.39    2.01  -63.60  169.53   27.90\n",
      "  438        629E 629E N   CA   4878 4879 -178.69  136.80                  -40.30\n",
      "  439  20348 629E 630R C   N    4885 4887 -144.81 -125.20   40.60    1.45  -63.00  164.53   27.73\n",
      "  439        630R 630R N   CA   4887 4888  176.15  140.60                  -41.10\n",
      "  440  20349 630R 631A C   N    4896 4898 -174.88 -134.00   43.35    1.01  -62.50 -166.36   37.66\n",
      "  440        631A 631A N   CA   4898 4899  161.41  147.00                  -40.90\n",
      "  441  20351 632V 633D C   N    4908 4910 -176.50  -96.50   95.20    3.96  -63.30 -168.72   30.74\n",
      "  441        633D 633D N   CA   4910 4911  165.81  114.20                  -40.00\n",
      "  442  20352 633D 634Y C   N    4916 4918 -139.67 -124.30   27.76    1.10  -63.50  175.47   31.58\n",
      "  442        634Y 634Y N   CA   4918 4919  158.52  135.40                  -43.40\n",
      "  443  20354 635V 636F C   N    4935 4937 -150.53 -124.20   32.32    0.84  -63.20  176.74   30.06\n",
      "  443        636F 636F N   CA   4937 4938  162.04  143.30                  -44.30\n",
      "  444  20355 636F 637S C   N    4946 4948 -159.89 -136.60   36.91    1.26  -64.10  173.93   19.52\n",
      "  444        637S 637S N   CA   4948 4949  179.83  151.20                  -35.00\n",
      "  445  20356 637S 638H C   N    4952 4954 -149.07 -125.60   40.45    1.06  -63.20  169.34   26.70\n",
      "  445        638H 638H N   CA   4954 4955  171.74  138.80                  -42.30\n",
      "  446  20357 638H 639E C   N    4962 4964 -140.72 -117.80   30.53    0.95  -63.60 -179.93   28.33\n",
      "  446        639E 639E N   CA   4964 4965  156.98  136.80                  -40.30\n",
      "  447  20358 639E 640N C   N    4971 4973 -140.31 -119.90   30.64    0.91  -63.20  176.75   26.44\n",
      "  447        640N 640N N   CA   4973 4974  159.86  137.00                  -41.10\n",
      "  448  20360 641I 642A C   N    4987 4989 -136.52 -134.00   16.23    0.86  -62.50  172.74   32.27\n",
      "  448        642A 642A N   CA   4989 4990  163.03  147.00                  -40.90\n",
      "  449  20361 642A 643E C   N    4992 4994 -131.88 -117.80   40.40    1.71  -63.60  160.30   25.20\n",
      "  449        643E 643E N   CA   4994 4995  174.67  136.80                  -40.30\n",
      "  450  20362 643E 644E C   N    5001 5003 -152.83 -117.80   41.39    1.22  -63.60 -176.05   29.47\n",
      "  450        644E 644E N   CA   5003 5004  158.85  136.80                  -40.30\n",
      "  451  20363 644E 645A C   N    5010 5012 -143.01 -134.00   26.24    1.22  -62.50  168.01   31.84\n",
      "  451        645A 645A N   CA   5012 5013  171.64  147.00                  -40.90\n",
      "  452  20364 645A 646G C   N    5015 5017 -155.62 -167.20   21.81    0.62   82.20 -168.38   14.39\n",
      "  452        646G 646G N   CA   5017 5018  156.12  174.60                    8.50\n",
      "  453  20365 646G 647L C   N    5019 5021 -134.30 -108.50   28.42    1.26  -63.50 -171.81   30.52\n",
      "  453        647L 647L N   CA   5021 5022  144.43  132.50                  -41.20\n",
      "  454  20366 647L 648A C   N    5027 5029 -132.68 -134.00    5.74    0.35  -62.50 -179.30   33.37\n",
      "  454        648A 648A N   CA   5029 5030  152.59  147.00                  -40.90\n",
      "  455  20367 648A 649E C   N    5032 5034 -135.65 -117.80   26.90    0.90  -63.60  178.00   27.78\n",
      "  455        649E 649E N   CA   5034 5035  156.93  136.80                  -40.30\n",
      "  456  20369 650I 651S C   N    5049 5051 -144.44 -136.60   13.90    0.50  -64.10 -178.89   18.88\n",
      "  456        651S 651S N   CA   5051 5052  162.68  151.20                  -35.00\n",
      "  457  20370 651S 652A C   N    5055 5057 -152.15 -134.00   20.57    0.47  -62.50 -174.47   35.19\n",
      "  457        652A 652A N   CA   5057 5058  156.67  147.00                  -40.90\n",
      "  458  20371 652A 653T C   N    5060 5062 -137.22 -124.80   21.95    0.68  -63.20  172.94   27.01\n",
      "  458        653T 653T N   CA   5062 5063  161.60  143.50                  -42.10\n",
      "  459  20372 653T 654A C   N    5067 5069 -150.64 -134.00   19.72    0.47  -62.50 -176.00   34.86\n",
      "  459        654A 654A N   CA   5069 5070  157.59  147.00                  -40.90\n",
      "  460  20373 654A 655S C   N    5072 5074 -144.44 -136.60    9.60    0.29  -64.10 -173.55   19.25\n",
      "  460        655S 655S N   CA   5074 5075  156.74  151.20                  -35.00\n",
      "  461  20374 655S 656E C   N    5078 5080 -146.79 -117.80   36.43    1.10  -63.60 -178.92   28.78\n",
      "  461        656E 656E N   CA   5080 5081  158.86  136.80                  -40.30\n",
      "  462  20375 656E 657S C   N    5087 5089 -143.06 -136.60    8.43    0.26  -64.10 -174.03   19.12\n",
      "  462        657S 657S N   CA   5089 5090  156.62  151.20                  -35.00\n",
      "  463  20376 657S 658Q C   N    5093 5095 -139.76 -121.10   19.82    0.63  -63.80 -170.78   31.81\n",
      "  463        658Q 658Q N   CA   5095 5096  146.39  139.70                  -40.30\n",
      "  464  20378 659P 660A C   N    5109 5111 -142.61 -134.00   11.82    0.34  -62.50 -177.47   34.20\n",
      "  464        660A 660A N   CA   5111 5112  155.09  147.00                  -40.90\n",
      "  465  20379 660A 661H C   N    5114 5116 -139.08 -125.60   21.55    0.55  -63.20  178.97   27.32\n",
      "  465        661H 661H N   CA   5116 5117  155.61  138.80                  -42.30\n",
      "  466  20381 662V 663L C   N    5131 5133 -124.62 -108.50   19.15    0.85  -63.50 -173.73   29.64\n",
      "  466        663L 663L N   CA   5133 5134  142.84  132.50                  -41.20\n",
      "  467  20382 663L 664D C   N    5139 5141 -125.02  -96.50   39.63    1.66  -63.30 -171.34   27.28\n",
      "  467        664D 664D N   CA   5141 5142  141.72  114.20                  -40.00\n",
      "  468  20383 664D 665G C   N    5147 5149 -153.35 -167.20   51.62    1.80   82.20  170.38   13.14\n",
      "  468        665G 665G N   CA   5149 5150  124.87  174.60                    8.50\n",
      "  469  20385 666P 667A C   N    5158 5160 -155.28 -134.00   21.60    0.59  -62.50 -167.70   36.47\n",
      "  469        667A 667A N   CA   5160 5161  150.66  147.00                  -40.90\n",
      "  470  20386 667A 668K C   N    5163 5165 -151.09 -118.00   33.33    1.19  -62.90 -163.04   30.85\n",
      "  470        668K 668K N   CA   5165 5166  143.09  139.10                  -40.80\n",
      "  471  20387 668K 669Y C   N    5172 5174 -142.85 -124.30   27.12    1.17  -63.50  179.87   32.42\n",
      "  471        669Y 669Y N   CA   5174 5175  155.18  135.40                  -43.40\n",
      "  472  20388 669Y 670R C   N    5184 5186 -146.01 -125.20   21.71    0.72  -63.00 -168.92   31.47\n",
      "  472        670R 670R N   CA   5186 5187  146.79  140.60                  -41.10\n",
      "  473  20389 670R 671L C   N    5195 5197 -143.08 -108.50   40.45    1.78  -63.50 -176.53   30.40\n",
      "  473        671L 671L N   CA   5197 5198  153.49  132.50                  -41.20\n",
      "  474  20390 671L 672H C   N    5203 5205 -145.79 -125.60   23.58    0.60  -63.20 -173.95   28.63\n",
      "  474        672H 672H N   CA   5205 5206  150.98  138.80                  -42.30\n",
      "  475  20391 672H 673A C   N    5213 5215 -158.07 -134.00   26.15    0.60  -62.50 -172.02   35.90\n",
      "  475        673A 673A N   CA   5215 5216  157.23  147.00                  -40.90\n",
      "  476  20392 673A 674M C   N    5218 5220 -144.98 -125.60   21.29    0.65  -63.40 -171.28   34.58\n",
      "  476        674M 674M N   CA   5220 5221  149.32  140.50                  -40.50\n",
      "  477  20394 675I 676S C   N    5234 5236 -148.54 -136.60   13.05    0.41  -64.10 -171.49   19.71\n",
      "  477        676S 676S N   CA   5236 5237  156.46  151.20                  -35.00\n",
      "  478  20395 676S 677H C   N    5240 5242 -153.99 -125.60   33.57    0.85  -63.20 -175.19   28.97\n",
      "  478        677H 677H N   CA   5242 5243  156.72  138.80                  -42.30\n",
      "  479  20397 678V 679G C   N    5257 5259  174.00 -167.20   26.75    1.00   82.20  173.36   12.45\n",
      "  479        679G 679G N   CA   5259 5260  155.57  174.60                    8.50\n",
      "  480  20398 679G 680A C   N    5261 5263 -165.78 -134.00   34.47    0.79  -62.50 -170.61   36.52\n",
      "  480        680A 680A N   CA   5263 5264  160.35  147.00                  -40.90\n",
      "  481  20399 680A 681S C   N    5266 5268 -154.52 -136.60   20.55    0.63  -64.10 -172.95   20.06\n",
      "  481        681S 681S N   CA   5268 5269  161.26  151.20                  -35.00\n",
      "  482  20400 681S 682A C   N    5272 5274 -157.48 -134.00   25.51    0.58  -62.50 -172.08   35.86\n",
      "  482        682A 682A N   CA   5274 5275  156.95  147.00                  -40.90\n",
      "  483  20401 682A 683K C   N    5277 5279 -147.64 -118.00   32.39    1.07  -62.90 -172.70   29.39\n",
      "  483        683K 683K N   CA   5279 5280  152.17  139.10                  -40.80\n",
      "  484  20402 683K 684T C   N    5286 5288 -146.18 -124.80   34.72    1.08  -63.20  168.84   26.98\n",
      "  484        684T 684T N   CA   5288 5289  170.86  143.50                  -42.10\n",
      "  485  20403 684T 685G C   N    5293 5295  174.65 -167.20   26.59    1.01   82.20  173.37   12.48\n",
      "  485        685G 685G N   CA   5295 5296  155.17  174.60                    8.50\n",
      "  486  20404 685G 686H C   N    5297 5299 -152.54 -125.60   28.95    0.80  -63.20 -169.45   29.61\n",
      "  486        686H 686H N   CA   5299 5300  149.39  138.80                  -42.30\n",
      "  487  20405 686H 687Y C   N    5307 5309 -142.90 -124.30   25.46    1.16  -63.50 -177.95   32.78\n",
      "  487        687Y 687Y N   CA   5309 5310  152.78  135.40                  -43.40\n",
      "  488  20408 689C 690H C   N    5332 5334 -127.46 -125.60   88.90    3.42  -63.20  112.40   12.04\n",
      "  488        690H 690H N   CA   5334 5335   49.92  138.80                  -42.30\n",
      "  489  20411 692C 693D C   N    5356 5358  142.73  -96.50  124.71    5.12  -63.30 -127.13   38.24\n",
      "  489        693D 693D N   CA   5358 5359  145.30  114.20                  -40.00\n",
      "  490  20412 693D 694A C   N    5364 5366 -173.25 -134.00   42.66    0.97  -62.50 -169.17   37.11\n",
      "  490        694A 694A N   CA   5366 5367  163.70  147.00                  -40.90\n",
      "  491  20413 694A 695Q C   N    5369 5371 -152.05 -121.10   44.31    1.45  -63.80  172.57   29.89\n",
      "  491        695Q 695Q N   CA   5371 5372  171.40  139.70                  -40.30\n",
      "  492  20414 695Q 696T C   N    5378 5380 -151.00 -124.80   42.92    1.33  -63.20  165.60   26.81\n",
      "  492        696T 696T N   CA   5380 5381  177.49  143.50                  -42.10\n",
      "  493  20415 696T 697G C   N    5385 5387 -164.40 -167.20    8.68    0.36   82.20 -152.05   15.05\n",
      "  493        697G 697G N   CA   5387 5388 -177.19  174.60                    8.50\n",
      "  494  20416 697G 698K C   N    5389 5391 -167.16 -118.00   52.11    1.75  -62.90 -166.67   31.29\n",
      "  494        698K 698K N   CA   5391 5392  156.40  139.10                  -40.80\n",
      "  495  20417 698K 699W C   N    5398 5400 -154.84 -124.90   49.29    1.65  -63.00  161.83   26.37\n",
      "  495        699W 699W N   CA   5400 5401 -177.45  143.40                  -44.20\n",
      "  496  20418 699W 700L C   N    5412 5414 -159.36 -108.50   68.47    3.06  -63.50  170.05   29.45\n",
      "  496        700L 700L N   CA   5414 5415  178.34  132.50                  -41.20\n",
      "  497  20419 700L 701L C   N    5420 5422 -156.37 -108.50   74.15    3.39  -63.50  159.49   27.78\n",
      "  497        701L 701L N   CA   5422 5423 -170.87  132.50                  -41.20\n",
      "  498  20420 701L 702F C   N    5428 5430 -150.49 -124.20   50.52    1.61  -63.20  155.98   27.03\n",
      "  498        702F 702F N   CA   5430 5431 -173.56  143.30                  -44.30\n",
      "  499  20421 702F 703N C   N    5439 5441 -163.53 -119.90   61.82    1.81  -63.20  170.70   26.80\n",
      "  499        703N 703N N   CA   5441 5442 -179.20  137.00                  -41.10\n",
      "  500  20422 703N 704D C   N    5447 5449 -164.06  -96.50   91.30    3.81  -63.30  176.07   28.10\n",
      "  500        704D 704D N   CA   5449 5450  175.61  114.20                  -40.00\n",
      "  501  20423 704D 705E C   N    5455 5457 -149.63 -117.80   50.11    1.71  -63.60  167.91   27.12\n",
      "  501        705E 705E N   CA   5457 5458  175.51  136.80                  -40.30\n",
      "  502  20424 705E 706K C   N    5464 5466 -161.65 -118.00   72.35    2.70  -62.90  157.26   26.19\n",
      "  502        706K 706K N   CA   5466 5467 -163.19  139.10                  -40.80\n",
      "  503  20426 707V 708A C   N    5480 5482 -175.16 -134.00   47.09    1.09  -62.50 -173.03   36.55\n",
      "  503        708A 708A N   CA   5482 5483  169.88  147.00                  -40.90\n",
      "  504  20427 708A 709E C   N    5485 5487 -152.42 -117.80   72.35    2.80  -63.60  148.79   24.61\n",
      "  504        709E 709E N   CA   5487 5488 -159.67  136.80                  -40.30\n",
      "  505  20428 709E 710S C   N    5494 5496  144.59 -136.60   88.70    2.72  -64.10 -158.50   25.34\n",
      "  505        710S 710S N   CA   5496 5497 -168.09  151.20                  -35.00\n",
      "  506  20429 710S 711L C   N    5500 5502 -117.13 -108.50   79.92    4.22  -63.50  119.55   19.91\n",
      "  506        711L 711L N   CA   5502 5503 -148.04  132.50                  -41.20\n",
      "  507  20430 711L 712N C   N    5508 5510  158.01 -119.90   90.20    2.85  -63.20 -159.64   32.28\n",
      "  507        712N 712N N   CA   5510 5511  174.39  137.00                  -41.10\n",
      "  508  20432 713P 714P C   N    5523 5525  175.00  -64.50  120.60   12.50  -64.50  120.60   12.50\n",
      "  508        714P 714P N   CA   5525 5526  142.18  147.20                  147.20\n",
      "  509  20433 714P 715F C   N    5530 5532  105.27 -124.20  134.76    4.29  -63.20 -141.65   25.96\n",
      "  509        715F 715F N   CA   5532 5533  176.79  143.30                  -44.30\n",
      "  510  20434 715F 716S C   N    5541 5543  122.01 -136.60  102.45    3.70  -64.10 -124.32   29.45\n",
      "  510        716S 716S N   CA   5543 5544  165.91  151.20                  -35.00\n",
      "  511  20435 716S 717M C   N    5547 5549 -122.31 -125.60   29.06    1.45  -63.40  161.27   28.75\n",
      "  511        717M 717M N   CA   5549 5550  169.37  140.50                  -40.50\n",
      "  512  20436 717M 718A C   N    5555 5557 -155.54 -134.00   24.30    0.56  -62.50 -174.15   35.42\n",
      "  512        718A 718A N   CA   5557 5558  158.23  147.00                  -40.90\n",
      "  513  20437 718A 719F C   N    5560 5562 -130.38 -124.20   43.01    1.87  -63.20  146.19   24.60\n",
      "  513        719F 719F N   CA   5562 5563 -174.14  143.30                  -44.30\n",
      "  514  20438 719F 720L C   N    5571 5573  125.74 -108.50  130.63    5.88  -63.50 -132.08   41.42\n",
      "  514        720L 720L N   CA   5573 5574  167.83  132.50                  -41.20\n",
      "  515  20439 720L 721Y C   N    5579 5581  152.55 -124.30   83.15    6.23  -63.50 -129.97   31.74\n",
      "  515        721Y 721Y N   CA   5581 5582  136.03  135.40                  -43.40\n",
      "  516  20440 721Y 722F C   N    5591 5593 -174.77 -124.20  105.27    3.48  -63.20  137.33   24.78\n",
      "  516        722F 722F N   CA   5593 5594 -124.37  143.30                  -44.30\n",
      "  517  20441 722F 723Y C   N    5602 5604  173.44 -124.30   75.04    3.87  -63.50 -174.12   34.89\n",
      "  517        723Y 723Y N   CA   5604 5605  177.29  135.40                  -43.40\n",
      "  518  20442 723Y 724K C   N    5614 5616 -173.74 -118.00   61.13    2.03  -62.90 -169.46   31.28\n",
      "  518        724K 724K N   CA   5616 5617  164.21  139.10                  -40.80\n",
      "  519  20443 724K 725R C   N    5623 5625 -105.16 -125.20   23.18    0.69  -63.00  175.20   21.15\n",
      "  519        725R 725R N   CA   5625 5626  128.95  140.60                  -41.10\n",
      "  520  20445 726V 727G C   N    5641 5643 -102.24  -80.20   33.83    0.97   82.20 -116.50    7.79\n",
      "  520        727G 727G N   CA   5643 5644 -160.23  174.10                    8.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  28118 233P 607L CB  CD2  1763 4719   13.67    4.45    9.22    5.81    4.45    9.22    5.81\n",
      "    2  28184 252F 606M CB  SD   1910 4710   27.89    4.79   23.10   18.27    4.79   23.10   18.27\n",
      "    3  28185 253A 606M CB  CG   1921 4709   22.18    4.50   17.68   11.70    4.50   17.68   11.70\n",
      "    4  28186 253A 606M CB  SD   1921 4710   24.03    3.87   20.16   13.69    3.87   20.16   13.69\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   50   97  352  407  473  435  486  508  612  635\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00192.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      728     5656     5656\n",
      "openf___224_> Open           EAO00192.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18229916   17802.652    17.385\n",
      "read_mo_297_> Segments, residues, atoms:        1      728     5656\n",
      "read_mo_298_> Segment:        1  1:A  728:A     5656\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     5656     5656        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    23948\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24395\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    24673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    25693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    26848\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27084\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    27270\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28042\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28044\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28188\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     5656    28194\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.8610        5  5\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.4883       10  10\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.4442       17  17\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.9776       18  18\n",
      "iupac_m_486_> OE1/2 will be swapped:     -106.8463       34  34\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.1529       45  45\n",
      "iupac_m_486_> OE1/2 will be swapped:     -131.9317       61  61\n",
      "iupac_m_485_> OD1/2 will be swapped:     -161.2804       86  86\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.0535       87  87\n",
      "iupac_m_485_> OD1/2 will be swapped:     -136.5060       88  88\n",
      "iupac_m_485_> OD1/2 will be swapped:      134.9953       94  94\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.9886      128  128\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -101.4739      170  170\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.0419      186  186\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      107.1646      241  241\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.1629      245  245\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.2818      246  246\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.9076      275  275\n",
      "iupac_m_485_> OD1/2 will be swapped:       94.1691      279  279\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5918      288  288\n",
      "iupac_m_485_> OD1/2 will be swapped:     -152.4290      294  294\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.7324      327  327\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      347  347\n",
      "iupac_m_487_> NH1/2 swapped:      179.7797      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.2125      368  368\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9374      375  375\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.3216      384  384\n",
      "iupac_m_487_> NH1/2 swapped:      179.9051      387  387\n",
      "iupac_m_485_> OD1/2 will be swapped:      180.0000      395  395\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.3931      406  406\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.1621      434  434\n",
      "iupac_m_487_> NH1/2 swapped:      179.9802      443  443\n",
      "iupac_m_487_> NH1/2 swapped:      179.7230      453  453\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5166      456  456\n",
      "iupac_m_486_> OE1/2 will be swapped:      180.0000      457  457\n",
      "iupac_m_485_> OD1/2 will be swapped:      148.1046      459  459\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.6169      469  469\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.8571      476  476\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.6028      484  484\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.1062      485  485\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.3283      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      120.6042      509  509\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4949      526  526\n",
      "iupac_m_487_> NH1/2 swapped:      179.7888      528  528\n",
      "iupac_m_485_> OD1/2 will be swapped:      169.9538      532  532\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7070      540  540\n",
      "iupac_m_485_> OD1/2 will be swapped:      130.9591      544  544\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.8539      554  554\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      106.9963      556  556\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      112.5934      558  558\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.4194      562  562\n",
      "iupac_m_485_> OD1/2 will be swapped:       98.0222      564  564\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      110.9337      567  567\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.7620      569  569\n",
      "iupac_m_485_> OD1/2 will be swapped:     -120.7421      576  576\n",
      "iupac_m_486_> OE1/2 will be swapped:      165.6761      585  585\n",
      "iupac_m_485_> OD1/2 will be swapped:     -149.0040      586  586\n",
      "iupac_m_485_> OD1/2 will be swapped:     -122.7248      600  600\n",
      "iupac_m_486_> OE1/2 will be swapped:     -160.6864      601  601\n",
      "iupac_m_486_> OE1/2 will be swapped:      126.6030      602  602\n",
      "iupac_m_486_> OE1/2 will be swapped:     -159.4137      613  613\n",
      "iupac_m_487_> NH1/2 swapped:      153.3216      618  618\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -111.7693      619  619\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.3526      630  630\n",
      "iupac_m_486_> OE1/2 will be swapped:     -150.0215      639  639\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.0239      644  644\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9011      649  649\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.7944      656  656\n",
      "iupac_m_485_> OD1/2 will be swapped:      162.7204      664  664\n",
      "iupac_m_487_> NH1/2 swapped:      179.6478      670  670\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.1437      702  702\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.5688      705  705\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      135.5612      719  719\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.7284      721  721\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -98.4579      722  722\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      142.6076      723  723\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   476895\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -18607.2324\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -18607.232422\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    53.912998\n",
      "Sequence length          :          728\n",
      "Compactness              :     0.000335\n",
      "Native energy (pair)     :  1001.331760\n",
      "Native energy (surface)  :   114.493981\n",
      "Native energy (combined) :    31.361687\n",
      "Z score (pair)           :    -1.565269\n",
      "Z score (surface)        :    -0.654882\n",
      "Z score (combined)       :    -1.555245\n",
      "GA341 score              :     0.323514\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      728\n",
      "Number of all, selected real atoms                :     5656    5656\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28194   28194\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     8804\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      389528.9375\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    5789     432    546   0.431   0.431     0.31806E+06   1.000\n",
      " 2 Bond angle potential               :    7845     506    851  10.106  10.106      16795.       1.000\n",
      " 3 Stereochemical cosine torsion poten:    3676       0    301  56.310  56.310      2036.1       1.000\n",
      " 4 Stereochemical improper torsion pot:    2409       8     33   2.674   2.674      342.65       1.000\n",
      " 5 Soft-sphere overlap restraints     :    8804       3     10   0.010   0.010      96.576       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    1992     222    406   4.647   4.647      25696.       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2093     192    383   5.123   5.123      23170.       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     727      11     46   6.333   6.333      343.89       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     627       0     52  88.663  88.663      422.24       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     449       0     27  77.769  77.769      325.68       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     214       0      6 108.320 108.320      163.41       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      2  92.682  92.682      49.772       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1112       0      0   0.753   0.753      53.514       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     726     504    233  52.956 152.339      1640.8       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     471       4     22   2.826   2.826      338.10       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00192.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00192.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  231951.4688\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     84  10E  10E C   CA     81   75    1.65    1.49    0.16    4.66    1.49    0.16    4.66\n",
      "    2     89  11A  11A C   CA     86   84    1.66    1.49    0.17    4.95    1.49    0.17    4.95\n",
      "    3     93  12C  12C N   CA     88   89    1.58    1.43    0.15    4.77    1.43    0.15    4.77\n",
      "    4     95  12C  12C C   CA     92   89    1.68    1.49    0.19    5.40    1.49    0.19    5.40\n",
      "    5     96  12C  13A C   N      92   94    1.48    1.35    0.14    4.83    1.35    0.14    4.83\n",
      "    6     98  13A  13A N   CA     94   95    1.59    1.43    0.16    5.18    1.43    0.16    5.18\n",
      "    7    100  13A  13A C   CA     97   95    1.69    1.49    0.20    5.68    1.49    0.20    5.68\n",
      "    8    101  13A  14H C   N      97   99    1.49    1.35    0.14    5.10    1.35    0.14    5.10\n",
      "    9    108  14H  14H N   CA     99  100    1.60    1.43    0.17    5.48    1.43    0.17    5.48\n",
      "   10    110  14H  14H C   CA    107  100    1.71    1.49    0.22    6.28    1.49    0.22    6.28\n",
      "   11    111  14H  15C C   N     107  109    1.50    1.35    0.16    5.57    1.35    0.16    5.57\n",
      "   12    115  15C  15C N   CA    109  110    1.61    1.43    0.18    6.01    1.43    0.18    6.01\n",
      "   13    117  15C  15C C   CA    113  110    1.72    1.49    0.23    6.70    1.49    0.23    6.70\n",
      "   14    118  15C  16W C   N     113  115    1.52    1.35    0.17    6.13    1.35    0.17    6.13\n",
      "   15    128  16W  16W N   CA    115  116    1.63    1.43    0.20    6.52    1.43    0.20    6.52\n",
      "   16    130  16W  16W C   CA    127  116    1.76    1.49    0.27    7.87    1.49    0.27    7.87\n",
      "   17    131  16W  17D C   N     127  129    1.54    1.35    0.20    7.09    1.35    0.20    7.09\n",
      "   18    139  17D  17D N   CA    129  130    1.66    1.43    0.23    7.70    1.43    0.23    7.70\n",
      "   19    141  17D  17D C   CA    135  130    1.78    1.49    0.29    8.41    1.49    0.29    8.41\n",
      "   20    142  17D  18E C   N     135  137    1.55    1.35    0.21    7.41    1.35    0.21    7.41\n",
      "   21    148  18E  18E N   CA    137  138    1.67    1.43    0.24    8.01    1.43    0.24    8.01\n",
      "   22    150  18E  18E C   CA    144  138    1.80    1.49    0.31    8.92    1.49    0.31    8.92\n",
      "   23    151  18E  19S C   N     144  146    1.56    1.35    0.22    7.77    1.35    0.22    7.77\n",
      "   24    154  19S  19S N   CA    146  147    1.68    1.43    0.25    8.37    1.43    0.25    8.37\n",
      "   25    156  19S  19S C   CA    150  147    1.80    1.49    0.31    9.15    1.49    0.31    9.15\n",
      "   26    157  19S  20V C   N     150  152    1.57    1.35    0.23    8.04    1.35    0.23    8.04\n",
      "   27    161  20V  20V N   CA    152  153    1.69    1.43    0.26    8.67    1.43    0.26    8.67\n",
      "   28    163  20V  20V C   CA    157  153    1.81    1.49    0.32    9.44    1.49    0.32    9.44\n",
      "   29    164  20V  21M C   N     157  159    1.58    1.35    0.23    8.17    1.35    0.23    8.17\n",
      "   30    169  21M  21M N   CA    159  160    1.70    1.43    0.27    8.81    1.43    0.27    8.81\n",
      "   31    171  21M  21M C   CA    165  160    1.83    1.49    0.34    9.83    1.49    0.34    9.83\n",
      "   32    172  21M  22L C   N     165  167    1.58    1.35    0.24    8.44    1.35    0.24    8.44\n",
      "   33    177  22L  22L N   CA    167  168    1.71    1.43    0.28    9.08    1.43    0.28    9.08\n",
      "   34    179  22L  22L C   CA    173  168    1.84    1.49    0.35   10.06    1.49    0.35   10.06\n",
      "   35    180  22L  23V C   N     173  175    1.59    1.35    0.25    8.76    1.35    0.25    8.76\n",
      "   36    184  23V  23V N   CA    175  176    1.72    1.43    0.29    9.40    1.43    0.29    9.40\n",
      "   37    186  23V  23V C   CA    180  176    1.85    1.49    0.36   10.53    1.49    0.36   10.53\n",
      "   38    187  23V  24P C   N     180  182    1.64    1.30    0.34   10.17    1.30    0.34   10.17\n",
      "   39    189  24P  24P C   CA    187  183    1.85    1.49    0.36   10.37    1.49    0.36   10.37\n",
      "   40    190  24P  25N C   N     187  189    1.59    1.35    0.25    8.86    1.35    0.25    8.86\n",
      "   41    191  24P  24P N   CA    182  183    1.73    1.43    0.29    9.68    1.43    0.29    9.68\n",
      "   42    200  25N  25N N   CA    189  190    1.72    1.43    0.29    9.60    1.43    0.29    9.60\n",
      "   43    202  25N  25N C   CA    195  190    1.85    1.49    0.36   10.58    1.49    0.36   10.58\n",
      "   44    203  25N  26H C   N     195  197    1.60    1.35    0.26    9.07    1.35    0.26    9.07\n",
      "   45    210  26H  26H N   CA    197  198    1.73    1.43    0.30    9.87    1.43    0.30    9.87\n",
      "   46    212  26H  26H C   CA    205  198    1.87    1.49    0.38   11.07    1.49    0.38   11.07\n",
      "   47    213  26H  27R C   N     205  207    1.61    1.35    0.27    9.57    1.35    0.27    9.57\n",
      "   48    222  27R  27R N   CA    207  208    1.74    1.43    0.31   10.26    1.43    0.31   10.26\n",
      "   49    224  27R  27R C   CA    216  208    1.88    1.49    0.39   11.42    1.49    0.39   11.42\n",
      "   50    225  27R  28S C   N     216  218    1.62    1.35    0.28    9.77    1.35    0.28    9.77\n",
      "   51    228  28S  28S N   CA    218  219    1.75    1.43    0.32   10.63    1.43    0.32   10.63\n",
      "   52    230  28S  28S C   CA    222  219    1.90    1.49    0.41   11.82    1.49    0.41   11.82\n",
      "   53    231  28S  29A C   N     222  224    1.63    1.35    0.29   10.15    1.35    0.29   10.15\n",
      "   54    233  29A  29A N   CA    224  225    1.76    1.43    0.33   10.99    1.43    0.33   10.99\n",
      "   55    235  29A  29A C   CA    227  225    1.91    1.49    0.42   12.27    1.49    0.42   12.27\n",
      "   56    236  29A  30V C   N     227  229    1.65    1.35    0.30   10.75    1.35    0.30   10.75\n",
      "   57    240  30V  30V N   CA    229  230    1.79    1.43    0.36   11.69    1.43    0.36   11.69\n",
      "   58    242  30V  30V C   CA    234  230    1.94    1.49    0.45   13.06    1.49    0.45   13.06\n",
      "   59    243  30V  31H C   N     234  236    1.67    1.35    0.33   11.54    1.35    0.33   11.54\n",
      "   60    250  31H  31H N   CA    236  237    1.81    1.43    0.38   12.62    1.43    0.38   12.62\n",
      "   61    252  31H  31H C   CA    244  237    2.00    1.49    0.51   14.89    1.49    0.51   14.89\n",
      "   62    253  31H  32K C   N     244  246    1.70    1.35    0.35   12.42    1.35    0.35   12.42\n",
      "   63    260  32K  32K N   CA    246  247    1.84    1.43    0.41   13.40    1.43    0.41   13.40\n",
      "   64    262  32K  32K C   CA    253  247    2.01    1.49    0.52   15.21    1.49    0.52   15.21\n",
      "   65    263  32K  33E C   N     253  255    1.72    1.35    0.37   13.21    1.35    0.37   13.21\n",
      "   66    269  33E  33E N   CA    255  256    1.88    1.43    0.45   14.70    1.43    0.45   14.70\n",
      "   67    271  33E  33E C   CA    262  256    2.05    1.49    0.56   16.39    1.49    0.56   16.39\n",
      "   68    272  33E  34E C   N     262  264    1.72    1.35    0.38   13.30    1.35    0.38   13.30\n",
      "   69    278  34E  34E N   CA    264  265    1.87    1.43    0.44   14.45    1.43    0.44   14.45\n",
      "   70    280  34E  34E C   CA    271  265    2.09    1.49    0.60   17.34    1.49    0.60   17.34\n",
      "   71    281  34E  35C C   N     271  273    1.75    1.35    0.40   14.22    1.35    0.40   14.22\n",
      "   72    284  35C  35C N   CA    273  274    1.90    1.43    0.47   15.31    1.43    0.47   15.31\n",
      "   73    286  35C  35C C   CA    277  274    2.08    1.49    0.59   17.09    1.49    0.59   17.09\n",
      "   74    287  35C  36A C   N     277  279    1.71    1.35    0.37   13.05    1.35    0.37   13.05\n",
      "   75    289  36A  36A N   CA    279  280    1.90    1.43    0.47   15.56    1.43    0.47   15.56\n",
      "   76    291  36A  36A C   CA    282  280    2.08    1.49    0.59   17.14    1.49    0.59   17.14\n",
      "   77    292  36A  37Y C   N     282  284    1.78    1.35    0.43   15.30    1.35    0.43   15.30\n",
      "   78    302  37Y  37Y N   CA    284  285    1.96    1.43    0.53   17.33    1.43    0.53   17.33\n",
      "   79    304  37Y  37Y C   CA    294  285    2.27    1.49    0.78   22.79    1.49    0.78   22.79\n",
      "   80    305  37Y  38C C   N     294  296    1.84    1.35    0.49   17.52    1.35    0.49   17.52\n",
      "   81    308  38C  38C N   CA    296  297    2.01    1.43    0.58   19.10    1.43    0.58   19.10\n",
      "   82    310  38C  38C C   CA    300  297    2.21    1.49    0.72   20.96    1.49    0.72   20.96\n",
      "   83    311  38C  39C C   N     300  302    1.87    1.35    0.52   18.53    1.35    0.52   18.53\n",
      "   84    314  39C  39C N   CA    302  303    2.04    1.43    0.61   20.17    1.43    0.61   20.17\n",
      "   85    316  39C  39C C   CA    306  303    2.18    1.49    0.69   20.07    1.49    0.69   20.07\n",
      "   86    317  39C  40R C   N     306  308    1.83    1.35    0.49   17.20    1.35    0.49   17.20\n",
      "   87    325  40R  40R N   CA    308  309    2.01    1.43    0.58   19.12    1.43    0.58   19.12\n",
      "   88    327  40R  40R C   CA    317  309    2.24    1.49    0.75   21.70    1.49    0.75   21.70\n",
      "   89    328  40R  41T C   N     317  319    1.85    1.35    0.50   17.85    1.35    0.50   17.85\n",
      "   90    332  41T  41T N   CA    319  320    2.03    1.43    0.60   19.59    1.43    0.60   19.59\n",
      "   91    334  41T  41T C   CA    324  320    2.25    1.49    0.76   22.10    1.49    0.76   22.10\n",
      "   92    335  41T  42C C   N     324  326    1.87    1.35    0.53   18.75    1.35    0.53   18.75\n",
      "   93    338  42C  42C N   CA    326  327    2.05    1.43    0.62   20.43    1.43    0.62   20.43\n",
      "   94    340  42C  42C C   CA    330  327    2.29    1.49    0.80   23.31    1.49    0.80   23.31\n",
      "   95    341  42C  43R C   N     330  332    1.90    1.35    0.55   19.59    1.35    0.55   19.59\n",
      "   96    349  43R  43R N   CA    332  333    2.07    1.43    0.64   21.07    1.43    0.64   21.07\n",
      "   97    351  43R  43R C   CA    341  333    2.28    1.49    0.79   23.05    1.49    0.79   23.05\n",
      "   98    352  43R  44H C   N     341  343    1.89    1.35    0.54   19.20    1.35    0.54   19.20\n",
      "   99    359  44H  44H N   CA    343  344    2.07    1.43    0.64   21.16    1.43    0.64   21.16\n",
      "  100    361  44H  44H C   CA    351  344    2.36    1.49    0.87   25.21    1.49    0.87   25.21\n",
      "  101    362  44H  45E C   N     351  353    1.96    1.35    0.61   21.67    1.35    0.61   21.67\n",
      "  102    369  45E  45E N   CA    353  354    2.14    1.43    0.71   23.36    1.43    0.71   23.36\n",
      "  103    371  45E  45E C   CA    360  354    2.40    1.49    0.91   26.34    1.49    0.91   26.34\n",
      "  104    372  45E  46M C   N     360  362    1.97    1.35    0.62   22.00    1.35    0.62   22.00\n",
      "  105    377  46M  46M N   CA    362  363    2.18    1.43    0.75   24.82    1.43    0.75   24.82\n",
      "  106    379  46M  46M C   CA    368  363    2.48    1.49    0.99   28.70    1.49    0.99   28.70\n",
      "  107    380  46M  47G C   N     368  370    2.01    1.35    0.67   23.67    1.35    0.67   23.67\n",
      "  108    381  47G  47G N   CA    370  371    2.16    1.43    0.73   23.85    1.43    0.73   23.85\n",
      "  109    383  47G  47G C   CA    372  371    2.19    1.49    0.70   20.25    1.49    0.70   20.25\n",
      "  110    384  47G  48T C   N     372  374    1.67    1.35    0.32   11.40    1.35    0.32   11.40\n",
      "  111    388  48T  48T N   CA    374  375    1.78    1.43    0.35   11.52    1.43    0.35   11.52\n",
      "  112   1825 236A 236A C   CA   1780 1778    1.65    1.49    0.16    4.74    1.49    0.16    4.74\n",
      "  113   1845 238F 238F C   CA   1799 1791    1.67    1.49    0.18    5.12    1.49    0.18    5.12\n",
      "  114   1923 247V 247V C   CA   1875 1871    1.65    1.49    0.16    4.56    1.49    0.16    4.56\n",
      "  115   2043 261I 261I C   CA   1990 1985    1.78    1.49    0.29    8.32    1.49    0.29    8.32\n",
      "  116   2044 261I 262D C   N    1990 1992    1.57    1.35    0.22    7.93    1.35    0.22    7.93\n",
      "  117   2049 262D 262D N   CA   1992 1993    1.73    1.43    0.30   10.02    1.43    0.30   10.02\n",
      "  118   2051 262D 262D C   CA   1998 1993    1.87    1.49    0.38   11.15    1.49    0.38   11.15\n",
      "  119   2052 262D 263I C   N    1998 2000    1.60    1.35    0.26    9.17    1.35    0.26    9.17\n",
      "  120   2057 263I 263I N   CA   2000 2001    1.73    1.43    0.30    9.80    1.43    0.30    9.80\n",
      "  121   2059 263I 263I C   CA   2006 2001    1.85    1.49    0.36   10.48    1.49    0.36   10.48\n",
      "  122   2060 263I 264Q C   N    2006 2008    1.59    1.35    0.25    8.73    1.35    0.25    8.73\n",
      "  123   2066 264Q 264Q N   CA   2008 2009    1.72    1.43    0.29    9.44    1.43    0.29    9.44\n",
      "  124   2068 264Q 264Q C   CA   2015 2009    1.84    1.49    0.35   10.03    1.49    0.35   10.03\n",
      "  125   2069 264Q 265T C   N    2015 2017    1.58    1.35    0.23    8.25    1.35    0.23    8.25\n",
      "  126   2073 265T 265T N   CA   2017 2018    1.69    1.43    0.26    8.61    1.43    0.26    8.61\n",
      "  127   2075 265T 265T C   CA   2022 2018    1.82    1.49    0.33    9.65    1.49    0.33    9.65\n",
      "  128   2076 265T 266A C   N    2022 2024    1.57    1.35    0.22    7.96    1.35    0.22    7.96\n",
      "  129   2078 266A 266A N   CA   2024 2025    1.69    1.43    0.26    8.47    1.43    0.26    8.47\n",
      "  130   2080 266A 266A C   CA   2027 2025    1.80    1.49    0.31    8.97    1.49    0.31    8.97\n",
      "  131   2081 266A 267K C   N    2027 2029    1.56    1.35    0.21    7.57    1.35    0.21    7.57\n",
      "  132   2087 267K 267K N   CA   2029 2030    1.68    1.43    0.25    8.17    1.43    0.25    8.17\n",
      "  133   2089 267K 267K C   CA   2036 2030    1.79    1.49    0.30    8.62    1.49    0.30    8.62\n",
      "  134   2090 267K 268K C   N    2036 2038    1.55    1.35    0.21    7.43    1.35    0.21    7.43\n",
      "  135   2096 268K 268K N   CA   2038 2039    1.67    1.43    0.24    7.92    1.43    0.24    7.92\n",
      "  136   2098 268K 268K C   CA   2045 2039    1.78    1.49    0.29    8.40    1.49    0.29    8.40\n",
      "  137   2099 268K 269T C   N    2045 2047    1.55    1.35    0.20    7.21    1.35    0.20    7.21\n",
      "  138   2103 269T 269T N   CA   2047 2048    1.66    1.43    0.23    7.56    1.43    0.23    7.56\n",
      "  139   2105 269T 269T C   CA   2052 2048    1.77    1.49    0.28    8.08    1.49    0.28    8.08\n",
      "  140   2106 269T 270A C   N    2052 2054    1.54    1.35    0.20    6.95    1.35    0.20    6.95\n",
      "  141   2108 270A 270A N   CA   2054 2055    1.66    1.43    0.23    7.42    1.43    0.23    7.42\n",
      "  142   2110 270A 270A C   CA   2057 2055    1.77    1.49    0.28    8.00    1.49    0.28    8.00\n",
      "  143   2111 270A 271K C   N    2057 2059    1.54    1.35    0.19    6.84    1.35    0.19    6.84\n",
      "  144   2117 271K 271K N   CA   2059 2060    1.65    1.43    0.22    7.31    1.43    0.22    7.31\n",
      "  145   2119 271K 271K C   CA   2066 2060    1.75    1.49    0.26    7.61    1.49    0.26    7.61\n",
      "  146   2120 271K 272T C   N    2066 2068    1.53    1.35    0.18    6.53    1.35    0.18    6.53\n",
      "  147   2124 272T 272T N   CA   2068 2069    1.64    1.43    0.21    6.98    1.43    0.21    6.98\n",
      "  148   2126 272T 272T C   CA   2073 2069    1.74    1.49    0.25    7.21    1.49    0.25    7.21\n",
      "  149   2127 272T 273L C   N    2073 2075    1.52    1.35    0.17    6.17    1.35    0.17    6.17\n",
      "  150   2132 273L 273L N   CA   2075 2076    1.63    1.43    0.20    6.56    1.43    0.20    6.56\n",
      "  151   2134 273L 273L C   CA   2081 2076    1.73    1.49    0.24    6.92    1.49    0.24    6.92\n",
      "  152   2135 273L 274G C   N    2081 2083    1.51    1.35    0.17    5.92    1.35    0.17    5.92\n",
      "  153   2136 274G 274G N   CA   2083 2084    1.62    1.43    0.19    6.31    1.43    0.19    6.31\n",
      "  154   2138 274G 274G C   CA   2085 2084    1.72    1.49    0.23    6.73    1.49    0.23    6.73\n",
      "  155   2139 274G 275E C   N    2085 2087    1.51    1.35    0.16    5.75    1.35    0.16    5.75\n",
      "  156   2145 275E 275E N   CA   2087 2088    1.62    1.43    0.19    6.09    1.43    0.19    6.09\n",
      "  157   2147 275E 275E C   CA   2094 2088    1.71    1.49    0.22    6.36    1.49    0.22    6.36\n",
      "  158   2148 275E 276M C   N    2094 2096    1.50    1.35    0.15    5.40    1.35    0.15    5.40\n",
      "  159   2153 276M 276M N   CA   2096 2097    1.60    1.43    0.17    5.67    1.43    0.17    5.67\n",
      "  160   2155 276M 276M C   CA   2102 2097    1.69    1.49    0.20    5.93    1.49    0.20    5.93\n",
      "  161   2156 276M 277Q C   N    2102 2104    1.49    1.35    0.14    5.08    1.35    0.14    5.08\n",
      "  162   2162 277Q 277Q N   CA   2104 2105    1.59    1.43    0.16    5.27    1.43    0.16    5.27\n",
      "  163   2164 277Q 277Q C   CA   2111 2105    1.67    1.49    0.18    5.38    1.49    0.18    5.38\n",
      "  164   2165 277Q 278Y C   N    2111 2113    1.48    1.35    0.13    4.63    1.35    0.13    4.63\n",
      "  165   2175 278Y 278Y N   CA   2113 2114    1.58    1.43    0.15    4.90    1.43    0.15    4.90\n",
      "  166   2177 278Y 278Y C   CA   2123 2114    1.67    1.49    0.18    5.22    1.49    0.18    5.22\n",
      "  167   2185 279D 279D C   CA   2131 2126    1.65    1.49    0.16    4.69    1.49    0.16    4.69\n",
      "  168   4422 555V 555V C   CA   4318 4314    1.65    1.49    0.16    4.56    1.49    0.16    4.56\n",
      "  169   4432 556F 556F N   CA   4320 4321    1.57    1.43    0.14    4.62    1.43    0.14    4.62\n",
      "  170   4434 556F 556F C   CA   4329 4321    1.68    1.49    0.19    5.50    1.49    0.19    5.50\n",
      "  171   4435 556F 557V C   N    4329 4331    1.48    1.35    0.14    4.89    1.35    0.14    4.89\n",
      "  172   4439 557V 557V N   CA   4331 4332    1.59    1.43    0.16    5.28    1.43    0.16    5.28\n",
      "  173   4441 557V 557V C   CA   4336 4332    1.70    1.49    0.21    6.19    1.49    0.21    6.19\n",
      "  174   4442 557V 558D C   N    4336 4338    1.50    1.35    0.15    5.45    1.35    0.15    5.45\n",
      "  175   4447 558D 558D N   CA   4338 4339    1.60    1.43    0.17    5.72    1.43    0.17    5.72\n",
      "  176   4449 558D 558D C   CA   4344 4339    1.73    1.49    0.24    6.88    1.49    0.24    6.88\n",
      "  177   4450 558D 559V C   N    4344 4346    1.52    1.35    0.17    6.07    1.35    0.17    6.07\n",
      "  178   4454 559V 559V N   CA   4346 4347    1.63    1.43    0.20    6.64    1.43    0.20    6.64\n",
      "  179   4456 559V 559V C   CA   4351 4347    1.76    1.49    0.27    7.90    1.49    0.27    7.90\n",
      "  180   4457 559V 560P C   N    4351 4353    1.56    1.30    0.26    7.67    1.30    0.26    7.67\n",
      "  181   4459 560P 560P C   CA   4358 4354    1.78    1.49    0.29    8.42    1.49    0.29    8.42\n",
      "  182   4460 560P 561L C   N    4358 4360    1.55    1.35    0.21    7.41    1.35    0.21    7.41\n",
      "  183   4461 560P 560P N   CA   4353 4354    1.67    1.43    0.24    7.86    1.43    0.24    7.86\n",
      "  184   4470 561L 561L N   CA   4360 4361    1.67    1.43    0.24    8.05    1.43    0.24    8.05\n",
      "  185   4472 561L 561L C   CA   4366 4361    1.80    1.49    0.31    8.92    1.49    0.31    8.92\n",
      "  186   4473 561L 562E C   N    4366 4368    1.57    1.35    0.22    7.90    1.35    0.22    7.90\n",
      "  187   4479 562E 562E N   CA   4368 4369    1.69    1.43    0.26    8.53    1.43    0.26    8.53\n",
      "  188   4481 562E 562E C   CA   4375 4369    1.82    1.49    0.33    9.67    1.49    0.33    9.67\n",
      "  189   4482 562E 563L C   N    4375 4377    1.59    1.35    0.25    8.75    1.35    0.25    8.75\n",
      "  190   4487 563L 563L N   CA   4377 4378    1.72    1.43    0.29    9.43    1.43    0.29    9.43\n",
      "  191   4489 563L 563L C   CA   4383 4378    1.85    1.49    0.36   10.55    1.49    0.36   10.55\n",
      "  192   4490 563L 564D C   N    4383 4385    1.60    1.35    0.26    9.20    1.35    0.26    9.20\n",
      "  193   4495 564D 564D N   CA   4385 4386    1.73    1.43    0.30    9.91    1.43    0.30    9.91\n",
      "  194   4497 564D 564D C   CA   4391 4386    1.88    1.49    0.39   11.32    1.49    0.39   11.32\n",
      "  195   4498 564D 565L C   N    4391 4393    1.62    1.35    0.28    9.90    1.35    0.28    9.90\n",
      "  196   4503 565L 565L N   CA   4393 4394    1.75    1.43    0.32   10.68    1.43    0.32   10.68\n",
      "  197   4505 565L 565L C   CA   4399 4394    1.91    1.49    0.42   12.14    1.49    0.42   12.14\n",
      "  198   4506 565L 566E C   N    4399 4401    1.64    1.35    0.29   10.41    1.35    0.29   10.41\n",
      "  199   4512 566E 566E N   CA   4401 4402    1.78    1.43    0.35   11.39    1.43    0.35   11.39\n",
      "  200   4514 566E 566E C   CA   4408 4402    1.93    1.49    0.44   12.92    1.49    0.44   12.92\n",
      "  201   4515 566E 567F C   N    4408 4410    1.66    1.35    0.32   11.21    1.35    0.32   11.21\n",
      "  202   4524 567F 567F N   CA   4410 4411    1.80    1.43    0.37   12.22    1.43    0.37   12.22\n",
      "  203   4526 567F 567F C   CA   4419 4411    1.98    1.49    0.49   14.11    1.49    0.49   14.11\n",
      "  204   4527 567F 568L C   N    4419 4421    1.69    1.35    0.35   12.30    1.35    0.35   12.30\n",
      "  205   4532 568L 568L N   CA   4421 4422    1.84    1.43    0.41   13.46    1.43    0.41   13.46\n",
      "  206   4534 568L 568L C   CA   4427 4422    2.03    1.49    0.54   15.71    1.49    0.54   15.71\n",
      "  207   4535 568L 569R C   N    4427 4429    1.73    1.35    0.38   13.59    1.35    0.38   13.59\n",
      "  208   4543 569R 569R N   CA   4429 4430    1.88    1.43    0.45   14.77    1.43    0.45   14.77\n",
      "  209   4545 569R 569R C   CA   4438 4430    2.09    1.49    0.60   17.37    1.49    0.60   17.37\n",
      "  210   4546 569R 570G C   N    4438 4440    1.76    1.35    0.41   14.61    1.35    0.41   14.61\n",
      "  211   4547 570G 570G N   CA   4440 4441    1.92    1.43    0.49   16.01    1.43    0.49   16.01\n",
      "  212   4549 570G 570G C   CA   4442 4441    2.10    1.49    0.61   17.87    1.49    0.61   17.87\n",
      "  213   4550 570G 571K C   N    4442 4444    1.78    1.35    0.44   15.43    1.35    0.44   15.43\n",
      "  214   4556 571K 571K N   CA   4444 4445    1.94    1.43    0.51   16.68    1.43    0.51   16.68\n",
      "  215   4558 571K 571K C   CA   4451 4445    2.18    1.49    0.69   19.99    1.49    0.69   19.99\n",
      "  216   4559 571K 572G C   N    4451 4453    1.81    1.35    0.46   16.46    1.35    0.46   16.46\n",
      "  217   4560 572G 572G N   CA   4453 4454    1.97    1.43    0.54   17.82    1.43    0.54   17.82\n",
      "  218   4562 572G 572G C   CA   4455 4454    2.16    1.49    0.67   19.60    1.49    0.67   19.60\n",
      "  219   4563 572G 573L C   N    4455 4457    1.82    1.35    0.48   16.93    1.35    0.48   16.93\n",
      "  220   4568 573L 573L N   CA   4457 4458    1.99    1.43    0.56   18.56    1.43    0.56   18.56\n",
      "  221   4570 573L 573L C   CA   4463 4458    2.21    1.49    0.72   20.82    1.49    0.72   20.82\n",
      "  222   4571 573L 574Q C   N    4463 4465    1.83    1.35    0.48   17.12    1.35    0.48   17.12\n",
      "  223   4577 574Q 574Q N   CA   4465 4466    2.00    1.43    0.57   18.89    1.43    0.57   18.89\n",
      "  224   4579 574Q 574Q C   CA   4472 4466    2.19    1.49    0.70   20.35    1.49    0.70   20.35\n",
      "  225   4580 574Q 575S C   N    4472 4474    1.83    1.35    0.49   17.32    1.35    0.49   17.32\n",
      "  226   4583 575S 575S N   CA   4474 4475    2.00    1.43    0.57   18.85    1.43    0.57   18.85\n",
      "  227   4585 575S 575S C   CA   4478 4475    2.25    1.49    0.76   22.11    1.49    0.76   22.11\n",
      "  228   4586 575S 576D C   N    4478 4480    1.86    1.35    0.51   18.23    1.35    0.51   18.23\n",
      "  229   4591 576D 576D N   CA   4480 4481    2.04    1.43    0.61   19.99    1.43    0.61   19.99\n",
      "  230   4593 576D 576D C   CA   4486 4481    2.34    1.49    0.85   24.79    1.49    0.85   24.79\n",
      "  231   4594 576D 577E C   N    4486 4488    1.96    1.35    0.62   21.86    1.35    0.62   21.86\n",
      "  232   4600 577E 577E N   CA   4488 4489    2.15    1.43    0.72   23.78    1.43    0.72   23.78\n",
      "  233   4602 577E 577E C   CA   4495 4489    2.43    1.49    0.94   27.41    1.49    0.94   27.41\n",
      "  234   4603 577E 578V C   N    4495 4497    1.99    1.35    0.65   22.90    1.35    0.65   22.90\n",
      "  235   4607 578V 578V N   CA   4497 4498    2.16    1.43    0.73   23.93    1.43    0.73   23.93\n",
      "  236   4609 578V 578V C   CA   4502 4498    2.48    1.49    0.99   28.73    1.49    0.99   28.73\n",
      "  237   4610 578V 579S C   N    4502 4504    2.02    1.35    0.68   24.05    1.35    0.68   24.05\n",
      "  238   4613 579S 579S N   CA   4504 4505    2.23    1.43    0.80   26.20    1.43    0.80   26.20\n",
      "  239   4615 579S 579S C   CA   4508 4505    2.53    1.49    1.04   30.18    1.49    1.04   30.18\n",
      "  240   4616 579S 580M C   N    4508 4510    2.07    1.35    0.73   25.72    1.35    0.73   25.72\n",
      "  241   4621 580M 580M N   CA   4510 4511    2.30    1.43    0.87   28.65    1.43    0.87   28.65\n",
      "  242   4623 580M 580M C   CA   4516 4511    2.61    1.49    1.12   32.54    1.49    1.12   32.54\n",
      "  243   4624 580M 581P C   N    4516 4518    2.42    1.30    1.12   33.13    1.30    1.12   33.13\n",
      "  244   4626 581P 581P C   CA   4523 4519    2.73    1.49    1.24   36.17    1.49    1.24   36.17\n",
      "  245   4627 581P 582E C   N    4523 4525    2.22    1.35    0.87   30.91    1.35    0.87   30.91\n",
      "  246   4628 581P 581P N   CA   4518 4519    2.34    1.43    0.91   29.86    1.43    0.91   29.86\n",
      "  247   4638 582E 582E N   CA   4525 4526    2.49    1.43    1.06   34.90    1.43    1.06   34.90\n",
      "  248   4640 582E 582E C   CA   4532 4526    2.91    1.49    1.42   41.15    1.49    1.42   41.15\n",
      "  249   4641 582E 583A C   N    4532 4534    2.33    1.35    0.98   34.81    1.35    0.98   34.81\n",
      "  250   4643 583A 583A N   CA   4534 4535    2.60    1.43    1.17   38.38    1.43    1.17   38.38\n",
      "  251   4645 583A 583A C   CA   4537 4535    3.03    1.49    1.54   44.83    1.49    1.54   44.83\n",
      "  252   4646 583A 584Q C   N    4537 4539    2.45    1.35    1.10   39.02    1.35    1.10   39.02\n",
      "  253   4652 584Q 584Q N   CA   4539 4540    2.74    1.43    1.31   43.02    1.43    1.31   43.02\n",
      "  254   4654 584Q 584Q C   CA   4546 4540    3.25    1.49    1.76   51.18    1.49    1.76   51.18\n",
      "  255   4655 584Q 585E C   N    4546 4548    2.58    1.35    1.24   43.83    1.35    1.24   43.83\n",
      "  256   4661 585E 585E N   CA   4548 4549    2.89    1.43    1.46   47.87    1.43    1.46   47.87\n",
      "  257   4663 585E 585E C   CA   4555 4549    3.43    1.49    1.94   56.51    1.49    1.94   56.51\n",
      "  258   4664 585E 586D C   N    4555 4557    2.71    1.35    1.37   48.58    1.35    1.37   48.58\n",
      "  259   4669 586D 586D N   CA   4557 4558    3.05    1.43    1.62   53.13    1.43    1.62   53.13\n",
      "  260   4671 586D 586D C   CA   4563 4558    3.66    1.49    2.17   63.01    1.49    2.17   63.01\n",
      "  261   4672 586D 587L C   N    4563 4565    2.84    1.35    1.50   53.14    1.35    1.50   53.14\n",
      "  262   4677 587L 587L N   CA   4565 4566    3.25    1.43    1.82   59.82    1.43    1.82   59.82\n",
      "  263   4679 587L 587L C   CA   4571 4566    3.99    1.49    2.50   72.68    1.49    2.50   72.68\n",
      "  264   4680 587L 588P C   N    4571 4573    3.76    1.30    2.46   72.98    1.30    2.46   72.98\n",
      "  265   4682 588P 588P C   CA   4578 4574    4.34    1.49    2.85   82.92    1.49    2.85   82.92\n",
      "  266   4683 588P 589H C   N    4578 4580    3.32    1.35    1.98   70.18    1.35    1.98   70.18\n",
      "  267   4684 588P 588P N   CA   4573 4574    3.45    1.43    2.02   66.46    1.43    2.02   66.46\n",
      "  268   4685 588P 588P CA  CB   4574 4576    1.70    1.53    0.17    4.62    1.53    0.17    4.62\n",
      "  269   4689 589H 589H CB  CA   4584 4581    1.71    1.54    0.17    4.63    1.54    0.17    4.63\n",
      "  270   4695 589H 589H N   CA   4580 4581    3.77    1.43    2.34   77.01    1.43    2.34   77.01\n",
      "  271   4697 589H 589H C   CA   4588 4581    4.67    1.49    3.18   92.37    1.49    3.18   92.37\n",
      "  272   4698 589H 590K C   N    4588 4590    3.57    1.35    2.23   78.93    1.35    2.23   78.93\n",
      "  273   4705 590K 590K N   CA   4590 4591    4.03    1.43    2.60   85.63    1.43    2.60   85.63\n",
      "  274   4707 590K 590K C   CA   4597 4591    4.94    1.49    3.45  100.16    1.49    3.45  100.16\n",
      "  275   4708 590K 591P C   N    4597 4599    4.69    1.30    3.39  100.51    1.30    3.39  100.51\n",
      "  276   4710 591P 591P C   CA   4604 4600    5.27    1.49    3.78  109.76    1.49    3.78  109.76\n",
      "  277   4711 591P 592T C   N    4604 4606    3.96    1.35    2.62   92.79    1.35    2.62   92.79\n",
      "  278   4712 591P 591P N   CA   4599 4600    4.14    1.43    2.71   89.02    1.43    2.71   89.02\n",
      "  279   4713 591P 591P CA  CB   4600 4602    1.78    1.53    0.25    6.85    1.53    0.25    6.85\n",
      "  280   4714 591P 591P CB  CG   4602 4603    1.75    1.54    0.21    5.86    1.54    0.21    5.86\n",
      "  281   4720 592T 592T N   CA   4606 4607    4.51    1.43    3.08  101.17    1.43    3.08  101.17\n",
      "  282   4722 592T 592T C   CA   4611 4607    5.61    1.49    4.12  119.68    1.49    4.12  119.68\n",
      "  283   4723 592T 593S C   N    4611 4613    4.16    1.35    2.81   99.67    1.35    2.81   99.67\n",
      "  284   4726 593S 593S N   CA   4613 4614    4.74    1.43    3.31  109.00    1.43    3.31  109.00\n",
      "  285   4728 593S 593S C   CA   4617 4614    5.83    1.49    4.34  126.07    1.49    4.34  126.07\n",
      "  286   4729 593S 594S C   N    4617 4619    4.31    1.35    2.97  105.19    1.35    2.97  105.19\n",
      "  287   4732 594S 594S N   CA   4619 4620    4.90    1.43    3.47  114.27    1.43    3.47  114.27\n",
      "  288   4734 594S 594S C   CA   4623 4620    6.08    1.49    4.59  133.48    1.49    4.59  133.48\n",
      "  289   4735 594S 595S C   N    4623 4625    4.47    1.35    3.13  110.87    1.35    3.13  110.87\n",
      "  290   4738 595S 595S N   CA   4625 4626    5.11    1.43    3.68  121.19    1.43    3.68  121.19\n",
      "  291   4740 595S 595S C   CA   4629 4626    6.31    1.49    4.82  140.08    1.49    4.82  140.08\n",
      "  292   4741 595S 596M C   N    4629 4631    4.63    1.35    3.29  116.60    1.35    3.29  116.60\n",
      "  293   4746 596M 596M N   CA   4631 4632    5.27    1.43    3.84  126.46    1.43    3.84  126.46\n",
      "  294   4748 596M 596M C   CA   4637 4632    6.51    1.49    5.02  145.84    1.49    5.02  145.84\n",
      "  295   4749 596M 597K C   N    4637 4639    4.78    1.35    3.44  121.87    1.35    3.44  121.87\n",
      "  296   4750 597K 597K CB  CA   4641 4640    1.72    1.54    0.18    5.08    1.54    0.18    5.08\n",
      "  297   4755 597K 597K N   CA   4639 4640    5.44    1.43    4.01  131.80    1.43    4.01  131.80\n",
      "  298   4757 597K 597K C   CA   4646 4640    6.67    1.49    5.18  150.72    1.49    5.18  150.72\n",
      "  299   4758 597K 598T C   N    4646 4648    4.89    1.35    3.55  125.76    1.35    3.55  125.76\n",
      "  300   4762 598T 598T N   CA   4648 4649    5.58    1.43    4.15  136.54    1.43    4.15  136.54\n",
      "  301   4764 598T 598T C   CA   4653 4649    6.98    1.49    5.49  159.66    1.49    5.49  159.66\n",
      "  302   4765 598T 599V C   N    4653 4655    5.05    1.35    3.70  131.23    1.35    3.70  131.23\n",
      "  303   4769 599V 599V N   CA   4655 4656    5.71    1.43    4.28  140.93    1.43    4.28  140.93\n",
      "  304   4771 599V 599V C   CA   4660 4656    7.06    1.49    5.57  162.06    1.49    5.57  162.06\n",
      "  305   4772 599V 600D C   N    4660 4662    5.15    1.35    3.80  134.84    1.35    3.80  134.84\n",
      "  306   4777 600D 600D N   CA   4662 4663    5.83    1.43    4.40  144.77    1.43    4.40  144.77\n",
      "  307   4779 600D 600D C   CA   4668 4663    7.16    1.49    5.67  164.76    1.49    5.67  164.76\n",
      "  308   4780 600D 601E C   N    4668 4670    5.19    1.35    3.85  136.46    1.35    3.85  136.46\n",
      "  309   4786 601E 601E N   CA   4670 4671    5.92    1.43    4.49  147.77    1.43    4.49  147.77\n",
      "  310   4788 601E 601E C   CA   4677 4671    7.28    1.49    5.79  168.33    1.49    5.79  168.33\n",
      "  311   4789 601E 602E C   N    4677 4679    5.25    1.35    3.91  138.57    1.35    3.91  138.57\n",
      "  312   4795 602E 602E N   CA   4679 4680    5.98    1.43    4.55  149.79    1.43    4.55  149.79\n",
      "  313   4797 602E 602E C   CA   4686 4680    7.38    1.49    5.89  171.26    1.49    5.89  171.26\n",
      "  314   4798 602E 603A C   N    4686 4688    5.30    1.35    3.96  140.38    1.35    3.96  140.38\n",
      "  315   4800 603A 603A N   CA   4688 4689    6.04    1.43    4.61  151.54    1.43    4.61  151.54\n",
      "  316   4802 603A 603A C   CA   4691 4689    7.37    1.49    5.88  171.07    1.49    5.88  171.07\n",
      "  317   4803 603A 604L C   N    4691 4693    5.31    1.35    3.97  140.77    1.35    3.97  140.77\n",
      "  318   4808 604L 604L N   CA   4693 4694    6.03    1.43    4.60  151.24    1.43    4.60  151.24\n",
      "  319   4810 604L 604L C   CA   4699 4694    7.38    1.49    5.89  171.17    1.49    5.89  171.17\n",
      "  320   4811 604L 605A C   N    4699 4701    5.30    1.35    3.96  140.41    1.35    3.96  140.41\n",
      "  321   4813 605A 605A N   CA   4701 4702    6.01    1.43    4.58  150.71    1.43    4.58  150.71\n",
      "  322   4815 605A 605A C   CA   4704 4702    7.37    1.49    5.88  170.90    1.49    5.88  170.90\n",
      "  323   4816 605A 606M C   N    4704 4706    5.30    1.35    3.95  140.24    1.35    3.95  140.24\n",
      "  324   4821 606M 606M N   CA   4706 4707    4.96    1.43    3.53  116.16    1.43    3.53  116.16\n",
      "  325   4822 606M 606M O   C    4713 4712    1.53    1.23    0.30   13.73    1.23    0.30   13.73\n",
      "  326   4823 606M 606M C   CA   4712 4707    3.37    1.49    1.88   54.55    1.49    1.88   54.55\n",
      "  327   4824 606M 607L C   N    4712 4714    2.00    1.35    0.65   23.20    1.35    0.65   23.20\n",
      "  328   4829 607L 607L N   CA   4714 4715    1.97    1.43    0.54   17.63    1.43    0.54   17.63\n",
      "  329   4831 607L 607L C   CA   4720 4715    1.67    1.49    0.18    5.34    1.49    0.18    5.34\n",
      "  330   4926 619Y 619Y C   CA   4814 4805    1.71    1.49    0.22    6.41    1.49    0.22    6.41\n",
      "  331   4927 619Y 620A C   N    4814 4816    1.49    1.35    0.15    5.28    1.35    0.15    5.28\n",
      "  332   4929 620A 620A N   CA   4816 4817    1.60    1.43    0.17    5.67    1.43    0.17    5.67\n",
      "  333   4931 620A 620A C   CA   4819 4817    1.71    1.49    0.22    6.40    1.49    0.22    6.40\n",
      "  334   4932 620A 621L C   N    4819 4821    1.49    1.35    0.15    5.24    1.35    0.15    5.24\n",
      "  335   4937 621L 621L N   CA   4821 4822    1.59    1.43    0.16    5.43    1.43    0.16    5.43\n",
      "  336   4939 621L 621L C   CA   4827 4822    1.65    1.49    0.16    4.56    1.49    0.16    4.56\n",
      "  337   4945 622L 622L N   CA   4829 4830    1.57    1.43    0.14    4.56    1.43    0.14    4.56\n",
      "  338   4947 622L 622L C   CA   4835 4830    1.67    1.49    0.18    5.31    1.49    0.18    5.31\n",
      "  339   4948 622L 623Q C   N    4835 4837    1.48    1.35    0.13    4.63    1.35    0.13    4.63\n",
      "  340   4954 623Q 623Q N   CA   4837 4838    1.58    1.43    0.15    5.02    1.43    0.15    5.02\n",
      "  341   4956 623Q 623Q C   CA   4844 4838    1.68    1.49    0.19    5.65    1.49    0.19    5.65\n",
      "  342   4957 623Q 624T C   N    4844 4846    1.48    1.35    0.14    4.79    1.35    0.14    4.79\n",
      "  343   4961 624T 624T N   CA   4846 4847    1.58    1.43    0.15    4.97    1.43    0.15    4.97\n",
      "  344   5535 699W 699W C   CA   5412 5401    1.68    1.49    0.19    5.61    1.49    0.19    5.61\n",
      "  345   5536 699W 700L C   N    5412 5414    1.49    1.35    0.14    5.05    1.35    0.14    5.05\n",
      "  346   5544 700L 700L N   CA   5414 5415    1.60    1.43    0.17    5.57    1.43    0.17    5.57\n",
      "  347   5546 700L 700L C   CA   5420 5415    1.71    1.49    0.22    6.25    1.49    0.22    6.25\n",
      "  348   5547 700L 701L C   N    5420 5422    1.50    1.35    0.16    5.61    1.35    0.16    5.61\n",
      "  349   5552 701L 701L N   CA   5422 5423    1.62    1.43    0.19    6.09    1.43    0.19    6.09\n",
      "  350   5554 701L 701L C   CA   5428 5423    1.72    1.49    0.23    6.61    1.49    0.23    6.61\n",
      "  351   5555 701L 702F C   N    5428 5430    1.52    1.35    0.17    6.12    1.35    0.17    6.12\n",
      "  352   5564 702F 702F N   CA   5430 5431    1.63    1.43    0.20    6.47    1.43    0.20    6.47\n",
      "  353   5566 702F 702F C   CA   5439 5431    1.76    1.49    0.27    7.80    1.49    0.27    7.80\n",
      "  354   5567 702F 703N C   N    5439 5441    1.53    1.35    0.19    6.72    1.35    0.19    6.72\n",
      "  355   5572 703N 703N N   CA   5441 5442    1.66    1.43    0.23    7.41    1.43    0.23    7.41\n",
      "  356   5574 703N 703N C   CA   5447 5442    1.76    1.49    0.27    7.90    1.49    0.27    7.90\n",
      "  357   5575 703N 704D C   N    5447 5449    1.56    1.35    0.21    7.60    1.35    0.21    7.60\n",
      "  358   5580 704D 704D N   CA   5449 5450    1.68    1.43    0.25    8.11    1.43    0.25    8.11\n",
      "  359   5582 704D 704D C   CA   5455 5450    1.79    1.49    0.30    8.79    1.49    0.30    8.79\n",
      "  360   5583 704D 705E C   N    5455 5457    1.56    1.35    0.22    7.78    1.35    0.22    7.78\n",
      "  361   5589 705E 705E N   CA   5457 5458    1.69    1.43    0.26    8.48    1.43    0.26    8.48\n",
      "  362   5591 705E 705E C   CA   5464 5458    1.82    1.49    0.33    9.69    1.49    0.33    9.69\n",
      "  363   5592 705E 706K C   N    5464 5466    1.59    1.35    0.24    8.57    1.35    0.24    8.57\n",
      "  364   5598 706K 706K N   CA   5466 5467    1.71    1.43    0.28    9.18    1.43    0.28    9.18\n",
      "  365   5600 706K 706K C   CA   5473 5467    1.84    1.49    0.35   10.25    1.49    0.35   10.25\n",
      "  366   5601 706K 707V C   N    5473 5475    1.60    1.35    0.25    8.96    1.35    0.25    8.96\n",
      "  367   5605 707V 707V N   CA   5475 5476    1.73    1.43    0.30    9.81    1.43    0.30    9.81\n",
      "  368   5607 707V 707V C   CA   5480 5476    1.85    1.49    0.36   10.44    1.49    0.36   10.44\n",
      "  369   5608 707V 708A C   N    5480 5482    1.62    1.35    0.27    9.62    1.35    0.27    9.62\n",
      "  370   5610 708A 708A N   CA   5482 5483    1.75    1.43    0.32   10.54    1.43    0.32   10.54\n",
      "  371   5612 708A 708A C   CA   5485 5483    1.91    1.49    0.42   12.29    1.49    0.42   12.29\n",
      "  372   5613 708A 709E C   N    5485 5487    1.65    1.35    0.30   10.67    1.35    0.30   10.67\n",
      "  373   5619 709E 709E N   CA   5487 5488    1.79    1.43    0.36   11.84    1.43    0.36   11.84\n",
      "  374   5621 709E 709E C   CA   5494 5488    1.93    1.49    0.44   12.82    1.49    0.44   12.82\n",
      "  375   5622 709E 710S C   N    5494 5496    1.66    1.35    0.32   11.34    1.35    0.32   11.34\n",
      "  376   5625 710S 710S N   CA   5496 5497    1.80    1.43    0.37   12.07    1.43    0.37   12.07\n",
      "  377   5627 710S 710S C   CA   5500 5497    1.89    1.49    0.40   11.68    1.49    0.40   11.68\n",
      "  378   5628 710S 711L C   N    5500 5502    1.63    1.35    0.29   10.27    1.35    0.29   10.27\n",
      "  379   5633 711L 711L N   CA   5502 5503    1.77    1.43    0.34   11.25    1.43    0.34   11.25\n",
      "  380   5635 711L 711L C   CA   5508 5503    1.97    1.49    0.48   14.05    1.49    0.48   14.05\n",
      "  381   5636 711L 712N C   N    5508 5510    1.63    1.35    0.28   10.08    1.35    0.28   10.08\n",
      "  382   5641 712N 712N N   CA   5510 5511    1.79    1.43    0.36   11.83    1.43    0.36   11.83\n",
      "  383   5643 712N 712N C   CA   5516 5511    1.94    1.49    0.45   13.07    1.49    0.45   13.07\n",
      "  384   5644 712N 713P C   N    5516 5518    1.72    1.30    0.42   12.61    1.30    0.42   12.61\n",
      "  385   5646 713P 713P C   CA   5523 5519    1.90    1.49    0.41   11.96    1.49    0.41   11.96\n",
      "  386   5647 713P 714P C   N    5523 5525    1.73    1.30    0.43   12.81    1.30    0.43   12.81\n",
      "  387   5648 713P 713P N   CA   5518 5519    1.75    1.43    0.32   10.41    1.43    0.32   10.41\n",
      "  388   5654 714P 714P C   CA   5530 5526    1.95    1.49    0.46   13.24    1.49    0.46   13.24\n",
      "  389   5655 714P 715F C   N    5530 5532    1.68    1.35    0.33   11.82    1.35    0.33   11.82\n",
      "  390   5656 714P 714P N   CA   5525 5526    1.80    1.43    0.37   12.06    1.43    0.37   12.06\n",
      "  391   5669 715F 715F N   CA   5532 5533    1.86    1.43    0.43   13.98    1.43    0.43   13.98\n",
      "  392   5671 715F 715F C   CA   5541 5533    1.97    1.49    0.48   13.97    1.49    0.48   13.97\n",
      "  393   5672 715F 716S C   N    5541 5543    1.69    1.35    0.35   12.40    1.35    0.35   12.40\n",
      "  394   5675 716S 716S N   CA   5543 5544    1.87    1.43    0.44   14.59    1.43    0.44   14.59\n",
      "  395   5677 716S 716S C   CA   5547 5544    2.04    1.49    0.55   16.04    1.49    0.55   16.04\n",
      "  396   5678 716S 717M C   N    5547 5549    1.75    1.35    0.41   14.36    1.35    0.41   14.36\n",
      "  397   5683 717M 717M N   CA   5549 5550    1.91    1.43    0.48   15.71    1.43    0.48   15.71\n",
      "  398   5685 717M 717M C   CA   5555 5550    2.14    1.49    0.65   18.75    1.49    0.65   18.75\n",
      "  399   5686 717M 718A C   N    5555 5557    1.82    1.35    0.47   16.76    1.35    0.47   16.76\n",
      "  400   5688 718A 718A N   CA   5557 5558    2.01    1.43    0.58   18.99    1.43    0.58   18.99\n",
      "  401   5690 718A 718A C   CA   5560 5558    2.26    1.49    0.77   22.24    1.49    0.77   22.24\n",
      "  402   5691 718A 719F C   N    5560 5562    1.86    1.35    0.51   18.21    1.35    0.51   18.21\n",
      "  403   5700 719F 719F N   CA   5562 5563    2.06    1.43    0.63   20.66    1.43    0.63   20.66\n",
      "  404   5702 719F 719F C   CA   5571 5563    2.39    1.49    0.90   26.16    1.49    0.90   26.16\n",
      "  405   5703 719F 720L C   N    5571 5573    1.94    1.35    0.60   21.25    1.35    0.60   21.25\n",
      "  406   5708 720L 720L N   CA   5573 5574    2.12    1.43    0.69   22.77    1.43    0.69   22.77\n",
      "  407   5710 720L 720L C   CA   5579 5574    2.40    1.49    0.91   26.38    1.49    0.91   26.38\n",
      "  408   5711 720L 721Y C   N    5579 5581    1.99    1.35    0.65   23.03    1.35    0.65   23.03\n",
      "  409   5721 721Y 721Y N   CA   5581 5582    2.18    1.43    0.75   24.78    1.43    0.75   24.78\n",
      "  410   5723 721Y 721Y C   CA   5591 5582    2.57    1.49    1.08   31.30    1.49    1.08   31.30\n",
      "  411   5724 721Y 722F C   N    5591 5593    2.12    1.35    0.78   27.51    1.35    0.78   27.51\n",
      "  412   5733 722F 722F N   CA   5593 5594    2.37    1.43    0.94   31.03    1.43    0.94   31.03\n",
      "  413   5735 722F 722F C   CA   5602 5594    2.69    1.49    1.20   34.80    1.49    1.20   34.80\n",
      "  414   5736 722F 723Y C   N    5602 5604    2.15    1.35    0.81   28.65    1.35    0.81   28.65\n",
      "  415   5746 723Y 723Y N   CA   5604 5605    2.42    1.43    0.99   32.62    1.43    0.99   32.62\n",
      "  416   5748 723Y 723Y C   CA   5614 5605    2.93    1.49    1.44   41.77    1.49    1.44   41.77\n",
      "  417   5749 723Y 724K C   N    5614 5616    2.32    1.35    0.98   34.67    1.35    0.98   34.67\n",
      "  418   5755 724K 724K N   CA   5616 5617    2.60    1.43    1.17   38.45    1.43    1.17   38.45\n",
      "  419   5757 724K 724K C   CA   5623 5617    3.02    1.49    1.53   44.50    1.49    1.53   44.50\n",
      "  420   5758 724K 725R C   N    5623 5625    2.40    1.35    1.05   37.34    1.35    1.05   37.34\n",
      "  421   5766 725R 725R N   CA   5625 5626    2.68    1.43    1.25   40.98    1.43    1.25   40.98\n",
      "  422   5768 725R 725R C   CA   5634 5626    3.14    1.49    1.65   47.90    1.49    1.65   47.90\n",
      "  423   5769 725R 726V C   N    5634 5636    2.47    1.35    1.12   39.73    1.35    1.12   39.73\n",
      "  424   5773 726V 726V N   CA   5636 5637    2.75    1.43    1.32   43.56    1.43    1.32   43.56\n",
      "  425   5775 726V 726V C   CA   5641 5637    3.22    1.49    1.73   50.15    1.49    1.73   50.15\n",
      "  426   5776 726V 727G C   N    5641 5643    2.53    1.35    1.19   42.04    1.35    1.19   42.04\n",
      "  427   5777 727G 727G N   CA   5643 5644    2.85    1.43    1.42   46.65    1.43    1.42   46.65\n",
      "  428   5779 727G 727G C   CA   5645 5644    3.28    1.49    1.79   52.16    1.49    1.79   52.16\n",
      "  429   5780 727G 728K C   N    5645 5647    2.57    1.35    1.23   43.48    1.35    1.23   43.48\n",
      "  430   5786 728K 728K N   CA   5647 5648    2.40    1.43    0.97   31.94    1.43    0.97   31.94\n",
      "  431   5787 728K 728K O   C    5655 5654    1.43    1.26    0.17    7.05    1.26    0.17    7.05\n",
      "  432   5788 728K 728K C   CA   5654 5648    1.87    1.52    0.35    9.10    1.52    0.35    9.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5907  11A  11A N   CA     83   84  123.02  107.00   16.02    4.61  107.00   16.02    4.61\n",
      "    2   5914  12C  12C N   CA     88   89  124.35  107.00   17.35    4.99  107.00   17.35    4.99\n",
      "    3   5922  13A  13A N   CA     94   95  124.78  107.00   17.78    5.11  107.00   17.78    5.11\n",
      "    4   5929  14H  14H N   CA     99  100  124.24  107.00   17.24    4.96  107.00   17.24    4.96\n",
      "    5   5944  15C  15C N   CA    109  110  126.89  107.00   19.89    5.72  107.00   19.89    5.72\n",
      "    6   5952  16W  16W N   CA    115  116  126.78  107.00   19.78    5.69  107.00   19.78    5.69\n",
      "    7   5975  17D  17D N   CA    129  130  130.27  107.00   23.27    6.69  107.00   23.27    6.69\n",
      "    8   5986  18E  18E N   CA    137  138  130.94  107.00   23.94    6.88  107.00   23.94    6.88\n",
      "    9   5989  18E  18E CA  C     138  144  132.27  116.50   15.77    4.53  116.50   15.77    4.53\n",
      "   10   5998  19S  19S N   CA    146  147  131.86  107.00   24.86    7.15  107.00   24.86    7.15\n",
      "   11   6001  19S  19S CA  C     147  150  132.89  116.50   16.39    4.71  116.50   16.39    4.71\n",
      "   12   6006  20V  20V N   CA    152  153  133.06  107.00   26.06    7.49  107.00   26.06    7.49\n",
      "   13   6010  20V  20V CA  C     153  157  132.86  116.50   16.36    4.70  116.50   16.36    4.70\n",
      "   14   6016  21M  21M N   CA    159  160  133.17  107.00   26.17    7.52  107.00   26.17    7.52\n",
      "   15   6019  21M  21M CA  C     160  165  133.98  116.50   17.48    5.03  116.50   17.48    5.03\n",
      "   16   6026  22L  22L N   CA    167  168  133.28  107.00   26.28    7.56  107.00   26.28    7.56\n",
      "   17   6029  22L  22L CA  C     168  173  134.42  116.50   17.92    5.15  116.50   17.92    5.15\n",
      "   18   6037  23V  23V N   CA    175  176  136.59  107.00   29.59    8.51  107.00   29.59    8.51\n",
      "   19   6044  23V  24P C   N     180  182  144.48  117.00   27.49    6.84  117.00   27.49    6.84\n",
      "   20   6047  24P  24P N   CA    182  183  142.59  108.20   34.39    7.82  108.20   34.39    7.82\n",
      "   21   6051  24P  24P CA  C     183  187  135.27  116.50   18.77    5.40  116.50   18.77    5.40\n",
      "   22   6059  25N  25N N   CA    189  190  134.81  107.00   27.81    8.00  107.00   27.81    8.00\n",
      "   23   6062  25N  25N CA  C     190  195  134.53  116.50   18.03    5.18  116.50   18.03    5.18\n",
      "   24   6067  25N  26H C   N     195  197  140.14  120.00   20.14    4.58  120.00   20.14    4.58\n",
      "   25   6070  26H  26H N   CA    197  198  135.72  107.00   28.72    8.26  107.00   28.72    8.26\n",
      "   26   6072  26H  26H CA  C     198  205  135.74  116.50   19.24    5.53  116.50   19.24    5.53\n",
      "   27   6082  26H  27R C   N     205  207  141.36  120.00   21.36    4.85  120.00   21.36    4.85\n",
      "   28   6085  27R  27R N   CA    207  208  134.89  107.00   27.89    8.02  107.00   27.89    8.02\n",
      "   29   6088  27R  27R CA  C     208  216  136.32  116.50   19.82    5.70  116.50   19.82    5.70\n",
      "   30   6096  27R  28S C   N     216  218  141.14  120.00   21.14    4.80  120.00   21.14    4.80\n",
      "   31   6099  28S  28S N   CA    218  219  136.46  107.00   29.46    8.47  107.00   29.46    8.47\n",
      "   32   6102  28S  28S CA  C     219  222  136.46  116.50   19.96    5.74  116.50   19.96    5.74\n",
      "   33   6104  28S  29A C   N     222  224  141.67  120.00   21.67    4.93  120.00   21.67    4.93\n",
      "   34   6107  29A  29A N   CA    224  225  137.25  107.00   30.25    8.70  107.00   30.25    8.70\n",
      "   35   6109  29A  29A CA  C     225  227  137.68  116.50   21.18    6.09  116.50   21.18    6.09\n",
      "   36   6111  29A  30V C   N     227  229  142.73  120.00   22.72    5.16  120.00   22.72    5.16\n",
      "   37   6114  30V  30V N   CA    229  230  138.49  107.00   31.49    9.06  107.00   31.49    9.06\n",
      "   38   6118  30V  30V CA  C     230  234  138.46  116.50   21.96    6.32  116.50   21.96    6.32\n",
      "   39   6121  30V  31H C   N     234  236  143.76  120.00   23.76    5.40  120.00   23.76    5.40\n",
      "   40   6124  31H  31H N   CA    236  237  139.52  107.00   32.52    9.35  107.00   32.52    9.35\n",
      "   41   6126  31H  31H CA  C     237  244  141.54  116.50   25.04    7.20  116.50   25.04    7.20\n",
      "   42   6136  31H  32K C   N     244  246  144.40  120.00   24.40    5.55  120.00   24.40    5.55\n",
      "   43   6139  32K  32K N   CA    246  247  140.85  107.00   33.85    9.73  107.00   33.85    9.73\n",
      "   44   6142  32K  32K CA  C     247  253  139.73  116.50   23.24    6.68  116.50   23.24    6.68\n",
      "   45   6147  32K  33E C   N     253  255  148.02  120.00   28.02    6.37  120.00   28.02    6.37\n",
      "   46   6150  33E  33E N   CA    255  256  140.87  107.00   33.87    9.74  107.00   33.87    9.74\n",
      "   47   6153  33E  33E CA  C     256  262  142.28  116.50   25.78    7.41  116.50   25.78    7.41\n",
      "   48   6159  33E  34E C   N     262  264  147.61  120.00   27.61    6.28  120.00   27.61    6.28\n",
      "   49   6162  34E  34E N   CA    264  265  145.63  107.00   38.63   11.11  107.00   38.63   11.11\n",
      "   50   6165  34E  34E CA  C     265  271  142.71  116.50   26.21    7.54  116.50   26.21    7.54\n",
      "   51   6171  34E  35C C   N     271  273  147.19  120.00   27.19    6.18  120.00   27.19    6.18\n",
      "   52   6174  35C  35C N   CA    273  274  145.68  107.00   38.68   11.12  107.00   38.68   11.12\n",
      "   53   6177  35C  35C CA  C     274  277  141.31  116.50   24.81    7.13  116.50   24.81    7.13\n",
      "   54   6179  35C  36A C   N     277  279  151.89  120.00   31.89    7.25  120.00   31.89    7.25\n",
      "   55   6182  36A  36A N   CA    279  280  145.93  107.00   38.93   11.19  107.00   38.93   11.19\n",
      "   56   6184  36A  36A CA  C     280  282  143.86  116.50   27.36    7.87  116.50   27.36    7.87\n",
      "   57   6186  36A  37Y C   N     282  284  148.02  120.00   28.02    6.37  120.00   28.02    6.37\n",
      "   58   6189  37Y  37Y N   CA    284  285  150.00  107.00   43.00   12.36  107.00   43.00   12.36\n",
      "   59   6190  37Y  37Y CA  CB    285  286  130.51  107.50   23.02    5.32  107.50   23.02    5.32\n",
      "   60   6192  37Y  37Y CA  C     285  294  149.71  116.50   33.21    9.55  116.50   33.21    9.55\n",
      "   61   6204  37Y  38C C   N     294  296  148.77  120.00   28.77    6.54  120.00   28.77    6.54\n",
      "   62   6207  38C  38C N   CA    296  297  149.06  107.00   42.06   12.09  107.00   42.06   12.09\n",
      "   63   6210  38C  38C CA  C     297  300  146.41  116.50   29.91    8.60  116.50   29.91    8.60\n",
      "   64   6212  38C  39C C   N     300  302  154.19  120.00   34.19    7.77  120.00   34.19    7.77\n",
      "   65   6215  39C  39C N   CA    302  303  151.21  107.00   44.21   12.71  107.00   44.21   12.71\n",
      "   66   6218  39C  39C CA  C     303  306  145.20  116.50   28.70    8.25  116.50   28.70    8.25\n",
      "   67   6220  39C  40R C   N     306  308  151.98  120.00   31.98    7.27  120.00   31.98    7.27\n",
      "   68   6223  40R  40R N   CA    308  309  148.96  107.00   41.96   12.07  107.00   41.96   12.07\n",
      "   69   6226  40R  40R CA  C     309  317  147.17  116.50   30.67    8.82  116.50   30.67    8.82\n",
      "   70   6234  40R  41T C   N     317  319  151.93  120.00   31.93    7.26  120.00   31.93    7.26\n",
      "   71   6237  41T  41T N   CA    319  320  149.29  107.00   42.29   12.16  107.00   42.29   12.16\n",
      "   72   6241  41T  41T CA  C     320  324  148.95  116.50   32.45    9.33  116.50   32.45    9.33\n",
      "   73   6244  41T  42C C   N     324  326  151.85  120.00   31.85    7.24  120.00   31.85    7.24\n",
      "   74   6247  42C  42C N   CA    326  327  149.28  107.00   42.28   12.16  107.00   42.28   12.16\n",
      "   75   6250  42C  42C CA  C     327  330  148.37  116.50   31.88    9.17  116.50   31.88    9.17\n",
      "   76   6252  42C  43R C   N     330  332  153.15  120.00   33.15    7.53  120.00   33.15    7.53\n",
      "   77   6255  43R  43R N   CA    332  333  151.52  107.00   44.52   12.80  107.00   44.52   12.80\n",
      "   78   6258  43R  43R CA  C     333  341  149.37  116.50   32.87    9.45  116.50   32.87    9.45\n",
      "   79   6266  43R  44H C   N     341  343  150.78  120.00   30.77    6.99  120.00   30.77    6.99\n",
      "   80   6269  44H  44H N   CA    343  344  153.37  107.00   46.37   13.33  107.00   46.37   13.33\n",
      "   81   6271  44H  44H CA  C     344  351  150.69  116.50   34.19    9.83  116.50   34.19    9.83\n",
      "   82   6281  44H  45E C   N     351  353  155.01  120.00   35.01    7.96  120.00   35.01    7.96\n",
      "   83   6284  45E  45E N   CA    353  354  155.25  107.00   48.25   13.87  107.00   48.25   13.87\n",
      "   84   6287  45E  45E CA  C     354  360  152.64  116.50   36.14   10.39  116.50   36.14   10.39\n",
      "   85   6293  45E  46M C   N     360  362  159.00  120.00   39.00    8.86  120.00   39.00    8.86\n",
      "   86   6296  46M  46M N   CA    362  363  159.17  107.00   52.17   15.00  107.00   52.17   15.00\n",
      "   87   6299  46M  46M CA  C     363  368  152.84  116.50   36.34   10.45  116.50   36.34   10.45\n",
      "   88   6303  46M  47G C   N     368  370  164.46  120.00   44.46   10.10  120.00   44.46   10.10\n",
      "   89   6305  47G  47G N   CA    370  371  159.26  107.00   52.26   15.03  107.00   52.26   15.03\n",
      "   90   6307  47G  47G CA  C     371  372  139.72  116.50   23.22    6.68  116.50   23.22    6.68\n",
      "   91   6308  47G  48T C   N     372  374  143.13  120.00   23.13    5.26  120.00   23.13    5.26\n",
      "   92   6311  48T  48T N   CA    374  375  131.35  107.00   24.35    7.00  107.00   24.35    7.00\n",
      "   93   6432  60V  61E C   N     463  465  140.25  120.00   20.25    4.60  120.00   20.25    4.60\n",
      "   94   7064 120V 120V N   CA    918  919  125.76  107.00   18.76    5.39  107.00   18.76    5.39\n",
      "   95   7079 121S 122P C   N     929  931  138.98  117.00   21.98    5.47  117.00   21.98    5.47\n",
      "   96   7082 122P 122P N   CA    931  932  136.37  108.20   28.17    6.40  108.20   28.17    6.40\n",
      "   97   7415 154R 154R N   CA   1166 1167  126.70  107.00   19.70    5.67  107.00   19.70    5.67\n",
      "   98   7460 158P 158P N   CA   1200 1201  129.34  108.20   21.14    4.81  108.20   21.14    4.81\n",
      "   99   7582 169P 169P N   CA   1285 1286  135.22  108.20   27.02    6.14  108.20   27.02    6.14\n",
      "  100   8269 236A 236A N   CA   1777 1778  125.59  107.00   18.59    5.34  107.00   18.59    5.34\n",
      "  101   8418 249D 249D N   CA   1883 1884  124.43  107.00   17.43    5.01  107.00   17.43    5.01\n",
      "  102   8561 261I 261I N   CA   1984 1985  131.79  107.00   24.79    7.13  107.00   24.79    7.13\n",
      "  103   8569 261I 262D C   N    1990 1992  150.70  120.00   30.70    6.98  120.00   30.70    6.98\n",
      "  104   8572 262D 262D N   CA   1992 1993  133.55  107.00   26.55    7.64  107.00   26.55    7.64\n",
      "  105   8575 262D 262D CA  C    1993 1998  140.75  116.50   24.25    6.97  116.50   24.25    6.97\n",
      "  106   8583 263I 263I N   CA   2000 2001  134.99  107.00   27.99    8.05  107.00   27.99    8.05\n",
      "  107   8587 263I 263I CA  C    2001 2006  133.80  116.50   17.30    4.98  116.50   17.30    4.98\n",
      "  108   8591 263I 264Q C   N    2006 2008  141.17  120.00   21.17    4.81  120.00   21.17    4.81\n",
      "  109   8594 264Q 264Q N   CA   2008 2009  134.53  107.00   27.53    7.92  107.00   27.53    7.92\n",
      "  110   8597 264Q 264Q CA  C    2009 2015  133.59  116.50   17.10    4.92  116.50   17.10    4.92\n",
      "  111   8606 265T 265T N   CA   2017 2018  133.72  107.00   26.72    7.68  107.00   26.72    7.68\n",
      "  112   8610 265T 265T CA  C    2018 2022  132.59  116.50   16.09    4.63  116.50   16.09    4.63\n",
      "  113   8613 265T 266A C   N    2022 2024  141.16  120.00   21.16    4.81  120.00   21.16    4.81\n",
      "  114   8616 266A 266A N   CA   2024 2025  133.95  107.00   26.95    7.75  107.00   26.95    7.75\n",
      "  115   8623 267K 267K N   CA   2029 2030  131.20  107.00   24.20    6.96  107.00   24.20    6.96\n",
      "  116   8634 268K 268K N   CA   2038 2039  130.97  107.00   23.97    6.89  107.00   23.97    6.89\n",
      "  117   8645 269T 269T N   CA   2047 2048  130.23  107.00   23.23    6.68  107.00   23.23    6.68\n",
      "  118   8655 270A 270A N   CA   2054 2055  130.45  107.00   23.45    6.74  107.00   23.45    6.74\n",
      "  119   8662 271K 271K N   CA   2059 2060  129.04  107.00   22.04    6.34  107.00   22.04    6.34\n",
      "  120   8673 272T 272T N   CA   2068 2069  128.94  107.00   21.94    6.31  107.00   21.94    6.31\n",
      "  121   8683 273L 273L N   CA   2075 2076  128.35  107.00   21.35    6.14  107.00   21.35    6.14\n",
      "  122   8693 274G 274G N   CA   2083 2084  128.36  107.00   21.36    6.14  107.00   21.36    6.14\n",
      "  123   8699 275E 275E N   CA   2087 2088  126.99  107.00   19.99    5.75  107.00   19.99    5.75\n",
      "  124   8711 276M 276M N   CA   2096 2097  125.81  107.00   18.81    5.41  107.00   18.81    5.41\n",
      "  125   8721 277Q 277Q N   CA   2104 2105  124.70  107.00   17.70    5.09  107.00   17.70    5.09\n",
      "  126   8751 279D 279D N   CA   2125 2126  123.07  107.00   16.07    4.62  107.00   16.07    4.62\n",
      "  127  10041 396F 396F CB  CG   3056 3057  152.96  122.30   30.66    6.66  122.30   30.66    6.66\n",
      "  128  11786 556F 556F N   CA   4320 4321  123.62  107.00   16.62    4.78  107.00   16.62    4.78\n",
      "  129  11802 557V 557V N   CA   4331 4332  125.30  107.00   18.30    5.26  107.00   18.30    5.26\n",
      "  130  11812 558D 558D N   CA   4338 4339  126.31  107.00   19.31    5.55  107.00   19.31    5.55\n",
      "  131  11823 559V 559V N   CA   4346 4347  132.17  107.00   25.17    7.24  107.00   25.17    7.24\n",
      "  132  11830 559V 560P C   N    4351 4353  141.79  117.00   24.79    6.17  117.00   24.79    6.17\n",
      "  133  11833 560P 560P N   CA   4353 4354  141.39  108.20   33.20    7.54  108.20   33.20    7.54\n",
      "  134  11845 561L 561L N   CA   4360 4361  131.76  107.00   24.76    7.12  107.00   24.76    7.12\n",
      "  135  11856 562E 562E N   CA   4368 4369  132.00  107.00   25.00    7.19  107.00   25.00    7.19\n",
      "  136  11859 562E 562E CA  C    4369 4375  134.00  116.50   17.50    5.03  116.50   17.50    5.03\n",
      "  137  11865 562E 563L C   N    4375 4377  139.96  120.00   19.96    4.54  120.00   19.96    4.54\n",
      "  138  11868 563L 563L N   CA   4377 4378  134.12  107.00   27.12    7.80  107.00   27.12    7.80\n",
      "  139  11871 563L 563L CA  C    4378 4383  135.23  116.50   18.73    5.39  116.50   18.73    5.39\n",
      "  140  11876 563L 564D C   N    4383 4385  139.85  120.00   19.85    4.51  120.00   19.85    4.51\n",
      "  141  11879 564D 564D N   CA   4385 4386  135.92  107.00   28.93    8.32  107.00   28.93    8.32\n",
      "  142  11882 564D 564D CA  C    4386 4391  135.98  116.50   19.48    5.60  116.50   19.48    5.60\n",
      "  143  11887 564D 565L C   N    4391 4393  141.94  120.00   21.94    4.99  120.00   21.94    4.99\n",
      "  144  11890 565L 565L N   CA   4393 4394  135.20  107.00   28.20    8.11  107.00   28.20    8.11\n",
      "  145  11893 565L 565L CA  C    4394 4399  137.69  116.50   21.19    6.09  116.50   21.19    6.09\n",
      "  146  11898 565L 566E C   N    4399 4401  141.45  120.00   21.45    4.87  120.00   21.45    4.87\n",
      "  147  11901 566E 566E N   CA   4401 4402  138.27  107.00   31.27    8.99  107.00   31.27    8.99\n",
      "  148  11904 566E 566E CA  C    4402 4408  138.59  116.50   22.10    6.35  116.50   22.10    6.35\n",
      "  149  11910 566E 567F C   N    4408 4410  142.92  120.00   22.92    5.21  120.00   22.92    5.21\n",
      "  150  11913 567F 567F N   CA   4410 4411  137.63  107.00   30.63    8.81  107.00   30.63    8.81\n",
      "  151  11916 567F 567F CA  C    4411 4419  141.09  116.50   24.59    7.07  116.50   24.59    7.07\n",
      "  152  11926 567F 568L C   N    4419 4421  143.99  120.00   23.99    5.45  120.00   23.99    5.45\n",
      "  153  11929 568L 568L N   CA   4421 4422  140.36  107.00   33.36    9.59  107.00   33.36    9.59\n",
      "  154  11932 568L 568L CA  C    4422 4427  142.85  116.50   26.35    7.58  116.50   26.35    7.58\n",
      "  155  11937 568L 569R C   N    4427 4429  146.42  120.00   26.42    6.00  120.00   26.42    6.00\n",
      "  156  11940 569R 569R N   CA   4429 4430  141.79  107.00   34.79   10.00  107.00   34.79   10.00\n",
      "  157  11943 569R 569R CA  C    4430 4438  144.30  116.50   27.80    7.99  116.50   27.80    7.99\n",
      "  158  11951 569R 570G C   N    4438 4440  148.15  120.00   28.15    6.40  120.00   28.15    6.40\n",
      "  159  11953 570G 570G N   CA   4440 4441  146.17  107.00   39.17   11.26  107.00   39.17   11.26\n",
      "  160  11955 570G 570G CA  C    4441 4442  143.82  116.50   27.32    7.85  116.50   27.32    7.85\n",
      "  161  11956 570G 571K C   N    4442 4444  148.35  120.00   28.35    6.44  120.00   28.35    6.44\n",
      "  162  11959 571K 571K N   CA   4444 4445  146.21  107.00   39.21   11.27  107.00   39.21   11.27\n",
      "  163  11962 571K 571K CA  C    4445 4451  147.21  116.50   30.71    8.83  116.50   30.71    8.83\n",
      "  164  11967 571K 572G C   N    4451 4453  149.88  120.00   29.88    6.79  120.00   29.88    6.79\n",
      "  165  11969 572G 572G N   CA   4453 4454  147.65  107.00   40.65   11.69  107.00   40.65   11.69\n",
      "  166  11971 572G 572G CA  C    4454 4455  146.00  116.50   29.50    8.48  116.50   29.50    8.48\n",
      "  167  11972 572G 573L C   N    4455 4457  150.19  120.00   30.19    6.86  120.00   30.19    6.86\n",
      "  168  11975 573L 573L N   CA   4457 4458  149.04  107.00   42.04   12.09  107.00   42.04   12.09\n",
      "  169  11978 573L 573L CA  C    4458 4463  146.59  116.50   30.09    8.65  116.50   30.09    8.65\n",
      "  170  11983 573L 574Q C   N    4463 4465  151.40  120.00   31.40    7.14  120.00   31.40    7.14\n",
      "  171  11986 574Q 574Q N   CA   4465 4466  149.12  107.00   42.12   12.11  107.00   42.12   12.11\n",
      "  172  11989 574Q 574Q CA  C    4466 4472  145.55  116.50   29.05    8.35  116.50   29.05    8.35\n",
      "  173  11995 574Q 575S C   N    4472 4474  150.76  120.00   30.76    6.99  120.00   30.76    6.99\n",
      "  174  11998 575S 575S N   CA   4474 4475  148.01  107.00   41.01   11.79  107.00   41.01   11.79\n",
      "  175  12001 575S 575S CA  C    4475 4478  147.65  116.50   31.15    8.96  116.50   31.15    8.96\n",
      "  176  12003 575S 576D C   N    4478 4480  148.87  120.00   28.87    6.56  120.00   28.87    6.56\n",
      "  177  12006 576D 576D N   CA   4480 4481  149.31  107.00   42.31   12.17  107.00   42.31   12.17\n",
      "  178  12007 576D 576D CA  CB   4481 4482  130.85  108.00   22.85    5.30  108.00   22.85    5.30\n",
      "  179  12009 576D 576D CA  C    4481 4486  151.66  116.50   35.16   10.11  116.50   35.16   10.11\n",
      "  180  12014 576D 577E C   N    4486 4488  155.34  120.00   35.34    8.03  120.00   35.34    8.03\n",
      "  181  12017 577E 577E N   CA   4488 4489  153.15  107.00   46.15   13.27  107.00   46.15   13.27\n",
      "  182  12020 577E 577E CA  C    4489 4495  154.38  116.50   37.88   10.89  116.50   37.88   10.89\n",
      "  183  12026 577E 578V C   N    4495 4497  152.95  120.00   32.95    7.49  120.00   32.95    7.49\n",
      "  184  12029 578V 578V N   CA   4497 4498  154.87  107.00   47.87   13.77  107.00   47.87   13.77\n",
      "  185  12033 578V 578V CA  C    4498 4502  153.94  116.50   37.44   10.76  116.50   37.44   10.76\n",
      "  186  12036 578V 579S C   N    4502 4504  156.60  120.00   36.60    8.32  120.00   36.60    8.32\n",
      "  187  12039 579S 579S N   CA   4504 4505  155.60  107.00   48.60   13.97  107.00   48.60   13.97\n",
      "  188  12042 579S 579S CA  C    4505 4508  154.93  116.50   38.43   11.05  116.50   38.43   11.05\n",
      "  189  12044 579S 580M C   N    4508 4510  159.25  120.00   39.25    8.92  120.00   39.25    8.92\n",
      "  190  12047 580M 580M N   CA   4510 4511  160.59  107.00   53.59   15.41  107.00   53.59   15.41\n",
      "  191  12050 580M 580M CA  C    4511 4516  162.02  112.50   49.52    7.11  112.50   49.52    7.11\n",
      "  192  12054 580M 581P C   N    4516 4518  166.15  117.00   49.15   12.24  117.00   49.15   12.24\n",
      "  193  12057 581P 581P N   CA   4518 4519  164.18  108.20   55.99   12.72  108.20   55.99   12.72\n",
      "  194  12061 581P 581P CA  C    4519 4523  159.49  116.50   42.99   12.36  116.50   42.99   12.36\n",
      "  195  12066 581P 582E C   N    4523 4525  162.39  120.00   42.39    9.63  120.00   42.39    9.63\n",
      "  196  12069 582E 582E N   CA   4525 4526  163.45  107.00   56.45   16.23  107.00   56.45   16.23\n",
      "  197  12072 582E 582E CA  C    4526 4532  161.14  116.50   44.64   12.84  116.50   44.64   12.84\n",
      "  198  12078 582E 583A C   N    4532 4534  163.93  120.00   43.93    9.98  120.00   43.93    9.98\n",
      "  199  12081 583A 583A N   CA   4534 4535  162.05  107.00   55.05   15.83  107.00   55.05   15.83\n",
      "  200  12083 583A 583A CA  C    4535 4537  161.70  116.50   45.20   13.00  116.50   45.20   13.00\n",
      "  201  12085 583A 584Q C   N    4537 4539  169.61  120.00   49.61   11.27  120.00   49.61   11.27\n",
      "  202  12088 584Q 584Q N   CA   4539 4540  170.57  107.00   63.57   18.28  107.00   63.57   18.28\n",
      "  203  12091 584Q 584Q CA  C    4540 4546  165.40  116.50   48.90   14.06  116.50   48.90   14.06\n",
      "  204  12097 584Q 585E C   N    4546 4548  166.81  120.00   46.81   10.64  120.00   46.81   10.64\n",
      "  205  12100 585E 585E N   CA   4548 4549  165.95  107.00   58.95   16.95  107.00   58.95   16.95\n",
      "  206  12103 585E 585E CA  C    4549 4555  167.12  116.50   50.62   14.56  116.50   50.62   14.56\n",
      "  207  12109 585E 586D C   N    4555 4557  171.97  120.00   51.97   11.81  120.00   51.97   11.81\n",
      "  208  12112 586D 586D N   CA   4557 4558  171.97  107.00   64.97   18.68  107.00   64.97   18.68\n",
      "  209  12115 586D 586D CA  C    4558 4563  165.67  116.50   49.17   14.14  116.50   49.17   14.14\n",
      "  210  12120 586D 587L C   N    4563 4565  168.15  120.00   48.15   10.94  120.00   48.15   10.94\n",
      "  211  12123 587L 587L N   CA   4565 4566  173.03  107.00   66.04   18.99  107.00   66.04   18.99\n",
      "  212  12126 587L 587L CA  C    4566 4571  177.68  112.50   65.18    9.36  112.50   65.18    9.36\n",
      "  213  12131 587L 588P C   N    4571 4573  173.68  117.00   56.68   14.11  117.00   56.68   14.11\n",
      "  214  12134 588P 588P N   CA   4573 4574  173.39  108.20   65.19   14.82  108.20   65.19   14.82\n",
      "  215  12138 588P 588P CA  C    4574 4578  173.46  116.50   56.96   16.38  116.50   56.96   16.38\n",
      "  216  12141 588P 588P CB  CG   4576 4577  127.63  108.50   19.12    5.14  108.50   19.12    5.14\n",
      "  217  12143 588P 589H C   N    4578 4580  172.91  120.00   52.91   12.02  120.00   52.91   12.02\n",
      "  218  12146 589H 589H N   CA   4580 4581  170.89  107.00   63.89   18.37  107.00   63.89   18.37\n",
      "  219  12148 589H 589H CA  C    4581 4588  170.33  116.50   53.83   15.48  116.50   53.83   15.48\n",
      "  220  12158 589H 590K C   N    4588 4590  176.17  120.00   56.17   12.77  120.00   56.17   12.77\n",
      "  221  12161 590K 590K N   CA   4590 4591  175.87  107.00   68.87   19.80  107.00   68.87   19.80\n",
      "  222  12164 590K 590K CA  C    4591 4597  176.04  112.50   63.54    9.13  112.50   63.54    9.13\n",
      "  223  12169 590K 591P C   N    4597 4599  177.52  117.00   60.52   15.07  117.00   60.52   15.07\n",
      "  224  12172 591P 591P N   CA   4599 4600  178.02  108.20   69.82   15.87  108.20   69.82   15.87\n",
      "  225  12176 591P 591P CA  C    4600 4604  175.54  116.50   59.04   16.98  116.50   59.04   16.98\n",
      "  226  12177 591P 591P CA  CB   4600 4602  132.01  108.50   23.51    6.32  108.50   23.51    6.32\n",
      "  227  12179 591P 591P CB  CG   4602 4603  134.67  108.50   26.17    7.04  108.50   26.17    7.04\n",
      "  228  12181 591P 592T C   N    4604 4606  176.80  120.00   56.80   12.91  120.00   56.80   12.91\n",
      "  229  12184 592T 592T N   CA   4606 4607  176.07  107.00   69.07   19.86  107.00   69.07   19.86\n",
      "  230  12188 592T 592T CA  C    4607 4611  174.38  116.50   57.88   16.64  116.50   57.88   16.64\n",
      "  231  12191 592T 593S C   N    4611 4613  174.98  120.00   54.98   12.50  120.00   54.98   12.50\n",
      "  232  12194 593S 593S N   CA   4613 4614  177.27  107.00   70.27   20.20  107.00   70.27   20.20\n",
      "  233  12197 593S 593S CA  C    4614 4617  178.54  116.50   62.04   17.84  116.50   62.04   17.84\n",
      "  234  12199 593S 594S C   N    4617 4619  177.60  120.00   57.60   13.09  120.00   57.60   13.09\n",
      "  235  12202 594S 594S N   CA   4619 4620  179.80  107.00   72.80   20.93  107.00   72.80   20.93\n",
      "  236  12205 594S 594S CA  C    4620 4623  176.79  116.50   60.29   17.33  116.50   60.29   17.33\n",
      "  237  12207 594S 595S C   N    4623 4625  175.35  120.00   55.35   12.58  120.00   55.35   12.58\n",
      "  238  12210 595S 595S N   CA   4625 4626  178.33  107.00   71.33   20.51  107.00   71.33   20.51\n",
      "  239  12213 595S 595S CA  C    4626 4629  177.32  116.50   60.82   17.49  116.50   60.82   17.49\n",
      "  240  12215 595S 596M C   N    4629 4631  176.67  120.00   56.67   12.88  120.00   56.67   12.88\n",
      "  241  12218 596M 596M N   CA   4631 4632  176.47  107.00   69.47   19.98  107.00   69.47   19.98\n",
      "  242  12221 596M 596M CA  C    4632 4637  175.92  116.50   59.42   17.09  116.50   59.42   17.09\n",
      "  243  12225 596M 597K C   N    4637 4639  179.19  120.00   59.19   13.45  120.00   59.19   13.45\n",
      "  244  12228 597K 597K N   CA   4639 4640  177.19  107.00   70.19   20.18  107.00   70.19   20.18\n",
      "  245  12231 597K 597K CA  C    4640 4646  176.56  116.50   60.06   17.27  116.50   60.06   17.27\n",
      "  246  12236 597K 598T C   N    4646 4648  177.43  120.00   57.43   13.05  120.00   57.43   13.05\n",
      "  247  12239 598T 598T N   CA   4648 4649  176.64  107.00   69.64   20.02  107.00   69.64   20.02\n",
      "  248  12243 598T 598T CA  C    4649 4653  174.68  116.50   58.18   16.73  116.50   58.18   16.73\n",
      "  249  12246 598T 599V C   N    4653 4655  176.45  120.00   56.45   12.83  120.00   56.45   12.83\n",
      "  250  12249 599V 599V N   CA   4655 4656  176.96  107.00   69.96   20.12  107.00   69.96   20.12\n",
      "  251  12253 599V 599V CA  C    4656 4660  175.43  116.50   58.93   16.95  116.50   58.93   16.95\n",
      "  252  12256 599V 600D C   N    4660 4662  176.48  120.00   56.48   12.84  120.00   56.48   12.84\n",
      "  253  12259 600D 600D N   CA   4662 4663  176.23  107.00   69.23   19.91  107.00   69.23   19.91\n",
      "  254  12262 600D 600D CA  C    4663 4668  176.41  116.50   59.91   17.23  116.50   59.91   17.23\n",
      "  255  12267 600D 601E C   N    4668 4670  177.03  120.00   57.03   12.96  120.00   57.03   12.96\n",
      "  256  12270 601E 601E N   CA   4670 4671  177.27  107.00   70.27   20.20  107.00   70.27   20.20\n",
      "  257  12273 601E 601E CA  C    4671 4677  175.06  116.50   58.57   16.84  116.50   58.57   16.84\n",
      "  258  12279 601E 602E C   N    4677 4679  177.59  120.00   57.59   13.09  120.00   57.59   13.09\n",
      "  259  12282 602E 602E N   CA   4679 4680  177.49  107.00   70.49   20.27  107.00   70.49   20.27\n",
      "  260  12285 602E 602E CA  C    4680 4686  175.83  116.50   59.33   17.06  116.50   59.33   17.06\n",
      "  261  12291 602E 603A C   N    4686 4688  177.65  120.00   57.65   13.10  120.00   57.65   13.10\n",
      "  262  12294 603A 603A N   CA   4688 4689  178.63  107.00   71.63   20.60  107.00   71.63   20.60\n",
      "  263  12296 603A 603A CA  C    4689 4691  176.88  116.50   60.38   17.36  116.50   60.38   17.36\n",
      "  264  12298 603A 604L C   N    4691 4693  177.52  120.00   57.52   13.07  120.00   57.52   13.07\n",
      "  265  12301 604L 604L N   CA   4693 4694  178.28  107.00   71.28   20.49  107.00   71.28   20.49\n",
      "  266  12302 604L 604L CA  CB   4694 4695  141.11  113.50   27.61    6.78  113.50   27.61    6.78\n",
      "  267  12304 604L 604L CA  C    4694 4699  176.83  116.50   60.34   17.35  116.50   60.34   17.35\n",
      "  268  12309 604L 605A C   N    4699 4701  176.97  120.00   56.97   12.95  120.00   56.97   12.95\n",
      "  269  12312 605A 605A N   CA   4701 4702  176.69  107.00   69.69   20.04  107.00   69.69   20.04\n",
      "  270  12314 605A 605A CA  C    4702 4704  175.39  116.50   58.89   16.93  116.50   58.89   16.93\n",
      "  271  12316 605A 606M C   N    4704 4706  178.20  120.00   58.20   13.23  120.00   58.20   13.23\n",
      "  272  12319 606M 606M N   CA   4706 4707  175.38  107.00   68.38   19.66  107.00   68.38   19.66\n",
      "  273  12321 606M 606M CA  C    4707 4712  152.19  121.00   31.19    8.97  121.00   31.19    8.97\n",
      "  274  12322 606M 606M CA  C    4707 4712  160.45  116.50   43.95   12.64  116.50   43.95   12.64\n",
      "  275  12326 606M 607L C   N    4712 4714  152.68  120.00   32.68    7.43  120.00   32.68    7.43\n",
      "  276  12329 607L 607L N   CA   4714 4715  145.13  107.00   38.13   10.96  107.00   38.13   10.96\n",
      "  277  12340 608L 608L N   CA   4722 4723  124.78  107.00   17.78    5.11  107.00   17.78    5.11\n",
      "  278  12351 609S 609S N   CA   4730 4731  131.36  107.00   24.36    7.00  107.00   24.36    7.00\n",
      "  279  12436 618R 618R N   CA   4793 4794  127.08  107.00   20.08    5.77  107.00   20.08    5.77\n",
      "  280  12451 619Y 619Y CA  CB   4805 4806  137.97  107.50   30.47    7.04  107.50   30.47    7.04\n",
      "  281  12468 620A 620A N   CA   4816 4817  128.68  107.00   21.68    6.23  107.00   21.68    6.23\n",
      "  282  12475 621L 621L N   CA   4821 4822  128.53  107.00   21.53    6.19  107.00   21.53    6.19\n",
      "  283  12486 622L 622L N   CA   4829 4830  125.74  107.00   18.74    5.39  107.00   18.74    5.39\n",
      "  284  12497 623Q 623Q N   CA   4837 4838  125.62  107.00   18.62    5.35  107.00   18.62    5.35\n",
      "  285  13300 700L 700L N   CA   5414 5415  125.94  107.00   18.94    5.44  107.00   18.94    5.44\n",
      "  286  13311 701L 701L N   CA   5422 5423  125.33  107.00   18.33    5.27  107.00   18.33    5.27\n",
      "  287  13322 702F 702F N   CA   5430 5431  127.07  107.00   20.07    5.77  107.00   20.07    5.77\n",
      "  288  13338 703N 703N N   CA   5441 5442  129.03  107.00   22.03    6.33  107.00   22.03    6.33\n",
      "  289  13349 704D 704D N   CA   5449 5450  133.41  107.00   26.41    7.59  107.00   26.41    7.59\n",
      "  290  13360 705E 705E N   CA   5457 5458  131.46  107.00   24.46    7.03  107.00   24.46    7.03\n",
      "  291  13363 705E 705E CA  C    5458 5464  134.41  116.50   17.91    5.15  116.50   17.91    5.15\n",
      "  292  13372 706K 706K N   CA   5466 5467  133.91  107.00   26.91    7.74  107.00   26.91    7.74\n",
      "  293  13375 706K 706K CA  C    5467 5473  133.30  116.50   16.80    4.83  116.50   16.80    4.83\n",
      "  294  13380 706K 707V C   N    5473 5475  141.27  120.00   21.27    4.83  120.00   21.27    4.83\n",
      "  295  13383 707V 707V N   CA   5475 5476  136.65  107.00   29.65    8.53  107.00   29.65    8.53\n",
      "  296  13387 707V 707V CA  C    5476 5480  133.88  116.50   17.38    5.00  116.50   17.38    5.00\n",
      "  297  13390 707V 708A C   N    5480 5482  143.43  120.00   23.43    5.33  120.00   23.43    5.33\n",
      "  298  13393 708A 708A N   CA   5482 5483  135.70  107.00   28.70    8.25  107.00   28.70    8.25\n",
      "  299  13395 708A 708A CA  C    5483 5485  137.33  116.50   20.83    5.99  116.50   20.83    5.99\n",
      "  300  13397 708A 709E C   N    5485 5487  143.01  120.00   23.01    5.23  120.00   23.01    5.23\n",
      "  301  13400 709E 709E N   CA   5487 5488  139.33  107.00   32.33    9.30  107.00   32.33    9.30\n",
      "  302  13403 709E 709E CA  C    5488 5494  137.38  116.50   20.88    6.00  116.50   20.88    6.00\n",
      "  303  13409 709E 710S C   N    5494 5496  144.97  120.00   24.97    5.67  120.00   24.97    5.67\n",
      "  304  13412 710S 710S N   CA   5496 5497  139.45  107.00   32.45    9.33  107.00   32.45    9.33\n",
      "  305  13415 710S 710S CA  C    5497 5500  138.43  116.50   21.93    6.31  116.50   21.93    6.31\n",
      "  306  13420 711L 711L N   CA   5502 5503  136.76  107.00   29.76    8.56  107.00   29.76    8.56\n",
      "  307  13423 711L 711L CA  C    5503 5508  138.58  116.50   22.08    6.35  116.50   22.08    6.35\n",
      "  308  13428 711L 712N C   N    5508 5510  139.85  120.00   19.85    4.51  120.00   19.85    4.51\n",
      "  309  13431 712N 712N N   CA   5510 5511  142.99  107.00   36.00   10.35  107.00   36.00   10.35\n",
      "  310  13439 712N 713P C   N    5516 5518  145.26  117.00   28.26    7.04  117.00   28.26    7.04\n",
      "  311  13442 713P 713P N   CA   5518 5519  139.34  108.20   31.14    7.08  108.20   31.14    7.08\n",
      "  312  13446 713P 713P CA  C    5519 5523  148.21  112.50   35.71    5.13  112.50   35.71    5.13\n",
      "  313  13451 713P 714P C   N    5523 5525  160.01  117.00   43.01   10.71  117.00   43.01   10.71\n",
      "  314  13454 714P 714P N   CA   5525 5526  147.09  108.20   38.89    8.84  108.20   38.89    8.84\n",
      "  315  13458 714P 714P CA  C    5526 5530  140.71  116.50   24.21    6.96  116.50   24.21    6.96\n",
      "  316  13463 714P 715F C   N    5530 5532  151.43  120.00   31.43    7.14  120.00   31.43    7.14\n",
      "  317  13466 715F 715F N   CA   5532 5533  144.54  107.00   37.54   10.79  107.00   37.54   10.79\n",
      "  318  13469 715F 715F CA  C    5533 5541  136.73  116.50   20.23    5.82  116.50   20.23    5.82\n",
      "  319  13479 715F 716S C   N    5541 5543  154.66  120.00   34.66    7.88  120.00   34.66    7.88\n",
      "  320  13482 716S 716S N   CA   5543 5544  147.53  107.00   40.53   11.65  107.00   40.53   11.65\n",
      "  321  13485 716S 716S CA  C    5544 5547  139.15  116.50   22.65    6.51  116.50   22.65    6.51\n",
      "  322  13487 716S 717M C   N    5547 5549  148.79  120.00   28.79    6.54  120.00   28.79    6.54\n",
      "  323  13490 717M 717M N   CA   5549 5550  146.14  107.00   39.14   11.25  107.00   39.14   11.25\n",
      "  324  13493 717M 717M CA  C    5550 5555  144.68  116.50   28.18    8.10  116.50   28.18    8.10\n",
      "  325  13497 717M 718A C   N    5555 5557  152.33  120.00   32.33    7.35  120.00   32.33    7.35\n",
      "  326  13500 718A 718A N   CA   5557 5558  151.39  107.00   44.39   12.76  107.00   44.39   12.76\n",
      "  327  13502 718A 718A CA  C    5558 5560  150.78  116.50   34.28    9.86  116.50   34.28    9.86\n",
      "  328  13504 718A 719F C   N    5560 5562  167.13  120.00   47.13   10.71  120.00   47.13   10.71\n",
      "  329  13507 719F 719F N   CA   5562 5563  153.32  107.00   46.32   13.32  107.00   46.32   13.32\n",
      "  330  13510 719F 719F CA  C    5563 5571  149.83  116.50   33.33    9.58  116.50   33.33    9.58\n",
      "  331  13520 719F 720L C   N    5571 5573  154.41  120.00   34.41    7.82  120.00   34.41    7.82\n",
      "  332  13523 720L 720L N   CA   5573 5574  155.15  107.00   48.15   13.84  107.00   48.15   13.84\n",
      "  333  13526 720L 720L CA  C    5574 5579  149.67  116.50   33.17    9.54  116.50   33.17    9.54\n",
      "  334  13531 720L 721Y C   N    5579 5581  155.26  120.00   35.26    8.01  120.00   35.26    8.01\n",
      "  335  13534 721Y 721Y N   CA   5581 5582  152.10  107.00   45.10   12.97  107.00   45.10   12.97\n",
      "  336  13537 721Y 721Y CA  C    5582 5591  151.18  116.50   34.68    9.97  116.50   34.68    9.97\n",
      "  337  13549 721Y 722F C   N    5591 5593  170.19  120.00   50.19   11.41  120.00   50.19   11.41\n",
      "  338  13552 722F 722F N   CA   5593 5594  162.27  107.00   55.27   15.89  107.00   55.27   15.89\n",
      "  339  13555 722F 722F CA  C    5594 5602  155.13  116.50   38.63   11.11  116.50   38.63   11.11\n",
      "  340  13565 722F 723Y C   N    5602 5604  169.95  120.00   49.95   11.35  120.00   49.95   11.35\n",
      "  341  13568 723Y 723Y N   CA   5604 5605  163.43  107.00   56.43   16.23  107.00   56.43   16.23\n",
      "  342  13571 723Y 723Y CA  C    5605 5614  162.15  116.50   45.65   13.13  116.50   45.65   13.13\n",
      "  343  13583 723Y 724K C   N    5614 5616  163.42  120.00   43.42    9.87  120.00   43.42    9.87\n",
      "  344  13586 724K 724K N   CA   5616 5617  168.67  107.00   61.67   17.73  107.00   61.67   17.73\n",
      "  345  13589 724K 724K CA  C    5617 5623  167.44  116.50   50.94   14.65  116.50   50.94   14.65\n",
      "  346  13594 724K 725R C   N    5623 5625  164.65  120.00   44.65   10.15  120.00   44.65   10.15\n",
      "  347  13597 725R 725R N   CA   5625 5626  166.49  107.00   59.49   17.11  107.00   59.49   17.11\n",
      "  348  13600 725R 725R CA  C    5626 5634  162.01  116.50   45.51   13.09  116.50   45.51   13.09\n",
      "  349  13608 725R 726V C   N    5634 5636  165.72  120.00   45.72   10.39  120.00   45.72   10.39\n",
      "  350  13611 726V 726V N   CA   5636 5637  167.70  107.00   60.70   17.45  107.00   60.70   17.45\n",
      "  351  13615 726V 726V CA  C    5637 5641  164.18  116.50   47.68   13.71  116.50   47.68   13.71\n",
      "  352  13618 726V 727G C   N    5641 5643  170.12  120.00   50.12   11.39  120.00   50.12   11.39\n",
      "  353  13620 727G 727G N   CA   5643 5644  170.78  107.00   63.78   18.34  107.00   63.78   18.34\n",
      "  354  13622 727G 727G CA  C    5644 5645  167.37  116.50   50.87   14.63  116.50   50.87   14.63\n",
      "  355  13623 727G 728K C   N    5645 5647  169.14  120.00   49.14   11.17  120.00   49.14   11.17\n",
      "  356  13626 728K 728K N   CA   5647 5648  152.05  107.00   45.05   10.24  107.00   45.05   10.24\n",
      "  357  13628 728K 728K CA  C    5648 5654  142.99  118.00   24.99    5.08  118.00   24.99    5.08\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19289 601E 601E CA  N    4671 4670   54.28   35.27   19.01    4.53   35.27   19.01    4.53\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22527  47G  60V CA  CA    371  459   26.70    5.82   20.88    7.03    5.82   20.88    7.03\n",
      "    2  22528  47G  61E CA  CA    371  466   30.02    7.97   22.05    8.38    7.97   22.05    8.38\n",
      "    3  22529  47G  62H CA  CA    371  475   31.80    9.00   22.80    9.30    9.00   22.80    9.30\n",
      "    4  22530  47G  63V CA  CA    371  485   33.40   12.77   20.63    9.67   12.77   20.63    9.67\n",
      "    5  22531  47G 243C CA  CA    371 1840   23.44   13.26   10.17   14.77   13.26   10.17   14.77\n",
      "    6  22532  47G 244D CA  CA    371 1846   21.42   12.85    8.57   10.39   12.85    8.57   10.39\n",
      "    7  22533  47G 245D CA  CA    371 1854   23.86   10.17   13.69   18.48   10.17   13.69   18.48\n",
      "    8  22534  47G 246E CA  CA    371 1862   27.42   10.81   16.61   25.68   10.81   16.61   25.68\n",
      "    9  22535  47G 247V CA  CA    371 1871   30.03   11.44   18.59   30.27   11.44   18.59   30.27\n",
      "   10  22536  47G 248S CA  CA    371 1878   33.89   13.40   20.49   24.39   13.40   20.49   24.39\n",
      "   11  22537  48T  60V CA  CA    375  459   21.43    3.89   17.53    6.42    3.89   17.53    6.42\n",
      "   12  22538  48T  61E CA  CA    375  466   24.71    6.49   18.22    7.52    6.49   18.22    7.52\n",
      "   13  22539  48T  62H CA  CA    375  475   26.57    9.04   17.53    7.69    9.04   17.53    7.69\n",
      "   14  22540  48T  63V CA  CA    375  485   28.24   12.76   15.47    7.99   12.76   15.47    7.99\n",
      "   15  22541  48T 237D CA  CA    375 1783   31.93   13.73   18.20   30.03   13.73   18.20   30.03\n",
      "   16  22542  48T 239Y CA  CA    375 1802   24.60   13.74   10.86   20.03   13.74   10.86   20.03\n",
      "   17  22544  48T 245D CA  CA    375 1854   18.52   10.05    8.47   13.88   10.05    8.47   13.88\n",
      "   18  22545  48T 246E CA  CA    375 1862   22.08    9.43   12.65   25.60    9.43   12.65   25.60\n",
      "   19  22546  48T 247V CA  CA    375 1871   24.72   10.45   14.27   30.18   10.45   14.27   30.18\n",
      "   20  22547  48T 248S CA  CA    375 1878   28.61   12.09   16.51   25.57   12.09   16.51   25.57\n",
      "   21  22566  61E 235G CA  CA    466 1774   17.12   12.03    5.10    9.34   12.03    5.10    9.34\n",
      "   22  22567  61E 236A CA  CA    466 1778   14.71   11.52    3.19    6.67   11.52    3.19    6.67\n",
      "   23  22580  61E 252F CA  CA    466 1909   16.37   13.72    2.65    4.73   13.72    2.65    4.73\n",
      "   24  22912  86D 254N CA  CA    676 1925   22.71   13.93    8.78    7.93   13.93    8.78    7.93\n",
      "   25  22913  86D 255H CA  CA    676 1933   19.02   12.25    6.76    8.40   12.25    6.76    8.40\n",
      "   26  22914  86D 256M CA  CA    676 1943   20.11   13.51    6.60    7.58   13.51    6.60    7.58\n",
      "   27  22915  86D 257Q CA  CA    676 1951   21.58   12.79    8.79    6.09   12.79    8.79    6.09\n",
      "   28  22916  86D 258H CA  CA    676 1960   18.29    8.96    9.34    6.20    8.96    9.34    6.20\n",
      "   29  22928  96N 255H CA  CA    744 1933   22.29   13.60    8.69    9.96   13.60    8.69    9.96\n",
      "   30  22929  96N 256M CA  CA    744 1943   22.90   13.80    9.11    9.98   13.80    9.11    9.98\n",
      "   31  22930  96N 257Q CA  CA    744 1951   24.24   12.80   11.44    7.70   12.80   11.44    7.70\n",
      "   32  22931  96N 258H CA  CA    744 1960   20.88    9.26   11.62    7.51    9.26   11.62    7.51\n",
      "   33  22932  96N 259F CA  CA    744 1970   19.51    9.20   10.30    5.63    9.20   10.30    5.63\n",
      "   34  22946 103P 254N CA  CA    793 1925   20.92   13.24    7.67    9.61   13.24    7.67    9.61\n",
      "   35  22947 103P 255H CA  CA    793 1933   17.46   11.11    6.35   11.37   11.11    6.35   11.37\n",
      "   36  22948 103P 256M CA  CA    793 1943   17.54   10.74    6.81   11.70   10.74    6.81   11.70\n",
      "   37  22949 103P 257Q CA  CA    793 1951   18.31   10.47    7.84    7.13   10.47    7.84    7.13\n",
      "   38  22950 103P 258H CA  CA    793 1960   14.86    7.11    7.75    6.19    7.11    7.75    6.19\n",
      "   39  22951 103P 259F CA  CA    793 1970   13.33    5.98    7.34    4.83    5.98    7.34    4.83\n",
      "   40  22997 117K 606M CA  CA    893 4707   39.38   13.64   25.74   10.38   13.64   25.74   10.38\n",
      "   41  23023 118S 231V CA  CA    902 1747   15.56   12.22    3.34    8.12   12.22    3.34    8.12\n",
      "   42  23024 118S 253A CA  CA    902 1920   19.48   13.92    5.56    8.50   13.92    5.56    8.50\n",
      "   43  23025 118S 254N CA  CA    902 1925   18.06   13.99    4.07    5.38   13.99    4.07    5.38\n",
      "   44  23026 118S 255H CA  CA    902 1933   15.47   12.02    3.46    6.58   12.02    3.46    6.58\n",
      "   45  23027 118S 256M CA  CA    902 1943   13.33    9.04    4.29    8.45    9.04    4.29    8.45\n",
      "   46  23034 118S 606M CA  CA    902 4707   41.40   11.87   29.53   10.91   11.87   29.53   10.91\n",
      "   47  23053 122P 231V CA  CA    932 1747   14.68   11.96    2.72    5.56   11.96    2.72    5.56\n",
      "   48  23054 122P 256M CA  CA    932 1943   15.85   11.27    4.58    6.98   11.27    4.58    6.98\n",
      "   49  23061 122P 606M CA  CA    932 4707   42.24   11.70   30.54   10.76   11.70   30.54   10.76\n",
      "   50  23079 123P 231V CA  CA    939 1747   11.82    9.36    2.46    4.79    9.36    2.46    4.79\n",
      "   51  23080 123P 232T CA  CA    939 1754   15.01   10.94    4.07    6.51   10.94    4.07    6.51\n",
      "   52  23081 123P 233P CA  CA    939 1761   18.21   13.32    4.90    7.16   13.32    4.90    7.16\n",
      "   53  23090 123P 606M CA  CA    939 4707   39.42    9.60   29.82   10.29    9.60   29.82   10.29\n",
      "   54  23091 123P 607L CA  CA    939 4715   32.32   12.23   20.09    6.21   12.23   20.09    6.21\n",
      "   55  23108 129C 229G CA  CA    986 1736   11.32    7.57    3.75    7.15    7.57    3.75    7.15\n",
      "   56  23109 129C 230T CA  CA    986 1740   15.04   11.11    3.93    6.92   11.11    3.93    6.92\n",
      "   57  23110 129C 231V CA  CA    986 1747   17.07   10.93    6.14   11.32   10.93    6.14   11.32\n",
      "   58  23111 129C 232T CA  CA    986 1754   20.38   12.90    7.48   11.22   12.90    7.48   11.22\n",
      "   59  23116 129C 606M CA  CA    986 4707   45.40   12.81   32.60   10.88   12.81   32.60   10.88\n",
      "   60  23133 140G 227K CA  CA   1076 1719   16.78   13.32    3.46    5.54   13.32    3.46    5.54\n",
      "   61  23134 140G 228L CA  CA   1076 1728   16.11    9.80    6.31   13.04    9.80    6.31   13.04\n",
      "   62  23135 140G 229G CA  CA   1076 1736   19.15    9.76    9.39   15.83    9.76    9.39   15.83\n",
      "   63  23136 140G 230T CA  CA   1076 1740   22.43   13.58    8.85   14.57   13.58    8.85   14.57\n",
      "   64  23160 155L 228L CA  CA   1178 1728   11.67    8.19    3.47    9.53    8.19    3.47    9.53\n",
      "   65  23161 155L 229G CA  CA   1178 1736   15.09    9.59    5.50   11.34    9.59    5.50   11.34\n",
      "   66  23162 155L 230T CA  CA   1178 1740   18.02   13.09    4.93   10.44   13.09    4.93   10.44\n",
      "   67  23163 155L 231V CA  CA   1178 1747   19.50   13.69    5.81   12.81   13.69    5.81   12.81\n",
      "   68  23190 156Q 228L CA  CA   1186 1728   12.37   10.24    2.13    5.00   10.24    2.13    5.00\n",
      "   69  23191 156Q 229G CA  CA   1186 1736   15.32   12.03    3.29    5.75   12.03    3.29    5.75\n",
      "   70  23224 157C 228L CA  CA   1195 1728   13.02   10.94    2.08    5.83   10.94    2.08    5.83\n",
      "   71  23225 157C 229G CA  CA   1195 1736   16.08   13.47    2.62    5.15   13.47    2.62    5.15\n",
      "   72  23235 168S 193M CA  CA   1280 1469   17.10   10.88    6.23    5.93   10.88    6.23    5.93\n",
      "   73  23236 168S 194C CA  CA   1280 1477   16.35   12.24    4.10    5.47   12.24    4.10    5.47\n",
      "   74  23237 168S 197C CA  CA   1280 1498   16.63   13.00    3.64    5.00   13.00    3.64    5.00\n",
      "   75  23238 168S 198G CA  CA   1280 1504   16.59   11.08    5.52    8.22   11.08    5.52    8.22\n",
      "   76  23239 168S 199T CA  CA   1280 1508   12.88    7.73    5.15    7.52    7.73    5.15    7.52\n",
      "   77  23240 168S 200I CA  CA   1280 1515   13.35    6.23    7.12    7.50    6.23    7.12    7.50\n",
      "   78  23252 168S 228L CA  CA   1280 1728   21.43   13.67    7.76   14.93   13.67    7.76   14.93\n",
      "   79  23261 179K 198G CA  CA   1359 1504   18.90   13.88    5.02    7.61   13.88    5.02    7.61\n",
      "   80  23262 179K 199T CA  CA   1359 1508   15.01   10.15    4.86    6.93   10.15    4.86    6.93\n",
      "   81  23263 179K 200I CA  CA   1359 1515   13.91    8.82    5.09    5.21    8.82    5.09    5.21\n",
      "   82  23284 180C 200I CA  CA   1368 1515   13.71   10.55    3.16    5.09   10.55    3.16    5.09\n",
      "   83  23334 182L 227K CA  CA   1379 1719   15.92   13.34    2.58    5.24   13.34    2.58    5.24\n",
      "   84  23335 182L 228L CA  CA   1379 1728   15.75   12.33    3.41    8.30   12.33    3.41    8.30\n",
      "   85  23336 182L 229G CA  CA   1379 1736   18.12   13.56    4.57    8.20   13.56    4.57    8.20\n",
      "   86  23435 191N 231V CA  CA   1447 1747   14.16   12.10    2.06    4.79   12.10    2.06    4.79\n",
      "   87  23471 192W 231V CA  CA   1455 1747   13.60   11.32    2.28    5.05   11.32    2.28    5.05\n",
      "   88  23526 193M 606M CA  CA   1469 4707   39.16   13.92   25.25   26.77   13.92   25.25   26.77\n",
      "   89  23685 198G 252F CA  CA   1504 1909   15.57   13.63    1.94    5.01   13.63    1.94    5.01\n",
      "   90  23815 203P 230T CA  CA   1533 1740   15.82   13.52    2.30    4.93   13.52    2.30    4.93\n",
      "   91  24079 225V 231V CA  CA   1705 1747   14.06   11.82    2.24    5.09   11.82    2.24    5.09\n",
      "   92  24107 226V 231V CA  CA   1712 1747   10.28    8.04    2.24    5.31    8.04    2.24    5.31\n",
      "   93  24135 226V 606M CA  CA   1712 4707   35.65   11.91   23.74   63.46   11.91   23.74   63.46\n",
      "   94  24138 227K 231V CA  CA   1719 1747    9.66    7.49    2.17    4.66    7.49    2.17    4.66\n",
      "   95  24158 227K 606M CA  CA   1719 4707   36.85   12.70   24.15   52.46   12.70   24.15   52.46\n",
      "   96  24175 228L 606M CA  CA   1728 4707   36.57   11.25   25.33   66.77   11.25   25.33   66.77\n",
      "   97  24187 229G 606M CA  CA   1736 4707   34.21   10.88   23.33   47.29   10.88   23.33   47.29\n",
      "   98  24188 229G 607L CA  CA   1736 4715   27.01   13.87   13.14   19.03   13.87   13.14   19.03\n",
      "   99  24189 229G 728K CA  CA   1736 5648   20.66   12.91    7.75   13.57   12.91    7.75   13.57\n",
      "  100  24198 230T 246E CA  CA   1740 1862   16.19   12.94    3.25    6.19   12.94    3.25    6.19\n",
      "  101  24206 230T 606M CA  CA   1740 4707   31.02    9.84   21.17   50.21    9.84   21.17   50.21\n",
      "  102  24207 230T 607L CA  CA   1740 4715   23.84   12.62   11.22   19.24   12.62   11.22   19.24\n",
      "  103  24208 230T 728K CA  CA   1740 5648   24.34   12.71   11.64   23.46   12.71   11.64   23.46\n",
      "  104  24214 231V 238F CA  CA   1747 1791   12.09    9.55    2.54    5.89    9.55    2.54    5.89\n",
      "  105  24215 231V 239Y CA  CA   1747 1802   15.72   13.01    2.71    5.70   13.01    2.71    5.70\n",
      "  106  24216 231V 248S CA  CA   1747 1878   15.57   12.98    2.59    4.72   12.98    2.59    4.72\n",
      "  107  24229 231V 606M CA  CA   1747 4707   28.38    6.06   22.32   60.63    6.06   22.32   60.63\n",
      "  108  24230 231V 607L CA  CA   1747 4715   21.18    9.09   12.09   24.40    9.09   12.09   24.40\n",
      "  109  24231 231V 728K CA  CA   1747 5648   26.26    9.49   16.76   38.35    9.49   16.76   38.35\n",
      "  110  24245 232T 606M CA  CA   1754 4707   25.20    5.37   19.83   49.02    5.37   19.83   49.02\n",
      "  111  24246 232T 607L CA  CA   1754 4715   18.03    6.72   11.31   23.01    6.72   11.31   23.01\n",
      "  112  24247 232T 728K CA  CA   1754 5648   28.83    7.50   21.33   45.64    7.50   21.33   45.64\n",
      "  113  24261 233P 262D CA  CA   1761 1993   14.00    8.58    5.41   10.30    8.58    5.41   10.30\n",
      "  114  24262 233P 606M CA  CA   1761 4707   21.62    5.20   16.42   39.94    5.20   16.42   39.94\n",
      "  115  24263 233P 607L CA  CA   1761 4715   14.45    4.36   10.09   24.13    4.36   10.09   24.13\n",
      "  116  24264 233P 728K CA  CA   1761 5648   32.25    6.73   25.51   55.50    6.73   25.51   55.50\n",
      "  117  24268 234S 248S CA  CA   1768 1878   15.61   11.90    3.71    5.26   11.90    3.71    5.26\n",
      "  118  24277 234S 606M CA  CA   1768 4707   22.20    8.18   14.02   26.28    8.18   14.02   26.28\n",
      "  119  24278 234S 607L CA  CA   1768 4715   15.16    8.02    7.14   11.68    8.02    7.14   11.68\n",
      "  120  24279 234S 728K CA  CA   1768 5648   32.79   10.59   22.21   35.30   10.59   22.21   35.30\n",
      "  121  24283 235G 246E CA  CA   1774 1862   17.46   12.57    4.89    9.11   12.57    4.89    9.11\n",
      "  122  24284 235G 247V CA  CA   1774 1871   15.23   11.30    3.94    8.14   11.30    3.94    8.14\n",
      "  123  24285 235G 248S CA  CA   1774 1878   11.97    8.95    3.02    5.61    8.95    3.02    5.61\n",
      "  124  24297 235G 606M CA  CA   1774 4707   24.91    7.52   17.38   40.46    7.52   17.38   40.46\n",
      "  125  24298 235G 607L CA  CA   1774 4715   17.89    9.05    8.84   16.33    9.05    8.84   16.33\n",
      "  126  24299 235G 728K CA  CA   1774 5648   31.67   11.48   20.18   39.22   11.48   20.18   39.22\n",
      "  127  24303 236A 245D CA  CA   1778 1854   17.71   13.89    3.82    6.52   13.89    3.82    6.52\n",
      "  128  24304 236A 246E CA  CA   1778 1862   14.15   10.23    3.92    8.37   10.23    3.92    8.37\n",
      "  129  24305 236A 247V CA  CA   1778 1871   12.46    9.89    2.57    6.04    9.89    2.57    6.04\n",
      "  130  24318 236A 606M CA  CA   1778 4707   28.85    8.77   20.08   51.35    8.77   20.08   51.35\n",
      "  131  24319 236A 607L CA  CA   1778 4715   21.82   11.42   10.39   19.65   11.42   10.39   19.65\n",
      "  132  24320 236A 728K CA  CA   1778 5648   28.72   12.98   15.74   33.77   12.98   15.74   33.77\n",
      "  133  24324 237D 244D CA  CA   1783 1846   16.40   13.33    3.07    4.73   13.33    3.07    4.73\n",
      "  134  24325 237D 245D CA  CA   1783 1854   13.82   10.62    3.21    5.23   10.62    3.21    5.23\n",
      "  135  24326 237D 246E CA  CA   1783 1862   10.24    7.05    3.19    6.53    7.05    3.19    6.53\n",
      "  136  24336 237D 606M CA  CA   1783 4707   32.42   12.37   20.04   41.02   12.37   20.04   41.02\n",
      "  137  24354 238F 606M CA  CA   1791 4707   34.61   13.44   21.18   51.54   13.44   21.18   51.54\n",
      "  138  24405 246E 252F CA  CA   1862 1909   15.94   13.43    2.50    5.23   13.43    2.50    5.23\n",
      "  139  24409 247V 252F CA  CA   1871 1909   13.08   10.88    2.21    5.48   10.88    2.21    5.48\n",
      "  140  24424 249D 606M CA  CA   1884 4707   30.85   13.37   17.48   42.05   13.37   17.48   42.05\n",
      "  141  24431 250V 606M CA  CA   1892 4707   27.49   13.43   14.07   25.83   13.43   14.07   25.83\n",
      "  142  24441 251H 606M CA  CA   1899 4707   27.19   12.08   15.11   30.40   12.08   15.11   30.40\n",
      "  143  24442 251H 607L CA  CA   1899 4715   21.07   13.68    7.39   10.51   13.68    7.39   10.51\n",
      "  144  24452 252F 606M CA  CA   1909 4707   27.05    8.96   18.09   42.36    8.96   18.09   42.36\n",
      "  145  24453 252F 607L CA  CA   1909 4715   20.40   11.27    9.13   14.63   11.27    9.13   14.63\n",
      "  146  24454 252F 728K CA  CA   1909 5648   33.90   13.91   19.99   35.75   13.91   19.99   35.75\n",
      "  147  24462 253A 262D CA  CA   1920 1993   11.82    7.50    4.32    7.08    7.50    4.32    7.08\n",
      "  148  24463 253A 606M CA  CA   1920 4707   24.03    6.73   17.30   34.39    6.73   17.30   34.39\n",
      "  149  24464 253A 607L CA  CA   1920 4715   17.28    8.40    8.88   12.83    8.40    8.88   12.83\n",
      "  150  24465 253A 728K CA  CA   1920 5648   34.70   11.13   23.58   34.87   11.13   23.58   34.87\n",
      "  151  24472 254N 262D CA  CA   1925 1993   12.95    9.29    3.65    4.77    9.29    3.65    4.77\n",
      "  152  24473 254N 606M CA  CA   1925 4707   26.12    9.84   16.29   24.19    9.84   16.29   24.19\n",
      "  153  24474 254N 607L CA  CA   1925 4715   19.69   11.52    8.17    8.88   11.52    8.17    8.88\n",
      "  154  24475 254N 728K CA  CA   1925 5648   35.29   13.83   21.46   25.26   13.83   21.46   25.26\n",
      "  155  24482 255H 606M CA  CA   1933 4707   29.85   10.39   19.46   37.55   10.39   19.46   37.55\n",
      "  156  24483 255H 607L CA  CA   1933 4715   23.30   13.12   10.18   13.61   13.12   10.18   13.61\n",
      "  157  24489 256M 606M CA  CA   1943 4707   29.71    7.34   22.37   45.22    7.34   22.37   45.22\n",
      "  158  24490 256M 607L CA  CA   1943 4715   22.82   10.55   12.27   17.05   10.55   12.27   17.05\n",
      "  159  24491 256M 728K CA  CA   1943 5648   30.00   11.69   18.30   29.13   11.69   18.30   29.13\n",
      "  160  24494 257Q 261I CA  CA   1951 1985    8.10    5.03    3.07    5.48    5.03    3.07    5.48\n",
      "  161  24495 257Q 262D CA  CA   1951 1993   12.29    6.08    6.21    6.11    6.08    6.21    6.11\n",
      "  162  24496 257Q 606M CA  CA   1951 4707   28.57    8.14   20.43   20.41    8.14   20.43   20.41\n",
      "  163  24497 257Q 607L CA  CA   1951 4715   21.88   10.33   11.55    9.20   10.33   11.55    9.20\n",
      "  164  24498 257Q 728K CA  CA   1951 5648   31.39   11.32   20.08   17.35   11.32   20.08   17.35\n",
      "  165  24502 258H 606M CA  CA   1960 4707   31.98   11.57   20.41   15.39   11.57   20.41   15.39\n",
      "  166  24505 259F 606M CA  CA   1970 4707   34.02   10.95   23.07   14.48   10.95   23.07   14.48\n",
      "  167  24506 259F 728K CA  CA   1970 5648   27.32   13.89   13.43    7.85   13.89   13.43    7.85\n",
      "  168  24508 260G 606M CA  CA   1981 4707   31.74    9.52   22.22   10.63    9.52   22.22   10.63\n",
      "  169  24509 260G 607L CA  CA   1981 4715   24.71   11.87   12.84    5.39   11.87   12.84    5.39\n",
      "  170  24510 260G 728K CA  CA   1981 5648   26.40   11.23   15.17    6.86   11.23   15.17    6.86\n",
      "  171  24511 261I 606M CA  CA   1985 4707   32.81    5.99   26.82   11.97    5.99   26.82   11.97\n",
      "  172  24512 261I 607L CA  CA   1985 4715   25.69    8.92   16.77    6.74    8.92   16.77    6.74\n",
      "  173  24513 261I 728K CA  CA   1985 5648   25.42    8.38   17.04    7.23    8.38   17.04    7.23\n",
      "  174  24514 262D 606M CA  CA   1993 4707   34.12    3.83   30.29   11.59    3.83   30.29   11.59\n",
      "  175  24515 262D 607L CA  CA   1993 4715   27.05    5.40   21.66    7.71    5.40   21.66    7.71\n",
      "  176  24516 262D 728K CA  CA   1993 5648   25.34    5.28   20.06    7.39    5.28   20.06    7.39\n",
      "  177  24517 606M 728K CA  CA   4707 5648   53.44    5.10   48.35   18.46    5.10   48.35   18.46\n",
      "  178  24518 607L 728K CA  CA   4715 5648   46.45    3.83   42.62   15.88    3.83   42.62   15.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24519  47G  60V N   O     370  464   29.73    5.46   24.27    7.54    5.46   24.27    7.54\n",
      "    2  24520  47G  61E N   O     370  473   31.44    9.39   22.05    7.62    9.39   22.05    7.62\n",
      "    3  24521  47G  62H N   O     370  483   34.06   10.24   23.82    8.59   10.24   23.82    8.59\n",
      "    4  24522  47G 243C N   O     370 1844   23.71   10.34   13.37   19.72   10.34   13.37   19.72\n",
      "    5  24523  47G 244D N   O     370 1852   23.18    9.80   13.38   15.80    9.80   13.38   15.80\n",
      "    6  24524  47G 246E N   O     370 1869   29.41    8.86   20.55   30.87    8.86   20.55   30.87\n",
      "    7  24525  48T  60V N   O     374  464   24.08    4.55   19.53    6.62    4.55   19.53    6.62\n",
      "    8  24526  48T  61E N   O     374  473   25.74    8.51   17.23    6.51    8.51   17.23    6.51\n",
      "    9  24527  48T  62H N   O     374  483   28.32   10.49   17.82    7.11   10.49   17.82    7.11\n",
      "   10  24528  48T 246E N   O     374 1869   23.64    8.70   14.94   28.15    8.70   14.94   28.15\n",
      "   11  24529  60V  47G N   O     458  373   22.81    3.45   19.36    6.28    3.45   19.36    6.28\n",
      "   12  24530  60V  48T N   O     458  380   18.01    2.29   15.72    5.58    2.29   15.72    5.58\n",
      "   13  24534  61E  47G N   O     465  373   26.30    5.39   20.91    7.80    5.39   20.91    7.80\n",
      "   14  24535  61E  48T N   O     465  380   21.32    3.58   17.74    7.32    3.58   17.74    7.32\n",
      "   15  24537  61E 236A N   O     465 1781   13.32    9.83    3.48    7.09    9.83    3.48    7.09\n",
      "   16  24543  62H  47G N   O     474  373   28.99    6.52   22.47    8.79    6.52   22.47    8.79\n",
      "   17  24544  62H  48T N   O     474  380   23.87    6.51   17.36    7.48    6.51   17.36    7.48\n",
      "   18  24557  63V  47G N   O     484  373   30.75    9.89   20.87    9.57    9.89   20.87    9.57\n",
      "   19  24558  63V  48T N   O     484  380   25.58   10.09   15.49    7.84   10.09   15.49    7.84\n",
      "   20  24681  79I 249D N   O     620 1890   12.66   10.58    2.09    5.28   10.58    2.09    5.28\n",
      "   21  24682  79I 250V N   O     620 1897   13.47   10.95    2.52    4.67   10.95    2.52    4.67\n",
      "   22  24683  79I 251H N   O     620 1907   11.81    8.69    3.13    6.63    8.69    3.13    6.63\n",
      "   23  24746  86D 196T N   O     675 1496    9.13    6.06    3.07    4.71    6.06    3.07    4.71\n",
      "   24  24748  86D 255H N   O     675 1941   17.66   10.89    6.76    7.90   10.89    6.76    7.90\n",
      "   25  24749  86D 258H N   O     675 1968   17.54    8.43    9.11    5.65    8.43    9.11    5.65\n",
      "   26  24756  96N 195M N   O     743 1489   16.49   10.61    5.88    6.76   10.61    5.88    6.76\n",
      "   27  24757  96N 196T N   O     743 1496   13.81    8.31    5.50    7.28    8.31    5.50    7.28\n",
      "   28  24759  96N 257Q N   O     743 1958   24.06   10.58   13.48    8.62   10.58   13.48    8.62\n",
      "   29  24760  96N 258H N   O     743 1968   21.32    7.30   14.02    8.77    7.30   14.02    8.77\n",
      "   30  24761  96N 259F N   O     743 1979   22.17   10.74   11.43    5.68   10.74   11.43    5.68\n",
      "   31  24774 103P 255H N   O     792 1941   15.60    9.83    5.77    9.68    9.83    5.77    9.68\n",
      "   32  24775 103P 256M N   O     792 1949   17.34   10.09    7.25   11.42   10.09    7.25   11.42\n",
      "   33  24776 103P 257Q N   O     792 1958   16.76    8.93    7.83    6.81    8.93    7.83    6.81\n",
      "   34  24777 103P 258H N   O     792 1968   13.99    5.45    8.54    6.67    5.45    8.54    6.67\n",
      "   35  24791 116A 255H N   O     887 1941   13.85    9.88    3.98    5.50    9.88    3.98    5.50\n",
      "   36  24792 116A 256M N   O     887 1949   14.42    9.39    5.03    6.74    9.39    5.03    6.74\n",
      "   37  24806 117K 256M N   O     892 1949   11.65    8.74    2.91    4.80    8.74    2.91    4.80\n",
      "   38  24822 118S 255H N   O     901 1941   12.13    9.39    2.74    5.18    9.39    2.74    5.18\n",
      "   39  24823 118S 256M N   O     901 1949   11.33    7.05    4.28    8.42    7.05    4.28    8.42\n",
      "   40  24840 122P 256M N   O     931 1949   15.94    9.12    6.82    9.97    9.12    6.82    9.97\n",
      "   41  24854 123P 231V N   O     938 1752   13.83    9.86    3.97    6.98    9.86    3.97    6.98\n",
      "   42  24855 123P 256M N   O     938 1949   14.16   10.24    3.92    4.75   10.24    3.92    4.75\n",
      "   43  24870 129C 229G N   O     985 1738   12.89    9.63    3.26    4.84    9.63    3.26    4.84\n",
      "   44  24871 129C 231V N   O     985 1752   18.29    9.74    8.55   15.13    9.74    8.55   15.13\n",
      "   45  24887 140G 228L N   O    1075 1734   16.98    8.93    8.05   15.68    8.93    8.05   15.68\n",
      "   46  24888 140G 229G N   O    1075 1738   21.62   10.45   11.17   16.36   10.45   11.17   16.36\n",
      "   47  24906 155L 228L N   O    1177 1734   13.93    8.66    5.26   13.35    8.66    5.26   13.35\n",
      "   48  24926 156Q 228L N   O    1185 1734   12.74    9.87    2.87    6.32    9.87    2.87    6.32\n",
      "   49  24948 157C 228L N   O    1194 1734   13.04   10.65    2.39    6.53   10.65    2.39    6.53\n",
      "   50  24961 168S 192W N   O    1279 1467   15.20    8.00    7.19    4.96    8.00    7.19    4.96\n",
      "   51  24962 168S 198G N   O    1279 1506   15.78    8.70    7.08   10.29    8.70    7.08   10.29\n",
      "   52  24963 168S 199T N   O    1279 1513   14.38    7.75    6.63    8.50    7.75    6.63    8.50\n",
      "   53  24964 168S 200I N   O    1279 1521   12.23    3.06    9.17    9.95    3.06    9.17    9.95\n",
      "   54  24965 168S 201G N   O    1279 1525   13.35    6.91    6.43    4.62    6.91    6.43    4.62\n",
      "   55  24980 179K 198G N   O    1358 1506   18.10   10.77    7.33   10.87   10.77    7.33   10.87\n",
      "   56  24981 179K 199T N   O    1358 1513   16.11    9.61    6.50    8.40    9.61    6.50    8.40\n",
      "   57  24982 179K 200I N   O    1358 1521   13.12    5.15    7.96    8.40    5.15    7.96    8.40\n",
      "   58  24997 180C 200I N   O    1367 1521   11.47    7.05    4.43    7.29    7.05    4.43    7.29\n",
      "   59  25224 195M 252F N   O    1482 1918   11.58    9.69    1.89    5.18    9.69    1.89    5.18\n",
      "   60  25254 196T 252F N   O    1490 1918   12.41   10.58    1.83    4.85   10.58    1.83    4.85\n",
      "   61  25304 198G 256M N   O    1503 1949   11.58    9.83    1.75    4.71    9.83    1.75    4.71\n",
      "   62  25316 199T 168S N   O    1507 1284   15.73   10.73    5.00    5.70   10.73    5.00    5.70\n",
      "   63  25341 200I 168S N   O    1514 1284   14.37    8.25    6.13    5.52    8.25    6.13    5.52\n",
      "   64  25342 200I 180C N   O    1514 1372   14.35   10.23    4.12    5.99   10.23    4.12    5.99\n",
      "   65  25877 227K 231V N   O    1718 1752   11.79    9.38    2.40    4.98    9.38    2.40    4.98\n",
      "   66  25892 228L 182L N   O    1727 1385   13.62   10.86    2.76    6.54   10.86    2.76    6.54\n",
      "   67  25909 228L 231V N   O    1727 1752   10.77    8.46    2.31    5.70    8.46    2.31    5.70\n",
      "   68  25918 229G 129C N   O    1735  990   12.16    9.03    3.13    4.74    9.03    3.13    4.74\n",
      "   69  25919 229G 140G N   O    1735 1078   19.16   10.74    8.42   12.23   10.74    8.42   12.23\n",
      "   70  25920 229G 155L N   O    1735 1184   12.18    9.45    2.74    5.19    9.45    2.74    5.19\n",
      "   71  25936 229G 606M N   O    1735 4713   30.23   10.86   19.37   36.53   10.86   19.37   36.53\n",
      "   72  25937 229G 728K N   O    1735 5655   17.70   10.98    6.72   11.63   10.98    6.72   11.63\n",
      "   73  25940 230T 188R N   O    1739 1429   13.08   10.35    2.72    4.92   10.35    2.72    4.92\n",
      "   74  25954 230T 606M N   O    1739 4713   27.52   10.05   17.47   38.47   10.05   17.47   38.47\n",
      "   75  25955 230T 728K N   O    1739 5655   20.04   10.90    9.14   18.02   10.90    9.14   18.02\n",
      "   76  25957 231V 123P N   O    1746  944   11.09    8.38    2.71    4.94    8.38    2.71    4.94\n",
      "   77  25958 231V 191N N   O    1746 1453   11.01    8.88    2.12    5.01    8.88    2.12    5.01\n",
      "   78  25973 231V 606M N   O    1746 4713   24.93    7.57   17.35   42.40    7.57   17.35   42.40\n",
      "   79  25974 231V 728K N   O    1746 5655   22.12    9.06   13.06   28.02    9.06   13.06   28.02\n",
      "   80  25975 232T 123P N   O    1753  944   14.28    9.84    4.44    6.69    9.84    4.44    6.69\n",
      "   81  25988 232T 262D N   O    1753 1999   12.10    7.25    4.85    8.75    7.25    4.85    8.75\n",
      "   82  25989 232T 606M N   O    1753 4713   21.54    4.82   16.72   36.72    4.82   16.72   36.72\n",
      "   83  25990 232T 607L N   O    1753 4721   16.77    8.14    8.63   14.12    8.14    8.63   14.12\n",
      "   84  25991 232T 728K N   O    1753 5655   25.10    7.27   17.83   34.31    7.27   17.83   34.31\n",
      "   85  25999 233P 262D N   O    1760 1999   15.28    6.83    8.45   15.21    6.83    8.45   15.21\n",
      "   86  26000 233P 606M N   O    1760 4713   18.41    3.34   15.06   33.59    3.34   15.06   33.59\n",
      "   87  26001 233P 607L N   O    1760 4721   13.75    5.27    8.47   15.11    5.27    8.47   15.11\n",
      "   88  26002 233P 728K N   O    1760 5655   27.77    6.42   21.35   41.38    6.42   21.35   41.38\n",
      "   89  26009 234S 262D N   O    1767 1999   14.83    9.20    5.63    7.76    9.20    5.63    7.76\n",
      "   90  26010 234S 606M N   O    1767 4713   17.60    5.81   11.79   21.12    5.81   11.79   21.12\n",
      "   91  26011 234S 607L N   O    1767 4721   12.93    7.38    5.55    7.41    7.38    5.55    7.41\n",
      "   92  26012 234S 728K N   O    1767 5655   29.14    9.21   19.93   29.92    9.21   19.93   29.92\n",
      "   93  26020 235G 247V N   O    1773 1876   14.06   10.30    3.77    7.98   10.30    3.77    7.98\n",
      "   94  26026 235G 606M N   O    1773 4713   19.37    6.83   12.53   27.17    6.83   12.53   27.17\n",
      "   95  26027 235G 607L N   O    1773 4721   14.65    9.30    5.35    8.89    9.30    5.35    8.89\n",
      "   96  26028 235G 728K N   O    1773 5655   28.67   10.50   18.16   33.86   10.50   18.16   33.86\n",
      "   97  26039 236A 238F N   O    1777 1800   10.31    8.28    2.03    5.24    8.28    2.03    5.24\n",
      "   98  26040 236A 247V N   O    1777 1876   10.67    7.56    3.10    7.90    7.56    3.10    7.90\n",
      "   99  26048 236A 606M N   O    1777 4713   22.64    8.78   13.86   33.86    8.78   13.86   33.86\n",
      "  100  26063 237D 245D N   O    1782 1860   13.16   10.25    2.91    4.55   10.25    2.91    4.55\n",
      "  101  26064 237D 246E N   O    1782 1869   12.76    9.79    2.97    5.57    9.79    2.97    5.57\n",
      "  102  26082 238F 231V N   O    1790 1752   13.14   10.98    2.16    5.17   10.98    2.16    5.17\n",
      "  103  26083 238F 232T N   O    1790 1759   13.29   10.92    2.37    4.72   10.92    2.37    4.72\n",
      "  104  26210 245D  47G N   O    1853  373   20.77    9.49   11.28   12.95    9.49   11.28   12.95\n",
      "  105  26211 245D  48T N   O    1853  380   15.45   10.73    4.72    6.72   10.73    4.72    6.72\n",
      "  106  26225 246E  47G N   O    1861  373   23.45    8.04   15.41   24.05    8.04   15.41   24.05\n",
      "  107  26226 246E  48T N   O    1861  380   18.23    8.02   10.21   19.78    8.02   10.21   19.78\n",
      "  108  26233 246E 236A N   O    1861 1781   13.36    9.53    3.83    8.38    9.53    3.83    8.38\n",
      "  109  26234 246E 237D N   O    1861 1789   11.41    8.66    2.75    5.26    8.66    2.75    5.26\n",
      "  110  26243 247V  47G N   O    1870  373   26.75    9.09   17.66   28.80    9.09   17.66   28.80\n",
      "  111  26244 247V  48T N   O    1870  380   21.46    8.35   13.11   27.37    8.35   13.11   27.37\n",
      "  112  26253 247V 236A N   O    1870 1781   10.35    7.22    3.14    8.00    7.22    3.14    8.00\n",
      "  113  26264 248S  47G N   O    1877  373   30.08   10.21   19.87   23.73   10.21   19.87   23.73\n",
      "  114  26265 248S  48T N   O    1877  380   24.76    9.22   15.54   24.59    9.22   15.54   24.59\n",
      "  115  26276 248S 234S N   O    1877 1772   15.53   10.72    4.81    6.43   10.72    4.81    6.43\n",
      "  116  26287 249D  61E N   O    1883  473    9.80    7.26    2.55    6.01    7.26    2.55    6.01\n",
      "  117  26309 250V  61E N   O    1891  473   12.52    8.91    3.61    6.35    8.91    3.61    6.35\n",
      "  118  26355 252F 247V N   O    1908 1876   11.10    9.25    1.86    4.54    9.25    1.86    4.54\n",
      "  119  26379 253A 606M N   O    1919 4713   20.00    9.45   10.55   16.36    9.45   10.55   16.36\n",
      "  120  26395 254N 606M N   O    1924 4713   20.63   10.87    9.77   12.47   10.87    9.77   12.47\n",
      "  121  26425 256M 118S N   O    1942  906   14.72   10.15    4.57    7.94   10.15    4.57    7.94\n",
      "  122  26446 256M 606M N   O    1942 4713   24.73   10.48   14.25   23.51   10.48   14.25   23.51\n",
      "  123  26448 257Q 118S N   O    1950  906   14.71    9.15    5.55    5.03    9.15    5.55    5.03\n",
      "  124  26461 257Q 261I N   O    1950 1991    7.52    3.50    4.02    6.80    3.50    4.02    6.80\n",
      "  125  26463 257Q 606M N   O    1950 4713   24.00    9.90   14.10   12.29    9.90   14.10   12.29\n",
      "  126  26468 258H  96N N   O    1959  750   22.26   10.98   11.28    6.43   10.98   11.28    6.43\n",
      "  127  26469 258H 103P N   O    1959  798   17.32    8.91    8.41    6.07    8.91    8.41    6.07\n",
      "  128  26488 259F  86D N   O    1969  682   19.00    9.86    9.14    4.69    9.86    9.14    4.69\n",
      "  129  26489 259F  96N N   O    1969  750   19.99    9.14   10.85    5.55    9.14   10.85    5.55\n",
      "  130  26490 259F 103P N   O    1969  798   15.27    6.79    8.49    5.29    6.79    8.49    5.29\n",
      "  131  26512 260G 103P N   O    1980  798   17.11    7.16    9.95    4.61    7.16    9.95    4.61\n",
      "  132  26543 261I 606M N   O    1984 4713   27.42    9.30   18.12    7.49    9.30   18.12    7.49\n",
      "  133  26544 261I 728K N   O    1984 5655   23.34    9.61   13.73    5.48    9.61   13.73    5.48\n",
      "  134  26552 262D 233P N   O    1992 1766   13.81   10.81    3.00    4.87   10.81    3.00    4.87\n",
      "  135  26555 262D 253A N   O    1992 1923   11.12    6.80    4.32    6.37    6.80    4.32    6.37\n",
      "  136  26558 262D 256M N   O    1992 1949    8.41    5.39    3.01    5.02    5.39    3.01    5.02\n",
      "  137  26563 262D 606M N   O    1992 4713   28.89    6.20   22.69    8.05    6.20   22.69    8.05\n",
      "  138  26564 262D 607L N   O    1992 4721   24.06    8.01   16.05    5.15    8.01   16.05    5.15\n",
      "  139  26565 262D 728K N   O    1992 5655   22.40    6.56   15.84    5.46    6.56   15.84    5.46\n",
      "  140  26566 606M 118S N   O    4706  906   47.53    9.51   38.02   13.74    9.51   38.02   13.74\n",
      "  141  26567 606M 122P N   O    4706  937   45.18   10.15   35.03   11.92   10.15   35.03   11.92\n",
      "  142  26568 606M 123P N   O    4706  944   44.88   10.43   34.45   11.17   10.43   34.45   11.17\n",
      "  143  26569 606M 230T N   O    4706 1745   33.36    9.71   23.65   52.45    9.71   23.65   52.45\n",
      "  144  26570 606M 231V N   O    4706 1752   32.21    7.12   25.08   61.84    7.12   25.08   61.84\n",
      "  145  26571 606M 232T N   O    4706 1759   28.50    5.78   22.72   48.62    5.78   22.72   48.62\n",
      "  146  26572 606M 233P N   O    4706 1766   25.29    7.57   17.72   35.66    7.57   17.72   35.66\n",
      "  147  26573 606M 234S N   O    4706 1772   26.90   10.52   16.38   25.76   10.52   16.38   25.76\n",
      "  148  26574 606M 235G N   O    4706 1776   31.24    7.86   23.38   49.69    7.86   23.38   49.69\n",
      "  149  26575 606M 251H N   O    4706 1907   31.14   10.34   20.81   40.24   10.34   20.81   40.24\n",
      "  150  26576 606M 252F N   O    4706 1918   31.33    7.37   23.97   55.09    7.37   23.97   55.09\n",
      "  151  26577 606M 253A N   O    4706 1923   29.55    6.24   23.31   41.86    6.24   23.31   41.86\n",
      "  152  26578 606M 254N N   O    4706 1931   32.72    9.32   23.39   31.68    9.32   23.39   31.68\n",
      "  153  26579 606M 255H N   O    4706 1941   35.85    9.46   26.39   47.76    9.46   26.39   47.76\n",
      "  154  26580 606M 256M N   O    4706 1949   34.60    6.87   27.74   52.83    6.87   27.74   52.83\n",
      "  155  26581 606M 257Q N   O    4706 1958   35.22    8.68   26.54   23.92    8.68   26.54   23.92\n",
      "  156  26582 606M 259F N   O    4706 1979   38.50    9.56   28.94   17.61    9.56   28.94   17.61\n",
      "  157  26583 606M 260G N   O    4706 1983   37.55    7.12   30.43   14.33    7.12   30.43   14.33\n",
      "  158  26584 606M 261I N   O    4706 1991   36.93    3.73   33.20   14.80    3.73   33.20   14.80\n",
      "  159  26585 606M 262D N   O    4706 1999   40.32    2.24   38.08   14.35    2.24   38.08   14.35\n",
      "  160  26586 606M 728K N   O    4706 5655   55.03    5.74   49.28   17.98    5.74   49.28   17.98\n",
      "  161  26587 607L 230T N   O    4714 1745   23.19   10.06   13.13   22.61   10.06   13.13   22.61\n",
      "  162  26588 607L 231V N   O    4714 1752   22.07    7.47   14.59   28.11    7.47   14.59   28.11\n",
      "  163  26589 607L 232T N   O    4714 1759   18.29    5.05   13.24   24.21    5.05   13.24   24.21\n",
      "  164  26590 607L 233P N   O    4714 1766   15.12    5.75    9.37   16.41    5.75    9.37   16.41\n",
      "  165  26591 607L 234S N   O    4714 1772   16.93    9.53    7.40    9.20    9.53    7.40    9.20\n",
      "  166  26592 607L 235G N   O    4714 1776   21.13    7.92   13.21   22.80    7.92   13.21   22.80\n",
      "  167  26593 607L 252F N   O    4714 1918   21.56    8.95   12.60   19.95    8.95   12.60   19.95\n",
      "  168  26594 607L 253A N   O    4714 1923   19.80    7.87   11.93   15.34    7.87   11.93   15.34\n",
      "  169  26595 607L 256M N   O    4714 1949   24.70    9.47   15.24   19.61    9.47   15.24   19.61\n",
      "  170  26596 607L 257Q N   O    4714 1958   25.56   10.98   14.58   10.32   10.98   14.58   10.32\n",
      "  171  26597 607L 260G N   O    4714 1983   27.42    9.44   17.98    7.38    9.44   17.98    7.38\n",
      "  172  26598 607L 261I N   O    4714 1991   26.87    6.26   20.61    8.19    6.26   20.61    8.19\n",
      "  173  26599 607L 262D N   O    4714 1999   30.38    3.40   26.98    9.40    3.40   26.98    9.40\n",
      "  174  26600 607L 728K N   O    4714 5655   45.11    5.32   39.80   13.69    5.32   39.80   13.69\n",
      "  175  26601 728K 231V N   O    5647 1752   28.90    8.18   20.72   45.32    8.18   20.72   45.32\n",
      "  176  26602 728K 232T N   O    5647 1759   33.20    6.93   26.27   51.07    6.93   26.27   51.07\n",
      "  177  26603 728K 233P N   O    5647 1766   36.41    7.68   28.73   53.85    7.68   28.73   53.85\n",
      "  178  26604 728K 235G N   O    5647 1776   31.90    9.89   22.01   41.52    9.89   22.01   41.52\n",
      "  179  26605 728K 253A N   O    5647 1923   36.24   10.01   26.24   36.45   10.01   26.24   36.45\n",
      "  180  26606 728K 256M N   O    5647 1949   31.34   10.79   20.56   29.98   10.79   20.56   29.98\n",
      "  181  26607 728K 260G N   O    5647 1983   26.81    9.33   17.48    7.68    9.33   17.48    7.68\n",
      "  182  26608 728K 261I N   O    5647 1991   29.50    7.36   22.13    9.10    7.36   22.13    9.10\n",
      "  183  26609 728K 262D N   O    5647 1999   28.65    3.27   25.37    9.22    3.27   25.37    9.22\n",
      "  184  26610 728K 606M N   O    5647 4713   51.50    3.30   48.20   18.21    3.30   48.20   18.21\n",
      "  185  26611 728K 607L N   O    5647 4721   46.95    2.29   44.66   16.22    2.29   44.66   16.22\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20599 154R 154R CA  C    1167 1175 -156.95 -180.00   23.05    4.61 -180.00   23.05    4.61\n",
      "    2  20613 168S 168S CA  C    1280 1283 -150.95 -180.00   29.04    5.81 -180.00   29.04    5.81\n",
      "    3  21035 590K 590K CA  C    4591 4597 -157.11 -180.00   22.89    4.58 -180.00   22.89    4.58\n",
      "    4  21053 608L 608L CA  C    4723 4728 -154.34 -180.00   25.66    5.13 -180.00   25.66    5.13\n",
      "    5  21062 617A 617A CA  C    4789 4791 -157.44 -180.00   22.56    4.51 -180.00   22.56    4.51\n",
      "    6  21063 618R 618R CA  C    4794 4802 -147.27 -180.00   32.73    6.54 -180.00   32.73    6.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19720   1M   2K C   N       7    9 -156.11 -118.00   39.60    1.35  -62.90 -166.72   30.67\n",
      "    1          2K   2K N   CA      9   10  149.88  139.10                  -40.80\n",
      "    2  19723   4P   5F C   N      30   32 -153.55 -124.20   40.27    1.07  -63.20  170.70   29.34\n",
      "    2          5F   5F N   CA     32   33  170.87  143.30                  -44.30\n",
      "    3  19724   5F   6N C   N      41   43 -156.72 -119.90   55.23    1.64  -63.20  168.98   26.27\n",
      "    3          6N   6N N   CA     43   44  178.16  137.00                  -41.10\n",
      "    4  19726   7I   8T C   N      57   59 -155.18 -124.80   46.28    1.46  -63.20  167.09   27.23\n",
      "    4          8T   8T N   CA     59   60  178.41  143.50                  -42.10\n",
      "    5  19727   8T   9D C   N      64   66 -162.14  -96.50   96.31    4.03  -63.30  167.59   26.91\n",
      "    5          9D   9D N   CA     66   67 -175.33  114.20                  -40.00\n",
      "    6  19728   9D  10E C   N      72   74 -167.26 -117.80   62.51    1.89  -63.60  177.98   29.33\n",
      "    6         10E  10E N   CA     74   75  175.02  136.80                  -40.30\n",
      "    7  19729  10E  11A C   N      81   83 -171.26 -134.00   46.56    1.19  -62.50 -179.39   35.31\n",
      "    7         11A  11A N   CA     83   84  174.91  147.00                  -40.90\n",
      "    8  19731  12C  13A C   N      92   94 -171.57 -134.00   49.74    1.38  -62.50  177.09   34.73\n",
      "    8         13A  13A N   CA     94   95  179.59  147.00                  -40.90\n",
      "    9  19732  13A  14H C   N      97   99 -165.59 -125.60   55.65    1.37  -63.20  173.62   28.17\n",
      "    9         14H  14H N   CA     99  100  177.49  138.80                  -42.30\n",
      "   10  19734  15C  16W C   N     113  115 -169.19 -124.90   50.62    1.67  -63.00 -177.93   29.85\n",
      "   10         16W  16W N   CA    115  116  167.90  143.40                  -44.20\n",
      "   11  19735  16W  17D C   N     127  129  176.50  -96.50  103.34    4.29  -63.30 -167.75   31.23\n",
      "   11         17D  17D N   CA    129  130  169.97  114.20                  -40.00\n",
      "   12  19736  17D  18E C   N     135  137 -171.81 -117.80   65.37    1.94  -63.60 -178.22   30.05\n",
      "   12         18E  18E N   CA    137  138  173.64  136.80                  -40.30\n",
      "   13  19737  18E  19S C   N     144  146  175.28 -136.60   53.06    1.65  -64.10 -166.40   22.72\n",
      "   13         19S  19S N   CA    146  147  173.57  151.20                  -35.00\n",
      "   14  19739  20V  21M C   N     157  159  173.78 -125.60   67.24    2.05  -63.40 -166.20   38.02\n",
      "   14         21M  21M N   CA    159  160  169.58  140.50                  -40.50\n",
      "   15  19740  21M  22L C   N     165  167  177.22 -108.50   83.04    3.67  -63.50 -169.00   33.68\n",
      "   15         22L  22L N   CA    167  168  169.63  132.50                  -41.20\n",
      "   16  19743  24P  25N C   N     187  189 -178.15 -119.90   70.48    2.07  -63.20 -177.12   29.03\n",
      "   16         25N  25N N   CA    189  190  176.67  137.00                  -41.10\n",
      "   17  19744  25N  26H C   N     195  197  178.34 -125.60   64.78    1.66  -63.20 -171.64   30.94\n",
      "   17         26H  26H N   CA    197  198  171.25  138.80                  -42.30\n",
      "   18  19745  26H  27R C   N     205  207  173.79 -125.20   69.57    2.09  -63.00 -169.83   33.58\n",
      "   18         27R  27R N   CA    207  208  174.04  140.60                  -41.10\n",
      "   19  19746  27R  28S C   N     216  218 -178.77 -136.60   47.81    1.46  -64.10 -170.16   22.03\n",
      "   19         28S  28S N   CA    218  219  173.71  151.20                  -35.00\n",
      "   20  19747  28S  29A C   N     222  224 -179.44 -134.00   54.15    1.31  -62.50 -175.55   36.32\n",
      "   20         29A  29A N   CA    224  225  176.45  147.00                  -40.90\n",
      "   21  19749  30V  31H C   N     234  236 -174.48 -125.60   59.93    1.49  -63.20 -177.83   29.76\n",
      "   21         31H  31H N   CA    236  237  173.46  138.80                  -42.30\n",
      "   22  19750  31H  32K C   N     244  246 -171.13 -118.00   66.16    2.23  -62.90  177.49   29.39\n",
      "   22         32K  32K N   CA    246  247  178.53  139.10                  -40.80\n",
      "   23  19751  32K  33E C   N     253  255 -179.79 -117.80   78.34    2.36  -63.60  178.13   29.86\n",
      "   23         33E  33E N   CA    255  256 -175.31  136.80                  -40.30\n",
      "   24  19752  33E  34E C   N     262  264 -142.57 -117.80   59.89    2.43  -63.60  150.72   24.43\n",
      "   24         34E  34E N   CA    264  265 -168.67  136.80                  -40.30\n",
      "   25  19754  35C  36A C   N     277  279  116.03 -134.00  114.12    2.80  -62.50 -132.13   36.21\n",
      "   25         36A  36A N   CA    279  280  177.49  147.00                  -40.90\n",
      "   26  19755  36A  37Y C   N     282  284  135.63 -124.30  106.82    8.80  -63.50 -145.80   30.06\n",
      "   26         37Y  37Y N   CA    284  285   98.03  135.40                  -43.40\n",
      "   27  19758  39C  40R C   N     306  308 -151.08 -125.20   36.84    1.13  -63.00  175.75   29.66\n",
      "   27         40R  40R N   CA    308  309  166.82  140.60                  -41.10\n",
      "   28  19759  40R  41T C   N     317  319 -145.61 -124.80   46.21    1.47  -63.20  156.58   25.32\n",
      "   28         41T  41T N   CA    319  320 -175.24  143.50                  -42.10\n",
      "   29  19761  42C  43R C   N     330  332  166.47 -125.20   77.49    2.33  -63.00 -167.31   34.27\n",
      "   29         43R  43R N   CA    332  333  177.15  140.60                  -41.10\n",
      "   30  19762  43R  44H C   N     341  343  172.86 -125.60   65.17    1.84  -63.20 -159.59   32.82\n",
      "   30         44H  44H N   CA    343  344  160.21  138.80                  -42.30\n",
      "   31  19763  44H  45E C   N     351  353 -179.03 -117.80   74.33    2.20  -63.60 -177.96   30.39\n",
      "   31         45E  45E N   CA    353  354  178.93  136.80                  -40.30\n",
      "   32  19764  45E  46M C   N     360  362 -112.52 -125.60   62.68    3.23  -63.40  127.53   22.93\n",
      "   32         46M  46M N   CA    362  363 -158.19  140.50                  -40.50\n",
      "   33  19767  48T  49L C   N     379  381 -139.68 -108.50   35.81    1.93  -63.50  173.69   21.30\n",
      "   33         49L  49L N   CA    381  382  114.89  132.50                  -41.20\n",
      "   34  19769  50V  51C C   N     394  396  -62.98  -63.00   19.62    2.39 -117.90  167.44    7.25\n",
      "   34         51C  51C N   CA    396  397  -60.72  -41.10                  141.10\n",
      "   35  19770  51C  52M C   N     400  402  -90.89  -73.00   32.56    2.51  -63.40  158.70   21.70\n",
      "   35         52M  52M N   CA    402  403  115.80  143.00                  -40.50\n",
      "   36  19773  54C  55H C   N     420  422 -140.40 -125.60   26.57    0.71  -63.20  174.81   26.88\n",
      "   36         55H  55H N   CA    422  423  160.87  138.80                  -42.30\n",
      "   37  19774  55H  56M C   N     430  432 -155.89 -125.60   46.42    1.47  -63.40  171.01   32.61\n",
      "   37         56M  56M N   CA    432  433  175.67  140.50                  -40.50\n",
      "   38  19775  56M  57G C   N     438  440 -161.07 -167.20    7.29    0.24   82.20 -153.75   15.06\n",
      "   38         57G  57G N   CA    440  441  178.54  174.60                    8.50\n",
      "   39  19776  57G  58L C   N     442  444 -149.91 -108.50   64.83    2.97  -63.50  161.48   27.71\n",
      "   39         58L  58L N   CA    444  445 -177.62  132.50                  -41.20\n",
      "   40  19779  60V  61E C   N     463  465    8.57  -69.30   86.57    7.67  -69.30   86.57    7.67\n",
      "   40         61E  61E N   CA    465  466 -179.68  142.50                  142.50\n",
      "   41  19780  61E  62H C   N     472  474  127.25 -125.60  112.17    3.24  -67.60  168.22   11.67\n",
      "   41         62H  62H N   CA    474  475  171.99  138.80                  140.00\n",
      "   42  19787  68L  69N C   N     532  534 -157.94  -63.20  112.36   18.47  -63.20  112.36   18.47\n",
      "   42         69N  69N N   CA    534  535 -101.50  -41.10                  -41.10\n",
      "   43  19790  71P  72T C   N     553  555 -151.47 -124.80   33.41    1.17  -63.20  177.74   28.46\n",
      "   43         72T  72T N   CA    555  556  163.63  143.50                  -42.10\n",
      "   44  19791  72T  73H C   N     560  562   89.82   56.30  152.37   16.50  -63.20 -160.34   23.29\n",
      "   44         73H  73H N   CA    562  563 -170.56   40.80                  -42.30\n",
      "   45  19792  73H  74V C   N     570  572  -59.28  -62.40   14.99    1.74 -125.40  172.79    9.77\n",
      "   45         74V  74V N   CA    572  573  -57.06  -42.40                  143.30\n",
      "   46  19793  74V  75M C   N     577  579 -163.03 -125.60   62.54    3.41  -63.40  164.50   20.50\n",
      "   46         75M  75M N   CA    579  580   90.40  140.50                  -40.50\n",
      "   47  19794  75M  76Y C   N     585  587 -161.97 -124.30   65.95    2.64  -63.50  160.75   29.97\n",
      "   47         76Y  76Y N   CA    587  588 -170.47  135.40                  -43.40\n",
      "   48  19795  76Y  77V C   N     597  599   54.98  -73.50  128.48    9.13 -125.40  179.65    7.12\n",
      "   48         77V  77V N   CA    599  600  140.10  139.20                  143.30\n",
      "   49  19796  77V  78W C   N     604  606 -171.02 -124.90   52.88    1.74  -63.00 -177.95   29.96\n",
      "   49         78W  78W N   CA    606  607  169.27  143.40                  -44.20\n",
      "   50  19800  81E  82L C   N     644  646 -176.56 -108.50   69.60    3.16  -70.70  106.01    9.46\n",
      "   50         82L  82L N   CA    646  647  147.07  132.50                  141.60\n",
      "   51  19802  83P  84A C   N     659  661  -69.73  -68.20   24.46    1.92  -62.50  149.56   24.87\n",
      "   51         84A  84A N   CA    661  662  169.72  145.30                  -40.90\n",
      "   52  19803  84A  85K C   N     664  666  -80.21  -70.20   53.02    4.07  -62.90  130.29   16.05\n",
      "   52         85K  85K N   CA    666  667   88.33  140.40                  -40.80\n",
      "   53  19805  86D  87E C   N     681  683 -151.75 -117.80   45.64    1.42  -63.60  176.06   28.34\n",
      "   53         87E  87E N   CA    683  684  167.30  136.80                  -40.30\n",
      "   54  19806  87E  88D C   N     690  692 -148.59  -96.50   64.19    2.67  -63.30 -171.34   28.78\n",
      "   54         88D  88D N   CA    692  693  151.72  114.20                  -40.00\n",
      "   55  19807  88D  89A C   N     698  700 -134.17 -134.00    4.04    0.23  -62.50 -177.28   33.78\n",
      "   55         89A  89A N   CA    700  701  151.03  147.00                  -40.90\n",
      "   56  19808  89A  90G C   N     703  705  136.66 -167.20   57.32    1.39   82.20  163.82   10.53\n",
      "   56         90G  90G N   CA    705  706  163.00  174.60                    8.50\n",
      "   57  19809  90G  91G C   N     707  709   60.25   78.70   27.97    0.56   82.20  155.14    8.62\n",
      "   57         91G  91G N   CA    709  710 -145.08 -166.10                    8.50\n",
      "   58  19811  92S  93K C   N     717  719  179.52 -118.00   62.77    2.27  -62.90 -149.96   34.23\n",
      "   58         93K  93K N   CA    719  720  145.16  139.10                  -40.80\n",
      "   59  19812  93K  94D C   N     726  728 -144.87  -70.90   79.28    4.80  -63.30  163.05   25.35\n",
      "   59         94D  94D N   CA    728  729  178.82  150.30                  -40.00\n",
      "   60  19814  95V  96N C   N     741  743  -43.56  -63.20   21.29    3.48   55.90  123.01   15.01\n",
      "   60         96N  96N N   CA    743  744  -32.88  -41.10                   39.50\n",
      "   61  19815  96N  97K C   N     749  751 -156.92 -118.00   53.41    1.86  -62.90  171.57   27.85\n",
      "   61         97K  97K N   CA    751  752  175.68  139.10                  -40.80\n",
      "   62  19817  98L  99G C   N     766  768  -99.85  -80.20   72.41    1.95   82.20 -142.70    8.39\n",
      "   62         99G  99G N   CA    768  769 -116.21  174.10                    8.50\n",
      "   63  19819 100V 101L C   N     777  779 -140.01 -108.50  104.33    5.26  -63.50  115.74   20.61\n",
      "   63        101L 101L N   CA    779  780 -128.04  132.50                  -41.20\n",
      "   64  19820 101L 102A C   N     785  787  -56.34  -68.20   31.66    2.93  -62.50  144.58   23.40\n",
      "   64        102A 102A N   CA    787  788  174.66  145.30                  -40.90\n",
      "   65  19822 103P 104K C   N     797  799  127.81 -118.00  118.38    4.03  -62.90 -134.55   38.74\n",
      "   65        104K 104K N   CA    799  800  170.31  139.10                  -40.80\n",
      "   66  19823 104K 105E C   N     806  808 -154.28 -117.80   50.79    1.62  -63.60  173.20   28.07\n",
      "   66        105E 105E N   CA    808  809  172.14  136.80                  -40.30\n",
      "   67  19824 105E 106Y C   N     815  817 -138.93 -124.30   24.07    0.98  -63.50  178.78   32.08\n",
      "   67        106Y 106Y N   CA    817  818  154.51  135.40                  -43.40\n",
      "   68  19825 106Y 107E C   N     827  829 -118.72 -117.80   40.50    2.05  -63.60  147.32   17.30\n",
      "   68        107E 107E N   CA    829  830   96.31  136.80                  -40.30\n",
      "   69  19827 108N 109A C   N     844  846 -129.26 -134.00   16.52    1.01  -62.50  169.94   31.43\n",
      "   69        109A 109A N   CA    846  847  162.83  147.00                  -40.90\n",
      "   70  19828 109A 110L C   N     849  851  -51.76  -70.70   34.14    2.11  -63.50  154.84   22.19\n",
      "   70        110L 110L N   CA    851  852  113.20  141.60                  -41.20\n",
      "   71  19831 112C 113A C   N     869  871 -149.48 -134.00   42.86    1.97  -62.50  158.20   30.54\n",
      "   71        113A 113A N   CA    871  872 -173.04  147.00                  -40.90\n",
      "   72  19832 113A 114A C   N     874  876 -171.06 -134.00   62.31    2.22  -62.50  163.32   32.36\n",
      "   72        114A 114A N   CA    876  877 -162.92  147.00                  -40.90\n",
      "   73  19834 115C 116A C   N     885  887  144.03 -134.00  100.01    2.49  -68.20  159.11   11.38\n",
      "   73        116A 116A N   CA    887  888 -155.71  147.00                  145.30\n",
      "   74  19836 117K 118S C   N     899  901  131.74 -136.60  112.21    3.43  -72.40  168.31    9.88\n",
      "   74        118S 118S N   CA    901  902 -144.08  151.20                  152.40\n",
      "   75  19837 118S 119F C   N     905  907  164.03 -124.20  127.51    3.89  -63.20  148.73   26.67\n",
      "   75        119F 119F N   CA    907  908 -111.31  143.30                  -44.30\n",
      "   76  19838 119F 120V C   N     916  918  -83.93  -62.40   21.55    3.46 -125.40  178.20    9.72\n",
      "   76        120V 120V N   CA    918  919  -43.39  -42.40                  143.30\n",
      "   77  19839 120V 121S C   N     923  925 -157.02 -136.60   54.59    3.07  -64.10  164.36   11.42\n",
      "   77        121S 121S N   CA    925  926  100.57  151.20                  -35.00\n",
      "   78  19840 121S 122P C   N     929  931  -44.45  -58.70   68.94    4.64  -64.50  116.59    9.79\n",
      "   78        122P 122P N   CA    931  932  -97.95  -30.50                  147.20\n",
      "   79  19842 123P 124E C   N     943  945  175.75 -117.80   66.57    2.34  -63.60 -144.22   35.33\n",
      "   79        124E 124E N   CA    945  946  140.80  136.80                  -40.30\n",
      "   80  19843 124E 125L C   N     952  954 -103.33  -70.70   77.71    4.89  -63.50  113.86   18.28\n",
      "   80        125L 125L N   CA    954  955 -147.87  141.60                  -41.20\n",
      "   81  19845 126T 127I C   N     967  969  -73.80  -63.40   13.19    2.49 -120.60 -175.96   12.29\n",
      "   81        127I 127I N   CA    969  970  -51.71  -43.60                  130.30\n",
      "   82  19846 127I 128D C   N     975  977  179.06  -96.50  118.06    4.94  -63.30  170.41   28.21\n",
      "   82        128D 128D N   CA    977  978 -163.29  114.20                  -40.00\n",
      "   83  19848 129C 130Y C   N     989  991  147.82 -124.30   88.03    6.46  -63.50 -129.58   43.16\n",
      "   83        130Y 130Y N   CA    991  992  140.57  135.40                  -43.40\n",
      "   84  19849 130Y 131Q C   N    1001 1003 -150.12 -121.10   29.86    0.99  -63.80 -166.68   32.92\n",
      "   84        131Q 131Q N   CA   1003 1004  146.72  139.70                  -40.30\n",
      "   85  19850 131Q 132W C   N    1010 1012 -163.60 -124.90   74.03    2.61  -63.00  148.54   25.16\n",
      "   85        132W 132W N   CA   1012 1013 -153.49  143.40                  -44.20\n",
      "   86  19853 134I 135N C   N    1040 1042  172.51 -119.90  104.31    3.12  -63.20  161.08   26.31\n",
      "   86        135N 135N N   CA   1042 1043 -143.56  137.00                  -41.10\n",
      "   87  19854 135N 136A C   N    1048 1050  -63.33  -68.20    9.59    0.62  -62.50  177.94   29.12\n",
      "   87        136A 136A N   CA   1050 1051  137.04  145.30                  -40.90\n",
      "   88  19855 136A 137T C   N    1053 1055 -134.09 -124.80   23.46    0.77  -63.20  168.50   26.24\n",
      "   88        137T 137T N   CA   1055 1056  165.04  143.50                  -42.10\n",
      "   89  19856 137T 138S C   N    1060 1062 -169.32 -136.60   58.11    3.30  -64.10  173.68   12.55\n",
      "   89        138S 138S N   CA   1062 1063  103.18  151.20                  -35.00\n",
      "   90  19857 138S 139T C   N    1066 1068 -147.90 -124.80   25.34    1.52  -63.20 -165.42   20.77\n",
      "   90        139T 139T N   CA   1068 1069  133.08  143.50                  -42.10\n",
      "   91  19859 140G 141A C   N    1077 1079   95.29 -134.00  133.19    3.53  -62.50 -144.68   33.56\n",
      "   91        141A 141A N   CA   1079 1080  172.59  147.00                  -40.90\n",
      "   92  19860 141A 142Q C   N    1082 1084  167.64 -121.10   75.44    2.41  -63.80 -158.45   35.88\n",
      "   92        142Q 142Q N   CA   1084 1085  164.47  139.70                  -40.30\n",
      "   93  19861 142Q 143A C   N    1091 1093 -163.95 -134.00   34.33    0.79  -62.50 -174.48   35.78\n",
      "   93        143A 143A N   CA   1093 1094  163.78  147.00                  -40.90\n",
      "   94  19862 143A 144A C   N    1096 1098 -167.10 -134.00   36.55    0.83  -62.50 -171.67   36.40\n",
      "   94        144A 144A N   CA   1098 1099  162.50  147.00                  -40.90\n",
      "   95  19864 145I 146E C   N    1109 1111  -73.81  -69.30   47.41    3.46  -63.60  130.40   17.95\n",
      "   95        146E 146E N   CA   1111 1112 -170.30  142.50                  -40.30\n",
      "   96  19866 147P 148E C   N    1125 1127   80.52  -69.30  153.89   10.39  -63.60 -153.65   34.92\n",
      "   96        148E 148E N   CA   1127 1128  107.37  142.50                  -40.30\n",
      "   97  19867 148E 149G C   N    1134 1136 -174.50 -167.20   13.76    0.39   82.20 -154.41   14.58\n",
      "   97        149G 149G N   CA   1136 1137 -173.74  174.60                    8.50\n",
      "   98  19869 150V 151A C   N    1145 1147 -167.52 -134.00   47.36    1.43  -62.50  173.93   34.01\n",
      "   98        151A 151A N   CA   1147 1148 -179.54  147.00                  -40.90\n",
      "   99  19870 151A 152S C   N    1150 1152 -158.96 -136.60   32.08    1.04  -64.10  178.15   19.75\n",
      "   99        152S 152S N   CA   1152 1153  174.20  151.20                  -35.00\n",
      "  100  19871 152S 153M C   N    1156 1158 -161.47 -125.60   38.40    1.21  -63.40 -167.79   36.27\n",
      "  100        153M 153M N   CA   1158 1159  154.19  140.50                  -40.50\n",
      "  101  19872 153M 154R C   N    1164 1166 -159.46  -63.00  106.11   19.27  -63.00  106.11   19.27\n",
      "  101        154R 154R N   CA   1166 1167  -85.31  -41.10                  -41.10\n",
      "  102  19873 154R 155L C   N    1175 1177  -99.40 -108.50  129.20    6.88  -70.70  140.93   11.79\n",
      "  102        155L 155L N   CA   1177 1178    3.62  132.50                  141.60\n",
      "  103  19874 155L 156Q C   N    1183 1185  -91.69  -73.00  146.29    9.19 -121.10  149.03    7.51\n",
      "  103        156Q 156Q N   CA   1185 1186  -74.21  140.70                  139.70\n",
      "  104  19877 158P 159H C   N    1205 1207 -176.13 -125.60   52.14    1.56  -63.20 -159.18   32.29\n",
      "  104        159H 159H N   CA   1207 1208  151.64  138.80                  -42.30\n",
      "  105  19878 159H 160L C   N    1215 1217 -156.55 -108.50   53.44    2.36  -63.50 -172.38   31.78\n",
      "  105        160L 160L N   CA   1217 1218  155.88  132.50                  -41.20\n",
      "  106  19881 162C 163L C   N    1236 1238 -110.20 -108.50   22.24    1.22  -63.50  158.56   19.88\n",
      "  106        163L 163L N   CA   1238 1239  110.33  132.50                  -41.20\n",
      "  107  19882 163L 164E C   N    1244 1246   65.16  -69.30  138.46    9.29  -63.60 -162.52   33.10\n",
      "  107        164E 164E N   CA   1246 1247  109.44  142.50                  -40.30\n",
      "  108  19883 164E 165Q C   N    1253 1255 -144.22 -121.10   36.48    1.24  -63.80  171.77   29.42\n",
      "  108        165Q 165Q N   CA   1255 1256  167.92  139.70                  -40.30\n",
      "  109  19884 165Q 166L C   N    1262 1264 -146.63 -108.50   38.17    1.80  -63.50 -165.82   23.83\n",
      "  109        166L 166L N   CA   1264 1265  134.28  132.50                  -41.20\n",
      "  110  19886 167P 168S C   N    1277 1279  -52.41 -136.60   93.66    2.89  -64.10  145.63   11.29\n",
      "  110        168S 168S N   CA   1279 1280  110.15  151.20                  -35.00\n",
      "  111  19888 169P 170F C   N    1290 1292 -128.42 -124.20   38.35    1.70  -63.20  149.28   24.93\n",
      "  111        170F 170F N   CA   1292 1293 -178.58  143.30                  -44.30\n",
      "  112  19889 170F 171Q C   N    1301 1303  -96.58  -73.00   29.99    1.77  -63.80  163.78   25.83\n",
      "  112        171Q 171Q N   CA   1303 1304  159.23  140.70                  -40.30\n",
      "  113  19891 172T 173A C   N    1317 1319  -57.05  -68.20   11.94    0.86  -62.50  178.16   28.94\n",
      "  113        173A 173A N   CA   1319 1320  141.02  145.30                  -40.90\n",
      "  114  19893 174P 175T C   N    1329 1331 -170.12 -124.80   62.59    2.05  -63.20  169.28   28.24\n",
      "  114        175T 175T N   CA   1331 1332 -173.34  143.50                  -42.10\n",
      "  115  19894 175T 176S C   N    1336 1338 -149.38 -136.60   13.70    0.44  -64.10 -170.82   19.82\n",
      "  115        176S 176S N   CA   1338 1339  156.13  151.20                  -35.00\n",
      "  116  19895 176S 177S C   N    1342 1344 -160.81 -136.60   27.30    0.84  -64.10 -172.05   20.59\n",
      "  116        177S 177S N   CA   1344 1345  163.83  151.20                  -35.00\n",
      "  117  19896 177S 178D C   N    1348 1350 -167.14  -96.50   70.93    2.90  -63.30 -168.80   20.39\n",
      "  117        178D 178D N   CA   1350 1351  120.54  114.20                  -40.00\n",
      "  118  19900 181A 182L C   N    1376 1378   48.57  -70.70  162.45   16.25  -70.70  162.45   16.25\n",
      "  118        182L 182L N   CA   1378 1379 -108.10  141.60                  141.60\n",
      "  119  19901 182L 183D C   N    1384 1386 -160.57  -63.30  111.55   14.30  -63.30  111.55   14.30\n",
      "  119        183D 183D N   CA   1386 1387   14.61  -40.00                  -40.00\n",
      "  120  19904 185C 186E C   N    1402 1404  -68.74  -69.30   57.41    4.36  -63.60  125.50   16.46\n",
      "  120        186E 186E N   CA   1404 1405   85.10  142.50                  -40.30\n",
      "  121  19906 187C 188R C   N    1417 1419 -156.59 -125.20   33.51    1.07  -63.00 -168.93   32.12\n",
      "  121        188R 188R N   CA   1419 1420  152.32  140.60                  -41.10\n",
      "  122  19924 205E 206E C   N    1557 1559   64.03   54.60   25.20    1.33  -63.60  140.75   24.13\n",
      "  122        206E 206E N   CA   1559 1560   19.03   42.40                  -40.30\n",
      "  123  19925 206E 207A C   N    1566 1568  145.43 -134.00   80.85    2.43  -68.20  146.61   12.14\n",
      "  123        207A 207A N   CA   1568 1569  153.72  147.00                  145.30\n",
      "  124  19982 263I 264Q C   N    2006 2008  158.63 -121.10   90.33    2.79  -63.80 -164.73   35.13\n",
      "  124        264Q 264Q N   CA   2008 2009 -178.88  139.70                  -40.30\n",
      "  125  19983 264Q 265T C   N    2015 2017  176.88 -124.80   61.55    2.70  -63.20 -164.27   32.49\n",
      "  125        265T 265T N   CA   2017 2018  163.20  143.50                  -42.10\n",
      "  126  19984 265T 266A C   N    2022 2024  150.39 -134.00   78.88    1.91  -62.50 -150.15   41.88\n",
      "  126        266A 266A N   CA   2024 2025  169.45  147.00                  -40.90\n",
      "  127  19985 266A 267K C   N    2027 2029  169.18 -118.00   79.12    2.63  -62.90 -163.51   32.94\n",
      "  127        267K 267K N   CA   2029 2030  170.05  139.10                  -40.80\n",
      "  128  19986 267K 268K C   N    2036 2038  179.92 -118.00   70.06    2.32  -62.90 -171.52   31.33\n",
      "  128        268K 268K N   CA   2038 2039  171.57  139.10                  -40.80\n",
      "  129  19987 268K 269T C   N    2045 2047 -169.24 -124.80   58.31    1.97  -63.20  172.98   28.71\n",
      "  129        269T 269T N   CA   2047 2048 -178.76  143.50                  -42.10\n",
      "  130  19988 269T 270A C   N    2052 2054  179.04 -134.00   53.47    1.23  -62.50 -171.58   37.05\n",
      "  130        270A 270A N   CA   2054 2055  172.58  147.00                  -40.90\n",
      "  131  19989 270A 271K C   N    2057 2059  178.11 -118.00   70.28    2.33  -62.90 -167.89   31.91\n",
      "  131        271K 271K N   CA   2059 2060  168.38  139.10                  -40.80\n",
      "  132  19990 271K 272T C   N    2066 2068 -171.13 -124.80   57.70    2.03  -63.20  176.78   29.33\n",
      "  132        272T 272T N   CA   2068 2069  177.89  143.50                  -42.10\n",
      "  133  19991 272T 273L C   N    2073 2075  179.14 -108.50   81.15    3.58  -63.50 -169.89   33.46\n",
      "  133        273L 273L N   CA   2075 2076  169.24  132.50                  -41.20\n",
      "  134  19992 273L 274G C   N    2081 2083  170.90 -167.20   22.12    0.50   82.20 -174.46   13.00\n",
      "  134        274G 274G N   CA   2083 2084  171.47  174.60                    8.50\n",
      "  135  19993 274G 275E C   N    2085 2087  174.19 -117.80   73.91    2.18  -63.60 -163.43   32.71\n",
      "  135        275E 275E N   CA   2087 2088  165.74  136.80                  -40.30\n",
      "  136  19994 275E 276M C   N    2094 2096  171.00 -125.60   69.85    2.14  -63.40 -164.60   38.43\n",
      "  136        276M 276M N   CA   2096 2097  169.81  140.50                  -40.50\n",
      "  137  19995 276M 277Q C   N    2102 2104 -180.00 -121.10   64.25    2.01  -63.80 -166.82   34.14\n",
      "  137        277Q 277Q N   CA   2104 2105  165.38  139.70                  -40.30\n",
      "  138  19996 277Q 278Y C   N    2111 2113 -169.97 -124.30   60.95    2.83  -63.50  176.55   32.86\n",
      "  138        278Y 278Y N   CA   2113 2114  175.76  135.40                  -43.40\n",
      "  139  19997 278Y 279D C   N    2123 2125 -176.78  -96.50   91.95    3.81  -63.30 -163.04   31.48\n",
      "  139        279D 279D N   CA   2125 2126  159.02  114.20                  -40.00\n",
      "  140  19998 279D 280Y C   N    2131 2133 -162.10 -124.30   55.42    2.39  -63.50  171.79   31.82\n",
      "  140        280Y 280Y N   CA   2133 2134  175.92  135.40                  -43.40\n",
      "  141  19999 280Y 281S C   N    2143 2145 -167.12 -136.60   38.29    1.18  -64.10 -177.48   20.66\n",
      "  141        281S 281S N   CA   2145 2146  174.33  151.20                  -35.00\n",
      "  142  20001 282C 283Q C   N    2155 2157 -169.52 -121.10   55.36    1.71  -63.80 -173.90   32.66\n",
      "  142        283Q 283Q N   CA   2157 2158  166.54  139.70                  -40.30\n",
      "  143  20002 283Q 284F C   N    2164 2166 -177.99 -124.20   60.28    1.64  -63.20 -174.92   32.56\n",
      "  143        284F 284F N   CA   2166 2167  170.51  143.30                  -44.30\n",
      "  144  20003 284F 285D C   N    2175 2177 -163.30  -96.50   87.98    3.67  -63.30  179.07   28.45\n",
      "  144        285D 285D N   CA   2177 2178  171.46  114.20                  -40.00\n",
      "  145  20004 285D 286F C   N    2183 2185 -168.38 -124.20   56.59    1.47  -63.20  172.75   30.31\n",
      "  145        286F 286F N   CA   2185 2186  178.67  143.30                  -44.30\n",
      "  146  20005 286F 287K C   N    2194 2196 -161.83 -118.00   49.03    1.62  -62.90 -173.48   30.10\n",
      "  146        287K 287K N   CA   2196 2197  161.08  139.10                  -40.80\n",
      "  147  20006 287K 288R C   N    2203 2205 -159.16 -125.20   55.73    1.81  -63.00  165.02   28.60\n",
      "  147        288R 288R N   CA   2205 2206 -175.21  140.60                  -41.10\n",
      "  148  20008 289I 290T C   N    2222 2224 -158.41 -124.80   53.67    1.67  -63.20  163.21   26.87\n",
      "  148        290T 290T N   CA   2224 2225 -174.66  143.50                  -42.10\n",
      "  149  20009 290T 291E C   N    2229 2231 -144.42 -117.80   41.46    1.41  -63.60  171.38   27.34\n",
      "  149        291E 291E N   CA   2231 2232  168.58  136.80                  -40.30\n",
      "  150  20010 291E 292N C   N    2238 2240 -151.01 -119.90   58.28    1.87  -63.20  159.06   24.72\n",
      "  150        292N 292N N   CA   2240 2241 -173.72  137.00                  -41.10\n",
      "  151  20011 292N 293G C   N    2246 2248 -159.46 -167.20    8.43    0.12   82.20 -158.78   14.83\n",
      "  151        293G 293G N   CA   2248 2249  171.25  174.60                    8.50\n",
      "  152  20012 293G 294D C   N    2250 2252 -148.54  -96.50   70.91    2.96  -63.30  179.20   27.60\n",
      "  152        294D 294D N   CA   2252 2253  162.37  114.20                  -40.00\n",
      "  153  20013 294D 295S C   N    2258 2260 -154.71 -136.60   22.61    0.69  -64.10 -175.89   19.86\n",
      "  153        295S 295S N   CA   2260 2261  164.73  151.20                  -35.00\n",
      "  154  20014 295S 296L C   N    2264 2266 -155.59 -108.50   49.25    2.21  -63.50 -165.00   32.76\n",
      "  154        296L 296L N   CA   2266 2267  146.92  132.50                  -41.20\n",
      "  155  20017 298P 299A C   N    2286 2288 -159.08 -134.00   29.07    0.68  -62.50 -175.32   35.41\n",
      "  155        299A 299A N   CA   2288 2289  161.69  147.00                  -40.90\n",
      "  156  20018 299A 300F C   N    2291 2293 -159.86 -124.20   41.38    1.09  -63.20  179.62   30.95\n",
      "  156        300F 300F N   CA   2293 2294  164.31  143.30                  -44.30\n",
      "  157  20019 300F 301G C   N    2302 2304 -170.49 -167.20   35.48    1.42   82.20  169.17   12.69\n",
      "  157        301G 301G N   CA   2304 2305  139.27  174.60                    8.50\n",
      "  158  20021 302P 303G C   N    2313 2315 -160.70 -167.20   12.16    0.34   82.20 -165.08   14.44\n",
      "  158        303G 303G N   CA   2315 2316  164.32  174.60                    8.50\n",
      "  159  20022 303G 304R C   N    2317 2319 -154.07 -125.20   40.65    1.24  -63.00  175.21   29.76\n",
      "  159        304R 304R N   CA   2319 2320  169.21  140.60                  -41.10\n",
      "  160  20023 304R 305T C   N    2328 2330 -149.07 -124.80   43.14    1.34  -63.20  163.16   26.38\n",
      "  160        305T 305T N   CA   2330 2331  179.17  143.50                  -42.10\n",
      "  161  20024 305T 306G C   N    2335 2337  178.24 -167.20   19.61    0.72   82.20 -179.39   12.99\n",
      "  161        306G 306G N   CA   2337 2338  161.46  174.60                    8.50\n",
      "  162  20025 306G 307M C   N    2339 2341 -165.77 -125.60   46.39    1.38  -63.40 -173.58   35.65\n",
      "  162        307M 307M N   CA   2341 2342  163.70  140.50                  -40.50\n",
      "  163  20026 307M 308H C   N    2347 2349 -159.58 -125.60   42.30    1.05  -63.20 -178.57   28.86\n",
      "  163        308H 308H N   CA   2349 2350  163.99  138.80                  -42.30\n",
      "  164  20027 308H 309N C   N    2357 2359 -156.90 -119.90   43.41    1.29  -63.20 -175.26   28.31\n",
      "  164        309N 309N N   CA   2359 2360  159.69  137.00                  -41.10\n",
      "  165  20028 309N 310F C   N    2365 2367 -163.81 -124.20   45.54    1.21  -63.20 -179.44   31.27\n",
      "  165        310F 310F N   CA   2367 2368  165.77  143.30                  -44.30\n",
      "  166  20029 310F 311G C   N    2376 2378  177.74 -167.20   21.03    0.78   82.20  179.03   12.89\n",
      "  166        311G 311G N   CA   2378 2379  159.91  174.60                    8.50\n",
      "  167  20030 311G 312N C   N    2380 2382 -160.52 -119.90   49.76    1.46  -63.20 -178.54   28.06\n",
      "  167        312N 312N N   CA   2382 2383  165.74  137.00                  -41.10\n",
      "  168  20031 312N 313S C   N    2388 2390 -160.58 -136.60   27.93    0.85  -64.10 -173.61   20.46\n",
      "  168        313S 313S N   CA   2390 2391  165.51  151.20                  -35.00\n",
      "  169  20033 314C 315Y C   N    2400 2402 -144.58 -124.30   34.76    1.40  -63.50  173.12   31.40\n",
      "  169        315Y 315Y N   CA   2402 2403  163.64  135.40                  -43.40\n",
      "  170  20034 315Y 316M C   N    2412 2414 -160.37 -125.60   38.29    1.17  -63.40 -170.37   35.79\n",
      "  170        316M 316M N   CA   2414 2415  156.54  140.50                  -40.50\n",
      "  171  20035 316M 317A C   N    2420 2422 -168.70 -134.00   37.08    0.86  -62.50 -168.76   36.97\n",
      "  171        317A 317A N   CA   2422 2423  160.06  147.00                  -40.90\n",
      "  172  20036 317A 318S C   N    2425 2427 -167.73 -136.60   32.95    1.06  -64.10 -166.85   21.47\n",
      "  172        318S 318S N   CA   2427 2428  162.00  151.20                  -35.00\n",
      "  173  20038 319V 320L C   N    2438 2440 -145.34 -108.50   42.46    1.87  -63.50 -175.66   30.66\n",
      "  173        320L 320L N   CA   2440 2441  153.62  132.50                  -41.20\n",
      "  174  20039 320L 321Q C   N    2446 2448 -155.93 -121.10   38.48    1.20  -63.80 -172.21   32.35\n",
      "  174        321Q 321Q N   CA   2448 2449  156.06  139.70                  -40.30\n",
      "  175  20041 322C 323L C   N    2461 2463 -144.45 -108.50   43.36    1.92  -63.50 -178.85   30.16\n",
      "  175        323L 323L N   CA   2463 2464  156.75  132.50                  -41.20\n",
      "  176  20042 323L 324F C   N    2469 2471 -151.42 -124.20   29.80    0.84  -63.20 -177.05   31.01\n",
      "  176        324F 324F N   CA   2471 2472  155.42  143.30                  -44.30\n",
      "  177  20043 324F 325S C   N    2480 2482 -156.06 -136.60   20.17    0.68  -64.10 -168.03   20.52\n",
      "  177        325S 325S N   CA   2482 2483  156.49  151.20                  -35.00\n",
      "  178  20044 325S 326L C   N    2486 2488 -153.41 -108.50   47.15    2.11  -63.50 -165.97   32.49\n",
      "  178        326L 326L N   CA   2488 2489  146.86  132.50                  -41.20\n",
      "  179  20045 326L 327E C   N    2494 2496 -132.45 -117.80   16.17    0.77  -63.60 -176.35   21.56\n",
      "  179        327E 327E N   CA   2496 2497  129.96  136.80                  -40.30\n",
      "  180  20046 327E 328P C   N    2503 2505  -66.18  -58.70   13.98    1.75  -64.50  170.50   12.63\n",
      "  180        328P 328P N   CA   2505 2506  -42.31  -30.50                  147.20\n",
      "  181  20047 328P 329F C   N    2510 2512   62.98  -71.40  134.77    9.44  -63.20 -152.46   23.45\n",
      "  181        329F 329F N   CA   2512 2513  150.92  140.70                  -44.30\n",
      "  182  20048 329F 330K C   N    2521 2523 -156.77 -118.00   41.40    1.38  -62.90 -169.68   30.32\n",
      "  182        330K 330K N   CA   2523 2524  153.64  139.10                  -40.80\n",
      "  183  20049 330K 331E C   N    2530 2532 -160.31 -117.80   47.06    1.38  -63.60 -170.72   30.57\n",
      "  183        331E 331E N   CA   2532 2533  156.99  136.80                  -40.30\n",
      "  184  20050 331E 332A C   N    2539 2541 -160.63 -134.00   28.23    0.66  -62.50 -169.96   36.37\n",
      "  184        332A 332A N   CA   2541 2542  156.36  147.00                  -40.90\n",
      "  185  20051 332A 333F C   N    2544 2546 -154.70 -124.20   32.24    0.96  -63.20 -173.99   31.62\n",
      "  185        333F 333F N   CA   2546 2547  153.75  143.30                  -44.30\n",
      "  186  20052 333F 334Y C   N    2555 2557 -140.15 -124.30   20.27    0.98  -63.50 -174.83   33.17\n",
      "  186        334Y 334Y N   CA   2557 2558  148.04  135.40                  -43.40\n",
      "  187  20053 334Y 335Y C   N    2567 2569 -142.38 -124.30   28.89    1.19  -63.50  177.19   31.97\n",
      "  187        335Y 335Y N   CA   2569 2570  157.94  135.40                  -43.40\n",
      "  188  20054 335Y 336N C   N    2579 2581 -147.30 -119.90   35.19    1.02  -63.20 -179.40   27.30\n",
      "  188        336N 336N N   CA   2581 2582  159.07  137.00                  -41.10\n",
      "  189  20056 337C 338E C   N    2593 2595 -156.57 -117.80   39.91    1.26  -63.60 -163.21   31.40\n",
      "  189        338E 338E N   CA   2595 2596  146.25  136.80                  -40.30\n",
      "  190  20057 338E 339T C   N    2602 2604 -148.75 -124.80   27.26    1.06  -63.20 -177.36   28.95\n",
      "  190        339T 339T N   CA   2604 2605  156.53  143.50                  -42.10\n",
      "  191  20058 339T 340R C   N    2609 2611 -159.07 -125.20   37.75    1.15  -63.00 -171.98   31.84\n",
      "  191        340R 340R N   CA   2611 2612  157.27  140.60                  -41.10\n",
      "  192  20059 340R 341H C   N    2620 2622 -163.26 -125.60   41.25    1.11  -63.20 -169.54   30.23\n",
      "  192        341H 341H N   CA   2622 2623  155.64  138.80                  -42.30\n",
      "  193  20060 341H 342Q C   N    2630 2632 -160.35 -121.10   41.74    1.33  -63.80 -168.14   33.16\n",
      "  193        342Q 342Q N   CA   2632 2633  153.90  139.70                  -40.30\n",
      "  194  20061 342Q 343N C   N    2639 2641 -152.68 -119.90   38.14    1.14  -63.20 -174.56   28.19\n",
      "  194        343N 343N N   CA   2641 2642  156.49  137.00                  -41.10\n",
      "  195  20062 343N 344A C   N    2647 2649 -160.32 -134.00   28.04    0.65  -62.50 -170.37   36.29\n",
      "  195        344A 344A N   CA   2649 2650  156.65  147.00                  -40.90\n",
      "  196  20064 345C 346Q C   N    2658 2660 -155.03 -121.10   38.02    1.18  -63.80 -173.36   32.14\n",
      "  196        346Q 346Q N   CA   2660 2661  156.87  139.70                  -40.30\n",
      "  197  20065 346Q 347E C   N    2667 2669 -148.10 -117.80   34.73    1.02  -63.60 -173.79   29.54\n",
      "  197        347E 347E N   CA   2669 2670  153.77  136.80                  -40.30\n",
      "  198  20066 347E 348N C   N    2676 2678 -138.01 -119.90   18.51    0.67  -63.20 -166.80   28.39\n",
      "  198        348N 348N N   CA   2678 2679  140.78  137.00                  -41.10\n",
      "  199  20068 349P 350Y C   N    2691 2693 -137.31 -124.30   21.90    0.89  -63.50  179.46   32.12\n",
      "  199        350Y 350Y N   CA   2693 2694  153.02  135.40                  -43.40\n",
      "  200  20069 350Y 351K C   N    2703 2705 -142.68 -118.00   27.82    0.92  -62.90 -174.69   28.84\n",
      "  200        351K 351K N   CA   2705 2706  151.94  139.10                  -40.80\n",
      "  201  20071 352C 353H C   N    2718 2720 -152.80 -125.60   31.05    0.80  -63.20 -173.19   29.15\n",
      "  201        353H 353H N   CA   2720 2721  153.78  138.80                  -42.30\n",
      "  202  20074 355C 356Q C   N    2740 2742 -153.53 -121.10   36.18    1.12  -63.80 -173.09   32.11\n",
      "  202        356Q 356Q N   CA   2742 2743  155.74  139.70                  -40.30\n",
      "  203  20075 356Q 357T C   N    2749 2751 -143.98 -124.80   25.71    0.86  -63.20  176.81   27.91\n",
      "  203        357T 357T N   CA   2751 2752  160.62  143.50                  -42.10\n",
      "  204  20076 357T 358E C   N    2756 2758 -153.17 -117.80   40.21    1.18  -63.60 -173.34   29.86\n",
      "  204        358E 358E N   CA   2758 2759  155.93  136.80                  -40.30\n",
      "  205  20077 358E 359R C   N    2765 2767 -150.43 -125.20   28.42    0.86  -63.00 -173.08   31.17\n",
      "  205        359R 359R N   CA   2767 2768  153.69  140.60                  -41.10\n",
      "  206  20079 360V 361A C   N    2783 2785 -159.92 -134.00   27.85    0.64  -62.50 -171.02   36.16\n",
      "  206        361A 361A N   CA   2785 2786  157.16  147.00                  -40.90\n",
      "  207  20080 361A 362S C   N    2788 2790 -164.86 -136.60   34.04    1.04  -64.10 -175.26   20.66\n",
      "  207        362S 362S N   CA   2790 2791  170.16  151.20                  -35.00\n",
      "  208  20081 362S 363G C   N    2794 2796  173.51 -167.20   31.18    1.22   82.20  168.48   12.17\n",
      "  208        363G 363G N   CA   2796 2797  150.10  174.60                    8.50\n",
      "  209  20082 363G 364L C   N    2798 2800 -143.15 -108.50   38.98    1.72  -63.50 -173.67   30.80\n",
      "  209        364L 364L N   CA   2800 2801  150.35  132.50                  -41.20\n",
      "  210  20083 364L 365L C   N    2806 2808 -135.95 -108.50   30.43    1.34  -63.50 -172.29   30.55\n",
      "  210        365L 365L N   CA   2808 2809  145.64  132.50                  -41.20\n",
      "  211  20084 365L 366S C   N    2814 2816 -155.20 -136.60   24.79    0.78  -64.10 -178.13   19.74\n",
      "  211        366S 366S N   CA   2816 2817  167.58  151.20                  -35.00\n",
      "  212  20085 366S 367G C   N    2820 2822  172.46 -167.20   27.35    1.00   82.20  173.19   12.39\n",
      "  212        367G 367G N   CA   2822 2823  156.31  174.60                    8.50\n",
      "  213  20086 367G 368E C   N    2824 2826 -154.71 -117.80   42.59    1.25  -63.60 -174.44   29.78\n",
      "  213        368E 368E N   CA   2826 2827  158.05  136.80                  -40.30\n",
      "  214  20087 368E 369F C   N    2833 2835 -149.74 -124.20   30.16    0.79  -63.20  178.71   30.31\n",
      "  214        369F 369F N   CA   2835 2836  159.34  143.30                  -44.30\n",
      "  215  20088 369F 370S C   N    2844 2846 -152.58 -136.60   18.62    0.57  -64.10 -173.44   19.88\n",
      "  215        370S 370S N   CA   2846 2847  160.76  151.20                  -35.00\n",
      "  216  20090 371V 372E C   N    2857 2859 -155.10 -117.80   41.90    1.23  -63.60 -172.35   30.09\n",
      "  216        372E 372E N   CA   2859 2860  155.87  136.80                  -40.30\n",
      "  217  20091 372E 373G C   N    2866 2868  178.93 -167.20   24.16    0.96   82.20  175.40   12.73\n",
      "  217        373G 373G N   CA   2868 2869  154.82  174.60                    8.50\n",
      "  218  20092 373G 374Q C   N    2870 2872 -158.46 -121.10   41.45    1.29  -63.80 -172.33   32.44\n",
      "  218        374Q 374Q N   CA   2872 2873  157.65  139.70                  -40.30\n",
      "  219  20093 374Q 375E C   N    2879 2881 -146.25 -117.80   29.71    0.91  -63.60 -167.05   30.35\n",
      "  219        375E 375E N   CA   2881 2882  145.34  136.80                  -40.30\n",
      "  220  20094 375E 376L C   N    2888 2890 -142.33 -108.50   34.06    1.58  -63.50 -165.67   23.89\n",
      "  220        376L 376L N   CA   2890 2891  136.42  132.50                  -41.20\n",
      "  221  20095 376L 377T C   N    2896 2898 -137.69 -124.80   15.05    0.57  -63.20 -177.48   28.29\n",
      "  221        377T 377T N   CA   2898 2899  151.27  143.50                  -42.10\n",
      "  222  20096 377T 378N C   N    2903 2905 -160.32 -119.90   49.01    1.44  -63.20 -177.77   28.15\n",
      "  222        378N 378N N   CA   2905 2906  164.71  137.00                  -41.10\n",
      "  223  20097 378N 379G C   N    2911 2913  170.58 -167.20   40.65    1.63   82.20  158.90   11.55\n",
      "  223        379G 379G N   CA   2913 2914  140.55  174.60                    8.50\n",
      "  224  20099 380I 381T C   N    2923 2925 -142.11 -124.80   21.90    0.76  -63.20  179.29   28.13\n",
      "  224        381T 381T N   CA   2925 2926  156.91  143.50                  -42.10\n",
      "  225  20100 381T 382A C   N    2930 2932 -164.56 -134.00   31.66    0.78  -62.50 -166.97   37.06\n",
      "  225        382A 382A N   CA   2932 2933  155.27  147.00                  -40.90\n",
      "  226  20101 382A 383R C   N    2935 2937 -153.34 -125.20   31.74    0.96  -63.00 -173.09   31.35\n",
      "  226        383R 383R N   CA   2937 2938  155.28  140.60                  -41.10\n",
      "  227  20102 383R 384E C   N    2946 2948 -149.33 -117.80   35.20    1.03  -63.60 -172.06   29.84\n",
      "  227        384E 384E N   CA   2948 2949  152.45  136.80                  -40.30\n",
      "  228  20103 384E 385F C   N    2955 2957 -144.97 -124.20   23.89    0.64  -63.20 -179.79   30.27\n",
      "  228        385F 385F N   CA   2957 2958  155.11  143.30                  -44.30\n",
      "  229  20104 385F 386K C   N    2966 2968 -148.29 -118.00   32.45    1.08  -62.90 -171.15   29.63\n",
      "  229        386K 386K N   CA   2968 2969  150.76  139.10                  -40.80\n",
      "  230  20105 386K 387R C   N    2975 2977 -148.63 -125.20   25.90    0.79  -63.00 -172.09   31.20\n",
      "  230        387R 387R N   CA   2977 2978  151.63  140.60                  -41.10\n",
      "  231  20107 388V 389F C   N    2993 2995 -153.79 -124.20   32.52    0.91  -63.20 -177.08   31.13\n",
      "  231        389F 389F N   CA   2995 2996  156.78  143.30                  -44.30\n",
      "  232  20108 389F 390A C   N    3004 3006 -160.95 -134.00   28.73    0.67  -62.50 -170.31   36.33\n",
      "  232        390A 390A N   CA   3006 3007  156.96  147.00                  -40.90\n",
      "  233  20109 390A 391Q C   N    3009 3011 -148.69 -121.10   33.10    1.02  -63.80 -177.37   31.25\n",
      "  233        391Q 391Q N   CA   3011 3012  157.99  139.70                  -40.30\n",
      "  234  20110 391Q 392K C   N    3018 3020 -141.47 -118.00   26.45    0.88  -62.90 -174.63   28.77\n",
      "  234        392K 392K N   CA   3020 3021  151.31  139.10                  -40.80\n",
      "  235  20111 392K 393H C   N    3027 3029 -142.94 -125.60   17.43    0.57  -63.20 -165.73   29.49\n",
      "  235        393H 393H N   CA   3029 3030  140.55  138.80                  -42.30\n",
      "  236  20113 394P 395D C   N    3044 3046 -137.22  -96.50   45.00    1.86  -63.30 -171.56   20.11\n",
      "  236        395D 395D N   CA   3046 3047  133.33  114.20                  -40.00\n",
      "  237  20114 395D 396F C   N    3052 3054 -132.36 -124.20    8.91    0.25  -63.20 -177.56   29.86\n",
      "  237        396F 396F N   CA   3054 3055  146.88  143.30                  -44.30\n",
      "  238  20115 396F 397S C   N    3063 3065 -154.90 -136.60   19.19    0.63  -64.10 -168.99   20.37\n",
      "  238        397S 397S N   CA   3065 3066  156.96  151.20                  -35.00\n",
      "  239  20116 397S 398T C   N    3069 3071 -142.68 -124.80   23.87    0.80  -63.20  177.40   27.91\n",
      "  239        398T 398T N   CA   3071 3072  159.31  143.50                  -42.10\n",
      "  240  20117 398T 399A C   N    3076 3078 -156.04 -134.00   23.39    0.55  -62.50 -170.94   35.98\n",
      "  240        399A 399A N   CA   3078 3079  154.80  147.00                  -40.90\n",
      "  241  20118 399A 400E C   N    3081 3083 -151.66 -117.80   35.89    1.08  -63.60 -167.65   30.55\n",
      "  241        400E 400E N   CA   3083 3084  148.69  136.80                  -40.30\n",
      "  242  20119 400E 401Q C   N    3090 3092 -153.39 -121.10   34.80    1.10  -63.80 -170.46   32.50\n",
      "  242        401Q 401Q N   CA   3092 3093  152.67  139.70                  -40.30\n",
      "  243  20120 401Q 402Q C   N    3099 3101 -144.91 -121.10   24.29    0.82  -63.80 -166.91   32.64\n",
      "  243        402Q 402Q N   CA   3101 3102  144.47  139.70                  -40.30\n",
      "  244  20121 402Q 403D C   N    3108 3110 -144.07  -96.50   50.94    2.10  -63.30 -169.59   20.24\n",
      "  244        403D 403D N   CA   3110 3111  132.43  114.20                  -40.00\n",
      "  245  20122 403D 404A C   N    3116 3118 -144.23 -134.00   10.52    0.27  -62.50 -171.66   35.24\n",
      "  245        404A 404A N   CA   3118 3119  149.43  147.00                  -40.90\n",
      "  246  20123 404A 405Q C   N    3121 3123 -156.20 -121.10   37.56    1.19  -63.80 -169.48   32.77\n",
      "  246        405Q 405Q N   CA   3123 3124  153.08  139.70                  -40.30\n",
      "  247  20124 405Q 406E C   N    3130 3132 -137.36 -117.80   19.56    0.71  -63.60 -167.73   22.53\n",
      "  247        406E 406E N   CA   3132 3133  137.25  136.80                  -40.30\n",
      "  248  20125 406E 407Y C   N    3139 3141 -136.28 -124.30   12.82    1.06  -63.50 -171.18   26.75\n",
      "  248        407Y 407Y N   CA   3141 3142  130.83  135.40                  -43.40\n",
      "  249  20126 407Y 408F C   N    3151 3153 -158.61 -124.20   34.89    1.19  -63.20 -168.00   32.69\n",
      "  249        408F 408F N   CA   3153 3154  149.08  143.30                  -44.30\n",
      "  250  20127 408F 409L C   N    3162 3164 -144.45 -108.50   41.10    1.81  -63.50 -174.98   30.70\n",
      "  250        409L 409L N   CA   3164 3165  152.42  132.50                  -41.20\n",
      "  251  20128 409L 410Y C   N    3170 3172 -132.50 -124.30   13.78    0.56  -63.50 -176.41   32.57\n",
      "  251        410Y 410Y N   CA   3172 3173  146.47  135.40                  -43.40\n",
      "  252  20129 410Y 411L C   N    3182 3184 -130.54 -108.50   25.85    1.14  -63.50 -174.65   29.89\n",
      "  252        411L 411L N   CA   3184 3185  146.00  132.50                  -41.20\n",
      "  253  20131 412V 413E C   N    3197 3199 -149.24 -117.80   33.44    1.00  -63.60 -168.28   30.34\n",
      "  253        413E 413E N   CA   3199 3200  148.17  136.80                  -40.30\n",
      "  254  20132 413E 414Q C   N    3206 3208 -151.60 -121.10   32.42    1.03  -63.80 -169.56   32.55\n",
      "  254        414Q 414Q N   CA   3208 3209  150.71  139.70                  -40.30\n",
      "  255  20133 414Q 415M C   N    3215 3217 -150.95 -125.60   26.97    0.86  -63.40 -168.98   35.36\n",
      "  255        415M 415M N   CA   3217 3218  149.72  140.50                  -40.50\n",
      "  256  20134 415M 416R C   N    3223 3225 -150.03 -125.20   27.08    0.84  -63.00 -171.25   31.40\n",
      "  256        416R 416R N   CA   3225 3226  151.42  140.60                  -41.10\n",
      "  257  20135 416R 417R C   N    3234 3236 -144.39 -125.20   19.24    0.81  -63.00 -162.74   32.21\n",
      "  257        417R 417R N   CA   3236 3237  139.22  140.60                  -41.10\n",
      "  258  20136 417R 418Y C   N    3245 3247 -140.31 -124.30   22.86    1.01  -63.50 -178.10   32.64\n",
      "  258        418Y 418Y N   CA   3247 3248  151.71  135.40                  -43.40\n",
      "  259  20138 419V 420K C   N    3264 3266 -142.98 -118.00   25.06    0.91  -62.90 -164.79   30.14\n",
      "  259        420K 420K N   CA   3266 3267  141.17  139.10                  -40.80\n",
      "  260  20140 421P 422S C   N    3280 3282 -144.07 -136.60    7.47    0.30  -64.10 -168.64   19.57\n",
      "  260        422S 422S N   CA   3282 3283  151.15  151.20                  -35.00\n",
      "  261  20141 422S 423N C   N    3286 3288 -141.26 -119.90   23.75    0.74  -63.20 -171.54   27.97\n",
      "  261        423N 423N N   CA   3288 3289  147.37  137.00                  -41.10\n",
      "  262  20144 425V 426G C   N    3309 3311  164.93 -167.20   33.88    1.16   82.20  168.53   11.88\n",
      "  262        426G 426G N   CA   3311 3312  155.33  174.60                    8.50\n",
      "  263  20145 426G 427A C   N    3313 3315 -158.11 -134.00   25.92    0.59  -62.50 -171.38   36.01\n",
      "  263        427A 427A N   CA   3315 3316  156.51  147.00                  -40.90\n",
      "  264  20146 427A 428N C   N    3318 3320 -140.45 -119.90   22.49    0.71  -63.20 -170.77   28.03\n",
      "  264        428N 428N N   CA   3320 3321  146.15  137.00                  -41.10\n",
      "  265  20148 429I 430S C   N    3334 3336 -145.32 -136.60    9.20    0.30  -64.10 -170.81   19.52\n",
      "  265        430S 430S N   CA   3336 3337  154.12  151.20                  -35.00\n",
      "  266  20149 430S 431H C   N    3340 3342 -141.21 -125.60   16.57    0.68  -63.20 -167.89   20.56\n",
      "  266        431H 431H N   CA   3342 3343  133.25  138.80                  -42.30\n",
      "  267  20152 433V 434D C   N    3364 3366 -127.77  -96.50   35.88    1.49  -63.30 -176.50   19.73\n",
      "  267        434D 434D N   CA   3366 3367  131.80  114.20                  -40.00\n",
      "  268  20154 435I 436F C   N    3380 3382 -138.54 -124.20   14.50    0.50  -63.20 -173.87   30.75\n",
      "  268        436F 436F N   CA   3382 3383  145.50  143.30                  -44.30\n",
      "  269  20155 436F 437K C   N    3391 3393 -155.46 -118.00   39.15    1.32  -62.90 -167.57   30.52\n",
      "  269        437K 437K N   CA   3393 3394  150.50  139.10                  -40.80\n",
      "  270  20156 437K 438M C   N    3400 3402 -146.07 -125.60   21.65    0.70  -63.40 -169.21   34.98\n",
      "  270        438M 438M N   CA   3402 3403  147.54  140.50                  -40.50\n",
      "  271  20157 438M 439T C   N    3408 3410 -141.56 -124.80   20.19    0.73  -63.20 -179.01   28.32\n",
      "  271        439T 439T N   CA   3410 3411  154.76  143.50                  -42.10\n",
      "  272  20159 440V 441E C   N    3422 3424 -151.96 -117.80   36.21    1.09  -63.60 -167.64   30.57\n",
      "  272        441E 441E N   CA   3424 3425  148.83  136.80                  -40.30\n",
      "  273  20160 441E 442H C   N    3431 3433 -149.96 -125.60   26.27    0.72  -63.20 -169.95   29.39\n",
      "  273        442H 442H N   CA   3433 3434  148.61  138.80                  -42.30\n",
      "  274  20161 442H 443R C   N    3441 3443 -148.05 -125.20   25.51    0.78  -63.00 -172.63   31.09\n",
      "  274        443R 443R N   CA   3443 3444  151.94  140.60                  -41.10\n",
      "  275  20163 444V 445Q C   N    3459 3461 -144.07 -121.10   24.31    0.78  -63.80 -170.16   32.11\n",
      "  275        445Q 445Q N   CA   3461 3462  147.66  139.70                  -40.30\n",
      "  276  20165 446C 447G C   N    3474 3476  169.10 -167.20   31.72    1.16   82.20  169.06   12.06\n",
      "  276        447G 447G N   CA   3476 3477  153.52  174.60                    8.50\n",
      "  277  20166 447G 448S C   N    3478 3480 -157.93 -136.60   22.90    0.73  -64.10 -169.78   20.54\n",
      "  277        448S 448S N   CA   3480 3481  159.53  151.20                  -35.00\n",
      "  278  20168 449C 450H C   N    3490 3492 -145.63 -125.60   22.06    0.59  -63.20 -171.37   28.95\n",
      "  278        450H 450H N   CA   3492 3493  148.04  138.80                  -42.30\n",
      "  279  20169 450H 451K C   N    3500 3502 -143.46 -118.00   26.52    0.90  -62.90 -169.45   29.57\n",
      "  279        451K 451K N   CA   3502 3503  146.52  139.10                  -40.80\n",
      "  280  20171 452V 453R C   N    3516 3518 -141.73 -125.20   16.60    0.63  -63.00 -166.48   31.53\n",
      "  280        453R 453R N   CA   3518 3519  142.12  140.60                  -41.10\n",
      "  281  20172 453R 454Y C   N    3527 3529 -134.78 -124.30   12.12    0.66  -63.50 -170.92   33.56\n",
      "  281        454Y 454Y N   CA   3529 3530  141.47  135.40                  -43.40\n",
      "  282  20173 454Y 455T C   N    3539 3541 -137.99 -124.80   15.86    0.58  -63.20 -178.30   28.20\n",
      "  282        455T 455T N   CA   3541 3542  152.31  143.50                  -42.10\n",
      "  283  20174 455T 456R C   N    3546 3548 -141.50 -125.20   17.20    0.56  -63.00 -170.22   31.00\n",
      "  283        456R 456R N   CA   3548 3549  146.11  140.60                  -41.10\n",
      "  284  20175 456R 457E C   N    3557 3559 -142.74 -117.80   25.31    0.83  -63.60 -164.67   30.49\n",
      "  284        457E 457E N   CA   3559 3560  141.12  136.80                  -40.30\n",
      "  285  20176 457E 458T C   N    3566 3568 -134.02 -124.80    9.35    0.53  -63.20 -170.32   28.99\n",
      "  285        458T 458T N   CA   3568 3569  141.94  143.50                  -42.10\n",
      "  286  20177 458T 459D C   N    3573 3575 -131.78  -96.50   36.89    1.52  -63.30  178.60   19.09\n",
      "  286        459D 459D N   CA   3575 3576  124.95  114.20                  -40.00\n",
      "  287  20180 461C 462L C   N    3593 3595 -130.47 -108.50   23.16    1.03  -63.50 -168.92   30.68\n",
      "  287        462L 462L N   CA   3595 3596  139.84  132.50                  -41.20\n",
      "  288  20181 462L 463S C   N    3601 3603 -135.92 -136.60    5.96    0.29  -64.10 -166.46   19.11\n",
      "  288        463S 463S N   CA   3603 3604  145.28  151.20                  -35.00\n",
      "  289  20182 463S 464L C   N    3607 3609 -130.74 -108.50   26.15    1.15  -63.50 -174.82   29.88\n",
      "  289        464L 464L N   CA   3609 3610  146.26  132.50                  -41.20\n",
      "  290  20186 467P 468L C   N    3637 3639 -115.01 -108.50    7.35    0.39  -63.50  177.91   22.33\n",
      "  290        468L 468L N   CA   3639 3640  129.09  132.50                  -41.20\n",
      "  291  20187 468L 469D C   N    3645 3647 -113.62  -96.50   18.40    0.76  -63.30  168.62   18.38\n",
      "  291        469D 469D N   CA   3647 3648  120.94  114.20                  -40.00\n",
      "  292  20189 470P 471N C   N    3660 3662 -139.67 -119.90   26.34    0.76  -63.20 -178.59   27.00\n",
      "  292        471N 471N N   CA   3662 3663  154.40  137.00                  -41.10\n",
      "  293  20190 471N 472L C   N    3668 3670 -122.65 -108.50   14.40    0.66  -63.50 -173.98   23.20\n",
      "  293        472L 472L N   CA   3670 3671  135.16  132.50                  -41.20\n",
      "  294  20191 472L 473K C   N    3676 3678 -134.15 -118.00   17.27    0.58  -62.90 -172.00   28.68\n",
      "  294        473K 473K N   CA   3678 3679  145.23  139.10                  -40.80\n",
      "  295  20192 473K 474N C   N    3685 3687 -139.74 -119.90   22.29    0.69  -63.20 -171.98   27.84\n",
      "  295        474N 474N N   CA   3687 3688  147.16  137.00                  -41.10\n",
      "  296  20193 474N 475S C   N    3693 3695 -131.82 -136.60   10.49    0.41  -64.10 -170.62   11.70\n",
      "  296        475S 475S N   CA   3695 3696  141.85  151.20                  -35.00\n",
      "  297  20194 475S 476D C   N    3699 3701 -138.23  -96.50   54.55    2.28  -63.30 -173.60   27.85\n",
      "  297        476D 476D N   CA   3701 3702  149.33  114.20                  -40.00\n",
      "  298  20195 476D 477G C   N    3707 3709 -166.53 -167.20   26.08    1.01   82.20  178.85   13.35\n",
      "  298        477G 477G N   CA   3709 3710  148.53  174.60                    8.50\n",
      "  299  20196 477G 478N C   N    3711 3713 -136.24 -119.90   19.53    0.58  -63.20 -173.85   27.41\n",
      "  299        478N 478N N   CA   3713 3714  147.68  137.00                  -41.10\n",
      "  300  20198 479P 480K C   N    3726 3728 -132.60 -118.00   14.67    0.53  -62.90 -168.21   29.07\n",
      "  300        480K 480K N   CA   3728 3729  140.53  139.10                  -40.80\n",
      "  301  20199 480K 481Q C   N    3735 3737 -131.02 -121.10   12.73    0.40  -63.80 -175.31   30.71\n",
      "  301        481Q 481Q N   CA   3737 3738  147.67  139.70                  -40.30\n",
      "  302  20200 481Q 482T C   N    3744 3746 -133.78 -124.80   17.47    0.54  -63.20  174.35   26.98\n",
      "  302        482T 482T N   CA   3746 3747  158.48  143.50                  -42.10\n",
      "  303  20201 482T 483E C   N    3751 3753 -128.60 -117.80   19.52    0.71  -63.60  178.86   27.51\n",
      "  303        483E 483E N   CA   3753 3754  153.06  136.80                  -40.30\n",
      "  304  20202 483E 484E C   N    3760 3762  -84.14  -69.30   25.06    1.43  -63.60  158.34   22.26\n",
      "  304        484E 484E N   CA   3762 3763  162.70  142.50                  -40.30\n",
      "  305  20203 484E 485E C   N    3769 3771 -115.95 -117.80   32.46    1.66  -63.60  159.34   24.20\n",
      "  305        485E 485E N   CA   3771 3772  169.21  136.80                  -40.30\n",
      "  306  20205 486I 487F C   N    3786 3788 -109.64 -124.20   16.16    0.82  -63.20  171.77   26.96\n",
      "  306        487F 487F N   CA   3788 3789  150.32  143.30                  -44.30\n",
      "  307  20206 487F 488A C   N    3797 3799 -134.83 -134.00    9.93    0.54  -62.50  177.60   32.98\n",
      "  307        488A 488A N   CA   3799 3800  156.90  147.00                  -40.90\n",
      "  308  20207 488A 489S C   N    3802 3804 -132.28 -136.60   15.62    0.86  -64.10  172.81   17.37\n",
      "  308        489S 489S N   CA   3804 3805  166.20  151.20                  -35.00\n",
      "  309  20208 489S 490R C   N    3808 3810 -123.76 -125.20    3.23    0.12  -63.00 -171.15   22.13\n",
      "  309        490R 490R N   CA   3810 3811  137.72  140.60                  -41.10\n",
      "  310  20210 491P 492H C   N    3826 3828 -134.31 -125.60   14.42    0.37  -63.20 -178.12   27.38\n",
      "  310        492H 492H N   CA   3828 3829  150.29  138.80                  -42.30\n",
      "  311  20212 493I 494S C   N    3844 3846 -140.68 -136.60    4.69    0.14  -64.10 -172.18   19.07\n",
      "  311        494S 494S N   CA   3846 3847  153.50  151.20                  -35.00\n",
      "  312  20213 494S 495L C   N    3850 3852 -131.46 -108.50   25.23    1.12  -63.50 -171.49   30.38\n",
      "  312        495L 495L N   CA   3852 3853  142.97  132.50                  -41.20\n",
      "  313  20214 495L 496D C   N    3858 3860 -129.80  -96.50   45.01    1.88  -63.30 -172.30   27.47\n",
      "  313        496D 496D N   CA   3860 3861  144.48  114.20                  -40.00\n",
      "  314  20215 496D 497A C   N    3866 3868 -144.18 -134.00   11.46    0.26  -62.50 -174.25   34.81\n",
      "  314        497A 497A N   CA   3868 3869  152.28  147.00                  -40.90\n",
      "  315  20217 498C 499L C   N    3877 3879 -136.76 -108.50   40.15    1.81  -63.50  173.96   28.69\n",
      "  315        499L 499L N   CA   3879 3880  161.02  132.50                  -41.20\n",
      "  316  20218 499L 500G C   N    3885 3887 -171.40 -167.20   17.36    0.71   82.20 -176.71   13.47\n",
      "  316        500G 500G N   CA   3887 3888  157.75  174.60                    8.50\n",
      "  317  20219 500G 501S C   N    3889 3891 -148.22 -136.60   13.50    0.41  -64.10 -173.06   19.58\n",
      "  317        501S 501S N   CA   3891 3892  158.06  151.20                  -35.00\n",
      "  318  20220 501S 502L C   N    3895 3897 -133.01 -108.50   27.92    1.23  -63.50 -173.61   30.18\n",
      "  318        502L 502L N   CA   3897 3898  145.86  132.50                  -41.20\n",
      "  319  20221 502L 503M C   N    3903 3905 -130.07 -125.60   13.82    0.55  -63.40  178.82   31.98\n",
      "  319        503M 503M N   CA   3905 3906  153.57  140.50                  -40.50\n",
      "  320  20222 503M 504Q C   N    3911 3913 -129.59 -121.10   13.10    0.44  -63.80 -177.68   30.29\n",
      "  320        504Q 504Q N   CA   3913 3914  149.67  139.70                  -40.30\n",
      "  321  20223 504Q 505A C   N    3920 3922  -86.08  -68.20   27.17    1.71  -62.50  155.14   26.64\n",
      "  321        505A 505A N   CA   3922 3923  165.76  145.30                  -40.90\n",
      "  322  20224 505A 506T C   N    3925 3927 -122.12 -124.80    6.72    0.36  -63.20  178.26   26.76\n",
      "  322        506T 506T N   CA   3927 3928  149.65  143.50                  -42.10\n",
      "  323  20225 506T 507D C   N    3932 3934  -87.04  -70.90   29.19    1.00  -63.30  147.31   19.69\n",
      "  323        507D 507D N   CA   3934 3935  174.62  150.30                  -40.00\n",
      "  324  20227 508I 509D C   N    3948 3950 -113.09  -96.50   30.57    1.29  -63.30 -173.36   20.54\n",
      "  324        509D 509D N   CA   3950 3951  139.87  114.20                  -40.00\n",
      "  325  20229 510C 511R C   N    3962 3964 -136.38 -125.20   17.74    0.57  -63.00 -179.86   29.36\n",
      "  325        511R 511R N   CA   3964 3965  154.38  140.60                  -41.10\n",
      "  326  20231 512C 513S C   N    3979 3981 -149.59 -136.60   15.96    0.49  -64.10 -174.58   19.57\n",
      "  326        513S 513S N   CA   3981 3982  160.46  151.20                  -35.00\n",
      "  327  20232 513S 514A C   N    3985 3987 -149.67 -134.00   17.05    0.39  -62.50 -173.04   35.30\n",
      "  327        514A 514A N   CA   3987 3988  153.71  147.00                  -40.90\n",
      "  328  20234 515C 516G C   N    3996 3998 -177.38 -167.20   26.62    1.09   82.20  173.51   12.74\n",
      "  328        516G 516G N   CA   3998 3999  150.00  174.60                    8.50\n",
      "  329  20238 519V 520T C   N    4022 4024 -133.72 -124.80   11.21    0.39  -63.20 -178.15   27.96\n",
      "  329        520T 520T N   CA   4024 4025  150.28  143.50                  -42.10\n",
      "  330  20239 520T 521Y C   N    4029 4031 -139.94 -124.30   24.39    1.02  -63.50  179.57   32.25\n",
      "  330        521Y 521Y N   CA   4031 4032  154.11  135.40                  -43.40\n",
      "  331  20241 522C 523K C   N    4047 4049 -142.98 -118.00   32.56    1.12  -62.90  178.22   27.93\n",
      "  331        523K 523K N   CA   4049 4050  159.98  139.10                  -40.80\n",
      "  332  20242 523K 524T C   N    4056 4058 -138.68 -124.80   22.25    0.69  -63.20  174.21   27.26\n",
      "  332        524T 524T N   CA   4058 4059  160.89  143.50                  -42.10\n",
      "  333  20243 524T 525T C   N    4063 4065 -143.64 -124.80   30.08    0.94  -63.20  171.04   27.13\n",
      "  333        525T 525T N   CA   4065 4066  166.96  143.50                  -42.10\n",
      "  334  20244 525T 526R C   N    4070 4072 -140.69 -125.20   20.86    0.63  -63.00 -178.23   29.85\n",
      "  334        526R 526R N   CA   4072 4073  154.57  140.60                  -41.10\n",
      "  335  20245 526R 527L C   N    4081 4083 -143.03 -108.50   89.23    4.39  -63.50  130.94   22.98\n",
      "  335        527L 527L N   CA   4083 4084 -145.22  132.50                  -41.20\n",
      "  336  20246 527L 528R C   N    4089 4091   56.84   57.30   31.88    2.06  -63.00  163.33   29.40\n",
      "  336        528R 528R N   CA   4091 4092   69.87   38.00                  -41.10\n",
      "  337  20247 528R 529T C   N    4100 4102  -91.36  -78.10   30.37    1.00  -63.20  143.56   20.29\n",
      "  337        529T 529T N   CA   4102 4103  177.13  149.80                  -42.10\n",
      "  338  20248 529T 530F C   N    4107 4109 -134.30 -124.20   10.16    0.43  -63.20 -172.51   30.69\n",
      "  338        530F 530F N   CA   4109 4110  142.21  143.30                  -44.30\n",
      "  339  20250 531P 532D C   N    4125 4127 -132.29  -70.90   63.66    4.15  -63.30  167.69   25.15\n",
      "  339        532D 532D N   CA   4127 4128  167.16  150.30                  -40.00\n",
      "  340  20252 533V 534L C   N    4140 4142 -135.55 -108.50   27.44    1.26  -63.50 -167.68   23.74\n",
      "  340        534L 534L N   CA   4142 4143  137.11  132.50                  -41.20\n",
      "  341  20253 534L 535P C   N    4148 4150  -65.16  -58.70   15.83    1.83  -64.50  167.85   12.49\n",
      "  341        535P 535P N   CA   4150 4151  -44.95  -30.50                  147.20\n",
      "  342  20254 535P 536I C   N    4155 4157   53.78  -97.30  151.62    6.27 -120.60  174.65   12.43\n",
      "  342        536I 536I N   CA   4157 4158  139.99  127.20                  130.30\n",
      "  343  20255 536I 537F C   N    4163 4165 -146.09 -124.20   30.25    0.81  -63.20  172.72   29.25\n",
      "  343        537F 537F N   CA   4165 4166  164.17  143.30                  -44.30\n",
      "  344  20256 537F 538L C   N    4174 4176 -155.62 -108.50   50.39    2.24  -63.50 -168.01   32.34\n",
      "  344        538L 538L N   CA   4176 4177  150.35  132.50                  -41.20\n",
      "  345  20257 538L 539R C   N    4182 4184 -151.54 -125.20   32.76    0.97  -63.00 -178.16   30.53\n",
      "  345        539R 539R N   CA   4184 4185  160.07  140.60                  -41.10\n",
      "  346  20258 539R 540R C   N    4193 4195 -162.38 -125.20   47.01    1.39  -63.00  179.54   30.84\n",
      "  346        540R 540R N   CA   4195 4196  169.37  140.60                  -41.10\n",
      "  347  20259 540R 541A C   N    4204 4206 -171.70 -134.00   41.98    0.96  -62.50 -171.50   36.65\n",
      "  347        541A 541A N   CA   4206 4207  165.46  147.00                  -40.90\n",
      "  348  20260 541A 542Q C   N    4209 4211 -167.89 -121.10   54.22    1.67  -63.80 -175.27   32.39\n",
      "  348        542Q 542Q N   CA   4211 4212  167.09  139.70                  -40.30\n",
      "  349  20261 542Q 543F C   N    4218 4220 -165.90 -124.20   46.37    1.27  -63.20 -176.45   31.81\n",
      "  349        543F 543F N   CA   4220 4221  163.58  143.30                  -44.30\n",
      "  350  20262 543F 544D C   N    4229 4231 -161.10  -96.50   82.20    3.43  -63.30 -176.75   28.85\n",
      "  350        544D 544D N   CA   4231 4232  165.03  114.20                  -40.00\n",
      "  351  20263 544D 545M C   N    4237 4239 -172.03 -125.60   52.87    1.59  -63.40 -171.77   36.32\n",
      "  351        545M 545M N   CA   4239 4240  165.78  140.50                  -40.50\n",
      "  352  20264 545M 546G C   N    4245 4247 -179.44 -167.20   44.09    1.81   82.20  158.06   11.80\n",
      "  352        546G 546G N   CA   4247 4248  132.24  174.60                    8.50\n",
      "  353  20266 547T 548M C   N    4256 4258   78.28  -73.00  152.35   11.79  -63.40 -147.39   26.99\n",
      "  353        548M 548M N   CA   4258 4259  160.97  143.00                  -40.50\n",
      "  354  20267 548M 549S C   N    4264 4266 -176.61 -136.60   41.03    1.41  -64.10 -160.52   22.58\n",
      "  354        549S 549S N   CA   4266 4267  160.28  151.20                  -35.00\n",
      "  355  20269 550V 551K C   N    4277 4279 -158.09 -118.00   46.90    1.56  -62.90 -177.47   29.37\n",
      "  355        551K 551K N   CA   4279 4280  163.45  139.10                  -40.80\n",
      "  356  20270 551K 552K C   N    4286 4288 -171.82 -118.00   69.45    2.37  -62.90  174.40   29.01\n",
      "  356        552K 552K N   CA   4288 4289 -177.01  139.10                  -40.80\n",
      "  357  20271 552K 553L C   N    4295 4297  173.99 -108.50   78.14    3.61  -63.50 -145.23   37.27\n",
      "  357        553L 553L N   CA   4297 4298  142.40  132.50                  -41.20\n",
      "  358  20272 553L 554D C   N    4303 4305  155.01  -96.50  125.41    5.20  -63.30 -158.77   33.44\n",
      "  358        554D 554D N   CA   4305 4306  177.12  114.20                  -40.00\n",
      "  359  20274 555V 556F C   N    4318 4320  136.21 -124.20  102.77    3.28  -63.20 -142.27   39.09\n",
      "  359        556F 556F N   CA   4320 4321  168.68  143.30                  -44.30\n",
      "  360  20276 557V 558D C   N    4336 4338 -155.52  -70.90   93.61    5.32  -63.30  159.13   25.46\n",
      "  360        558D 558D N   CA   4338 4339 -169.68  150.30                  -40.00\n",
      "  361  20279 560P 561L C   N    4358 4360 -153.25 -108.50   70.59    3.24  -63.50  159.38   27.60\n",
      "  361        561L 561L N   CA   4360 4361 -172.91  132.50                  -41.20\n",
      "  362  20280 561L 562E C   N    4366 4368 -161.66 -117.80   56.92    1.74  -63.60  176.39   28.86\n",
      "  362        562E 562E N   CA   4368 4369  173.08  136.80                  -40.30\n",
      "  363  20281 562E 563L C   N    4375 4377  179.75 -108.50   84.35    3.72  -63.50 -176.20   32.51\n",
      "  363        563L 563L N   CA   4377 4378  176.84  132.50                  -41.20\n",
      "  364  20282 563L 564D C   N    4383 4385 -178.53  -96.50   99.67    4.15  -63.30 -171.49   30.49\n",
      "  364        564D 564D N   CA   4385 4386  170.81  114.20                  -40.00\n",
      "  365  20283 564D 565L C   N    4391 4393  178.16 -108.50   83.27    3.67  -63.50 -171.40   33.29\n",
      "  365        565L 565L N   CA   4393 4394  171.95  132.50                  -41.20\n",
      "  366  20284 565L 566E C   N    4399 4401  178.94 -117.80   72.32    2.11  -63.60 -171.17   31.43\n",
      "  366        566E 566E N   CA   4401 4402  171.84  136.80                  -40.30\n",
      "  367  20285 566E 567F C   N    4408 4410 -178.56 -124.20   59.46    1.67  -63.20 -172.11   33.01\n",
      "  367        567F 567F N   CA   4410 4411  167.40  143.30                  -44.30\n",
      "  368  20286 567F 568L C   N    4419 4421 -171.09 -108.50   75.32    3.33  -63.50 -179.94   31.51\n",
      "  368        568L 568L N   CA   4421 4422  174.41  132.50                  -41.20\n",
      "  369  20287 568L 569R C   N    4427 4429 -168.21 -125.20   62.64    1.93  -63.00  169.40   29.69\n",
      "  369        569R 569R N   CA   4429 4430 -173.86  140.60                  -41.10\n",
      "  370  20288 569R 570G C   N    4438 4440  171.21 -167.20   22.57    0.60   82.20 -177.34   12.86\n",
      "  370        570G 570G N   CA   4440 4441  168.01  174.60                    8.50\n",
      "  371  20289 570G 571K C   N    4442 4444 -165.24 -118.00   65.29    2.29  -62.90  169.43   28.01\n",
      "  371        571K 571K N   CA   4444 4445 -175.83  139.10                  -40.80\n",
      "  372  20290 571K 572G C   N    4451 4453 -177.53 -167.20   10.37    0.18   82.20 -165.26   13.89\n",
      "  372        572G 572G N   CA   4453 4454  175.45  174.60                    8.50\n",
      "  373  20291 572G 573L C   N    4455 4457 -165.08 -108.50   80.86    3.65  -63.50  163.82   28.85\n",
      "  373        573L 573L N   CA   4457 4458 -169.73  132.50                  -41.20\n",
      "  374  20292 573L 574Q C   N    4463 4465 -158.94 -121.10   59.82    2.04  -63.80  164.07   28.87\n",
      "  374        574Q 574Q N   CA   4465 4466 -173.97  139.70                  -40.30\n",
      "  375  20293 574Q 575S C   N    4472 4474  171.64 -136.60   57.59    1.78  -64.10 -166.32   22.98\n",
      "  375        575S 575S N   CA   4474 4475  176.43  151.20                  -35.00\n",
      "  376  20294 575S 576D C   N    4478 4480  170.19  -96.50  103.16    4.27  -63.30 -154.59   33.27\n",
      "  376        576D 576D N   CA   4480 4481  158.18  114.20                  -40.00\n",
      "  377  20295 576D 577E C   N    4486 4488 -174.01 -117.80   62.49    1.83  -63.60 -169.19   31.41\n",
      "  377        577E 577E N   CA   4488 4489  164.08  136.80                  -40.30\n",
      "  378  20297 578V 579S C   N    4502 4504 -175.29 -136.60   52.59    1.67  -64.10  177.35   20.87\n",
      "  378        579S 579S N   CA   4504 4505 -173.18  151.20                  -35.00\n",
      "  379  20298 579S 580M C   N    4508 4510  172.40 -125.60   62.00    2.45  -63.40 -142.11   41.95\n",
      "  379        580M 580M N   CA   4510 4511  140.47  140.50                  -40.50\n",
      "  380  20300 581P 582E C   N    4523 4525 -147.88 -117.80   52.50    1.88  -63.60  163.30   26.41\n",
      "  380        582E 582E N   CA   4525 4526  179.83  136.80                  -40.30\n",
      "  381  20301 582E 583A C   N    4532 4534 -167.73 -134.00   50.55    1.62  -62.50  170.73   33.48\n",
      "  381        583A 583A N   CA   4534 4535 -175.34  147.00                  -40.90\n",
      "  382  20302 583A 584Q C   N    4537 4539   91.35 -121.10  149.77    5.13  -63.80 -141.18   28.56\n",
      "  382        584Q 584Q N   CA   4539 4540  165.38  139.70                  -40.30\n",
      "  383  20303 584Q 585E C   N    4546 4548  174.26 -117.80   76.79    2.24  -63.60 -168.80   31.95\n",
      "  383        585E 585E N   CA   4548 4549  172.59  136.80                  -40.30\n",
      "  384  20304 585E 586D C   N    4555 4557  -90.46  -70.90   31.25    1.14  -63.30  147.85   19.98\n",
      "  384        586D 586D N   CA   4557 4558  174.67  150.30                  -40.00\n",
      "  385  20305 586D 587L C   N    4563 4565 -153.82 -108.50   64.92    3.65  -63.50  156.02   19.30\n",
      "  385        587L 587L N   CA   4565 4566   86.02  132.50                  -41.20\n",
      "  386  20306 587L 588P C   N    4571 4573  155.51  -64.50  142.52   13.37  -64.50  142.52   13.37\n",
      "  386        588P 588P N   CA   4573 4574  173.93  147.20                  147.20\n",
      "  387  20307 588P 589H C   N    4578 4580  178.70 -125.60   66.09    1.66  -63.20 -174.28   30.57\n",
      "  387        589H 589H N   CA   4580 4581  174.37  138.80                  -42.30\n",
      "  388  20308 589H 590K C   N    4588 4590  109.72 -118.00  162.35    5.46  -62.90 -167.16   26.73\n",
      "  388        590K 590K N   CA   4590 4591 -126.77  139.10                  -40.80\n",
      "  389  20310 591P 592T C   N    4604 4606    3.44  -78.10   85.88    5.65  -63.20  156.10   16.69\n",
      "  389        592T 592T N   CA   4606 4607  176.73  149.80                  -42.10\n",
      "  390  20311 592T 593S C   N    4611 4613  169.18 -136.60   73.51    4.07  -64.10 -173.70   14.61\n",
      "  390        593S 593S N   CA   4613 4614  101.56  151.20                  -35.00\n",
      "  391  20312 593S 594S C   N    4617 4619  113.79 -136.60  135.67    7.31  -64.10 -152.79   27.24\n",
      "  391        594S 594S N   CA   4619 4620   71.25  151.20                  -35.00\n",
      "  392  20313 594S 595S C   N    4623 4625  175.93  -64.10  119.97   15.43  -64.10  119.97   15.43\n",
      "  392        595S 595S N   CA   4625 4626  -36.25  -35.00                  -35.00\n",
      "  393  20314 595S 596M C   N    4629 4631 -161.67 -125.60   46.16    1.37  -63.40  179.49   34.30\n",
      "  393        596M 596M N   CA   4631 4632  169.30  140.50                  -40.50\n",
      "  394  20315 596M 597K C   N    4637 4639  -57.36  -70.20   21.66    1.80  -62.90  161.44   20.67\n",
      "  394        597K 597K N   CA   4639 4640  157.85  140.40                  -40.80\n",
      "  395  20316 597K 598T C   N    4646 4648  150.79 -124.80   84.41    4.48  -63.20 -132.04   38.12\n",
      "  395        598T 598T N   CA   4648 4649  142.83  143.50                  -42.10\n",
      "  396  20318 599V 600D C   N    4660 4662 -174.79  -96.50   83.40    3.44  -63.30 -150.76   32.93\n",
      "  396        600D 600D N   CA   4662 4663  142.94  114.20                  -40.00\n",
      "  397  20319 600D 601E C   N    4668 4670 -109.01 -117.80   42.87    2.28  -63.60  148.08   22.31\n",
      "  397        601E 601E N   CA   4670 4671  178.76  136.80                  -40.30\n",
      "  398  20320 601E 602E C   N    4677 4679    4.86  -69.30   89.47    8.12  -63.60  144.41   16.71\n",
      "  398        602E 602E N   CA   4679 4680 -167.45  142.50                  -40.30\n",
      "  399  20321 602E 603A C   N    4686 4688   31.46  -68.20  100.91    9.06  -62.50 -176.21   27.54\n",
      "  399        603A 603A N   CA   4688 4689  161.15  145.30                  -40.90\n",
      "  400  20322 603A 604L C   N    4691 4693 -171.82 -108.50   83.18    4.63  -63.50  161.48   20.48\n",
      "  400        604L 604L N   CA   4693 4694   78.56  132.50                  -41.20\n",
      "  401  20323 604L 605A C   N    4699 4701 -146.05 -134.00   30.99    1.39  -62.50  166.09   31.68\n",
      "  401        605A 605A N   CA   4701 4702  175.55  147.00                  -40.90\n",
      "  402  20326 607L 608L C   N    4720 4722   96.30  -63.50  169.42   26.04  -63.50  169.42   26.04\n",
      "  402        608L 608L N   CA   4722 4723  -97.46  -41.20                  -41.20\n",
      "  403  20327 608L 609S C   N    4728 4730  -42.04  -72.40   30.50    2.09  -64.10  176.95   11.69\n",
      "  403        609S 609S N   CA   4730 4731  149.43  152.40                  -35.00\n",
      "  404  20328 609S 610M C   N    4734 4736  -79.79  -73.00   20.12    1.27  -63.40  158.41   24.95\n",
      "  404        610M 610M N   CA   4736 4737  161.93  143.00                  -40.50\n",
      "  405  20329 610M 611G C   N    4742 4744  160.77 -167.20   53.24    2.09   82.20  146.43   10.55\n",
      "  405        611G 611G N   CA   4744 4745  132.07  174.60                    8.50\n",
      "  406  20331 612I 613E C   N    4754 4756 -154.28 -117.80   61.33    2.16  -63.60  161.46   26.46\n",
      "  406        613E 613E N   CA   4756 4757 -173.89  136.80                  -40.30\n",
      "  407  20332 613E 614E C   N    4763 4765 -154.09 -117.80   63.63    2.29  -63.60  158.91   26.09\n",
      "  407        614E 614E N   CA   4765 4766 -170.93  136.80                  -40.30\n",
      "  408  20333 614E 615T C   N    4772 4774  166.36 -124.80   83.49    3.01  -63.20 -177.83   30.97\n",
      "  408        615T 615T N   CA   4774 4775 -169.27  143.50                  -42.10\n",
      "  409  20335 616V 617A C   N    4786 4788 -120.11 -134.00   29.32    1.81  -62.50  157.22   28.85\n",
      "  409        617A 617A N   CA   4788 4789  172.82  147.00                  -40.90\n",
      "  410  20336 617A 618R C   N    4791 4793 -151.18  -63.00   95.68   17.33  -63.00   95.68   17.33\n",
      "  410        618R 618R N   CA   4793 4794  -78.22  -41.10                  -41.10\n",
      "  411  20337 618R 619Y C   N    4802 4804 -154.45 -124.30   30.72    2.46  -63.50 -164.62   27.26\n",
      "  411        619Y 619Y N   CA   4804 4805  129.52  135.40                  -43.40\n",
      "  412  20338 619Y 620A C   N    4814 4816  164.60 -134.00   63.32    2.70  -62.50 -142.28   32.93\n",
      "  412        620A 620A N   CA   4816 4817  131.55  147.00                  -40.90\n",
      "  413  20339 620A 621L C   N    4819 4821 -112.81 -108.50   44.11    2.34  -63.50  150.69   23.97\n",
      "  413        621L 621L N   CA   4821 4822  176.40  132.50                  -41.20\n",
      "  414  20340 621L 622L C   N    4827 4829 -130.66 -108.50   57.40    2.83  -63.50  149.31   24.88\n",
      "  414        622L 622L N   CA   4829 4830 -174.55  132.50                  -41.20\n",
      "  415  20341 622L 623Q C   N    4835 4837 -173.44 -121.10   59.90    1.85  -63.80 -173.50   32.87\n",
      "  415        623Q 623Q N   CA   4837 4838  168.83  139.70                  -40.30\n",
      "  416  20342 623Q 624T C   N    4844 4846 -121.88 -124.80   45.25    2.03  -63.20  141.95   22.04\n",
      "  416        624T 624T N   CA   4846 4847 -171.35  143.50                  -42.10\n",
      "  417  20343 624T 625G C   N    4851 4853  175.44 -167.20   23.95    0.89   82.20  176.28   12.66\n",
      "  417        625G 625G N   CA   4853 4854  158.10  174.60                    8.50\n",
      "  418  20344 625G 626M C   N    4855 4857 -171.96 -125.60   53.17    1.59  -63.40 -172.42   36.21\n",
      "  418        626M 626M N   CA   4857 4858  166.52  140.50                  -40.50\n",
      "  419  20345 626M 627N C   N    4863 4865 -169.22 -119.90   63.22    1.83  -63.20  177.50   27.94\n",
      "  419        627N 627N N   CA   4865 4866  176.55  137.00                  -41.10\n",
      "  420  20346 627N 628A C   N    4871 4873 -179.41 -134.00   49.48    1.13  -62.50 -167.87   37.61\n",
      "  420        628A 628A N   CA   4873 4874  166.64  147.00                  -40.90\n",
      "  421  20347 628A 629E C   N    4876 4878 -171.48 -117.80   66.54    1.99  -63.60  179.60   29.73\n",
      "  421        629E 629E N   CA   4878 4879  176.11  136.80                  -40.30\n",
      "  422  20348 629E 630R C   N    4885 4887 -160.45 -125.20   46.25    1.38  -63.00  177.51   30.45\n",
      "  422        630R 630R N   CA   4887 4888  170.54  140.60                  -41.10\n",
      "  423  20349 630R 631A C   N    4896 4898 -174.13 -134.00   48.58    1.19  -62.50 -177.23   35.80\n",
      "  423        631A 631A N   CA   4898 4899  174.38  147.00                  -40.90\n",
      "  424  20351 632V 633D C   N    4908 4910 -162.97  -96.50   89.60    3.74  -63.30  176.54   28.10\n",
      "  424        633D 633D N   CA   4910 4911  174.29  114.20                  -40.00\n",
      "  425  20352 633D 634Y C   N    4916 4918 -142.73 -124.30   28.29    1.19  -63.50  178.31   32.17\n",
      "  425        634Y 634Y N   CA   4918 4919  156.86  135.40                  -43.40\n",
      "  426  20354 635V 636F C   N    4935 4937 -146.11 -124.20   27.57    0.71  -63.20  176.37   29.78\n",
      "  426        636F 636F N   CA   4937 4938  160.03  143.30                  -44.30\n",
      "  427  20355 636F 637S C   N    4946 4948 -169.77 -136.60   40.66    1.24  -64.10 -176.29   20.94\n",
      "  427        637S 637S N   CA   4948 4949  174.72  151.20                  -35.00\n",
      "  428  20356 637S 638H C   N    4952 4954 -156.46 -125.60   34.70    0.91  -63.20 -172.18   29.50\n",
      "  428        638H 638H N   CA   4954 4955  154.67  138.80                  -42.30\n",
      "  429  20357 638H 639E C   N    4962 4964 -174.19 -117.80   67.99    2.01  -63.60 -177.70   30.23\n",
      "  429        639E 639E N   CA   4964 4965  174.78  136.80                  -40.30\n",
      "  430  20358 639E 640N C   N    4971 4973 -140.21 -119.90   31.33    0.94  -63.20  175.80   26.32\n",
      "  430        640N 640N N   CA   4973 4974  160.86  137.00                  -41.10\n",
      "  431  20360 641I 642A C   N    4987 4989 -148.38 -134.00   23.63    0.83  -62.50  175.76   33.39\n",
      "  431        642A 642A N   CA   4989 4990  165.75  147.00                  -40.90\n",
      "  432  20361 642A 643E C   N    4992 4994 -135.58 -117.80   37.31    1.45  -63.60  166.47   26.22\n",
      "  432        643E 643E N   CA   4994 4995  169.60  136.80                  -40.30\n",
      "  433  20362 643E 644E C   N    5001 5003 -142.28 -117.80   44.36    1.62  -63.60  165.77   26.47\n",
      "  433        644E 644E N   CA   5003 5004  173.79  136.80                  -40.30\n",
      "  434  20363 644E 645A C   N    5010 5012 -142.80 -134.00   28.59    1.37  -62.50  165.67   31.44\n",
      "  434        645A 645A N   CA   5012 5013  174.20  147.00                  -40.90\n",
      "  435  20364 645A 646G C   N    5015 5017 -147.09 -167.20   20.22    0.34   82.20 -150.28   15.67\n",
      "  435        646G 646G N   CA   5017 5018  172.50  174.60                    8.50\n",
      "  436  20365 646G 647L C   N    5019 5021 -126.59 -108.50   42.35    2.06  -63.50  160.88   26.25\n",
      "  436        647L 647L N   CA   5021 5022  170.80  132.50                  -41.20\n",
      "  437  20366 647L 648A C   N    5027 5029 -139.22 -134.00   14.19    0.65  -62.50  176.46   33.03\n",
      "  437        648A 648A N   CA   5029 5030  160.19  147.00                  -40.90\n",
      "  438  20367 648A 649E C   N    5032 5034 -132.10 -117.80   25.26    0.91  -63.60  175.96   27.32\n",
      "  438        649E 649E N   CA   5034 5035  157.62  136.80                  -40.30\n",
      "  439  20369 650I 651S C   N    5049 5051 -148.38 -136.60   18.45    0.63  -64.10 -179.50   19.13\n",
      "  439        651S 651S N   CA   5051 5052  165.39  151.20                  -35.00\n",
      "  440  20370 651S 652A C   N    5055 5057 -158.01 -134.00   27.15    0.62  -62.50 -174.15   35.55\n",
      "  440        652A 652A N   CA   5057 5058  159.67  147.00                  -40.90\n",
      "  441  20371 652A 653T C   N    5060 5062 -143.01 -124.80   28.28    0.89  -63.20  172.36   27.27\n",
      "  441        653T 653T N   CA   5062 5063  165.13  143.50                  -42.10\n",
      "  442  20372 653T 654A C   N    5067 5069 -157.72 -134.00   27.48    0.64  -62.50 -175.33   35.33\n",
      "  442        654A 654A N   CA   5069 5070  160.88  147.00                  -40.90\n",
      "  443  20373 654A 655S C   N    5072 5074 -153.22 -136.60   18.64    0.57  -64.10 -172.15   20.02\n",
      "  443        655S 655S N   CA   5074 5075  159.63  151.20                  -35.00\n",
      "  444  20374 655S 656E C   N    5078 5080 -154.28 -117.80   43.88    1.30  -63.60 -177.39   29.36\n",
      "  444        656E 656E N   CA   5080 5081  161.19  136.80                  -40.30\n",
      "  445  20375 656E 657S C   N    5087 5089 -149.30 -136.60   14.63    0.45  -64.10 -172.95   19.66\n",
      "  445        657S 657S N   CA   5089 5090  158.47  151.20                  -35.00\n",
      "  446  20376 657S 658Q C   N    5093 5095 -145.47 -121.10   25.71    0.82  -63.80 -169.78   32.24\n",
      "  446        658Q 658Q N   CA   5095 5096  147.91  139.70                  -40.30\n",
      "  447  20378 659P 660A C   N    5109 5111 -146.53 -134.00   15.57    0.40  -62.50 -176.73   34.53\n",
      "  447        660A 660A N   CA   5111 5112  156.23  147.00                  -40.90\n",
      "  448  20379 660A 661H C   N    5114 5116 -142.62 -125.60   26.27    0.66  -63.20  177.64   27.37\n",
      "  448        661H 661H N   CA   5116 5117  158.81  138.80                  -42.30\n",
      "  449  20381 662V 663L C   N    5131 5133 -129.85 -108.50   27.18    1.21  -63.50 -177.99   29.38\n",
      "  449        663L 663L N   CA   5133 5134  149.32  132.50                  -41.20\n",
      "  450  20382 663L 664D C   N    5139 5141 -122.70  -96.50   51.24    2.16  -63.30  172.33   25.11\n",
      "  450        664D 664D N   CA   5141 5142  158.23  114.20                  -40.00\n",
      "  451  20383 664D 665G C   N    5147 5149 -159.32 -167.20   41.67    1.52   82.20  172.36   13.15\n",
      "  451        665G 665G N   CA   5149 5150  133.68  174.60                    8.50\n",
      "  452  20385 666P 667A C   N    5158 5160 -152.07 -134.00   19.10    0.45  -62.50 -171.46   35.69\n",
      "  452        667A 667A N   CA   5160 5161  153.20  147.00                  -40.90\n",
      "  453  20386 667A 668K C   N    5163 5165 -145.57 -118.00   27.80    0.99  -62.90 -165.08   30.26\n",
      "  453        668K 668K N   CA   5165 5166  142.68  139.10                  -40.80\n",
      "  454  20387 668K 669Y C   N    5172 5174 -139.41 -124.30   20.59    0.94  -63.50 -176.36   32.89\n",
      "  454        669Y 669Y N   CA   5174 5175  149.38  135.40                  -43.40\n",
      "  455  20388 669Y 670R C   N    5184 5186 -136.59 -125.20   11.49    0.42  -63.00 -168.53   30.91\n",
      "  455        670R 670R N   CA   5186 5187  142.14  140.60                  -41.10\n",
      "  456  20389 670R 671L C   N    5195 5197 -144.14 -108.50   36.13    1.66  -63.50 -163.13   24.19\n",
      "  456        671L 671L N   CA   5197 5198  138.39  132.50                  -41.20\n",
      "  457  20390 671L 672H C   N    5203 5205 -146.22 -125.60   23.75    0.61  -63.20 -173.40   28.73\n",
      "  457        672H 672H N   CA   5205 5206  150.59  138.80                  -42.30\n",
      "  458  20391 672H 673A C   N    5213 5215 -162.44 -134.00   30.78    0.70  -62.50 -171.06   36.28\n",
      "  458        673A 673A N   CA   5215 5216  158.76  147.00                  -40.90\n",
      "  459  20392 673A 674M C   N    5218 5220 -150.12 -125.60   26.46    0.83  -63.40 -170.01   35.14\n",
      "  459        674M 674M N   CA   5220 5221  150.45  140.50                  -40.50\n",
      "  460  20394 675I 676S C   N    5234 5236 -150.23 -136.60   15.27    0.47  -64.10 -172.16   19.79\n",
      "  460        676S 676S N   CA   5236 5237  158.07  151.20                  -35.00\n",
      "  461  20395 676S 677H C   N    5240 5242 -154.40 -125.60   34.32    0.86  -63.20 -175.63   28.94\n",
      "  461        677H 677H N   CA   5242 5243  157.47  138.80                  -42.30\n",
      "  462  20397 678V 679G C   N    5257 5259 -178.91 -167.20   22.78    0.92   82.20  176.80   12.87\n",
      "  462        679G 679G N   CA   5259 5260  155.06  174.60                    8.50\n",
      "  463  20398 679G 680A C   N    5261 5263 -165.23 -134.00   33.69    0.77  -62.50 -170.30   36.54\n",
      "  463        680A 680A N   CA   5263 5264  159.63  147.00                  -40.90\n",
      "  464  20399 680A 681S C   N    5266 5268 -156.20 -136.60   21.11    0.67  -64.10 -170.21   20.38\n",
      "  464        681S 681S N   CA   5268 5269  159.05  151.20                  -35.00\n",
      "  465  20400 681S 682A C   N    5272 5274 -157.08 -134.00   24.51    0.57  -62.50 -170.80   36.05\n",
      "  465        682A 682A N   CA   5274 5275  155.24  147.00                  -40.90\n",
      "  466  20401 682A 683K C   N    5277 5279 -147.71 -118.00   32.69    1.08  -62.90 -173.19   29.33\n",
      "  466        683K 683K N   CA   5279 5280  152.75  139.10                  -40.80\n",
      "  467  20402 683K 684T C   N    5286 5288 -145.98 -124.80   31.47    1.00  -63.20  172.31   27.43\n",
      "  467        684T 684T N   CA   5288 5289  166.78  143.50                  -42.10\n",
      "  468  20403 684T 685G C   N    5293 5295  177.85 -167.20   27.02    1.08   82.20  172.53   12.53\n",
      "  468        685G 685G N   CA   5295 5296  152.09  174.60                    8.50\n",
      "  469  20404 685G 686H C   N    5297 5299 -156.61 -125.60   31.99    0.96  -63.20 -165.13   30.41\n",
      "  469        686H 686H N   CA   5299 5300  146.68  138.80                  -42.30\n",
      "  470  20405 686H 687Y C   N    5307 5309 -141.83 -124.30   25.64    1.11  -63.50 -179.61   32.46\n",
      "  470        687Y 687Y N   CA   5309 5310  154.11  135.40                  -43.40\n",
      "  471  20408 689C 690H C   N    5332 5334 -131.10 -125.60  102.28    3.99  -63.20  104.15   11.42\n",
      "  471        690H 690H N   CA   5334 5335   36.67  138.80                  -42.30\n",
      "  472  20411 692C 693D C   N    5356 5358 -170.41  -96.50  100.75    4.21  -63.30  174.16   28.20\n",
      "  472        693D 693D N   CA   5358 5359 -177.32  114.20                  -40.00\n",
      "  473  20412 693D 694A C   N    5364 5366 -175.94 -134.00   44.19    1.04  -62.50 -165.35   37.88\n",
      "  473        694A 694A N   CA   5366 5367  160.93  147.00                  -40.90\n",
      "  474  20413 694A 695Q C   N    5369 5371 -177.50 -121.10   59.95    1.91  -63.80 -163.98   34.49\n",
      "  474        695Q 695Q N   CA   5371 5372  160.03  139.70                  -40.30\n",
      "  475  20414 695Q 696T C   N    5378 5380 -156.94 -124.80   49.12    1.55  -63.20  166.22   27.20\n",
      "  475        696T 696T N   CA   5380 5381 -179.36  143.50                  -42.10\n",
      "  476  20415 696T 697G C   N    5385 5387 -172.03 -167.20    7.17    0.17   82.20 -158.60   14.44\n",
      "  476        697G 697G N   CA   5387 5388  179.90  174.60                    8.50\n",
      "  477  20416 697G 698K C   N    5389 5391  172.67 -118.00   69.48    2.55  -62.90 -144.72   35.30\n",
      "  477        698K 698K N   CA   5391 5392  143.53  139.10                  -40.80\n",
      "  478  20417 698K 699W C   N    5398 5400 -163.45 -124.90   44.18    1.45  -63.00 -178.79   29.38\n",
      "  478        699W 699W N   CA   5400 5401  164.98  143.40                  -44.20\n",
      "  479  20418 699W 700L C   N    5412 5414 -159.54 -108.50   69.08    3.09  -63.50  169.57   29.39\n",
      "  479        700L 700L N   CA   5414 5415  179.05  132.50                  -41.20\n",
      "  480  20419 700L 701L C   N    5420 5422 -160.72 -108.50   63.66    2.81  -63.50  178.65   30.75\n",
      "  480        701L 701L N   CA   5422 5423  168.92  132.50                  -41.20\n",
      "  481  20420 701L 702F C   N    5428 5430 -172.48 -124.20   51.32    1.51  -63.20 -170.36   33.01\n",
      "  481        702F 702F N   CA   5430 5431  160.71  143.30                  -44.30\n",
      "  482  20421 702F 703N C   N    5439 5441  170.78 -119.90   75.62    2.41  -63.20 -162.80   31.38\n",
      "  482        703N 703N N   CA   5441 5442  167.22  137.00                  -41.10\n",
      "  483  20422 703N 704D C   N    5447 5449  163.05  -96.50  115.91    4.81  -63.30 -160.61   32.83\n",
      "  483        704D 704D N   CA   5449 5450  172.04  114.20                  -40.00\n",
      "  484  20423 704D 705E C   N    5455 5457  179.96 -117.80   69.98    2.05  -63.60 -169.38   31.64\n",
      "  484        705E 705E N   CA   5457 5458  168.78  136.80                  -40.30\n",
      "  485  20424 705E 706K C   N    5464 5466 -168.87 -118.00   62.89    2.12  -62.90  178.07   29.36\n",
      "  485        706K 706K N   CA   5466 5467  176.09  139.10                  -40.80\n",
      "  486  20426 707V 708A C   N    5480 5482  169.17 -134.00   67.33    1.61  -62.50 -173.01   37.21\n",
      "  486        708A 708A N   CA   5482 5483 -176.90  147.00                  -40.90\n",
      "  487  20427 708A 709E C   N    5485 5487 -178.32 -117.80   69.29    2.03  -63.60 -171.81   31.23\n",
      "  487        709E 709E N   CA   5487 5488  170.53  136.80                  -40.30\n",
      "  488  20428 709E 710S C   N    5494 5496 -169.14 -136.60   43.58    1.37  -64.10  178.90   20.54\n",
      "  488        710S 710S N   CA   5496 5497 -179.82  151.20                  -35.00\n",
      "  489  20429 710S 711L C   N    5500 5502  169.54 -108.50   82.27    3.84  -63.50 -140.34   38.21\n",
      "  489        711L 711L N   CA   5502 5503  139.55  132.50                  -41.20\n",
      "  490  20430 711L 712N C   N    5508 5510 -177.95 -119.90   62.44    2.03  -63.20 -163.97   30.76\n",
      "  490        712N 712N N   CA   5510 5511  159.97  137.00                  -41.10\n",
      "  491  20431 712N 713P C   N    5516 5518 -145.55  -64.50   88.40    7.27  -64.50   88.40    7.27\n",
      "  491        713P 713P N   CA   5518 5519 -177.51  147.20                  147.20\n",
      "  492  20432 713P 714P C   N    5523 5525  170.05  -64.50  127.42   12.03  -64.50  127.42   12.03\n",
      "  492        714P 714P N   CA   5525 5526  169.51  147.20                  147.20\n",
      "  493  20433 714P 715F C   N    5530 5532  -97.04 -124.20   56.29    3.10  -63.20  127.66   19.99\n",
      "  493        715F 715F N   CA   5532 5533 -167.39  143.30                  -44.30\n",
      "  494  20434 715F 716S C   N    5541 5543  109.07 -136.60  115.58    4.16  -64.10 -126.36   19.73\n",
      "  494        716S 716S N   CA   5543 5544  168.16  151.20                  -35.00\n",
      "  495  20435 716S 717M C   N    5547 5549 -150.56 -125.60   57.03    2.11  -63.40  154.62   29.71\n",
      "  495        717M 717M N   CA   5549 5550 -168.22  140.50                  -40.50\n",
      "  496  20436 717M 718A C   N    5555 5557  142.00 -134.00   84.64    3.20  -62.50 -123.99   36.03\n",
      "  496        718A 718A N   CA   5557 5558  136.64  147.00                  -40.90\n",
      "  497  20437 718A 719F C   N    5560 5562  -52.63  -71.40   24.08    2.08  -63.20  160.27   21.55\n",
      "  497        719F 719F N   CA   5562 5563  155.77  140.70                  -44.30\n",
      "  498  20438 719F 720L C   N    5571 5573  167.88 -108.50   89.12    3.97  -63.50 -158.22   35.71\n",
      "  498        720L 720L N   CA   5573 5574  163.32  132.50                  -41.20\n",
      "  499  20439 720L 721Y C   N    5579 5581 -148.68 -124.30   48.88    1.92  -63.50  162.88   29.89\n",
      "  499        721Y 721Y N   CA   5581 5582  177.77  135.40                  -43.40\n",
      "  500  20440 721Y 722F C   N    5591 5593   40.70  -71.40  119.57    9.43  -63.20  169.09   19.12\n",
      "  500        722F 722F N   CA   5593 5594 -177.71  140.70                  -44.30\n",
      "  501  20441 722F 723Y C   N    5602 5604   81.94 -124.30  173.56    9.78  -63.50  176.90   25.43\n",
      "  501        723Y 723Y N   CA   5604 5605 -144.11  135.40                  -43.40\n",
      "  502  20442 723Y 724K C   N    5614 5616 -121.68  -62.90   92.12   15.38  -62.90   92.12   15.38\n",
      "  502        724K 724K N   CA   5616 5617 -111.73  -40.80                  -40.80\n",
      "  503  20443 724K 725R C   N    5623 5625 -149.30 -125.20   59.64    2.27  -63.00  150.87   26.07\n",
      "  503        725R 725R N   CA   5625 5626 -164.85  140.60                  -41.10\n",
      "  504  20445 726V 727G C   N    5641 5643  -93.34  -80.20   15.49    0.67   82.20 -112.97    7.77\n",
      "  504        727G 727G N   CA   5643 5644 -177.69  174.10                    8.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  28118 233P 607L CB  CD2  1763 4719   12.96    4.45    8.51    5.36    4.45    8.51    5.36\n",
      "    2  28184 252F 606M CB  SD   1910 4710   25.74    4.79   20.95   16.58    4.79   20.95   16.58\n",
      "    3  28185 253A 606M CB  CG   1921 4709   21.75    4.50   17.25   11.42    4.50   17.25   11.42\n",
      "    4  28186 253A 606M CB  SD   1921 4710   21.74    3.87   17.87   12.14    3.87   17.87   12.14\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   58  101  311  421  475  436  436  550  656  649\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00192.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      728     5656     5656\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00192.B99990001.pdb      376139.37500   -17505.79492        0.40234\n",
      "EAO00192.B99990002.pdb      373309.87500   -18096.42773        0.41025\n",
      "EAO00192.B99990003.pdb      382246.93750   -18842.24609        0.59032\n",
      "EAO00192.B99990004.pdb      375530.37500   -17959.56445        0.45047\n",
      "EAO00192.B99990005.pdb      389528.93750   -18607.23242        0.32351\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18421258   17989.510    17.568\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18421786   17990.025    17.568\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18470386   18037.486    17.615\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18483686   18050.475    17.627\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     18752952   18313.430    17.884\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     22314075   21791.090    21.280\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22316317   21793.277    21.282\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22317767   21794.695    21.284\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22320667   21797.527    21.287\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     22326467   21803.191    21.292\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     35764866   34926.625    34.108\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     35776466   34937.953    34.119\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     35799666   34960.609    34.141\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     35846066   35005.922    34.185\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     35938866   35096.547    34.274\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     42658066   41658.266    40.682\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00193.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18755272   18315.695    17.886\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18756722   18317.111    17.888\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18759622   18319.943    17.891\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18765422   18325.607    17.896\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18766486   18326.646    17.897\n",
      "\n",
      "Read the alignment from file       : ali/EAO00193.ali\n",
      "\n",
      "Total number of alignment positions:   267\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00193     267      1    EAO00193\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     18767887   18328.016    17.898\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19093062   18645.568    18.209\n",
      "profile_iteration_> processing sequence:       1    267      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19094140   18646.621    18.210\n",
      "profile_iteration_> processing sequence:    6720    267      1     2.5781250     0.0003836     0.0000015  2607\n",
      "profile_iteration_> processing sequence:   13439    267      1     5.0781250     0.0003779     0.0000015  2646\n",
      "profile_iteration_> processing sequence:   20158    267      1     7.5625000     0.0003752     0.0000015  2666\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19094515   18646.988    18.210\n",
      "profile_iteration_> processing sequence:   26877    267      1    10.0781250     0.0003750     0.0000015  2667\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19094751   18647.219    18.210\n",
      "profile_iteration_> processing sequence:   33596    267      1    12.5937500     0.0003749     0.0000015  2668\n",
      "profile_iteration_> processing sequence:   40315    267      1    15.0781250     0.0003740     0.0000015  2674\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19097701   18650.098    18.213\n",
      "profile_iteration_> processing sequence:   47034    267      1    17.6093750     0.0003744     0.0000015  2671\n",
      "profile_iteration_> processing sequence:   53753    267      1    20.1875000     0.0003756     0.0000015  2663\n",
      "profile_iteration_> processing sequence:   60472    267      1    22.7187500     0.0003757     0.0000015  2662\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19101211   18653.527    18.216\n",
      "profile_iteration_> processing sequence:   67191    267      1    25.3281250     0.0003770     0.0000015  2653\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    48.80215\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    249.1817\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0242,      121,        0.0124,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00030   0.00001\n",
      "Z:         1  -2.45000   0.00074   0.00004\n",
      "Z:         1  -2.35000   0.00104   0.00016\n",
      "Z:         1  -2.25000   0.00149   0.00055\n",
      "Z:         1  -2.15000   0.00283   0.00163\n",
      "Z:         1  -2.05000   0.00625   0.00416\n",
      "Z:         1  -1.95000   0.01206   0.00933\n",
      "Z:         1  -1.85000   0.02813   0.01872\n",
      "Z:         1  -1.75000   0.04093   0.03400\n",
      "Z:         1  -1.65000   0.06355   0.05658\n",
      "Z:         1  -1.55000   0.08959   0.08721\n",
      "Z:         1  -1.45000   0.12621   0.12565\n",
      "Z:         1  -1.35000   0.16817   0.17059\n",
      "Z:         1  -1.25000   0.21401   0.21981\n",
      "Z:         1  -1.15000   0.26447   0.27052\n",
      "Z:         1  -1.05000   0.31358   0.31973\n",
      "Z:         1  -0.95000   0.35629   0.36469\n",
      "Z:         1  -0.85000   0.39543   0.40316\n",
      "Z:         1  -0.75000   0.41166   0.43360\n",
      "Z:         1  -0.65000   0.44693   0.45519\n",
      "Z:         1  -0.55000   0.45199   0.46779\n",
      "Z:         1  -0.45000   0.46821   0.47182\n",
      "Z:         1  -0.35000   0.42907   0.46811\n",
      "Z:         1  -0.25000   0.46895   0.45775\n",
      "Z:         1  -0.15000   0.40779   0.44196\n",
      "Z:         1  -0.05000   0.48280   0.42194\n",
      "Z:         1   0.05000   0.38263   0.39889\n",
      "Z:         1   0.15000   0.38249   0.37383\n",
      "Z:         1   0.25000   0.33308   0.34769\n",
      "Z:         1   0.35000   0.33456   0.32121\n",
      "Z:         1   0.45000   0.27950   0.29500\n",
      "Z:         1   0.55000   0.26759   0.26953\n",
      "Z:         1   0.65000   0.23440   0.24513\n",
      "Z:         1   0.75000   0.23753   0.22205\n",
      "Z:         1   0.85000   0.19824   0.20043\n",
      "Z:         1   0.95000   0.18633   0.18036\n",
      "Z:         1   1.05000   0.15642   0.16185\n",
      "Z:         1   1.15000   0.14719   0.14489\n",
      "Z:         1   1.25000   0.14020   0.12944\n",
      "Z:         1   1.35000   0.12457   0.11542\n",
      "Z:         1   1.45000   0.11088   0.10275\n",
      "Z:         1   1.55000   0.09287   0.09133\n",
      "Z:         1   1.65000   0.07605   0.08109\n",
      "Z:         1   1.75000   0.06920   0.07191\n",
      "Z:         1   1.85000   0.06727   0.06371\n",
      "Z:         1   1.95000   0.06028   0.05639\n",
      "Z:         1   2.05000   0.04807   0.04988\n",
      "Z:         1   2.15000   0.05164   0.04409\n",
      "Z:         1   2.25000   0.03736   0.03895\n",
      "Z:         1   2.35000   0.04137   0.03439\n",
      "Z:         1   2.45000   0.03036   0.03035\n",
      "Z:         1   2.55000   0.02619   0.02678\n",
      "Z:         1   2.65000   0.02218   0.02361\n",
      "Z:         1   2.75000   0.02173   0.02082\n",
      "Z:         1   2.85000   0.02247   0.01835\n",
      "Z:         1   2.95000   0.01444   0.01617\n",
      "Z:         1   3.05000   0.01578   0.01424\n",
      "Z:         1   3.15000   0.01399   0.01255\n",
      "Z:         1   3.25000   0.00997   0.01105\n",
      "Z:         1   3.35000   0.01131   0.00973\n",
      "Z:         1   3.45000   0.00640   0.00857\n",
      "Z:         1   3.55000   0.00685   0.00754\n",
      "Z:         1   3.65000   0.00625   0.00664\n",
      "Z:         1   3.75000   0.00595   0.00584\n",
      "Z:         1   3.85000   0.00521   0.00514\n",
      "Z:         1   3.95000   0.00506   0.00453\n",
      "Z:         1   4.05000   0.00446   0.00398\n",
      "Z:         1   4.15000   0.00432   0.00350\n",
      "Z:         1   4.25000   0.00387   0.00308\n",
      "Z:         1   4.35000   0.00402   0.00271\n",
      "Z:         1   4.45000   0.00208   0.00239\n",
      "Z:         1   4.55000   0.00372   0.00210\n",
      "Z:         1   4.65000   0.00119   0.00185\n",
      "Z:         1   4.75000   0.00149   0.00163\n",
      "Z:         1   4.85000   0.00149   0.00143\n",
      "Z:         1   4.95000   0.00179   0.00126\n",
      "Z:         1   5.05000   0.00119   0.00111\n",
      "Z:         1   5.15000   0.00089   0.00097\n",
      "Z:         1   5.25000   0.00060   0.00086\n",
      "Z:         1   5.35000   0.00074   0.00075\n",
      "Z:         1   5.45000   0.00104   0.00066\n",
      "Z:         1   5.55000   0.00015   0.00058\n",
      "Z:         1   5.65000   0.00074   0.00051\n",
      "Z:         1   5.75000   0.00060   0.00045\n",
      "Z:         1   5.85000   0.00030   0.00040\n",
      "Z:         1   5.95000   0.00015   0.00035\n",
      "Z:         1   6.05000   0.00074   0.00031\n",
      "Z:         1   6.15000   0.00030   0.00027\n",
      "Z:         1   6.25000   0.00089   0.00024\n",
      "Z:         1   6.35000   0.00030   0.00021\n",
      "Z:         1   6.45000   0.00015   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00015   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00015   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00015   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     19102837   18655.113    18.218\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     19333437   18880.309    18.438\n",
      "> 1wz8A                      1     596    9750     263     267   27.85     0.0           2   230    31   264    26   262\n",
      "> 1ef8C                      1    1070   12050     250     267   31.20     0.0           3   230    24   264    14   247\n",
      "> 3r9qA                      1    1264   16300     233     267   38.71     0.0           4   216    14   236     2   218\n",
      "> 3r9tA                      1    1266   19750     260     267   37.90     0.0           5   243    20   262     8   255\n",
      "> 2iexA                      1    1450   17350     253     267   38.30     0.0           6   229    18   264    12   246\n",
      "> 4b3hA                      1    1594   11500     729     267   32.32     0.0           7   182    24   209    24   221\n",
      "> 2a7kA                      1    1929    8350     229     267   29.52    0.30E-10       8   206    24   231    10   219\n",
      "> 2a7kF                      1    1930    8700     226     267   29.38    0.75E-11       9   207    24   231    10   220\n",
      "> 1nzyA                      1    2548   10250     269     267   28.27     0.0          10   181    24   207    13   203\n",
      "> 3qxiA                      1    3190   20500     244     267   40.66     0.0          11   240    14   264     1   241\n",
      "> 3qxzA                      1    3201   14450     259     267   32.66     0.0          12   243    21   264     9   256\n",
      "> 1wdkA                      1    3581   12400     708     267   34.97     0.0          13   172    19   195    12   194\n",
      "> 3zwcA                      1    4301   13200     725     267   37.21     0.0          14   171    35   212    26   197\n",
      "> 5kjpA                      1    4842   20850     252     267   40.16     0.0          15   248    13   264     1   249\n",
      "> 6lvoA                      1    5370   14500     237     267   30.09     0.0          16   226    24   258    12   237\n",
      "> 6lvpA                      1    5371   24200     263     267   45.08     0.0          17   241    24   265    18   261\n",
      "> 4zu2A                      1    5863   11450     258     267   31.56     0.0          18   238    24   262    11   254\n",
      "> 3fduA                      1    6745   12900     248     267   31.05     0.0          19   238    11   257     1   248\n",
      "> 3fduB                      1    6746   12450     232     267   35.68     0.0          20   185    21   213    10   194\n",
      "> 3fduF                      1    6747   12100     215     267   37.43     0.0          21   179    21   213     7   185\n",
      "> 4f47A                      1    8751   17100     238     267   39.13     0.0          22   230    20   266     8   237\n",
      "> 3qkaA                      1    8859   15400     233     267   35.56     0.0          23   225    15   246     2   226\n",
      "> 3qkaC                      1    8860   15400     243     267   35.56     0.0          24   225    15   246     4   228\n",
      "> 3qk8A                      1    8876   12750     259     267   31.69     0.0          25   231    24   264    17   259\n",
      "> 2vx2A                      1    9191   15650     256     267   33.47     0.0          26   241    20   262     8   252\n",
      "> 5zaiA                      1    9307   28550     257     267   45.63     0.0          27   249    16   265     6   257\n",
      "> 5e0nX                      1   10184   13250     209     267   39.06     0.0          28   191    24   232    13   204\n",
      "> 3swxA                      1   10616   14700     257     267   31.60     0.0          29   247    16   264     5   254\n",
      "> 3hrxA                      1   12252   18250     254     267   35.46     0.0          30   250    15   265     2   252\n",
      "> 2vssA                      1   12873   12300     239     267   27.40     0.0          31   214    16   235     8   226\n",
      "> 4jotA                      1   13322   11200     255     267   33.97     0.0          32   198    20   225    22   230\n",
      "> 4di1A                      1   15118   14150     238     267   31.39     0.0          33   221    24   245    12   234\n",
      "> 3t3wA                      1   16255   14350     241     267   36.54     0.0          34   203    16   226     4   211\n",
      "> 3t3wB                      1   16256   14350     237     267   36.54     0.0          35   203    16   226     4   211\n",
      "> 4izbA                      1   18049   10700     245     267   31.94     0.0          36   187    24   212    18   208\n",
      "> 4izcB                      1   18050   10400     266     267   31.41     0.0          37   187    24   212    20   210\n",
      "> 3hinA                      1   18426   14050     258     267   33.05     0.0          38   235    22   260    16   254\n",
      "> 3i47A                      1   19708   13600     259     267   32.94     0.0          39   243    18   264     7   258\n",
      "> 4i42A                      1   19740   14750     285     267   31.43     0.0          40   236    24   264    34   278\n",
      "> 4i42D                      1   19741   14750     268     267   33.62     0.0          41   224    24   264    33   261\n",
      "> 4i4zA                      1   19758   15700     275     267   31.91     0.0          42   244    16   264    12   268\n",
      "> 2zqqA                      1   20194   18200     266     267   36.55     0.0          43   242    24   265    16   264\n",
      "> 4kpkA                      1   20244   11550     265     267   27.24     0.0          44   234    22   264    20   265\n",
      "> 3p5mA                      1   20445   15700     250     267   31.22     0.0          45   237    22   265    12   248\n",
      "> 6p5uA                      1   20463   12250     239     267   28.91     0.0          46   208    16   223     5   215\n",
      "> 4mi2A                      1   20818   13000     260     267   29.96     0.0          47   237    24   264    11   257\n",
      "> 1q52A                      1   21176   12700     269     267   32.48     0.0          48   226    23   264    29   262\n",
      "> 1q52B                      1   21177   13050     279     267   31.56     0.0          49   236    23   264    29   272\n",
      "> 4jjtA                      1   21970   16100     229     267   39.60     0.0          50   202    24   236    11   212\n",
      "> 3kqfA                      1   22947   22450     259     267   41.63     0.0          51   241    24   265    13   257\n",
      "> 3rrvA                      1   23526   13150     253     267   30.83     0.0          52   236    22   259    13   252\n",
      "> 4u19A                      1   26687    9800     255     267   28.70     0.0          53   221    24   248    21   250\n",
      "> 4u19C                      1   26688    9800     242     267   28.32     0.0          54   223    24   248    13   238\n",
      "> 3ot6A                      1   27073    7550     230     267   30.28    0.18E-08      55   209    11   229     1   218\n",
      "> 3isaA                      1   27254    8700     240     267   28.42    0.75E-11      56   186    20   211    10   199\n",
      "> 1xx4A                      1   27351    7750     254     267   23.17    0.72E-09      57   240    20   261     9   254\n",
      "> 3r0oA                      1   27552   20350     262     267   39.04     0.0          58   246    20   265    10   260\n",
      "> 1hnuA                      1   27698    6650     248     267   27.33    0.20E-06      59   171    16   188     4   175\n",
      "> 7boqD                      1   27936   12500     247     267   27.76     0.0          60   244    15   259     3   247\n",
      "> 6dv2G                      1   28029   15000     725     267   32.48     0.0          61   226    16   243     4   237\n",
      "> 6dv2I                      1   28030   15000     714     267   32.48     0.0          62   226    16   243     4   237\n",
      "> 6n92A                      1   28137   12300     260     267   30.61     0.0          63   240    24   264    13   257\n",
      "> 4emlA                      1   28271   15250     261     267   33.33     0.0          64   233    16   264    12   254\n",
      "> 2np9A                      1   28512    9900     423     267   32.04     0.0          65   179    16   195   157   362\n",
      "> 5dufA                      1   28630   15850     245     267   32.51     0.0          66   243    15   267     3   245\n",
      "> 3ju1A                      1   29425    8550     361     267   28.43    0.15E-10      67   182    24   211    18   214\n",
      "> 3pe8A                      1   30834   15450     219     267   39.00     0.0          68   198    14   227     2   201\n",
      "> 3peaA                      1   30837   19900     257     267   38.27     0.0          69   240    24   264    13   255\n",
      "> 5fusA                      1   31472    6500     269     267   40.51    0.48E-06      70    79   108   186   117   195\n",
      "> 3t89C                      1   31975   12950     246     267   33.86     0.0          71   187    24   222    32   220\n",
      "> 3t8aA                      1   31977   11400     253     267   31.70     0.0          72   215    23   252    30   253\n",
      "> 3t8bA                      1   31978   11100     237     267   32.50     0.0          73   192    23   222    31   230\n",
      "> 3t8bB                      1   31979    8600     204     267   35.00    0.75E-11      74   152    23   195    30   189\n",
      "> 4q1iA                      1   32239    8350     215     267   26.74    0.30E-10      75   172    21   207    17   188\n",
      "> 4q1kA                      1   32256    9700     236     267   25.56     0.0          76   220    21   244    13   235\n",
      "> 4q1kB                      1   32257    9850     247     267   25.33     0.0          77   226    21   250    12   240\n",
      "> 3q1tA                      1   32260   11200     253     267   30.34     0.0          78   219    24   255    17   250\n",
      "> 2hw5A                      1   33001   33150     260     267   53.20     0.0          79   250    17   266    10   259\n",
      "> 4qfeA                      1   33143   14000     225     267   37.61     0.0          80   217    16   246     7   224\n",
      "> 4elwF                      1   33979   13400     259     267   32.88     0.0          81   217    24   264    31   252\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 4fn7A                      1   34369    6550     231     267   28.87    0.31E-06      82   189    13   207     3   196\n",
      "> 2gtrA                      1   34443   10500     258     267   26.29     0.0          83   245    15   260     5   255\n",
      "> 2w3pA                      1   34602    7300     547     267   28.57    0.19E-07      84   198    57   259    66   275\n",
      "> 4qiiA                      1   35877   12850     300     267   29.77     0.0          85   241    23   264    32   293\n",
      "> 4k2nA                      1   36227   17800     267     267   37.50     0.0          86   250    11   264     9   264\n",
      "> 4k29A                      1   36267   14700     269     267   37.16     0.0          87   211    16   233     5   222\n",
      "> 3mybA                      1   36353   17950     259     267   34.57     0.0          88   240    24   264    13   255\n",
      "> 4jvtA                      1   36447   11750     257     267   32.16     0.0          89   211    22   241    21   247\n",
      "> 5xzdA                      1   36676   26400     233     267   48.65     0.0          90   221    24   244    12   233\n",
      "> 5xzdB                      1   36677   24450     222     267   49.76     0.0          91   211    24   244    12   222\n",
      "> 5jbxA                      1   36701   20700     258     267   38.43     0.0          92   250    14   265     2   256\n",
      "> 4z0mA                      1   36951   15500     241     267   36.05     0.0          93   233    15   254     2   234\n",
      "> 4z0mC                      1   36952   15450     230     267   36.44     0.0          94   225    15   246     2   226\n",
      "> 3qreA                      1   38451   12700     198     267   36.17     0.0          95   187    13   218     2   189\n",
      "> 4jylA                      1   38533   15100     249     267   33.62     0.0          96   230    15   249     8   239\n",
      "> 4jyjA                      1   38558    8400     270     267   29.38    0.30E-10      97   204    16   226     3   213\n",
      "> 6ojmA                      1   38580   17050     266     267   34.41     0.0          98   239    16   264    13   259\n",
      "> 6ojmB                      1   38581   17750     278     267   34.36     0.0          99   248    16   264    13   271\n",
      "> 2q35A                      1   38944    9900     243     267   26.75     0.0         100   227    15   244     3   230\n",
      "> 3sllA                      1   39620   15600     262     267   34.94     0.0         101   242    24   266    12   260\n",
      "> 3oc7A                      1   40695   14750     251     267   34.00     0.0         102   243    12   264     1   250\n",
      "> 5yloA                      1   41639   15000     235     267   33.05     0.0         103   228     9   237     1   233\n",
      "> 2ej5A                      1   42109   20350     250     267   37.30     0.0         104   244    16   265     5   248\n",
      "> 1sg4A                      1   43139    7100     249     267   24.89    0.20E-07     105   219    22   253    11   247\n",
      "> 5z7rA                      1   43233   27550     254     267   46.12     0.0         106   242    20   261    10   254\n",
      "> 4jfcA                      1   43349   17550     257     267   33.33     0.0         107   250    16   265     3   257\n",
      "> 2vreA                      1   43436   13350     272     267   30.31     0.0         108   239    16   255    17   270\n",
      "> 2vreB                      1   43437   13300     260     267   30.77     0.0         109   232    16   248    12   258\n",
      "> 4jcsA                      1   43492   13500     265     267   28.51     0.0         110   239    27   265    18   259\n",
      "> 4fzwA                      1   43716   26700     254     267   46.69     0.0         111   242    24   265    11   252\n",
      "> 4fzwC                      1   43717   19250     249     267   38.78     0.0         112   245    15   264     2   246\n",
      "> 3m6nA                      1   45179    7650     264     267   36.19    0.13E-08     113   102    89   190    89   193\n",
      "> 3tlfA                      1   45528   17400     253     267   34.55     0.0         114   242    16   265     6   251\n",
      "> 3tlfD                      1   45529   18000     266     267   33.85     0.0         115   248    16   265     5   264\n",
      "> 4k3wA                      1   45950   12700     265     267   30.20     0.0         116   244    20   264     8   262\n",
      "> 1uiyA                      1   47237   17950     253     267   35.34     0.0         117   244    20   264     5   253\n",
      "> 3laoA                      1   47715   11000     223     267   30.81     0.0         118   207    10   217     4   214\n",
      "> 3laoB                      1   47716   10850     234     267   31.58     0.0         119   205    10   217     3   211\n",
      "> 3g64A                      1   48589   13300     275     267   30.52     0.0         120   241    24   265    25   273\n",
      "> 5c9gA                      1   49883   15900     273     267   33.88     0.0         121   237    27   264    29   270\n",
      "> 5c9gB                      1   49884   15900     261     267   33.88     0.0         122   237    27   264    17   258\n",
      "> 4og1A                      1   50081   10100     257     267   27.85     0.0         123   234    24   264    18   254\n",
      "> 5o34A                      1   50180    9300     209     267   34.12     0.0         124   169    31   222    22   191\n",
      "> 5o34C                      1   50181    8700     196     267   33.14    0.75E-11     125   171    23   222    11   182\n",
      "> 4hdtA                      1   50337   10550     340     267   31.11     0.0         126   169    21   195     9   188\n",
      "> 3moyA                      1   50756   30000     258     267   52.10     0.0         127   238    27   264    18   255\n",
      "> 4mouA                      1   50777   14850     261     267   30.92     0.0         128   244    16   264     8   256\n",
      "> 2ppyA                      1   50909   16400     264     267   37.85     0.0         129   210    56   265    49   262\n",
      "> 6eqoA                      1   51179   12700    1804     267   43.10     0.0         130   159    24   186   867  1040\n",
      "> 2uzfA                      1   51439   15950     260     267   35.66     0.0         131   238    16   264    10   253\n",
      "> 4jwvA                      1   51899   16850     254     267   35.80     0.0         132   241    16   262     7   249\n",
      "> 6iumB                      1   52117   13650     692     267   36.56     0.0         133   180    17   206     3   188\n",
      "> 3r6hA                      1   53544    5750     225     267   27.51    0.18E-04     134   184    16   207     3   191\n",
      "> 5dtpB                      1   53747   13900     232     267   32.90     0.0         135   231    15   267     2   232\n",
      "> 1mj3A                      1   54653   32900     258     267   54.03     0.0         136   248    17   266    10   257\n",
      "> 1szoA                      1   54750    9200     249     267   27.73     0.0         137   221    14   236     4   241\n",
      "> 3hp0A                      1   54792    6300     250     267   27.87    0.12E-05     138   169    16   196     5   187\n",
      "> 4wczA                      1   55386   14250     249     267   31.58     0.0         139   242    16   264     3   249\n",
      "> 6tnmA                      1   55503   12400     719     267   27.13     0.0         140   240    24   266    17   274\n",
      "> 4kd6A                      1   55583   12750     215     267   32.34     0.0         141   200    16   226     2   202\n",
      "> 3p85A                      1   55749   15050     224     267   42.13     0.0         142   178    24   214    11   188\n",
      "> 2wtbA                      1   56127   14550     676     267   39.04     0.0         143   186    23   218    13   199\n",
      "> 5ve2A                      1   56610   16700     251     267   35.00     0.0         144   240    24   264    11   250\n",
      "> 3bptA                      1   56722   12750     362     267   34.23     0.0         145   212    16   235     7   228\n",
      "> 3trrA                      1   56894   21900     252     267   43.93     0.0         146   238    20   262     9   247\n",
      "> 3h81A                      1   57288   32500     256     267   52.94     0.0         147   238    27   264    16   253\n",
      "> 3njdA                      1   57854   10650     290     267   26.41     0.0         148   214    18   237    11   241\n",
      "> 3pzkA                      1   58616   30800     244     267   54.82     0.0         149   228    27   264    16   243\n",
      "> 2j5iA                      1   58825   12650     248     267   26.87     0.0         150   220    16   235     9   235\n",
      "> 2j5gA                      1   58840    8000     249     267   25.28    0.19E-09     151   174    61   235    57   234\n",
      "> 4zdcA                      1   59581    6350     266     267   26.34    0.10E-05     152   172    16   188     8   193\n",
      "> 4zdfA                      1   59594    6750     256     267   27.37    0.12E-06     153   172    16   188     8   186\n",
      "> 4nnqA                      1   59685    8250     228     267   32.72    0.45E-10     154   156    16   190     2   163\n",
      "> 3l3sA                      1   60085   11200     231     267   33.65     0.0         155   205    24   234    10   220\n",
      "> 3omeA                      1   60216   15450     243     267   37.32     0.0         156   204    16   226     4   212\n",
      "> 5wydA                      1   61483   16250     262     267   33.33     0.0         157   243    22   265    17   262\n",
      "> 5wybB                      1   61515   13750     244     267   32.44     0.0         158   222    22   244    17   241\n",
      "> 3gkbA                      1   61826    9400     274     267   27.23     0.0         159   226    16   245     5   239\n",
      "> 2fw2A                      1   63095   10300     258     267   25.49     0.0         160   248    15   264     4   258\n",
      "> 1dciA                      1   63267   13150     275     267   28.96     0.0         161   244    16   260     7   265\n",
      "> 4j2uA                      1   64098   10400     344     267   28.81     0.0         162   223     5   231     1   236\n",
      "> 4lk5A                      1   64143   13150     240     267   32.30     0.0         163   226    24   255    15   240\n",
      "> 3lkeA                      1   64157   10150     252     267   26.16     0.0         164   229    27   255    15   251\n",
      "> 6ijkA                      1   64706   18200     269     267   38.43     0.0         165   238    16   264     5   246\n",
      "> 4nekA                      1   65271   16050     251     267   34.02     0.0         166   235    21   264     9   249\n",
      "> 3qmjA                      1   65880   13500     211     267   39.69     0.0         167   188    24   214    11   204\n",
      "> 2pbpA                      1   65931   29350     255     267   47.15     0.0         168   246    20   265     8   253\n",
      "> 3h0uA                      1   66195   11550     278     267   33.20     0.0         169   225    16   246     9   261\n",
      "> 3rsiA                      1   66516   19500     243     267   40.95     0.0         170   232    22   264     9   240\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    25.328125\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000377\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  2662.717041\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.171875\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000381\n",
      "profile_iteration__> Overall performance [seq/sec]:  2625.328369\n",
      "openf___224_> Open           buildprofile/EAO00193build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     19509281   19052.031    18.605\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19510345   19053.070    18.607\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19511261   19053.965    18.607\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19512177   19054.859    18.608\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19513037   19055.699    18.609\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19514005   19056.645    18.610\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19514917   19057.535    18.611\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19515641   19058.242    18.612\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19516461   19059.043    18.612\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19517285   19059.848    18.613\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19518005   19060.551    18.614\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19518961   19061.484    18.615\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19519929   19062.430    18.616\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19520613   19063.098    18.616\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19521293   19063.762    18.617\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19522281   19064.727    18.618\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19523181   19065.605    18.619\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19524141   19066.543    18.620\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19525089   19067.469    18.621\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19526037   19068.395    18.621\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19526773   19069.113    18.622\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19527485   19069.809    18.623\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19528401   19070.703    18.624\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19529297   19071.578    18.625\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19530193   19072.453    18.625\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19531113   19073.352    18.626\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19532073   19074.289    18.627\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19533065   19075.258    18.628\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19533825   19076.000    18.629\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19534809   19076.961    18.630\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19535805   19077.934    18.631\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19536657   19078.766    18.632\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19537445   19079.535    18.632\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19538325   19080.395    18.633\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19539133   19081.184    18.634\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19539941   19081.973    18.635\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19540685   19082.699    18.635\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19541429   19083.426    18.636\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19542365   19084.340    18.637\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19543333   19085.285    18.638\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19544273   19086.203    18.639\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19545165   19087.074    18.640\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19546137   19088.023    18.641\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19547101   19088.965    18.642\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19548033   19089.875    18.642\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19548977   19090.797    18.643\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19549805   19091.605    18.644\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19550749   19092.527    18.645\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19551649   19093.406    18.646\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19552589   19094.324    18.647\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19553393   19095.109    18.648\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19554353   19096.047    18.648\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19555293   19096.965    18.649\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19556173   19097.824    18.650\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19557061   19098.691    18.651\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19557893   19099.504    18.652\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19558633   19100.227    18.653\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19559589   19101.160    18.653\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19560569   19102.117    18.654\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19561249   19102.781    18.655\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19562221   19103.730    18.656\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19563121   19104.609    18.657\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19564021   19105.488    18.658\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19564977   19106.422    18.659\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19565905   19107.328    18.660\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19566617   19108.023    18.660\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19567585   19108.969    18.661\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19568309   19109.676    18.662\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19569097   19110.445    18.663\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19570053   19111.379    18.663\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19570365   19111.684    18.664\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19571109   19112.410    18.664\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19571965   19113.246    18.665\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19572729   19113.992    18.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19573333   19114.582    18.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19574017   19115.250    18.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19574893   19116.105    18.668\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19575793   19116.984    18.669\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19576665   19117.836    18.670\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19577661   19118.809    18.671\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19578525   19119.652    18.672\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19579389   19120.496    18.672\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19580141   19121.230    18.673\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19581117   19122.184    18.674\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19581905   19122.953    18.675\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19582865   19123.891    18.676\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19583861   19124.863    18.677\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19584701   19125.684    18.677\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19585657   19126.617    18.678\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19586497   19127.438    18.679\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19587377   19128.297    18.680\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19588217   19129.117    18.681\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19589213   19130.090    18.682\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19590141   19130.996    18.683\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19591037   19131.871    18.683\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19591781   19132.598    18.684\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19592697   19133.492    18.685\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19593509   19134.285    18.686\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19594461   19135.215    18.687\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19595449   19136.180    18.688\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19596353   19137.062    18.689\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19597317   19138.004    18.689\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19598285   19138.949    18.690\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19599193   19139.836    18.691\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19600165   19140.785    18.692\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19601037   19141.637    18.693\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19602001   19142.578    18.694\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19602997   19143.551    18.695\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19603949   19144.480    18.696\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19604873   19145.383    18.697\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19605825   19146.312    18.698\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19606789   19147.254    18.698\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19607765   19148.207    18.699\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19608169   19148.602    18.700\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19609133   19149.543    18.701\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19610121   19150.508    18.702\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19611093   19151.457    18.703\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19612065   19152.406    18.704\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19612889   19153.211    18.704\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19613705   19154.008    18.705\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19614665   19154.945    18.706\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19615609   19155.867    18.707\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19616553   19156.789    18.708\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19617485   19157.699    18.709\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19618157   19158.355    18.709\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19618837   19159.020    18.710\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19619509   19159.676    18.711\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19620457   19160.602    18.712\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19621429   19161.551    18.712\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19622265   19162.367    18.713\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19622897   19162.984    18.714\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19623845   19163.910    18.715\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19624805   19164.848    18.716\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19625521   19165.547    18.716\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19626253   19166.262    18.717\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19627173   19167.160    18.718\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19628161   19168.125    18.719\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19629041   19168.984    18.720\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19629713   19169.641    18.720\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19630677   19170.582    18.721\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19631633   19171.516    18.722\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19632429   19172.293    18.723\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19633137   19172.984    18.724\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19633877   19173.707    18.724\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19634833   19174.641    18.725\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19635677   19175.465    18.726\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19636625   19176.391    18.727\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19637573   19177.316    18.728\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19638425   19178.148    18.729\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19639333   19179.035    18.730\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19640209   19179.891    18.730\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19640901   19180.566    18.731\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19641585   19181.234    18.732\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19642269   19181.902    18.732\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19642889   19182.508    18.733\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19643705   19183.305    18.734\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19644517   19184.098    18.734\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19645485   19185.043    18.735\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19646369   19185.906    18.736\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19647269   19186.785    18.737\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19648257   19187.750    18.738\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19649229   19188.699    18.739\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19650117   19189.566    18.740\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19651017   19190.445    18.741\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19651929   19191.336    18.742\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19652877   19192.262    18.742\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19653813   19193.176    18.743\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19654561   19193.906    18.744\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19655541   19194.863    18.745\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19656437   19195.738    18.746\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19657361   19196.641    18.747\n",
      "openf___224_> Open           buildprofile/EAO00193build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18421258   17989.510    17.568\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18421786   17990.025    17.568\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18470386   18037.486    17.615\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18483686   18050.475    17.627\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           1mj3.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18495896   18062.398    17.639\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18495972   18062.473    17.639\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18496143   18062.641    17.639\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18496993   18063.469    17.640\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18498251   18064.699    17.641\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18500155   18066.559    17.643\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18500195   18066.598    17.643\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18500285   18066.684    17.643\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18503141   18069.473    17.646\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18503201   18069.531    17.646\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18503336   18069.664    17.646\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18507620   18073.848    17.650\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18507708   18073.934    17.650\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18507906   18074.127    17.651\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18514332   18080.402    17.657\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18514464   18080.531    17.657\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18514761   18080.820    17.657\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18524383   18090.219    17.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18524583   18090.414    17.666\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18525033   18090.852    17.667\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18539483   18104.965    17.681\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18539783   18105.258    17.681\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18540458   18105.916    17.682\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18562116   18127.066    17.702\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18562564   18127.504    17.703\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18563572   18128.488    17.704\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18596076   18160.230    17.735\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18827948   18386.668    17.956\n",
      "read_mo_297_> Segments, residues, atoms:        1      258     1955\n",
      "read_mo_298_> Segment:        1  31:A  290:A     1955\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18835401   18393.945    17.963\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18982637   18537.730    18.103\n",
      "openf___224_> Open           ali/EAO00193.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     19072227   18625.223    18.189\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     19151691   18702.824    18.264\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     19152755   18703.863    18.265\n",
      "\n",
      "Read the alignment from file       : ali/EAO00193.ali\n",
      "\n",
      "Total number of alignment positions:   267\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      1mj3A     258      1    1mj3.pdb MOL_ID  1; MOLECULE  ENOYL-COA HYDRATASE, MITOCHONDRIAL; CHAIN  A, B, C, D, E, F\n",
      "  2   EAO00193     267      1    EAO00193\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          267\n",
      "  Score                    :  178665.6719\n",
      "openf___224_> Open           ali/EAO00193-1mj3A.ali\n",
      "openf___224_> Open           ali/EAO00193-1mj3A.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18421258   17989.510    17.568\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18421786   17990.025    17.568\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18470386   18037.486    17.615\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     18483686   18050.475    17.627\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00193-1mj3A.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18495651   18062.160    17.639\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18497101   18063.574    17.640\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18500001   18066.406    17.643\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18505801   18072.070    17.649\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18635009   18198.250    17.772\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18636037   18199.254    17.773\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18637101   18200.293    17.774\n",
      "\n",
      "Read the alignment from file       : ali/EAO00193-1mj3A.ali\n",
      "\n",
      "Total number of alignment positions:   267\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      1mj3A     258      1    1mj3.pdb MOL_ID  1; MOLECULE  ENOYL-COA HYDRATASE, MITOCHONDRIAL; CHAIN  A, B, C, D, E, F\n",
      "  2   EAO00193     267      1    EAO00193\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           1mj3.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18716226   18277.564    17.849\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18716226   18277.564    17.849\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18716397   18277.730    17.849\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18717247   18278.562    17.850\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18718505   18279.789    17.851\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18720409   18281.648    17.853\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18720409   18281.648    17.853\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18720499   18281.738    17.853\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18723355   18284.527    17.856\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18723355   18284.527    17.856\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18723490   18284.658    17.856\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18727774   18288.842    17.860\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18727774   18288.842    17.860\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18727972   18289.035    17.860\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18734398   18295.311    17.867\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18734398   18295.311    17.867\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18734695   18295.602    17.867\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18744317   18304.996    17.876\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18744317   18304.996    17.876\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18744767   18305.438    17.876\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18759217   18319.547    17.890\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18759217   18319.547    17.890\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18759892   18320.207    17.891\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18781550   18341.357    17.911\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18781550   18341.357    17.911\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18782558   18342.342    17.912\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18815062   18374.084    17.943\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18814351   18373.391    17.943\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18953603   18509.379    18.076\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18955203   18510.941    18.077\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18959183   18514.828    18.081\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18961183   18516.781    18.083\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18964183   18519.711    18.086\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18968683   18524.105    18.090\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18974255   18529.547    18.095\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18977055   18532.281    18.098\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18981255   18536.383    18.102\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18987555   18542.535    18.108\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18990327   18545.242    18.111\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     18991727   18546.609    18.112\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     19033489   18587.391    18.152\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     19232821   18782.051    18.342\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     19261885   18810.434    18.370\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      267   2015   2041      0         0       796\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     19346253   18892.824    18.450\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      267   2015   2041   2753      3210       796\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     19438349   18982.762    18.538\n",
      "patch_______> segment topology patched using RTF:  267 ;  ALA  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      267   2016   2042   2755      3212       797\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      267   2016   2042   2755      3212       797\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1693      323\n",
      "openf___224_> Open           EAO00193.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      267     2016     2016\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19545798   19087.693    18.640\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19553990   19095.693    18.648\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19570374   19111.693    18.664\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19603142   19143.693    18.695\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19635910   19175.693    18.726\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19701446   19239.693    18.789\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19734214   19271.693    18.820\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19766982   19303.693    18.851\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19832518   19367.693    18.914\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19963590   19495.693    19.039\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     20029126   19559.693    19.101\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     20094662   19623.693    19.164\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     20225734   19751.693    19.289\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     2042    2755    3027     788\n",
      "              Total number of restraints before, now         :        0     8612\n",
      "make_re_422_> Number of previous, current restraints         :        0     8612\n",
      "make_re_423_> Number of previous, current selected restraints:        0     8612\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     20375816   19898.258    19.432\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        2  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        2  ASN\n",
      "getdata_643_> Protein accepted:  1mj3A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :     8612     8877\n",
      "make_re_423_> Number of previous, current selected restraints:     8612     8877\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     20375832   19898.273    19.432\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  1mj3A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      275\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :     8877     9143\n",
      "make_re_423_> Number of previous, current selected restraints:     8877     8868\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     20375832   19898.273    19.432\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  1mj3A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     20637976   20154.273    19.682\n",
      "make_re_422_> Number of previous, current restraints         :     9143     9354\n",
      "make_re_423_> Number of previous, current selected restraints:     8868     9079\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     20637976   20154.273    19.682\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  1mj3A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     20769048   20282.273    19.807\n",
      "make_re_422_> Number of previous, current restraints         :     9354     9515\n",
      "make_re_423_> Number of previous, current selected restraints:     9079     9240\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     20769048   20282.273    19.807\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  1mj3A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :     9515     9594\n",
      "make_re_423_> Number of previous, current selected restraints:     9240     9319\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     20769048   20282.273    19.807\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  1mj3A\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :     9594     9626\n",
      "make_re_423_> Number of previous, current selected restraints:     9319     9351\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     21349620   20849.238    20.361\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     21480692   20977.238    20.486\n",
      "make_re_422_> Number of previous, current restraints         :     9626    14759\n",
      "make_re_423_> Number of previous, current selected restraints:     9351    14484\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     21742836   21233.238    20.736\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     22267124   21745.238    21.236\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     22529268   22001.238    21.486\n",
      "make_re_422_> Number of previous, current restraints         :    14759    20163\n",
      "make_re_423_> Number of previous, current selected restraints:    14484    19888\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    20163    22901\n",
      "make_re_423_> Number of previous, current selected restraints:    19888    22626\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    22901    23655\n",
      "make_re_423_> Number of previous, current selected restraints:    22626    23380\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23655    23655\n",
      "make_re_423_> Number of previous, current selected restraints:    23380    23380\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23655    23655\n",
      "make_re_423_> Number of previous, current selected restraints:    23380    23380\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23655    23655\n",
      "make_re_423_> Number of previous, current selected restraints:    23380    23380\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23655    23655\n",
      "make_re_423_> Number of previous, current selected restraints:    23380    23380\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23655    23655\n",
      "make_re_423_> Number of previous, current selected restraints:    23380    23380\n",
      "rmdupl__427_>     1589 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    23655    21791\n",
      "openf___224_> Open           EAO00193.rsr\n",
      "openf___224_> Open           EAO00193.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     22581375   22052.125    21.535\n",
      "rdcsr2__307_> Number of restraints read    :    21791\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    21791    21791\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00193.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     22581407   22052.156    21.535\n",
      "read_mo_297_> Segments, residues, atoms:        1      267     2016\n",
      "read_mo_298_> Segment:        1  1:A  267:A     2016\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2016     2016        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    11338\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    12816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    13751\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    14812\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    15872\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    16933\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    18990\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    20526\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     22922363   22385.121    21.860\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21125\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21760\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21791\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.1361       13  13\n",
      "iupac_m_487_> NH1/2 swapped:     -177.2996       31  31\n",
      "iupac_m_486_> OE1/2 will be swapped:       98.4053       41  41\n",
      "iupac_m_486_> OE1/2 will be swapped:      -98.7737       48  48\n",
      "iupac_m_486_> OE1/2 will be swapped:      122.5869       87  87\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9344       95  95\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0214      104  104\n",
      "iupac_m_486_> OE1/2 will be swapped:     -147.0021      181  181\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7023      182  182\n",
      "iupac_m_487_> NH1/2 swapped:      178.6555      188  188\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.1377      193  193\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.2954      194  194\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.6108      204  204\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9782      238  238\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23263519   22718.281    22.186\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23360287   22812.781    22.278\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23553823   23001.781    22.463\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23940895   23379.781    22.832\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24715039   24135.781    23.570\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     26263327   25647.781    25.047\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   348281\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -27448.5762\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -27448.576172\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    51.938000\n",
      "Sequence length          :          267\n",
      "Compactness              :     0.191534\n",
      "Native energy (pair)     :  -285.203274\n",
      "Native energy (surface)  :    -3.438165\n",
      "Native energy (combined) :    -6.716266\n",
      "Z score (pair)           :    -7.565790\n",
      "Z score (surface)        :    -4.141161\n",
      "Z score (combined)       :    -8.533169\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4414\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1187.0774\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2042       0      0   0.005   0.005      14.173       1.000\n",
      " 2 Bond angle potential               :    2755       0      9   1.953   1.953      209.08       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1163       0     32  49.508  49.508      447.60       1.000\n",
      " 4 Stereochemical improper torsion pot:     788       0      0   1.226   1.226      27.322       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4414       0      0   0.002   0.002      1.6581       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5133       0      3   0.176   0.176      75.672       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5404       0      3   0.194   0.194      130.18       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     266       0      2   4.258   4.258      56.875       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     211       0      0  69.127  69.127      32.935       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     161       0      1  67.848  67.848      49.356       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      79       0      0  65.399  65.399      48.415       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  75.065  75.065      19.790       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2738       0      0   0.332   0.332      51.321       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     265      11     25  26.642  38.570     -23.261       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     754       0      0   0.649   0.649      45.969       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00193.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00193.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   21905.5488\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6749   1M   2L C   N       7    9  -70.62  -70.70    3.58    0.27  -63.50  173.77   24.54\n",
      "    1          2L   2L N   CA      9   10  145.18  141.60                  -41.20\n",
      "    2   6750   2L   3R C   N      15   17  -77.72  -72.10   25.76    1.86  -63.00  152.57   21.62\n",
      "    2          3R   3R N   CA     17   18  167.04  141.90                  -41.10\n",
      "    3   6759  11S  12M C   N      90   92 -135.63 -125.60   59.44    2.62  -63.40  140.42   26.53\n",
      "    3         12M  12M N   CA     92   93 -160.91  140.50                  -40.50\n",
      "    4   6762  14P  15I C   N     113  115   25.98  -97.30  124.94    5.79 -120.60  148.43    9.50\n",
      "    4         15I  15I N   CA    115  116  106.90  127.20                  130.30\n",
      "    5   6766  18Y  19A C   N     149  151   89.39 -134.00  140.50    3.56  -62.50 -153.91   32.17\n",
      "    5         19A  19A N   CA    151  152  179.81  147.00                  -40.90\n",
      "    6   6777  29L  30N C   N     225  227 -133.33 -119.90   63.89    3.18  -63.20  135.24   14.53\n",
      "    6         30N  30N N   CA    227  228   74.53  137.00                  -41.10\n",
      "    7   6836  88F  89Y C   N     666  668  -37.48  -98.40   61.45    2.41  -63.50 -178.27   30.18\n",
      "    7         89Y  89Y N   CA    668  669  136.46  128.40                  -43.40\n",
      "    8   6837  89Y  90K C   N     678  680   52.60  -62.90  174.36   20.79  -62.90  174.36   20.79\n",
      "    8         90K  90K N   CA    680  681 -171.41  -40.80                  -40.80\n",
      "    9   6838  90K  91H C   N     687  689  -80.41  -63.20   90.74   10.38  -63.20   90.74   10.38\n",
      "    9         91H  91H N   CA    689  690   46.79  -42.30                  -42.30\n",
      "   10   6841  93M  94L C   N     713  715   60.46   60.20    1.89    0.09 -108.50 -161.19    8.77\n",
      "   10         94L  94L N   CA    715  716   27.73   29.60                  132.50\n",
      "   11   6845  97L  98D C   N     744  746 -111.19  -96.50   74.58    3.12  -63.30   94.17   10.00\n",
      "   11         98D  98D N   CA    746  747   41.08  114.20                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    7   21  113  158  219  195  258  326  341  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00193.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      267     2016     2016\n",
      "openf___224_> Open           EAO00193.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     25970587   25361.902    24.767\n",
      "read_mo_297_> Segments, residues, atoms:        1      267     2016\n",
      "read_mo_298_> Segment:        1  1:A  267:A     2016\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2016     2016        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    11338\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    12816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    13751\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    14812\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    15872\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    16933\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    18990\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    20526\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21125\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21760\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21791\n",
      "iupac_m_487_> NH1/2 swapped:      179.5670        3  3\n",
      "iupac_m_486_> OE1/2 will be swapped:      107.7389       48  48\n",
      "iupac_m_486_> OE1/2 will be swapped:     -108.7834       67  67\n",
      "iupac_m_487_> NH1/2 swapped:      176.3786       95  95\n",
      "iupac_m_486_> OE1/2 will be swapped:     -158.9532      133  133\n",
      "iupac_m_487_> NH1/2 swapped:      178.0013      155  155\n",
      "iupac_m_487_> NH1/2 swapped:      179.5546      158  158\n",
      "iupac_m_486_> OE1/2 will be swapped:     -137.5706      179  179\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7353      191  191\n",
      "iupac_m_486_> OE1/2 will be swapped:      162.0099      204  204\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.9065      248  248\n",
      "iupac_m_486_> OE1/2 will be swapped:     -112.6149      258  258\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   349643\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -27727.9180\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -27727.917969\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    51.938000\n",
      "Sequence length          :          267\n",
      "Compactness              :     0.199461\n",
      "Native energy (pair)     :  -298.700603\n",
      "Native energy (surface)  :    -2.905181\n",
      "Native energy (combined) :    -6.899351\n",
      "Z score (pair)           :    -7.555762\n",
      "Z score (surface)        :    -3.921575\n",
      "Z score (combined)       :    -6.977693\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4352\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1169.4932\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2042       0      0   0.005   0.005      14.356       1.000\n",
      " 2 Bond angle potential               :    2755       0      8   1.944   1.944      208.26       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1163       0     32  49.060  49.060      444.05       1.000\n",
      " 4 Stereochemical improper torsion pot:     788       0      0   1.252   1.252      28.002       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4352       0      0   0.002   0.002      2.1612       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5133       0      3   0.160   0.160      73.517       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5404       0      3   0.204   0.204      135.58       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     266       0      3   4.219   4.219      55.850       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     211       0      1  63.509  63.509      23.032       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     161       0      1  71.629  71.629      56.648       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      79       0      0  70.861  70.861      48.046       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0 105.062 105.062      22.890       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2738       0      0   0.294   0.294      39.742       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     265      12     25  23.483  39.545     -31.220       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     754       0      0   0.654   0.654      48.582       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00193.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00193.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   22033.1523\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6749   1M   2L C   N       7    9  -98.56  -70.70   62.30    3.90  -63.50  126.44   19.72\n",
      "    1          2L   2L N   CA      9   10 -162.68  141.60                  -41.20\n",
      "    2   6750   2L   3R C   N      15   17  -65.11  -72.10   17.48    1.48  -63.00  161.00   21.89\n",
      "    2          3R   3R N   CA     17   18  157.91  141.90                  -41.10\n",
      "    3   6759  11S  12M C   N      90   92 -131.73 -125.60   67.00    3.07  -63.40  131.44   24.88\n",
      "    3         12M  12M N   CA     92   93 -152.78  140.50                  -40.50\n",
      "    4   6761  13D  14P C   N     106  108  -50.94  -58.70   35.98    2.40  -64.50  147.79   11.76\n",
      "    4         14P  14P N   CA    108  109  -65.64  -30.50                  147.20\n",
      "    5   6762  14P  15I C   N     113  115  -69.14  -63.40   13.65    2.45 -120.60 -178.83   12.29\n",
      "    5         15I  15I N   CA    115  116  -55.99  -43.60                  130.30\n",
      "    6   6766  18Y  19A C   N     149  151   93.99 -134.00  135.68    3.45  -62.50 -149.53   32.93\n",
      "    6         19A  19A N   CA    151  152  178.36  147.00                  -40.90\n",
      "    7   6777  29L  30N C   N     225  227 -133.85 -119.90   59.59    2.99  -63.20  139.40   14.98\n",
      "    7         30N  30N N   CA    227  228   79.07  137.00                  -41.10\n",
      "    8   6837  89Y  90K C   N     678  680  -49.63 -118.00   75.88    3.48  -62.90  147.78   18.50\n",
      "    8         90K  90K N   CA    680  681  172.02  139.10                  -40.80\n",
      "    9   6838  90K  91H C   N     687  689  -57.26  -67.60   11.24    0.96  -63.20  173.37   21.31\n",
      "    9         91H  91H N   CA    689  690  144.43  140.00                  -42.30\n",
      "   10   6839  91H  92N C   N     697  699   50.91   55.90    7.35    0.42  -63.20  142.88   23.42\n",
      "   10         92N  92N N   CA    699  700   44.90   39.50                  -41.10\n",
      "   11   6841  93M  94L C   N     713  715   53.41   60.20    8.93    0.53 -108.50 -171.20    8.33\n",
      "   11         94L  94L N   CA    715  716   35.39   29.60                  132.50\n",
      "   12   6845  97L  98D C   N     744  746 -114.12  -96.50   72.82    3.04  -63.30   97.78   10.40\n",
      "   12         98D  98D N   CA    746  747   43.54  114.20                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   21  107  164  210  175  251  319  325  308\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00193.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      267     2016     2016\n",
      "openf___224_> Open           EAO00193.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     25970587   25361.902    24.767\n",
      "read_mo_297_> Segments, residues, atoms:        1      267     2016\n",
      "read_mo_298_> Segment:        1  1:A  267:A     2016\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2016     2016        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    11338\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    12816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    13751\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    14812\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    15872\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    16933\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    18990\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    20526\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21125\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21760\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21791\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8736        3  3\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.1919        7  7\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.0093       31  31\n",
      "iupac_m_487_> NH1/2 swapped:     -177.4469       69  69\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.1654       88  88\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:       96.6013      121  121\n",
      "iupac_m_487_> NH1/2 swapped:      179.7771      158  158\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -118.3500      175  175\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.4912      181  181\n",
      "iupac_m_487_> NH1/2 swapped:      177.4698      182  182\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9619      188  188\n",
      "iupac_m_487_> NH1/2 swapped:     -177.3177      191  191\n",
      "iupac_m_486_> OE1/2 will be swapped:     -105.6527      226  226\n",
      "iupac_m_486_> OE1/2 will be swapped:     -123.8308      236  236\n",
      "iupac_m_487_> NH1/2 swapped:      179.6625      238  238\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   347839\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -26910.7148\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -26910.714844\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    51.938000\n",
      "Sequence length          :          267\n",
      "Compactness              :     0.189735\n",
      "Native energy (pair)     :  -286.212141\n",
      "Native energy (surface)  :    -3.848223\n",
      "Native energy (combined) :    -7.001606\n",
      "Z score (pair)           :    -7.713665\n",
      "Z score (surface)        :    -4.288954\n",
      "Z score (combined)       :    -7.376454\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4485\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1235.9978\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2042       0      0   0.005   0.005      15.880       1.000\n",
      " 2 Bond angle potential               :    2755       0      7   2.015   2.015      219.88       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1163       0     33  49.248  49.248      448.34       1.000\n",
      " 4 Stereochemical improper torsion pot:     788       0      0   1.305   1.305      29.862       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4485       0      0   0.002   0.002      2.3044       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5133       0      2   0.144   0.144      74.126       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5404       1      4   0.202   0.202      148.07       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     266       0      1   4.114   4.114      53.082       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     211       0      1  69.110  69.110      46.074       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     161       0      1  72.472  72.472      55.931       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      79       0      0  64.057  64.057      49.580       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  85.330  85.330      24.012       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2738       0      0   0.324   0.324      47.947       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     265      13     23  24.688  41.579     -32.014       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     754       0      0   0.674   0.674      52.914       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00193.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00193.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   23299.5664\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6749   1M   2L C   N       7    9  -62.87  -70.70   21.01    1.34  -63.50  163.30   22.71\n",
      "    1          2L   2L N   CA      9   10  122.10  141.60                  -41.20\n",
      "    2   6750   2L   3R C   N      15   17  -75.42  -72.10    7.77    0.53  -63.00  170.43   23.87\n",
      "    2          3R   3R N   CA     17   18  148.93  141.90                  -41.10\n",
      "    3   6759  11S  12M C   N      90   92  164.29 -125.60   75.74    2.36  -63.40 -159.72   39.58\n",
      "    3         12M  12M N   CA     92   93  169.15  140.50                  -40.50\n",
      "    4   6760  12M  13D C   N      98  100   71.81  -70.90  151.40   12.59  -63.30 -179.85   20.88\n",
      "    4         13D  13D N   CA    100  101 -159.15  150.30                  -40.00\n",
      "    5   6761  13D  14P C   N     106  108  -52.84  -58.70   10.43    0.63  -64.50  174.07   13.61\n",
      "    5         14P  14P N   CA    108  109  -39.13  -30.50                  147.20\n",
      "    6   6766  18Y  19A C   N     149  151   91.01 -134.00  139.11    3.50  -62.50 -153.25   32.33\n",
      "    6         19A  19A N   CA    151  152 -179.38  147.00                  -40.90\n",
      "    7   6777  29L  30N C   N     225  227 -135.38 -119.90   66.37    3.33  -63.20  134.56   14.47\n",
      "    7         30N  30N N   CA    227  228   72.46  137.00                  -41.10\n",
      "    8   6837  89Y  90K C   N     678  680 -103.55 -118.00   81.24    4.10  -62.90  108.08   16.47\n",
      "    8         90K  90K N   CA    680  681 -140.95  139.10                  -40.80\n",
      "    9   6838  90K  91H C   N     687  689  -52.38  -67.60   23.12    2.08  -63.20  160.66   19.43\n",
      "    9         91H  91H N   CA    689  690  157.40  140.00                  -42.30\n",
      "   10   6839  91H  92N C   N     697  699   71.56   55.90   23.47    1.32  -63.20  148.81   24.40\n",
      "   10         92N  92N N   CA    699  700   22.02   39.50                  -41.10\n",
      "   11   6842  94L  95R C   N     721  723 -144.74 -125.20   31.96    1.04  -63.00  173.47   28.96\n",
      "   11         95R  95R N   CA    723  724  165.89  140.60                  -41.10\n",
      "   12   6843  95R  96G C   N     732  734   77.05   82.20   10.90    0.76  -62.40  145.10   27.15\n",
      "   12         96G  96G N   CA    734  735   -1.10    8.50                  -41.20\n",
      "   13   6845  97L  98D C   N     744  746 -114.01  -96.50   74.93    3.13  -63.30   95.86   10.20\n",
      "   13         98D  98D N   CA    746  747   41.35  114.20                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   10   29   96  154  225  185  251  338  331  342\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00193.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      267     2016     2016\n",
      "openf___224_> Open           EAO00193.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     25970587   25361.902    24.767\n",
      "read_mo_297_> Segments, residues, atoms:        1      267     2016\n",
      "read_mo_298_> Segment:        1  1:A  267:A     2016\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2016     2016        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    11338\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    12816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    13751\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    14812\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    15872\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    16933\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    18990\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    20526\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21125\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21760\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21791\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.8147        7  7\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -152.2412       13  13\n",
      "iupac_m_486_> OE1/2 will be swapped:      -99.3822       41  41\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.1465       48  48\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0353      104  104\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.8604      114  114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.1905\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.5003      121  121\n",
      "iupac_m_486_> OE1/2 will be swapped:     -144.5658      167  167\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0055      182  182\n",
      "iupac_m_487_> NH1/2 swapped:     -176.9842      188  188\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2254      191  191\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1225      238  238\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.2043      247  247\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   348289\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -27624.9473\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -27624.947266\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    51.938000\n",
      "Sequence length          :          267\n",
      "Compactness              :     0.195726\n",
      "Native energy (pair)     :  -287.608806\n",
      "Native energy (surface)  :    -0.659380\n",
      "Native energy (combined) :    -6.454218\n",
      "Z score (pair)           :    -7.533552\n",
      "Z score (surface)        :    -3.687821\n",
      "Z score (combined)       :    -6.909256\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4373\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1159.8188\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2042       0      0   0.005   0.005      14.097       1.000\n",
      " 2 Bond angle potential               :    2755       0      6   1.909   1.909      199.99       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1163       0     34  49.298  49.298      446.98       1.000\n",
      " 4 Stereochemical improper torsion pot:     788       0      0   1.153   1.153      24.841       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4373       0      0   0.002   0.002      1.9301       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5133       0      1   0.155   0.155      72.151       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5404       1      3   0.212   0.212      147.43       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     266       0      2   4.121   4.121      53.266       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     211       0      1  68.721  68.721      29.434       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     161       0      1  70.691  70.691      45.263       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      79       0      0  64.366  64.366      51.211       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  92.135  92.135      19.308       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2738       0      0   0.334   0.334      53.364       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     265      11     20  22.707  38.776     -52.677       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     754       0      0   0.679   0.679      53.226       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00193.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00193.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   22497.1133\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6749   1M   2L C   N       7    9 -123.88 -108.50   20.79    0.93  -63.50 -177.42   29.09\n",
      "    1          2L   2L N   CA      9   10  146.49  132.50                  -41.20\n",
      "    2   6750   2L   3R C   N      15   17 -125.80 -125.20   12.33    0.58  -63.00  177.47   28.30\n",
      "    2          3R   3R N   CA     17   18  152.91  140.60                  -41.10\n",
      "    3   6759  11S  12M C   N      90   92 -152.06 -125.60   96.59    4.00  -63.40  123.60   24.74\n",
      "    3         12M  12M N   CA     92   93 -126.61  140.50                  -40.50\n",
      "    4   6761  13D  14P C   N     106  108  -66.30  -58.70   11.98    1.57  -64.50  173.05   12.82\n",
      "    4         14P  14P N   CA    108  109  -39.76  -30.50                  147.20\n",
      "    5   6766  18Y  19A C   N     149  151   92.24 -134.00  137.36    3.50  -62.50 -150.73   32.69\n",
      "    5         19A  19A N   CA    151  152  178.21  147.00                  -40.90\n",
      "    6   6777  29L  30N C   N     225  227 -133.21 -119.90   66.70    3.31  -63.20  132.71   14.26\n",
      "    6         30N  30N N   CA    227  228   71.64  137.00                  -41.10\n",
      "    7   6838  90K  91H C   N     687  689 -177.43 -125.60   66.84    1.64  -63.20  178.14   29.37\n",
      "    7         91H  91H N   CA    689  690 -179.00  138.80                  -42.30\n",
      "    8   6839  91H  92N C   N     697  699   59.96   55.90    8.55    0.41  -63.20  143.21   23.55\n",
      "    8         92N  92N N   CA    699  700   31.98   39.50                  -41.10\n",
      "    9   6842  94L  95R C   N     721  723 -139.50 -125.20   33.52    1.25  -63.00  166.58   27.69\n",
      "    9         95R  95R N   CA    723  724  170.92  140.60                  -41.10\n",
      "   10   6843  95R  96G C   N     732  734   77.89   82.20   10.86    0.73  -62.40  145.81   27.27\n",
      "   10         96G  96G N   CA    734  735   -1.46    8.50                  -41.20\n",
      "   11   6845  97L  98D C   N     744  746 -114.74  -96.50   74.77    3.12  -63.30   96.54   10.28\n",
      "   11         98D  98D N   CA    746  747   41.69  114.20                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   14   16  103  158  221  181  257  330  326  293\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00193.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      267     2016     2016\n",
      "openf___224_> Open           EAO00193.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     25970587   25361.902    24.767\n",
      "read_mo_297_> Segments, residues, atoms:        1      267     2016\n",
      "read_mo_298_> Segment:        1  1:A  267:A     2016\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2016     2016        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    11338\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    12816\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    13751\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    14812\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    15872\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    16933\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    18990\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    20526\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21125\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21760\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2016    21791\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4305        3  3\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.9134        7  7\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      116.6531       13  13\n",
      "iupac_m_487_> NH1/2 swapped:      178.0485       31  31\n",
      "iupac_m_486_> OE1/2 will be swapped:      146.5838       41  41\n",
      "iupac_m_486_> OE1/2 will be swapped:     -153.0430       67  67\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.4508       85  85\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.4042       88  88\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      137.0403      133  133\n",
      "iupac_m_487_> NH1/2 swapped:      174.8324      158  158\n",
      "iupac_m_486_> OE1/2 will be swapped:     -132.4695      167  167\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.9623      173  173\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -105.1986      175  175\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.5770      188  188\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8953      191  191\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.5752      193  193\n",
      "iupac_m_486_> OE1/2 will be swapped:     -102.2271      226  226\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.3636      235  235\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.9821      238  238\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   349597\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -27328.0762\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -27328.076172\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    51.938000\n",
      "Sequence length          :          267\n",
      "Compactness              :     0.191493\n",
      "Native energy (pair)     :  -280.368937\n",
      "Native energy (surface)  :    -2.223470\n",
      "Native energy (combined) :    -6.585794\n",
      "Z score (pair)           :    -7.593906\n",
      "Z score (surface)        :    -3.851731\n",
      "Z score (combined)       :    -6.995704\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      267\n",
      "Number of all, selected real atoms                :     2016    2016\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21791   21791\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4327\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1267.1495\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2042       0      0   0.005   0.005      14.238       1.000\n",
      " 2 Bond angle potential               :    2755       0     10   1.969   1.969      211.99       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1163       0     34  49.491  49.491      453.24       1.000\n",
      " 4 Stereochemical improper torsion pot:     788       0      0   1.302   1.302      29.770       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4327       0      0   0.002   0.002      1.6327       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5133       0      1   0.158   0.158      75.393       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5404       1      5   0.221   0.221      157.90       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     266       0      0   4.158   4.158      54.242       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     211       0      1  69.941  69.941      41.085       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     161       0      1  72.147  72.147      56.690       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      79       0      0  69.825  69.825      49.372       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  92.935  92.935      21.096       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2738       0      0   0.347   0.347      56.423       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     265      14     25  26.650  42.498     -11.292       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     754       0      1   0.696   0.696      55.370       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00193.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00193.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   23852.8535\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6749   1M   2L C   N       7    9  -65.19  -70.70    7.70    0.49  -63.50  177.42   24.54\n",
      "    1          2L   2L N   CA      9   10  136.21  141.60                  -41.20\n",
      "    2   6750   2L   3R C   N      15   17 -117.38 -125.20   15.84    0.88  -63.00  173.28   27.16\n",
      "    2          3R   3R N   CA     17   18  154.37  140.60                  -41.10\n",
      "    3   6759  11S  12M C   N      90   92  175.49  -63.40  124.38   24.52  -63.40  124.38   24.52\n",
      "    3         12M  12M N   CA     92   93  -68.84  -40.50                  -40.50\n",
      "    4   6762  14P  15I C   N     113  115  -53.49  -63.40   11.05    2.08 -120.60 -178.15    8.04\n",
      "    4         15I  15I N   CA    115  116  -38.71  -43.60                  130.30\n",
      "    5   6766  18Y  19A C   N     149  151   89.25 -134.00  140.50    3.58  -62.50 -153.62   32.19\n",
      "    5         19A  19A N   CA    151  152  179.23  147.00                  -40.90\n",
      "    6   6777  29L  30N C   N     225  227 -130.59 -119.90   63.46    3.11  -63.20  133.76   14.38\n",
      "    6         30N  30N N   CA    227  228   74.44  137.00                  -41.10\n",
      "    7   6836  88F  89Y C   N     666  668  -65.46  -98.40   35.13    1.92  -63.50  159.60   25.22\n",
      "    7         89Y  89Y N   CA    668  669  116.19  128.40                  -43.40\n",
      "    8   6837  89Y  90K C   N     678  680 -165.63 -118.00   83.79    3.19  -62.90  151.35   25.56\n",
      "    8         90K  90K N   CA    680  681 -151.95  139.10                  -40.80\n",
      "    9   6838  90K  91H C   N     687  689  -82.64  -67.60   81.40    6.68  -63.20  104.14   11.93\n",
      "    9         91H  91H N   CA    689  690   60.01  140.00                  -42.30\n",
      "   10   6839  91H  92N C   N     697  699   72.70   55.90  120.81    9.72  -63.20 -150.25   22.95\n",
      "   10         92N  92N N   CA    699  700  159.13   39.50                  -41.10\n",
      "   11   6841  93M  94L C   N     713  715   63.42   60.20  126.72    7.82 -108.50  173.56    8.56\n",
      "   11         94L  94L N   CA    715  716  156.28   29.60                  132.50\n",
      "   12   6842  94L  95R C   N     721  723 -144.66 -125.20   26.22    0.79  -63.00 -179.72   29.90\n",
      "   12         95R  95R N   CA    723  724  158.17  140.60                  -41.10\n",
      "   13   6843  95R  96G C   N     732  734   72.63   82.20   10.60    0.82  -62.40  142.38   26.75\n",
      "   13         96G  96G N   CA    734  735    3.96    8.50                  -41.20\n",
      "   14   6845  97L  98D C   N     744  746 -113.06  -96.50   75.35    3.15  -63.30   94.81   10.09\n",
      "   14         98D  98D N   CA    746  747   40.70  114.20                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    9   16   96  175  200  194  256  320  308  321\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00193.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      267     2016     2016\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00193.B99990001.pdb        1187.07739   -27448.57617        1.00000\n",
      "EAO00193.B99990002.pdb        1169.49316   -27727.91797        1.00000\n",
      "EAO00193.B99990003.pdb        1235.99780   -26910.71484        1.00000\n",
      "EAO00193.B99990004.pdb        1159.81885   -27624.94727        1.00000\n",
      "EAO00193.B99990005.pdb        1267.14954   -27328.07617        1.00000\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26161929   25548.758    24.950\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26162457   25549.273    24.950\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26211057   25596.734    24.997\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26224357   25609.723    25.009\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26493623   25872.680    25.266\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     30054746   29350.338    28.662\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     30056988   29352.527    28.665\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     30058438   29353.943    28.666\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     30061338   29356.775    28.669\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     30067138   29362.439    28.674\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     43505537   42485.875    41.490\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     43517137   42497.203    41.501\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     43540337   42519.859    41.523\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     43586737   42565.172    41.568\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     43679537   42655.797    41.656\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     50398737   49217.516    48.064\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00194.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26495943   25874.945    25.269\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26497393   25876.359    25.270\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26500293   25879.191    25.273\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26506093   25884.855    25.278\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26507577   25886.305    25.280\n",
      "\n",
      "Read the alignment from file       : ali/EAO00194.ali\n",
      "\n",
      "Total number of alignment positions:   372\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00194     372      1    EAO00194\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     26509398   25888.084    25.281\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26834573   26205.637    25.591\n",
      "profile_iteration_> processing sequence:       1    372      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26835651   26206.691    25.592\n",
      "profile_iteration_> processing sequence:    6720    372      1     3.5781250     0.0005325     0.0000020  1878\n",
      "profile_iteration_> processing sequence:   13439    372      1     7.0625000     0.0005255     0.0000021  1903\n",
      "profile_iteration_> processing sequence:   20158    372      1    10.5312500     0.0005224     0.0000021  1914\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26836026   26207.057    25.593\n",
      "profile_iteration_> processing sequence:   26877    372      1    14.0468750     0.0005226     0.0000021  1913\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26836262   26207.287    25.593\n",
      "profile_iteration_> processing sequence:   33596    372      1    17.5625000     0.0005228     0.0000021  1913\n",
      "profile_iteration_> processing sequence:   40315    372      1    21.0156250     0.0005213     0.0000021  1918\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26839212   26210.168    25.596\n",
      "profile_iteration_> processing sequence:   47034    372      1    24.5468750     0.0005219     0.0000021  1916\n",
      "profile_iteration_> processing sequence:   53753    372      1    28.1250000     0.0005232     0.0000021  1911\n",
      "profile_iteration_> processing sequence:   60472    372      1    31.6250000     0.0005230     0.0000021  1912\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     26842722   26213.596    25.599\n",
      "profile_iteration_> processing sequence:   67191    372      1    35.2656250     0.0005249     0.0000021  1905\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    28.17937\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    250.5631\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0224,      121,        0.0145,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00015   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00030   0.00016\n",
      "Z:         1  -2.25000   0.00074   0.00055\n",
      "Z:         1  -2.15000   0.00491   0.00163\n",
      "Z:         1  -2.05000   0.00387   0.00416\n",
      "Z:         1  -1.95000   0.01533   0.00933\n",
      "Z:         1  -1.85000   0.01667   0.01872\n",
      "Z:         1  -1.75000   0.03051   0.03400\n",
      "Z:         1  -1.65000   0.05209   0.05658\n",
      "Z:         1  -1.55000   0.07992   0.08721\n",
      "Z:         1  -1.45000   0.11013   0.12565\n",
      "Z:         1  -1.35000   0.16728   0.17059\n",
      "Z:         1  -1.25000   0.19481   0.21981\n",
      "Z:         1  -1.15000   0.28024   0.27052\n",
      "Z:         1  -1.05000   0.30584   0.31973\n",
      "Z:         1  -0.95000   0.34364   0.36469\n",
      "Z:         1  -0.85000   0.37594   0.40316\n",
      "Z:         1  -0.75000   0.42490   0.43360\n",
      "Z:         1  -0.65000   0.44961   0.45519\n",
      "Z:         1  -0.55000   0.51494   0.46779\n",
      "Z:         1  -0.45000   0.45571   0.47182\n",
      "Z:         1  -0.35000   0.48027   0.46811\n",
      "Z:         1  -0.25000   0.43785   0.45775\n",
      "Z:         1  -0.15000   0.43353   0.44196\n",
      "Z:         1  -0.05000   0.42059   0.42194\n",
      "Z:         1   0.05000   0.40436   0.39889\n",
      "Z:         1   0.15000   0.36299   0.37383\n",
      "Z:         1   0.25000   0.34200   0.34769\n",
      "Z:         1   0.35000   0.32370   0.32121\n",
      "Z:         1   0.45000   0.30331   0.29500\n",
      "Z:         1   0.55000   0.25256   0.26953\n",
      "Z:         1   0.65000   0.25092   0.24513\n",
      "Z:         1   0.75000   0.21178   0.22205\n",
      "Z:         1   0.85000   0.19824   0.20043\n",
      "Z:         1   0.95000   0.17443   0.18036\n",
      "Z:         1   1.05000   0.16490   0.16185\n",
      "Z:         1   1.15000   0.15359   0.14489\n",
      "Z:         1   1.25000   0.13752   0.12944\n",
      "Z:         1   1.35000   0.10805   0.11542\n",
      "Z:         1   1.45000   0.10507   0.10275\n",
      "Z:         1   1.55000   0.08305   0.09133\n",
      "Z:         1   1.65000   0.08409   0.08109\n",
      "Z:         1   1.75000   0.07248   0.07191\n",
      "Z:         1   1.85000   0.06013   0.06371\n",
      "Z:         1   1.95000   0.05268   0.05639\n",
      "Z:         1   2.05000   0.04926   0.04988\n",
      "Z:         1   2.15000   0.04584   0.04409\n",
      "Z:         1   2.25000   0.03929   0.03895\n",
      "Z:         1   2.35000   0.03185   0.03439\n",
      "Z:         1   2.45000   0.03200   0.03035\n",
      "Z:         1   2.55000   0.02857   0.02678\n",
      "Z:         1   2.65000   0.01950   0.02361\n",
      "Z:         1   2.75000   0.01712   0.02082\n",
      "Z:         1   2.85000   0.01816   0.01835\n",
      "Z:         1   2.95000   0.01637   0.01617\n",
      "Z:         1   3.05000   0.01429   0.01424\n",
      "Z:         1   3.15000   0.01086   0.01255\n",
      "Z:         1   3.25000   0.00863   0.01105\n",
      "Z:         1   3.35000   0.01086   0.00973\n",
      "Z:         1   3.45000   0.00997   0.00857\n",
      "Z:         1   3.55000   0.00729   0.00754\n",
      "Z:         1   3.65000   0.00476   0.00664\n",
      "Z:         1   3.75000   0.00402   0.00584\n",
      "Z:         1   3.85000   0.00417   0.00514\n",
      "Z:         1   3.95000   0.00506   0.00453\n",
      "Z:         1   4.05000   0.00521   0.00398\n",
      "Z:         1   4.15000   0.00283   0.00350\n",
      "Z:         1   4.25000   0.00372   0.00308\n",
      "Z:         1   4.35000   0.00313   0.00271\n",
      "Z:         1   4.45000   0.00193   0.00239\n",
      "Z:         1   4.55000   0.00193   0.00210\n",
      "Z:         1   4.65000   0.00074   0.00185\n",
      "Z:         1   4.75000   0.00193   0.00163\n",
      "Z:         1   4.85000   0.00074   0.00143\n",
      "Z:         1   4.95000   0.00045   0.00126\n",
      "Z:         1   5.05000   0.00119   0.00111\n",
      "Z:         1   5.15000   0.00089   0.00097\n",
      "Z:         1   5.25000   0.00193   0.00086\n",
      "Z:         1   5.35000   0.00030   0.00075\n",
      "Z:         1   5.45000   0.00074   0.00066\n",
      "Z:         1   5.55000   0.00015   0.00058\n",
      "Z:         1   5.65000   0.00015   0.00051\n",
      "Z:         1   5.75000   0.00045   0.00045\n",
      "Z:         1   5.85000   0.00060   0.00040\n",
      "Z:         1   5.95000   0.00000   0.00035\n",
      "Z:         1   6.05000   0.00015   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00015   0.00024\n",
      "Z:         1   6.35000   0.00030   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00030   0.00016\n",
      "Z:         1   6.65000   0.00015   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00030   0.00007\n",
      "Z:         1   7.25000   0.00060   0.00007\n",
      "Z:         1   7.35000   0.00015   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00030   0.00003\n",
      "Z:         1   7.85000   0.00015   0.00003\n",
      "Z:         1   7.95000   0.00030   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     27157322   26520.822    25.899\n",
      "> 2xk7A                      1      64    6900     246     372   34.85    0.89E-07       2   132   156   316   105   236\n",
      "> 4lg4A                      1     163   17950     289     372   34.63     0.0           3   276    54   348     1   283\n",
      "> 4lg4D                      1     164   14150     250     372   33.61     0.0           4   235    80   348     7   244\n",
      "> 4lg4E                      1     165   16300     275     372   33.60     0.0           5   251    74   348    17   269\n",
      "> 4lgdA                      1     176   18800     350     372   34.35     0.0           6   287    54   357     3   296\n",
      "> 4lgdD                      1     177   18950     338     372   33.99     0.0           7   296    54   368     8   310\n",
      "> 4lggB                      1     180    7500     245     372   33.07    0.41E-08       8   126   160   308    94   220\n",
      "> 3brbA                      1     232    6850     260     372   26.42    0.12E-06       9   232    57   312     4   249\n",
      "> 3brbB                      1     233    7150     271     372   26.03    0.28E-07      10   224    66   312    17   258\n",
      "> 4eojC                      1     383   13050     282     372   31.82     0.0          11   252    68   323    14   277\n",
      "> 3utoA                      1     450    9300     559     372   23.69     0.0          12   246    68   317   151   399\n",
      "> 6vnvA                      1     583    8850     274     372   31.25    0.75E-11      13   145   166   313   103   262\n",
      "> 3wzdA                      1     602    7000     278     372   32.09    0.62E-07      14   133   165   315   137   270\n",
      "> 5t6aA                      1     680   13600     431     372   30.77     0.0          15   299    65   372    10   321\n",
      "> 6t6fA                      1     707    9050     289     372   28.29     0.0          16   238    65   339    15   272\n",
      "> 5jgaA                      1     853    9000     296     372   33.05     0.0          17   221    65   314    14   246\n",
      "> 6g78A                      1     919    9400     283     372   27.73     0.0          18   226    66   318    27   264\n",
      "> 6g76B                      1     943    8400     269     372   30.36    0.45E-10      19   212    66   318    27   250\n",
      "> 3vo3A                      1    1028    7300     303     372   30.00    0.15E-07      20   145   165   315   145   294\n",
      "> 5efqA                      1    1099   10400     321     372   27.50     0.0          21   247    67   319    16   295\n",
      "> 3efkA                      1    1104    7000     276     372   29.57    0.61E-07      22   217    67   313    30   259\n",
      "> 5z33A                      1    1175    8750     375     372   28.36    0.15E-10      23   169   155   325   117   317\n",
      "> 5yksA                      1    1240    9850     312     372   37.16     0.0          24   146   153   320    99   246\n",
      "> 5yksB                      1    1241   10900     325     372   33.51     0.0          25   183   153   353    98   282\n",
      "> 3dajA                      1    1327   10900     245     372   31.60     0.0          26   225    66   319    11   241\n",
      "> 5da3A                      1    1370    8950     263     372   30.67     0.0          27   230    66   308    11   248\n",
      "> 3iecA                      1    1487    8950     311     372   33.12     0.0          28   152   155   314    95   248\n",
      "> 6b3eC                      1    1617   10100     307     372   28.32     0.0          29   245    67   319    14   292\n",
      "> 6iqnB                      1    1696    6900     258     372   37.80    0.94E-07      30   123   166   312   121   247\n",
      "> 4xg8C                      1    1763    7400     254     372   25.97    0.72E-08      31   227    71   310    15   245\n",
      "> 6ckxA                      1    1833   10100     292     372   28.32     0.0          32   245    67   319    11   289\n",
      "> 4ckjA                      1    1876    9400     292     372   31.28     0.0          33   216    87   314    56   282\n",
      "> 3ckxA                      1    1918   16950     276     372   35.02     0.0          34   255    68   342    11   267\n",
      "> 3ckwA                      1    1920   15000     255     372   37.62     0.0          35   209   101   342    37   246\n",
      "> 3a7iA                      1    2001   16950     286     372   34.75     0.0          36   256    68   342    18   276\n",
      "> 2qluA                      1    2010    5350     297     372   28.92    0.31E-03      37   171    67   253     8   211\n",
      "> 4fyoA                      1    2109    7650     267     372   25.64    0.21E-08      38   230    68   310    15   248\n",
      "> 6fyiA                      1    2140    6650     336     372   22.83    0.47E-06      39   245    68   318    22   332\n",
      "> 6fylA                      1    2142    6650     340     372   22.83    0.47E-06      40   245    68   318    26   336\n",
      "> 6fyrA                      1    2144    6050     355     372   22.58    0.11E-04      41   160   160   322   133   349\n",
      "> 6fyvA                      1    2147    6500     329     372   23.32    0.98E-06      42   245    58   318    13   325\n",
      "> 1q8yA                      1    2178    6900     351     372   27.36    0.14E-06      43   164   154   319   121   321\n",
      "> 1q8yB                      1    2179    6700     357     372   26.89    0.39E-06      44   164   154   319   116   327\n",
      "> 4xyfA                      1    2402    7600     290     372   27.94    0.30E-08      45   229    67   313    32   278\n",
      "> 5xyzA                      1    2434    5600     230     372   28.92    0.64E-04      46   165   129   315    59   224\n",
      "> 4nzwB                      1    2545   14850     266     372   35.86     0.0          47   232    68   326    16   252\n",
      "> 3d7tA                      1    2626    5650     249     372   32.00    0.54E-04      48   122   167   312   111   235\n",
      "> 3q4zA                      1    2707   14300     282     372   34.96     0.0          49   241    60   319    13   258\n",
      "> 6q49A                      1    2741   12150     287     372   31.73     0.0          50   243    68   323    10   280\n",
      "> 6q4gA                      1    2742   12000     282     372   31.95     0.0          51   238    68   323    10   275\n",
      "> 1x8bA                      1    2756    8300     259     372   30.99    0.75E-10      52   228    68   315    15   256\n",
      "> 3ubdA                      1    2804    9750     287     372   26.94     0.0          53   231    66   318    24   268\n",
      "> 3d5vA                      1    3133    9850     260     372   33.03     0.0          54   219    66   318    19   239\n",
      "> 3soaA                      1    3236    9700     436     372   26.15     0.0          55   205   143   351    78   295\n",
      "> 3socA                      1    3239    5200     307     372   28.96    0.70E-03      56   165    70   242    21   203\n",
      "> 3e3pA                      1    3440    7150     337     372   24.34    0.36E-07      57   160   159   318   109   297\n",
      "> 4e3cA                      1    3487    9100     624     372   36.70     0.0          58   106   154   260   101   209\n",
      "> 5wdyA                      1    3587   10300     260     372   35.11     0.0          59   184   122   319    72   259\n",
      "> 4btfA                      1    3663    6950     413     372   25.89    0.13E-06      60   192   115   314   213   409\n",
      "> 6cthA                      1    3789   10100     295     372   32.47     0.0          61   184    68   263    22   215\n",
      "> 3orkA                      1    3808    8000     274     372   26.61    0.37E-09      62   220    67   310    18   250\n",
      "> 4c2vA                      1    3971   10500     270     372   25.68     0.0          63   253    57   319     1   257\n",
      "> 6gu2A                      1    4119   11600     294     372   29.08     0.0          64   249    61   318     6   287\n",
      "> 6gu7A                      1    4122   11400     284     372   29.35     0.0          65   243    61   318     3   278\n",
      "> 3h4jB                      1    4214   10850     330     372   28.92     0.0          66   244    65   318    11   259\n",
      "> 3h4jA                      1    4215   10200     316     372   29.83     0.0          67   233    65   318    10   247\n",
      "> 1zwsA                      1    4289   10350     278     372   25.81     0.0          68   213   100   317    57   273\n",
      "> 4d1sA                      1    4394   11200     280     372   28.46     0.0          69   237    69   312    17   269\n",
      "> 4d1zA                      1    4396   11000     258     372   33.88     0.0          70   217    74   323    10   254\n",
      "> 5d1jA                      1    4400   10500     265     372   33.06     0.0          71   220    71   323    11   258\n",
      "> 6d1zA                      1    4431    9150     290     372   29.67     0.0          72   230    68   312    26   271\n",
      "> 4xh0A                      1    4435    5250     350     372   27.27    0.63E-03      73   178    68   258     9   206\n",
      "> 4xhlA                      1    4461    5050     330     372   27.27    0.16E-02      74   178    68   258     7   204\n",
      "> 2jedB                      1    4580   10700     320     372   28.97     0.0          75   239    66   319    10   261\n",
      "> 6hxfA                      1    4868   16700     282     372   34.25     0.0          76   247    68   327    19   272\n",
      "> 6hx1A                      1    4876   11350     387     372   34.16     0.0          77   229    65   318    13   255\n",
      "> 3hx4A                      1    4920   13400     467     372   29.81     0.0          78   296    65   371    25   336\n",
      "> 5y90A                      1    4971   14650     304     372   31.43     0.0          79   237    68   319    24   268\n",
      "> 4y93A                      1    4985    7500     429     372   27.60    0.81E-08      80   244    61   315   172   421\n",
      "> 4pf4A                      1    5041   10250     278     372   27.85     0.0          81   208   100   319    59   277\n",
      "> 3pfqA                      1    5113    8450     520     372   25.20    0.82E-10      82   236    67   319   201   454\n",
      "> 5lvoA                      1    5364   12700     286     372   32.69     0.0          83   245    66   322    14   273\n",
      "> 6gzdA                      1    5398    4750     297     372   29.58    0.67E-02      84   133    66   207    13   154\n",
      "> 1gzkA                      1    5451    9100     271     372   36.03     0.0          85   131   166   318   104   239\n",
      "> 4gv1A                      1    5589   11150     329     372   29.30     0.0          86   243    66   320    12   267\n",
      "> 3eb0A                      1    5699    6450     311     372   26.79    0.12E-05      87   196   119   318    56   279\n",
      "> 5ebzA                      1    5747    9350     655     372   37.38     0.0          88   104   154   258   110   216\n",
      "> 2eb2A                      1    5748    9600     305     372   28.63     0.0          89   252    52   313     4   265\n",
      "> 4yfiA                      1    6034    8550     268     372   31.47    0.22E-10      90   191    51   259    12   208\n",
      "> 6u2hC                      1    6138   10400     274     372   32.35     0.0          91   225    68   313    16   253\n",
      "> 2fstX                      1    6158    7900     324     372   30.49    0.75E-09      92   137   163   318   117   280\n",
      "> 6in4A                      1    6312    9750     265     372   29.81     0.0          93   200   100   319    57   264\n",
      "> 3vumA                      1    6403    9400     347     372   30.18     0.0          94   226    68   317    26   300\n",
      "> 3vutA                      1    6425   10700     203     372   41.48     0.0          95   133   155   317    47   181\n",
      "> 3vutB                      1    6426   11200     260     372   33.64     0.0          96   205    84   317    40   253\n",
      "> 5lwmA                      1    6471    9750     290     372   25.76     0.0          97   242    68   312    15   278\n",
      "> 3lw0A                      1    6513    8700     292     372   28.68    0.75E-11      98   244    55   313    12   269\n",
      "> 3qa8A                      1    6529    9300     622     372   37.61     0.0          99   106   154   260   101   209\n",
      "> 3qa8G                      1    6530    9300     541     372   37.61     0.0         100   106   154   260   101   209\n",
      "> 6qatB                      1    6561    8350     245     372   29.63    0.52E-10     101   225    53   319     1   243\n",
      "> 6qauB                      1    6569    8600     265     372   27.97    0.15E-10     102   236    58   319     3   263\n",
      "> 6qavA                      1    6570    9600     273     372   26.57     0.0         103   246    58   319     3   273\n",
      "> 6qavD                      1    6571    8950     254     372   28.06     0.0         104   230    58   319     2   254\n",
      "> 3w18A                      1    6652   10100     234     372   34.10     0.0         105   213    66   317    12   228\n",
      "> 3f3zA                      1    6688   10300     271     372   29.13     0.0         106   242    63   319     7   260\n",
      "> 6f3iA                      1    6737    6850     258     372   42.31    0.12E-06     107    78   169   254   106   183\n",
      "> 6fdyU                      1    6753   11050     262     372   31.91     0.0         108   183   127   317    70   257\n",
      "> 6fd3A                      1    6808   16150     296     372   35.77     0.0         109   241    68   319    30   275\n",
      "> 5fd2B                      1    6827    8950     234     372   35.62     0.0         110   141   156   313    80   225\n",
      "> 1rjbA                      1    6841    8550     298     372   28.35    0.22E-10     111   229    66   321    43   296\n",
      "> 3nizA                      1    6942   12050     288     372   29.85     0.0         112   239    68   319    15   282\n",
      "> 3nieA                      1    6944    8100     369     372   31.63    0.31E-09     113   182    53   245     5   200\n",
      "> 4nifA                      1    6956   10000     313     372   35.37     0.0         114   142   173   317   113   259\n",
      "> 1ua2A                      1    7243    9050     286     372   27.55     0.0         115   237    68   318     6   270\n",
      "> 3rp9A                      1    7322    8800     350     372   28.11    0.75E-11     116   179   155   334   100   316\n",
      "> 2jamA                      1    7529   10900     279     372   31.45     0.0         117   233    67   322    11   258\n",
      "> 5lpbA                      1    7642    9100     280     372   32.46     0.0         118   187    67   257    22   212\n",
      "> 4bgqA                      1    7703    7900     276     372   25.48    0.61E-09     119   232    67   319    10   272\n",
      "> 6bg2C                      1    7706   10850     334     372   28.24     0.0         120   240    68   320    30   284\n",
      "> 3zh8A                      1    7735   11100     318     372   28.79     0.0         121   244    66   319    11   267\n",
      "> 4zhxA                      1    7757    8050     408     372   29.76    0.46E-09     122   167   145   319    93   260\n",
      "> 3lm5A                      1    7975    9200     274     372   26.54     0.0         123   206   100   319    57   267\n",
      "> 6ft8A                      1    8112    6350     338     372   23.00    0.22E-05     124   156   160   318   120   332\n",
      "> 5kbrA                      1    8133   13600     259     372   38.99     0.0         125   214    68   319    22   239\n",
      "> 5kbrB                      1    8134   12000     260     372   37.70     0.0         126   187   113   319    50   240\n",
      "> 5kbqB                      1    8144   13300     273     372   36.09     0.0         127   225    74   319    24   253\n",
      "> 4kb8A                      1    8182    5600     265     372   29.51    0.76E-04     128   174    66   258    11   193\n",
      "> 4kbaA                      1    8197    5600     263     372   29.51    0.75E-04     129   174    66   258    11   193\n",
      "> 5a4eC                      1    8266    5700     262     372   27.61    0.45E-04     130   129   134   265    99   232\n",
      "> 4m3qB                      1    8337    5900     236     372   30.60    0.14E-04     131   133   157   312    92   225\n",
      "> 2cn8A                      1    8387   11000     266     372   35.78     0.0         132   211    92   319    33   250\n",
      "> 6cnhA                      1    8423    4950     321     372   33.33    0.27E-02     133    86   180   284   148   234\n",
      "> 5drbA                      1    8478   10950     273     372   33.66     0.0         134   193   122   319    71   272\n",
      "> 6srhA                      1    8601    5750     288     372   29.75    0.39E-04     135   103   140   248    80   200\n",
      "> 6srhB                      1    8602    6000     295     372   28.79    0.11E-04     136   109   140   248    76   207\n",
      "> 2srcA                      1    8620    8500     449     372   30.14    0.52E-10     137   142   160   308   280   425\n",
      "> 3qklA                      1    8798   11150     317     372   29.30     0.0         138   243    66   320    11   266\n",
      "> 2qkrA                      1    8799    9650     273     372   27.86     0.0         139   235    65   319     7   268\n",
      "> 2qkwB                      1    8812   10850     290     372   29.77     0.0         140   241    65   313    15   276\n",
      "> 5ek7A                      1    8901    6800     286     372   24.70    0.18E-06     141   240    63   313    29   279\n",
      "> 4r1yA                      1    8974    7100     265     372   30.14    0.35E-07     142   140   166   313   108   253\n",
      "> 4r1vA                      1    8982    7450     289     372   28.00    0.65E-08     143   232    67   313    29   278\n",
      "> 4l44A                      1    9048   10300     321     372   27.56     0.0         144   232    66   319    15   268\n",
      "> 5l4qA                      1    9081    6900     293     372   24.16    0.11E-06     145   242    63   331    15   283\n",
      "> 5vamA                      1    9296    9900     269     372   30.61     0.0         146   232    68   313    15   259\n",
      "> 4e1zA                      1    9408    7600     288     372   27.92    0.30E-08     147   180   130   313    81   277\n",
      "> 3ag9B                      1    9450    8500     317     372   27.90    0.30E-10     148   226    66   306    33   265\n",
      "> 4aguA                      1    9480   10500     277     372   27.44     0.0         149   237    68   319     9   274\n",
      "> 5lohA                      1    9652   10550     248     372   33.16     0.0         150   189   114   318    40   229\n",
      "> 3enmA                      1    9848   14650     279     372   36.02     0.0         151   243    68   327    14   274\n",
      "> 3enmD                      1    9849   14650     282     372   36.02     0.0         152   243    68   327    17   277\n",
      "> 3kc3D                      1    9921    7750     269     372   28.28    0.13E-08     153   233    59   323    14   257\n",
      "> 5kcvA                      1    9981    9450     353     372   33.81     0.0         154   134   166   320   187   325\n",
      "> 4au8A                      1   10015   10800     276     372   29.28     0.0         155   234    68   318    11   273\n",
      "> 2w9zB                      1   10095    9400     287     372   26.81     0.0         156   243    68   317    11   286\n",
      "> 2w96B                      1   10121    9450     267     372   27.52     0.0         157   238    68   317     9   266\n",
      "> 4w9wA                      1   10140    6700     306     372   23.47    0.32E-06     158   252    63   330    14   290\n",
      "> 3kulA                      1   10258    7000     267     372   28.21    0.59E-07     159   186    65   258    12   206\n",
      "> 2weiA                      1   10311   11450     278     372   32.41     0.0         160   238    67   319    21   273\n",
      "> 2welA                      1   10361    9150     304     372   27.60     0.0         161   179   136   317    72   263\n",
      "> 6lnmA                      1   10475   10350     325     372   33.33     0.0         162   179   166   349   119   301\n",
      "> 4yr8F                      1   10488    8550     300     372   30.54    0.22E-10     163   205    87   317    35   273\n",
      "> 2w4oA                      1   10551    9300     278     372   32.00     0.0         164   222    68   320    19   243\n",
      "> 3fxzA                      1   10674   15750     292     372   35.37     0.0         165   241    68   319    28   273\n",
      "> 5eymA                      1   11110   16500     292     372   39.29     0.0         166   240    58   319    22   273\n",
      "> 2reiA                      1   11236    6000     277     372   25.75    0.10E-04     167   224    65   313    30   262\n",
      "> 4redA                      1   11260    8100     327     372   33.53    0.27E-09     168   166   129   319    75   247\n",
      "> 4rerA                      1   11279    8300     458     372   30.27    0.15E-09     169   182   129   319    76   260\n",
      "> 3repA                      1   11290    5100     268     372   26.21    0.99E-03     170   195    96   307    48   253\n",
      "> 1kwpA                      1   11465    7050     319     372   28.11    0.57E-07     171   266    59   366    14   294\n",
      "> 4fr4A                      1   11550   10750     350     372   29.75     0.0         172   232    66   306    14   255\n",
      "> 5hu3A                      1   11593    8700     266     372   26.49    0.75E-11     173   172   143   317    81   265\n",
      "> 5hu9A                      1   11638    9350     270     372   30.36     0.0         174   237    58   312    14   260\n",
      "> 1u59A                      1   11678    8000     282     372   28.15    0.37E-09     175   229    68   307    17   254\n",
      "> 1u5qA                      1   11692   15700     308     372   34.50     0.0         176   245    61   317    11   268\n",
      "> 6u5lA                      1   11736    9300     276     372   24.90     0.0         177   241    68   319    12   272\n",
      "> 1phkA                      1   11875   11000     277     372   26.97     0.0         178   243    65   319     8   274\n",
      "> 4itjA                      1   11942    8850     261     372   33.80    0.75E-11     179   137   163   308   103   244\n",
      "> 4itjB                      1   11943    8500     276     372   32.43    0.30E-10     180   141   163   308   112   259\n",
      "> 5itaA                      1   11972   10650     250     372   33.33     0.0         181   218    68   313    16   243\n",
      "> 4uwcA                      1   12129    8500     301     372   35.81    0.30E-10     182   141   165   314   141   288\n",
      "> 4y83A                      1   12341    9900     299     372   30.21     0.0         183   224    68   315    61   295\n",
      "> 4y85A                      1   12343   10000     307     372   28.08     0.0         184   277    19   315    11   302\n",
      "> 4y8dA                      1   12348    6700     262     372   23.98    0.27E-06     185   231    65   314    17   262\n",
      "> 5y80A                      1   12366    6950     306     372   24.15    0.90E-07     186   234    65   313    20   284\n",
      "> 5y86A                      1   12378    7600     391     372   28.09    0.43E-08     187   224    66   301    76   310\n",
      "> 4zslA                      1   12605    9700     341     372   26.92     0.0         188   244    68   317     9   294\n",
      "> 5jzjA                      1   12675    9150     271     372   30.92     0.0         189   148   163   317   118   269\n",
      "> 4wb8A                      1   12751    8550     334     372   28.45    0.30E-10     190   225    66   306    34   265\n",
      "> 4wb7A                      1   12760    8650     402     372   26.62    0.22E-10     191   271    19   306    56   333\n",
      "> 4jlcA                      1   12783    6600     646     372   28.29    0.13E-05     192   184    67   262    15   219\n",
      "> 3vs1A                      1   12837    8550     424     372   30.94    0.37E-10     193   213    68   308   180   402\n",
      "> 3qd2B                      1   13021    9500     271     372   28.91     0.0         194   239    68   319    14   269\n",
      "> 4usfA                      1   13249   14500     277     372   35.46     0.0         195   244    55   317     8   258\n",
      "> 4joaA                      1   13288    5100     251     372   26.00    0.91E-03     196   146   167   329    98   247\n",
      "> 6joiA                      1   13347    7050     296     372   27.43    0.52E-07     197   225    66   308    29   265\n",
      "> 2zv7A                      1   13415    8550     257     372   29.57    0.22E-10     198   220    68   308    16   245\n",
      "> 5dbxA                      1   13524   13000     284     372   33.33     0.0         199   244    65   319    13   270\n",
      "> 5dbxB                      1   13525   13150     282     372   33.85     0.0         200   244    65   319    13   269\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     27471922   26828.049    26.199\n",
      "> 6db4A                      1   13527    9500     276     372   28.06     0.0         201   235    68   312    15   267\n",
      "> 3dlsA                      1   13639    8500     285     372   33.11    0.30E-10     202   148   155   317   119   269\n",
      "> 2ofvA                      1   13904    7550     244     372   32.12    0.32E-08     203   130   154   308    96   232\n",
      "> 4cfeC                      1   14104    8050     403     372   29.76    0.45E-09     204   167   145   319    92   259\n",
      "> 4cfhA                      1   14154    8150     401     372   29.73    0.27E-09     205   182   129   319    75   259\n",
      "> 3g0eA                      1   14233    7650     332     372   29.35    0.28E-08     206   176   118   314   140   323\n",
      "> 3sxsA                      1   14380    8300     264     372   30.71    0.75E-10     207   232    68   313    16   256\n",
      "> 2w5aA                      1   14465   10050     267     372   34.00     0.0         208   150   156   316   108   257\n",
      "> 2a19B                      1   14568   10850     263     372   31.25     0.0         209   224    67   314    17   256\n",
      "> 2a19C                      1   14569    7250     203     372   34.25    0.12E-07     210   171    67   314    16   196\n",
      "> 4ic8A                      1   14614    8950     325     372   30.64     0.0         211   147   160   317   111   283\n",
      "> 5icpA                      1   14642    9500     343     372   25.86     0.0         212   242    68   318    27   316\n",
      "> 6khfA                      1   14735    6700     327     372   24.02    0.35E-06     213   160   160   322   119   322\n",
      "> 3mdyA                      1   14842    5100     320     372   32.65    0.12E-02     214    84   157   246   126   223\n",
      "> 6aamA                      1   15068    8950     269     372   29.32     0.0         215   235    44   313     9   257\n",
      "> 4aaaA                      1   15079   11950     295     372   30.60     0.0         216   237    67   319    14   281\n",
      "> 6di1A                      1   15153    7450     271     372   28.00    0.60E-08     217   244    61   315    13   262\n",
      "> 6di9A                      1   15155    6200     252     372   28.41    0.33E-05     218   173   129   315    68   243\n",
      "> 3kk8A                      1   15226    8650     283     372   26.44    0.15E-10     219   195   136   335    74   281\n",
      "> 4aszA                      1   15397    8650     289     372   26.64    0.15E-10     220   226    87   327    40   283\n",
      "> 6pnxA                      1   15485    8750     291     372   35.00    0.75E-11     221   150   156   314   121   280\n",
      "> 5knjA                      1   15489    5600     252     372   29.77    0.71E-04     222   128   170   314   118   248\n",
      "> 4knbD                      1   15557    7050     247     372   32.87    0.41E-07     223   208    67   313    20   235\n",
      "> 3kn6A                      1   15561    6700     282     372   34.40    0.29E-06     224   125   157   306   104   228\n",
      "> 3ddqC                      1   15694   12550     266     372   33.20     0.0         225   239    68   323    11   260\n",
      "> 1cm8A                      1   15832    9500     327     372   29.26     0.0         226   161   153   319   110   297\n",
      "> 2cmwA                      1   15857    4850     294     372   25.50    0.40E-02     227   233    68   307     7   257\n",
      "> 5t3qA                      1   16228    5800     278     372   28.32    0.29E-04     228   213    71   313    36   261\n",
      "> 3m2wA                      1   16332    6700     290     372   27.01    0.30E-06     229   260    59   366    13   286\n",
      "> 3qc4B                      1   16360   13500     278     372   32.69     0.0         230   253    51   322     6   265\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 3qc9B                      1   16364   11750     481     372   30.80     0.0         231   248    66   319   161   423\n",
      "> 3lxpA                      1   16502    9600     285     372   28.85     0.0         232   239    66   313    14   273\n",
      "> 5lxcA                      1   16542    7500     383     372   25.81    0.70E-08     233   246    66   319    74   383\n",
      "> 5lxcB                      1   16543    7500     385     372   25.81    0.71E-08     234   246    66   319    74   383\n",
      "> 4xbrA                      1   16576   13850     315     372   31.92     0.0         235   255    68   332    54   313\n",
      "> 3a62A                      1   16727   10000     263     372   29.06     0.0         236   218    66   319    11   244\n",
      "> 6s14A                      1   16781    5950     339     372   26.78    0.17E-04     237   174   104   297    80   262\n",
      "> 6s1bA                      1   16811    5950     337     372   26.78    0.17E-04     238   174   104   297    80   262\n",
      "> 2bdjA                      1   17057    8300     254     372   31.85    0.75E-10     239   134   160   308   107   241\n",
      "> 6bdnA                      1   17063   15500     307     372   34.54     0.0         240   241    68   319    23   271\n",
      "> 2bdwA                      1   17071    8700     309     372   23.66    0.15E-10     241   212   136   351    74   297\n",
      "> 6ed6B                      1   17191    9150     364     372   28.72     0.0         242   271    19   319    29   310\n",
      "> 2q0nA                      1   17349   13850     293     372   31.92     0.0         243   255    68   332    30   289\n",
      "> 2qg5A                      1   17460   10300     284     372   29.13     0.0         244   242    63   319    27   280\n",
      "> 6qg7A                      1   17482   14600     282     372   30.92     0.0         245   259    40   319     1   262\n",
      "> 6okoA                      1   17587    5950     263     372   32.79    0.12E-04     246   119   166   308   111   232\n",
      "> 6qp5A                      1   17606    8500     266     372   31.93    0.30E-10     247   163   143   339    97   262\n",
      "> 1qpcA                      1   17609    8450     271     372   30.72    0.37E-10     248   143   154   308   103   255\n",
      "> 4qpsA                      1   17667    9150     270     372   26.72     0.0         249   235    68   312    15   261\n",
      "> 3hkoA                      1   17772    8650     319     372   33.92    0.15E-10     250   159   149   317   137   307\n",
      "> 6hknA                      1   17840    9050     333     372   28.72     0.0         251   157   160   317    99   286\n",
      "> 4zloB                      1   17880   12900     263     372   37.78     0.0         252   176   136   319    63   242\n",
      "> 1t46A                      1   17930    7750     297     372   27.91    0.14E-08     253   236    66   314    29   286\n",
      "> 2qokA                      1   17970    6200     281     372   24.15    0.38E-05     254   227    66   313    22   257\n",
      "> 2qolA                      1   18015    5200     273     372   24.45    0.61E-03     255   220    66   313    20   248\n",
      "> 4ynoA                      1   18137   11050     342     372   26.98     0.0         256   276    37   335     2   316\n",
      "> 4ynzA                      1   18147    7050     313     372   35.35    0.55E-07     257    98   161   273   112   210\n",
      "> 1tkiA                      1   18164    7400     321     372   29.41    0.95E-08     258   148   166   319   110   262\n",
      "> 3tkuA                      1   18179    9000     395     372   27.55     0.0         259   250    66   320    79   343\n",
      "> 3tkiA                      1   18204    8400     272     372   32.74    0.45E-10     260   109   154   263    93   205\n",
      "> 4pdoA                      1   18257    5500     261     372   25.65    0.12E-03     261   225    63   313    25   254\n",
      "> 4pdpA                      1   18258   12000     261     372   33.96     0.0         262   209    92   319    27   238\n",
      "> 4pdsA                      1   18259   10650     249     372   33.50     0.0         263   194   103   319    33   229\n",
      "> 5j0aA                      1   18315    7300     299     372   24.01    0.15E-07     264   247    68   314    16   294\n",
      "> 2j0iA                      1   18324   13850     289     372   31.92     0.0         265   255    68   332    28   287\n",
      "> 2j0jA                      1   18327    8800     611     372   29.54    0.15E-10     266   227    66   312   361   597\n",
      "> 2j0kA                      1   18343    7750     598     372   29.48    0.34E-08     267   230    58   312   334   584\n",
      "> 6ae3C                      1   18499    9050     331     372   25.98     0.0         268   255    52   318     8   288\n",
      "> 4i93A                      1   18584    6550     289     372   26.45    0.65E-06     269   224    69   313    27   268\n",
      "> 4i93B                      1   18585    5950     283     372   26.69    0.14E-04     270   218    69   313    27   262\n",
      "> 4u3yA                      1   18706   14700     295     372   32.57     0.0         271   246    67   319    14   274\n",
      "> 4u3zA                      1   18707   13400     282     372   33.33     0.0         272   229    81   319    22   261\n",
      "> 1p4oA                      1   18774    8050     308     372   27.78    0.32E-09     273   244    55   313    17   286\n",
      "> 4p4cA                      1   18804    5300     266     372   24.35    0.35E-03     274   221    66   313    13   242\n",
      "> 6b5jD                      1   19315    8350     257     372   29.95    0.60E-10     275   194    51   259     1   197\n",
      "> 5hhwA                      1   19398    7700     306     372   28.11    0.19E-08     276   251    55   318    14   294\n",
      "> 6hhjA                      1   19416   11200     410     372   29.57     0.0         277   243    66   320   120   376\n",
      "> 6hhgA                      1   19428   11100     386     372   33.55     0.0         278   147   166   320   201   352\n",
      "> 2ozoA                      1   19539    7800     545     372   29.59    0.23E-08     279   192    95   307   321   516\n",
      "> 2ozaA                      1   19552    9250     332     372   27.89     0.0         280   280    59   366     9   302\n",
      "> 3oz6A                      1   19584    8900     343     372   24.82    0.75E-11     281   232    82   317    22   299\n",
      "> 1i44A                      1   19698    7600     282     372   30.94    0.29E-08     282   238    55   318     6   270\n",
      "> 4i4eA                      1   19747    7350     254     372   30.26    0.92E-08     283   219    66   312    13   240\n",
      "> 5u6bA                      1   19957    8050     270     372   30.00    0.28E-09     284   233    57   312     7   256\n",
      "> 5u6bB                      1   19958    8900     291     372   28.79    0.75E-11     285   247    57   312    16   279\n",
      "> 3comA                      1   19999   16550     264     372   34.52     0.0         286   249    74   342    10   261\n",
      "> 3comB                      1   20000   16600     279     372   34.94     0.0         287   263    54   342     6   274\n",
      "> 3cokA                      1   20060    8650     234     372   29.52    0.75E-11     288   223    55   317     6   232\n",
      "> 2oo8X                      1   20124    7350     259     372   38.17    0.95E-08     289   124   167   313   101   231\n",
      "> 6p5sA                      1   20449    4700     331     372   24.76    0.99E-02     290   172   160   347   105   310\n",
      "> 5ig1A                      1   20592   10600     300     372   28.85     0.0         291   252    60   317    13   272\n",
      "> 3miyA                      1   20824    6450     239     372   28.07    0.86E-06     292   223    59   314     3   230\n",
      "> 3mi9A                      1   20836    8450     328     372   24.82    0.45E-10     293   247    68   318    18   299\n",
      "> 4bfmA                      1   20876    7700     313     372   33.10    0.20E-08     294   141   155   317   103   244\n",
      "> 6bfnA                      1   20886    6650     301     372   33.71    0.41E-06     295    88   162   255   120   208\n",
      "> 6n0qA                      1   20913    9550     257     372   32.61     0.0         296   219    68   313    18   247\n",
      "> 6n0qB                      1   20914   10200     254     372   32.75     0.0         297   219    68   313    16   244\n",
      "> 3gopA                      1   21124    7950     300     372   27.31    0.52E-09     298   235    52   313    24   261\n",
      "> 1q5kA                      1   21185    9100     345     372   27.60     0.0         299   247    63   318    23   301\n",
      "> 3q5iA                      1   21189    9950     470     372   27.46     0.0         300   241    71   319    36   279\n",
      "> 4fg8A                      1   21233    9300     263     372   32.68     0.0         301   150   163   334   102   254\n",
      "> 4fgbA                      1   21262   11850     282     372   33.53     0.0         302   167   163   334   104   273\n",
      "> 4fg7A                      1   21269    9650     250     372   33.10     0.0         303   142   163   317   104   248\n",
      "> 4fg9A                      1   21319    9000     278     372   32.24     0.0         304   149   163   334   103   254\n",
      "> 1zy4A                      1   21329    8550     271     372   31.38    0.22E-10     305   226    67   313    13   251\n",
      "> 1zy4B                      1   21330    7950     249     372   30.87    0.42E-09     306   219    67   313    13   242\n",
      "> 3fe3A                      1   21407    8950     317     372   29.10     0.0         307   239    66   314    11   254\n",
      "> 4feqA                      1   21453    8650     224     372   35.12    0.75E-11     308   197    63   308     5   209\n",
      "> 6ferA                      1   21479    8200     271     372   25.20    0.13E-09     309   244    63   312    12   265\n",
      "> 6ferI                      1   21480    7900     267     372   25.79    0.59E-09     310   241    63   312    10   261\n",
      "> 6fexA                      1   21482    7650     289     372   25.09    0.23E-08     311   241    65   312    12   282\n",
      "> 2yexA                      1   21667    8400     269     372   32.74    0.45E-10     312   109   154   263   100   212\n",
      "> 2c0tA                      1   21796    8600     427     372   27.62    0.30E-10     313   276     3   308   120   405\n",
      "> 3c0iA                      1   21803   10300     298     372   36.36     0.0         314   150   166   319   119   272\n",
      "> 4c0tA                      1   21812   11800     290     372   34.13     0.0         315   158   155   319   104   270\n",
      "> 3c0gB                      1   21830   10350     320     372   33.33     0.0         316   179   166   349   120   302\n",
      "> 6t2wA                      1   21908    7550     304     372   27.24    0.41E-08     317   233    66   312    38   294\n",
      "> 4x3jA                      1   22216    6950     234     372   36.22    0.65E-07     318   120   167   313    80   206\n",
      "> 5x3fB                      1   22242    8800     348     372   28.31    0.75E-11     319   263    24   306    41   312\n",
      "> 3mn3A                      1   22328    9800     252     372   32.17     0.0         320   228    66   319    10   239\n",
      "> 6mnhA                      1   22384   10750     265     372   32.62     0.0         321   183   121   317    66   252\n",
      "> 3plsA                      1   22421    7150     298     372   24.80    0.31E-07     322   228    64   308    25   274\n",
      "> 4pl3A                      1   22434   10350     388     372   33.06     0.0         323   230    65   318    15   256\n",
      "> 4pl4D                      1   22436    9950     377     372   32.78     0.0         324   229    65   318    13   253\n",
      "> 4b4lA                      1   22510   10250     299     372   27.85     0.0         325   208   100   319    54   272\n",
      "> 3nynA                      1   22612   10400     553     372   32.50     0.0         326   153   161   319   286   445\n",
      "> 3nyvA                      1   22639   13900     457     372   30.25     0.0         327   299    65   371    13   326\n",
      "> 3rcdA                      1   22656    9150     281     372   31.00     0.0         328   217    66   313    15   243\n",
      "> 6rcgA                      1   22704    4850     295     372   28.87    0.40E-02     329   135    66   207    15   156\n",
      "> 5dewB                      1   23244   15300     281     372   36.67     0.0         330   235    68   319    22   261\n",
      "> 3gniB                      1   23553    9450     304     372   33.17     0.0         331   186   115   319    81   279\n",
      "> 5wngB                      1   23706    8350     365     372   29.11    0.82E-10     332   230    66   319    76   312\n",
      "> 4wnoA                      1   23727   10250     272     372   29.41     0.0         333   190   121   317    66   269\n",
      "> 4wnkA                      1   23732   12050     499     372   28.85     0.0         334   243    66   319   167   426\n",
      "> 5wnlA                      1   23748    8450     262     372   30.67    0.37E-10     335   139   162   312   104   253\n",
      "> 5wnfB                      1   23750    8050     358     372   30.00    0.39E-09     336   223    66   319    76   305\n",
      "> 5wnfC                      1   23751    9200     386     372   29.88     0.0         337   238    66   319    77   327\n",
      "> 2wntA                      1   23757    8450     281     372   34.75    0.37E-10     338   136   173   317   107   247\n",
      "> 5wnoA                      1   23773    8450     302     372   27.84    0.37E-10     339   247    51   321     3   257\n",
      "> 3qupA                      1   23809    8300     241     372   33.04    0.67E-10     340   215    59   308     4   227\n",
      "> 2qu6A                      1   23810    6950     283     372   33.59    0.82E-07     341   130   165   315   134   264\n",
      "> 2qu6B                      1   23811    6450     260     372   32.82    0.95E-06     342   130   165   315   124   254\n",
      "> 2y6oA                      1   23885    4700     263     372   29.67    0.75E-02     343   169    65   258    16   197\n",
      "> 6y6hA                      1   23901    9600     292     372   24.89     0.0         344   213   100   319    57   277\n",
      "> 4ff8A                      1   23954    6450     169     372   33.87    0.57E-06     345   123   157   308    34   157\n",
      "> 2exeA                      1   24106    6300     262     372   23.11    0.21E-05     346   160   160   322    48   259\n",
      "> 5hesB                      1   24249    6600     278     372   31.54    0.48E-06     347   128   167   313   108   237\n",
      "> 5li9A                      1   24365   10950     329     372   29.64     0.0         348   240    66   319    12   264\n",
      "> 3lijA                      1   24381   13000     465     372   32.26     0.0         349   205   163   371   119   335\n",
      "> 5tx3B                      1   24467    7250     300     372   35.04    0.19E-07     350   137   155   317    92   228\n",
      "> 3txoA                      1   24512   10600     317     372   28.23     0.0         351   230    66   319     7   254\n",
      "> 3p1aA                      1   24542    7750     281     372   30.13    0.14E-08     352   232    68   319    41   279\n",
      "> 6p1lC                      1   24596    9150     280     372   29.65     0.0         353   222    66   313    15   240\n",
      "> 5z1dA                      1   24725   14100     274     372   32.47     0.0         354   228    71   319    22   252\n",
      "> 3v8sA                      1   24968    9550     395     372   28.85     0.0         355   243    66   319    73   332\n",
      "> 4uy9A                      1   25034    8600     313     372   28.91    0.22E-10     356   241    63   313    11   266\n",
      "> 4uyaA                      1   25035    7450     274     372   31.16    0.61E-08     357   133   166   314   101   238\n",
      "> 4uynA                      1   25039   12050     249     372   33.33     0.0         358   221    66   317    13   240\n",
      "> 5uyjA                      1   25047   12350     281     372   37.95     0.0         359   162   144   317   112   277\n",
      "> 6uyaC                      1   25053    7100     275     372   32.35    0.37E-07     360   167    68   254    31   200\n",
      "> 1jpaA                      1   25150    4750     273     372   24.00    0.61E-02     361   124   167   313   134   258\n",
      "> 6jpeA                      1   25219    9000     298     372   35.81     0.0         362   142   165   314   141   288\n",
      "> 6sp9A                      1   25228   10450     348     372   29.04     0.0         363   237    68   318    26   297\n",
      "> 4zjvB                      1   25323    9650     273     372   30.45     0.0         364   234    66   313    14   256\n",
      "> 5zj6A                      1   25330    8700     271     372   29.66    0.75E-11     365   223    68   308    21   256\n",
      "> 5tobA                      1   25407    6450     245     372   26.43    0.89E-06     366   220    61   313     5   231\n",
      "> 5tosA                      1   25419    5950     279     372   24.44    0.13E-04     367   247    33   314     1   266\n",
      "> 4qnyA                      1   25442    8700     333     372   25.60    0.15E-10     368   291    45   346     1   332\n",
      "> 5byzA                      1   25679    9700     348     372   26.92     0.0         369   244    68   317    14   299\n",
      "> 1bygA                      1   25717    5350     246     372   32.28    0.25E-03     370   124   167   312   108   234\n",
      "> 6dkiA                      1   25831    8350     279     372   31.15    0.60E-10     371   224    68   312    17   260\n",
      "> 3dkcA                      1   25919    7900     312     372   27.38    0.71E-09     372   234    67   313    33   284\n",
      "> 1ywnA                      1   25938    6200     279     372   30.28    0.37E-05     373   140   165   315   126   267\n",
      "> 5vczA                      1   26047    8300     294     372   28.57    0.82E-10     374   238    68   319    49   293\n",
      "> 6bqqA                      1   26409   11700     271     372   31.42     0.0         375   241    63   317     8   268\n",
      "> 4mh7A                      1   26800    6200     248     372   28.25    0.32E-05     376   211    66   312    15   237\n",
      "> 2h34A                      1   27011    6000     243     372   28.85    0.88E-05     377   103   182   306   124   227\n",
      "> 5oteA                      1   27074    9250     407     372   27.76     0.0         378   250    66   320    82   344\n",
      "> 4otdA                      1   27075   12000     334     372   31.37     0.0         379   241    67   319    16   270\n",
      "> 3ot8A                      1   27082    8400     255     372   32.74    0.45E-10     380   109   154   263    84   196\n",
      "> 4othA                      1   27094    9050     305     372   32.60     0.0         381   213    67   319    15   241\n",
      "> 4otgA                      1   27117   12000     322     372   31.37     0.0         382   241    67   319    16   270\n",
      "> 3is5A                      1   27248   11000     260     372   29.37     0.0         383   239    60   319     8   259\n",
      "> 3is5D                      1   27249    9550     236     372   31.00     0.0         384   216    60   319     8   236\n",
      "> 3is5E                      1   27250   10050     252     372   29.17     0.0         385   224    60   315    11   250\n",
      "> 5isoA                      1   27295    8700     453     372   30.18    0.15E-10     386   168   145   319    93   261\n",
      "> 5isoC                      1   27297    8700     429     372   30.18    0.15E-10     387   168   145   319    93   261\n",
      "> 6xx8A                      1   27350    7900     327     372   23.76    0.75E-09     388   243    66   318    30   311\n",
      "> 1mrvA                      1   27470    8350     261     372   35.34    0.60E-10     389   128   166   318    98   230\n",
      "> 2r0iA                      1   27568    7750     300     372   30.70    0.15E-08     390   222    66   314    10   237\n",
      "> 5n1fA                      1   27638    8600     337     372   27.27    0.22E-10     391   257    33   306     5   268\n",
      "> 3hngA                      1   27687    8050     288     372   34.69    0.30E-09     392   141   165   314   136   282\n",
      "> 5hniX                      1   27743    6800     245     372   30.56    0.15E-06     393   173   117   313    44   223\n",
      "> 5hnvA                      1   27745    6350     283     372   26.88    0.18E-05     394   178   126   315    97   282\n",
      "> 5ng0A                      1   27794    8150     274     372   27.76    0.17E-09     395   231    66   313    17   261\n",
      "> 5ng0B                      1   27795    9150     282     372   27.67     0.0         396   239    66   313    18   270\n",
      "> 6ng0B                      1   27813   12650     279     372   31.38     0.0         397   233    74   315    19   257\n",
      "> 5uq3A                      1   27955   11300     270     372   30.65     0.0         398   233    84   323     9   269\n",
      "> 5dvrA                      1   28025   13250     387     372   29.61     0.0         399   294    65   368    11   314\n",
      "> 3n9xA                      1   28185    8050     374     372   26.10    0.41E-09     400   242    65   319    21   315\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     27786522   27135.275    26.499\n",
      "> 6emhA                      1   28287    9750     344     372   28.03     0.0         401   218    87   317    39   302\n",
      "> 6emhC                      1   28288    9150     334     372   28.96     0.0         402   213    87   317    40   298\n",
      "> 1fotA                      1   28341   11050     299     372   29.08     0.0         403   239    66   319    10   260\n",
      "> 2fo0A                      1   28351    9900     465     372   29.59     0.0         404   257    58   329   192   458\n",
      "> 3aqvA                      1   28470    7900     267     372   30.67    0.59E-09     405   162   145   319    95   257\n",
      "> 3npcA                      1   28513   10550     357     372   26.99     0.0         406   242    68   317    26   314\n",
      "> 4oh4A                      1   28729    9750     299     372   30.20     0.0         407   240    67   313    37   291\n",
      "> 3ohtA                      1   28780    9450     331     372   28.63     0.0         408   227    68   318    27   288\n",
      "> 2eu9A                      1   28900    6100     345     372   22.69    0.80E-05     409   160   160   322   125   340\n",
      "> 4euuA                      1   28941    6750     307     372   27.49    0.25E-06     410   183    67   262    15   225\n",
      "> 4eutA                      1   28951    6950     384     372   27.14    0.12E-06     411   184    67   262    15   224\n",
      "> 3nupA                      1   29074   10300     252     372   30.45     0.0         412   217    74   318    10   252\n",
      "> 1yojA                      1   29195    6500     234     372   32.00    0.65E-06     413   124   160   308    85   209\n",
      "> 1yojB                      1   29196    7300     215     372   32.28    0.98E-08     414   126   160   308    64   190\n",
      "> 5nxdA                      1   29370    6700     271     372   35.11    0.28E-06     415   128   167   312   106   236\n",
      "> 1nxkA                      1   29384    7950     290     372   27.52    0.50E-09     416   247    59   323    18   275\n",
      "> 1nxkB                      1   29385    8100     283     372   28.06    0.22E-09     417   239    59   323    16   268\n",
      "> 1nxkC                      1   29386    8150     306     372   28.22    0.19E-09     418   271    59   362    19   305\n",
      "> 4csvA                      1   29535    8400     244     372   29.52    0.37E-10     419   223    63   313    10   236\n",
      "> 5ew8A                      1   29648    8600     291     372   34.57    0.15E-10     420   152   154   314   117   278\n",
      "> 4yzbA                      1   29796   13350     466     372   30.15     0.0         421   262    65   332    24   295\n",
      "> 4yznA                      1   29797    8550     269     372   26.29    0.22E-10     422   237    68   314    14   264\n",
      "> 4yz9B                      1   29811    9950     375     372   34.47     0.0         423   223    65   318    12   246\n",
      "> 5kmiA                      1   29944    8100     300     372   35.86    0.25E-09     424   138   166   312   137   281\n",
      "> 3uc4A                      1   29992    9100     295     372   38.16     0.0         425   146   157   318   103   254\n",
      "> 3uc4B                      1   29993    9250     277     372   38.56     0.0         426   147   157   318   100   252\n",
      "> 3uc3A                      1   30016    7250     272     372   35.86    0.17E-07     427   140   157   318   105   249\n",
      "> 5zn2A                      1   30147    7850     324     372   23.30    0.96E-09     428   244    66   318    41   319\n",
      "> 2ycfA                      1   30165   13050     286     372   34.94     0.0         429   241    66   319    15   263\n",
      "> 5kx7A                      1   30366    7050     237     372   32.92    0.39E-07     430   158    68   254     1   161\n",
      "> 3q6uA                      1   30438    7250     287     372   29.31    0.18E-07     431   219    67   313    36   267\n",
      "> 6o8bA                      1   30815    6750     656     372   26.89    0.63E-06     432   184    67   262    20   231\n",
      "> 2vz6A                      1   30895    9800     291     372   26.57     0.0         433   194   143   339    78   284\n",
      "> 6vzkA                      1   30910    9200     258     372   28.18     0.0         434   171   143   317    74   254\n",
      "> 1vzoA                      1   30943    9450     319     372   26.89     0.0         435   249    66   319    30   293\n",
      "> 1pkgB                      1   31013    7550     277     372   28.98    0.37E-08     436   229    66   314    26   270\n",
      "> 2pk9C                      1   31042   10300     266     372   28.40     0.0         437   220    87   318    14   263\n",
      "> 1lufA                      1   31316    8100     275     372   29.51    0.22E-09     438   222    66   308    19   262\n",
      "> 4ludB                      1   31350    8450     409     372   31.22    0.60E-10     439   211    68   308   169   389\n",
      "> 4d4rA                      1   31596    7000     251     372   30.09    0.55E-07     440   219    66   312    13   238\n",
      "> 4d4rB                      1   31597    8500     270     372   29.51    0.30E-10     441   231    66   312    15   258\n",
      "> 2j90A                      1   31710    9850     263     372   27.27     0.0         442   205   100   319    53   261\n",
      "> 4j99A                      1   31715    7400     280     372   34.03    0.80E-08     443   136   165   314   128   271\n",
      "> 5n3fA                      1   31812    8050     318     372   31.11    0.34E-09     444   176   115   306    86   265\n",
      "> 6n33A                      1   31817    7350     292     372   27.00    0.11E-07     445   233    66   312    21   283\n",
      "> 6n3lA                      1   31825    8500     231     372   33.33    0.22E-10     446   206    66   314    11   226\n",
      "> 6n3oA                      1   31826   10400     252     372   34.04     0.0         447   223    66   314    11   245\n",
      "> 4pteA                      1   31990    9950     341     372   29.60     0.0         448   232    63   305    22   271\n",
      "> 2ptkA                      1   32002    7950     425     372   33.85    0.80E-09     449   129   160   308   277   406\n",
      "> 3ggfA                      1   32046   15700     275     372   36.65     0.0         450   245    68   342    14   264\n",
      "> 3ggfB                      1   32047   15550     270     372   36.80     0.0         451   244    68   342    13   262\n",
      "> 3lffA                      1   32297    9150     335     372   28.20     0.0         452   232    68   318    26   291\n",
      "> 4czuA                      1   32600   10100     297     372   29.64     0.0         453   242    66   319    17   269\n",
      "> 4czuB                      1   32601   10100     287     372   29.64     0.0         454   242    66   319    15   267\n",
      "> 5czoA                      1   32619    4900     365     372   31.91    0.40E-02     455   131    66   206    13   153\n",
      "> 6cz4A                      1   32627    9350     257     372   31.62     0.0         456   226    66   307    13   246\n",
      "> 2hw6A                      1   33003    8450     242     372   30.71    0.30E-10     457   231    51   320     2   242\n",
      "> 4qfrA                      1   33147    8150     361     372   29.73    0.24E-09     458   182   129   319    76   260\n",
      "> 4nstA                      1   33213   10500     322     372   28.47     0.0         459   247    67   319    17   297\n",
      "> 4nstC                      1   33214   10250     317     372   27.70     0.0         460   246    67   319    16   293\n",
      "> 6nssA                      1   33236    8100     303     372   35.86    0.25E-09     461   138   166   312   141   285\n",
      "> 3f61A                      1   33244    7500     271     372   26.29    0.46E-08     462   219    67   310    16   247\n",
      "> 3f66B                      1   33253    7150     269     372   30.04    0.28E-07     463   224    67   313    22   254\n",
      "> 4f6uA                      1   33294    8850     334     372   25.89    0.75E-11     464   233    68   318    29   310\n",
      "> 5es1A                      1   33378    8000     304     372   30.30    0.41E-09     465   225    66   314    10   240\n",
      "> 6es0A                      1   33399    7800     288     372   27.69    0.11E-08     466   228    66   313    18   259\n",
      "> 3gbzA                      1   33515    8550     274     372   28.50    0.22E-10     467   163   143   318    75   267\n",
      "> 4wuaA                      1   33606    6350     344     372   26.20    0.22E-05     468   161   152   317   113   341\n",
      "> 4wunA                      1   33615    7300     282     372   37.69    0.14E-07     469   126   165   314   140   269\n",
      "> 4wunB                      1   33616    7300     268     372   38.93    0.13E-07     470   127   165   314   128   258\n",
      "> 6egeD                      1   33763    6650     284     372   28.33    0.38E-06     471   177    68   254    31   210\n",
      "> 6egeA                      1   33764    6350     270     372   40.79    0.17E-05     472    76   169   254   119   194\n",
      "> 2f57A                      1   33797   13750     297     372   32.52     0.0         473   242    68   319    38   283\n",
      "> 4pwnA                      1   33893   10400     245     372   35.87     0.0         474   180   122   319    60   243\n",
      "> 4un0D                      1   33922    9700     276     372   30.30     0.0         475   231    67   319    10   273\n",
      "> 3eljA                      1   34031    9950     347     372   30.74     0.0         476   234    68   317    26   308\n",
      "> 4hcuA                      1   34192    7750     263     372   27.27    0.12E-08     477   235    68   314    14   255\n",
      "> 1kobA                      1   34256   10900     352     372   25.79     0.0         478   246    68   320    37   288\n",
      "> 1koaA                      1   34262    9300     447     372   23.69     0.0         479   246    68   317    34   282\n",
      "> 3gp0A                      1   34287    8500     333     372   26.32    0.37E-10     480   231    71   318    28   293\n",
      "> 4fnwA                      1   34401    7000     304     372   25.38    0.69E-07     481   246    70   336    37   300\n",
      "> 6fnkA                      1   34407    6000     274     372   22.73    0.10E-04     482   233    65   313    15   256\n",
      "> 6fnjA                      1   34414    5450     262     372   23.50    0.16E-03     483   225    65   313    12   245\n",
      "> 6fnhB                      1   34421    6350     271     372   24.49    0.17E-05     484   238    63   313    13   257\n",
      "> 4gt4A                      1   34477    6700     274     372   27.78    0.28E-06     485   104   166   270   121   228\n",
      "> 4pmtA                      1   34531    7400     273     372   35.00    0.78E-08     486   136   166   312   120   259\n",
      "> 2pmiA                      1   34533   10050     284     372   29.38     0.0         487   164   154   318    86   279\n",
      "> 2pmlX                      1   34537    9100     340     372   30.46     0.0         488   164   146   315   142   315\n",
      "> 4pmpA                      1   34539    8150     280     372   38.03    0.17E-09     489   135   166   312   125   266\n",
      "> 5x5oA                      1   34707    7650     291     372   31.47    0.23E-08     490   140   167   313   107   249\n",
      "> 3be2A                      1   34801    6950     290     372   33.59    0.84E-07     491   130   165   315   143   273\n",
      "> 3bhhA                      1   34891    9100     289     372   27.60     0.0         492   179   136   317    68   259\n",
      "> 3bhyA                      1   34902   10400     268     372   27.01     0.0         493   207   100   319    51   261\n",
      "> 6cqeA                      1   34960   13250     287     372   30.86     0.0         494   237    74   315    22   264\n",
      "> 6cq0A                      1   35027    5850     604     372   31.40    0.57E-04     495   160    67   262    15   186\n",
      "> 3bizA                      1   35286    7850     250     372   30.97    0.70E-09     496   216    80   315    22   247\n",
      "> 1bi8A                      1   35307   11950     264     372   29.69     0.0         497   227    68   319     9   264\n",
      "> 3dxnA                      1   35355   12000     258     372   31.91     0.0         498   249    60   323     1   257\n",
      "> 3fhrA                      1   35513    6950     273     372   28.05    0.78E-07     499   212    59   317     7   227\n",
      "> 6v66D                      1   35672   10200     314     372   28.00     0.0         500   239    66   313    16   265\n",
      "> 4f9cA                      1   35694    8050     306     372   28.83    0.32E-09     501   241    68   319    24   304\n",
      "> 4f9aA                      1   35737    7800     315     372   28.12    0.12E-08     502   241    68   319    26   313\n",
      "> 5f9eA                      1   35773   10700     334     372   28.97     0.0         503   239    66   319    11   262\n",
      "> 5f9eB                      1   35774   10700     293     372   28.97     0.0         504   239    66   319    10   261\n",
      "> 5qinA                      1   35868    6250     295     372   28.14    0.31E-05     505   176    67   248    10   208\n",
      "> 6babA                      1   36050    9350     290     372   25.70     0.0         506   201   136   339    70   283\n",
      "> 5xvuA                      1   36103    8250     321     372   23.13    0.12E-09     507   244    68   319    38   318\n",
      "> 4xv9A                      1   36130   10300     261     372   31.91     0.0         508   222    68   313    16   250\n",
      "> 3my1A                      1   36325    9050     314     372   24.73     0.0         509   248    68   318    20   302\n",
      "> 3my0U                      1   36372    5550     285     372   26.63    0.11E-03     510   142   170   314   111   279\n",
      "> 5my8A                      1   36385    6050     355     372   25.53    0.11E-04     511   161   152   317   118   352\n",
      "> 1j1bA                      1   36557    9100     354     372   28.52     0.0         512   234    63   305    23   278\n",
      "> 5j1wC                      1   36609    7400     316     372   25.13    0.93E-08     513   156   160   318   120   310\n",
      "> 5xzwA                      1   36682   10750     378     372   34.63     0.0         514   223    59   317   148   378\n",
      "> 5xzwB                      1   36683   12100     379     372   33.61     0.0         515   234    59   317   137   377\n",
      "> 2jbpJ                      1   36724    5700     225     372   28.57    0.37E-04     516   201    59   317    12   221\n",
      "> 2xrwA                      1   36772    8800     349     372   30.77    0.75E-11     517   224    68   317    32   304\n",
      "> 4z55A                      1   37033    6700     294     372   26.34    0.31E-06     518   244    66   329    28   289\n",
      "> 1xjdA                      1   37519   11250     280     372   29.08     0.0         519   239    66   319     8   258\n",
      "> 1ob3A                      1   37854   12450     277     372   30.34     0.0         520   237    68   319    10   276\n",
      "> 1ob3B                      1   37855   11800     258     372   32.53     0.0         521   223    68   319     9   257\n",
      "> 4oboA                      1   37899   14000     275     372   33.61     0.0         522   231    67   319    15   255\n",
      "> 3fbvA                      1   37967    9550     415     372   27.01     0.0         523   245    64   318     7   280\n",
      "> 2xirA                      1   38117    7300     296     372   30.00    0.14E-07     524   145   165   315   138   287\n",
      "> 4xi2A                      1   38125    7500     421     372   28.40    0.79E-08     525   244    61   315   165   414\n",
      "> 2xikA                      1   38154   17700     288     372   35.06     0.0         526   247    68   326    24   274\n",
      "> 4ysjA                      1   38259   12850     459     372   31.44     0.0         527   255    65   332    24   287\n",
      "> 6a32A                      1   38350    7400     336     372   25.95    0.10E-07     528   237    66   308    44   305\n",
      "> 2qr7A                      1   38463   10450     298     372   34.38     0.0         529   154   163   317   103   262\n",
      "> 2oj9A                      1   38649    8150     295     372   28.24    0.19E-09     530   244    55   313    11   272\n",
      "> 3zbfA                      1   38703    8500     282     372   26.89    0.30E-10     531   226    58   308    22   259\n",
      "> 6mzqB                      1   39020    7350     260     372   38.76    0.95E-08     532   125   165   314   123   251\n",
      "> 2r4bA                      1   39102    8600     292     372   26.75    0.15E-10     533   231    66   312    29   271\n",
      "> 3pg1A                      1   39186    6700     341     372   25.89    0.37E-06     534   234    63   319    20   301\n",
      "> 6b1uA                      1   39311    8050     441     372   29.76    0.50E-09     535   167   145   319    94   261\n",
      "> 6ib2A                      1   39447   13050     265     372   32.29     0.0         536   220    71   319    23   245\n",
      "> 3zclA                      1   39459    7750     291     372   28.11    0.14E-08     537   231    67   313    32   280\n",
      "> 3slsB                      1   39592   18300     295     372   38.98     0.0         538   248    58   319    21   274\n",
      "> 6slgA                      1   39599    8650     332     372   28.80    0.15E-10     539   152   165   319   114   297\n",
      "> 3wf6A                      1   39649   10500     277     372   28.46     0.0         540   227    66   319    11   256\n",
      "> 3wf7A                      1   39651   10000     288     372   27.45     0.0         541   232    66   319    13   267\n",
      "> 4i5mA                      1   39797   11900     273     372   30.68     0.0         542   238    59   318     3   253\n",
      "> 4i5pA                      1   39798   12700     284     372   30.53     0.0         543   248    59   318     3   264\n",
      "> 6i5hA                      1   39821    6900     325     372   24.26    0.12E-06     544   156   160   318   120   321\n",
      "> 3lzbA                      1   39832    9400     265     372   28.21     0.0         545   226    66   313    14   247\n",
      "> 3lzbC                      1   39833    9200     259     372   29.69     0.0         546   221    66   313    13   241\n",
      "> 3pozA                      1   40053    9300     293     372   29.74     0.0         547   229    66   313    16   247\n",
      "> 1uu3A                      1   40111   12850     277     372   34.40     0.0         548   237    66   322    14   263\n",
      "> 5uuuA                      1   40137    9950     414     372   27.78     0.0         549   267    40   319    56   343\n",
      "> 5l8jA                      1   40157   12300     253     372   32.62     0.0         550   227    66   317    12   244\n",
      "> 5l8lA                      1   40197   13600     263     372   31.84     0.0         551   239    66   317    12   256\n",
      "> 3a99A                      1   40305   10100     273     372   27.31     0.0         552   239    67   319    11   259\n",
      "> 3bbtB                      1   40459    8950     274     372   28.76     0.0         553   220    66   312    15   247\n",
      "> 3bbtD                      1   40460    8450     259     372   28.07    0.37E-10     554   217    66   312    14   241\n",
      "> 4bbmA                      1   40478   12150     314     372   32.69     0.0         555   179   139   319    87   294\n",
      "> 4bbmB                      1   40479   12150     292     372   32.69     0.0         556   179   139   319    65   272\n",
      "> 4af3A                      1   40640    9700     253     372   25.94     0.0         557   233    60   319     5   243\n",
      "> 6n7aA                      1   40921    8300     276     372   30.38    0.82E-10     558   145   166   314   112   269\n",
      "> 6n7aB                      1   40922    8650     292     372   30.00    0.15E-10     559   147   166   314   126   285\n",
      "> 6n7cA                      1   40931    8450     273     372   29.68    0.37E-10     560   145   166   314   112   266\n",
      "> 2yjrA                      1   41086    6450     281     372   26.05    0.10E-05     561   243    66   336    20   280\n",
      "> 4yjrA                      1   41104    7050     263     372   24.88    0.45E-07     562   202   100   310    46   250\n",
      "> 5yj9D                      1   41119    5500     343     372   25.50    0.17E-03     563   224    68   306     3   253\n",
      "> 2hakA                      1   41133    8750     309     372   30.80    0.75E-11     564   231    65   314     9   245\n",
      "> 2hakB                      1   41134    6500     279     372   29.34    0.80E-06     565   248    65   358    11   269\n",
      "> 2hakG                      1   41135    7350     297     372   31.56    0.11E-07     566   220    65   314     9   233\n",
      "> 5te0A                      1   41423    7000     316     372   23.27    0.72E-07     567   251    63   331    22   296\n",
      "> 2v7oA                      1   41535    8850     305     372   24.66    0.75E-11     568   210   136   349    83   305\n",
      "> 5e8sA                      1   41655    5250     301     372   30.84    0.53E-03     569    89   157   246   103   209\n",
      "> 4jr7A                      1   41772    6550     367     372   23.41    0.87E-06     570   171   147   318   154   358\n",
      "> 4jrnA                      1   41783    6700     349     372   34.43    0.38E-06     571   122   180   319   212   333\n",
      "> 5jrqB                      1   41807    9900     246     372   37.23     0.0         572   136   156   313   101   237\n",
      "> 5o1vA                      1   41877   10950     278     372   33.50     0.0         573   191   122   318    72   271\n",
      "> 5o1sA                      1   41891   10200     290     372   33.76     0.0         574   152   163   317   100   256\n",
      "> 6e2mB                      1   41977   12300     248     372   35.42     0.0         575   232    61   319     8   247\n",
      "> 6e2nA                      1   41978   13350     263     372   34.90     0.0         576   244    61   319     9   263\n",
      "> 6e2nB                      1   41979   12750     250     372   35.95     0.0         577   234    61   319     9   250\n",
      "> 4crsA                      1   42050   11800     331     372   31.89     0.0         578   240    67   319    19   272\n",
      "> 4ejnA                      1   42126   11100     376     372   29.71     0.0         579   229    66   320   110   348\n",
      "> 1b6cB                      1   42264    5250     326     372   30.84    0.58E-03     580    89   157   246   126   232\n",
      "> 4b6lA                      1   42291   12100     281     372   30.31     0.0         581   245    66   322    15   268\n",
      "> 2o2uA                      1   42364    8950     319     372   28.52     0.0         582   210    87   317    43   298\n",
      "> 5o2cA                      1   42375   10950     368     372   33.33     0.0         583   192   122   318    77   277\n",
      "> 4o2zA                      1   42380    9800     363     372   30.81     0.0         584   161   155   317   135   319\n",
      "> 5o2bB                      1   42387    9400     257     372   33.86     0.0         585   180   122   318    68   256\n",
      "> 5vdkA                      1   42399    7600     260     372   27.98    0.26E-08     586   226    68   315    15   257\n",
      "> 2vd5A                      1   42431    9450     390     372   30.80     0.0         587   246    66   320    65   327\n",
      "> 3udbC                      1   42637    6300     261     372   35.25    0.21E-05     588   134   157   318    98   236\n",
      "> 3e7oA                      1   42693   11000     342     372   27.53     0.0         589   242    68   317    24   310\n",
      "> 3e7oB                      1   42694    8850     326     372   27.41    0.75E-11     590   224    68   317    25   294\n",
      "> 4e7wA                      1   42720   10150     359     372   28.01     0.0         591   246    63   318    24   305\n",
      "> 5e7rA                      1   42734    8250     273     372   37.31    0.97E-10     592   129   166   314    90   223\n",
      "> 6jqrA                      1   42906    7850     301     372   29.18    0.88E-09     593   233    66   326    44   300\n",
      "> 3cblA                      1   42935    8050     356     372   26.25    0.39E-09     594   231    68   308   102   341\n",
      "> 4ow8A                      1   43031    8700     276     372   35.46    0.75E-11     595   139   180   333   130   270\n",
      "> 5owqA                      1   43050   13400     255     372   34.93     0.0         596   227    68   327    13   241\n",
      "> 4z7gA                      1   43188    9750     366     372   33.91     0.0         597   223    65   318    13   245\n",
      "> 2z7qA                      1   43228    9000     255     372   28.32     0.0         598   216    66   318    11   236\n",
      "> 4ra4A                      1   43275    9000     317     372   27.17     0.0         599   235    67   319    13   266\n",
      "> 6ra7A                      1   43291   14650     300     372   32.05     0.0         600   245    67   319    18   276\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     28101122   27442.502    26.799\n",
      "> 5jfwA                      1   43373    7650     277     372   29.91    0.22E-08     601   216    68   312    28   261\n",
      "> 2jc6A                      1   43461    8150     278     372   28.63    0.17E-09     602   234    65   339    16   263\n",
      "> 3fzsA                      1   43721    6000     254     372   25.42    0.93E-05     603   226    61   313     5   240\n",
      "> 3rnyA                      1   43785    8450     264     372   34.72    0.37E-10     604   139   173   317    88   231\n",
      "> 2f2uA                      1   43819    9850     386     372   29.00     0.0         605   283    19   319    30   329\n",
      "> 2f2uB                      1   43820    9850     383     372   29.00     0.0         606   283    19   319    30   329\n",
      "> 2f2cB                      1   43858   11450     280     372   29.71     0.0         607   165   154   319   102   276\n",
      "> 3iw4A                      1   44043    8750     330     372   27.06    0.75E-11     608   235    67   319    14   268\n",
      "> 4yhjA                      1   44150    9650     501     372   29.31     0.0         609   167   161   330   264   437\n",
      "> 4yhjB                      1   44151    9650     490     372   29.31     0.0         610   167   161   330   264   437\n",
      "> 4js8A                      1   44215    8550     259     372   39.84    0.22E-10     611   123   180   317   129   256\n",
      "> 3kexA                      1   44296    7100     292     372   31.43    0.39E-07     612   139   166   312   120   259\n",
      "> 3lltA                      1   44365    5550     341     372   24.54    0.13E-03     613   150   166   319   126   341\n",
      "> 4c62B                      1   44448    9700     271     372   29.55     0.0         614   228    68   312    13   259\n",
      "> 4im0A                      1   44505    5800     619     372   29.57    0.76E-04     615   170    67   262    16   201\n",
      "> 4im2A                      1   44533    5800     618     372   29.57    0.76E-04     616   170    67   262    16   201\n",
      "> 5imxA                      1   44572    5700     266     372   26.32    0.45E-04     617   148   167   329   109   260\n",
      "> 4ueuA                      1   45017    9800     256     372   29.83     0.0         618   231    63   313    12   249\n",
      "> 4hyuA                      1   45074    9350     322     372   30.71     0.0         619   225    68   317    25   291\n",
      "> 3hyhA                      1   45087    9200     238     372   31.56     0.0         620   224    66   319     6   230\n",
      "> 4m68A                      1   45199    7000     268     372   25.00    0.59E-07     621   191   115   314    66   261\n",
      "> 4m69A                      1   45200    6500     284     372   32.14    0.82E-06     622   129   166   308   113   252\n",
      "> 4rlkA                      1   45417    8450     327     372   23.32    0.45E-10     623   247    66   319    37   319\n",
      "> 3tl8A                      1   45523    9400     300     372   28.73     0.0         624   237    63   314    18   285\n",
      "> 4qqtA                      1   45606    9000     283     372   35.81     0.0         625   142   165   314   127   274\n",
      "> 1k9aA                      1   45714    6200     439     372   30.83    0.64E-05     626   130   167   312   293   425\n",
      "> 4rgjA                      1   45771   11750     463     372   30.10     0.0         627   281    66   371    31   329\n",
      "> 3rgfA                      1   45775    8100     321     372   25.18    0.26E-09     628   228    74   318    30   303\n",
      "> 1opkA                      1   45818    9900     449     372   29.59     0.0         629   257    58   329   175   441\n",
      "> 6op9A                      1   45875    7050     276     372   31.43    0.47E-07     630   139   166   312   119   258\n",
      "> 5k3yA                      1   45972   10400     267     372   25.59     0.0         631   250    60   319     3   256\n",
      "> 6k3lB                      1   45979    9650     329     372   26.37     0.0         632   256    66   328    38   329\n",
      "> 1g3nA                      1   46191   12150     293     372   26.86     0.0         633   248    68   319    11   293\n",
      "> 4g31A                      1   46207   10050     248     372   30.77     0.0         634   228    68   319    13   246\n",
      "> 4g3cB                      1   46215    7300     327     372   27.68    0.16E-07     635   255    39   312    32   302\n",
      "> 4g3dA                      1   46216    6850     324     372   28.22    0.16E-06     636   225    69   312    59   299\n",
      "> 4g3dD                      1   46217    6950     290     372   29.39    0.84E-07     637   218    69   312    53   280\n",
      "> 4g3fA                      1   46218    7600     312     372   29.34    0.33E-08     638   248    39   312    27   285\n",
      "> 4s31A                      1   46564    8650     347     372   28.80    0.15E-10     639   152   165   319   121   304\n",
      "> 5uv4A                      1   46680    6400     290     372   23.33    0.14E-05     640   197    56   255    12   221\n",
      "> 5uvcA                      1   46717    9950     457     372   27.78     0.0         641   267    40   319   111   398\n",
      "> 6nvhA                      1   47057    8250     271     372   36.84    0.97E-10     642   130   165   314   130   262\n",
      "> 4oliA                      1   47129    7800     539     372   29.35    0.23E-08     643   185   119   313   328   528\n",
      "> 3uibA                      1   47171    6550     309     372   28.40    0.70E-06     644   139   165   319   102   270\n",
      "> 3uimA                      1   47203    8800     291     372   30.30    0.75E-11     645   232    61   314    16   279\n",
      "> 5uisD                      1   47214    6850     250     372   42.31    0.12E-06     646    78   169   254   100   177\n",
      "> 5ui0A                      1   47230    7800     301     372   36.20    0.11E-08     647   155   136   314   128   290\n",
      "> 6uipC                      1   47234    5800     305     372   25.00    0.32E-04     648   197   134   346    86   293\n",
      "> 5nkgA                      1   47260    6800     292     372   24.31    0.18E-06     649   240    63   313    10   264\n",
      "> 4nkaB                      1   47265    7250     287     372   36.62    0.18E-07     650   135   165   314   133   274\n",
      "> 5wvdB                      1   47384    6500     212     372   32.84    0.58E-06     651   194    64   317    11   211\n",
      "> 5vi9A                      1   47435    8500     306     372   27.90    0.30E-10     652   226    66   306    28   260\n",
      "> 4mvfA                      1   47463    6350     408     372   36.07    0.27E-05     653   119   163   324   113   234\n",
      "> 4ks7A                      1   47580   14200     286     372   32.38     0.0         654   242    68   317    28   271\n",
      "> 6urcA                      1   47655   11000     400     372   32.82     0.0         655   240    65   318    15   273\n",
      "> 5larA                      1   47676   11250     350     372   29.03     0.0         656   244    68   318    27   305\n",
      "> 2bujA                      1   47789    7600     291     372   26.83    0.30E-08     657   223    81   313    34   279\n",
      "> 3bu3A                      1   47801    8150     294     372   28.78    0.19E-09     658   251    55   318     5   282\n",
      "> 1s9iA                      1   47879   17500     303     372   35.40     0.0         659   252    58   319     7   280\n",
      "> 1s9iB                      1   47880   17300     291     372   35.85     0.0         660   250    58   319     8   272\n",
      "> 3s95A                      1   47897    8200     290     372   35.77    0.14E-09     661   133   167   312   117   253\n",
      "> 2xnmA                      1   48079    8600     258     372   34.04    0.15E-10     662   141   156   316   108   248\n",
      "> 3i6uA                      1   48137   11550     390     372   33.45     0.0         663   272    27   319    96   376\n",
      "> 3i6uB                      1   48138    9800     376     372   32.58     0.0         664   258    27   319    96   362\n",
      "> 2i6lA                      1   48149    9600     292     372   28.68     0.0         665   224    87   317    29   286\n",
      "> 2i6lB                      1   48150    9050     269     372   29.75     0.0         666   219    87   317    23   264\n",
      "> 4l9iA                      1   48341   11750     504     372   30.80     0.0         667   248    66   319   162   424\n",
      "> 4b8lA                      1   48352    8900     249     372   26.03     0.0         668   238    57   319     1   242\n",
      "> 3v5pA                      1   48443   14050     450     372   30.35     0.0         669   299    65   371    12   324\n",
      "> 3v5wA                      1   48486    9950     623     372   27.78     0.0         670   267    40   319   138   425\n",
      "> 3v5lB                      1   48497    6500     241     372   28.57    0.67E-06     671   164   130   314    65   232\n",
      "> 5g6vA                      1   48642    9500     308     372   28.41     0.0         672   240    68   318    10   280\n",
      "> 4q9zB                      1   48838   10700     308     372   28.97     0.0         673   239    66   319    11   262\n",
      "> 3bkbA                      1   48976    8050     373     372   26.25    0.41E-09     674   231    68   308   119   358\n",
      "> 4bkjB                      1   49004    7300     297     372   25.73    0.14E-07     675   226    81   312    45   285\n",
      "> 4krdA                      1   49512   11250     294     372   27.96     0.0         676   245    68   318    12   290\n",
      "> 5cekA                      1   49724    6150     237     372   28.77    0.40E-05     677   146   156   319    89   234\n",
      "> 5cemA                      1   49725    6600     252     372   30.92    0.43E-06     678   150   156   319    90   241\n",
      "> 5cenA                      1   49726   10800     281     372   29.18     0.0         679   232    68   315    15   247\n",
      "> 5ceqA                      1   49727    9500     269     372   30.00     0.0         680   219    68   315    15   234\n",
      "> 4aw2A                      1   49771   10250     398     372   29.43     0.0         681   249    66   320    80   344\n",
      "> 5awmA                      1   49777    9050     345     372   25.00     0.0         682   242    87   339    40   327\n",
      "> 6c9dA                      1   49899    9000     425     372   29.39     0.0         683   239    65   314    11   255\n",
      "> 6c9fA                      1   49901    8150     372     372   29.73    0.25E-09     684   182   129   319    76   260\n",
      "> 6c9gA                      1   49902    8150     385     372   29.73    0.25E-09     685   182   129   319    76   260\n",
      "> 6c9hA                      1   49903    9200     435     372   28.51     0.0         686   218   129   357    76   296\n",
      "> 5xs2A                      1   49956    8750     366     372   24.27    0.75E-11     687   246    68   318    30   338\n",
      "> 4o38A                      1   50217    6600     267     372   23.79    0.46E-06     688   232    65   314    19   266\n",
      "> 6hdrA                      1   50332    7650     404     372   26.06    0.35E-08     689   246    66   319    98   404\n",
      "> 3mfrA                      1   50377   11250     302     372   30.34     0.0         690   271    67   349    13   302\n",
      "> 4f0gA                      1   50548    6600     263     372   24.79    0.45E-06     691   224    68   314    17   258\n",
      "> 6qy8A                      1   50655    8500     321     372   23.93    0.37E-10     692   245    66   318    37   316\n",
      "> 2bvaA                      1   50657   12450     272     372   36.99     0.0         693   170   155   332   100   272\n",
      "> 5bvdA                      1   50679    7250     330     372   28.66    0.21E-07     694   138   165   319   123   286\n",
      "> 5bvwA                      1   50702    7250     274     372   25.88    0.17E-07     695   240    65   312    10   264\n",
      "> 5mo4A                      1   50784    8150     429     372   29.55    0.29E-09     696   240    58   329   175   421\n",
      "> 6mobA                      1   50796    7650     306     372   29.35    0.25E-08     697   176   118   314   115   298\n",
      "> 3ppzA                      1   50913    9400     264     372   30.17     0.0         698   236    65   313    15   256\n",
      "> 3pp0A                      1   50918   10400     286     372   28.68     0.0         699   247    52   313     2   266\n",
      "> 5tq6A                      1   51076   11050     274     372   28.92     0.0         700   238    68   312    13   261\n",
      "> 3eqrA                      1   51181    8150     270     372   34.01    0.16E-09     701   138   166   312   113   259\n",
      "> 3eqcA                      1   51200   18900     312     372   37.55     0.0         702   254    58   319    24   292\n",
      "> 4equA                      1   51211   15450     274     372   34.43     0.0         703   236    71   327    17   260\n",
      "> 4eqmA                      1   51212    6950     269     372   34.57    0.77E-07     704    81   166   258   116   196\n",
      "> 3dfaA                      1   51303   11650     265     372   32.26     0.0         705   241    67   319    15   262\n",
      "> 4zznA                      1   51384    8350     338     372   28.96    0.75E-10     706   151   165   319   121   303\n",
      "> 3uzsA                      1   51440    9250     609     372   29.10     0.0         707   247    40   319   139   406\n",
      "> 3uztA                      1   51469    9650     586     372   28.62     0.0         708   255    40   319   138   413\n",
      "> 5ci6A                      1   51621    8450     335     372   27.59    0.45E-10     709   171   140   325   104   306\n",
      "> 5l2qC                      1   51792    7950     282     372   30.14    0.48E-09     710   199   104   314    65   273\n",
      "> 5l2qA                      1   51793    7400     293     372   33.11    0.85E-08     711   145   163   314   136   283\n",
      "> 5l2sA                      1   51794   11300     263     372   30.20     0.0         712   226    68   319     9   263\n",
      "> 4wihA                      1   52212    8750     350     372   26.84    0.75E-11     713   265    26   306    10   281\n",
      "> 4rt7A                      1   52424    8800     274     372   28.96    0.75E-11     714   234    66   326    16   274\n",
      "> 3zfmA                      1   52466    4750     249     372   24.00    0.54E-02     715   124   167   313   108   232\n",
      "> 3zfxA                      1   52475    6400     267     372   25.64    0.13E-05     716   226    71   313    16   249\n",
      "> 3zfxB                      1   52476    5950     254     372   24.34    0.12E-04     717   223    65   313    12   237\n",
      "> 3zfyA                      1   52477    5350     260     372   25.00    0.27E-03     718   139   167   313   105   244\n",
      "> 5ap1A                      1   52710    8450     268     372   39.26    0.37E-10     719   127   180   317   129   263\n",
      "> 6tszU                      1   52776    8800     288     372   24.35    0.75E-11     720   241    68   319    14   284\n",
      "> 4nw6A                      1   52838    9600     302     372   25.78     0.0         721   238    66   318    22   277\n",
      "> 6nw2B                      1   52868    9000     246     372   33.09     0.0         722   136   163   308    91   229\n",
      "> 4lqsA                      1   53048   10450     372     372   25.40     0.0         723   290    12   320     8   322\n",
      "> 4lqqA                      1   53067   10250     388     372   25.08     0.0         724   290    12   320    20   338\n",
      "> 4lqqD                      1   53069    9550     329     372   26.52     0.0         725   272    12   320     8   286\n",
      "> 4ux9B                      1   53105    9350     327     372   27.80     0.0         726   217    87   317    41   299\n",
      "> 4idtA                      1   53189    7950     334     372   28.63    0.60E-09     727   251    39   312    45   306\n",
      "> 4l68A                      1   53263    6200     313     372   27.46    0.43E-05     728   281    32   336     5   299\n",
      "> 4l68B                      1   53264    6200     325     372   27.24    0.45E-05     729   282    32   336    11   311\n",
      "> 5l6oA                      1   53288    5550     267     372   22.67    0.98E-04     730   236    65   313     5   251\n",
      "> 6q7dA                      1   53436    5750     275     372   25.32    0.37E-04     731   227    63   313    15   247\n",
      "> 6q7gA                      1   53437    5650     273     372   25.00    0.61E-04     732   226    63   313    14   245\n",
      "> 5k00A                      1   53481    7750     311     372   32.19    0.15E-08     733   145   155   317   101   246\n",
      "> 6k0jA                      1   53499    7350     325     372   25.57    0.12E-07     734   246    66   319    15   323\n",
      "> 3tubA                      1   53632    6250     253     372   25.33    0.26E-05     735   219    68   310    16   240\n",
      "> 6tu9A                      1   53663    7100     268     372   25.13    0.35E-07     736   192    49   248     7   201\n",
      "> 6tuaA                      1   53664    6850     294     372   26.34    0.14E-06     737   246    54   314    20   281\n",
      "> 5x2aA                      1   53712   10150     297     372   29.11     0.0         738   229    66   313    17   253\n",
      "> 3dtcA                      1   53757    7750     245     372   31.17    0.11E-08     739   227    63   313    10   240\n",
      "> 6dtlA                      1   53788    8650     356     372   27.24    0.15E-10     740   242    68   325    38   316\n",
      "> 5ar8A                      1   53964    5650     255     372   27.41    0.56E-04     741   129   163   313    98   232\n",
      "> 6hvfB                      1   54060    7150     244     372   32.28    0.25E-07     742   152   131   308    62   219\n",
      "> 6hveA                      1   54088    8700     267     372   29.33    0.75E-11     743   218    68   308    18   242\n",
      "> 6hveB                      1   54089    7650     246     372   32.70    0.19E-08     744   153   131   308    63   221\n",
      "> 4hvsA                      1   54092    7900     326     372   29.35    0.75E-09     745   176   118   314   134   317\n",
      "> 6hvdA                      1   54103   15200     286     372   34.35     0.0         746   251    55   317     7   268\n",
      "> 5hvjA                      1   54118    8050     283     372   35.56    0.29E-09     747   131   167   312   116   250\n",
      "> 6mm7A                      1   54129    8500     325     372   28.76    0.37E-10     748   224    66   306    32   264\n",
      "> 2r5tA                      1   54372    8350     284     372   26.97    0.60E-10     749   232    66   318    21   261\n",
      "> 4gcjA                      1   54439   12350     304     372   30.85     0.0         750   251    68   323    16   297\n",
      "> 3gc9B                      1   54483    9100     339     372   26.84     0.0         751   235    66   318    27   298\n",
      "> 6gr8A                      1   54575   12850     270     372   27.41     0.0         752   255    53   317     1   259\n",
      "> 5grnA                      1   54619    7650     286     372   26.17    0.23E-08     753   237    66   308    15   270\n",
      "> 5mjaA                      1   54654    6350     271     372   23.65    0.17E-05     754   233    65   313    14   254\n",
      "> 3mj1A                      1   54689    6100     228     372   30.49    0.49E-05     755   160   130   314    60   223\n",
      "> 3vvhA                      1   54913   18200     287     372   37.22     0.0         756   251    58   319     2   267\n",
      "> 3vvhC                      1   54914   19650     283     372   38.40     0.0         757   256    58   319     2   264\n",
      "> 5xd6A                      1   55070    7750     282     372   32.65    0.14E-08     758   181    51   257    12   207\n",
      "> 4an3A                      1   55123   18300     280     372   38.91     0.0         759   251    58   319     5   261\n",
      "> 2r2pA                      1   55256    7300     283     372   25.79    0.14E-07     760   241    63   313    15   266\n",
      "> 3r2yA                      1   55296    6450     259     372   29.87    0.95E-06     761   219    59   323    14   244\n",
      "> 5dzcA                      1   55427   10750     816     372   29.69     0.0         762   229    64   318   529   784\n",
      "> 4tndA                      1   55498   11950     529     372   28.85     0.0         763   243    66   319   176   435\n",
      "> 3tnhA                      1   55521    9000     288     372   26.74     0.0         764   240    68   318    18   275\n",
      "> 3ma6A                      1   55617   12750     267     372   30.89     0.0         765   252    65   323     9   267\n",
      "> 6p8eB                      1   55719    8050     260     372   29.59    0.26E-09     766   148   154   317    87   255\n",
      "> 3p86A                      1   55741    7900     250     372   29.31    0.54E-09     767   224    65   313    11   242\n",
      "> 3p86B                      1   55742    7450     239     372   29.55    0.51E-08     768   215    65   313    12   231\n",
      "> 3warA                      1   55829    8600     334     372   24.11    0.22E-10     769   246    66   318    42   323\n",
      "> 4wa9A                      1   55849    9250     246     372   30.16     0.0         770   182   139   329    58   246\n",
      "> 5waxA                      1   55878    9200     279     372   33.19     0.0         771   230    66   318    15   252\n",
      "> 1fmkA                      1   55902    7900     437     372   31.82    0.11E-08     772   131   160   308   282   413\n",
      "> 3fmeA                      1   55906   14100     268     372   36.80     0.0         773   235    68   327    14   263\n",
      "> 6kzdA                      1   56051   10400     282     372   28.68     0.0         774   242    61   314    10   267\n",
      "> 2wtkB                      1   56091    9850     311     372   32.02     0.0         775   190   115   319    66   268\n",
      "> 2wtkC                      1   56092    9800     285     372   33.77     0.0         776   144   164   319   108   261\n",
      "> 2wtkF                      1   56093    8950     268     372   35.66     0.0         777   137   164   319   108   250\n",
      "> 4j7bA                      1   56243   11450     284     372   33.05     0.0         778   235    66   318    26   264\n",
      "> 5j7sA                      1   56279    7900     279     372   36.84    0.62E-09     779   128   166   314    97   229\n",
      "> 6bx6A                      1   56553    7000     243     372   32.65    0.53E-07     780   147   145   319    89   235\n",
      "> 6bxiA                      1   56560    8600     333     372   25.25    0.22E-10     781   255    61   320     7   303\n",
      "> 3pvuA                      1   56682    9950     609     372   27.78     0.0         782   267    40   319   138   425\n",
      "> 2pvfA                      1   56688    8000     285     372   35.40    0.38E-09     783   155   138   314   115   275\n",
      "> 2c3iB                      1   56957   10100     266     372   27.64     0.0         784   239    67   319     7   252\n",
      "> 4c3fA                      1   56996    8400     257     372   29.33    0.45E-10     785   143   154   308    96   245\n",
      "> 4fl2A                      1   57071    6800     555     372   25.00    0.40E-06     786   233    68   310   297   540\n",
      "> 4fl3A                      1   57072    7000     538     372   25.54    0.14E-06     787   225    68   310   293   523\n",
      "> 3mtlA                      1   57126    9200     294     372   29.46     0.0         788   226    68   318    10   267\n",
      "> 3g2fA                      1   57347    6850     302     372   25.86    0.15E-06     789   246    67   335    12   274\n",
      "> 4b9dA                      1   57471   12700     290     372   28.33     0.0         790   233    75   317    24   263\n",
      "> 2b9hA                      1   57526    9250     337     372   29.25     0.0         791   254    51   319     1   294\n",
      "> 3cd3A                      1   57930    7600     352     372   26.81    0.38E-08     792   229    68   308   103   337\n",
      "> 4qtbA                      1   58008    9050     351     372   29.35     0.0         793   152   165   319   125   308\n",
      "> 3mwuA                      1   58108   11800     447     372   31.95     0.0         794   286    67   371    16   328\n",
      "> 4mwiA                      1   58120    6050     266     372   27.74    0.76E-05     795   134   170   314   124   260\n",
      "> 6mweA                      1   58142    7600     292     372   34.29    0.30E-08     796   133   167   313   120   259\n",
      "> 2rkuA                      1   58154   11550     294     372   30.92     0.0         797   244    66   318    21   269\n",
      "> 4iirB                      1   58247    6350     327     372   28.14    0.21E-05     798   166   180   372   139   305\n",
      "> 3kl8A                      1   58353    8450     256     372   25.13    0.30E-10     799   179   136   317    61   251\n",
      "> 5hoaA                      1   58426    6100     225     372   31.30    0.48E-05     800   128   166   313    73   203\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     28415722   27749.729    27.099\n",
      "> 1o6lA                      1   58466   10250     316     372   33.77     0.0         801   146   166   318   113   263\n",
      "> 1o6yA                      1   58474    7550     260     372   25.11    0.34E-08     802   223    67   310    14   248\n",
      "> 6o6qA                      1   58532    7200     326     372   23.47    0.27E-07     803   243    66   318    44   320\n",
      "> 1ym7A                      1   58542    9950     608     372   27.78     0.0         804   267    40   319   139   426\n",
      "> 2ym3A                      1   58543    8400     247     372   32.74    0.45E-10     805   109   154   263    78   190\n",
      "> 4ymjA                      1   58581    9650     270     372   30.28     0.0         806   234    61   314    11   261\n",
      "> 2ym7A                      1   58583    8400     248     372   32.74    0.45E-10     807   109   154   263    78   190\n",
      "> 2x0gA                      1   58724   10250     318     372   27.85     0.0         808   208   100   319    56   274\n",
      "> 5j5tA                      1   58816   12500     297     372   30.36     0.0         809   242    68   317    11   257\n",
      "> 2i0yA                      1   58848    7550     289     372   27.09    0.39E-08     810   233    66   312    29   279\n",
      "> 2i0eA                      1   58876    8450     326     372   25.20    0.45E-10     811   236    67   319     9   262\n",
      "> 2i0eB                      1   58877    8450     300     372   25.20    0.37E-10     812   236    67   319     9   262\n",
      "> 3ujgA                      1   59022    9350     290     372   37.27     0.0         813   155   157   318    93   253\n",
      "> 1zxeD                      1   59061    8350     256     372   34.78    0.52E-10     814   136   163   313   111   248\n",
      "> 3zxtA                      1   59090    9450     249     372   28.30     0.0         815   205    94   319    38   249\n",
      "> 5zxbA                      1   59130    7450     240     372   36.51    0.52E-08     816   123   166   312   107   232\n",
      "> 2i1mA                      1   59183    7100     311     372   26.25    0.42E-07     817   235    66   312    38   296\n",
      "> 6i1sA                      1   59226    6000     322     372   28.79    0.12E-04     818   109   140   248   103   234\n",
      "> 5yv8A                      1   59288   13200     259     372   33.60     0.0         819   238    63   317     7   256\n",
      "> 2vwiA                      1   59463   13450     237     372   36.09     0.0         820   225    65   319     8   237\n",
      "> 2vwiB                      1   59464   13850     264     372   32.58     0.0         821   253    52   319     1   264\n",
      "> 2vwiD                      1   59465   15200     284     372   32.97     0.0         822   260    51   319     3   278\n",
      "> 3vw8A                      1   59488    8200     303     372   28.63    0.15E-09     823   231    67   313    45   285\n",
      "> 5dykA                      1   59527   10500     809     372   28.90     0.0         824   249    56   318   517   779\n",
      "> 3dy7A                      1   59545   19000     260     372   38.76     0.0         825   253    58   319     1   258\n",
      "> 3zduA                      1   59589   12400     297     372   30.19     0.0         826   236    68   318    10   274\n",
      "> 6ccfA                      1   59936   13350     260     372   33.33     0.0         827   237    68   317    12   257\n",
      "> 6ccyA                      1   59939   11500     325     372   30.47     0.0         828   243    66   320    14   269\n",
      "> 3cc6A                      1   59951    6700     269     372   25.82    0.28E-06     829   233    61   313    10   253\n",
      "> 5ajqB                      1   60014   15100     267     372   34.85     0.0         830   233    68   327    18   258\n",
      "> 4l3jA                      1   60100   10900     291     372   27.80     0.0         831   236    66   319    11   269\n",
      "> 3blhA                      1   60148    8750     291     372   26.89    0.75E-11     832   237    68   318    18   281\n",
      "> 1blxA                      1   60150   12150     305     372   26.86     0.0         833   248    68   319    15   297\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     28417047   27751.023    27.101\n",
      "> 3omvA                      1   60226    9450     264     372   29.80     0.0         834   233    68   315    16   260\n",
      "> 2ivsB                      1   60277    9000     277     372   33.33     0.0         835   208    87   314    53   268\n",
      "> 4ivaA                      1   60324   10900     290     372   28.14     0.0         836   241    68   312    15   277\n",
      "> 3o0gA                      1   60452    9800     289     372   26.71     0.0         837   243    68   318    10   286\n",
      "> 3o0gB                      1   60454    9350     264     372   28.04     0.0         838   162   155   318    74   262\n",
      "> 5o0yA                      1   60511    9900     288     372   29.05     0.0         839   163   155   317   107   285\n",
      "> 4d28A                      1   60559    9250     274     372   26.94     0.0         840   240    68   319     6   250\n",
      "> 4d2sA                      1   60567    8400     249     372   29.82    0.45E-10     841   221    68   317    19   246\n",
      "> 4d2wD                      1   60568    7600     326     372   30.92    0.35E-08     842   151   155   317   104   255\n",
      "> 2y4iB                      1   60668    5750     265     372   26.56    0.35E-04     843   227    67   313    19   259\n",
      "> 2y4pB                      1   60698    8550     184     372   30.87    0.15E-10     844   145   153   317    35   183\n",
      "> 6nfzA                      1   60871   12900     268     372   32.61     0.0         845   225    71   315    18   247\n",
      "> 4aojA                      1   61102    7850     260     372   30.17    0.74E-09     846   218    68   312    17   248\n",
      "> 6ao5A                      1   61134   15750     321     372   33.33     0.0         847   262    68   371     6   269\n",
      "> 6c19A                      1   61231    7300     276     372   38.93    0.13E-07     848   127   165   314   133   263\n",
      "> 4kikA                      1   61270    8850     618     372   40.40    0.15E-10     849    97   154   260   101   199\n",
      "> 4kikB                      1   61271    8900     648     372   39.42    0.75E-11     850   101   154   260   116   219\n",
      "> 5u8lA                      1   61330   10000     289     372   28.00     0.0         851   239    66   313    15   264\n",
      "> 3c51B                      1   61477   11100     461     372   31.69     0.0         852   232    66   319   154   396\n",
      "> 3rhkB                      1   61590    8000     299     372   27.78    0.40E-09     853   234    67   313    38   289\n",
      "> 5hg8A                      1   61785   10000     300     372   28.00     0.0         854   239    66   313    17   266\n",
      "> 4gk2A                      1   61848    5650     265     372   24.57    0.58E-04     855   223    66   313    17   248\n",
      "> 6d3kB                      1   61973   10750     254     372   32.17     0.0         856   216    67   314    18   247\n",
      "> 2a2aA                      1   62193   10350     304     372   25.81     0.0         857   213   100   317    58   274\n",
      "> 2a2aB                      1   62194   10350     303     372   25.81     0.0         858   213   100   317    57   273\n",
      "> 3hmiA                      1   62305    9200     268     372   29.48     0.0         859   241    58   311     6   256\n",
      "> 3hmmA                      1   62307    4850     293     372   31.37    0.40E-02     860    85   157   246   100   201\n",
      "> 6hmqA                      1   62351    8700     327     372   23.76    0.15E-10     861   246    66   318    38   319\n",
      "> 3kfaA                      1   62522   10150     284     372   29.96     0.0         862   257    58   329    11   277\n",
      "> 1v0oB                      1   62575   12800     275     372   31.58     0.0         863   237    68   319     8   273\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 2x4fA                      1   62679   10250     289     372   28.88     0.0         864   226    84   317    45   276\n",
      "> 5vhbA                      1   62837    8450     325     372   28.02    0.45E-10     865   225    66   306    30   261\n",
      "> 3pj3A                      1   62963    5300     240     372   27.56    0.31E-03     866   222    60   315     7   231\n",
      "> 6b2qA                      1   63051    7600     261     372   30.71    0.27E-08     867   235    65   333    14   254\n",
      "> 2ac3A                      1   63138    7900     277     372   35.90    0.61E-09     868   136   167   317   120   275\n",
      "> 2acxA                      1   63152   10750     495     372   31.90     0.0         869   156   161   319   267   429\n",
      "> 3w2cA                      1   63195    5850     203     372   33.51    0.15E-04     870   179    66   303    10   194\n",
      "> 3w2pA                      1   63205    9800     308     372   28.00     0.0         871   239    66   313    20   269\n",
      "> 6dc0A                      1   63288    6300     265     372   27.81    0.21E-05     872   163   156   327    92   260\n",
      "> 5oatA                      1   63462    5250     359     372   24.26    0.65E-03     873   228    68   306    13   284\n",
      "> 4x7kA                      1   63639    9450     260     372   28.40     0.0         874   243    55   319     2   251\n",
      "> 4x7qB                      1   63642    8000     243     372   29.18    0.31E-09     875   229    67   319    11   243\n",
      "> 4x7qA                      1   63643    6850     217     372   33.58    0.99E-07     876   134   161   319    81   217\n",
      "> 2x7fD                      1   63655   13000     236     372   37.76     0.0         877   192   101   319    25   220\n",
      "> 2x7fE                      1   63656   11100     251     372   33.80     0.0         878   211    74   319    20   232\n",
      "> 2x7gA                      1   63658    6000     344     372   25.33    0.13E-04     879   161   152   317   113   341\n",
      "> 5jt2D                      1   63723    9100     234     372   31.90     0.0         880   207    68   313    18   227\n",
      "> 2hziB                      1   63814    9450     264     372   30.24     0.0         881   238    58   312     6   253\n",
      "> 4hzrA                      1   63837    8150     261     372   34.01    0.16E-09     882   138   166   312   107   253\n",
      "> 3hztA                      1   63847   11600     421     372   32.08     0.0         883   249    68   332    10   274\n",
      "> 4hzsA                      1   63853    8250     319     372   37.31    0.12E-09     884   131   166   312   121   254\n",
      "> 6hzvD                      1   63873   10450     257     372   29.54     0.0         885   228    68   312    14   250\n",
      "> 2hz0B                      1   63880    7600     243     372   30.54    0.24E-08     886   163   129   312    69   235\n",
      "> 6gqmA                      1   64064    8350     297     372   26.04    0.67E-10     887   242    48   314    24   288\n",
      "> 6gqmB                      1   64065    8150     288     372   26.52    0.18E-09     888   244    48   314    16   279\n",
      "> 4xeyB                      1   64236    9400     359     372   29.65     0.0         889   192   129   329   161   359\n",
      "> 5ax9A                      1   64273   14050     282     372   33.60     0.0         890   233    67   319    19   265\n",
      "> 5ax9B                      1   64274   14000     274     372   34.17     0.0         891   231    67   319    18   257\n",
      "> 4p2kA                      1   64555    6800     259     372   25.32    0.16E-06     892   232    63   313    15   251\n",
      "> 2dq7X                      1   64748    8150     262     372   26.81    0.16E-09     893   228    68   308    17   251\n",
      "> 3alnA                      1   64811   12300     270     372   35.78     0.0         894   223    56   317    16   247\n",
      "> 3alnC                      1   64812    6350     183     372   40.37    0.10E-05     895   106   158   317    51   159\n",
      "> 3aloA                      1   64815   15150     289     372   34.22     0.0         896   253    56   317     4   266\n",
      "> 3c4wB                      1   64892   11750     518     372   30.80     0.0         897   248    66   319   184   446\n",
      "> 3c4yB                      1   64894   11250     466     372   37.80     0.0         898   155   161   319   245   408\n",
      "> 3c4zA                      1   64896   11750     493     372   30.80     0.0         899   248    66   319   158   420\n",
      "> 5p9jA                      1   65073    7500     263     372   27.94    0.45E-08     900   241    61   315     8   254\n",
      "> 2wqbA                      1   65136    7600     289     372   34.75    0.30E-08     901   134   167   313   118   258\n",
      "> 2wqmA                      1   65144   13700     258     372   37.10     0.0         902   214    68   307    21   241\n",
      "> 2wqnA                      1   65145   14050     259     372   36.77     0.0         903   216    68   307    20   242\n",
      "> 5jk3A                      1   65206    7550     269     372   36.36    0.36E-08     904   127   166   314    88   219\n",
      "> 5jkgA                      1   65207    7750     266     372   38.17    0.13E-08     905   128   165   314   127   257\n",
      "> 4neuA                      1   65280    8800     286     372   33.13    0.75E-11     906   156   163   331   113   275\n",
      "> 4neuB                      1   65281    8250     268     372   33.33    0.97E-10     907   149   163   331   105   260\n",
      "> 3oy1A                      1   65323    9250     329     372   29.41     0.0         908   226    68   317    25   296\n",
      "> 6oytA                      1   65387   10050     222     372   36.67     0.0         909   203    61   319    13   222\n",
      "> 6oytB                      1   65388   12400     238     372   37.39     0.0         910   223    61   319     9   238\n",
      "> 6oytC                      1   65389   11800     236     372   34.21     0.0         911   220    61   317     8   235\n",
      "> 2h6dA                      1   65435    7550     256     372   30.13    0.33E-08     912   156   145   319    92   247\n",
      "> 3ncgA                      1   65520   11650     426     372   31.73     0.0         913   285    67   371    16   327\n",
      "> 5nclA                      1   65564   10350     418     372   25.08     0.0         914   290    12   320    36   358\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     28421269   27755.145    27.105\n",
      "> 3dj6A                      1   65653   12150     252     372   30.96     0.0         915   233    66   319    10   248\n",
      "> 4rz7A                      1   65686   10450     801     372   30.00     0.0         916   227    64   318   528   777\n",
      "> 2rioA                      1   65780    9350     396     372   30.20     0.0         917   227    64   318    12   266\n",
      "> 6eifC                      1   66010    6050     316     372   24.05    0.93E-05     918   198   104   318    80   316\n",
      "> 6eimA                      1   66043   16100     286     372   33.86     0.0         919   244    76   327    19   272\n",
      "> 2y7jA                      1   66067   10350     284     372   24.44     0.0         920   247    65   319    18   283\n",
      "> 3u4wA                      1   66124    8700     275     372   30.82    0.75E-11     921   142   160   308   105   250\n",
      "> 5h0bA                      1   66220    8800     446     372   26.94    0.75E-11     922   284     5   308   128   424\n",
      "> 6h0oA                      1   66241    6750     225     372   29.57    0.17E-06     923   184   105   316    30   215\n",
      "> 3ugcA                      1   66274   10600     274     372   28.97     0.0         924   232    68   312    13   264\n",
      "> 5ug9A                      1   66278    9900     280     372   28.86     0.0         925   236    66   313    15   260\n",
      "> 5ugaA                      1   66279   10000     283     372   28.00     0.0         926   239    66   313    14   263\n",
      "> 5uglB                      1   66281    8000     271     372   29.34    0.36E-09     927   225    68   314    21   262\n",
      "> 5m5aA                      1   66442    8100     315     372   34.51    0.26E-09     928   142   155   317   103   244\n",
      "> 4tthB                      1   66469   12300     256     372   28.87     0.0         929   230    81   319    15   253\n",
      "> 5tc0A                      1   66707    5250     238     372   28.50    0.40E-03     930   202    66   312    14   227\n",
      "> 5tc0B                      1   66708    6100     250     372   26.78    0.54E-05     931   226    57   312     1   239\n",
      "> 2onlC                      1   66860    8550     317     372   31.35    0.22E-10     932   182   153   366   100   284\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    35.265625\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000525\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1909.541748\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     1.296875\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000546\n",
      "profile_iteration__> Overall performance [seq/sec]:  1833.029785\n",
      "openf___224_> Open           buildprofile/EAO00194build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     31739934   30996.029    30.270\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31741418   30997.479    30.271\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31741942   30997.990    30.271\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31743042   30999.064    30.273\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31743978   30999.979    30.273\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31744978   31000.955    30.274\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31746122   31002.072    30.275\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31747302   31003.225    30.277\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31747802   31003.713    30.277\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31748726   31004.615    30.278\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31749618   31005.486    30.279\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31750622   31006.467    30.280\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31751602   31007.424    30.281\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31752178   31007.986    30.281\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31752706   31008.502    30.282\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31753898   31009.666    30.283\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31754846   31010.592    30.284\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31755726   31011.451    30.285\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31756626   31012.330    30.285\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31757470   31013.154    30.286\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31758046   31013.717    30.287\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31759030   31014.678    30.288\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31759894   31015.521    30.289\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31760566   31016.178    30.289\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31761146   31016.744    30.290\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31761874   31017.455    30.290\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31762770   31018.330    30.291\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31763686   31019.225    30.292\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31764290   31019.814    30.293\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31765266   31020.768    30.294\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31765754   31021.244    30.294\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31766658   31022.127    30.295\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31767634   31023.080    30.296\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31768494   31023.920    30.297\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31769510   31024.912    30.298\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31770342   31025.725    30.299\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31771362   31026.721    30.300\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31772042   31027.385    30.300\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31772958   31028.279    30.301\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31773934   31029.232    30.302\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31774910   31030.186    30.303\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31775546   31030.807    30.304\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31776522   31031.760    30.304\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31777174   31032.396    30.305\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31777826   31033.033    30.306\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31778738   31033.924    30.307\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31779394   31034.564    30.307\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31780318   31035.467    30.308\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31780802   31035.939    30.309\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31781762   31036.877    30.309\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31782730   31037.822    30.310\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31783678   31038.748    30.311\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31784586   31039.635    30.312\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31785506   31040.533    30.313\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31786378   31041.385    30.314\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31787194   31042.182    30.315\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31787850   31042.822    30.315\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31788486   31043.443    30.316\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31788906   31043.854    30.316\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31789638   31044.568    30.317\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31790402   31045.314    30.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31791134   31046.029    30.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31792010   31046.885    30.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31793018   31047.869    30.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31794010   31048.838    30.321\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31794978   31049.783    30.322\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31795950   31050.732    30.323\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31796878   31051.639    30.324\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31797726   31052.467    30.325\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31798670   31053.389    30.326\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31799534   31054.232    30.326\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31800410   31055.088    30.327\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31801326   31055.982    30.328\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31802034   31056.674    30.329\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31802742   31057.365    30.329\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31803694   31058.295    30.330\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31804678   31059.256    30.331\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31805590   31060.146    30.332\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31806770   31061.299    30.333\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31807714   31062.221    30.334\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31808686   31063.170    30.335\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31809514   31063.979    30.336\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31810454   31064.896    30.337\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31811430   31065.850    30.338\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31811958   31066.365    30.338\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31812478   31066.873    30.339\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31813446   31067.818    30.340\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31814226   31068.580    30.340\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31814638   31068.982    30.341\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31815642   31069.963    30.342\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31816402   31070.705    30.342\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31817298   31071.580    30.343\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31817842   31072.111    30.344\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31818638   31072.889    30.345\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31819538   31073.768    30.345\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31820066   31074.283    30.346\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31820882   31075.080    30.347\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31821846   31076.021    30.348\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31822818   31076.971    30.349\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31823238   31077.381    30.349\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31823658   31077.791    30.349\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31824554   31078.666    30.350\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31825494   31079.584    30.351\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31826474   31080.541    30.352\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31827390   31081.436    30.353\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31828238   31082.264    30.354\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31829202   31083.205    30.355\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31829510   31083.506    30.355\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31830238   31084.217    30.356\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31831198   31085.154    30.357\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31831758   31085.701    30.357\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31832670   31086.592    30.358\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31833622   31087.521    30.359\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31834346   31088.229    30.360\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31834910   31088.779    30.360\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31835854   31089.701    30.361\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31836566   31090.396    30.362\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31837494   31091.303    30.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31838238   31092.029    30.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31839162   31092.932    30.364\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31840118   31093.865    30.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31841090   31094.814    30.366\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31841754   31095.463    30.367\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31842574   31096.264    30.367\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31843194   31096.869    30.368\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31844046   31097.701    30.369\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31844790   31098.428    30.370\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31845686   31099.303    30.370\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31846378   31099.979    30.371\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31847070   31100.654    30.372\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31847582   31101.154    30.372\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31848110   31101.670    30.373\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31848950   31102.490    30.374\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31849290   31102.822    30.374\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31850058   31103.572    30.375\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31850466   31103.971    30.375\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31850898   31104.393    30.375\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31851462   31104.943    30.376\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31852430   31105.889    30.377\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31853366   31106.803    30.378\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31854326   31107.740    30.379\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31855282   31108.674    30.380\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31855838   31109.217    30.380\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31856762   31110.119    30.381\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31857686   31111.021    30.382\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31858650   31111.963    30.383\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31859574   31112.865    30.384\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31860290   31113.564    30.384\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31861190   31114.443    30.385\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31862134   31115.365    30.386\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31862886   31116.100    30.387\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31863854   31117.045    30.388\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31864822   31117.990    30.389\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31865750   31118.896    30.390\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31866282   31119.416    30.390\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31867214   31120.326    30.391\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31868182   31121.271    30.392\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31869130   31122.197    30.393\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31870134   31123.178    30.394\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31870874   31123.900    30.394\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31871822   31124.826    30.395\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31872534   31125.521    30.396\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31873246   31126.217    30.397\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31874062   31127.014    30.397\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31874946   31127.877    30.398\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31875906   31128.814    30.399\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31876862   31129.748    30.400\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31877754   31130.619    30.401\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31878414   31131.264    30.402\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31879138   31131.971    30.402\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31879914   31132.729    30.403\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31880974   31133.764    30.404\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31881898   31134.666    30.405\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31882582   31135.334    30.406\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31883526   31136.256    30.406\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31884438   31137.146    30.407\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31885414   31138.100    30.408\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31886374   31139.037    30.409\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31887342   31139.982    30.410\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31887886   31140.514    30.411\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31888446   31141.061    30.411\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31889314   31141.908    30.412\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31889874   31142.455    30.413\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31890766   31143.326    30.413\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31891870   31144.404    30.414\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31892790   31145.303    30.415\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31893722   31146.213    30.416\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31894614   31147.084    30.417\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31895586   31148.033    30.418\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31896174   31148.607    30.419\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31897070   31149.482    30.419\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31898150   31150.537    30.420\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31898882   31151.252    30.421\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31899730   31152.080    30.422\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31900682   31153.010    30.423\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31901654   31153.959    30.424\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31902234   31154.525    30.424\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31903130   31155.400    30.425\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31904006   31156.256    30.426\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31904978   31157.205    30.427\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31905950   31158.154    30.428\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31906886   31159.068    30.429\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31907474   31159.643    30.429\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31907990   31160.146    30.430\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31908654   31160.795    30.430\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31909378   31161.502    30.431\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31910078   31162.186    30.432\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31911002   31163.088    30.433\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31911598   31163.670    30.433\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31912490   31164.541    30.434\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31913170   31165.205    30.435\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31913754   31165.775    30.435\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31914718   31166.717    30.436\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31915354   31167.338    30.437\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31915686   31167.662    30.437\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31916622   31168.576    30.438\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31917566   31169.498    30.439\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31918538   31170.447    30.440\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31919226   31171.119    30.441\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31920002   31171.877    30.441\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31920902   31172.756    30.442\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31921498   31173.338    30.443\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31922006   31173.834    30.443\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31922834   31174.643    30.444\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31923330   31175.127    30.444\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31924282   31176.057    30.445\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31924922   31176.682    30.446\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31925850   31177.588    30.447\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31926698   31178.416    30.448\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31927734   31179.428    30.449\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31928742   31180.412    30.450\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31929730   31181.377    30.451\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31930682   31182.307    30.451\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31931662   31183.264    30.452\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31932642   31184.221    30.453\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31933658   31185.213    30.454\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31934526   31186.061    30.455\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31935218   31186.736    30.456\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31935910   31187.412    30.456\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31936442   31187.932    30.457\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31937402   31188.869    30.458\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31938246   31189.693    30.459\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31939326   31190.748    30.460\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31940342   31191.740    30.461\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31941306   31192.682    30.462\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31942338   31193.689    30.463\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31942810   31194.150    30.463\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31943458   31194.783    30.464\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31944026   31195.338    30.464\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31944962   31196.252    30.465\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31945594   31196.869    30.466\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31946218   31197.479    30.466\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31946918   31198.162    30.467\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31947858   31199.080    30.468\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31948762   31199.963    30.469\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31949638   31200.818    30.470\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31950738   31201.893    30.471\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31951126   31202.271    30.471\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31951714   31202.846    30.472\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31952710   31203.818    30.472\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31953142   31204.240    30.473\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31954038   31205.115    30.474\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31954870   31205.928    30.475\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31955642   31206.682    30.475\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31956626   31207.643    30.476\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31957642   31208.635    30.477\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31958546   31209.518    30.478\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31959462   31210.412    30.479\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31960478   31211.404    30.480\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31961370   31212.275    30.481\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31962238   31213.123    30.482\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31963218   31214.080    30.483\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31964130   31214.971    30.483\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31965102   31215.920    30.484\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31965982   31216.779    30.485\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31966754   31217.533    30.486\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31967754   31218.510    30.487\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31968722   31219.455    30.488\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31969306   31220.025    30.488\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31970070   31220.771    30.489\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31971186   31221.861    30.490\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31972110   31222.764    30.491\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31973058   31223.689    30.492\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31973930   31224.541    30.493\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31974858   31225.447    30.494\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31975842   31226.408    30.495\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31976834   31227.377    30.495\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31977882   31228.400    30.496\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31978770   31229.268    30.497\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31979262   31229.748    30.498\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31979946   31230.416    30.498\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31980950   31231.396    30.499\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31981838   31232.264    30.500\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31982822   31233.225    30.501\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31983382   31233.771    30.502\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31983730   31234.111    30.502\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31984602   31234.963    30.503\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31985474   31235.814    30.504\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31986410   31236.729    30.505\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31987394   31237.689    30.506\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31988354   31238.627    30.506\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31988950   31239.209    30.507\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31989614   31239.857    30.508\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31990178   31240.408    30.508\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31990770   31240.986    30.509\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31991670   31241.865    30.510\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31992542   31242.717    30.510\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31993494   31243.646    30.511\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31994278   31244.412    30.512\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31995250   31245.361    30.513\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31996210   31246.299    30.514\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31997170   31247.236    30.515\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31997602   31247.658    30.515\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31998702   31248.732    30.516\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31999298   31249.314    30.517\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     31999926   31249.928    30.518\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32000638   31250.623    30.518\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32001566   31251.529    30.519\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32002042   31251.994    30.520\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32003090   31253.018    30.521\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32003998   31253.904    30.521\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32004726   31254.615    30.522\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32005634   31255.502    30.523\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32006550   31256.396    30.524\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32007462   31257.287    30.525\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32008290   31258.096    30.525\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32008898   31258.689    30.526\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32010090   31259.854    30.527\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32010954   31260.697    30.528\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32011490   31261.221    30.529\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32012426   31262.135    30.529\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32013166   31262.857    30.530\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32014082   31263.752    30.531\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32014838   31264.490    30.532\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32015806   31265.436    30.533\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32016358   31265.975    30.533\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32017246   31266.842    30.534\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32018194   31267.768    30.535\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32018734   31268.295    30.535\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32019718   31269.256    30.536\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32020574   31270.092    30.537\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32021090   31270.596    30.538\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32021606   31271.100    30.538\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32022278   31271.756    30.539\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32023126   31272.584    30.540\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32023614   31273.061    30.540\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32024250   31273.682    30.541\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32024758   31274.178    30.541\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32025714   31275.111    30.542\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32026530   31275.908    30.543\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32027074   31276.439    30.543\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32027990   31277.334    30.544\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32028914   31278.236    30.545\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32029798   31279.100    30.546\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32030706   31279.986    30.547\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32031674   31280.932    30.548\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32032634   31281.869    30.549\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32033162   31282.385    30.549\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32034042   31283.244    30.550\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32034686   31283.873    30.551\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32035350   31284.521    30.551\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32035842   31285.002    30.552\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32036406   31285.553    30.552\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32037350   31286.475    30.553\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32038282   31287.385    30.554\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32039170   31288.252    30.555\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32040046   31289.107    30.556\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32041030   31290.068    30.557\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32042190   31291.201    30.558\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32043162   31292.150    30.559\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32043654   31292.631    30.559\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32044546   31293.502    30.560\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32045478   31294.412    30.561\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32046034   31294.955    30.561\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32046982   31295.881    30.562\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32047942   31296.818    30.563\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32048782   31297.639    30.564\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32049190   31298.037    30.564\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32050186   31299.010    30.565\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32051146   31299.947    30.566\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32051578   31300.369    30.567\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32052426   31301.197    30.568\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32053386   31302.135    30.568\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32054338   31303.064    30.569\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32055198   31303.904    30.570\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32056090   31304.775    30.571\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32056758   31305.428    30.572\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32057426   31306.080    30.572\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32058394   31307.025    30.573\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32058902   31307.521    30.574\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32059786   31308.385    30.575\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32060810   31309.385    30.576\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32061370   31309.932    30.576\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32062058   31310.604    30.577\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32062766   31311.295    30.577\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32063686   31312.193    30.578\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32064638   31313.123    30.579\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32065566   31314.029    30.580\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32066494   31314.936    30.581\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32067666   31316.080    30.582\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32068630   31317.021    30.583\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32069498   31317.869    30.584\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32070346   31318.697    30.585\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32071298   31319.627    30.586\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32072322   31320.627    30.587\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32072966   31321.256    30.587\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32073930   31322.197    30.588\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32074886   31323.131    30.589\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32075790   31324.014    30.590\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32076426   31324.635    30.590\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32077154   31325.346    30.591\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32077886   31326.061    30.592\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32078750   31326.904    30.593\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32079242   31327.385    30.593\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32079742   31327.873    30.594\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32080250   31328.369    30.594\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32081234   31329.330    30.595\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32082186   31330.260    30.596\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32083266   31331.314    30.597\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32084154   31332.182    30.598\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32084758   31332.771    30.598\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32085802   31333.791    30.599\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32086746   31334.713    30.600\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32087634   31335.580    30.601\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32088182   31336.115    30.602\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32088762   31336.682    30.602\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32089346   31337.252    30.603\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32089902   31337.795    30.603\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32090874   31338.744    30.604\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32091834   31339.682    30.605\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32092462   31340.295    30.606\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32093334   31341.146    30.607\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32094066   31341.861    30.607\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32094838   31342.615    30.608\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32095518   31343.279    30.609\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32096510   31344.248    30.610\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32097422   31345.139    30.610\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32098298   31345.994    30.611\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32099182   31346.857    30.612\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32100022   31347.678    30.613\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32100894   31348.529    30.614\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32101814   31349.428    30.615\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32102630   31350.225    30.615\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32103170   31350.752    30.616\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32103870   31351.436    30.617\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32104798   31352.342    30.618\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32105618   31353.143    30.618\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32106506   31354.010    30.619\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32107430   31354.912    30.620\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32107942   31355.412    30.621\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32108918   31356.365    30.621\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32109890   31357.314    30.622\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32110814   31358.217    30.623\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32111778   31359.158    30.624\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32112742   31360.100    30.625\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32113262   31360.607    30.626\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32114162   31361.486    30.626\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32115082   31362.385    30.627\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32115806   31363.092    30.628\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32116790   31364.053    30.629\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32117770   31365.010    30.630\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32118318   31365.545    30.630\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32119190   31366.396    30.631\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32120082   31367.268    30.632\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32121010   31368.174    30.633\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32121906   31369.049    30.634\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32122814   31369.936    30.635\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32123462   31370.568    30.635\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32124102   31371.193    30.636\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32124602   31371.682    30.636\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32125106   31372.174    30.637\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32125810   31372.861    30.638\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32126110   31373.154    30.638\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32127074   31374.096    30.639\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32127790   31374.795    30.639\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32128710   31375.693    30.640\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32129642   31376.604    30.641\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32130578   31377.518    30.642\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32131558   31378.475    30.643\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32132538   31379.432    30.644\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32133458   31380.330    30.645\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32134438   31381.287    30.646\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32135366   31382.193    30.647\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32136262   31383.068    30.648\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32137210   31383.994    30.648\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32137622   31384.396    30.649\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32138162   31384.924    30.649\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32138814   31385.561    30.650\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32139466   31386.197    30.651\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32140002   31386.721    30.651\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32140558   31387.264    30.652\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32141074   31387.768    30.652\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32141786   31388.463    30.653\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32142610   31389.268    30.654\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32143554   31390.189    30.654\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32144190   31390.811    30.655\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32145050   31391.650    30.656\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32145954   31392.533    30.657\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32146946   31393.502    30.658\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32147790   31394.326    30.659\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32148742   31395.256    30.659\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32149702   31396.193    30.660\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32150662   31397.131    30.661\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32151614   31398.061    30.662\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32152566   31398.990    30.663\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32153266   31399.674    30.664\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32154066   31400.455    30.665\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32155038   31401.404    30.665\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32155922   31402.268    30.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32156910   31403.232    30.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32157474   31403.783    30.668\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32158114   31404.408    30.668\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32159046   31405.318    30.669\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32159666   31405.924    30.670\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32160554   31406.791    30.671\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32161486   31407.701    30.672\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32162286   31408.482    30.672\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32163178   31409.354    30.673\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32164150   31410.303    30.674\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32165102   31411.232    30.675\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32166046   31412.154    30.676\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32166934   31413.021    30.677\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32167854   31413.920    30.678\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32168830   31414.873    30.679\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32169406   31415.436    30.679\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32170378   31416.385    30.680\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32171362   31417.346    30.681\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32172378   31418.338    30.682\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32173322   31419.260    30.683\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32173934   31419.857    30.683\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32174906   31420.807    30.684\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32175806   31421.686    30.685\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32176302   31422.170    30.686\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32177222   31423.068    30.687\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32178154   31423.979    30.687\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32178818   31424.627    30.688\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32179694   31425.482    30.689\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32180614   31426.381    30.690\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32181602   31427.346    30.691\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32182206   31427.936    30.691\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32183110   31428.818    30.692\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32184034   31429.721    30.693\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32184982   31430.646    30.694\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32185970   31431.611    30.695\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32186590   31432.217    30.696\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32187490   31433.096    30.696\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32188370   31433.955    30.697\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32189282   31434.846    30.698\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32190226   31435.768    30.699\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32191290   31436.807    30.700\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32192194   31437.689    30.701\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32193146   31438.619    30.702\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32194098   31439.549    30.703\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32194974   31440.404    30.704\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32195838   31441.248    30.704\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32196550   31441.943    30.705\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32197262   31442.639    30.706\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32198190   31443.545    30.707\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32198766   31444.107    30.707\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32199350   31444.678    30.708\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32199926   31445.240    30.708\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32200894   31446.186    30.709\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32201698   31446.971    30.710\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32202590   31447.842    30.711\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32203510   31448.740    30.712\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32204498   31449.705    30.713\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32205374   31450.561    30.713\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32206374   31451.537    30.714\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32207210   31452.354    30.715\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32207562   31452.697    30.716\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32208242   31453.361    30.716\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32208726   31453.834    30.717\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32209266   31454.361    30.717\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32210026   31455.104    30.718\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32210630   31455.693    30.718\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32211554   31456.596    30.719\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32212526   31457.545    30.720\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32213458   31458.455    30.721\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32214414   31459.389    30.722\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32215326   31460.279    30.723\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32215678   31460.623    30.723\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32216654   31461.576    30.724\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32217490   31462.393    30.725\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32218254   31463.139    30.726\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32218894   31463.764    30.726\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32219610   31464.463    30.727\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32220510   31465.342    30.728\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32221490   31466.299    30.729\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32222022   31466.818    30.729\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32222986   31467.760    30.730\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32223878   31468.631    30.731\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32224858   31469.588    30.732\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32225370   31470.088    30.733\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32226298   31470.994    30.733\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32227218   31471.893    30.734\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32227770   31472.432    30.735\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32228674   31473.314    30.736\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32229562   31474.182    30.737\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32230422   31475.021    30.737\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32231358   31475.936    30.738\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32232334   31476.889    30.739\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32233194   31477.729    30.740\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32234126   31478.639    30.741\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32235026   31479.518    30.742\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32235578   31480.057    30.742\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32236706   31481.158    30.743\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32237834   31482.260    30.744\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32238490   31482.900    30.745\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32239426   31483.814    30.746\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32240090   31484.463    30.747\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32240754   31485.111    30.747\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32241242   31485.588    30.748\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32241794   31486.127    30.748\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32242390   31486.709    30.749\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32243298   31487.596    30.750\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32243974   31488.256    30.750\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32244650   31488.916    30.751\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32245238   31489.490    30.751\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32246158   31490.389    30.752\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32247054   31491.264    30.753\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32247946   31492.135    30.754\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32248706   31492.877    30.755\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32249218   31493.377    30.755\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32250202   31494.338    30.756\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32251146   31495.260    30.757\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32251710   31495.811    30.758\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32252226   31496.314    30.758\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32253346   31497.408    30.759\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32254254   31498.295    30.760\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32255278   31499.295    30.761\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32255830   31499.834    30.762\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32256826   31500.807    30.763\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32257846   31501.803    30.763\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32258834   31502.768    30.764\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32259742   31503.654    30.765\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32260758   31504.646    30.766\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32261654   31505.521    30.767\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32262522   31506.369    30.768\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32263510   31507.334    30.769\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32264114   31507.924    30.769\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32264898   31508.689    30.770\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32265962   31509.729    30.771\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32266478   31510.232    30.772\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32267214   31510.951    30.772\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32267766   31511.490    30.773\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32268690   31512.393    30.774\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32268998   31512.693    30.774\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32269614   31513.295    30.775\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32270398   31514.061    30.775\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32271354   31514.994    30.776\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32271890   31515.518    30.777\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32272662   31516.271    30.778\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32273562   31517.150    30.778\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32274034   31517.611    30.779\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32274998   31518.553    30.780\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32275954   31519.486    30.781\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32276926   31520.436    30.782\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32277814   31521.303    30.783\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32278814   31522.279    30.783\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32279818   31523.260    30.784\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32280814   31524.232    30.785\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32281342   31524.748    30.786\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32281902   31525.295    30.786\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32282986   31526.354    30.787\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32284014   31527.357    30.788\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32284906   31528.229    30.789\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32285778   31529.080    30.790\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32286766   31530.045    30.791\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32287714   31530.971    30.792\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32288906   31532.135    30.793\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32289970   31533.174    30.794\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32290622   31533.811    30.795\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32291578   31534.744    30.796\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32292530   31535.674    30.797\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32293450   31536.572    30.797\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32294350   31537.451    30.798\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32295326   31538.404    30.799\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32295906   31538.971    30.800\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32296502   31539.553    30.800\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32297426   31540.455    30.801\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32298298   31541.307    30.802\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32299290   31542.275    30.803\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32300254   31543.217    30.804\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32301206   31544.146    30.805\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32301930   31544.854    30.806\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32302654   31545.561    30.806\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32303522   31546.408    30.807\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32304502   31547.365    30.808\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32305426   31548.268    30.809\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32306406   31549.225    30.810\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32307486   31550.279    30.811\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32308378   31551.150    30.812\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32309354   31552.104    30.813\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32310030   31552.764    30.813\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32310578   31553.299    30.814\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32311534   31554.232    30.815\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32312490   31555.166    30.816\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32313190   31555.850    30.816\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32314130   31556.768    30.817\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32315114   31557.729    30.818\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32316062   31558.654    30.819\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32316610   31559.189    30.820\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32317622   31560.178    30.820\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32318562   31561.096    30.821\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32318882   31561.408    30.822\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32319842   31562.346    30.823\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32320442   31562.932    30.823\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32321426   31563.893    30.824\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32322442   31564.885    30.825\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32323122   31565.549    30.826\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32323914   31566.322    30.826\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32324490   31566.885    30.827\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32325390   31567.764    30.828\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32326446   31568.795    30.829\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32327378   31569.705    30.830\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32327870   31570.186    30.830\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32328770   31571.064    30.831\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32329658   31571.932    30.832\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32330210   31572.471    30.832\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32330714   31572.963    30.833\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32331674   31573.900    30.834\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32332622   31574.826    30.835\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32333162   31575.354    30.835\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32334318   31576.482    30.836\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32335474   31577.611    30.838\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32336558   31578.670    30.839\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32337422   31579.514    30.839\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32338422   31580.490    30.840\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32339542   31581.584    30.841\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32340666   31582.682    30.842\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32341606   31583.600    30.843\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32342510   31584.482    30.844\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32343410   31585.361    30.845\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32343986   31585.924    30.846\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32344966   31586.881    30.847\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32345838   31587.732    30.847\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32346602   31588.479    30.848\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32347582   31589.436    30.849\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32348494   31590.326    30.850\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32349398   31591.209    30.851\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32350362   31592.150    30.852\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32350874   31592.650    30.852\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32351478   31593.240    30.853\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32352346   31594.088    30.854\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32352954   31594.682    30.854\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32353654   31595.365    30.855\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32354654   31596.342    30.856\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32355174   31596.850    30.856\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32356066   31597.721    30.857\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32356990   31598.623    30.858\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32357990   31599.600    30.859\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32358926   31600.514    30.860\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32359942   31601.506    30.861\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32360886   31602.428    30.862\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32361814   31603.334    30.863\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32362450   31603.955    30.863\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32363450   31604.932    30.864\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32364470   31605.928    30.865\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32365190   31606.631    30.866\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32366190   31607.607    30.867\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32367150   31608.545    30.868\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32368022   31609.396    30.869\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32368934   31610.287    30.869\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32369902   31611.232    30.870\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32370858   31612.166    30.871\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32371862   31613.146    30.872\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32372450   31613.721    30.873\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32373342   31614.592    30.874\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32374198   31615.428    30.874\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32375178   31616.385    30.875\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32375902   31617.092    30.876\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32376818   31617.986    30.877\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32377338   31618.494    30.877\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32378274   31619.408    30.878\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32379238   31620.350    30.879\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32379994   31621.088    30.880\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32380566   31621.646    30.881\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32381110   31622.178    30.881\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32382046   31623.092    30.882\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32382554   31623.588    30.882\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32383138   31624.158    30.883\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32384154   31625.150    30.884\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32385218   31626.189    30.885\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32385834   31626.791    30.886\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32386786   31627.721    30.886\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32387354   31628.275    30.887\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32388282   31629.182    30.888\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32389178   31630.057    30.889\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32390078   31630.936    30.890\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32391058   31631.893    30.891\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32391986   31632.799    30.891\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32392998   31633.787    30.892\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32393910   31634.678    30.893\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32394514   31635.268    30.894\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32395654   31636.381    30.895\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32396186   31636.900    30.895\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32396714   31637.416    30.896\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32397686   31638.365    30.897\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32398346   31639.010    30.897\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32399058   31639.705    30.898\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32399566   31640.201    30.899\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32400146   31640.768    30.899\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32401034   31641.635    30.900\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32402002   31642.580    30.901\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32403066   31643.619    30.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32403498   31644.041    30.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32404430   31644.951    30.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32404862   31645.373    30.904\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32405690   31646.182    30.904\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32406654   31647.123    30.905\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32407582   31648.029    30.906\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32408522   31648.947    30.907\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32409462   31649.865    30.908\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32410078   31650.467    30.909\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32410618   31650.994    30.909\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32411434   31651.791    30.910\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32411922   31652.268    30.910\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32412858   31653.182    30.911\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32413290   31653.604    30.912\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32414238   31654.529    30.913\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32415134   31655.404    30.913\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32416142   31656.389    30.914\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32417178   31657.400    30.915\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32418098   31658.299    30.916\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32419090   31659.268    30.917\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32420098   31660.252    30.918\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32421038   31661.170    30.919\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32421982   31662.092    30.920\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32422950   31663.037    30.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32423878   31663.943    30.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32424806   31664.850    30.923\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32425746   31665.768    30.924\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32426690   31666.689    30.925\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32427678   31667.654    30.925\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32428606   31668.561    30.926\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32429434   31669.369    30.927\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32430394   31670.307    30.928\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32431362   31671.252    30.929\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32432006   31671.881    30.930\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32432654   31672.514    30.930\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32433610   31673.447    30.931\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32434490   31674.307    30.932\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32435090   31674.893    30.933\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32435994   31675.775    30.933\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32436570   31676.338    30.934\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32437466   31677.213    30.935\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32438334   31678.061    30.936\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32439378   31679.080    30.937\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32439882   31679.572    30.937\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32440266   31679.947    30.937\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32440666   31680.338    30.938\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32441618   31681.268    30.939\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32442542   31682.170    30.940\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32443474   31683.080    30.941\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32444426   31684.010    30.941\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32445314   31684.877    30.942\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32446174   31685.717    30.943\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32447022   31686.545    30.944\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32447870   31687.373    30.945\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32448830   31688.311    30.946\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32449166   31688.639    30.946\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32450146   31689.596    30.947\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32451170   31690.596    30.948\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32452114   31691.518    30.949\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32453014   31692.396    30.950\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32453910   31693.271    30.950\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32454794   31694.135    30.951\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32455730   31695.049    30.952\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32456270   31695.576    30.953\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32456890   31696.182    30.953\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32457602   31696.877    30.954\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32458554   31697.807    30.955\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32459202   31698.439    30.956\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32460110   31699.326    30.956\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32461078   31700.271    30.957\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32461990   31701.162    30.958\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32462522   31701.682    30.959\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32463286   31702.428    30.959\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32464126   31703.248    30.960\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32464766   31703.873    30.961\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32465590   31704.678    30.962\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32466538   31705.604    30.963\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32467086   31706.139    30.963\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32468078   31707.107    30.964\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32468598   31707.615    30.964\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32469506   31708.502    30.965\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32470154   31709.135    30.966\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32471118   31710.076    30.967\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32472090   31711.025    30.968\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32472854   31711.771    30.969\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32473782   31712.678    30.969\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32474702   31713.576    30.970\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32475626   31714.479    30.971\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32476534   31715.365    30.972\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32477422   31716.232    30.973\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32477842   31716.643    30.973\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32478850   31717.627    30.974\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32479838   31718.592    30.975\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32480454   31719.193    30.976\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32481442   31720.158    30.977\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32482402   31721.096    30.978\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32482934   31721.615    30.978\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32483786   31722.447    30.979\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32484646   31723.287    30.980\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32485150   31723.779    30.980\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32485658   31724.275    30.981\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32486278   31724.881    30.981\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32486870   31725.459    30.982\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32487770   31726.338    30.983\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32488578   31727.127    30.984\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32489466   31727.994    30.984\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32490342   31728.850    30.985\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32490962   31729.455    30.986\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32492098   31730.564    30.987\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32493254   31731.693    30.988\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32494182   31732.600    30.989\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32495086   31733.482    30.990\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32495990   31734.365    30.991\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32496778   31735.135    30.991\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32497750   31736.084    30.992\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32498734   31737.045    30.993\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32499298   31737.596    30.994\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32500430   31738.701    30.995\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32501162   31739.416    30.996\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32502086   31740.318    30.996\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32503026   31741.236    30.997\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32503978   31742.166    30.998\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32504874   31743.041    30.999\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32505438   31743.592    31.000\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32506354   31744.486    31.000\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32507158   31745.271    31.001\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32508058   31746.150    31.002\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     32508782   31746.857    31.003\n",
      "openf___224_> Open           buildprofile/EAO00194build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26161929   25548.758    24.950\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26162457   25549.273    24.950\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26211057   25596.734    24.997\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26224357   25609.723    25.009\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           5uis.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26236567   25621.648    25.021\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26236643   25621.723    25.021\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26236814   25621.889    25.021\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26237664   25622.719    25.022\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26238922   25623.947    25.023\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26240826   25625.807    25.025\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26243682   25628.596    25.028\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26243722   25628.635    25.028\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26243812   25628.723    25.028\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26243872   25628.781    25.028\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26244007   25628.914    25.028\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26248291   25633.098    25.032\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26248379   25633.184    25.032\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26248577   25633.375    25.033\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26255003   25639.652    25.039\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26255135   25639.781    25.039\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26255432   25640.070    25.039\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26265054   25649.467    25.048\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26265254   25649.662    25.048\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26265704   25650.102    25.049\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26280154   25664.213    25.063\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26280454   25664.506    25.063\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26281129   25665.164    25.064\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26302787   25686.316    25.084\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26303235   25686.754    25.085\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26304243   25687.738    25.086\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26336747   25719.480    25.117\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     26568619   25945.918    25.338\n",
      "read_mo_297_> Segments, residues, atoms:        1      269     2113\n",
      "read_mo_298_> Segment:        1  165:A  458:A     2113\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26576391   25953.508    25.345\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26732442   26105.900    25.494\n",
      "openf___224_> Open           ali/EAO00194.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26825101   26196.387    25.582\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26907953   26277.297    25.661\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26909437   26278.746    25.663\n",
      "\n",
      "Read the alignment from file       : ali/EAO00194.ali\n",
      "\n",
      "Total number of alignment positions:   372\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      5uisD     269      1    5uis.pdb MOL_ID  1; MOLECULE  INTERLEUKIN-1 RECEPTOR-ASSOCIATED KINASE 4; CHAIN  A, B, C,\n",
      "  2   EAO00194     372      1    EAO00194\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          375\n",
      "  Score                    :  152252.8125\n",
      "openf___224_> Open           ali/EAO00194-5uisD.ali\n",
      "openf___224_> Open           ali/EAO00194-5uisD.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26161929   25548.758    24.950\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26162457   25549.273    24.950\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26211057   25596.734    24.997\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     26224357   25609.723    25.009\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00194-5uisD.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26236322   25621.408    25.021\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26237772   25622.824    25.022\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26240672   25625.656    25.025\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26246472   25631.320    25.031\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     26375680   25757.500    25.154\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26376752   25758.547    25.155\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26378236   25759.996    25.156\n",
      "\n",
      "Read the alignment from file       : ali/EAO00194-5uisD.ali\n",
      "\n",
      "Total number of alignment positions:   375\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      5uisD     269      1    5uis.pdb MOL_ID  1; MOLECULE  INTERLEUKIN-1 RECEPTOR-ASSOCIATED KINASE 4; CHAIN  A, B, C,\n",
      "  2   EAO00194     372      1    EAO00194\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           5uis.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26460661   25840.488    25.235\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26460661   25840.488    25.235\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26460832   25840.656    25.235\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26461682   25841.486    25.236\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26462940   25842.715    25.237\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26464844   25844.574    25.239\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26467700   25847.363    25.242\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26467700   25847.363    25.242\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26467790   25847.451    25.242\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26467790   25847.451    25.242\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26467925   25847.582    25.242\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26472209   25851.766    25.246\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26472209   25851.766    25.246\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26472407   25851.961    25.246\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26478833   25858.234    25.252\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26478833   25858.234    25.252\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26479130   25858.525    25.252\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26488752   25867.922    25.262\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26488752   25867.922    25.262\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26489202   25868.361    25.262\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26503652   25882.473    25.276\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26503652   25882.473    25.276\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26504327   25883.133    25.276\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26525985   25904.281    25.297\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     26525985   25904.281    25.297\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26526993   25905.266    25.298\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     26559497   25937.008    25.329\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26558885   25936.410    25.329\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     92     1  215   227      I     P    8.508\n",
      "    217     1  335   344      V     S   13.022\n",
      "    282     1  405   409      P     T    8.112\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26698137   26072.398    25.461\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26699737   26073.961    25.463\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26703717   26077.848    25.467\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26705717   26079.801    25.469\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26708717   26082.730    25.471\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26713217   26087.125    25.476\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26718789   26092.566    25.481\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26721589   26095.301    25.484\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26725789   26099.402    25.488\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26732089   26105.555    25.494\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26734861   26108.262    25.496\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     26736261   26109.629    25.498\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     26794508   26166.512    25.553\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     27083118   26448.357    25.828\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     27126318   26490.545    25.870\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      372   2926   2986      0         0      1207\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     27250826   26612.135    25.988\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      372   2926   2986   4034      4758      1207\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     27383402   26741.604    26.115\n",
      "patch_______> segment topology patched using RTF:  372 ;  LEU  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      372   2927   2987   4036      4760      1208\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      372   2927   2987   4036      4760      1208\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1677     1250\n",
      "openf___224_> Open           EAO00194.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      372     2927     2927\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27490851   26846.535    26.217\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27499043   26854.535    26.225\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27515427   26870.535    26.241\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27548195   26902.535    26.272\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27580963   26934.535    26.303\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27646499   26998.535    26.366\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27679267   27030.535    26.397\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27712035   27062.535    26.428\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27777571   27126.535    26.491\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27908643   27254.535    26.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27974179   27318.535    26.678\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28039715   27382.535    26.741\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28170787   27510.535    26.866\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28432931   27766.535    27.116\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28564003   27894.535    27.241\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     2987    4036    4538    1190\n",
      "              Total number of restraints before, now         :        0    12751\n",
      "make_re_422_> Number of previous, current restraints         :        0    12751\n",
      "make_re_423_> Number of previous, current selected restraints:        0    12751\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     28777504   28103.031    27.444\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       51  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       51  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       51  GLN\n",
      "getdata_643_> Protein accepted:  5uisD\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    12751    13121\n",
      "make_re_423_> Number of previous, current selected restraints:    12751    13121\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     28777520   28103.047    27.444\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  5uisD\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      389\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    13121    13492\n",
      "make_re_423_> Number of previous, current selected restraints:    13121    13103\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     28777520   28103.047    27.444\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  5uisD\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28908592   28231.047    27.569\n",
      "make_re_422_> Number of previous, current restraints         :    13492    13814\n",
      "make_re_423_> Number of previous, current selected restraints:    13103    13425\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     28908592   28231.047    27.569\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  5uisD\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    13814    14058\n",
      "make_re_423_> Number of previous, current selected restraints:    13425    13669\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     28908592   28231.047    27.569\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  5uisD\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     29170736   28487.047    27.819\n",
      "make_re_422_> Number of previous, current restraints         :    14058    14168\n",
      "make_re_423_> Number of previous, current selected restraints:    13669    13779\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     29170736   28487.047    27.819\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  5uisD\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    14168    14217\n",
      "make_re_423_> Number of previous, current selected restraints:    13779    13828\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     30013676   29310.230    28.623\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     30275820   29566.230    28.873\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     30800108   30078.230    29.373\n",
      "make_re_422_> Number of previous, current restraints         :    14217    19763\n",
      "make_re_423_> Number of previous, current selected restraints:    13828    19374\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     31062252   30334.230    29.623\n",
      "make_re_422_> Number of previous, current restraints         :    19763    25268\n",
      "make_re_423_> Number of previous, current selected restraints:    19374    24879\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    25268    27501\n",
      "make_re_423_> Number of previous, current selected restraints:    24879    27112\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    27501    28204\n",
      "make_re_423_> Number of previous, current selected restraints:    27112    27815\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    28204    28204\n",
      "make_re_423_> Number of previous, current selected restraints:    27815    27815\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    28204    28204\n",
      "make_re_423_> Number of previous, current selected restraints:    27815    27815\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    28204    28204\n",
      "make_re_423_> Number of previous, current selected restraints:    27815    27815\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    28204    28204\n",
      "make_re_423_> Number of previous, current selected restraints:    27815    27815\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    28204    28204\n",
      "make_re_423_> Number of previous, current selected restraints:    27815    27815\n",
      "rmdupl__427_>     2294 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    28204    25521\n",
      "openf___224_> Open           EAO00194.rsr\n",
      "openf___224_> Open           EAO00194.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     31047077   30319.410    29.609\n",
      "rdcsr2__307_> Number of restraints read    :    25521\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    25521    25521\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           EAO00194.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     31047109   30319.441    29.609\n",
      "read_mo_297_> Segments, residues, atoms:        1      372     2927\n",
      "read_mo_298_> Segment:        1  1:A  372:A     2927\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2927     2927        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    14510\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    15739\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    16544\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    17530\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    19055\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    20020\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    21585\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    23455\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    24867\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     31409929   30673.758    29.955\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     31480177   30742.359    30.022\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25435\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "iupac_m_487_> NH1/2 swapped:     -173.4738       18  18\n",
      "iupac_m_485_> OD1/2 will be swapped:      146.1301       19  19\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.4266       23  23\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      174.9388       34  34\n",
      "iupac_m_487_> NH1/2 swapped:     -171.9580       49  49\n",
      "iupac_m_486_> OE1/2 will be swapped:      145.3870       58  58\n",
      "iupac_m_487_> NH1/2 swapped:     -175.5893       66  66\n",
      "iupac_m_487_> NH1/2 swapped:     -175.8763       70  70\n",
      "iupac_m_487_> NH1/2 swapped:     -176.5178      100  100\n",
      "iupac_m_485_> OD1/2 will be swapped:      160.4390      104  104\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.8018      118  118\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.8357      130  130\n",
      "iupac_m_487_> NH1/2 swapped:     -175.4966      161  161\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -109.9670      164  164\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      109.8565      165  165\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.0928      176  176\n",
      "iupac_m_487_> NH1/2 swapped:      179.4688      179  179\n",
      "iupac_m_485_> OD1/2 will be swapped:     -173.3349      194  194\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.8439      213  213\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.7069      218  218\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8275      229  229\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.4909      231  231\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.5938      235  235\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.0109      262  262\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.3102      263  263\n",
      "iupac_m_487_> NH1/2 swapped:      177.1476      272  272\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.5335      283  283\n",
      "iupac_m_487_> NH1/2 swapped:      176.2309      292  292\n",
      "iupac_m_486_> OE1/2 will be swapped:      159.7321      305  305\n",
      "iupac_m_487_> NH1/2 swapped:     -176.2636      306  306\n",
      "iupac_m_485_> OD1/2 will be swapped:     -166.7689      335  335\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5666      339  339\n",
      "iupac_m_485_> OD1/2 will be swapped:     -164.5544      371  371\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     31913445   31165.473    30.435\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     32194437   31439.879    30.703\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     32756421   31988.691    31.239\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     33880389   33086.316    32.311\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     36128325   35281.566    34.455\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     40624197   39672.066    38.742\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   581303\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -31535.4043\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -31535.404297\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    33.828999\n",
      "Sequence length          :          372\n",
      "Compactness              :     0.206674\n",
      "Native energy (pair)     :   137.688055\n",
      "Native energy (surface)  :    10.399493\n",
      "Native energy (combined) :     4.715150\n",
      "Z score (pair)           :    -3.830423\n",
      "Z score (surface)        :    -3.187827\n",
      "Z score (combined)       :    -5.148936\n",
      "GA341 score              :     0.995818\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     6722\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2849.3772\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2987       0      0   0.005   0.005      25.516       1.000\n",
      " 2 Bond angle potential               :    4036       0      8   2.159   2.159      360.13       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1855       0     69  47.946  47.946      674.00       1.000\n",
      " 4 Stereochemical improper torsion pot:    1190       1      2   1.543   1.543      56.120       1.000\n",
      " 5 Soft-sphere overlap restraints     :    6722       0      3   0.005   0.005      22.219       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5546       0      8   0.761   0.761      389.75       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5505       0     17   0.891   0.891      471.83       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     371       3      7  14.161  14.161      117.45       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     322       0      4  77.101  77.101      103.91       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     244       0      1  71.687  71.687      102.48       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     110       0      0  83.532  83.532      71.948       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      49       0      0  93.006  93.006      30.876       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2233       0      0   0.571   0.571      37.585       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     370     101     64  36.887  90.719      313.30       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     703       0      0   2.278   2.278      72.266       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00194.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00194.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   51418.0703\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10530  92I  92I CA  C     697  702  178.87   -0.75  179.62   36.65   -0.75  179.62   36.65\n",
      "    2  10581 143Y 143Y CA  C    1107 1116 -148.52 -180.00   31.48    6.29 -180.00   31.48    6.29\n",
      "    3  10704 266I 266I CA  C    2093 2098    0.05 -179.18  179.23   41.66 -179.18  179.23   41.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10070   2K   3K C   N      16   18 -167.16 -118.00   79.80    2.96  -62.90  156.89   26.40\n",
      "    1          3K   3K N   CA     18   19 -158.04  139.10                  -40.80\n",
      "    2  10071   3K   4L C   N      25   27  -98.66  -70.70   45.40    2.82  -63.50  145.74   22.39\n",
      "    2          4L   4L N   CA     27   28  177.36  141.60                  -41.20\n",
      "    3  10072   4L   5G C   N      33   35 -158.00 -167.20   26.39    0.88   82.20 -174.70   13.96\n",
      "    3          5G   5G N   CA     35   36  149.87  174.60                    8.50\n",
      "    4  10076   8L   9D C   N      60   62  -80.49  -70.90   12.47    0.55  -63.30  162.65   21.15\n",
      "    4          9D   9D N   CA     62   63  158.27  150.30                  -40.00\n",
      "    5  10077   9D  10A C   N      68   70 -162.91 -134.00   37.22    0.99  -62.50  179.39   34.71\n",
      "    5         10A  10A N   CA     70   71  170.44  147.00                  -40.90\n",
      "    6  10078  10A  11A C   N      73   75  -82.01  -68.20   40.80    2.81  -62.50  136.81   23.43\n",
      "    6         11A  11A N   CA     75   76 -176.31  145.30                  -40.90\n",
      "    7  10081  13V  14E C   N      90   92  -89.83  -69.30   26.37    2.41  -63.60  168.31   21.11\n",
      "    7         14E  14E N   CA     92   93  125.96  142.50                  -40.30\n",
      "    8  10082  14E  15K C   N      99  101 -133.58 -118.00   31.60    1.26  -62.90  168.18   26.07\n",
      "    8         15K  15K N   CA    101  102  166.60  139.10                  -40.80\n",
      "    9  10084  16S  17Q C   N     114  116  -58.38  -73.00   28.59    1.65  -63.80  156.53   23.30\n",
      "    9         17Q  17Q N   CA    116  117  116.14  140.70                  -40.30\n",
      "   10  10085  17Q  18R C   N     123  125  -90.65  -72.10   23.89    1.55  -63.00  164.30   24.10\n",
      "   10         18R  18R N   CA    125  126  156.95  141.90                  -41.10\n",
      "   11  10097  29N  30S C   N     227  229 -139.61 -136.60   10.11    0.56   56.90 -165.62   24.50\n",
      "   11         30S  30S N   CA    229  230  141.55  151.20                   36.40\n",
      "   12  10098  30S  31L C   N     233  235 -139.53 -108.50   32.50    1.46  -63.50 -167.69   31.41\n",
      "   12         31L  31L N   CA    235  236  142.16  132.50                  -41.20\n",
      "   13  10099  31L  32Q C   N     241  243 -138.19 -121.10   49.56    2.02  -63.80  152.81   26.30\n",
      "   13         32Q  32Q N   CA    243  244 -173.78  139.70                  -40.30\n",
      "   14  10100  32Q  33L C   N     250  252  -90.02 -108.50   43.77    2.13  -63.50  136.63   17.63\n",
      "   14         33L  33L N   CA    252  253   92.83  132.50                  -41.20\n",
      "   15  10101  33L  34R C   N     258  260 -179.91 -125.20   64.13    1.90  -63.00 -173.86   32.69\n",
      "   15         34R  34R N   CA    260  261  174.05  140.60                  -41.10\n",
      "   16  10102  34R  35A C   N     269  271  -95.35  -68.20   40.90    4.02  -62.50  159.04   24.67\n",
      "   16         35A  35A N   CA    271  272  114.71  145.30                  -40.90\n",
      "   17  10103  35A  36F C   N     274  276  120.80 -124.20  115.11    4.35  -63.20 -117.07   43.55\n",
      "   17         36F  36F N   CA    276  277  148.26  143.30                  -44.30\n",
      "   18  10104  36F  37L C   N     285  287 -102.77 -108.50   20.63    1.16  -63.50  171.05   26.15\n",
      "   18         37L  37L N   CA    287  288  152.31  132.50                  -41.20\n",
      "   19  10105  37L  38L C   N     293  295  -83.65  -70.70   30.83    1.94  -63.50  150.57   22.11\n",
      "   19         38L  38L N   CA    295  296  169.58  141.60                  -41.20\n",
      "   20  10106  38L  39N C   N     301  303 -149.86 -119.90   46.48    1.39   55.90 -156.31   24.50\n",
      "   20         39N  39N N   CA    303  304  172.54  137.00                   39.50\n",
      "   21  10107  39N  40E C   N     309  311  -72.95  -69.30    4.81    0.27  -63.60  174.32   23.76\n",
      "   21         40E  40E N   CA    311  312  145.63  142.50                  -40.30\n",
      "   22  10108  40E  41S C   N     318  320 -118.05 -136.60   63.55    2.78  -64.10  136.53    8.51\n",
      "   22         41S  41S N   CA    320  321   90.42  151.20                  -35.00\n",
      "   23  10110  42G  43M C   N     328  330 -156.25 -125.60  113.96    4.73  -63.40  115.82   23.49\n",
      "   23         43M  43M N   CA    330  331 -109.74  140.50                  -40.50\n",
      "   24  10111  43M  44F C   N     336  338 -145.80 -124.20   29.12    0.77   58.10 -156.90   30.55\n",
      "   24         44F  44F N   CA    338  339  162.83  143.30                   32.90\n",
      "   25  10113  45T  46K C   N     354  356   62.13   56.60   11.71    1.26  -62.90  153.89   26.73\n",
      "   25         46K  46K N   CA    356  357   48.92   38.60                  -40.80\n",
      "   26  10115  47D  48G C   N     371  373   97.80   82.20   16.61    1.27  -62.40  169.51   31.87\n",
      "   26         48G  48G N   CA    373  374   14.22    8.50                  -41.20\n",
      "   27  10116  48G  49R C   N     375  377  -62.88  -63.00   11.15    1.50  -72.10  166.11   12.98\n",
      "   27         49R  49R N   CA    377  378  -52.25  -41.10                  141.90\n",
      "   28  10118  50K  51L C   N     395  397 -116.61 -108.50   28.43    1.60  -63.50  155.79   19.34\n",
      "   28         51L  51L N   CA    397  398  105.25  132.50                  -41.20\n",
      "   29  10120  52P  53G C   N     410  412   48.46   82.20   47.05    1.50  -62.40  138.19   26.00\n",
      "   29         53G  53G N   CA    412  413   41.30    8.50                  -41.20\n",
      "   30  10121  53G  54S C   N     414  416  -61.88  -72.40   21.72    1.00  -64.10  168.42   12.34\n",
      "   30         54S  54S N   CA    416  417  133.40  152.40                  -35.00\n",
      "   31  10122  54S  55V C   N     420  422  -98.36 -125.40   28.52    0.85  -62.40 -179.74   20.96\n",
      "   31         55V  55V N   CA    422  423  134.24  143.30                  -42.40\n",
      "   32  10123  55V  56T C   N     427  429 -134.13 -124.80   32.51    1.14  -63.20  159.86   25.11\n",
      "   32         56T  56T N   CA    429  430  174.64  143.50                  -42.10\n",
      "   33  10124  56T  57K C   N     434  436   81.69   56.60  119.66   10.50  -62.90 -141.66   26.03\n",
      "   33         57K  57K N   CA    436  437  155.60   38.60                  -40.80\n",
      "   34  10128  60I  61D C   N     467  469 -156.02  -96.50   60.22    2.45  -63.30  172.15   18.35\n",
      "   34         61D  61D N   CA    469  470  105.05  114.20                  -40.00\n",
      "   35  10129  61D  62D C   N     475  477  -74.59  -70.90   23.23    0.97   54.50 -175.13   11.14\n",
      "   35         62D  62D N   CA    477  478  173.23  150.30                   40.90\n",
      "   36  10133  65K  66R C   N     505  507  175.81 -125.20   88.49    2.77  -63.00  165.25   29.74\n",
      "   36         66R  66R N   CA    507  508 -153.44  140.60                  -41.10\n",
      "   37  10138  70R  71G C   N     547  549   69.37   82.20   72.72    2.88  -62.40  179.10   33.36\n",
      "   37         71G  71G N   CA    549  550   80.09    8.50                  -41.20\n",
      "   38  10139  71G  72A C   N     551  553  -40.29  -68.20   46.14    2.91  -62.50  151.10   25.91\n",
      "   38         72A  72A N   CA    553  554  108.56  145.30                  -40.90\n",
      "   39  10140  72A  73S C   N     556  558  174.90 -136.60   86.49    3.12  -64.10  158.37   20.04\n",
      "   39         73S  73S N   CA    558  559 -137.19  151.20                  -35.00\n",
      "   40  10150  82K  83D C   N     632  634  -67.47  -96.50   34.28    1.40  -63.30  172.48   21.04\n",
      "   40         83D  83D N   CA    634  635  132.43  114.20                  -40.00\n",
      "   41  10151  83D  84G C   N     640  642   60.97   78.70   45.50    0.69   82.20  134.39    7.57\n",
      "   41         84G  84G N   CA    642  643 -124.20 -166.10                    8.50\n",
      "   42  10160  92I  93P C   N     702  704  -81.41  -64.50   27.23    2.89  -58.70  158.00   11.31\n",
      "   42         93P  93P N   CA    704  705  125.86  147.20                  -30.50\n",
      "   43  10162  94I  95T C   N     717  719 -114.72 -124.80   26.15    1.41  -63.20  158.86   23.79\n",
      "   43         95T  95T N   CA    719  720  167.62  143.50                  -42.10\n",
      "   44  10163  95T  96S C   N     724  726  -70.04  -72.40   16.03    0.86  -64.10  171.65   12.10\n",
      "   44         96S  96S N   CA    726  727  136.55  152.40                  -35.00\n",
      "   45  10164  96S  97K C   N     730  732   99.22   56.60  142.72    7.56  -62.90  171.79   25.10\n",
      "   45         97K  97K N   CA    732  733  -97.61   38.60                  -40.80\n",
      "   46  10166  98P  99H C   N     746  748 -108.59  -63.20   76.71   12.46  -63.20   76.71   12.46\n",
      "   46         99H  99H N   CA    748  749 -104.13  -42.30                  -42.30\n",
      "   47  10196 128D 129S C   N     996  998   65.91  -72.40  155.24   12.43  -64.10  165.30   14.79\n",
      "   47        129S 129S N   CA    998  999 -137.09  152.40                  -35.00\n",
      "   48  10209 141M 142A C   N    1099 1101   70.46  -68.20  146.90   13.86  -62.50 -177.32   28.42\n",
      "   48        142A 142A N   CA   1101 1102 -166.17  145.30                  -40.90\n",
      "   49  10219 151F 152W C   N    1179 1181   64.14  -63.00  130.53   20.16  -63.00  130.53   20.16\n",
      "   49        152W 152W N   CA   1181 1182  -73.75  -44.20                  -44.20\n",
      "   50  10220 152W 153E C   N    1193 1195  -55.76  -69.30   13.87    1.15   54.60  151.03    9.58\n",
      "   50        153E 153E N   CA   1195 1196  145.51  142.50                   42.40\n",
      "   51  10221 153E 154G C   N    1202 1204  -63.49  -62.40    9.73    1.40   82.20  151.09   11.31\n",
      "   51        154G 154G N   CA   1204 1205  -31.53  -41.20                    8.50\n",
      "   52  10222 154G 155I C   N    1206 1208  -14.29  -97.30   83.03    3.44 -120.60  106.32    7.34\n",
      "   52        155I 155I N   CA   1208 1209  128.89  127.20                  130.30\n",
      "   53  10226 158T 159M C   N    1239 1241  -81.04  -73.00   44.49    3.15  -63.40  140.85   19.72\n",
      "   53        159M 159M N   CA   1241 1242   99.24  143.00                  -40.50\n",
      "   54  10251 183R 184D C   N    1453 1455 -105.56  -63.30   60.15    6.78  -63.30   60.15    6.78\n",
      "   54        184D 184D N   CA   1455 1456    2.80  -40.00                  -40.00\n",
      "   55  10275 207K 208L C   N    1639 1641 -162.21 -108.50   53.75    2.54  -63.50 -158.40   24.70\n",
      "   55        208L 208L N   CA   1641 1642  134.58  132.50                  -41.20\n",
      "   56  10278 210H 211T C   N    1664 1666  -71.85  -78.10    6.49    0.42  -63.20  166.57   21.93\n",
      "   56        211T 211T N   CA   1666 1667  151.56  149.80                  -42.10\n",
      "   57  10279 211T 212L C   N    1671 1673  -79.37  -70.70   37.71    3.19  -63.50  146.96   19.55\n",
      "   57        212L 212L N   CA   1673 1674  104.90  141.60                  -41.20\n",
      "   58  10280 212L 213D C   N    1679 1681 -121.26  -96.50   77.22    3.22  -63.30   99.64   10.71\n",
      "   58        213D 213D N   CA   1681 1682   41.06  114.20                  -40.00\n",
      "   59  10283 215S 216S C   N    1700 1702 -153.22 -136.60   67.88    3.06  -64.10  140.01   16.57\n",
      "   59        216S 216S N   CA   1702 1703 -142.99  151.20                  -35.00\n",
      "   60  10298 230L 231E C   N    1822 1824  -63.80  -69.30   30.64    2.14  -63.60  152.66   20.34\n",
      "   60        231E 231E N   CA   1824 1825  112.35  142.50                  -40.30\n",
      "   61  10300 232Q 233G C   N    1840 1842  -63.39  -62.40    1.01    0.19   82.20  153.90   11.70\n",
      "   61        233G 233G N   CA   1842 1843  -41.36  -41.20                    8.50\n",
      "   62  10323 255K 256N C   N    2008 2010  -47.89  -71.20   23.38    1.74  -63.20  175.00   21.07\n",
      "   62        256N 256N N   CA   2010 2011  144.57  142.80                  -41.10\n",
      "   63  10344 276D 277T C   N    2174 2176  -89.69 -124.80   93.79    3.10  -63.20  102.13   11.55\n",
      "   63        277T 277T N   CA   2176 2177   56.53  143.50                  -42.10\n",
      "   64  10346 278T 279P C   N    2188 2190  -73.13  -64.50   67.47    4.60  -58.70  116.29   10.80\n",
      "   64        279P 279P N   CA   2190 2191 -145.89  147.20                  -30.50\n",
      "   65  10349 281L 282P C   N    2210 2212  -80.15  -64.50   22.92    1.43  -58.70  166.95   15.56\n",
      "   65        282P 282P N   CA   2212 2213  163.94  147.20                  -30.50\n",
      "   66  10350 282P 283E C   N    2217 2219  -79.54  -69.30   23.17    1.41  -63.60  157.23   21.85\n",
      "   66        283E 283E N   CA   2219 2220  163.28  142.50                  -40.30\n",
      "   67  10351 283E 284K C   N    2226 2228  -83.25  -70.20   13.51    1.06  -62.90  178.86   22.19\n",
      "   67        284K 284K N   CA   2228 2229  136.90  140.40                  -40.80\n",
      "   68  10353 285P 286A C   N    2242 2244   83.18 -134.00  146.77    3.73  -62.50 -159.16   31.23\n",
      "   68        286A 286A N   CA   2244 2245 -179.16  147.00                  -40.90\n",
      "   69  10354 286A 287Y C   N    2247 2249  -68.15  -98.40   31.17    1.49  -63.50  164.36   25.84\n",
      "   69        287Y 287Y N   CA   2249 2250  120.89  128.40                  -43.40\n",
      "   70  10355 287Y 288S C   N    2259 2261 -128.47 -136.60   18.28    0.72  -64.10 -178.38   11.21\n",
      "   70        288S 288S N   CA   2261 2262  134.83  151.20                  -35.00\n",
      "   71  10357 289E 290E C   N    2274 2276 -137.66 -117.80   50.24    2.06  -63.60  155.52   24.85\n",
      "   71        290E 290E N   CA   2276 2277 -177.06  136.80                  -40.30\n",
      "   72  10358 290E 291A C   N    2283 2285 -143.86 -134.00   21.42    0.90  -62.50  173.36   32.76\n",
      "   72        291A 291A N   CA   2285 2286  166.01  147.00                  -40.90\n",
      "   73  10359 291A 292R C   N    2288 2290 -109.09 -125.20   16.76    0.56  -63.00 -177.02   21.99\n",
      "   73        292R 292R N   CA   2290 2291  135.98  140.60                  -41.10\n",
      "   74  10364 296K 297K C   N    2334 2336 -116.59  -62.90   82.83   13.87  -62.90   82.83   13.87\n",
      "   74        297K 297K N   CA   2336 2337 -103.87  -40.80                  -40.80\n",
      "   75  10369 301R 302D C   N    2375 2377 -110.71  -96.50   69.97    2.93  -63.30   97.92   10.39\n",
      "   75        302D 302D N   CA   2377 2378   45.69  114.20                  -40.00\n",
      "   76  10370 302D 303P C   N    2383 2385  -65.97  -58.70   17.23    1.01  -64.50  162.09   12.17\n",
      "   76        303P 303P N   CA   2385 2386  -14.88  -30.50                  147.20\n",
      "   77  10372 304Q 305E C   N    2399 2401  -53.97  -69.30   21.09    1.94  -63.60  163.00   21.22\n",
      "   77        305E 305E N   CA   2401 2402  156.98  142.50                  -40.30\n",
      "   78  10381 313L 314K C   N    2469 2471  -71.99 -118.00   87.77    4.54  -62.90  105.74   14.25\n",
      "   78        314K 314K N   CA   2471 2472 -146.15  139.10                  -40.80\n",
      "   79  10382 314K 315H C   N    2478 2480  -86.06  -67.60   45.38    2.93  -63.20  138.14   18.75\n",
      "   79        315H 315H N   CA   2480 2481 -178.54  140.00                  -42.30\n",
      "   80  10384 316P 317F C   N    2495 2497 -141.64 -124.20   33.94    1.09  -63.20  163.35   27.67\n",
      "   80        317F 317F N   CA   2497 2498  172.41  143.30                  -44.30\n",
      "   81  10386 318F 319K C   N    2517 2519 -125.98 -118.00   11.77    0.42  -62.90 -177.32   27.49\n",
      "   81        319K 319K N   CA   2519 2520  147.75  139.10                  -40.80\n",
      "   82  10387 319K 320G C   N    2526 2528  130.40   78.70   53.00    2.35   82.20 -179.75   11.07\n",
      "   82        320G 320G N   CA   2528 2529 -177.81 -166.10                    8.50\n",
      "   83  10389 321C 322T C   N    2536 2538 -128.37 -124.80   55.95    2.51  -63.20  145.21   15.46\n",
      "   83        322T 322T N   CA   2538 2539   87.67  143.50                  -42.10\n",
      "   84  10398 330V 331K C   N    2595 2597  -79.20  -70.20   16.03    0.97  -62.90  166.34   22.53\n",
      "   84        331K 331K N   CA   2597 2598  153.66  140.40                  -40.80\n",
      "   85  10399 331K 332Q C   N    2604 2606 -124.93 -121.10    7.38    0.39  -63.80 -175.86   24.35\n",
      "   85        332Q 332Q N   CA   2606 2607  133.39  139.70                  -40.30\n",
      "   86  10400 332Q 333G C   N    2613 2615 -150.05 -167.20   53.19    1.80   82.20  172.38   13.32\n",
      "   86        333G 333G N   CA   2615 2616  124.25  174.60                    8.50\n",
      "   87  10401 333G 334V C   N    2617 2619  -64.88  -62.40    3.26    0.55 -125.40 -177.50   10.22\n",
      "   87        334V 334V N   CA   2619 2620  -44.52  -42.40                  143.30\n",
      "   88  10402 334V 335D C   N    2624 2626  -76.67  -70.90   20.27    0.75  -63.30  150.87   19.45\n",
      "   88        335D 335D N   CA   2626 2627  169.73  150.30                  -40.00\n",
      "   89  10403 335D 336Y C   N    2632 2634  -98.24  -98.40    3.35    0.36  -63.50  178.56   26.63\n",
      "   89        336Y 336Y N   CA   2634 2635  131.75  128.40                  -43.40\n",
      "   90  10404 336Y 337M C   N    2644 2646  -83.92  -73.00   28.81    2.14  -63.40  158.18   22.09\n",
      "   90        337M 337M N   CA   2646 2647  116.34  143.00                  -40.50\n",
      "   91  10406 338T 339R C   N    2659 2661  -64.59  -72.10   29.27    2.40  -63.00  148.72   20.20\n",
      "   91        339R 339R N   CA   2661 2662  170.19  141.90                  -41.10\n",
      "   92  10407 339R 340L C   N    2670 2672  -65.55  -70.70   15.79    1.39  -63.50  162.28   22.65\n",
      "   92        340L 340L N   CA   2672 2673  156.53  141.60                  -41.20\n",
      "   93  10408 340L 341I C   N    2678 2680  -58.90  -63.40   12.08    1.86 -120.60 -174.55   12.89\n",
      "   93        341I 341I N   CA   2680 2681  -54.82  -43.60                  130.30\n",
      "   94  10409 341I 342N C   N    2686 2688   56.78   55.90   21.74    1.46  -63.20  133.64   21.93\n",
      "   94        342N 342N N   CA   2688 2689   17.78   39.50                  -41.10\n",
      "   95  10410 342N 343S C   N    2694 2696 -149.11 -136.60   21.72    1.23  -64.10 -171.32   12.04\n",
      "   95        343S 343S N   CA   2696 2697  133.45  151.20                  -35.00\n",
      "   96  10411 343S 344D C   N    2700 2702  -98.31  -70.90   45.47    1.62  -63.30  137.94   19.28\n",
      "   96        344D 344D N   CA   2702 2703 -173.43  150.30                  -40.00\n",
      "   97  10415 347K 348N C   N    2731 2733 -101.05 -119.90   19.27    0.90  -63.20 -178.13   24.89\n",
      "   97        348N 348N N   CA   2733 2734  141.01  137.00                  -41.10\n",
      "   98  10417 349G 350L C   N    2743 2745  -99.79  -63.50   74.29   12.55  -63.50   74.29   12.55\n",
      "   98        350L 350L N   CA   2745 2746 -106.02  -41.20                  -41.20\n",
      "   99  10421 353R 354S C   N    2779 2781  -56.74  -72.40   17.99    0.98  -64.10  178.69   13.40\n",
      "   99        354S 354S N   CA   2781 2782  143.54  152.40                  -35.00\n",
      "  100  10427 359T 360K C   N    2823 2825 -102.69  -62.90   56.80    9.65  -62.90   56.80    9.65\n",
      "  100        360K 360K N   CA   2825 2826  -81.33  -40.80                  -40.80\n",
      "  101  10437 369L 370G C   N    2905 2907 -126.79 -167.20   57.75    1.31   82.20 -164.06   15.23\n",
      "  101        370G 370G N   CA   2907 2908  133.35  174.60                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   24   60  204  251  352  338  406  411  495  532\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00194.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      372     2927     2927\n",
      "openf___224_> Open           EAO00194.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     40331457   39386.188    38.463\n",
      "read_mo_297_> Segments, residues, atoms:        1      372     2927\n",
      "read_mo_298_> Segment:        1  1:A  372:A     2927\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2927     2927        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    14510\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    15739\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    16544\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    17530\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    19055\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    20020\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    21585\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    23455\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    24867\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25435\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.1078       14  14\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4582       18  18\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0482       34  34\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.2585       40  40\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.7141       58  58\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4868       66  66\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8783      100  100\n",
      "iupac_m_486_> OE1/2 will be swapped:     -168.3885      102  102\n",
      "iupac_m_487_> NH1/2 swapped:      179.1282      105  105\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.8640      106  106\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.3957      110  110\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.0680      131  131\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.3283      143  143\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      103.7131      151  151\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8442      161  161\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.4133      165  165\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      154.1635      176  176\n",
      "iupac_m_487_> NH1/2 swapped:      178.1300      179  179\n",
      "iupac_m_487_> NH1/2 swapped:     -176.8486      183  183\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.4770      194  194\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.6562      195  195\n",
      "iupac_m_485_> OD1/2 will be swapped:      109.5053      202  202\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.4333      213  213\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.1982      231  231\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.9036      235  235\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.8162      262  262\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.8152      263  263\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0615      267  267\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.7421      272  272\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.8652      283  283\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9747      292  292\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.2827      293  293\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3113      301  301\n",
      "iupac_m_485_> OD1/2 will be swapped:      166.7968      302  302\n",
      "iupac_m_487_> NH1/2 swapped:      179.6557      306  306\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.3783      335  335\n",
      "iupac_m_485_> OD1/2 will be swapped:     -165.4733      344  344\n",
      "iupac_m_487_> NH1/2 swapped:      179.7521      353  353\n",
      "iupac_m_486_> OE1/2 will be swapped:     -141.7038      356  356\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   580313\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -31141.9727\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -31141.972656\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    33.828999\n",
      "Sequence length          :          372\n",
      "Compactness              :     0.169607\n",
      "Native energy (pair)     :   184.214733\n",
      "Native energy (surface)  :     7.815052\n",
      "Native energy (combined) :     5.117562\n",
      "Z score (pair)           :    -3.292651\n",
      "Z score (surface)        :    -3.749604\n",
      "Z score (combined)       :    -4.686319\n",
      "GA341 score              :     0.959799\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     6988\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2545.6304\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2987       0      0   0.005   0.005      19.403       1.000\n",
      " 2 Bond angle potential               :    4036       0      7   2.007   2.007      305.65       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1855       0     92  49.290  49.290      696.69       1.000\n",
      " 4 Stereochemical improper torsion pot:    1190       0      0   1.307   1.307      41.177       1.000\n",
      " 5 Soft-sphere overlap restraints     :    6988       0      0   0.003   0.003      9.5131       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5546       0      5   0.632   0.632      285.96       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5505       0      8   0.769   0.769      396.22       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     371       1      2  10.253  10.253      76.723       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     322       0      4  82.449  82.449      98.942       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     244       0      2  75.494  75.494      104.42       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     110       0      0  87.852  87.852      65.105       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      49       0      0  97.541  97.541      34.188       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2233       0      0   0.603   0.603      40.243       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     370      99     68  38.274  91.698      311.36       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     703       0      3   1.460   1.460      60.029       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00194.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00194.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   49935.9961\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10070   2K   3K C   N      16   18 -175.68 -118.00   80.26    2.82  -62.90  167.83   28.30\n",
      "    1          3K   3K N   CA     18   19 -165.09  139.10                  -40.80\n",
      "    2  10071   3K   4L C   N      25   27  -71.81  -70.70    2.23    0.21  -63.50  179.32   25.38\n",
      "    2          4L   4L N   CA     27   28  139.67  141.60                  -41.20\n",
      "    3  10072   4L   5G C   N      33   35  -96.08  -80.20   30.81    0.73   82.20 -115.04   18.88\n",
      "    3          5G   5G N   CA     35   36 -159.49  174.10                    8.50\n",
      "    4  10073   5G   6L C   N      37   39  -66.38  -70.70    4.41    0.42  -63.50  176.34   24.65\n",
      "    4          6L   6L N   CA     39   40  142.48  141.60                  -41.20\n",
      "    5  10075   7V   8L C   N      52   54 -126.76 -108.50   43.05    2.10  -63.50  160.32   26.18\n",
      "    5          8L   8L N   CA     54   55  171.48  132.50                  -41.20\n",
      "    6  10077   9D  10A C   N      68   70  -69.66  -68.20    8.27    0.62  -62.50  165.81   27.53\n",
      "    6         10A  10A N   CA     70   71  153.44  145.30                  -40.90\n",
      "    7  10078  10A  11A C   N      73   75  -65.13  -68.20    5.52    0.35  -62.50  178.41   29.37\n",
      "    7         11A  11A N   CA     75   76  140.71  145.30                  -40.90\n",
      "    8  10079  11A  12A C   N      78   80  -78.24  -68.20   18.13    1.16  -62.50  159.49   26.94\n",
      "    8         12A  12A N   CA     80   81  160.40  145.30                  -40.90\n",
      "    9  10081  13V  14E C   N      90   92  -58.66  -69.30   12.19    0.72  -63.60  176.91   23.86\n",
      "    9         14E  14E N   CA     92   93  136.54  142.50                  -40.30\n",
      "   10  10082  14E  15K C   N      99  101  -77.77  -70.20   10.34    0.87  -62.90  174.79   21.93\n",
      "   10         15K  15K N   CA    101  102  133.36  140.40                  -40.80\n",
      "   11  10084  16S  17Q C   N     114  116  -45.62  -73.00   48.71    2.80  -63.80  141.88   21.83\n",
      "   11         17Q  17Q N   CA    116  117  100.41  140.70                  -40.30\n",
      "   12  10085  17Q  18R C   N     123  125  161.77 -125.20  137.56    4.75  -63.00  148.64   26.99\n",
      "   12         18R  18R N   CA    125  126 -102.82  140.60                  -41.10\n",
      "   13  10097  29N  30S C   N     227  229 -143.36 -136.60   12.40    0.70   56.90 -169.17   24.02\n",
      "   13         30S  30S N   CA    229  230  140.80  151.20                   36.40\n",
      "   14  10098  30S  31L C   N     233  235 -118.53 -108.50   11.19    0.49  -63.50 -173.05   23.44\n",
      "   14         31L  31L N   CA    235  236  137.47  132.50                  -41.20\n",
      "   15  10099  31L  32Q C   N     241  243 -109.50 -121.10   23.39    1.22  -63.80  166.10   26.86\n",
      "   15         32Q  32Q N   CA    243  244  160.01  139.70                  -40.30\n",
      "   16  10100  32Q  33L C   N     250  252  -82.28 -108.50   48.29    2.27  -63.50  134.47   17.68\n",
      "   16         33L  33L N   CA    252  253   91.95  132.50                  -41.20\n",
      "   17  10101  33L  34R C   N     258  260 -163.98 -125.20   48.36    1.43  -63.00 -179.67   31.04\n",
      "   17         34R  34R N   CA    260  261  169.50  140.60                  -41.10\n",
      "   18  10102  34R  35A C   N     269  271  -67.91  -68.20    2.16    0.16  -62.50  176.02   29.11\n",
      "   18         35A  35A N   CA    271  272  143.17  145.30                  -40.90\n",
      "   19  10103  35A  36F C   N     274  276   69.35  -71.40  141.50   10.04  -63.20 -151.91   23.59\n",
      "   19         36F  36F N   CA    276  277  155.28  140.70                  -44.30\n",
      "   20  10104  36F  37L C   N     285  287 -100.40 -108.50    8.25    0.41  -63.50  179.11   23.02\n",
      "   20         37L  37L N   CA    287  288  134.06  132.50                  -41.20\n",
      "   21  10105  37L  38L C   N     293  295 -139.61 -108.50   31.16    1.46  -63.50 -168.79   23.53\n",
      "   21         38L  38L N   CA    295  296  134.21  132.50                  -41.20\n",
      "   22  10106  38L  39N C   N     301  303 -100.14 -119.90   65.62    2.49   55.90  159.90   16.03\n",
      "   22         39N  39N N   CA    303  304   74.43  137.00                   39.50\n",
      "   23  10107  39N  40E C   N     309  311  -71.31  -69.30   35.67    2.79  -63.60  147.40   19.24\n",
      "   23         40E  40E N   CA    311  312  106.89  142.50                  -40.30\n",
      "   24  10108  40E  41S C   N     318  320  152.45 -136.60   99.29    3.19  -64.10  177.39   22.89\n",
      "   24         41S  41S N   CA    320  321 -139.35  151.20                  -35.00\n",
      "   25  10109  41S  42G C   N     324  326  -78.69  -80.20   27.41    1.16   82.20 -128.08   17.72\n",
      "   25         42G  42G N   CA    326  327 -158.53  174.10                    8.50\n",
      "   26  10111  43M  44F C   N     336  338 -119.25 -124.20   12.42    0.44   58.10 -156.88   19.36\n",
      "   26         44F  44F N   CA    338  339  131.91  143.30                   32.90\n",
      "   27  10112  44F  45T C   N     347  349  -84.69  -63.20   83.35   12.12  -63.20   83.35   12.12\n",
      "   27         45T  45T N   CA    349  350 -122.63  -42.10                  -42.10\n",
      "   28  10114  46K  47D C   N     363  365  -57.84  -63.30   17.44    2.51  -70.90  174.22    7.94\n",
      "   28         47D  47D N   CA    365  366  -23.43  -40.00                  150.30\n",
      "   29  10119  51L  52P C   N     403  405  -70.88  -58.70   20.55    1.27  -64.50  161.27   12.37\n",
      "   29         52P  52P N   CA    405  406  -13.95  -30.50                  147.20\n",
      "   30  10122  54S  55V C   N     420  422  -65.43 -125.40   66.30    1.83  -62.40  157.46   19.75\n",
      "   30         55V  55V N   CA    422  423  115.03  143.30                  -42.40\n",
      "   31  10123  55V  56T C   N     427  429 -128.04 -124.80   13.67    0.49  -63.20  173.68   26.54\n",
      "   31         56T  56T N   CA    429  430  156.78  143.50                  -42.10\n",
      "   32  10124  56T  57K C   N     434  436  109.35 -118.00  137.36    4.69  -62.90 -135.20   28.22\n",
      "   32         57K  57K N   CA    436  437  174.75  139.10                  -40.80\n",
      "   33  10125  57K  58E C   N     443  445  -75.84  -69.30   39.73    3.25  -63.60  144.14   18.57\n",
      "   33         58E  58E N   CA    445  446  103.32  142.50                  -40.30\n",
      "   34  10128  60I  61D C   N     467  469 -160.46  -96.50   80.62    3.29  -63.30  143.15   15.94\n",
      "   34         61D  61D N   CA    469  470   65.13  114.20                  -40.00\n",
      "   35  10129  61D  62D C   N     475  477 -109.41  -96.50   14.16    0.59   54.50 -178.00   15.90\n",
      "   35         62D  62D N   CA    477  478  120.00  114.20                   40.90\n",
      "   36  10138  70R  71G C   N     547  549   74.40   82.20   57.49    2.40  -62.40  173.46   32.58\n",
      "   36         71G  71G N   CA    549  550   65.46    8.50                  -41.20\n",
      "   37  10139  71G  72A C   N     551  553  -43.91  -68.20   45.66    2.93  -62.50  148.71   25.32\n",
      "   37         72A  72A N   CA    553  554  106.64  145.30                  -40.90\n",
      "   38  10140  72A  73S C   N     556  558  168.24 -136.60  104.61    3.88  -64.10  153.32   19.97\n",
      "   38         73S  73S N   CA    558  559 -119.92  151.20                  -35.00\n",
      "   39  10149  81L  82K C   N     623  625   49.72  -70.20  129.88   10.50 -118.00  175.35    7.57\n",
      "   39         82K  82K N   CA    625  626 -169.72  140.40                  139.10\n",
      "   40  10151  83D  84G C   N     640  642  -53.56  -62.40    9.31    1.52   82.20  145.61   11.16\n",
      "   40         84G  84G N   CA    642  643  -44.13  -41.20                    8.50\n",
      "   41  10159  91H  92I C   N     694  696   76.69 -120.60  163.91   10.71 -120.60  163.91   10.71\n",
      "   41         92I  92I N   CA    696  697  150.09  130.30                  130.30\n",
      "   42  10160  92I  93P C   N     702  704  -77.65  -64.50   25.68    2.61  -58.70  156.79   11.49\n",
      "   42         93P  93P N   CA    704  705  125.14  147.20                  -30.50\n",
      "   43  10162  94I  95T C   N     717  719  169.26 -124.80   81.80    2.89  -63.20  179.29   30.46\n",
      "   43         95T  95T N   CA    719  720 -168.11  143.50                  -42.10\n",
      "   44  10163  95T  96S C   N     724  726  -93.10  -72.40   61.27    4.31  -64.10  132.93    8.36\n",
      "   44         96S  96S N   CA    726  727   94.73  152.40                  -35.00\n",
      "   45  10164  96S  97K C   N     730  732 -170.30 -118.00   71.91    2.51  -62.90  169.19   28.23\n",
      "   45         97K  97K N   CA    732  733 -171.54  139.10                  -40.80\n",
      "   46  10166  98P  99H C   N     746  748   58.96   56.30   14.96    1.14  -63.20  139.99   24.05\n",
      "   46         99H  99H N   CA    748  749   26.08   40.80                  -42.30\n",
      "   47  10187 119V 120M C   N     921  923  -49.07  -73.00   42.94    2.67  -63.40  148.54   23.31\n",
      "   47        120M 120M N   CA    923  924  107.35  143.00                  -40.50\n",
      "   48  10209 141M 142A C   N    1099 1101   82.72 -134.00  153.88    3.53  -62.50 -174.15   29.51\n",
      "   48        142A 142A N   CA   1101 1102 -156.88  147.00                  -40.90\n",
      "   49  10219 151F 152W C   N    1179 1181   58.00  -71.30  141.88   11.62  -63.00  169.28   20.27\n",
      "   49        152W 152W N   CA   1181 1182 -162.58  139.00                  -44.20\n",
      "   50  10220 152W 153E C   N    1193 1195 -126.78 -117.80   25.51    1.08   54.60 -145.77   27.99\n",
      "   50        153E 153E N   CA   1195 1196  160.68  136.80                   42.40\n",
      "   51  10226 158T 159M C   N    1239 1241  -91.48  -73.00   47.28    3.53  -63.40  142.77   19.30\n",
      "   51        159M 159M N   CA   1241 1242   99.48  143.00                  -40.50\n",
      "   52  10244 176D 177T C   N    1390 1392  -91.78 -124.80   36.96    1.47  -63.20  171.40   20.22\n",
      "   52        177T 177T N   CA   1392 1393  126.90  143.50                  -42.10\n",
      "   53  10245 177T 178K C   N    1397 1399  -58.36  -70.20   37.94    3.00   56.60  179.49   10.42\n",
      "   53        178K 178K N   CA   1399 1400  176.44  140.40                   38.60\n",
      "   54  10246 178K 179R C   N    1406 1408   59.23   57.30    3.83    0.40  -63.00  147.42   26.85\n",
      "   54        179R 179R N   CA   1408 1409   41.31   38.00                  -41.10\n",
      "   55  10251 183R 184D C   N    1453 1455 -107.55  -63.30   61.61    7.00  -63.30   61.61    7.00\n",
      "   55        184D 184D N   CA   1455 1456    2.87  -40.00                  -40.00\n",
      "   56  10275 207K 208L C   N    1639 1641  157.00 -108.50   94.73    4.44  -63.50 -132.60   39.98\n",
      "   56        208L 208L N   CA   1641 1642  139.21  132.50                  -41.20\n",
      "   57  10278 210H 211T C   N    1664 1666  -82.12  -78.10   14.69    0.51  -63.20  155.13   21.15\n",
      "   57        211T 211T N   CA   1666 1667  163.93  149.80                  -42.10\n",
      "   58  10279 211T 212L C   N    1671 1673  -80.95  -70.70   39.48    3.39  -63.50  145.73   19.30\n",
      "   58        212L 212L N   CA   1673 1674  103.48  141.60                  -41.20\n",
      "   59  10280 212L 213D C   N    1679 1681 -131.79  -96.50   75.46    3.13  -63.30  111.12   12.06\n",
      "   59        213D 213D N   CA   1681 1682   47.50  114.20                  -40.00\n",
      "   60  10282 214V 215S C   N    1694 1696 -167.63 -136.60   33.43    1.65  -72.40   96.20    7.23\n",
      "   60        215S 215S N   CA   1696 1697  138.78  151.20                  152.40\n",
      "   61  10283 215S 216S C   N    1700 1702 -154.51 -136.60   70.63    3.88  -64.10  148.56   10.77\n",
      "   61        216S 216S N   CA   1702 1703   82.88  151.20                  -35.00\n",
      "   62  10324 256N 257P C   N    2016 2018  -70.61  -58.70   28.16    1.65  -64.50  152.30   11.69\n",
      "   62        257P 257P N   CA   2018 2019   -4.98  -30.50                  147.20\n",
      "   63  10327 259A 260P C   N    2042 2044  -85.26  -58.70   34.69    2.66  -64.50  156.77   12.82\n",
      "   63        260P 260P N   CA   2044 2045   -8.19  -30.50                  147.20\n",
      "   64  10330 262E 263E C   N    2065 2067  -58.95  -69.30   43.56    3.66  -63.60  134.97   17.75\n",
      "   64        263E 263E N   CA   2067 2068 -175.19  142.50                  -40.30\n",
      "   65  10344 276D 277T C   N    2174 2176  -91.41  -63.20   90.56   10.03  -63.20   90.56   10.03\n",
      "   65        277T 277T N   CA   2176 2177   43.95  -42.10                  -42.10\n",
      "   66  10350 282P 283E C   N    2217 2219   77.00  -69.30  146.57   11.42  -63.60 -140.77   25.65\n",
      "   66        283E 283E N   CA   2219 2220  151.49  142.50                  -40.30\n",
      "   67  10351 283E 284K C   N    2226 2228  -71.34  -70.20   26.10    1.86  -62.90  152.96   20.33\n",
      "   67        284K 284K N   CA   2228 2229  166.48  140.40                  -40.80\n",
      "   68  10353 285P 286A C   N    2242 2244 -149.50 -134.00   29.61    1.16  -62.50  170.71   32.61\n",
      "   68        286A 286A N   CA   2244 2245  172.23  147.00                  -40.90\n",
      "   69  10354 286A 287Y C   N    2247 2249  -98.05  -98.40   43.98    4.68  -63.50  132.41   19.37\n",
      "   69        287Y 287Y N   CA   2249 2250   84.42  128.40                  -43.40\n",
      "   70  10355 287Y 288S C   N    2259 2261 -135.80 -136.60   43.73    2.20  -64.10  159.50   10.17\n",
      "   70        288S 288S N   CA   2261 2262  107.47  151.20                  -35.00\n",
      "   71  10358 290E 291A C   N    2283 2285  178.39 -134.00   54.37    1.25  -62.50 -171.69   37.06\n",
      "   71        291A 291A N   CA   2285 2286  173.25  147.00                  -40.90\n",
      "   72  10359 291A 292R C   N    2288 2290  -98.36  -72.10   29.72    1.98  -63.00  166.88   24.99\n",
      "   72        292R 292R N   CA   2290 2291  155.81  141.90                  -41.10\n",
      "   73  10364 296K 297K C   N    2334 2336 -110.93 -118.00   15.12    0.61  -62.90  173.33   20.41\n",
      "   73        297K 297K N   CA   2336 2337  125.74  139.10                  -40.80\n",
      "   74  10365 297K 298C C   N    2343 2345   66.85   57.40   43.11    1.67 -117.90 -131.15   10.00\n",
      "   74        298C 298C N   CA   2345 2346   -6.06   36.00                  141.10\n",
      "   75  10371 303P 304Q C   N    2390 2392  -66.23  -73.00   17.26    1.26  -63.80  163.14   24.11\n",
      "   75        304Q 304Q N   CA   2392 2393  156.58  140.70                  -40.30\n",
      "   76  10382 314K 315H C   N    2478 2480  -20.62  -67.60   47.60    3.80  -63.20  175.28   19.63\n",
      "   76        315H 315H N   CA   2480 2481  147.67  140.00                  -42.30\n",
      "   77  10384 316P 317F C   N    2495 2497  -51.25  -71.40   29.82    1.75  -63.20  163.46   23.50\n",
      "   77        317F 317F N   CA   2497 2498  118.72  140.70                  -44.30\n",
      "   78  10385 317F 318F C   N    2506 2508 -160.82 -124.20   56.93    1.61  -63.20  161.61   28.33\n",
      "   78        318F 318F N   CA   2508 2509 -173.10  143.30                  -44.30\n",
      "   79  10386 318F 319K C   N    2517 2519 -103.63 -118.00   31.55    1.28  -62.90  157.18   18.59\n",
      "   79        319K 319K N   CA   2519 2520  111.01  139.10                  -40.80\n",
      "   80  10388 320G 321C C   N    2530 2532  -65.33  -63.00    3.80    0.63 -117.90 -177.48    7.84\n",
      "   80        321C 321C N   CA   2532 2533  -44.11  -41.10                  141.10\n",
      "   81  10389 321C 322T C   N    2536 2538 -120.60 -124.80   11.08    0.37  -63.20 -175.50   20.43\n",
      "   81        322T 322T N   CA   2538 2539  133.24  143.50                  -42.10\n",
      "   82  10398 330V 331K C   N    2595 2597 -126.52 -118.00   26.31    1.14  -62.90  167.74   25.59\n",
      "   82        331K 331K N   CA   2597 2598  164.00  139.10                  -40.80\n",
      "   83  10400 332Q 333G C   N    2613 2615 -161.03 -167.20   27.02    0.96   82.20 -177.85   13.69\n",
      "   83        333G 333G N   CA   2615 2616  148.30  174.60                    8.50\n",
      "   84  10402 334V 335D C   N    2624 2626 -106.04  -96.50   20.63    0.87  -63.30  177.71   19.74\n",
      "   84        335D 335D N   CA   2626 2627  132.49  114.20                  -40.00\n",
      "   85  10403 335D 336Y C   N    2632 2634  -76.76  -98.40   22.29    1.07  -63.50  166.97   25.82\n",
      "   85        336Y 336Y N   CA   2634 2635  123.04  128.40                  -43.40\n",
      "   86  10404 336Y 337M C   N    2644 2646 -166.66 -125.60   57.30    1.75  -63.40  173.18   33.61\n",
      "   86        337M 337M N   CA   2646 2647 -179.53  140.50                  -40.50\n",
      "   87  10405 337M 338T C   N    2652 2654  -95.76 -124.80   53.71    1.67  -63.20  144.15   16.54\n",
      "   87        338T 338T N   CA   2654 2655   98.32  143.50                  -42.10\n",
      "   88  10406 338T 339R C   N    2659 2661 -146.18 -125.20   23.69    1.21  -63.00 -170.12   21.62\n",
      "   88        339R 339R N   CA   2661 2662  129.59  140.60                  -41.10\n",
      "   89  10408 340L 341I C   N    2678 2680  -60.59  -63.40    3.34    0.63 -120.60 -177.74    8.26\n",
      "   89        341I 341I N   CA   2680 2681  -41.80  -43.60                  130.30\n",
      "   90  10409 341I 342N C   N    2686 2688  -57.76  -71.20   16.62    0.93  -63.20  174.21   22.08\n",
      "   90        342N 342N N   CA   2688 2689  133.02  142.80                  -41.10\n",
      "   91  10410 342N 343S C   N    2694 2696 -128.42 -136.60    8.26    0.36  -64.10 -175.74   17.88\n",
      "   91        343S 343S N   CA   2696 2697  152.33  151.20                  -35.00\n",
      "   92  10411 343S 344D C   N    2700 2702 -127.44  -96.50   49.86    2.09  -63.30  178.61   26.19\n",
      "   92        344D 344D N   CA   2702 2703  153.31  114.20                  -40.00\n",
      "   93  10415 347K 348N C   N    2731 2733   62.22  -63.20  134.70   17.74  -63.20  134.70   17.74\n",
      "   93        348N 348N N   CA   2733 2734  -90.24  -41.10                  -41.10\n",
      "   94  10417 349G 350L C   N    2743 2745  -97.47  -70.70   79.13    5.11  -63.50  108.20   17.12\n",
      "   94        350L 350L N   CA   2745 2746 -143.93  141.60                  -41.20\n",
      "   95  10422 354S 355P C   N    2785 2787  -71.47  -58.70   42.52    2.64  -64.50  137.32   10.62\n",
      "   95        355P 355P N   CA   2787 2788   10.06  -30.50                  147.20\n",
      "   96  10423 355P 356E C   N    2792 2794  -54.78  -69.30   14.53    1.08  -63.60  177.96   23.26\n",
      "   96        356E 356E N   CA   2794 2795  141.96  142.50                  -40.30\n",
      "   97  10424 356E 357D C   N    2801 2803   74.56  -70.90  159.23    9.24  -70.90  159.23    9.24\n",
      "   97        357D 357D N   CA   2803 2804   85.52  150.30                  150.30\n",
      "   98  10437 369L 370G C   N    2905 2907  -82.17  -80.20   13.28    0.62   82.20 -135.81    7.32\n",
      "   98        370G 370G N   CA   2907 2908  160.96  174.10                    8.50\n",
      "   99  10438 370G 371D C   N    2909 2911 -108.65  -96.50   17.86    0.75  -63.30  173.32   19.11\n",
      "   99        371D 371D N   CA   2911 2912  127.29  114.20                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   38   60  270  255  409  340  439  419  508  536\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00194.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      372     2927     2927\n",
      "openf___224_> Open           EAO00194.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     40331457   39386.188    38.463\n",
      "read_mo_297_> Segments, residues, atoms:        1      372     2927\n",
      "read_mo_298_> Segment:        1  1:A  372:A     2927\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2927     2927        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    14510\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    15739\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    16544\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    17530\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    19055\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    20020\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    21585\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    23455\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    24867\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25435\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.1432       44  44\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.9294       58  58\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.8961       61  61\n",
      "iupac_m_485_> OD1/2 will be swapped:     -173.4957       62  62\n",
      "iupac_m_487_> NH1/2 swapped:     -175.0568       70  70\n",
      "iupac_m_487_> NH1/2 swapped:      179.7490      100  100\n",
      "iupac_m_485_> OD1/2 will be swapped:     -160.1220      101  101\n",
      "iupac_m_485_> OD1/2 will be swapped:     -140.4244      104  104\n",
      "iupac_m_487_> NH1/2 swapped:     -174.6060      105  105\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.1772      131  131\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.4244      153  153\n",
      "iupac_m_486_> OE1/2 will be swapped:     -163.5750      157  157\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4597      161  161\n",
      "iupac_m_487_> NH1/2 swapped:      174.7970      183  183\n",
      "iupac_m_485_> OD1/2 will be swapped:      137.3253      194  194\n",
      "iupac_m_485_> OD1/2 will be swapped:     -145.7809      195  195\n",
      "iupac_m_485_> OD1/2 will be swapped:      115.2040      202  202\n",
      "iupac_m_486_> OE1/2 will be swapped:     -142.0423      228  228\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.0417      235  235\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.9017      267  267\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.8520      272  272\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.9511      289  289\n",
      "iupac_m_487_> NH1/2 swapped:      179.7039      292  292\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.9511      293  293\n",
      "iupac_m_487_> NH1/2 swapped:      176.8152      301  301\n",
      "iupac_m_485_> OD1/2 will be swapped:     -134.7072      302  302\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.5139      305  305\n",
      "iupac_m_486_> OE1/2 will be swapped:     -121.4488      311  311\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.0785      318  318\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -165.7092      323  323\n",
      "iupac_m_485_> OD1/2 will be swapped:     -172.0304      335  335\n",
      "iupac_m_487_> NH1/2 swapped:      179.6987      339  339\n",
      "iupac_m_485_> OD1/2 will be swapped:      118.1794      357  357\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4615      368  368\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   588631\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -30937.3828\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -30937.382812\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    33.828999\n",
      "Sequence length          :          372\n",
      "Compactness              :     0.182821\n",
      "Native energy (pair)     :   188.641657\n",
      "Native energy (surface)  :    12.413206\n",
      "Native energy (combined) :     5.759398\n",
      "Z score (pair)           :    -3.398069\n",
      "Z score (surface)        :    -2.955295\n",
      "Z score (combined)       :    -4.284014\n",
      "GA341 score              :     0.889781\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     6964\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2655.4807\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2987       0      0   0.005   0.005      22.314       1.000\n",
      " 2 Bond angle potential               :    4036       1      9   2.086   2.086      332.49       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1855       0     70  47.796  47.796      661.85       1.000\n",
      " 4 Stereochemical improper torsion pot:    1190       1      1   1.434   1.434      48.147       1.000\n",
      " 5 Soft-sphere overlap restraints     :    6964       0      1   0.005   0.005      18.566       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5546       0      2   0.649   0.649      325.67       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5505       0     14   0.764   0.764      405.18       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     371       4      8  13.896  13.896      110.46       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     322       0      6  80.625  80.625      108.38       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     244       0      3  72.039  72.039      97.053       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     110       0      0  90.303  90.303      79.386       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      49       0      0  92.039  92.039      36.493       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2233       0      0   0.656   0.656      43.747       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     370      96     72  36.278  88.649      309.23       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     703       0      0   1.516   1.516      56.511       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00194.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00194.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   49675.4219\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4504 143Y 143Y N   CA   1106 1107  123.22  107.00   16.22    4.66  107.00   16.22    4.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10530  92I  92I CA  C     697  702  172.70   -0.75  173.44   35.39   -0.75  173.44   35.39\n",
      "    2  10557 119V 119V CA  C     917  921 -157.41 -180.00   22.59    4.52 -180.00   22.59    4.52\n",
      "    3  10581 143Y 143Y CA  C    1107 1116 -147.00 -180.00   33.00    6.60 -180.00   33.00    6.60\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10070   2K   3K C   N      16   18 -152.51 -118.00   45.41    1.56  -62.90  175.22   28.09\n",
      "    1          3K   3K N   CA     18   19  168.62  139.10                  -40.80\n",
      "    2  10071   3K   4L C   N      25   27 -121.22 -108.50   52.52    2.69  -63.50  147.14   24.00\n",
      "    2          4L   4L N   CA     27   28 -176.54  132.50                  -41.20\n",
      "    3  10073   5G   6L C   N      37   39  -66.15  -70.70   12.49    1.12  -63.50  165.59   23.14\n",
      "    3          6L   6L N   CA     39   40  153.23  141.60                  -41.20\n",
      "    4  10075   7V   8L C   N      52   54  -74.61  -70.70   42.57    3.08  -63.50  135.26   19.44\n",
      "    4          8L   8L N   CA     54   55 -176.01  141.60                  -41.20\n",
      "    5  10077   9D  10A C   N      68   70 -139.44 -134.00   32.32    1.94  -62.50  173.98   26.09\n",
      "    5         10A  10A N   CA     70   71  115.14  147.00                  -40.90\n",
      "    6  10078  10A  11A C   N      73   75 -155.44 -134.00   21.73    0.86  -62.50 -161.31   37.53\n",
      "    6         11A  11A N   CA     75   76  143.48  147.00                  -40.90\n",
      "    7  10079  11A  12A C   N      78   80  -79.24  -68.20   11.07    0.97  -62.50  175.47   29.61\n",
      "    7         12A  12A N   CA     80   81  144.43  145.30                  -40.90\n",
      "    8  10081  13V  14E C   N      90   92 -125.45 -117.80   24.00    1.03  -63.60  171.68   26.38\n",
      "    8         14E  14E N   CA     92   93  159.54  136.80                  -40.30\n",
      "    9  10082  14E  15K C   N      99  101  -66.88  -70.20   12.77    1.00  -62.90  166.52   21.84\n",
      "    9         15K  15K N   CA    101  102  152.73  140.40                  -40.80\n",
      "   10  10084  16S  17Q C   N     114  116   37.07  -73.00  120.99    7.61  -63.80  165.17   29.25\n",
      "   10         17Q  17Q N   CA    116  117   90.49  140.70                  -40.30\n",
      "   11  10085  17Q  18R C   N     123  125   65.64   57.30   29.84    1.50  -63.00  138.18   24.97\n",
      "   11         18R  18R N   CA    125  126    9.34   38.00                  -41.10\n",
      "   12  10086  18R  19D C   N     134  136 -134.39  -63.30   75.26   10.51  -63.30   75.26   10.51\n",
      "   12         19D  19D N   CA    136  137  -15.31  -40.00                  -40.00\n",
      "   13  10097  29N  30S C   N     227  229   80.66   56.90  138.63   10.89   56.90  138.63   10.89\n",
      "   13         30S  30S N   CA    229  230  172.98   36.40                   36.40\n",
      "   14  10098  30S  31L C   N     233  235 -133.94 -108.50   92.99    4.72  -63.50  119.76   20.90\n",
      "   14         31L  31L N   CA    235  236 -138.06  132.50                  -41.20\n",
      "   15  10099  31L  32Q C   N     241  243  -59.57  -73.00   14.86    1.15  -63.80  172.69   25.17\n",
      "   15         32Q  32Q N   CA    243  244  147.06  140.70                  -40.30\n",
      "   16  10100  32Q  33L C   N     250  252  -64.06  -70.70   15.87    1.00  -63.50  168.39   23.35\n",
      "   16         33L  33L N   CA    252  253  127.19  141.60                  -41.20\n",
      "   17  10101  33L  34R C   N     258  260 -179.28 -125.20   58.76    1.83  -63.00 -165.97   33.80\n",
      "   17         34R  34R N   CA    260  261  163.58  140.60                  -41.10\n",
      "   18  10102  34R  35A C   N     269  271  -99.77  -68.20   55.99    5.43  -62.50  144.84   22.22\n",
      "   18         35A  35A N   CA    271  272   99.06  145.30                  -40.90\n",
      "   19  10103  35A  36F C   N     274  276 -161.60 -124.20   62.13    1.83  -63.20  157.35   27.71\n",
      "   19         36F  36F N   CA    276  277 -167.09  143.30                  -44.30\n",
      "   20  10104  36F  37L C   N     285  287  -85.41 -108.50   29.47    1.31  -63.50  156.93   20.63\n",
      "   20         37L  37L N   CA    287  288  114.19  132.50                  -41.20\n",
      "   21  10105  37L  38L C   N     293  295 -100.94 -108.50   14.50    0.83  -63.50  177.91   26.99\n",
      "   21         38L  38L N   CA    295  296  144.87  132.50                  -41.20\n",
      "   22  10106  38L  39N C   N     301  303 -152.86 -119.90   63.94    2.08   55.90 -145.37   25.33\n",
      "   22         39N  39N N   CA    303  304 -168.21  137.00                   39.50\n",
      "   23  10107  39N  40E C   N     309  311 -124.09 -117.80   10.27    0.56  -63.60  179.49   21.25\n",
      "   23         40E  40E N   CA    311  312  128.69  136.80                  -40.30\n",
      "   24  10111  43M  44F C   N     336  338 -151.53 -124.20   28.92    0.86   58.10 -167.70   29.04\n",
      "   24         44F  44F N   CA    338  339  152.77  143.30                   32.90\n",
      "   25  10112  44F  45T C   N     347  349  -73.25  -63.20   82.29   11.22  -63.20   82.29   11.22\n",
      "   25         45T  45T N   CA    349  350 -123.77  -42.10                  -42.10\n",
      "   26  10114  46K  47D C   N     363  365  -63.51  -63.30    5.95    0.72  -70.90  175.80    9.03\n",
      "   26         47D  47D N   CA    365  366  -34.05  -40.00                  150.30\n",
      "   27  10118  50K  51L C   N     395  397  -56.31  -70.70   15.25    1.20  -63.50  177.90   25.12\n",
      "   27         51L  51L N   CA    397  398  136.55  141.60                  -41.20\n",
      "   28  10122  54S  55V C   N     420  422  -69.61 -125.40   65.93    1.74  -62.40  150.74   18.67\n",
      "   28         55V  55V N   CA    422  423  108.17  143.30                  -42.40\n",
      "   29  10123  55V  56T C   N     427  429 -154.81 -124.80   39.70    1.33  -63.20  174.41   28.19\n",
      "   29         56T  56T N   CA    429  430  169.49  143.50                  -42.10\n",
      "   30  10124  56T  57K C   N     434  436  -87.63  -70.20   34.83    2.14  -62.90  150.69   21.00\n",
      "   30         57K  57K N   CA    436  437  170.56  140.40                  -40.80\n",
      "   31  10125  57K  58E C   N     443  445  -64.23  -69.30   45.68    3.31  -63.60  137.41   18.29\n",
      "   31         58E  58E N   CA    445  446   97.11  142.50                  -40.30\n",
      "   32  10128  60I  61D C   N     467  469 -150.98  -96.50   54.68    2.23  -63.30  173.35   18.41\n",
      "   32         61D  61D N   CA    469  470  109.55  114.20                  -40.00\n",
      "   33  10129  61D  62D C   N     475  477 -111.99  -70.90   42.47    2.79   54.50 -154.69   15.03\n",
      "   33         62D  62D N   CA    477  478  161.03  150.30                   40.90\n",
      "   34  10131  63T  64S C   N     490  492  -66.88  -72.40    8.72    0.68  -64.10  165.87   12.18\n",
      "   34         64S  64S N   CA    492  493  159.16  152.40                  -35.00\n",
      "   35  10132  64S  65K C   N     496  498   58.88   56.60   11.34    0.99 -118.00 -161.81    6.56\n",
      "   35         65K  65K N   CA    498  499   49.71   38.60                  139.10\n",
      "   36  10133  65K  66R C   N     505  507  -74.14  -72.10   58.50    4.54  -63.00  125.03   16.17\n",
      "   36         66R  66R N   CA    507  508   83.43  141.90                  -41.10\n",
      "   37  10134  66R  67I C   N     516  518   15.27  -97.30  125.46    7.87 -120.60  147.93    8.06\n",
      "   37         67I  67I N   CA    518  519   71.82  127.20                  130.30\n",
      "   38  10138  70R  71G C   N     547  549   94.98   82.20   33.57    1.03  -62.40  158.48   28.99\n",
      "   38         71G  71G N   CA    549  550  -22.54    8.50                  -41.20\n",
      "   39  10139  71G  72A C   N     551  553  -79.26  -68.20   38.65    2.73  -62.50  137.79   23.44\n",
      "   39         72A  72A N   CA    553  554 -177.66  145.30                  -40.90\n",
      "   40  10140  72A  73S C   N     556  558  -55.51  -72.40   45.72    2.19  -64.10  145.17   11.05\n",
      "   40         73S  73S N   CA    558  559  109.91  152.40                  -35.00\n",
      "   41  10141  73S  74G C   N     562  564   56.41   78.70   42.56    0.66  -80.20  147.66   10.87\n",
      "   41         74G  74G N   CA    564  565 -129.84 -166.10                  174.10\n",
      "   42  10149  81L  82K C   N     623  625  -65.06  -62.90    7.54    1.11 -118.00 -179.20    9.29\n",
      "   42         82K  82K N   CA    625  626  -48.02  -40.80                  139.10\n",
      "   43  10151  83D  84G C   N     640  642  -88.37  -80.20   27.66    0.72   82.20 -120.60   18.38\n",
      "   43         84G  84G N   CA    642  643 -159.48  174.10                    8.50\n",
      "   44  10160  92I  93P C   N     702  704  -80.72  -64.50   31.06    3.17  -58.70  152.80   10.94\n",
      "   44         93P  93P N   CA    704  705  120.71  147.20                  -30.50\n",
      "   45  10162  94I  95T C   N     717  719  -50.35  -78.10   53.62    1.76  -63.20  146.59   19.65\n",
      "   45         95T  95T N   CA    719  720  103.92  149.80                  -42.10\n",
      "   46  10163  95T  96S C   N     724  726  176.40 -136.60   87.45    3.21  -64.10  155.85   19.75\n",
      "   46         96S  96S N   CA    726  727 -135.06  151.20                  -35.00\n",
      "   47  10164  96S  97K C   N     730  732  -63.10  -70.20   31.99    2.15  -62.90  150.01   19.47\n",
      "   47         97K  97K N   CA    732  733  109.21  140.40                  -40.80\n",
      "   48  10166  98P  99H C   N     746  748  169.52 -125.60   70.63    2.98  -63.20 -160.82   21.73\n",
      "   48         99H  99H N   CA    748  749  110.90  138.80                  -42.30\n",
      "   49  10209 141M 142A C   N    1099 1101   69.98  -68.20  146.57   13.84  -62.50 -177.89   28.33\n",
      "   49        142A 142A N   CA   1101 1102 -165.84  145.30                  -40.90\n",
      "   50  10217 149S 150C C   N    1162 1164   62.45   57.40   11.35    0.37 -117.90 -146.55    9.10\n",
      "   50        150C 150C N   CA   1164 1165   25.84   36.00                  141.10\n",
      "   51  10219 151F 152W C   N    1179 1181  171.27 -124.90   98.09    3.21  -63.00  159.36   27.82\n",
      "   51        152W 152W N   CA   1181 1182 -142.11  143.40                  -44.20\n",
      "   52  10220 152W 153E C   N    1193 1195  -62.74  -63.60    8.93    1.24   54.60  138.62   18.14\n",
      "   52        153E 153E N   CA   1195 1196  -31.41  -40.30                   42.40\n",
      "   53  10221 153E 154G C   N    1202 1204  120.63   78.70   97.42    1.45   82.20  104.76    6.88\n",
      "   53        154G 154G N   CA   1204 1205  105.96 -166.10                    8.50\n",
      "   54  10226 158T 159M C   N    1239 1241  -81.66  -73.00   48.14    3.41  -63.40  137.37   19.16\n",
      "   54        159M 159M N   CA   1241 1242   95.65  143.00                  -40.50\n",
      "   55  10244 176D 177T C   N    1390 1392  -87.96 -124.80   37.17    1.84  -63.20 -178.97   24.86\n",
      "   55        177T 177T N   CA   1392 1393  138.57  143.50                  -42.10\n",
      "   56  10245 177T 178K C   N    1397 1399  -59.51  -62.90   13.61    1.97   56.60  133.67   17.09\n",
      "   56        178K 178K N   CA   1399 1400  -27.62  -40.80                   38.60\n",
      "   57  10251 183R 184D C   N    1453 1455 -106.08  -63.30   62.13    6.95  -63.30   62.13    6.95\n",
      "   57        184D 184D N   CA   1455 1456    5.06  -40.00                  -40.00\n",
      "   58  10275 207K 208L C   N    1639 1641 -163.80 -108.50   63.38    3.42  -63.50  174.45   21.56\n",
      "   58        208L 208L N   CA   1641 1642  101.53  132.50                  -41.20\n",
      "   59  10279 211T 212L C   N    1671 1673 -132.11 -108.50   23.91    1.10  -63.50 -169.70   23.54\n",
      "   59        212L 212L N   CA   1673 1674  136.30  132.50                  -41.20\n",
      "   60  10280 212L 213D C   N    1679 1681  -86.58  -70.90   48.35    1.72  -63.30  126.13   17.05\n",
      "   60        213D 213D N   CA   1681 1682 -163.96  150.30                  -40.00\n",
      "   61  10283 215S 216S C   N    1700 1702 -173.56 -136.60   57.25    1.93  -64.10  170.01   20.20\n",
      "   61        216S 216S N   CA   1702 1703 -165.08  151.20                  -35.00\n",
      "   62  10325 257P 258W C   N    2023 2025  -25.85  -71.30   46.22    3.07  -63.00  178.67   24.76\n",
      "   62        258W 258W N   CA   2025 2026  130.57  139.00                  -44.20\n",
      "   63  10330 262E 263E C   N    2065 2067 -156.85  -63.60  114.36   14.40  -63.60  114.36   14.40\n",
      "   63        263E 263E N   CA   2067 2068   25.89  -40.30                  -40.30\n",
      "   64  10344 276D 277T C   N    2174 2176  -90.82  -63.20   91.40   10.16  -63.20   91.40   10.16\n",
      "   64        277T 277T N   CA   2176 2177   45.02  -42.10                  -42.10\n",
      "   65  10350 282P 283E C   N    2217 2219  147.92 -117.80  101.38    3.02  -63.60 -152.03   35.29\n",
      "   65        283E 283E N   CA   2219 2220  174.08  136.80                  -40.30\n",
      "   66  10351 283E 284K C   N    2226 2228  -66.91  -70.20   19.39    1.48  -62.90  159.74   20.96\n",
      "   66        284K 284K N   CA   2228 2229  159.51  140.40                  -40.80\n",
      "   67  10352 284K 285P C   N    2235 2237  -66.00  -58.70   13.19    0.79  -64.50  166.71   12.52\n",
      "   67        285P 285P N   CA   2237 2238  -19.51  -30.50                  147.20\n",
      "   68  10353 285P 286A C   N    2242 2244  -68.56  -68.20   13.63    1.09  -62.50  160.29   26.57\n",
      "   68        286A 286A N   CA   2244 2245  158.92  145.30                  -40.90\n",
      "   69  10354 286A 287Y C   N    2247 2249  -62.11  -98.40   36.62    1.55  -63.50  166.93   26.56\n",
      "   69        287Y 287Y N   CA   2249 2250  123.52  128.40                  -43.40\n",
      "   70  10355 287Y 288S C   N    2259 2261 -130.80 -136.60   17.43    0.74  -64.10 -177.61   11.28\n",
      "   70        288S 288S N   CA   2261 2262  134.76  151.20                  -35.00\n",
      "   71  10357 289E 290E C   N    2274 2276  -91.09  -69.30   24.73    1.47  -63.60  167.77   23.92\n",
      "   71        290E 290E N   CA   2276 2277  154.19  142.50                  -40.30\n",
      "   72  10358 290E 291A C   N    2283 2285  -81.98  -68.20   23.79    1.51  -62.50  155.63   26.50\n",
      "   72        291A 291A N   CA   2285 2286  164.70  145.30                  -40.90\n",
      "   73  10359 291A 292R C   N    2288 2290  -81.07  -72.10   23.10    1.58  -63.00  156.76   22.41\n",
      "   73        292R 292R N   CA   2290 2291  163.19  141.90                  -41.10\n",
      "   74  10365 297K 298C C   N    2343 2345  -63.58  -63.00    0.75    0.09 -117.90 -173.62    8.01\n",
      "   74        298C 298C N   CA   2345 2346  -40.61  -41.10                  141.10\n",
      "   75  10370 302D 303P C   N    2383 2385  -49.31  -58.70   11.83    1.64  -64.50  171.18   11.99\n",
      "   75        303P 303P N   CA   2385 2386  -23.31  -30.50                  147.20\n",
      "   76  10373 305E 306R C   N    2408 2410  -64.36  -63.00    8.23    1.03 -125.20 -176.07    7.37\n",
      "   76        306R 306R N   CA   2410 2411  -32.98  -41.10                  140.60\n",
      "   77  10381 313L 314K C   N    2469 2471  -99.02  -62.90   74.45   11.83  -62.90   74.45   11.83\n",
      "   77        314K 314K N   CA   2471 2472 -105.91  -40.80                  -40.80\n",
      "   78  10382 314K 315H C   N    2478 2480 -140.32  -63.20   83.24   14.23  -63.20   83.24   14.23\n",
      "   78        315H 315H N   CA   2480 2481  -73.61  -42.30                  -42.30\n",
      "   79  10384 316P 317F C   N    2495 2497 -164.78 -124.20   49.02    1.27  -63.20  176.96   30.79\n",
      "   79        317F 317F N   CA   2497 2498  170.80  143.30                  -44.30\n",
      "   80  10389 321C 322T C   N    2536 2538 -121.79 -124.80   69.79    3.08  -63.20  119.96   19.15\n",
      "   80        322T 322T N   CA   2538 2539 -146.78  143.50                  -42.10\n",
      "   81  10398 330V 331K C   N    2595 2597 -107.67 -118.00   19.76    1.02  -62.90  169.28   24.63\n",
      "   81        331K 331K N   CA   2597 2598  155.95  139.10                  -40.80\n",
      "   82  10399 331K 332Q C   N    2604 2606  -91.63  -73.00   37.23    2.15  -63.80  149.38   23.45\n",
      "   82        332Q 332Q N   CA   2606 2607  172.93  140.70                  -40.30\n",
      "   83  10400 332Q 333G C   N    2613 2615  -65.77  -80.20   22.15    0.63   82.20 -150.15    6.54\n",
      "   83        333G 333G N   CA   2615 2616  157.30  174.10                    8.50\n",
      "   84  10402 334V 335D C   N    2624 2626  -55.19  -70.90   17.29    0.99  -63.30  177.13   21.38\n",
      "   84        335D 335D N   CA   2626 2627  143.06  150.30                  -40.00\n",
      "   85  10403 335D 336Y C   N    2632 2634  -61.69  -98.40   36.77    1.41  -63.50  173.98   27.70\n",
      "   85        336Y 336Y N   CA   2634 2635  130.57  128.40                  -43.40\n",
      "   86  10404 336Y 337M C   N    2644 2646 -107.10 -125.60   22.26    0.66  -63.40  174.19   22.99\n",
      "   86        337M 337M N   CA   2646 2647  128.11  140.50                  -40.50\n",
      "   87  10407 339R 340L C   N    2670 2672  127.08 -108.50  131.30    5.86  -63.50 -137.43   40.54\n",
      "   87        340L 340L N   CA   2672 2673  174.45  132.50                  -41.20\n",
      "   88  10409 341I 342N C   N    2686 2688  -75.14  -71.20    4.03    0.27  -63.20  175.68   22.63\n",
      "   88        342N 342N N   CA   2688 2689  143.63  142.80                  -41.10\n",
      "   89  10411 343S 344D C   N    2700 2702  -61.55  -70.90   11.43    0.55  -63.30  176.30   21.66\n",
      "   89        344D 344D N   CA   2702 2703  143.71  150.30                  -40.00\n",
      "   90  10415 347K 348N C   N    2731 2733 -137.25 -119.90   76.96    3.07  -63.20  130.07   20.31\n",
      "   90        348N 348N N   CA   2733 2734 -148.03  137.00                  -41.10\n",
      "   91  10416 348N 349G C   N    2739 2741   63.81   78.70   40.99    0.64   82.20  137.65    7.59\n",
      "   91        349G 349G N   CA   2741 2742 -127.91 -166.10                    8.50\n",
      "   92  10417 349G 350L C   N    2743 2745  -64.63  -70.70   21.93    1.45  -63.50  161.74   22.39\n",
      "   92        350L 350L N   CA   2745 2746  120.53  141.60                  -41.20\n",
      "   93  10418 350L 351L C   N    2751 2753  -85.19 -108.50   54.86    2.67  -63.50  125.93   16.37\n",
      "   93        351L 351L N   CA   2753 2754   82.84  132.50                  -41.20\n",
      "   94  10427 359T 360K C   N    2823 2825 -119.57  -62.90   61.17   10.66  -62.90   61.17   10.66\n",
      "   94        360K 360K N   CA   2825 2826  -63.83  -40.80                  -40.80\n",
      "   95  10437 369L 370G C   N    2905 2907  -69.74  -80.20   14.18    0.48   82.20 -142.21    6.72\n",
      "   95        370G 370G N   CA   2907 2908  164.53  174.10                    8.50\n",
      "   96  10438 370G 371D C   N    2909 2911  -77.06  -70.90   24.83    0.95  -63.30  146.30   18.92\n",
      "   96        371D 371D N   CA   2911 2912  174.35  150.30                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   33   80  250  246  371  333  399  469  484  558\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00194.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      372     2927     2927\n",
      "openf___224_> Open           EAO00194.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     40331457   39386.188    38.463\n",
      "read_mo_297_> Segments, residues, atoms:        1      372     2927\n",
      "read_mo_298_> Segment:        1  1:A  372:A     2927\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2927     2927        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    14510\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    15739\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    16544\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    17530\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    19055\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    20020\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    21585\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    23455\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    24867\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25435\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "iupac_m_486_> OE1/2 will be swapped:     -147.5486       14  14\n",
      "iupac_m_487_> NH1/2 swapped:      179.8718       18  18\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.7825       19  19\n",
      "iupac_m_487_> NH1/2 swapped:     -177.7405       34  34\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      101.6320       36  36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.5197       61  61\n",
      "iupac_m_485_> OD1/2 will be swapped:      127.3575       62  62\n",
      "iupac_m_487_> NH1/2 swapped:      178.6827       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.4715      102  102\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.1068      126  126\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.7928      139  139\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.6313      151  151\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.1589      153  153\n",
      "iupac_m_486_> OE1/2 will be swapped:     -147.2576      156  156\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.9082      157  157\n",
      "iupac_m_485_> OD1/2 will be swapped:     -101.1873      162  162\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      123.2972      165  165\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      169.2885      176  176\n",
      "iupac_m_487_> NH1/2 swapped:      178.7781      179  179\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.0902      194  194\n",
      "iupac_m_485_> OD1/2 will be swapped:      165.5066      213  213\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.6005      228  228\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.4475      235  235\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.3864      262  262\n",
      "iupac_m_486_> OE1/2 will be swapped:     -165.1534      263  263\n",
      "iupac_m_487_> NH1/2 swapped:      179.7668      272  272\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.2580      283  283\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.2491      289  289\n",
      "iupac_m_486_> OE1/2 will be swapped:      149.2197      290  290\n",
      "iupac_m_487_> NH1/2 swapped:     -177.8000      301  301\n",
      "iupac_m_485_> OD1/2 will be swapped:      117.8889      302  302\n",
      "iupac_m_486_> OE1/2 will be swapped:     -154.7807      323  323\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3048      339  339\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.9500      361  361\n",
      "iupac_m_485_> OD1/2 will be swapped:      128.8901      371  371\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   574964\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -32249.1387\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -32249.138672\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    33.828999\n",
      "Sequence length          :          372\n",
      "Compactness              :     0.180981\n",
      "Native energy (pair)     :   142.332213\n",
      "Native energy (surface)  :    10.184574\n",
      "Native energy (combined) :     4.702274\n",
      "Z score (pair)           :    -4.048699\n",
      "Z score (surface)        :    -3.379714\n",
      "Z score (combined)       :    -5.059450\n",
      "GA341 score              :     0.991536\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     6603\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2701.9258\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2987       0      0   0.005   0.005      21.325       1.000\n",
      " 2 Bond angle potential               :    4036       1      8   1.993   1.993      306.89       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1855       0     87  49.497  49.497      699.92       1.000\n",
      " 4 Stereochemical improper torsion pot:    1190       1      1   1.373   1.373      44.508       1.000\n",
      " 5 Soft-sphere overlap restraints     :    6603       0      1   0.005   0.005      18.042       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5546       0      3   0.714   0.714      347.68       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5505       0     17   0.871   0.871      463.99       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     371       3      6  10.518  10.518      103.17       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     322       0      2  78.560  78.560      98.655       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     244       0      2  75.606  75.606      89.270       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     110       0      0  88.084  88.084      72.507       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      49       0      0  84.169  84.169      32.141       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2233       0      0   0.591   0.591      36.361       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     370      96     63  37.854  88.568      315.33       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     703       0      0   1.688   1.688      52.152       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00194.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00194.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   49918.7031\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4504 143Y 143Y N   CA   1106 1107  122.73  107.00   15.73    4.52  107.00   15.73    4.52\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10581 143Y 143Y CA  C    1107 1116 -148.01 -180.00   31.99    6.40 -180.00   31.99    6.40\n",
      "    2  10696 258W 258W CA  C    2026 2037 -153.68 -180.00   26.32    5.26 -180.00   26.32    5.26\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10070   2K   3K C   N      16   18 -156.29 -118.00   81.30    3.27  -62.90  143.07   23.99\n",
      "    1          3K   3K N   CA     18   19 -149.18  139.10                  -40.80\n",
      "    2  10071   3K   4L C   N      25   27  -63.69  -70.70    7.53    0.58  -63.50  179.97   25.00\n",
      "    2          4L   4L N   CA     27   28  138.83  141.60                  -41.20\n",
      "    3  10072   4L   5G C   N      33   35  -90.59  -80.20   27.68    0.67   82.20 -118.48   18.56\n",
      "    3          5G   5G N   CA     35   36 -160.24  174.10                    8.50\n",
      "    4  10073   5G   6L C   N      37   39  -65.69  -70.70   11.46    1.06  -63.50  166.90   23.30\n",
      "    4          6L   6L N   CA     39   40  151.91  141.60                  -41.20\n",
      "    5  10075   7V   8L C   N      52   54 -142.25 -108.50   42.49    1.88  -63.50  178.75   29.69\n",
      "    5          8L   8L N   CA     54   55  158.32  132.50                  -41.20\n",
      "    6  10076   8L   9D C   N      60   62  -63.61  -70.90   23.20    0.83  -63.30  168.28   20.75\n",
      "    6          9D   9D N   CA     62   63  128.28  150.30                  -40.00\n",
      "    7  10077   9D  10A C   N      68   70 -173.57 -134.00   44.21    1.01  -62.50 -171.43   36.75\n",
      "    7         10A  10A N   CA     70   71  166.71  147.00                  -40.90\n",
      "    8  10078  10A  11A C   N      73   75 -109.10 -134.00   27.22    0.62  -62.50 -177.07   28.08\n",
      "    8         11A  11A N   CA     75   76  136.00  147.00                  -40.90\n",
      "    9  10079  11A  12A C   N      78   80 -158.14 -134.00   26.99    0.61  -62.50 -173.55   35.65\n",
      "    9         12A  12A N   CA     80   81  159.06  147.00                  -40.90\n",
      "   10  10081  13V  14E C   N      90   92  -73.68  -69.30   18.46    1.55  -63.60  165.18   21.49\n",
      "   10         14E  14E N   CA     92   93  124.57  142.50                  -40.30\n",
      "   11  10082  14E  15K C   N      99  101  157.61 -118.00   97.79    3.24  -62.90 -168.87   32.69\n",
      "   11         15K  15K N   CA    101  102 -171.47  139.10                  -40.80\n",
      "   12  10085  17Q  18R C   N     123  125  -59.19  -72.10   15.84    1.04  -63.00  173.87   23.74\n",
      "   12         18R  18R N   CA    125  126  132.73  141.90                  -41.10\n",
      "   13  10086  18R  19D C   N     134  136  -81.97  -96.50   53.04    2.24  -63.30  104.86   11.93\n",
      "   13         19D  19D N   CA    136  137   63.19  114.20                  -40.00\n",
      "   14  10096  28N  29N C   N     219  221  -84.71 -119.90   75.90    2.56  -63.20  112.92   13.04\n",
      "   14         29N  29N N   CA    221  222   69.75  137.00                  -41.10\n",
      "   15  10097  29N  30S C   N     227  229  -85.25  -72.40   49.31    2.49   56.90 -143.26   11.71\n",
      "   15         30S  30S N   CA    229  230 -160.00  152.40                   36.40\n",
      "   16  10098  30S  31L C   N     233  235 -108.14  -70.70   82.45    5.15  -63.50  112.93   18.45\n",
      "   16         31L  31L N   CA    235  236 -144.94  141.60                  -41.20\n",
      "   17  10099  31L  32Q C   N     241  243  -58.07  -73.00   44.61    3.22  -63.80  137.08   19.86\n",
      "   17         32Q  32Q N   CA    243  244 -177.26  140.70                  -40.30\n",
      "   18  10101  33L  34R C   N     258  260 -114.15 -125.20   30.01    1.17  -63.00  162.08   19.03\n",
      "   18         34R  34R N   CA    260  261  112.69  140.60                  -41.10\n",
      "   19  10102  34R  35A C   N     269  271 -113.43 -134.00   30.45    0.97  -62.50  173.10   26.38\n",
      "   19         35A  35A N   CA    271  272  124.54  147.00                  -40.90\n",
      "   20  10104  36F  37L C   N     285  287  -84.79 -108.50   33.86    1.53  -63.50  151.04   19.85\n",
      "   20         37L  37L N   CA    287  288  108.33  132.50                  -41.20\n",
      "   21  10106  38L  39N C   N     301  303 -152.72  -63.20  113.20   18.54   55.90 -146.98   12.78\n",
      "   21         39N  39N N   CA    303  304 -110.37  -41.10                   39.50\n",
      "   22  10107  39N  40E C   N     309  311  -49.46  -69.30   41.96    2.51  -63.60  146.51   20.32\n",
      "   22         40E  40E N   CA    311  312  105.53  142.50                  -40.30\n",
      "   23  10110  42G  43M C   N     328  330 -131.19 -125.60   48.11    2.18  -63.40  147.69   27.35\n",
      "   23         43M  43M N   CA    330  331 -171.71  140.50                  -40.50\n",
      "   24  10111  43M  44F C   N     336  338  -60.11  -63.20    3.16    0.54   58.10  140.81   21.55\n",
      "   24         44F  44F N   CA    338  339  -43.62  -44.30                   32.90\n",
      "   25  10112  44F  45T C   N     347  349 -156.03 -124.80   48.95    1.53  -63.20  165.26   27.03\n",
      "   25         45T  45T N   CA    349  350 -178.82  143.50                  -42.10\n",
      "   26  10114  46K  47D C   N     363  365  177.36  -96.50  102.12    4.24  -70.90  113.31    7.89\n",
      "   26         47D  47D N   CA    365  366  169.04  114.20                  150.30\n",
      "   27  10116  48G  49R C   N     375  377 -165.98 -125.20   45.86    1.39  -72.10   95.92    6.89\n",
      "   27         49R  49R N   CA    377  378  161.59  140.60                  141.90\n",
      "   28  10118  50K  51L C   N     395  397 -107.95 -108.50   33.78    1.82  -63.50  146.82   18.37\n",
      "   28         51L  51L N   CA    397  398   98.72  132.50                  -41.20\n",
      "   29  10120  52P  53G C   N     410  412   80.79   82.20   14.36    0.77  -62.40  147.51   27.49\n",
      "   29         53G  53G N   CA    412  413   -5.79    8.50                  -41.20\n",
      "   30  10121  53G  54S C   N     414  416  -67.06  -72.40   18.79    0.94  -64.10  169.42   12.10\n",
      "   30         54S  54S N   CA    416  417  134.39  152.40                  -35.00\n",
      "   31  10122  54S  55V C   N     420  422 -156.29 -125.40   43.43    1.21  -62.40  171.72   27.40\n",
      "   31         55V  55V N   CA    422  423  173.83  143.30                  -42.40\n",
      "   32  10123  55V  56T C   N     427  429 -119.26 -124.80   16.69    0.87  -63.20  168.28   25.29\n",
      "   32         56T  56T N   CA    429  430  159.24  143.50                  -42.10\n",
      "   33  10128  60I  61D C   N     467  469  -77.88  -70.90   17.90    0.61  -63.30  153.91   19.91\n",
      "   33         61D  61D N   CA    469  470  166.78  150.30                  -40.00\n",
      "   34  10129  61D  62D C   N     475  477 -112.77  -96.50   26.01    1.07   54.50  175.47   17.26\n",
      "   34         62D  62D N   CA    477  478   93.92  114.20                   40.90\n",
      "   35  10133  65K  66R C   N     505  507  -82.69  -72.10   69.11    5.08  -63.00  110.48   16.28\n",
      "   35         66R  66R N   CA    507  508 -149.81  141.90                  -41.10\n",
      "   36  10139  71G  72A C   N     551  553  170.69 -134.00   57.84    1.39  -62.50 -159.57   39.45\n",
      "   36         72A  72A N   CA    553  554  163.89  147.00                  -40.90\n",
      "   37  10140  72A  73S C   N     556  558   54.74  -64.10  136.87   13.68  -64.10  136.87   13.68\n",
      "   37         73S  73S N   CA    558  559 -102.89  -35.00                  -35.00\n",
      "   38  10149  81L  82K C   N     623  625   51.89  -70.20  131.39   10.59 -118.00  177.05    7.61\n",
      "   38         82K  82K N   CA    625  626 -171.06  140.40                  139.10\n",
      "   39  10150  82K  83D C   N     632  634  -79.36  -96.50   33.68    1.39  -63.30  177.53   22.91\n",
      "   39         83D  83D N   CA    634  635  143.20  114.20                  -40.00\n",
      "   40  10159  91H  92I C   N     694  696   81.93 -120.60  158.63   10.36 -120.60  158.63   10.36\n",
      "   40         92I  92I N   CA    696  697  149.45  130.30                  130.30\n",
      "   41  10160  92I  93P C   N     702  704  -94.64  -64.50   31.46    3.47  -58.70  172.49   11.60\n",
      "   41         93P  93P N   CA    704  705  138.21  147.20                  -30.50\n",
      "   42  10162  94I  95T C   N     717  719 -169.73 -124.80   55.50    1.97  -63.20  177.38   29.34\n",
      "   42         95T  95T N   CA    719  720  176.07  143.50                  -42.10\n",
      "   43  10164  96S  97K C   N     730  732  -66.42  -70.20   23.06    1.76  -62.90  156.10   20.47\n",
      "   43         97K  97K N   CA    732  733  163.14  140.40                  -40.80\n",
      "   44  10165  97K  98P C   N     739  741  -52.07  -58.70    6.64    0.84  -64.50  178.40   12.67\n",
      "   44         98P  98P N   CA    741  742  -30.77  -30.50                  147.20\n",
      "   45  10166  98P  99H C   N     746  748  -81.52  -67.60   40.83    3.50  -63.20  145.08   17.07\n",
      "   45         99H  99H N   CA    748  749  101.62  140.00                  -42.30\n",
      "   46  10167  99H 100R C   N     756  758 -156.01  -63.00   93.29   15.14  -63.00   93.29   15.14\n",
      "   46        100R 100R N   CA    758  759  -33.84  -41.10                  -41.10\n",
      "   47  10196 128D 129S C   N     996  998   63.38  -64.10  150.10   14.61  -64.10  150.10   14.61\n",
      "   47        129S 129S N   CA    998  999 -114.23  -35.00                  -35.00\n",
      "   48  10209 141M 142A C   N    1099 1101   69.47  -68.20  146.63   13.88  -62.50 -179.38   28.13\n",
      "   48        142A 142A N   CA   1101 1102 -164.23  145.30                  -40.90\n",
      "   49  10219 151F 152W C   N    1179 1181   80.30  -63.00  143.47   24.21  -63.00  143.47   24.21\n",
      "   49        152W 152W N   CA   1181 1182  -37.36  -44.20                  -44.20\n",
      "   50  10221 153E 154G C   N    1202 1204  -64.42  -62.40    4.05    0.71   82.20  155.98   11.90\n",
      "   50        154G 154G N   CA   1204 1205  -44.71  -41.20                    8.50\n",
      "   51  10223 155I 156E C   N    1214 1216  -67.33  -63.60   11.04    1.68  -69.30  166.82   12.81\n",
      "   51        156E 156E N   CA   1216 1217  -50.69  -40.30                  142.50\n",
      "   52  10226 158T 159M C   N    1239 1241  -97.11 -125.60   57.35    2.03  -63.40  135.49   17.90\n",
      "   52        159M 159M N   CA   1241 1242   90.73  140.50                  -40.50\n",
      "   53  10244 176D 177T C   N    1390 1392  -88.48 -124.80   61.49    1.91  -63.20  138.32   16.19\n",
      "   53        177T 177T N   CA   1392 1393   93.88  143.50                  -42.10\n",
      "   54  10246 178K 179R C   N    1406 1408   50.16   57.30    9.55    0.66  -63.00  141.78   25.76\n",
      "   54        179R 179R N   CA   1408 1409   44.33   38.00                  -41.10\n",
      "   55  10251 183R 184D C   N    1453 1455 -105.67  -63.30   61.72    6.90  -63.30   61.72    6.90\n",
      "   55        184D 184D N   CA   1455 1456    4.87  -40.00                  -40.00\n",
      "   56  10275 207K 208L C   N    1639 1641 -145.32 -108.50   37.57    1.88  -63.50 -174.74   22.71\n",
      "   56        208L 208L N   CA   1641 1642  125.01  132.50                  -41.20\n",
      "   57  10278 210H 211T C   N    1664 1666  -75.88  -78.10   14.82    0.71  -63.20  153.97   20.59\n",
      "   57        211T 211T N   CA   1666 1667  164.45  149.80                  -42.10\n",
      "   58  10280 212L 213D C   N    1679 1681  -61.16  -70.90   24.60    0.84  -63.30  167.73   20.83\n",
      "   58        213D 213D N   CA   1681 1682  127.71  150.30                  -40.00\n",
      "   59  10282 214V 215S C   N    1694 1696  170.34 -136.60   57.01    1.81  -72.40  118.89    7.88\n",
      "   59        215S 215S N   CA   1696 1697  172.06  151.20                  152.40\n",
      "   60  10283 215S 216S C   N    1700 1702 -147.13 -136.60   34.63    1.92  -64.10  174.26   11.29\n",
      "   60        216S 216S N   CA   1702 1703  118.21  151.20                  -35.00\n",
      "   61  10325 257P 258W C   N    2023 2025  -46.06  -71.30   29.07    1.73  -63.00  169.62   22.27\n",
      "   61        258W 258W N   CA   2025 2026  124.58  139.00                  -44.20\n",
      "   62  10330 262E 263E C   N    2065 2067  -50.30  -69.30   19.74    1.67  -63.60  172.36   22.28\n",
      "   62        263E 263E N   CA   2067 2068  147.86  142.50                  -40.30\n",
      "   63  10344 276D 277T C   N    2174 2176  -88.78  -63.20   91.02   10.20  -63.20   91.02   10.20\n",
      "   63        277T 277T N   CA   2176 2177   45.24  -42.10                  -42.10\n",
      "   64  10349 281L 282P C   N    2210 2212  -76.52  -64.50   18.09    1.11  -58.70  169.73   15.49\n",
      "   64        282P 282P N   CA   2212 2213  160.71  147.20                  -30.50\n",
      "   65  10350 282P 283E C   N    2217 2219 -122.66 -117.80    6.09    0.31  -63.60 -176.79   21.77\n",
      "   65        283E 283E N   CA   2219 2220  133.13  136.80                  -40.30\n",
      "   66  10351 283E 284K C   N    2226 2228 -138.91 -118.00   25.03    0.84  -62.90 -177.11   28.30\n",
      "   66        284K 284K N   CA   2228 2229  152.86  139.10                  -40.80\n",
      "   67  10353 285P 286A C   N    2242 2244 -139.19 -134.00   14.41    0.66  -62.50  176.23   32.99\n",
      "   67        286A 286A N   CA   2244 2245  160.44  147.00                  -40.90\n",
      "   68  10354 286A 287Y C   N    2247 2249  -86.18  -98.40   55.16    5.80  -63.50  120.17   17.96\n",
      "   68        287Y 287Y N   CA   2249 2250   74.61  128.40                  -43.40\n",
      "   69  10355 287Y 288S C   N    2259 2261 -179.79 -136.60   47.17    1.47  -64.10 -166.72   22.35\n",
      "   69        288S 288S N   CA   2261 2262  170.16  151.20                  -35.00\n",
      "   70  10356 288S 289E C   N    2265 2267  -99.36 -117.80   31.93    1.14  -63.60  155.22   18.96\n",
      "   70        289E 289E N   CA   2267 2268  110.74  136.80                  -40.30\n",
      "   71  10357 289E 290E C   N    2274 2276 -138.49 -117.80   22.39    0.66  -63.60 -170.25   29.51\n",
      "   71        290E 290E N   CA   2276 2277  145.36  136.80                  -40.30\n",
      "   72  10358 290E 291A C   N    2283 2285 -131.36 -134.00   13.01    0.67  -62.50 -171.79   28.39\n",
      "   72        291A 291A N   CA   2285 2286  134.26  147.00                  -40.90\n",
      "   73  10359 291A 292R C   N    2288 2290 -119.56 -125.20   41.97    1.90  -63.00  151.10   17.43\n",
      "   73        292R 292R N   CA   2290 2291   99.01  140.60                  -41.10\n",
      "   74  10364 296K 297K C   N    2334 2336  -99.36  -62.90   92.21   14.16  -62.90   92.21   14.16\n",
      "   74        297K 297K N   CA   2336 2337 -125.49  -40.80                  -40.80\n",
      "   75  10368 300V 301R C   N    2364 2366  -91.42 -125.20   34.49    1.56  -63.00  173.72   25.43\n",
      "   75        301R 301R N   CA   2366 2367  147.52  140.60                  -41.10\n",
      "   76  10369 301R 302D C   N    2375 2377   54.45   54.50    9.12    0.67  -63.30  148.21   25.11\n",
      "   76        302D 302D N   CA   2377 2378   50.01   40.90                  -40.00\n",
      "   77  10371 303P 304Q C   N    2390 2392   49.41  -73.00  137.36    8.53  -63.80  164.02   29.45\n",
      "   77        304Q 304Q N   CA   2392 2393   78.38  140.70                  -40.30\n",
      "   78  10372 304Q 305E C   N    2399 2401  -64.95  -69.30   43.58    3.17  -63.60  139.44   18.52\n",
      "   78        305E 305E N   CA   2401 2402   99.13  142.50                  -40.30\n",
      "   79  10373 305E 306R C   N    2408 2410   66.60  -72.10  139.72   10.29 -125.20  168.91    7.19\n",
      "   79        306R 306R N   CA   2410 2411  125.04  141.90                  140.60\n",
      "   80  10382 314K 315H C   N    2478 2480 -155.58 -125.60   30.26    1.14  -63.20 -160.37   21.23\n",
      "   80        315H 315H N   CA   2480 2481  134.67  138.80                  -42.30\n",
      "   81  10385 317F 318F C   N    2506 2508  -52.01  -71.40   20.47    1.60  -63.20  168.82   22.70\n",
      "   81        318F 318F N   CA   2508 2509  147.25  140.70                  -44.30\n",
      "   82  10386 318F 319K C   N    2517 2519  -56.71  -70.20   14.13    0.93  -62.90  177.08   23.34\n",
      "   82        319K 319K N   CA   2519 2520  136.18  140.40                  -40.80\n",
      "   83  10389 321C 322T C   N    2536 2538 -112.08 -124.80   16.77    1.01  -63.20  170.63   25.13\n",
      "   83        322T 322T N   CA   2538 2539  154.42  143.50                  -42.10\n",
      "   84  10398 330V 331K C   N    2595 2597  -95.12 -118.00   25.93    1.21  -62.90  170.96   24.08\n",
      "   84        331K 331K N   CA   2597 2598  151.30  139.10                  -40.80\n",
      "   85  10399 331K 332Q C   N    2604 2606 -121.85 -121.10   45.98    2.20  -63.80  146.06   19.04\n",
      "   85        332Q 332Q N   CA   2606 2607   93.72  139.70                  -40.30\n",
      "   86  10402 334V 335D C   N    2624 2626 -102.50  -70.90   51.37    1.85  -63.30  135.01   19.19\n",
      "   86        335D 335D N   CA   2626 2627 -169.20  150.30                  -40.00\n",
      "   87  10403 335D 336Y C   N    2632 2634  -69.60  -98.40   29.03    1.13  -63.50  175.56   27.54\n",
      "   87        336Y 336Y N   CA   2634 2635  132.05  128.40                  -43.40\n",
      "   88  10404 336Y 337M C   N    2644 2646 -139.80 -125.60   51.38    2.12  -63.40  150.46   28.37\n",
      "   88        337M 337M N   CA   2646 2647 -170.13  140.50                  -40.50\n",
      "   89  10406 338T 339R C   N    2659 2661  -59.80  -72.10   13.50    0.91  -63.00  177.48   24.19\n",
      "   89        339R 339R N   CA   2661 2662  136.35  141.90                  -41.10\n",
      "   90  10409 341I 342N C   N    2686 2688  -83.43  -71.20   15.67    0.87  -63.20  167.54   22.08\n",
      "   90        342N 342N N   CA   2688 2689  152.59  142.80                  -41.10\n",
      "   91  10410 342N 343S C   N    2694 2696  -93.80  -72.40   25.91    2.09  -64.10  175.33   11.28\n",
      "   91        343S 343S N   CA   2696 2697  137.80  152.40                  -35.00\n",
      "   92  10411 343S 344D C   N    2700 2702 -128.81  -70.90   69.08    3.46  -63.30  147.41   22.42\n",
      "   92        344D 344D N   CA   2702 2703 -172.05  150.30                  -40.00\n",
      "   93  10415 347K 348N C   N    2731 2733   46.18  -63.20  166.98   18.31  -63.20  166.98   18.31\n",
      "   93        348N 348N N   CA   2733 2734 -167.27  -41.10                  -41.10\n",
      "   94  10416 348N 349G C   N    2739 2741  -93.39  -80.20   50.93    1.40   82.20 -132.15   17.71\n",
      "   94        349G 349G N   CA   2741 2742 -136.71  174.10                    8.50\n",
      "   95  10437 369L 370G C   N    2905 2907 -150.41 -167.20   29.59    0.80   82.20 -169.44   14.45\n",
      "   95        370G 370G N   CA   2907 2908  150.24  174.60                    8.50\n",
      "   96  10438 370G 371D C   N    2909 2911  -73.89  -70.90    3.03    0.24  -63.30  170.53   21.70\n",
      "   96        371D 371D N   CA   2911 2912  149.81  150.30                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   31   62  237  220  378  332  360  431  484  468\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00194.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      372     2927     2927\n",
      "openf___224_> Open           EAO00194.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     40331457   39386.188    38.463\n",
      "read_mo_297_> Segments, residues, atoms:        1      372     2927\n",
      "read_mo_298_> Segment:        1  1:A  372:A     2927\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2927     2927        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    14510\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    15739\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    16544\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    17530\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    19055\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    20020\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    21585\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    23455\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    24867\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25435\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2927    25521\n",
      "iupac_m_485_> OD1/2 will be swapped:       92.3725        9  9\n",
      "iupac_m_487_> NH1/2 swapped:     -175.7990       18  18\n",
      "iupac_m_485_> OD1/2 will be swapped:     -157.0492       19  19\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.3522       36  36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.3672       44  44\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -142.7233       47  47\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9031      100  100\n",
      "iupac_m_487_> NH1/2 swapped:      179.2699      105  105\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.1734      118  118\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      127.2435      128  128\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.1410      139  139\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      107.4656      143  143\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.5617      147  147\n",
      "iupac_m_486_> OE1/2 will be swapped:      147.5866      153  153\n",
      "iupac_m_487_> NH1/2 swapped:      173.6106      183  183\n",
      "iupac_m_485_> OD1/2 will be swapped:      122.1855      194  194\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0508      197  197\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2489\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.3618      202  202\n",
      "iupac_m_486_> OE1/2 will be swapped:       94.9876      231  231\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.2184      235  235\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      155.5913      262  262\n",
      "iupac_m_485_> OD1/2 will be swapped:      153.0502      276  276\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.3191      283  283\n",
      "iupac_m_486_> OE1/2 will be swapped:      129.2027      289  289\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.4536      290  290\n",
      "iupac_m_487_> NH1/2 swapped:      179.4148      301  301\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.9163      302  302\n",
      "iupac_m_486_> OE1/2 will be swapped:       99.8453      305  305\n",
      "iupac_m_487_> NH1/2 swapped:     -176.5842      306  306\n",
      "iupac_m_486_> OE1/2 will be swapped:     -160.1545      323  323\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.8649      335  335\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.7582      344  344\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.8462      356  356\n",
      "iupac_m_485_> OD1/2 will be swapped:      159.7612      357  357\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.7788      361  361\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   592862\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -33037.6484\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -33037.648438\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    33.828999\n",
      "Sequence length          :          372\n",
      "Compactness              :     0.200056\n",
      "Native energy (pair)     :   158.445160\n",
      "Native energy (surface)  :    11.334930\n",
      "Native energy (combined) :     5.321446\n",
      "Z score (pair)           :    -3.974887\n",
      "Z score (surface)        :    -3.445463\n",
      "Z score (combined)       :    -4.937336\n",
      "GA341 score              :     0.987456\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      372\n",
      "Number of all, selected real atoms                :     2927    2927\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25521   25521\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     6678\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2477.7708\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2987       0      0   0.005   0.005      20.168       1.000\n",
      " 2 Bond angle potential               :    4036       0      4   1.990   1.990      298.05       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1855       0     83  48.640  48.640      674.92       1.000\n",
      " 4 Stereochemical improper torsion pot:    1190       0      1   1.291   1.291      40.640       1.000\n",
      " 5 Soft-sphere overlap restraints     :    6678       0      0   0.004   0.004      11.328       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    5546       0      1   0.644   0.644      285.78       1.000\n",
      "10 Distance restraints 2 (N-O)        :    5505       0     16   0.830   0.830      413.05       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     371       3      1  13.490  13.490      78.526       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     322       0      3  77.228  77.228      87.512       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     244       0      1  72.762  72.762      95.648       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     110       0      1  86.826  86.826      80.783       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      49       0      0  93.608  93.608      34.202       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2233       0      0   0.564   0.564      33.212       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     370      90     61  35.909  86.076      280.09       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     703       0      1   1.226   1.226      43.855       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00194.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00194.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   48418.3750\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10530  92I  92I CA  C     697  702  169.76   -0.75  170.50   34.79   -0.75  170.50   34.79\n",
      "    2  10696 258W 258W CA  C    2026 2037 -156.67 -180.00   23.33    4.66 -180.00   23.33    4.66\n",
      "    3  10704 266I 266I CA  C    2093 2098   -0.88 -179.18  178.30   41.44 -179.18  178.30   41.44\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10070   2K   3K C   N      16   18   62.25   56.60   14.07    1.44  -62.90  155.50   26.98\n",
      "    1          3K   3K N   CA     18   19   51.48   38.60                  -40.80\n",
      "    2  10071   3K   4L C   N      25   27 -117.67 -108.50   10.65    0.47  -63.50 -172.87   23.49\n",
      "    2          4L   4L N   CA     27   28  137.92  132.50                  -41.20\n",
      "    3  10072   4L   5G C   N      33   35  -78.76  -80.20   35.09    1.31   82.20 -152.76    7.34\n",
      "    3          5G   5G N   CA     35   36  139.04  174.10                    8.50\n",
      "    4  10075   7V   8L C   N      52   54  -70.88  -70.70   53.07    4.03  -63.50  129.94   17.63\n",
      "    4          8L   8L N   CA     54   55   88.53  141.60                  -41.20\n",
      "    5  10076   8L   9D C   N      60   62 -159.17  -96.50   76.31    3.17  -63.30 -171.52   29.40\n",
      "    5          9D   9D N   CA     62   63  157.73  114.20                  -40.00\n",
      "    6  10077   9D  10A C   N      68   70  -78.13  -68.20   21.24    2.02  -62.50  168.16   26.83\n",
      "    6         10A  10A N   CA     70   71  126.53  145.30                  -40.90\n",
      "    7  10078  10A  11A C   N      73   75 -160.02 -134.00   27.04    0.66  -62.50 -168.53   36.58\n",
      "    7         11A  11A N   CA     75   76  154.33  147.00                  -40.90\n",
      "    8  10079  11A  12A C   N      78   80  -80.45  -68.20   20.27    1.28  -62.50  158.68   26.92\n",
      "    8         12A  12A N   CA     80   81  161.45  145.30                  -40.90\n",
      "    9  10081  13V  14E C   N      90   92  -82.08  -69.30   19.61    1.10  -63.60  163.37   22.81\n",
      "    9         14E  14E N   CA     92   93  157.38  142.50                  -40.30\n",
      "   10  10082  14E  15K C   N      99  101  -86.34  -70.20   21.64    1.30  -62.90  166.04   22.91\n",
      "   10         15K  15K N   CA    101  102  154.82  140.40                  -40.80\n",
      "   11  10083  15K  16S C   N     108  110  -65.19  -64.10   15.97    1.10 -136.60 -175.37    7.59\n",
      "   11         16S  16S N   CA    110  111  -19.07  -35.00                  151.20\n",
      "   12  10084  16S  17Q C   N     114  116  -61.51  -73.00   39.65    2.38  -63.80  143.07   21.16\n",
      "   12         17Q  17Q N   CA    116  117  102.75  140.70                  -40.30\n",
      "   13  10085  17Q  18R C   N     123  125  178.56 -125.20  105.68    3.65  -63.00  148.05   26.91\n",
      "   13         18R  18R N   CA    125  126 -129.93  140.60                  -41.10\n",
      "   14  10097  29N  30S C   N     227  229 -143.24 -136.60   21.04    1.17   56.90 -174.13   23.53\n",
      "   14         30S  30S N   CA    229  230  131.23  151.20                   36.40\n",
      "   15  10098  30S  31L C   N     233  235 -113.16 -108.50    7.64    0.35  -63.50 -173.50   23.55\n",
      "   15         31L  31L N   CA    235  236  138.56  132.50                  -41.20\n",
      "   16  10099  31L  32Q C   N     241  243 -106.06 -121.10   19.98    1.01  -63.80  172.12   27.57\n",
      "   16         32Q  32Q N   CA    243  244  152.84  139.70                  -40.30\n",
      "   17  10101  33L  34R C   N     258  260 -117.12 -125.20   67.06    3.05  -63.00  127.21   14.51\n",
      "   17         34R  34R N   CA    260  261   74.03  140.60                  -41.10\n",
      "   18  10102  34R  35A C   N     269  271 -149.32 -134.00   24.83    1.50  -62.50 -170.57   28.39\n",
      "   18         35A  35A N   CA    271  272  127.47  147.00                  -40.90\n",
      "   19  10104  36F  37L C   N     285  287 -145.92 -108.50   38.89    1.75  -63.50 -165.93   32.05\n",
      "   19         37L  37L N   CA    287  288  143.10  132.50                  -41.20\n",
      "   20  10105  37L  38L C   N     293  295 -132.04 -108.50   41.17    1.92  -63.50  167.22   27.46\n",
      "   20         38L  38L N   CA    295  296  166.28  132.50                  -41.20\n",
      "   21  10106  38L  39N C   N     301  303 -145.51 -119.90   56.15    1.90   55.90 -143.43   25.83\n",
      "   21         39N  39N N   CA    303  304 -173.03  137.00                   39.50\n",
      "   22  10107  39N  40E C   N     309  311  -84.83  -69.30   39.60    3.48  -63.60  147.90   18.63\n",
      "   22         40E  40E N   CA    311  312  106.07  142.50                  -40.30\n",
      "   23  10109  41S  42G C   N     324  326 -101.76  -80.20   50.94    1.19   82.20 -129.86    7.96\n",
      "   23         42G  42G N   CA    326  327 -139.74  174.10                    8.50\n",
      "   24  10111  43M  44F C   N     336  338 -126.30 -124.20   44.37    2.04   58.10 -125.96   35.00\n",
      "   24         44F  44F N   CA    338  339 -172.38  143.30                   32.90\n",
      "   25  10112  44F  45T C   N     347  349  -60.27  -78.10   36.80    2.16  -63.20  135.94   17.24\n",
      "   25         45T  45T N   CA    349  350 -178.01  149.80                  -42.10\n",
      "   26  10116  48G  49R C   N     375  377   66.04   57.30  104.31    7.51  -72.10  138.14   10.55\n",
      "   26         49R  49R N   CA    377  378  141.94   38.00                  141.90\n",
      "   27  10118  50K  51L C   N     395  397 -141.02  -63.50   97.65   17.88  -63.50   97.65   17.88\n",
      "   27         51L  51L N   CA    397  398 -100.59  -41.20                  -41.20\n",
      "   28  10121  53G  54S C   N     414  416  -94.81  -72.40   43.98    3.31  -64.10  152.69    9.67\n",
      "   28         54S  54S N   CA    416  417  114.56  152.40                  -35.00\n",
      "   29  10123  55V  56T C   N     427  429 -136.32 -124.80   25.90    0.83  -63.20  167.95   26.30\n",
      "   29         56T  56T N   CA    429  430  166.70  143.50                  -42.10\n",
      "   30  10124  56T  57K C   N     434  436  -74.16  -70.20    9.40    0.76  -62.90  173.04   21.88\n",
      "   30         57K  57K N   CA    436  437  131.88  140.40                  -40.80\n",
      "   31  10125  57K  58E C   N     443  445  177.07 -117.80   92.40    2.97  -63.60  167.37   28.40\n",
      "   31         58E  58E N   CA    445  446 -157.66  136.80                  -40.30\n",
      "   32  10128  60I  61D C   N     467  469 -129.44  -96.50   45.88    1.88  -63.30  139.01   14.75\n",
      "   32         61D  61D N   CA    469  470   82.27  114.20                  -40.00\n",
      "   33  10129  61D  62D C   N     475  477  -79.54  -70.90   15.64    0.54   54.50 -178.46   11.67\n",
      "   33         62D  62D N   CA    477  478  163.34  150.30                   40.90\n",
      "   34  10133  65K  66R C   N     505  507  -77.89  -72.10   57.67    4.31  -63.00  120.55   17.31\n",
      "   34         66R  66R N   CA    507  508 -160.73  141.90                  -41.10\n",
      "   35  10139  71G  72A C   N     551  553  165.56 -134.00   63.73    2.90  -62.50 -146.62   32.32\n",
      "   35         72A  72A N   CA    553  554  126.80  147.00                  -40.90\n",
      "   36  10140  72A  73S C   N     556  558  115.11 -136.60  143.20    4.49  -64.10 -163.70   20.94\n",
      "   36         73S  73S N   CA    558  559 -115.10  151.20                  -35.00\n",
      "   37  10149  81L  82K C   N     623  625  -61.03  -62.90    6.69    0.79 -118.00 -177.23    9.40\n",
      "   37         82K  82K N   CA    625  626  -47.23  -40.80                  139.10\n",
      "   38  10150  82K  83D C   N     632  634  -96.32  -96.50   24.48    1.03  -63.30 -178.29   20.64\n",
      "   38         83D  83D N   CA    634  635  138.68  114.20                  -40.00\n",
      "   39  10162  94I  95T C   N     717  719 -168.37 -124.80   64.74    2.05  -63.20  164.52   27.50\n",
      "   39         95T  95T N   CA    719  720 -168.61  143.50                  -42.10\n",
      "   40  10163  95T  96S C   N     724  726  -94.46  -72.40   28.58    2.29  -64.10  171.93   11.02\n",
      "   40         96S  96S N   CA    726  727  134.22  152.40                  -35.00\n",
      "   41  10164  96S  97K C   N     730  732   65.61   56.60   16.84    0.88  -62.90  144.10   25.18\n",
      "   41         97K  97K N   CA    732  733   24.37   38.60                  -40.80\n",
      "   42  10167  99H 100R C   N     756  758  -51.10  -72.10   21.07    1.57  -63.00  179.20   23.43\n",
      "   42        100R 100R N   CA    758  759  140.09  141.90                  -41.10\n",
      "   43  10196 128D 129S C   N     996  998   80.30 -136.60  161.88    4.96  -64.10  174.58   16.49\n",
      "   43        129S 129S N   CA    998  999 -133.11  151.20                  -35.00\n",
      "   44  10209 141M 142A C   N    1099 1101   85.72 -134.00  154.58    3.51  -62.50 -177.09   29.43\n",
      "   44        142A 142A N   CA   1101 1102 -148.07  147.00                  -40.90\n",
      "   45  10220 152W 153E C   N    1193 1195  -53.12  -63.60   12.21    2.10   54.60  132.08   17.20\n",
      "   45        153E 153E N   CA   1195 1196  -34.04  -40.30                   42.40\n",
      "   46  10226 158T 159M C   N    1239 1241  -87.92  -73.00   49.96    3.64  -63.40  138.02   18.83\n",
      "   46        159M 159M N   CA   1241 1242   95.32  143.00                  -40.50\n",
      "   47  10245 177T 178K C   N    1397 1399  172.91 -118.00   93.43    3.24   56.60 -159.43   22.99\n",
      "   47        178K 178K N   CA   1399 1400 -158.00  139.10                   38.60\n",
      "   48  10246 178K 179R C   N    1406 1408  -78.76  -72.10   64.98    5.14  -63.00  119.41   15.13\n",
      "   48        179R 179R N   CA   1408 1409   77.27  141.90                  -41.10\n",
      "   49  10251 183R 184D C   N    1453 1455 -105.95  -63.30   62.11    6.94  -63.30   62.11    6.94\n",
      "   49        184D 184D N   CA   1455 1456    5.15  -40.00                  -40.00\n",
      "   50  10275 207K 208L C   N    1639 1641 -150.36 -108.50   41.87    1.98  -63.50 -164.78   23.93\n",
      "   50        208L 208L N   CA   1641 1642  133.63  132.50                  -41.20\n",
      "   51  10278 210H 211T C   N    1664 1666  -85.94  -78.10   21.37    0.71  -63.20  149.96   20.75\n",
      "   51        211T 211T N   CA   1666 1667  169.68  149.80                  -42.10\n",
      "   52  10279 211T 212L C   N    1671 1673  -69.92  -70.70   13.65    1.00  -63.50  169.30   23.15\n",
      "   52        212L 212L N   CA   1673 1674  127.97  141.60                  -41.20\n",
      "   53  10280 212L 213D C   N    1679 1681 -159.24  -63.30  106.24   14.08  -63.30  106.24   14.08\n",
      "   53        213D 213D N   CA   1681 1682    5.64  -40.00                  -40.00\n",
      "   54  10283 215S 216S C   N    1700 1702 -173.33 -136.60   57.66    1.96  -64.10  169.31   20.13\n",
      "   54        216S 216S N   CA   1702 1703 -164.36  151.20                  -35.00\n",
      "   55  10323 255K 256N C   N    2008 2010   44.73   55.90   16.17    0.94  -63.20  142.01   23.16\n",
      "   55        256N 256N N   CA   2010 2011   51.20   39.50                  -41.10\n",
      "   56  10325 257P 258W C   N    2023 2025   27.75  -71.30  106.63    6.59  -63.00  169.95   27.34\n",
      "   56        258W 258W N   CA   2025 2026   99.50  139.00                  -44.20\n",
      "   57  10327 259A 260P C   N    2042 2044  -74.00  -58.70   37.57    2.21  -64.50  143.70   11.23\n",
      "   57        260P 260P N   CA   2044 2045    3.81  -30.50                  147.20\n",
      "   58  10344 276D 277T C   N    2174 2176  -86.82  -63.20   92.18   10.44  -63.20   92.18   10.44\n",
      "   58        277T 277T N   CA   2176 2177   47.00  -42.10                  -42.10\n",
      "   59  10350 282P 283E C   N    2217 2219  -49.60  -69.30   23.84    1.37  -63.60  169.94   23.44\n",
      "   59        283E 283E N   CA   2219 2220  129.06  142.50                  -40.30\n",
      "   60  10351 283E 284K C   N    2226 2228 -119.16 -118.00    4.50    0.20  -62.90 -175.46   27.31\n",
      "   60        284K 284K N   CA   2228 2229  143.45  139.10                  -40.80\n",
      "   61  10353 285P 286A C   N    2242 2244  -69.06  -68.20   32.15    2.56  -62.50  141.81   23.57\n",
      "   61        286A 286A N   CA   2244 2245  177.44  145.30                  -40.90\n",
      "   62  10354 286A 287Y C   N    2247 2249  -53.10  -98.40   46.47    2.18  -63.50  161.75   26.20\n",
      "   62        287Y 287Y N   CA   2249 2250  118.01  128.40                  -43.40\n",
      "   63  10355 287Y 288S C   N    2259 2261  167.46 -136.60  103.13    3.78  -64.10  155.21   20.18\n",
      "   63        288S 288S N   CA   2261 2262 -122.16  151.20                  -35.00\n",
      "   64  10356 288S 289E C   N    2265 2267  -58.41  -69.30   18.76    1.71  -63.60  162.00   21.32\n",
      "   64        289E 289E N   CA   2267 2268  157.78  142.50                  -40.30\n",
      "   65  10357 289E 290E C   N    2274 2276  -53.62  -69.30   15.68    1.18  -63.60  177.72   23.16\n",
      "   65        290E 290E N   CA   2276 2277  142.26  142.50                  -40.30\n",
      "   66  10359 291A 292R C   N    2288 2290  -76.84  -72.10   33.74    2.70  -63.00  150.23   19.40\n",
      "   66        292R 292R N   CA   2290 2291  108.50  141.90                  -41.10\n",
      "   67  10365 297K 298C C   N    2343 2345  -63.30  -63.00    4.94    0.62 -117.90 -178.71    7.81\n",
      "   67        298C 298C N   CA   2345 2346  -46.03  -41.10                  141.10\n",
      "   68  10371 303P 304Q C   N    2390 2392 -120.12 -121.10   44.78    2.15  -63.80  146.49   24.51\n",
      "   68        304Q 304Q N   CA   2392 2393 -175.53  139.70                  -40.30\n",
      "   69  10381 313L 314K C   N    2469 2471  -91.44  -62.90   78.37   11.88  -62.90   78.37   11.88\n",
      "   69        314K 314K N   CA   2471 2472 -113.79  -40.80                  -40.80\n",
      "   70  10382 314K 315H C   N    2478 2480  109.47   56.30  114.28   15.05  -63.20 -113.62   27.62\n",
      "   70        315H 315H N   CA   2480 2481  141.95   40.80                  -42.30\n",
      "   71  10384 316P 317F C   N    2495 2497  -63.02  -71.40   18.70    1.24  -63.20  168.29   23.38\n",
      "   71        317F 317F N   CA   2497 2498  123.98  140.70                  -44.30\n",
      "   72  10385 317F 318F C   N    2506 2508 -168.27 -124.20   54.96    1.42  -63.20  174.70   30.60\n",
      "   72        318F 318F N   CA   2508 2509  176.14  143.30                  -44.30\n",
      "   73  10386 318F 319K C   N    2517 2519  -62.44  -70.20   22.82    1.82  -62.90  157.34   20.41\n",
      "   73        319K 319K N   CA   2519 2520  161.86  140.40                  -40.80\n",
      "   74  10387 319K 320G C   N    2526 2528   79.21   78.70    6.33    0.20   82.20  168.32    8.32\n",
      "   74        320G 320G N   CA   2528 2529 -159.79 -166.10                    8.50\n",
      "   75  10389 321C 322T C   N    2536 2538 -107.54 -124.80   30.08    0.93  -63.20  166.96   18.83\n",
      "   75        322T 322T N   CA   2538 2539  118.86  143.50                  -42.10\n",
      "   76  10398 330V 331K C   N    2595 2597 -113.96 -118.00   19.26    0.87  -62.90  168.97   19.77\n",
      "   76        331K 331K N   CA   2597 2598  120.27  139.10                  -40.80\n",
      "   77  10399 331K 332Q C   N    2604 2606 -134.23 -121.10   55.87    2.41  -63.80  144.08   24.81\n",
      "   77        332Q 332Q N   CA   2606 2607 -166.00  139.70                  -40.30\n",
      "   78  10403 335D 336Y C   N    2632 2634 -141.74 -124.30   17.59    1.25  -63.50 -164.75   34.86\n",
      "   78        336Y 336Y N   CA   2634 2635  137.71  135.40                  -43.40\n",
      "   79  10404 336Y 337M C   N    2644 2646 -131.48 -125.60   23.84    1.00  -63.40  170.11   30.77\n",
      "   79        337M 337M N   CA   2646 2647  163.61  140.50                  -40.50\n",
      "   80  10406 338T 339R C   N    2659 2661  -59.74  -72.10   18.62    1.62  -63.00  163.11   21.82\n",
      "   80        339R 339R N   CA   2661 2662  155.82  141.90                  -41.10\n",
      "   81  10409 341I 342N C   N    2686 2688 -123.52 -119.90   82.58    3.67  -63.20  116.27   17.88\n",
      "   81        342N 342N N   CA   2688 2689 -140.50  137.00                  -41.10\n",
      "   82  10411 343S 344D C   N    2700 2702   43.82   54.50   23.33    1.14  -63.30  147.67   24.67\n",
      "   82        344D 344D N   CA   2702 2703   61.64   40.90                  -40.00\n",
      "   83  10413 345A 346K C   N    2713 2715   62.32  -70.20  137.41    9.17 -118.00 -176.94    7.55\n",
      "   83        346K 346K N   CA   2715 2716  104.09  140.40                  139.10\n",
      "   84  10415 347K 348N C   N    2731 2733  -53.86 -119.90   69.85    2.33  -63.20  155.61   19.97\n",
      "   84        348N 348N N   CA   2733 2734  114.23  137.00                  -41.10\n",
      "   85  10416 348N 349G C   N    2739 2741  -61.06  -80.20   57.97    1.45   82.20 -178.84    6.60\n",
      "   85        349G 349G N   CA   2741 2742  119.38  174.10                    8.50\n",
      "   86  10417 349G 350L C   N    2743 2745  170.54 -108.50  119.34    5.42  -63.50  159.97   29.27\n",
      "   86        350L 350L N   CA   2745 2746 -139.82  132.50                  -41.20\n",
      "   87  10421 353R 354S C   N    2779 2781  -54.03  -72.40   18.93    1.46  -64.10  168.31   11.63\n",
      "   87        354S 354S N   CA   2781 2782  156.99  152.40                  -35.00\n",
      "   88  10427 359T 360K C   N    2823 2825  -81.78  -62.90   70.49   10.27  -62.90   70.49   10.27\n",
      "   88        360K 360K N   CA   2825 2826 -108.71  -40.80                  -40.80\n",
      "   89  10437 369L 370G C   N    2905 2907  -62.53  -80.20   47.63    1.15   82.20 -171.11    6.55\n",
      "   89        370G 370G N   CA   2907 2908  129.87  174.10                    8.50\n",
      "   90  10438 370G 371D C   N    2909 2911 -125.41  -70.90   70.57    3.15  -63.30  139.49   21.22\n",
      "   90        371D 371D N   CA   2911 2912 -164.89  150.30                  -40.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   40   61  228  238  354  332  367  423  491  488\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00194.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      372     2927     2927\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00194.B99990001.pdb        2849.37720   -31535.40430        0.99582\n",
      "EAO00194.B99990002.pdb        2545.63037   -31141.97266        0.95980\n",
      "EAO00194.B99990003.pdb        2655.48071   -30937.38281        0.88978\n",
      "EAO00194.B99990004.pdb        2701.92578   -32249.13867        0.99154\n",
      "EAO00194.B99990005.pdb        2477.77075   -33037.64844        0.98746\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40522799   39573.047    38.646\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40523327   39573.562    38.646\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40571927   39621.023    38.692\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40585227   39634.012    38.705\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     40854493   39896.965    38.962\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     44415616   43374.625    42.358\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44417858   43376.812    42.360\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44419308   43378.230    42.362\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44422208   43381.062    42.364\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44428008   43386.727    42.370\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     57866407   56510.164    55.186\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57878007   56521.492    55.197\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57901207   56544.148    55.219\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57947607   56589.461    55.263\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     58040407   56680.086    55.352\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     64759607   63241.805    61.760\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00195.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40856813   39899.230    38.964\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40858263   39900.648    38.965\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40861163   39903.480    38.968\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40866963   39909.145    38.974\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40868003   39910.160    38.975\n",
      "\n",
      "Read the alignment from file       : ali/EAO00195.ali\n",
      "\n",
      "Total number of alignment positions:   261\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00195     261      1    EAO00195\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     40869380   39911.504    38.976\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41194555   40229.059    39.286\n",
      "profile_iteration_> processing sequence:       1    261      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41195633   40230.109    39.287\n",
      "profile_iteration_> processing sequence:    6720    261      1     2.5781250     0.0003836     0.0000015  2607\n",
      "profile_iteration_> processing sequence:   13439    261      1     5.0781250     0.0003779     0.0000015  2646\n",
      "profile_iteration_> processing sequence:   20158    261      1     7.5468750     0.0003744     0.0000015  2671\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41196008   40230.477    39.288\n",
      "profile_iteration_> processing sequence:   26877    261      1    10.0312500     0.0003732     0.0000015  2679\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41196244   40230.707    39.288\n",
      "profile_iteration_> processing sequence:   33596    261      1    12.5156250     0.0003725     0.0000015  2684\n",
      "profile_iteration_> processing sequence:   40315    261      1    14.9687500     0.0003713     0.0000015  2693\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41199194   40233.586    39.291\n",
      "profile_iteration_> processing sequence:   47034    261      1    17.4843750     0.0003717     0.0000015  2690\n",
      "profile_iteration_> processing sequence:   53753    261      1    20.1718750     0.0003753     0.0000015  2665\n",
      "profile_iteration_> processing sequence:   60472    261      1    22.6718750     0.0003749     0.0000015  2667\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41202704   40237.016    39.294\n",
      "profile_iteration_> processing sequence:   67191    261      1    25.2656250     0.0003760     0.0000015  2659\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    21.55036\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    230.2134\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0222,      121,        0.0063,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00015   0.00000\n",
      "Z:         1  -2.85000   0.00015   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00045   0.00000\n",
      "Z:         1  -2.55000   0.00089   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00060   0.00016\n",
      "Z:         1  -2.25000   0.00119   0.00055\n",
      "Z:         1  -2.15000   0.00372   0.00163\n",
      "Z:         1  -2.05000   0.00670   0.00416\n",
      "Z:         1  -1.95000   0.01414   0.00933\n",
      "Z:         1  -1.85000   0.01994   0.01872\n",
      "Z:         1  -1.75000   0.04539   0.03400\n",
      "Z:         1  -1.65000   0.06995   0.05658\n",
      "Z:         1  -1.55000   0.09852   0.08721\n",
      "Z:         1  -1.45000   0.12442   0.12565\n",
      "Z:         1  -1.35000   0.17175   0.17059\n",
      "Z:         1  -1.25000   0.22145   0.21981\n",
      "Z:         1  -1.15000   0.26804   0.27052\n",
      "Z:         1  -1.05000   0.30644   0.31973\n",
      "Z:         1  -0.95000   0.36299   0.36469\n",
      "Z:         1  -0.85000   0.37579   0.40316\n",
      "Z:         1  -0.75000   0.45943   0.43360\n",
      "Z:         1  -0.65000   0.43100   0.45519\n",
      "Z:         1  -0.55000   0.44291   0.46779\n",
      "Z:         1  -0.45000   0.46300   0.47182\n",
      "Z:         1  -0.35000   0.45243   0.46811\n",
      "Z:         1  -0.25000   0.46910   0.45775\n",
      "Z:         1  -0.15000   0.41954   0.44196\n",
      "Z:         1  -0.05000   0.43219   0.42194\n",
      "Z:         1   0.05000   0.39424   0.39889\n",
      "Z:         1   0.15000   0.41151   0.37383\n",
      "Z:         1   0.25000   0.32980   0.34769\n",
      "Z:         1   0.35000   0.31938   0.32121\n",
      "Z:         1   0.45000   0.29051   0.29500\n",
      "Z:         1   0.55000   0.27474   0.26953\n",
      "Z:         1   0.65000   0.24884   0.24513\n",
      "Z:         1   0.75000   0.22562   0.22205\n",
      "Z:         1   0.85000   0.19154   0.20043\n",
      "Z:         1   0.95000   0.17770   0.18036\n",
      "Z:         1   1.05000   0.15597   0.16185\n",
      "Z:         1   1.15000   0.14585   0.14489\n",
      "Z:         1   1.25000   0.13186   0.12944\n",
      "Z:         1   1.35000   0.11177   0.11542\n",
      "Z:         1   1.45000   0.11594   0.10275\n",
      "Z:         1   1.55000   0.09331   0.09133\n",
      "Z:         1   1.65000   0.08126   0.08109\n",
      "Z:         1   1.75000   0.07665   0.07191\n",
      "Z:         1   1.85000   0.06608   0.06371\n",
      "Z:         1   1.95000   0.05507   0.05639\n",
      "Z:         1   2.05000   0.04837   0.04988\n",
      "Z:         1   2.15000   0.04688   0.04409\n",
      "Z:         1   2.25000   0.04361   0.03895\n",
      "Z:         1   2.35000   0.03423   0.03439\n",
      "Z:         1   2.45000   0.03110   0.03035\n",
      "Z:         1   2.55000   0.02724   0.02678\n",
      "Z:         1   2.65000   0.02724   0.02361\n",
      "Z:         1   2.75000   0.02426   0.02082\n",
      "Z:         1   2.85000   0.01831   0.01835\n",
      "Z:         1   2.95000   0.01548   0.01617\n",
      "Z:         1   3.05000   0.01563   0.01424\n",
      "Z:         1   3.15000   0.01444   0.01255\n",
      "Z:         1   3.25000   0.01101   0.01105\n",
      "Z:         1   3.35000   0.01072   0.00973\n",
      "Z:         1   3.45000   0.00789   0.00857\n",
      "Z:         1   3.55000   0.00938   0.00754\n",
      "Z:         1   3.65000   0.00625   0.00664\n",
      "Z:         1   3.75000   0.00580   0.00584\n",
      "Z:         1   3.85000   0.00595   0.00514\n",
      "Z:         1   3.95000   0.00372   0.00453\n",
      "Z:         1   4.05000   0.00283   0.00398\n",
      "Z:         1   4.15000   0.00327   0.00350\n",
      "Z:         1   4.25000   0.00446   0.00308\n",
      "Z:         1   4.35000   0.00342   0.00271\n",
      "Z:         1   4.45000   0.00208   0.00239\n",
      "Z:         1   4.55000   0.00223   0.00210\n",
      "Z:         1   4.65000   0.00193   0.00185\n",
      "Z:         1   4.75000   0.00208   0.00163\n",
      "Z:         1   4.85000   0.00089   0.00143\n",
      "Z:         1   4.95000   0.00119   0.00126\n",
      "Z:         1   5.05000   0.00089   0.00111\n",
      "Z:         1   5.15000   0.00074   0.00097\n",
      "Z:         1   5.25000   0.00030   0.00086\n",
      "Z:         1   5.35000   0.00089   0.00075\n",
      "Z:         1   5.45000   0.00119   0.00066\n",
      "Z:         1   5.55000   0.00015   0.00058\n",
      "Z:         1   5.65000   0.00074   0.00051\n",
      "Z:         1   5.75000   0.00015   0.00045\n",
      "Z:         1   5.85000   0.00074   0.00040\n",
      "Z:         1   5.95000   0.00015   0.00035\n",
      "Z:         1   6.05000   0.00030   0.00031\n",
      "Z:         1   6.15000   0.00000   0.00027\n",
      "Z:         1   6.25000   0.00000   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00030   0.00018\n",
      "Z:         1   6.55000   0.00000   0.00016\n",
      "Z:         1   6.65000   0.00030   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00030   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    25.265625\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000376\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  2669.328369\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000377\n",
      "profile_iteration__> Overall performance [seq/sec]:  2651.225586\n",
      "openf___224_> Open           buildprofile/EAO00195build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40877901   39919.824    38.984\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40878941   39920.840    38.985\n",
      "openf___224_> Open           buildprofile/EAO00195build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40522799   39573.047    38.646\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40523327   39573.562    38.646\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40571927   39621.023    38.692\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40585227   39634.012    38.705\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     40854493   39896.965    38.962\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     44415616   43374.625    42.358\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44417858   43376.812    42.360\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44419308   43378.230    42.362\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44422208   43381.062    42.364\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44428008   43386.727    42.370\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     57866407   56510.164    55.186\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57878007   56521.492    55.197\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57901207   56544.148    55.219\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57947607   56589.461    55.263\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     58040407   56680.086    55.352\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     64759607   63241.805    61.760\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00196.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40856813   39899.230    38.964\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40858263   39900.648    38.965\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40861163   39903.480    38.968\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40866963   39909.145    38.974\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40868435   39910.582    38.975\n",
      "\n",
      "Read the alignment from file       : ali/EAO00196.ali\n",
      "\n",
      "Total number of alignment positions:   369\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00196     369      1    EAO00196\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     40870244   39912.348    38.977\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41195419   40229.902    39.287\n",
      "profile_iteration_> processing sequence:       1    369      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41196497   40230.953    39.288\n",
      "profile_iteration_> processing sequence:    6720    369      1     3.6093750     0.0005371     0.0000021  1862\n",
      "profile_iteration_> processing sequence:   13439    369      1     7.0625000     0.0005255     0.0000021  1903\n",
      "profile_iteration_> processing sequence:   20158    369      1    10.5156250     0.0005217     0.0000020  1917\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41196872   40231.320    39.288\n",
      "profile_iteration_> processing sequence:   26877    369      1    14.0156250     0.0005215     0.0000020  1918\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41197108   40231.551    39.289\n",
      "profile_iteration_> processing sequence:   33596    369      1    17.5156250     0.0005214     0.0000020  1918\n",
      "profile_iteration_> processing sequence:   40315    369      1    20.9843750     0.0005205     0.0000020  1921\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41200058   40234.430    39.291\n",
      "profile_iteration_> processing sequence:   47034    369      1    24.5000000     0.0005209     0.0000020  1920\n",
      "profile_iteration_> processing sequence:   53753    369      1    28.0781250     0.0005224     0.0000020  1914\n",
      "profile_iteration_> processing sequence:   60472    369      1    31.5625000     0.0005219     0.0000020  1916\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41203568   40237.859    39.295\n",
      "profile_iteration_> processing sequence:   67191    369      1    35.2031250     0.0005239     0.0000020  1909\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    48.62099\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    260.3616\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0292,      121,        0.0102,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00015   0.00000\n",
      "Z:         1  -2.95000   0.00015   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00015   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00045   0.00016\n",
      "Z:         1  -2.25000   0.00074   0.00055\n",
      "Z:         1  -2.15000   0.00268   0.00163\n",
      "Z:         1  -2.05000   0.00461   0.00416\n",
      "Z:         1  -1.95000   0.01339   0.00933\n",
      "Z:         1  -1.85000   0.01756   0.01872\n",
      "Z:         1  -1.75000   0.03661   0.03400\n",
      "Z:         1  -1.65000   0.05566   0.05658\n",
      "Z:         1  -1.55000   0.08632   0.08721\n",
      "Z:         1  -1.45000   0.13186   0.12565\n",
      "Z:         1  -1.35000   0.17175   0.17059\n",
      "Z:         1  -1.25000   0.21625   0.21981\n",
      "Z:         1  -1.15000   0.27042   0.27052\n",
      "Z:         1  -1.05000   0.30941   0.31973\n",
      "Z:         1  -0.95000   0.33947   0.36469\n",
      "Z:         1  -0.85000   0.38844   0.40316\n",
      "Z:         1  -0.75000   0.42133   0.43360\n",
      "Z:         1  -0.65000   0.47193   0.45519\n",
      "Z:         1  -0.55000   0.43725   0.46779\n",
      "Z:         1  -0.45000   0.47312   0.47182\n",
      "Z:         1  -0.35000   0.43740   0.46811\n",
      "Z:         1  -0.25000   0.46389   0.45775\n",
      "Z:         1  -0.15000   0.56301   0.44196\n",
      "Z:         1  -0.05000   0.43859   0.42194\n",
      "Z:         1   0.05000   0.37430   0.39889\n",
      "Z:         1   0.15000   0.36701   0.37383\n",
      "Z:         1   0.25000   0.34156   0.34769\n",
      "Z:         1   0.35000   0.30673   0.32121\n",
      "Z:         1   0.45000   0.28724   0.29500\n",
      "Z:         1   0.55000   0.26625   0.26953\n",
      "Z:         1   0.65000   0.24542   0.24513\n",
      "Z:         1   0.75000   0.21878   0.22205\n",
      "Z:         1   0.85000   0.19005   0.20043\n",
      "Z:         1   0.95000   0.17710   0.18036\n",
      "Z:         1   1.05000   0.15552   0.16185\n",
      "Z:         1   1.15000   0.13796   0.14489\n",
      "Z:         1   1.25000   0.12725   0.12944\n",
      "Z:         1   1.35000   0.11475   0.11542\n",
      "Z:         1   1.45000   0.10403   0.10275\n",
      "Z:         1   1.55000   0.08945   0.09133\n",
      "Z:         1   1.65000   0.08424   0.08109\n",
      "Z:         1   1.75000   0.07322   0.07191\n",
      "Z:         1   1.85000   0.07010   0.06371\n",
      "Z:         1   1.95000   0.05923   0.05639\n",
      "Z:         1   2.05000   0.04971   0.04988\n",
      "Z:         1   2.15000   0.04703   0.04409\n",
      "Z:         1   2.25000   0.04599   0.03895\n",
      "Z:         1   2.35000   0.03646   0.03439\n",
      "Z:         1   2.45000   0.03259   0.03035\n",
      "Z:         1   2.55000   0.02828   0.02678\n",
      "Z:         1   2.65000   0.02366   0.02361\n",
      "Z:         1   2.75000   0.02232   0.02082\n",
      "Z:         1   2.85000   0.02084   0.01835\n",
      "Z:         1   2.95000   0.02084   0.01617\n",
      "Z:         1   3.05000   0.01667   0.01424\n",
      "Z:         1   3.15000   0.01042   0.01255\n",
      "Z:         1   3.25000   0.01220   0.01105\n",
      "Z:         1   3.35000   0.01131   0.00973\n",
      "Z:         1   3.45000   0.00908   0.00857\n",
      "Z:         1   3.55000   0.00640   0.00754\n",
      "Z:         1   3.65000   0.00685   0.00664\n",
      "Z:         1   3.75000   0.00536   0.00584\n",
      "Z:         1   3.85000   0.00699   0.00514\n",
      "Z:         1   3.95000   0.00521   0.00453\n",
      "Z:         1   4.05000   0.00446   0.00398\n",
      "Z:         1   4.15000   0.00372   0.00350\n",
      "Z:         1   4.25000   0.00298   0.00308\n",
      "Z:         1   4.35000   0.00283   0.00271\n",
      "Z:         1   4.45000   0.00342   0.00239\n",
      "Z:         1   4.55000   0.00149   0.00210\n",
      "Z:         1   4.65000   0.00223   0.00185\n",
      "Z:         1   4.75000   0.00223   0.00163\n",
      "Z:         1   4.85000   0.00208   0.00143\n",
      "Z:         1   4.95000   0.00164   0.00126\n",
      "Z:         1   5.05000   0.00119   0.00111\n",
      "Z:         1   5.15000   0.00134   0.00097\n",
      "Z:         1   5.25000   0.00119   0.00086\n",
      "Z:         1   5.35000   0.00045   0.00075\n",
      "Z:         1   5.45000   0.00060   0.00066\n",
      "Z:         1   5.55000   0.00060   0.00058\n",
      "Z:         1   5.65000   0.00104   0.00051\n",
      "Z:         1   5.75000   0.00104   0.00045\n",
      "Z:         1   5.85000   0.00045   0.00040\n",
      "Z:         1   5.95000   0.00045   0.00035\n",
      "Z:         1   6.05000   0.00074   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00060   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00015   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00030   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00015   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00015   0.00003\n",
      "Z:         1   7.95000   0.00015   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    35.203125\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000524\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1909.541748\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000525\n",
      "profile_iteration__> Overall performance [seq/sec]:  1903.624634\n",
      "openf___224_> Open           buildprofile/EAO00196build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40881897   39923.727    38.988\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40883369   39925.164    38.989\n",
      "openf___224_> Open           buildprofile/EAO00196build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40522799   39573.047    38.646\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40523327   39573.562    38.646\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40571927   39621.023    38.692\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40585227   39634.012    38.705\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     40854493   39896.965    38.962\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     44415616   43374.625    42.358\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44417858   43376.812    42.360\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44419308   43378.230    42.362\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44422208   43381.062    42.364\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44428008   43386.727    42.370\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     57866407   56510.164    55.186\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57878007   56521.492    55.197\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57901207   56544.148    55.219\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57947607   56589.461    55.263\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     58040407   56680.086    55.352\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     64759607   63241.805    61.760\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00197.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40856813   39899.230    38.964\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40858263   39900.648    38.965\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40861163   39903.480    38.968\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40866963   39909.145    38.974\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40867779   39909.941    38.975\n",
      "\n",
      "Read the alignment from file       : ali/EAO00197.ali\n",
      "\n",
      "Total number of alignment positions:   205\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00197     205      1    EAO00197\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     40868932   39911.066    38.976\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41194107   40228.621    39.286\n",
      "profile_iteration_> processing sequence:       1    205      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41195185   40229.672    39.287\n",
      "profile_iteration_> processing sequence:    6720    205      1     1.9843750     0.0002953     0.0000011  3386\n",
      "profile_iteration_> processing sequence:   13439    205      1     3.9062500     0.0002907     0.0000011  3440\n",
      "profile_iteration_> processing sequence:   20158    205      1     5.8437500     0.0002899     0.0000011  3449\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41195560   40230.039    39.287\n",
      "profile_iteration_> processing sequence:   26877    205      1     7.7968750     0.0002901     0.0000011  3447\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41195796   40230.270    39.287\n",
      "profile_iteration_> processing sequence:   33596    205      1     9.7343750     0.0002897     0.0000011  3451\n",
      "profile_iteration_> processing sequence:   40315    205      1    11.6562500     0.0002891     0.0000011  3459\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41198746   40233.148    39.290\n",
      "profile_iteration_> processing sequence:   47034    205      1    13.6093750     0.0002894     0.0000011  3456\n",
      "profile_iteration_> processing sequence:   53753    205      1    15.5937500     0.0002901     0.0000011  3447\n",
      "profile_iteration_> processing sequence:   60472    205      1    17.5312500     0.0002899     0.0000011  3449\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41202256   40236.578    39.294\n",
      "profile_iteration_> processing sequence:   67191    205      1    19.5468750     0.0002909     0.0000011  3437\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    30.96826\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    250.8956\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0209,      121,        0.0085,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00015   0.00000\n",
      "Z:         1  -2.55000   0.00045   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00015   0.00016\n",
      "Z:         1  -2.25000   0.00089   0.00055\n",
      "Z:         1  -2.15000   0.00193   0.00163\n",
      "Z:         1  -2.05000   0.00595   0.00416\n",
      "Z:         1  -1.95000   0.00982   0.00933\n",
      "Z:         1  -1.85000   0.02366   0.01872\n",
      "Z:         1  -1.75000   0.03155   0.03400\n",
      "Z:         1  -1.65000   0.05432   0.05658\n",
      "Z:         1  -1.55000   0.07918   0.08721\n",
      "Z:         1  -1.45000   0.12278   0.12565\n",
      "Z:         1  -1.35000   0.22324   0.17059\n",
      "Z:         1  -1.25000   0.22160   0.21981\n",
      "Z:         1  -1.15000   0.26402   0.27052\n",
      "Z:         1  -1.05000   0.30629   0.31973\n",
      "Z:         1  -0.95000   0.37891   0.36469\n",
      "Z:         1  -0.85000   0.42788   0.40316\n",
      "Z:         1  -0.75000   0.41061   0.43360\n",
      "Z:         1  -0.65000   0.45690   0.45519\n",
      "Z:         1  -0.55000   0.45050   0.46779\n",
      "Z:         1  -0.45000   0.45824   0.47182\n",
      "Z:         1  -0.35000   0.42862   0.46811\n",
      "Z:         1  -0.25000   0.44886   0.45775\n",
      "Z:         1  -0.15000   0.41553   0.44196\n",
      "Z:         1  -0.05000   0.42550   0.42194\n",
      "Z:         1   0.05000   0.38784   0.39889\n",
      "Z:         1   0.15000   0.37311   0.37383\n",
      "Z:         1   0.25000   0.34096   0.34769\n",
      "Z:         1   0.35000   0.31894   0.32121\n",
      "Z:         1   0.45000   0.29438   0.29500\n",
      "Z:         1   0.55000   0.27191   0.26953\n",
      "Z:         1   0.65000   0.26194   0.24513\n",
      "Z:         1   0.75000   0.22637   0.22205\n",
      "Z:         1   0.85000   0.20985   0.20043\n",
      "Z:         1   0.95000   0.18574   0.18036\n",
      "Z:         1   1.05000   0.16014   0.16185\n",
      "Z:         1   1.15000   0.15672   0.14489\n",
      "Z:         1   1.25000   0.12829   0.12944\n",
      "Z:         1   1.35000   0.11415   0.11542\n",
      "Z:         1   1.45000   0.10373   0.10275\n",
      "Z:         1   1.55000   0.08840   0.09133\n",
      "Z:         1   1.65000   0.08483   0.08109\n",
      "Z:         1   1.75000   0.07293   0.07191\n",
      "Z:         1   1.85000   0.06295   0.06371\n",
      "Z:         1   1.95000   0.05715   0.05639\n",
      "Z:         1   2.05000   0.05254   0.04988\n",
      "Z:         1   2.15000   0.04718   0.04409\n",
      "Z:         1   2.25000   0.04152   0.03895\n",
      "Z:         1   2.35000   0.03468   0.03439\n",
      "Z:         1   2.45000   0.03259   0.03035\n",
      "Z:         1   2.55000   0.02724   0.02678\n",
      "Z:         1   2.65000   0.02694   0.02361\n",
      "Z:         1   2.75000   0.02113   0.02082\n",
      "Z:         1   2.85000   0.02098   0.01835\n",
      "Z:         1   2.95000   0.01816   0.01617\n",
      "Z:         1   3.05000   0.01652   0.01424\n",
      "Z:         1   3.15000   0.01563   0.01255\n",
      "Z:         1   3.25000   0.01250   0.01105\n",
      "Z:         1   3.35000   0.00982   0.00973\n",
      "Z:         1   3.45000   0.00997   0.00857\n",
      "Z:         1   3.55000   0.00685   0.00754\n",
      "Z:         1   3.65000   0.00774   0.00664\n",
      "Z:         1   3.75000   0.00551   0.00584\n",
      "Z:         1   3.85000   0.00491   0.00514\n",
      "Z:         1   3.95000   0.00476   0.00453\n",
      "Z:         1   4.05000   0.00432   0.00398\n",
      "Z:         1   4.15000   0.00268   0.00350\n",
      "Z:         1   4.25000   0.00208   0.00308\n",
      "Z:         1   4.35000   0.00327   0.00271\n",
      "Z:         1   4.45000   0.00223   0.00239\n",
      "Z:         1   4.55000   0.00193   0.00210\n",
      "Z:         1   4.65000   0.00238   0.00185\n",
      "Z:         1   4.75000   0.00179   0.00163\n",
      "Z:         1   4.85000   0.00119   0.00143\n",
      "Z:         1   4.95000   0.00074   0.00126\n",
      "Z:         1   5.05000   0.00149   0.00111\n",
      "Z:         1   5.15000   0.00119   0.00097\n",
      "Z:         1   5.25000   0.00149   0.00086\n",
      "Z:         1   5.35000   0.00119   0.00075\n",
      "Z:         1   5.45000   0.00074   0.00066\n",
      "Z:         1   5.55000   0.00074   0.00058\n",
      "Z:         1   5.65000   0.00060   0.00051\n",
      "Z:         1   5.75000   0.00060   0.00045\n",
      "Z:         1   5.85000   0.00015   0.00040\n",
      "Z:         1   5.95000   0.00060   0.00035\n",
      "Z:         1   6.05000   0.00060   0.00031\n",
      "Z:         1   6.15000   0.00000   0.00027\n",
      "Z:         1   6.25000   0.00074   0.00024\n",
      "Z:         1   6.35000   0.00030   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00045   0.00016\n",
      "Z:         1   6.65000   0.00030   0.00014\n",
      "Z:         1   6.75000   0.00000   0.00013\n",
      "Z:         1   6.85000   0.00015   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00015   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    19.546875\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000291\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  3454.046631\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.000000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000293\n",
      "profile_iteration__> Overall performance [seq/sec]:  3418.353027\n",
      "openf___224_> Open           buildprofile/EAO00197build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40875829   39917.801    38.982\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40876645   39918.598    38.983\n",
      "openf___224_> Open           buildprofile/EAO00197build_profile.ali\n",
      "next file\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40522799   39573.047    38.646\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40523327   39573.562    38.646\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40571927   39621.023    38.692\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40585227   39634.012    38.705\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     40854493   39896.965    38.962\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     44415616   43374.625    42.358\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44417858   43376.812    42.360\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44419308   43378.230    42.362\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44422208   43381.062    42.364\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     44428008   43386.727    42.370\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     57866407   56510.164    55.186\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57878007   56521.492    55.197\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57901207   56544.148    55.219\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     57947607   56589.461    55.263\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     58040407   56680.086    55.352\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     64759607   63241.805    61.760\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00198.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40856813   39899.230    38.964\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40858263   39900.648    38.965\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40861163   39903.480    38.968\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40866963   39909.145    38.974\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40878563   39920.473    38.985\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40880359   39922.227    38.987\n",
      "\n",
      "Read the alignment from file       : ali/EAO00198.ali\n",
      "\n",
      "Total number of alignment positions:   450\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00198     450      1    EAO00198\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     40882492   39924.309    38.989\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41207667   40241.863    39.299\n",
      "profile_iteration_> processing sequence:       1    450      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41208745   40242.914    39.300\n",
      "profile_iteration_> processing sequence:    6720    450      1     4.3750000     0.0006510     0.0000025  1536\n",
      "profile_iteration_> processing sequence:   13439    450      1     8.5937500     0.0006395     0.0000025  1564\n",
      "profile_iteration_> processing sequence:   20158    450      1    12.7968750     0.0006348     0.0000025  1575\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41209120   40243.281    39.300\n",
      "profile_iteration_> processing sequence:   26877    450      1    17.0468750     0.0006343     0.0000025  1577\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41209356   40243.512    39.300\n",
      "profile_iteration_> processing sequence:   33596    450      1    21.3125000     0.0006344     0.0000025  1576\n",
      "profile_iteration_> processing sequence:   40315    450      1    25.5156250     0.0006329     0.0000025  1580\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41212306   40246.391    39.303\n",
      "profile_iteration_> processing sequence:   47034    450      1    29.7968750     0.0006335     0.0000025  1578\n",
      "profile_iteration_> processing sequence:   53753    450      1    34.1406250     0.0006351     0.0000025  1574\n",
      "profile_iteration_> processing sequence:   60472    450      1    38.3750000     0.0006346     0.0000025  1576\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     41215816   40249.820    39.306\n",
      "profile_iteration_> processing sequence:   67191    450      1    42.7968750     0.0006369     0.0000025  1570\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    31.47346\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    235.7506\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0186,      121,        0.0057,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00015   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00015   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00074   0.00016\n",
      "Z:         1  -2.25000   0.00298   0.00055\n",
      "Z:         1  -2.15000   0.00268   0.00163\n",
      "Z:         1  -2.05000   0.00610   0.00416\n",
      "Z:         1  -1.95000   0.01310   0.00933\n",
      "Z:         1  -1.85000   0.02069   0.01872\n",
      "Z:         1  -1.75000   0.03319   0.03400\n",
      "Z:         1  -1.65000   0.05194   0.05658\n",
      "Z:         1  -1.55000   0.08483   0.08721\n",
      "Z:         1  -1.45000   0.12219   0.12565\n",
      "Z:         1  -1.35000   0.15835   0.17059\n",
      "Z:         1  -1.25000   0.21238   0.21981\n",
      "Z:         1  -1.15000   0.26194   0.27052\n",
      "Z:         1  -1.05000   0.36210   0.31973\n",
      "Z:         1  -0.95000   0.35540   0.36469\n",
      "Z:         1  -0.85000   0.41880   0.40316\n",
      "Z:         1  -0.75000   0.41255   0.43360\n",
      "Z:         1  -0.65000   0.48175   0.45519\n",
      "Z:         1  -0.55000   0.46717   0.46779\n",
      "Z:         1  -0.45000   0.47282   0.47182\n",
      "Z:         1  -0.35000   0.46151   0.46811\n",
      "Z:         1  -0.25000   0.43591   0.45775\n",
      "Z:         1  -0.15000   0.43472   0.44196\n",
      "Z:         1  -0.05000   0.40838   0.42194\n",
      "Z:         1   0.05000   0.39603   0.39889\n",
      "Z:         1   0.15000   0.36358   0.37383\n",
      "Z:         1   0.25000   0.34320   0.34769\n",
      "Z:         1   0.35000   0.31760   0.32121\n",
      "Z:         1   0.45000   0.30361   0.29500\n",
      "Z:         1   0.55000   0.25569   0.26953\n",
      "Z:         1   0.65000   0.25896   0.24513\n",
      "Z:         1   0.75000   0.21982   0.22205\n",
      "Z:         1   0.85000   0.19303   0.20043\n",
      "Z:         1   0.95000   0.17264   0.18036\n",
      "Z:         1   1.05000   0.15999   0.16185\n",
      "Z:         1   1.15000   0.15106   0.14489\n",
      "Z:         1   1.25000   0.12621   0.12944\n",
      "Z:         1   1.35000   0.12382   0.11542\n",
      "Z:         1   1.45000   0.10135   0.10275\n",
      "Z:         1   1.55000   0.09689   0.09133\n",
      "Z:         1   1.65000   0.07739   0.08109\n",
      "Z:         1   1.75000   0.07516   0.07191\n",
      "Z:         1   1.85000   0.06281   0.06371\n",
      "Z:         1   1.95000   0.05670   0.05639\n",
      "Z:         1   2.05000   0.05313   0.04988\n",
      "Z:         1   2.15000   0.04748   0.04409\n",
      "Z:         1   2.25000   0.04852   0.03895\n",
      "Z:         1   2.35000   0.03959   0.03439\n",
      "Z:         1   2.45000   0.03185   0.03035\n",
      "Z:         1   2.55000   0.02456   0.02678\n",
      "Z:         1   2.65000   0.02530   0.02361\n",
      "Z:         1   2.75000   0.02262   0.02082\n",
      "Z:         1   2.85000   0.02188   0.01835\n",
      "Z:         1   2.95000   0.01697   0.01617\n",
      "Z:         1   3.05000   0.01518   0.01424\n",
      "Z:         1   3.15000   0.01503   0.01255\n",
      "Z:         1   3.25000   0.01206   0.01105\n",
      "Z:         1   3.35000   0.01072   0.00973\n",
      "Z:         1   3.45000   0.00893   0.00857\n",
      "Z:         1   3.55000   0.00759   0.00754\n",
      "Z:         1   3.65000   0.00536   0.00664\n",
      "Z:         1   3.75000   0.00789   0.00584\n",
      "Z:         1   3.85000   0.00595   0.00514\n",
      "Z:         1   3.95000   0.00536   0.00453\n",
      "Z:         1   4.05000   0.00446   0.00398\n",
      "Z:         1   4.15000   0.00283   0.00350\n",
      "Z:         1   4.25000   0.00357   0.00308\n",
      "Z:         1   4.35000   0.00223   0.00271\n",
      "Z:         1   4.45000   0.00164   0.00239\n",
      "Z:         1   4.55000   0.00164   0.00210\n",
      "Z:         1   4.65000   0.00119   0.00185\n",
      "Z:         1   4.75000   0.00134   0.00163\n",
      "Z:         1   4.85000   0.00253   0.00143\n",
      "Z:         1   4.95000   0.00208   0.00126\n",
      "Z:         1   5.05000   0.00134   0.00111\n",
      "Z:         1   5.15000   0.00074   0.00097\n",
      "Z:         1   5.25000   0.00060   0.00086\n",
      "Z:         1   5.35000   0.00104   0.00075\n",
      "Z:         1   5.45000   0.00030   0.00066\n",
      "Z:         1   5.55000   0.00089   0.00058\n",
      "Z:         1   5.65000   0.00045   0.00051\n",
      "Z:         1   5.75000   0.00074   0.00045\n",
      "Z:         1   5.85000   0.00015   0.00040\n",
      "Z:         1   5.95000   0.00030   0.00035\n",
      "Z:         1   6.05000   0.00045   0.00031\n",
      "Z:         1   6.15000   0.00045   0.00027\n",
      "Z:         1   6.25000   0.00045   0.00024\n",
      "Z:         1   6.35000   0.00030   0.00021\n",
      "Z:         1   6.45000   0.00030   0.00018\n",
      "Z:         1   6.55000   0.00030   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00015   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00060   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     41592816   40617.984    39.666\n",
      "> 5jgfA                      1     876   32050     460     450   34.30     0.0           2   431    10   443     5   453\n",
      "> 3vatA                      1    9251   55050     458     450   49.08     0.0           3   425    12   440    12   445\n",
      "> 3varA                      1    9287   45000     414     450   48.72     0.0           4   384    12   440    12   401\n",
      "> 5jm6A                      1   23106   31350     445     450   37.88     0.0           5   403    15   439     9   433\n",
      "> 4emeA                      1   28265   33300     474     450   33.12     0.0           6   432    10   445     8   466\n",
      "> 4r8fA                      1   34765   29100     438     450   34.20     0.0           7   411    10   443     5   428\n",
      "> 4oidA                      1   37455   37600     407     450   43.00     0.0           8   395    10   442     4   403\n",
      "> 5jh9A                      1   39910   32050     475     450   34.30     0.0           9   431    10   443    19   467\n",
      "> 3wt4A                      1   56106   41400     425     450   43.78     0.0          10   413    10   442     4   421\n",
      "> 4dyoA                      1   59516   56400     457     450   50.11     0.0          11   426    12   440    12   446\n",
      "> 2glfA                      1   61036   14300     450     450   27.10     0.0          12   401    12   429    20   436\n",
      "> 2gljA                      1   61074   13400     456     450   23.50     0.0          13   420    11   442    23   456\n",
      "> 2ijzA                      1   64677   39000     379     450   45.89     0.0          14   376    16   442     2   378\n",
      "> 1y7eA                      1   66060   11100     404     450   28.31     0.0          15   377    15   440    18   402\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.015625\n",
      "profile_iteration__> Score Collection        [sec]:    42.781250\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000637\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1571.742676\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.031250\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000639\n",
      "profile_iteration__> Overall performance [seq/sec]:  1565.448853\n",
      "openf___224_> Open           buildprofile/EAO00198build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41338146   40369.281    39.423\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41339942   40371.039    39.425\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41341662   40372.719    39.426\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41343358   40374.375    39.428\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41344890   40375.867    39.430\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41346498   40377.438    39.431\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41348222   40379.125    39.433\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41349862   40380.727    39.434\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41351438   40382.266    39.436\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41353158   40383.945    39.437\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41354806   40385.555    39.439\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41356506   40387.211    39.441\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41358106   40388.773    39.442\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41359782   40390.414    39.444\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41361282   40391.875    39.445\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41362786   40393.344    39.447\n",
      "openf___224_> Open           buildprofile/EAO00198build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40522799   39573.047    38.646\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40523327   39573.562    38.646\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40571927   39621.023    38.692\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40585227   39634.012    38.705\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           3vat.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40597437   39645.934    38.717\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40597513   39646.008    38.717\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40597684   39646.176    38.717\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40598534   39647.008    38.718\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40599792   39648.234    38.719\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40601696   39650.094    38.721\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40601736   39650.133    38.721\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40601826   39650.219    38.721\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40604682   39653.008    38.724\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40604742   39653.070    38.724\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40604877   39653.199    38.724\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40609161   39657.383    38.728\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40609249   39657.469    38.728\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40609447   39657.664    38.728\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40615873   39663.938    38.734\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40616005   39664.066    38.734\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40616302   39664.359    38.735\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40625924   39673.754    38.744\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40626124   39673.949    38.744\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40626574   39674.391    38.745\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40641024   39688.500    38.758\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40641324   39688.793    38.759\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40641999   39689.453    38.759\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40663657   39710.602    38.780\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40664105   39711.039    38.780\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40665113   39712.023    38.781\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40697617   39743.766    38.812\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40698289   39744.422    38.813\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40699801   39745.898    38.814\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     40748557   39793.512    38.861\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     41096397   40133.199    39.193\n",
      "read_mo_297_> Segments, residues, atoms:        1      458     3548\n",
      "read_mo_298_> Segment:        1  28:A  496:A     3548\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41109650   40146.141    39.205\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     41373648   40403.953    39.457\n",
      "openf___224_> Open           ali/EAO00198.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41519038   40545.938    39.596\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41520834   40547.688    39.597\n",
      "\n",
      "Read the alignment from file       : ali/EAO00198.ali\n",
      "\n",
      "Total number of alignment positions:   458\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      3vatA     458      1    3vat.pdb MOL_ID  1; MOLECULE  ASPARTYL AMINOPEPTIDASE; CHAIN  A; SYNONYM  DNPEP; EC  3.4.\n",
      "  2   EAO00198     450      1    EAO00198\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     41522066   40548.891    39.599\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          462\n",
      "  Score                    :  303106.9375\n",
      "openf___224_> Open           ali/EAO00198-3vatA.ali\n",
      "openf___224_> Open           ali/EAO00198-3vatA.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40522799   39573.047    38.646\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40523327   39573.562    38.646\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40571927   39621.023    38.692\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     40585227   39634.012    38.705\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00198-3vatA.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40597192   39645.695    38.716\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40598642   39647.109    38.718\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40601542   39649.945    38.721\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40607342   39655.609    38.726\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40618942   39666.938    38.737\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     40859750   39902.102    38.967\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40861578   39903.883    38.969\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     40863374   39905.641    38.970\n",
      "\n",
      "Read the alignment from file       : ali/EAO00198-3vatA.ali\n",
      "\n",
      "Total number of alignment positions:   462\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      3vatA     458      1    3vat.pdb MOL_ID  1; MOLECULE  ASPARTYL AMINOPEPTIDASE; CHAIN  A; SYNONYM  DNPEP; EC  3.4.\n",
      "  2   EAO00198     450      1    EAO00198\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           3vat.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41002499   40041.504    39.103\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41002499   40041.504    39.103\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41002670   40041.672    39.103\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41003520   40042.500    39.104\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41004778   40043.727    39.105\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41006682   40045.586    39.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41006682   40045.586    39.107\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41006772   40045.676    39.107\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41009628   40048.465    39.110\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41009628   40048.465    39.110\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41009763   40048.598    39.110\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41014047   40052.781    39.114\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41014047   40052.781    39.114\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41014245   40052.973    39.114\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41020671   40059.250    39.120\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41020671   40059.250    39.120\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41020968   40059.539    39.121\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41030590   40068.938    39.130\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41030590   40068.938    39.130\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41031040   40069.375    39.130\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41045490   40083.484    39.144\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41045490   40083.484    39.144\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41046165   40084.145    39.145\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41067823   40105.297    39.165\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41067823   40105.297    39.165\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41068831   40106.281    39.166\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41101335   40138.023    39.197\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     41101335   40138.023    39.197\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41102847   40139.500    39.199\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     41151603   40187.113    39.245\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     41151180   40186.699    39.245\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    200     1  223   235      D     K   17.704\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41290432   40322.688    39.378\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41292032   40324.250    39.379\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41296012   40328.137    39.383\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41298012   40330.090    39.385\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41301012   40333.020    39.388\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41305512   40337.414    39.392\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41311084   40342.855    39.397\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41313884   40345.590    39.400\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41318084   40349.691    39.404\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41324384   40355.844    39.410\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41327156   40358.551    39.413\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     41328556   40359.918    39.414\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     41399049   40428.758    39.481\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     41742049   40763.719    39.808\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     41794169   40814.617    39.858\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      450   3481   3557      0         0      1479\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     41943417   40960.367    40.000\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      450   3481   3557   4821      5714      1479\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     42102473   41115.695    40.152\n",
      "patch_______> segment topology patched using RTF:  450 ;  SER  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      450   3482   3558   4823      5716      1480\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      450   3482   3558   4823      5716      1480\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     2984      498\n",
      "openf___224_> Open           EAO00198.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      450     3482     3482\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42209922   41220.625    40.255\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42218114   41228.625    40.262\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42234498   41244.625    40.278\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42267266   41276.625    40.309\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42300034   41308.625    40.340\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42365570   41372.625    40.403\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42398338   41404.625    40.434\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42431106   41436.625    40.465\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42496642   41500.625    40.528\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42627714   41628.625    40.653\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42693250   41692.625    40.715\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42758786   41756.625    40.778\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     42889858   41884.625    40.903\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     43152002   42140.625    41.153\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     43283074   42268.625    41.278\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     3558    4823    5458    1458\n",
      "              Total number of restraints before, now         :        0    15297\n",
      "make_re_422_> Number of previous, current restraints         :        0    15297\n",
      "make_re_423_> Number of previous, current selected restraints:        0    15297\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     43539547   42519.090    41.523\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        6  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        6  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        6  GLU\n",
      "getdata_643_> Protein accepted:  3vatA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     43670635   42647.105    41.648\n",
      "make_re_422_> Number of previous, current restraints         :    15297    15745\n",
      "make_re_423_> Number of previous, current selected restraints:    15297    15745\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     43670635   42647.105    41.648\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  3vatA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      471\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    15745    16194\n",
      "make_re_423_> Number of previous, current selected restraints:    15745    15723\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     43670635   42647.105    41.648\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  3vatA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     43932779   42903.105    41.898\n",
      "make_re_422_> Number of previous, current restraints         :    16194    16577\n",
      "make_re_423_> Number of previous, current selected restraints:    15723    16106\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     43932779   42903.105    41.898\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  3vatA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     44194923   43159.105    42.148\n",
      "make_re_422_> Number of previous, current restraints         :    16577    16860\n",
      "make_re_423_> Number of previous, current selected restraints:    16106    16389\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     44194923   43159.105    42.148\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  3vatA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    16860    16973\n",
      "make_re_423_> Number of previous, current selected restraints:    16389    16502\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     44194923   43159.105    42.148\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  3vatA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    16973    17018\n",
      "make_re_423_> Number of previous, current selected restraints:    16502    16547\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     45197703   44138.383    43.104\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     45721991   44650.383    43.604\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     45984135   44906.383    43.854\n",
      "make_re_422_> Number of previous, current restraints         :    17018    26822\n",
      "make_re_423_> Number of previous, current selected restraints:    16547    26351\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     46508423   45418.383    44.354\n",
      "make_re_422_> Number of previous, current restraints         :    26822    36511\n",
      "make_re_423_> Number of previous, current selected restraints:    26351    36040\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     47556999   46442.383    45.354\n",
      "make_re_422_> Number of previous, current restraints         :    36511    41744\n",
      "make_re_423_> Number of previous, current selected restraints:    36040    41273\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    41744    43913\n",
      "make_re_423_> Number of previous, current selected restraints:    41273    43442\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    43913    43913\n",
      "make_re_423_> Number of previous, current selected restraints:    43442    43442\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    43913    43913\n",
      "make_re_423_> Number of previous, current selected restraints:    43442    43442\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    43913    43913\n",
      "make_re_423_> Number of previous, current selected restraints:    43442    43442\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    43913    43913\n",
      "make_re_423_> Number of previous, current selected restraints:    43442    43442\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    43913    43913\n",
      "make_re_423_> Number of previous, current selected restraints:    43442    43442\n",
      "rmdupl__427_>     2732 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    43913    40710\n",
      "openf___224_> Open           EAO00198.rsr\n",
      "openf___224_> Open           EAO00198.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     47791715   46671.598    45.578\n",
      "rdcsr2__307_> Number of restraints read    :    40710\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    40710    40710\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00198.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     47791747   46671.629    45.578\n",
      "read_mo_297_> Segments, residues, atoms:        1      450     3482\n",
      "read_mo_298_> Segment:        1  1:A  450:A     3482\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3482     3482        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    20554\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    23058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    24424\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    25868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    27516\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    28693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    30389\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    32707\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    34802\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     48167887   47038.953    45.936\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    37511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    39984\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    40710\n",
      "iupac_m_486_> OE1/2 will be swapped:      153.2510       14  14\n",
      "iupac_m_486_> OE1/2 will be swapped:     -154.9754       38  38\n",
      "iupac_m_487_> NH1/2 swapped:      179.5528       43  43\n",
      "iupac_m_485_> OD1/2 will be swapped:     -177.9874       47  47\n",
      "iupac_m_487_> NH1/2 swapped:      179.4659       58  58\n",
      "iupac_m_485_> OD1/2 will be swapped:     -172.9201       65  65\n",
      "iupac_m_486_> OE1/2 will be swapped:     -149.3389       78  78\n",
      "iupac_m_487_> NH1/2 swapped:      179.0293      102  102\n",
      "iupac_m_487_> NH1/2 swapped:      178.8696      127  127\n",
      "iupac_m_487_> NH1/2 swapped:      179.6417      134  134\n",
      "iupac_m_486_> OE1/2 will be swapped:     -160.7284      143  143\n",
      "iupac_m_487_> NH1/2 swapped:     -176.6980      145  145\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3803      151  151\n",
      "iupac_m_485_> OD1/2 will be swapped:      130.2986      195  195\n",
      "iupac_m_486_> OE1/2 will be swapped:      115.5985      210  210\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.2064      217  217\n",
      "iupac_m_486_> OE1/2 will be swapped:     -110.9532      218  218\n",
      "iupac_m_486_> OE1/2 will be swapped:      145.6846      283  283\n",
      "iupac_m_486_> OE1/2 will be swapped:      157.5715      287  287\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3403      311  311\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.1242      333  333\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.8607      338  338\n",
      "iupac_m_487_> NH1/2 swapped:      178.4969      347  347\n",
      "iupac_m_487_> NH1/2 swapped:      174.8068      358  358\n",
      "iupac_m_487_> NH1/2 swapped:      177.8632      421  421\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -115.8862      443  443\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     48544227   47406.473    46.295\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     48711363   47569.691    46.455\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49045635   47896.129    46.774\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49714179   48549.004    47.411\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     51051267   49854.754    48.686\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     53725443   52466.254    51.237\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     59073795   57689.254    56.337\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   715085\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -51507.0508\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -51507.050781\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    47.556000\n",
      "Sequence length          :          450\n",
      "Compactness              :     0.213591\n",
      "Native energy (pair)     :  -275.726328\n",
      "Native energy (surface)  :   -19.803242\n",
      "Native energy (combined) :    -7.713671\n",
      "Z score (pair)           :    -7.998062\n",
      "Z score (surface)        :    -8.059435\n",
      "Z score (combined)       :   -10.630957\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7659\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2532.2007\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3558       0      0   0.006   0.006      30.363       1.000\n",
      " 2 Bond angle potential               :    4823       2     14   2.095   2.095      410.64       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2255       0     76  48.713  48.713      839.78       1.000\n",
      " 4 Stereochemical improper torsion pot:    1458       0      1   1.217   1.217      46.759       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7659       0      0   0.002   0.002      3.7995       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    9804       3     12   0.261   0.261      244.50       1.000\n",
      "10 Distance restraints 2 (N-O)        :    9689       0      9   0.309   0.309      276.30       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     449       0      3   4.169   4.169      92.010       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     383       0      1  69.797  69.797      62.439       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     283       0      0  66.404  66.404      83.508       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     113       0      0  68.379  68.379      73.108       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      45       0      0  97.938  97.938      28.826       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5233       0      0   0.407   0.407      124.35       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     448      15     33  26.602  36.915      104.17       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2169       0      0   0.597   0.597      111.65       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00198.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00198.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   40400.0117\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6262 248D 249N C   N    1943 1945  155.23  120.00   35.23    8.01  120.00   35.23    8.01\n",
      "    2   6265 249N 249N N   CA   1945 1946  123.41  107.00   16.41    4.72  107.00   16.41    4.72\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12097   3F   4G C   N      24   26  -57.49  -62.40    5.02    0.93   82.20  147.92   11.26\n",
      "    1          4G   4G N   CA     26   27  -40.13  -41.20                    8.50\n",
      "    2  12115  21K  22A C   N     168  170  -74.57  -62.50   54.84    8.48  -62.50   54.84    8.48\n",
      "    2         22A  22A N   CA    170  171   12.60  -40.90                  -40.90\n",
      "    3  12159  65D  66S C   N     529  531 -133.69  -64.10   74.65    8.21  -64.10   74.65    8.21\n",
      "    3         66S  66S N   CA    531  532   -7.99  -35.00                  -35.00\n",
      "    4  12173  79P  80A C   N     629  631   89.44   55.40   52.60    2.46  -62.50  156.87   31.44\n",
      "    4         80A  80A N   CA    631  632   -1.89   38.20                  -40.90\n",
      "    5  12232 138S 139S C   N    1082 1084  172.24 -136.60   76.37    2.53  -64.10  170.31   21.16\n",
      "    5        139S 139S N   CA   1084 1085 -152.11  151.20                  -35.00\n",
      "    6  12234 140P 141K C   N    1095 1097   71.14   56.60   96.32    8.01 -118.00  170.94    6.54\n",
      "    6        141K 141K N   CA   1097 1098  133.82   38.60                  139.10\n",
      "    7  12287 193N 194G C   N    1513 1515  128.08   78.70   52.44    2.10  -62.40 -138.49   41.46\n",
      "    7        194G 194G N   CA   1515 1516  176.23 -166.10                  -41.20\n",
      "    8  12288 194G 195D C   N    1517 1519  -50.99  -70.90   21.14    1.30  -63.30  177.26   21.16\n",
      "    8        195D 195D N   CA   1519 1520  143.17  150.30                  -40.00\n",
      "    9  12289 195D 196N C   N    1525 1527  -72.60  -71.20   14.19    0.87  -63.20  162.26   20.81\n",
      "    9        196N 196N N   CA   1527 1528  156.92  142.80                  -41.10\n",
      "   10  12290 196N 197D C   N    1533 1535  -90.09  -96.50   35.38    1.48  -63.30  173.09   23.06\n",
      "   10        197D 197D N   CA   1535 1536  149.00  114.20                  -40.00\n",
      "   11  12342 248D 249N C   N    1943 1945   90.64  -63.20  155.98   24.65  -63.20  155.98   24.65\n",
      "   11        249N 249N N   CA   1945 1946  -15.33  -41.10                  -41.10\n",
      "   12  12356 262Q 263T C   N    2047 2049  -61.26 -124.80   76.29    2.78  -63.20  143.39   18.51\n",
      "   12        263T 263T N   CA   2049 2050  101.27  143.50                  -42.10\n",
      "   13  12399 305I 306A C   N    2374 2376   55.26  -68.20  130.49   12.29  -68.20  130.49   12.29\n",
      "   13        306A 306A N   CA   2376 2377 -172.43  145.30                  145.30\n",
      "   14  12401 307S 308K C   N    2385 2387  -94.13  -62.90   74.71   11.57  -62.90   74.71   11.57\n",
      "   14        308K 308K N   CA   2387 2388 -108.67  -40.80                  -40.80\n",
      "   15  12402 308K 309T C   N    2394 2396  -85.58  -63.20   85.94    9.71  -63.20   85.94    9.71\n",
      "   15        309T 309T N   CA   2396 2397   40.87  -42.10                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   16   49  184  269  329  315  422  534  540  541\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00198.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      450     3482     3482\n",
      "openf___224_> Open           EAO00198.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     58781055   57403.375    56.058\n",
      "read_mo_297_> Segments, residues, atoms:        1      450     3482\n",
      "read_mo_298_> Segment:        1  1:A  450:A     3482\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3482     3482        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    20554\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    23058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    24424\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    25868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    27516\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    28693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    30389\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    32707\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    34802\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    37511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    39984\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    40710\n",
      "iupac_m_487_> NH1/2 swapped:      179.1619       43  43\n",
      "iupac_m_486_> OE1/2 will be swapped:      153.2389       46  46\n",
      "iupac_m_485_> OD1/2 will be swapped:      136.7251       47  47\n",
      "iupac_m_485_> OD1/2 will be swapped:     -143.7545       57  57\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.8406       78  78\n",
      "iupac_m_487_> NH1/2 swapped:      178.0614      100  100\n",
      "iupac_m_487_> NH1/2 swapped:     -175.8104      102  102\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5693      127  127\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.8246      170  170\n",
      "iupac_m_485_> OD1/2 will be swapped:      145.0350      195  195\n",
      "iupac_m_487_> NH1/2 swapped:     -176.3581      199  199\n",
      "iupac_m_487_> NH1/2 swapped:     -173.0440      246  246\n",
      "iupac_m_487_> NH1/2 swapped:     -173.7229      274  274\n",
      "iupac_m_486_> OE1/2 will be swapped:     -108.4236      287  287\n",
      "iupac_m_485_> OD1/2 will be swapped:     -129.0070      299  299\n",
      "iupac_m_485_> OD1/2 will be swapped:      137.6020      335  335\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.8797      338  338\n",
      "iupac_m_487_> NH1/2 swapped:      178.5652      347  347\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.7972      382  382\n",
      "iupac_m_485_> OD1/2 will be swapped:      161.6165      428  428\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.2274      437  437\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -105.9686      443  443\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   718970\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -51315.4375\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -51315.437500\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    47.556000\n",
      "Sequence length          :          450\n",
      "Compactness              :     0.215808\n",
      "Native energy (pair)     :  -268.161187\n",
      "Native energy (surface)  :   -20.083308\n",
      "Native energy (combined) :    -8.114413\n",
      "Z score (pair)           :    -8.915755\n",
      "Z score (surface)        :    -8.051985\n",
      "Z score (combined)       :   -11.303559\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7731\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2609.3782\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3558       0      0   0.005   0.005      29.751       1.000\n",
      " 2 Bond angle potential               :    4823       3     16   2.067   2.067      407.28       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2255       0     78  49.020  49.020      845.46       1.000\n",
      " 4 Stereochemical improper torsion pot:    1458       0      2   1.338   1.338      58.503       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7731       0      0   0.002   0.002      3.8955       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    9804       3     12   0.256   0.256      238.38       1.000\n",
      "10 Distance restraints 2 (N-O)        :    9689       1     11   0.296   0.296      278.93       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     449       1      6   5.231   5.231      144.90       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     383       0      3  73.667  73.667      88.075       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     283       0      0  65.169  65.169      91.288       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     113       0      1  68.004  68.004      71.800       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      45       0      0  86.538  86.538      30.519       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5233       0      0   0.381   0.381      118.90       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     448      14     33  24.787  33.679      78.593       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2169       0      1   0.609   0.609      123.11       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00198.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00198.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   40187.3945\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   3789  21K  22A C   N     168  170  139.84  120.00   19.84    4.51  120.00   19.84    4.51\n",
      "    2   6265 249N 249N N   CA   1945 1946  135.20  107.00   28.20    8.11  107.00   28.20    8.11\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  27482 197D 188V N   O    1535 1483   12.59    7.81    4.78    4.74    7.81    4.78    4.74\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12790 248D 248D CA  C    1938 1943 -117.73 -180.00   62.27   12.45 -180.00   62.27   12.45\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12097   3F   4G C   N      24   26  -57.04  -62.40    6.57    1.24   82.20  146.62   11.12\n",
      "    1          4G   4G N   CA     26   27  -37.40  -41.20                    8.50\n",
      "    2  12115  21K  22A C   N     168  170  -73.88  -62.50   53.42    8.27  -62.50   53.42    8.27\n",
      "    2         22A  22A N   CA    170  171   11.30  -40.90                  -40.90\n",
      "    3  12159  65D  66S C   N     529  531 -132.52  -64.10   73.63    8.06  -64.10   73.63    8.06\n",
      "    3         66S  66S N   CA    531  532   -7.79  -35.00                  -35.00\n",
      "    4  12173  79P  80A C   N     629  631   85.32   55.40   50.43    2.13  -62.50  152.75   30.63\n",
      "    4         80A  80A N   CA    631  632   -2.39   38.20                  -40.90\n",
      "    5  12232 138S 139S C   N    1082 1084 -164.56 -136.60   58.89    2.27  -64.10  158.01   18.69\n",
      "    5        139S 139S N   CA   1084 1085 -156.97  151.20                  -35.00\n",
      "    6  12233 139S 140P C   N    1088 1090  -67.21  -58.70   10.07    0.88  -64.50  172.34   13.00\n",
      "    6        140P 140P N   CA   1090 1091  -25.12  -30.50                  147.20\n",
      "    7  12286 192L 193N C   N    1505 1507 -105.38 -119.90   46.45    1.75  -63.20  140.46   15.63\n",
      "    7        193N 193N N   CA   1507 1508   92.88  137.00                  -41.10\n",
      "    8  12288 194G 195D C   N    1517 1519   40.03  -70.90  119.97   10.03  -63.30  161.40   17.67\n",
      "    8        195D 195D N   CA   1519 1520 -163.98  150.30                  -40.00\n",
      "    9  12289 195D 196N C   N    1525 1527 -129.87 -119.90   19.34    1.04  -63.20  174.74   19.05\n",
      "    9        196N 196N N   CA   1527 1528  120.42  137.00                  -41.10\n",
      "   10  12290 196N 197D C   N    1533 1535 -110.10  -96.50   47.82    1.99  -63.30  118.03   12.59\n",
      "   10        197D 197D N   CA   1535 1536   68.36  114.20                  -40.00\n",
      "   11  12342 248D 249N C   N    1943 1945  -47.40  -63.20   98.11   13.16  -63.20   98.11   13.16\n",
      "   11        249N 249N N   CA   1945 1946   55.73  -41.10                  -41.10\n",
      "   12  12356 262Q 263T C   N    2047 2049  -60.99 -124.80   76.27    2.80  -63.20  143.85   18.59\n",
      "   12        263T 263T N   CA   2049 2050  101.73  143.50                  -42.10\n",
      "   13  12402 308K 309T C   N    2394 2396 -142.24 -124.80  109.17    5.20  -63.20  110.93   12.21\n",
      "   13        309T 309T N   CA   2396 2397   35.73  143.50                  -42.10\n",
      "   14  12410 316A 317N C   N    2452 2454  -97.93  -63.20   81.66    8.83  -63.20   81.66    8.83\n",
      "   14        317N 317N N   CA   2454 2455   32.81  -41.10                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   19   60  189  261  354  318  447  533  537  577\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00198.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      450     3482     3482\n",
      "openf___224_> Open           EAO00198.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     58781055   57403.375    56.058\n",
      "read_mo_297_> Segments, residues, atoms:        1      450     3482\n",
      "read_mo_298_> Segment:        1  1:A  450:A     3482\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3482     3482        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    20554\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    23058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    24424\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    25868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    27516\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    28693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    30389\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    32707\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    34802\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    37511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    39984\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    40710\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      101.6833        3  3\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      159.6512       14  14\n",
      "iupac_m_486_> OE1/2 will be swapped:      120.8028       38  38\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7805       43  43\n",
      "iupac_m_485_> OD1/2 will be swapped:      131.8772       47  47\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.3693       59  59\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.7556      102  102\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.3543      107  107\n",
      "iupac_m_485_> OD1/2 will be swapped:     -104.5936      126  126\n",
      "iupac_m_487_> NH1/2 swapped:      179.8012      151  151\n",
      "iupac_m_486_> OE1/2 will be swapped:      132.7074      168  168\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.1749      187  187\n",
      "iupac_m_485_> OD1/2 will be swapped:      169.0028      195  195\n",
      "iupac_m_485_> OD1/2 will be swapped:     -168.0709      197  197\n",
      "iupac_m_487_> NH1/2 swapped:      178.7150      199  199\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.8265      210  210\n",
      "iupac_m_486_> OE1/2 will be swapped:      144.5051      240  240\n",
      "iupac_m_487_> NH1/2 swapped:      175.2288      246  246\n",
      "iupac_m_485_> OD1/2 will be swapped:     -167.3528      264  264\n",
      "iupac_m_487_> NH1/2 swapped:      174.9908      274  274\n",
      "iupac_m_487_> NH1/2 swapped:     -177.7139      311  311\n",
      "iupac_m_487_> NH1/2 swapped:     -174.5917      342  342\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.8878      375  375\n",
      "iupac_m_487_> NH1/2 swapped:      172.1647      421  421\n",
      "iupac_m_485_> OD1/2 will be swapped:     -159.8105      428  428\n",
      "iupac_m_486_> OE1/2 will be swapped:     -161.6947      437  437\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -98.6838      443  443\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   717443\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -51178.8125\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -51178.812500\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    47.556000\n",
      "Sequence length          :          450\n",
      "Compactness              :     0.212683\n",
      "Native energy (pair)     :  -272.915451\n",
      "Native energy (surface)  :   -20.928826\n",
      "Native energy (combined) :    -8.501134\n",
      "Z score (pair)           :    -9.093685\n",
      "Z score (surface)        :    -8.282096\n",
      "Z score (combined)       :   -11.846745\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7705\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2788.4448\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3558       0      0   0.005   0.005      30.370       1.000\n",
      " 2 Bond angle potential               :    4823       2     18   2.174   2.174      440.43       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2255       0     79  49.020  49.020      854.18       1.000\n",
      " 4 Stereochemical improper torsion pot:    1458       0      1   1.304   1.304      52.628       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7705       0      0   0.002   0.002      2.4401       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    9804       3     11   0.263   0.263      267.28       1.000\n",
      "10 Distance restraints 2 (N-O)        :    9689       3     24   0.327   0.327      383.94       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     449       0      2   4.360   4.360      100.66       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     383       0      1  72.643  72.643      88.216       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     283       0      3  67.766  67.766      94.783       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     113       0      0  64.458  64.458      67.168       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      45       0      0  99.334  99.334      31.901       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5233       0      0   0.392   0.392      111.68       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     448      24     34  28.062  41.899      136.05       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2169       0      1   0.624   0.624      126.72       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00198.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00198.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   42498.1758\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6262 248D 249N C   N    1943 1945  154.99  120.00   34.99    7.95  120.00   34.99    7.95\n",
      "    2   6265 249N 249N N   CA   1945 1946  123.51  107.00   16.51    4.75  107.00   16.51    4.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12095   1M   2S C   N       7    9 -135.48 -136.60   25.44    1.31  -64.10  164.66   17.04\n",
      "    1          2S   2S N   CA      9   10  176.62  151.20                  -35.00\n",
      "    2  12096   2S   3F C   N      13   15   75.55   58.10   72.56    4.40  -63.20  138.92   22.79\n",
      "    2          3F   3F N   CA     15   16  -37.53   32.90                  -44.30\n",
      "    3  12097   3F   4G C   N      24   26  -74.33  -62.40   12.32    2.30   82.20  165.19   12.55\n",
      "    3          4G   4G N   CA     26   27  -44.27  -41.20                    8.50\n",
      "    4  12115  21K  22A C   N     168  170  -74.78  -62.50   55.18    8.53  -62.50   55.18    8.53\n",
      "    4         22A  22A N   CA    170  171   12.90  -40.90                  -40.90\n",
      "    5  12142  48E  49P C   N     385  387  -69.04  -58.70   13.26    1.04  -64.50  169.46   12.89\n",
      "    5         49P  49P N   CA    387  388  -22.21  -30.50                  147.20\n",
      "    6  12143  49P  50W C   N     392  394   88.84 -124.90  149.41    7.33 -124.90  149.41    7.33\n",
      "    6         50W  50W N   CA    394  395  112.87  143.40                  143.40\n",
      "    7  12144  50W  51P C   N     406  408  -61.41  -58.70    4.19    0.55  -64.50  179.13   13.53\n",
      "    7         51P  51P N   CA    408  409  -33.70  -30.50                  147.20\n",
      "    8  12159  65D  66S C   N     529  531 -133.48  -64.10   74.56    8.17  -64.10   74.56    8.17\n",
      "    8         66S  66S N   CA    531  532   -7.68  -35.00                  -35.00\n",
      "    9  12172  78E  79P C   N     622  624  -40.44  -58.70   30.66    1.90  -64.50  159.50   13.20\n",
      "    9         79P  79P N   CA    624  625  -55.12  -30.50                  147.20\n",
      "   10  12232 138S 139S C   N    1082 1084  143.84 -136.60   98.08    3.00  -64.10 -168.48   24.56\n",
      "   10        139S 139S N   CA   1084 1085 -151.45  151.20                  -35.00\n",
      "   11  12234 140P 141K C   N    1095 1097   64.71   56.60   96.37    7.51 -118.00  177.34    6.77\n",
      "   11        141K 141K N   CA   1097 1098  134.62   38.60                  139.10\n",
      "   12  12267 173A 174P C   N    1364 1366  -71.57  -58.70   13.84    1.42  -64.50  172.77   13.27\n",
      "   12        174P 174P N   CA   1366 1367  -25.42  -30.50                  147.20\n",
      "   13  12268 174P 175N C   N    1371 1373   52.34   55.90   39.95    2.54  -71.20  138.91    8.29\n",
      "   13        175N 175N N   CA   1373 1374   79.29   39.50                  142.80\n",
      "   14  12286 192L 193N C   N    1505 1507  -75.81 -119.90   47.34    2.35  -63.20  165.14   21.35\n",
      "   14        193N 193N N   CA   1507 1508  154.24  137.00                  -41.10\n",
      "   15  12288 194G 195D C   N    1517 1519 -106.06  -96.50   27.82    1.17  -63.30 -175.30   25.54\n",
      "   15        195D 195D N   CA   1519 1520  140.32  114.20                  -40.00\n",
      "   16  12289 195D 196N C   N    1525 1527 -117.04 -119.90   16.73    0.69  -63.20  170.35   18.87\n",
      "   16        196N 196N N   CA   1527 1528  120.52  137.00                  -41.10\n",
      "   17  12290 196N 197D C   N    1533 1535 -119.33  -96.50   47.92    2.02  -63.30  172.99   24.97\n",
      "   17        197D 197D N   CA   1535 1536  156.34  114.20                  -40.00\n",
      "   18  12342 248D 249N C   N    1943 1945   92.62  -63.20  158.00   24.97  -63.20  158.00   24.97\n",
      "   18        249N 249N N   CA   1945 1946  -14.96  -41.10                  -41.10\n",
      "   19  12356 262Q 263T C   N    2047 2049  -58.23 -124.80   76.82    2.93  -63.20  147.35   19.22\n",
      "   19        263T 263T N   CA   2049 2050  105.17  143.50                  -42.10\n",
      "   20  12399 305I 306A C   N    2374 2376   51.05  -68.20  125.29   11.75  -68.20  125.29   11.75\n",
      "   20        306A 306A N   CA   2376 2377 -176.26  145.30                  145.30\n",
      "   21  12401 307S 308K C   N    2385 2387  -62.03  -70.20   75.83    5.68  -62.90  103.42   13.38\n",
      "   21        308K 308K N   CA   2387 2388 -144.22  140.40                  -40.80\n",
      "   22  12402 308K 309T C   N    2394 2396  -90.92 -124.80   96.43    3.23  -63.20   99.27   11.13\n",
      "   22        309T 309T N   CA   2396 2397   53.22  143.50                  -42.10\n",
      "   23  12426 332N 333Y C   N    2571 2573  -79.44  -63.50   46.39    6.64  -63.50   46.39    6.64\n",
      "   23        333Y 333Y N   CA   2573 2574    0.17  -43.40                  -43.40\n",
      "   24  12449 355A 356N C   N    2770 2772  -78.53  -63.20   70.90    8.11  -63.20   70.90    8.11\n",
      "   24        356N 356N N   CA   2772 2773   28.12  -41.10                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   10   51  159  250  347  327  431  525  578  573\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00198.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      450     3482     3482\n",
      "openf___224_> Open           EAO00198.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     58781055   57403.375    56.058\n",
      "read_mo_297_> Segments, residues, atoms:        1      450     3482\n",
      "read_mo_298_> Segment:        1  1:A  450:A     3482\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3482     3482        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    20554\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    23058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    24424\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    25868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    27516\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    28693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    30389\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    32707\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    34802\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    37511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    39984\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    40710\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.8275        3  3\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -118.5571       10  10\n",
      "iupac_m_486_> OE1/2 will be swapped:     -124.5276       14  14\n",
      "iupac_m_487_> NH1/2 swapped:      178.2544       63  63\n",
      "iupac_m_485_> OD1/2 will be swapped:      103.3803       65  65\n",
      "iupac_m_487_> NH1/2 swapped:      179.1494      100  100\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3716      102  102\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.7077      104  104\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.5987      116  116\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.1706      127  127\n",
      "iupac_m_487_> NH1/2 swapped:      173.8714      151  151\n",
      "iupac_m_486_> OE1/2 will be swapped:      149.5387      168  168\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.3958      195  195\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.4510      197  197\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.6689      217  217\n",
      "iupac_m_487_> NH1/2 swapped:     -173.0436      246  246\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.6971      264  264\n",
      "iupac_m_485_> OD1/2 will be swapped:      145.0831      268  268\n",
      "iupac_m_487_> NH1/2 swapped:     -177.6936      274  274\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.7271      338  338\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.1979      339  339\n",
      "iupac_m_487_> NH1/2 swapped:      176.2115      347  347\n",
      "iupac_m_487_> NH1/2 swapped:      178.6419      358  358\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2925      406  406\n",
      "iupac_m_486_> OE1/2 will be swapped:      -99.6864      422  422\n",
      "iupac_m_485_> OD1/2 will be swapped:      142.9844      428  428\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.6073      437  437\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -117.1480      443  443\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   714371\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -51569.1641\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -51569.164062\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    47.556000\n",
      "Sequence length          :          450\n",
      "Compactness              :     0.213483\n",
      "Native energy (pair)     :  -237.105583\n",
      "Native energy (surface)  :   -20.458861\n",
      "Native energy (combined) :    -7.707770\n",
      "Z score (pair)           :    -9.013592\n",
      "Z score (surface)        :    -8.100793\n",
      "Z score (combined)       :   -11.748560\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7543\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2640.1277\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3558       0      0   0.006   0.006      30.418       1.000\n",
      " 2 Bond angle potential               :    4823       2     10   2.111   2.111      416.98       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2255       0     81  49.284  49.284      854.96       1.000\n",
      " 4 Stereochemical improper torsion pot:    1458       0      1   1.302   1.302      52.526       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7543       0      0   0.002   0.002      3.3837       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    9804       3     10   0.253   0.253      244.13       1.000\n",
      "10 Distance restraints 2 (N-O)        :    9689       0     12   0.297   0.297      304.57       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     449       0      3   4.165   4.165      91.859       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     383       0      4  68.034  68.034      68.798       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     283       0      0  68.862  68.862      93.917       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     113       0      1  59.685  59.685      65.372       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      45       0      0  92.466  92.466      29.146       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5233       0      0   0.386   0.386      111.10       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     448      19     38  28.326  39.375      138.72       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2169       0      1   0.649   0.649      134.24       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00198.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00198.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   40579.7109\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6262 248D 249N C   N    1943 1945  155.58  120.00   35.58    8.08  120.00   35.58    8.08\n",
      "    2   6265 249N 249N N   CA   1945 1946  122.88  107.00   15.88    4.57  107.00   15.88    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12095   1M   2S C   N       7    9   71.44   56.90   43.48    1.82  -64.10  138.91   18.50\n",
      "    1          2S   2S N   CA      9   10   -4.58   36.40                  -35.00\n",
      "    2  12096   2S   3F C   N      13   15 -117.87 -124.20   14.04    0.48  -63.20 -176.59   22.14\n",
      "    2          3F   3F N   CA     15   16  130.77  143.30                  -44.30\n",
      "    3  12115  21K  22A C   N     168  170  -74.03  -62.50   55.37    8.58  -62.50   55.37    8.58\n",
      "    3         22A  22A N   CA    170  171   13.26  -40.90                  -40.90\n",
      "    4  12144  50W  51P C   N     406  408  -52.38  -58.70    9.21    0.63  -64.50  176.03   13.78\n",
      "    4         51P  51P N   CA    408  409  -37.19  -30.50                  147.20\n",
      "    5  12173  79P  80A C   N     629  631   87.61   55.40   52.71    2.30  -62.50  154.69   30.98\n",
      "    5         80A  80A N   CA    631  632   -3.52   38.20                  -40.90\n",
      "    6  12199 105K 106G C   N     820  822   59.73   78.70   30.08    0.57   82.20  152.92    8.54\n",
      "    6        106G 106G N   CA    822  823 -142.76 -166.10                    8.50\n",
      "    7  12200 106G 107E C   N     824  826  -77.19  -63.60   21.65    2.53   54.60  147.33   19.30\n",
      "    7        107E 107E N   CA    826  827  -23.46  -40.30                   42.40\n",
      "    8  12232 138S 139S C   N    1082 1084  160.32 -136.60   85.25    2.70  -64.10  178.74   22.56\n",
      "    8        139S 139S N   CA   1084 1085 -151.47  151.20                  -35.00\n",
      "    9  12287 193N 194G C   N    1513 1515   95.85   82.20   16.56    1.29  -62.40  168.92   31.81\n",
      "    9        194G 194G N   CA   1515 1516   17.88    8.50                  -41.20\n",
      "   10  12288 194G 195D C   N    1517 1519  -70.80  -96.50   26.34    1.07  -63.30  160.19   19.33\n",
      "   10        195D 195D N   CA   1519 1520  120.02  114.20                  -40.00\n",
      "   11  12290 196N 197D C   N    1533 1535   70.67  -63.30  170.54   28.85  -63.30  170.54   28.85\n",
      "   11        197D 197D N   CA   1535 1536   65.53  -40.00                  -40.00\n",
      "   12  12342 248D 249N C   N    1943 1945   84.92  -63.20  151.23   24.08  -63.20  151.23   24.08\n",
      "   12        249N 249N N   CA   1945 1946  -10.58  -41.10                  -41.10\n",
      "   13  12356 262Q 263T C   N    2047 2049  -62.65 -124.80   73.88    2.72  -63.20  145.66   18.71\n",
      "   13        263T 263T N   CA   2049 2050  103.56  143.50                  -42.10\n",
      "   14  12398 304I 305I C   N    2366 2368 -103.17 -120.60   88.66    4.41  -63.40   95.63   14.62\n",
      "   14        305I 305I N   CA   2368 2369   43.37  130.30                  -43.60\n",
      "   15  12401 307S 308K C   N    2385 2387  -69.91  -70.20   86.69    6.29  -62.90   92.37   12.39\n",
      "   15        308K 308K N   CA   2387 2388 -132.91  140.40                  -40.80\n",
      "   16  12402 308K 309T C   N    2394 2396  -96.99  -63.20   94.52   10.30  -63.20   94.52   10.30\n",
      "   16        309T 309T N   CA   2396 2397   46.18  -42.10                  -42.10\n",
      "   17  12424 330H 331P C   N    2556 2558  -70.35  -64.50   17.33    1.61  -58.70  161.80   12.45\n",
      "   17        331P 331P N   CA   2558 2559  130.88  147.20                  -30.50\n",
      "   18  12425 331P 332N C   N    2563 2565  114.33  -63.20  178.16   26.28  -63.20  178.16   26.28\n",
      "   18        332N 332N N   CA   2565 2566  -56.12  -41.10                  -41.10\n",
      "   19  12449 355A 356N C   N    2770 2772  -76.08  -63.20   65.48    7.55  -63.20   65.48    7.55\n",
      "   19        356N 356N N   CA   2772 2773   23.10  -41.10                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    9   50  172  251  337  289  397  543  539  536\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00198.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      450     3482     3482\n",
      "openf___224_> Open           EAO00198.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     58781055   57403.375    56.058\n",
      "read_mo_297_> Segments, residues, atoms:        1      450     3482\n",
      "read_mo_298_> Segment:        1  1:A  450:A     3482\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3482     3482        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    20554\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    23058\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    24424\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    25868\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    27516\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    28693\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    30389\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    32707\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    34802\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    37511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    39984\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3482    40710\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0614        3  3\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2169\n",
      "iupac_m_486_> OE1/2 will be swapped:      -92.9627       38  38\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.1218       48  48\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.2935       59  59\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.8458       60  60\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -151.2146       78  78\n",
      "iupac_m_485_> OD1/2 will be swapped:      106.0521      104  104\n",
      "iupac_m_485_> OD1/2 will be swapped:      158.6119      126  126\n",
      "iupac_m_487_> NH1/2 swapped:      177.1544      127  127\n",
      "iupac_m_487_> NH1/2 swapped:      177.8851      134  134\n",
      "iupac_m_487_> NH1/2 swapped:     -176.2495      145  145\n",
      "iupac_m_487_> NH1/2 swapped:      179.9184      151  151\n",
      "iupac_m_486_> OE1/2 will be swapped:      102.8615      168  168\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.4918      187  187\n",
      "iupac_m_485_> OD1/2 will be swapped:     -176.5856      195  195\n",
      "iupac_m_486_> OE1/2 will be swapped:     -104.8738      217  217\n",
      "iupac_m_485_> OD1/2 will be swapped:     -164.9199      221  221\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.8038      223  223\n",
      "iupac_m_485_> OD1/2 will be swapped:      128.1931      239  239\n",
      "iupac_m_486_> OE1/2 will be swapped:      127.2505      240  240\n",
      "iupac_m_485_> OD1/2 will be swapped:      129.2553      268  268\n",
      "iupac_m_487_> NH1/2 swapped:     -174.4596      274  274\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.6960      338  338\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6014      347  347\n",
      "iupac_m_487_> NH1/2 swapped:     -174.9700      358  358\n",
      "iupac_m_485_> OD1/2 will be swapped:      111.3006      428  428\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -99.2705      431  431\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.3048      437  437\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -115.6788      443  443\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   714019\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -51667.7070\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -51667.707031\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    47.556000\n",
      "Sequence length          :          450\n",
      "Compactness              :     0.213507\n",
      "Native energy (pair)     :  -255.151992\n",
      "Native energy (surface)  :   -18.927451\n",
      "Native energy (combined) :    -7.879782\n",
      "Z score (pair)           :    -8.951094\n",
      "Z score (surface)        :    -7.903492\n",
      "Z score (combined)       :   -11.277540\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      450\n",
      "Number of all, selected real atoms                :     3482    3482\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    40710   40710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7538\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2683.5410\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3558       0      0   0.006   0.006      31.022       1.000\n",
      " 2 Bond angle potential               :    4823       1     18   2.170   2.170      437.44       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2255       0     77  49.180  49.180      851.17       1.000\n",
      " 4 Stereochemical improper torsion pot:    1458       0      2   1.275   1.275      51.117       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7538       0      0   0.002   0.002      3.0173       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    9804       3     12   0.256   0.256      246.75       1.000\n",
      "10 Distance restraints 2 (N-O)        :    9689       3     17   0.294   0.294      322.02       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     449       0      3   4.162   4.162      91.720       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     383       0      5  65.843  65.843      76.820       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     283       0      1  69.673  69.673      92.765       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     113       0      0  61.484  61.484      70.570       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      45       0      0  87.793  87.793      28.087       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    5233       0      0   0.395   0.395      126.21       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     448      18     33  25.479  37.187      106.75       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2169       0      3   0.661   0.661      148.09       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00198.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00198.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   40510.9844\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6262 248D 249N C   N    1943 1945  160.85  120.00   40.85    9.28  120.00   40.85    9.28\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12097   3F   4G C   N      24   26  -59.16  -62.40    3.33    0.62   82.20  149.59   11.38\n",
      "    1          4G   4G N   CA     26   27  -40.41  -41.20                    8.50\n",
      "    2  12115  21K  22A C   N     168  170  -74.44  -62.50   55.77    8.63  -62.50   55.77    8.63\n",
      "    2         22A  22A N   CA    170  171   13.58  -40.90                  -40.90\n",
      "    3  12144  50W  51P C   N     406  408  -53.84  -58.70    8.37    0.51  -64.50  175.81   13.69\n",
      "    3         51P  51P N   CA    408  409  -37.31  -30.50                  147.20\n",
      "    4  12169  75G  76K C   N     593  595 -104.67  -62.90   85.27    9.80  -62.90   85.27    9.80\n",
      "    4         76K  76K N   CA    595  596   33.54  -40.80                  -40.80\n",
      "    5  12172  78E  79P C   N     622  624  -48.56  -58.70   23.97    1.40  -64.50  161.38   12.90\n",
      "    5         79P  79P N   CA    624  625  -52.21  -30.50                  147.20\n",
      "    6  12232 138S 139S C   N    1082 1084 -166.67 -136.60   53.63    1.94  -64.10  165.12   19.36\n",
      "    6        139S 139S N   CA   1084 1085 -164.40  151.20                  -35.00\n",
      "    7  12233 139S 140P C   N    1088 1090  -66.80  -58.70    9.94    0.83  -64.50  171.95   12.95\n",
      "    7        140P 140P N   CA   1090 1091  -24.74  -30.50                  147.20\n",
      "    8  12287 193N 194G C   N    1513 1515   89.39   78.70   12.68    0.40  -62.40 -159.03   28.87\n",
      "    8        194G 194G N   CA   1515 1516 -172.92 -166.10                  -41.20\n",
      "    9  12288 194G 195D C   N    1517 1519   51.04   54.50   10.58    0.55  -63.30  146.06   24.69\n",
      "    9        195D 195D N   CA   1519 1520   50.89   40.90                  -40.00\n",
      "   10  12289 195D 196N C   N    1525 1527  -64.51  -71.20    6.73    0.47  -63.20  176.78   22.17\n",
      "   10        196N 196N N   CA   1527 1528  142.13  142.80                  -41.10\n",
      "   11  12290 196N 197D C   N    1533 1535 -108.11  -96.50   35.73    1.51  -63.30  177.75   24.82\n",
      "   11        197D 197D N   CA   1535 1536  147.99  114.20                  -40.00\n",
      "   12  12342 248D 249N C   N    1943 1945   81.79  -63.20  148.79   23.79  -63.20  148.79   23.79\n",
      "   12        249N 249N N   CA   1945 1946   -7.66  -41.10                  -41.10\n",
      "   13  12356 262Q 263T C   N    2047 2049  -61.54 -124.80   75.19    2.77  -63.20  144.97   18.70\n",
      "   13        263T 263T N   CA   2049 2050  102.86  143.50                  -42.10\n",
      "   14  12401 307S 308K C   N    2385 2387  -69.33  -70.20   88.53    6.43  -62.90   90.50   12.11\n",
      "   14        308K 308K N   CA   2387 2388 -131.07  140.40                  -40.80\n",
      "   15  12402 308K 309T C   N    2394 2396  -95.27 -124.80   95.48    3.28  -63.20  100.08   11.04\n",
      "   15        309T 309T N   CA   2396 2397   52.70  143.50                  -42.10\n",
      "   16  12426 332N 333Y C   N    2571 2573  -79.52  -63.50   47.22    6.76  -63.50   47.22    6.76\n",
      "   16        333Y 333Y N   CA   2573 2574    1.02  -43.40                  -43.40\n",
      "   17  12541 447V 448I C   N    3460 3462  -94.60 -120.60   70.18    3.10  -63.40  113.10   17.59\n",
      "   17        448I 448I N   CA   3462 3463   65.11  130.30                  -43.60\n",
      "   18  12542 448I 449S C   N    3468 3470 -156.80  -64.10   97.99   11.02  -64.10   97.99   11.02\n",
      "   18        449S 449S N   CA   3470 3471   -3.25  -35.00                  -35.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   11   42  161  246  331  296  446  521  540  555\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00198.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      450     3482     3482\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00198.B99990001.pdb        2532.20068   -51507.05078        1.00000\n",
      "EAO00198.B99990002.pdb        2609.37817   -51315.43750        1.00000\n",
      "EAO00198.B99990003.pdb        2788.44482   -51178.81250        1.00000\n",
      "EAO00198.B99990004.pdb        2640.12769   -51569.16406        1.00000\n",
      "EAO00198.B99990005.pdb        2683.54102   -51667.70703        1.00000\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     58972397   57590.230    56.240\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     58972925   57590.746    56.241\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     59021525   57638.207    56.287\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     59034825   57651.195    56.300\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59304091   57914.152    56.557\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     62865214   61391.812    59.953\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     62867456   61394.000    59.955\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     62868906   61395.414    59.956\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     62871806   61398.250    59.959\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     62877606   61403.914    59.965\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     76316005   74527.352    72.781\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     76327605   74538.680    72.792\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     76350805   74561.336    72.814\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     76397205   74606.648    72.858\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     76490005   74697.273    72.947\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     83209205   81258.992    79.354\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00199.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59306411   57916.418    56.559\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59307861   57917.832    56.560\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59310761   57920.664    56.563\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59316561   57926.328    56.569\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59328161   57937.656    56.580\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59330465   57939.906    56.582\n",
      "\n",
      "Read the alignment from file       : ali/EAO00199.ali\n",
      "\n",
      "Total number of alignment positions:   577\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00199     577      1    EAO00199\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     59333106   57942.484    56.584\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59658281   58260.039    56.895\n",
      "profile_iteration_> processing sequence:       1    577      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59659359   58261.094    56.896\n",
      "profile_iteration_> processing sequence:    6720    577      1     5.6093750     0.0008347     0.0000032  1198\n",
      "profile_iteration_> processing sequence:   13439    577      1    11.0312500     0.0008208     0.0000032  1218\n",
      "profile_iteration_> processing sequence:   20158    577      1    16.4218750     0.0008147     0.0000032  1228\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59659734   58261.461    56.896\n",
      "profile_iteration_> processing sequence:   26877    577      1    21.8906250     0.0008145     0.0000032  1228\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59659970   58261.688    56.896\n",
      "profile_iteration_> processing sequence:   33596    577      1    27.3750000     0.0008148     0.0000032  1227\n",
      "profile_iteration_> processing sequence:   40315    577      1    32.7812500     0.0008131     0.0000032  1230\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59662920   58264.570    56.899\n",
      "profile_iteration_> processing sequence:   47034    577      1    38.2968750     0.0008142     0.0000032  1228\n",
      "profile_iteration_> processing sequence:   53753    577      1    43.9062500     0.0008168     0.0000032  1224\n",
      "profile_iteration_> processing sequence:   60472    577      1    49.3593750     0.0008162     0.0000032  1225\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     59666430   58268.000    56.902\n",
      "profile_iteration_> processing sequence:   67191    577      1    55.0312500     0.0008190     0.0000032  1221\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    29.31267\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    262.1087\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0175,      121,        0.0092,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00015   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00060   0.00016\n",
      "Z:         1  -2.25000   0.00030   0.00055\n",
      "Z:         1  -2.15000   0.00164   0.00163\n",
      "Z:         1  -2.05000   0.00476   0.00416\n",
      "Z:         1  -1.95000   0.00923   0.00933\n",
      "Z:         1  -1.85000   0.01607   0.01872\n",
      "Z:         1  -1.75000   0.03170   0.03400\n",
      "Z:         1  -1.65000   0.05417   0.05658\n",
      "Z:         1  -1.55000   0.07471   0.08721\n",
      "Z:         1  -1.45000   0.13231   0.12565\n",
      "Z:         1  -1.35000   0.15939   0.17059\n",
      "Z:         1  -1.25000   0.21759   0.21981\n",
      "Z:         1  -1.15000   0.25375   0.27052\n",
      "Z:         1  -1.05000   0.31120   0.31973\n",
      "Z:         1  -0.95000   0.34424   0.36469\n",
      "Z:         1  -0.85000   0.40466   0.40316\n",
      "Z:         1  -0.75000   0.45318   0.43360\n",
      "Z:         1  -0.65000   0.48696   0.45519\n",
      "Z:         1  -0.55000   0.46047   0.46779\n",
      "Z:         1  -0.45000   0.49917   0.47182\n",
      "Z:         1  -0.35000   0.46523   0.46811\n",
      "Z:         1  -0.25000   0.46300   0.45775\n",
      "Z:         1  -0.15000   0.43725   0.44196\n",
      "Z:         1  -0.05000   0.43026   0.42194\n",
      "Z:         1   0.05000   0.40913   0.39889\n",
      "Z:         1   0.15000   0.35570   0.37383\n",
      "Z:         1   0.25000   0.35153   0.34769\n",
      "Z:         1   0.35000   0.30599   0.32121\n",
      "Z:         1   0.45000   0.29453   0.29500\n",
      "Z:         1   0.55000   0.26238   0.26953\n",
      "Z:         1   0.65000   0.24184   0.24513\n",
      "Z:         1   0.75000   0.21267   0.22205\n",
      "Z:         1   0.85000   0.19720   0.20043\n",
      "Z:         1   0.95000   0.17309   0.18036\n",
      "Z:         1   1.05000   0.15552   0.16185\n",
      "Z:         1   1.15000   0.13380   0.14489\n",
      "Z:         1   1.25000   0.12710   0.12944\n",
      "Z:         1   1.35000   0.11296   0.11542\n",
      "Z:         1   1.45000   0.10596   0.10275\n",
      "Z:         1   1.55000   0.08915   0.09133\n",
      "Z:         1   1.65000   0.08349   0.08109\n",
      "Z:         1   1.75000   0.07501   0.07191\n",
      "Z:         1   1.85000   0.06995   0.06371\n",
      "Z:         1   1.95000   0.05462   0.05639\n",
      "Z:         1   2.05000   0.04896   0.04988\n",
      "Z:         1   2.15000   0.04509   0.04409\n",
      "Z:         1   2.25000   0.04197   0.03895\n",
      "Z:         1   2.35000   0.03230   0.03439\n",
      "Z:         1   2.45000   0.03378   0.03035\n",
      "Z:         1   2.55000   0.02575   0.02678\n",
      "Z:         1   2.65000   0.02560   0.02361\n",
      "Z:         1   2.75000   0.02337   0.02082\n",
      "Z:         1   2.85000   0.02218   0.01835\n",
      "Z:         1   2.95000   0.01890   0.01617\n",
      "Z:         1   3.05000   0.01741   0.01424\n",
      "Z:         1   3.15000   0.01325   0.01255\n",
      "Z:         1   3.25000   0.01444   0.01105\n",
      "Z:         1   3.35000   0.01191   0.00973\n",
      "Z:         1   3.45000   0.00744   0.00857\n",
      "Z:         1   3.55000   0.00938   0.00754\n",
      "Z:         1   3.65000   0.00655   0.00664\n",
      "Z:         1   3.75000   0.00744   0.00584\n",
      "Z:         1   3.85000   0.00699   0.00514\n",
      "Z:         1   3.95000   0.00729   0.00453\n",
      "Z:         1   4.05000   0.00476   0.00398\n",
      "Z:         1   4.15000   0.00372   0.00350\n",
      "Z:         1   4.25000   0.00357   0.00308\n",
      "Z:         1   4.35000   0.00506   0.00271\n",
      "Z:         1   4.45000   0.00283   0.00239\n",
      "Z:         1   4.55000   0.00298   0.00210\n",
      "Z:         1   4.65000   0.00208   0.00185\n",
      "Z:         1   4.75000   0.00223   0.00163\n",
      "Z:         1   4.85000   0.00060   0.00143\n",
      "Z:         1   4.95000   0.00149   0.00126\n",
      "Z:         1   5.05000   0.00134   0.00111\n",
      "Z:         1   5.15000   0.00149   0.00097\n",
      "Z:         1   5.25000   0.00104   0.00086\n",
      "Z:         1   5.35000   0.00119   0.00075\n",
      "Z:         1   5.45000   0.00193   0.00066\n",
      "Z:         1   5.55000   0.00045   0.00058\n",
      "Z:         1   5.65000   0.00134   0.00051\n",
      "Z:         1   5.75000   0.00030   0.00045\n",
      "Z:         1   5.85000   0.00060   0.00040\n",
      "Z:         1   5.95000   0.00015   0.00035\n",
      "Z:         1   6.05000   0.00060   0.00031\n",
      "Z:         1   6.15000   0.00045   0.00027\n",
      "Z:         1   6.25000   0.00104   0.00024\n",
      "Z:         1   6.35000   0.00030   0.00021\n",
      "Z:         1   6.45000   0.00045   0.00018\n",
      "Z:         1   6.55000   0.00060   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00045   0.00013\n",
      "Z:         1   6.85000   0.00015   0.00011\n",
      "Z:         1   6.95000   0.00045   0.00010\n",
      "Z:         1   7.05000   0.00015   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00030   0.00007\n",
      "Z:         1   7.35000   0.00015   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     60145030   58735.383    57.359\n",
      "> 1x88A                      1    2754   21850     332     577   37.69     0.0           2   308    11   343     2   330\n",
      "> 4ubfA                      1    2828   13000     309     577   41.26     0.0           3   220    51   334    82   304\n",
      "> 5wdeA                      1    3567   20050     320     577   39.79     0.0           4   284    31   335    28   316\n",
      "> 5wdhA                      1    3577   15300     298     577   38.96     0.0           5   245    51   334    44   292\n",
      "> 3h4sA                      1    4201   19900     359     577   37.77     0.0           6   273    53   353    50   327\n",
      "> 1bg2A                      1    7654   25000     323     577   38.87     0.0           7   299    22   340    23   323\n",
      "> 3cnzA                      1    8394   17750     333     577   37.59     0.0           8   263    53   353    44   309\n",
      "> 4a5yB                      1    8685   20500     302     577   40.59     0.0           9   257    51   343    32   302\n",
      "> 4bn2A                      1    9693   18650     326     577   37.94     0.0          10   274    51   348    41   322\n",
      "> 4bn2C                      1    9694   18250     311     577   37.79     0.0          11   304    11   348     4   310\n",
      "> 3u06A                      1   10774   16100     359     577   33.67     0.0          12   290    11   334    58   351\n",
      "> 3u06B                      1   10775   13800     344     577   34.78     0.0          13   269    11   334    60   335\n",
      "> 4frzB                      1   11562   17850     349     577   37.45     0.0          14   262    53   353    58   324\n",
      "> 1n6mB                      1   13177   13600     335     577   33.94     0.0          15   265    11   334    58   331\n",
      "> 1mkjA                      1   13729   23250     336     577   37.93     0.0          16   312     4   347     1   319\n",
      "> 6a1zA                      1   14531   22500     352     577   38.60     0.0          17   314     9   361     1   342\n",
      "> 4a14A                      1   14535   19250     296     577   41.91     0.0          18   241    60   337    55   295\n",
      "> 3wpnA                      1   15809   18350     309     577   37.95     0.0          19   292    11   348     5   307\n",
      "> 4etpA                      1   16413   15600     379     577   36.50     0.0          20   259    51   349   106   379\n",
      "> 4ozqA                      1   19585   23650     683     577   43.64     0.0          21   287    23   337   392   682\n",
      "> 3cobA                      1   19976   22250     355     577   38.31     0.0          22   303    53   372    48   355\n",
      "> 3cobC                      1   19977   20050     340     577   38.14     0.0          23   279    53   359    48   338\n",
      "> 5mioC                      1   20804   18250     477     577   36.86     0.0          24   289    51   361    83   375\n",
      "> 1gojA                      1   21118   26800     354     577   38.33     0.0          25   335     5   358     1   347\n",
      "> 5x3eA                      1   22218   14350     375     577   29.03     0.0          26   312    11   340     4   375\n",
      "> 5x3eB                      1   22219   13050     358     577   29.78     0.0          27   302    11   340     3   358\n",
      "> 2y65A                      1   23876   21450     328     577   38.99     0.0          28   275    47   348    42   318\n",
      "> 2y65B                      1   23877   21350     341     577   38.35     0.0          29   277    47   348    41   319\n",
      "> 2hehA                      1   24256   14550     324     577   39.75     0.0          30   236    51   334    81   319\n",
      "> 1v8kA                      1   24955   15650     332     577   39.84     0.0          31   246    51   341    82   332\n",
      "> 1vfvA                      1   29856   20200     324     577   41.02     0.0          32   253    61   347    68   323\n",
      "> 5hleA                      1   30091   19150     303     577   38.28     0.0          33   253    51   334    47   302\n",
      "> 1cz7A                      1   32563   16850     342     577   32.79     0.0          34   296    11   334    30   334\n",
      "> 3gbjA                      1   33518   21000     302     577   40.86     0.0          35   283    10   337     1   301\n",
      "> 3gbjC                      1   33519   20600     294     577   40.96     0.0          36   278    10   337     1   293\n",
      "> 5w3dB                      1   34603   11200     313     577   35.86     0.0          37   245    11   334    58   308\n",
      "> 1f9tA                      1   35688   15350     304     577   39.60     0.0          38   237    51   334    52   301\n",
      "> 1f9vA                      1   35704   15050     313     577   38.22     0.0          39   248    51   349    52   310\n",
      "> 5zbrB                      1   38732   22200     330     577   39.88     0.0          40   300     9   348     1   321\n",
      "> 3zcwA                      1   39489   21350     321     577   38.24     0.0          41   300    11   343     2   320\n",
      "> 6bbnE                      1   40497   16700     378     577   39.23     0.0          42   256    51   334   114   373\n",
      "> 3b6uA                      1   42271   21350     323     577   38.94     0.0          43   301    11   348     3   323\n",
      "> 3b6uB                      1   42272   22450     343     577   38.97     0.0          44   315    11   352     7   337\n",
      "> 3b6vA                      1   42273   17000     306     577   39.46     0.0          45   255    42   352    42   302\n",
      "> 2owmA                      1   43026   22150     328     577   38.44     0.0          46   308    10   348     8   327\n",
      "> 1t5cA                      1   49621   22000     322     577   38.51     0.0          47   316     6   350     1   322\n",
      "> 1t5cB                      1   49622   21700     310     577   39.72     0.0          48   285    38   350    24   310\n",
      "> 2zflA                      1   52454   19800     313     577   43.32     0.0          49   245    61   342    67   313\n",
      "> 3zfdA                      1   52460   24750     340     577   41.92     0.0          50   285    51   347    46   336\n",
      "> 2zfiA                      1   52487   22150     329     577   40.84     0.0          51   259    61   337    67   328\n",
      "> 3nwnA                      1   52817   16100     308     577   37.50     0.0          52   266    28   334    31   302\n",
      "> 4h1gA                      1   52945   15950     670     577   32.13     0.0          53   303     1   334   361   665\n",
      "> 3x2tA                      1   53687   20300     311     577   38.43     0.0          54   266    51   347    44   311\n",
      "> 3x2tB                      1   53688   21800     326     577   38.85     0.0          55   276    51   347    44   321\n",
      "> 1ry6A                      1   53816   13600     319     577   33.84     0.0          56   252    51   334    52   314\n",
      "> 3t0qA                      1   54261   14550     304     577   34.68     0.0          57   281    11   334     3   299\n",
      "> 2gryA                      1   54590   16050     327     577   40.00     0.0          58   247    51   334    75   324\n",
      "> 2vvgA                      1   54932   19150     308     577   41.35     0.0          59   257    51   354    42   307\n",
      "> 3lreA                      1   55211   18000     289     577   40.28     0.0          60   279    11   337     1   288\n",
      "> 3lreB                      1   55212   18850     300     577   40.27     0.0          61   289    11   340     3   300\n",
      "> 6njeA                      1   57846   14950     297     577   31.83     0.0          62   289     9   340     8   296\n",
      "> 1ii6A                      1   58279   22850     340     577   37.98     0.0          63   316    11   346     4   340\n",
      "> 3kinB                      1   61261    6600     117     577   35.40    0.83E-06      64   113   267   391     1   113\n",
      "> 2kinA                      1   61324   16100     238     577   38.54     0.0          65   190    51   249    46   237\n",
      "> 2kinB                      1   61325    6650     100     577   38.55    0.54E-06      66    83   263   347     1    83\n",
      "> 4gkrA                      1   61877   14900     296     577   35.86     0.0          67   277    11   334     2   291\n",
      "> 4gkrB                      1   61878   15500     307     577   36.54     0.0          68   289    11   349     3   303\n",
      "> 6a20A                      1   62255   24150     395     577   39.56     0.0          69   348     9   420     3   366\n",
      "> 3dc4A                      1   63280   13850     291     577   37.10     0.0          70   277     9   339     4   286\n",
      "> 5lt0A                      1   63785   22550     303     577   38.28     0.0          71   296     9   340     1   303\n",
      "> 5lt1A                      1   63786   23150     309     577   38.11     0.0          72   300     9   337     1   307\n",
      "> 5gszA                      1   66643   17850     306     577   35.15     0.0          73   290     7   334    12   304\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    55.046875\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000819\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1223.062622\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.171875\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000823\n",
      "profile_iteration__> Overall performance [seq/sec]:  1214.770630\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           buildprofile/EAO00199build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     60244183   58832.211    57.453\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60246487   58834.461    57.456\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60247715   58835.660    57.457\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60248591   58836.516    57.458\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60249723   58837.621    57.459\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60250699   58838.574    57.460\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60251787   58839.637    57.461\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60252979   58840.801    57.462\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60254027   58841.824    57.463\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60255051   58842.824    57.464\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60256143   58843.891    57.465\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60257355   58845.074    57.466\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60258511   58846.203    57.467\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60259583   58847.250    57.468\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60260627   58848.270    57.469\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60261683   58849.301    57.470\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60262927   58850.516    57.471\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60264179   58851.738    57.472\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60265139   58852.676    57.473\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60266303   58853.812    57.474\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60267335   58854.820    57.475\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60268479   58855.938    57.477\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60269687   58857.117    57.478\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60270799   58858.203    57.479\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60271951   58859.328    57.480\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60273287   58860.633    57.481\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60274531   58861.848    57.482\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60275735   58863.023    57.483\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60276831   58864.094    57.484\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60277935   58865.172    57.486\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60278875   58866.090    57.486\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60279855   58867.047    57.487\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60280863   58868.031    57.488\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60281871   58869.016    57.489\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60283051   58870.168    57.490\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60284179   58871.270    57.491\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60285287   58872.352    57.493\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60286263   58873.305    57.493\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60287207   58874.227    57.494\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60288195   58875.191    57.495\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60289391   58876.359    57.496\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60290587   58877.527    57.498\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60291607   58878.523    57.499\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60292807   58879.695    57.500\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60294063   58880.922    57.501\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60295079   58881.914    57.502\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60296307   58883.113    57.503\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60297567   58884.344    57.504\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60298703   58885.453    57.505\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60299679   58886.406    57.506\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60300815   58887.516    57.507\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60301847   58888.523    57.508\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60302907   58889.559    57.509\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60304115   58890.738    57.510\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60305175   58891.773    57.511\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60306275   58892.848    57.513\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60307279   58893.828    57.514\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60308399   58894.922    57.515\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60309383   58895.883    57.516\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60310407   58896.883    57.516\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60311519   58897.969    57.518\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60312671   58899.094    57.519\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60313823   58900.219    57.520\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60315083   58901.449    57.521\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60315531   58901.887    57.521\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60316287   58902.625    57.522\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60316615   58902.945    57.522\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60317719   58904.023    57.523\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60318871   58905.148    57.525\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60320259   58906.504    57.526\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60321363   58907.582    57.527\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60322543   58908.734    57.528\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60323739   58909.902    57.529\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     60324895   58911.031    57.530\n",
      "openf___224_> Open           buildprofile/EAO00199build_profile.ali\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     58972397   57590.230    56.240\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     58972925   57590.746    56.241\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     59021525   57638.207    56.287\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     59034825   57651.195    56.300\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           2zfl.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59047035   57663.121    56.312\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59047111   57663.195    56.312\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59047282   57663.359    56.312\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59048132   57664.191    56.313\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59049390   57665.422    56.314\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59051294   57667.281    56.316\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59051334   57667.320    56.316\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59051424   57667.406    56.316\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59054280   57670.195    56.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59054340   57670.254    56.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59054475   57670.387    56.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59058759   57674.570    56.323\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59058847   57674.656    56.323\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59059045   57674.848    56.323\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59065471   57681.125    56.329\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59065603   57681.254    56.329\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59065900   57681.543    56.330\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59075522   57690.938    56.339\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59075722   57691.133    56.339\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59076172   57691.574    56.339\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59090622   57705.688    56.353\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59090922   57705.977    56.353\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59091597   57706.637    56.354\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59113255   57727.789    56.375\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59113703   57728.227    56.375\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59114711   57729.211    56.376\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59147215   57760.953    56.407\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     59379087   57987.391    56.628\n",
      "read_mo_297_> Segments, residues, atoms:        1      313     2467\n",
      "read_mo_298_> Segment:        1  4:A  356:A     2467\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59388135   57996.227    56.637\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     59569994   58173.820    56.810\n",
      "openf___224_> Open           ali/EAO00199.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59674929   58276.297    56.910\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59771333   58370.441    57.002\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59773637   58372.691    57.005\n",
      "\n",
      "Read the alignment from file       : ali/EAO00199.ali\n",
      "\n",
      "Total number of alignment positions:   577\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      2zflA     313      1    2zfl.pdb MOL_ID  1; MOLECULE  KINESIN-LIKE PROTEIN KIF1A, KINESIN HEAVY CHAIN I CHAIN  A;\n",
      "  2   EAO00199     577      1    EAO00199\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          587\n",
      "  Score                    :  177109.6094\n",
      "openf___224_> Open           ali/EAO00199-2zflA.ali\n",
      "openf___224_> Open           ali/EAO00199-2zflA.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     58972397   57590.230    56.240\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     58972925   57590.746    56.241\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     59021525   57638.207    56.287\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     59034825   57651.195    56.300\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00199-2zflA.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59046790   57662.883    56.311\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59048240   57664.297    56.313\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59051140   57667.129    56.316\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59056940   57672.793    56.321\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59068540   57684.121    56.332\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     59309348   57919.285    56.562\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59310596   57920.504    56.563\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59312900   57922.754    56.565\n",
      "\n",
      "Read the alignment from file       : ali/EAO00199-2zflA.ali\n",
      "\n",
      "Total number of alignment positions:   587\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      2zflA     313      1    2zfl.pdb MOL_ID  1; MOLECULE  KINESIN-LIKE PROTEIN KIF1A, KINESIN HEAVY CHAIN I CHAIN  A;\n",
      "  2   EAO00199     577      1    EAO00199\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           2zfl.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59408525   58016.137    56.656\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59408525   58016.137    56.656\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59408696   58016.305    56.657\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59409546   58017.133    56.657\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59410804   58018.363    56.659\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59412708   58020.223    56.660\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59412708   58020.223    56.660\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59412798   58020.312    56.660\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59415654   58023.102    56.663\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59415654   58023.102    56.663\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59415789   58023.230    56.663\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59420073   58027.414    56.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59420073   58027.414    56.667\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59420271   58027.609    56.668\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59426697   58033.883    56.674\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59426697   58033.883    56.674\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59426994   58034.172    56.674\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59436616   58043.570    56.683\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59436616   58043.570    56.683\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59437066   58044.008    56.684\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59451516   58058.121    56.697\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59451516   58058.121    56.697\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59452191   58058.781    56.698\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59473849   58079.930    56.719\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     59473849   58079.930    56.719\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59474857   58080.914    56.720\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     59507361   58112.656    56.751\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     59507145   58112.445    56.750\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    263     1  204   209      T     E   12.229\n",
      "    351     1  253   270      T     N   10.900\n",
      "    391     1  286   303      A     Q   12.272\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59646397   58248.434    56.883\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59647997   58249.996    56.885\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59651977   58253.883    56.889\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59653977   58255.836    56.890\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59656977   58258.766    56.893\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59661477   58263.160    56.898\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59667049   58268.602    56.903\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59669849   58271.336    56.906\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59674049   58275.438    56.910\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59680349   58281.590    56.916\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59683121   58284.297    56.918\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     59684521   58285.664    56.920\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     59774953   58373.977    57.006\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     60215365   58804.066    57.426\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     60280941   58868.105    57.488\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      577   4475   4545      0         0      1826\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     60468757   59051.520    57.667\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      577   4475   4545   6127      7145      1826\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     60659613   59237.902    57.850\n",
      "patch_______> segment topology patched using RTF:  577 ;  SER  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      577   4476   4546   6129      7147      1827\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      577   4476   4546   6129      7147      1827\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1926     2550\n",
      "openf___224_> Open           EAO00199.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      577     4476     4476\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60767062   59342.836    57.952\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60775254   59350.836    57.960\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60791638   59366.836    57.975\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60824406   59398.836    58.007\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60857174   59430.836    58.038\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60922710   59494.836    58.100\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     60955478   59526.836    58.132\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61021014   59590.836    58.194\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61053782   59622.836    58.225\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61184854   59750.836    58.350\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61250390   59814.836    58.413\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61315926   59878.836    58.475\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61446998   60006.836    58.600\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61709142   60262.836    58.850\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61840214   60390.836    58.975\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     61971286   60518.836    59.100\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     62233430   60774.836    59.350\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     62757718   61286.836    59.850\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     4546    6129    6722    1811\n",
      "              Total number of restraints before, now         :        0    19208\n",
      "make_re_422_> Number of previous, current restraints         :        0    19208\n",
      "make_re_423_> Number of previous, current selected restraints:        0    19208\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     63087230   61608.625    60.165\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      198  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      201  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      208  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      222  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      226  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      198  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      201  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      208  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      222  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      226  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      198  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      208  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      208  ARG\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     63349390   61864.641    60.415\n",
      "make_re_422_> Number of previous, current restraints         :    19208    19783\n",
      "make_re_423_> Number of previous, current selected restraints:    19208    19783\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     63349390   61864.641    60.415\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      592\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    19783    20359\n",
      "make_re_423_> Number of previous, current selected restraints:    19783    19767\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     63349390   61864.641    60.415\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     63611534   62120.641    60.665\n",
      "make_re_422_> Number of previous, current restraints         :    20359    20847\n",
      "make_re_423_> Number of previous, current selected restraints:    19767    20255\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     63611534   62120.641    60.665\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    20847    21203\n",
      "make_re_423_> Number of previous, current selected restraints:    20255    20611\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     63611534   62120.641    60.665\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    21203    21376\n",
      "make_re_423_> Number of previous, current selected restraints:    20611    20784\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     63611534   62120.641    60.665\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "make_re_422_> Number of previous, current restraints         :    21376    21441\n",
      "make_re_423_> Number of previous, current selected restraints:    20784    20849\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     64900586   63379.477    61.894\n",
      "make_re_422_> Number of previous, current restraints         :    21441    27741\n",
      "make_re_423_> Number of previous, current selected restraints:    20849    27149\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     65424874   63891.477    62.394\n",
      "make_re_422_> Number of previous, current restraints         :    27741    34175\n",
      "make_re_423_> Number of previous, current selected restraints:    27149    33583\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     65949162   64403.477    62.894\n",
      "make_re_422_> Number of previous, current restraints         :    34175    36952\n",
      "make_re_423_> Number of previous, current selected restraints:    33583    36360\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     66997738   65427.477    63.894\n",
      "make_re_422_> Number of previous, current restraints         :    36952    37833\n",
      "make_re_423_> Number of previous, current selected restraints:    36360    37241\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37833    37833\n",
      "make_re_423_> Number of previous, current selected restraints:    37241    37241\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37833    37833\n",
      "make_re_423_> Number of previous, current selected restraints:    37241    37241\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37833    37833\n",
      "make_re_423_> Number of previous, current selected restraints:    37241    37241\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37833    37833\n",
      "make_re_423_> Number of previous, current selected restraints:    37241    37241\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37833    37833\n",
      "make_re_423_> Number of previous, current selected restraints:    37241    37241\n",
      "rmdupl__427_>     3460 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37833    33781\n",
      "openf___224_> Open           EAO00199.rsr\n",
      "openf___224_> Open           EAO00199.rsr\n",
      "rdcsr2__307_> Number of restraints read    :    33781\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    33781    33781\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00199.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     66204504   64652.836    63.138\n",
      "read_mo_297_> Segments, residues, atoms:        1      577     4476\n",
      "read_mo_298_> Segment:        1  1:A  577:A     4476\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4476     4476        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    21012\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22310\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22981\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    23873\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    24919\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    25728\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    27228\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    28107\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    29246\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    30620\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     66604500   65043.457    63.519\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     66711924   65148.363    63.621\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    31777\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33640\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33781\n",
      "iupac_m_487_> NH1/2 swapped:      178.9932       21  21\n",
      "iupac_m_486_> OE1/2 will be swapped:     -132.3827       24  24\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.9604       31  31\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.2505       33  33\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.3899       34  34\n",
      "iupac_m_487_> NH1/2 swapped:      178.7268       36  36\n",
      "iupac_m_485_> OD1/2 will be swapped:      127.1290       41  41\n",
      "iupac_m_486_> OE1/2 will be swapped:     -126.4870       43  43\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      104.3029       51  51\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      129.3772       54  54\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3710       55  55\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.8258       62  62\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000       75  75\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.9524       87  87\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.2135      120  120\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.1924      121  121\n",
      "iupac_m_487_> NH1/2 swapped:      179.4236      125  125\n",
      "iupac_m_487_> NH1/2 swapped:      178.2089      149  149\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.4633      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.7444      162  162\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.7163      165  165\n",
      "iupac_m_487_> NH1/2 swapped:      176.7289      173  173\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3925      189  189\n",
      "iupac_m_486_> OE1/2 will be swapped:      164.2512      205  205\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3473      233  233\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.4820      243  243\n",
      "iupac_m_486_> OE1/2 will be swapped:      153.3611      248  248\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6720      249  249\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.4495      254  254\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.8282      261  261\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.8340      262  262\n",
      "iupac_m_487_> NH1/2 swapped:     -177.4072      275  275\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8922      283  283\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.7648      289  289\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6124      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:     -180.0000      303  303\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.6601      352  352\n",
      "iupac_m_486_> OE1/2 will be swapped:      146.7419      354  354\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.9646      355  355\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      165.9539      371  371\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.4549      389  389\n",
      "iupac_m_485_> OD1/2 will be swapped:      160.6320      391  391\n",
      "iupac_m_485_> OD1/2 will be swapped:     -174.8002      402  402\n",
      "iupac_m_485_> OD1/2 will be swapped:      108.4604      415  415\n",
      "iupac_m_487_> NH1/2 swapped:      175.6403      416  416\n",
      "iupac_m_487_> NH1/2 swapped:      179.8718      424  424\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.8433      429  429\n",
      "iupac_m_487_> NH1/2 swapped:      178.7406      431  431\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.4044      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.5281      488  488\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.5196      489  489\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8506      499  499\n",
      "iupac_m_485_> OD1/2 will be swapped:     -158.4145      506  506\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.2341      519  519\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.4364      569  569\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     67219544   65644.086    64.106\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     67649240   66063.711    64.515\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     68508632   66902.961    65.335\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     70227416   68581.461    66.974\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     73664984   71938.461    70.252\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     80540120   78652.461    76.809\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   912705\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -40490.2852\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -40490.285156\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    32.907001\n",
      "Sequence length          :          577\n",
      "Compactness              :     0.145218\n",
      "Native energy (pair)     :   545.603106\n",
      "Native energy (surface)  :    37.544783\n",
      "Native energy (combined) :    14.870977\n",
      "Z score (pair)           :    -4.269842\n",
      "Z score (surface)        :    -2.838513\n",
      "Z score (combined)       :    -5.069613\n",
      "GA341 score              :     0.984422\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10878\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4719.9028\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4546       0      0   0.006   0.006      55.274       1.000\n",
      " 2 Bond angle potential               :    6129       2     24   2.099   2.099      512.45       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2670       0    140  51.850  51.850      1159.1       1.000\n",
      " 4 Stereochemical improper torsion pot:    1811       0      2   1.487   1.487      81.825       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10878       0      0   0.005   0.005      37.235       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6300       1     20   0.701   0.701      494.91       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6434       6     50   0.887   0.887      731.56       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     576       0     15   4.834   4.834      158.20       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     488       0      4  84.459  84.459      148.95       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     356       0      0  79.387  79.387      154.25       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     173       0      1  90.948  90.948      128.34       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      65       0      0 101.377 101.377      41.665       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2777       0      2   0.698   0.698      80.554       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     575     242    124  41.778 113.535      831.59       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     881       0      4   1.542   1.542      104.05       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00199.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00199.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   92841.4609\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8400 367I 367I N   CA   2829 2830  126.08  107.00   19.08    5.49  107.00   19.08    5.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24549  51F  36R N   O     367  271    5.04    2.83    2.21    6.93    2.83    2.21    6.93\n",
      "    2  28470 342N 144Y N   O    2622 1108   10.90    6.52    4.39    5.84    6.52    4.39    5.84\n",
      "    3  28567 369L 143V N   O    2848 1096   10.74    7.94    2.80    5.40    7.94    2.80    5.40\n",
      "    4  28568 369L 144Y N   O    2848 1108   13.87   10.51    3.36    4.87   10.51    3.36    4.87\n",
      "    5  28569 369L 145M N   O    2848 1116   13.42    9.65    3.77    4.95    9.65    3.77    4.95\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15157   1M   2G C   N       7    9  -65.79  -80.20   49.46    1.30   82.20 -170.55    6.77\n",
      "    1          2G   2G N   CA      9   10  126.78  174.10                    8.50\n",
      "    2  15158   2G   3E C   N      11   13 -174.73 -117.80   96.43    3.41  -63.60  152.93   26.01\n",
      "    2          3E   3E N   CA     13   14 -145.37  136.80                  -40.30\n",
      "    3  15176  20P  21R C   N     141  143  -87.07 -125.20   68.93    2.34  -63.00  126.58   15.62\n",
      "    3         21R  21R N   CA    143  144   83.17  140.60                  -41.10\n",
      "    4  15177  21R  22Q C   N     152  154  -58.47  -73.00   24.89    1.90  -63.80  158.88   23.09\n",
      "    4         22Q  22Q N   CA    154  155  160.90  140.70                  -40.30\n",
      "    5  15186  30A  31D C   N     220  222   71.54   54.50   34.11    1.68  -63.30  144.29   24.52\n",
      "    5         31D  31D N   CA    222  223   11.35   40.90                  -40.00\n",
      "    6  15188  32S  33D C   N     234  236  -59.38  -70.90   35.52    1.26  -63.30  156.75   19.59\n",
      "    6         33D  33D N   CA    236  237  116.70  150.30                  -40.00\n",
      "    7  15189  33D  34D C   N     242  244   15.78  -63.30  118.58   19.51  -63.30  118.58   19.51\n",
      "    7         34D  34D N   CA    244  245   48.36  -40.00                  -40.00\n",
      "    8  15199  43E  44G C   N     320  322   57.79   78.70   84.43    1.59  -62.40  127.69   20.68\n",
      "    8         44G  44G N   CA    322  323  -84.30 -166.10                  -41.20\n",
      "    9  15260 104S 105K C   N     789  791  -75.79  -70.20   30.33    2.06   56.60 -173.32   11.94\n",
      "    9        105K 105K N   CA    791  792  170.20  140.40                   38.60\n",
      "   10  15261 105K 106E C   N     798  800  -64.27  -69.30   33.04    2.69  -63.60  144.54   19.31\n",
      "   10        106E 106E N   CA    800  801  175.16  142.50                  -40.30\n",
      "   11  15262 106E 107G C   N     807  809  131.44   82.20   52.33    2.72  -62.40  169.21   28.76\n",
      "   11        107G 107G N   CA    809  810   -9.20    8.50                  -41.20\n",
      "   12  15275 119I 120E C   N     906  908  -52.16  -69.30   31.88    2.89  -63.60  150.75   19.50\n",
      "   12        120E 120E N   CA    908  909  169.38  142.50                  -40.30\n",
      "   13  15284 128S 129D C   N     979  981  -56.31  -63.30    7.91    1.35  -70.90  174.02    9.31\n",
      "   13        129D 129D N   CA    981  982  -36.29  -40.00                  150.30\n",
      "   14  15285 129D 130S C   N     987  989   78.15  -64.10  146.42   17.23  -64.10  146.42   17.23\n",
      "   14        130S 130S N   CA    989  990  -69.67  -35.00                  -35.00\n",
      "   15  15311 155R 156A C   N    1198 1200 -125.55 -134.00   12.45    0.39  -62.50 -170.46   28.72\n",
      "   15        156A 156A N   CA   1200 1201  137.85  147.00                  -40.90\n",
      "   16  15314 158G 159Q C   N    1218 1220  -98.97 -121.10   41.70    1.52  -63.80  148.87   20.21\n",
      "   16        159Q 159Q N   CA   1220 1221  104.36  139.70                  -40.30\n",
      "   17  15315 159Q 160E C   N    1227 1229 -163.61 -117.80   67.78    2.23  -63.60  166.37   27.55\n",
      "   17        160E 160E N   CA   1229 1230 -173.25  136.80                  -40.30\n",
      "   18  15317 161P 162E C   N    1243 1245   56.42  -69.30  136.47    8.45  -63.60  176.73   29.79\n",
      "   18        162E 162E N   CA   1245 1246   89.42  142.50                  -40.30\n",
      "   19  15319 163S 164L C   N    1258 1260  130.82   60.20  105.06   11.80   60.20  105.06   11.80\n",
      "   19        164L 164L N   CA   1260 1261  107.39   29.60                   29.60\n",
      "   20  15325 169D 170S C   N    1310 1312 -104.46 -136.60   35.24    1.10  -64.10  176.42   11.05\n",
      "   20        170S 170S N   CA   1312 1313  136.74  151.20                  -35.00\n",
      "   21  15326 170S 171Q C   N    1316 1318 -178.96 -121.10   72.85    2.27  -63.80  178.00   31.73\n",
      "   21        171Q 171Q N   CA   1318 1319 -176.03  139.70                  -40.30\n",
      "   22  15327 171Q 172H C   N    1325 1327  -72.51  -67.60    9.62    0.60  -63.20  169.68   21.80\n",
      "   22        172H 172H N   CA   1327 1328  148.28  140.00                  -42.30\n",
      "   23  15328 172H 173R C   N    1335 1337  -67.33  -72.10    9.92    0.85  -63.00  168.36   23.03\n",
      "   23        173R 173R N   CA   1337 1338  150.60  141.90                  -41.10\n",
      "   24  15330 174V 175Y C   N    1353 1355 -140.26 -124.30   17.76    1.47  -63.50 -172.56   26.42\n",
      "   24        175Y 175Y N   CA   1355 1356  127.61  135.40                  -43.40\n",
      "   25  15332 176V 177K C   N    1372 1374 -115.14 -118.00    5.74    0.30  -62.90 -177.26   26.83\n",
      "   25        177K 177K N   CA   1374 1375  144.08  139.10                  -40.80\n",
      "   26  15333 177K 178G C   N    1381 1383  163.70 -167.20   31.10    0.45   82.20 -165.07   13.21\n",
      "   26        178G 178G N   CA   1383 1384 -174.42  174.60                    8.50\n",
      "   27  15334 178G 179A C   N    1385 1387  -65.96  -68.20    4.58    0.44  -62.50  169.84   28.00\n",
      "   27        179A 179A N   CA   1387 1388  149.30  145.30                  -40.90\n",
      "   28  15335 179A 180G C   N    1390 1392 -137.44 -167.20   39.61    0.82   82.20 -161.79   15.20\n",
      "   28        180G 180G N   CA   1392 1393  148.46  174.60                    8.50\n",
      "   29  15337 181V 182H C   N    1401 1403   72.21   56.30  122.74   12.25  -63.20 -154.04   22.63\n",
      "   29        182H 182H N   CA   1403 1404  162.51   40.80                  -42.30\n",
      "   30  15338 182H 183S C   N    1411 1413 -141.05 -136.60   13.02    0.55  -64.10  178.96   18.47\n",
      "   30        183S 183S N   CA   1413 1414  163.43  151.20                  -35.00\n",
      "   31  15341 185L 186S C   N    1432 1434 -151.19 -136.60   58.07    3.19  -64.10  156.47   10.78\n",
      "   31        186S 186S N   CA   1434 1435   94.99  151.20                  -35.00\n",
      "   32  15343 187L 188E C   N    1446 1448  -76.83  -69.30   26.69    1.77  -63.60  152.17   21.02\n",
      "   32        188E 188E N   CA   1448 1449  168.10  142.50                  -40.30\n",
      "   33  15352 196K 197G C   N    1525 1527  -91.08  -80.20  105.94    4.72   82.20 -176.55    9.62\n",
      "   33        197G 197G N   CA   1527 1528   68.72  174.10                    8.50\n",
      "   34  15353 197G 198N C   N    1529 1531 -162.24 -119.90   54.94    1.59  -63.20  177.17   27.59\n",
      "   34        198N 198N N   CA   1531 1532  172.00  137.00                  -41.10\n",
      "   35  15354 198N 199A C   N    1537 1539   78.63  -68.20  147.21   12.83  -62.50 -144.18   32.91\n",
      "   35        199A 199A N   CA   1539 1540  155.83  145.30                  -40.90\n",
      "   36  15355 199A 200N C   N    1542 1544 -121.95 -119.90   13.44    0.65  -63.20  174.98   19.28\n",
      "   36        200N 200N N   CA   1544 1545  123.72  137.00                  -41.10\n",
      "   37  15356 200N 201R C   N    1550 1552 -145.76 -125.20   22.77    0.70  -63.00 -172.27   30.99\n",
      "   37        201R 201R N   CA   1552 1553  150.40  140.60                  -41.10\n",
      "   38  15357 201R 202Q C   N    1561 1563 -104.71 -121.10   46.71    1.90  -63.80  142.27   19.03\n",
      "   38        202Q 202Q N   CA   1563 1564   95.96  139.70                  -40.30\n",
      "   39  15358 202Q 203T C   N    1570 1572 -154.84 -124.80   47.93    1.49  -63.20  164.87   26.92\n",
      "   39        203T 203T N   CA   1572 1573 -179.16  143.50                  -42.10\n",
      "   40  15359 203T 204G C   N    1577 1579   84.03   78.70    8.78    0.16   82.20  178.43    8.75\n",
      "   40        204G 204G N   CA   1579 1580 -173.08 -166.10                    8.50\n",
      "   41  15360 204G 205E C   N    1581 1583 -116.14 -117.80   28.65    1.47  -63.60  163.00   24.70\n",
      "   41        205E 205E N   CA   1583 1584  165.40  136.80                  -40.30\n",
      "   42  15361 205E 206T C   N    1590 1592  -67.03  -78.10   11.60    0.60  -63.20  171.61   22.26\n",
      "   42        206T 206T N   CA   1592 1593  146.33  149.80                  -42.10\n",
      "   43  15362 206T 207K C   N    1597 1599  -78.62  -70.20   12.58    0.75  -62.90  170.18   22.99\n",
      "   43        207K 207K N   CA   1599 1600  149.75  140.40                  -40.80\n",
      "   44  15364 208M 209N C   N    1614 1616  -73.93  -71.20    3.15    0.18  -63.20  174.86   22.46\n",
      "   44        209N 209N N   CA   1616 1617  144.37  142.80                  -41.10\n",
      "   45  15365 209N 210E C   N    1622 1624  -99.95 -117.80   17.92    0.70  -63.60 -177.65   22.51\n",
      "   45        210E 210E N   CA   1624 1625  138.38  136.80                  -40.30\n",
      "   46  15366 210E 211T C   N    1631 1633 -127.17 -124.80   25.19    1.01  -63.20  162.45   25.03\n",
      "   46        211T 211T N   CA   1633 1634  168.58  143.50                  -42.10\n",
      "   47  15367 211T 212S C   N    1638 1640  -67.73  -72.40    7.66    0.60  -64.10  166.57   12.29\n",
      "   47        212S 212S N   CA   1640 1641  158.47  152.40                  -35.00\n",
      "   48  15368 212S 213S C   N    1644 1646  -62.34  -72.40   32.94    1.62  -64.10  156.05   11.41\n",
      "   48        213S 213S N   CA   1646 1647  121.04  152.40                  -35.00\n",
      "   49  15369 213S 214R C   N    1650 1652 -172.74 -125.20   50.26    1.63  -63.00 -164.35   33.68\n",
      "   49        214R 214R N   CA   1652 1653  156.92  140.60                  -41.10\n",
      "   50  15370 214R 215S C   N    1661 1663 -137.72 -136.60   12.51    0.66  -64.10 -171.30   11.74\n",
      "   50        215S 215S N   CA   1663 1664  138.74  151.20                  -35.00\n",
      "   51  15371 215S 216H C   N    1667 1669 -160.57 -125.60   40.23    1.03  -63.20 -173.54   29.56\n",
      "   51        216H 216H N   CA   1669 1670  158.69  138.80                  -42.30\n",
      "   52  15372 216H 217A C   N    1677 1679 -116.28 -134.00   28.04    0.95  -62.50  174.65   26.56\n",
      "   52        217A 217A N   CA   1679 1680  125.26  147.00                  -40.90\n",
      "   53  15375 219V 220Q C   N    1697 1699  173.44 -121.10   92.54    3.01  -73.00  130.56    8.00\n",
      "   53        220Q 220Q N   CA   1699 1700 -154.89  139.70                  140.70\n",
      "   54  15379 223I 224S C   N    1729 1731  -43.41  -72.40   32.98    1.81  -64.10  172.93   13.87\n",
      "   54        224S 224S N   CA   1731 1732  136.68  152.40                  -35.00\n",
      "   55  15380 224S 225Q C   N    1735 1737  -31.32  -73.00   46.71    2.90  -63.80  163.18   25.73\n",
      "   55        225Q 225Q N   CA   1737 1738  119.62  140.70                  -40.30\n",
      "   56  15383 227Y 228G C   N    1765 1767  118.63   82.20   45.02    1.70  -62.40 -179.53   31.21\n",
      "   56        228G 228G N   CA   1767 1768  -17.94    8.50                  -41.20\n",
      "   57  15386 230L 231D C   N    1783 1785  -87.66  -70.90   44.05    1.51  -63.30  131.25   17.75\n",
      "   57        231D 231D N   CA   1785 1786 -168.97  150.30                  -40.00\n",
      "   58  15388 232M 233R C   N    1799 1801 -166.17 -125.20   69.31    3.84  -63.00  162.69   18.71\n",
      "   58        233R 233R N   CA   1801 1802   84.70  140.60                  -41.10\n",
      "   59  15392 236E 237S C   N    1834 1836  -42.11  -72.40   36.19    1.90  -64.10  169.02   13.67\n",
      "   59        237S 237S N   CA   1836 1837  132.58  152.40                  -35.00\n",
      "   60  15395 239V 240L C   N    1854 1856  -63.67  -70.70   21.75    1.41  -63.50  162.22   22.51\n",
      "   60        240L 240L N   CA   1856 1857  121.02  141.60                  -41.20\n",
      "   61  15396 240L 241L C   N    1862 1864   85.16  -70.70  155.92   14.02  -63.50 -128.00   41.10\n",
      "   61        241L 241L N   CA   1864 1865  136.92  141.60                  -41.20\n",
      "   62  15398 242V 243D C   N    1877 1879  -61.84  -63.30   61.31    7.48  -63.30   61.31    7.48\n",
      "   62        243D 243D N   CA   1879 1880 -101.29  -40.00                  -40.00\n",
      "   63  15409 253T 254E C   N    1959 1961   61.22   54.60  131.83   10.67  -63.60 -168.19   22.49\n",
      "   63        254E 254E N   CA   1961 1962  174.06   42.40                  -40.30\n",
      "   64  15410 254E 255S C   N    1968 1970 -159.56 -136.60   23.06    0.97  -64.10 -159.84   21.37\n",
      "   64        255S 255S N   CA   1970 1971  149.07  151.20                  -35.00\n",
      "   65  15411 255S 256T C   N    1974 1976 -164.36 -124.80   73.65    2.29  -63.20  151.13   25.45\n",
      "   65        256T 256T N   CA   1976 1977 -154.38  143.50                  -42.10\n",
      "   66  15412 256T 257G C   N    1981 1983 -138.29 -167.20   31.21    0.91   82.20 -133.97   16.85\n",
      "   66        257G 257G N   CA   1983 1984 -173.66  174.60                    8.50\n",
      "   67  15414 258V 259A C   N    1992 1994 -136.76 -134.00   13.98    0.84  -62.50 -170.64   28.49\n",
      "   67        259A 259A N   CA   1994 1995  133.29  147.00                  -40.90\n",
      "   68  15415 259A 260F C   N    1997 1999  129.58 -124.20  106.22    4.16  -63.20 -119.45   42.94\n",
      "   68        260F 260F N   CA   1999 2000  142.77  143.30                  -44.30\n",
      "   69  15416 260F 261E C   N    2008 2010 -138.71 -117.80   24.63    0.72  -63.60 -174.25   28.98\n",
      "   69        261E 261E N   CA   2010 2011  149.82  136.80                  -40.30\n",
      "   70  15417 261E 262E C   N    2017 2019 -103.45 -117.80   14.36    0.54  -63.60 -177.85   22.34\n",
      "   70        262E 262E N   CA   2019 2020  137.44  136.80                  -40.30\n",
      "   71  15418 262E 263A C   N    2026 2028  -65.37  -68.20   10.16    0.72  -62.50  176.46   28.78\n",
      "   71        263A 263A N   CA   2028 2029  135.54  145.30                  -40.90\n",
      "   72  15419 263A 264K C   N    2031 2033 -157.89 -118.00   41.18    1.41  -62.90 -165.37   30.94\n",
      "   72        264K 264K N   CA   2033 2034  149.33  139.10                  -40.80\n",
      "   73  15420 264K 265K C   N    2040 2042 -114.56 -118.00    8.53    0.35  -62.90  179.68   21.10\n",
      "   73        265K 265K N   CA   2042 2043  131.29  139.10                  -40.80\n",
      "   74  15422 266I 267N C   N    2057 2059 -103.65 -119.90   34.24    1.15  -63.20  153.39   17.27\n",
      "   74        267N 267N N   CA   2059 2060  106.87  137.00                  -41.10\n",
      "   75  15423 267N 268Q C   N    2065 2067 -145.11 -121.10   27.78    0.86  -63.80 -175.13   31.42\n",
      "   75        268Q 268Q N   CA   2067 2068  153.68  139.70                  -40.30\n",
      "   76  15425 269S 270L C   N    2080 2082  -99.75 -108.50    9.32    0.48  -63.50 -179.42   23.25\n",
      "   76        270L 270L N   CA   2082 2083  135.71  132.50                  -41.20\n",
      "   77  15427 271L 272M C   N    2096 2098 -103.46 -125.60   22.29    0.95  -63.40 -179.11   30.21\n",
      "   77        272M 272M N   CA   2098 2099  143.10  140.50                  -40.50\n",
      "   78  15428 272M 273L C   N    2104 2106  -74.45  -70.70    6.83    0.42  -63.50  171.85   24.50\n",
      "   78        273L 273L N   CA   2106 2107  147.30  141.60                  -41.20\n",
      "   79  15429 273L 274G C   N    2112 2114  -63.26  -62.40    1.35    0.19   82.20  153.39   11.64\n",
      "   79        274G 274G N   CA   2114 2115  -40.16  -41.20                    8.50\n",
      "   80  15430 274G 275R C   N    2116 2118  -73.74  -72.10    6.24    0.51  -63.00  177.30   23.25\n",
      "   80        275R 275R N   CA   2118 2119  135.88  141.90                  -41.10\n",
      "   81  15431 275R 276A C   N    2127 2129  171.17 -134.00   57.91    1.36  -62.50 -161.24   39.15\n",
      "   81        276A 276A N   CA   2129 2130  165.65  147.00                  -40.90\n",
      "   82  15434 278N 279S C   N    2148 2150  -53.71  -72.40   54.23    2.62  -64.10  136.89   10.57\n",
      "   82        279S 279S N   CA   2150 2151  101.49  152.40                  -35.00\n",
      "   83  15454 298T 299R C   N    2299 2301 -158.84 -125.20   33.64    1.36  -63.00 -157.13   33.89\n",
      "   83        299R 299R N   CA   2301 2302  140.09  140.60                  -41.10\n",
      "   84  15455 299R 300L C   N    2310 2312 -131.70 -108.50   26.60    1.17  -63.50 -173.80   30.08\n",
      "   84        300L 300L N   CA   2312 2313  145.53  132.50                  -41.20\n",
      "   85  15456 300L 301L C   N    2318 2320  -97.92 -108.50   34.04    1.71  -63.50  145.48   18.53\n",
      "   85        301L 301L N   CA   2320 2321  100.15  132.50                  -41.20\n",
      "   86  15457 301L 302S C   N    2326 2328 -172.13 -136.60   40.81    1.25  -64.10 -172.10   21.41\n",
      "   86        302S 302S N   CA   2328 2329  171.26  151.20                  -35.00\n",
      "   87  15458 302S 303E C   N    2332 2334  -77.35  -69.30   11.44    0.64  -63.60  169.62   23.38\n",
      "   87        303E 303E N   CA   2334 2335  150.64  142.50                  -40.30\n",
      "   88  15459 303E 304S C   N    2341 2343 -113.72 -136.60   23.02    0.98  -64.10  178.24   16.35\n",
      "   88        304S 304S N   CA   2343 2344  153.80  151.20                  -35.00\n",
      "   89  15460 304S 305F C   N    2347 2349 -101.95 -124.20   27.53    0.71  -63.20  175.72   21.94\n",
      "   89        305F 305F N   CA   2349 2350  127.09  143.30                  -44.30\n",
      "   90  15461 305F 306G C   N    2358 2360   90.44   78.70   54.58    1.09   82.20  132.35    6.84\n",
      "   90        306G 306G N   CA   2360 2361  140.60 -166.10                    8.50\n",
      "   91  15462 306G 307G C   N    2362 2364  -64.69  -62.40    3.97    0.71   82.20  156.15   11.91\n",
      "   91        307G 307G N   CA   2364 2365  -44.45  -41.20                    8.50\n",
      "   92  15463 307G 308N C   N    2366 2368 -125.57 -119.90    7.14    0.21  -63.20 -171.79   27.08\n",
      "   92        308N 308N N   CA   2368 2369  141.33  137.00                  -41.10\n",
      "   93  15464 308N 309S C   N    2374 2376 -156.73 -136.60   45.83    1.81  -64.10  161.78   18.41\n",
      "   93        309S 309S N   CA   2376 2377 -167.64  151.20                  -35.00\n",
      "   94  15465 309S 310K C   N    2380 2382  -61.91  -70.20   24.80    1.97  -62.90  155.43   20.13\n",
      "   94        310K 310K N   CA   2382 2383  163.77  140.40                  -40.80\n",
      "   95  15467 311T 312W C   N    2396 2398 -141.09 -124.90   32.18    1.15  -63.00  164.33   25.80\n",
      "   95        312W 312W N   CA   2398 2399  171.21  143.40                  -44.20\n",
      "   96  15468 312W 313M C   N    2410 2412  -67.89  -73.00   10.80    0.82  -63.40  167.04   25.29\n",
      "   96        313M 313M N   CA   2412 2413  152.51  143.00                  -40.50\n",
      "   97  15469 313M 314L C   N    2418 2420  -67.52  -70.70   11.02    0.73  -63.50  172.30   23.70\n",
      "   97        314L 314L N   CA   2420 2421  131.05  141.60                  -41.20\n",
      "   98  15470 314L 315A C   N    2426 2428  166.21 -134.00   70.07    1.66  -62.50 -171.27   37.62\n",
      "   98        315A 315A N   CA   2428 2429 -176.47  147.00                  -40.90\n",
      "   99  15471 315A 316T C   N    2431 2433  -68.57  -78.10   24.07    1.35  -63.20  146.09   19.09\n",
      "   99        316T 316T N   CA   2433 2434  171.91  149.80                  -42.10\n",
      "  100  15474 318S 319T C   N    2451 2453 -122.00 -124.80   18.86    0.89  -63.20  166.48   25.24\n",
      "  100        319T 319T N   CA   2453 2454  162.14  143.50                  -42.10\n",
      "  101  15477 321A 322N C   N    2468 2470 -166.09 -119.90   68.07    3.69   55.90  145.96   17.88\n",
      "  101        322N 322N N   CA   2470 2471   87.01  137.00                   39.50\n",
      "  102  15478 322N 323N C   N    2476 2478 -160.18 -119.90   40.38    1.61  -71.20   89.04    6.52\n",
      "  102        323N 323N N   CA   2478 2479  139.73  137.00                  142.80\n",
      "  103  15495 339A 340I C   N    2605 2607  -68.62  -63.40    5.52    0.92 -120.60  179.82    8.37\n",
      "  103        340I 340I N   CA   2607 2608  -41.84  -43.60                  130.30\n",
      "  104  15497 341T 342N C   N    2620 2622   16.72 -119.90  141.32    4.96  -63.20  162.90   24.83\n",
      "  104        342N 342N N   CA   2622 2623  100.85  137.00                  -41.10\n",
      "  105  15498 342N 343K C   N    2628 2630 -102.62 -118.00   78.21    3.96  -62.90  110.78   16.76\n",
      "  105        343K 343K N   CA   2630 2631 -144.21  139.10                  -40.80\n",
      "  106  15499 343K 344A C   N    2637 2639  -59.07  -68.20   12.92    0.81  -62.50  177.10   29.19\n",
      "  106        344A 344A N   CA   2639 2640  136.16  145.30                  -40.90\n",
      "  107  15500 344A 345K C   N    2642 2644   73.09   56.60   68.97    3.84  -62.90  136.55   22.87\n",
      "  107        345K 345K N   CA   2644 2645  -28.37   38.60                  -40.80\n",
      "  108  15502 346V 347N C   N    2658 2660 -123.49 -119.90   49.73    2.34   55.90 -174.32   18.12\n",
      "  108        347N 347N N   CA   2660 2661   87.40  137.00                   39.50\n",
      "  109  15504 348K 349T C   N    2675 2677   -8.35 -124.80  116.56    6.04  -63.20 -172.38   20.93\n",
      "  109        349T 349T N   CA   2677 2678  138.48  143.50                  -42.10\n",
      "  110  15505 349T 350A C   N    2682 2684 -136.67 -134.00    7.33    0.34  -62.50 -178.85   33.66\n",
      "  110        350A 350A N   CA   2684 2685  153.83  147.00                  -40.90\n",
      "  111  15506 350A 351K C   N    2687 2689 -155.07 -118.00   86.67    3.56  -62.90  137.30   23.14\n",
      "  111        351K 351K N   CA   2689 2690 -142.55  139.10                  -40.80\n",
      "  112  15507 351K 352D C   N    2696 2698  -70.83  -70.90   27.52    1.36  -63.30  142.38   18.04\n",
      "  112        352D 352D N   CA   2698 2699  177.82  150.30                  -40.00\n",
      "  113  15508 352D 353L C   N    2704 2706 -132.32 -108.50   58.01    2.84  -63.50  150.10   25.09\n",
      "  113        353L 353L N   CA   2706 2707 -174.60  132.50                  -41.20\n",
      "  114  15509 353L 354E C   N    2712 2714 -145.90 -117.80   83.37    3.55  -63.60  132.95   22.11\n",
      "  114        354E 354E N   CA   2714 2715 -144.71  136.80                  -40.30\n",
      "  115  15510 354E 355Y C   N    2721 2723 -133.49 -124.30   19.36    1.41  -63.50  176.25   24.91\n",
      "  115        355Y 355Y N   CA   2723 2724  118.36  135.40                  -43.40\n",
      "  116  15511 355Y 356D C   N    2733 2735 -121.32  -96.50   45.11    1.90  -63.30  177.87   25.70\n",
      "  116        356D 356D N   CA   2735 2736  151.86  114.20                  -40.00\n",
      "  117  15512 356D 357E C   N    2741 2743 -122.02 -117.80   12.08    0.51  -63.60 -178.75   27.47\n",
      "  117        357E 357E N   CA   2743 2744  148.12  136.80                  -40.30\n",
      "  118  15513 357E 358I C   N    2750 2752  -64.89  -63.40    6.26    0.98 -120.60  176.83    8.09\n",
      "  118        358I 358I N   CA   2752 2753  -37.52  -43.60                  130.30\n",
      "  119  15515 359Q 360K C   N    2767 2769 -112.23 -118.00   13.17    0.54  -62.90  175.16   20.60\n",
      "  119        360K 360K N   CA   2769 2770  127.27  139.10                  -40.80\n",
      "  120  15517 361L 362A C   N    2784 2786  -59.41  -68.20    9.49    0.90  -62.50  170.25   27.75\n",
      "  120        362A 362A N   CA   2786 2787  148.88  145.30                  -40.90\n",
      "  121  15518 362A 363W C   N    2789 2791 -116.80 -124.90   25.70    1.40  -63.00  157.49   23.29\n",
      "  121        363W 363W N   CA   2791 2792  167.79  143.40                  -44.20\n",
      "  122  15520 364V 365L C   N    2810 2812 -150.59 -108.50   42.37    1.96  -63.50 -161.32   24.36\n",
      "  122        365L 365L N   CA   2812 2813  137.37  132.50                  -41.20\n",
      "  123  15521 365L 366Q C   N    2818 2820 -166.66 -121.10   51.69    1.60  -63.80 -173.48   32.61\n",
      "  123        366Q 366Q N   CA   2820 2821  164.11  139.70                  -40.30\n",
      "  124  15522 366Q 367I C   N    2827 2829   71.82 -120.60  168.11   11.23 -120.60  168.11   11.23\n",
      "  124        367I 367I N   CA   2829 2830  143.62  130.30                  130.30\n",
      "  125  15523 367I 368R C   N    2835 2837  167.49 -125.20   73.76    3.70  -63.00 -160.68   23.03\n",
      "  125        368R 368R N   CA   2837 2838  110.41  140.60                  -41.10\n",
      "  126  15525 369L 370D C   N    2854 2856 -166.87  -63.30  108.43   18.31  -63.30  108.43   18.31\n",
      "  126        370D 370D N   CA   2856 2857  -72.08  -40.00                  -40.00\n",
      "  127  15534 378A 379L C   N    2916 2918  -62.78  -63.50    1.43    0.24  -70.70  178.60   13.84\n",
      "  127        379L 379L N   CA   2918 2919  -39.97  -41.20                  141.60\n",
      "  128  15539 383I 384Q C   N    2954 2956  -65.90  -73.00   29.81    2.09  -63.80  150.06   22.17\n",
      "  128        384Q 384Q N   CA   2956 2957  169.65  140.70                  -40.30\n",
      "  129  15540 384Q 385G C   N    2963 2965  107.34   82.20   30.11    1.20  -80.20 -112.28    7.71\n",
      "  129        385G 385G N   CA   2965 2966   -8.07    8.50                  174.10\n",
      "  130  15542 386L 387R C   N    2975 2977 -118.54 -125.20   60.60    3.06  -63.00  130.48   21.42\n",
      "  130        387R 387R N   CA   2977 2978 -159.17  140.60                  -41.10\n",
      "  131  15543 387R 388G C   N    2986 2988 -152.85 -167.20   14.57    0.23   82.20 -154.17   15.28\n",
      "  131        388G 388G N   CA   2988 2989  172.07  174.60                    8.50\n",
      "  132  15544 388G 389E C   N    2990 2992  -66.59  -69.30    3.17    0.29  -63.60  175.57   23.57\n",
      "  132        389E 389E N   CA   2992 2993  144.15  142.50                  -40.30\n",
      "  133  15546 390I 391D C   N    3007 3009 -131.81  -96.50   36.51    1.49  -63.30  160.32   17.04\n",
      "  133        391D 391D N   CA   3009 3010  104.95  114.20                  -40.00\n",
      "  134  15548 392C 393M C   N    3021 3023 -119.76 -125.60   32.03    1.40  -63.40  159.79   20.30\n",
      "  134        393M 393M N   CA   3023 3024  109.01  140.50                  -40.50\n",
      "  135  15549 393M 394K C   N    3029 3031 -151.14 -118.00   47.33    1.68  -62.90  170.86   27.44\n",
      "  135        394K 394K N   CA   3031 3032  172.89  139.10                  -40.80\n",
      "  136  15552 396K 397L C   N    3053 3055  102.49 -108.50  149.20    7.20  -63.50 -125.17   42.30\n",
      "  136        397L 397L N   CA   3055 3056  124.90  132.50                  -41.20\n",
      "  137  15553 397L 398T C   N    3061 3063 -142.59 -124.80   30.21    0.94  -63.20  169.70   26.89\n",
      "  137        398T 398T N   CA   3063 3064  167.92  143.50                  -42.10\n",
      "  138  15555 399V 400A C   N    3075 3077 -104.85 -134.00   35.28    0.87  -62.50  173.28   26.66\n",
      "  138        400A 400A N   CA   3077 3078  127.12  147.00                  -40.90\n",
      "  139  15556 400A 401S C   N    3080 3082 -163.87 -136.60   30.74    0.94  -64.10 -171.77   20.84\n",
      "  139        401S 401S N   CA   3082 3083  165.38  151.20                  -35.00\n",
      "  140  15564 408E 409L C   N    3141 3143  -74.68  -63.50   51.58    6.61  -63.50   51.58    6.61\n",
      "  140        409L 409L N   CA   3143 3144    9.15  -41.20                  -41.20\n",
      "  141  15568 412A 413L C   N    3172 3174  -73.26  -63.50   19.25    3.27 -108.50  173.32    9.68\n",
      "  141        413L 413L N   CA   3174 3175  -57.80  -41.20                  132.50\n",
      "  142  15580 424R 425K C   N    3275 3277  -88.61 -118.00   72.48    3.75  -62.90  116.72   16.67\n",
      "  142        425K 425K N   CA   3277 3278 -154.65  139.10                  -40.80\n",
      "  143  15581 425K 426G C   N    3284 3286  153.36 -167.20   73.22    2.93   82.20  126.35    9.22\n",
      "  143        426G 426G N   CA   3286 3287  112.91  174.60                    8.50\n",
      "  144  15582 426G 427L C   N    3288 3290  -67.60  -70.70    9.95    0.65  -63.50  173.40   23.84\n",
      "  144        427L 427L N   CA   3290 3291  132.15  141.60                  -41.20\n",
      "  145  15584 428I 429E C   N    3304 3306   77.49  -69.30  148.22   11.93  -63.60 -149.19   24.84\n",
      "  145        429E 429E N   CA   3306 3307  163.06  142.50                  -40.30\n",
      "  146  15585 429E 430Q C   N    3313 3315  -71.46  -73.00   14.02    0.95  -63.80  165.24   24.70\n",
      "  146        430Q 430Q N   CA   3315 3316  154.63  140.70                  -40.30\n",
      "  147  15587 431R 432K C   N    3333 3335 -145.71 -118.00   35.79    1.22  -62.90  177.90   28.05\n",
      "  147        432K 432K N   CA   3335 3336  161.75  139.10                  -40.80\n",
      "  148  15589 433I 434S C   N    3350 3352 -148.99 -136.60   43.05    2.38  -64.10  167.99   11.10\n",
      "  148        434S 434S N   CA   3352 3353  109.97  151.20                  -35.00\n",
      "  149  15590 434S 435G C   N    3356 3358   57.14   78.70   26.86    0.76   82.20  160.56    9.03\n",
      "  149        435G 435G N   CA   3358 3359 -150.09 -166.10                    8.50\n",
      "  150  15591 435G 436F C   N    3360 3362 -132.32 -124.20    8.59    0.26  -63.20 -176.87   29.96\n",
      "  150        436F 436F N   CA   3362 3363  146.12  143.30                  -44.30\n",
      "  151  15592 436F 437S C   N    3371 3373  -70.57  -72.40   17.55    0.97  -64.10  170.07   11.95\n",
      "  151        437S 437S N   CA   3373 3374  134.94  152.40                  -35.00\n",
      "  152  15593 437S 438A C   N    3377 3379 -137.95 -134.00    5.84    0.35  -62.50 -168.14   35.48\n",
      "  152        438A 438A N   CA   3379 3380  142.70  147.00                  -40.90\n",
      "  153  15594 438A 439A C   N    3382 3384  -82.38  -68.20   23.86    1.51  -62.50  155.88   26.57\n",
      "  153        439A 439A N   CA   3384 3385  164.49  145.30                  -40.90\n",
      "  154  15595 439A 440L C   N    3387 3389  -64.41  -70.70    6.52    0.64  -63.50  175.50   24.42\n",
      "  154        440L 440L N   CA   3389 3390  143.30  141.60                  -41.20\n",
      "  155  15598 442C 443F C   N    3408 3410  -89.33  -71.40   30.89    2.75  -63.20  161.97   20.80\n",
      "  155        443F 443F N   CA   3410 3411  115.55  140.70                  -44.30\n",
      "  156  15599 443F 444S C   N    3419 3421  106.71 -136.60  124.54    3.98  -64.10 -145.17   19.44\n",
      "  156        444S 444S N   CA   3421 3422 -165.28  151.20                  -35.00\n",
      "  157  15600 444S 445G C   N    3425 3427  100.55   78.70   37.27    0.64   82.20  156.28    8.50\n",
      "  157        445G 445G N   CA   3427 3428  163.70 -166.10                    8.50\n",
      "  158  15601 445G 446T C   N    3429 3431 -108.98 -124.80   31.44    0.98  -63.20  164.91   18.50\n",
      "  158        446T 446T N   CA   3431 3432  116.33  143.50                  -42.10\n",
      "  159  15603 447C 448K C   N    3442 3444  -55.34  -70.20   15.40    1.21  -62.90  174.91   22.31\n",
      "  159        448K 448K N   CA   3444 3445  144.45  140.40                  -40.80\n",
      "  160  15604 448K 449T C   N    3451 3453 -123.87 -124.80    8.95    0.41  -63.20  176.27   26.62\n",
      "  160        449T 449T N   CA   3453 3454  152.39  143.50                  -42.10\n",
      "  161  15605 449T 450S C   N    3458 3460 -137.23 -136.60   11.45    0.59  -64.10 -170.55   11.77\n",
      "  161        450S 450S N   CA   3460 3461  139.77  151.20                  -35.00\n",
      "  162  15606 450S 451L C   N    3464 3466 -129.73 -108.50   34.10    1.57  -63.50  172.81   28.10\n",
      "  162        451L 451L N   CA   3466 3467  159.19  132.50                  -41.20\n",
      "  163  15607 451L 452A C   N    3472 3474  -64.28  -62.50   20.84    3.33  -68.20  165.48   13.21\n",
      "  163        452A 452A N   CA   3474 3475  -20.13  -40.90                  145.30\n",
      "  164  15611 455P 456F C   N    3500 3502   55.70  -71.40  142.33   11.65  -63.20  162.62   18.95\n",
      "  164        456F 456F N   CA   3502 3503 -155.25  140.70                  -44.30\n",
      "  165  15612 456F 457G C   N    3511 3513  -80.99  -80.20   89.78    3.57   82.20  179.95    8.53\n",
      "  165        457G 457G N   CA   3513 3514   84.32  174.10                    8.50\n",
      "  166  15618 462T 463T C   N    3556 3558  -33.17  -78.10   46.19    2.50  -63.20 -178.68   21.40\n",
      "  166        463T 463T N   CA   3558 3559  139.08  149.80                  -42.10\n",
      "  167  15619 463T 464L C   N    3563 3565 -100.70  -70.70   67.42    4.22  -63.50  122.59   19.32\n",
      "  167        464L 464L N   CA   3565 3566 -158.02  141.60                  -41.20\n",
      "  168  15625 469Y 470M C   N    3610 3612 -163.84 -125.60   45.38    1.34  -63.40 -175.67   35.20\n",
      "  168        470M 470M N   CA   3612 3613  164.94  140.50                  -40.50\n",
      "  169  15626 470M 471L C   N    3618 3620  -84.29  -70.70   28.02    1.74  -63.50  154.10   22.64\n",
      "  169        471L 471L N   CA   3620 3621  166.10  141.60                  -41.20\n",
      "  170  15627 471L 472A C   N    3626 3628  -63.03  -68.20    9.69    0.93  -62.50  165.61   27.17\n",
      "  170        472A 472A N   CA   3628 3629  153.50  145.30                  -40.90\n",
      "  171  15628 472A 473G C   N    3631 3633   87.92   78.70   13.60    0.29   82.20  175.50    8.81\n",
      "  171        473G 473G N   CA   3633 3634 -176.09 -166.10                    8.50\n",
      "  172  15632 476P 477F C   N    3656 3658  -80.27  -71.40   12.22    0.70  -63.20  167.47   24.41\n",
      "  172        477F 477F N   CA   3658 3659  149.11  140.70                  -44.30\n",
      "  173  15633 477F 478L C   N    3667 3669   69.75  -70.70  142.05   13.53  -63.50 -154.88   25.83\n",
      "  173        478L 478L N   CA   3669 3670  162.85  141.60                  -41.20\n",
      "  174  15636 480C 481H C   N    3689 3691  -71.37  -67.60   14.53    1.00  -63.20  163.87   21.00\n",
      "  174        481H 481H N   CA   3691 3692  154.03  140.00                  -42.30\n",
      "  175  15638 482I 483H C   N    3707 3709  -67.99  -67.60    4.65    0.35  -63.20  173.13   21.95\n",
      "  175        483H 483H N   CA   3709 3710  144.63  140.00                  -42.30\n",
      "  176  15640 484V 485Y C   N    3724 3726 -138.15 -124.30   56.91    2.48  -63.50  146.45   26.80\n",
      "  176        485Y 485Y N   CA   3726 3727 -169.40  135.40                  -43.40\n",
      "  177  15641 485Y 486L C   N    3736 3738  -68.79  -70.70   75.60    5.80  -63.50  101.76   14.44\n",
      "  177        486L 486L N   CA   3738 3739 -142.82  141.60                  -41.20\n",
      "  178  15642 486L 487T C   N    3744 3746  -54.57  -78.10   25.19    1.26  -63.20  177.28   22.18\n",
      "  178        487T 487T N   CA   3746 3747  140.83  149.80                  -42.10\n",
      "  179  15643 487T 488E C   N    3751 3753  -64.36  -69.30   10.25    0.61  -63.60  173.82   23.13\n",
      "  179        488E 488E N   CA   3753 3754  133.52  142.50                  -40.30\n",
      "  180  15645 489E 490S C   N    3769 3771 -118.11 -136.60   18.53    0.77  -64.10 -179.21   16.86\n",
      "  180        490S 490S N   CA   3771 3772  152.46  151.20                  -35.00\n",
      "  181  15646 490S 491S C   N    3775 3777 -109.06 -136.60   35.26    1.09  -64.10  170.21   10.55\n",
      "  181        491S 491S N   CA   3777 3778  129.17  151.20                  -35.00\n",
      "  182  15648 492V 493A C   N    3788 3790  175.57 -134.00   58.72    1.38  -62.50 -172.82   36.99\n",
      "  182        493A 493A N   CA   3790 3791  177.08  147.00                  -40.90\n",
      "  183  15651 495L 496L C   N    3808 3810  -67.51  -63.50   64.41    9.18  -63.50   64.41    9.18\n",
      "  183        496L 496L N   CA   3810 3811 -105.49  -41.20                  -41.20\n",
      "  184  15655 499R 500E C   N    3846 3848  159.48 -117.80   93.62    2.74  -63.60 -164.86   33.04\n",
      "  184        500E 500E N   CA   3848 3849 -179.34  136.80                  -40.30\n",
      "  185  15656 500E 501L C   N    3855 3857  -72.81  -70.70    4.65    0.29  -63.50  173.30   24.61\n",
      "  185        501L 501L N   CA   3857 3858  145.75  141.60                  -41.20\n",
      "  186  15657 501L 502Q C   N    3863 3865  175.22 -121.10   81.63    2.56  -63.80  176.80   31.72\n",
      "  186        502Q 502Q N   CA   3865 3866 -169.23  139.70                  -40.30\n",
      "  187  15659 503H 504L C   N    3882 3884  -71.98  -70.70    9.39    0.66  -63.50  168.11   23.84\n",
      "  187        504L 504L N   CA   3884 3885  150.90  141.60                  -41.20\n",
      "  188  15661 505P 506D C   N    3897 3899  -82.10  -70.90   14.71    0.64  -63.30  161.26   21.08\n",
      "  188        506D 506D N   CA   3899 3900  159.84  150.30                  -40.00\n",
      "  189  15662 506D 507Y C   N    3905 3907 -150.57  -63.50  101.11   19.10  -63.50  101.11   19.10\n",
      "  189        507Y 507Y N   CA   3907 3908  -94.80  -43.40                  -43.40\n",
      "  190  15670 514V 515T C   N    3960 3962 -108.19 -124.80   19.20    1.16  -63.20  170.81   24.90\n",
      "  190        515T 515T N   CA   3962 3963  153.12  143.50                  -42.10\n",
      "  191  15672 516V 517W C   N    3974 3976 -132.21 -124.90   22.72    1.24  -63.00  179.93   19.79\n",
      "  191        517W 517W N   CA   3976 3977  121.89  143.40                  -44.20\n",
      "  192  15673 517W 518F C   N    3988 3990 -146.93 -124.20   22.75    0.92  -63.20 -167.50   32.13\n",
      "  192        518F 518F N   CA   3990 3991  142.37  143.30                  -44.30\n",
      "  193  15674 518F 519E C   N    3999 4001  178.94 -117.80   76.11    2.25  -63.60 -176.81   30.63\n",
      "  193        519E 519E N   CA   4001 4002  179.12  136.80                  -40.30\n",
      "  194  15675 519E 520G C   N    4008 4010  103.45   78.70   56.33    0.84   82.20  136.46    7.67\n",
      "  194        520G 520G N   CA   4010 4011  143.29 -166.10                    8.50\n",
      "  195  15676 520G 521H C   N    4012 4014  -74.68  -67.60    8.75    0.52  -63.20  172.93   22.35\n",
      "  195        521H 521H N   CA   4014 4015  145.15  140.00                  -42.30\n",
      "  196  15677 521H 522Y C   N    4022 4024 -137.74 -124.30   36.36    1.47  -63.50  165.05   29.81\n",
      "  196        522Y 522Y N   CA   4024 4025  169.18  135.40                  -43.40\n",
      "  197  15678 522Y 523S C   N    4034 4036  -69.83  -72.40   19.00    1.21  -64.10  153.88   11.50\n",
      "  197        523S 523S N   CA   4036 4037  171.23  152.40                  -35.00\n",
      "  198  15679 523S 524S C   N    4040 4042  -60.64  -72.40   23.11    1.06  -64.10  167.55   12.35\n",
      "  198        524S 524S N   CA   4042 4043  132.51  152.40                  -35.00\n",
      "  199  15680 524S 525A C   N    4046 4048 -167.18 -134.00   35.62    0.82  -62.50 -169.49   36.77\n",
      "  199        525A 525A N   CA   4048 4049  159.93  147.00                  -40.90\n",
      "  200  15681 525A 526V C   N    4051 4053  -61.67  -62.40    8.89    1.08 -125.40  177.29    9.98\n",
      "  200        526V 526V N   CA   4053 4054  -51.26  -42.40                  143.30\n",
      "  201  15682 526V 527Q C   N    4058 4060  168.56 -121.10   71.11    2.47  -63.80 -147.75   37.51\n",
      "  201        527Q 527Q N   CA   4060 4061  150.12  139.70                  -40.30\n",
      "  202  15683 527Q 528T C   N    4067 4069 -114.69 -124.80   46.63    1.71  -63.20  149.24   16.33\n",
      "  202        528T 528T N   CA   4069 4070   97.98  143.50                  -42.10\n",
      "  203  15687 531V 532R C   N    4096 4098  -48.71 -125.20   76.49    3.07  -63.00  179.12   23.27\n",
      "  203        532R 532R N   CA   4098 4099  140.36  140.60                  -41.10\n",
      "  204  15689 533P 534N C   N    4114 4116  -94.79  -71.20   43.62    2.32  -63.20  142.94   19.67\n",
      "  204        534N 534N N   CA   4116 4117  179.49  142.80                  -41.10\n",
      "  205  15690 534N 535G C   N    4122 4124 -134.75 -167.20   32.57    0.69   82.20 -138.70   16.67\n",
      "  205        535G 535G N   CA   4124 4125  177.36  174.60                    8.50\n",
      "  206  15691 535G 536G C   N    4126 4128 -160.14 -167.20   13.31    0.53   82.20 -147.14   15.45\n",
      "  206        536G 536G N   CA   4128 4129 -174.11  174.60                    8.50\n",
      "  207  15692 536G 537N C   N    4130 4132  -97.12 -119.90   30.46    0.88  -63.20  161.47   18.52\n",
      "  207        537N 537N N   CA   4132 4133  116.76  137.00                  -41.10\n",
      "  208  15694 538P 539Q C   N    4145 4147 -142.59 -121.10   76.88    3.94  -63.80  132.23   16.96\n",
      "  208        539Q 539Q N   CA   4147 4148   65.88  139.70                  -40.30\n",
      "  209  15695 539Q 540F C   N    4154 4156 -160.43 -124.20   36.81    1.25  -63.20 -167.72   32.82\n",
      "  209        540F 540F N   CA   4156 4157  149.81  143.30                  -44.30\n",
      "  210  15696 540F 541S C   N    4165 4167  -93.65  -72.40   23.35    1.34  -64.10  165.58   13.96\n",
      "  210        541S 541S N   CA   4167 4168  162.07  152.40                  -35.00\n",
      "  211  15697 541S 542L C   N    4171 4173 -121.31 -108.50   20.52    0.94  -63.50  179.81   28.54\n",
      "  211        542L 542L N   CA   4173 4174  148.54  132.50                  -41.20\n",
      "  212  15698 542L 543R C   N    4179 4181  -71.71  -72.10    1.60    0.11  -63.00  178.76   24.74\n",
      "  212        543R 543R N   CA   4181 4182  140.35  141.90                  -41.10\n",
      "  213  15699 543R 544Q C   N    4190 4192  -90.95  -73.00   22.38    1.33  -63.80  167.84   26.13\n",
      "  213        544Q 544Q N   CA   4192 4193  154.06  140.70                  -40.30\n",
      "  214  15700 544Q 545M C   N    4199 4201 -129.93 -125.60   28.42    1.26  -63.40  164.92   29.87\n",
      "  214        545M 545M N   CA   4201 4202  168.59  140.50                  -40.50\n",
      "  215  15701 545M 546Y C   N    4207 4209 -104.90  -98.40   19.86    1.98  -63.50  158.53   23.19\n",
      "  215        546Y 546Y N   CA   4209 4210  109.63  128.40                  -43.40\n",
      "  216  15702 546Y 547L C   N    4219 4221 -143.85 -108.50   73.95    3.55  -63.50  145.53   25.11\n",
      "  216        547L 547L N   CA   4221 4222 -162.54  132.50                  -41.20\n",
      "  217  15703 547L 548F C   N    4227 4229  -72.16  -71.40   42.50    3.41  -63.20  132.81   19.05\n",
      "  217        548F 548F N   CA   4229 4230 -176.81  140.70                  -44.30\n",
      "  218  15704 548F 549G C   N    4238 4240  -98.10  -62.40   98.13   13.41   82.20 -175.53   13.65\n",
      "  218        549G 549G N   CA   4240 4241   50.20  -41.20                    8.50\n",
      "  219  15706 550P 551H C   N    4249 4251  -74.42  -67.60   11.42    0.69  -63.20  168.92   21.83\n",
      "  219        551H 551H N   CA   4251 4252  149.15  140.00                  -42.30\n",
      "  220  15707 551H 552T C   N    4259 4261  -88.90  -78.10   35.64    1.89  -63.20  160.01   18.93\n",
      "  220        552T 552T N   CA   4261 4262  115.83  149.80                  -42.10\n",
      "  221  15708 552T 553E C   N    4266 4268 -106.67 -117.80   18.61    0.65  -63.60  167.80   20.32\n",
      "  221        553E 553E N   CA   4268 4269  121.88  136.80                  -40.30\n",
      "  222  15709 553E 554D C   N    4275 4277 -123.22  -96.50   28.92    1.18  -63.30  155.17   16.58\n",
      "  222        554D 554D N   CA   4277 4278  103.14  114.20                  -40.00\n",
      "  223  15710 554D 555L C   N    4283 4285  -76.21  -70.70   31.79    2.62  -63.50  152.03   20.41\n",
      "  223        555L 555L N   CA   4285 4286  110.29  141.60                  -41.20\n",
      "  224  15711 555L 556E C   N    4291 4293 -127.43 -117.80   32.83    1.44  -63.60  164.41   25.51\n",
      "  224        556E 556E N   CA   4293 4294  168.19  136.80                  -40.30\n",
      "  225  15712 556E 557R C   N    4300 4302 -171.36 -125.20   47.04    2.12  -63.00 -156.20   23.06\n",
      "  225        557R 557R N   CA   4302 4303  131.51  140.60                  -41.10\n",
      "  226  15713 557R 558F C   N    4311 4313 -120.19 -124.20   19.25    0.80  -63.20  178.14   21.31\n",
      "  226        558F 558F N   CA   4313 4314  124.48  143.30                  -44.30\n",
      "  227  15714 558F 559F C   N    4322 4324  -65.81  -71.40    6.48    0.36  -63.20  178.30   24.93\n",
      "  227        559F 559F N   CA   4324 4325  137.42  140.70                  -44.30\n",
      "  228  15715 559F 560Q C   N    4333 4335  -69.21  -73.00   29.32    1.99  -63.80  150.02   22.34\n",
      "  228        560Q 560Q N   CA   4335 4336  169.78  140.70                  -40.30\n",
      "  229  15716 560Q 561L C   N    4342 4344  -77.61 -108.50   58.65    2.77  -63.50  124.65   16.55\n",
      "  229        561L 561L N   CA   4344 4345   82.65  132.50                  -41.20\n",
      "  230  15717 561L 562D C   N    4350 4352   47.61   54.50   13.34    0.66  -63.30  144.30   24.33\n",
      "  230        562D 562D N   CA   4352 4353   52.31   40.90                  -40.00\n",
      "  231  15719 563V 564L C   N    4365 4367  -73.89  -70.70   18.45    1.52  -63.50  164.96   22.33\n",
      "  231        564L 564L N   CA   4367 4368  123.43  141.60                  -41.20\n",
      "  232  15720 564L 565N C   N    4373 4375 -177.24 -119.90  115.67    3.81  -63.20  140.15   23.00\n",
      "  232        565N 565N N   CA   4375 4376 -122.55  137.00                  -41.10\n",
      "  233  15721 565N 566F C   N    4381 4383  -59.09  -71.40   22.92    2.04  -63.20  155.72   21.35\n",
      "  233        566F 566F N   CA   4383 4384  160.04  140.70                  -44.30\n",
      "  234  15722 566F 567R C   N    4392 4394  -61.41  -72.10   12.39    1.06  -63.00  170.76   22.96\n",
      "  234        567R 567R N   CA   4394 4395  148.15  141.90                  -41.10\n",
      "  235  15724 568V 569E C   N    4410 4412  -89.35  -69.30   25.40    1.43  -63.60  163.64   23.27\n",
      "  235        569E 569E N   CA   4412 4413  158.09  142.50                  -40.30\n",
      "  236  15725 569E 570G C   N    4419 4421  -76.78  -80.20    3.86    0.28   82.20 -129.16    7.03\n",
      "  236        570G 570G N   CA   4421 4422  175.87  174.10                    8.50\n",
      "  237  15726 570G 571I C   N    4423 4425  -61.43  -63.40    2.10    0.39 -120.60 -177.00    8.32\n",
      "  237        571I 571I N   CA   4425 4426  -42.87  -43.60                  130.30\n",
      "  238  15727 571I 572M C   N    4431 4433  -68.41  -73.00    5.98    0.48  -63.40  172.75   26.18\n",
      "  238        572M 572M N   CA   4433 4434  146.82  143.00                  -40.50\n",
      "  239  15728 572M 573S C   N    4439 4441 -114.94 -136.60   37.10    1.31  -64.10  164.15   10.11\n",
      "  239        573S 573S N   CA   4441 4442  121.08  151.20                  -35.00\n",
      "  240  15729 573S 574D C   N    4445 4447 -132.93  -96.50   36.55    1.49  -63.30  171.90   18.31\n",
      "  240        574D 574D N   CA   4447 4448  117.16  114.20                  -40.00\n",
      "  241  15730 574D 575L C   N    4453 4455 -127.29 -108.50   29.38    1.35  -63.50  175.70   28.35\n",
      "  241        575L 575L N   CA   4455 4456  155.09  132.50                  -41.20\n",
      "  242  15731 575L 576T C   N    4461 4463 -104.59  -78.10   67.30    2.23  -63.20  114.01   17.35\n",
      "  242        576T 576T N   CA   4463 4464 -148.33  149.80                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3   70  125  484  403  678  501  595  660  732  797\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00199.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      577     4476     4476\n",
      "openf___224_> Open           EAO00199.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     47436912   46325.109    45.239\n",
      "read_mo_297_> Segments, residues, atoms:        1      577     4476\n",
      "read_mo_298_> Segment:        1  1:A  577:A     4476\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4476     4476        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    21012\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22310\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22981\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    23873\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    24919\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    25728\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    27228\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    28107\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    29246\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    30620\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    31777\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33640\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33781\n",
      "iupac_m_487_> NH1/2 swapped:      179.3608       16  16\n",
      "iupac_m_487_> NH1/2 swapped:      179.5550       21  21\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.7121       31  31\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.8081       33  33\n",
      "iupac_m_486_> OE1/2 will be swapped:      124.9461       43  43\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5748       46  46\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0964       55  55\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.5047       62  62\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.7585       66  66\n",
      "iupac_m_485_> OD1/2 will be swapped:      150.8639      101  101\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1781      113  113\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.0375      120  120\n",
      "iupac_m_487_> NH1/2 swapped:      178.3916      125  125\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.4945      127  127\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.2996      137  137\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.4569      149  149\n",
      "iupac_m_485_> OD1/2 will be swapped:     -155.2959      150  150\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1382      155  155\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8287      157  157\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.3173      162  162\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9802      173  173\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.1980      175  175\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.1657      188  188\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.4138      205  205\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.8229      210  210\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.8505      236  236\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7553      249  249\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      103.2224      260  260\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.3962      261  261\n",
      "iupac_m_485_> OD1/2 will be swapped:      115.3944      282  282\n",
      "iupac_m_487_> NH1/2 swapped:     -175.9510      283  283\n",
      "iupac_m_485_> OD1/2 will be swapped:      131.4810      287  287\n",
      "iupac_m_487_> NH1/2 swapped:      178.8332      293  293\n",
      "iupac_m_487_> NH1/2 swapped:      179.1197      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.6580      303  303\n",
      "iupac_m_486_> OE1/2 will be swapped:      138.1778      332  332\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.3266      355  355\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.4839\n",
      "iupac_m_485_> OD1/2 will be swapped:      148.0892      356  356\n",
      "iupac_m_486_> OE1/2 will be swapped:     -107.2793      357  357\n",
      "iupac_m_487_> NH1/2 swapped:      178.9987      387  387\n",
      "iupac_m_485_> OD1/2 will be swapped:      126.0171      402  402\n",
      "iupac_m_486_> OE1/2 will be swapped:     -112.9678      408  408\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.2707      414  414\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8253      416  416\n",
      "iupac_m_486_> OE1/2 will be swapped:      131.8892      417  417\n",
      "iupac_m_487_> NH1/2 swapped:     -169.6131      424  424\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.4073      488  488\n",
      "iupac_m_486_> OE1/2 will be swapped:      142.3443      489  489\n",
      "iupac_m_487_> NH1/2 swapped:     -173.3893      499  499\n",
      "iupac_m_486_> OE1/2 will be swapped:      149.6894      500  500\n",
      "iupac_m_485_> OD1/2 will be swapped:      155.6712      506  506\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      109.5213      507  507\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      176.6025      543  543\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.3216      556  556\n",
      "iupac_m_487_> NH1/2 swapped:      179.6103      557  557\n",
      "iupac_m_487_> NH1/2 swapped:     -177.2196      567  567\n",
      "iupac_m_485_> OD1/2 will be swapped:      139.4167      574  574\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   950669\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -41533.2734\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -41533.273438\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    32.907001\n",
      "Sequence length          :          577\n",
      "Compactness              :     0.110550\n",
      "Native energy (pair)     :   686.842125\n",
      "Native energy (surface)  :    53.097920\n",
      "Native energy (combined) :    19.300198\n",
      "Z score (pair)           :    -4.109617\n",
      "Z score (surface)        :    -1.747872\n",
      "Z score (combined)       :    -3.783464\n",
      "GA341 score              :     0.655619\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    11299\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        5297.7935\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4546       0      0   0.006   0.006      47.025       1.000\n",
      " 2 Bond angle potential               :    6129       2     37   2.135   2.135      531.06       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2670       0    131  51.521  51.521      1142.9       1.000\n",
      " 4 Stereochemical improper torsion pot:    1811       0      3   1.472   1.472      81.256       1.000\n",
      " 5 Soft-sphere overlap restraints     :   11299       0      0   0.005   0.005      39.570       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6300       3     35   0.791   0.791      620.66       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6434       9     88   1.024   1.024      965.11       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     576       2     19   5.866   5.866      233.19       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     488       0      9  83.558  83.558      175.80       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     356       0      4  79.467  79.467      166.37       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     173       0      2  89.078  89.078      135.36       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      65       0      0  98.338  98.338      39.607       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2777       0      0   0.680   0.680      80.894       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     575     248    131  41.096 112.395      925.72       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     881       0      2   1.519   1.519      113.31       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00199.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00199.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   93544.2422\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8388 366Q 366Q N   CA   2820 2821  123.28  107.00   16.28    4.68  107.00   16.28    4.68\n",
      "    2  10168 533P 533P N   CA   4109 4110  131.81  108.20   23.61    5.37  108.20   23.61    5.37\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19902  89Q 496L CA  CA    677 3811   12.20    9.44    2.76    4.87    9.44    2.76    4.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24549  51F  36R N   O     367  271    4.38    2.83    1.54    4.85    2.83    1.54    4.85\n",
      "    2  24609  53F 498L N   O     387 3836   11.93    8.62    3.31    4.61    8.62    3.31    4.61\n",
      "    3  25469  89Q 341T N   O     676 2621    9.25    5.85    3.40    5.07    5.85    3.40    5.07\n",
      "    4  28470 342N 144Y N   O    2622 1108   10.35    6.52    3.83    5.11    6.52    3.83    5.11\n",
      "    5  29191 413L 577S N   O    3174 4475    5.69    3.36    2.33    4.82    3.36    2.33    4.82\n",
      "    6  29560 452A  11I N   O    3474   76   12.01    9.09    2.92    4.53    9.09    2.92    4.53\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16097 366Q 366Q CA  C    2821 2827 -153.73 -180.00   26.27    5.25 -180.00   26.27    5.25\n",
      "    2  16228 497R 497R CA  C    3819 3827 -112.28 -180.00   67.72   13.54 -180.00   67.72   13.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15157   1M   2G C   N       7    9 -172.54   82.20  105.55    7.35   82.20  105.55    7.35\n",
      "    1          2G   2G N   CA      9   10   16.38    8.50                    8.50\n",
      "    2  15176  20P  21R C   N     141  143  -78.02 -125.20   50.49    1.61  -63.00  164.42   21.24\n",
      "    2         21R  21R N   CA    143  144  122.64  140.60                  -41.10\n",
      "    3  15177  21R  22Q C   N     152  154  -73.15  -73.00   13.61    0.89  -63.80  165.65   24.85\n",
      "    3         22Q  22Q N   CA    154  155  154.31  140.70                  -40.30\n",
      "    4  15186  30A  31D C   N     220  222  -90.55  -96.50   42.61    1.79  -63.30  115.27   12.82\n",
      "    4         31D  31D N   CA    222  223   72.01  114.20                  -40.00\n",
      "    5  15189  33D  34D C   N     242  244 -152.90  -96.50  104.30    4.31  -63.30  111.56   13.44\n",
      "    5         34D  34D N   CA    244  245   26.47  114.20                  -40.00\n",
      "    6  15199  43E  44G C   N     320  322   44.69   82.20   83.45    2.39  -62.40  164.03   30.04\n",
      "    6         44G  44G N   CA    322  323   83.04    8.50                  -41.20\n",
      "    7  15200  44G  45G C   N     324  326  115.75   82.20   43.09    1.53  -62.40  179.59   32.90\n",
      "    7         45G  45G N   CA    326  327  -18.53    8.50                  -41.20\n",
      "    8  15202  46R  47A C   N     339  341   84.11  -62.50  160.84   27.50  -68.20 -173.49   18.34\n",
      "    8         47A  47A N   CA    341  342 -107.05  -40.90                  145.30\n",
      "    9  15203  47A  48S C   N     344  346   60.51   56.90   16.77    1.38  -72.40  166.10    8.40\n",
      "    9         48S  48S N   CA    346  347   52.78   36.40                  152.40\n",
      "   10  15213  57F  58T C   N     433  435 -143.12  -63.20   90.37   11.35 -124.80  144.59    6.75\n",
      "   10         58T  58T N   CA    435  436    0.07  -42.10                  143.50\n",
      "   11  15260 104S 105K C   N     789  791  -66.97  -70.20   20.17    1.53   56.60  173.44   11.15\n",
      "   11        105K 105K N   CA    791  792  160.30  140.40                   38.60\n",
      "   12  15261 105K 106E C   N     798  800  -69.17  -69.30   36.66    2.80  -63.60  140.65   19.06\n",
      "   12        106E 106E N   CA    800  801  179.16  142.50                  -40.30\n",
      "   13  15262 106E 107G C   N     807  809  123.57   82.20   42.44    2.45  -62.40  178.61   30.03\n",
      "   13        107G 107G N   CA    809  810   -1.01    8.50                  -41.20\n",
      "   14  15275 119I 120E C   N     906  908 -103.77 -117.80   82.13    3.84  -63.60  104.24   12.21\n",
      "   14        120E 120E N   CA    908  909   55.88  136.80                  -40.30\n",
      "   15  15276 120E 121E C   N     915  917 -147.88 -117.80   66.15    2.60  -63.60  149.91   24.57\n",
      "   15        121E 121E N   CA    917  918 -164.28  136.80                  -40.30\n",
      "   16  15300 144Y 145M C   N    1107 1109   75.14  -73.00  159.72   12.74   56.30  168.97   13.65\n",
      "   16        145M 145M N   CA   1109 1110 -157.28  143.00                   34.80\n",
      "   17  15301 145M 146E C   N    1115 1117  -71.45  -63.60   20.87    2.45   54.60  141.08   18.48\n",
      "   17        146E 146E N   CA   1117 1118  -20.96  -40.30                   42.40\n",
      "   18  15311 155R 156A C   N    1198 1200 -125.26 -134.00   21.25    0.93  -62.50  179.83   27.18\n",
      "   18        156A 156A N   CA   1200 1201  127.63  147.00                  -40.90\n",
      "   19  15312 156A 157R C   N    1203 1205   48.72  -72.10  126.53   10.42 -125.20  178.21    8.15\n",
      "   19        157R 157R N   CA   1205 1206  179.47  141.90                  140.60\n",
      "   20  15314 158G 159Q C   N    1218 1220  -98.56 -121.10   43.07    1.57  -63.80  147.45   20.02\n",
      "   20        159Q 159Q N   CA   1220 1221  103.00  139.70                  -40.30\n",
      "   21  15315 159Q 160E C   N    1227 1229  178.55 -117.80   72.29    2.11  -63.60 -170.32   31.56\n",
      "   21        160E 160E N   CA   1229 1230  171.08  136.80                  -40.30\n",
      "   22  15317 161P 162E C   N    1243 1245   32.29  -69.30  111.98    6.82  -63.60  166.14   27.34\n",
      "   22        162E 162E N   CA   1245 1246   95.38  142.50                  -40.30\n",
      "   23  15318 162E 163S C   N    1252 1254 -136.34 -136.60   14.27    0.73  -64.10  175.12   17.84\n",
      "   23        163S 163S N   CA   1254 1255  165.47  151.20                  -35.00\n",
      "   24  15325 169D 170S C   N    1310 1312  -96.23 -136.60   47.47    1.45  -64.10  164.38   10.44\n",
      "   24        170S 170S N   CA   1312 1313  126.21  151.20                  -35.00\n",
      "   25  15326 170S 171Q C   N    1316 1318 -111.24 -121.10   17.14    0.61  -63.80  172.62   23.17\n",
      "   25        171Q 171Q N   CA   1318 1319  125.67  139.70                  -40.30\n",
      "   26  15327 171Q 172H C   N    1325 1327 -112.71 -125.60   14.72    0.38  -63.20 -179.10   20.03\n",
      "   26        172H 172H N   CA   1327 1328  131.69  138.80                  -42.30\n",
      "   27  15328 172H 173R C   N    1335 1337 -151.02 -125.20   32.23    1.73  -63.00 -175.27   20.93\n",
      "   27        173R 173R N   CA   1337 1338  121.31  140.60                  -41.10\n",
      "   28  15330 174V 175Y C   N    1353 1355 -108.74  -98.40   14.09    1.05  -63.50  168.42   24.58\n",
      "   28        175Y 175Y N   CA   1355 1356  118.83  128.40                  -43.40\n",
      "   29  15332 176V 177K C   N    1372 1374 -110.34 -118.00    8.46    0.28  -62.90 -177.41   21.59\n",
      "   29        177K 177K N   CA   1374 1375  135.52  139.10                  -40.80\n",
      "   30  15334 178G 179A C   N    1385 1387  -76.89  -68.20   12.06    0.76  -62.50  166.06   27.94\n",
      "   30        179A 179A N   CA   1387 1388  153.67  145.30                  -40.90\n",
      "   31  15335 179A 180G C   N    1390 1392 -147.29 -167.20   26.62    0.55   82.20 -162.35   14.95\n",
      "   31        180G 180G N   CA   1392 1393  156.93  174.60                    8.50\n",
      "   32  15337 181V 182H C   N    1401 1403 -110.02 -125.60   29.51    1.33  -63.20  160.81   23.17\n",
      "   32        182H 182H N   CA   1403 1404  163.86  138.80                  -42.30\n",
      "   33  15338 182H 183S C   N    1411 1413  -61.85  -72.40   20.52    0.94  -64.10  169.82   12.44\n",
      "   33        183S 183S N   CA   1413 1414  134.80  152.40                  -35.00\n",
      "   34  15341 185L 186S C   N    1432 1434 -131.96 -136.60   17.47    0.78  -64.10 -177.55   11.30\n",
      "   34        186S 186S N   CA   1434 1435  134.36  151.20                  -35.00\n",
      "   35  15343 187L 188E C   N    1446 1448 -147.28 -117.80   72.76    2.97  -63.60  143.34   23.63\n",
      "   35        188E 188E N   CA   1448 1449 -156.68  136.80                  -40.30\n",
      "   36  15352 196K 197G C   N    1525 1527 -109.44  -62.40  101.44   13.77   82.20  173.08   12.83\n",
      "   36        197G 197G N   CA   1527 1528   48.67  -41.20                    8.50\n",
      "   37  15353 197G 198N C   N    1529 1531 -139.56 -119.90   21.26    1.06  -63.20 -173.63   20.21\n",
      "   37        198N 198N N   CA   1531 1532  128.91  137.00                  -41.10\n",
      "   38  15354 198N 199A C   N    1537 1539 -157.46 -134.00   23.50    0.83  -62.50 -162.29   37.48\n",
      "   38        199A 199A N   CA   1539 1540  145.69  147.00                  -40.90\n",
      "   39  15355 199A 200N C   N    1542 1544 -127.51 -119.90   32.53    1.63  -63.20  159.97   17.37\n",
      "   39        200N 200N N   CA   1544 1545  105.38  137.00                  -41.10\n",
      "   40  15356 200N 201R C   N    1550 1552 -172.30 -125.20   50.85    1.60  -63.00 -166.94   33.29\n",
      "   40        201R 201R N   CA   1552 1553  159.76  140.60                  -41.10\n",
      "   41  15357 201R 202Q C   N    1561 1563 -117.18 -121.10   18.99    0.83  -63.80  170.02   22.58\n",
      "   41        202Q 202Q N   CA   1563 1564  121.12  139.70                  -40.30\n",
      "   42  15358 202Q 203T C   N    1570 1572 -137.89 -124.80   18.28    0.59  -63.20  178.07   27.72\n",
      "   42        203T 203T N   CA   1572 1573  156.25  143.50                  -42.10\n",
      "   43  15359 203T 204G C   N    1577 1579 -138.21 -167.20   53.76    1.51   82.20 -175.38   14.31\n",
      "   43        204G 204G N   CA   1579 1580  129.33  174.60                    8.50\n",
      "   44  15360 204G 205E C   N    1581 1583 -142.56 -117.80   33.57    1.05  -63.60  178.62   28.23\n",
      "   44        205E 205E N   CA   1583 1584  159.48  136.80                  -40.30\n",
      "   45  15361 205E 206T C   N    1590 1592 -131.20 -124.80   21.74    0.76  -63.20  168.01   26.00\n",
      "   45        206T 206T N   CA   1592 1593  164.27  143.50                  -42.10\n",
      "   46  15362 206T 207K C   N    1597 1599   74.44   56.60  115.94    9.67  -62.90 -144.52   25.46\n",
      "   46        207K 207K N   CA   1599 1600  153.16   38.60                  -40.80\n",
      "   47  15363 207K 208M C   N    1606 1608 -121.69 -125.60    9.64    0.52  -63.40  179.89   31.50\n",
      "   47        208M 208M N   CA   1608 1609  149.31  140.50                  -40.50\n",
      "   48  15364 208M 209N C   N    1614 1616 -126.35 -119.90   23.23    0.89  -63.20  171.63   25.04\n",
      "   48        209N 209N N   CA   1616 1617  159.32  137.00                  -41.10\n",
      "   49  15366 210E 211T C   N    1631 1633  -82.29  -78.10    5.64    0.21  -63.20  165.44   22.48\n",
      "   49        211T 211T N   CA   1633 1634  153.57  149.80                  -42.10\n",
      "   50  15367 211T 212S C   N    1638 1640  -69.97  -72.40   12.82    0.67  -64.10  174.91   12.34\n",
      "   50        212S 212S N   CA   1640 1641  139.81  152.40                  -35.00\n",
      "   51  15368 212S 213S C   N    1644 1646 -119.03 -136.60   38.85    1.52  -64.10  161.20    9.94\n",
      "   51        213S 213S N   CA   1646 1647  116.55  151.20                  -35.00\n",
      "   52  15369 213S 214R C   N    1650 1652 -164.37 -125.20   51.48    1.53  -63.00  176.83   30.56\n",
      "   52        214R 214R N   CA   1652 1653  174.01  140.60                  -41.10\n",
      "   53  15371 215S 216H C   N    1667 1669 -144.76 -125.60   21.82    0.56  -63.20 -172.84   28.71\n",
      "   53        216H 216H N   CA   1669 1670  149.24  138.80                  -42.30\n",
      "   54  15372 216H 217A C   N    1677 1679  -77.92  -68.20   18.58    1.19  -62.50  158.72   26.79\n",
      "   54        217A 217A N   CA   1679 1680  161.14  145.30                  -40.90\n",
      "   55  15377 221I 222T C   N    1714 1716  -70.52  -63.20   54.84    6.57  -63.20   54.84    6.57\n",
      "   55        222T 222T N   CA   1716 1717   12.25  -42.10                  -42.10\n",
      "   56  15378 222T 223I C   N    1721 1723  -41.97  -63.40   23.23    4.36 -120.60 -177.28    7.81\n",
      "   56        223I 223I N   CA   1723 1724  -34.63  -43.60                  130.30\n",
      "   57  15379 223I 224S C   N    1729 1731  -60.22  -72.40   48.05    2.44  -64.10  140.97   10.45\n",
      "   57        224S 224S N   CA   1731 1732  105.92  152.40                  -35.00\n",
      "   58  15380 224S 225Q C   N    1735 1737   41.78  -73.00  140.53    8.40  -63.80  145.36   26.23\n",
      "   58        225Q 225Q N   CA   1737 1738   59.62  140.70                  -40.30\n",
      "   59  15384 228G 229S C   N    1769 1771 -166.49 -136.60   33.00    1.02  -64.10 -170.20   21.14\n",
      "   59        229S 229S N   CA   1771 1772  165.18  151.20                  -35.00\n",
      "   60  15385 229S 230L C   N    1775 1777 -139.86 -108.50   74.48    3.63  -63.50  141.17   24.27\n",
      "   60        230L 230L N   CA   1777 1778 -159.94  132.50                  -41.20\n",
      "   61  15387 231D 232M C   N    1791 1793 -110.80 -125.60   29.54    1.04  -63.40  162.51   21.09\n",
      "   61        232M 232M N   CA   1793 1794  114.94  140.50                  -40.50\n",
      "   62  15388 232M 233R C   N    1799 1801  -66.33  -72.10   20.21    1.43  -63.00  163.66   21.89\n",
      "   62        233R 233R N   CA   1801 1802  122.53  141.90                  -41.10\n",
      "   63  15392 236E 237S C   N    1834 1836   90.95 -136.60  139.42    4.54  -64.10 -157.49   17.64\n",
      "   63        237S 237S N   CA   1836 1837 -165.27  151.20                  -35.00\n",
      "   64  15394 238V 239V C   N    1847 1849  -59.00  -62.40    4.20    0.52 -125.40 -175.79   10.37\n",
      "   64        239V 239V N   CA   1849 1850  -44.87  -42.40                  143.30\n",
      "   65  15395 239V 240L C   N    1854 1856  -68.06  -70.70   27.23    2.16  -63.50  150.17   21.12\n",
      "   65        240L 240L N   CA   1856 1857  168.70  141.60                  -41.20\n",
      "   66  15396 240L 241L C   N    1862 1864  -59.27  -70.70   18.18    1.13  -63.50  168.72   23.67\n",
      "   66        241L 241L N   CA   1864 1865  127.46  141.60                  -41.20\n",
      "   67  15398 242V 243D C   N    1877 1879  -56.52  -63.30   66.39    7.81  -63.30   66.39    7.81\n",
      "   67        243D 243D N   CA   1879 1880 -106.04  -40.00                  -40.00\n",
      "   68  15409 253T 254E C   N    1959 1961 -115.38 -117.80   11.56    0.62  -63.60  179.24   26.83\n",
      "   68        254E 254E N   CA   1961 1962  148.10  136.80                  -40.30\n",
      "   69  15410 254E 255S C   N    1968 1970 -166.80 -136.60   60.91    2.31  -64.10  158.64   18.89\n",
      "   69        255S 255S N   CA   1970 1971 -155.91  151.20                  -35.00\n",
      "   70  15411 255S 256T C   N    1974 1976 -134.27 -124.80   33.84    1.19  -63.20  158.72   24.97\n",
      "   70        256T 256T N   CA   1976 1977  175.98  143.50                  -42.10\n",
      "   71  15412 256T 257G C   N    1981 1983 -163.88 -167.20    5.94    0.16   82.20 -162.63   14.48\n",
      "   71        257G 257G N   CA   1983 1984  169.67  174.60                    8.50\n",
      "   72  15414 258V 259A C   N    1992 1994 -155.34 -134.00   29.55    0.87  -62.50  177.82   34.08\n",
      "   72        259A 259A N   CA   1994 1995  167.44  147.00                  -40.90\n",
      "   73  15416 260F 261E C   N    2008 2010 -171.58 -117.80   53.89    1.89  -63.60 -150.49   33.88\n",
      "   73        261E 261E N   CA   2010 2011  140.16  136.80                  -40.30\n",
      "   74  15417 261E 262E C   N    2017 2019 -171.06 -117.80   53.33    1.89  -63.60 -150.47   24.20\n",
      "   74        262E 262E N   CA   2019 2020  139.58  136.80                  -40.30\n",
      "   75  15418 262E 263A C   N    2026 2028 -117.82 -134.00   34.07    1.40  -62.50  167.32   25.36\n",
      "   75        263A 263A N   CA   2028 2029  117.01  147.00                  -40.90\n",
      "   76  15419 263A 264K C   N    2031 2033 -173.49 -118.00   56.43    2.32  -62.90 -157.50   23.41\n",
      "   76        264K 264K N   CA   2033 2034  128.83  139.10                  -40.80\n",
      "   77  15420 264K 265K C   N    2040 2042 -126.37 -118.00    8.38    0.31  -62.90 -169.56   28.52\n",
      "   77        265K 265K N   CA   2042 2043  139.65  139.10                  -40.80\n",
      "   78  15422 266I 267N C   N    2057 2059  -68.81  -71.20    8.30    0.47  -63.20  176.03   21.71\n",
      "   78        267N 267N N   CA   2059 2060  134.85  142.80                  -41.10\n",
      "   79  15423 267N 268Q C   N    2065 2067 -169.40 -121.10   67.25    2.17  -63.80  169.99   30.16\n",
      "   79        268Q 268Q N   CA   2067 2068 -173.51  139.70                  -40.30\n",
      "   80  15424 268Q 269S C   N    2074 2076  -73.20  -72.40   43.84    2.55  -64.10  129.08    9.92\n",
      "   80        269S 269S N   CA   2076 2077 -163.76  152.40                  -35.00\n",
      "   81  15426 270L 271L C   N    2088 2090  -64.62  -70.70    7.34    0.51  -63.50  178.70   24.75\n",
      "   81        271L 271L N   CA   2090 2091  137.49  141.60                  -41.20\n",
      "   82  15427 271L 272M C   N    2096 2098 -118.05 -125.60   12.98    0.71  -63.40  177.08   30.80\n",
      "   82        272M 272M N   CA   2098 2099  151.07  140.50                  -40.50\n",
      "   83  15428 272M 273L C   N    2104 2106 -107.35 -108.50   46.99    2.55  -63.50  146.05   22.99\n",
      "   83        273L 273L N   CA   2106 2107  179.48  132.50                  -41.20\n",
      "   84  15430 274G 275R C   N    2116 2118 -136.96 -125.20   54.72    2.36  -63.00  145.12   24.56\n",
      "   84        275R 275R N   CA   2118 2119 -165.96  140.60                  -41.10\n",
      "   85  15431 275R 276A C   N    2127 2129  -64.49  -68.20   29.72    2.53  -62.50  144.33   23.75\n",
      "   85        276A 276A N   CA   2129 2130  174.78  145.30                  -40.90\n",
      "   86  15434 278N 279S C   N    2148 2150  -57.13  -72.40   52.67    2.62  -64.10  137.17   10.37\n",
      "   86        279S 279S N   CA   2150 2151  101.99  152.40                  -35.00\n",
      "   87  15454 298T 299R C   N    2299 2301 -148.35 -125.20   32.67    1.79  -63.00 -179.86   20.42\n",
      "   87        299R 299R N   CA   2301 2302  117.54  140.60                  -41.10\n",
      "   88  15455 299R 300L C   N    2310 2312 -179.65 -108.50   72.73    3.31  -63.50 -153.09   35.81\n",
      "   88        300L 300L N   CA   2312 2313  147.57  132.50                  -41.20\n",
      "   89  15456 300L 301L C   N    2318 2320  -62.58  -70.70   10.60    0.70  -63.50  176.00   24.49\n",
      "   89        301L 301L N   CA   2320 2321  134.80  141.60                  -41.20\n",
      "   90  15457 301L 302S C   N    2326 2328 -135.25 -136.60    7.64    0.41  -64.10 -179.14   18.16\n",
      "   90        302S 302S N   CA   2328 2329  158.72  151.20                  -35.00\n",
      "   91  15458 302S 303E C   N    2332 2334  -62.23  -69.30   12.25    0.70  -63.60  172.80   23.12\n",
      "   91        303E 303E N   CA   2334 2335  132.50  142.50                  -40.30\n",
      "   92  15459 303E 304S C   N    2341 2343 -132.83 -136.60   20.67    0.96  -64.10  179.55   11.15\n",
      "   92        304S 304S N   CA   2343 2344  130.87  151.20                  -35.00\n",
      "   93  15460 304S 305F C   N    2347 2349 -165.26 -124.20   45.17    1.26  -63.20 -175.60   31.91\n",
      "   93        305F 305F N   CA   2349 2350  162.12  143.30                  -44.30\n",
      "   94  15461 305F 306G C   N    2358 2360  171.17 -167.20   37.67    1.49   82.20  161.89   11.74\n",
      "   94        306G 306G N   CA   2360 2361  143.76  174.60                    8.50\n",
      "   95  15462 306G 307G C   N    2362 2364  -64.72  -62.40    4.18    0.75   82.20  156.25   11.92\n",
      "   95        307G 307G N   CA   2364 2365  -44.68  -41.20                    8.50\n",
      "   96  15463 307G 308N C   N    2366 2368 -139.96 -119.90   26.67    0.77  -63.20 -178.62   27.01\n",
      "   96        308N 308N N   CA   2368 2369  154.57  137.00                  -41.10\n",
      "   97  15464 308N 309S C   N    2374 2376 -148.62 -136.60   29.41    1.19  -64.10  169.53   18.37\n",
      "   97        309S 309S N   CA   2376 2377  178.04  151.20                  -35.00\n",
      "   98  15465 309S 310K C   N    2380 2382  -90.72  -70.20   83.84    5.59  -62.90  101.40   14.82\n",
      "   98        310K 310K N   CA   2382 2383 -138.31  140.40                  -40.80\n",
      "   99  15467 311T 312W C   N    2396 2398 -122.50 -124.90   17.18    0.89  -63.00  166.39   24.79\n",
      "   99        312W 312W N   CA   2398 2399  160.41  143.40                  -44.20\n",
      "  100  15468 312W 313M C   N    2410 2412  -70.02  -73.00   18.86    1.33  -63.40  158.01   24.11\n",
      "  100        313M 313M N   CA   2412 2413  161.63  143.00                  -40.50\n",
      "  101  15469 313M 314L C   N    2418 2420  -61.46  -70.70   24.11    1.53  -63.50  160.55   22.41\n",
      "  101        314L 314L N   CA   2420 2421  119.33  141.60                  -41.20\n",
      "  102  15470 314L 315A C   N    2426 2428  165.36 -134.00   69.50    1.61  -62.50 -168.83   38.08\n",
      "  102        315A 315A N   CA   2428 2429 -179.05  147.00                  -40.90\n",
      "  103  15471 315A 316T C   N    2431 2433  -77.48  -78.10    8.81    0.40  -63.20  159.95   21.46\n",
      "  103        316T 316T N   CA   2433 2434  158.59  149.80                  -42.10\n",
      "  104  15474 318S 319T C   N    2451 2453  -83.80  -78.10   13.82    0.46  -63.20  156.86   21.49\n",
      "  104        319T 319T N   CA   2453 2454  162.39  149.80                  -42.10\n",
      "  105  15477 321A 322N C   N    2468 2470  -85.45  -71.20   31.81    1.72   55.90 -166.78   11.98\n",
      "  105        322N 322N N   CA   2470 2471  171.24  142.80                   39.50\n",
      "  106  15497 341T 342N C   N    2620 2622  -81.06 -119.90   43.22    1.34  -63.20  160.13   19.09\n",
      "  106        342N 342N N   CA   2622 2623  118.03  137.00                  -41.10\n",
      "  107  15500 344A 345K C   N    2642 2644 -109.49 -118.00   22.39    1.16  -62.90  166.06   24.33\n",
      "  107        345K 345K N   CA   2644 2645  159.81  139.10                  -40.80\n",
      "  108  15504 348K 349T C   N    2675 2677  -42.15 -124.80   86.30    5.05  -63.20  151.05   18.06\n",
      "  108        349T 349T N   CA   2677 2678  168.32  143.50                  -42.10\n",
      "  109  15505 349T 350A C   N    2682 2684 -151.84 -134.00   21.88    0.55  -62.50 -177.22   34.72\n",
      "  109        350A 350A N   CA   2684 2685  159.65  147.00                  -40.90\n",
      "  110  15506 350A 351K C   N    2687 2689  168.66 -118.00  100.34    5.02  -62.90  170.03   21.21\n",
      "  110        351K 351K N   CA   2689 2690   70.62  139.10                  -40.80\n",
      "  111  15507 351K 352D C   N    2696 2698 -149.65  -96.50   54.22    2.22  -63.30 -173.87   19.75\n",
      "  111        352D 352D N   CA   2698 2699  124.89  114.20                  -40.00\n",
      "  112  15508 352D 353L C   N    2704 2706 -146.98 -108.50   39.95    2.04  -63.50 -176.89   22.44\n",
      "  112        353L 353L N   CA   2706 2707  121.78  132.50                  -41.20\n",
      "  113  15509 353L 354E C   N    2712 2714 -147.77 -117.80   30.30    1.22  -63.60 -167.96   22.30\n",
      "  113        354E 354E N   CA   2714 2715  132.31  136.80                  -40.30\n",
      "  114  15510 354E 355Y C   N    2721 2723  -86.87  -98.40   17.44    1.41  -63.50  176.66   29.23\n",
      "  114        355Y 355Y N   CA   2723 2724  141.49  128.40                  -43.40\n",
      "  115  15511 355Y 356D C   N    2733 2735  -74.69  -70.90   20.99    1.23  -63.30  170.03   20.32\n",
      "  115        356D 356D N   CA   2735 2736  129.65  150.30                  -40.00\n",
      "  116  15514 358I 359Q C   N    2758 2760  -50.25  -73.00   22.92    1.69  -63.80  176.71   25.29\n",
      "  116        359Q 359Q N   CA   2760 2761  143.50  140.70                  -40.30\n",
      "  117  15515 359Q 360K C   N    2767 2769  -57.82  -70.20   35.08    2.25  -62.90  148.46   19.56\n",
      "  117        360K 360K N   CA   2769 2770  107.57  140.40                  -40.80\n",
      "  118  15516 360K 361L C   N    2776 2778 -129.25 -108.50   90.62    4.65  -63.50  118.08   20.41\n",
      "  118        361L 361L N   CA   2778 2779 -139.28  132.50                  -41.20\n",
      "  119  15518 362A 363W C   N    2789 2791  -53.70  -71.30   18.02    1.39  -63.00  173.17   21.11\n",
      "  119        363W 363W N   CA   2791 2792  142.89  139.00                  -44.20\n",
      "  120  15520 364V 365L C   N    2810 2812 -132.58 -108.50   25.40    1.31  -63.50  179.45   22.12\n",
      "  120        365L 365L N   CA   2812 2813  124.42  132.50                  -41.20\n",
      "  121  15521 365L 366Q C   N    2818 2820  -19.63  -73.00   71.18    5.54  -63.80  139.09   18.46\n",
      "  121        366Q 366Q N   CA   2820 2821 -172.20  140.70                  -40.30\n",
      "  122  15523 367I 368R C   N    2835 2837 -153.22 -125.20   47.40    1.56  -63.00  166.60   28.51\n",
      "  122        368R 368R N   CA   2837 2838  178.84  140.60                  -41.10\n",
      "  123  15525 369L 370D C   N    2854 2856  177.72  -63.30  121.06   20.09  -63.30  121.06   20.09\n",
      "  123        370D 370D N   CA   2856 2857  -62.36  -40.00                  -40.00\n",
      "  124  15529 373A 374K C   N    2884 2886  -96.09 -118.00   44.41    1.76  -62.90  145.13   17.31\n",
      "  124        374K 374K N   CA   2886 2887  100.48  139.10                  -40.80\n",
      "  125  15530 374K 375G C   N    2893 2895  139.23   82.20   76.12    2.56  -62.40  158.38   28.32\n",
      "  125        375G 375G N   CA   2895 2896  -41.92    8.50                  -41.20\n",
      "  126  15534 378A 379L C   N    2916 2918  -62.50  -63.50    7.55    1.11  -70.70  175.51   12.98\n",
      "  126        379L 379L N   CA   2918 2919  -33.72  -41.20                  141.60\n",
      "  127  15539 383I 384Q C   N    2954 2956  -85.00  -73.00   31.35    1.84  -63.80  151.52   23.41\n",
      "  127        384Q 384Q N   CA   2956 2957  169.67  140.70                  -40.30\n",
      "  128  15540 384Q 385G C   N    2963 2965   90.21   82.20   10.58    0.82  -80.20 -127.14    7.80\n",
      "  128        385G 385G N   CA   2965 2966   15.41    8.50                  174.10\n",
      "  129  15542 386L 387R C   N    2975 2977 -119.99 -125.20   14.59    0.79  -63.00  174.26   27.47\n",
      "  129        387R 387R N   CA   2977 2978  154.22  140.60                  -41.10\n",
      "  130  15543 387R 388G C   N    2986 2988 -172.95 -167.20   52.11    2.10   82.20  155.10   11.79\n",
      "  130        388G 388G N   CA   2988 2989  122.80  174.60                    8.50\n",
      "  131  15544 388G 389E C   N    2990 2992 -172.07 -117.80   60.30    1.77  -63.60 -169.49   31.28\n",
      "  131        389E 389E N   CA   2992 2993  163.09  136.80                  -40.30\n",
      "  132  15546 390I 391D C   N    3007 3009 -161.09  -96.50   84.00    3.44  -63.30  140.22   15.76\n",
      "  132        391D 391D N   CA   3009 3010   60.50  114.20                  -40.00\n",
      "  133  15548 392C 393M C   N    3021 3023  -84.15  -73.00   48.35    3.47  -63.40  138.02   19.08\n",
      "  133        393M 393M N   CA   3023 3024   95.95  143.00                  -40.50\n",
      "  134  15549 393M 394K C   N    3029 3031  178.31 -118.00   70.56    2.34  -62.90 -168.88   31.76\n",
      "  134        394K 394K N   CA   3031 3032  169.48  139.10                  -40.80\n",
      "  135  15550 394K 395S C   N    3038 3040  -61.55  -72.40   12.77    1.03  -64.10  165.87   11.87\n",
      "  135        395S 395S N   CA   3040 3041  159.14  152.40                  -35.00\n",
      "  136  15551 395S 396K C   N    3044 3046  -79.81  -70.20   20.42    1.67  -62.90  164.06   20.44\n",
      "  136        396K 396K N   CA   3046 3047  122.39  140.40                  -40.80\n",
      "  137  15552 396K 397L C   N    3053 3055 -128.85 -108.50   39.13    1.85  -63.50  166.25   27.13\n",
      "  137        397L 397L N   CA   3055 3056  165.93  132.50                  -41.20\n",
      "  138  15553 397L 398T C   N    3061 3063  -95.04  -78.10   34.93    2.05  -63.20  164.47   19.15\n",
      "  138        398T 398T N   CA   3063 3064  119.25  149.80                  -42.10\n",
      "  139  15555 399V 400A C   N    3075 3077  -64.68  -68.20    5.83    0.37  -62.50  178.46   29.35\n",
      "  139        400A 400A N   CA   3077 3078  140.65  145.30                  -40.90\n",
      "  140  15556 400A 401S C   N    3080 3082 -147.76 -136.60   19.87    0.72  -64.10  178.22   18.92\n",
      "  140        401S 401S N   CA   3082 3083  167.64  151.20                  -35.00\n",
      "  141  15557 401S 402D C   N    3086 3088  -86.97  -63.30  101.61   14.06  -63.30  101.61   14.06\n",
      "  141        402D 402D N   CA   3088 3089 -138.81  -40.00                  -40.00\n",
      "  142  15564 408E 409L C   N    3141 3143  -76.09  -63.50   53.53    6.82  -63.50   53.53    6.82\n",
      "  142        409L 409L N   CA   3143 3144   10.83  -41.20                  -41.20\n",
      "  143  15568 412A 413L C   N    3172 3174  -71.88  -63.50   21.90    3.56 -108.50  170.05    9.51\n",
      "  143        413L 413L N   CA   3174 3175  -61.43  -41.20                  132.50\n",
      "  144  15580 424R 425K C   N    3275 3277  -65.73 -118.00   95.97    4.96  -62.90   99.65   13.10\n",
      "  144        425K 425K N   CA   3277 3278 -140.41  139.10                  -40.80\n",
      "  145  15582 426G 427L C   N    3288 3290 -144.47 -108.50   72.89    3.48  -63.50  147.17   25.38\n",
      "  145        427L 427L N   CA   3290 3291 -164.10  132.50                  -41.20\n",
      "  146  15584 428I 429E C   N    3304 3306  178.43 -117.80   63.90    2.44  -63.60 -150.54   24.23\n",
      "  146        429E 429E N   CA   3306 3307  132.77  136.80                  -40.30\n",
      "  147  15585 429E 430Q C   N    3313 3315 -165.43 -121.10   44.71    1.57  -63.80 -158.35   34.86\n",
      "  147        430Q 430Q N   CA   3315 3316  145.53  139.70                  -40.30\n",
      "  148  15586 430Q 431R C   N    3322 3324 -121.46 -125.20   37.79    1.74  -63.00  155.50   17.93\n",
      "  148        431R 431R N   CA   3324 3325  102.99  140.60                  -41.10\n",
      "  149  15587 431R 432K C   N    3333 3335  179.58 -118.00   74.55    2.49  -62.90 -177.73   30.50\n",
      "  149        432K 432K N   CA   3335 3336  179.87  139.10                  -40.80\n",
      "  150  15589 433I 434S C   N    3350 3352  -98.55  -72.40   41.35    1.91  -64.10  144.72   12.84\n",
      "  150        434S 434S N   CA   3352 3353 -175.57  152.40                  -35.00\n",
      "  151  15590 434S 435G C   N    3356 3358  -66.89  -80.20   31.02    0.72   82.20 -157.13    6.64\n",
      "  151        435G 435G N   CA   3358 3359  146.08  174.10                    8.50\n",
      "  152  15591 435G 436F C   N    3360 3362  -80.42  -71.40   26.23    1.83  -63.20  151.35   22.18\n",
      "  152        436F 436F N   CA   3362 3363  165.33  140.70                  -44.30\n",
      "  153  15592 436F 437S C   N    3371 3373  -55.60  -72.40   27.73    1.27  -64.10  165.56   12.52\n",
      "  153        437S 437S N   CA   3373 3374  130.34  152.40                  -35.00\n",
      "  154  15593 437S 438A C   N    3377 3379  -75.93  -68.20    9.14    0.61  -62.50  169.46   28.45\n",
      "  154        438A 438A N   CA   3379 3380  150.18  145.30                  -40.90\n",
      "  155  15594 438A 439A C   N    3382 3384  -76.03  -68.20   20.30    1.37  -62.50  155.65   26.19\n",
      "  155        439A 439A N   CA   3384 3385  164.04  145.30                  -40.90\n",
      "  156  15595 439A 440L C   N    3387 3389  -63.82  -70.70    6.94    0.66  -63.50  176.25   24.49\n",
      "  156        440L 440L N   CA   3389 3390  142.54  141.60                  -41.20\n",
      "  157  15599 443F 444S C   N    3419 3421  -65.03  -72.40   17.95    0.84  -64.10  171.04   12.34\n",
      "  157        444S 444S N   CA   3421 3422  136.03  152.40                  -35.00\n",
      "  158  15600 444S 445G C   N    3425 3427   87.88   78.70   47.00    0.97   82.20  139.42    7.05\n",
      "  158        445G 445G N   CA   3427 3428  147.80 -166.10                    8.50\n",
      "  159  15601 445G 446T C   N    3429 3431 -133.26 -124.80    8.54    0.42  -63.20 -173.16   28.57\n",
      "  159        446T 446T N   CA   3431 3432  144.70  143.50                  -42.10\n",
      "  160  15603 447C 448K C   N    3442 3444 -147.19 -118.00   35.00    1.68  -62.90 -178.63   20.83\n",
      "  160        448K 448K N   CA   3444 3445  119.79  139.10                  -40.80\n",
      "  161  15604 448K 449T C   N    3451 3453 -130.85 -124.80   31.86    1.54  -63.20  168.49   18.12\n",
      "  161        449T 449T N   CA   3453 3454  112.21  143.50                  -42.10\n",
      "  162  15605 449T 450S C   N    3458 3460  162.62 -136.60   61.15    2.60  -64.10 -136.45   16.00\n",
      "  162        450S 450S N   CA   3460 3461  144.48  151.20                  -35.00\n",
      "  163  15606 450S 451L C   N    3464 3466 -130.55 -108.50   51.04    2.48  -63.50  155.46   25.73\n",
      "  163        451L 451L N   CA   3466 3467  178.54  132.50                  -41.20\n",
      "  164  15607 451L 452A C   N    3472 3474   12.73  -62.50  114.00   22.57  -68.20  129.08    8.13\n",
      "  164        452A 452A N   CA   3474 3475   44.75  -40.90                  145.30\n",
      "  165  15608 452A 453N C   N    3477 3479 -148.74  -63.20   85.66   13.17  -63.20   85.66   13.17\n",
      "  165        453N 453N N   CA   3479 3480  -45.58  -41.10                  -41.10\n",
      "  166  15611 455P 456F C   N    3500 3502  -50.34  -71.40   21.06    1.41  -63.20  175.73   23.55\n",
      "  166        456F 456F N   CA   3502 3503  140.44  140.70                  -44.30\n",
      "  167  15613 457G 458S C   N    3515 3517  -64.86 -136.60   81.76    4.23  -64.10  134.58    9.80\n",
      "  167        458S 458S N   CA   3517 3518 -169.58  151.20                  -35.00\n",
      "  168  15618 462T 463T C   N    3556 3558   74.70  -78.10  156.84    8.53  -63.20 -151.39   35.06\n",
      "  168        463T 463T N   CA   3558 3559  114.44  149.80                  -42.10\n",
      "  169  15619 463T 464L C   N    3563 3565 -179.03 -108.50   70.61    3.40  -63.50 -154.21   25.37\n",
      "  169        464L 464L N   CA   3565 3566  129.10  132.50                  -41.20\n",
      "  170  15624 468C 469Y C   N    3598 3600 -110.63  -98.40   13.45    0.71  -63.50  172.76   25.18\n",
      "  170        469Y 469Y N   CA   3600 3601  122.81  128.40                  -43.40\n",
      "  171  15625 469Y 470M C   N    3610 3612 -124.93 -125.60   15.33    0.75  -63.40  174.87   31.00\n",
      "  171        470M 470M N   CA   3612 3613  155.81  140.50                  -40.50\n",
      "  172  15626 470M 471L C   N    3618 3620  -74.01  -70.70    4.34    0.29  -63.50  174.71   24.87\n",
      "  172        471L 471L N   CA   3620 3621  144.40  141.60                  -41.20\n",
      "  173  15627 471L 472A C   N    3626 3628   69.17  -68.20  138.16   12.19  -62.50 -153.56   31.37\n",
      "  173        472A 472A N   CA   3628 3629  160.10  145.30                  -40.90\n",
      "  174  15628 472A 473G C   N    3631 3633   92.38   78.70   37.09    0.58   82.20  151.26    7.85\n",
      "  174        473G 473G N   CA   3633 3634  159.42 -166.10                    8.50\n",
      "  175  15632 476P 477F C   N    3656 3658 -125.40 -124.20   64.88    3.03  -63.20  124.22   21.17\n",
      "  175        477F 477F N   CA   3658 3659 -151.83  143.30                  -44.30\n",
      "  176  15636 480C 481H C   N    3689 3691 -119.67 -125.60    6.41    0.18  -63.20 -172.62   20.56\n",
      "  176        481H 481H N   CA   3691 3692  136.37  138.80                  -42.30\n",
      "  177  15640 484V 485Y C   N    3724 3726 -115.22 -124.30   10.18    0.58  -63.50 -178.29   26.40\n",
      "  177        485Y 485Y N   CA   3726 3727  130.80  135.40                  -43.40\n",
      "  178  15641 485Y 486L C   N    3736 3738 -112.88 -108.50   28.09    1.47  -63.50  166.07   26.10\n",
      "  178        486L 486L N   CA   3738 3739  160.24  132.50                  -41.20\n",
      "  179  15642 486L 487T C   N    3744 3746  -81.17  -78.10   46.60    1.89  -63.20  122.93   16.96\n",
      "  179        487T 487T N   CA   3746 3747 -163.71  149.80                  -42.10\n",
      "  180  15643 487T 488E C   N    3751 3753  -62.57  -69.30   14.68    0.88  -63.60  169.76   22.69\n",
      "  180        488E 488E N   CA   3753 3754  129.45  142.50                  -40.30\n",
      "  181  15644 488E 489E C   N    3760 3762  -80.12  -69.30   10.97    0.89  -63.60  179.73   24.88\n",
      "  181        489E 489E N   CA   3762 3763  140.73  142.50                  -40.30\n",
      "  182  15646 490S 491S C   N    3775 3777  -89.92  -72.40   26.73    1.24  -64.10  154.57   12.91\n",
      "  182        491S 491S N   CA   3777 3778  172.60  152.40                  -35.00\n",
      "  183  15648 492V 493A C   N    3788 3790  -60.08  -68.20   21.11    1.43  -62.50  166.74   27.44\n",
      "  183        493A 493A N   CA   3790 3791  125.82  145.30                  -40.90\n",
      "  184  15649 493A 494V C   N    3793 3795  -65.59  -62.40   20.41    2.78 -125.40  165.33    9.31\n",
      "  184        494V 494V N   CA   3795 3796  -62.56  -42.40                  143.30\n",
      "  185  15650 494V 495L C   N    3800 3802  -70.50  -63.50   57.57    7.62  -63.50   57.57    7.62\n",
      "  185        495L 495L N   CA   3802 3803   15.94  -41.20                  -41.20\n",
      "  186  15652 496L 497R C   N    3816 3818  -80.94  -63.00  101.01   12.53  -63.00  101.01   12.53\n",
      "  186        497R 497R N   CA   3818 3819   58.30  -41.10                  -41.10\n",
      "  187  15653 497R 498L C   N    3827 3829  -12.81  -63.50   77.51    9.77  -63.50   77.51    9.77\n",
      "  187        498L 498L N   CA   3829 3830  -99.83  -41.20                  -41.20\n",
      "  188  15655 499R 500E C   N    3846 3848   80.35  -69.30  154.20   12.90  -63.60 -159.19   24.04\n",
      "  188        500E 500E N   CA   3848 3849  179.68  142.50                  -40.30\n",
      "  189  15656 500E 501L C   N    3855 3857  -59.71  -70.70   11.03    1.03  -63.50  176.27   24.26\n",
      "  189        501L 501L N   CA   3857 3858  142.57  141.60                  -41.20\n",
      "  190  15657 501L 502Q C   N    3863 3865  -75.39  -73.00   13.36    0.92  -63.80  168.26   24.14\n",
      "  190        502Q 502Q N   CA   3865 3866  127.55  140.70                  -40.30\n",
      "  191  15658 502Q 503H C   N    3872 3874 -174.95 -125.60   99.62    2.76  -63.20  144.99   24.65\n",
      "  191        503H 503H N   CA   3874 3875 -134.67  138.80                  -42.30\n",
      "  192  15659 503H 504L C   N    3882 3884 -107.58 -108.50   40.21    2.16  -63.50  140.59   17.55\n",
      "  192        504L 504L N   CA   3884 3885   92.30  132.50                  -41.20\n",
      "  193  15661 505P 506D C   N    3897 3899 -141.42  -96.50   49.63    2.02  -63.30  154.34   16.39\n",
      "  193        506D 506D N   CA   3899 3900   93.11  114.20                  -40.00\n",
      "  194  15662 506D 507Y C   N    3905 3907 -130.19  -63.50   88.76   16.80  -63.50   88.76   16.80\n",
      "  194        507Y 507Y N   CA   3907 3908 -101.98  -43.40                  -43.40\n",
      "  195  15668 512V 513Q C   N    3944 3946 -108.82 -121.10   42.33    1.78  -63.80  146.57   19.50\n",
      "  195        513Q 513Q N   CA   3946 3947   99.18  139.70                  -40.30\n",
      "  196  15670 514V 515T C   N    3960 3962  -47.56  -78.10   35.47    1.57  -63.20  174.57   23.42\n",
      "  196        515T 515T N   CA   3962 3963  131.76  149.80                  -42.10\n",
      "  197  15671 515T 516V C   N    3967 3969  -70.69  -62.40   10.59    1.28 -125.40 -172.72    7.75\n",
      "  197        516V 516V N   CA   3969 3970  -35.81  -42.40                  143.30\n",
      "  198  15673 517W 518F C   N    3988 3990 -100.12 -124.20   34.86    0.95  -63.20  166.54   20.79\n",
      "  198        518F 518F N   CA   3990 3991  118.10  143.30                  -44.30\n",
      "  199  15674 518F 519E C   N    3999 4001  -80.71  -69.30   11.43    0.85  -63.60  177.44   24.61\n",
      "  199        519E 519E N   CA   4001 4002  143.08  142.50                  -40.30\n",
      "  200  15675 519E 520G C   N    4008 4010  -93.13  -80.20   17.64    0.59   82.20 -110.44    7.75\n",
      "  200        520G 520G N   CA   4010 4011 -173.91  174.10                    8.50\n",
      "  201  15676 520G 521H C   N    4012 4014 -147.17 -125.60   21.65    0.79  -63.20 -162.04   21.13\n",
      "  201        521H 521H N   CA   4014 4015  136.97  138.80                  -42.30\n",
      "  202  15677 521H 522Y C   N    4022 4024 -137.74 -124.30   19.16    0.84  -63.50 -176.74   32.75\n",
      "  202        522Y 522Y N   CA   4024 4025  149.05  135.40                  -43.40\n",
      "  203  15678 522Y 523S C   N    4034 4036  -63.92  -72.40   27.59    1.36  -64.10  161.15   11.69\n",
      "  203        523S 523S N   CA   4036 4037  126.15  152.40                  -35.00\n",
      "  204  15679 523S 524S C   N    4040 4042 -145.11 -136.60   15.11    0.54  -64.10 -179.49   18.89\n",
      "  204        524S 524S N   CA   4042 4043  163.68  151.20                  -35.00\n",
      "  205  15680 524S 525A C   N    4046 4048  -70.11  -68.20    4.67    0.44  -62.50  178.23   29.59\n",
      "  205        525A 525A N   CA   4048 4049  141.04  145.30                  -40.90\n",
      "  206  15681 525A 526V C   N    4051 4053  -64.76  -62.40    7.31    0.82 -125.40 -171.22    7.64\n",
      "  206        526V 526V N   CA   4053 4054  -35.48  -42.40                  143.30\n",
      "  207  15682 526V 527Q C   N    4058 4060  -75.41  -63.80   78.74   12.20  -63.80   78.74   12.20\n",
      "  207        527Q 527Q N   CA   4060 4061 -118.18  -40.30                  -40.30\n",
      "  208  15687 531V 532R C   N    4096 4098  -64.50 -125.20   69.22    2.08  -63.00  148.44   19.95\n",
      "  208        532R 532R N   CA   4098 4099  107.33  140.60                  -41.10\n",
      "  209  15688 532R 533P C   N    4107 4109  -57.08  -58.70    2.31    0.16  -64.50  179.50   13.01\n",
      "  209        533P 533P N   CA   4109 4110  -32.15  -30.50                  147.20\n",
      "  210  15689 533P 534N C   N    4114 4116 -115.58 -119.90    9.17    0.49  -63.20 -178.47   25.68\n",
      "  210        534N 534N N   CA   4116 4117  145.09  137.00                  -41.10\n",
      "  211  15690 534N 535G C   N    4122 4124 -102.46  -80.20   36.45    0.97   82.20 -118.87    7.79\n",
      "  211        535G 535G N   CA   4124 4125 -157.03  174.10                    8.50\n",
      "  212  15691 535G 536G C   N    4126 4128 -176.66 -167.20   10.50    0.16   82.20 -161.64   14.12\n",
      "  212        536G 536G N   CA   4128 4129  179.14  174.60                    8.50\n",
      "  213  15692 536G 537N C   N    4130 4132 -105.95 -119.90   14.94    0.74  -63.20 -178.36   25.14\n",
      "  213        537N 537N N   CA   4132 4133  142.36  137.00                  -41.10\n",
      "  214  15694 538P 539Q C   N    4145 4147 -169.75 -121.10   83.33    2.93  -63.80  154.42   27.71\n",
      "  214        539Q 539Q N   CA   4147 4148 -152.64  139.70                  -40.30\n",
      "  215  15695 539Q 540F C   N    4154 4156  -55.60  -71.40   16.85    1.33  -63.20  169.33   23.00\n",
      "  215        540F 540F N   CA   4156 4157  146.54  140.70                  -44.30\n",
      "  216  15696 540F 541S C   N    4165 4167 -142.90 -136.60   12.63    0.48  -64.10 -179.08   18.75\n",
      "  216        541S 541S N   CA   4167 4168  162.14  151.20                  -35.00\n",
      "  217  15697 541S 542L C   N    4171 4173 -107.64 -108.50   18.34    1.00  -63.50  173.68   26.83\n",
      "  217        542L 542L N   CA   4173 4174  150.82  132.50                  -41.20\n",
      "  218  15698 542L 543R C   N    4179 4181 -117.09 -125.20    8.12    0.32  -63.00 -173.50   28.93\n",
      "  218        543R 543R N   CA   4181 4182  140.42  140.60                  -41.10\n",
      "  219  15699 543R 544Q C   N    4190 4192  -84.30  -73.00   26.31    1.53  -63.80  156.58   24.12\n",
      "  219        544Q 544Q N   CA   4192 4193  164.46  140.70                  -40.30\n",
      "  220  15700 544Q 545M C   N    4199 4201  -79.96  -73.00   90.72    6.25  -63.40   94.51   12.91\n",
      "  220        545M 545M N   CA   4201 4202   52.54  143.00                  -40.50\n",
      "  221  15701 545M 546Y C   N    4207 4209 -138.51 -124.30   16.10    1.34  -63.50 -173.05   26.40\n",
      "  221        546Y 546Y N   CA   4209 4210  127.84  135.40                  -43.40\n",
      "  222  15702 546Y 547L C   N    4219 4221   50.51  -70.70  128.83   12.77  -63.50  175.59   22.11\n",
      "  222        547L 547L N   CA   4221 4222 -174.74  141.60                  -41.20\n",
      "  223  15703 547L 548F C   N    4227 4229 -105.91 -124.20   24.10    1.31  -63.20  162.43   25.41\n",
      "  223        548F 548F N   CA   4229 4230  158.99  143.30                  -44.30\n",
      "  224  15704 548F 549G C   N    4238 4240  104.90   78.70   62.92    0.94   82.20  130.19    7.44\n",
      "  224        549G 549G N   CA   4240 4241  136.70 -166.10                    8.50\n",
      "  225  15706 550P 551H C   N    4249 4251 -100.95 -125.60   25.35    1.00  -63.20  177.05   24.59\n",
      "  225        551H 551H N   CA   4251 4252  144.73  138.80                  -42.30\n",
      "  226  15707 551H 552T C   N    4259 4261  -74.52  -78.10   12.18    0.42  -63.20 -179.91   23.84\n",
      "  226        552T 552T N   CA   4261 4262  138.16  149.80                  -42.10\n",
      "  227  15708 552T 553E C   N    4266 4268 -113.92 -117.80   22.27    1.04  -63.60  163.12   19.46\n",
      "  227        553E 553E N   CA   4268 4269  114.87  136.80                  -40.30\n",
      "  228  15709 553E 554D C   N    4275 4277 -103.85  -70.90   40.31    1.94  -63.30  152.00   21.37\n",
      "  228        554D 554D N   CA   4277 4278  173.51  150.30                  -40.00\n",
      "  229  15710 554D 555L C   N    4283 4285  165.25 -108.50   86.28    4.08  -63.50 -140.33   27.27\n",
      "  229        555L 555L N   CA   4285 4286  134.94  132.50                  -41.20\n",
      "  230  15711 555L 556E C   N    4291 4293 -131.63 -117.80   18.81    0.59  -63.60 -176.76   28.27\n",
      "  230        556E 556E N   CA   4293 4294  149.56  136.80                  -40.30\n",
      "  231  15712 556E 557R C   N    4300 4302 -103.36 -125.20   27.79    1.50  -63.00  166.10   25.23\n",
      "  231        557R 557R N   CA   4302 4303  157.78  140.60                  -41.10\n",
      "  232  15713 557R 558F C   N    4311 4313 -103.81 -124.20   21.05    0.67  -63.20 -177.79   28.03\n",
      "  232        558F 558F N   CA   4313 4314  138.07  143.30                  -44.30\n",
      "  233  15714 558F 559F C   N    4322 4324 -118.10 -124.20   19.93    1.06  -63.20  162.95   26.26\n",
      "  233        559F 559F N   CA   4324 4325  162.28  143.30                  -44.30\n",
      "  234  15715 559F 560Q C   N    4333 4335  -57.10  -73.00   21.32    1.25  -63.80  166.94   24.90\n",
      "  234        560Q 560Q N   CA   4335 4336  126.50  140.70                  -40.30\n",
      "  235  15716 560Q 561L C   N    4342 4344 -131.44 -108.50   53.53    2.60  -63.50  153.75   25.55\n",
      "  235        561L 561L N   CA   4344 4345 -179.13  132.50                  -41.20\n",
      "  236  15717 561L 562D C   N    4350 4352  -77.40  -96.50   73.34    3.09  -63.30   84.57    9.67\n",
      "  236        562D 562D N   CA   4352 4353   43.39  114.20                  -40.00\n",
      "  237  15718 562D 563V C   N    4358 4360  -62.52  -62.40   12.24    1.56 -125.40  173.83    9.79\n",
      "  237        563V 563V N   CA   4360 4361  -54.64  -42.40                  143.30\n",
      "  238  15719 563V 564L C   N    4365 4367   70.53  -63.50  159.80   22.25  -63.50  159.80   22.25\n",
      "  238        564L 564L N   CA   4367 4368 -128.22  -41.20                  -41.20\n",
      "  239  15721 565N 566F C   N    4381 4383 -115.17 -124.20   12.49    0.69  -63.20  171.82   27.32\n",
      "  239        566F 566F N   CA   4383 4384  151.93  143.30                  -44.30\n",
      "  240  15722 566F 567R C   N    4392 4394 -113.51 -125.20   30.40    1.66  -63.00  158.50   24.90\n",
      "  240        567R 567R N   CA   4394 4395  168.66  140.60                  -41.10\n",
      "  241  15724 568V 569E C   N    4410 4412  -97.44 -117.80   90.26    4.11  -63.60   95.37   11.24\n",
      "  241        569E 569E N   CA   4412 4413   48.86  136.80                  -40.30\n",
      "  242  15725 569E 570G C   N    4419 4421 -173.85 -167.20   85.48    3.42   82.20  131.71   10.26\n",
      "  242        570G 570G N   CA   4421 4422   89.38  174.60                    8.50\n",
      "  243  15726 570G 571I C   N    4423 4425  -69.96  -63.40    8.78    1.36 -120.60  175.53    8.17\n",
      "  243        571I 571I N   CA   4425 4426  -37.76  -43.60                  130.30\n",
      "  244  15727 571I 572M C   N    4431 4433  -50.73  -73.00   25.97    1.73  -63.40  170.60   26.47\n",
      "  244        572M 572M N   CA   4433 4434  129.63  143.00                  -40.50\n",
      "  245  15728 572M 573S C   N    4439 4441 -142.88 -136.60    7.80    0.42  -64.10 -164.97   19.74\n",
      "  245        573S 573S N   CA   4441 4442  146.58  151.20                  -35.00\n",
      "  246  15729 573S 574D C   N    4445 4447 -116.11  -70.90   65.39    2.58  -63.30  133.36   19.87\n",
      "  246        574D 574D N   CA   4447 4448 -162.46  150.30                  -40.00\n",
      "  247  15730 574D 575L C   N    4453 4455 -113.38 -108.50   11.76    0.57  -63.50 -177.46   28.42\n",
      "  247        575L 575L N   CA   4455 4456  143.20  132.50                  -41.20\n",
      "  248  15731 575L 576T C   N    4461 4463 -143.57 -124.80   81.28    2.95  -63.20  124.68   20.87\n",
      "  248        576T 576T N   CA   4463 4464 -137.42  143.50                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   87  120  487  433  659  525  617  680  748  833\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00199.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      577     4476     4476\n",
      "openf___224_> Open           EAO00199.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     47436912   46325.109    45.239\n",
      "read_mo_297_> Segments, residues, atoms:        1      577     4476\n",
      "read_mo_298_> Segment:        1  1:A  577:A     4476\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4476     4476        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    21012\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22310\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22981\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    23873\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    24919\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    25728\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    27228\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    28107\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    29246\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    30620\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    31777\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33640\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33781\n",
      "iupac_m_486_> OE1/2 will be swapped:       94.0231        3  3\n",
      "iupac_m_487_> NH1/2 swapped:     -163.6044       10  10\n",
      "iupac_m_487_> NH1/2 swapped:      179.0463       16  16\n",
      "iupac_m_487_> NH1/2 swapped:      177.5693       21  21\n",
      "iupac_m_486_> OE1/2 will be swapped:      152.3752       24  24\n",
      "iupac_m_485_> OD1/2 will be swapped:     -178.1529       33  33\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.2448       34  34\n",
      "iupac_m_485_> OD1/2 will be swapped:      160.4137       41  41\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.1623       51  51\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.4168       54  54\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.9622       62  62\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8381       70  70\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.8789       80  80\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -112.5754       85  85\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      109.9004       87  87\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.7694       99  99\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.0864      106  106\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.5914      114  114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.6413      121  121\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7797      125  125\n",
      "iupac_m_485_> OD1/2 will be swapped:     -169.3095      129  129\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -99.1447      137  137\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.1796      157  157\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.5990      162  162\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4815      189  189\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.5784      192  192\n",
      "iupac_m_487_> NH1/2 swapped:      179.8533      201  201\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.5773      210  210\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9477      214  214\n",
      "iupac_m_485_> OD1/2 will be swapped:      103.7910      231  231\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8430      233  233\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.1476      236  236\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.5113      248  248\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8749      249  249\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.0745      254  254\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.5274      260  260\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.0414      261  261\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.9784      262  262\n",
      "iupac_m_487_> NH1/2 swapped:      179.1820      275  275\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      105.8622      280  280\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -176.9851      287  287\n",
      "iupac_m_487_> NH1/2 swapped:      179.6368      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.7113      303  303\n",
      "iupac_m_486_> OE1/2 will be swapped:     -131.5843      354  354\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -122.1268      355  355\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -154.0215      368  368\n",
      "iupac_m_485_> OD1/2 will be swapped:      106.9681      371  371\n",
      "iupac_m_485_> OD1/2 will be swapped:      131.1256      380  380\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.6749      389  389\n",
      "iupac_m_485_> OD1/2 will be swapped:     -147.1913      402  402\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.2328      406  406\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.2449      408  408\n",
      "iupac_m_486_> OE1/2 will be swapped:      143.7923      414  414\n",
      "iupac_m_485_> OD1/2 will be swapped:      148.8528      415  415\n",
      "iupac_m_487_> NH1/2 swapped:      179.9161      416  416\n",
      "iupac_m_486_> OE1/2 will be swapped:     -121.7451      418  418\n",
      "iupac_m_487_> NH1/2 swapped:      179.1739      424  424\n",
      "iupac_m_487_> NH1/2 swapped:      179.2197      431  431\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.6944      456  456\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.2586      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -116.4629      460  460\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.8629      488  488\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.1268      489  489\n",
      "iupac_m_487_> NH1/2 swapped:     -176.3146      497  497\n",
      "iupac_m_486_> OE1/2 will be swapped:      154.3990      500  500\n",
      "iupac_m_485_> OD1/2 will be swapped:      125.5696      506  506\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      158.4980      507  507\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2840      532  532\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0954      543  543\n",
      "iupac_m_486_> OE1/2 will be swapped:     -142.4159      553  553\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.9148      554  554\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.9079      556  556\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.9262      558  558\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.5441      562  562\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.3874      569  569\n",
      "iupac_m_485_> OD1/2 will be swapped:      167.4361      574  574\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   924142\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -39699.6914\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -39699.691406\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    32.907001\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sequence length          :          577\n",
      "Compactness              :     0.130636\n",
      "Native energy (pair)     :   624.110802\n",
      "Native energy (surface)  :    48.927767\n",
      "Native energy (combined) :    17.207032\n",
      "Z score (pair)           :    -4.492025\n",
      "Z score (surface)        :    -1.963639\n",
      "Z score (combined)       :    -4.227874\n",
      "GA341 score              :     0.824211\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    11106\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        6942.6592\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4546       5     16   0.011   0.011      162.38       1.000\n",
      " 2 Bond angle potential               :    6129      15     55   2.654   2.654      813.09       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2670       0    152  52.699  52.699      1194.1       1.000\n",
      " 4 Stereochemical improper torsion pot:    1811       0      4   1.517   1.517      86.405       1.000\n",
      " 5 Soft-sphere overlap restraints     :   11106       4     15   0.011   0.011      145.28       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6300      23     94   0.940   0.940      1276.2       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6434      22    115   1.084   1.084      1357.9       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     576       2     21   5.284   5.284      189.03       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     488       0      9  88.267  88.267      192.17       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     356       0      5  78.462  78.462      164.28       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     173       0      0  93.954  93.954      129.30       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      65       0      0  93.634  93.634      42.004       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2777       0      0   0.751   0.751      88.339       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     575     246    137  42.991 112.332      958.39       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     881       0      7   1.967   1.967      143.79       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00199.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00199.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   98061.2812\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2873 367I 367I C   CA   2835 2830    1.72    1.49    0.23    6.74    1.49    0.23    6.74\n",
      "    2   2884 368R 368R C   CA   2846 2838    1.81    1.49    0.32    9.39    1.49    0.32    9.39\n",
      "    3   2885 368R 369L C   N    2846 2848    1.48    1.35    0.14    4.96    1.35    0.14    4.96\n",
      "    4   2890 369L 369L N   CA   2848 2849    1.60    1.43    0.17    5.58    1.43    0.17    5.58\n",
      "    5   3288 421L 421L C   CA   3250 3245    1.68    1.49    0.19    5.50    1.49    0.19    5.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4962  41D  42A C   N     306  308  143.08  120.00   23.08    5.25  120.00   23.08    5.25\n",
      "    2   4965  42A  42A N   CA    308  309  124.52  107.00   17.52    5.04  107.00   17.52    5.04\n",
      "    3   8236 353L 354E C   N    2712 2714  139.90  120.00   19.90    4.52  120.00   19.90    4.52\n",
      "    4   8397 366Q 367I C   N    2827 2829  145.77  120.00   25.77    5.86  120.00   25.77    5.86\n",
      "    5   8401 367I 367I CA  CB   2830 2831  131.64  111.00   20.65    4.84  111.00   20.65    4.84\n",
      "    6   8407 367I 367I C   CA   2835 2830  140.41  108.00   32.41    7.51  108.00   32.41    7.51\n",
      "    7   8408 367I 368R C   N    2835 2837  165.27  120.00   45.27   10.29  120.00   45.27   10.29\n",
      "    8   8411 368R 368R N   CA   2837 2838  128.10  107.00   21.10    6.07  107.00   21.10    6.07\n",
      "    9   8414 368R 368R CA  C    2838 2846  136.61  116.50   20.11    5.78  116.50   20.11    5.78\n",
      "   10   8425 369L 369L N   CA   2848 2849  146.33  107.00   39.33   11.31  107.00   39.33   11.31\n",
      "   11   8952 420L 421L C   N    3242 3244  143.24  120.00   23.24    5.28  120.00   23.24    5.28\n",
      "   12   9305 456F 456F N   CA   3502 3503  126.08  107.00   19.08    5.49  107.00   19.08    5.49\n",
      "   13   9318 456F 457G C   N    3511 3513  155.60  120.00   35.60    8.09  120.00   35.60    8.09\n",
      "   14   9735 495L 495L N   CA   3802 3803  123.76  107.00   16.76    4.82  107.00   16.76    4.82\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  17715   9S 421L CA  CA     53 3245    5.96    4.39    1.57    5.04    4.39    1.57    5.04\n",
      "    2  21234 142E 369L CA  CA   1082 2849   16.18   13.27    2.91    4.69   13.27    2.91    4.69\n",
      "    3  21287 143V 369L CA  CA   1091 2849   14.88   10.52    4.36    7.97   10.52    4.36    7.97\n",
      "    4  21334 144Y 369L CA  CA   1098 2849   15.91   10.69    5.23    8.39   10.69    5.23    8.39\n",
      "    5  21371 145M 369L CA  CA   1110 2849   15.94    9.55    6.39    9.45    9.55    6.39    9.45\n",
      "    6  21409 146E 369L CA  CA   1118 2849   16.21   10.18    6.03    7.86   10.18    6.03    7.86\n",
      "    7  21453 147K 369L CA  CA   1127 2849   19.04   13.29    5.75    6.09   13.29    5.75    6.09\n",
      "    8  22365 283R 369L CA  CA   2182 2849   13.49    8.79    4.70    6.62    8.79    4.70    6.62\n",
      "    9  22387 284K 369L CA  CA   2193 2849   13.23    9.25    3.98    6.12    9.25    3.98    6.12\n",
      "   10  22412 285G 369L CA  CA   2202 2849   11.18    8.48    2.70    4.66    8.48    2.70    4.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  23900  10R 421L N   O      58 3251    4.76    2.76    2.00    6.64    2.76    2.00    6.64\n",
      "    2  24456  43E  40I N   O     313  299    5.72    3.17    2.55    5.50    3.17    2.55    5.50\n",
      "    3  24485  46R  43E N   O     330  321    5.22    2.79    2.43    5.31    2.79    2.43    5.31\n",
      "    4  25413  87Y 420L N   O     660 3243    4.81    3.01    1.80    6.15    3.01    1.80    6.15\n",
      "    5  25608  94K 420L N   O     706 3243    9.16    6.90    2.26    5.07    6.90    2.26    5.07\n",
      "    6  25697  97S 420L N   O     732 3243   10.71    8.28    2.43    4.54    8.28    2.43    4.54\n",
      "    7  27573 277M 283R N   O    2134 2191   15.97   10.79    5.19    4.51   10.79    5.19    4.51\n",
      "    8  28470 342N 144Y N   O    2622 1108   10.38    6.52    3.86    5.14    6.52    3.86    5.14\n",
      "    9  28567 369L 143V N   O    2848 1096   13.85    7.94    5.91   11.39    7.94    5.91   11.39\n",
      "   10  28568 369L 144Y N   O    2848 1108   17.90   10.51    7.40   10.71   10.51    7.40   10.71\n",
      "   11  28569 369L 145M N   O    2848 1116   18.23    9.65    8.59   11.26    9.65    8.59   11.26\n",
      "   12  28570 369L 146E N   O    2848 1125   18.78   10.73    8.04    9.16   10.73    8.04    9.16\n",
      "   13  28572 369L 283R N   O    2848 2191   14.25    8.61    5.64    7.09    8.61    5.64    7.09\n",
      "   14  28574 369L 285G N   O    2848 2204   13.40    9.94    3.46    4.79    9.94    3.46    4.79\n",
      "   15  29391 421L   8A N   O    3244   51    4.18    2.59    1.59    4.93    2.59    1.59    4.93\n",
      "   16  29559 452A  10R N   O    3474   68   12.71    9.23    3.48    4.86    9.23    3.48    4.86\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16152 421L 421L CA  C    3245 3250 -154.18 -180.00   25.82    5.16 -180.00   25.82    5.16\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15157   1M   2G C   N       7    9   93.66   78.70   45.40    0.75   82.20  142.99    7.52\n",
      "    1          2G   2G N   CA      9   10  151.04 -166.10                    8.50\n",
      "    2  15158   2G   3E C   N      11   13 -177.03 -117.80  107.12    3.91  -63.60  147.09   25.17\n",
      "    2          3E   3E N   CA     13   14 -133.94  136.80                  -40.30\n",
      "    3  15175  19P  20P C   N     134  136  -63.28  -64.50   11.42    0.92  -58.70  171.01   14.50\n",
      "    3         20P  20P N   CA    136  137  158.55  147.20                  -30.50\n",
      "    4  15176  20P  21R C   N     141  143   58.10  -63.00  168.93   30.31  -63.00  168.93   30.31\n",
      "    4         21R  21R N   CA    143  144   76.68  -41.10                  -41.10\n",
      "    5  15177  21R  22Q C   N     152  154  -63.80  -73.00    9.21    0.66  -63.80  178.62   26.26\n",
      "    5         22Q  22Q N   CA    154  155  141.08  140.70                  -40.30\n",
      "    6  15186  30A  31D C   N     220  222   59.47   54.50   21.55    1.22  -63.30  136.61   23.32\n",
      "    6         31D  31D N   CA    222  223   19.92   40.90                  -40.00\n",
      "    7  15188  32S  33D C   N     234  236   47.68  -70.90  129.32   10.83  -63.30  162.06   18.09\n",
      "    7         33D  33D N   CA    236  237 -158.10  150.30                  -40.00\n",
      "    8  15189  33D  34D C   N     242  244  -94.18  -96.50   69.27    2.91  -63.30   90.41    9.74\n",
      "    8         34D  34D N   CA    244  245   44.97  114.20                  -40.00\n",
      "    9  15199  43E  44G C   N     320  322   70.21   82.20   12.17    0.88  -62.40  140.91   26.52\n",
      "    9         44G  44G N   CA    322  323    6.46    8.50                  -41.20\n",
      "   10  15202  46R  47A C   N     339  341  -64.21  -62.50    8.68    1.51  -68.20  165.34   13.48\n",
      "   10         47A  47A N   CA    341  342  -49.40  -40.90                  145.30\n",
      "   11  15203  47A  48S C   N     344  346  141.73 -136.60   88.09    4.35  -72.40  149.83   11.53\n",
      "   11         48S  48S N   CA    346  347  118.19  151.20                  152.40\n",
      "   12  15213  57F  58T C   N     433  435 -143.25  -63.20   93.13   11.40 -124.80  139.24    6.52\n",
      "   12         58T  58T N   CA    435  436    5.48  -42.10                  143.50\n",
      "   13  15257 101D 102L C   N     767  769  -60.30  -63.50   12.33    1.56 -108.50 -179.07   10.18\n",
      "   13        102L 102L N   CA    769  770  -53.11  -41.20                  132.50\n",
      "   14  15259 103N 104S C   N     783  785  164.72 -136.60   77.91    2.45  -64.10  179.51   22.36\n",
      "   14        104S 104S N   CA    785  786 -157.54  151.20                  -35.00\n",
      "   15  15260 104S 105K C   N     789  791  -62.40  -62.90    2.56    0.36   56.60  141.68   18.03\n",
      "   15        105K 105K N   CA    791  792  -38.29  -40.80                   38.60\n",
      "   16  15261 105K 106E C   N     798  800 -128.52 -117.80  100.59    4.85  -63.60  105.29   17.50\n",
      "   16        106E 106E N   CA    800  801 -123.19  136.80                  -40.30\n",
      "   17  15262 106E 107G C   N     807  809  112.35   82.20   41.49    1.34  -62.40  176.03   32.22\n",
      "   17        107G 107G N   CA    809  810  -20.00    8.50                  -41.20\n",
      "   18  15275 119I 120E C   N     906  908  -67.20  -69.30   13.64    0.97  -63.60  169.37   22.39\n",
      "   18        120E 120E N   CA    908  909  129.03  142.50                  -40.30\n",
      "   19  15276 120E 121E C   N     915  917   56.64  -63.60  135.13   18.00  -63.60  135.13   18.00\n",
      "   19        121E 121E N   CA    917  918 -101.96  -40.30                  -40.30\n",
      "   20  15284 128S 129D C   N     979  981  -54.94  -63.30    8.42    1.38  -70.90  171.41    9.25\n",
      "   20        129D 129D N   CA    981  982  -39.04  -40.00                  150.30\n",
      "   21  15285 129D 130S C   N     987  989   78.10  -64.10  146.00   17.26  -64.10  146.00   17.26\n",
      "   21        130S 130S N   CA    989  990  -68.06  -35.00                  -35.00\n",
      "   22  15311 155R 156A C   N    1198 1200  -74.89  -68.20    8.87    0.56  -62.50  168.44   28.23\n",
      "   22        156A 156A N   CA   1200 1201  151.12  145.30                  -40.90\n",
      "   23  15314 158G 159Q C   N    1218 1220 -133.48 -121.10   17.39    0.89  -63.80 -178.31   23.75\n",
      "   23        159Q 159Q N   CA   1220 1221  127.49  139.70                  -40.30\n",
      "   24  15315 159Q 160E C   N    1227 1229  167.81 -117.80   91.94    2.75  -63.60 -177.95   30.85\n",
      "   24        160E 160E N   CA   1229 1230 -169.17  136.80                  -40.30\n",
      "   25  15317 161P 162E C   N    1243 1245   38.00  -69.30  118.55    7.21  -63.60  166.88   27.68\n",
      "   25        162E 162E N   CA   1245 1246   92.09  142.50                  -40.30\n",
      "   26  15318 162E 163S C   N    1252 1254  -96.33 -136.60   48.58    1.48  -64.10  162.26   10.29\n",
      "   26        163S 163S N   CA   1254 1255  124.02  151.20                  -35.00\n",
      "   27  15325 169D 170S C   N    1310 1312  -64.42 -136.60   73.22    2.60  -64.10  173.90   12.58\n",
      "   27        170S 170S N   CA   1312 1313  138.90  151.20                  -35.00\n",
      "   28  15326 170S 171Q C   N    1316 1318  -71.59  -73.00   15.31    0.97  -63.80  165.95   24.00\n",
      "   28        171Q 171Q N   CA   1318 1319  125.46  140.70                  -40.30\n",
      "   29  15327 171Q 172H C   N    1325 1327 -126.71 -125.60   47.43    1.82  -63.20  148.00   15.79\n",
      "   29        172H 172H N   CA   1327 1328   91.38  138.80                  -42.30\n",
      "   30  15328 172H 173R C   N    1335 1337 -119.06 -125.20   28.45    1.23  -63.00  163.81   19.07\n",
      "   30        173R 173R N   CA   1337 1338  112.82  140.60                  -41.10\n",
      "   31  15330 174V 175Y C   N    1353 1355   69.79 -124.30  165.93   12.55  -63.50 -138.88   41.16\n",
      "   31        175Y 175Y N   CA   1355 1356  133.02  135.40                  -43.40\n",
      "   32  15332 176V 177K C   N    1372 1374  -61.34  -70.20   11.37    0.95  -62.90  171.70   22.21\n",
      "   32        177K 177K N   CA   1374 1375  147.51  140.40                  -40.80\n",
      "   33  15333 177K 178G C   N    1381 1383 -168.80 -167.20    4.04    0.17   82.20 -164.42   14.23\n",
      "   33        178G 178G N   CA   1383 1384  170.89  174.60                    8.50\n",
      "   34  15334 178G 179A C   N    1385 1387  -69.51  -68.20    1.50    0.10  -62.50  173.22   28.74\n",
      "   34        179A 179A N   CA   1387 1388  146.02  145.30                  -40.90\n",
      "   35  15335 179A 180G C   N    1390 1392  139.46 -167.20   53.76    1.19   82.20  169.40   10.93\n",
      "   35        180G 180G N   CA   1392 1393  167.93  174.60                    8.50\n",
      "   36  15337 181V 182H C   N    1401 1403 -149.64 -125.60   39.75    1.03  -63.20  170.74   26.91\n",
      "   36        182H 182H N   CA   1403 1404  170.45  138.80                  -42.30\n",
      "   37  15338 182H 183S C   N    1411 1413 -123.71 -136.60   13.72    0.44  -64.10 -171.82   17.80\n",
      "   37        183S 183S N   CA   1413 1414  146.51  151.20                  -35.00\n",
      "   38  15341 185L 186S C   N    1432 1434  128.22 -136.60   95.40    3.95  -64.10 -114.15   19.32\n",
      "   38        186S 186S N   CA   1434 1435  144.79  151.20                  -35.00\n",
      "   39  15343 187L 188E C   N    1446 1448  -32.05  -69.30   37.92    2.62  -63.60  178.49   25.57\n",
      "   39        188E 188E N   CA   1448 1449  135.38  142.50                  -40.30\n",
      "   40  15352 196K 197G C   N    1525 1527   76.84   78.70   15.39    0.51   82.20  170.21    7.99\n",
      "   40        197G 197G N   CA   1527 1528  178.63 -166.10                    8.50\n",
      "   41  15353 197G 198N C   N    1529 1531 -129.51 -119.90   10.74    0.33  -63.20 -170.88   27.41\n",
      "   41        198N 198N N   CA   1531 1532  141.79  137.00                  -41.10\n",
      "   42  15354 198N 199A C   N    1537 1539  -73.07  -68.20    5.10    0.38  -62.50  172.61   28.82\n",
      "   42        199A 199A N   CA   1539 1540  146.81  145.30                  -40.90\n",
      "   43  15355 199A 200N C   N    1542 1544 -110.18 -119.90   26.10    0.94  -63.20  160.89   17.95\n",
      "   43        200N 200N N   CA   1544 1545  112.78  137.00                  -41.10\n",
      "   44  15356 200N 201R C   N    1550 1552 -157.60 -125.20   36.44    1.10  -63.00 -172.73   31.65\n",
      "   44        201R 201R N   CA   1552 1553  157.28  140.60                  -41.10\n",
      "   45  15357 201R 202Q C   N    1561 1563 -103.42 -121.10   34.62    1.28  -63.80  155.37   20.98\n",
      "   45        202Q 202Q N   CA   1563 1564  109.93  139.70                  -40.30\n",
      "   46  15358 202Q 203T C   N    1570 1572 -143.23 -124.80   30.46    0.95  -63.20  170.14   26.99\n",
      "   46        203T 203T N   CA   1572 1573  167.75  143.50                  -42.10\n",
      "   47  15359 203T 204G C   N    1577 1579 -161.43 -167.20   30.26    1.10   82.20  179.28   13.51\n",
      "   47        204G 204G N   CA   1579 1580  144.89  174.60                    8.50\n",
      "   48  15360 204G 205E C   N    1581 1583 -127.78 -117.80   10.35    0.32  -63.60 -169.04   22.61\n",
      "   48        205E 205E N   CA   1583 1584  139.55  136.80                  -40.30\n",
      "   49  15361 205E 206T C   N    1590 1592 -109.32 -124.80   51.73    1.79  -63.20  143.83   15.87\n",
      "   49        206T 206T N   CA   1592 1593   94.13  143.50                  -42.10\n",
      "   50  15362 206T 207K C   N    1597 1599 -176.12 -118.00   61.73    2.07  -62.90 -164.58   32.06\n",
      "   50        207K 207K N   CA   1599 1600  159.92  139.10                  -40.80\n",
      "   51  15363 207K 208M C   N    1606 1608  -99.00 -125.60   64.17    2.42  -63.40  127.67   16.65\n",
      "   51        208M 208M N   CA   1608 1609   82.11  140.50                  -40.50\n",
      "   52  15364 208M 209N C   N    1614 1616 -163.01 -119.90   45.01    1.54  -63.20 -163.75   30.09\n",
      "   52        209N 209N N   CA   1616 1617  149.93  137.00                  -41.10\n",
      "   53  15365 209N 210E C   N    1622 1624 -134.47 -117.80   16.68    0.62  -63.60 -169.73   22.35\n",
      "   53        210E 210E N   CA   1624 1625  136.27  136.80                  -40.30\n",
      "   54  15366 210E 211T C   N    1631 1633  -90.63  -78.10   19.51    0.68  -63.20  155.58   21.78\n",
      "   54        211T 211T N   CA   1633 1634  164.76  149.80                  -42.10\n",
      "   55  15368 212S 213S C   N    1644 1646 -159.93 -136.60   25.41    0.80  -64.10 -170.30   20.65\n",
      "   55        213S 213S N   CA   1646 1647  161.28  151.20                  -35.00\n",
      "   56  15369 213S 214R C   N    1650 1652  -80.67  -72.10   12.88    0.84  -63.00  168.32   23.95\n",
      "   56        214R 214R N   CA   1652 1653  151.51  141.90                  -41.10\n",
      "   57  15370 214R 215S C   N    1661 1663  -63.34  -72.40   46.76    2.45  -64.10  141.53   10.30\n",
      "   57        215S 215S N   CA   1663 1664  106.52  152.40                  -35.00\n",
      "   58  15371 215S 216H C   N    1667 1669 -169.45 -125.60   50.21    1.29  -63.20 -172.54   30.19\n",
      "   58        216H 216H N   CA   1669 1670  163.26  138.80                  -42.30\n",
      "   59  15372 216H 217A C   N    1677 1679  -99.80  -68.20   63.48    6.07  -62.50  136.35   20.86\n",
      "   59        217A 217A N   CA   1679 1680   90.25  145.30                  -40.90\n",
      "   60  15375 219V 220Q C   N    1697 1699   71.09  -73.00  144.61   10.55  -73.00  144.61   10.55\n",
      "   60        220Q 220Q N   CA   1699 1700  153.06  140.70                  140.70\n",
      "   61  15380 224S 225Q C   N    1735 1737  -84.04  -63.80   72.05    9.66  -63.80   72.05    9.66\n",
      "   61        225Q 225Q N   CA   1737 1738   28.85  -40.30                  -40.30\n",
      "   62  15383 227Y 228G C   N    1765 1767  104.25   82.20   33.31    0.98  -62.40  168.48   30.98\n",
      "   62        228G 228G N   CA   1767 1768  -16.47    8.50                  -41.20\n",
      "   63  15386 230L 231D C   N    1783 1785  -85.24  -70.90   36.24    1.23  -63.30  138.18   18.45\n",
      "   63        231D 231D N   CA   1785 1786 -176.42  150.30                  -40.00\n",
      "   64  15388 232M 233R C   N    1799 1801 -162.19 -125.20   85.26    4.68  -63.00  144.35   16.91\n",
      "   64        233R 233R N   CA   1801 1802   63.77  140.60                  -41.10\n",
      "   65  15389 233R 234D C   N    1810 1812  -71.97  -63.30   56.13    7.48  -63.30   56.13    7.48\n",
      "   65        234D 234D N   CA   1812 1813  -95.46  -40.00                  -40.00\n",
      "   66  15392 236E 237S C   N    1834 1836   66.11  -72.40  141.44   10.81  -64.10 -165.90   15.06\n",
      "   66        237S 237S N   CA   1836 1837 -178.95  152.40                  -35.00\n",
      "   67  15395 239V 240L C   N    1854 1856 -136.57 -108.50   60.78    2.93  -63.50  151.22   25.50\n",
      "   67        240L 240L N   CA   1856 1857 -173.59  132.50                  -41.20\n",
      "   68  15396 240L 241L C   N    1862 1864  -64.82  -70.70   32.20    2.23  -63.50  151.14   20.91\n",
      "   68        241L 241L N   CA   1864 1865  109.94  141.60                  -41.20\n",
      "   69  15398 242V 243D C   N    1877 1879  -53.49  -63.30   60.60    6.94  -63.30   60.60    6.94\n",
      "   69        243D 243D N   CA   1879 1880  -99.80  -40.00                  -40.00\n",
      "   70  15409 253T 254E C   N    1959 1961  105.26 -117.80  153.44    4.49  -63.60 -156.44   25.87\n",
      "   70        254E 254E N   CA   1961 1962 -153.98  136.80                  -40.30\n",
      "   71  15410 254E 255S C   N    1968 1970 -155.42 -136.60   26.44    0.85  -64.10 -179.91   19.63\n",
      "   71        255S 255S N   CA   1970 1971  169.78  151.20                  -35.00\n",
      "   72  15411 255S 256T C   N    1974 1976 -152.83 -124.80   55.92    1.75  -63.20  154.64   25.43\n",
      "   72        256T 256T N   CA   1976 1977 -168.11  143.50                  -42.10\n",
      "   73  15412 256T 257G C   N    1981 1983 -143.90 -167.20   26.07    0.40   82.20 -155.63   15.44\n",
      "   73        257G 257G N   CA   1983 1984  162.90  174.60                    8.50\n",
      "   74  15414 258V 259A C   N    1992 1994 -171.73 -134.00   38.57    1.01  -62.50 -162.83   38.10\n",
      "   74        259A 259A N   CA   1994 1995  154.96  147.00                  -40.90\n",
      "   75  15415 259A 260F C   N    1997 1999 -171.16 -124.20   53.17    1.43  -63.20 -177.26   31.93\n",
      "   75        260F 260F N   CA   1999 2000  168.25  143.30                  -44.30\n",
      "   76  15416 260F 261E C   N    2008 2010  -75.43  -69.30   29.68    2.47  -63.60  154.21   19.94\n",
      "   76        261E 261E N   CA   2010 2011  113.46  142.50                  -40.30\n",
      "   77  15418 262E 263A C   N    2026 2028  141.99 -134.00   91.89    2.09  -62.50 -154.16   41.45\n",
      "   77        263A 263A N   CA   2028 2029 -175.76  147.00                  -40.90\n",
      "   78  15419 263A 264K C   N    2031 2033  -68.41  -70.20    2.47    0.21  -62.90  177.19   23.31\n",
      "   78        264K 264K N   CA   2033 2034  142.10  140.40                  -40.80\n",
      "   79  15420 264K 265K C   N    2040 2042  -82.34  -70.20   17.90    1.07  -62.90  166.78   22.77\n",
      "   79        265K 265K N   CA   2042 2043  153.55  140.40                  -40.80\n",
      "   80  15422 266I 267N C   N    2057 2059  -77.11  -71.20    7.88    0.43  -63.20  171.45   22.21\n",
      "   80        267N 267N N   CA   2059 2060  148.01  142.80                  -41.10\n",
      "   81  15423 267N 268Q C   N    2065 2067 -128.06 -121.10   17.69    0.70  -63.80  175.89   29.26\n",
      "   81        268Q 268Q N   CA   2067 2068  155.96  139.70                  -40.30\n",
      "   82  15426 270L 271L C   N    2088 2090   77.97  -70.70  149.16   13.15  -63.50 -138.28   39.25\n",
      "   82        271L 271L N   CA   2090 2091  129.51  141.60                  -41.20\n",
      "   83  15427 271L 272M C   N    2096 2098  -97.33  -73.00   34.39    2.73  -63.40  162.77   21.89\n",
      "   83        272M 272M N   CA   2098 2099  118.70  143.00                  -40.50\n",
      "   84  15428 272M 273L C   N    2104 2106  -78.98  -70.70   11.75    0.75  -63.50  169.57   24.46\n",
      "   84        273L 273L N   CA   2106 2107  149.93  141.60                  -41.20\n",
      "   85  15429 273L 274G C   N    2112 2114  -88.18  -80.20  106.18    4.58   82.20 -179.45    9.45\n",
      "   85        274G 274G N   CA   2114 2115   68.22  174.10                    8.50\n",
      "   86  15430 274G 275R C   N    2116 2118 -126.61 -125.20   56.06    2.74  -63.00  140.84   16.00\n",
      "   86        275R 275R N   CA   2118 2119   84.56  140.60                  -41.10\n",
      "   87  15431 275R 276A C   N    2127 2129 -169.86 -134.00   37.17    1.61  -62.50 -152.01   31.18\n",
      "   87        276A 276A N   CA   2129 2130  137.24  147.00                  -40.90\n",
      "   88  15434 278N 279S C   N    2148 2150  -58.11  -72.40   14.32    0.99  -64.10  173.64   12.24\n",
      "   88        279S 279S N   CA   2150 2151  151.46  152.40                  -35.00\n",
      "   89  15454 298T 299R C   N    2299 2301 -156.13 -125.20   30.94    1.27  -63.00 -158.03   33.60\n",
      "   89        299R 299R N   CA   2301 2302  139.69  140.60                  -41.10\n",
      "   90  15455 299R 300L C   N    2310 2312 -133.74 -108.50   26.51    1.19  -63.50 -168.48   30.94\n",
      "   90        300L 300L N   CA   2312 2313  140.62  132.50                  -41.20\n",
      "   91  15456 300L 301L C   N    2318 2320 -111.18 -108.50    2.71    0.12  -63.50 -179.48   22.81\n",
      "   91        301L 301L N   CA   2320 2321  132.91  132.50                  -41.20\n",
      "   92  15457 301L 302S C   N    2326 2328 -143.85 -136.60   13.58    0.50  -64.10 -179.16   18.81\n",
      "   92        302S 302S N   CA   2328 2329  162.69  151.20                  -35.00\n",
      "   93  15458 302S 303E C   N    2332 2334 -101.36 -117.80   16.99    0.53  -63.60  176.89   21.73\n",
      "   93        303E 303E N   CA   2334 2335  132.51  136.80                  -40.30\n",
      "   94  15459 303E 304S C   N    2341 2343 -133.53 -136.60    7.68    0.43  -64.10 -179.37   18.02\n",
      "   94        304S 304S N   CA   2343 2344  158.24  151.20                  -35.00\n",
      "   95  15460 304S 305F C   N    2347 2349 -108.65 -124.20   19.66    0.51  -63.20 -178.63   22.34\n",
      "   95        305F 305F N   CA   2349 2350  131.28  143.30                  -44.30\n",
      "   96  15461 305F 306G C   N    2358 2360   59.74   78.70   29.57    0.57   82.20  153.57    8.57\n",
      "   96        306G 306G N   CA   2360 2361 -143.41 -166.10                    8.50\n",
      "   97  15462 306G 307G C   N    2362 2364 -177.65 -167.20   18.78    0.75   82.20 -179.24   13.13\n",
      "   97        307G 307G N   CA   2364 2365  158.99  174.60                    8.50\n",
      "   98  15463 307G 308N C   N    2366 2368 -127.25 -119.90   18.51    0.66  -63.20  176.92   25.76\n",
      "   98        308N 308N N   CA   2368 2369  153.99  137.00                  -41.10\n",
      "   99  15464 308N 309S C   N    2374 2376  -65.86  -72.40   18.85    0.91  -64.10  169.73   12.19\n",
      "   99        309S 309S N   CA   2376 2377  134.72  152.40                  -35.00\n",
      "  100  15465 309S 310K C   N    2380 2382 -171.51 -118.00   87.55    3.26  -62.90  155.16   26.35\n",
      "  100        310K 310K N   CA   2382 2383 -151.60  139.10                  -40.80\n",
      "  101  15466 310K 311T C   N    2389 2391  -75.47  -78.10   46.03    2.05  -63.20  122.76   16.56\n",
      "  101        311T 311T N   CA   2391 2392 -164.25  149.80                  -42.10\n",
      "  102  15468 312W 313M C   N    2410 2412 -121.69 -125.60    4.17    0.20  -63.40 -173.12   32.55\n",
      "  102        313M 313M N   CA   2412 2413  141.94  140.50                  -40.50\n",
      "  103  15469 313M 314L C   N    2418 2420 -114.35 -108.50    9.21    0.52  -63.50  174.18   21.85\n",
      "  103        314L 314L N   CA   2420 2421  125.39  132.50                  -41.20\n",
      "  104  15470 314L 315A C   N    2426 2428 -169.65 -134.00   40.73    0.94  -62.50 -173.68   36.19\n",
      "  104        315A 315A N   CA   2428 2429  166.68  147.00                  -40.90\n",
      "  105  15471 315A 316T C   N    2431 2433  -93.64 -124.80   82.48    2.72  -63.20  113.40   12.77\n",
      "  105        316T 316T N   CA   2433 2434   67.14  143.50                  -42.10\n",
      "  106  15473 317V 318S C   N    2445 2447 -115.62 -136.60   31.22    1.03  -64.10  171.02   10.54\n",
      "  106        318S 318S N   CA   2447 2448  128.07  151.20                  -35.00\n",
      "  107  15474 318S 319T C   N    2451 2453 -128.46 -124.80    8.38    0.46  -63.20 -170.36   20.75\n",
      "  107        319T 319T N   CA   2453 2454  135.96  143.50                  -42.10\n",
      "  108  15477 321A 322N C   N    2468 2470 -157.79 -119.90   43.65    2.27   55.90  164.79   20.34\n",
      "  108        322N 322N N   CA   2470 2471  115.33  137.00                   39.50\n",
      "  109  15495 339A 340I C   N    2605 2607  -62.67  -63.40    4.08    0.69 -120.60  179.50    8.17\n",
      "  109        340I 340I N   CA   2607 2608  -39.59  -43.60                  130.30\n",
      "  110  15497 341T 342N C   N    2620 2622 -106.22 -119.90   35.93    1.88  -63.20  154.78   21.80\n",
      "  110        342N 342N N   CA   2622 2623  170.22  137.00                  -41.10\n",
      "  111  15499 343K 344A C   N    2637 2639  -58.10  -68.20   11.37    0.78  -62.50  179.07   29.13\n",
      "  111        344A 344A N   CA   2639 2640  140.09  145.30                  -40.90\n",
      "  112  15501 345K 346V C   N    2651 2653  -67.61  -62.40   50.17    6.66  -62.40   50.17    6.66\n",
      "  112        346V 346V N   CA   2653 2654  -92.30  -42.40                  -42.40\n",
      "  113  15504 348K 349T C   N    2675 2677  -84.54 -124.80   41.90    2.44  -63.20  164.20   22.48\n",
      "  113        349T 349T N   CA   2677 2678  155.09  143.50                  -42.10\n",
      "  114  15505 349T 350A C   N    2682 2684 -174.38 -134.00   40.93    1.12  -62.50 -160.24   38.66\n",
      "  114        350A 350A N   CA   2684 2685  153.62  147.00                  -40.90\n",
      "  115  15506 350A 351K C   N    2687 2689  -78.11  -70.20   29.69    1.96  -62.90  150.95   20.47\n",
      "  115        351K 351K N   CA   2689 2690  169.01  140.40                  -40.80\n",
      "  116  15507 351K 352D C   N    2696 2698  -71.34  -70.90   17.42    0.84  -63.30  152.50   19.32\n",
      "  116        352D 352D N   CA   2698 2699  167.71  150.30                  -40.00\n",
      "  117  15508 352D 353L C   N    2704 2706 -144.56 -108.50   86.61    4.23  -63.50  134.67   23.60\n",
      "  117        353L 353L N   CA   2706 2707 -148.75  132.50                  -41.20\n",
      "  118  15509 353L 354E C   N    2712 2714  -65.97  -69.30   14.86    1.24  -63.60  162.73   21.83\n",
      "  118        354E 354E N   CA   2714 2715  156.99  142.50                  -40.30\n",
      "  119  15510 354E 355Y C   N    2721 2723 -140.20 -124.30   38.45    1.53  -63.50  165.09   29.92\n",
      "  119        355Y 355Y N   CA   2723 2724  170.41  135.40                  -43.40\n",
      "  120  15511 355Y 356D C   N    2733 2735  179.63  -96.50  107.31    4.47  -63.30 -178.38   29.68\n",
      "  120        356D 356D N   CA   2735 2736 -178.85  114.20                  -40.00\n",
      "  121  15512 356D 357E C   N    2741 2743  -91.60  -69.30   23.00    1.93  -63.60  179.39   22.49\n",
      "  121        357E 357E N   CA   2743 2744  136.89  142.50                  -40.30\n",
      "  122  15514 358I 359Q C   N    2758 2760 -133.32 -121.10   58.59    2.56  -63.80  141.02   24.31\n",
      "  122        359Q 359Q N   CA   2760 2761 -163.00  139.70                  -40.30\n",
      "  123  15516 360K 361L C   N    2776 2778  -91.92 -108.50   22.16    0.99  -63.50  161.53   20.97\n",
      "  123        361L 361L N   CA   2778 2779  117.81  132.50                  -41.20\n",
      "  124  15517 361L 362A C   N    2784 2786  -92.60  -68.20   30.64    1.98  -62.50  158.16   27.49\n",
      "  124        362A 362A N   CA   2786 2787  163.83  145.30                  -40.90\n",
      "  125  15518 362A 363W C   N    2789 2791 -136.05 -124.90   39.81    1.65  -63.00  152.78   24.02\n",
      "  125        363W 363W N   CA   2791 2792 -178.38  143.40                  -44.20\n",
      "  126  15519 363W 364V C   N    2803 2805   64.08  -73.50  137.59    9.74 -125.40  170.61    6.83\n",
      "  126        364V 364V N   CA   2805 2806  137.90  139.20                  143.30\n",
      "  127  15520 364V 365L C   N    2810 2812 -165.92  -63.50  114.68   16.72  -63.50  114.68   16.72\n",
      "  127        365L 365L N   CA   2812 2813   10.39  -41.20                  -41.20\n",
      "  128  15521 365L 366Q C   N    2818 2820 -175.03 -121.10   95.62    4.99  -63.80  150.27   19.79\n",
      "  128        366Q 366Q N   CA   2820 2821   60.74  139.70                  -40.30\n",
      "  129  15522 366Q 367I C   N    2827 2829  -96.92  -63.40   58.32   10.74 -120.60  140.38    8.93\n",
      "  129        367I 367I N   CA   2829 2830  -91.33  -43.60                  130.30\n",
      "  130  15523 367I 368R C   N    2835 2837 -169.41 -125.20   78.85    4.37  -63.00  157.71   18.39\n",
      "  130        368R 368R N   CA   2837 2838   75.30  140.60                  -41.10\n",
      "  131  15524 368R 369L C   N    2846 2848  -90.97  -63.50   39.01    7.02  -63.50   39.01    7.02\n",
      "  131        369L 369L N   CA   2848 2849  -68.90  -41.20                  -41.20\n",
      "  132  15525 369L 370D C   N    2854 2856  165.82  -96.50   97.68    3.99  -63.30 -158.32   22.16\n",
      "  132        370D 370D N   CA   2856 2857  113.45  114.20                  -40.00\n",
      "  133  15527 371D 372E C   N    2870 2872  -71.47  -63.60   53.81    6.77  -63.60   53.81    6.77\n",
      "  133        372E 372E N   CA   2872 2873   12.93  -40.30                  -40.30\n",
      "  134  15534 378A 379L C   N    2916 2918  -62.44  -63.50    1.48    0.27  -70.70  178.42   13.84\n",
      "  134        379L 379L N   CA   2918 2919  -40.17  -41.20                  141.60\n",
      "  135  15539 383I 384Q C   N    2954 2956  -98.82 -121.10   26.42    1.29  -63.80  169.46   26.79\n",
      "  135        384Q 384Q N   CA   2956 2957  153.90  139.70                  -40.30\n",
      "  136  15540 384Q 385G C   N    2963 2965   88.39   82.20    7.16    0.56  -80.20 -126.20    7.66\n",
      "  136        385G 385G N   CA   2965 2966   12.11    8.50                  174.10\n",
      "  137  15542 386L 387R C   N    2975 2977 -136.69 -125.20   53.22    2.30  -63.00  146.26   24.71\n",
      "  137        387R 387R N   CA   2977 2978 -167.44  140.60                  -41.10\n",
      "  138  15543 387R 388G C   N    2986 2988  171.74 -167.20   41.76    1.69   82.20  157.87   11.53\n",
      "  138        388G 388G N   CA   2988 2989  138.53  174.60                    8.50\n",
      "  139  15544 388G 389E C   N    2990 2992 -138.97 -117.80   27.57    1.44  -63.60  176.35   20.51\n",
      "  139        389E 389E N   CA   2992 2993  119.13  136.80                  -40.30\n",
      "  140  15546 390I 391D C   N    3007 3009 -101.02  -96.50   21.14    0.88  -63.30  138.78   15.25\n",
      "  140        391D 391D N   CA   3009 3010   93.56  114.20                  -40.00\n",
      "  141  15548 392C 393M C   N    3021 3023 -127.19 -125.60    3.60    0.13  -63.40 -173.02   32.99\n",
      "  141        393M 393M N   CA   3023 3024  143.73  140.50                  -40.50\n",
      "  142  15549 393M 394K C   N    3029 3031 -114.60 -118.00   22.31    1.12  -62.90  166.29   24.67\n",
      "  142        394K 394K N   CA   3031 3032  161.15  139.10                  -40.80\n",
      "  143  15552 396K 397L C   N    3053 3055  174.28 -108.50   77.49    3.62  -63.50 -142.63   37.63\n",
      "  143        397L 397L N   CA   3055 3056  139.04  132.50                  -41.20\n",
      "  144  15553 397L 398T C   N    3061 3063 -126.75 -124.80    2.07    0.09  -63.20 -175.02   27.93\n",
      "  144        398T 398T N   CA   3063 3064  144.18  143.50                  -42.10\n",
      "  145  15555 399V 400A C   N    3075 3077 -108.71 -134.00   38.67    1.27  -62.50  165.24   25.25\n",
      "  145        400A 400A N   CA   3077 3078  117.75  147.00                  -40.90\n",
      "  146  15556 400A 401S C   N    3080 3082 -169.94 -136.60   36.42    1.82  -64.10 -158.45   13.52\n",
      "  146        401S 401S N   CA   3082 3083  136.53  151.20                  -35.00\n",
      "  147  15564 408E 409L C   N    3141 3143  -75.42  -63.50   59.48    7.66  -63.50   59.48    7.66\n",
      "  147        409L 409L N   CA   3143 3144   17.07  -41.20                  -41.20\n",
      "  148  15567 411E 412A C   N    3167 3169  -58.89  -62.50   39.20    6.61  -62.50   39.20    6.61\n",
      "  148        412A 412A N   CA   3169 3170   -1.87  -40.90                  -40.90\n",
      "  149  15568 412A 413L C   N    3172 3174  -70.28  -63.50   24.47    3.81 -108.50  167.21    9.37\n",
      "  149        413L 413L N   CA   3174 3175  -64.71  -41.20                  132.50\n",
      "  150  15580 424R 425K C   N    3275 3277  -85.62 -118.00   73.90    3.83  -62.90  115.92   16.38\n",
      "  150        425K 425K N   CA   3277 3278 -154.47  139.10                  -40.80\n",
      "  151  15582 426G 427L C   N    3288 3290 -135.31 -108.50   33.93    1.51  -63.50 -179.59   29.50\n",
      "  151        427L 427L N   CA   3290 3291  153.30  132.50                  -41.20\n",
      "  152  15584 428I 429E C   N    3304 3306  178.90 -117.80   77.42    3.94  -63.60  177.11   20.83\n",
      "  152        429E 429E N   CA   3306 3307   92.22  136.80                  -40.30\n",
      "  153  15585 429E 430Q C   N    3313 3315 -155.68 -121.10   39.00    1.20  -63.80 -173.79   32.10\n",
      "  153        430Q 430Q N   CA   3315 3316  157.73  139.70                  -40.30\n",
      "  154  15586 430Q 431R C   N    3322 3324 -109.30 -125.20   29.85    1.03  -63.00  163.14   19.37\n",
      "  154        431R 431R N   CA   3324 3325  115.34  140.60                  -41.10\n",
      "  155  15587 431R 432K C   N    3333 3335 -132.03 -118.00   20.49    0.73  -62.90  179.04   27.39\n",
      "  155        432K 432K N   CA   3335 3336  154.04  139.10                  -40.80\n",
      "  156  15589 433I 434S C   N    3350 3352 -169.60 -136.60   44.57    1.41  -64.10  178.38   20.54\n",
      "  156        434S 434S N   CA   3352 3353 -178.84  151.20                  -35.00\n",
      "  157  15590 434S 435G C   N    3356 3358 -137.00 -167.20   42.06    0.92   82.20 -163.68   15.09\n",
      "  157        435G 435G N   CA   3358 3359  145.32  174.60                    8.50\n",
      "  158  15591 435G 436F C   N    3360 3362 -139.25 -124.20   27.10    0.83  -63.20  168.06   28.21\n",
      "  158        436F 436F N   CA   3362 3363  165.83  143.30                  -44.30\n",
      "  159  15592 436F 437S C   N    3371 3373  -77.35  -72.40   14.38    0.70  -64.10  159.64   12.40\n",
      "  159        437S 437S N   CA   3373 3374  165.90  152.40                  -35.00\n",
      "  160  15593 437S 438A C   N    3377 3379  -74.11  -68.20   11.44    0.74  -62.50  164.42   27.53\n",
      "  160        438A 438A N   CA   3379 3380  155.09  145.30                  -40.90\n",
      "  161  15594 438A 439A C   N    3382 3384 -120.85 -134.00   13.66    0.60  -62.50  178.22   32.32\n",
      "  161        439A 439A N   CA   3384 3385  150.71  147.00                  -40.90\n",
      "  162  15598 442C 443F C   N    3408 3410 -140.61 -124.20   31.79    1.02  -63.20  164.52   27.78\n",
      "  162        443F 443F N   CA   3410 3411  170.53  143.30                  -44.30\n",
      "  163  15599 443F 444S C   N    3419 3421  -66.63  -72.40    9.68    0.75  -64.10  164.84   12.09\n",
      "  163        444S 444S N   CA   3421 3422  160.17  152.40                  -35.00\n",
      "  164  15600 444S 445G C   N    3425 3427   99.74   78.70   49.29    0.73   82.20  141.91    7.76\n",
      "  164        445G 445G N   CA   3427 3428  149.33 -166.10                    8.50\n",
      "  165  15603 447C 448K C   N    3442 3444  -84.57  -70.20   16.51    1.02  -62.90  172.05   23.59\n",
      "  165        448K 448K N   CA   3444 3445  148.52  140.40                  -40.80\n",
      "  166  15604 448K 449T C   N    3451 3453 -145.72 -124.80   29.66    0.96  -63.20  174.17   27.66\n",
      "  166        449T 449T N   CA   3453 3454  164.52  143.50                  -42.10\n",
      "  167  15605 449T 450S C   N    3458 3460 -116.94 -136.60   23.12    0.70  -64.10 -178.13   11.22\n",
      "  167        450S 450S N   CA   3460 3461  139.02  151.20                  -35.00\n",
      "  168  15606 450S 451L C   N    3464 3466 -149.09 -108.50   60.29    2.74  -63.50  165.55   28.24\n",
      "  168        451L 451L N   CA   3466 3467  177.09  132.50                  -41.20\n",
      "  169  15607 451L 452A C   N    3472 3474  -65.67  -62.50   14.51    2.24  -68.20  172.06   13.79\n",
      "  169        452A 452A N   CA   3474 3475  -26.74  -40.90                  145.30\n",
      "  170  15608 452A 453N C   N    3477 3479 -102.45 -119.90   66.17    2.57  -63.20  120.82   13.35\n",
      "  170        453N 453N N   CA   3479 3480   73.17  137.00                  -41.10\n",
      "  171  15612 456F 457G C   N    3511 3513 -161.73 -167.20   65.41    2.49   82.20  153.80   11.92\n",
      "  171        457G 457G N   CA   3513 3514  109.42  174.60                    8.50\n",
      "  172  15618 462T 463T C   N    3556 3558  -96.06  -78.10   70.56    2.50  -63.20  105.13   15.66\n",
      "  172        463T 463T N   CA   3558 3559 -141.97  149.80                  -42.10\n",
      "  173  15619 463T 464L C   N    3563 3565 -156.14 -108.50   48.10    2.22  -63.50 -157.87   33.79\n",
      "  173        464L 464L N   CA   3565 3566  139.15  132.50                  -41.20\n",
      "  174  15624 468C 469Y C   N    3598 3600 -118.19 -124.30    6.56    0.54  -63.50 -173.03   32.44\n",
      "  174        469Y 469Y N   CA   3600 3601  137.81  135.40                  -43.40\n",
      "  175  15625 469Y 470M C   N    3610 3612  -76.94  -73.00    4.80    0.31  -63.40  174.30   27.09\n",
      "  175        470M 470M N   CA   3612 3613  145.73  143.00                  -40.50\n",
      "  176  15626 470M 471L C   N    3618 3620  -80.52  -70.70   18.15    1.12  -63.50  162.83   23.62\n",
      "  176        471L 471L N   CA   3620 3621  156.86  141.60                  -41.20\n",
      "  177  15627 471L 472A C   N    3626 3628  -67.31  -68.20    0.96    0.09  -62.50  173.51   28.67\n",
      "  177        472A 472A N   CA   3628 3629  145.65  145.30                  -40.90\n",
      "  178  15628 472A 473G C   N    3631 3633  155.91 -167.20   42.31    1.36   82.20  162.99   11.26\n",
      "  178        473G 473G N   CA   3633 3634  153.88  174.60                    8.50\n",
      "  179  15630 474P 475P C   N    3642 3644  -69.37  -58.70   10.92    1.26  -64.50  175.47   13.35\n",
      "  179        475P 475P N   CA   3644 3645  -28.20  -30.50                  147.20\n",
      "  180  15632 476P 477F C   N    3656 3658 -112.08 -124.20   30.85    1.68  -63.20  152.10   24.36\n",
      "  180        477F 477F N   CA   3658 3659  171.67  143.30                  -44.30\n",
      "  181  15635 479I 480C C   N    3683 3685  -58.88  -63.00    6.81    0.73 -117.90 -177.79    7.89\n",
      "  181        480C 480C N   CA   3685 3686  -46.52  -41.10                  141.10\n",
      "  182  15636 480C 481H C   N    3689 3691  -64.24  -67.60    4.08    0.36  -63.20  175.39   21.99\n",
      "  182        481H 481H N   CA   3691 3692  142.31  140.00                  -42.30\n",
      "  183  15637 481H 482I C   N    3699 3701  -63.72  -63.40    4.56    0.76 -120.60 -172.70    8.63\n",
      "  183        482I 482I N   CA   3701 3702  -48.15  -43.60                  130.30\n",
      "  184  15638 482I 483H C   N    3707 3709   53.32   56.30    3.04    0.44  -63.20  143.46   24.54\n",
      "  184        483H 483H N   CA   3709 3710   41.39   40.80                  -42.30\n",
      "  185  15640 484V 485Y C   N    3724 3726 -127.99 -124.30   20.70    0.95  -63.50  173.28   30.71\n",
      "  185        485Y 485Y N   CA   3726 3727  155.77  135.40                  -43.40\n",
      "  186  15641 485Y 486L C   N    3736 3738  -64.03  -70.70    7.51    0.55  -63.50  179.36   24.87\n",
      "  186        486L 486L N   CA   3738 3739  138.16  141.60                  -41.20\n",
      "  187  15642 486L 487T C   N    3744 3746 -104.77 -124.80   21.29    0.92  -63.20 -176.84   21.00\n",
      "  187        487T 487T N   CA   3746 3747  136.28  143.50                  -42.10\n",
      "  188  15643 487T 488E C   N    3751 3753  -66.50  -69.30   11.39    0.77  -63.60  171.78   22.75\n",
      "  188        488E 488E N   CA   3753 3754  131.45  142.50                  -40.30\n",
      "  189  15644 488E 489E C   N    3760 3762 -161.66 -117.80   69.51    2.38  -63.60  162.02   26.85\n",
      "  189        489E 489E N   CA   3762 3763 -169.27  136.80                  -40.30\n",
      "  190  15646 490S 491S C   N    3775 3777 -140.92 -136.60   13.09    0.73  -64.10 -169.95   11.88\n",
      "  190        491S 491S N   CA   3777 3778  138.84  151.20                  -35.00\n",
      "  191  15648 492V 493A C   N    3788 3790  172.52 -134.00   62.02    1.45  -62.50 -171.80   37.29\n",
      "  191        493A 493A N   CA   3790 3791  178.40  147.00                  -40.90\n",
      "  192  15650 494V 495L C   N    3800 3802  -80.46  -63.50   63.56    9.87  -63.50   63.56    9.87\n",
      "  192        495L 495L N   CA   3802 3803 -102.46  -41.20                  -41.20\n",
      "  193  15655 499R 500E C   N    3846 3848  142.53 -117.80  108.36    3.20  -63.60 -151.71   35.49\n",
      "  193        500E 500E N   CA   3848 3849  179.32  136.80                  -40.30\n",
      "  194  15656 500E 501L C   N    3855 3857  -87.72  -70.70   69.26    4.65  -63.50  112.70   17.13\n",
      "  194        501L 501L N   CA   3857 3858 -151.26  141.60                  -41.20\n",
      "  195  15657 501L 502Q C   N    3863 3865  -98.62 -121.10   52.49    2.04  -63.80  137.06   18.51\n",
      "  195        502Q 502Q N   CA   3865 3866   92.26  139.70                  -40.30\n",
      "  196  15658 502Q 503H C   N    3872 3874 -163.69 -125.60   51.21    1.26  -63.20  176.15   28.41\n",
      "  196        503H 503H N   CA   3874 3875  173.02  138.80                  -42.30\n",
      "  197  15661 505P 506D C   N    3897 3899 -141.00  -70.90   80.53    4.29  -63.30  151.51   23.67\n",
      "  197        506D 506D N   CA   3899 3900 -170.07  150.30                  -40.00\n",
      "  198  15662 506D 507Y C   N    3905 3907 -125.20  -63.50   93.93   17.62  -63.50   93.93   17.62\n",
      "  198        507Y 507Y N   CA   3907 3908 -114.23  -43.40                  -43.40\n",
      "  199  15670 514V 515T C   N    3960 3962 -158.57 -124.80   78.71    2.53  -63.20  140.59   23.72\n",
      "  199        515T 515T N   CA   3962 3963 -145.40  143.50                  -42.10\n",
      "  200  15672 516V 517W C   N    3974 3976  -57.88  -71.30   13.68    1.05  -63.00  174.19   21.48\n",
      "  200        517W 517W N   CA   3976 3977  141.69  139.00                  -44.20\n",
      "  201  15673 517W 518F C   N    3988 3990  -71.25  -71.40   13.94    1.12  -63.20  171.25   23.24\n",
      "  201        518F 518F N   CA   3990 3991  126.76  140.70                  -44.30\n",
      "  202  15674 518F 519E C   N    3999 4001 -173.88 -117.80   65.71    1.93  -63.60 -174.91   30.60\n",
      "  202        519E 519E N   CA   4001 4002  171.05  136.80                  -40.30\n",
      "  203  15675 519E 520G C   N    4008 4010   98.11   78.70   44.16    0.65   82.20  146.60    7.91\n",
      "  203        520G 520G N   CA   4010 4011  154.24 -166.10                    8.50\n",
      "  204  15676 520G 521H C   N    4012 4014 -130.98 -125.60    5.84    0.16  -63.20 -170.81   28.09\n",
      "  204        521H 521H N   CA   4014 4015  141.07  138.80                  -42.30\n",
      "  205  15677 521H 522Y C   N    4022 4024 -114.56 -124.30   15.60    0.64  -63.50  174.27   25.26\n",
      "  205        522Y 522Y N   CA   4024 4025  123.22  135.40                  -43.40\n",
      "  206  15678 522Y 523S C   N    4034 4036 -116.85 -136.60   35.70    1.30  -64.10  165.11   10.17\n",
      "  206        523S 523S N   CA   4036 4037  121.45  151.20                  -35.00\n",
      "  207  15679 523S 524S C   N    4040 4042  173.83 -136.60   55.04    1.70  -64.10 -166.68   22.80\n",
      "  207        524S 524S N   CA   4042 4043  175.10  151.20                  -35.00\n",
      "  208  15680 524S 525A C   N    4046 4048  -88.79  -68.20   63.09    5.76  -62.50  129.26   20.06\n",
      "  208        525A 525A N   CA   4048 4049   85.67  145.30                  -40.90\n",
      "  209  15682 526V 527Q C   N    4058 4060 -113.15 -121.10   73.05    3.34  -63.80  118.19   15.35\n",
      "  209        527Q 527Q N   CA   4060 4061   67.09  139.70                  -40.30\n",
      "  210  15683 527Q 528T C   N    4067 4069 -126.67 -124.80   59.43    2.61  -63.20  141.26   15.04\n",
      "  210        528T 528T N   CA   4069 4070   84.10  143.50                  -42.10\n",
      "  211  15684 528T 529P C   N    4074 4076  -65.61  -58.70   14.15    1.72  -64.50  169.96   12.62\n",
      "  211        529P 529P N   CA   4076 4077  -42.84  -30.50                  147.20\n",
      "  212  15687 531V 532R C   N    4096 4098  -55.82 -125.20   71.48    2.44  -63.00  164.66   22.73\n",
      "  212        532R 532R N   CA   4098 4099  123.41  140.60                  -41.10\n",
      "  213  15688 532R 533P C   N    4107 4109  -55.13  -58.70    3.57    0.45  -64.50  178.02   12.80\n",
      "  213        533P 533P N   CA   4109 4110  -30.58  -30.50                  147.20\n",
      "  214  15689 533P 534N C   N    4114 4116  -60.58  -71.20   10.63    0.76  -63.20  176.59   21.94\n",
      "  214        534N 534N N   CA   4116 4117  142.33  142.80                  -41.10\n",
      "  215  15690 534N 535G C   N    4122 4124  160.21 -167.20   33.01    0.76   82.20  178.78   12.24\n",
      "  215        535G 535G N   CA   4124 4125  169.36  174.60                    8.50\n",
      "  216  15691 535G 536G C   N    4126 4128   93.94   78.70   41.87    0.66   82.20  146.87    7.72\n",
      "  216        536G 536G N   CA   4128 4129  154.90 -166.10                    8.50\n",
      "  217  15692 536G 537N C   N    4130 4132 -124.54 -119.90    4.89    0.16  -63.20 -170.18   20.99\n",
      "  217        537N 537N N   CA   4132 4133  138.54  137.00                  -41.10\n",
      "  218  15694 538P 539Q C   N    4145 4147 -149.11 -121.10   29.15    1.27  -63.80 -168.09   24.81\n",
      "  218        539Q 539Q N   CA   4147 4148  131.61  139.70                  -40.30\n",
      "  219  15695 539Q 540F C   N    4154 4156 -153.15 -124.20   40.89    1.10  -63.20  169.38   29.13\n",
      "  219        540F 540F N   CA   4156 4157  172.18  143.30                  -44.30\n",
      "  220  15696 540F 541S C   N    4165 4167 -122.08 -136.60   22.79    1.29  -64.10  166.64   16.16\n",
      "  220        541S 541S N   CA   4167 4168  168.77  151.20                  -35.00\n",
      "  221  15697 541S 542L C   N    4171 4173  -62.90  -70.70   17.44    1.09  -63.50  167.20   23.25\n",
      "  221        542L 542L N   CA   4173 4174  126.00  141.60                  -41.20\n",
      "  222  15698 542L 543R C   N    4179 4181 -148.98 -125.20   31.82    0.95  -63.00  179.15   30.01\n",
      "  222        543R 543R N   CA   4181 4182  161.73  140.60                  -41.10\n",
      "  223  15699 543R 544Q C   N    4190 4192  -79.73  -73.00   11.54    0.88  -63.80  172.37   24.53\n",
      "  223        544Q 544Q N   CA   4192 4193  131.33  140.70                  -40.30\n",
      "  224  15700 544Q 545M C   N    4199 4201 -149.18 -125.60   26.57    0.80  -63.40 -172.48   34.70\n",
      "  224        545M 545M N   CA   4201 4202  152.75  140.50                  -40.50\n",
      "  225  15701 545M 546Y C   N    4207 4209 -106.95  -98.40   35.46    3.64  -63.50  144.10   20.84\n",
      "  225        546Y 546Y N   CA   4209 4210   93.99  128.40                  -43.40\n",
      "  226  15702 546Y 547L C   N    4219 4221 -152.86 -108.50   44.60    2.18  -63.50 -168.74   23.43\n",
      "  226        547L 547L N   CA   4221 4222  127.90  132.50                  -41.20\n",
      "  227  15703 547L 548F C   N    4227 4229 -108.65 -124.20   39.18    1.40  -63.20  158.31   19.20\n",
      "  227        548F 548F N   CA   4229 4230  107.34  143.30                  -44.30\n",
      "  228  15704 548F 549G C   N    4238 4240   89.70   78.70   53.88    1.10   82.20  132.86    6.83\n",
      "  228        549G 549G N   CA   4240 4241  141.15 -166.10                    8.50\n",
      "  229  15706 550P 551H C   N    4249 4251 -152.35 -125.60   40.47    1.02  -63.20  173.23   27.39\n",
      "  229        551H 551H N   CA   4251 4252  169.17  138.80                  -42.30\n",
      "  230  15707 551H 552T C   N    4259 4261 -115.28 -124.80   12.84    0.43  -63.20 -175.51   20.66\n",
      "  230        552T 552T N   CA   4261 4262  134.89  143.50                  -42.10\n",
      "  231  15708 552T 553E C   N    4266 4268  -83.04  -69.30   16.89    0.96  -63.60  168.50   23.55\n",
      "  231        553E 553E N   CA   4268 4269  152.33  142.50                  -40.30\n",
      "  232  15709 553E 554D C   N    4275 4277 -101.03  -96.50   56.92    2.39  -63.30  104.51   11.22\n",
      "  232        554D 554D N   CA   4277 4278   57.46  114.20                  -40.00\n",
      "  233  15710 554D 555L C   N    4283 4285 -135.15 -108.50   55.91    2.68  -63.50  154.74   25.91\n",
      "  233        555L 555L N   CA   4285 4286 -178.36  132.50                  -41.20\n",
      "  234  15711 555L 556E C   N    4291 4293  -60.21  -69.30   10.02    0.61  -63.60  178.62   24.00\n",
      "  234        556E 556E N   CA   4293 4294  138.29  142.50                  -40.30\n",
      "  235  15713 557R 558F C   N    4311 4313 -178.67 -124.20   59.53    1.67  -63.20 -171.98   33.03\n",
      "  235        558F 558F N   CA   4313 4314  167.31  143.30                  -44.30\n",
      "  236  15714 558F 559F C   N    4322 4324 -132.79 -124.20   14.66    0.44  -63.20  174.96   28.83\n",
      "  236        559F 559F N   CA   4324 4325  155.18  143.30                  -44.30\n",
      "  237  15715 559F 560Q C   N    4333 4335 -131.26 -121.10   41.50    1.78  -63.80  155.19   26.34\n",
      "  237        560Q 560Q N   CA   4335 4336  179.94  139.70                  -40.30\n",
      "  238  15717 561L 562D C   N    4350 4352  -75.24  -70.90   16.42    0.62  -63.30  154.33   19.79\n",
      "  238        562D 562D N   CA   4352 4353  166.13  150.30                  -40.00\n",
      "  239  15719 563V 564L C   N    4365 4367 -135.66 -108.50   36.16    1.62  -63.50  177.73   29.15\n",
      "  239        564L 564L N   CA   4367 4368  156.37  132.50                  -41.20\n",
      "  240  15720 564L 565N C   N    4373 4375 -137.34 -119.90   53.79    2.01  -63.20  150.54   22.96\n",
      "  240        565N 565N N   CA   4375 4376 -172.12  137.00                  -41.10\n",
      "  241  15724 568V 569E C   N    4410 4412   67.17  -69.30  136.52   10.48  -63.60 -142.85   25.22\n",
      "  241        569E 569E N   CA   4412 4413  146.34  142.50                  -40.30\n",
      "  242  15725 569E 570G C   N    4419 4421  178.59 -167.20   15.02    0.22   82.20 -163.73   13.84\n",
      "  242        570G 570G N   CA   4421 4422  179.48  174.60                    8.50\n",
      "  243  15727 571I 572M C   N    4431 4433  -65.02  -73.00   43.23    2.80  -63.40  141.02   20.93\n",
      "  243        572M 572M N   CA   4433 4434  100.51  143.00                  -40.50\n",
      "  244  15728 572M 573S C   N    4439 4441 -125.67 -136.60   12.08    0.37  -64.10 -170.76   18.03\n",
      "  244        573S 573S N   CA   4441 4442  146.05  151.20                  -35.00\n",
      "  245  15729 573S 574D C   N    4445 4447  -98.13  -96.50   29.86    1.25  -63.30  129.17   14.20\n",
      "  245        574D 574D N   CA   4447 4448   84.39  114.20                  -40.00\n",
      "  246  15730 574D 575L C   N    4453 4455  -87.15  -70.70   22.06    1.44  -63.50  164.21   24.22\n",
      "  246        575L 575L N   CA   4455 4456  156.30  141.60                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  serious non-bonded atom clash:  2837 3244   2.271\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    6   80  125  473  408  610  540  637  618  736  818\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00199.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      577     4476     4476\n",
      "openf___224_> Open           EAO00199.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     47436912   46325.109    45.239\n",
      "read_mo_297_> Segments, residues, atoms:        1      577     4476\n",
      "read_mo_298_> Segment:        1  1:A  577:A     4476\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4476     4476        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    21012\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22310\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22981\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    23873\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    24919\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    25728\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    27228\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    28107\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    29246\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    30620\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    31777\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33640\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33781\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.3368        3  3\n",
      "iupac_m_487_> NH1/2 swapped:      177.4153       10  10\n",
      "iupac_m_487_> NH1/2 swapped:      179.1411       16  16\n",
      "iupac_m_487_> NH1/2 swapped:      179.0625       21  21\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.3813       33  33\n",
      "iupac_m_485_> OD1/2 will be swapped:      133.7129       54  54\n",
      "iupac_m_487_> NH1/2 swapped:      179.5941       55  55\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.1278       63  63\n",
      "iupac_m_487_> NH1/2 swapped:      179.2972       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.2113       75  75\n",
      "iupac_m_485_> OD1/2 will be swapped:     -163.2182      101  101\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3903      113  113\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.4557      120  120\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.3260      121  121\n",
      "iupac_m_487_> NH1/2 swapped:      179.8587      149  149\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.5326      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.8343      162  162\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -113.3654      175  175\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.8198      189  189\n",
      "iupac_m_486_> OE1/2 will be swapped:      164.5866      192  192\n",
      "iupac_m_486_> OE1/2 will be swapped:      157.6384      210  210\n",
      "iupac_m_487_> NH1/2 swapped:     -177.0504      214  214\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.2762      231  231\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.5832      243  243\n",
      "iupac_m_487_> NH1/2 swapped:     -178.6898      249  249\n",
      "iupac_m_487_> NH1/2 swapped:      178.1337      275  275\n",
      "iupac_m_485_> OD1/2 will be swapped:      140.4590      282  282\n",
      "iupac_m_487_> NH1/2 swapped:     -176.8523      293  293\n",
      "iupac_m_487_> NH1/2 swapped:     -173.4060      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      144.9829      332  332\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.3573      354  354\n",
      "iupac_m_486_> OE1/2 will be swapped:      131.4936      372  372\n",
      "iupac_m_485_> OD1/2 will be swapped:      103.1660      380  380\n",
      "iupac_m_487_> NH1/2 swapped:      178.4452      387  387\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.1280      389  389\n",
      "iupac_m_485_> OD1/2 will be swapped:     -142.3225      391  391\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.0079      414  414\n",
      "iupac_m_485_> OD1/2 will be swapped:     -117.3131      415  415\n",
      "iupac_m_487_> NH1/2 swapped:      174.5867      416  416\n",
      "iupac_m_487_> NH1/2 swapped:      179.8520      431  431\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.7127      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -150.7384      488  488\n",
      "iupac_m_487_> NH1/2 swapped:      179.7421      499  499\n",
      "iupac_m_487_> NH1/2 swapped:     -175.9834      543  543\n",
      "iupac_m_486_> OE1/2 will be swapped:     -145.1897      553  553\n",
      "iupac_m_486_> OE1/2 will be swapped:     -161.0356      556  556\n",
      "iupac_m_487_> NH1/2 swapped:      179.9802      557  557\n",
      "iupac_m_485_> OD1/2 will be swapped:     -137.2299      562  562\n",
      "iupac_m_487_> NH1/2 swapped:      177.9867      567  567\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.3947      569  569\n",
      "iupac_m_485_> OD1/2 will be swapped:      134.6588      574  574\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   958720\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -39459.0156\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -39459.015625\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    32.907001\n",
      "Sequence length          :          577\n",
      "Compactness              :     0.151827\n",
      "Native energy (pair)     :   753.565747\n",
      "Native energy (surface)  :    67.276895\n",
      "Native energy (combined) :    20.639730\n",
      "Z score (pair)           :    -2.793872\n",
      "Z score (surface)        :    -1.369136\n",
      "Z score (combined)       :    -2.709180\n",
      "GA341 score              :     0.328520\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    11516\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        7071.8784\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4546       9     10   0.011   0.011      164.26       1.000\n",
      " 2 Bond angle potential               :    6129      10     34   2.455   2.455      697.29       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2670       0    140  52.001  52.001      1175.0       1.000\n",
      " 4 Stereochemical improper torsion pot:    1811       1      5   1.633   1.633      99.138       1.000\n",
      " 5 Soft-sphere overlap restraints     :   11516       3      7   0.010   0.010      124.73       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6300      35    155   1.079   1.079      1588.9       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6434      23    126   1.184   1.184      1413.9       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     576       2     18   9.248   9.248      211.11       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     488       0     10  87.785  87.785      202.01       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     356       0      7  78.162  78.162      165.45       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     173       0      0  92.737  92.737      137.13       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      65       0      0  88.634  88.634      46.170       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2777       0      2   0.772   0.772      95.654       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     575     239    126  39.223 113.605      777.04       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     881       1     14   2.046   2.046      174.04       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00199.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00199.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   99866.4688\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2866 366Q 367I C   N    2827 2829    1.49    1.35    0.14    5.05    1.35    0.14    5.05\n",
      "    2   2871 367I 367I N   CA   2829 2830    1.61    1.43    0.18    5.78    1.43    0.18    5.78\n",
      "    3   2873 367I 367I C   CA   2835 2830    1.78    1.49    0.29    8.55    1.49    0.29    8.55\n",
      "    4   2874 367I 368R C   N    2835 2837    1.55    1.35    0.21    7.41    1.35    0.21    7.41\n",
      "    5   2882 368R 368R N   CA   2837 2838    1.66    1.43    0.23    7.65    1.43    0.23    7.65\n",
      "    6   2884 368R 368R C   CA   2846 2838    1.69    1.49    0.20    5.76    1.49    0.20    5.76\n",
      "    7   2885 368R 369L C   N    2846 2848    1.49    1.35    0.15    5.22    1.35    0.15    5.22\n",
      "    8   2890 369L 369L N   CA   2848 2849    1.57    1.43    0.14    4.63    1.43    0.14    4.63\n",
      "    9   3280 420L 420L C   CA   3242 3237    1.67    1.49    0.18    5.23    1.49    0.18    5.23\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8397 366Q 367I C   N    2827 2829  143.64  120.00   23.64    5.37  120.00   23.64    5.37\n",
      "    2   8400 367I 367I N   CA   2829 2830  128.77  107.00   21.77    6.26  107.00   21.77    6.26\n",
      "    3   8404 367I 367I CA  C    2830 2835  132.46  116.50   15.96    4.59  116.50   15.96    4.59\n",
      "    4   8408 367I 368R C   N    2835 2837  158.84  120.00   38.84    8.83  120.00   38.84    8.83\n",
      "    5   8411 368R 368R N   CA   2837 2838  131.46  107.00   24.46    7.03  107.00   24.46    7.03\n",
      "    6   8422 368R 369L C   N    2846 2848  147.08  120.00   27.08    6.15  120.00   27.08    6.15\n",
      "    7   8425 369L 369L N   CA   2848 2849  128.11  107.00   21.11    6.07  107.00   21.11    6.07\n",
      "    8   8951 420L 420L C   CA   3242 3237  130.73  108.00   22.73    5.27  108.00   22.73    5.27\n",
      "    9   8952 420L 421L C   N    3242 3244  149.39  120.00   29.39    6.68  120.00   29.39    6.68\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  18444  38V 495L CA  CA    278 3803   11.29    7.42    3.87    5.50    7.42    3.87    5.50\n",
      "    2  19895  89Q 454I CA  CA    677 3488   17.72   12.45    5.27    8.17   12.45    5.27    8.17\n",
      "    3  19941  90T 454I CA  CA    686 3488   18.21   13.45    4.76    5.51   13.45    4.76    5.51\n",
      "    4  20008  92S 454I CA  CA    697 3488   16.34   12.48    3.86    5.77   12.48    3.86    5.77\n",
      "    5  21371 145M 369L CA  CA   1110 2849   13.14    9.55    3.59    5.31    9.55    3.59    5.31\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24485  46R  43E N   O     330  321    4.96    2.79    2.17    4.73    2.79    2.17    4.73\n",
      "    2  25608  94K 420L N   O     706 3243    8.94    6.90    2.04    4.58    6.90    2.04    4.58\n",
      "    3  25697  97S 420L N   O     732 3243   11.07    8.28    2.79    5.21    8.28    2.79    5.21\n",
      "    4  25729  98L 420L N   O     738 3243   10.54    7.92    2.63    4.84    7.92    2.63    4.84\n",
      "    5  28470 342N 144Y N   O    2622 1108   10.32    6.52    3.80    5.06    6.52    3.80    5.06\n",
      "    6  28567 369L 143V N   O    2848 1096   11.60    7.94    3.65    7.04    7.94    3.65    7.04\n",
      "    7  28568 369L 144Y N   O    2848 1108   15.80   10.51    5.30    7.67   10.51    5.30    7.67\n",
      "    8  28569 369L 145M N   O    2848 1116   16.17    9.65    6.53    8.56    9.65    6.53    8.56\n",
      "    9  28570 369L 146E N   O    2848 1125   16.44   10.73    5.71    6.50   10.73    5.71    6.50\n",
      "   10  28572 369L 283R N   O    2848 2191   12.91    8.61    4.30    5.40    8.61    4.30    5.40\n",
      "   11  29177 413L 379L N   O    3174 2925   11.96    8.08    3.88    5.27    8.08    3.88    5.27\n",
      "   12  29391 421L   8A N   O    3244   51    4.81    2.59    2.22    6.88    2.59    2.22    6.88\n",
      "   13  29559 452A  10R N   O    3474   68   12.93    9.23    3.70    5.17    9.23    3.70    5.17\n",
      "   14  29578 452A  92S N   O    3474  701   11.81    8.91    2.90    5.35    8.91    2.90    5.35\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15888 157R 157R CA  C    1206 1214   -2.05 -179.18  177.13   41.17 -179.18  177.13   41.17\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15157   1M   2G C   N       7    9  -68.89  -80.20   40.84    1.09   82.20 -163.04    6.84\n",
      "    1          2G   2G N   CA      9   10  134.85  174.10                    8.50\n",
      "    2  15158   2G   3E C   N      11   13 -177.03 -117.80   97.15    3.38  -63.60  155.17   26.41\n",
      "    2          3E   3E N   CA     13   14 -146.19  136.80                  -40.30\n",
      "    3  15176  20P  21R C   N     141  143  -74.58 -125.20   53.04    1.75  -63.00  166.26   21.70\n",
      "    3         21R  21R N   CA    143  144  124.76  140.60                  -41.10\n",
      "    4  15177  21R  22Q C   N     152  154  -55.73  -73.00   21.96    1.30  -63.80  167.63   25.07\n",
      "    4         22Q  22Q N   CA    154  155  127.14  140.70                  -40.30\n",
      "    5  15178  22Q  23G C   N     161  163   83.09   78.70   29.15    0.65   82.20  156.58    7.65\n",
      "    5         23G  23G N   CA    163  164  165.08 -166.10                    8.50\n",
      "    6  15179  23G  24E C   N     165  167 -165.76 -117.80   48.05    1.83  -69.30   96.84    7.63\n",
      "    6         24E  24E N   CA    167  168  133.91  136.80                  142.50\n",
      "    7  15186  30A  31D C   N     220  222   70.54   54.50   31.95    1.57  -63.30  144.05   24.50\n",
      "    7         31D  31D N   CA    222  223   13.27   40.90                  -40.00\n",
      "    8  15189  33D  34D C   N     242  244 -132.46  -63.30   90.42   10.59  -63.30   90.42   10.59\n",
      "    8         34D  34D N   CA    244  245   18.25  -40.00                  -40.00\n",
      "    9  15199  43E  44G C   N     320  322   75.74   82.20   43.08    2.41  -62.40  138.32   24.99\n",
      "    9         44G  44G N   CA    322  323  -34.09    8.50                  -41.20\n",
      "   10  15202  46R  47A C   N     339  341 -175.12 -134.00   65.97    2.26  -68.20  119.46    8.24\n",
      "   10         47A  47A N   CA    341  342 -161.41  147.00                  145.30\n",
      "   11  15213  57F  58T C   N     433  435 -143.80  -63.20   88.66   11.45 -124.80  149.88    6.99\n",
      "   11         58T  58T N   CA    435  436   -5.18  -42.10                  143.50\n",
      "   12  15259 103N 104S C   N     783  785 -170.34 -136.60   41.69    1.28  -64.10 -176.75   20.95\n",
      "   12        104S 104S N   CA    785  786  175.69  151.20                  -35.00\n",
      "   13  15260 104S 105K C   N     789  791  -68.36  -70.20    7.88    0.53   56.60  156.45   11.58\n",
      "   13        105K 105K N   CA    791  792  132.73  140.40                   38.60\n",
      "   14  15261 105K 106E C   N     798  800   50.93  -69.30  127.96   11.06  -63.60  175.84   20.62\n",
      "   14        106E 106E N   CA    800  801 -173.72  142.50                  -40.30\n",
      "   15  15262 106E 107G C   N     807  809   98.16   82.20   24.97    0.72  -62.40  163.43   30.23\n",
      "   15        107G 107G N   CA    809  810  -10.70    8.50                  -41.20\n",
      "   16  15275 119I 120E C   N     906  908  -58.87  -69.30   36.84    3.14  -63.60  141.94   18.67\n",
      "   16        120E 120E N   CA    908  909  177.83  142.50                  -40.30\n",
      "   17  15311 155R 156A C   N    1198 1200 -111.94 -134.00   40.03    1.51  -62.50  162.20   24.68\n",
      "   17        156A 156A N   CA   1200 1201  113.59  147.00                  -40.90\n",
      "   18  15315 159Q 160E C   N    1227 1229  -48.96  -69.30   23.42    2.10  -63.60  166.24   21.39\n",
      "   18        160E 160E N   CA   1229 1230  154.11  142.50                  -40.30\n",
      "   19  15317 161P 162E C   N    1243 1245   34.33  -69.30  115.18    6.97  -63.60  164.79   27.24\n",
      "   19        162E 162E N   CA   1245 1246   92.24  142.50                  -40.30\n",
      "   20  15318 162E 163S C   N    1252 1254 -131.79 -136.60   12.41    0.70  -64.10  175.91   17.55\n",
      "   20        163S 163S N   CA   1254 1255  162.63  151.20                  -35.00\n",
      "   21  15324 168E 169D C   N    1302 1304 -125.47  -96.50   40.34    1.65  -70.90   84.24    6.65\n",
      "   21        169D 169D N   CA   1304 1305   86.12  114.20                  150.30\n",
      "   22  15325 169D 170S C   N    1310 1312 -159.48 -136.60   23.61    0.80  -64.10 -166.81   20.86\n",
      "   22        170S 170S N   CA   1312 1313  157.00  151.20                  -35.00\n",
      "   23  15326 170S 171Q C   N    1316 1318  -72.70  -73.00    5.28    0.35  -63.80  173.95   26.05\n",
      "   23        171Q 171Q N   CA   1318 1319  145.98  140.70                  -40.30\n",
      "   24  15327 171Q 172H C   N    1325 1327  -71.44  -67.60    5.20    0.47  -63.20  178.99   21.88\n",
      "   24        172H 172H N   CA   1327 1328  136.50  140.00                  -42.30\n",
      "   25  15328 172H 173R C   N    1335 1337 -112.59 -125.20   12.72    0.47  -63.00 -173.35   28.64\n",
      "   25        173R 173R N   CA   1337 1338  138.96  140.60                  -41.10\n",
      "   26  15330 174V 175Y C   N    1353 1355 -103.40  -98.40   16.63    1.68  -63.50  160.96   23.63\n",
      "   26        175Y 175Y N   CA   1355 1356  112.53  128.40                  -43.40\n",
      "   27  15332 176V 177K C   N    1372 1374  -68.32  -70.20   13.75    1.04  -62.90  165.27   21.76\n",
      "   27        177K 177K N   CA   1374 1375  154.01  140.40                  -40.80\n",
      "   28  15333 177K 178G C   N    1381 1383   86.32   78.70   40.00    0.83   82.20  146.19    7.30\n",
      "   28        178G 178G N   CA   1383 1384  154.63 -166.10                    8.50\n",
      "   29  15334 178G 179A C   N    1385 1387 -148.52 -134.00   24.85    0.90  -62.50  174.61   33.21\n",
      "   29        179A 179A N   CA   1387 1388  167.16  147.00                  -40.90\n",
      "   30  15335 179A 180G C   N    1390 1392 -149.90 -167.20   31.42    0.87   82.20 -170.47   14.40\n",
      "   30        180G 180G N   CA   1392 1393  148.38  174.60                    8.50\n",
      "   31  15337 181V 182H C   N    1401 1403 -156.21 -125.60   33.11    0.90  -63.20 -169.46   29.83\n",
      "   31        182H 182H N   CA   1403 1404  151.40  138.80                  -42.30\n",
      "   32  15338 182H 183S C   N    1411 1413  -81.43  -72.40   18.25    0.84  -64.10  157.70   12.54\n",
      "   32        183S 183S N   CA   1413 1414  168.26  152.40                  -35.00\n",
      "   33  15341 185L 186S C   N    1432 1434 -108.08 -136.60   28.72    1.23  -64.10  175.99   15.77\n",
      "   33        186S 186S N   CA   1434 1435  154.59  151.20                  -35.00\n",
      "   34  15343 187L 188E C   N    1446 1448  177.26 -117.80   93.51    3.03  -63.60  166.02   28.19\n",
      "   34        188E 188E N   CA   1448 1449 -155.92  136.80                  -40.30\n",
      "   35  15352 196K 197G C   N    1525 1527   53.52   78.70   27.23    1.04   82.20  166.72    9.51\n",
      "   35        197G 197G N   CA   1527 1528 -155.73 -166.10                    8.50\n",
      "   36  15353 197G 198N C   N    1529 1531  -96.69 -119.90   23.87    0.85  -63.20  175.74   20.30\n",
      "   36        198N 198N N   CA   1531 1532  131.42  137.00                  -41.10\n",
      "   37  15354 198N 199A C   N    1537 1539 -134.95 -134.00    6.26    0.33  -62.50 -178.95   33.55\n",
      "   37        199A 199A N   CA   1539 1540  153.19  147.00                  -40.90\n",
      "   38  15355 199A 200N C   N    1542 1544  -92.78  -71.20   33.21    2.65  -63.20  161.38   18.69\n",
      "   38        200N 200N N   CA   1544 1545  117.55  142.80                  -41.10\n",
      "   39  15356 200N 201R C   N    1550 1552 -134.72 -125.20   34.05    1.41  -63.00  162.31   26.81\n",
      "   39        201R 201R N   CA   1552 1553  173.29  140.60                  -41.10\n",
      "   40  15357 201R 202Q C   N    1561 1563  -63.48  -73.00   14.22    1.10  -63.80  168.43   24.75\n",
      "   40        202Q 202Q N   CA   1563 1564  151.27  140.70                  -40.30\n",
      "   41  15358 202Q 203T C   N    1570 1572  -73.69  -78.10   46.05    2.12  -63.20  122.72   16.43\n",
      "   41        203T 203T N   CA   1572 1573 -164.37  149.80                  -42.10\n",
      "   42  15360 204G 205E C   N    1581 1583  -84.32  -69.30   46.10    3.98  -63.60  140.75   17.71\n",
      "   42        205E 205E N   CA   1583 1584   98.92  142.50                  -40.30\n",
      "   43  15361 205E 206T C   N    1590 1592 -133.08 -124.80   11.06    0.67  -63.20 -168.53   20.81\n",
      "   43        206T 206T N   CA   1592 1593  136.16  143.50                  -42.10\n",
      "   44  15362 206T 207K C   N    1597 1599 -142.95 -118.00   34.95    1.23  -62.90  175.00   27.51\n",
      "   44        207K 207K N   CA   1599 1600  163.58  139.10                  -40.80\n",
      "   45  15363 207K 208M C   N    1606 1608  -96.18  -73.00   49.59    3.76  -63.40  143.46   19.12\n",
      "   45        208M 208M N   CA   1608 1609   99.16  143.00                  -40.50\n",
      "   46  15364 208M 209N C   N    1614 1616 -161.49 -119.90   51.37    1.50  -63.20 -179.20   28.02\n",
      "   46        209N 209N N   CA   1616 1617  167.15  137.00                  -41.10\n",
      "   47  15365 209N 210E C   N    1622 1624 -127.35 -117.80   16.20    0.57  -63.60 -178.62   27.78\n",
      "   47        210E 210E N   CA   1624 1625  149.89  136.80                  -40.30\n",
      "   48  15366 210E 211T C   N    1631 1633 -126.91 -124.80    2.41    0.09  -63.20 -175.43   27.88\n",
      "   48        211T 211T N   CA   1633 1634  144.67  143.50                  -42.10\n",
      "   49  15367 211T 212S C   N    1638 1640  -96.68  -72.40   41.62    1.90  -64.10  142.57   12.55\n",
      "   49        212S 212S N   CA   1640 1641 -173.80  152.40                  -35.00\n",
      "   50  15369 213S 214R C   N    1650 1652 -107.24 -125.20   22.88    1.23  -63.00  169.99   26.02\n",
      "   50        214R 214R N   CA   1652 1653  154.77  140.60                  -41.10\n",
      "   51  15370 214R 215S C   N    1661 1663 -154.69 -136.60   40.31    1.58  -64.10  164.90   18.49\n",
      "   51        215S 215S N   CA   1663 1664 -172.79  151.20                  -35.00\n",
      "   52  15371 215S 216H C   N    1667 1669 -158.34 -125.60   33.93    1.36  -63.20 -163.26   20.91\n",
      "   52        216H 216H N   CA   1669 1670  129.91  138.80                  -42.30\n",
      "   53  15372 216H 217A C   N    1677 1679 -169.81 -134.00   46.49    1.26  -62.50  178.35   34.86\n",
      "   53        217A 217A N   CA   1679 1680  176.65  147.00                  -40.90\n",
      "   54  15374 218I 219V C   N    1690 1692   53.04  -73.50  144.88   11.70 -125.40 -169.59    9.40\n",
      "   54        219V 219V N   CA   1692 1693 -150.25  139.20                  143.30\n",
      "   55  15379 223I 224S C   N    1729 1731  -28.59  -72.40   50.62    2.74  -64.10  165.89   14.42\n",
      "   55        224S 224S N   CA   1731 1732  127.04  152.40                  -35.00\n",
      "   56  15380 224S 225Q C   N    1735 1737  -63.75  -73.00    9.57    0.63  -63.80  178.55   26.26\n",
      "   56        225Q 225Q N   CA   1737 1738  138.25  140.70                  -40.30\n",
      "   57  15383 227Y 228G C   N    1765 1767  107.28   82.20   25.52    1.52  -62.40  175.53   32.77\n",
      "   57        228G 228G N   CA   1767 1768    3.73    8.50                  -41.20\n",
      "   58  15385 229S 230L C   N    1775 1777  -79.31  -70.70   45.08    3.10  -63.50  133.89   19.53\n",
      "   58        230L 230L N   CA   1777 1778 -174.15  141.60                  -41.20\n",
      "   59  15387 231D 232M C   N    1791 1793 -137.71 -125.60   33.97    1.81  -63.40  166.74   20.71\n",
      "   59        232M 232M N   CA   1793 1794  108.77  140.50                  -40.50\n",
      "   60  15388 232M 233R C   N    1799 1801 -149.74 -125.20   24.92    1.11  -63.00 -162.55   22.48\n",
      "   60        233R 233R N   CA   1801 1802  136.28  140.60                  -41.10\n",
      "   61  15392 236E 237S C   N    1834 1836  108.51 -136.60  115.18    4.36  -64.10 -120.70   19.71\n",
      "   61        237S 237S N   CA   1836 1837  159.25  151.20                  -35.00\n",
      "   62  15396 240L 241L C   N    1862 1864  -94.76 -108.50   20.26    1.14  -63.50  174.23   26.09\n",
      "   62        241L 241L N   CA   1864 1865  147.39  132.50                  -41.20\n",
      "   63  15409 253T 254E C   N    1959 1961   36.37  -69.30  116.42   10.28  -63.60  162.67   18.93\n",
      "   63        254E 254E N   CA   1961 1962 -168.63  142.50                  -40.30\n",
      "   64  15410 254E 255S C   N    1968 1970 -148.13 -136.60   16.29    0.53  -64.10 -177.25   19.27\n",
      "   64        255S 255S N   CA   1970 1971  162.71  151.20                  -35.00\n",
      "   65  15411 255S 256T C   N    1974 1976 -156.46 -124.80   61.68    1.92  -63.20  153.14   25.39\n",
      "   65        256T 256T N   CA   1976 1977 -163.57  143.50                  -42.10\n",
      "   66  15412 256T 257G C   N    1981 1983 -146.70 -167.20   22.87    0.71   82.20 -140.35   16.24\n",
      "   66        257G 257G N   CA   1983 1984 -175.26  174.60                    8.50\n",
      "   67  15414 258V 259A C   N    1992 1994 -154.76 -134.00   23.29    0.53  -62.50 -173.96   35.41\n",
      "   67        259A 259A N   CA   1994 1995  157.55  147.00                  -40.90\n",
      "   68  15415 259A 260F C   N    1997 1999  128.15 -124.20  111.04    3.55  -63.20 -137.48   40.04\n",
      "   68        260F 260F N   CA   1999 2000  170.54  143.30                  -44.30\n",
      "   69  15416 260F 261E C   N    2008 2010 -112.24 -117.80   22.90    1.03  -63.60  162.34   19.41\n",
      "   69        261E 261E N   CA   2010 2011  114.58  136.80                  -40.30\n",
      "   70  15417 261E 262E C   N    2017 2019 -168.85 -117.80   59.78    2.97  -63.60  179.98   20.86\n",
      "   70        262E 262E N   CA   2019 2020  105.69  136.80                  -40.30\n",
      "   71  15418 262E 263A C   N    2026 2028 -155.73 -134.00   23.55    0.54  -62.50 -172.19   35.75\n",
      "   71        263A 263A N   CA   2028 2029  156.07  147.00                  -40.90\n",
      "   72  15419 263A 264K C   N    2031 2033 -119.67 -118.00   16.62    0.78  -62.90  173.13   25.87\n",
      "   72        264K 264K N   CA   2033 2034  155.64  139.10                  -40.80\n",
      "   73  15420 264K 265K C   N    2040 2042  -67.20  -70.20    3.47    0.29  -62.90  177.11   23.24\n",
      "   73        265K 265K N   CA   2042 2043  142.14  140.40                  -40.80\n",
      "   74  15422 266I 267N C   N    2057 2059  -66.76  -71.20   15.56    0.88  -63.20  169.02   20.94\n",
      "   74        267N 267N N   CA   2059 2060  127.89  142.80                  -41.10\n",
      "   75  15423 267N 268Q C   N    2065 2067   68.85  -73.00  142.05   10.01  -63.80 -141.61   38.65\n",
      "   75        268Q 268Q N   CA   2067 2068  133.19  140.70                  -40.30\n",
      "   76  15424 268Q 269S C   N    2074 2076 -111.74 -136.60   56.72    2.25  -64.10  143.36    8.83\n",
      "   76        269S 269S N   CA   2076 2077  100.21  151.20                  -35.00\n",
      "   77  15426 270L 271L C   N    2088 2090 -102.76 -108.50   24.27    1.36  -63.50  167.39   25.65\n",
      "   77        271L 271L N   CA   2090 2091  156.08  132.50                  -41.20\n",
      "   78  15427 271L 272M C   N    2096 2098  -74.68  -73.00   15.13    1.00  -63.40  161.85   25.05\n",
      "   78        272M 272M N   CA   2098 2099  158.04  143.00                  -40.50\n",
      "   79  15428 272M 273L C   N    2104 2106  -76.30  -70.70    9.88    0.61  -63.50  169.55   24.30\n",
      "   79        273L 273L N   CA   2106 2107  149.73  141.60                  -41.20\n",
      "   80  15430 274G 275R C   N    2116 2118  -64.58  -72.10   10.55    0.68  -63.00  175.61   23.62\n",
      "   80        275R 275R N   CA   2118 2119  134.51  141.90                  -41.10\n",
      "   81  15431 275R 276A C   N    2127 2129  -68.18  -68.20   11.31    0.91  -62.50  174.99   28.40\n",
      "   81        276A 276A N   CA   2129 2130  134.00  145.30                  -40.90\n",
      "   82  15432 276A 277M C   N    2132 2134  -67.72  -63.40   11.04    1.39 -125.60 -179.62    7.25\n",
      "   82        277M 277M N   CA   2134 2135  -30.35  -40.50                  140.50\n",
      "   83  15433 277M 278N C   N    2140 2142  -54.57  -63.20    9.86    1.62 -119.90 -174.77    6.77\n",
      "   83        278N 278N N   CA   2142 2143  -36.33  -41.10                  137.00\n",
      "   84  15434 278N 279S C   N    2148 2150  -86.81  -72.40   41.52    2.93  -64.10  150.19    9.76\n",
      "   84        279S 279S N   CA   2150 2151  113.46  152.40                  -35.00\n",
      "   85  15454 298T 299R C   N    2299 2301 -150.41 -125.20   26.15    1.23  -63.00 -164.63   22.21\n",
      "   85        299R 299R N   CA   2301 2302  133.63  140.60                  -41.10\n",
      "   86  15455 299R 300L C   N    2310 2312 -128.00 -108.50   34.96    1.99  -63.50  158.41   19.47\n",
      "   86        300L 300L N   CA   2312 2313  103.48  132.50                  -41.20\n",
      "   87  15456 300L 301L C   N    2318 2320 -116.24 -108.50   30.77    1.73  -63.50  153.28   19.02\n",
      "   87        301L 301L N   CA   2320 2321  102.72  132.50                  -41.20\n",
      "   88  15457 301L 302S C   N    2326 2328 -167.63 -136.60   33.97    1.06  -64.10 -169.45   21.28\n",
      "   88        302S 302S N   CA   2328 2329  165.02  151.20                  -35.00\n",
      "   89  15458 302S 303E C   N    2332 2334 -111.26 -117.80   18.33    0.77  -63.60  166.93   20.04\n",
      "   89        303E 303E N   CA   2334 2335  119.68  136.80                  -40.30\n",
      "   90  15459 303E 304S C   N    2341 2343 -142.64 -136.60   13.43    0.53  -64.10  179.86   18.65\n",
      "   90        304S 304S N   CA   2343 2344  163.20  151.20                  -35.00\n",
      "   91  15460 304S 305F C   N    2347 2349 -116.54 -124.20    8.57    0.23  -63.20 -175.86   29.13\n",
      "   91        305F 305F N   CA   2349 2350  139.45  143.30                  -44.30\n",
      "   92  15461 305F 306G C   N    2358 2360   89.02   78.70   41.60    0.78   82.20  145.26    7.40\n",
      "   92        306G 306G N   CA   2360 2361  153.60 -166.10                    8.50\n",
      "   93  15462 306G 307G C   N    2362 2364   88.84   78.70   48.11    0.97   82.20  138.53    7.06\n",
      "   93        307G 307G N   CA   2364 2365  146.87 -166.10                    8.50\n",
      "   94  15463 307G 308N C   N    2366 2368 -124.37 -119.90    5.61    0.30  -63.20 -174.88   20.43\n",
      "   94        308N 308N N   CA   2368 2369  133.62  137.00                  -41.10\n",
      "   95  15464 308N 309S C   N    2374 2376 -134.76 -136.60    9.43    0.51  -64.10  179.08   18.00\n",
      "   95        309S 309S N   CA   2376 2377  160.45  151.20                  -35.00\n",
      "   96  15465 309S 310K C   N    2380 2382 -121.07 -118.00   71.27    3.42  -62.90  123.45   19.51\n",
      "   96        310K 310K N   CA   2382 2383 -149.69  139.10                  -40.80\n",
      "   97  15467 311T 312W C   N    2396 2398  -78.11  -71.30   11.23    0.67  -63.00  168.56   22.02\n",
      "   97        312W 312W N   CA   2398 2399  147.92  139.00                  -44.20\n",
      "   98  15468 312W 313M C   N    2410 2412 -116.36 -125.60   22.48    1.21  -63.40  167.11   29.17\n",
      "   98        313M 313M N   CA   2412 2413  161.00  140.50                  -40.50\n",
      "   99  15469 313M 314L C   N    2418 2420  -95.52 -108.50   13.61    0.61  -63.50  172.58   22.33\n",
      "   99        314L 314L N   CA   2420 2421  128.39  132.50                  -41.20\n",
      "  100  15470 314L 315A C   N    2426 2428 -164.90 -134.00   35.58    0.83  -62.50 -174.68   35.80\n",
      "  100        315A 315A N   CA   2428 2429  164.65  147.00                  -40.90\n",
      "  101  15471 315A 316T C   N    2431 2433  -89.95  -78.10   88.74    4.19  -63.20  107.35   12.19\n",
      "  101        316T 316T N   CA   2433 2434   61.86  149.80                  -42.10\n",
      "  102  15473 317V 318S C   N    2445 2447  -72.93  -72.40    6.97    0.39  -64.10  165.89   12.56\n",
      "  102        318S 318S N   CA   2447 2448  159.35  152.40                  -35.00\n",
      "  103  15474 318S 319T C   N    2451 2453 -117.02 -124.80    8.93    0.54  -63.20  178.34   26.45\n",
      "  103        319T 319T N   CA   2453 2454  147.87  143.50                  -42.10\n",
      "  104  15477 321A 322N C   N    2468 2470 -142.92  -63.20   83.61   13.54   55.90 -167.19   14.34\n",
      "  104        322N 322N N   CA   2470 2471  -66.30  -41.10                   39.50\n",
      "  105  15495 339A 340I C   N    2605 2607  -55.58  -63.40    7.96    1.47 -120.60 -175.74    8.22\n",
      "  105        340I 340I N   CA   2607 2608  -42.11  -43.60                  130.30\n",
      "  106  15497 341T 342N C   N    2620 2622  -87.50 -119.90   34.66    1.72  -63.20  171.30   22.79\n",
      "  106        342N 342N N   CA   2622 2623  149.33  137.00                  -41.10\n",
      "  107  15500 344A 345K C   N    2642 2644   72.77   56.60   21.19    1.48  -62.90  150.74   26.33\n",
      "  107        345K 345K N   CA   2644 2645   24.89   38.60                  -40.80\n",
      "  108  15504 348K 349T C   N    2675 2677  -44.94 -124.80   89.49    3.55  -63.20  146.35   19.98\n",
      "  108        349T 349T N   CA   2677 2678  103.10  143.50                  -42.10\n",
      "  109  15505 349T 350A C   N    2682 2684  -84.99  -68.20   30.34    2.94  -62.50  162.49   25.62\n",
      "  109        350A 350A N   CA   2684 2685  120.03  145.30                  -40.90\n",
      "  110  15506 350A 351K C   N    2687 2689 -153.34 -118.00   36.92    1.59  -62.90 -168.13   22.04\n",
      "  110        351K 351K N   CA   2689 2690  128.41  139.10                  -40.80\n",
      "  111  15507 351K 352D C   N    2696 2698 -157.98  -96.50   73.94    3.07  -63.30 -169.99   29.52\n",
      "  111        352D 352D N   CA   2698 2699  155.27  114.20                  -40.00\n",
      "  112  15508 352D 353L C   N    2704 2706 -123.25 -108.50   23.84    1.10  -63.50  177.90   28.40\n",
      "  112        353L 353L N   CA   2706 2707  151.23  132.50                  -41.20\n",
      "  113  15509 353L 354E C   N    2712 2714 -158.16 -117.80   40.50    1.40  -63.60 -157.11   32.31\n",
      "  113        354E 354E N   CA   2714 2715  140.19  136.80                  -40.30\n",
      "  114  15510 354E 355Y C   N    2721 2723  156.90 -124.30   79.17    5.70  -63.50 -137.27   41.61\n",
      "  114        355Y 355Y N   CA   2723 2724  143.05  135.40                  -43.40\n",
      "  115  15511 355Y 356D C   N    2733 2735 -111.48  -63.30   77.82   12.62  -63.30   77.82   12.62\n",
      "  115        356D 356D N   CA   2735 2736 -101.11  -40.00                  -40.00\n",
      "  116  15512 356D 357E C   N    2741 2743 -149.92 -117.80   61.15    2.28  -63.60  156.77   25.61\n",
      "  116        357E 357E N   CA   2743 2744 -171.17  136.80                  -40.30\n",
      "  117  15514 358I 359Q C   N    2758 2760 -145.89 -121.10   33.98    1.09  -63.80  176.95   30.27\n",
      "  117        359Q 359Q N   CA   2760 2761  162.94  139.70                  -40.30\n",
      "  118  15516 360K 361L C   N    2776 2778  -80.85  -70.70   24.77    1.56  -63.50  155.57   22.64\n",
      "  118        361L 361L N   CA   2778 2779  164.19  141.60                  -41.20\n",
      "  119  15517 361L 362A C   N    2784 2786  -69.69  -68.20   16.96    1.31  -62.50  157.07   26.10\n",
      "  119        362A 362A N   CA   2786 2787  162.19  145.30                  -40.90\n",
      "  120  15518 362A 363W C   N    2789 2791 -124.16 -124.90   18.36    0.91  -63.00  165.75   24.83\n",
      "  120        363W 363W N   CA   2791 2792  161.75  143.40                  -44.20\n",
      "  121  15520 364V 365L C   N    2810 2812 -162.63 -108.50   91.79    4.26  -63.50  149.70   26.67\n",
      "  121        365L 365L N   CA   2812 2813 -153.37  132.50                  -41.20\n",
      "  122  15521 365L 366Q C   N    2818 2820   85.48 -121.10  153.48    5.68  -63.80 -129.40   29.73\n",
      "  122        366Q 366Q N   CA   2820 2821  143.93  139.70                  -40.30\n",
      "  123  15522 366Q 367I C   N    2827 2829  -76.25  -63.40   23.29    3.54 -120.60  160.72    7.55\n",
      "  123        367I 367I N   CA   2829 2830  -24.18  -43.60                  130.30\n",
      "  124  15523 367I 368R C   N    2835 2837  -59.31  -72.10   16.04    1.39  -63.00  167.36   22.36\n",
      "  124        368R 368R N   CA   2837 2838  151.58  141.90                  -41.10\n",
      "  125  15525 369L 370D C   N    2854 2856   -2.95  -96.50  108.23    4.49  -63.30  116.60   18.25\n",
      "  125        370D 370D N   CA   2856 2857   59.77  114.20                  -40.00\n",
      "  126  15535 379L 380D C   N    2924 2926   64.97   54.50   25.62    1.26  -63.30  140.57   23.97\n",
      "  126        380D 380D N   CA   2926 2927   17.51   40.90                  -40.00\n",
      "  127  15539 383I 384Q C   N    2954 2956 -112.99 -121.10   20.94    1.09  -63.80  168.05   27.33\n",
      "  127        384Q 384Q N   CA   2956 2957  159.01  139.70                  -40.30\n",
      "  128  15540 384Q 385G C   N    2963 2965   88.30   82.20    9.81    0.73  -80.20 -129.07    7.70\n",
      "  128        385G 385G N   CA   2965 2966   16.19    8.50                  174.10\n",
      "  129  15542 386L 387R C   N    2975 2977 -113.15 -125.20   96.41    4.90  -63.00   96.67   16.41\n",
      "  129        387R 387R N   CA   2977 2978 -123.75  140.60                  -41.10\n",
      "  130  15543 387R 388G C   N    2986 2988  168.23 -167.20   46.75    1.88   82.20  152.83   11.15\n",
      "  130        388G 388G N   CA   2988 2989  134.82  174.60                    8.50\n",
      "  131  15544 388G 389E C   N    2990 2992 -156.90 -117.80   40.50    1.26  -63.60 -164.04   31.31\n",
      "  131        389E 389E N   CA   2992 2993  147.37  136.80                  -40.30\n",
      "  132  15546 390I 391D C   N    3007 3009  -83.86  -96.50   26.06    1.10  -63.30  133.01   15.28\n",
      "  132        391D 391D N   CA   3009 3010   91.41  114.20                  -40.00\n",
      "  133  15548 392C 393M C   N    3021 3023  -97.93 -125.60   73.69    2.85  -63.40  117.87   15.29\n",
      "  133        393M 393M N   CA   3023 3024   72.20  140.50                  -40.50\n",
      "  134  15549 393M 394K C   N    3029 3031 -146.59 -118.00   37.92    1.88  -62.90  176.15   20.24\n",
      "  134        394K 394K N   CA   3031 3032  114.19  139.10                  -40.80\n",
      "  135  15552 396K 397L C   N    3053 3055  -71.48  -70.70   16.85    1.31  -63.50  166.16   22.63\n",
      "  135        397L 397L N   CA   3055 3056  124.77  141.60                  -41.20\n",
      "  136  15553 397L 398T C   N    3061 3063 -141.55 -124.80   22.82    0.75  -63.20  177.18   27.82\n",
      "  136        398T 398T N   CA   3063 3064  158.99  143.50                  -42.10\n",
      "  137  15555 399V 400A C   N    3075 3077  -71.98  -68.20    3.80    0.31  -62.50  173.66   28.93\n",
      "  137        400A 400A N   CA   3077 3078  145.70  145.30                  -40.90\n",
      "  138  15556 400A 401S C   N    3080 3082 -170.26 -136.60   59.64    2.15  -64.10  163.66   19.50\n",
      "  138        401S 401S N   CA   3082 3083 -159.56  151.20                  -35.00\n",
      "  139  15557 401S 402D C   N    3086 3088   76.54  -96.50  175.96    7.16  -63.30 -136.86   24.31\n",
      "  139        402D 402D N   CA   3088 3089  146.12  114.20                  -40.00\n",
      "  140  15568 412A 413L C   N    3172 3174  -71.04  -63.50   20.68    3.34 -108.50  171.19    9.58\n",
      "  140        413L 413L N   CA   3174 3175  -60.46  -41.20                  132.50\n",
      "  141  15580 424R 425K C   N    3275 3277 -152.49 -118.00   37.38    1.68  -62.90 -171.81   21.62\n",
      "  141        425K 425K N   CA   3277 3278  124.70  139.10                  -40.80\n",
      "  142  15581 425K 426G C   N    3284 3286  178.73 -167.20   18.28    0.37   82.20 -157.71   14.16\n",
      "  142        426G 426G N   CA   3286 3287 -173.72  174.60                    8.50\n",
      "  143  15582 426G 427L C   N    3288 3290 -109.69 -108.50   16.12    0.86  -63.50  176.37   27.33\n",
      "  143        427L 427L N   CA   3290 3291  148.58  132.50                  -41.20\n",
      "  144  15584 428I 429E C   N    3304 3306 -123.11 -117.80    6.30    0.19  -63.60 -170.88   28.58\n",
      "  144        429E 429E N   CA   3306 3307  140.19  136.80                  -40.30\n",
      "  145  15585 429E 430Q C   N    3313 3315 -102.44 -121.10   26.95    1.38  -63.80  165.13   26.35\n",
      "  145        430Q 430Q N   CA   3315 3316  159.15  139.70                  -40.30\n",
      "  146  15586 430Q 431R C   N    3322 3324  -71.26  -72.10   43.52    3.33  -63.00  139.73   18.33\n",
      "  146        431R 431R N   CA   3324 3325   98.38  141.90                  -41.10\n",
      "  147  15587 431R 432K C   N    3333 3335  171.93 -118.00   76.58    2.54  -62.90 -165.25   32.57\n",
      "  147        432K 432K N   CA   3335 3336  170.00  139.10                  -40.80\n",
      "  148  15589 433I 434S C   N    3350 3352 -165.67 -136.60   33.98    1.04  -64.10 -173.67   20.83\n",
      "  148        434S 434S N   CA   3352 3353  168.79  151.20                  -35.00\n",
      "  149  15590 434S 435G C   N    3356 3358   96.23   78.70   42.46    0.64   82.20  147.39    7.86\n",
      "  149        435G 435G N   CA   3358 3359  155.23 -166.10                    8.50\n",
      "  150  15591 435G 436F C   N    3360 3362 -132.25 -124.20   18.62    0.64  -63.20  170.24   28.13\n",
      "  150        436F 436F N   CA   3362 3363  160.09  143.30                  -44.30\n",
      "  151  15592 436F 437S C   N    3371 3373 -130.86 -136.60   16.24    0.91  -64.10  172.09   17.21\n",
      "  151        437S 437S N   CA   3373 3374  166.38  151.20                  -35.00\n",
      "  152  15593 437S 438A C   N    3377 3379  -69.72  -68.20    4.36    0.30  -62.50  169.87   28.20\n",
      "  152        438A 438A N   CA   3379 3380  149.38  145.30                  -40.90\n",
      "  153  15594 438A 439A C   N    3382 3384 -114.33 -134.00   19.96    0.79  -62.50  176.51   31.68\n",
      "  153        439A 439A N   CA   3384 3385  150.38  147.00                  -40.90\n",
      "  154  15595 439A 440L C   N    3387 3389  -72.15  -70.70    9.86    0.69  -63.50  167.67   23.79\n",
      "  154        440L 440L N   CA   3389 3390  151.35  141.60                  -41.20\n",
      "  155  15596 440L 441V C   N    3395 3397  -61.72  -62.40    0.72    0.10 -125.40 -174.66   10.40\n",
      "  155        441V 441V N   CA   3397 3398  -42.64  -42.40                  143.30\n",
      "  156  15598 442C 443F C   N    3408 3410  -66.92  -71.40    4.55    0.33  -63.20  174.23   24.44\n",
      "  156        443F 443F N   CA   3410 3411  141.51  140.70                  -44.30\n",
      "  157  15599 443F 444S C   N    3419 3421 -147.60 -136.60   24.57    0.97  -64.10  173.27   18.56\n",
      "  157        444S 444S N   CA   3421 3422  173.17  151.20                  -35.00\n",
      "  158  15600 444S 445G C   N    3425 3427  -75.15  -80.20    9.18    0.23   82.20 -137.06    6.96\n",
      "  158        445G 445G N   CA   3427 3428  166.43  174.10                    8.50\n",
      "  159  15603 447C 448K C   N    3442 3444 -125.81 -118.00   39.63    1.80  -62.90  154.62   23.84\n",
      "  159        448K 448K N   CA   3444 3445  177.96  139.10                  -40.80\n",
      "  160  15604 448K 449T C   N    3451 3453  -75.30  -78.10    8.82    0.30  -63.20  176.88   23.48\n",
      "  160        449T 449T N   CA   3453 3454  141.43  149.80                  -42.10\n",
      "  161  15605 449T 450S C   N    3458 3460 -127.64 -136.60   13.60    0.77  -64.10  175.48   17.21\n",
      "  161        450S 450S N   CA   3460 3461  161.43  151.20                  -35.00\n",
      "  162  15606 450S 451L C   N    3464 3466 -146.11 -108.50   62.63    2.90  -63.50  159.31   27.19\n",
      "  162        451L 451L N   CA   3466 3467 -177.42  132.50                  -41.20\n",
      "  163  15607 451L 452A C   N    3472 3474  -56.08  -62.50    6.42    1.22  -68.20  173.98   14.46\n",
      "  163        452A 452A N   CA   3474 3475  -41.14  -40.90                  145.30\n",
      "  164  15608 452A 453N C   N    3477 3479 -143.45 -119.90   29.59    1.58  -63.20  179.15   19.32\n",
      "  164        453N 453N N   CA   3479 3480  119.08  137.00                  -41.10\n",
      "  165  15611 455P 456F C   N    3500 3502  -87.70  -71.40   20.52    1.16  -63.20  164.36   24.48\n",
      "  165        456F 456F N   CA   3502 3503  153.18  140.70                  -44.30\n",
      "  166  15613 457G 458S C   N    3515 3517   65.59  -64.10  137.25   18.36  -64.10  137.25   18.36\n",
      "  166        458S 458S N   CA   3517 3518    9.91  -35.00                  -35.00\n",
      "  167  15618 462T 463T C   N    3556 3558 -130.71 -124.80   30.77    1.49  -63.20  169.43   18.24\n",
      "  167        463T 463T N   CA   3558 3559  113.30  143.50                  -42.10\n",
      "  168  15619 463T 464L C   N    3563 3565  -92.91  -70.70   33.47    2.11  -63.50  154.97   23.30\n",
      "  168        464L 464L N   CA   3565 3566  166.65  141.60                  -41.20\n",
      "  169  15624 468C 469Y C   N    3598 3600  -49.24  -98.40   50.08    2.04  -63.50  179.19   27.71\n",
      "  169        469Y 469Y N   CA   3600 3601  137.97  128.40                  -43.40\n",
      "  170  15625 469Y 470M C   N    3610 3612 -128.79 -125.60   53.50    2.49  -63.40  141.60   26.25\n",
      "  170        470M 470M N   CA   3612 3613 -166.10  140.50                  -40.50\n",
      "  171  15626 470M 471L C   N    3618 3620  -71.45  -70.70   18.40    1.37  -63.50  159.01   22.54\n",
      "  171        471L 471L N   CA   3620 3621  159.99  141.60                  -41.20\n",
      "  172  15627 471L 472A C   N    3626 3628 -124.68 -134.00   22.84    1.41  -62.50  163.54   30.13\n",
      "  172        472A 472A N   CA   3628 3629  167.85  147.00                  -40.90\n",
      "  173  15632 476P 477F C   N    3656 3658 -128.52 -124.20    7.28    0.40  -63.20 -170.16   30.67\n",
      "  173        477F 477F N   CA   3658 3659  137.45  143.30                  -44.30\n",
      "  174  15633 477F 478L C   N    3667 3669 -122.74 -108.50   14.51    0.66  -63.50 -173.82   23.22\n",
      "  174        478L 478L N   CA   3669 3670  135.30  132.50                  -41.20\n",
      "  175  15634 478L 479I C   N    3675 3677  -63.14  -63.40    0.42    0.06 -120.60 -176.53    8.40\n",
      "  175        479I 479I N   CA   3677 3678  -43.94  -43.60                  130.30\n",
      "  176  15636 480C 481H C   N    3689 3691  -54.05  -67.60   14.37    1.21  -63.20  173.16   21.09\n",
      "  176        481H 481H N   CA   3691 3692  144.78  140.00                  -42.30\n",
      "  177  15638 482I 483H C   N    3707 3709 -141.52 -125.60   27.84    0.73  -63.20  174.61   26.92\n",
      "  177        483H 483H N   CA   3709 3710  161.64  138.80                  -42.30\n",
      "  178  15640 484V 485Y C   N    3724 3726 -128.25 -124.30   18.36    0.82  -63.50  175.64   31.11\n",
      "  178        485Y 485Y N   CA   3726 3727  153.33  135.40                  -43.40\n",
      "  179  15642 486L 487T C   N    3744 3746 -101.92  -78.10   56.86    1.87  -63.20  122.74   18.31\n",
      "  179        487T 487T N   CA   3746 3747 -158.57  149.80                  -42.10\n",
      "  180  15643 487T 488E C   N    3751 3753  -71.93  -69.30   21.56    1.55  -63.60  156.03   21.26\n",
      "  180        488E 488E N   CA   3753 3754  163.90  142.50                  -40.30\n",
      "  181  15644 488E 489E C   N    3760 3762  -60.48  -69.30    9.06    0.61  -63.60  179.31   23.74\n",
      "  181        489E 489E N   CA   3762 3763  140.42  142.50                  -40.30\n",
      "  182  15645 489E 490S C   N    3769 3771 -146.74 -136.60   14.42    0.47  -64.10 -176.75   19.20\n",
      "  182        490S 490S N   CA   3771 3772  161.44  151.20                  -35.00\n",
      "  183  15648 492V 493A C   N    3788 3790 -177.81 -134.00   82.53    3.19  -62.50  154.06   30.99\n",
      "  183        493A 493A N   CA   3790 3791 -143.06  147.00                  -40.90\n",
      "  184  15650 494V 495L C   N    3800 3802  -76.26  -63.50   44.99    7.03  -63.50   44.99    7.03\n",
      "  184        495L 495L N   CA   3802 3803  -84.35  -41.20                  -41.20\n",
      "  185  15655 499R 500E C   N    3846 3848  118.12 -117.80  130.48    3.97  -63.60 -131.76   39.11\n",
      "  185        500E 500E N   CA   3848 3849  177.18  136.80                  -40.30\n",
      "  186  15656 500E 501L C   N    3855 3857  -70.80  -70.70    4.83    0.36  -63.50  172.52   24.38\n",
      "  186        501L 501L N   CA   3857 3858  146.43  141.60                  -41.20\n",
      "  187  15657 501L 502Q C   N    3863 3865  -57.82  -73.00   39.02    2.29  -63.80  145.18   21.66\n",
      "  187        502Q 502Q N   CA   3865 3866  104.75  140.70                  -40.30\n",
      "  188  15658 502Q 503H C   N    3872 3874 -168.57 -125.60   61.49    1.53  -63.20  171.20   28.01\n",
      "  188        503H 503H N   CA   3874 3875 -177.23  138.80                  -42.30\n",
      "  189  15661 505P 506D C   N    3897 3899 -133.67  -70.90   71.59    3.86  -63.30  152.50   23.35\n",
      "  189        506D 506D N   CA   3899 3900 -175.29  150.30                  -40.00\n",
      "  190  15662 506D 507Y C   N    3905 3907 -121.81  -63.50   78.67   14.88  -63.50   78.67   14.88\n",
      "  190        507Y 507Y N   CA   3907 3908  -96.21  -43.40                  -43.40\n",
      "  191  15670 514V 515T C   N    3960 3962  -69.66  -78.10   17.30    0.57  -63.20  176.91   22.27\n",
      "  191        515T 515T N   CA   3962 3963  134.69  149.80                  -42.10\n",
      "  192  15672 516V 517W C   N    3974 3976 -140.72 -124.90   27.55    0.94  -63.00  168.81   26.34\n",
      "  192        517W 517W N   CA   3976 3977  165.95  143.40                  -44.20\n",
      "  193  15673 517W 518F C   N    3988 3990  -56.64  -71.40   24.69    2.19  -63.20  155.35   21.13\n",
      "  193        518F 518F N   CA   3990 3991  160.48  140.70                  -44.30\n",
      "  194  15674 518F 519E C   N    3999 4001 -107.89 -117.80   26.42    1.44  -63.60  164.48   24.44\n",
      "  194        519E 519E N   CA   4001 4002  161.29  136.80                  -40.30\n",
      "  195  15675 519E 520G C   N    4008 4010   98.99   78.70   56.85    0.90   82.20  133.35    7.31\n",
      "  195        520G 520G N   CA   4010 4011  140.79 -166.10                    8.50\n",
      "  196  15676 520G 521H C   N    4012 4014 -116.97 -125.60   51.44    1.78  -63.20  141.05   15.16\n",
      "  196        521H 521H N   CA   4014 4015   88.09  138.80                  -42.30\n",
      "  197  15677 521H 522Y C   N    4022 4024 -131.87 -124.30    9.76    0.47  -63.50 -172.08   33.24\n",
      "  197        522Y 522Y N   CA   4024 4025  141.56  135.40                  -43.40\n",
      "  198  15678 522Y 523S C   N    4034 4036  -62.00  -72.40   26.59    1.26  -64.10  162.95   11.93\n",
      "  198        523S 523S N   CA   4036 4037  127.93  152.40                  -35.00\n",
      "  199  15679 523S 524S C   N    4040 4042 -146.46 -136.60   37.13    1.65  -64.10  160.71   17.58\n",
      "  199        524S 524S N   CA   4042 4043 -173.01  151.20                  -35.00\n",
      "  200  15680 524S 525A C   N    4046 4048  -60.73  -68.20   11.48    1.13  -62.50  165.08   26.97\n",
      "  200        525A 525A N   CA   4048 4049  154.03  145.30                  -40.90\n",
      "  201  15682 526V 527Q C   N    4058 4060 -122.17 -121.10   36.50    1.72  -63.80  154.94   25.86\n",
      "  201        527Q 527Q N   CA   4060 4061  176.18  139.70                  -40.30\n",
      "  202  15683 527Q 528T C   N    4067 4069  -67.17  -78.10   12.38    0.57  -63.20  173.97   22.57\n",
      "  202        528T 528T N   CA   4069 4070  143.98  149.80                  -42.10\n",
      "  203  15685 529P 530I C   N    4081 4083   42.68  -97.30  139.98    5.83 -120.60  163.30   11.26\n",
      "  203        530I 530I N   CA   4083 4084  127.72  127.20                  130.30\n",
      "  204  15687 531V 532R C   N    4096 4098  -58.22 -125.20   74.08    2.27  -63.00  150.13   20.60\n",
      "  204        532R 532R N   CA   4098 4099  108.96  140.60                  -41.10\n",
      "  205  15689 533P 534N C   N    4114 4116 -145.01 -119.90   35.78    1.05  -63.20  176.51   26.66\n",
      "  205        534N 534N N   CA   4116 4117  162.49  137.00                  -41.10\n",
      "  206  15690 534N 535G C   N    4122 4124  158.21 -167.20   35.58    0.90   82.20  175.10   11.98\n",
      "  206        535G 535G N   CA   4124 4125  166.25  174.60                    8.50\n",
      "  207  15691 535G 536G C   N    4126 4128 -179.41 -167.20   30.93    1.27   82.20  169.22   12.44\n",
      "  207        536G 536G N   CA   4128 4129  146.17  174.60                    8.50\n",
      "  208  15692 536G 537N C   N    4130 4132 -142.19 -119.90   26.37    0.78  -63.20 -174.53   27.64\n",
      "  208        537N 537N N   CA   4132 4133  151.09  137.00                  -41.10\n",
      "  209  15694 538P 539Q C   N    4145 4147 -155.69 -121.10   50.62    1.67  -63.80  170.02   29.65\n",
      "  209        539Q 539Q N   CA   4147 4148  176.65  139.70                  -40.30\n",
      "  210  15695 539Q 540F C   N    4154 4156 -127.82 -124.20    3.73    0.12  -63.20 -176.73   29.71\n",
      "  210        540F 540F N   CA   4156 4157  144.20  143.30                  -44.30\n",
      "  211  15696 540F 541S C   N    4165 4167 -148.34 -136.60   33.61    1.42  -64.10  165.37   18.05\n",
      "  211        541S 541S N   CA   4167 4168 -177.31  151.20                  -35.00\n",
      "  212  15697 541S 542L C   N    4171 4173 -112.30 -108.50   13.81    0.78  -63.50  167.69   21.04\n",
      "  212        542L 542L N   CA   4173 4174  119.23  132.50                  -41.20\n",
      "  213  15698 542L 543R C   N    4179 4181 -162.78 -125.20   53.62    1.64  -63.00  171.95   29.78\n",
      "  213        543R 543R N   CA   4181 4182  178.86  140.60                  -41.10\n",
      "  214  15699 543R 544Q C   N    4190 4192  -92.04  -73.00   58.68    4.22  -63.80  128.64   17.56\n",
      "  214        544Q 544Q N   CA   4192 4193   85.20  140.70                  -40.30\n",
      "  215  15700 544Q 545M C   N    4199 4201 -160.36 -125.60   34.84    1.43  -63.40 -156.68   25.16\n",
      "  215        545M 545M N   CA   4201 4202  138.21  140.50                  -40.50\n",
      "  216  15702 546Y 547L C   N    4219 4221 -113.77 -108.50   35.94    1.88  -63.50  158.91   25.17\n",
      "  216        547L 547L N   CA   4221 4222  168.05  132.50                  -41.20\n",
      "  217  15703 547L 548F C   N    4227 4229 -103.96 -124.20   20.36    0.86  -63.20  174.98   27.04\n",
      "  217        548F 548F N   CA   4229 4230  145.53  143.30                  -44.30\n",
      "  218  15704 548F 549G C   N    4238 4240  -83.18  -80.20   17.61    0.84   82.20 -137.90    7.40\n",
      "  218        549G 549G N   CA   4240 4241  156.74  174.10                    8.50\n",
      "  219  15706 550P 551H C   N    4249 4251 -115.11 -125.60   28.47    1.24  -63.20  161.03   23.53\n",
      "  219        551H 551H N   CA   4251 4252  165.26  138.80                  -42.30\n",
      "  220  15707 551H 552T C   N    4259 4261 -102.67 -124.80   22.63    1.07  -63.20 -176.57   26.16\n",
      "  220        552T 552T N   CA   4261 4262  138.77  143.50                  -42.10\n",
      "  221  15708 552T 553E C   N    4266 4268  175.91 -117.80   89.67    2.81  -63.60  171.83   29.09\n",
      "  221        553E 553E N   CA   4268 4269 -162.82  136.80                  -40.30\n",
      "  222  15709 553E 554D C   N    4275 4277  121.80   54.50   92.45   11.60  -63.30 -133.26   26.67\n",
      "  222        554D 554D N   CA   4277 4278  104.29   40.90                  -40.00\n",
      "  223  15710 554D 555L C   N    4283 4285 -118.79 -108.50   46.02    2.58  -63.50  140.21   17.26\n",
      "  223        555L 555L N   CA   4285 4286   87.64  132.50                  -41.20\n",
      "  224  15711 555L 556E C   N    4291 4293 -165.39 -117.80   50.44    1.52  -63.60 -165.14   31.57\n",
      "  224        556E 556E N   CA   4293 4294  153.53  136.80                  -40.30\n",
      "  225  15712 556E 557R C   N    4300 4302 -168.60 -125.20   57.89    1.73  -63.00  175.35   30.57\n",
      "  225        557R 557R N   CA   4302 4303  178.92  140.60                  -41.10\n",
      "  226  15713 557R 558F C   N    4311 4313  -62.86  -71.40   29.98    2.15  -63.20  156.26   21.72\n",
      "  226        558F 558F N   CA   4313 4314  111.96  140.70                  -44.30\n",
      "  227  15715 559F 560Q C   N    4333 4335  -86.43  -73.00   18.16    1.41  -63.80  170.29   23.90\n",
      "  227        560Q 560Q N   CA   4335 4336  128.47  140.70                  -40.30\n",
      "  228  15716 560Q 561L C   N    4342 4344 -127.01 -108.50   22.54    1.24  -63.50  172.92   21.37\n",
      "  228        561L 561L N   CA   4344 4345  119.64  132.50                  -41.20\n",
      "  229  15718 562D 563V C   N    4358 4360  -61.49  -62.40    2.48    0.27 -125.40 -176.51   10.31\n",
      "  229        563V 563V N   CA   4360 4361  -44.70  -42.40                  143.30\n",
      "  230  15719 563V 564L C   N    4365 4367  -76.85  -70.70   50.97    4.10  -63.50  132.88   17.73\n",
      "  230        564L 564L N   CA   4367 4368   91.00  141.60                  -41.20\n",
      "  231  15720 564L 565N C   N    4373 4375 -122.65 -119.90   52.96    2.47  -63.20  138.61   14.99\n",
      "  231        565N 565N N   CA   4375 4376   84.11  137.00                  -41.10\n",
      "  232  15721 565N 566F C   N    4381 4383 -129.38 -124.20   26.01    1.09  -63.20  161.12   26.67\n",
      "  232        566F 566F N   CA   4383 4384  168.79  143.30                  -44.30\n",
      "  233  15724 568V 569E C   N    4410 4412  -59.09  -69.30   11.12    0.69  -63.60  178.45   24.04\n",
      "  233        569E 569E N   CA   4412 4413  138.09  142.50                  -40.30\n",
      "  234  15725 569E 570G C   N    4419 4421 -130.57 -167.20   72.43    2.10   82.20 -179.97   14.06\n",
      "  234        570G 570G N   CA   4421 4422  112.12  174.60                    8.50\n",
      "  235  15727 571I 572M C   N    4431 4433  -91.95  -73.00   21.04    1.43  -63.40  169.78   27.63\n",
      "  235        572M 572M N   CA   4433 4434  152.13  143.00                  -40.50\n",
      "  236  15728 572M 573S C   N    4439 4441 -122.79 -136.60   23.66    0.84  -64.10  177.00   10.90\n",
      "  236        573S 573S N   CA   4441 4442  131.99  151.20                  -35.00\n",
      "  237  15729 573S 574D C   N    4445 4447  -75.15  -70.90    4.34    0.30  -63.30  169.28   21.63\n",
      "  237        574D 574D N   CA   4447 4448  151.14  150.30                  -40.00\n",
      "  238  15730 574D 575L C   N    4453 4455 -140.02 -108.50   52.26    2.42  -63.50  163.61   27.44\n",
      "  238        575L 575L N   CA   4455 4456  174.18  132.50                  -41.20\n",
      "  239  15731 575L 576T C   N    4461 4463 -140.75 -124.80   16.03    0.81  -63.20 -170.63   29.36\n",
      "  239        576T 576T N   CA   4463 4464  145.14  143.50                  -42.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33021  38V 498L CB  CD1   279 3833   11.70    4.05    7.65    5.27    4.05    7.65    5.27\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    5   76  137  558  445  655  554  609  666  767  843\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00199.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      577     4476     4476\n",
      "openf___224_> Open           EAO00199.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     47436912   46325.109    45.239\n",
      "read_mo_297_> Segments, residues, atoms:        1      577     4476\n",
      "read_mo_298_> Segment:        1  1:A  577:A     4476\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4476     4476        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    21012\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22310\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    22981\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    23873\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    24919\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    25728\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    27228\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    28107\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    29246\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    30620\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    31777\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33640\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4476    33781\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4996       16  16\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.9478       33  33\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7577       36  36\n",
      "iupac_m_485_> OD1/2 will be swapped:     -144.1068       41  41\n",
      "iupac_m_487_> NH1/2 swapped:      179.1233       46  46\n",
      "iupac_m_485_> OD1/2 will be swapped:     -142.5406       54  54\n",
      "iupac_m_487_> NH1/2 swapped:     -177.7964       55  55\n",
      "iupac_m_485_> OD1/2 will be swapped:      117.0451       62  62\n",
      "iupac_m_487_> NH1/2 swapped:     -178.2574       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.6195      120  120\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.6497      121  121\n",
      "iupac_m_487_> NH1/2 swapped:      178.5737      125  125\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.2647      127  127\n",
      "iupac_m_485_> OD1/2 will be swapped:     -175.9987      129  129\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      106.4046      136  136\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.0136      155  155\n",
      "iupac_m_487_> NH1/2 swapped:      177.1286      157  157\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.1213      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.2740      162  162\n",
      "iupac_m_486_> OE1/2 will be swapped:      161.3432      168  168\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.3076      175  175\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -136.0201      188  188\n",
      "iupac_m_487_> NH1/2 swapped:     -176.9434      189  189\n",
      "iupac_m_486_> OE1/2 will be swapped:      -90.5199      192  192\n",
      "iupac_m_486_> OE1/2 will be swapped:      145.4049      205  205\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.9622      210  210\n",
      "iupac_m_485_> OD1/2 will be swapped:     -147.8623      231  231\n",
      "iupac_m_487_> NH1/2 swapped:      177.9465      233  233\n",
      "iupac_m_485_> OD1/2 will be swapped:      106.8248      234  234\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.1803      254  254\n",
      "iupac_m_486_> OE1/2 will be swapped:     -103.5290      262  262\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0101      275  275\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5195      283  283\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.2211      289  289\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.8261\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5628      293  293\n",
      "iupac_m_487_> NH1/2 swapped:      178.2754      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.6983      326  326\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.0854      333  333\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.3518\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.1642      355  355\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      173.7927      368  368\n",
      "iupac_m_486_> OE1/2 will be swapped:     -116.9132      372  372\n",
      "iupac_m_485_> OD1/2 will be swapped:      127.9241      380  380\n",
      "iupac_m_487_> NH1/2 swapped:     -176.2940      387  387\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.4440      389  389\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.3484      391  391\n",
      "iupac_m_486_> OE1/2 will be swapped:      -96.2817      406  406\n",
      "iupac_m_486_> OE1/2 will be swapped:       90.9219      408  408\n",
      "iupac_m_485_> OD1/2 will be swapped:      119.0136      415  415\n",
      "iupac_m_487_> NH1/2 swapped:      178.4825      416  416\n",
      "iupac_m_486_> OE1/2 will be swapped:     -159.1896      417  417\n",
      "iupac_m_486_> OE1/2 will be swapped:     -142.3473      429  429\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.2649      443  443\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.3729      460  460\n",
      "iupac_m_486_> OE1/2 will be swapped:      134.8882      488  488\n",
      "iupac_m_486_> OE1/2 will be swapped:     -166.7990      489  489\n",
      "iupac_m_486_> OE1/2 will be swapped:     -115.4463      519  519\n",
      "iupac_m_487_> NH1/2 swapped:      175.6811      543  543\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.1067      546  546\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.1188      548  548\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.7063      553  553\n",
      "iupac_m_487_> NH1/2 swapped:      177.0332      557  557\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.4746      559  559\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.2798      566  566\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      133.2087      569  569\n",
      "iupac_m_485_> OD1/2 will be swapped:      135.8898      574  574\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  SER:OXT  SER\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   941652\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -41837.7383\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -41837.738281\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "% sequence identity      :    32.907001\n",
      "Sequence length          :          577\n",
      "Compactness              :     0.146116\n",
      "Native energy (pair)     :   628.530097\n",
      "Native energy (surface)  :    48.396206\n",
      "Native energy (combined) :    18.030312\n",
      "Z score (pair)           :    -4.115109\n",
      "Z score (surface)        :    -1.705373\n",
      "Z score (combined)       :    -3.802695\n",
      "GA341 score              :     0.691064\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      577\n",
      "Number of all, selected real atoms                :     4476    4476\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781   33781\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    10990\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        5928.5049\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4546       0      2   0.008   0.008      83.525       1.000\n",
      " 2 Bond angle potential               :    6129       6     37   2.402   2.402      670.46       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2670       0    151  52.426  52.426      1176.2       1.000\n",
      " 4 Stereochemical improper torsion pot:    1811       0      3   1.560   1.560      91.953       1.000\n",
      " 5 Soft-sphere overlap restraints     :   10990       0      2   0.006   0.006      50.589       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6300      11     61   0.893   0.893      867.23       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6434       8     95   1.092   1.092      1064.1       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     576       2     18   5.615   5.615      213.52       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     488       0     11  84.125  84.125      188.96       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     356       0      3  83.142  83.142      184.31       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     173       0      2  93.575  93.575      144.90       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      65       0      0 105.467 105.467      48.209       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2777       0      0   0.686   0.686      67.196       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     575     233    153  41.757 111.230      958.12       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     881       0      5   1.976   1.976      119.18       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00199.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00199.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   95849.3516\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5912 132A 132A N   CA   1000 1001  123.53  107.00   16.53    4.75  107.00   16.53    4.75\n",
      "    2   9282 454I 454I N   CA   3487 3488  123.11  107.00   16.12    4.63  107.00   16.12    4.63\n",
      "    3   9302 455P 456F C   N    3500 3502  139.82  120.00   19.82    4.50  120.00   19.82    4.50\n",
      "    4   9305 456F 456F N   CA   3502 3503  125.30  107.00   18.30    5.26  107.00   18.30    5.26\n",
      "    5   9318 456F 457G C   N    3511 3513  143.66  120.00   23.66    5.38  120.00   23.66    5.38\n",
      "    6   9402 464L 465V C   N    3571 3573  142.97  120.00   22.97    5.22  120.00   22.97    5.22\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19895  89Q 454I CA  CA    677 3488   16.76   12.45    4.31    6.67   12.45    4.31    6.67\n",
      "    2  19941  90T 454I CA  CA    686 3488   17.89   13.45    4.45    5.14   13.45    4.45    5.14\n",
      "    3  22290 277M 283R CA  CA   2135 2182   14.65    9.94    4.71    4.72    9.94    4.71    4.72\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24549  51F  36R N   O     367  271    5.13    2.83    2.30    7.21    2.83    2.30    7.21\n",
      "    2  25469  89Q 341T N   O     676 2621    9.31    5.85    3.46    5.16    5.85    3.46    5.16\n",
      "    3  26380 138V 219V N   O    1051 1698    4.82    3.07    1.75    4.92    3.07    1.75    4.92\n",
      "    4  28470 342N 144Y N   O    2622 1108   10.15    6.52    3.63    4.83    6.52    3.63    4.83\n",
      "    5  28567 369L 143V N   O    2848 1096   10.30    7.94    2.36    4.55    7.94    2.36    4.55\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16189 458S 458S CA  C    3518 3521 -151.66 -180.00   28.33    5.66 -180.00   28.33    5.66\n",
      "    2  16228 497R 497R CA  C    3819 3827 -134.41 -180.00   45.59    9.11 -180.00   45.59    9.11\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15157   1M   2G C   N       7    9 -115.64  -80.20   43.50    3.18   82.20 -145.52   16.63\n",
      "    1          2G   2G N   CA      9   10  148.89  174.10                    8.50\n",
      "    2  15158   2G   3E C   N      11   13  177.92 -117.80   90.33    2.89  -63.60  168.23   28.50\n",
      "    2          3E   3E N   CA     13   14 -159.73  136.80                  -40.30\n",
      "    3  15175  19P  20P C   N     134  136  -59.67  -64.50   34.38    2.32  -58.70  143.66   11.79\n",
      "    3         20P  20P N   CA    136  137  113.16  147.20                  -30.50\n",
      "    4  15176  20P  21R C   N     141  143  126.90 -125.20  122.86    3.69  -63.00 -152.09   37.85\n",
      "    4         21R  21R N   CA    143  144 -160.65  140.60                  -41.10\n",
      "    5  15178  22Q  23G C   N     161  163  125.20   78.70   76.40    1.37   82.20  131.98    8.45\n",
      "    5         23G  23G N   CA    163  164  133.28 -166.10                    8.50\n",
      "    6  15189  33D  34D C   N     242  244 -114.22  -63.30   67.49    7.85  -63.30   67.49    7.85\n",
      "    6         34D  34D N   CA    244  245    4.29  -40.00                  -40.00\n",
      "    7  15199  43E  44G C   N     320  322   56.97   78.70   81.29    1.48  -62.40  128.14   20.56\n",
      "    7         44G  44G N   CA    322  323  -87.77 -166.10                  -41.20\n",
      "    8  15258 102L 103N C   N     775  777  -51.90  -63.20   11.37    1.77 -119.90 -170.52    6.90\n",
      "    8        103N 103N N   CA    777  778  -39.85  -41.10                  137.00\n",
      "    9  15259 103N 104S C   N     783  785  -60.62  -72.40   11.87    0.89  -64.10  171.20   12.20\n",
      "    9        104S 104S N   CA    785  786  153.83  152.40                  -35.00\n",
      "   10  15261 105K 106E C   N     798  800   67.27   54.60   42.95    2.41  -63.60  137.34   23.26\n",
      "   10        106E 106E N   CA    800  801    1.36   42.40                  -40.30\n",
      "   11  15262 106E 107G C   N     807  809   28.18   78.70   87.75    1.47  -62.40  105.02   15.98\n",
      "   11        107G 107G N   CA    809  810  -94.36 -166.10                  -41.20\n",
      "   12  15275 119I 120E C   N     906  908  -55.79  -69.30   31.79    2.82  -63.60  148.63   19.40\n",
      "   12        120E 120E N   CA    908  909  171.28  142.50                  -40.30\n",
      "   13  15287 131A 132A C   N     998 1000  -65.19  -62.50   28.41    4.53 -134.00  173.81    7.74\n",
      "   13        132A 132A N   CA   1000 1001  -12.61  -40.90                  147.00\n",
      "   14  15314 158G 159Q C   N    1218 1220  -93.62  -73.00   37.32    2.83  -63.80  152.83   21.02\n",
      "   14        159Q 159Q N   CA   1220 1221  109.59  140.70                  -40.30\n",
      "   15  15315 159Q 160E C   N    1227 1229 -170.75 -117.80   79.65    2.65  -63.60  163.43   27.41\n",
      "   15        160E 160E N   CA   1229 1230 -163.70  136.80                  -40.30\n",
      "   16  15317 161P 162E C   N    1243 1245  -33.01  -69.30   51.87    2.89  -63.60  148.92   21.58\n",
      "   16        162E 162E N   CA   1245 1246  105.44  142.50                  -40.30\n",
      "   17  15318 162E 163S C   N    1252 1254 -124.27 -136.60   24.46    1.39  -64.10  164.10   16.15\n",
      "   17        163S 163S N   CA   1254 1255  172.33  151.20                  -35.00\n",
      "   18  15325 169D 170S C   N    1310 1312  -76.93 -136.60   67.39    2.07  -64.10  155.40   10.56\n",
      "   18        170S 170S N   CA   1312 1313  119.86  151.20                  -35.00\n",
      "   19  15326 170S 171Q C   N    1316 1318  -75.34  -73.00   21.49    1.45  -63.80  160.06   22.94\n",
      "   19        171Q 171Q N   CA   1318 1319  119.34  140.70                  -40.30\n",
      "   20  15327 171Q 172H C   N    1325 1327  -92.21  -67.60   73.63    6.30  -63.20  116.58   13.03\n",
      "   20        172H 172H N   CA   1327 1328   70.60  140.00                  -42.30\n",
      "   21  15328 172H 173R C   N    1335 1337 -144.85 -125.20   54.62    2.96  -63.00  154.23   17.45\n",
      "   21        173R 173R N   CA   1337 1338   89.63  140.60                  -41.10\n",
      "   22  15330 174V 175Y C   N    1353 1355 -106.98  -98.40    8.68    0.35  -63.50  175.89   25.83\n",
      "   22        175Y 175Y N   CA   1355 1356  127.04  128.40                  -43.40\n",
      "   23  15332 176V 177K C   N    1372 1374  -97.10 -118.00   21.72    0.74  -62.90  177.32   21.39\n",
      "   23        177K 177K N   CA   1374 1375  133.19  139.10                  -40.80\n",
      "   24  15333 177K 178G C   N    1381 1383  -65.80  -80.20   47.97    1.25   82.20 -169.56    6.75\n",
      "   24        178G 178G N   CA   1383 1384  128.34  174.10                    8.50\n",
      "   25  15334 178G 179A C   N    1385 1387 -146.44 -134.00   12.72    0.53  -62.50 -166.11   36.27\n",
      "   25        179A 179A N   CA   1387 1388  144.32  147.00                  -40.90\n",
      "   26  15335 179A 180G C   N    1390 1392   62.54   78.70   18.93    0.61   82.20  165.91    9.03\n",
      "   26        180G 180G N   CA   1392 1393 -156.24 -166.10                    8.50\n",
      "   27  15337 181V 182H C   N    1401 1403 -103.32 -125.60   26.62    1.18  -63.20  169.16   23.77\n",
      "   27        182H 182H N   CA   1403 1404  153.36  138.80                  -42.30\n",
      "   28  15338 182H 183S C   N    1411 1413 -141.51 -136.60   30.79    1.45  -64.10  162.96   17.37\n",
      "   28        183S 183S N   CA   1413 1414 -178.41  151.20                  -35.00\n",
      "   29  15341 185L 186S C   N    1432 1434  -65.91  -72.40   62.11    3.43  -64.10  125.65    9.00\n",
      "   29        186S 186S N   CA   1434 1435   90.63  152.40                  -35.00\n",
      "   30  15343 187L 188E C   N    1446 1448   77.01   54.60  131.60   11.92  -63.60 -156.13   24.17\n",
      "   30        188E 188E N   CA   1448 1449  172.08   42.40                  -40.30\n",
      "   31  15352 196K 197G C   N    1525 1527 -116.30  -80.20   87.41    5.20   82.20 -177.04   14.25\n",
      "   31        197G 197G N   CA   1527 1528   94.49  174.10                    8.50\n",
      "   32  15353 197G 198N C   N    1529 1531 -145.01 -119.90   30.43    0.89  -63.20 -176.09   27.60\n",
      "   32        198N 198N N   CA   1531 1532  154.19  137.00                  -41.10\n",
      "   33  15354 198N 199A C   N    1537 1539  124.28 -134.00  102.05    3.08  -62.50 -121.44   47.81\n",
      "   33        199A 199A N   CA   1539 1540  155.07  147.00                  -40.90\n",
      "   34  15355 199A 200N C   N    1542 1544 -129.03 -119.90   22.31    1.18  -63.20  170.93   18.62\n",
      "   34        200N 200N N   CA   1544 1545  116.64  137.00                  -41.10\n",
      "   35  15356 200N 201R C   N    1550 1552  158.00 -125.20   81.24    2.63  -63.00 -154.18   36.59\n",
      "   35        201R 201R N   CA   1552 1553  167.10  140.60                  -41.10\n",
      "   36  15357 201R 202Q C   N    1561 1563  -79.77  -73.00   26.79    1.89  -63.80  155.90   22.11\n",
      "   36        202Q 202Q N   CA   1563 1564  114.78  140.70                  -40.30\n",
      "   37  15358 202Q 203T C   N    1570 1572 -140.01 -124.80   26.05    0.81  -63.20  171.42   26.97\n",
      "   37        203T 203T N   CA   1572 1573  164.65  143.50                  -42.10\n",
      "   38  15359 203T 204G C   N    1577 1579   89.56   78.70   38.42    0.68   82.20  148.73    7.59\n",
      "   38        204G 204G N   CA   1579 1580  157.05 -166.10                    8.50\n",
      "   39  15360 204G 205E C   N    1581 1583 -103.34 -117.80   15.60    0.72  -63.60 -178.54   26.44\n",
      "   39        205E 205E N   CA   1583 1584  142.65  136.80                  -40.30\n",
      "   40  15361 205E 206T C   N    1590 1592  -72.31  -78.10    7.30    0.29  -63.20  172.79   22.76\n",
      "   40        206T 206T N   CA   1592 1593  145.35  149.80                  -42.10\n",
      "   41  15362 206T 207K C   N    1597 1599 -106.42 -118.00   12.30    0.41  -62.90 -178.94   21.53\n",
      "   41        207K 207K N   CA   1599 1600  134.96  139.10                  -40.80\n",
      "   42  15363 207K 208M C   N    1606 1608 -109.56 -125.60   20.40    0.61  -63.40  174.61   22.91\n",
      "   42        208M 208M N   CA   1608 1609  127.89  140.50                  -40.50\n",
      "   43  15364 208M 209N C   N    1614 1616 -149.72 -119.90   39.08    1.13  -63.20  178.96   27.21\n",
      "   43        209N 209N N   CA   1616 1617  162.25  137.00                  -41.10\n",
      "   44  15366 210E 211T C   N    1631 1633  -73.25  -78.10   10.16    0.33  -63.20  177.31   23.40\n",
      "   44        211T 211T N   CA   1633 1634  140.87  149.80                  -42.10\n",
      "   45  15367 211T 212S C   N    1638 1640  -87.35  -72.40   23.97    1.10  -64.10  155.61   12.80\n",
      "   45        212S 212S N   CA   1640 1641  171.14  152.40                  -35.00\n",
      "   46  15368 212S 213S C   N    1644 1646 -120.55 -136.60   16.64    0.56  -64.10 -173.09   17.48\n",
      "   46        213S 213S N   CA   1646 1647  146.81  151.20                  -35.00\n",
      "   47  15369 213S 214R C   N    1650 1652  -87.99  -72.10   22.65    1.47  -63.00  162.78   23.71\n",
      "   47        214R 214R N   CA   1652 1653  158.05  141.90                  -41.10\n",
      "   48  15371 215S 216H C   N    1667 1669  -62.23  -67.60   12.74    1.11  -63.20  166.15   20.71\n",
      "   48        216H 216H N   CA   1669 1670  151.55  140.00                  -42.30\n",
      "   49  15372 216H 217A C   N    1677 1679 -150.46 -134.00   28.77    1.06  -62.50  172.60   32.97\n",
      "   49        217A 217A N   CA   1679 1680  170.60  147.00                  -40.90\n",
      "   50  15375 219V 220Q C   N    1697 1699   92.47  -63.80  163.77   24.75  -73.00 -149.57   16.41\n",
      "   50        220Q 220Q N   CA   1699 1700  -89.30  -40.30                  140.70\n",
      "   51  15378 222T 223I C   N    1721 1723  -56.48  -63.40    6.92    1.23 -120.60 -174.34    8.33\n",
      "   51        223I 223I N   CA   1723 1724  -43.94  -43.60                  130.30\n",
      "   52  15379 223I 224S C   N    1729 1731  -72.80  -72.40   45.77    2.71  -64.10  141.90    9.80\n",
      "   52        224S 224S N   CA   1731 1732  106.63  152.40                  -35.00\n",
      "   53  15384 228G 229S C   N    1769 1771    1.72  -72.40   88.56    7.13  -64.10  140.50    9.06\n",
      "   53        229S 229S N   CA   1771 1772 -159.12  152.40                  -35.00\n",
      "   54  15385 229S 230L C   N    1775 1777  -55.89  -70.70   29.81    1.85  -63.50  157.11   22.26\n",
      "   54        230L 230L N   CA   1777 1778  115.73  141.60                  -41.20\n",
      "   55  15386 230L 231D C   N    1783 1785  -94.65  -96.50   49.41    2.08  -63.30  109.42   11.97\n",
      "   55        231D 231D N   CA   1785 1786   64.83  114.20                  -40.00\n",
      "   56  15388 232M 233R C   N    1799 1801 -178.84 -125.20   78.40    4.32  -63.00  170.06   19.88\n",
      "   56        233R 233R N   CA   1801 1802   83.42  140.60                  -41.10\n",
      "   57  15389 233R 234D C   N    1810 1812  -68.58  -63.30   75.59    9.66  -63.30   75.59    9.66\n",
      "   57        234D 234D N   CA   1812 1813 -115.41  -40.00                  -40.00\n",
      "   58  15392 236E 237S C   N    1834 1836  111.60 -136.60  119.37    3.81  -64.10 -140.26   20.00\n",
      "   58        237S 237S N   CA   1836 1837 -166.95  151.20                  -35.00\n",
      "   59  15396 240L 241L C   N    1862 1864  -76.19  -70.70   13.82    0.88  -63.50  165.00   23.66\n",
      "   59        241L 241L N   CA   1864 1865  154.28  141.60                  -41.20\n",
      "   60  15406 250Q 251S C   N    1937 1939  -67.43 -136.60   91.68    5.05  -64.10  113.67    8.44\n",
      "   60        251S 251S N   CA   1939 1940 -148.63  151.20                  -35.00\n",
      "   61  15409 253T 254E C   N    1959 1961   57.80  -69.30  135.85   11.78  -63.60  177.31   20.99\n",
      "   61        254E 254E N   CA   1961 1962 -169.54  142.50                  -40.30\n",
      "   62  15410 254E 255S C   N    1968 1970 -150.42 -136.60   14.98    0.47  -64.10 -171.11   19.88\n",
      "   62        255S 255S N   CA   1970 1971  156.98  151.20                  -35.00\n",
      "   63  15411 255S 256T C   N    1974 1976 -158.34 -124.80   46.54    1.52  -63.20  171.03   27.92\n",
      "   63        256T 256T N   CA   1976 1977  175.77  143.50                  -42.10\n",
      "   64  15412 256T 257G C   N    1981 1983  179.91 -167.20   24.49    0.98   82.20  175.07   12.74\n",
      "   64        257G 257G N   CA   1983 1984  153.78  174.60                    8.50\n",
      "   65  15414 258V 259A C   N    1992 1994 -148.39 -134.00   19.35    0.55  -62.50 -179.12   34.23\n",
      "   65        259A 259A N   CA   1994 1995  159.92  147.00                  -40.90\n",
      "   66  15415 259A 260F C   N    1997 1999 -124.00 -124.20   26.66    1.26  -63.20  157.91   25.90\n",
      "   66        260F 260F N   CA   1999 2000  169.97  143.30                  -44.30\n",
      "   67  15416 260F 261E C   N    2008 2010 -124.87 -117.80   26.10    1.41  -63.60  163.86   19.24\n",
      "   67        261E 261E N   CA   2010 2011  111.67  136.80                  -40.30\n",
      "   68  15417 261E 262E C   N    2017 2019 -136.23 -117.80   19.99    0.59  -63.60 -170.36   29.37\n",
      "   68        262E 262E N   CA   2019 2020  144.52  136.80                  -40.30\n",
      "   69  15418 262E 263A C   N    2026 2028 -106.78 -134.00   32.47    0.78  -62.50  175.87   27.01\n",
      "   69        263A 263A N   CA   2028 2029  129.30  147.00                  -40.90\n",
      "   70  15419 263A 264K C   N    2031 2033 -125.23 -118.00   48.90    2.27  -62.90  145.73   22.66\n",
      "   70        264K 264K N   CA   2033 2034 -172.53  139.10                  -40.80\n",
      "   71  15420 264K 265K C   N    2040 2042  -78.83  -70.20   76.58    5.75  -62.90  106.30   13.02\n",
      "   71        265K 265K N   CA   2042 2043   64.30  140.40                  -40.80\n",
      "   72  15422 266I 267N C   N    2057 2059 -118.75 -119.90    6.02    0.30  -63.20 -175.45   26.24\n",
      "   72        267N 267N N   CA   2059 2060  142.91  137.00                  -41.10\n",
      "   73  15423 267N 268Q C   N    2065 2067   64.85  -73.00  138.70   10.18  -63.80 -151.83   26.75\n",
      "   73        268Q 268Q N   CA   2067 2068  156.03  140.70                  -40.30\n",
      "   74  15424 268Q 269S C   N    2074 2076  -90.92  -72.40   23.99    1.92  -64.10  174.24   11.30\n",
      "   74        269S 269S N   CA   2076 2077  137.16  152.40                  -35.00\n",
      "   75  15425 269S 270L C   N    2080 2082  -90.70  -70.70   20.79    1.68  -63.50  173.69   25.76\n",
      "   75        270L 270L N   CA   2082 2083  147.25  141.60                  -41.20\n",
      "   76  15427 271L 272M C   N    2096 2098  163.64 -125.60   74.22    2.42  -63.40 -154.56   40.45\n",
      "   76        272M 272M N   CA   2098 2099  162.89  140.50                  -40.50\n",
      "   77  15428 272M 273L C   N    2104 2106  -98.47 -108.50   12.99    0.72  -63.50 -178.56   27.32\n",
      "   77        273L 273L N   CA   2106 2107  140.76  132.50                  -41.20\n",
      "   78  15429 273L 274G C   N    2112 2114   79.26   78.70   14.36    0.39   82.20  171.07    8.16\n",
      "   78        274G 274G N   CA   2114 2115  179.55 -166.10                    8.50\n",
      "   79  15430 274G 275R C   N    2116 2118 -107.86 -125.20   49.73    1.97  -63.00  142.34   16.72\n",
      "   79        275R 275R N   CA   2118 2119   93.99  140.60                  -41.10\n",
      "   80  15431 275R 276A C   N    2127 2129  161.86 -134.00   67.44    1.60  -62.50 -156.82   40.28\n",
      "   80        276A 276A N   CA   2129 2130  167.83  147.00                  -40.90\n",
      "   81  15434 278N 279S C   N    2148 2150  -75.90  -72.40   13.23    0.67  -64.10  160.27   12.35\n",
      "   81        279S 279S N   CA   2150 2151  165.16  152.40                  -35.00\n",
      "   82  15454 298T 299R C   N    2299 2301 -160.41 -125.20   45.73    1.36  -63.00  178.10   30.53\n",
      "   82        299R 299R N   CA   2301 2302  169.79  140.60                  -41.10\n",
      "   83  15455 299R 300L C   N    2310 2312 -108.62 -108.50    3.85    0.21  -63.50  175.75   22.25\n",
      "   83        300L 300L N   CA   2312 2313  128.66  132.50                  -41.20\n",
      "   84  15456 300L 301L C   N    2318 2320  -88.54  -70.70   33.59    2.08  -63.50  150.83   22.46\n",
      "   84        301L 301L N   CA   2320 2321  170.06  141.60                  -41.20\n",
      "   85  15457 301L 302S C   N    2326 2328   60.19  -72.40  140.86   11.10  -64.10  176.31   14.24\n",
      "   85        302S 302S N   CA   2328 2329 -160.04  152.40                  -35.00\n",
      "   86  15458 302S 303E C   N    2332 2334  -56.83  -69.30   13.28    0.84  -63.60  178.39   24.16\n",
      "   86        303E 303E N   CA   2334 2335  137.95  142.50                  -40.30\n",
      "   87  15459 303E 304S C   N    2341 2343 -129.45 -136.60   10.76    0.36  -64.10 -170.23   11.70\n",
      "   87        304S 304S N   CA   2343 2344  143.16  151.20                  -35.00\n",
      "   88  15460 304S 305F C   N    2347 2349 -133.59 -124.20   14.11    0.39  -63.20  176.51   29.10\n",
      "   88        305F 305F N   CA   2349 2350  153.83  143.30                  -44.30\n",
      "   89  15461 305F 306G C   N    2358 2360   78.57   78.70   11.01    0.32   82.20  174.42    8.28\n",
      "   89        306G 306G N   CA   2360 2361 -177.11 -166.10                    8.50\n",
      "   90  15462 306G 307G C   N    2362 2364 -165.03 -167.20   35.33    1.36   82.20  172.73   13.04\n",
      "   90        307G 307G N   CA   2364 2365  139.34  174.60                    8.50\n",
      "   91  15463 307G 308N C   N    2366 2368 -133.38 -119.90   14.11    0.48  -63.20 -168.89   27.88\n",
      "   91        308N 308N N   CA   2368 2369  141.15  137.00                  -41.10\n",
      "   92  15464 308N 309S C   N    2374 2376 -139.85 -136.60    7.37    0.42  -64.10 -165.08   12.12\n",
      "   92        309S 309S N   CA   2376 2377  144.59  151.20                  -35.00\n",
      "   93  15465 309S 310K C   N    2380 2382 -140.10 -118.00   41.17    1.60  -62.90  164.59   25.99\n",
      "   93        310K 310K N   CA   2382 2383  173.83  139.10                  -40.80\n",
      "   94  15468 312W 313M C   N    2410 2412  -78.80  -73.00   14.78    0.93  -63.40  163.63   25.65\n",
      "   94        313M 313M N   CA   2412 2413  156.59  143.00                  -40.50\n",
      "   95  15469 313M 314L C   N    2418 2420 -100.08 -108.50   42.52    2.37  -63.50  149.17   22.96\n",
      "   95        314L 314L N   CA   2420 2421  174.18  132.50                  -41.20\n",
      "   96  15470 314L 315A C   N    2426 2428  -73.57  -68.20   36.45    3.13  -62.50  150.56   24.15\n",
      "   96        315A 315A N   CA   2428 2429  109.25  145.30                  -40.90\n",
      "   97  15471 315A 316T C   N    2431 2433 -152.97 -124.80   31.52    1.25  -63.20 -176.30   29.33\n",
      "   97        316T 316T N   CA   2433 2434  157.63  143.50                  -42.10\n",
      "   98  15473 317V 318S C   N    2445 2447 -156.51 -136.60   27.97    0.90  -64.10  179.73   19.68\n",
      "   98        318S 318S N   CA   2447 2448  170.84  151.20                  -35.00\n",
      "   99  15474 318S 319T C   N    2451 2453 -116.13 -124.80   34.63    1.24  -63.20  161.02   17.72\n",
      "   99        319T 319T N   CA   2453 2454  109.97  143.50                  -42.10\n",
      "  100  15475 319T 320A C   N    2458 2460 -153.91 -134.00   24.11    1.31  -68.20   86.53    7.72\n",
      "  100        320A 320A N   CA   2460 2461  133.40  147.00                  145.30\n",
      "  101  15477 321A 322N C   N    2468 2470 -136.06 -119.90   74.56    2.99   55.90 -120.76   28.21\n",
      "  101        322N 322N N   CA   2470 2471 -150.21  137.00                   39.50\n",
      "  102  15497 341T 342N C   N    2620 2622  -81.48 -119.90   43.57    1.33  -63.20  158.61   18.88\n",
      "  102        342N 342N N   CA   2622 2623  116.45  137.00                  -41.10\n",
      "  103  15498 342N 343K C   N    2628 2630   20.91  -70.20  122.89   10.28  -62.90  127.70   15.19\n",
      "  103        343K 343K N   CA   2630 2631 -137.14  140.40                  -40.80\n",
      "  104  15499 343K 344A C   N    2637 2639  -83.87  -68.20   45.23    4.16  -62.50  145.35   22.87\n",
      "  104        344A 344A N   CA   2639 2640  102.88  145.30                  -40.90\n",
      "  105  15500 344A 345K C   N    2642 2644   49.09   56.60    7.57    0.84  -62.90  137.84   23.94\n",
      "  105        345K 345K N   CA   2644 2645   39.56   38.60                  -40.80\n",
      "  106  15502 346V 347N C   N    2658 2660  -90.47 -119.90   60.32    2.00   55.90  153.08   14.57\n",
      "  106        347N 347N N   CA   2660 2661   84.34  137.00                   39.50\n",
      "  107  15504 348K 349T C   N    2675 2677   35.53 -124.80  163.45    9.32  -63.20  173.46   18.35\n",
      "  107        349T 349T N   CA   2677 2678  175.27  143.50                  -42.10\n",
      "  108  15505 349T 350A C   N    2682 2684 -176.61 -134.00   45.85    1.05  -62.50 -167.40   37.56\n",
      "  108        350A 350A N   CA   2684 2685  163.94  147.00                  -40.90\n",
      "  109  15506 350A 351K C   N    2687 2689 -160.99 -118.00   49.18    1.63  -62.90 -175.54   29.78\n",
      "  109        351K 351K N   CA   2689 2690  162.98  139.10                  -40.80\n",
      "  110  15507 351K 352D C   N    2696 2698 -133.30  -96.50   39.50    1.63  -63.30 -177.50   19.51\n",
      "  110        352D 352D N   CA   2698 2699  128.55  114.20                  -40.00\n",
      "  111  15508 352D 353L C   N    2704 2706 -159.50 -108.50   51.81    2.58  -63.50 -169.50   23.35\n",
      "  111        353L 353L N   CA   2706 2707  123.34  132.50                  -41.20\n",
      "  112  15509 353L 354E C   N    2712 2714 -155.49 -117.80   37.77    1.33  -63.60 -158.39   23.37\n",
      "  112        354E 354E N   CA   2714 2715  139.15  136.80                  -40.30\n",
      "  113  15510 354E 355Y C   N    2721 2723 -150.00 -124.30   25.76    1.88  -63.50 -160.81   35.85\n",
      "  113        355Y 355Y N   CA   2723 2724  137.17  135.40                  -43.40\n",
      "  114  15511 355Y 356D C   N    2733 2735 -174.33  -96.50   86.72    3.59  -63.30 -158.98   31.86\n",
      "  114        356D 356D N   CA   2735 2736  152.43  114.20                  -40.00\n",
      "  115  15512 356D 357E C   N    2741 2743  -71.28  -69.30    2.38    0.14  -63.60  176.03   23.89\n",
      "  115        357E 357E N   CA   2743 2744  143.83  142.50                  -40.30\n",
      "  116  15514 358I 359Q C   N    2758 2760 -143.13 -121.10   26.29    0.81  -63.80 -176.35   31.15\n",
      "  116        359Q 359Q N   CA   2760 2761  154.06  139.70                  -40.30\n",
      "  117  15516 360K 361L C   N    2776 2778 -112.60 -108.50    8.16    0.47  -63.50  173.73   21.85\n",
      "  117        361L 361L N   CA   2778 2779  125.44  132.50                  -41.20\n",
      "  118  15517 361L 362A C   N    2784 2786 -168.67 -134.00   56.94    1.99  -62.50  165.49   32.64\n",
      "  118        362A 362A N   CA   2786 2787 -167.84  147.00                  -40.90\n",
      "  119  15518 362A 363W C   N    2789 2791  -53.88  -71.30   25.66    2.21  -63.00  158.22   19.25\n",
      "  119        363W 363W N   CA   2791 2792  157.84  139.00                  -44.20\n",
      "  120  15520 364V 365L C   N    2810 2812 -145.12 -108.50   52.82    2.39  -63.50  169.21   28.53\n",
      "  120        365L 365L N   CA   2812 2813  170.58  132.50                  -41.20\n",
      "  121  15521 365L 366Q C   N    2818 2820 -121.97 -121.10   20.37    0.96  -63.80  169.92   28.07\n",
      "  121        366Q 366Q N   CA   2820 2821  160.05  139.70                  -40.30\n",
      "  122  15523 367I 368R C   N    2835 2837  163.06 -125.20   73.15    3.30  -63.00 -145.61   24.59\n",
      "  122        368R 368R N   CA   2837 2838  126.31  140.60                  -41.10\n",
      "  123  15525 369L 370D C   N    2854 2856  172.53  -63.30  125.41   20.62  -63.30  125.41   20.62\n",
      "  123        370D 370D N   CA   2856 2857  -57.54  -40.00                  -40.00\n",
      "  124  15529 373A 374K C   N    2884 2886  -89.44 -118.00   39.91    1.40  -62.90  154.33   18.75\n",
      "  124        374K 374K N   CA   2886 2887  111.23  139.10                  -40.80\n",
      "  125  15530 374K 375G C   N    2893 2895  129.57   82.20   74.28    2.13  -62.40  168.20   30.34\n",
      "  125        375G 375G N   CA   2895 2896  -48.72    8.50                  -41.20\n",
      "  126  15534 378A 379L C   N    2916 2918  -57.62  -63.50    8.38    1.08  -70.70  171.73   13.52\n",
      "  126        379L 379L N   CA   2918 2919  -47.17  -41.20                  141.60\n",
      "  127  15535 379L 380D C   N    2924 2926  -77.28  -96.50   36.53    1.54  -63.30  123.92   14.50\n",
      "  127        380D 380D N   CA   2926 2927   83.13  114.20                  -40.00\n",
      "  128  15540 384Q 385G C   N    2963 2965  -57.96  -62.40   15.15    2.50  -80.20  160.73    7.45\n",
      "  128        385G 385G N   CA   2965 2966  -26.71  -41.20                  174.10\n",
      "  129  15542 386L 387R C   N    2975 2977  -89.72  -63.00   73.19   11.73  -63.00   73.19   11.73\n",
      "  129        387R 387R N   CA   2977 2978 -109.24  -41.10                  -41.10\n",
      "  130  15543 387R 388G C   N    2986 2988  167.91 -167.20   29.34    0.97   82.20  173.25   12.24\n",
      "  130        388G 388G N   CA   2988 2989  159.06  174.60                    8.50\n",
      "  131  15544 388G 389E C   N    2990 2992 -117.53 -117.80    7.36    0.37  -63.60  178.10   21.28\n",
      "  131        389E 389E N   CA   2992 2993  129.44  136.80                  -40.30\n",
      "  132  15548 392C 393M C   N    3021 3023  -70.08  -73.00    7.26    0.45  -63.40  176.98   25.91\n",
      "  132        393M 393M N   CA   3023 3024  136.35  143.00                  -40.50\n",
      "  133  15549 393M 394K C   N    3029 3031 -160.29 -118.00  100.33    4.14  -62.90  132.01   22.62\n",
      "  133        394K 394K N   CA   3031 3032 -129.92  139.10                  -40.80\n",
      "  134  15550 394K 395S C   N    3038 3040  -71.51  -72.40   13.33    0.81  -64.10  159.47   12.01\n",
      "  134        395S 395S N   CA   3040 3041  165.70  152.40                  -35.00\n",
      "  135  15552 396K 397L C   N    3053 3055  -89.29 -108.50   27.24    1.23  -63.50  156.54   20.40\n",
      "  135        397L 397L N   CA   3055 3056  113.20  132.50                  -41.20\n",
      "  136  15553 397L 398T C   N    3061 3063 -143.92 -124.80   31.66    0.98  -63.20  169.61   26.96\n",
      "  136        398T 398T N   CA   3063 3064  168.73  143.50                  -42.10\n",
      "  137  15555 399V 400A C   N    3075 3077 -110.44 -134.00   30.12    0.80  -62.50  175.79   26.90\n",
      "  137        400A 400A N   CA   3077 3078  128.23  147.00                  -40.90\n",
      "  138  15557 401S 402D C   N    3086 3088  -74.45  -96.50   48.33    2.00  -63.30  163.17   20.83\n",
      "  138        402D 402D N   CA   3088 3089  157.21  114.20                  -40.00\n",
      "  139  15564 408E 409L C   N    3141 3143  -75.09  -63.50   52.45    6.71  -63.50   52.45    6.71\n",
      "  139        409L 409L N   CA   3143 3144    9.95  -41.20                  -41.20\n",
      "  140  15568 412A 413L C   N    3172 3174  -72.60  -63.50   20.93    3.47 -108.50  171.26    9.57\n",
      "  140        413L 413L N   CA   3174 3175  -60.05  -41.20                  132.50\n",
      "  141  15581 425K 426G C   N    3284 3286  108.82   78.70   51.58    0.88   82.20  145.98    8.40\n",
      "  141        426G 426G N   CA   3286 3287  152.03 -166.10                    8.50\n",
      "  142  15582 426G 427L C   N    3288 3290 -149.71 -108.50   54.44    2.43  -63.50  173.64   29.42\n",
      "  142        427L 427L N   CA   3290 3291  168.07  132.50                  -41.20\n",
      "  143  15584 428I 429E C   N    3304 3306  -61.55  -69.30   11.45    0.64  -63.60  174.38   23.36\n",
      "  143        429E 429E N   CA   3306 3307  134.07  142.50                  -40.30\n",
      "  144  15585 429E 430Q C   N    3313 3315 -158.56 -121.10   62.84    3.27  -63.80  160.51   20.58\n",
      "  144        430Q 430Q N   CA   3315 3316   89.25  139.70                  -40.30\n",
      "  145  15587 431R 432K C   N    3333 3335 -116.91 -118.00   13.37    0.66  -62.90  175.30   25.98\n",
      "  145        432K 432K N   CA   3335 3336  152.42  139.10                  -40.80\n",
      "  146  15589 433I 434S C   N    3350 3352 -117.49 -136.60   19.74    0.67  -64.10 -173.44   17.22\n",
      "  146        434S 434S N   CA   3352 3353  146.24  151.20                  -35.00\n",
      "  147  15590 434S 435G C   N    3356 3358 -108.81  -80.20   83.83    4.71   82.20 -170.02   14.77\n",
      "  147        435G 435G N   CA   3358 3359   95.31  174.10                    8.50\n",
      "  148  15591 435G 436F C   N    3360 3362 -131.62 -124.20   19.42    1.05  -63.20 -177.07   21.45\n",
      "  148        436F 436F N   CA   3362 3363  125.35  143.30                  -44.30\n",
      "  149  15592 436F 437S C   N    3371 3373 -172.20 -136.60   38.45    1.90  -64.10 -157.13   13.70\n",
      "  149        437S 437S N   CA   3373 3374  136.67  151.20                  -35.00\n",
      "  150  15593 437S 438A C   N    3377 3379  -92.25  -68.20   26.38    1.85  -62.50  165.65   28.70\n",
      "  150        438A 438A N   CA   3379 3380  156.14  145.30                  -40.90\n",
      "  151  15596 440L 441V C   N    3395 3397  -58.78  -62.40    5.71    0.64 -125.40 -177.52   10.28\n",
      "  151        441V 441V N   CA   3397 3398  -46.82  -42.40                  143.30\n",
      "  152  15598 442C 443F C   N    3408 3410  -99.63 -124.20   31.09    0.81  -63.20  172.45   21.63\n",
      "  152        443F 443F N   CA   3410 3411  124.25  143.30                  -44.30\n",
      "  153  15600 444S 445G C   N    3425 3427  -63.45  -62.40    5.70    0.91   82.20  155.80   11.93\n",
      "  153        445G 445G N   CA   3427 3428  -46.81  -41.20                    8.50\n",
      "  154  15601 445G 446T C   N    3429 3431  -81.67  -78.10    3.78    0.19  -63.20  167.89   22.76\n",
      "  154        446T 446T N   CA   3431 3432  151.03  149.80                  -42.10\n",
      "  155  15603 447C 448K C   N    3442 3444 -108.77 -118.00   85.99    4.04  -62.90  104.96   12.06\n",
      "  155        448K 448K N   CA   3444 3445   53.61  139.10                  -40.80\n",
      "  156  15604 448K 449T C   N    3451 3453 -110.14 -124.80   21.38    0.68  -63.20  176.40   19.89\n",
      "  156        449T 449T N   CA   3453 3454  127.94  143.50                  -42.10\n",
      "  157  15606 450S 451L C   N    3464 3466  -58.88  -70.70   14.50    1.46  -63.50  168.85   23.18\n",
      "  157        451L 451L N   CA   3466 3467  150.01  141.60                  -41.20\n",
      "  158  15607 451L 452A C   N    3472 3474   14.67  -68.20  105.98    6.80  -68.20  105.98    6.80\n",
      "  158        452A 452A N   CA   3474 3475   79.24  145.30                  145.30\n",
      "  159  15608 452A 453N C   N    3477 3479  -75.41 -119.90   47.13    1.57  -63.20  162.99   19.73\n",
      "  159        453N 453N N   CA   3479 3480  121.44  137.00                  -41.10\n",
      "  160  15611 455P 456F C   N    3500 3502   34.58  -71.40  123.67    6.96  -63.20  155.77   27.45\n",
      "  160        456F 456F N   CA   3502 3503   76.96  140.70                  -44.30\n",
      "  161  15612 456F 457G C   N    3511 3513  107.73   78.70   77.18    1.19   82.20  116.71    6.91\n",
      "  161        457G 457G N   CA   3513 3514  122.39 -166.10                    8.50\n",
      "  162  15613 457G 458S C   N    3515 3517  -74.04 -136.60   88.13    2.84  -64.10  124.52    8.48\n",
      "  162        458S 458S N   CA   3517 3518   89.12  151.20                  -35.00\n",
      "  163  15618 462T 463T C   N    3556 3558  -21.89  -78.10   59.90    3.02  -63.20  176.13   25.34\n",
      "  163        463T 463T N   CA   3558 3559  129.12  149.80                  -42.10\n",
      "  164  15619 463T 464L C   N    3563 3565 -121.07 -108.50   70.32    3.66  -63.50  130.49   21.68\n",
      "  164        464L 464L N   CA   3565 3566 -158.31  132.50                  -41.20\n",
      "  165  15624 468C 469Y C   N    3598 3600 -132.29 -124.30    9.98    0.49  -63.50 -171.75   33.31\n",
      "  165        469Y 469Y N   CA   3600 3601  141.37  135.40                  -43.40\n",
      "  166  15625 469Y 470M C   N    3610 3612 -139.11 -125.60   14.77    0.72  -63.40 -169.27   23.93\n",
      "  166        470M 470M N   CA   3612 3613  134.55  140.50                  -40.50\n",
      "  167  15626 470M 471L C   N    3618 3620 -122.65 -108.50   29.23    1.66  -63.50  159.50   19.70\n",
      "  167        471L 471L N   CA   3620 3621  106.93  132.50                  -41.20\n",
      "  168  15627 471L 472A C   N    3626 3628  162.57 -134.00   63.45    2.17  -62.50 -139.98   43.12\n",
      "  168        472A 472A N   CA   3628 3629  145.31  147.00                  -40.90\n",
      "  169  15628 472A 473G C   N    3631 3633   54.15   78.70   24.96    1.14   82.20  172.43    9.76\n",
      "  169        473G 473G N   CA   3633 3634 -161.63 -166.10                    8.50\n",
      "  170  15632 476P 477F C   N    3656 3658  -68.52  -71.40   12.28    1.06  -63.20  163.15   23.01\n",
      "  170        477F 477F N   CA   3658 3659  152.63  140.70                  -44.30\n",
      "  171  15633 477F 478L C   N    3667 3669  -95.89 -108.50   65.21    3.38  -63.50  114.41   14.38\n",
      "  171        478L 478L N   CA   3669 3670   68.53  132.50                  -41.20\n",
      "  172  15638 482I 483H C   N    3707 3709 -120.56 -125.60    8.47    0.38  -63.20 -178.60   26.43\n",
      "  172        483H 483H N   CA   3709 3710  145.61  138.80                  -42.30\n",
      "  173  15639 483H 484V C   N    3717 3719  -67.00  -62.40    8.13    0.90 -125.40 -171.72    7.68\n",
      "  173        484V 484V N   CA   3719 3720  -35.69  -42.40                  143.30\n",
      "  174  15642 486L 487T C   N    3744 3746  -61.32  -78.10   25.74    1.63  -63.20  148.59   18.93\n",
      "  174        487T 487T N   CA   3746 3747  169.32  149.80                  -42.10\n",
      "  175  15643 487T 488E C   N    3751 3753  -74.71  -69.30   29.10    2.40  -63.60  154.61   20.03\n",
      "  175        488E 488E N   CA   3753 3754  113.90  142.50                  -40.30\n",
      "  176  15644 488E 489E C   N    3760 3762 -162.84 -117.80   57.39    1.74  -63.60  177.64   29.08\n",
      "  176        489E 489E N   CA   3762 3763  172.36  136.80                  -40.30\n",
      "  177  15645 489E 490S C   N    3769 3771  -65.87  -72.40   18.73    0.90  -64.10  169.85   12.20\n",
      "  177        490S 490S N   CA   3771 3772  134.84  152.40                  -35.00\n",
      "  178  15646 490S 491S C   N    3775 3777  -97.19  -72.40   29.69    1.55  -64.10  159.71   13.80\n",
      "  178        491S 491S N   CA   3777 3778  168.75  152.40                  -35.00\n",
      "  179  15648 492V 493A C   N    3788 3790 -113.23 -134.00   20.80    0.73  -62.50  178.33   31.92\n",
      "  179        493A 493A N   CA   3790 3791  148.14  147.00                  -40.90\n",
      "  180  15652 496L 497R C   N    3816 3818  -78.40  -63.00   83.08   10.27  -63.00   83.08   10.27\n",
      "  180        497R 497R N   CA   3818 3819   40.54  -41.10                  -41.10\n",
      "  181  15653 497R 498L C   N    3827 3829   32.03  -63.50   97.62   15.77  -63.50   97.62   15.77\n",
      "  181        498L 498L N   CA   3829 3830  -61.32  -41.20                  -41.20\n",
      "  182  15655 499R 500E C   N    3846 3848  169.04 -117.80   82.61    2.42  -63.60 -167.37   32.35\n",
      "  182        500E 500E N   CA   3848 3849  175.17  136.80                  -40.30\n",
      "  183  15656 500E 501L C   N    3855 3857 -153.56 -108.50   66.46    3.02  -63.50  164.33   28.32\n",
      "  183        501L 501L N   CA   3857 3858 -178.65  132.50                  -41.20\n",
      "  184  15658 502Q 503H C   N    3872 3874 -114.89 -125.60   81.71    3.30  -63.20  110.71   17.19\n",
      "  184        503H 503H N   CA   3874 3875 -140.20  138.80                  -42.30\n",
      "  185  15659 503H 504L C   N    3882 3884  -53.65  -70.70   28.93    2.80  -63.50  154.14   20.86\n",
      "  185        504L 504L N   CA   3884 3885  164.97  141.60                  -41.20\n",
      "  186  15661 505P 506D C   N    3897 3899  115.85   54.50   68.53    8.90  -63.30 -149.02   35.98\n",
      "  186        506D 506D N   CA   3899 3900   71.44   40.90                  -40.00\n",
      "  187  15662 506D 507Y C   N    3905 3907 -134.67  -63.50   93.33   17.67  -63.50   93.33   17.67\n",
      "  187        507Y 507Y N   CA   3907 3908 -103.79  -43.40                  -43.40\n",
      "  188  15670 514V 515T C   N    3960 3962  -81.66  -78.10    6.72    0.22  -63.20  163.44   22.19\n",
      "  188        515T 515T N   CA   3962 3963  155.50  149.80                  -42.10\n",
      "  189  15672 516V 517W C   N    3974 3976  -88.87  -71.30   19.27    1.58  -63.00  177.18   20.75\n",
      "  189        517W 517W N   CA   3976 3977  131.08  139.00                  -44.20\n",
      "  190  15673 517W 518F C   N    3988 3990 -156.95 -124.20   32.75    1.29  -63.20 -163.47   33.26\n",
      "  190        518F 518F N   CA   3990 3991  142.97  143.30                  -44.30\n",
      "  191  15674 518F 519E C   N    3999 4001  179.45 -117.80   71.78    2.10  -63.60 -171.34   31.38\n",
      "  191        519E 519E N   CA   4001 4002  171.66  136.80                  -40.30\n",
      "  192  15675 519E 520G C   N    4008 4010 -173.31 -167.20   62.74    2.52   82.20  147.18   11.30\n",
      "  192        520G 520G N   CA   4010 4011  112.16  174.60                    8.50\n",
      "  193  15676 520G 521H C   N    4012 4014 -120.07  -63.20   93.13   10.07  -63.20   93.13   10.07\n",
      "  193        521H 521H N   CA   4014 4015   31.45  -42.30                  -42.30\n",
      "  194  15677 521H 522Y C   N    4022 4024 -137.08 -124.30   31.58    1.26  -63.50  169.16   30.45\n",
      "  194        522Y 522Y N   CA   4024 4025  164.28  135.40                  -43.40\n",
      "  195  15678 522Y 523S C   N    4034 4036 -109.14 -136.60   36.98    1.17  -64.10  167.60   10.38\n",
      "  195        523S 523S N   CA   4036 4037  126.43  151.20                  -35.00\n",
      "  196  15679 523S 524S C   N    4040 4042 -176.08 -136.60   47.36    1.44  -64.10 -174.68   21.51\n",
      "  196        524S 524S N   CA   4042 4043  177.34  151.20                  -35.00\n",
      "  197  15680 524S 525A C   N    4046 4048  -68.31  -68.20    2.62    0.21  -62.50  171.28   28.36\n",
      "  197        525A 525A N   CA   4048 4049  147.92  145.30                  -40.90\n",
      "  198  15681 525A 526V C   N    4051 4053  -66.54  -62.40    4.66    0.80 -125.40 -178.06   10.17\n",
      "  198        526V 526V N   CA   4053 4054  -44.54  -42.40                  143.30\n",
      "  199  15682 526V 527Q C   N    4058 4060  -63.67  -73.00    9.62    0.64  -63.80  178.66   26.28\n",
      "  199        527Q 527Q N   CA   4060 4061  138.36  140.70                  -40.30\n",
      "  200  15683 527Q 528T C   N    4067 4069  -97.16  -78.10   40.47    1.32  -63.20  136.68   19.79\n",
      "  200        528T 528T N   CA   4069 4070 -174.50  149.80                  -42.10\n",
      "  201  15685 529P 530I C   N    4081 4083    1.39  -97.30   99.05    4.31 -120.60  122.53    8.13\n",
      "  201        530I 530I N   CA   4083 4084  118.87  127.20                  130.30\n",
      "  202  15689 533P 534N C   N    4114 4116 -172.82 -119.90   66.32    1.93  -63.20  179.34   28.34\n",
      "  202        534N 534N N   CA   4116 4117  176.96  137.00                  -41.10\n",
      "  203  15690 534N 535G C   N    4122 4124 -166.84 -167.20    1.83    0.07   82.20 -161.73   14.44\n",
      "  203        535G 535G N   CA   4124 4125  172.81  174.60                    8.50\n",
      "  204  15692 536G 537N C   N    4130 4132 -108.89 -119.90   13.17    0.69  -63.20 -179.46   25.17\n",
      "  204        537N 537N N   CA   4132 4133  144.24  137.00                  -41.10\n",
      "  205  15694 538P 539Q C   N    4145 4147 -176.85 -121.10   59.91    2.72  -63.80 -165.66   24.91\n",
      "  205        539Q 539Q N   CA   4147 4148  117.77  139.70                  -40.30\n",
      "  206  15695 539Q 540F C   N    4154 4156 -170.68 -124.20   56.71    1.47  -63.20  176.42   30.96\n",
      "  206        540F 540F N   CA   4156 4157  175.80  143.30                  -44.30\n",
      "  207  15696 540F 541S C   N    4165 4167  -82.42  -72.40   19.19    0.88  -64.10  157.31   12.58\n",
      "  207        541S 541S N   CA   4167 4168  168.76  152.40                  -35.00\n",
      "  208  15697 541S 542L C   N    4171 4173 -117.73 -108.50   24.32    1.20  -63.50  172.54   27.31\n",
      "  208        542L 542L N   CA   4173 4174  155.00  132.50                  -41.20\n",
      "  209  15698 542L 543R C   N    4179 4181 -123.10 -125.20   12.54    0.65  -63.00  176.49   27.98\n",
      "  209        543R 543R N   CA   4181 4182  152.96  140.60                  -41.10\n",
      "  210  15699 543R 544Q C   N    4190 4192  -53.79  -73.00   37.82    2.18  -63.80  148.76   22.40\n",
      "  210        544Q 544Q N   CA   4192 4193  108.12  140.70                  -40.30\n",
      "  211  15700 544Q 545M C   N    4199 4201  160.98 -125.60   74.19    2.67  -63.40 -143.83   42.34\n",
      "  211        545M 545M N   CA   4201 4202  151.16  140.50                  -40.50\n",
      "  212  15702 546Y 547L C   N    4219 4221  -72.62  -70.70    5.72    0.51  -63.50  177.65   24.16\n",
      "  212        547L 547L N   CA   4221 4222  136.21  141.60                  -41.20\n",
      "  213  15703 547L 548F C   N    4227 4229 -151.68 -124.20   28.69    0.88  -63.20 -173.52   31.53\n",
      "  213        548F 548F N   CA   4229 4230  151.54  143.30                  -44.30\n",
      "  214  15704 548F 549G C   N    4238 4240 -125.59 -167.20   62.67    1.50   82.20 -166.65   15.06\n",
      "  214        549G 549G N   CA   4240 4241  127.74  174.60                    8.50\n",
      "  215  15706 550P 551H C   N    4249 4251 -111.49 -125.60   14.94    0.42  -63.20 -177.31   20.30\n",
      "  215        551H 551H N   CA   4251 4252  133.89  138.80                  -42.30\n",
      "  216  15707 551H 552T C   N    4259 4261 -113.39 -124.80   13.47    0.82  -63.20  174.61   25.73\n",
      "  216        552T 552T N   CA   4261 4262  150.66  143.50                  -42.10\n",
      "  217  15708 552T 553E C   N    4266 4268  -74.68  -69.30   18.58    1.59  -63.60  165.39   21.46\n",
      "  217        553E 553E N   CA   4268 4269  124.72  142.50                  -40.30\n",
      "  218  15709 553E 554D C   N    4275 4277 -106.27  -70.90   35.39    2.71  -63.30  176.03   24.49\n",
      "  218        554D 554D N   CA   4277 4278  149.29  150.30                  -40.00\n",
      "  219  15710 554D 555L C   N    4283 4285 -115.37 -108.50   25.32    1.29  -63.50  170.03   26.81\n",
      "  219        555L 555L N   CA   4285 4286  156.88  132.50                  -41.20\n",
      "  220  15711 555L 556E C   N    4291 4293  -80.65  -69.30   12.22    1.06  -63.60  179.11   22.99\n",
      "  220        556E 556E N   CA   4293 4294  137.99  142.50                  -40.30\n",
      "  221  15712 556E 557R C   N    4300 4302  -76.71  -72.10    4.88    0.34  -63.00  175.94   24.70\n",
      "  221        557R 557R N   CA   4302 4303  143.50  141.90                  -41.10\n",
      "  222  15713 557R 558F C   N    4311 4313  -97.70  -71.40   34.96    3.04  -63.20  165.60   20.80\n",
      "  222        558F 558F N   CA   4313 4314  117.67  140.70                  -44.30\n",
      "  223  15714 558F 559F C   N    4322 4324  -79.68  -71.40   25.97    2.27  -63.20  161.24   21.30\n",
      "  223        559F 559F N   CA   4324 4325  116.09  140.70                  -44.30\n",
      "  224  15715 559F 560Q C   N    4333 4335 -151.47 -121.10   45.25    1.50  -63.80  170.69   29.58\n",
      "  224        560Q 560Q N   CA   4335 4336  173.24  139.70                  -40.30\n",
      "  225  15716 560Q 561L C   N    4342 4344  -68.65  -70.70    9.37    0.79  -63.50  168.13   23.64\n",
      "  225        561L 561L N   CA   4344 4345  150.74  141.60                  -41.20\n",
      "  226  15717 561L 562D C   N    4350 4352   66.83   54.50   61.18    5.50  -63.30 -168.26   31.64\n",
      "  226        562D 562D N   CA   4352 4353  100.82   40.90                  -40.00\n",
      "  227  15719 563V 564L C   N    4365 4367 -128.41 -108.50   38.20    1.81  -63.50  166.84   27.19\n",
      "  227        564L 564L N   CA   4367 4368  165.11  132.50                  -41.20\n",
      "  228  15720 564L 565N C   N    4373 4375 -117.37 -119.90   23.15    1.11  -63.20  167.87   24.07\n",
      "  228        565N 565N N   CA   4375 4376  160.01  137.00                  -41.10\n",
      "  229  15721 565N 566F C   N    4381 4383 -100.03 -124.20   24.17    0.93  -63.20  176.66   27.01\n",
      "  229        566F 566F N   CA   4383 4384  142.92  143.30                  -44.30\n",
      "  230  15724 568V 569E C   N    4410 4412  -60.61  -69.30   16.35    1.48  -63.60  163.37   21.62\n",
      "  230        569E 569E N   CA   4412 4413  156.35  142.50                  -40.30\n",
      "  231  15729 573S 574D C   N    4445 4447  -84.41  -70.90   22.38    0.80  -63.30  153.32   20.26\n",
      "  231        574D 574D N   CA   4447 4448  168.14  150.30                  -40.00\n",
      "  232  15730 574D 575L C   N    4453 4455 -125.27 -108.50   21.32    0.95  -63.50 -176.18   29.34\n",
      "  232        575L 575L N   CA   4455 4456  145.67  132.50                  -41.20\n",
      "  233  15731 575L 576T C   N    4461 4463 -107.91 -124.80   37.73    2.09  -63.20  147.60   21.90\n",
      "  233        576T 576T N   CA   4463 4464  177.24  143.50                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3   67  134  432  466  620  522  583  670  751  815\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00199.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      577     4476     4476\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00199.B99990001.pdb        4719.90283   -40490.28516        0.98442\n",
      "EAO00199.B99990002.pdb        5297.79346   -41533.27344        0.65562\n",
      "EAO00199.B99990003.pdb        6942.65918   -39699.69141        0.82421\n",
      "EAO00199.B99990004.pdb        7071.87842   -39459.01562        0.32852\n",
      "EAO00199.B99990005.pdb        5928.50488   -41837.73828        0.69106\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47628254   46511.969    45.422\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47628782   46512.484    45.422\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47677382   46559.945    45.469\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47690682   46572.930    45.481\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     47959948   46835.887    45.738\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     51521071   50313.547    49.134\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     51523313   50315.734    49.136\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     51524763   50317.152    49.138\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     51527663   50319.984    49.141\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     51533463   50325.648    49.146\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     64971862   63449.086    61.962\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     64983462   63460.414    61.973\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     65006662   63483.070    61.995\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     65053062   63528.383    62.039\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     65145862   63619.008    62.128\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     71865062   70180.727    68.536\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00200.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47962268   46838.152    45.740\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47963718   46839.570    45.742\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47966618   46842.398    45.745\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47972418   46848.062    45.750\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47984018   46859.391    45.761\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47986374   46861.695    45.763\n",
      "\n",
      "Read the alignment from file       : ali/EAO00200.ali\n",
      "\n",
      "Total number of alignment positions:   590\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00200     590      1    EAO00200\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     47989067   46864.324    45.766\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     48314242   47181.875    46.076\n",
      "profile_iteration_> processing sequence:       1    590      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     48315320   47182.930    46.077\n",
      "profile_iteration_> processing sequence:    6720    590      1     5.7500000     0.0008557     0.0000033  1169\n",
      "profile_iteration_> processing sequence:   13439    590      1    11.2968750     0.0008406     0.0000033  1190\n",
      "profile_iteration_> processing sequence:   20158    590      1    16.8281250     0.0008348     0.0000033  1198\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     48315695   47183.297    46.077\n",
      "profile_iteration_> processing sequence:   26877    590      1    22.4687500     0.0008360     0.0000033  1196\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     48315931   47183.527    46.078\n",
      "profile_iteration_> processing sequence:   33596    590      1    28.0781250     0.0008358     0.0000033  1197\n",
      "profile_iteration_> processing sequence:   40315    590      1    33.6093750     0.0008337     0.0000033  1200\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     48318881   47186.406    46.080\n",
      "profile_iteration_> processing sequence:   47034    590      1    39.2343750     0.0008342     0.0000033  1199\n",
      "profile_iteration_> processing sequence:   53753    590      1    44.9843750     0.0008369     0.0000033  1195\n",
      "profile_iteration_> processing sequence:   60472    590      1    50.5781250     0.0008364     0.0000033  1196\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     48322391   47189.836    46.084\n",
      "profile_iteration_> processing sequence:   67191    590      1    56.3906250     0.0008393     0.0000033  1192\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    45.12504\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    271.3778\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0219,      121,        0.0074,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00015   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00015   0.00004\n",
      "Z:         1  -2.35000   0.00000   0.00016\n",
      "Z:         1  -2.25000   0.00060   0.00055\n",
      "Z:         1  -2.15000   0.00119   0.00163\n",
      "Z:         1  -2.05000   0.00417   0.00416\n",
      "Z:         1  -1.95000   0.00670   0.00933\n",
      "Z:         1  -1.85000   0.01607   0.01872\n",
      "Z:         1  -1.75000   0.02768   0.03400\n",
      "Z:         1  -1.65000   0.05135   0.05658\n",
      "Z:         1  -1.55000   0.07739   0.08721\n",
      "Z:         1  -1.45000   0.12546   0.12565\n",
      "Z:         1  -1.35000   0.15954   0.17059\n",
      "Z:         1  -1.25000   0.22428   0.21981\n",
      "Z:         1  -1.15000   0.24750   0.27052\n",
      "Z:         1  -1.05000   0.35332   0.31973\n",
      "Z:         1  -0.95000   0.35466   0.36469\n",
      "Z:         1  -0.85000   0.41240   0.40316\n",
      "Z:         1  -0.75000   0.42148   0.43360\n",
      "Z:         1  -0.65000   0.45363   0.45519\n",
      "Z:         1  -0.55000   0.47565   0.46779\n",
      "Z:         1  -0.45000   0.52313   0.47182\n",
      "Z:         1  -0.35000   0.45526   0.46811\n",
      "Z:         1  -0.25000   0.45675   0.45775\n",
      "Z:         1  -0.15000   0.42669   0.44196\n",
      "Z:         1  -0.05000   0.41806   0.42194\n",
      "Z:         1   0.05000   0.41508   0.39889\n",
      "Z:         1   0.15000   0.36835   0.37383\n",
      "Z:         1   0.25000   0.35540   0.34769\n",
      "Z:         1   0.35000   0.30644   0.32121\n",
      "Z:         1   0.45000   0.30078   0.29500\n",
      "Z:         1   0.55000   0.24616   0.26953\n",
      "Z:         1   0.65000   0.24750   0.24513\n",
      "Z:         1   0.75000   0.20955   0.22205\n",
      "Z:         1   0.85000   0.19898   0.20043\n",
      "Z:         1   0.95000   0.17398   0.18036\n",
      "Z:         1   1.05000   0.15850   0.16185\n",
      "Z:         1   1.15000   0.13514   0.14489\n",
      "Z:         1   1.25000   0.13365   0.12944\n",
      "Z:         1   1.35000   0.11192   0.11542\n",
      "Z:         1   1.45000   0.09317   0.10275\n",
      "Z:         1   1.55000   0.08394   0.09133\n",
      "Z:         1   1.65000   0.08825   0.08109\n",
      "Z:         1   1.75000   0.07635   0.07191\n",
      "Z:         1   1.85000   0.06534   0.06371\n",
      "Z:         1   1.95000   0.05462   0.05639\n",
      "Z:         1   2.05000   0.05239   0.04988\n",
      "Z:         1   2.15000   0.04643   0.04409\n",
      "Z:         1   2.25000   0.04093   0.03895\n",
      "Z:         1   2.35000   0.03557   0.03439\n",
      "Z:         1   2.45000   0.03602   0.03035\n",
      "Z:         1   2.55000   0.03244   0.02678\n",
      "Z:         1   2.65000   0.02307   0.02361\n",
      "Z:         1   2.75000   0.02471   0.02082\n",
      "Z:         1   2.85000   0.01607   0.01835\n",
      "Z:         1   2.95000   0.01994   0.01617\n",
      "Z:         1   3.05000   0.01503   0.01424\n",
      "Z:         1   3.15000   0.01399   0.01255\n",
      "Z:         1   3.25000   0.01414   0.01105\n",
      "Z:         1   3.35000   0.01086   0.00973\n",
      "Z:         1   3.45000   0.01027   0.00857\n",
      "Z:         1   3.55000   0.00848   0.00754\n",
      "Z:         1   3.65000   0.00744   0.00664\n",
      "Z:         1   3.75000   0.00878   0.00584\n",
      "Z:         1   3.85000   0.00625   0.00514\n",
      "Z:         1   3.95000   0.00417   0.00453\n",
      "Z:         1   4.05000   0.00506   0.00398\n",
      "Z:         1   4.15000   0.00432   0.00350\n",
      "Z:         1   4.25000   0.00372   0.00308\n",
      "Z:         1   4.35000   0.00298   0.00271\n",
      "Z:         1   4.45000   0.00327   0.00239\n",
      "Z:         1   4.55000   0.00223   0.00210\n",
      "Z:         1   4.65000   0.00253   0.00185\n",
      "Z:         1   4.75000   0.00238   0.00163\n",
      "Z:         1   4.85000   0.00193   0.00143\n",
      "Z:         1   4.95000   0.00164   0.00126\n",
      "Z:         1   5.05000   0.00134   0.00111\n",
      "Z:         1   5.15000   0.00104   0.00097\n",
      "Z:         1   5.25000   0.00208   0.00086\n",
      "Z:         1   5.35000   0.00089   0.00075\n",
      "Z:         1   5.45000   0.00179   0.00066\n",
      "Z:         1   5.55000   0.00119   0.00058\n",
      "Z:         1   5.65000   0.00074   0.00051\n",
      "Z:         1   5.75000   0.00015   0.00045\n",
      "Z:         1   5.85000   0.00060   0.00040\n",
      "Z:         1   5.95000   0.00030   0.00035\n",
      "Z:         1   6.05000   0.00089   0.00031\n",
      "Z:         1   6.15000   0.00030   0.00027\n",
      "Z:         1   6.25000   0.00060   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00030   0.00016\n",
      "Z:         1   6.65000   0.00045   0.00014\n",
      "Z:         1   6.75000   0.00030   0.00013\n",
      "Z:         1   6.85000   0.00060   0.00011\n",
      "Z:         1   6.95000   0.00045   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00015   0.00007\n",
      "Z:         1   7.35000   0.00015   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00045   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00030   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     48811391   47667.375    46.550\n",
      "> 1x88A                      1    2754   21850     332     590   37.69     0.0           2   308    11   343     2   330\n",
      "> 4ubfA                      1    2828   13000     309     590   41.26     0.0           3   220    51   334    82   304\n",
      "> 5wdeA                      1    3567   20050     320     590   39.79     0.0           4   284    31   335    28   316\n",
      "> 5wdhA                      1    3577   15300     298     590   38.96     0.0           5   245    51   334    44   292\n",
      "> 3h4sA                      1    4201   20700     359     590   37.74     0.0           6   302    53   392    50   359\n",
      "> 1bg2A                      1    7654   25000     323     590   38.87     0.0           7   299    22   340    23   323\n",
      "> 3cnzA                      1    8394   18300     333     590   39.05     0.0           8   268    53   363    44   317\n",
      "> 4a5yB                      1    8685   20500     302     590   40.59     0.0           9   257    51   343    32   302\n",
      "> 4bn2A                      1    9693   18700     326     590   37.94     0.0          10   274    51   348    41   322\n",
      "> 4bn2C                      1    9694   18300     311     590   37.79     0.0          11   304    11   348     4   310\n",
      "> 3u06A                      1   10774   16100     359     590   33.67     0.0          12   290    11   334    58   351\n",
      "> 3u06B                      1   10775   13800     344     590   34.78     0.0          13   269    11   334    60   335\n",
      "> 4frzB                      1   11562   18750     349     590   38.49     0.0          14   280    53   376    58   348\n",
      "> 1n6mB                      1   13177   13900     335     590   34.06     0.0          15   267     9   334    56   331\n",
      "> 1mkjA                      1   13729   23200     336     590   38.85     0.0          16   294    22   347    24   319\n",
      "> 6a1zA                      1   14531   22700     352     590   38.26     0.0          17   322    10   369     2   346\n",
      "> 4a14A                      1   14535   19250     296     590   41.91     0.0          18   241    60   337    55   295\n",
      "> 3wpnA                      1   15809   18300     309     590   38.08     0.0          19   291    11   347     5   306\n",
      "> 4etpA                      1   16413   15650     379     590   32.93     0.0          20   307     3   349    49   379\n",
      "> 4ozqA                      1   19585   23650     683     590   43.64     0.0          21   287    23   337   392   682\n",
      "> 3cobA                      1   19976   22650     355     590   39.53     0.0          22   295    53   366    48   348\n",
      "> 3cobC                      1   19977   20550     340     590   40.14     0.0          23   278    53   363    48   331\n",
      "> 5mioC                      1   20804   17800     477     590   36.15     0.0          24   292    51   364    83   378\n",
      "> 1gojA                      1   21118   27000     354     590   38.62     0.0          25   335     5   358     1   347\n",
      "> 5x3eA                      1   22218   14350     375     590   29.03     0.0          26   312    11   340     4   375\n",
      "> 5x3eB                      1   22219   13050     358     590   29.78     0.0          27   302    11   340     3   358\n",
      "> 2y65A                      1   23876   21700     328     590   38.81     0.0          28   284    47   357    42   327\n",
      "> 2y65B                      1   23877   21750     341     590   37.67     0.0          29   298    47   369    41   340\n",
      "> 2hehA                      1   24256   14550     324     590   39.75     0.0          30   236    51   334    81   319\n",
      "> 1v8kA                      1   24955   15650     332     590   39.84     0.0          31   246    51   341    82   332\n",
      "> 1vfvA                      1   29856   20200     324     590   41.02     0.0          32   253    61   347    68   323\n",
      "> 5hleA                      1   30091   19150     303     590   38.28     0.0          33   253    51   334    47   302\n",
      "> 1cz7A                      1   32563   17150     342     590   32.90     0.0          34   298     9   334    28   334\n",
      "> 3gbjA                      1   33518   21000     302     590   40.86     0.0          35   283    10   337     1   301\n",
      "> 3gbjC                      1   33519   20600     294     590   40.96     0.0          36   278    10   337     1   293\n",
      "> 5w3dB                      1   34603   11500     313     590   35.97     0.0          37   247     9   334    56   308\n",
      "> 1f9tA                      1   35688   15350     304     590   39.60     0.0          38   237    51   334    52   301\n",
      "> 1f9vA                      1   35704   15150     313     590   38.08     0.0          39   249    51   350    52   311\n",
      "> 5zbrB                      1   38732   22050     330     590   39.69     0.0          40   299    10   348     2   321\n",
      "> 3zcwA                      1   39489   21350     321     590   38.24     0.0          41   300    11   343     2   320\n",
      "> 6bbnE                      1   40497   16700     378     590   39.23     0.0          42   256    51   334   114   373\n",
      "> 3b6uA                      1   42271   21400     323     590   38.94     0.0          43   301    11   348     3   323\n",
      "> 3b6uB                      1   42272   22350     343     590   38.51     0.0          44   319    11   356     7   341\n",
      "> 3b6vA                      1   42273   16900     306     590   38.87     0.0          45   259    42   356    42   306\n",
      "> 2owmA                      1   43026   22150     328     590   38.39     0.0          46   311     6   347     4   326\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 1t5cA                      1   49621   22150     322     590   38.87     0.0          47   311     9   348     2   320\n",
      "> 1t5cB                      1   49622   21750     310     590   39.29     0.0          48   306     9   348     1   308\n",
      "> 2zflA                      1   52454   19800     313     590   43.32     0.0          49   245    61   342    67   313\n",
      "> 3zfdA                      1   52460   24750     340     590   41.92     0.0          50   285    51   347    46   336\n",
      "> 2zfiA                      1   52487   22150     329     590   40.84     0.0          51   259    61   337    67   328\n",
      "> 3nwnA                      1   52817   16100     308     590   37.50     0.0          52   266    28   334    31   302\n",
      "> 4h1gA                      1   52945   15950     670     590   33.67     0.0          53   295     5   334   369   665\n",
      "> 3x2tA                      1   53687   20300     311     590   38.43     0.0          54   266    51   347    44   311\n",
      "> 3x2tB                      1   53688   21800     326     590   38.85     0.0          55   276    51   347    44   321\n",
      "> 1ry6A                      1   53816   13600     319     590   33.84     0.0          56   252    51   334    52   314\n",
      "> 3t0qA                      1   54261   14850     304     590   34.78     0.0          57   283     9   334     1   299\n",
      "> 2gryA                      1   54590   16050     327     590   40.00     0.0          58   247    51   334    75   324\n",
      "> 2vvgA                      1   54932   19050     308     590   40.82     0.0          59   258    51   355    42   308\n",
      "> 3lreA                      1   55211   18000     289     590   40.28     0.0          60   279    11   337     1   288\n",
      "> 3lreB                      1   55212   18850     300     590   40.27     0.0          61   289    11   340     3   300\n",
      "> 6njeA                      1   57846   15100     297     590   31.85     0.0          62   292     6   340     5   296\n",
      "> 1ii6A                      1   58279   22850     340     590   37.98     0.0          63   316    11   346     4   340\n",
      "> 3kinB                      1   61261    6550     117     590   33.63    0.25E-05      64   102   267   370     1   113\n",
      "> 2kinA                      1   61324   16100     238     590   38.54     0.0          65   190    51   249    46   237\n",
      "> 2kinB                      1   61325    6650     100     590   38.55    0.13E-05      66    83   263   347     1    83\n",
      "> 4gkrA                      1   61877   14900     296     590   35.86     0.0          67   277    11   334     2   291\n",
      "> 4gkrB                      1   61878   16100     307     590   36.84     0.0          68   292     9   350     1   304\n",
      "> 6a20A                      1   62255   24550     395     590   37.80     0.0          69   358    10   398     4   384\n",
      "> 3dc4A                      1   63280   13700     291     590   37.01     0.0          70   275    11   339     6   286\n",
      "> 5lt0A                      1   63785   22500     303     590   38.95     0.0          71   283    22   340    19   303\n",
      "> 5lt1A                      1   63786   23100     309     590   38.75     0.0          72   287    22   337    19   307\n",
      "> 5gszA                      1   66643   17800     306     590   35.15     0.0          73   290     7   334    12   304\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    56.406250\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000839\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1191.875854\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.171875\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000843\n",
      "profile_iteration__> Overall performance [seq/sec]:  1185.959229\n",
      "openf___224_> Open           buildprofile/EAO00200build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     48919345   47772.797    46.653\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48921701   47775.098    46.655\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48922929   47776.297    46.657\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48923805   47777.152    46.657\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48924937   47778.258    46.658\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48925913   47779.211    46.659\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48927117   47780.387    46.661\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48928309   47781.551    46.662\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48929377   47782.594    46.663\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48930401   47783.594    46.664\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48931493   47784.660    46.665\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48932705   47785.844    46.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48933861   47786.973    46.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48934933   47788.020    46.668\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48936049   47789.109    46.669\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48937113   47790.148    46.670\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48938285   47791.293    46.671\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48939569   47792.547    46.672\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48940529   47793.484    46.673\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48941689   47794.617    46.674\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48942913   47795.812    46.676\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48944057   47796.930    46.677\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48945233   47798.078    46.678\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48946341   47799.160    46.679\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48947505   47800.297    46.680\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48948841   47801.602    46.681\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48950085   47802.816    46.682\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48951289   47803.992    46.684\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48952421   47805.098    46.685\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48953609   47806.258    46.686\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48954549   47807.176    46.687\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48955529   47808.133    46.688\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48956537   47809.117    46.689\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48957545   47810.102    46.690\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48958733   47811.262    46.691\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48959861   47812.363    46.692\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48960969   47813.445    46.693\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48961953   47814.406    46.694\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48962897   47815.328    46.695\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48963889   47816.297    46.696\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48965081   47817.461    46.697\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48966277   47818.629    46.698\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48967297   47819.625    46.699\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48968497   47820.797    46.700\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48969769   47822.039    46.701\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48970801   47823.047    46.702\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48972041   47824.258    46.703\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48973281   47825.469    46.705\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48974501   47826.660    46.706\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48975477   47827.613    46.707\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48976613   47828.723    46.708\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48977645   47829.730    46.709\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48978705   47830.766    46.710\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48979881   47831.914    46.711\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48980941   47832.949    46.712\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48982041   47834.023    46.713\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48983045   47835.004    46.714\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48984173   47836.105    46.715\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48985157   47837.066    46.716\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48986185   47838.070    46.717\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48987297   47839.156    46.718\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48988449   47840.281    46.719\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48989613   47841.418    46.720\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48990873   47842.648    46.721\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48991277   47843.043    46.722\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48992033   47843.781    46.722\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48992361   47844.102    46.723\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48993465   47845.180    46.724\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48994629   47846.316    46.725\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48996057   47847.711    46.726\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48997153   47848.781    46.727\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48998281   47849.883    46.728\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48999425   47851.000    46.729\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     49000581   47852.129    46.731\n",
      "openf___224_> Open           buildprofile/EAO00200build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47628254   46511.969    45.422\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47628782   46512.484    45.422\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47677382   46559.945    45.469\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47690682   46572.930    45.481\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           2zfl.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47702892   46584.855    45.493\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47702968   46584.930    45.493\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47703139   46585.098    45.493\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47703989   46585.926    45.494\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47705247   46587.156    45.495\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47707151   46589.016    45.497\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47707191   46589.055    45.497\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47707281   46589.141    45.497\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47710137   46591.930    45.500\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47710197   46591.988    45.500\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47710332   46592.121    45.500\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47714616   46596.305    45.504\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47714704   46596.391    45.504\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47714902   46596.586    45.504\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47721328   46602.859    45.511\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47721460   46602.988    45.511\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47721757   46603.277    45.511\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47731379   46612.676    45.520\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47731579   46612.871    45.520\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47732029   46613.309    45.521\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47746479   46627.422    45.535\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47746779   46627.715    45.535\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47747454   46628.375    45.536\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47769112   46649.523    45.556\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47769560   46649.961    45.557\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47770568   46650.945    45.558\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     47803072   46682.688    45.589\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     48034944   46909.125    45.810\n",
      "read_mo_297_> Segments, residues, atoms:        1      313     2467\n",
      "read_mo_298_> Segment:        1  4:A  356:A     2467\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     48043992   46917.961    45.818\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     48225851   47095.559    45.992\n",
      "openf___224_> Open           ali/EAO00200.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     48330786   47198.031    46.092\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     48427190   47292.180    46.184\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48429546   47294.477    46.186\n",
      "\n",
      "Read the alignment from file       : ali/EAO00200.ali\n",
      "\n",
      "Total number of alignment positions:   590\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      2zflA     313      1    2zfl.pdb MOL_ID  1; MOLECULE  KINESIN-LIKE PROTEIN KIF1A, KINESIN HEAVY CHAIN I CHAIN  A;\n",
      "  2   EAO00200     590      1    EAO00200\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          603\n",
      "  Score                    :  176280.3594\n",
      "openf___224_> Open           ali/EAO00200-2zflA.ali\n",
      "openf___224_> Open           ali/EAO00200-2zflA.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47628254   46511.969    45.422\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47628782   46512.484    45.422\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47677382   46559.945    45.469\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     47690682   46572.930    45.481\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00200-2zflA.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47702647   46584.617    45.493\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47704097   46586.031    45.494\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47706997   46588.863    45.497\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47712797   46594.527    45.502\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47724397   46605.855    45.514\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     47965205   46841.020    45.743\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47966453   46842.238    45.744\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     47968809   46844.539    45.747\n",
      "\n",
      "Read the alignment from file       : ali/EAO00200-2zflA.ali\n",
      "\n",
      "Total number of alignment positions:   603\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      2zflA     313      1    2zfl.pdb MOL_ID  1; MOLECULE  KINESIN-LIKE PROTEIN KIF1A, KINESIN HEAVY CHAIN I CHAIN  A;\n",
      "  2   EAO00200     590      1    EAO00200\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           2zfl.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48064434   46937.922    45.838\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48064434   46937.922    45.838\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48064605   46938.090    45.838\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48065455   46938.922    45.839\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48066713   46940.148    45.840\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48068617   46942.008    45.842\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48068617   46942.008    45.842\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48068707   46942.098    45.842\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48071563   46944.887    45.845\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48071563   46944.887    45.845\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48071698   46945.016    45.845\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48075982   46949.203    45.849\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48075982   46949.203    45.849\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48076180   46949.395    45.849\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48082606   46955.672    45.855\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48082606   46955.672    45.855\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48082903   46955.961    45.855\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48092525   46965.355    45.865\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48092525   46965.355    45.865\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48092975   46965.797    45.865\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48107425   46979.906    45.879\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48107425   46979.906    45.879\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48108100   46980.566    45.879\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48129758   47001.719    45.900\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     48129758   47001.719    45.900\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48130766   47002.703    45.901\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     48163270   47034.445    45.932\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     48163054   47034.234    45.932\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    266     1  204   209      T     E   12.229\n",
      "    354     1  253   270      T     N   10.900\n",
      "    386     1  286   303      A     L   12.272\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48302306   47170.219    46.065\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48303906   47171.781    46.066\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48307886   47175.672    46.070\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48309886   47177.625    46.072\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48312886   47180.555    46.075\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48317386   47184.945    46.079\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48322958   47190.391    46.084\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48325758   47193.125    46.087\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48329958   47197.227    46.091\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48336258   47203.375    46.097\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48339030   47206.086    46.100\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     48340430   47207.453    46.101\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     48432903   47297.758    46.189\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     48889289   47743.445    46.624\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     48957465   47810.023    46.689\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      590   4638   4710      0         0      1906\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     49151141   47999.160    46.874\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      590   4638   4710   6334      7356      1906\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     49341997   48185.543    47.056\n",
      "patch_______> segment topology patched using RTF:  590 ;  ALA  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      590   4639   4711   6336      7358      1907\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      590   4639   4711   6336      7358      1907\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1914     2725\n",
      "openf___224_> Open           EAO00200.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      590     4639     4639\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49449446   48290.477    47.159\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49457638   48298.477    47.166\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49474022   48314.477    47.182\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49506790   48346.477    47.213\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49539558   48378.477    47.245\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49605094   48442.477    47.307\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49637862   48474.477    47.338\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49703398   48538.477    47.401\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49736166   48570.477    47.432\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49867238   48698.477    47.557\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49932774   48762.477    47.620\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     49998310   48826.477    47.682\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     50129382   48954.477    47.807\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     50391526   49210.477    48.057\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     50522598   49338.477    48.182\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     50653670   49466.477    48.307\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     50915814   49722.477    48.557\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     51440102   50234.477    49.057\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     4711    6336    6872    1896\n",
      "              Total number of restraints before, now         :        0    19815\n",
      "make_re_422_> Number of previous, current restraints         :        0    19815\n",
      "make_re_423_> Number of previous, current selected restraints:        0    19815\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     51779173   50565.598    49.380\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      198  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      201  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      208  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      222  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      226  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      198  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      201  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      208  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      222  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      226  ASN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      198  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      208  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      208  ARG\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     52041333   50821.613    49.630\n",
      "make_re_422_> Number of previous, current restraints         :    19815    20403\n",
      "make_re_423_> Number of previous, current selected restraints:    19815    20403\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     52041333   50821.613    49.630\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      600\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    20403    20992\n",
      "make_re_423_> Number of previous, current selected restraints:    20403    20392\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     52041333   50821.613    49.630\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     52303477   51077.613    49.880\n",
      "make_re_422_> Number of previous, current restraints         :    20992    21491\n",
      "make_re_423_> Number of previous, current selected restraints:    20392    20891\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     52303477   51077.613    49.880\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    21491    21862\n",
      "make_re_423_> Number of previous, current selected restraints:    20891    21262\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     52303477   51077.613    49.880\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    21862    22041\n",
      "make_re_423_> Number of previous, current selected restraints:    21262    21441\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     52303477   51077.613    49.880\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  2zflA\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    22041    22118\n",
      "make_re_423_> Number of previous, current selected restraints:    21441    21518\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     53639473   52382.297    51.155\n",
      "make_re_422_> Number of previous, current restraints         :    22118    28369\n",
      "make_re_423_> Number of previous, current selected restraints:    21518    27769\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     54163761   52894.297    51.655\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     54688049   53406.297    52.155\n",
      "make_re_422_> Number of previous, current restraints         :    28369    34746\n",
      "make_re_423_> Number of previous, current selected restraints:    27769    34146\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     55736625   54430.297    53.155\n",
      "make_re_422_> Number of previous, current restraints         :    34746    37464\n",
      "make_re_423_> Number of previous, current selected restraints:    34146    36864\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "make_re_422_> Number of previous, current restraints         :    37464    38393\n",
      "make_re_423_> Number of previous, current selected restraints:    36864    37793\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    38393    38393\n",
      "make_re_423_> Number of previous, current selected restraints:    37793    37793\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    38393    38393\n",
      "make_re_423_> Number of previous, current selected restraints:    37793    37793\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    38393    38393\n",
      "make_re_423_> Number of previous, current selected restraints:    37793    37793\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    38393    38393\n",
      "make_re_423_> Number of previous, current selected restraints:    37793    37793\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    38393    38393\n",
      "make_re_423_> Number of previous, current selected restraints:    37793    37793\n",
      "rmdupl__427_>     3521 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38393    34272\n",
      "openf___224_> Open           EAO00200.rsr\n",
      "openf___224_> Open           EAO00200.rsr\n",
      "rdcsr2__307_> Number of restraints read    :    34272\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    34272    34272\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00200.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     54933780   53646.270    52.389\n",
      "read_mo_297_> Segments, residues, atoms:        1      590     4639\n",
      "read_mo_298_> Segment:        1  1:A  590:A     4639\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4639     4639        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    21546\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    22778\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    23393\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    24312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    25407\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    26173\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    27672\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    28645\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    29568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    31057\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     55337688   54040.711    52.774\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     55449024   54149.438    52.880\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    32296\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34090\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34272\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.8087        3  3\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4497       10  10\n",
      "iupac_m_487_> NH1/2 swapped:      161.1426       16  16\n",
      "iupac_m_487_> NH1/2 swapped:      179.6004       21  21\n",
      "iupac_m_485_> OD1/2 will be swapped:     -178.4418       31  31\n",
      "iupac_m_487_> NH1/2 swapped:      179.6993       36  36\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6034       46  46\n",
      "iupac_m_485_> OD1/2 will be swapped:      162.3302       54  54\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3341       55  55\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.3368       62  62\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.6838       87  87\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -168.9672      101  101\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      105.5668      114  114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.2165      120  120\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.9938      121  121\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8155      125  125\n",
      "iupac_m_486_> OE1/2 will be swapped:       97.1659      127  127\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.8996      129  129\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8477      155  155\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2467      157  157\n",
      "iupac_m_486_> OE1/2 will be swapped:     -168.4336      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.7319      162  162\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.7696      165  165\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9788      173  173\n",
      "iupac_m_487_> NH1/2 swapped:      179.8830      233  233\n",
      "iupac_m_485_> OD1/2 will be swapped:      163.8433      243  243\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9834      249  249\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.4121      261  261\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.2062      262  262\n",
      "iupac_m_487_> NH1/2 swapped:      179.6495      275  275\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0105      283  283\n",
      "iupac_m_485_> OD1/2 will be swapped:     -177.8943      287  287\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      109.7155      289  289\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -153.2637      366  366\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7086      368  368\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.2025      370  370\n",
      "iupac_m_486_> OE1/2 will be swapped:      138.8837      375  375\n",
      "iupac_m_486_> OE1/2 will be swapped:     -140.2502      381  381\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8198      385  385\n",
      "iupac_m_486_> OE1/2 will be swapped:      151.6504      396  396\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      138.1954      399  399\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -171.8572      401  401\n",
      "iupac_m_485_> OD1/2 will be swapped:     -164.1157      405  405\n",
      "iupac_m_487_> NH1/2 swapped:      176.7174      418  418\n",
      "iupac_m_487_> NH1/2 swapped:      179.3082      425  425\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8134      426  426\n",
      "iupac_m_485_> OD1/2 will be swapped:      -91.1400      440  440\n",
      "iupac_m_487_> NH1/2 swapped:     -176.8671      451  451\n",
      "iupac_m_486_> OE1/2 will be swapped:       99.4774      457  457\n",
      "iupac_m_487_> NH1/2 swapped:     -176.5038      464  464\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.9729      480  480\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.6163      496  496\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8123      497  497\n",
      "iupac_m_485_> OD1/2 will be swapped:      116.2668      537  537\n",
      "iupac_m_487_> NH1/2 swapped:      174.1221      538  538\n",
      "iupac_m_486_> OE1/2 will be swapped:     -118.7767      540  540\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.4593      542  542\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.9421      549  549\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.9666      563  563\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.4637      577  577\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.3488      584  584\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     55964468   54652.801    53.372\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     56409812   55087.707    53.797\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     57300500   55957.520    54.646\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     59081876   57697.145    56.345\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     62644628   61176.395    59.743\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     69770132   68134.891    66.538\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):  1050575\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -41066.5742\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -41066.574219\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    31.629000\n",
      "Sequence length          :          590\n",
      "Compactness              :     0.188367\n",
      "Native energy (pair)     :   901.278335\n",
      "Native energy (surface)  :    97.179159\n",
      "Native energy (combined) :    25.386992\n",
      "Z score (pair)           :    -1.764734\n",
      "Z score (surface)        :    -1.177886\n",
      "Z score (combined)       :    -2.255497\n",
      "GA341 score              :     0.217251\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    12613\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        7219.5117\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4711       0      0   0.007   0.007      61.604       1.000\n",
      " 2 Bond angle potential               :    6336       3     43   2.378   2.378      680.85       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2751       0    160  53.126  53.126      1258.8       1.000\n",
      " 4 Stereochemical improper torsion pot:    1896       2      3   1.811   1.811      122.17       1.000\n",
      " 5 Soft-sphere overlap restraints     :   12613       0      4   0.008   0.008      93.896       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6251      24    135   1.349   1.349      1335.4       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6377      23    183   1.602   1.602      1496.2       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     589       3     15   8.697   8.697      186.30       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     499       0      9  87.004  87.004      227.45       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     371       0      6  81.567  81.567      170.92       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     179       0      0  98.524  98.524      140.07       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      77       0      0  92.427  92.427      52.171       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2718       0      0   0.725   0.725      76.713       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     588     271    136  44.017 117.399      1043.7       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     929       1     43   3.002   3.002      273.28       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00200.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00200.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  106904.0234\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10425 538R 538R N   CA   4194 4195  124.94  107.00   17.94    5.16  107.00   17.94    5.16\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19009  38V 537D CA  CA    276 4187   18.61   13.45    5.15    4.63   13.45    5.15    4.63\n",
      "    2  19147  49T 537D CA  CA    351 4187   19.26   12.38    6.88    5.93   12.38    6.88    5.93\n",
      "    3  19163  50M 537D CA  CA    358 4187   16.00   10.61    5.39    5.35   10.61    5.39    5.35\n",
      "    4  19164  50M 538R CA  CA    358 4195   19.61   14.00    5.61    5.03   14.00    5.61    5.03\n",
      "    5  19180  51F 537D CA  CA    366 4187   12.59    7.18    5.40    7.15    7.18    5.40    7.15\n",
      "    6  19181  51F 538R CA  CA    366 4195   16.20   10.77    5.43    6.07   10.77    5.43    6.07\n",
      "    7  20127  83C 458S CA  CA    629 3561    8.43    5.99    2.44    7.57    5.99    2.44    7.57\n",
      "    8  20177  84L 458S CA  CA    635 3561   11.59    9.00    2.59    5.97    9.00    2.59    5.97\n",
      "    9  20242  85F 458S CA  CA    643 3561   13.32    9.83    3.48    6.72    9.83    3.48    6.72\n",
      "   10  20295  86A 458S CA  CA    654 3561   16.76   13.45    3.31    5.57   13.45    3.31    5.57\n",
      "   11  21977 146E 410L CA  CA   1116 3178   10.77    7.43    3.34    5.36    7.43    3.34    5.36\n",
      "   12  23023 287D 458S CA  CA   2208 3561   16.90   12.20    4.70    6.89   12.20    4.70    6.89\n",
      "   13  23040 288A 458S CA  CA   2216 3561   17.32   13.43    3.89    5.36   13.43    3.89    5.36\n",
      "   14  23239 332E 458S CA  CA   2545 3561   11.33    8.18    3.15    5.08    8.18    3.15    5.08\n",
      "   15  23631 359R 538R CA  CA   2761 4195   19.42    9.45    9.97    6.45    9.45    9.97    6.45\n",
      "   16  23658 360E 538R CA  CA   2772 4195   22.15   10.89   11.26    4.97   10.89   11.26    4.97\n",
      "   17  24209 509R 538R CA  CA   3969 4195   17.32    6.39   10.93    4.85    6.39   10.93    4.85\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  25071  51F 537D N   O     365 4193   15.52   10.14    5.38    5.29   10.14    5.38    5.29\n",
      "    2  29187 359R 538R N   O    2760 4204   19.36   10.91    8.45    4.86   10.91    8.45    4.86\n",
      "    3  29347 397V 360E N   O    3077 2779   17.26    7.05   10.21    4.72    7.05   10.21    4.72\n",
      "    4  29353 397V 374L N   O    3077 2881   18.91    2.26   16.65    5.53    2.26   16.65    5.53\n",
      "    5  29363 398A 360E N   O    3084 2779   13.82    6.19    7.63    4.64    6.19    7.63    4.64\n",
      "    6  29484 410L 146E N   O    3177 1123   10.90    6.64    4.26    6.36    6.64    4.26    6.36\n",
      "    7  29656 458S  84L N   O    3560  641   13.46   10.73    2.73    4.91   10.73    2.73    4.91\n",
      "    8  30271 506K 475S N   O    3945 3705   14.37   10.31    4.06    4.78   10.31    4.06    4.78\n",
      "    9  30272 506K 476G N   O    3945 3709   16.05   10.57    5.48    5.60   10.57    5.48    5.60\n",
      "   10  30273 506K 477V N   O    3945 3716   16.86   10.89    5.97    5.68   10.89    5.97    5.68\n",
      "   11  30274 506K 478L N   O    3945 3724   12.59    6.32    6.26    5.33    6.32    6.26    5.33\n",
      "   12  30275 506K 479T N   O    3945 3731   11.54    4.80    6.74    4.50    4.80    6.74    4.50\n",
      "   13  30276 506K 480E N   O    3945 3740   11.30    2.71    8.60    4.68    2.71    8.60    4.68\n",
      "   14  30419 537D  50M N   O    4186  364   15.55    9.63    5.91    5.64    9.63    5.91    5.64\n",
      "   15  30424 537D  87Y N   O    4186  669   14.08   10.32    3.76    5.19   10.32    3.76    5.19\n",
      "   16  30425 537D 359R N   O    4186 2770   20.49   10.84    9.65    5.36   10.84    9.65    5.36\n",
      "   17  30435 537D 508L N   O    4186 3967   14.16    2.68   11.47    5.67    2.68   11.47    5.67\n",
      "   18  30436 537D 509R N   O    4186 3978   14.21    3.33   10.88    4.94    3.33   10.88    4.94\n",
      "   19  30450 538R 359R N   O    4194 2770   19.72    9.13   10.59    6.44    9.13   10.59    6.44\n",
      "   20  30459 538R 508L N   O    4194 3967   15.16    4.90   10.26    4.82    4.90   10.26    4.82\n",
      "   21  30460 538R 509R N   O    4194 3978   14.79    3.14   11.65    5.43    3.14   11.65    5.43\n",
      "   22  30546 571V 538R N   O    4474 4204   12.73    2.67   10.05    4.52    2.67   10.05    4.52\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16740 458S 458S CA  C    3561 3564   -9.32 -179.18  169.86   39.48 -179.18  169.86   39.48\n",
      "    2  16788 506K 506K CA  C    3946 3952 -134.67 -180.00   45.33    9.06 -180.00   45.33    9.06\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15695   1M   2A C   N       7    9  -61.79  -62.50    2.97    0.46 -134.00 -176.02   11.35\n",
      "    1          2A   2A N   CA      9   10  -43.78  -40.90                  147.00\n",
      "    2  15696   2A   3E C   N      12   14   79.24  -69.30  155.14   10.14  -63.60 -161.36   33.74\n",
      "    2          3E   3E N   CA     14   15   97.73  142.50                  -40.30\n",
      "    3  15698   4L   5A C   N      29   31  -79.60  -68.20   32.87    2.26  -62.50  143.99   24.47\n",
      "    3          5A   5A N   CA     31   32  176.13  145.30                  -40.90\n",
      "    4  15699   5A   6G C   N      34   36   81.55   78.70    6.70    0.10   82.20  179.33    8.67\n",
      "    4          6G   6G N   CA     36   37 -172.17 -166.10                    8.50\n",
      "    5  15700   6G   7T C   N      38   40  -63.06  -78.10   45.01    2.43  -63.20  125.67   16.11\n",
      "    5          7T   7T N   CA     40   41 -167.77  149.80                  -42.10\n",
      "    6  15703   9G  10R C   N      54   56 -162.84 -125.20   38.91    1.32  -72.10   91.14    6.77\n",
      "    6         10R  10R N   CA     56   57  150.50  140.60                  141.90\n",
      "    7  15710  16R  17M C   N     111  113  -78.90  -73.00   23.77    1.52  -63.40  154.25   24.26\n",
      "    7         17M  17M N   CA    113  114  166.03  143.00                  -40.50\n",
      "    8  15737  43E  44G C   N     318  320   56.95   78.70   81.83    1.49  -62.40  127.91   20.55\n",
      "    8         44G  44G N   CA    320  321  -87.22 -166.10                  -41.20\n",
      "    9  15752  58T  59G C   N     438  440 -138.33 -167.20   31.43    0.93   82.20 -133.46   16.87\n",
      "    9         59G  59G N   CA    440  441 -172.98  174.60                    8.50\n",
      "   10  15797 103N 104S C   N     781  783  -62.12  -72.40   43.90    2.25  -64.10  144.74   10.61\n",
      "   10        104S 104S N   CA    783  784  109.72  152.40                  -35.00\n",
      "   11  15798 104S 105K C   N     787  789  163.30 -118.00   96.85    3.26   56.60 -170.20   21.54\n",
      "   11        105K 105K N   CA    789  790 -164.44  139.10                   38.60\n",
      "   12  15800 106E 107G C   N     805  807   58.45   78.70   66.89    1.15  -62.40  135.44   21.04\n",
      "   12        107G 107G N   CA    807  808 -102.35 -166.10                  -41.20\n",
      "   13  15813 119I 120E C   N     904  906  -60.83  -69.30   36.62    3.07  -63.60  141.60   18.73\n",
      "   13        120E 120E N   CA    906  907  178.13  142.50                  -40.30\n",
      "   14  15838 144Y 145M C   N    1105 1107  -51.85  -63.40   11.58    2.08   56.30  132.32   23.71\n",
      "   14        145M 145M N   CA   1107 1108  -41.44  -40.50                   34.80\n",
      "   15  15839 145M 146E C   N    1113 1115 -159.23  -63.60  101.70   14.27   54.60  153.89   14.93\n",
      "   15        146E 146E N   CA   1115 1116   -5.70  -40.30                   42.40\n",
      "   16  15849 155R 156A C   N    1196 1198 -111.95 -134.00   41.02    1.57  -62.50  161.09   24.50\n",
      "   16        156A 156A N   CA   1198 1199  112.41  147.00                  -40.90\n",
      "   17  15850 156A 157R C   N    1201 1203   44.88  -72.10  123.75   10.26 -125.20  175.11    8.09\n",
      "   17        157R 157R N   CA   1203 1204 -177.72  141.90                  140.60\n",
      "   18  15852 158G 159Q C   N    1216 1218 -104.16 -121.10   29.96    1.07  -63.80  160.45   21.69\n",
      "   18        159Q 159Q N   CA   1218 1219  114.98  139.70                  -40.30\n",
      "   19  15853 159Q 160E C   N    1225 1227  172.24 -117.80   95.65    3.01  -63.60  171.06   29.09\n",
      "   19        160E 160E N   CA   1227 1228 -157.97  136.80                  -40.30\n",
      "   20  15855 161P 162E C   N    1241 1243  -46.95  -69.30   27.32    1.56  -63.60  167.92   23.32\n",
      "   20        162E 162E N   CA   1243 1244  126.79  142.50                  -40.30\n",
      "   21  15856 162E 163S C   N    1250 1252 -152.66 -136.60   53.36    2.32  -64.10  151.49   17.37\n",
      "   21        163S 163S N   CA   1252 1253 -157.92  151.20                  -35.00\n",
      "   22  15859 165E 166I C   N    1273 1275  -19.61  -97.30   80.52    3.68 -120.60  102.60    7.64\n",
      "   22        166I 166I N   CA   1275 1276  148.36  127.20                  130.30\n",
      "   23  15862 168E 169D C   N    1300 1302  -57.82  -63.30   12.89    1.37  -70.90  158.58    8.47\n",
      "   23        169D 169D N   CA   1302 1303  -51.66  -40.00                  150.30\n",
      "   24  15863 169D 170S C   N    1308 1310 -145.23 -136.60   13.37    0.75  -64.10 -166.21   12.18\n",
      "   24        170S 170S N   CA   1310 1311  140.99  151.20                  -35.00\n",
      "   25  15864 170S 171Q C   N    1314 1316 -122.55 -121.10   21.36    1.04  -63.80  169.22   22.29\n",
      "   25        171Q 171Q N   CA   1316 1317  118.39  139.70                  -40.30\n",
      "   26  15865 171Q 172H C   N    1323 1325 -163.75 -125.60   50.86    1.25  -63.20  176.67   28.47\n",
      "   26        172H 172H N   CA   1325 1326  172.44  138.80                  -42.30\n",
      "   27  15866 172H 173R C   N    1333 1335  -93.82  -72.10   63.57    5.31  -63.00  127.04   15.33\n",
      "   27        173R 173R N   CA   1335 1336   82.15  141.90                  -41.10\n",
      "   28  15868 174V 175Y C   N    1351 1353 -117.02 -124.30    8.48    0.46  -63.50 -177.51   26.45\n",
      "   28        175Y 175Y N   CA   1353 1354  131.06  135.40                  -43.40\n",
      "   29  15870 176V 177K C   N    1370 1372 -121.81 -118.00   18.80    0.85  -62.90  172.08   25.86\n",
      "   29        177K 177K N   CA   1372 1373  157.52  139.10                  -40.80\n",
      "   30  15871 177K 178G C   N    1379 1381 -102.74  -80.20   57.56    3.37   82.20 -151.84   16.27\n",
      "   30        178G 178G N   CA   1381 1382  121.14  174.10                    8.50\n",
      "   31  15872 178G 179A C   N    1383 1385 -153.54 -134.00   20.29    0.50  -62.50 -170.13   35.98\n",
      "   31        179A 179A N   CA   1385 1386  152.48  147.00                  -40.90\n",
      "   32  15873 179A 180G C   N    1388 1390   66.79   78.70   12.08    0.56   82.20  173.32    9.18\n",
      "   32        180G 180G N   CA   1390 1391 -164.13 -166.10                    8.50\n",
      "   33  15875 181V 182H C   N    1399 1401 -148.67 -125.60   30.39    0.75  -63.20 -179.38   28.12\n",
      "   33        182H 182H N   CA   1401 1402  158.58  138.80                  -42.30\n",
      "   34  15876 182H 183S C   N    1409 1411  -74.66  -72.40    8.34    0.57  -64.10  179.68   12.43\n",
      "   34        183S 183S N   CA   1411 1412  144.37  152.40                  -35.00\n",
      "   35  15879 185L 186S C   N    1430 1432 -129.43 -136.60    9.67    0.31  -64.10 -168.79   11.79\n",
      "   35        186S 186S N   CA   1432 1433  144.70  151.20                  -35.00\n",
      "   36  15881 187L 188E C   N    1444 1446 -142.47 -117.80   40.13    1.39  -63.60  170.58   27.13\n",
      "   36        188E 188E N   CA   1446 1447  168.45  136.80                  -40.30\n",
      "   37  15891 197G 198N C   N    1527 1529 -117.41 -119.90    2.50    0.11  -63.20 -173.56   20.81\n",
      "   37        198N 198N N   CA   1529 1530  137.29  137.00                  -41.10\n",
      "   38  15892 198N 199A C   N    1535 1537 -136.40 -134.00    8.39    0.41  -62.50  179.94   33.45\n",
      "   38        199A 199A N   CA   1537 1538  155.04  147.00                  -40.90\n",
      "   39  15893 199A 200N C   N    1540 1542  -86.98  -71.20   39.18    2.93  -63.20  149.93   17.54\n",
      "   39        200N 200N N   CA   1542 1543  106.94  142.80                  -41.10\n",
      "   40  15894 200N 201R C   N    1548 1550 -150.52 -125.20   39.41    1.25  -63.00  172.03   29.11\n",
      "   40        201R 201R N   CA   1550 1551  170.80  140.60                  -41.10\n",
      "   41  15895 201R 202Q C   N    1559 1561  -71.62  -73.00   17.65    1.18  -63.80  161.60   24.17\n",
      "   41        202Q 202Q N   CA   1561 1562  158.29  140.70                  -40.30\n",
      "   42  15896 202Q 203T C   N    1568 1570  -76.06  -78.10   18.89    0.88  -63.20  149.87   20.07\n",
      "   42        203T 203T N   CA   1570 1571  168.58  149.80                  -42.10\n",
      "   43  15897 203T 204G C   N    1575 1577  102.75   78.70   60.52    0.92   82.20  131.48    7.40\n",
      "   43        204G 204G N   CA   1577 1578  138.37 -166.10                    8.50\n",
      "   44  15898 204G 205E C   N    1579 1581 -131.60 -117.80   21.22    0.71  -63.60 -179.90   27.84\n",
      "   44        205E 205E N   CA   1581 1582  152.93  136.80                  -40.30\n",
      "   45  15899 205E 206T C   N    1588 1590  -71.93  -78.10   15.54    0.51  -63.20  177.86   22.24\n",
      "   45        206T 206T N   CA   1590 1591  135.54  149.80                  -42.10\n",
      "   46  15900 206T 207K C   N    1595 1597 -104.40 -118.00   34.18    1.43  -62.90  154.24   18.19\n",
      "   46        207K 207K N   CA   1597 1598  107.75  139.10                  -40.80\n",
      "   47  15901 207K 208M C   N    1604 1606 -144.58 -125.60   27.25    0.84  -63.40  178.92   33.06\n",
      "   47        208M 208M N   CA   1606 1607  160.05  140.50                  -40.50\n",
      "   48  15902 208M 209N C   N    1612 1614 -105.67 -119.90   14.92    0.51  -63.20  178.73   20.30\n",
      "   48        209N 209N N   CA   1614 1615  132.51  137.00                  -41.10\n",
      "   49  15903 209N 210E C   N    1620 1622 -124.09 -117.80    7.83    0.40  -63.60 -177.25   21.67\n",
      "   49        210E 210E N   CA   1622 1623  132.14  136.80                  -40.30\n",
      "   50  15905 211T 212S C   N    1636 1638 -143.26 -136.60   12.59    0.71  -64.10 -167.46   12.06\n",
      "   50        212S 212S N   CA   1638 1639  140.51  151.20                  -35.00\n",
      "   51  15906 212S 213S C   N    1642 1644 -119.09 -136.60   17.67    0.76  -64.10  179.99   16.88\n",
      "   51        213S 213S N   CA   1644 1645  153.61  151.20                  -35.00\n",
      "   52  15907 213S 214R C   N    1648 1650 -103.60 -125.20   30.33    0.92  -63.00  165.46   19.94\n",
      "   52        214R 214R N   CA   1650 1651  119.31  140.60                  -41.10\n",
      "   53  15908 214R 215S C   N    1659 1661 -175.12 -136.60   38.72    1.44  -64.10 -157.13   22.71\n",
      "   53        215S 215S N   CA   1661 1662  155.20  151.20                  -35.00\n",
      "   54  15909 215S 216H C   N    1665 1667  -80.66  -67.60   14.62    0.91  -63.20  172.04   22.62\n",
      "   54        216H 216H N   CA   1667 1668  146.55  140.00                  -42.30\n",
      "   55  15910 216H 217A C   N    1675 1677 -101.47  -68.20   44.60    4.40  -62.50  161.29   24.83\n",
      "   55        217A 217A N   CA   1677 1678  115.61  145.30                  -40.90\n",
      "   56  15913 219V 220Q C   N    1695 1697  -77.18  -63.80   14.12    2.12  -73.00  176.54   11.63\n",
      "   56        220Q 220Q N   CA   1697 1698  -35.79  -40.30                  140.70\n",
      "   57  15914 220Q 221I C   N    1704 1706   60.45  -97.30  159.06    6.65 -120.60  179.77   11.90\n",
      "   57        221I 221I N   CA   1706 1707  147.55  127.20                  130.30\n",
      "   58  15916 222T 223I C   N    1719 1721  -59.36  -63.40    4.05    0.71 -120.60 -175.31    8.36\n",
      "   58        223I 223I N   CA   1721 1722  -43.94  -43.60                  130.30\n",
      "   59  15918 224S 225Q C   N    1733 1735 -170.65 -121.10   96.32    3.54  -63.80  144.58   26.12\n",
      "   59        225Q 225Q N   CA   1735 1736 -137.70  139.70                  -40.30\n",
      "   60  15921 227Y 228G C   N    1763 1765   93.53   82.20   14.14    1.10  -62.40  166.43   31.34\n",
      "   60        228G 228G N   CA   1765 1766   16.96    8.50                  -41.20\n",
      "   61  15923 229S 230L C   N    1773 1775  -76.81  -70.70    6.50    0.51  -63.50  175.47   25.15\n",
      "   61        230L 230L N   CA   1775 1776  143.83  141.60                  -41.20\n",
      "   62  15924 230L 231D C   N    1781 1783  -74.43  -96.50   36.68    1.54  -63.30  125.39   14.84\n",
      "   62        231D 231D N   CA   1783 1784   84.90  114.20                  -40.00\n",
      "   63  15925 231D 232M C   N    1789 1791 -136.99  -63.40  100.05   13.14  -63.40  100.05   13.14\n",
      "   63        232M 232M N   CA   1791 1792   27.28  -40.50                  -40.50\n",
      "   64  15926 232M 233R C   N    1797 1799   57.56   57.30    8.27    0.53  -63.00  139.82   25.49\n",
      "   64        233R 233R N   CA   1799 1800   29.73   38.00                  -41.10\n",
      "   65  15927 233R 234D C   N    1808 1810  -65.73  -63.30   63.03    7.93  -63.30   63.03    7.93\n",
      "   65        234D 234D N   CA   1810 1811 -102.98  -40.00                  -40.00\n",
      "   66  15930 236E 237S C   N    1832 1834   78.93 -136.60  144.53    5.62  -64.10 -138.29   16.71\n",
      "   66        237S 237S N   CA   1834 1835  155.60  151.20                  -35.00\n",
      "   67  15934 240L 241L C   N    1860 1862 -113.33 -108.50   20.78    1.16  -63.50  161.38   20.17\n",
      "   67        241L 241L N   CA   1862 1863  112.29  132.50                  -41.20\n",
      "   68  15939 245A 246G C   N    1896 1898  -75.72  -62.40  138.57   21.36  -62.40  138.57   21.36\n",
      "   68        246G 246G N   CA   1898 1899 -179.13  -41.20                  -41.20\n",
      "   69  15940 246G 247S C   N    1900 1902   75.17  -64.10  142.88   16.92  -64.10  142.88   16.92\n",
      "   69        247S 247S N   CA   1902 1903  -66.87  -35.00                  -35.00\n",
      "   70  15947 253T 254E C   N    1957 1959   65.63   54.60   19.03    1.00  -63.60  145.65   25.03\n",
      "   70        254E 254E N   CA   1959 1960   26.89   42.40                  -40.30\n",
      "   71  15948 254E 255S C   N    1966 1968  -73.06  -72.40   14.29    0.86  -64.10  173.36   12.06\n",
      "   71        255S 255S N   CA   1968 1969  138.13  152.40                  -35.00\n",
      "   72  15949 255S 256T C   N    1972 1974 -148.76 -124.80   43.16    1.34  -63.20  162.80   26.32\n",
      "   72        256T 256T N   CA   1974 1975  179.40  143.50                  -42.10\n",
      "   73  15950 256T 257G C   N    1979 1981 -110.15  -62.40   89.55   12.19   82.20  169.66   12.23\n",
      "   73        257G 257G N   CA   1981 1982   34.56  -41.20                    8.50\n",
      "   74  15952 258V 259A C   N    1990 1992 -148.25 -134.00   17.64    0.45  -62.50 -176.97   34.58\n",
      "   74        259A 259A N   CA   1992 1993  157.40  147.00                  -40.90\n",
      "   75  15953 259A 260F C   N    1995 1997  -73.22  -71.40    5.47    0.38  -63.20  170.14   24.30\n",
      "   75        260F 260F N   CA   1997 1998  145.86  140.70                  -44.30\n",
      "   76  15954 260F 261E C   N    2006 2008  -85.43  -69.30   35.32    3.16  -63.60  152.95   19.27\n",
      "   76        261E 261E N   CA   2008 2009  111.08  142.50                  -40.30\n",
      "   77  15955 261E 262E C   N    2015 2017 -130.14 -117.80   15.35    0.79  -63.60 -179.33   21.24\n",
      "   77        262E 262E N   CA   2017 2018  127.67  136.80                  -40.30\n",
      "   78  15956 262E 263A C   N    2024 2026  168.39 -134.00   64.98    1.49  -62.50 -168.03   38.10\n",
      "   78        263A 263A N   CA   2026 2027  177.04  147.00                  -40.90\n",
      "   79  15957 263A 264K C   N    2029 2031  -67.00  -70.20   13.42    1.04  -62.90  165.82   21.76\n",
      "   79        264K 264K N   CA   2031 2032  153.43  140.40                  -40.80\n",
      "   80  15958 264K 265K C   N    2038 2040  -61.95  -70.20   30.55    2.02  -62.90  151.79   19.76\n",
      "   80        265K 265K N   CA   2040 2041  110.99  140.40                  -40.80\n",
      "   81  15960 266I 267N C   N    2055 2057  -65.95  -71.20    5.25    0.38  -63.20  176.07   22.16\n",
      "   81        267N 267N N   CA   2057 2058  142.85  142.80                  -41.10\n",
      "   82  15961 267N 268Q C   N    2063 2065  -72.88  -73.00    4.97    0.32  -63.80  176.27   25.45\n",
      "   82        268Q 268Q N   CA   2065 2066  135.73  140.70                  -40.30\n",
      "   83  15962 268Q 269S C   N    2072 2074 -152.09 -136.60   33.49    1.30  -64.10  168.85   18.58\n",
      "   83        269S 269S N   CA   2074 2075 -179.11  151.20                  -35.00\n",
      "   84  15963 269S 270L C   N    2078 2080  -67.84  -70.70    5.91    0.55  -63.50  172.08   24.14\n",
      "   84        270L 270L N   CA   2080 2081  146.77  141.60                  -41.20\n",
      "   85  15964 270L 271L C   N    2086 2088   69.14  -70.70  141.77   12.07  -63.50 -152.56   36.73\n",
      "   85        271L 271L N   CA   2088 2089  118.29  141.60                  -41.20\n",
      "   86  15965 271L 272M C   N    2094 2096  -90.02  -73.00   28.81    1.80  -63.40  155.55   25.35\n",
      "   86        272M 272M N   CA   2096 2097  166.25  143.00                  -40.50\n",
      "   87  15966 272M 273L C   N    2102 2104  -83.24  -70.70   56.19    3.81  -63.50  124.01   18.41\n",
      "   87        273L 273L N   CA   2104 2105 -163.63  141.60                  -41.20\n",
      "   88  15968 274G 275R C   N    2114 2116 -121.18 -125.20   19.64    1.02  -63.00  169.39   26.89\n",
      "   88        275R 275R N   CA   2116 2117  159.82  140.60                  -41.10\n",
      "   89  15969 275R 276A C   N    2125 2127  -72.34  -68.20   22.97    1.70  -62.50  151.53   25.33\n",
      "   89        276A 276A N   CA   2127 2128  167.89  145.30                  -40.90\n",
      "   90  15971 277M 278N C   N    2138 2140   49.49  -71.20  130.11   10.35 -119.90  177.91    8.64\n",
      "   90        278N 278N N   CA   2140 2141 -168.58  142.80                  137.00\n",
      "   91  15972 278N 279S C   N    2146 2148  -80.99  -72.40   87.86    5.47  -64.10  101.38    6.53\n",
      "   91        279S 279S N   CA   2148 2149   64.96  152.40                  -35.00\n",
      "   92  15992 298T 299R C   N    2297 2299 -160.30 -125.20   37.42    1.20  -63.00 -168.17   32.44\n",
      "   92        299R 299R N   CA   2299 2300  153.58  140.60                  -41.10\n",
      "   93  15993 299R 300L C   N    2308 2310 -117.23 -108.50   10.78    0.48  -63.50 -172.17   29.39\n",
      "   93        300L 300L N   CA   2310 2311  138.82  132.50                  -41.20\n",
      "   94  15994 300L 301L C   N    2316 2318  -98.60 -108.50   21.04    1.01  -63.50  159.06   20.35\n",
      "   94        301L 301L N   CA   2318 2319  113.94  132.50                  -41.20\n",
      "   95  15995 301L 302S C   N    2324 2326 -174.58 -136.60   47.56    1.46  -64.10 -177.56   21.19\n",
      "   95        302S 302S N   CA   2326 2327  179.81  151.20                  -35.00\n",
      "   96  15996 302S 303E C   N    2330 2332  -64.86  -69.30    4.48    0.36  -63.60  176.62   23.62\n",
      "   96        303E 303E N   CA   2332 2333  143.08  142.50                  -40.30\n",
      "   97  15997 303E 304S C   N    2339 2341 -124.35 -136.60   13.15    0.65  -64.10  179.44   17.24\n",
      "   97        304S 304S N   CA   2341 2342  155.98  151.20                  -35.00\n",
      "   98  15998 304S 305F C   N    2345 2347 -124.82 -124.20   20.64    0.96  -63.20  163.80   26.78\n",
      "   98        305F 305F N   CA   2347 2348  163.93  143.30                  -44.30\n",
      "   99  15999 305F 306G C   N    2356 2358  -65.97  -80.20   26.26    0.64   82.20 -153.72    6.57\n",
      "   99        306G 306G N   CA   2358 2359  152.03  174.10                    8.50\n",
      "  100  16000 306G 307G C   N    2360 2362 -176.51 -167.20   10.47    0.16   82.20 -161.37   14.14\n",
      "  100        307G 307G N   CA   2362 2363  179.37  174.60                    8.50\n",
      "  101  16001 307G 308N C   N    2364 2366  -71.94  -71.20    4.68    0.28  -63.20  171.71   21.95\n",
      "  101        308N 308N N   CA   2366 2367  147.42  142.80                  -41.10\n",
      "  102  16002 308N 309S C   N    2372 2374  -77.41  -72.40   11.27    0.52  -64.10  163.04   12.65\n",
      "  102        309S 309S N   CA   2374 2375  162.50  152.40                  -35.00\n",
      "  103  16003 309S 310K C   N    2378 2380  -75.12  -70.20   19.11    1.27  -62.90  160.80   21.57\n",
      "  103        310K 310K N   CA   2380 2381  158.86  140.40                  -40.80\n",
      "  104  16004 310K 311T C   N    2387 2389  -66.91  -78.10   11.24    0.65  -63.20  169.20   21.94\n",
      "  104        311T 311T N   CA   2389 2390  148.74  149.80                  -42.10\n",
      "  105  16006 312W 313M C   N    2408 2410  -62.87  -73.00   15.26    1.20  -63.40  165.08   24.61\n",
      "  105        313M 313M N   CA   2410 2411  154.42  143.00                  -40.50\n",
      "  106  16007 313M 314L C   N    2416 2418  -68.50  -70.70    3.70    0.23  -63.50  179.89   24.69\n",
      "  106        314L 314L N   CA   2418 2419  138.61  141.60                  -41.20\n",
      "  107  16008 314L 315A C   N    2424 2426 -159.87 -134.00   31.83    0.80  -62.50 -178.16   34.97\n",
      "  107        315A 315A N   CA   2426 2427  165.53  147.00                  -40.90\n",
      "  108  16009 315A 316T C   N    2429 2431  -93.80  -78.10   32.23    1.90  -63.20  166.59   19.49\n",
      "  108        316T 316T N   CA   2431 2432  121.65  149.80                  -42.10\n",
      "  109  16012 318S 319T C   N    2449 2451 -129.23 -124.80    6.23    0.20  -63.20 -177.61   27.75\n",
      "  109        319T 319T N   CA   2451 2452  147.88  143.50                  -42.10\n",
      "  110  16015 321A 322N C   N    2466 2468 -160.37 -119.90   43.51    2.17   55.90  165.24   20.38\n",
      "  110        322N 322N N   CA   2468 2469  121.03  137.00                   39.50\n",
      "  111  16016 322N 323N C   N    2474 2476 -167.73 -119.90   48.74    2.25  -71.20   97.71    7.42\n",
      "  111        323N 323N N   CA   2476 2477  127.64  137.00                  142.80\n",
      "  112  16033 339A 340I C   N    2603 2605  -60.78  -63.40    2.77    0.52 -120.60 -176.95    8.31\n",
      "  112        340I 340I N   CA   2605 2606  -42.69  -43.60                  130.30\n",
      "  113  16035 341T 342N C   N    2618 2620  -76.64 -119.90   48.43    2.48  -63.20  160.67   20.84\n",
      "  113        342N 342N N   CA   2620 2621  158.80  137.00                  -41.10\n",
      "  114  16037 343K 344A C   N    2635 2637  -80.96  -68.20   16.10    1.04  -62.50  165.03   27.99\n",
      "  114        344A 344A N   CA   2637 2638  155.11  145.30                  -40.90\n",
      "  115  16038 344A 345K C   N    2640 2642  -85.81  -62.90   86.85   10.26  -62.90   86.85   10.26\n",
      "  115        345K 345K N   CA   2642 2643   42.97  -40.80                  -40.80\n",
      "  116  16040 346V 347N C   N    2656 2658  -78.51 -119.90   80.70    2.63   55.90  137.34   13.88\n",
      "  116        347N 347N N   CA   2658 2659   67.73  137.00                   39.50\n",
      "  117  16043 349T 350K C   N    2679 2681   63.67   56.60    7.18    0.78  -62.90  148.74   25.95\n",
      "  117        350K 350K N   CA   2681 2682   37.33   38.60                  -40.80\n",
      "  118  16044 350K 351K C   N    2688 2690  -81.95  -70.20   69.03    5.27  -62.90  114.77   13.97\n",
      "  118        351K 351K N   CA   2690 2691   72.37  140.40                  -40.80\n",
      "  119  16045 351K 352N C   N    2697 2699 -150.29 -119.90   56.67    1.81  -63.20  159.87   24.79\n",
      "  119        352N 352N N   CA   2699 2700 -175.17  137.00                  -41.10\n",
      "  120  16046 352N 353I C   N    2705 2707  -62.64  -63.40    7.05    1.18 -120.60  176.67    8.01\n",
      "  120        353I 353I N   CA   2707 2708  -36.59  -43.60                  130.30\n",
      "  121  16047 353I 354E C   N    2713 2715 -147.60 -117.80   68.59    3.76  -63.60  142.68   16.54\n",
      "  121        354E 354E N   CA   2715 2716   75.02  136.80                  -40.30\n",
      "  122  16048 354E 355L C   N    2722 2724  -69.41  -63.50   47.75    6.98  -63.50   47.75    6.98\n",
      "  122        355L 355L N   CA   2724 2725  -88.58  -41.20                  -41.20\n",
      "  123  16049 355L 356K C   N    2730 2732  -96.70  -62.90   42.16    7.31  -62.90   42.16    7.31\n",
      "  123        356K 356K N   CA   2732 2733  -66.01  -40.80                  -40.80\n",
      "  124  16057 363A 364S C   N    2798 2800  -87.67  -72.40   27.82    1.27  -64.10  151.19   12.51\n",
      "  124        364S 364S N   CA   2800 2801  175.66  152.40                  -35.00\n",
      "  125  16058 364S 365L C   N    2804 2806  -80.28  -70.70   30.01    1.95  -63.50  149.70   21.79\n",
      "  125        365L 365L N   CA   2806 2807  170.04  141.60                  -41.20\n",
      "  126  16059 365L 366E C   N    2812 2814  -63.80  -69.30   13.35    1.18  -63.60  165.03   22.02\n",
      "  126        366E 366E N   CA   2814 2815  154.66  142.50                  -40.30\n",
      "  127  16061 367G 368R C   N    2825 2827 -128.23 -125.20    3.87    0.11  -63.00 -172.40   29.84\n",
      "  127        368R 368R N   CA   2827 2828  143.01  140.60                  -41.10\n",
      "  128  16063 369L 370D C   N    2844 2846 -172.85  -70.90  112.32    6.44  -63.30  164.41   27.04\n",
      "  128        370D 370D N   CA   2846 2847 -162.59  150.30                  -40.00\n",
      "  129  16065 371V 372L C   N    2859 2861  -92.47 -108.50  151.09    7.99  -70.70  160.82   13.03\n",
      "  129        372L 372L N   CA   2861 2862  -17.74  132.50                  141.60\n",
      "  130  16068 374L 375E C   N    2880 2882 -114.11 -117.80   49.32    2.54  -63.60  142.94   21.91\n",
      "  130        375E 375E N   CA   2882 2883 -174.01  136.80                  -40.30\n",
      "  131  16069 375E 376K C   N    2889 2891 -123.28 -118.00   16.54    0.85  -62.90  174.98   20.31\n",
      "  131        376K 376K N   CA   2891 2892  123.43  139.10                  -40.80\n",
      "  132  16070 376K 377Q C   N    2898 2900 -162.47 -121.10   50.48    1.56  -63.80 -179.56   31.51\n",
      "  132        377Q 377Q N   CA   2900 2901  168.62  139.70                  -40.30\n",
      "  133  16071 377Q 378R C   N    2907 2909 -149.19 -125.20   44.18    1.51  -63.00  165.42   28.11\n",
      "  133        378R 378R N   CA   2909 2910  177.70  140.60                  -41.10\n",
      "  134  16072 378R 379K C   N    2918 2920 -175.75 -118.00   82.05    2.90  -62.90  166.05   28.05\n",
      "  134        379K 379K N   CA   2920 2921 -162.61  139.10                  -40.80\n",
      "  135  16073 379K 380Q C   N    2927 2929  158.55 -121.10   82.89    2.74  -63.80 -149.21   37.62\n",
      "  135        380Q 380Q N   CA   2929 2930  160.06  139.70                  -40.30\n",
      "  136  16074 380Q 381E C   N    2936 2938 -155.19 -117.80   51.61    1.63  -63.60  173.47   28.15\n",
      "  136        381E 381E N   CA   2938 2939  172.38  136.80                  -40.30\n",
      "  137  16075 381E 382E C   N    2945 2947 -155.61 -117.80   53.55    1.72  -63.60  171.71   27.93\n",
      "  137        382E 382E N   CA   2947 2948  174.72  136.80                  -40.30\n",
      "  138  16077 383I 384G C   N    2962 2964 -165.52 -167.20    4.44    0.14   82.20 -162.90   14.41\n",
      "  138        384G 384G N   CA   2964 2965  170.49  174.60                    8.50\n",
      "  139  16078 384G 385R C   N    2966 2968 -161.03 -125.20   35.90    1.50  -63.00 -155.56   23.16\n",
      "  139        385R 385R N   CA   2968 2969  138.31  140.60                  -41.10\n",
      "  140  16079 385R 386L C   N    2977 2979 -122.21 -108.50   15.53    0.83  -63.50  176.46   21.94\n",
      "  140        386L 386L N   CA   2979 2980  125.21  132.50                  -41.20\n",
      "  141  16080 386L 387T C   N    2985 2987  -89.60  -78.10   47.76    1.71  -63.20  124.57   17.74\n",
      "  141        387T 387T N   CA   2987 2988 -163.84  149.80                  -42.10\n",
      "  142  16081 387T 388Q C   N    2992 2994  -69.77  -73.00   33.22    2.11  -63.80  148.06   21.46\n",
      "  142        388Q 388Q N   CA   2994 2995  107.64  140.70                  -40.30\n",
      "  143  16082 388Q 389E C   N    3001 3003 -154.44 -117.80   63.09    2.25  -63.60  159.85   26.24\n",
      "  143        389E 389E N   CA   3003 3004 -171.83  136.80                  -40.30\n",
      "  144  16083 389E 390R C   N    3010 3012  -64.15  -72.10   11.86    1.03  -63.00  168.20   22.80\n",
      "  144        390R 390R N   CA   3012 3013  150.71  141.90                  -41.10\n",
      "  145  16084 390R 391D C   N    3021 3023  -82.66  -70.90   48.40    1.85  -63.30  124.27   16.56\n",
      "  145        391D 391D N   CA   3023 3024 -162.75  150.30                  -40.00\n",
      "  146  16085 391D 392K C   N    3029 3031   57.20   56.60   11.04    0.83  -62.90  150.33   26.06\n",
      "  146        392K 392K N   CA   3031 3032   49.62   38.60                  -40.80\n",
      "  147  16086 392K 393L C   N    3038 3040  -66.31  -70.70    8.60    0.81  -63.50  169.83   23.74\n",
      "  147        393L 393L N   CA   3040 3041  148.99  141.60                  -41.20\n",
      "  148  16087 393L 394R C   N    3046 3048 -140.66 -125.20   33.57    1.22  -63.00  167.58   27.90\n",
      "  148        394R 394R N   CA   3048 3049  170.40  140.60                  -41.10\n",
      "  149  16088 394R 395Q C   N    3057 3059 -136.82 -121.10   37.62    1.47  -63.80  163.08   27.78\n",
      "  149        395Q 395Q N   CA   3059 3060  173.87  139.70                  -40.30\n",
      "  150  16089 395Q 396E C   N    3066 3068 -155.44 -117.80   49.66    1.53  -63.60  176.31   28.56\n",
      "  150        396E 396E N   CA   3068 3069  169.20  136.80                  -40.30\n",
      "  151  16091 397V 398A C   N    3082 3084  115.86 -134.00  136.33    7.51  -68.20 -167.28   18.50\n",
      "  151        398A 398A N   CA   3084 3085   66.66  147.00                  145.30\n",
      "  152  16094 400A 401D C   N    3103 3105 -159.42  -63.30  101.33   14.23  -63.30  101.33   14.23\n",
      "  152        401D 401D N   CA   3105 3106   -7.93  -40.00                  -40.00\n",
      "  153  16097 403V 404H C   N    3124 3126   68.11  -67.60  144.00   12.15  -63.20 -175.54   20.80\n",
      "  153        404H 404H N   CA   3126 3127 -171.85  140.00                  -42.30\n",
      "  154  16098 404H 405D C   N    3134 3136 -177.11  -96.50   84.34    3.47  -63.30 -147.88   22.60\n",
      "  154        405D 405D N   CA   3136 3137  139.00  114.20                  -40.00\n",
      "  155  16099 405D 406A C   N    3142 3144  154.70 -134.00   71.86    2.06  -62.50 -143.17   42.91\n",
      "  155        406A 406A N   CA   3144 3145  155.94  147.00                  -40.90\n",
      "  156  16101 407R 408D C   N    3158 3160 -132.31  -96.50   63.34    2.61  -63.30  123.11   13.17\n",
      "  156        408D 408D N   CA   3160 3161   61.95  114.20                  -40.00\n",
      "  157  16102 408D 409K C   N    3166 3168 -135.91  -62.90   95.42   16.44  -62.90   95.42   16.44\n",
      "  157        409K 409K N   CA   3168 3169 -102.23  -40.80                  -40.80\n",
      "  158  16104 410L 411E C   N    3183 3185  -90.27  -63.60   53.40    8.59  -63.60   53.40    8.59\n",
      "  158        411E 411E N   CA   3185 3186  -86.57  -40.30                  -40.30\n",
      "  159  16108 414L 415N C   N    3213 3215   86.92 -119.90  154.20    5.95  -63.20 -137.48   24.55\n",
      "  159        415N 415N N   CA   3215 3216  154.65  137.00                  -41.10\n",
      "  160  16109 415N 416N C   N    3221 3223  122.31 -119.90  118.54    4.58  -63.20 -117.40   39.30\n",
      "  160        416N 416N N   CA   3223 3224  150.36  137.00                  -41.10\n",
      "  161  16111 417I 418R C   N    3237 3239 -159.11 -125.20   41.32    1.22  -63.00 -177.88   31.02\n",
      "  161        418R 418R N   CA   3239 3240  164.21  140.60                  -41.10\n",
      "  162  16112 418R 419L C   N    3248 3250 -124.87 -108.50   32.90    1.57  -63.50  169.28   27.30\n",
      "  162        419L 419L N   CA   3250 3251  161.04  132.50                  -41.20\n",
      "  163  16113 419L 420S C   N    3256 3258 -131.21 -136.60   14.08    0.58  -64.10 -174.26   11.48\n",
      "  163        420S 420S N   CA   3258 3259  138.19  151.20                  -35.00\n",
      "  164  16114 420S 421N C   N    3262 3264 -147.98 -119.90   47.78    1.48  -63.20  166.45   25.52\n",
      "  164        421N 421N N   CA   3264 3265  175.66  137.00                  -41.10\n",
      "  165  16116 422I 423A C   N    3278 3280   62.07  -68.20  130.34   11.19  -62.50 -149.60   31.74\n",
      "  165        423A 423A N   CA   3280 3281  149.54  145.30                  -40.90\n",
      "  166  16117 423A 424L C   N    3283 3285 -147.55 -108.50   42.92    1.90  -63.50 -171.73   31.34\n",
      "  166        424L 424L N   CA   3285 3286  150.33  132.50                  -41.20\n",
      "  167  16118 424L 425R C   N    3291 3293 -125.95 -125.20    8.76    0.44  -63.00 -175.93   21.43\n",
      "  167        425R 425R N   CA   3293 3294  131.87  140.60                  -41.10\n",
      "  168  16119 425R 426R C   N    3302 3304 -150.92 -125.20   29.54    0.88  -63.00 -174.11   31.06\n",
      "  168        426R 426R N   CA   3304 3305  155.12  140.60                  -41.10\n",
      "  169  16120 426R 427R C   N    3313 3315  -88.70  -72.10   19.37    1.28  -63.00  168.99   24.60\n",
      "  169        427R 427R N   CA   3315 3316  151.88  141.90                  -41.10\n",
      "  170  16122 428V 429E C   N    3331 3333  -78.94  -69.30   22.81    1.40  -63.60  157.28   21.82\n",
      "  170        429E 429E N   CA   3333 3334  163.17  142.50                  -40.30\n",
      "  171  16123 429E 430A C   N    3340 3342  -64.10  -68.20    4.14    0.33  -62.50  174.39   28.66\n",
      "  171        430A 430A N   CA   3342 3343  144.72  145.30                  -40.90\n",
      "  172  16124 430A 431A C   N    3345 3347  176.71 -134.00   62.85    1.66  -62.50  179.75   35.67\n",
      "  172        431A 431A N   CA   3347 3348 -174.01  147.00                  -40.90\n",
      "  173  16125 431A 432S C   N    3350 3352  -68.09  -72.40    4.37    0.29  -64.10  173.34   12.80\n",
      "  173        432S 432S N   CA   3352 3353  151.70  152.40                  -35.00\n",
      "  174  16126 432S 433E C   N    3356 3358  -91.66  -69.30   32.04    1.79  -63.60  156.79   22.49\n",
      "  174        433E 433E N   CA   3358 3359  165.44  142.50                  -40.30\n",
      "  175  16127 433E 434G C   N    3365 3367 -148.17 -167.20   20.49    0.59   82.20 -143.27   16.04\n",
      "  175        434G 434G N   CA   3367 3368 -177.81  174.60                    8.50\n",
      "  176  16128 434G 435F C   N    3369 3371  -69.35  -71.40   20.84    1.73  -63.20  154.39   21.85\n",
      "  176        435F 435F N   CA   3371 3372  161.44  140.70                  -44.30\n",
      "  177  16130 436I 437H C   N    3388 3390  176.22 -125.60   93.55    2.40  -63.20  160.31   27.17\n",
      "  177        437H 437H N   CA   3390 3391 -147.94  138.80                  -42.30\n",
      "  178  16131 437H 438S C   N    3398 3400  -63.03  -72.40   29.91    1.47  -64.10  159.00   11.58\n",
      "  178        438S 438S N   CA   3400 3401  124.00  152.40                  -35.00\n",
      "  179  16133 439L 440D C   N    3412 3414 -106.21  -70.90   41.10    2.14  -63.30  154.73   21.86\n",
      "  179        440D 440D N   CA   3414 3415  171.34  150.30                  -40.00\n",
      "  180  16134 440D 441N C   N    3420 3422   52.33   55.90    4.28    0.32  -63.20  142.24   23.34\n",
      "  180        441N 441N N   CA   3422 3423   41.87   39.50                  -41.10\n",
      "  181  16135 441N 442K C   N    3428 3430  -84.09  -70.20   17.93    1.08  -62.90  168.81   23.14\n",
      "  181        442K 442K N   CA   3430 3431  151.72  140.40                  -40.80\n",
      "  182  16136 442K 443S C   N    3437 3439  -61.82  -72.40   18.63    0.85  -64.10  172.09   12.60\n",
      "  182        443S 443S N   CA   3439 3440  137.07  152.40                  -35.00\n",
      "  183  16138 444C 445F C   N    3449 3451 -176.02 -124.20  109.07    3.62  -63.20  136.27   24.59\n",
      "  183        445F 445F N   CA   3451 3452 -120.72  143.30                  -44.30\n",
      "  184  16139 445F 446L C   N    3460 3462  -61.49  -70.70   24.70    1.58  -63.50  159.90   22.32\n",
      "  184        446L 446L N   CA   3462 3463  118.68  141.60                  -41.20\n",
      "  185  16140 446L 447F C   N    3468 3470  -83.35  -71.40   42.68    3.70  -63.20  145.43   18.88\n",
      "  185        447F 447F N   CA   3470 3471   99.72  140.70                  -44.30\n",
      "  186  16141 447F 448F C   N    3479 3481 -151.70 -124.20   89.93    3.47  -63.20  123.94   22.18\n",
      "  186        448F 448F N   CA   3481 3482 -131.07  143.30                  -44.30\n",
      "  187  16142 448F 449K C   N    3490 3492  -11.43  -70.20   62.81    4.06  -62.90  167.15   24.77\n",
      "  187        449K 449K N   CA   3492 3493  118.23  140.40                  -40.80\n",
      "  188  16143 449K 450G C   N    3499 3501  -79.26  -80.20   13.53    0.58   82.20 -118.87    7.16\n",
      "  188        450G 450G N   CA   3501 3502 -172.41  174.10                    8.50\n",
      "  189  16144 450G 451R C   N    3503 3505   54.55   57.30   16.37    0.90  -63.00  151.29   27.42\n",
      "  189        451R 451R N   CA   3505 3506   54.14   38.00                  -41.10\n",
      "  190  16146 452C 453S C   N    3520 3522 -134.52 -136.60    5.73    0.32  -64.10 -177.42   18.23\n",
      "  190        453S 453S N   CA   3522 3523  156.54  151.20                  -35.00\n",
      "  191  16148 454I 455T C   N    3534 3536 -122.24 -124.80    5.34    0.30  -63.20  179.70   26.96\n",
      "  191        455T 455T N   CA   3536 3537  148.18  143.50                  -42.10\n",
      "  192  16149 455T 456L C   N    3541 3543 -149.76 -108.50  114.76    5.69  -63.50  117.12   21.23\n",
      "  192        456L 456L N   CA   3543 3544 -120.42  132.50                  -41.20\n",
      "  193  16151 457E 458S C   N    3558 3560  170.24 -136.60   74.12    4.12  -64.10 -175.90   14.48\n",
      "  193        458S 458S N   CA   3560 3561   99.55  151.20                  -35.00\n",
      "  194  16152 458S 459V C   N    3564 3566  160.37  -62.40  138.23   21.10 -125.40 -175.34   10.48\n",
      "  194        459V 459V N   CA   3566 3567  -25.78  -42.40                  143.30\n",
      "  195  16153 459V 460L C   N    3571 3573 -107.19  -63.50   85.77   10.52  -63.50   85.77   10.52\n",
      "  195        460L 460L N   CA   3573 3574   32.61  -41.20                  -41.20\n",
      "  196  16154 460L 461R C   N    3579 3581 -179.91  -63.00  119.25   20.97  -63.00  119.25   20.97\n",
      "  196        461R 461R N   CA   3581 3582  -64.63  -41.10                  -41.10\n",
      "  197  16155 461R 462G C   N    3590 3592 -146.71 -167.20  100.29    4.10   78.70  156.00    8.73\n",
      "  197        462G 462G N   CA   3592 3593  -87.23  174.60                 -166.10\n",
      "  198  16157 463Q 464R C   N    3603 3605   47.63  -72.10  122.80    9.95 -125.20  175.18    7.77\n",
      "  198        464R 464R N   CA   3605 3606  169.16  141.90                  140.60\n",
      "  199  16177 483L 484H C   N    3759 3761   54.83   56.30    9.21    0.71  -63.20  149.77   25.54\n",
      "  199        484H 484H N   CA   3761 3762   49.90   40.80                  -42.30\n",
      "  200  16179 485I 486Q C   N    3777 3779 -137.60 -121.10   18.92    0.58  -63.80 -174.00   31.23\n",
      "  200        486Q 486Q N   CA   3779 3780  148.96  139.70                  -40.30\n",
      "  201  16180 486Q 487L C   N    3786 3788 -108.92 -108.50    3.28    0.18  -63.50  176.40   22.33\n",
      "  201        487L 487L N   CA   3788 3789  129.25  132.50                  -41.20\n",
      "  202  16181 487L 488F C   N    3794 3796 -124.83 -124.20    3.45    0.18  -63.20 -173.72   29.94\n",
      "  202        488F 488F N   CA   3796 3797  139.91  143.30                  -44.30\n",
      "  203  16184 490C 491E C   N    3818 3820 -105.41 -117.80   14.24    0.42  -63.60  175.15   21.34\n",
      "  203        491E 491E N   CA   3820 3821  129.79  136.80                  -40.30\n",
      "  204  16185 491E 492H C   N    3827 3829 -129.42 -125.60    9.56    0.28  -63.20 -177.43   27.16\n",
      "  204        492H 492H N   CA   3829 3830  147.56  138.80                  -42.30\n",
      "  205  16186 492H 493H C   N    3837 3839 -120.55 -125.60    8.11    0.21  -63.20 -176.07   20.12\n",
      "  205        493H 493H N   CA   3839 3840  132.46  138.80                  -42.30\n",
      "  206  16187 493H 494A C   N    3847 3849 -132.38 -134.00    1.62    0.06  -62.50 -174.39   34.16\n",
      "  206        494A 494A N   CA   3849 3850  147.15  147.00                  -40.90\n",
      "  207  16188 494A 495N C   N    3852 3854 -103.37 -119.90   45.34    1.64  -63.20  141.70   15.85\n",
      "  207        495N 495N N   CA   3854 3855   94.78  137.00                  -41.10\n",
      "  208  16189 495N 496E C   N    3860 3862 -168.47 -117.80   67.68    2.10  -63.60  173.36   28.73\n",
      "  208        496E 496E N   CA   3862 3863 -178.34  136.80                  -40.30\n",
      "  209  16190 496E 497R C   N    3869 3871  -65.98  -72.10   16.36    1.38  -63.00  161.86   22.07\n",
      "  209        497R 497R N   CA   3871 3872  157.07  141.90                  -41.10\n",
      "  210  16191 497R 498N C   N    3880 3882  -74.41  -71.20   55.36    3.64  -63.20  129.11   15.55\n",
      "  210        498N 498N N   CA   3882 3883   87.53  142.80                  -41.10\n",
      "  211  16192 498N 499D C   N    3888 3890 -121.92  -70.90   52.83    3.46  -63.30  166.68   24.36\n",
      "  211        499D 499D N   CA   3890 3891  163.97  150.30                  -40.00\n",
      "  212  16194 500P 501M C   N    3903 3905 -113.72 -125.60   23.98    0.85  -63.40  167.89   21.72\n",
      "  212        501M 501M N   CA   3905 3906  119.67  140.50                  -40.50\n",
      "  213  16195 501M 502Q C   N    3911 3913 -148.75 -121.10   34.99    1.09  -63.80  179.87   30.84\n",
      "  213        502Q 502Q N   CA   3913 3914  161.15  139.70                  -40.30\n",
      "  214  16196 502Q 503F C   N    3920 3922  -94.28  -71.40   24.27    1.42  -63.20  169.77   25.67\n",
      "  214        503F 503F N   CA   3922 3923  148.80  140.70                  -44.30\n",
      "  215  16199 505G 506K C   N    3943 3945  -54.11  -62.90   83.47   10.39  -62.90   83.47   10.39\n",
      "  215        506K 506K N   CA   3945 3946 -123.80  -40.80                  -40.80\n",
      "  216  16200 506K 507S C   N    3952 3954   50.45  -64.10  144.36   13.04  -64.10  144.36   13.04\n",
      "  216        507S 507S N   CA   3954 3955 -122.85  -35.00                  -35.00\n",
      "  217  16201 507S 508L C   N    3958 3960 -110.16  -63.50   46.67    8.10  -63.50   46.67    8.10\n",
      "  217        508L 508L N   CA   3960 3961  -42.22  -41.20                  -41.20\n",
      "  218  16205 511C 512L C   N    3994 3996   51.79  -70.70  123.06   11.54  -63.50 -158.48   24.89\n",
      "  218        512L 512L N   CA   3996 3997  153.52  141.60                  -41.20\n",
      "  219  16206 512L 513H C   N    4002 4004 -165.73 -125.60   58.03    1.44  -63.20  171.11   27.85\n",
      "  219        513H 513H N   CA   4004 4005 -179.28  138.80                  -42.30\n",
      "  220  16209 515V 516G C   N    4026 4028   85.13   78.70   19.47    0.32   82.20  167.04    8.26\n",
      "  220        516G 516G N   CA   4028 4029  175.52 -166.10                    8.50\n",
      "  221  16210 516G 517A C   N    4030 4032 -108.80 -134.00   26.17    0.64  -62.50 -174.95   32.77\n",
      "  221        517A 517A N   CA   4032 4033  139.94  147.00                  -40.90\n",
      "  222  16211 517A 518S C   N    4035 4037 -127.36 -136.60   13.60    0.76  -64.10  175.61   17.19\n",
      "  222        518S 518S N   CA   4037 4038  161.17  151.20                  -35.00\n",
      "  223  16212 518S 519G C   N    4041 4043   89.19   78.70   25.86    0.39   82.20  161.91    8.21\n",
      "  223        519G 519G N   CA   4043 4044  170.26 -166.10                    8.50\n",
      "  224  16214 520I 521P C   N    4053 4055  -63.88  -58.70    5.85    0.55  -64.50  174.97   13.02\n",
      "  224        521P 521P N   CA   4055 4056  -27.77  -30.50                  147.20\n",
      "  225  16215 521P 522K C   N    4060 4062   51.82   56.60    7.33    0.43  -62.90  142.75   24.76\n",
      "  225        522K 522K N   CA   4062 4063   44.15   38.60                  -40.80\n",
      "  226  16216 522K 523A C   N    4069 4071  -63.72  -68.20    9.13    0.59  -62.50  178.25   29.15\n",
      "  226        523A 523A N   CA   4071 4072  137.34  145.30                  -40.90\n",
      "  227  16217 523A 524F C   N    4074 4076 -148.45 -124.20   31.69    0.83  -63.20  174.27   29.60\n",
      "  227        524F 524F N   CA   4076 4077  163.70  143.30                  -44.30\n",
      "  228  16219 525V 526A C   N    4092 4094  176.04 -134.00   56.37    1.29  -62.50 -170.08   37.44\n",
      "  228        526A 526A N   CA   4094 4095  173.10  147.00                  -40.90\n",
      "  229  16220 526A 527H C   N    4097 4099  -93.15  -67.60   40.43    3.63  -63.20  153.92   17.57\n",
      "  229        527H 527H N   CA   4099 4100  108.68  140.00                  -42.30\n",
      "  230  16221 527H 528T C   N    4107 4109 -142.97 -124.80   48.98    1.62  -63.20  151.60   24.52\n",
      "  230        528T 528T N   CA   4109 4110 -171.02  143.50                  -42.10\n",
      "  231  16222 528T 529F C   N    4114 4116  -65.24  -71.40    7.78    0.67  -63.20  170.27   23.78\n",
      "  231        529F 529F N   CA   4116 4117  145.44  140.70                  -44.30\n",
      "  232  16224 530C 531K C   N    4131 4133 -110.27 -118.00    8.60    0.40  -62.90 -177.42   26.51\n",
      "  232        531K 531K N   CA   4133 4134  142.88  139.10                  -40.80\n",
      "  233  16225 531K 532F C   N    4140 4142 -142.05 -124.20   38.31    1.28  -63.20  159.38   27.12\n",
      "  233        532F 532F N   CA   4142 4143  177.20  143.30                  -44.30\n",
      "  234  16227 533T 534L C   N    4158 4160  -80.14  -70.70   15.54    0.97  -63.50  165.70   23.99\n",
      "  234        534L 534L N   CA   4160 4161  153.94  141.60                  -41.20\n",
      "  235  16228 534L 535L C   N    4166 4168 -108.84 -108.50   61.65    3.33  -63.50  132.64   21.22\n",
      "  235        535L 535L N   CA   4168 4169 -165.85  132.50                  -41.20\n",
      "  236  16229 535L 536H C   N    4174 4176 -117.22 -125.60   10.81    0.27  -63.20 -177.56   20.05\n",
      "  236        536H 536H N   CA   4176 4177  131.96  138.80                  -42.30\n",
      "  237  16232 538R 539E C   N    4203 4205  -66.38  -69.30   17.29    1.21  -63.60  165.79   21.95\n",
      "  237        539E 539E N   CA   4205 4206  125.46  142.50                  -40.30\n",
      "  238  16233 539E 540E C   N    4212 4214 -167.62 -117.80   81.20    2.82  -63.60  157.89   26.50\n",
      "  238        540E 540E N   CA   4214 4215 -159.07  136.80                  -40.30\n",
      "  239  16234 540E 541R C   N    4221 4223 -177.52 -125.20   54.54    2.58  -63.00 -158.09   22.91\n",
      "  239        541R 541R N   CA   4223 4224  125.20  140.60                  -41.10\n",
      "  240  16235 541R 542Y C   N    4232 4234 -134.35 -124.30   21.86    0.86  -63.50  176.62   31.54\n",
      "  240        542Y 542Y N   CA   4234 4235  154.81  135.40                  -43.40\n",
      "  241  16236 542Y 543F C   N    4244 4246 -117.69 -124.20   15.72    0.86  -63.20  167.21   26.83\n",
      "  241        543F 543F N   CA   4246 4247  157.62  143.30                  -44.30\n",
      "  242  16237 543F 544T C   N    4255 4257 -175.51 -124.80   57.16    2.25  -63.20 -174.18   30.77\n",
      "  242        544T 544T N   CA   4257 4258  169.86  143.50                  -42.10\n",
      "  243  16238 544T 545T C   N    4262 4264  102.53 -124.80  132.67    7.05  -63.20 -118.57   26.27\n",
      "  243        545T 545T N   CA   4264 4265  142.34  143.50                  -42.10\n",
      "  244  16239 545T 546S C   N    4269 4271 -148.96 -136.60   15.45    0.84  -64.10 -163.78   12.40\n",
      "  244        546S 546S N   CA   4271 4272  141.92  151.20                  -35.00\n",
      "  245  16240 546S 547T C   N    4275 4277 -123.24 -124.80   33.77    1.50  -63.20  152.94   23.56\n",
      "  245        547T 547T N   CA   4277 4278  177.23  143.50                  -42.10\n",
      "  246  16241 547T 548A C   N    4282 4284 -135.42 -134.00    3.86    0.24  -62.50 -169.78   35.08\n",
      "  246        548A 548A N   CA   4284 4285  143.41  147.00                  -40.90\n",
      "  247  16242 548A 549E C   N    4287 4289  165.62 -117.80   81.57    2.45  -63.60 -157.34   33.91\n",
      "  247        549E 549E N   CA   4289 4290  164.89  136.80                  -40.30\n",
      "  248  16243 549E 550N C   N    4296 4298 -147.47 -119.90   28.08    1.03  -63.20 -164.33   29.22\n",
      "  248        550N 550N N   CA   4298 4299  142.31  137.00                  -41.10\n",
      "  249  16244 550N 551T C   N    4304 4306 -144.45 -124.80   19.80    0.99  -63.20 -169.77   29.70\n",
      "  249        551T 551T N   CA   4306 4307  145.90  143.50                  -42.10\n",
      "  250  16245 551T 552Q C   N    4311 4313 -126.19 -121.10   17.21    0.88  -63.80  175.06   23.01\n",
      "  250        552Q 552Q N   CA   4313 4314  123.26  139.70                  -40.30\n",
      "  251  16246 552Q 553N C   N    4320 4322 -148.02 -119.90   30.57    0.98  -63.20 -170.07   28.52\n",
      "  251        553N 553N N   CA   4322 4323  148.97  137.00                  -41.10\n",
      "  252  16248 554P 555R C   N    4335 4337 -155.50 -125.20   49.13    1.59  -63.00  167.49   28.76\n",
      "  252        555R 555R N   CA   4337 4338  179.27  140.60                  -41.10\n",
      "  253  16249 555R 556W C   N    4346 4348  -66.07  -71.30    7.41    0.43  -63.00  177.97   22.07\n",
      "  253        556W 556W N   CA   4348 4349  133.75  139.00                  -44.20\n",
      "  254  16250 556W 557G C   N    4360 4362   65.95   78.70   30.73    0.46   82.20  147.54    7.97\n",
      "  254        557G 557G N   CA   4362 4363 -138.14 -166.10                    8.50\n",
      "  255  16251 557G 558Y C   N    4364 4366 -122.78 -124.30   27.40    1.52  -63.50  164.87   29.13\n",
      "  255        558Y 558Y N   CA   4366 4367  162.76  135.40                  -43.40\n",
      "  256  16253 559V 560K C   N    4383 4385   55.19   56.60    1.60    0.14  -62.90  142.73   24.84\n",
      "  256        560K 560K N   CA   4385 4386   39.37   38.60                  -40.80\n",
      "  257  16255 561V 562F C   N    4399 4401 -147.36 -124.20   39.84    2.21  -63.20  176.54   20.12\n",
      "  257        562F 562F N   CA   4401 4402  110.88  143.30                  -44.30\n",
      "  258  16259 565P 566E C   N    4434 4436  -83.54  -69.30   15.56    0.96  -63.60  172.08   24.06\n",
      "  258        566E 566E N   CA   4436 4437  148.78  142.50                  -40.30\n",
      "  259  16260 566E 567L C   N    4443 4445 -105.75 -108.50   45.38    2.42  -63.50  135.18   16.88\n",
      "  259        567L 567L N   CA   4445 4446   87.21  132.50                  -41.20\n",
      "  260  16261 567L 568T C   N    4451 4453 -165.50 -124.80   40.72    2.12  -63.20 -158.86   32.34\n",
      "  260        568T 568T N   CA   4453 4454  144.72  143.50                  -42.10\n",
      "  261  16268 574C 575F C   N    4504 4506 -133.50 -124.20    9.83    0.29  -63.20 -176.78   30.04\n",
      "  261        575F 575F N   CA   4506 4507  146.50  143.30                  -44.30\n",
      "  262  16270 576C 577E C   N    4521 4523 -145.86 -117.80   37.04    1.14  -63.60  178.78   28.42\n",
      "  262        577E 577E N   CA   4523 4524  160.97  136.80                  -40.30\n",
      "  263  16271 577E 578H C   N    4530 4532 -120.81 -125.60   22.61    0.76  -63.20  169.12   18.35\n",
      "  263        578H 578H N   CA   4532 4533  116.71  138.80                  -42.30\n",
      "  264  16272 578H 579T C   N    4540 4542 -112.29 -124.80   15.73    0.55  -63.20 -177.21   20.59\n",
      "  264        579T 579T N   CA   4542 4543  133.97  143.50                  -42.10\n",
      "  265  16274 580V 581F C   N    4554 4556  136.28 -124.20  100.24    3.56  -63.20 -133.11   40.57\n",
      "  265        581F 581F N   CA   4556 4557  155.35  143.30                  -44.30\n",
      "  266  16275 581F 582A C   N    4565 4567 -159.54 -134.00   26.98    0.63  -62.50 -169.96   36.31\n",
      "  266        582A 582A N   CA   4567 4568  155.71  147.00                  -40.90\n",
      "  267  16276 582A 583F C   N    4570 4572 -159.82 -124.20   86.02    3.03  -63.20  134.87   24.15\n",
      "  267        583F 583F N   CA   4572 4573 -138.40  143.30                  -44.30\n",
      "  268  16279 585V 586F C   N    4597 4599 -122.47 -124.20    2.28    0.12  -63.20 -179.10   29.04\n",
      "  268        586F 586F N   CA   4599 4600  144.78  143.30                  -44.30\n",
      "  269  16280 586F 587A C   N    4608 4610  -74.33  -68.20    9.25    0.58  -62.50  167.30   28.01\n",
      "  269        587A 587A N   CA   4610 4611  152.22  145.30                  -40.90\n",
      "  270  16281 587A 588F C   N    4613 4615 -139.27 -124.20   31.52    1.04  -63.20  163.48   27.56\n",
      "  270        588F 588F N   CA   4615 4616  170.99  143.30                  -44.30\n",
      "  271  16282 588F 589N C   N    4624 4626  -75.06  -71.20   28.68    1.73  -63.20  148.16   19.18\n",
      "  271        589N 589N N   CA   4626 4627  171.22  142.80                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33677  83C 458S SG  CB    631 3562    8.57    3.46    5.11    4.52    3.46    5.11    4.52\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    5  114  153  587  520  762  601  704  759  782  896\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00200.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      590     4639     4639\n",
      "openf___224_> Open           EAO00200.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     69477392   67849.016    66.259\n",
      "read_mo_297_> Segments, residues, atoms:        1      590     4639\n",
      "read_mo_298_> Segment:        1  1:A  590:A     4639\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4639     4639        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    21546\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    22778\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    23393\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    24312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    25407\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    26173\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    27672\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    28645\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    29568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    31057\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    32296\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34090\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34272\n",
      "iupac_m_486_> OE1/2 will be swapped:      151.6522        3  3\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.5188       24  24\n",
      "iupac_m_485_> OD1/2 will be swapped:      118.2548       31  31\n",
      "iupac_m_485_> OD1/2 will be swapped:      161.8842       34  34\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0476       36  36\n",
      "iupac_m_485_> OD1/2 will be swapped:      125.4221       41  41\n",
      "iupac_m_487_> NH1/2 swapped:     -178.8445       46  46\n",
      "iupac_m_485_> OD1/2 will be swapped:       91.5871       54  54\n",
      "iupac_m_485_> OD1/2 will be swapped:     -171.6887       62  62\n",
      "iupac_m_486_> OE1/2 will be swapped:      161.7366       66  66\n",
      "iupac_m_486_> OE1/2 will be swapped:      164.5519      106  106\n",
      "iupac_m_487_> NH1/2 swapped:      179.0075      113  113\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.4789      120  120\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.7508      121  121\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4412      125  125\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -139.8246      137  137\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.1850      144  144\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3988      155  155\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7704      157  157\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.6761      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:     -137.6208      168  168\n",
      "iupac_m_487_> NH1/2 swapped:      179.8287      173  173\n",
      "iupac_m_487_> NH1/2 swapped:      178.9107      189  189\n",
      "iupac_m_487_> NH1/2 swapped:      179.1162      233  233\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.8275      236  236\n",
      "iupac_m_486_> OE1/2 will be swapped:     -165.3123      248  248\n",
      "iupac_m_487_> NH1/2 swapped:      178.2619      249  249\n",
      "iupac_m_486_> OE1/2 will be swapped:     -133.8952      254  254\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.6950      262  262\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5521      275  275\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.1144      280  280\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.9690      283  283\n",
      "iupac_m_487_> NH1/2 swapped:      179.6149      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.3195      303  303\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.9634      326  326\n",
      "iupac_m_486_> OE1/2 will be swapped:     -147.7195      332  332\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.4723      348  348\n",
      "iupac_m_487_> NH1/2 swapped:      179.9287      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.4218      360  360\n",
      "iupac_m_487_> NH1/2 swapped:      178.5443      368  368\n",
      "iupac_m_487_> NH1/2 swapped:      177.6008      378  378\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.2188      381  381\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9822      385  385\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.3240      389  389\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.3037      391  391\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -124.4331      399  399\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.3632      405  405\n",
      "iupac_m_486_> OE1/2 will be swapped:      143.9960      411  411\n",
      "iupac_m_487_> NH1/2 swapped:      175.7608      418  418\n",
      "iupac_m_487_> NH1/2 swapped:     -177.8773      425  425\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7626      426  426\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.3382      433  433\n",
      "iupac_m_485_> OD1/2 will be swapped:      164.3481      440  440\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -99.8878      445  445\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      112.5434      448  448\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.7410      457  457\n",
      "iupac_m_487_> NH1/2 swapped:      148.1998      464  464\n",
      "iupac_m_487_> NH1/2 swapped:      179.5743      465  465\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.9802      491  491\n",
      "iupac_m_487_> NH1/2 swapped:      179.2931      497  497\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.6478      499  499\n",
      "iupac_m_487_> NH1/2 swapped:      170.7204      538  538\n",
      "iupac_m_486_> OE1/2 will be swapped:     -161.3455      540  540\n",
      "iupac_m_487_> NH1/2 swapped:     -174.2280      541  541\n",
      "iupac_m_487_> NH1/2 swapped:      177.7662      555  555\n",
      "iupac_m_486_> OE1/2 will be swapped:     -122.1788      566  566\n",
      "iupac_m_486_> OE1/2 will be swapped:       97.1795      570  570\n",
      "iupac_m_486_> OE1/2 will be swapped:     -135.2728      577  577\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.7007      581  581\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):  1022275\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -41338.1992\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -41338.199219\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    31.629000\n",
      "Sequence length          :          590\n",
      "Compactness              :     0.189667\n",
      "Native energy (pair)     :   820.874905\n",
      "Native energy (surface)  :    82.632144\n",
      "Native energy (combined) :    23.276974\n",
      "Z score (pair)           :    -2.947036\n",
      "Z score (surface)        :    -1.419067\n",
      "Z score (combined)       :    -2.876250\n",
      "GA341 score              :     0.366723\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    12462\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        6887.0449\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4711       0      0   0.007   0.007      65.020       1.000\n",
      " 2 Bond angle potential               :    6336       9     63   2.424   2.424      717.51       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2751       0    148  52.458  52.458      1224.7       1.000\n",
      " 4 Stereochemical improper torsion pot:    1896       1      3   1.611   1.611      99.734       1.000\n",
      " 5 Soft-sphere overlap restraints     :   12462       0      0   0.007   0.007      70.575       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6251      11    131   1.288   1.288      1157.8       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6377      17    140   1.473   1.473      1317.3       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     589       6     25   6.601   6.601      301.43       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     499       0     10  85.570  85.570      203.98       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     371       0      7  87.049  87.049      210.19       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     179       0      2  94.547  94.547      137.83       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      77       0      0  96.170  96.170      58.028       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2718       0      0   0.700   0.700      78.279       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     588     268    143  42.851 116.280      1063.4       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     929       0     11   2.804   2.804      181.29       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00200.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00200.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  103448.5469\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10078 506K 506K N   CA   3945 3946  123.12  107.00   16.12    4.64  107.00   16.12    4.64\n",
      "    2  10422 537D 538R C   N    4192 4194  140.02  120.00   20.02    4.55  120.00   20.02    4.55\n",
      "    3  10425 538R 538R N   CA   4194 4195  132.16  107.00   25.17    7.24  107.00   25.17    7.24\n",
      "    4  10839 573R 573R N   CA   4489 4490  123.42  107.00   16.42    4.72  107.00   16.42    4.72\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  18039   8A  75E CA  CA     48  565   16.59   13.11    3.48    4.66   13.11    3.48    4.66\n",
      "    2  19225  53F 570E CA  CA    386 4466   10.68    7.50    3.18    4.53    7.50    3.18    4.53\n",
      "    3  21509 137F 220Q CA  CA   1039 1698    5.62    3.85    1.77    4.58    3.85    1.77    4.58\n",
      "    4  23610 359R 398A CA  CA   2761 3085   15.79   10.50    5.29    4.53   10.50    5.29    4.53\n",
      "    5  23642 360E 397V CA  CA   2772 3078   16.78    7.52    9.27    4.50    7.52    9.27    4.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  24269   8A  81N N   O      47  620   13.78   10.46    3.33    4.80   10.46    3.33    4.80\n",
      "    2  25712  81N   8A N   O     613   51   14.47   10.97    3.50    4.92   10.97    3.50    4.92\n",
      "    3  26886 138V 219V N   O    1049 1696    5.33    3.07    2.26    6.27    3.07    2.26    6.27\n",
      "    4  28079 277M 283R N   O    2132 2189   16.13   10.79    5.34    4.54   10.79    5.34    4.54\n",
      "    5  28976 342N 144Y N   O    2620 1106   10.94    6.52    4.43    5.77    6.52    4.43    5.77\n",
      "    6  29346 397V 359R N   O    3077 2770   19.04   10.54    8.50    4.88   10.54    8.50    4.88\n",
      "    7  29347 397V 360E N   O    3077 2779   17.39    7.05   10.34    4.78    7.05   10.34    4.78\n",
      "    8  29362 398A 359R N   O    3084 2770   16.20    9.28    6.92    5.68    9.28    6.92    5.68\n",
      "    9  29363 398A 360E N   O    3084 2779   14.24    6.19    8.05    4.90    6.19    8.05    4.90\n",
      "   10  30272 506K 476G N   O    3945 3709   15.44   10.57    4.87    4.98   10.57    4.87    4.98\n",
      "   11  30273 506K 477V N   O    3945 3716   16.28   10.89    5.39    5.13   10.89    5.39    5.13\n",
      "   12  30274 506K 478L N   O    3945 3724   12.02    6.32    5.70    4.85    6.32    5.70    4.85\n",
      "   13  30424 537D  87Y N   O    4186  669   13.85   10.32    3.53    4.87   10.32    3.53    4.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16500 218I 218I CA  C    1683 1688 -157.11 -180.00   22.89    4.58 -180.00   22.89    4.58\n",
      "    2  16681 399F 399F CA  C    3090 3098 -124.52 -180.00   55.48   11.09 -180.00   55.48   11.09\n",
      "    3  16682 400A 400A CA  C    3101 3103 -148.07 -180.00   31.93    6.38 -180.00   31.93    6.38\n",
      "    4  16788 506K 506K CA  C    3946 3952 -132.63 -180.00   47.37    9.47 -180.00   47.37    9.47\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15696   2A   3E C   N      12   14 -150.46 -117.80   39.09    1.97  -63.60  178.22   20.60\n",
      "    1          3E   3E N   CA     14   15  115.31  136.80                  -40.30\n",
      "    2  15697   3E   4L C   N      21   23 -153.41 -108.50   70.10    3.21  -63.50  160.10   27.71\n",
      "    2          4L   4L N   CA     23   24 -173.67  132.50                  -41.20\n",
      "    3  15698   4L   5A C   N      29   31  -84.15  -68.20   18.97    1.25  -62.50  164.97   28.15\n",
      "    3          5A   5A N   CA     31   32  155.56  145.30                  -40.90\n",
      "    4  15700   6G   7T C   N      38   40 -163.52 -124.80   66.00    3.84  -63.20  165.92   17.63\n",
      "    4          7T   7T N   CA     40   41   90.05  143.50                  -42.10\n",
      "    5  15701   7T   8A C   N      45   47   94.10 -134.00  137.49    6.06  -68.20  166.48   12.79\n",
      "    5          8A   8A N   CA     47   48  108.21  147.00                  145.30\n",
      "    6  15710  16R  17M C   N     111  113  -73.34  -73.00   28.80    1.94  -63.40  148.03   22.89\n",
      "    6         17M  17M N   CA    113  114  171.80  143.00                  -40.50\n",
      "    7  15714  20P  21R C   N     139  141 -130.18 -125.20   80.27    3.98  -63.00  121.78   13.80\n",
      "    7         21R  21R N   CA    141  142   60.48  140.60                  -41.10\n",
      "    8  15720  26V  27I C   N     188  190 -125.53 -120.60   94.22    5.58  -63.40  101.15   15.40\n",
      "    8         27I  27I N   CA    190  191   36.21  130.30                  -43.60\n",
      "    9  15737  43E  44G C   N     318  320   57.78   78.70   85.15    1.61  -62.40  127.42   20.67\n",
      "    9         44G  44G N   CA    320  321  -83.56 -166.10                  -41.20\n",
      "   10  15752  58T  59G C   N     438  440 -174.94 -167.20   11.90    0.29   82.20 -156.89   14.43\n",
      "   10         59G  59G N   CA    440  441 -176.36  174.60                    8.50\n",
      "   11  15783  89Q  90T C   N     681  683  -56.34  -63.20   21.39    2.36  -78.10  149.43    7.12\n",
      "   11         90T  90T N   CA    683  684  -62.37  -42.10                  149.80\n",
      "   12  15784  90T  91G C   N     688  690  -64.44  -62.40   31.32    4.58   82.20  147.80   10.53\n",
      "   12         91G  91G N   CA    690  691   -9.94  -41.20                    8.50\n",
      "   13  15795 101D 102L C   N     765  767  -64.92  -63.50    1.46    0.26 -108.50  179.43    9.18\n",
      "   13        102L 102L N   CA    767  768  -41.55  -41.20                  132.50\n",
      "   14  15798 104S 105K C   N     787  789  -57.59  -70.20   19.12    1.14   56.60  143.82   10.56\n",
      "   14        105K 105K N   CA    789  790  126.03  140.40                   38.60\n",
      "   15  15799 105K 106E C   N     796  798   69.50   54.60   51.88    2.93  -63.60  137.14   23.00\n",
      "   15        106E 106E N   CA    798  799   -7.29   42.40                  -40.30\n",
      "   16  15800 106E 107G C   N     805  807   79.40   78.70   29.91    0.82  -62.40 -150.08   29.31\n",
      "   16        107G 107G N   CA    807  808  164.00 -166.10                  -41.20\n",
      "   17  15813 119I 120E C   N     904  906  -57.61  -69.30   32.94    2.87  -63.60  146.52   19.22\n",
      "   17        120E 120E N   CA    906  907  173.30  142.50                  -40.30\n",
      "   18  15838 144Y 145M C   N    1105 1107   75.31  -73.00  165.16   13.22   56.30 -179.85    8.37\n",
      "   18        145M 145M N   CA   1107 1108 -144.34  143.00                   34.80\n",
      "   19  15839 145M 146E C   N    1113 1115  -83.64  -63.60   25.22    3.13   54.60  153.79   20.14\n",
      "   19        146E 146E N   CA   1115 1116  -24.99  -40.30                   42.40\n",
      "   20  15849 155R 156A C   N    1196 1198  -74.04  -68.20   28.08    2.06  -62.50  146.79   24.64\n",
      "   20        156A 156A N   CA   1198 1199  172.77  145.30                  -40.90\n",
      "   21  15852 158G 159Q C   N    1216 1218  -92.29  -73.00   79.66    5.60  -63.80  107.56   14.48\n",
      "   21        159Q 159Q N   CA   1218 1219   63.42  140.70                  -40.30\n",
      "   22  15853 159Q 160E C   N    1225 1227 -145.26 -117.80   44.77    1.55  -63.60  168.63   27.01\n",
      "   22        160E 160E N   CA   1227 1228  172.16  136.80                  -40.30\n",
      "   23  15855 161P 162E C   N    1241 1243   39.23  -69.30  120.17    7.29  -63.60  166.70   27.71\n",
      "   23        162E 162E N   CA   1243 1244   90.90  142.50                  -40.30\n",
      "   24  15856 162E 163S C   N    1250 1252 -143.32 -136.60   13.03    0.49  -64.10 -179.10   18.78\n",
      "   24        163S 163S N   CA   1252 1253  162.36  151.20                  -35.00\n",
      "   25  15859 165E 166I C   N    1273 1275   31.05  -97.30  128.64    5.53 -120.60  152.10   10.17\n",
      "   25        166I 166I N   CA   1275 1276  118.64  127.20                  130.30\n",
      "   26  15863 169D 170S C   N    1308 1310 -126.24 -136.60   16.83    0.95  -64.10  172.14   16.86\n",
      "   26        170S 170S N   CA   1310 1311  164.46  151.20                  -35.00\n",
      "   27  15864 170S 171Q C   N    1314 1316 -154.51 -121.10   55.66    1.94  -63.80  163.04   28.54\n",
      "   27        171Q 171Q N   CA   1316 1317 -175.78  139.70                  -40.30\n",
      "   28  15865 171Q 172H C   N    1323 1325  -62.63  -67.60    6.64    0.60  -63.20  173.29   21.63\n",
      "   28        172H 172H N   CA   1325 1326  144.41  140.00                  -42.30\n",
      "   29  15866 172H 173R C   N    1333 1335  -82.21  -72.10   57.44    4.63  -63.00  127.90   16.07\n",
      "   29        173R 173R N   CA   1335 1336   85.35  141.90                  -41.10\n",
      "   30  15868 174V 175Y C   N    1351 1353 -115.57  -98.40   28.27    2.40  -63.50  158.16   22.71\n",
      "   30        175Y 175Y N   CA   1353 1354  105.94  128.40                  -43.40\n",
      "   31  15870 176V 177K C   N    1370 1372  -70.64  -70.20    0.65    0.05  -62.90  179.44   23.73\n",
      "   31        177K 177K N   CA   1372 1373  139.93  140.40                  -40.80\n",
      "   32  15871 177K 178G C   N    1379 1381 -115.81  -80.20   42.88    3.15   82.20 -144.78   16.67\n",
      "   32        178G 178G N   CA   1381 1382  150.20  174.10                    8.50\n",
      "   33  15872 178G 179A C   N    1383 1385 -132.70 -134.00    3.73    0.23  -62.50 -177.36   33.69\n",
      "   33        179A 179A N   CA   1385 1386  150.49  147.00                  -40.90\n",
      "   34  15875 181V 182H C   N    1399 1401 -111.19 -125.60   47.47    1.51  -63.20  144.10   15.67\n",
      "   34        182H 182H N   CA   1401 1402   93.57  138.80                  -42.30\n",
      "   35  15876 182H 183S C   N    1409 1411 -149.36 -136.60   15.43    0.47  -64.10 -174.17   19.58\n",
      "   35        183S 183S N   CA   1411 1412  159.88  151.20                  -35.00\n",
      "   36  15879 185L 186S C   N    1430 1432 -135.53 -136.60    3.34    0.14  -64.10 -169.16   18.89\n",
      "   36        186S 186S N   CA   1432 1433  148.03  151.20                  -35.00\n",
      "   37  15881 187L 188E C   N    1444 1446  122.83 -117.80  133.74    3.91  -63.60 -147.50   36.52\n",
      "   37        188E 188E N   CA   1446 1447 -162.89  136.80                  -40.30\n",
      "   38  15889 195K 196K C   N    1514 1516  -71.70  -62.90   17.93    2.06 -118.00  170.68    7.54\n",
      "   38        196K 196K N   CA   1516 1517  -25.17  -40.80                  139.10\n",
      "   39  15891 197G 198N C   N    1527 1529 -122.17 -119.90   26.34    1.25  -63.20  162.90   17.83\n",
      "   39        198N 198N N   CA   1529 1530  110.76  137.00                  -41.10\n",
      "   40  15892 198N 199A C   N    1535 1537 -161.99 -134.00   29.06    0.71  -62.50 -167.95   36.77\n",
      "   40        199A 199A N   CA   1537 1538  154.82  147.00                  -40.90\n",
      "   41  15893 199A 200N C   N    1540 1542  -99.78 -119.90   51.11    1.81  -63.20  136.12   15.30\n",
      "   41        200N 200N N   CA   1542 1543   90.01  137.00                  -41.10\n",
      "   42  15894 200N 201R C   N    1548 1550 -146.14 -125.20   23.47    0.71  -63.00 -172.83   30.94\n",
      "   42        201R 201R N   CA   1550 1551  151.21  140.60                  -41.10\n",
      "   43  15895 201R 202Q C   N    1559 1561 -124.22 -121.10    5.16    0.27  -63.80 -174.02   24.64\n",
      "   43        202Q 202Q N   CA   1561 1562  135.59  139.70                  -40.30\n",
      "   44  15896 202Q 203T C   N    1568 1570 -108.06 -124.80   17.28    0.80  -63.20 -175.77   26.62\n",
      "   44        203T 203T N   CA   1570 1571  139.22  143.50                  -42.10\n",
      "   45  15897 203T 204G C   N    1575 1577   68.40   78.70   16.48    0.31   82.20  162.33    8.57\n",
      "   45        204G 204G N   CA   1577 1578 -153.24 -166.10                    8.50\n",
      "   46  15898 204G 205E C   N    1579 1581 -123.92 -117.80    9.80    0.34  -63.60 -174.67   28.12\n",
      "   46        205E 205E N   CA   1581 1582  144.46  136.80                  -40.30\n",
      "   47  15899 205E 206T C   N    1588 1590 -106.21 -124.80   28.36    0.89  -63.20  169.73   19.24\n",
      "   47        206T 206T N   CA   1590 1591  122.08  143.50                  -42.10\n",
      "   48  15901 207K 208M C   N    1604 1606 -121.31 -125.60    9.38    0.51  -63.40 -179.78   31.52\n",
      "   48        208M 208M N   CA   1606 1607  148.84  140.50                  -40.50\n",
      "   49  15902 208M 209N C   N    1612 1614  -86.72  -71.20   15.93    1.23  -63.20 -178.79   23.98\n",
      "   49        209N 209N N   CA   1614 1615  139.22  142.80                  -41.10\n",
      "   50  15903 209N 210E C   N    1620 1622  -77.47  -69.30    8.17    0.63  -63.60  178.04   24.51\n",
      "   50        210E 210E N   CA   1622 1623  142.20  142.50                  -40.30\n",
      "   51  15905 211T 212S C   N    1636 1638  -67.16  -72.40    5.39    0.34  -64.10  173.89   12.78\n",
      "   51        212S 212S N   CA   1638 1639  151.14  152.40                  -35.00\n",
      "   52  15906 212S 213S C   N    1642 1644  -87.65  -72.40   15.29    1.06  -64.10  173.06   14.07\n",
      "   52        213S 213S N   CA   1644 1645  153.55  152.40                  -35.00\n",
      "   53  15908 214R 215S C   N    1659 1661  -58.82  -72.40   18.98    1.51  -64.10  159.43   11.25\n",
      "   53        215S 215S N   CA   1661 1662  165.66  152.40                  -35.00\n",
      "   54  15909 215S 216H C   N    1665 1667   56.25   56.30    4.70    0.41  -63.20  148.25   25.34\n",
      "   54        216H 216H N   CA   1667 1668   45.50   40.80                  -42.30\n",
      "   55  15910 216H 217A C   N    1675 1677  -96.25  -68.20   30.22    2.88  -62.50  178.17   27.74\n",
      "   55        217A 217A N   CA   1677 1678  134.04  145.30                  -40.90\n",
      "   56  15912 218I 219V C   N    1688 1690   53.38  -73.50  163.71   13.73 -125.40 -155.47   10.79\n",
      "   56        219V 219V N   CA   1690 1691 -117.35  139.20                  143.30\n",
      "   57  15913 219V 220Q C   N    1695 1697  171.39 -121.10   91.80    4.67  -73.00  131.76   10.24\n",
      "   57        220Q 220Q N   CA   1697 1698   77.49  139.70                  140.70\n",
      "   58  15916 222T 223I C   N    1719 1721   -0.44  -97.30   96.91    4.09 -120.60  120.33    8.15\n",
      "   58        223I 223I N   CA   1721 1722  124.12  127.20                  130.30\n",
      "   59  15917 223I 224S C   N    1727 1729  134.11 -136.60  152.82    5.41  -64.10  169.27   22.64\n",
      "   59        224S 224S N   CA   1729 1730  -84.78  151.20                  -35.00\n",
      "   60  15921 227Y 228G C   N    1763 1765  117.23   82.20   51.36    1.55  -62.40 -179.96   32.63\n",
      "   60        228G 228G N   CA   1765 1766  -29.06    8.50                  -41.20\n",
      "   61  15923 229S 230L C   N    1773 1775  -76.05  -70.70   34.50    2.42  -63.50  143.67   20.69\n",
      "   61        230L 230L N   CA   1775 1776  175.68  141.60                  -41.20\n",
      "   62  15926 232M 233R C   N    1797 1799   62.71   57.30    8.26    0.49  -63.00  145.29   26.49\n",
      "   62        233R 233R N   CA   1799 1800   31.75   38.00                  -41.10\n",
      "   63  15930 236E 237S C   N    1832 1834   92.02 -136.60  132.08    4.90  -64.10 -136.30   17.91\n",
      "   63        237S 237S N   CA   1834 1835  164.79  151.20                  -35.00\n",
      "   64  15932 238V 239V C   N    1845 1847  -59.20  -62.40    3.47    0.59 -125.40 -172.29   10.55\n",
      "   64        239V 239V N   CA   1847 1848  -41.04  -42.40                  143.30\n",
      "   65  15934 240L 241L C   N    1860 1862  -90.88  -70.70   39.85    2.47  -63.50  145.43   21.85\n",
      "   65        241L 241L N   CA   1862 1863  175.97  141.60                  -41.20\n",
      "   66  15936 242V 243D C   N    1875 1877  -67.14  -63.30   52.01    6.66  -63.30   52.01    6.66\n",
      "   66        243D 243D N   CA   1877 1878  -91.87  -40.00                  -40.00\n",
      "   67  15942 248E 249R C   N    1915 1917  -69.55 -125.20   62.41    3.18  -63.00  150.21   20.73\n",
      "   67        249R 249R N   CA   1917 1918  168.84  140.60                  -41.10\n",
      "   68  15943 249R 250Q C   N    1926 1928   75.80  -63.80  139.62   23.37  -63.80  139.62   23.37\n",
      "   68        250Q 250Q N   CA   1928 1929  -38.11  -40.30                  -40.30\n",
      "   69  15947 253T 254E C   N    1957 1959   51.51   54.60   11.78    0.68  -63.60  148.66   25.45\n",
      "   69        254E 254E N   CA   1959 1960   53.77   42.40                  -40.30\n",
      "   70  15948 254E 255S C   N    1966 1968  -61.44  -72.40   18.36    1.42  -64.10  157.89   11.28\n",
      "   70        255S 255S N   CA   1968 1969  167.13  152.40                  -35.00\n",
      "   71  15949 255S 256T C   N    1972 1974  -58.60  -78.10   38.59    1.26  -63.20  158.66   20.65\n",
      "   71        256T 256T N   CA   1974 1975  116.50  149.80                  -42.10\n",
      "   72  15950 256T 257G C   N    1979 1981  -97.05  -80.20   34.63    0.81   82.20 -116.94   18.77\n",
      "   72        257G 257G N   CA   1981 1982 -155.65  174.10                    8.50\n",
      "   73  15952 258V 259A C   N    1990 1992  -76.24  -68.20    9.53    0.63  -62.50  169.24   28.43\n",
      "   73        259A 259A N   CA   1992 1993  150.42  145.30                  -40.90\n",
      "   74  15953 259A 260F C   N    1995 1997  -76.30  -71.40    7.46    0.66  -63.20  179.85   24.10\n",
      "   74        260F 260F N   CA   1997 1998  135.07  140.70                  -44.30\n",
      "   75  15954 260F 261E C   N    2006 2008  -76.13  -69.30    7.59    0.67  -63.60  179.92   23.32\n",
      "   75        261E 261E N   CA   2008 2009  139.18  142.50                  -40.30\n",
      "   76  15955 261E 262E C   N    2015 2017 -113.79 -117.80   13.41    0.73  -63.60  177.35   26.49\n",
      "   76        262E 262E N   CA   2017 2018  149.60  136.80                  -40.30\n",
      "   77  15956 262E 263A C   N    2024 2026  -93.84  -68.20   38.45    3.78  -62.50  160.63   24.98\n",
      "   77        263A 263A N   CA   2026 2027  116.64  145.30                  -40.90\n",
      "   78  15957 263A 264K C   N    2029 2031 -158.69 -118.00   46.03    1.52  -62.90 -174.73   29.76\n",
      "   78        264K 264K N   CA   2031 2032  160.61  139.10                  -40.80\n",
      "   79  15958 264K 265K C   N    2038 2040  -90.35  -70.20   32.95    2.74  -62.90  157.54   19.12\n",
      "   79        265K 265K N   CA   2040 2041  114.33  140.40                  -40.80\n",
      "   80  15961 267N 268Q C   N    2063 2065 -148.60 -121.10  109.30    4.68  -63.80  112.69   20.39\n",
      "   80        268Q 268Q N   CA   2065 2066 -114.52  139.70                  -40.30\n",
      "   81  15962 268Q 269S C   N    2072 2074  -54.42  -72.40   18.35    1.20  -64.10  176.53   12.25\n",
      "   81        269S 269S N   CA   2074 2075  148.73  152.40                  -35.00\n",
      "   82  15963 269S 270L C   N    2078 2080  -71.48  -70.70   11.18    0.88  -63.50  171.83   23.41\n",
      "   82        270L 270L N   CA   2080 2081  130.45  141.60                  -41.20\n",
      "   83  15964 270L 271L C   N    2086 2088 -129.75 -108.50   24.50    1.32  -63.50  174.57   21.53\n",
      "   83        271L 271L N   CA   2088 2089  120.31  132.50                  -41.20\n",
      "   84  15965 271L 272M C   N    2094 2096 -141.68 -125.60   24.67    0.78  -63.40  178.39   32.78\n",
      "   84        272M 272M N   CA   2096 2097  159.20  140.50                  -40.50\n",
      "   85  15966 272M 273L C   N    2102 2104  -79.86  -70.70    9.61    0.77  -63.50  175.07   25.28\n",
      "   85        273L 273L N   CA   2104 2105  144.50  141.60                  -41.20\n",
      "   86  15969 275R 276A C   N    2125 2127  -62.60  -68.20   21.92    1.57  -62.50  165.01   27.04\n",
      "   86        276A 276A N   CA   2127 2128  124.11  145.30                  -40.90\n",
      "   87  15970 276A 277M C   N    2130 2132 -154.26 -125.60   37.95    2.03  -73.00   85.74    6.81\n",
      "   87        277M 277M N   CA   2132 2133  115.63  140.50                  143.00\n",
      "   88  15972 278N 279S C   N    2146 2148  -74.44  -72.40   31.82    1.80  -64.10  141.22   10.88\n",
      "   88        279S 279S N   CA   2148 2149 -175.85  152.40                  -35.00\n",
      "   89  15992 298T 299R C   N    2297 2299 -127.24  -63.00   71.99    9.81  -63.00   71.99    9.81\n",
      "   89        299R 299R N   CA   2299 2300   -8.60  -41.10                  -41.10\n",
      "   90  15993 299R 300L C   N    2308 2310  -65.95  -70.70    9.51    0.59  -63.50  174.58   24.10\n",
      "   90        300L 300L N   CA   2310 2311  133.36  141.60                  -41.20\n",
      "   91  15994 300L 301L C   N    2316 2318 -100.75 -108.50   11.11    0.50  -63.50  169.87   21.75\n",
      "   91        301L 301L N   CA   2318 2319  124.54  132.50                  -41.20\n",
      "   92  15995 301L 302S C   N    2324 2326 -126.24 -136.60   11.50    0.58  -64.10  179.89   17.41\n",
      "   92        302S 302S N   CA   2326 2327  156.18  151.20                  -35.00\n",
      "   93  15996 302S 303E C   N    2330 2332  -62.20  -69.30   24.65    1.63  -63.60  159.20   21.30\n",
      "   93        303E 303E N   CA   2332 2333  118.89  142.50                  -40.30\n",
      "   94  15997 303E 304S C   N    2339 2341  -88.75  -72.40   26.76    1.23  -64.10  153.40   12.74\n",
      "   94        304S 304S N   CA   2341 2342  173.59  152.40                  -35.00\n",
      "   95  15998 304S 305F C   N    2345 2347  -90.69  -71.40   42.72    3.79  -63.20  149.43   18.98\n",
      "   95        305F 305F N   CA   2347 2348  102.58  140.70                  -44.30\n",
      "   96  15999 305F 306G C   N    2356 2358   72.48   78.70   13.40    0.62   82.20  173.80    7.95\n",
      "   96        306G 306G N   CA   2358 2359 -177.97 -166.10                    8.50\n",
      "   97  16000 306G 307G C   N    2360 2362   81.57   78.70   20.61    0.47   82.20  164.99    7.98\n",
      "   97        307G 307G N   CA   2362 2363  173.49 -166.10                    8.50\n",
      "   98  16001 307G 308N C   N    2364 2366 -148.29 -119.90   54.73    1.77  -63.20  159.68   24.67\n",
      "   98        308N 308N N   CA   2366 2367 -176.21  137.00                  -41.10\n",
      "   99  16002 308N 309S C   N    2372 2374  -62.76  -72.40   11.59    0.60  -64.10  179.04   12.89\n",
      "   99        309S 309S N   CA   2374 2375  145.96  152.40                  -35.00\n",
      "  100  16003 309S 310K C   N    2378 2380 -125.20 -118.00    9.55    0.47  -62.90 -175.53   21.43\n",
      "  100        310K 310K N   CA   2380 2381  132.83  139.10                  -40.80\n",
      "  101  16005 311T 312W C   N    2394 2396 -158.98 -124.90   35.20    1.33  -63.00 -170.32   30.17\n",
      "  101        312W 312W N   CA   2396 2397  152.19  143.40                  -44.20\n",
      "  102  16006 312W 313M C   N    2408 2410 -152.30 -125.60   43.96    1.44  -63.40  169.30   32.11\n",
      "  102        313M 313M N   CA   2410 2411  175.42  140.50                  -40.50\n",
      "  103  16007 313M 314L C   N    2416 2418  -71.46  -70.70    1.31    0.08  -63.50  176.31   24.94\n",
      "  103        314L 314L N   CA   2418 2419  142.66  141.60                  -41.20\n",
      "  104  16008 314L 315A C   N    2424 2426  -81.77  -68.20   19.40    1.22  -62.50  161.09   27.39\n",
      "  104        315A 315A N   CA   2426 2427  159.17  145.30                  -40.90\n",
      "  105  16009 315A 316T C   N    2429 2431 -121.05 -124.80   17.47    0.86  -63.20  167.64   25.32\n",
      "  105        316T 316T N   CA   2431 2432  160.56  143.50                  -42.10\n",
      "  106  16010 316T 317V C   N    2436 2438  -62.72  -62.40    1.86    0.22 -125.40 -173.05   10.47\n",
      "  106        317V 317V N   CA   2438 2439  -40.57  -42.40                  143.30\n",
      "  107  16012 318S 319T C   N    2449 2451 -150.59 -124.80   36.94    1.19  -63.20  171.84   27.62\n",
      "  107        319T 319T N   CA   2451 2452  169.95  143.50                  -42.10\n",
      "  108  16015 321A 322N C   N    2466 2468  -67.92  -63.20    8.16    1.28   55.90  151.47   18.52\n",
      "  108        322N 322N N   CA   2468 2469  -47.75  -41.10                   39.50\n",
      "  109  16033 339A 340I C   N    2603 2605  -66.19  -63.40    3.51    0.55 -120.60 -179.81    8.31\n",
      "  109        340I 340I N   CA   2605 2606  -41.47  -43.60                  130.30\n",
      "  110  16035 341T 342N C   N    2618 2620  -86.47 -119.90   37.64    1.94  -63.20  166.24   22.10\n",
      "  110        342N 342N N   CA   2620 2621  154.29  137.00                  -41.10\n",
      "  111  16038 344A 345K C   N    2640 2642   71.21   56.60   22.62    1.32  -62.90  147.80   25.81\n",
      "  111        345K 345K N   CA   2642 2643   21.33   38.60                  -40.80\n",
      "  112  16040 346V 347N C   N    2656 2658  -82.93 -119.90   66.79    2.11   55.90  145.00   13.84\n",
      "  112        347N 347N N   CA   2658 2659   81.38  137.00                   39.50\n",
      "  113  16043 349T 350K C   N    2679 2681   37.45  -70.20  109.27    8.40  -62.90 -171.13   21.79\n",
      "  113        350K 350K N   CA   2681 2682  159.19  140.40                  -40.80\n",
      "  114  16044 350K 351K C   N    2688 2690 -116.28 -118.00   65.27    3.15  -62.90  126.48   14.54\n",
      "  114        351K 351K N   CA   2690 2691   73.86  139.10                  -40.80\n",
      "  115  16045 351K 352N C   N    2697 2699 -176.38 -119.90   98.00    3.05  -63.20  152.24   24.77\n",
      "  115        352N 352N N   CA   2699 2700 -142.92  137.00                  -41.10\n",
      "  116  16046 352N 353I C   N    2705 2707  -72.66  -63.40    9.26    1.67 -120.60 -179.25    8.55\n",
      "  116        353I 353I N   CA   2707 2708  -43.97  -43.60                  130.30\n",
      "  117  16047 353I 354E C   N    2713 2715 -166.41 -117.80   50.05    2.14  -63.60 -165.43   22.47\n",
      "  117        354E 354E N   CA   2715 2716  124.89  136.80                  -40.30\n",
      "  118  16048 354E 355L C   N    2722 2724  -76.00  -63.50   47.13    7.31  -63.50   47.13    7.31\n",
      "  118        355L 355L N   CA   2724 2725  -86.64  -41.20                  -41.20\n",
      "  119  16056 362V 363A C   N    2793 2795 -177.41 -134.00   58.52    1.66  -68.20  116.63    8.42\n",
      "  119        363A 363A N   CA   2795 2796 -173.77  147.00                  145.30\n",
      "  120  16057 363A 364S C   N    2798 2800  -87.67  -72.40   15.61    1.01  -64.10  171.01   13.93\n",
      "  120        364S 364S N   CA   2800 2801  155.62  152.40                  -35.00\n",
      "  121  16058 364S 365L C   N    2804 2806 -143.55 -108.50   37.57    1.67  -63.50 -169.59   31.39\n",
      "  121        365L 365L N   CA   2806 2807  146.03  132.50                  -41.20\n",
      "  122  16059 365L 366E C   N    2812 2814  -58.51  -69.30   10.91    0.77  -63.60  178.91   23.58\n",
      "  122        366E 366E N   CA   2814 2815  140.86  142.50                  -40.30\n",
      "  123  16061 367G 368R C   N    2825 2827  -62.14  -72.10   15.15    1.32  -63.00  165.60   22.31\n",
      "  123        368R 368R N   CA   2827 2828  153.31  141.90                  -41.10\n",
      "  124  16062 368R 369L C   N    2836 2838  -63.75  -70.70    8.52    0.58  -63.50  177.86   24.68\n",
      "  124        369L 369L N   CA   2838 2839  136.66  141.60                  -41.20\n",
      "  125  16063 369L 370D C   N    2844 2846 -133.54  -96.50   64.46    2.71  -63.30  168.39   25.32\n",
      "  125        370D 370D N   CA   2846 2847  166.96  114.20                  -40.00\n",
      "  126  16068 374L 375E C   N    2880 2882  -50.67  -69.30   21.86    1.97  -63.60  166.27   21.49\n",
      "  126        375E 375E N   CA   2882 2883  153.93  142.50                  -40.30\n",
      "  127  16069 375E 376K C   N    2889 2891  -55.82  -70.20   17.46    1.06  -62.90  171.44   22.66\n",
      "  127        376K 376K N   CA   2891 2892  130.49  140.40                  -40.80\n",
      "  128  16070 376K 377Q C   N    2898 2900 -167.40 -121.10   62.23    1.98  -63.80  172.91   30.55\n",
      "  128        377Q 377Q N   CA   2900 2901 -178.73  139.70                  -40.30\n",
      "  129  16071 377Q 378R C   N    2907 2909 -138.33 -125.20   18.95    0.58  -63.00 -178.95   29.61\n",
      "  129        378R 378R N   CA   2909 2910  154.26  140.60                  -41.10\n",
      "  130  16072 378R 379K C   N    2918 2920 -157.92 -118.00   40.78    1.42  -62.90 -163.70   31.16\n",
      "  130        379K 379K N   CA   2920 2921  147.43  139.10                  -40.80\n",
      "  131  16073 379K 380Q C   N    2927 2929 -161.38 -121.10   40.72    1.66  -63.80 -160.46   25.66\n",
      "  131        380Q 380Q N   CA   2929 2930  133.75  139.70                  -40.30\n",
      "  132  16074 380Q 381E C   N    2936 2938 -101.67 -117.80   40.75    2.23  -63.60  150.37   22.20\n",
      "  132        381E 381E N   CA   2938 2939  174.22  136.80                  -40.30\n",
      "  133  16075 381E 382E C   N    2945 2947 -127.60 -117.80   19.02    0.72  -63.60  178.46   27.41\n",
      "  133        382E 382E N   CA   2947 2948  153.10  136.80                  -40.30\n",
      "  134  16077 383I 384G C   N    2962 2964   63.81   78.70   28.28    0.44   82.20  151.68    8.28\n",
      "  134        384G 384G N   CA   2964 2965 -142.06 -166.10                    8.50\n",
      "  135  16078 384G 385R C   N    2966 2968 -133.37 -125.20   16.16    0.57  -63.00  178.79   28.98\n",
      "  135        385R 385R N   CA   2968 2969  154.54  140.60                  -41.10\n",
      "  136  16079 385R 386L C   N    2977 2979  -72.34  -70.70    3.51    0.33  -63.50  179.92   24.49\n",
      "  136        386L 386L N   CA   2979 2980  138.50  141.60                  -41.20\n",
      "  137  16081 387T 388Q C   N    2992 2994 -166.86 -121.10   46.81    1.57  -63.80 -161.07   34.51\n",
      "  137        388Q 388Q N   CA   2994 2995  149.54  139.70                  -40.30\n",
      "  138  16082 388Q 389E C   N    3001 3003  -78.52  -69.30   11.78    0.66  -63.60  170.52   23.56\n",
      "  138        389E 389E N   CA   3003 3004  149.83  142.50                  -40.30\n",
      "  139  16083 389E 390R C   N    3010 3012  -76.67  -72.10   26.23    1.92  -63.00  151.79   21.44\n",
      "  139        390R 390R N   CA   3012 3013  167.72  141.90                  -41.10\n",
      "  140  16085 391D 392K C   N    3029 3031  -97.54 -118.00   23.30    1.09  -62.90  172.46   24.42\n",
      "  140        392K 392K N   CA   3031 3032  150.26  139.10                  -40.80\n",
      "  141  16086 392K 393L C   N    3038 3040  -72.71  -70.70   16.62    1.34  -63.50  166.56   22.62\n",
      "  141        393L 393L N   CA   3040 3041  125.11  141.60                  -41.20\n",
      "  142  16087 393L 394R C   N    3046 3048 -132.06 -125.20   37.36    1.94  -63.00  160.58   18.31\n",
      "  142        394R 394R N   CA   3048 3049  103.87  140.60                  -41.10\n",
      "  143  16088 394R 395Q C   N    3057 3059 -150.38 -121.10   36.52    1.14  -63.80 -179.69   30.98\n",
      "  143        395Q 395Q N   CA   3059 3060  161.53  139.70                  -40.30\n",
      "  144  16089 395Q 396E C   N    3066 3068 -149.89 -117.80   60.50    2.25  -63.60  157.38   25.69\n",
      "  144        396E 396E N   CA   3068 3069 -171.91  136.80                  -40.30\n",
      "  145  16091 397V 398A C   N    3082 3084 -129.43 -134.00   64.25    3.54  -68.20   87.42    8.61\n",
      "  145        398A 398A N   CA   3084 3085   82.91  147.00                  145.30\n",
      "  146  16093 399F 400A C   N    3098 3100  -48.34  -62.50   59.27    9.13  -62.50   59.27    9.13\n",
      "  146        400A 400A N   CA   3100 3101  -98.46  -40.90                  -40.90\n",
      "  147  16094 400A 401D C   N    3103 3105   33.63  -63.30  131.11   21.97  -63.30  131.11   21.97\n",
      "  147        401D 401D N   CA   3105 3106   48.29  -40.00                  -40.00\n",
      "  148  16096 402S 403V C   N    3117 3119  -64.54  -62.40   59.31    7.63  -62.40   59.31    7.63\n",
      "  148        403V 403V N   CA   3119 3120 -101.67  -42.40                  -42.40\n",
      "  149  16097 403V 404H C   N    3124 3126  107.53 -125.60  144.37    3.73  -63.20 -156.90   25.13\n",
      "  149        404H 404H N   CA   3126 3127 -152.30  138.80                  -42.30\n",
      "  150  16098 404H 405D C   N    3134 3136  -92.26  -70.90   23.68    1.99  -63.30 -177.77   24.33\n",
      "  150        405D 405D N   CA   3136 3137  140.09  150.30                  -40.00\n",
      "  151  16099 405D 406A C   N    3142 3144 -148.63 -134.00   18.33    0.47  -62.50 -177.35   34.54\n",
      "  151        406A 406A N   CA   3144 3145  158.03  147.00                  -40.90\n",
      "  152  16100 406A 407R C   N    3147 3149  126.07   57.30   97.31   11.37  -63.00 -133.94   27.46\n",
      "  152        407R 407R N   CA   3149 3150  106.84   38.00                  -41.10\n",
      "  153  16101 407R 408D C   N    3158 3160   45.16   54.50   11.05    0.86  -63.30  138.92   23.48\n",
      "  153        408D 408D N   CA   3160 3161   46.81   40.90                  -40.00\n",
      "  154  16102 408D 409K C   N    3166 3168 -129.11  -62.90  101.36   17.00  -62.90  101.36   17.00\n",
      "  154        409K 409K N   CA   3168 3169 -117.54  -40.80                  -40.80\n",
      "  155  16104 410L 411E C   N    3183 3185  -95.71  -63.60   42.20    7.21  -63.60   42.20    7.21\n",
      "  155        411E 411E N   CA   3185 3186  -67.69  -40.30                  -40.30\n",
      "  156  16108 414L 415N C   N    3213 3215   -8.10 -119.90  112.47    4.97  -63.20  178.34   19.79\n",
      "  156        415N 415N N   CA   3215 3216  149.29  137.00                  -41.10\n",
      "  157  16109 415N 416N C   N    3221 3223 -139.26 -119.90   70.25    2.71  -63.20  137.35   21.35\n",
      "  157        416N 416N N   CA   3223 3224 -155.47  137.00                  -41.10\n",
      "  158  16111 417I 418R C   N    3237 3239  158.88 -125.20   77.91    3.57  -63.00 -145.44   24.74\n",
      "  158        418R 418R N   CA   3239 3240  123.09  140.60                  -41.10\n",
      "  159  16112 418R 419L C   N    3248 3250 -120.14 -108.50   13.17    0.58  -63.50 -171.41   23.61\n",
      "  159        419L 419L N   CA   3250 3251  138.68  132.50                  -41.20\n",
      "  160  16113 419L 420S C   N    3256 3258 -109.09 -136.60   31.95    0.97  -64.10  175.80   10.92\n",
      "  160        420S 420S N   CA   3258 3259  134.95  151.20                  -35.00\n",
      "  161  16114 420S 421N C   N    3262 3264  178.10 -119.90   73.19    2.17  -63.20 -174.14   29.58\n",
      "  161        421N 421N N   CA   3264 3265  175.89  137.00                  -41.10\n",
      "  162  16116 422I 423A C   N    3278 3280 -138.03 -134.00   16.81    0.84  -62.50  173.13   32.42\n",
      "  162        423A 423A N   CA   3280 3281  163.32  147.00                  -40.90\n",
      "  163  16117 423A 424L C   N    3283 3285  -97.40 -108.50   11.43    0.52  -63.50  174.29   22.49\n",
      "  163        424L 424L N   CA   3285 3286  129.76  132.50                  -41.20\n",
      "  164  16118 424L 425R C   N    3291 3293  -78.20  -72.10   23.53    1.68  -63.00  155.03   21.98\n",
      "  164        425R 425R N   CA   3293 3294  164.62  141.90                  -41.10\n",
      "  165  16119 425R 426R C   N    3302 3304  -62.28  -72.10   17.92    1.54  -63.00  162.01   21.84\n",
      "  165        426R 426R N   CA   3304 3305  156.89  141.90                  -41.10\n",
      "  166  16120 426R 427R C   N    3313 3315  -61.65  -72.10   11.29    0.77  -63.00  178.75   24.24\n",
      "  166        427R 427R N   CA   3315 3316  137.65  141.90                  -41.10\n",
      "  167  16123 429E 430A C   N    3340 3342 -160.19 -134.00   26.50    0.74  -62.50 -165.55   37.08\n",
      "  167        430A 430A N   CA   3342 3343  150.97  147.00                  -40.90\n",
      "  168  16124 430A 431A C   N    3345 3347  -78.68  -68.20   15.45    0.97  -62.50  163.26   27.58\n",
      "  168        431A 431A N   CA   3347 3348  156.65  145.30                  -40.90\n",
      "  169  16125 431A 432S C   N    3350 3352  -71.06  -72.40    3.01    0.14  -64.10  175.43   13.14\n",
      "  169        432S 432S N   CA   3352 3353  149.70  152.40                  -35.00\n",
      "  170  16126 432S 433E C   N    3356 3358 -115.44 -117.80    2.36    0.08  -63.60 -175.58   22.17\n",
      "  170        433E 433E N   CA   3358 3359  136.68  136.80                  -40.30\n",
      "  171  16127 433E 434G C   N    3365 3367 -174.55 -167.20    8.91    0.30   82.20 -168.69   13.81\n",
      "  171        434G 434G N   CA   3367 3368  169.56  174.60                    8.50\n",
      "  172  16131 437H 438S C   N    3398 3400  -89.37  -72.40   45.06    3.22  -64.10  147.83    9.50\n",
      "  172        438S 438S N   CA   3400 3401  110.65  152.40                  -35.00\n",
      "  173  16132 438S 439L C   N    3404 3406  -71.33  -70.70   16.35    1.26  -63.50  166.64   22.70\n",
      "  173        439L 439L N   CA   3406 3407  125.26  141.60                  -41.20\n",
      "  174  16133 439L 440D C   N    3412 3414 -101.80  -63.30   47.76    8.11  -63.30   47.76    8.11\n",
      "  174        440D 440D N   CA   3414 3415  -68.26  -40.00                  -40.00\n",
      "  175  16134 440D 441N C   N    3420 3422 -107.86  -63.20   63.25   10.22  -63.20   63.25   10.22\n",
      "  175        441N 441N N   CA   3422 3423  -85.88  -41.10                  -41.10\n",
      "  176  16135 441N 442K C   N    3428 3430   56.71   56.60   14.51    1.04  -62.90  152.07   26.31\n",
      "  176        442K 442K N   CA   3430 3431   53.11   38.60                  -40.80\n",
      "  177  16136 442K 443S C   N    3437 3439  -66.64  -72.40   15.95    1.13  -64.10  157.74   11.58\n",
      "  177        443S 443S N   CA   3439 3440  167.28  152.40                  -35.00\n",
      "  178  16138 444C 445F C   N    3449 3451  -83.87  -71.40   15.85    0.90  -63.20  166.49   24.51\n",
      "  178        445F 445F N   CA   3451 3452  150.50  140.70                  -44.30\n",
      "  179  16140 446L 447F C   N    3468 3470  -57.13  -71.40   15.61    0.89  -63.20  178.78   25.23\n",
      "  179        447F 447F N   CA   3470 3471  134.37  140.70                  -44.30\n",
      "  180  16141 447F 448F C   N    3479 3481 -105.45 -124.20   20.68    0.57  -63.20 -176.19   22.86\n",
      "  180        448F 448F N   CA   3481 3482  134.59  143.30                  -44.30\n",
      "  181  16142 448F 449K C   N    3490 3492 -120.29 -118.00   75.78    3.65  -62.90  119.10   18.89\n",
      "  181        449K 449K N   CA   3492 3493 -145.16  139.10                  -40.80\n",
      "  182  16143 449K 450G C   N    3499 3501   93.70   78.70   15.66    0.66   82.20  179.49    8.13\n",
      "  182        450G 450G N   CA   3501 3502 -170.62 -166.10                    8.50\n",
      "  183  16144 450G 451R C   N    3503 3505 -139.39 -125.20   14.67    0.50  -63.00 -169.42   30.98\n",
      "  183        451R 451R N   CA   3505 3506  144.30  140.60                  -41.10\n",
      "  184  16145 451R 452C C   N    3514 3516  118.62   57.40   90.69    4.78 -117.90 -148.25    9.42\n",
      "  184        452C 452C N   CA   3516 3517  -30.91   36.00                  141.10\n",
      "  185  16146 452C 453S C   N    3520 3522  -45.62  -72.40   26.79    1.89  -64.10  174.51   11.67\n",
      "  185        453S 453S N   CA   3522 3523  151.47  152.40                  -35.00\n",
      "  186  16148 454I 455T C   N    3534 3536 -102.32 -124.80   36.89    1.15  -63.20  161.18   18.36\n",
      "  186        455T 455T N   CA   3536 3537  114.25  143.50                  -42.10\n",
      "  187  16149 455T 456L C   N    3541 3543 -137.47 -108.50   29.44    1.47  -63.50 -176.05   22.62\n",
      "  187        456L 456L N   CA   3543 3544  127.22  132.50                  -41.20\n",
      "  188  16153 459V 460L C   N    3571 3573  164.16 -108.50  126.04    5.70  -63.50  163.15   29.95\n",
      "  188        460L 460L N   CA   3573 3574 -136.63  132.50                  -41.20\n",
      "  189  16154 460L 461R C   N    3579 3581 -114.50 -125.20   46.02    1.97  -63.00  146.30   16.98\n",
      "  189        461R 461R N   CA   3581 3582   95.84  140.60                  -41.10\n",
      "  190  16177 483L 484H C   N    3759 3761  116.25 -125.60  118.89    3.88  -63.20 -115.73   27.91\n",
      "  190        484H 484H N   CA   3761 3762  151.96  138.80                  -42.30\n",
      "  191  16179 485I 486Q C   N    3777 3779 -150.44 -121.10   29.60    1.20  -63.80 -163.73   25.39\n",
      "  191        486Q 486Q N   CA   3779 3780  135.81  139.70                  -40.30\n",
      "  192  16180 486Q 487L C   N    3786 3788 -127.88 -108.50   22.55    1.22  -63.50  174.49   21.56\n",
      "  192        487L 487L N   CA   3788 3789  120.98  132.50                  -41.20\n",
      "  193  16181 487L 488F C   N    3794 3796 -154.24 -124.20   34.15    0.92  -63.20 -179.25   30.84\n",
      "  193        488F 488F N   CA   3796 3797  159.55  143.30                  -44.30\n",
      "  194  16184 490C 491E C   N    3818 3820 -126.56 -117.80   11.37    0.35  -63.60 -173.40   28.44\n",
      "  194        491E 491E N   CA   3820 3821  144.04  136.80                  -40.30\n",
      "  195  16185 491E 492H C   N    3827 3829  -71.06  -67.60    3.48    0.25  -63.20  177.51   22.69\n",
      "  195        492H 492H N   CA   3829 3830  140.36  140.00                  -42.30\n",
      "  196  16186 492H 493H C   N    3837 3839 -148.67 -125.60   75.40    2.39  -63.20  137.04   22.48\n",
      "  196        493H 493H N   CA   3839 3840 -149.42  138.80                  -42.30\n",
      "  197  16187 493H 494A C   N    3847 3849  -60.40  -68.20   10.05    0.64  -62.50  179.87   29.58\n",
      "  197        494A 494A N   CA   3849 3850  138.96  145.30                  -40.90\n",
      "  198  16188 494A 495N C   N    3852 3854  -86.71  -71.20   19.60    1.09  -63.20  165.79   22.05\n",
      "  198        495N 495N N   CA   3854 3855  154.78  142.80                  -41.10\n",
      "  199  16189 495N 496E C   N    3860 3862 -119.21 -117.80    2.87    0.11  -63.60 -171.99   22.51\n",
      "  199        496E 496E N   CA   3862 3863  139.30  136.80                  -40.30\n",
      "  200  16190 496E 497R C   N    3869 3871 -102.58 -125.20   34.50    1.09  -63.00  160.61   19.35\n",
      "  200        497R 497R N   CA   3871 3872  114.55  140.60                  -41.10\n",
      "  201  16191 497R 498N C   N    3880 3882   53.50   55.90   10.15    0.89  -63.20  136.46   22.44\n",
      "  201        498N 498N N   CA   3882 3883   29.64   39.50                  -41.10\n",
      "  202  16192 498N 499D C   N    3888 3890  -83.27  -70.90   18.83    0.71  -63.30  156.78   20.61\n",
      "  202        499D 499D N   CA   3890 3891  164.50  150.30                  -40.00\n",
      "  203  16194 500P 501M C   N    3903 3905  -67.82  -73.00   31.26    2.20  -63.40  145.73   22.10\n",
      "  203        501M 501M N   CA   3905 3906  173.83  143.00                  -40.50\n",
      "  204  16195 501M 502Q C   N    3911 3913  -58.76  -73.00   15.21    1.17  -63.80  173.72   25.28\n",
      "  204        502Q 502Q N   CA   3913 3914  146.05  140.70                  -40.30\n",
      "  205  16196 502Q 503F C   N    3920 3922 -133.23 -124.20    9.13    0.32  -63.20 -175.14   30.25\n",
      "  205        503F 503F N   CA   3922 3923  144.62  143.30                  -44.30\n",
      "  206  16198 504I 505G C   N    3939 3941  -90.51  -80.20   19.12    1.24   82.20 -131.58    7.77\n",
      "  206        505G 505G N   CA   3941 3942  158.00  174.10                    8.50\n",
      "  207  16199 505G 506K C   N    3943 3945  -49.20  -62.90   85.70   10.46  -62.90   85.70   10.46\n",
      "  207        506K 506K N   CA   3945 3946 -125.40  -40.80                  -40.80\n",
      "  208  16200 506K 507S C   N    3952 3954   50.57  -64.10  146.83   13.04  -64.10  146.83   13.04\n",
      "  208        507S 507S N   CA   3954 3955 -126.70  -35.00                  -35.00\n",
      "  209  16201 507S 508L C   N    3958 3960 -105.03  -63.50   42.02    6.92  -63.50   42.02    6.92\n",
      "  209        508L 508L N   CA   3960 3961  -34.76  -41.20                  -41.20\n",
      "  210  16205 511C 512L C   N    3994 3996   15.26  -70.70   85.97    7.78  -63.50 -165.13   23.96\n",
      "  210        512L 512L N   CA   3996 3997  140.55  141.60                  -41.20\n",
      "  211  16206 512L 513H C   N    4002 4004  105.99   56.30  117.95   15.05  -63.20 -120.21   26.95\n",
      "  211        513H 513H N   CA   4004 4005  147.77   40.80                  -42.30\n",
      "  212  16209 515V 516G C   N    4026 4028  -78.82  -80.20    8.17    0.39   82.20 -123.18    7.12\n",
      "  212        516G 516G N   CA   4028 4029 -177.85  174.10                    8.50\n",
      "  213  16211 517A 518S C   N    4035 4037   57.52  -72.40  130.79    9.74  -64.10 -160.96   14.46\n",
      "  213        518S 518S N   CA   4037 4038  167.44  152.40                  -35.00\n",
      "  214  16212 518S 519G C   N    4041 4043   72.41   78.70   12.55    0.60   82.20  174.81    7.99\n",
      "  214        519G 519G N   CA   4043 4044 -176.96 -166.10                    8.50\n",
      "  215  16215 521P 522K C   N    4060 4062 -123.19 -118.00   14.01    0.59  -62.90  177.63   26.67\n",
      "  215        522K 522K N   CA   4062 4063  152.12  139.10                  -40.80\n",
      "  216  16216 522K 523A C   N    4069 4071 -114.56 -134.00   22.24    0.51  -62.50 -175.42   28.19\n",
      "  216        523A 523A N   CA   4071 4072  136.18  147.00                  -40.90\n",
      "  217  16217 523A 524F C   N    4074 4076 -124.21 -124.20    9.59    0.45  -63.20  173.87   28.16\n",
      "  217        524F 524F N   CA   4076 4077  152.89  143.30                  -44.30\n",
      "  218  16219 525V 526A C   N    4092 4094 -170.13 -134.00   38.83    0.89  -62.50 -168.91   37.01\n",
      "  218        526A 526A N   CA   4094 4095  161.21  147.00                  -40.90\n",
      "  219  16220 526A 527H C   N    4097 4099  -99.04 -125.60   92.17    2.96  -63.20   99.52   10.75\n",
      "  219        527H 527H N   CA   4099 4100   50.54  138.80                  -42.30\n",
      "  220  16221 527H 528T C   N    4107 4109 -144.38 -124.80   25.03    0.86  -63.20  178.36   28.14\n",
      "  220        528T 528T N   CA   4109 4110  159.09  143.50                  -42.10\n",
      "  221  16222 528T 529F C   N    4114 4116 -111.05 -124.20   27.08    0.89  -63.20  170.77   20.77\n",
      "  221        529F 529F N   CA   4116 4117  119.63  143.30                  -44.30\n",
      "  222  16224 530C 531K C   N    4131 4133 -135.65 -118.00   34.88    1.81  -62.90  166.55   19.14\n",
      "  222        531K 531K N   CA   4133 4134  109.02  139.10                  -40.80\n",
      "  223  16225 531K 532F C   N    4140 4142 -117.32 -124.20   46.79    2.03  -63.20  151.33   17.85\n",
      "  223        532F 532F N   CA   4142 4143   97.02  143.30                  -44.30\n",
      "  224  16226 532F 533T C   N    4151 4153 -131.72  -63.20   96.16   10.58  -63.20   96.16   10.58\n",
      "  224        533T 533T N   CA   4153 4154   25.36  -42.10                  -42.10\n",
      "  225  16227 533T 534L C   N    4158 4160 -118.00 -108.50   17.97    0.85  -63.50  179.52   28.29\n",
      "  225        534L 534L N   CA   4160 4161  147.75  132.50                  -41.20\n",
      "  226  16229 535L 536H C   N    4174 4176   74.09   56.30  112.56   11.53  -63.20 -144.78   23.48\n",
      "  226        536H 536H N   CA   4176 4177  151.95   40.80                  -42.30\n",
      "  227  16230 536H 537D C   N    4184 4186  -80.67  -63.30  108.48   12.43  -63.30  108.48   12.43\n",
      "  227        537D 537D N   CA   4186 4187   67.08  -40.00                  -40.00\n",
      "  228  16232 538R 539E C   N    4203 4205 -147.42  -63.60  107.54   18.42  -63.60  107.54   18.42\n",
      "  228        539E 539E N   CA   4205 4206 -107.66  -40.30                  -40.30\n",
      "  229  16233 539E 540E C   N    4212 4214 -152.90 -117.80  110.95    4.79  -63.60  118.34   20.21\n",
      "  229        540E 540E N   CA   4214 4215 -117.95  136.80                  -40.30\n",
      "  230  16234 540E 541R C   N    4221 4223 -135.81 -125.20   79.09    3.58  -63.00  123.64   21.47\n",
      "  230        541R 541R N   CA   4223 4224 -141.03  140.60                  -41.10\n",
      "  231  16235 541R 542Y C   N    4232 4234 -117.78 -124.30   10.63    0.83  -63.50 -178.88   31.49\n",
      "  231        542Y 542Y N   CA   4234 4235  143.81  135.40                  -43.40\n",
      "  232  16236 542Y 543F C   N    4244 4246  -43.49  -71.40   27.92    1.89  -63.20  175.98   23.14\n",
      "  232        543F 543F N   CA   4246 4247  140.83  140.70                  -44.30\n",
      "  233  16237 543F 544T C   N    4255 4257 -124.65 -124.80    0.89    0.04  -63.20 -175.91   27.68\n",
      "  233        544T 544T N   CA   4257 4258  144.37  143.50                  -42.10\n",
      "  234  16238 544T 545T C   N    4262 4264 -101.39  -78.10   38.38    1.30  -63.20  142.80   20.86\n",
      "  234        545T 545T N   CA   4264 4265 -179.70  149.80                  -42.10\n",
      "  235  16239 545T 546S C   N    4269 4271 -128.41 -136.60   12.13    0.68  -64.10  176.96   17.37\n",
      "  235        546S 546S N   CA   4271 4272  160.14  151.20                  -35.00\n",
      "  236  16240 546S 547T C   N    4275 4277 -126.81 -124.80   23.81    0.96  -63.20  163.55   25.15\n",
      "  236        547T 547T N   CA   4277 4278  167.22  143.50                  -42.10\n",
      "  237  16241 547T 548A C   N    4282 4284 -158.52 -134.00   39.51    1.36  -62.50  170.69   33.05\n",
      "  237        548A 548A N   CA   4284 4285  177.98  147.00                  -40.90\n",
      "  238  16242 548A 549E C   N    4287 4289  -62.79  -69.30   12.72    1.15  -63.60  166.27   22.13\n",
      "  238        549E 549E N   CA   4289 4290  153.43  142.50                  -40.30\n",
      "  239  16243 549E 550N C   N    4296 4298  -59.29  -71.20   21.35    1.13  -63.20  166.22   21.00\n",
      "  239        550N 550N N   CA   4298 4299  125.08  142.80                  -41.10\n",
      "  240  16244 550N 551T C   N    4304 4306 -147.49 -124.80   29.21    1.00  -63.20  177.31   28.18\n",
      "  240        551T 551T N   CA   4306 4307  161.90  143.50                  -42.10\n",
      "  241  16245 551T 552Q C   N    4311 4313 -104.75 -121.10   18.29    0.57  -63.80  176.61   24.01\n",
      "  241        552Q 552Q N   CA   4313 4314  131.50  139.70                  -40.30\n",
      "  242  16246 552Q 553N C   N    4320 4322 -142.40 -119.90   32.88    0.97  -63.20  176.66   26.54\n",
      "  242        553N 553N N   CA   4322 4323  160.98  137.00                  -41.10\n",
      "  243  16247 553N 554P C   N    4328 4330  -71.24  -58.70   23.10    1.38  -64.50  158.44   12.18\n",
      "  243        554P 554P N   CA   4330 4331  -11.09  -30.50                  147.20\n",
      "  244  16248 554P 555R C   N    4335 4337  -60.89  -72.10   21.07    1.81  -63.00  159.18   21.36\n",
      "  244        555R 555R N   CA   4337 4338  159.74  141.90                  -41.10\n",
      "  245  16249 555R 556W C   N    4346 4348  -60.26  -71.30   11.64    0.73  -63.00  179.54   22.61\n",
      "  245        556W 556W N   CA   4348 4349  135.32  139.00                  -44.20\n",
      "  246  16250 556W 557G C   N    4360 4362 -161.67 -167.20   82.20    3.15   82.20  143.37   11.19\n",
      "  246        557G 557G N   CA   4362 4363   92.58  174.60                    8.50\n",
      "  247  16251 557G 558Y C   N    4364 4366 -148.16 -124.30   53.70    2.12  -63.50  157.73   29.02\n",
      "  247        558Y 558Y N   CA   4366 4367 -176.49  135.40                  -43.40\n",
      "  248  16252 558Y 559V C   N    4376 4378  -60.57  -62.40    1.83    0.29 -125.40 -173.97   10.45\n",
      "  248        559V 559V N   CA   4378 4379  -42.33  -42.40                  143.30\n",
      "  249  16253 559V 560K C   N    4383 4385 -126.18 -118.00   14.55    0.56  -62.90  179.59   27.10\n",
      "  249        560K 560K N   CA   4385 4386  151.13  139.10                  -40.80\n",
      "  250  16255 561V 562F C   N    4399 4401 -126.23 -124.20    4.57    0.25  -63.20 -172.59   30.19\n",
      "  250        562F 562F N   CA   4401 4402  139.21  143.30                  -44.30\n",
      "  251  16256 562F 563E C   N    4410 4412 -139.94 -117.80   22.31    0.89  -63.60 -169.66   22.23\n",
      "  251        563E 563E N   CA   4412 4413  134.06  136.80                  -40.30\n",
      "  252  16257 563E 564I C   N    4419 4421  -69.64  -63.40    6.70    1.26 -120.60 -176.44    8.61\n",
      "  252        564I 564I N   CA   4421 4422  -46.04  -43.60                  130.30\n",
      "  253  16259 565P 566E C   N    4434 4436 -140.77 -117.80   61.16    2.54  -63.60  147.94   23.96\n",
      "  253        566E 566E N   CA   4436 4437 -166.52  136.80                  -40.30\n",
      "  254  16260 566E 567L C   N    4443 4445  117.48 -108.50  140.96    6.30  -63.50 -131.13   41.84\n",
      "  254        567L 567L N   CA   4445 4446  176.21  132.50                  -41.20\n",
      "  255  16261 567L 568T C   N    4451 4453 -106.22  -78.10   75.37    2.51  -63.20  107.19   16.57\n",
      "  255        568T 568T N   CA   4453 4454 -140.27  149.80                  -42.10\n",
      "  256  16266 572I 573R C   N    4487 4489  -60.34  -63.00   69.39    9.20  -63.00   69.39    9.20\n",
      "  256        573R 573R N   CA   4489 4490 -110.44  -41.10                  -41.10\n",
      "  257  16268 574C 575F C   N    4504 4506 -162.93 -124.20   47.09    1.22  -63.20  176.48   30.63\n",
      "  257        575F 575F N   CA   4506 4507  170.10  143.30                  -44.30\n",
      "  258  16270 576C 577E C   N    4521 4523  -48.95  -69.30   23.77    2.14  -63.60  165.56   21.30\n",
      "  258        577E 577E N   CA   4523 4524  154.79  142.50                  -40.30\n",
      "  259  16271 577E 578H C   N    4530 4532  -64.65  -67.60   34.53    2.57  -63.20  147.91   18.40\n",
      "  259        578H 578H N   CA   4532 4533  105.60  140.00                  -42.30\n",
      "  260  16274 580V 581F C   N    4554 4556  -55.15  -71.40   18.84    1.57  -63.20  165.65   22.46\n",
      "  260        581F 581F N   CA   4556 4557  150.24  140.70                  -44.30\n",
      "  261  16275 581F 582A C   N    4565 4567 -153.68 -134.00   43.82    1.85  -62.50  161.21   31.24\n",
      "  261        582A 582A N   CA   4567 4568 -173.85  147.00                  -40.90\n",
      "  262  16276 582A 583F C   N    4570 4572 -102.30 -124.20   89.88    4.69  -63.20   93.76   15.56\n",
      "  262        583F 583F N   CA   4572 4573 -129.52  143.30                  -44.30\n",
      "  263  16277 583F 584E C   N    4581 4583  -56.59  -69.30   13.73    0.86  -63.60  177.76   24.09\n",
      "  263        584E 584E N   CA   4583 4584  137.32  142.50                  -40.30\n",
      "  264  16278 584E 585V C   N    4590 4592  -65.81  -62.40    3.50    0.51 -125.40 -175.05   10.33\n",
      "  264        585V 585V N   CA   4592 4593  -41.61  -42.40                  143.30\n",
      "  265  16279 585V 586F C   N    4597 4599  -69.36  -71.40    2.36    0.13  -63.20  176.29   24.89\n",
      "  265        586F 586F N   CA   4599 4600  139.51  140.70                  -44.30\n",
      "  266  16280 586F 587A C   N    4608 4610 -156.45 -134.00   24.68    1.21  -62.50 -159.03   30.11\n",
      "  266        587A 587A N   CA   4610 4611  136.76  147.00                  -40.90\n",
      "  267  16281 587A 588F C   N    4613 4615 -164.03 -124.20   39.86    1.60  -63.20 -159.03   34.27\n",
      "  267        588F 588F N   CA   4615 4616  141.86  143.30                  -44.30\n",
      "  268  16282 588F 589N C   N    4624 4626 -161.54 -119.90   79.71    2.58  -63.20  150.51   24.04\n",
      "  268        589N 589N N   CA   4626 4627 -155.03  137.00                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3   97  121  492  500  742  621  689  794  804  899\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00200.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      590     4639     4639\n",
      "openf___224_> Open           EAO00200.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     69477392   67849.016    66.259\n",
      "read_mo_297_> Segments, residues, atoms:        1      590     4639\n",
      "read_mo_298_> Segment:        1  1:A  590:A     4639\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4639     4639        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    21546\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    22778\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    23393\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    24312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    25407\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    26173\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    27672\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    28645\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    29568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    31057\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    32296\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34090\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34272\n",
      "iupac_m_486_> OE1/2 will be swapped:     -108.4967        3  3\n",
      "iupac_m_487_> NH1/2 swapped:      177.7649       10  10\n",
      "iupac_m_487_> NH1/2 swapped:      170.6588       16  16\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5265       21  21\n",
      "iupac_m_485_> OD1/2 will be swapped:     -168.5734       31  31\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.3700       43  43\n",
      "iupac_m_487_> NH1/2 swapped:      179.2157       46  46\n",
      "iupac_m_485_> OD1/2 will be swapped:      136.0279       62  62\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.6086       63  63\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.5869       78  78\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.8847      106  106\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.1315      108  108\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.6196      120  120\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.0499      121  121\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9668      125  125\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.6802      129  129\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.1111      144  144\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.0623\n",
      "iupac_m_487_> NH1/2 swapped:      178.9301      155  155\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.6738      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.9019      165  165\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9934      173  173\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.0808      205  205\n",
      "iupac_m_487_> NH1/2 swapped:      179.4380      214  214\n",
      "iupac_m_485_> OD1/2 will be swapped:      119.9386      231  231\n",
      "iupac_m_487_> NH1/2 swapped:      179.5207      233  233\n",
      "iupac_m_485_> OD1/2 will be swapped:     -168.3523      234  234\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.5338      248  248\n",
      "iupac_m_487_> NH1/2 swapped:      179.9802      249  249\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.5195      261  261\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.2411      262  262\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.1145      282  282\n",
      "iupac_m_487_> NH1/2 swapped:      168.8873      283  283\n",
      "iupac_m_487_> NH1/2 swapped:      179.6568      293  293\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7230      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      161.5775      332  332\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.1614      366  366\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5559      368  368\n",
      "iupac_m_485_> OD1/2 will be swapped:      131.5188      370  370\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.1753      375  375\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3609      378  378\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.7897      381  381\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.6065      382  382\n",
      "iupac_m_487_> NH1/2 swapped:      179.8405      385  385\n",
      "iupac_m_485_> OD1/2 will be swapped:     -162.4705      405  405\n",
      "iupac_m_487_> NH1/2 swapped:      179.8688      407  407\n",
      "iupac_m_486_> OE1/2 will be swapped:     -131.7819      411  411\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.9184      440  440\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -100.2979      447  447\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.0014      457  457\n",
      "iupac_m_487_> NH1/2 swapped:     -172.1695      464  464\n",
      "iupac_m_487_> NH1/2 swapped:      174.8006      465  465\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.9367      503  503\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      178.3331      509  509\n",
      "iupac_m_487_> NH1/2 swapped:      176.0715      538  538\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.7853      539  539\n",
      "iupac_m_486_> OE1/2 will be swapped:      111.2010      540  540\n",
      "iupac_m_487_> NH1/2 swapped:      179.8369      541  541\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      114.9616      543  543\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      154.4539      549  549\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8442      555  555\n",
      "iupac_m_487_> NH1/2 swapped:     -176.7140      573  573\n",
      "iupac_m_486_> OE1/2 will be swapped:      147.2395      577  577\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.9204      584  584\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):  1051091\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -43080.4609\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -43080.460938\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    31.629000\n",
      "Sequence length          :          590\n",
      "Compactness              :     0.167182\n",
      "Native energy (pair)     :   820.897504\n",
      "Native energy (surface)  :    77.517779\n",
      "Native energy (combined) :    21.756413\n",
      "Z score (pair)           :    -3.424942\n",
      "Z score (surface)        :    -1.933545\n",
      "Z score (combined)       :    -3.370274\n",
      "GA341 score              :     0.511029\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    12566\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        6516.7930\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4711       0      0   0.007   0.007      60.976       1.000\n",
      " 2 Bond angle potential               :    6336       7     46   2.377   2.377      686.10       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2751       0    155  53.150  53.150      1247.3       1.000\n",
      " 4 Stereochemical improper torsion pot:    1896       0      6   1.678   1.678      105.32       1.000\n",
      " 5 Soft-sphere overlap restraints     :   12566       0      1   0.006   0.006      56.428       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6251       6     87   1.151   1.151      1011.8       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6377      17    115   1.378   1.378      1215.2       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     589       5     19   5.874   5.874      238.57       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     499       0     20  86.142  86.142      247.92       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     371       0      3  75.087  75.087      177.85       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     179       0      0  96.251  96.251      140.22       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      77       0      0  91.508  91.508      47.039       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2718       0      0   0.711   0.711      84.504       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     588     265    154  42.760 116.100      1006.8       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     929       0     22   2.714   2.714      190.83       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00200.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00200.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  102610.2656\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8875 396E 396E N   CA   3068 3069  123.03  107.00   16.03    4.61  107.00   16.03    4.61\n",
      "    2   8917 399F 400A C   N    3098 3100  146.42  120.00   26.42    6.00  120.00   26.42    6.00\n",
      "    3   8920 400A 400A N   CA   3100 3101  124.44  107.00   17.44    5.01  107.00   17.44    5.01\n",
      "    4  10078 506K 506K N   CA   3945 3946  124.97  107.00   17.97    5.17  107.00   17.97    5.17\n",
      "    5  10425 538R 538R N   CA   4194 4195  124.02  107.00   17.02    4.89  107.00   17.02    4.89\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20242  85F 458S CA  CA    643 3561   12.60    9.83    2.76    5.33    9.83    2.76    5.33\n",
      "    2  20295  86A 458S CA  CA    654 3561   16.36   13.45    2.91    4.91   13.45    2.91    4.91\n",
      "    3  20461  89Q 506K CA  CA    675 3946   12.34    9.44    2.90    4.85    9.44    2.90    4.85\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  25857  85F 458S N   O     642 3565   12.41    8.59    3.82    7.04    8.59    3.82    7.04\n",
      "    2  25975  89Q 341T N   O     674 2619    8.96    5.85    3.11    4.55    5.85    3.11    4.55\n",
      "    3  26942 140M 195K N   O    1063 1515    6.77    4.83    1.94    4.68    4.83    1.94    4.68\n",
      "    4  27682 219V 138V N   O    1690 1055    4.66    2.79    1.87    5.39    2.79    1.87    5.39\n",
      "    5  28976 342N 144Y N   O    2620 1106   11.07    6.52    4.55    5.94    6.52    4.55    5.94\n",
      "    6  29347 397V 360E N   O    3077 2779   17.14    7.05   10.09    4.67    7.05   10.09    4.67\n",
      "    7  29362 398A 359R N   O    3084 2770   15.26    9.28    5.98    4.91    9.28    5.98    4.91\n",
      "    8  29363 398A 360E N   O    3084 2779   14.13    6.19    7.94    4.83    6.19    7.94    4.83\n",
      "    9  29484 410L 146E N   O    3177 1123    9.82    6.64    3.18    4.75    6.64    3.18    4.75\n",
      "   10  29858 468Y 458S N   O    3644 3565   13.06    9.27    3.79    4.68    9.27    3.79    4.68\n",
      "   11  30272 506K 476G N   O    3945 3709   15.28   10.57    4.71    4.82   10.57    4.71    4.82\n",
      "   12  30273 506K 477V N   O    3945 3716   16.35   10.89    5.46    5.20   10.89    5.46    5.20\n",
      "   13  30274 506K 478L N   O    3945 3724   12.19    6.32    5.87    4.99    6.32    5.87    4.99\n",
      "   14  30276 506K 480E N   O    3945 3740   11.50    2.71    8.79    4.79    2.71    8.79    4.79\n",
      "   15  30424 537D  87Y N   O    4186  669   13.82   10.32    3.50    4.83   10.32    3.50    4.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16682 400A 400A CA  C    3101 3103 -150.51 -180.00   29.49    5.90 -180.00   29.49    5.90\n",
      "    2  16684 402S 402S CA  C    3114 3117 -156.70 -180.00   23.30    4.66 -180.00   23.30    4.66\n",
      "    3  16788 506K 506K CA  C    3946 3952 -156.23 -180.00   23.77    4.75 -180.00   23.77    4.75\n",
      "    4  16789 507S 507S CA  C    3955 3958 -122.28 -180.00   57.72   11.54 -180.00   57.72   11.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15696   2A   3E C   N      12   14 -162.87 -117.80   83.51    4.57  -63.60  145.81   17.24\n",
      "    1          3E   3E N   CA     14   15   66.50  136.80                  -40.30\n",
      "    2  15697   3E   4L C   N      21   23 -108.19 -108.50   38.04    2.06  -63.50  154.85   24.26\n",
      "    2          4L   4L N   CA     23   24  170.54  132.50                  -41.20\n",
      "    3  15698   4L   5A C   N      29   31  -58.90  -68.20   27.29    1.88  -62.50  160.58   26.49\n",
      "    3          5A   5A N   CA     31   32  119.64  145.30                  -40.90\n",
      "    4  15700   6G   7T C   N      38   40 -148.65 -124.80   24.85    1.45  -63.20 -162.00   21.15\n",
      "    4          7T   7T N   CA     40   41  136.51  143.50                  -42.10\n",
      "    5  15701   7T   8A C   N      45   47 -158.04 -134.00   29.97    1.66  -68.20   91.29    8.25\n",
      "    5          8A   8A N   CA     47   48  129.10  147.00                  145.30\n",
      "    6  15710  16R  17M C   N     111  113  -78.21  -73.00   28.50    1.85  -63.40  149.21   23.45\n",
      "    6         17M  17M N   CA    113  114  171.02  143.00                  -40.50\n",
      "    7  15714  20P  21R C   N     139  141 -169.17 -125.20   57.55    3.12  -63.00  179.36   20.43\n",
      "    7         21R  21R N   CA    141  142  103.47  140.60                  -41.10\n",
      "    8  15720  26V  27I C   N     188  190 -121.41 -120.60   95.62    5.50  -63.40   97.44   14.82\n",
      "    8         27I  27I N   CA    190  191   34.69  130.30                  -43.60\n",
      "    9  15724  30A  31D C   N     218  220  -63.53  -70.90    7.61    0.50  -63.30  171.61   21.20\n",
      "    9         31D  31D N   CA    220  221  148.39  150.30                  -40.00\n",
      "   10  15725  31D  32S C   N     226  228  -58.63  -64.10   10.11    0.69 -136.60 -177.23   10.33\n",
      "   10         32S  32S N   CA    228  229  -43.50  -35.00                  151.20\n",
      "   11  15726  32S  33D C   N     232  234  -71.00  -70.90   52.36    2.57  -63.30  117.60   14.99\n",
      "   11         33D  33D N   CA    234  235 -157.35  150.30                  -40.00\n",
      "   12  15737  43E  44G C   N     318  320   49.83   78.70   67.98    1.01  -62.40  128.88   19.72\n",
      "   12         44G  44G N   CA    320  321 -104.55 -166.10                  -41.20\n",
      "   13  15738  44G  45G C   N     322  324   15.91  -62.40   87.07   13.60  -62.40   87.07   13.60\n",
      "   13         45G  45G N   CA    324  325  -79.28  -41.20                  -41.20\n",
      "   14  15752  58T  59G C   N     438  440 -136.83 -167.20   32.39    0.92   82.20 -133.45   16.92\n",
      "   14         59G  59G N   CA    440  441 -174.15  174.60                    8.50\n",
      "   15  15783  89Q  90T C   N     681  683  -58.12  -63.20   17.55    1.96  -78.10  152.61    7.19\n",
      "   15         90T  90T N   CA    683  684  -58.90  -42.10                  149.80\n",
      "   16  15784  90T  91G C   N     688  690  -66.13  -62.40   32.07    4.62   82.20  149.41   10.62\n",
      "   16         91G  91G N   CA    690  691   -9.35  -41.20                    8.50\n",
      "   17  15797 103N 104S C   N     781  783 -119.62 -136.60   84.81    3.92  -64.10  117.10    7.57\n",
      "   17        104S 104S N   CA    783  784   68.10  151.20                  -35.00\n",
      "   18  15799 105K 106E C   N     796  798  154.07 -117.80  103.85    3.05  -63.60 -168.61   32.64\n",
      "   18        106E 106E N   CA    798  799 -168.26  136.80                  -40.30\n",
      "   19  15800 106E 107G C   N     805  807   93.60   82.20   19.91    0.55  -62.40  159.54   29.61\n",
      "   19        107G 107G N   CA    807  808   -7.82    8.50                  -41.20\n",
      "   20  15813 119I 120E C   N     904  906  -59.94  -69.30   35.93    3.04  -63.60  142.56   18.81\n",
      "   20        120E 120E N   CA    906  907  177.19  142.50                  -40.30\n",
      "   21  15821 127E 128S C   N     971  973  -90.05  -72.40   34.94    1.60  -64.10  144.79   12.22\n",
      "   21        128S 128S N   CA    973  974 -177.45  152.40                  -35.00\n",
      "   22  15822 128S 129D C   N     977  979   73.15   54.50   64.74    6.26  -70.90  151.64    9.54\n",
      "   22        129D 129D N   CA    979  980  102.90   40.90                  150.30\n",
      "   23  15849 155R 156A C   N    1196 1198 -118.70 -134.00   33.57    1.41  -62.50  167.71   25.40\n",
      "   23        156A 156A N   CA   1198 1199  117.12  147.00                  -40.90\n",
      "   24  15850 156A 157R C   N    1201 1203   50.05  -72.10  125.93   10.25 -125.20  178.14    7.97\n",
      "   24        157R 157R N   CA   1203 1204  172.53  141.90                  140.60\n",
      "   25  15852 158G 159Q C   N    1216 1218  -84.45  -73.00   25.69    1.49  -63.80  157.36   24.24\n",
      "   25        159Q 159Q N   CA   1218 1219  163.70  140.70                  -40.30\n",
      "   26  15853 159Q 160E C   N    1225 1227   97.33 -117.80  152.71    4.63  -63.60 -150.21   25.69\n",
      "   26        160E 160E N   CA   1227 1228 -174.89  136.80                  -40.30\n",
      "   27  15855 161P 162E C   N    1241 1243    8.55  -69.30   84.92    5.23  -63.60  165.44   26.09\n",
      "   27        162E 162E N   CA   1243 1244  108.58  142.50                  -40.30\n",
      "   28  15856 162E 163S C   N    1250 1252 -144.55 -136.60   28.54    1.26  -64.10  167.04   17.89\n",
      "   28        163S 163S N   CA   1252 1253  178.61  151.20                  -35.00\n",
      "   29  15861 167H 168E C   N    1291 1293  -73.28  -63.60   11.36    1.96 -117.80 -177.53    9.80\n",
      "   29        168E 168E N   CA   1293 1294  -46.25  -40.30                  136.80\n",
      "   30  15863 169D 170S C   N    1308 1310  -84.54 -136.60   52.10    2.12  -64.10  172.97   13.85\n",
      "   30        170S 170S N   CA   1310 1311  153.24  151.20                  -35.00\n",
      "   31  15864 170S 171Q C   N    1314 1316  -99.14 -121.10   28.69    1.45  -63.80  165.36   26.20\n",
      "   31        171Q 171Q N   CA   1316 1317  158.16  139.70                  -40.30\n",
      "   32  15866 172H 173R C   N    1333 1335 -169.64 -125.20   44.45    1.75  -63.00 -153.28   35.07\n",
      "   32        173R 173R N   CA   1335 1336  141.81  140.60                  -41.10\n",
      "   33  15868 174V 175Y C   N    1351 1353 -123.35  -98.40   77.00    7.69  -63.50  115.65   16.03\n",
      "   33        175Y 175Y N   CA   1353 1354   55.55  128.40                  -43.40\n",
      "   34  15870 176V 177K C   N    1370 1372 -122.86 -118.00   10.36    0.42  -62.90 -178.84   27.10\n",
      "   34        177K 177K N   CA   1372 1373  148.25  139.10                  -40.80\n",
      "   35  15871 177K 178G C   N    1379 1381  147.05 -167.20   46.19    0.75   82.20 -175.77   11.99\n",
      "   35        178G 178G N   CA   1381 1382 -179.06  174.60                    8.50\n",
      "   36  15872 178G 179A C   N    1383 1385  -68.50  -68.20    3.49    0.29  -62.50  177.38   29.37\n",
      "   36        179A 179A N   CA   1385 1386  141.83  145.30                  -40.90\n",
      "   37  15873 179A 180G C   N    1388 1390  -81.44  -80.20   12.14    0.54   82.20 -135.62    7.28\n",
      "   37        180G 180G N   CA   1390 1391  162.02  174.10                    8.50\n",
      "   38  15875 181V 182H C   N    1399 1401   66.95   56.30  128.59   12.29  -63.20 -162.35   21.73\n",
      "   38        182H 182H N   CA   1401 1402  168.95   40.80                  -42.30\n",
      "   39  15876 182H 183S C   N    1409 1411  -76.52  -72.40   10.95    0.52  -64.10  162.93   12.59\n",
      "   39        183S 183S N   CA   1411 1412  162.54  152.40                  -35.00\n",
      "   40  15879 185L 186S C   N    1430 1432  -77.24  -72.40   62.11    3.82  -64.10  126.16    8.45\n",
      "   40        186S 186S N   CA   1432 1433   90.48  152.40                  -35.00\n",
      "   41  15881 187L 188E C   N    1444 1446  157.97 -117.80   88.99    2.69  -63.60 -152.78   34.85\n",
      "   41        188E 188E N   CA   1446 1447  165.50  136.80                  -40.30\n",
      "   42  15889 195K 196K C   N    1514 1516  -81.14  -62.90   18.72    3.21 -118.00  179.74    9.12\n",
      "   42        196K 196K N   CA   1516 1517  -44.98  -40.80                  139.10\n",
      "   43  15890 196K 197G C   N    1523 1525  -72.68  -80.20   17.21    0.40   82.20 -144.31    6.87\n",
      "   43        197G 197G N   CA   1525 1526  158.62  174.10                    8.50\n",
      "   44  15891 197G 198N C   N    1527 1529 -110.45 -119.90   12.52    0.67  -63.20 -180.00   25.20\n",
      "   44        198N 198N N   CA   1529 1530  145.21  137.00                  -41.10\n",
      "   45  15892 198N 199A C   N    1535 1537  -72.58  -68.20    6.71    0.42  -62.50  169.02   28.20\n",
      "   45        199A 199A N   CA   1537 1538  150.39  145.30                  -40.90\n",
      "   46  15893 199A 200N C   N    1540 1542  -76.34  -71.20    6.92    0.38  -63.20  171.98   22.23\n",
      "   46        200N 200N N   CA   1542 1543  147.43  142.80                  -41.10\n",
      "   47  15894 200N 201R C   N    1548 1550 -136.95 -125.20   43.40    1.81  -63.00  155.27   25.97\n",
      "   47        201R 201R N   CA   1550 1551 -177.62  140.60                  -41.10\n",
      "   48  15895 201R 202Q C   N    1559 1561  -62.22  -73.00   23.40    1.74  -63.80  158.23   23.18\n",
      "   48        202Q 202Q N   CA   1561 1562  161.47  140.70                  -40.30\n",
      "   49  15896 202Q 203T C   N    1568 1570  -71.20  -78.10   19.19    1.05  -63.20  150.41   19.82\n",
      "   49        203T 203T N   CA   1570 1571  167.70  149.80                  -42.10\n",
      "   50  15897 203T 204G C   N    1575 1577  104.28   78.70   76.96    1.26   82.20  114.95    6.66\n",
      "   50        204G 204G N   CA   1577 1578  121.32 -166.10                    8.50\n",
      "   51  15898 204G 205E C   N    1579 1581 -154.22 -117.80   44.85    1.34  -63.60 -178.96   29.14\n",
      "   51        205E 205E N   CA   1581 1582  162.97  136.80                  -40.30\n",
      "   52  15899 205E 206T C   N    1588 1590  -75.22  -78.10    6.36    0.21  -63.20  174.19   23.13\n",
      "   52        206T 206T N   CA   1590 1591  144.13  149.80                  -42.10\n",
      "   53  15900 206T 207K C   N    1595 1597 -106.44 -118.00   16.93    0.86  -62.90  173.29   25.08\n",
      "   53        207K 207K N   CA   1597 1598  151.47  139.10                  -40.80\n",
      "   54  15901 207K 208M C   N    1604 1606 -118.17 -125.60    7.43    0.29  -63.40 -172.73   32.33\n",
      "   54        208M 208M N   CA   1606 1607  140.42  140.50                  -40.50\n",
      "   55  15902 208M 209N C   N    1612 1614  -92.79  -71.20   33.32    2.65  -63.20  161.25   18.68\n",
      "   55        209N 209N N   CA   1614 1615  117.42  142.80                  -41.10\n",
      "   56  15903 209N 210E C   N    1620 1622 -127.11 -117.80   17.66    0.66  -63.60  179.51   27.52\n",
      "   56        210E 210E N   CA   1622 1623  151.80  136.80                  -40.30\n",
      "   57  15905 211T 212S C   N    1636 1638 -136.22 -136.60    2.20    0.12  -64.10 -173.83   18.61\n",
      "   57        212S 212S N   CA   1638 1639  153.36  151.20                  -35.00\n",
      "   58  15906 212S 213S C   N    1642 1644  -79.50  -72.40    7.10    0.51  -64.10  173.50   13.55\n",
      "   58        213S 213S N   CA   1644 1645  152.18  152.40                  -35.00\n",
      "   59  15907 213S 214R C   N    1648 1650 -152.96 -125.20   44.91    1.45  -63.00  168.94   28.82\n",
      "   59        214R 214R N   CA   1650 1651  175.90  140.60                  -41.10\n",
      "   60  15908 214R 215S C   N    1659 1661 -144.51 -136.60   41.59    1.93  -64.10  155.39   17.06\n",
      "   60        215S 215S N   CA   1661 1662 -167.97  151.20                  -35.00\n",
      "   61  15909 215S 216H C   N    1665 1667  103.11   56.30   87.49   12.09  -63.20 -131.28   38.56\n",
      "   61        216H 216H N   CA   1667 1668  114.72   40.80                  -42.30\n",
      "   62  15910 216H 217A C   N    1675 1677  173.90 -134.00   52.13    1.78  -62.50 -146.87   41.45\n",
      "   62        217A 217A N   CA   1677 1678  145.47  147.00                  -40.90\n",
      "   63  15914 220Q 221I C   N    1704 1706   57.71  -97.30  162.14    7.64 -120.60 -176.23   13.97\n",
      "   63        221I 221I N   CA   1706 1707  174.76  127.20                  130.30\n",
      "   64  15917 223I 224S C   N    1727 1729   54.97  -72.40  143.01    7.98  -64.10  170.75   20.90\n",
      "   64        224S 224S N   CA   1729 1730   87.38  152.40                  -35.00\n",
      "   65  15918 224S 225Q C   N    1733 1735  -87.01  -73.00   38.59    2.81  -63.80  146.89   20.45\n",
      "   65        225Q 225Q N   CA   1735 1736  104.74  140.70                  -40.30\n",
      "   66  15921 227Y 228G C   N    1763 1765  106.66   82.20   32.99    1.09  -62.40  171.29   31.56\n",
      "   66        228G 228G N   CA   1765 1766  -13.64    8.50                  -41.20\n",
      "   67  15923 229S 230L C   N    1773 1775  -90.89  -70.70   51.39    3.26  -63.50  132.80   20.11\n",
      "   67        230L 230L N   CA   1775 1776 -171.14  141.60                  -41.20\n",
      "   68  15926 232M 233R C   N    1797 1799   59.71   57.30   19.80    1.14  -63.00  136.35   24.79\n",
      "   68        233R 233R N   CA   1799 1800   18.34   38.00                  -41.10\n",
      "   69  15930 236E 237S C   N    1832 1834   83.79 -136.60  140.50    5.18  -64.10 -143.56   17.03\n",
      "   69        237S 237S N   CA   1834 1835  166.97  151.20                  -35.00\n",
      "   70  15947 253T 254E C   N    1957 1959  -14.48  -69.30   97.04    8.84  -63.60  108.83   12.62\n",
      "   70        254E 254E N   CA   1959 1960 -137.42  142.50                  -40.30\n",
      "   71  15948 254E 255S C   N    1966 1968 -166.20 -136.60   29.90    1.08  -64.10 -162.09   21.70\n",
      "   71        255S 255S N   CA   1968 1969  155.46  151.20                  -35.00\n",
      "   72  15949 255S 256T C   N    1972 1974 -160.71 -124.80   41.54    1.58  -63.20 -178.14   29.50\n",
      "   72        256T 256T N   CA   1974 1975  164.39  143.50                  -42.10\n",
      "   73  15950 256T 257G C   N    1979 1981 -177.47 -167.20   12.66    0.44   82.20 -172.26   13.52\n",
      "   73        257G 257G N   CA   1981 1982  167.19  174.60                    8.50\n",
      "   74  15952 258V 259A C   N    1990 1992 -145.06 -134.00   17.90    0.62  -62.50  178.30   33.64\n",
      "   74        259A 259A N   CA   1992 1993  161.07  147.00                  -40.90\n",
      "   75  15953 259A 260F C   N    1995 1997 -125.18 -124.20   14.76    0.67  -63.20  169.42   27.60\n",
      "   75        260F 260F N   CA   1997 1998  158.03  143.30                  -44.30\n",
      "   76  15954 260F 261E C   N    2006 2008 -147.94 -117.80   31.93    1.44  -63.60 -173.28   21.65\n",
      "   76        261E 261E N   CA   2008 2009  126.28  136.80                  -40.30\n",
      "   77  15955 261E 262E C   N    2015 2017 -174.53 -117.80   74.95    3.92  -63.60  169.47   19.89\n",
      "   77        262E 262E N   CA   2017 2018   87.81  136.80                  -40.30\n",
      "   78  15956 262E 263A C   N    2024 2026  160.18 -134.00   73.31    1.67  -62.50 -164.01   39.11\n",
      "   78        263A 263A N   CA   2026 2027  179.27  147.00                  -40.90\n",
      "   79  15957 263A 264K C   N    2029 2031  -69.16  -70.20    1.11    0.09  -62.90  178.50   23.53\n",
      "   79        264K 264K N   CA   2031 2032  140.81  140.40                  -40.80\n",
      "   80  15958 264K 265K C   N    2038 2040  -82.51  -70.20   12.53    0.86  -62.90  177.57   24.18\n",
      "   80        265K 265K N   CA   2040 2041  142.71  140.40                  -40.80\n",
      "   81  15960 266I 267N C   N    2055 2057 -114.73 -119.90    5.19    0.20  -63.20 -175.06   20.72\n",
      "   81        267N 267N N   CA   2057 2058  136.51  137.00                  -41.10\n",
      "   82  15961 267N 268Q C   N    2063 2065  -80.73  -73.00   19.32    1.13  -63.80  162.18   24.75\n",
      "   82        268Q 268Q N   CA   2065 2066  158.40  140.70                  -40.30\n",
      "   83  15963 269S 270L C   N    2078 2080 -107.28 -108.50    2.29    0.11  -63.50  177.26   22.51\n",
      "   83        270L 270L N   CA   2080 2081  130.57  132.50                  -41.20\n",
      "   84  15964 270L 271L C   N    2086 2088 -120.72 -108.50   18.92    0.87  -63.50 -178.88   28.68\n",
      "   84        271L 271L N   CA   2088 2089  146.95  132.50                  -41.20\n",
      "   85  15965 271L 272M C   N    2094 2096 -116.17 -125.60   19.09    1.04  -63.40  170.76   29.70\n",
      "   85        272M 272M N   CA   2096 2097  157.09  140.50                  -40.50\n",
      "   86  15966 272M 273L C   N    2102 2104  -71.70  -70.70    4.42    0.30  -63.50  173.09   24.51\n",
      "   86        273L 273L N   CA   2104 2105  145.90  141.60                  -41.20\n",
      "   87  15967 273L 274G C   N    2110 2112   75.20   78.70    5.89    0.10   82.20  170.02    8.61\n",
      "   87        274G 274G N   CA   2112 2113 -161.37 -166.10                    8.50\n",
      "   88  15968 274G 275R C   N    2114 2116  -67.37  -72.10    7.77    0.67  -63.00  170.90   23.38\n",
      "   88        275R 275R N   CA   2116 2117  148.06  141.90                  -41.10\n",
      "   89  15969 275R 276A C   N    2125 2127  -61.41  -68.20   19.09    1.31  -62.50  168.36   27.64\n",
      "   89        276A 276A N   CA   2127 2128  127.46  145.30                  -40.90\n",
      "   90  15970 276A 277M C   N    2130 2132  -68.85  -63.40   14.21    1.79  -73.00  170.43   11.45\n",
      "   90        277M 277M N   CA   2132 2133  -27.38  -40.50                  143.00\n",
      "   91  15971 277M 278N C   N    2138 2140  -54.11  -63.20    9.89    1.28 -119.90 -170.23    9.86\n",
      "   91        278N 278N N   CA   2140 2141  -45.00  -41.10                  137.00\n",
      "   92  15972 278N 279S C   N    2146 2148  -64.35  -72.40   18.24    0.85  -64.10  171.04   12.38\n",
      "   92        279S 279S N   CA   2148 2149  136.04  152.40                  -35.00\n",
      "   93  15992 298T 299R C   N    2297 2299 -156.64 -125.20   33.52    1.63  -63.00 -165.85   21.99\n",
      "   93        299R 299R N   CA   2299 2300  128.98  140.60                  -41.10\n",
      "   94  15993 299R 300L C   N    2308 2310 -132.19 -108.50   25.23    1.12  -63.50 -169.58   30.69\n",
      "   94        300L 300L N   CA   2310 2311  141.20  132.50                  -41.20\n",
      "   95  15994 300L 301L C   N    2316 2318  -79.22  -70.70    8.89    0.72  -63.50  175.37   25.28\n",
      "   95        301L 301L N   CA   2318 2319  144.13  141.60                  -41.20\n",
      "   96  15995 301L 302S C   N    2324 2326 -143.78 -136.60   32.23    1.47  -64.10  163.16   17.55\n",
      "   96        302S 302S N   CA   2326 2327 -177.38  151.20                  -35.00\n",
      "   97  15996 302S 303E C   N    2330 2332  -62.59  -69.30   15.50    1.38  -63.60  163.23   21.71\n",
      "   97        303E 303E N   CA   2332 2333  156.47  142.50                  -40.30\n",
      "   98  15997 303E 304S C   N    2339 2341  -56.46  -72.40   16.05    1.09  -64.10  174.59   12.22\n",
      "   98        304S 304S N   CA   2341 2342  150.57  152.40                  -35.00\n",
      "   99  15998 304S 305F C   N    2345 2347 -108.36 -124.20   17.04    0.84  -63.20  172.13   26.92\n",
      "   99        305F 305F N   CA   2347 2348  149.60  143.30                  -44.30\n",
      "  100  15999 305F 306G C   N    2356 2358  -61.51  -80.20   50.42    1.22   82.20 -173.56    6.52\n",
      "  100        306G 306G N   CA   2358 2359  127.27  174.10                    8.50\n",
      "  101  16000 306G 307G C   N    2360 2362   76.13   78.70   22.78    0.75   82.20  162.88    7.60\n",
      "  101        307G 307G N   CA   2362 2363  171.27 -166.10                    8.50\n",
      "  102  16001 307G 308N C   N    2364 2366 -128.21 -119.90   10.40    0.55  -63.20 -176.27   20.15\n",
      "  102        308N 308N N   CA   2366 2367  130.75  137.00                  -41.10\n",
      "  103  16002 308N 309S C   N    2372 2374 -132.82 -136.60   11.28    0.48  -64.10 -171.46   11.66\n",
      "  103        309S 309S N   CA   2374 2375  140.57  151.20                  -35.00\n",
      "  104  16003 309S 310K C   N    2378 2380 -128.60 -118.00   10.63    0.42  -62.90 -169.27   22.14\n",
      "  104        310K 310K N   CA   2380 2381  138.26  139.10                  -40.80\n",
      "  105  16004 310K 311T C   N    2387 2389 -108.26 -124.80   44.27    1.46  -63.20  151.40   16.85\n",
      "  105        311T 311T N   CA   2389 2390  102.44  143.50                  -42.10\n",
      "  106  16006 312W 313M C   N    2408 2410  -89.33  -73.00   25.35    1.59  -63.40  159.24   25.84\n",
      "  106        313M 313M N   CA   2410 2411  162.38  143.00                  -40.50\n",
      "  107  16007 313M 314L C   N    2416 2418  -68.47  -70.70    2.35    0.23  -63.50  176.52   24.80\n",
      "  107        314L 314L N   CA   2418 2419  142.35  141.60                  -41.20\n",
      "  108  16008 314L 315A C   N    2424 2426 -143.50 -134.00   17.29    0.65  -62.50  177.25   33.38\n",
      "  108        315A 315A N   CA   2426 2427  161.44  147.00                  -40.90\n",
      "  109  16009 315A 316T C   N    2429 2431  -65.56  -78.10   21.79    0.73  -63.20  174.09   22.17\n",
      "  109        316T 316T N   CA   2431 2432  131.97  149.80                  -42.10\n",
      "  110  16012 318S 319T C   N    2449 2451  -75.61  -78.10   28.16    1.29  -63.20  140.60   18.85\n",
      "  110        319T 319T N   CA   2451 2452  177.84  149.80                  -42.10\n",
      "  111  16015 321A 322N C   N    2466 2468  -69.07  -71.20   13.64    0.93   55.90  171.04   10.59\n",
      "  111        322N 322N N   CA   2468 2469  156.28  142.80                   39.50\n",
      "  112  16035 341T 342N C   N    2618 2620  -82.43 -119.90   56.17    1.67  -63.20  137.60   16.21\n",
      "  112        342N 342N N   CA   2620 2621   95.15  137.00                  -41.10\n",
      "  113  16036 342N 343K C   N    2626 2628  -85.36  -62.90   87.99   10.42  -62.90   87.99   10.42\n",
      "  113        343K 343K N   CA   2628 2629   44.28  -40.80                  -40.80\n",
      "  114  16037 343K 344A C   N    2635 2637 -105.94  -62.50   91.65   13.73  -62.50   91.65   13.73\n",
      "  114        344A 344A N   CA   2637 2638   39.79  -40.90                  -40.90\n",
      "  115  16038 344A 345K C   N    2640 2642   52.27   56.60   14.05    1.34  -62.90  132.75   23.18\n",
      "  115        345K 345K N   CA   2642 2643   25.23   38.60                  -40.80\n",
      "  116  16040 346V 347N C   N    2656 2658  -94.38 -119.90   43.83    1.36   55.90  162.51   14.40\n",
      "  116        347N 347N N   CA   2658 2659  101.36  137.00                   39.50\n",
      "  117  16043 349T 350K C   N    2679 2681   49.83   56.60   48.23    2.96  -62.90  169.93   28.58\n",
      "  117        350K 350K N   CA   2681 2682   86.35   38.60                  -40.80\n",
      "  118  16044 350K 351K C   N    2688 2690  -60.57  -70.20    9.85    0.67  -62.90  179.14   23.39\n",
      "  118        351K 351K N   CA   2690 2691  138.32  140.40                  -40.80\n",
      "  119  16045 351K 352N C   N    2697 2699 -172.20 -119.90   84.08    2.55  -63.20  159.23   25.61\n",
      "  119        352N 352N N   CA   2699 2700 -157.17  137.00                  -41.10\n",
      "  120  16046 352N 353I C   N    2705 2707  -61.86  -63.40    4.61    0.81 -120.60  179.45    8.14\n",
      "  120        353I 353I N   CA   2707 2708  -39.26  -43.60                  130.30\n",
      "  121  16047 353I 354E C   N    2713 2715 -158.64 -117.80   67.73    3.69  -63.60  155.50   18.09\n",
      "  121        354E 354E N   CA   2715 2716   82.77  136.80                  -40.30\n",
      "  122  16049 355L 356K C   N    2730 2732  -87.22  -62.90   39.12    6.50  -62.90   39.12    6.50\n",
      "  122        356K 356K N   CA   2732 2733  -71.45  -40.80                  -40.80\n",
      "  123  16056 362V 363A C   N    2793 2795  173.03 -134.00   53.30    1.99  -68.20  118.85   10.21\n",
      "  123        363A 363A N   CA   2795 2796  141.13  147.00                  145.30\n",
      "  124  16057 363A 364S C   N    2798 2800  -94.23  -72.40   35.54    1.63  -64.10  147.66   12.73\n",
      "  124        364S 364S N   CA   2800 2801 -179.56  152.40                  -35.00\n",
      "  125  16058 364S 365L C   N    2804 2806  -81.39  -70.70   19.46    1.20  -63.50  161.93   23.55\n",
      "  125        365L 365L N   CA   2806 2807  157.86  141.60                  -41.20\n",
      "  126  16059 365L 366E C   N    2812 2814  -79.18  -69.30   20.61    1.23  -63.60  159.87   22.18\n",
      "  126        366E 366E N   CA   2814 2815  160.59  142.50                  -40.30\n",
      "  127  16061 367G 368R C   N    2825 2827 -144.58 -125.20   52.74    2.06  -63.00  152.84   26.08\n",
      "  127        368R 368R N   CA   2827 2828 -170.35  140.60                  -41.10\n",
      "  128  16062 368R 369L C   N    2836 2838  -60.99  -70.70   10.90    0.80  -63.50  177.87   24.84\n",
      "  128        369L 369L N   CA   2838 2839  136.65  141.60                  -41.20\n",
      "  129  16063 369L 370D C   N    2844 2846 -137.62  -96.50   41.25    1.69  -63.30  174.11   18.51\n",
      "  129        370D 370D N   CA   2846 2847  117.46  114.20                  -40.00\n",
      "  130  16065 371V 372L C   N    2859 2861  -95.98 -108.50   68.52    3.56  -70.70   80.54    7.08\n",
      "  130        372L 372L N   CA   2861 2862   65.13  132.50                  141.60\n",
      "  131  16067 373A 374L C   N    2872 2874 -151.82 -108.50   43.78    2.02  -70.70   81.18    7.47\n",
      "  131        374L 374L N   CA   2874 2875  138.78  132.50                  141.60\n",
      "  132  16068 374L 375E C   N    2880 2882  -22.64  -69.30   59.27    3.34  -63.60  151.89   22.57\n",
      "  132        375E 375E N   CA   2882 2883  105.96  142.50                  -40.30\n",
      "  133  16069 375E 376K C   N    2889 2891  132.63 -118.00  109.90    4.34  -62.90 -124.16   28.54\n",
      "  133        376K 376K N   CA   2891 2892  128.23  139.10                  -40.80\n",
      "  134  16071 377Q 378R C   N    2907 2909  -42.91  -72.10   29.40    2.17  -63.00 -179.31   25.78\n",
      "  134        378R 378R N   CA   2909 2910  138.47  141.90                  -41.10\n",
      "  135  16072 378R 379K C   N    2918 2920 -178.67 -118.00   62.38    2.15  -62.90 -157.94   33.08\n",
      "  135        379K 379K N   CA   2920 2921  153.59  139.10                  -40.80\n",
      "  136  16073 379K 380Q C   N    2927 2929  173.67 -121.10   70.26    2.21  -63.80 -163.27   34.92\n",
      "  136        380Q 380Q N   CA   2929 2930  165.78  139.70                  -40.30\n",
      "  137  16074 380Q 381E C   N    2936 2938 -156.56 -117.80   39.66    1.27  -63.60 -162.30   31.53\n",
      "  137        381E 381E N   CA   2938 2939  145.22  136.80                  -40.30\n",
      "  138  16075 381E 382E C   N    2945 2947 -170.61 -117.80   53.01    2.06  -63.60 -157.00   23.45\n",
      "  138        382E 382E N   CA   2947 2948  132.20  136.80                  -40.30\n",
      "  139  16078 384G 385R C   N    2966 2968 -123.08 -125.20   24.99    1.25  -63.00  164.75   26.39\n",
      "  139        385R 385R N   CA   2968 2969  165.50  140.60                  -41.10\n",
      "  140  16079 385R 386L C   N    2977 2979 -111.53 -108.50   20.41    1.07  -63.50  172.92   26.97\n",
      "  140        386L 386L N   CA   2979 2980  152.68  132.50                  -41.20\n",
      "  141  16080 386L 387T C   N    2985 2987 -113.48 -124.80   11.67    0.54  -63.20 -175.79   26.97\n",
      "  141        387T 387T N   CA   2987 2988  140.69  143.50                  -42.10\n",
      "  142  16081 387T 388Q C   N    2992 2994  -96.37 -121.10   48.76    1.80  -63.80  141.77   19.29\n",
      "  142        388Q 388Q N   CA   2994 2995   97.68  139.70                  -40.30\n",
      "  143  16082 388Q 389E C   N    3001 3003 -146.50 -117.80   38.46    1.20  -63.60  177.80   28.32\n",
      "  143        389E 389E N   CA   3003 3004  162.41  136.80                  -40.30\n",
      "  144  16084 390R 391D C   N    3021 3023  -96.22  -96.50    0.99    0.04  -63.30  156.74   17.61\n",
      "  144        391D 391D N   CA   3023 3024  113.25  114.20                  -40.00\n",
      "  145  16085 391D 392K C   N    3029 3031 -127.44 -118.00   35.73    1.83  -62.90  159.12   18.32\n",
      "  145        392K 392K N   CA   3031 3032  104.64  139.10                  -40.80\n",
      "  146  16086 392K 393L C   N    3038 3040 -150.99 -108.50   42.83    2.11  -63.50 -170.37   23.23\n",
      "  146        393L 393L N   CA   3040 3041  127.04  132.50                  -41.20\n",
      "  147  16087 393L 394R C   N    3046 3048  -79.68  -72.10    9.11    0.60  -63.00  172.75   24.48\n",
      "  147        394R 394R N   CA   3048 3049  146.96  141.90                  -41.10\n",
      "  148  16088 394R 395Q C   N    3057 3059 -153.06 -121.10   34.70    1.59  -63.80 -171.08   24.34\n",
      "  148        395Q 395Q N   CA   3059 3060  126.19  139.70                  -40.30\n",
      "  149  16091 397V 398A C   N    3082 3084  167.86 -134.00   87.62    5.21  -68.20  139.42   13.50\n",
      "  149        398A 398A N   CA   3084 3085   81.45  147.00                  145.30\n",
      "  150  16092 398A 399F C   N    3087 3089  -92.01  -63.20   73.78    8.60  -63.20   73.78    8.60\n",
      "  150        399F 399F N   CA   3089 3090   23.62  -44.30                  -44.30\n",
      "  151  16094 400A 401D C   N    3103 3105   62.15  -63.30  133.11   18.53  -63.30  133.11   18.53\n",
      "  151        401D 401D N   CA   3105 3106  -84.51  -40.00                  -40.00\n",
      "  152  16097 403V 404H C   N    3124 3126  128.91 -125.60  107.04    3.36  -63.20 -127.57   39.14\n",
      "  152        404H 404H N   CA   3126 3127  156.96  138.80                  -42.30\n",
      "  153  16098 404H 405D C   N    3134 3136 -160.19  -96.50   63.87    2.61  -63.30 -173.77   19.80\n",
      "  153        405D 405D N   CA   3136 3137  119.05  114.20                  -40.00\n",
      "  154  16099 405D 406A C   N    3142 3144  140.32 -134.00   88.54    2.20  -62.50 -142.87   43.50\n",
      "  154        406A 406A N   CA   3144 3145  169.30  147.00                  -40.90\n",
      "  155  16101 407R 408D C   N    3158 3160 -147.52  -96.50   67.92    2.78  -63.30  138.03   14.95\n",
      "  155        408D 408D N   CA   3160 3161   69.36  114.20                  -40.00\n",
      "  156  16102 408D 409K C   N    3166 3168 -135.13  -62.90   92.60   16.00  -62.90   92.60   16.00\n",
      "  156        409K 409K N   CA   3168 3169  -98.75  -40.80                  -40.80\n",
      "  157  16103 409K 410L C   N    3175 3177  -64.93  -63.50   53.52    7.51  -63.50   53.52    7.51\n",
      "  157        410L 410L N   CA   3177 3178  -94.70  -41.20                  -41.20\n",
      "  158  16104 410L 411E C   N    3183 3185  -93.53  -63.60   49.66    8.23  -63.60   49.66    8.23\n",
      "  158        411E 411E N   CA   3185 3186  -79.93  -40.30                  -40.30\n",
      "  159  16108 414L 415N C   N    3213 3215   40.59 -119.90  160.62    6.83  -63.20 -156.24   21.90\n",
      "  159        415N 415N N   CA   3215 3216  143.55  137.00                  -41.10\n",
      "  160  16109 415N 416N C   N    3221 3223  174.61 -119.90   66.09    2.52  -63.20 -148.16   33.16\n",
      "  160        416N 416N N   CA   3223 3224  145.85  137.00                  -41.10\n",
      "  161  16111 417I 418R C   N    3237 3239 -176.31 -125.20   51.36    1.93  -63.00 -153.03   35.48\n",
      "  161        418R 418R N   CA   3239 3240  145.70  140.60                  -41.10\n",
      "  162  16112 418R 419L C   N    3248 3250 -134.56 -108.50   49.48    2.34  -63.50  160.79   26.72\n",
      "  162        419L 419L N   CA   3250 3251  174.56  132.50                  -41.20\n",
      "  163  16113 419L 420S C   N    3256 3258 -122.72 -136.60   22.08    0.76  -64.10  178.90   11.02\n",
      "  163        420S 420S N   CA   3258 3259  134.02  151.20                  -35.00\n",
      "  164  16114 420S 421N C   N    3262 3264 -174.54 -119.90   65.70    1.93  -63.20 -176.85   28.91\n",
      "  164        421N 421N N   CA   3264 3265  173.48  137.00                  -41.10\n",
      "  165  16116 422I 423A C   N    3278 3280  -68.68  -68.20    4.92    0.38  -62.50  169.02   28.01\n",
      "  165        423A 423A N   CA   3280 3281  150.20  145.30                  -40.90\n",
      "  166  16117 423A 424L C   N    3283 3285  -93.87  -70.70   34.03    2.15  -63.50  155.28   23.41\n",
      "  166        424L 424L N   CA   3285 3286  166.52  141.60                  -41.20\n",
      "  167  16118 424L 425R C   N    3291 3293  -68.65  -72.10    9.61    0.81  -63.00  168.12   23.09\n",
      "  167        425R 425R N   CA   3293 3294  150.87  141.90                  -41.10\n",
      "  168  16119 425R 426R C   N    3302 3304 -130.70 -125.20   23.69    1.01  -63.00  169.38   27.52\n",
      "  168        426R 426R N   CA   3304 3305  163.64  140.60                  -41.10\n",
      "  169  16120 426R 427R C   N    3313 3315  -98.09  -72.10   27.26    2.25  -63.00  178.24   21.92\n",
      "  169        427R 427R N   CA   3315 3316  133.66  141.90                  -41.10\n",
      "  170  16123 429E 430A C   N    3340 3342 -118.86 -134.00   16.64    0.82  -62.50  174.54   31.61\n",
      "  170        430A 430A N   CA   3342 3343  153.92  147.00                  -40.90\n",
      "  171  16124 430A 431A C   N    3345 3347 -109.91 -134.00   24.34    0.68  -62.50 -178.11   32.32\n",
      "  171        431A 431A N   CA   3347 3348  143.50  147.00                  -40.90\n",
      "  172  16125 431A 432S C   N    3350 3352 -143.85 -136.60   16.32    0.64  -64.10  178.03   18.61\n",
      "  172        432S 432S N   CA   3352 3353  165.82  151.20                  -35.00\n",
      "  173  16126 432S 433E C   N    3356 3358 -121.43 -117.80   13.85    0.62  -63.60  179.12   27.15\n",
      "  173        433E 433E N   CA   3358 3359  150.17  136.80                  -40.30\n",
      "  174  16127 433E 434G C   N    3365 3367 -177.44 -167.20   12.69    0.23   82.20 -159.48   14.21\n",
      "  174        434G 434G N   CA   3367 3368 -177.90  174.60                    8.50\n",
      "  175  16131 437H 438S C   N    3398 3400  -66.60  -72.40   10.38    0.47  -64.10  178.82   12.81\n",
      "  175        438S 438S N   CA   3400 3401  143.80  152.40                  -35.00\n",
      "  176  16132 438S 439L C   N    3404 3406 -134.93 -108.50   33.11    1.47  -63.50 -178.96   29.56\n",
      "  176        439L 439L N   CA   3406 3407  152.44  132.50                  -41.20\n",
      "  177  16133 439L 440D C   N    3412 3414 -135.31  -63.30   72.57   11.89  -63.30   72.57   11.89\n",
      "  177        440D 440D N   CA   3414 3415  -48.98  -40.00                  -40.00\n",
      "  178  16134 440D 441N C   N    3420 3422 -137.72 -119.90   66.77    2.59  -63.20  139.19   21.52\n",
      "  178        441N 441N N   CA   3422 3423 -158.65  137.00                  -41.10\n",
      "  179  16135 441N 442K C   N    3428 3430  -71.80  -70.20   63.58    4.57  -62.90  115.58   15.51\n",
      "  179        442K 442K N   CA   3430 3431 -156.04  140.40                  -40.80\n",
      "  180  16136 442K 443S C   N    3437 3439  -58.78  -72.40   15.39    0.85  -64.10  179.84   12.72\n",
      "  180        443S 443S N   CA   3439 3440  145.23  152.40                  -35.00\n",
      "  181  16138 444C 445F C   N    3449 3451  -84.15  -71.40   26.23    2.33  -63.20  163.43   21.32\n",
      "  181        445F 445F N   CA   3451 3452  117.78  140.70                  -44.30\n",
      "  182  16139 445F 446L C   N    3460 3462   -7.06  -70.70   74.54    5.27  -63.50  154.66   24.97\n",
      "  182        446L 446L N   CA   3462 3463  102.80  141.60                  -41.20\n",
      "  183  16140 446L 447F C   N    3468 3470   29.57  -71.40  103.37    7.71  -63.20  178.80   20.25\n",
      "  183        447F 447F N   CA   3470 3471  162.84  140.70                  -44.30\n",
      "  184  16141 447F 448F C   N    3479 3481 -118.89 -124.20   47.24    2.09  -63.20  151.29   17.78\n",
      "  184        448F 448F N   CA   3481 3482   96.36  143.30                  -44.30\n",
      "  185  16142 448F 449K C   N    3490 3492 -127.10  -62.90   79.95   13.87  -62.90   79.95   13.87\n",
      "  185        449K 449K N   CA   3492 3493  -88.45  -40.80                  -40.80\n",
      "  186  16143 449K 450G C   N    3499 3501  -86.90  -80.20   33.69    1.01   82.20 -126.25   18.00\n",
      "  186        450G 450G N   CA   3501 3502 -152.88  174.10                    8.50\n",
      "  187  16144 450G 451R C   N    3503 3505   70.64   57.30   83.41    6.49  -63.00 -150.43   36.90\n",
      "  187        451R 451R N   CA   3505 3506  120.33   38.00                  -41.10\n",
      "  188  16146 452C 453S C   N    3520 3522  -94.65  -72.40   56.83    2.69  -64.10  124.12   11.08\n",
      "  188        453S 453S N   CA   3522 3523 -155.31  152.40                  -35.00\n",
      "  189  16148 454I 455T C   N    3534 3536 -127.06 -124.80    5.20    0.17  -63.20 -178.66   27.48\n",
      "  189        455T 455T N   CA   3536 3537  148.17  143.50                  -42.10\n",
      "  190  16149 455T 456L C   N    3541 3543  -91.35 -108.50   84.92    4.39  -63.50   94.72   11.88\n",
      "  190        456L 456L N   CA   3543 3544   49.33  132.50                  -41.20\n",
      "  191  16151 457E 458S C   N    3558 3560 -125.88 -136.60   51.17    2.35  -64.10  149.53    9.37\n",
      "  191        458S 458S N   CA   3560 3561  101.17  151.20                  -35.00\n",
      "  192  16153 459V 460L C   N    3571 3573  168.29 -108.50   89.66    3.98  -63.50 -160.45   35.37\n",
      "  192        460L 460L N   CA   3573 3574  165.88  132.50                  -41.20\n",
      "  193  16154 460L 461R C   N    3579 3581  -79.89  -72.10   68.13    5.41  -63.00  116.54   14.68\n",
      "  193        461R 461R N   CA   3581 3582   74.21  141.90                  -41.10\n",
      "  194  16177 483L 484H C   N    3759 3761   50.69   56.30    8.97    0.77  -63.20  145.22   24.75\n",
      "  194        484H 484H N   CA   3761 3762   47.80   40.80                  -42.30\n",
      "  195  16179 485I 486Q C   N    3777 3779  -99.81 -121.10   26.12    1.29  -63.80  168.75   26.74\n",
      "  195        486Q 486Q N   CA   3779 3780  154.83  139.70                  -40.30\n",
      "  196  16180 486Q 487L C   N    3786 3788  -66.11  -70.70    7.53    0.47  -63.50  176.85   24.41\n",
      "  196        487L 487L N   CA   3788 3789  135.63  141.60                  -41.20\n",
      "  197  16181 487L 488F C   N    3794 3796 -128.48 -124.20   18.11    0.74  -63.20  168.00   27.59\n",
      "  197        488F 488F N   CA   3796 3797  160.90  143.30                  -44.30\n",
      "  198  16184 490C 491E C   N    3818 3820 -127.74 -117.80   10.17    0.43  -63.60 -173.67   22.02\n",
      "  198        491E 491E N   CA   3820 3821  134.64  136.80                  -40.30\n",
      "  199  16185 491E 492H C   N    3827 3829 -127.93 -125.60   13.01    0.45  -63.20  178.27   26.52\n",
      "  199        492H 492H N   CA   3829 3830  151.60  138.80                  -42.30\n",
      "  200  16186 492H 493H C   N    3837 3839  -92.19  -67.60   65.40    5.65  -63.20  125.11   14.07\n",
      "  200        493H 493H N   CA   3839 3840   79.40  140.00                  -42.30\n",
      "  201  16187 493H 494A C   N    3847 3849 -168.70 -134.00   38.10    0.87  -62.50 -170.99   36.59\n",
      "  201        494A 494A N   CA   3849 3850  162.74  147.00                  -40.90\n",
      "  202  16188 494A 495N C   N    3852 3854 -115.64 -119.90    4.64    0.15  -63.20 -176.12   20.56\n",
      "  202        495N 495N N   CA   3854 3855  135.15  137.00                  -41.10\n",
      "  203  16189 495N 496E C   N    3860 3862  -81.10  -69.30   56.79    4.72  -63.60  128.45   16.23\n",
      "  203        496E 496E N   CA   3862 3863   86.95  142.50                  -40.30\n",
      "  204  16191 497R 498N C   N    3880 3882 -147.84  -63.20   97.01   15.95  -63.20   97.01   15.95\n",
      "  204        498N 498N N   CA   3882 3883  -88.50  -41.10                  -41.10\n",
      "  205  16192 498N 499D C   N    3888 3890  -56.07  -70.90   18.09    1.51  -63.30  159.50   19.26\n",
      "  205        499D 499D N   CA   3890 3891  160.67  150.30                  -40.00\n",
      "  206  16195 501M 502Q C   N    3911 3913  -41.38  -73.00   34.14    2.63  -63.80  167.63   23.52\n",
      "  206        502Q 502Q N   CA   3913 3914  153.58  140.70                  -40.30\n",
      "  207  16196 502Q 503F C   N    3920 3922 -116.85 -124.20   41.02    2.10  -63.20  142.52   23.31\n",
      "  207        503F 503F N   CA   3922 3923 -176.34  143.30                  -44.30\n",
      "  208  16199 505G 506K C   N    3943 3945  -75.56  -62.90   46.78    6.83  -62.90   46.78    6.83\n",
      "  208        506K 506K N   CA   3945 3946  -85.84  -40.80                  -40.80\n",
      "  209  16200 506K 507S C   N    3952 3954   43.03  -64.10  155.20   18.93  -64.10  155.20   18.93\n",
      "  209        507S 507S N   CA   3954 3955   77.29  -35.00                  -35.00\n",
      "  210  16205 511C 512L C   N    3994 3996  -45.80  -70.70   25.63    2.49  -63.50  172.04   22.93\n",
      "  210        512L 512L N   CA   3996 3997  147.67  141.60                  -41.20\n",
      "  211  16206 512L 513H C   N    4002 4004 -143.19 -125.60   22.61    0.56  -63.20 -176.91   28.09\n",
      "  211        513H 513H N   CA   4004 4005  153.00  138.80                  -42.30\n",
      "  212  16209 515V 516G C   N    4026 4028 -130.87 -167.20   48.31    1.00   82.20 -160.98   15.37\n",
      "  212        516G 516G N   CA   4028 4029  142.75  174.60                    8.50\n",
      "  213  16210 516G 517A C   N    4030 4032 -113.72 -134.00   21.80    0.50  -62.50 -172.95   28.61\n",
      "  213        517A 517A N   CA   4032 4033  139.00  147.00                  -40.90\n",
      "  214  16211 517A 518S C   N    4035 4037 -119.22 -136.60   21.22    0.65  -64.10 -177.45   11.25\n",
      "  214        518S 518S N   CA   4037 4038  139.02  151.20                  -35.00\n",
      "  215  16212 518S 519G C   N    4041 4043   79.18   78.70   22.05    0.61   82.20  163.38    7.78\n",
      "  215        519G 519G N   CA   4043 4044  171.86 -166.10                    8.50\n",
      "  216  16215 521P 522K C   N    4060 4062  -66.83  -70.20   15.62    1.05  -62.90  165.99   21.34\n",
      "  216        522K 522K N   CA   4062 4063  125.15  140.40                  -40.80\n",
      "  217  16216 522K 523A C   N    4069 4071  -87.69  -68.20   22.91    1.52  -62.50  163.72   28.13\n",
      "  217        523A 523A N   CA   4071 4072  157.34  145.30                  -40.90\n",
      "  218  16217 523A 524F C   N    4074 4076  -78.91  -71.40   10.29    0.59  -63.20  168.70   24.49\n",
      "  218        524F 524F N   CA   4076 4077  147.73  140.70                  -44.30\n",
      "  219  16219 525V 526A C   N    4092 4094 -148.71 -134.00   18.74    0.49  -62.50 -177.82   34.47\n",
      "  219        526A 526A N   CA   4094 4095  158.61  147.00                  -40.90\n",
      "  220  16220 526A 527H C   N    4097 4099  -87.35  -67.60   60.59    5.17  -63.20  127.34   14.57\n",
      "  220        527H 527H N   CA   4099 4100   82.72  140.00                  -42.30\n",
      "  221  16221 527H 528T C   N    4107 4109 -143.31 -124.80   38.11    1.20  -63.20  162.25   25.95\n",
      "  221        528T 528T N   CA   4109 4110  176.81  143.50                  -42.10\n",
      "  222  16222 528T 529F C   N    4114 4116  -71.41  -71.40    2.99    0.24  -63.20  172.20   24.46\n",
      "  222        529F 529F N   CA   4116 4117  143.69  140.70                  -44.30\n",
      "  223  16224 530C 531K C   N    4131 4133  -99.77 -118.00   42.70    2.21  -62.90  146.21   21.17\n",
      "  223        531K 531K N   CA   4133 4134  177.72  139.10                  -40.80\n",
      "  224  16225 531K 532F C   N    4140 4142 -114.21 -124.20   93.30    4.63  -63.20   94.57   16.30\n",
      "  224        532F 532F N   CA   4142 4143 -123.94  143.30                  -44.30\n",
      "  225  16226 532F 533T C   N    4151 4153  -60.29  -78.10   17.91    1.13  -63.20  166.19   21.12\n",
      "  225        533T 533T N   CA   4153 4154  151.74  149.80                  -42.10\n",
      "  226  16228 534L 535L C   N    4166 4168   55.82  -70.70  129.24   10.81  -63.50 -163.26   34.52\n",
      "  226        535L 535L N   CA   4168 4169  115.22  141.60                  -41.20\n",
      "  227  16229 535L 536H C   N    4174 4176  171.70 -125.60   62.70    2.20  -63.20 -141.46   35.26\n",
      "  227        536H 536H N   CA   4176 4177  138.50  138.80                  -42.30\n",
      "  228  16232 538R 539E C   N    4203 4205   48.73  -69.30  122.60    8.09  -63.60 -172.88   30.98\n",
      "  228        539E 539E N   CA   4205 4206  109.35  142.50                  -40.30\n",
      "  229  16233 539E 540E C   N    4212 4214  120.15 -117.80  158.07    4.83  -63.60 -165.41   33.37\n",
      "  229        540E 540E N   CA   4214 4215 -122.76  136.80                  -40.30\n",
      "  230  16234 540E 541R C   N    4221 4223 -133.19 -125.20   26.82    1.43  -63.00  171.14   19.58\n",
      "  230        541R 541R N   CA   4223 4224  114.99  140.60                  -41.10\n",
      "  231  16235 541R 542Y C   N    4232 4234  -56.03  -98.40   48.51    2.83  -63.50  164.74   25.76\n",
      "  231        542Y 542Y N   CA   4234 4235  152.03  128.40                  -43.40\n",
      "  232  16236 542Y 543F C   N    4244 4246   94.90 -124.20  140.95    5.62  -63.20 -123.25   27.01\n",
      "  232        543F 543F N   CA   4246 4247  139.47  143.30                  -44.30\n",
      "  233  16237 543F 544T C   N    4255 4257 -127.35 -124.80    2.94    0.11  -63.20 -175.55   27.90\n",
      "  233        544T 544T N   CA   4257 4258  144.97  143.50                  -42.10\n",
      "  234  16238 544T 545T C   N    4262 4264 -164.31 -124.80   76.87    2.39  -63.20  148.28   25.04\n",
      "  234        545T 545T N   CA   4264 4265 -150.56  143.50                  -42.10\n",
      "  235  16239 545T 546S C   N    4269 4271 -153.22 -136.60   27.40    1.55  -64.10 -172.99   12.11\n",
      "  235        546S 546S N   CA   4271 4272  129.41  151.20                  -35.00\n",
      "  236  16240 546S 547T C   N    4275 4277 -100.69 -124.80   31.87    1.07  -63.20  168.97   19.42\n",
      "  236        547T 547T N   CA   4277 4278  122.66  143.50                  -42.10\n",
      "  237  16241 547T 548A C   N    4282 4284 -134.74 -134.00    1.47    0.09  -62.50 -172.18   34.65\n",
      "  237        548A 548A N   CA   4284 4285  145.74  147.00                  -40.90\n",
      "  238  16242 548A 549E C   N    4287 4289 -113.02 -117.80   13.23    0.72  -63.60  177.58   26.47\n",
      "  238        549E 549E N   CA   4289 4290  149.14  136.80                  -40.30\n",
      "  239  16243 549E 550N C   N    4296 4298  -68.29  -71.20   20.27    1.22  -63.20  163.92   20.22\n",
      "  239        550N 550N N   CA   4298 4299  122.74  142.80                  -41.10\n",
      "  240  16244 550N 551T C   N    4304 4306 -136.77 -124.80   12.76    0.55  -63.20 -174.77   28.58\n",
      "  240        551T 551T N   CA   4306 4307  147.91  143.50                  -42.10\n",
      "  241  16245 551T 552Q C   N    4311 4313 -125.00 -121.10   14.16    0.72  -63.80  177.29   23.37\n",
      "  241        552Q 552Q N   CA   4313 4314  126.09  139.70                  -40.30\n",
      "  242  16246 552Q 553N C   N    4320 4322 -152.17 -119.90   32.73    1.22  -63.20 -162.38   29.71\n",
      "  242        553N 553N N   CA   4322 4323  142.45  137.00                  -41.10\n",
      "  243  16248 554P 555R C   N    4335 4337  177.34 -125.20   59.95    1.99  -63.00 -159.23   34.94\n",
      "  243        555R 555R N   CA   4337 4338  157.69  140.60                  -41.10\n",
      "  244  16249 555R 556W C   N    4346 4348  -82.26  -71.30   14.33    0.84  -63.00  168.68   22.31\n",
      "  244        556W 556W N   CA   4348 4349  148.22  139.00                  -44.20\n",
      "  245  16250 556W 557G C   N    4360 4362  179.83 -167.20   16.15    0.56   82.20 -175.57   13.25\n",
      "  245        557G 557G N   CA   4362 4363  164.97  174.60                    8.50\n",
      "  246  16251 557G 558Y C   N    4364 4366 -118.44 -124.30    8.45    0.37  -63.50 -178.75   26.20\n",
      "  246        558Y 558Y N   CA   4366 4367  129.32  135.40                  -43.40\n",
      "  247  16252 558Y 559V C   N    4376 4378  -62.04  -62.40    5.64    0.69 -125.40 -179.82   10.13\n",
      "  247        559V 559V N   CA   4378 4379  -48.02  -42.40                  143.30\n",
      "  248  16253 559V 560K C   N    4383 4385  -73.13  -70.20    8.93    0.71  -62.90  173.06   21.93\n",
      "  248        560K 560K N   CA   4385 4386  131.96  140.40                  -40.80\n",
      "  249  16255 561V 562F C   N    4399 4401 -133.85 -124.20   13.75    0.37  -63.20  177.29   29.22\n",
      "  249        562F 562F N   CA   4401 4402  153.10  143.30                  -44.30\n",
      "  250  16256 562F 563E C   N    4410 4412 -160.13 -117.80   47.86    1.40  -63.60 -172.64   30.29\n",
      "  250        563E 563E N   CA   4412 4413  159.12  136.80                  -40.30\n",
      "  251  16259 565P 566E C   N    4434 4436 -135.08 -117.80   84.30    3.88  -63.60  123.24   20.29\n",
      "  251        566E 566E N   CA   4436 4437 -140.69  136.80                  -40.30\n",
      "  252  16260 566E 567L C   N    4443 4445 -113.26 -108.50   75.09    4.01  -63.50  121.97   20.02\n",
      "  252        567L 567L N   CA   4445 4446 -152.56  132.50                  -41.20\n",
      "  253  16261 567L 568T C   N    4451 4453  -98.29  -78.10   69.55    2.41  -63.20  107.44   16.10\n",
      "  253        568T 568T N   CA   4453 4454 -143.65  149.80                  -42.10\n",
      "  254  16268 574C 575F C   N    4504 4506 -170.03 -124.20   47.77    1.48  -63.20 -168.51   33.17\n",
      "  254        575F 575F N   CA   4506 4507  156.78  143.30                  -44.30\n",
      "  255  16271 577E 578H C   N    4530 4532 -132.58 -125.60   16.77    0.49  -63.20  177.75   26.76\n",
      "  255        578H 578H N   CA   4532 4533  154.05  138.80                  -42.30\n",
      "  256  16272 578H 579T C   N    4540 4542  -96.30 -124.80   49.24    1.52  -63.20  149.17   17.15\n",
      "  256        579T 579T N   CA   4542 4543  103.35  143.50                  -42.10\n",
      "  257  16274 580V 581F C   N    4554 4556  -66.01  -71.40    6.69    0.57  -63.20  171.07   23.94\n",
      "  257        581F 581F N   CA   4556 4557  144.65  140.70                  -44.30\n",
      "  258  16275 581F 582A C   N    4565 4567  -61.84  -68.20   16.63    1.13  -62.50  170.83   28.03\n",
      "  258        582A 582A N   CA   4567 4568  129.93  145.30                  -40.90\n",
      "  259  16276 582A 583F C   N    4570 4572  169.65 -124.20   67.72    2.22  -63.20 -157.25   35.72\n",
      "  259        583F 583F N   CA   4572 4573  157.78  143.30                  -44.30\n",
      "  260  16277 583F 584E C   N    4581 4583  -95.75 -117.80   66.92    2.87  -63.60  118.36   14.27\n",
      "  260        584E 584E N   CA   4583 4584   73.61  136.80                  -40.30\n",
      "  261  16278 584E 585V C   N    4590 4592  -67.96  -62.40    6.59    1.13 -125.40 -179.85   10.05\n",
      "  261        585V 585V N   CA   4592 4593  -45.95  -42.40                  143.30\n",
      "  262  16279 585V 586F C   N    4597 4599 -154.05 -124.20   49.90    1.47  -63.20  160.58   27.88\n",
      "  262        586F 586F N   CA   4599 4600 -176.71  143.30                  -44.30\n",
      "  263  16280 586F 587A C   N    4608 4610  -74.66  -68.20   25.99    1.87  -62.50  149.12   25.05\n",
      "  263        587A 587A N   CA   4610 4611  170.48  145.30                  -40.90\n",
      "  264  16281 587A 588F C   N    4613 4615 -131.37 -124.20   52.24    2.28  -63.20  138.58   23.56\n",
      "  264        588F 588F N   CA   4615 4616 -164.96  143.30                  -44.30\n",
      "  265  16282 588F 589N C   N    4624 4626  -81.10  -71.20   80.59    4.90  -63.20   97.78   13.24\n",
      "  265        589N 589N N   CA   4626 4627 -137.23  142.80                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    5   94  149  544  515  712  595  700  776  872  865\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00200.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      590     4639     4639\n",
      "openf___224_> Open           EAO00200.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     69477392   67849.016    66.259\n",
      "read_mo_297_> Segments, residues, atoms:        1      590     4639\n",
      "read_mo_298_> Segment:        1  1:A  590:A     4639\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4639     4639        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    21546\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    22778\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    23393\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    24312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    25407\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    26173\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    27672\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    28645\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    29568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    31057\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    32296\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34090\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34272\n",
      "iupac_m_487_> NH1/2 swapped:     -176.6939       10  10\n",
      "iupac_m_485_> OD1/2 will be swapped:      111.4894       31  31\n",
      "iupac_m_485_> OD1/2 will be swapped:     -162.1607       33  33\n",
      "iupac_m_485_> OD1/2 will be swapped:       99.2902       34  34\n",
      "iupac_m_485_> OD1/2 will be swapped:      116.3061       41  41\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3335       46  46\n",
      "iupac_m_485_> OD1/2 will be swapped:     -150.6918       54  54\n",
      "iupac_m_487_> NH1/2 swapped:      179.2087       55  55\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.4229       66  66\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.2790       87  87\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -177.2898      113  113\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.8991      121  121\n",
      "iupac_m_485_> OD1/2 will be swapped:     -161.7558      150  150\n",
      "iupac_m_487_> NH1/2 swapped:      179.7925      155  155\n",
      "iupac_m_486_> OE1/2 will be swapped:     -151.8580      162  162\n",
      "iupac_m_486_> OE1/2 will be swapped:     -115.8059      165  165\n",
      "iupac_m_485_> OD1/2 will be swapped:      114.1296      169  169\n",
      "iupac_m_487_> NH1/2 swapped:      178.2085      173  173\n",
      "iupac_m_486_> OE1/2 will be swapped:      125.2216      188  188\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9477      189  189\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1107      201  201\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.0472      205  205\n",
      "iupac_m_485_> OD1/2 will be swapped:      117.2673      231  231\n",
      "iupac_m_487_> NH1/2 swapped:      179.3673      249  249\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.9606      254  254\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -134.6186      260  260\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.6666      261  261\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.6854      262  262\n",
      "iupac_m_487_> NH1/2 swapped:     -178.9592      275  275\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.7548      282  282\n",
      "iupac_m_487_> NH1/2 swapped:      179.7033      283  283\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.6624      303  303\n",
      "iupac_m_487_> NH1/2 swapped:      178.1371      357  357\n",
      "iupac_m_487_> NH1/2 swapped:     -177.6920      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:      165.1573      360  360\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9287      368  368\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.6205      370  370\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.5323      381  381\n",
      "iupac_m_486_> OE1/2 will be swapped:     -139.6245      382  382\n",
      "iupac_m_487_> NH1/2 swapped:      177.9404      385  385\n",
      "iupac_m_486_> OE1/2 will be swapped:      157.6138      389  389\n",
      "iupac_m_487_> NH1/2 swapped:      178.8177      390  390\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.0502      391  391\n",
      "iupac_m_485_> OD1/2 will be swapped:      144.5616      401  401\n",
      "iupac_m_485_> OD1/2 will be swapped:     -115.1063      405  405\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4325      407  407\n",
      "iupac_m_486_> OE1/2 will be swapped:      164.6156      411  411\n",
      "iupac_m_487_> NH1/2 swapped:      174.8743      425  425\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.0853      429  429\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -143.9429      435  435\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1898      461  461\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      126.0759      467  467\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.4631      491  491\n",
      "iupac_m_486_> OE1/2 will be swapped:     -125.6532      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      177.7254      499  499\n",
      "iupac_m_487_> NH1/2 swapped:     -173.4628      509  509\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.0409      529  529\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.7976      532  532\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -167.8449      537  537\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.7154      539  539\n",
      "iupac_m_486_> OE1/2 will be swapped:      152.8533      540  540\n",
      "iupac_m_487_> NH1/2 swapped:     -174.9438      541  541\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.9021      549  549\n",
      "iupac_m_487_> NH1/2 swapped:      177.9270      555  555\n",
      "iupac_m_486_> OE1/2 will be swapped:     -135.8387      566  566\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2117      573  573\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.9851      581  581\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.9931      583  583\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.4917      584  584\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.4011      586  586\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   978402\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -41755.1875\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -41755.187500\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    31.629000\n",
      "Sequence length          :          590\n",
      "Compactness              :     0.190735\n",
      "Native energy (pair)     :   809.247223\n",
      "Native energy (surface)  :    92.815131\n",
      "Native energy (combined) :    22.925656\n",
      "Z score (pair)           :    -3.298540\n",
      "Z score (surface)        :    -1.047986\n",
      "Z score (combined)       :    -2.706335\n",
      "GA341 score              :     0.320415\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    12047\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        6572.1514\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4711       0      6   0.007   0.007      71.741       1.000\n",
      " 2 Bond angle potential               :    6336       4     48   2.404   2.404      701.89       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2751       0    145  51.941  51.941      1215.9       1.000\n",
      " 4 Stereochemical improper torsion pot:    1896       1      7   1.908   1.908      135.60       1.000\n",
      " 5 Soft-sphere overlap restraints     :   12047       0      4   0.007   0.007      77.272       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6251      11    114   1.204   1.204      1066.9       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6377      13    120   1.382   1.382      1214.4       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     589       5     22   5.816   5.816      234.07       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     499       0     14  85.783  85.783      224.47       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     371       0      9  74.258  74.258      187.05       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     179       0      4  97.552  97.552      154.33       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      77       0      1  95.723  95.723      53.228       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2718       0      4   0.717   0.717      111.92       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     588     259    143  40.993 115.665      914.09       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     929       1     22   2.918   2.918      209.22       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00200.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00200.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  102791.6016\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8887 397V 397V N   CA   3077 3078  123.62  107.00   16.62    4.78  107.00   16.62    4.78\n",
      "    2   9641 467F 467F N   CA   3633 3634  127.38  107.00   20.38    5.86  107.00   20.38    5.86\n",
      "    3  10425 538R 538R N   CA   4194 4195  124.26  107.00   17.26    4.96  107.00   17.26    4.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21690 141I 150D CA  CA   1072 1152    5.65    4.41    1.24    4.90    4.41    1.24    4.90\n",
      "    2  23609 359R 397V CA  CA   2761 3078   18.29   10.77    7.52    4.60   10.77    7.52    4.60\n",
      "    3  23631 359R 538R CA  CA   2761 4195   17.59    9.45    8.14    5.26    9.45    8.14    5.26\n",
      "    4  23642 360E 397V CA  CA   2772 3078   18.45    7.52   10.93    5.31    7.52   10.93    5.31\n",
      "    5  23643 360E 398A CA  CA   2772 3085   14.84    7.42    7.42    4.61    7.42    7.42    4.61\n",
      "    6  23667 361L 397V CA  CA   2781 3078   16.26    5.84   10.42    4.89    5.84   10.42    4.89\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  26997 142E 149R N   O    1079 1150    4.77    2.72    2.05    6.56    2.72    2.05    6.56\n",
      "    2  28976 342N 144Y N   O    2620 1106   10.89    6.52    4.37    5.71    6.52    4.37    5.71\n",
      "    3  29346 397V 359R N   O    3077 2770   19.57   10.54    9.03    5.18   10.54    9.03    5.18\n",
      "    4  29347 397V 360E N   O    3077 2779   19.38    7.05   12.33    5.70    7.05   12.33    5.70\n",
      "    5  29348 397V 361L N   O    3077 2787   18.79    7.78   11.01    4.62    7.78   11.01    4.62\n",
      "    6  29362 398A 359R N   O    3084 2770   16.78    9.28    7.50    6.16    9.28    7.50    6.16\n",
      "    7  29363 398A 360E N   O    3084 2779   16.23    6.19   10.04    6.11    6.19   10.04    6.11\n",
      "    8  29380 399F 359R N   O    3089 2770   14.93   10.55    4.38    4.66   10.55    4.38    4.66\n",
      "    9  29484 410L 146E N   O    3177 1123   10.22    6.64    3.58    5.34    6.64    3.58    5.34\n",
      "   10  30424 537D  87Y N   O    4186  669   14.28   10.32    3.96    5.46   10.32    3.96    5.46\n",
      "   11  30450 538R 359R N   O    4194 2770   16.66    9.13    7.53    4.58    9.13    7.53    4.58\n",
      "   12  30603 590A  83C N   O    4634  633   13.22   10.26    2.96    5.38   10.26    2.96    5.38\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16749 467F 467F CA  C    3634 3642 -153.49 -180.00   26.51    5.30 -180.00   26.51    5.30\n",
      "    2  16820 538R 538R CA  C    4195 4203 -157.18 -180.00   22.82    4.56 -180.00   22.82    4.56\n",
      "    3  16854 572I 572I CA  C    4482 4487 -138.36 -180.00   41.64    8.33 -180.00   41.64    8.33\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15696   2A   3E C   N      12   14  169.29 -117.80   86.77    2.56  -63.60 -173.97   31.39\n",
      "    1          3E   3E N   CA     14   15 -176.14  136.80                  -40.30\n",
      "    2  15697   3E   4L C   N      21   23   30.84  -70.70  101.59    9.13  -63.50 -157.16   33.99\n",
      "    2          4L   4L N   CA     23   24  138.37  141.60                  -41.20\n",
      "    3  15698   4L   5A C   N      29   31  -47.03  -68.20   28.06    2.77  -62.50  156.15   24.88\n",
      "    3          5A   5A N   CA     31   32  163.72  145.30                  -40.90\n",
      "    4  15700   6G   7T C   N      38   40 -119.59 -124.80    5.25    0.26  -63.20 -176.06   27.33\n",
      "    4          7T   7T N   CA     40   41  142.82  143.50                  -42.10\n",
      "    5  15710  16R  17M C   N     111  113  -79.98  -73.00   27.80    1.78  -63.40  150.50   23.79\n",
      "    5         17M  17M N   CA    113  114  169.91  143.00                  -40.50\n",
      "    6  15734  40I  41D C   N     296  298  -68.59  -96.50   56.37    2.33  -63.30  156.92   19.69\n",
      "    6         41D  41D N   CA    298  299  163.17  114.20                  -40.00\n",
      "    7  15735  41D  42A C   N     304  306   79.09  -62.50  144.47   26.41  -62.50  144.47   26.41\n",
      "    7         42A  42A N   CA    306  307  -69.61  -40.90                  -40.90\n",
      "    8  15737  43E  44G C   N     318  320   54.17   78.70   90.51    1.64  -62.40  122.54   20.05\n",
      "    8         44G  44G N   CA    320  321  -78.97 -166.10                  -41.20\n",
      "    9  15740  46R  47A C   N     337  339  -60.74  -62.50    2.98    0.45  -68.20  171.55   14.10\n",
      "    9         47A  47A N   CA    339  340  -43.31  -40.90                  145.30\n",
      "   10  15741  47A  48S C   N     342  344   92.86   56.90   95.53    9.17  -72.40  167.54   11.11\n",
      "   10         48S  48S N   CA    344  345  124.90   36.40                  152.40\n",
      "   11  15752  58T  59G C   N     438  440  -98.01  -80.20   23.19    0.84   82.20 -105.91    7.95\n",
      "   11         59G  59G N   CA    440  441 -171.05  174.10                    8.50\n",
      "   12  15795 101D 102L C   N     765  767  -68.40  -63.50    6.60    0.86 -108.50  173.96    8.93\n",
      "   12        102L 102L N   CA    767  768  -36.77  -41.20                  132.50\n",
      "   13  15797 103N 104S C   N     781  783   62.49  -72.40  134.89    9.67  -64.10 -147.02   15.21\n",
      "   13        104S 104S N   CA    783  784  153.72  152.40                  -35.00\n",
      "   14  15798 104S 105K C   N     787  789  -62.79  -62.90    6.73    0.87   56.60  147.21   18.60\n",
      "   14        105K 105K N   CA    789  790  -47.52  -40.80                   38.60\n",
      "   15  15799 105K 106E C   N     796  798  -88.25  -69.30   46.20    2.85  -63.60  137.30   19.70\n",
      "   15        106E 106E N   CA    798  799 -175.37  142.50                  -40.30\n",
      "   16  15800 106E 107G C   N     805  807  107.20   82.20   30.99    1.17  -62.40  172.48   31.89\n",
      "   16        107G 107G N   CA    807  808   -9.81    8.50                  -41.20\n",
      "   17  15813 119I 120E C   N     904  906  -62.37  -69.30   34.65    2.87  -63.60  143.25   19.03\n",
      "   17        120E 120E N   CA    906  907  176.45  142.50                  -40.30\n",
      "   18  15846 152L 153A C   N    1173 1175  -81.87  -62.50   36.12    6.95  -62.50   36.12    6.95\n",
      "   18        153A 153A N   CA   1175 1176  -71.39  -40.90                  -40.90\n",
      "   19  15847 153A 154P C   N    1178 1180 -115.81  -64.50   51.32    5.19  -58.70 -172.30   11.64\n",
      "   19        154P 154P N   CA   1180 1181  148.30  147.20                  -30.50\n",
      "   20  15849 155R 156A C   N    1196 1198  -62.55  -68.20   16.09    1.10  -62.50  171.14   28.04\n",
      "   20        156A 156A N   CA   1198 1199  130.24  145.30                  -40.90\n",
      "   21  15850 156A 157R C   N    1201 1203   57.40  -72.10  131.40   10.50 -125.20  178.95    6.58\n",
      "   21        157R 157R N   CA   1203 1204  164.13  141.90                  140.60\n",
      "   22  15853 159Q 160E C   N    1225 1227  -87.56  -69.30   51.77    3.30  -63.60  130.97   18.81\n",
      "   22        160E 160E N   CA   1227 1228 -169.06  142.50                  -40.30\n",
      "   23  15854 160E 161P C   N    1234 1236  -64.97  -58.70   27.75    1.84  -64.50  150.67   11.27\n",
      "   23        161P 161P N   CA   1236 1237   -3.47  -30.50                  147.20\n",
      "   24  15856 162E 163S C   N    1250 1252  -31.84 -136.60  112.02    3.57  -64.10  150.04   13.05\n",
      "   24        163S 163S N   CA   1252 1253  111.53  151.20                  -35.00\n",
      "   25  15857 163S 164L C   N    1256 1258  -66.50  -70.70   40.79    3.25   60.20 -161.68    9.07\n",
      "   25        164L 164L N   CA   1258 1259 -177.83  141.60                   29.60\n",
      "   26  15863 169D 170S C   N    1308 1310  -84.47 -136.60   53.91    1.81  -64.10  173.63   11.52\n",
      "   26        170S 170S N   CA   1310 1311  137.43  151.20                  -35.00\n",
      "   27  15864 170S 171Q C   N    1314 1316 -140.67 -121.10   45.08    1.74  -63.80  159.18   27.37\n",
      "   27        171Q 171Q N   CA   1316 1317 -179.69  139.70                  -40.30\n",
      "   28  15865 171Q 172H C   N    1323 1325  -71.32  -67.60   20.95    1.50  -63.20  157.29   20.18\n",
      "   28        172H 172H N   CA   1325 1326  160.62  140.00                  -42.30\n",
      "   29  15868 174V 175Y C   N    1351 1353 -103.44  -98.40    7.87    0.69  -63.50 -177.73   26.99\n",
      "   29        175Y 175Y N   CA   1353 1354  134.44  128.40                  -43.40\n",
      "   30  15870 176V 177K C   N    1370 1372 -146.92 -118.00   38.51    1.33  -62.90  176.02   27.88\n",
      "   30        177K 177K N   CA   1372 1373  164.53  139.10                  -40.80\n",
      "   31  15871 177K 178G C   N    1379 1381  138.17 -167.20   56.28    0.83   82.20 -171.86   11.81\n",
      "   31        178G 178G N   CA   1381 1382 -171.87  174.60                    8.50\n",
      "   32  15872 178G 179A C   N    1383 1385 -107.30 -134.00   27.56    0.69  -62.50 -175.54   32.60\n",
      "   32        179A 179A N   CA   1385 1386  140.16  147.00                  -40.90\n",
      "   33  15873 179A 180G C   N    1388 1390  160.27 -167.20   34.14    0.92   82.20  174.21   12.01\n",
      "   33        180G 180G N   CA   1390 1391  164.24  174.60                    8.50\n",
      "   34  15875 181V 182H C   N    1399 1401  174.64 -125.60   72.07    1.80  -63.20 -175.24   30.64\n",
      "   34        182H 182H N   CA   1401 1402  179.08  138.80                  -42.30\n",
      "   35  15876 182H 183S C   N    1409 1411  -74.81  -72.40    3.08    0.15  -64.10  171.00   13.06\n",
      "   35        183S 183S N   CA   1411 1412  154.33  152.40                  -35.00\n",
      "   36  15879 185L 186S C   N    1430 1432  -64.80  -72.40   51.62    2.78  -64.10  136.34    9.84\n",
      "   36        186S 186S N   CA   1432 1433  101.34  152.40                  -35.00\n",
      "   37  15881 187L 188E C   N    1444 1446  155.67 -117.80   92.72    2.76  -63.60 -154.62   34.66\n",
      "   37        188E 188E N   CA   1446 1447  170.12  136.80                  -40.30\n",
      "   38  15890 196K 197G C   N    1523 1525   63.76   78.70   15.34    0.84   82.20  179.03    9.61\n",
      "   38        197G 197G N   CA   1525 1526 -169.58 -166.10                    8.50\n",
      "   39  15891 197G 198N C   N    1527 1529 -109.63 -119.90   23.67    0.82  -63.20  163.51   18.28\n",
      "   39        198N 198N N   CA   1529 1530  115.68  137.00                  -41.10\n",
      "   40  15892 198N 199A C   N    1535 1537  -69.85  -68.20    9.06    0.79  -62.50  177.44   28.73\n",
      "   40        199A 199A N   CA   1537 1538  136.39  145.30                  -40.90\n",
      "   41  15893 199A 200N C   N    1540 1542 -149.85 -119.90   36.00    1.06  -63.20 -176.35   27.81\n",
      "   41        200N 200N N   CA   1542 1543  156.98  137.00                  -41.10\n",
      "   42  15894 200N 201R C   N    1548 1550 -108.82 -125.20   16.40    0.67  -63.00 -176.51   27.96\n",
      "   42        201R 201R N   CA   1550 1551  141.22  140.60                  -41.10\n",
      "   43  15895 201R 202Q C   N    1559 1561  -89.66  -73.00   55.15    3.94  -63.80  131.01   18.00\n",
      "   43        202Q 202Q N   CA   1561 1562   88.13  140.70                  -40.30\n",
      "   44  15896 202Q 203T C   N    1568 1570 -131.07 -124.80    7.19    0.43  -63.20 -169.58   28.90\n",
      "   44        203T 203T N   CA   1570 1571  139.99  143.50                  -42.10\n",
      "   45  15897 203T 204G C   N    1575 1577   61.69   78.70   22.72    0.57   82.20  160.86    8.83\n",
      "   45        204G 204G N   CA   1577 1578 -151.05 -166.10                    8.50\n",
      "   46  15898 204G 205E C   N    1579 1581 -127.50 -117.80   19.24    0.73  -63.60  178.14   27.36\n",
      "   46        205E 205E N   CA   1581 1582  153.41  136.80                  -40.30\n",
      "   47  15899 205E 206T C   N    1588 1590 -112.48 -124.80   17.32    1.04  -63.20  169.55   25.02\n",
      "   47        206T 206T N   CA   1590 1591  155.67  143.50                  -42.10\n",
      "   48  15900 206T 207K C   N    1595 1597 -105.97 -118.00   12.07    0.44  -62.90 -175.88   21.93\n",
      "   48        207K 207K N   CA   1597 1598  138.21  139.10                  -40.80\n",
      "   49  15901 207K 208M C   N    1604 1606 -114.78 -125.60   20.94    1.14  -63.40  169.06   29.34\n",
      "   49        208M 208M N   CA   1606 1607  158.44  140.50                  -40.50\n",
      "   50  15902 208M 209N C   N    1612 1614 -129.80 -119.90   60.29    2.50  -63.20  139.37   21.15\n",
      "   50        209N 209N N   CA   1614 1615 -163.52  137.00                  -41.10\n",
      "   51  15903 209N 210E C   N    1620 1622  -60.61  -69.30   14.40    1.32  -63.60  165.75   21.94\n",
      "   51        210E 210E N   CA   1622 1623  153.98  142.50                  -40.30\n",
      "   52  15905 211T 212S C   N    1636 1638  -82.23  -72.40   28.07    1.98  -64.10  162.12   10.80\n",
      "   52        212S 212S N   CA   1638 1639  126.11  152.40                  -35.00\n",
      "   53  15906 212S 213S C   N    1642 1644  164.37 -136.60   59.83    2.13  -64.10 -149.71   24.72\n",
      "   53        213S 213S N   CA   1644 1645  160.93  151.20                  -35.00\n",
      "   54  15907 213S 214R C   N    1648 1650   42.39   57.30   46.48    2.28  -63.00  162.07   28.63\n",
      "   54        214R 214R N   CA   1650 1651   82.02   38.00                  -41.10\n",
      "   55  15908 214R 215S C   N    1659 1661  -56.88  -72.40   30.60    1.40  -64.10  161.20   12.12\n",
      "   55        215S 215S N   CA   1661 1662  126.03  152.40                  -35.00\n",
      "   56  15909 215S 216H C   N    1665 1667  -99.62 -125.60   29.96    1.31  -63.20  167.96   23.37\n",
      "   56        216H 216H N   CA   1667 1668  153.74  138.80                  -42.30\n",
      "   57  15910 216H 217A C   N    1675 1677  -78.31  -68.20   39.16    2.80  -62.50  136.88   23.24\n",
      "   57        217A 217A N   CA   1677 1678 -176.86  145.30                  -40.90\n",
      "   58  15921 227Y 228G C   N    1763 1765  123.72   82.20   46.12    2.15  -62.40  176.38   30.17\n",
      "   58        228G 228G N   CA   1765 1766  -11.59    8.50                  -41.20\n",
      "   59  15923 229S 230L C   N    1773 1775  -81.11  -70.70   59.00    4.09  -63.50  120.42   17.78\n",
      "   59        230L 230L N   CA   1775 1776 -160.32  141.60                  -41.20\n",
      "   60  15925 231D 232M C   N    1789 1791 -108.74  -63.40   61.40    8.08  -63.40   61.40    8.08\n",
      "   60        232M 232M N   CA   1791 1792    0.89  -40.50                  -40.50\n",
      "   61  15926 232M 233R C   N    1797 1799   54.56   57.30    5.50    0.28  -63.00  144.41   26.28\n",
      "   61        233R 233R N   CA   1799 1800   42.77   38.00                  -41.10\n",
      "   62  15930 236E 237S C   N    1832 1834  108.14 -136.60  115.77    4.32  -64.10 -122.87   19.66\n",
      "   62        237S 237S N   CA   1834 1835  162.02  151.20                  -35.00\n",
      "   63  15933 239V 240L C   N    1852 1854  -63.90  -70.70   20.14    1.30  -63.50  163.84   22.73\n",
      "   63        240L 240L N   CA   1854 1855  122.64  141.60                  -41.20\n",
      "   64  15934 240L 241L C   N    1860 1862   62.64  -70.70  133.35   12.07  -63.50 -141.07   27.06\n",
      "   64        241L 241L N   CA   1862 1863  139.87  141.60                  -41.20\n",
      "   65  15947 253T 254E C   N    1957 1959  -53.13  -69.30   24.33    2.24  -63.60  159.37   20.69\n",
      "   65        254E 254E N   CA   1959 1960  160.67  142.50                  -40.30\n",
      "   66  15948 254E 255S C   N    1966 1968  -75.04  -72.40    2.64    0.19  -64.10  173.16   13.23\n",
      "   66        255S 255S N   CA   1968 1969  152.19  152.40                  -35.00\n",
      "   67  15949 255S 256T C   N    1972 1974 -177.12 -124.80   61.05    2.30  -63.20 -177.21   30.43\n",
      "   67        256T 256T N   CA   1974 1975  174.96  143.50                  -42.10\n",
      "   68  15950 256T 257G C   N    1979 1981  169.49 -167.20   23.60    0.54   82.20 -175.63   12.88\n",
      "   68        257G 257G N   CA   1981 1982  170.89  174.60                    8.50\n",
      "   69  15952 258V 259A C   N    1990 1992 -146.92 -134.00   25.28    1.00  -62.50  172.45   32.77\n",
      "   69        259A 259A N   CA   1992 1993  168.73  147.00                  -40.90\n",
      "   70  15953 259A 260F C   N    1995 1997 -123.76 -124.20   22.83    1.09  -63.20  161.36   26.38\n",
      "   70        260F 260F N   CA   1997 1998  166.13  143.30                  -44.30\n",
      "   71  15954 260F 261E C   N    2006 2008 -172.64 -117.80   61.53    1.80  -63.60 -170.49   31.17\n",
      "   71        261E 261E N   CA   2008 2009  164.71  136.80                  -40.30\n",
      "   72  15955 261E 262E C   N    2015 2017 -111.66 -117.80   12.53    0.69  -63.60  178.57   26.53\n",
      "   72        262E 262E N   CA   2017 2018  147.72  136.80                  -40.30\n",
      "   73  15956 262E 263A C   N    2024 2026  -64.70  -68.20   14.55    1.05  -62.50  172.09   28.09\n",
      "   73        263A 263A N   CA   2026 2027  131.18  145.30                  -40.90\n",
      "   74  15957 263A 264K C   N    2029 2031  173.68 -118.00   68.44    2.51  -62.90 -145.09   35.20\n",
      "   74        264K 264K N   CA   2031 2032  143.26  139.10                  -40.80\n",
      "   75  15958 264K 265K C   N    2038 2040 -117.83 -118.00   25.00    1.21  -62.90  164.35   19.11\n",
      "   75        265K 265K N   CA   2040 2041  114.10  139.10                  -40.80\n",
      "   76  15960 266I 267N C   N    2055 2057  -81.18  -71.20   46.33    3.25  -63.20  139.81   16.55\n",
      "   76        267N 267N N   CA   2057 2058   97.55  142.80                  -41.10\n",
      "   77  15961 267N 268Q C   N    2063 2065 -139.01 -121.10   20.07    0.62  -63.80 -173.24   31.41\n",
      "   77        268Q 268Q N   CA   2065 2066  148.75  139.70                  -40.30\n",
      "   78  15962 268Q 269S C   N    2072 2074 -135.71 -136.60    9.01    0.47  -64.10  179.72   18.12\n",
      "   78        269S 269S N   CA   2074 2075  160.16  151.20                  -35.00\n",
      "   79  15963 269S 270L C   N    2078 2080  -84.30  -70.70   30.35    1.90  -63.50  151.50   22.28\n",
      "   79        270L 270L N   CA   2080 2081  168.73  141.60                  -41.20\n",
      "   80  15964 270L 271L C   N    2086 2088  -64.72  -70.70    6.01    0.52  -63.50  177.89   24.77\n",
      "   80        271L 271L N   CA   2088 2089  140.91  141.60                  -41.20\n",
      "   81  15967 273L 274G C   N    2110 2112 -103.23  -80.20   28.09    2.06   82.20 -130.15   17.83\n",
      "   81        274G 274G N   CA   2112 2113  158.02  174.10                    8.50\n",
      "   82  15968 274G 275R C   N    2114 2116  -81.82  -72.10   21.46    1.45  -63.00  158.99   22.77\n",
      "   82        275R 275R N   CA   2116 2117  161.03  141.90                  -41.10\n",
      "   83  15991 297L 298T C   N    2290 2292  -62.35  -63.20   17.56    2.20 -124.80  168.84    9.01\n",
      "   83        298T 298T N   CA   2292 2293  -59.64  -42.10                  143.50\n",
      "   84  15992 298T 299R C   N    2297 2299   36.84   57.30   29.53    1.87  -63.00  141.58   25.34\n",
      "   84        299R 299R N   CA   2299 2300   59.28   38.00                  -41.10\n",
      "   85  15993 299R 300L C   N    2308 2310  -90.78 -108.50   19.28    0.85  -63.50  168.34   21.96\n",
      "   85        300L 300L N   CA   2310 2311  124.91  132.50                  -41.20\n",
      "   86  15994 300L 301L C   N    2316 2318 -122.01 -108.50   16.15    0.88  -63.50  174.92   21.74\n",
      "   86        301L 301L N   CA   2318 2319  123.65  132.50                  -41.20\n",
      "   87  15995 301L 302S C   N    2324 2326 -133.42 -136.60    6.75    0.38  -64.10 -178.40   18.08\n",
      "   87        302S 302S N   CA   2326 2327  157.15  151.20                  -35.00\n",
      "   88  15996 302S 303E C   N    2330 2332  -97.57 -117.80   29.75    0.98  -63.60  158.96   19.53\n",
      "   88        303E 303E N   CA   2332 2333  114.98  136.80                  -40.30\n",
      "   89  15997 303E 304S C   N    2339 2341 -156.04 -136.60   29.14    0.97  -64.10  177.72   19.51\n",
      "   89        304S 304S N   CA   2341 2342  172.90  151.20                  -35.00\n",
      "   90  15998 304S 305F C   N    2345 2347 -104.89 -124.20   20.77    0.60  -63.20 -175.27   23.01\n",
      "   90        305F 305F N   CA   2347 2348  135.67  143.30                  -44.30\n",
      "   91  15999 305F 306G C   N    2356 2358   65.95   78.70   26.50    0.40   82.20  152.24    8.20\n",
      "   91        306G 306G N   CA   2358 2359 -142.87 -166.10                    8.50\n",
      "   92  16000 306G 307G C   N    2360 2362  103.62   78.70   63.08    0.96   82.20  129.24    7.33\n",
      "   92        307G 307G N   CA   2362 2363  135.95 -166.10                    8.50\n",
      "   93  16001 307G 308N C   N    2364 2366 -122.17 -119.90   12.87    0.53  -63.20  179.22   25.76\n",
      "   93        308N 308N N   CA   2366 2367  149.67  137.00                  -41.10\n",
      "   94  16002 308N 309S C   N    2372 2374 -106.23 -136.60   31.18    1.07  -64.10 -176.00   11.52\n",
      "   94        309S 309S N   CA   2374 2375  144.11  151.20                  -35.00\n",
      "   95  16003 309S 310K C   N    2378 2380  -71.74  -70.20    5.80    0.38  -62.90  173.43   23.02\n",
      "   95        310K 310K N   CA   2380 2381  145.99  140.40                  -40.80\n",
      "   96  16006 312W 313M C   N    2408 2410 -164.37 -125.60   38.84    1.48  -63.40 -156.54   38.20\n",
      "   96        313M 313M N   CA   2410 2411  142.86  140.50                  -40.50\n",
      "   97  16007 313M 314L C   N    2416 2418   60.05  -70.70  131.99   12.53  -63.50 -158.57   25.10\n",
      "   97        314L 314L N   CA   2418 2419  159.71  141.60                  -41.20\n",
      "   98  16008 314L 315A C   N    2424 2426  -75.94  -68.20   13.45    0.85  -62.50  163.36   27.45\n",
      "   98        315A 315A N   CA   2426 2427  156.30  145.30                  -40.90\n",
      "   99  16009 315A 316T C   N    2429 2431  -75.30  -78.10   39.83    1.61  -63.20  152.65   18.80\n",
      "   99        316T 316T N   CA   2431 2432  110.07  149.80                  -42.10\n",
      "  100  16012 318S 319T C   N    2449 2451  -80.38  -78.10   35.20    1.43  -63.20  134.08   18.34\n",
      "  100        319T 319T N   CA   2451 2452 -175.08  149.80                  -42.10\n",
      "  101  16015 321A 322N C   N    2466 2468 -164.96 -119.90   51.61    2.68   55.90  156.82   19.36\n",
      "  101        322N 322N N   CA   2468 2469  111.84  137.00                   39.50\n",
      "  102  16016 322N 323N C   N    2474 2476 -161.76 -119.90   41.88    1.71  -71.20   90.67    6.68\n",
      "  102        323N 323N N   CA   2476 2477  138.25  137.00                  142.80\n",
      "  103  16035 341T 342N C   N    2618 2620 -105.68 -119.90   20.21    0.59  -63.20  169.15   19.12\n",
      "  103        342N 342N N   CA   2620 2621  122.63  137.00                  -41.10\n",
      "  104  16038 344A 345K C   N    2640 2642 -122.86 -118.00   30.62    1.41  -62.90  161.42   24.55\n",
      "  104        345K 345K N   CA   2642 2643  169.33  139.10                  -40.80\n",
      "  105  16043 349T 350K C   N    2679 2681   63.13   56.60   77.97    6.07  -62.90 -158.60   33.51\n",
      "  105        350K 350K N   CA   2681 2682  116.30   38.60                  -40.80\n",
      "  106  16044 350K 351K C   N    2688 2690  -83.74  -70.20   15.41    1.27  -62.90  175.08   21.68\n",
      "  106        351K 351K N   CA   2690 2691  133.04  140.40                  -40.80\n",
      "  107  16045 351K 352N C   N    2697 2699  172.69 -119.90   80.31    2.37  -63.20 -174.21   29.77\n",
      "  107        352N 352N N   CA   2699 2700 -179.35  137.00                  -41.10\n",
      "  108  16047 353I 354E C   N    2713 2715   48.52  -69.30  133.43    7.94  -63.60  164.35   27.72\n",
      "  108        354E 354E N   CA   2715 2716   79.87  142.50                  -40.30\n",
      "  109  16048 354E 355L C   N    2722 2724  -79.64  -63.50   39.88    6.54  -63.50   39.88    6.54\n",
      "  109        355L 355L N   CA   2724 2725  -77.67  -41.20                  -41.20\n",
      "  110  16049 355L 356K C   N    2730 2732  -92.26  -62.90   38.56    6.64  -62.90   38.56    6.64\n",
      "  110        356K 356K N   CA   2732 2733  -65.80  -40.80                  -40.80\n",
      "  111  16056 362V 363A C   N    2793 2795 -154.58 -134.00   35.10    1.27  -68.20   91.49    6.68\n",
      "  111        363A 363A N   CA   2795 2796  175.43  147.00                  145.30\n",
      "  112  16057 363A 364S C   N    2798 2800  -82.12  -72.40   12.89    1.03  -64.10  179.85   12.07\n",
      "  112        364S 364S N   CA   2800 2801  143.94  152.40                  -35.00\n",
      "  113  16058 364S 365L C   N    2804 2806 -136.81 -108.50   37.14    2.07  -63.50  166.66   20.43\n",
      "  113        365L 365L N   CA   2806 2807  108.46  132.50                  -41.20\n",
      "  114  16059 365L 366E C   N    2812 2814   59.35  -69.30  129.29   10.24  -63.60 -154.79   23.83\n",
      "  114        366E 366E N   CA   2814 2815  155.39  142.50                  -40.30\n",
      "  115  16060 366E 367G C   N    2821 2823   71.66   78.70   21.85    0.36   82.20  154.28    8.02\n",
      "  115        367G 367G N   CA   2823 2824 -145.41 -166.10                    8.50\n",
      "  116  16061 367G 368R C   N    2825 2827  -70.61  -72.10   16.16    1.28  -63.00  161.09   22.28\n",
      "  116        368R 368R N   CA   2827 2828  157.99  141.90                  -41.10\n",
      "  117  16062 368R 369L C   N    2836 2838  -69.46  -70.70    5.56    0.38  -63.50  177.48   24.31\n",
      "  117        369L 369L N   CA   2838 2839  136.18  141.60                  -41.20\n",
      "  118  16063 369L 370D C   N    2844 2846 -103.14  -70.90   44.62    1.84  -63.30  144.47   20.40\n",
      "  118        370D 370D N   CA   2846 2847 -178.86  150.30                  -40.00\n",
      "  119  16065 371V 372L C   N    2859 2861  -91.57 -108.50   84.01    4.34  -70.70   93.73    7.91\n",
      "  119        372L 372L N   CA   2861 2862   50.22  132.50                  141.60\n",
      "  120  16067 373A 374L C   N    2872 2874 -165.64 -108.50   67.25    2.97  -70.70   98.54    7.99\n",
      "  120        374L 374L N   CA   2874 2875  167.95  132.50                  141.60\n",
      "  121  16068 374L 375E C   N    2880 2882  -76.71  -69.30   16.59    1.01  -63.60  162.88   22.45\n",
      "  121        375E 375E N   CA   2882 2883  157.34  142.50                  -40.30\n",
      "  122  16069 375E 376K C   N    2889 2891 -120.41 -118.00   24.04    1.13  -62.90  166.43   25.05\n",
      "  122        376K 376K N   CA   2891 2892  163.02  139.10                  -40.80\n",
      "  123  16070 376K 377Q C   N    2898 2900 -109.81 -121.10   38.02    1.95  -63.80  150.88   24.63\n",
      "  123        377Q 377Q N   CA   2900 2901  176.01  139.70                  -40.30\n",
      "  124  16071 377Q 378R C   N    2907 2909 -102.00 -125.20   27.82    0.82  -63.00  170.85   20.74\n",
      "  124        378R 378R N   CA   2909 2910  125.24  140.60                  -41.10\n",
      "  125  16072 378R 379K C   N    2918 2920  165.93 -118.00   76.88    3.10  -62.90 -146.24   25.08\n",
      "  125        379K 379K N   CA   2920 2921  127.98  139.10                  -40.80\n",
      "  126  16073 379K 380Q C   N    2927 2929 -167.61 -121.10  122.73    4.90  -63.80  123.25   22.38\n",
      "  126        380Q 380Q N   CA   2929 2930 -106.73  139.70                  -40.30\n",
      "  127  16074 380Q 381E C   N    2936 2938 -160.92 -117.80   62.28    2.02  -63.60  168.83   27.77\n",
      "  127        381E 381E N   CA   2938 2939 -178.26  136.80                  -40.30\n",
      "  128  16075 381E 382E C   N    2945 2947 -178.48 -117.80   60.68    2.21  -63.60 -148.43   24.44\n",
      "  128        382E 382E N   CA   2947 2948  137.36  136.80                  -40.30\n",
      "  129  16077 383I 384G C   N    2962 2964 -111.49  -80.20   31.51    1.98   82.20 -122.68   18.17\n",
      "  129        384G 384G N   CA   2964 2965  177.81  174.10                    8.50\n",
      "  130  16078 384G 385R C   N    2966 2968  -71.57  -72.10   17.12    1.30  -63.00  166.11   21.88\n",
      "  130        385R 385R N   CA   2968 2969  124.79  141.90                  -41.10\n",
      "  131  16079 385R 386L C   N    2977 2979 -161.00 -108.50   53.53    2.44  -63.50 -158.97   33.92\n",
      "  131        386L 386L N   CA   2979 2980  142.99  132.50                  -41.20\n",
      "  132  16080 386L 387T C   N    2985 2987 -127.59 -124.80    5.51    0.17  -63.20 -178.54   27.52\n",
      "  132        387T 387T N   CA   2987 2988  148.25  143.50                  -42.10\n",
      "  133  16081 387T 388Q C   N    2992 2994 -153.97 -121.10   32.89    1.21  -63.80 -159.70   34.14\n",
      "  133        388Q 388Q N   CA   2994 2995  140.84  139.70                  -40.30\n",
      "  134  16083 389E 390R C   N    3010 3012  -63.21  -72.10    9.26    0.65  -63.00  179.59   24.27\n",
      "  134        390R 390R N   CA   3012 3013  139.31  141.90                  -41.10\n",
      "  135  16084 390R 391D C   N    3021 3023  -93.99  -96.50    5.22    0.22  -63.30  152.74   17.22\n",
      "  135        391D 391D N   CA   3023 3024  109.63  114.20                  -40.00\n",
      "  136  16085 391D 392K C   N    3029 3031 -125.56 -118.00   67.15    3.15  -62.90  129.54   20.56\n",
      "  136        392K 392K N   CA   3031 3032 -154.17  139.10                  -40.80\n",
      "  137  16086 392K 393L C   N    3038 3040  -61.93  -70.70   14.45    1.41  -63.50  165.73   22.92\n",
      "  137        393L 393L N   CA   3040 3041  153.08  141.60                  -41.20\n",
      "  138  16087 393L 394R C   N    3046 3048 -154.79  -63.00   94.12   16.63  -63.00   94.12   16.63\n",
      "  138        394R 394R N   CA   3048 3049  -61.89  -41.10                  -41.10\n",
      "  139  16088 394R 395Q C   N    3057 3059 -147.70 -121.10   27.24    1.15  -63.80 -166.71   25.03\n",
      "  139        395Q 395Q N   CA   3059 3060  133.83  139.70                  -40.30\n",
      "  140  16089 395Q 396E C   N    3066 3068 -160.95 -117.80   45.00    1.38  -63.60 -164.01   31.51\n",
      "  140        396E 396E N   CA   3068 3069  149.59  136.80                  -40.30\n",
      "  141  16091 397V 398A C   N    3082 3084 -168.54 -134.00   39.11    2.00  -68.20  101.71    9.15\n",
      "  141        398A 398A N   CA   3084 3085  128.66  147.00                  145.30\n",
      "  142  16094 400A 401D C   N    3103 3105  -63.49  -63.30   66.99    8.28  -63.30   66.99    8.28\n",
      "  142        401D 401D N   CA   3105 3106 -106.99  -40.00                  -40.00\n",
      "  143  16095 401D 402S C   N    3111 3113  -67.19  -64.10   35.44    2.76 -136.60  154.92    8.77\n",
      "  143        402S 402S N   CA   3113 3114  -70.31  -35.00                  151.20\n",
      "  144  16097 403V 404H C   N    3124 3126   52.33  -67.60  129.40   11.04  -63.20  173.26   19.11\n",
      "  144        404H 404H N   CA   3126 3127 -171.42  140.00                  -42.30\n",
      "  145  16098 404H 405D C   N    3134 3136 -150.19  -96.50   63.48    2.64  -63.30 -167.35   29.38\n",
      "  145        405D 405D N   CA   3136 3137  148.06  114.20                  -40.00\n",
      "  146  16099 405D 406A C   N    3142 3144  145.31 -134.00   99.27    2.49  -62.50 -169.68   38.54\n",
      "  146        406A 406A N   CA   3144 3145 -155.18  147.00                  -40.90\n",
      "  147  16100 406A 407R C   N    3147 3149 -137.14 -125.20   13.63    0.70  -63.00 -169.84   21.86\n",
      "  147        407R 407R N   CA   3149 3150  134.02  140.60                  -41.10\n",
      "  148  16101 407R 408D C   N    3158 3160   57.99   54.50    7.18    0.80  -63.30  149.36   25.37\n",
      "  148        408D 408D N   CA   3160 3161   47.17   40.90                  -40.00\n",
      "  149  16102 408D 409K C   N    3166 3168  -73.17  -70.20   47.06    3.34  -62.90  132.24   17.75\n",
      "  149        409K 409K N   CA   3168 3169 -172.64  140.40                  -40.80\n",
      "  150  16104 410L 411E C   N    3183 3185  -92.57  -63.60   40.15    6.82  -63.60   40.15    6.82\n",
      "  150        411E 411E N   CA   3185 3186  -68.10  -40.30                  -40.30\n",
      "  151  16108 414L 415N C   N    3213 3215  138.09 -119.90  102.35    4.45  -63.20 -127.50   25.72\n",
      "  151        415N 415N N   CA   3215 3216  128.80  137.00                  -41.10\n",
      "  152  16109 415N 416N C   N    3221 3223 -161.38 -119.90   43.05    1.50  -63.20 -163.35   30.06\n",
      "  152        416N 416N N   CA   3223 3224  148.52  137.00                  -41.10\n",
      "  153  16111 417I 418R C   N    3237 3239  140.12 -125.20   96.91    3.37  -63.00 -137.60   39.80\n",
      "  153        418R 418R N   CA   3239 3240  161.25  140.60                  -41.10\n",
      "  154  16112 418R 419L C   N    3248 3250 -119.23 -108.50   33.17    1.66  -63.50  164.63   26.31\n",
      "  154        419L 419L N   CA   3250 3251  163.89  132.50                  -41.20\n",
      "  155  16113 419L 420S C   N    3256 3258  -74.30  -72.40    6.78    0.34  -64.10  166.40   12.69\n",
      "  155        420S 420S N   CA   3258 3259  158.91  152.40                  -35.00\n",
      "  156  16114 420S 421N C   N    3262 3264  -84.42  -71.20   20.06    1.07  -63.20  162.40   21.50\n",
      "  156        421N 421N N   CA   3264 3265  157.90  142.80                  -41.10\n",
      "  157  16116 422I 423A C   N    3278 3280  -75.59  -68.20   12.77    1.24  -62.50  176.28   28.27\n",
      "  157        423A 423A N   CA   3280 3281  134.89  145.30                  -40.90\n",
      "  158  16117 423A 424L C   N    3283 3285 -142.72 -108.50   44.37    1.97  -63.50  176.79   29.44\n",
      "  158        424L 424L N   CA   3285 3286  160.75  132.50                  -41.20\n",
      "  159  16118 424L 425R C   N    3291 3293 -104.20 -125.20   33.43    1.07  -63.00  161.05   19.33\n",
      "  159        425R 425R N   CA   3293 3294  114.59  140.60                  -41.10\n",
      "  160  16119 425R 426R C   N    3302 3304 -163.16 -125.20   43.43    1.30  -63.00 -173.62   31.85\n",
      "  160        426R 426R N   CA   3304 3305  161.71  140.60                  -41.10\n",
      "  161  16120 426R 427R C   N    3313 3315  -91.52  -72.10   34.92    2.30  -63.00  150.71   22.33\n",
      "  161        427R 427R N   CA   3315 3316  170.92  141.90                  -41.10\n",
      "  162  16123 429E 430A C   N    3340 3342   59.56  -68.20  127.86   11.01  -62.50 -151.83   31.38\n",
      "  162        430A 430A N   CA   3342 3343  150.48  145.30                  -40.90\n",
      "  163  16124 430A 431A C   N    3345 3347   89.35 -134.00  142.06    3.47  -62.50 -157.96   31.70\n",
      "  163        431A 431A N   CA   3347 3348 -174.18  147.00                  -40.90\n",
      "  164  16125 431A 432S C   N    3350 3352  -68.18  -72.40    6.91    0.54  -64.10  167.17   12.36\n",
      "  164        432S 432S N   CA   3352 3353  157.88  152.40                  -35.00\n",
      "  165  16126 432S 433E C   N    3356 3358 -135.77  -63.60   77.58   10.76  -63.60   77.58   10.76\n",
      "  165        433E 433E N   CA   3358 3359  -11.84  -40.30                  -40.30\n",
      "  166  16127 433E 434G C   N    3365 3367  150.42 -167.20   68.78    2.69   82.20  131.07    9.38\n",
      "  166        434G 434G N   CA   3367 3368  120.43  174.60                    8.50\n",
      "  167  16128 434G 435F C   N    3369 3371 -111.29 -124.20   78.94    3.38  -63.20  119.80   13.92\n",
      "  167        435F 435F N   CA   3371 3372   65.42  143.30                  -44.30\n",
      "  168  16129 435F 436I C   N    3380 3382  -68.93  -63.40    6.06    0.99 -120.60  179.03    8.33\n",
      "  168        436I 436I N   CA   3382 3383  -41.11  -43.60                  130.30\n",
      "  169  16130 436I 437H C   N    3388 3390  -72.93  -67.60   23.33    1.94  -63.20  159.88   19.40\n",
      "  169        437H 437H N   CA   3390 3391  117.28  140.00                  -42.30\n",
      "  170  16131 437H 438S C   N    3398 3400   93.28 -136.60  130.13    5.13  -64.10 -126.61   18.18\n",
      "  170        438S 438S N   CA   3400 3401  152.65  151.20                  -35.00\n",
      "  171  16132 438S 439L C   N    3404 3406 -121.81 -108.50   19.69    0.89  -63.50 -178.58   28.79\n",
      "  171        439L 439L N   CA   3406 3407  147.01  132.50                  -41.20\n",
      "  172  16133 439L 440D C   N    3412 3414  -94.64  -96.50   20.12    0.84  -63.30  177.03   20.15\n",
      "  172        440D 440D N   CA   3414 3415  134.24  114.20                  -40.00\n",
      "  173  16134 440D 441N C   N    3420 3422 -147.72 -119.90   29.17    0.99  -63.20 -167.34   28.85\n",
      "  173        441N 441N N   CA   3422 3423  145.77  137.00                  -41.10\n",
      "  174  16135 441N 442K C   N    3428 3430 -131.14 -118.00   30.84    1.27  -62.90  166.80   25.75\n",
      "  174        442K 442K N   CA   3430 3431  167.00  139.10                  -40.80\n",
      "  175  16136 442K 443S C   N    3437 3439 -147.39 -136.60   35.94    1.56  -64.10  162.48   17.78\n",
      "  175        443S 443S N   CA   3439 3440 -174.52  151.20                  -35.00\n",
      "  176  16137 443S 444C C   N    3443 3445  -58.43  -63.00    8.09    0.86 -117.90 -178.83    7.85\n",
      "  176        444C 444C N   CA   3445 3446  -47.76  -41.10                  141.10\n",
      "  177  16138 444C 445F C   N    3449 3451  -84.55  -71.40   19.00    1.11  -63.20  162.70   24.03\n",
      "  177        445F 445F N   CA   3451 3452  154.41  140.70                  -44.30\n",
      "  178  16139 445F 446L C   N    3460 3462 -123.71 -108.50   31.17    1.49  -63.50  170.10   27.35\n",
      "  178        446L 446L N   CA   3462 3463  159.71  132.50                  -41.20\n",
      "  179  16140 446L 447F C   N    3468 3470  -90.94  -71.40   28.28    2.49  -63.20  166.88   21.38\n",
      "  179        447F 447F N   CA   3470 3471  120.26  140.70                  -44.30\n",
      "  180  16142 448F 449K C   N    3490 3492  -50.15  -70.20   20.05    1.47  -62.90  178.98   22.57\n",
      "  180        449K 449K N   CA   3492 3493  140.68  140.40                  -40.80\n",
      "  181  16143 449K 450G C   N    3499 3501 -177.27 -167.20   57.07    2.32   82.20  148.95   11.32\n",
      "  181        450G 450G N   CA   3501 3502  118.42  174.60                    8.50\n",
      "  182  16144 450G 451R C   N    3503 3505 -142.62 -125.20   23.99    0.72  -63.00 -179.66   29.78\n",
      "  182        451R 451R N   CA   3505 3506  157.09  140.60                  -41.10\n",
      "  183  16146 452C 453S C   N    3520 3522 -123.74 -136.60   13.44    0.64  -64.10 -179.95   17.23\n",
      "  183        453S 453S N   CA   3522 3523  155.11  151.20                  -35.00\n",
      "  184  16148 454I 455T C   N    3534 3536 -136.96 -124.80   15.34    0.53  -63.20 -179.21   28.02\n",
      "  184        455T 455T N   CA   3536 3537  152.84  143.50                  -42.10\n",
      "  185  16149 455T 456L C   N    3541 3543 -137.05 -108.50   30.96    1.37  -63.50 -170.80   30.83\n",
      "  185        456L 456L N   CA   3543 3544  144.48  132.50                  -41.20\n",
      "  186  16151 457E 458S C   N    3558 3560 -116.20 -136.60   34.58    1.22  -64.10  166.63   10.26\n",
      "  186        458S 458S N   CA   3560 3561  123.27  151.20                  -35.00\n",
      "  187  16153 459V 460L C   N    3571 3573   36.59  -70.70  123.47   12.44  -63.50  153.27   19.32\n",
      "  187        460L 460L N   CA   3573 3574 -157.29  141.60                  -41.20\n",
      "  188  16154 460L 461R C   N    3579 3581  -78.00  -63.00   86.93   10.81  -63.00   86.93   10.81\n",
      "  188        461R 461R N   CA   3581 3582   44.53  -41.10                  -41.10\n",
      "  189  16155 461R 462G C   N    3590 3592  -92.46 -167.20   74.97    1.58   78.70  171.68    8.96\n",
      "  189        462G 462G N   CA   3592 3593 -179.44  174.60                 -166.10\n",
      "  190  16177 483L 484H C   N    3759 3761  108.03   56.30  105.31   14.13  -63.20 -115.29   40.99\n",
      "  190        484H 484H N   CA   3761 3762  132.52   40.80                  -42.30\n",
      "  191  16179 485I 486Q C   N    3777 3779  -83.02  -73.00   11.97    0.72  -63.80  173.53   26.54\n",
      "  191        486Q 486Q N   CA   3779 3780  147.24  140.70                  -40.30\n",
      "  192  16180 486Q 487L C   N    3786 3788  -92.63 -108.50   51.72    2.60  -63.50  127.84   16.32\n",
      "  192        487L 487L N   CA   3788 3789   83.28  132.50                  -41.20\n",
      "  193  16181 487L 488F C   N    3794 3796 -167.06 -124.20   44.10    1.42  -63.20 -167.54   33.18\n",
      "  193        488F 488F N   CA   3796 3797  153.67  143.30                  -44.30\n",
      "  194  16184 490C 491E C   N    3818 3820 -100.09 -117.80   21.18    0.62  -63.60  169.46   20.80\n",
      "  194        491E 491E N   CA   3820 3821  125.18  136.80                  -40.30\n",
      "  195  16185 491E 492H C   N    3827 3829 -152.68 -125.60   31.43    0.80  -63.20 -174.10   29.03\n",
      "  195        492H 492H N   CA   3829 3830  154.75  138.80                  -42.30\n",
      "  196  16186 492H 493H C   N    3837 3839  -74.04  -67.60    7.68    0.68  -63.20  178.45   21.66\n",
      "  196        493H 493H N   CA   3839 3840  135.82  140.00                  -42.30\n",
      "  197  16187 493H 494A C   N    3847 3849 -108.58 -134.00   25.76    0.71  -62.50 -177.80   32.30\n",
      "  197        494A 494A N   CA   3849 3850  142.83  147.00                  -40.90\n",
      "  198  16188 494A 495N C   N    3852 3854 -105.02 -119.90   18.27    0.53  -63.20  172.63   19.57\n",
      "  198        495N 495N N   CA   3854 3855  126.39  137.00                  -41.10\n",
      "  199  16191 497R 498N C   N    3880 3882  -66.96  -71.20   17.79    1.03  -63.20  166.66   20.63\n",
      "  199        498N 498N N   CA   3882 3883  125.52  142.80                  -41.10\n",
      "  200  16194 500P 501M C   N    3903 3905  -79.37  -73.00   25.56    1.63  -63.40  152.58   24.05\n",
      "  200        501M 501M N   CA   3905 3906  167.75  143.00                  -40.50\n",
      "  201  16195 501M 502Q C   N    3911 3913  -75.95  -73.00   48.53    3.23  -63.80  133.12   18.95\n",
      "  201        502Q 502Q N   CA   3913 3914   92.26  140.70                  -40.30\n",
      "  202  16196 502Q 503F C   N    3920 3922 -136.84 -124.20   14.32    0.74  -63.20 -166.33   31.71\n",
      "  202        503F 503F N   CA   3922 3923  136.57  143.30                  -44.30\n",
      "  203  16198 504I 505G C   N    3939 3941 -121.44  -80.20   66.66    4.51   82.20 -166.95   15.07\n",
      "  203        505G 505G N   CA   3941 3942  121.73  174.10                    8.50\n",
      "  204  16205 511C 512L C   N    3994 3996   24.92  -70.70   96.03    8.99  -63.50 -169.80   23.30\n",
      "  204        512L 512L N   CA   3996 3997  150.41  141.60                  -41.20\n",
      "  205  16206 512L 513H C   N    4002 4004  175.20 -125.60   59.20    2.08  -63.20 -143.12   34.86\n",
      "  205        513H 513H N   CA   4004 4005  138.11  138.80                  -42.30\n",
      "  206  16209 515V 516G C   N    4026 4028 -136.24 -167.20   33.09    0.48   82.20 -150.53   15.92\n",
      "  206        516G 516G N   CA   4028 4029  162.91  174.60                    8.50\n",
      "  207  16210 516G 517A C   N    4030 4032  -73.19  -68.20   10.74    0.70  -62.50  164.64   27.52\n",
      "  207        517A 517A N   CA   4032 4033  154.81  145.30                  -40.90\n",
      "  208  16211 517A 518S C   N    4035 4037 -158.63 -136.60   31.78    1.04  -64.10  178.06   19.72\n",
      "  208        518S 518S N   CA   4037 4038  174.11  151.20                  -35.00\n",
      "  209  16212 518S 519G C   N    4041 4043   69.89   78.70   19.51    0.29   82.20  157.67    8.27\n",
      "  209        519G 519G N   CA   4043 4044 -148.69 -166.10                    8.50\n",
      "  210  16215 521P 522K C   N    4060 4062 -169.83 -118.00   61.34    2.92  -62.90 -178.14   21.19\n",
      "  210        522K 522K N   CA   4062 4063  106.30  139.10                  -40.80\n",
      "  211  16216 522K 523A C   N    4069 4071  174.73 -134.00   69.16    1.97  -62.50  175.69   35.04\n",
      "  211        523A 523A N   CA   4071 4072 -166.58  147.00                  -40.90\n",
      "  212  16217 523A 524F C   N    4074 4076  -61.46  -71.40   19.98    1.29  -63.20  167.69   23.40\n",
      "  212        524F 524F N   CA   4076 4077  123.38  140.70                  -44.30\n",
      "  213  16219 525V 526A C   N    4092 4094  -72.93  -68.20   42.34    3.58  -62.50  144.50   23.19\n",
      "  213        526A 526A N   CA   4094 4095  103.23  145.30                  -40.90\n",
      "  214  16221 527H 528T C   N    4107 4109  -87.12  -63.20   83.73   12.33  -63.20   83.73   12.33\n",
      "  214        528T 528T N   CA   4109 4110 -122.34  -42.10                  -42.10\n",
      "  215  16222 528T 529F C   N    4114 4116  -58.19  -71.40   13.60    0.82  -63.20  178.32   24.42\n",
      "  215        529F 529F N   CA   4116 4117  137.45  140.70                  -44.30\n",
      "  216  16224 530C 531K C   N    4131 4133 -135.65 -118.00   20.63    0.69  -62.90 -175.62   28.30\n",
      "  216        531K 531K N   CA   4133 4134  149.78  139.10                  -40.80\n",
      "  217  16225 531K 532F C   N    4140 4142 -102.97 -124.20   38.58    1.19  -63.20  160.39   19.78\n",
      "  217        532F 532F N   CA   4142 4143  111.08  143.30                  -44.30\n",
      "  218  16226 532F 533T C   N    4151 4153 -146.36 -124.80   23.93    0.96  -63.20 -176.11   28.98\n",
      "  218        533T 533T N   CA   4153 4154  153.89  143.50                  -42.10\n",
      "  219  16227 533T 534L C   N    4158 4160 -106.40 -108.50    2.21    0.11  -63.50  179.58   22.85\n",
      "  219        534L 534L N   CA   4160 4161  133.18  132.50                  -41.20\n",
      "  220  16228 534L 535L C   N    4166 4168 -100.61 -108.50   14.82    0.70  -63.50  165.38   21.14\n",
      "  220        535L 535L N   CA   4168 4169  119.96  132.50                  -41.20\n",
      "  221  16229 535L 536H C   N    4174 4176 -122.28  -63.20   81.95   13.79  -63.20   81.95   13.79\n",
      "  221        536H 536H N   CA   4176 4177  -99.09  -42.30                  -42.30\n",
      "  222  16232 538R 539E C   N    4203 4205 -138.84 -117.80   34.42    1.20  -63.60  172.89   27.26\n",
      "  222        539E 539E N   CA   4205 4206  164.04  136.80                  -40.30\n",
      "  223  16233 539E 540E C   N    4212 4214 -146.26 -117.80   54.73    2.05  -63.60  159.27   25.78\n",
      "  223        540E 540E N   CA   4214 4215 -176.45  136.80                  -40.30\n",
      "  224  16234 540E 541R C   N    4221 4223  -63.84  -72.10   12.71    1.10  -63.00  167.36   22.66\n",
      "  224        541R 541R N   CA   4223 4224  151.55  141.90                  -41.10\n",
      "  225  16235 541R 542Y C   N    4232 4234 -130.63 -124.30   13.46    0.98  -63.50  179.92   25.56\n",
      "  225        542Y 542Y N   CA   4234 4235  123.53  135.40                  -43.40\n",
      "  226  16236 542Y 543F C   N    4244 4246 -144.83 -124.20   27.49    1.50  -63.20 -171.93   21.65\n",
      "  226        543F 543F N   CA   4246 4247  125.13  143.30                  -44.30\n",
      "  227  16237 543F 544T C   N    4255 4257 -147.66 -124.80   26.41    1.60  -63.20 -168.06   20.46\n",
      "  227        544T 544T N   CA   4257 4258  130.25  143.50                  -42.10\n",
      "  228  16238 544T 545T C   N    4262 4264 -157.92 -124.80   33.33    1.85  -63.20 -158.20   32.00\n",
      "  228        545T 545T N   CA   4264 4265  139.71  143.50                  -42.10\n",
      "  229  16239 545T 546S C   N    4269 4271 -158.51 -136.60   22.01    0.93  -64.10 -160.42   21.25\n",
      "  229        546S 546S N   CA   4271 4272  149.16  151.20                  -35.00\n",
      "  230  16240 546S 547T C   N    4275 4277 -149.76 -124.80   26.98    1.13  -63.20 -174.43   29.40\n",
      "  230        547T 547T N   CA   4277 4278  153.75  143.50                  -42.10\n",
      "  231  16241 547T 548A C   N    4282 4284 -164.56 -134.00   30.68    1.12  -62.50 -157.54   38.62\n",
      "  231        548A 548A N   CA   4284 4285  144.25  147.00                  -40.90\n",
      "  232  16242 548A 549E C   N    4287 4289  144.98 -117.80   97.28    3.48  -63.60 -125.02   39.31\n",
      "  232        549E 549E N   CA   4289 4290  140.02  136.80                  -40.30\n",
      "  233  16243 549E 550N C   N    4296 4298 -147.20 -119.90   30.51    0.95  -63.20 -171.91   28.24\n",
      "  233        550N 550N N   CA   4298 4299  150.62  137.00                  -41.10\n",
      "  234  16245 551T 552Q C   N    4311 4313  -88.92  -73.00   27.23    1.56  -63.80  158.90   24.71\n",
      "  234        552Q 552Q N   CA   4313 4314  162.79  140.70                  -40.30\n",
      "  235  16246 552Q 553N C   N    4320 4322 -114.10 -119.90    9.94    0.54  -63.20 -178.87   25.54\n",
      "  235        553N 553N N   CA   4322 4323  145.08  137.00                  -41.10\n",
      "  236  16249 555R 556W C   N    4346 4348 -105.46 -124.90   22.86    0.74  -63.00 -179.36   20.50\n",
      "  236        556W 556W N   CA   4348 4349  131.38  143.40                  -44.20\n",
      "  237  16250 556W 557G C   N    4360 4362  177.55 -167.20   73.83    3.02   82.20  133.80   10.28\n",
      "  237        557G 557G N   CA   4362 4363  102.36  174.60                    8.50\n",
      "  238  16251 557G 558Y C   N    4364 4366 -156.04 -124.30   75.82    3.01  -63.50  145.55   27.23\n",
      "  238        558Y 558Y N   CA   4366 4367 -155.75  135.40                  -43.40\n",
      "  239  16253 559V 560K C   N    4383 4385  -65.96  -70.20   11.93    0.95  -62.90  167.68   21.94\n",
      "  239        560K 560K N   CA   4385 4386  151.55  140.40                  -40.80\n",
      "  240  16255 561V 562F C   N    4399 4401  -61.50  -71.40    9.98    0.72  -63.20  173.72   24.01\n",
      "  240        562F 562F N   CA   4401 4402  141.99  140.70                  -44.30\n",
      "  241  16256 562F 563E C   N    4410 4412 -154.60 -117.80   79.36    3.10  -63.60  144.77   24.13\n",
      "  241        563E 563E N   CA   4412 4413 -152.89  136.80                  -40.30\n",
      "  242  16257 563E 564I C   N    4419 4421  -21.61  -97.30   75.86    3.13 -120.60   99.01    6.94\n",
      "  242        564I 564I N   CA   4421 4422  132.41  127.20                  130.30\n",
      "  243  16259 565P 566E C   N    4434 4436  -65.61  -69.30    6.44    0.59  -63.60  171.93   23.03\n",
      "  243        566E 566E N   CA   4436 4437  147.78  142.50                  -40.30\n",
      "  244  16260 566E 567L C   N    4443 4445 -148.99 -108.50   57.17    2.58  -63.50  169.14   28.74\n",
      "  244        567L 567L N   CA   4445 4446  172.86  132.50                  -41.20\n",
      "  245  16261 567L 568T C   N    4451 4453 -101.36  -78.10   58.81    1.95  -63.20  120.30   17.96\n",
      "  245        568T 568T N   CA   4453 4454 -156.19  149.80                  -42.10\n",
      "  246  16265 571V 572I C   N    4479 4481  -35.14  -97.30   66.68    3.38 -120.60   88.01    6.68\n",
      "  246        572I 572I N   CA   4481 4482  151.33  127.20                  130.30\n",
      "  247  16266 572I 573R C   N    4487 4489  -35.81  -63.00   70.70    8.14  -63.00   70.70    8.14\n",
      "  247        573R 573R N   CA   4489 4490 -106.36  -41.10                  -41.10\n",
      "  248  16268 574C 575F C   N    4504 4506 -131.15 -124.20    9.26    0.50  -63.20 -168.99   31.00\n",
      "  248        575F 575F N   CA   4506 4507  137.18  143.30                  -44.30\n",
      "  249  16270 576C 577E C   N    4521 4523  -71.75  -69.30    4.28    0.25  -63.60  173.87   23.63\n",
      "  249        577E 577E N   CA   4523 4524  146.02  142.50                  -40.30\n",
      "  250  16271 577E 578H C   N    4530 4532 -110.72 -125.60   40.52    1.77  -63.20  148.99   21.74\n",
      "  250        578H 578H N   CA   4532 4533  176.49  138.80                  -42.30\n",
      "  251  16272 578H 579T C   N    4540 4542  -63.01  -78.10   21.87    1.40  -63.20  152.27   19.52\n",
      "  251        579T 579T N   CA   4542 4543  165.62  149.80                  -42.10\n",
      "  252  16274 580V 581F C   N    4554 4556 -138.88 -124.20   45.78    1.75  -63.20  149.59   25.54\n",
      "  252        581F 581F N   CA   4556 4557 -173.33  143.30                  -44.30\n",
      "  253  16275 581F 582A C   N    4565 4567  -66.56  -68.20   35.82    2.83  -62.50  150.48   24.46\n",
      "  253        582A 582A N   CA   4567 4568  109.52  145.30                  -40.90\n",
      "  254  16276 582A 583F C   N    4570 4572 -134.86 -124.20   20.00    1.11  -63.20 -174.89   21.61\n",
      "  254        583F 583F N   CA   4572 4573  126.38  143.30                  -44.30\n",
      "  255  16277 583F 584E C   N    4581 4583  -90.79  -69.30   22.11    1.49  -63.60  174.18   24.75\n",
      "  255        584E 584E N   CA   4583 4584  147.66  142.50                  -40.30\n",
      "  256  16279 585V 586F C   N    4597 4599 -135.39 -124.20   48.75    2.53  -63.20  157.66   18.04\n",
      "  256        586F 586F N   CA   4599 4600   95.86  143.30                  -44.30\n",
      "  257  16280 586F 587A C   N    4608 4610 -145.91 -134.00   12.82    0.61  -62.50 -164.46   36.51\n",
      "  257        587A 587A N   CA   4610 4611  142.25  147.00                  -40.90\n",
      "  258  16281 587A 588F C   N    4613 4615 -122.12 -124.20   12.45    0.63  -63.20  170.62   27.58\n",
      "  258        588F 588F N   CA   4615 4616  155.58  143.30                  -44.30\n",
      "  259  16282 588F 589N C   N    4624 4626 -140.05 -119.90   84.73    3.35  -63.20  125.81   19.88\n",
      "  259        589N 589N N   CA   4626 4627 -140.70  137.00                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33585  68I 467F CG2 CB    513 3635    8.96    3.51    5.45    4.93    3.51    5.45    4.93\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    6   87  134  510  475  686  619  708  712  831  908\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00200.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      590     4639     4639\n",
      "openf___224_> Open           EAO00200.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     69477392   67849.016    66.259\n",
      "read_mo_297_> Segments, residues, atoms:        1      590     4639\n",
      "read_mo_298_> Segment:        1  1:A  590:A     4639\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     4639     4639        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    21546\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    22778\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    23393\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    24312\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    25407\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    26173\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    27672\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    28645\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    29568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    31057\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    32296\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34090\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     4639    34272\n",
      "iupac_m_486_> OE1/2 will be swapped:     -151.4010        3  3\n",
      "iupac_m_487_> NH1/2 swapped:     -175.8673       10  10\n",
      "iupac_m_487_> NH1/2 swapped:      179.7093       21  21\n",
      "iupac_m_486_> OE1/2 will be swapped:      125.7463       24  24\n",
      "iupac_m_485_> OD1/2 will be swapped:     -174.5845       31  31\n",
      "iupac_m_487_> NH1/2 swapped:     -171.8981       55  55\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -98.5068       57  57\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -147.5882       63  63\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.8001       66  66\n",
      "iupac_m_486_> OE1/2 will be swapped:      102.4820       75  75\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.1376       87  87\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.8700      101  101\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8242      155  155\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.2493      160  160\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.1692      162  162\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2364      189  189\n",
      "iupac_m_487_> NH1/2 swapped:      179.7797      201  201\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9558      205  205\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8209      214  214\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.8977      227  227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9702\n",
      "iupac_m_485_> OD1/2 will be swapped:      178.5011      231  231\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9234      233  233\n",
      "iupac_m_486_> OE1/2 will be swapped:      -97.9377      248  248\n",
      "iupac_m_487_> NH1/2 swapped:      179.1124      249  249\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.2436      260  260\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.2025      261  261\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.6184      280  280\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7368      283  283\n",
      "iupac_m_485_> OD1/2 will be swapped:      125.3220      287  287\n",
      "iupac_m_487_> NH1/2 swapped:      179.9604      293  293\n",
      "iupac_m_487_> NH1/2 swapped:      179.7842      299  299\n",
      "iupac_m_486_> OE1/2 will be swapped:      135.1993      332  332\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.8041      348  348\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.0790      354  354\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3027      359  359\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.0288      366  366\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4567      368  368\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.1734      370  370\n",
      "iupac_m_486_> OE1/2 will be swapped:     -176.7263      375  375\n",
      "iupac_m_486_> OE1/2 will be swapped:      157.8393      381  381\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.0314      382  382\n",
      "iupac_m_487_> NH1/2 swapped:      179.4363      385  385\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.6896      389  389\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9260      390  390\n",
      "iupac_m_485_> OD1/2 will be swapped:      135.1443      391  391\n",
      "iupac_m_487_> NH1/2 swapped:      180.0000      394  394\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.8165      396  396\n",
      "iupac_m_485_> OD1/2 will be swapped:     -146.0128      401  401\n",
      "iupac_m_485_> OD1/2 will be swapped:     -138.6014      405  405\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3144      426  426\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6528      427  427\n",
      "iupac_m_486_> OE1/2 will be swapped:      165.5146      429  429\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.0259      433  433\n",
      "iupac_m_485_> OD1/2 will be swapped:      151.2560      440  440\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.1344      448  448\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1640      451  451\n",
      "iupac_m_487_> NH1/2 swapped:     -178.5760      461  461\n",
      "iupac_m_487_> NH1/2 swapped:      176.1063      464  464\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -110.4818      468  468\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.8899      480  480\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9802      496  496\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.2533      499  499\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1026      509  509\n",
      "iupac_m_487_> NH1/2 swapped:     -176.3663      541  541\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.5193      542  542\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.3137      558  558\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -98.7387      562  562\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.9261      563  563\n",
      "iupac_m_487_> NH1/2 swapped:      169.1454      573  573\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.3326      584  584\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ALA:OXT  ALA\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):  1030725\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -40617.1641\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -40617.164062\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    31.629000\n",
      "Sequence length          :          590\n",
      "Compactness              :     0.175586\n",
      "Native energy (pair)     :   753.144939\n",
      "Native energy (surface)  :    80.619057\n",
      "Native energy (combined) :    20.698874\n",
      "Z score (pair)           :    -3.336747\n",
      "Z score (surface)        :    -1.358251\n",
      "Z score (combined)       :    -2.912060\n",
      "GA341 score              :     0.368922\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      590\n",
      "Number of all, selected real atoms                :     4639    4639\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34272   34272\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):    12615\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        6858.9512\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    4711       0      0   0.007   0.007      65.850       1.000\n",
      " 2 Bond angle potential               :    6336       3     44   2.377   2.377      691.86       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2751       0    152  53.137  53.137      1248.2       1.000\n",
      " 4 Stereochemical improper torsion pot:    1896       1      3   1.656   1.656      103.10       1.000\n",
      " 5 Soft-sphere overlap restraints     :   12615       0      0   0.007   0.007      64.433       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6251      20    103   1.305   1.305      1149.3       1.000\n",
      "10 Distance restraints 2 (N-O)        :    6377      25    142   1.582   1.582      1393.9       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     589       7     25   6.096   6.096      256.92       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     499       0     19  89.728  89.728      230.20       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     371       0      7  86.176  86.176      203.91       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     179       0      1 101.149 101.149      143.80       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      77       0      0 103.492 103.492      49.340       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2718       0      0   0.694   0.694      79.795       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     588     262    128  40.016 115.927      900.51       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     929       5     34   2.960   2.960      277.89       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00200.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00200.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:  105920.2812\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8904 399F 399F N   CA   3089 3090  131.83  107.00   24.83    7.14  107.00   24.83    7.14\n",
      "    2   9025 410L 410L N   CA   3177 3178  123.71  107.00   16.71    4.80  107.00   16.71    4.80\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  19009  38V 537D CA  CA    276 4187   19.76   13.45    6.30    5.66   13.45    6.30    5.66\n",
      "    2  19147  49T 537D CA  CA    351 4187   19.87   12.38    7.49    6.45   12.38    7.49    6.45\n",
      "    3  19163  50M 537D CA  CA    358 4187   16.73   10.61    6.12    6.07   10.61    6.12    6.07\n",
      "    4  19164  50M 538R CA  CA    358 4195   19.37   14.00    5.38    4.81   14.00    5.38    4.81\n",
      "    5  19180  51F 537D CA  CA    366 4187   13.14    7.18    5.96    7.89    7.18    5.96    7.89\n",
      "    6  19181  51F 538R CA  CA    366 4195   15.70   10.77    4.92    5.50   10.77    4.92    5.50\n",
      "    7  23631 359R 538R CA  CA   2761 4195   17.33    9.45    7.88    5.10    9.45    7.88    5.10\n",
      "    8  23642 360E 397V CA  CA   2772 3078   18.25    7.52   10.73    5.22    7.52   10.73    5.22\n",
      "    9  23643 360E 398A CA  CA   2772 3085   14.86    7.42    7.43    4.63    7.42    7.43    4.63\n",
      "   10  23667 361L 397V CA  CA   2781 3078   16.20    5.84   10.36    4.86    5.84   10.36    4.86\n",
      "   11  23817 374L 397V CA  CA   2875 3078   17.26    3.81   13.45    4.54    3.81   13.45    4.54\n",
      "   12  24208 509R 537D CA  CA   3969 4187   15.61    4.99   10.62    4.84    4.99   10.62    4.84\n",
      "   13  24209 509R 538R CA  CA   3969 4195   16.68    6.39   10.29    4.57    6.39   10.29    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  25071  51F 537D N   O     365 4193   14.81   10.14    4.67    4.59   10.14    4.67    4.59\n",
      "    2  27682 219V 138V N   O    1690 1055    4.69    2.79    1.90    5.49    2.79    1.90    5.49\n",
      "    3  28976 342N 144Y N   O    2620 1106   11.15    6.52    4.63    6.04    6.52    4.63    6.04\n",
      "    4  29187 359R 538R N   O    2760 4204   18.91   10.91    7.99    4.60   10.91    7.99    4.60\n",
      "    5  29346 397V 359R N   O    3077 2770   19.64   10.54    9.11    5.23   10.54    9.11    5.23\n",
      "    6  29347 397V 360E N   O    3077 2779   19.85    7.05   12.80    5.92    7.05   12.80    5.92\n",
      "    7  29348 397V 361L N   O    3077 2787   18.91    7.78   11.13    4.67    7.78   11.13    4.67\n",
      "    8  29351 397V 371V N   O    3077 2860   23.47    7.53   15.94    4.75    7.53   15.94    4.75\n",
      "    9  29353 397V 374L N   O    3077 2881   17.29    2.26   15.03    4.99    2.26   15.03    4.99\n",
      "   10  29362 398A 359R N   O    3084 2770   16.34    9.28    7.06    5.80    9.28    7.06    5.80\n",
      "   11  29363 398A 360E N   O    3084 2779   16.29    6.19   10.10    6.14    6.19   10.10    6.14\n",
      "   12  29483 410L 145M N   O    3177 1114   12.33    8.77    3.55    4.99    8.77    3.55    4.99\n",
      "   13  29484 410L 146E N   O    3177 1123   10.32    6.64    3.69    5.50    6.64    3.69    5.50\n",
      "   14  30419 537D  50M N   O    4186  364   16.36    9.63    6.73    6.42    9.63    6.73    6.42\n",
      "   15  30424 537D  87Y N   O    4186  669   14.72   10.32    4.40    6.08   10.32    4.40    6.08\n",
      "   16  30425 537D 359R N   O    4186 2770   19.14   10.84    8.30    4.61   10.84    8.30    4.61\n",
      "   17  30435 537D 508L N   O    4186 3967   15.50    2.68   12.82    6.33    2.68   12.82    6.33\n",
      "   18  30436 537D 509R N   O    4186 3978   15.47    3.33   12.14    5.51    3.33   12.14    5.51\n",
      "   19  30450 538R 359R N   O    4194 2770   17.11    9.13    7.98    4.85    9.13    7.98    4.85\n",
      "   20  30459 538R 508L N   O    4194 3967   15.56    4.90   10.66    5.01    4.90   10.66    5.01\n",
      "   21  30460 538R 509R N   O    4194 3978   14.79    3.14   11.65    5.43    3.14   11.65    5.43\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16680 398A 398A CA  C    3085 3087 -153.11 -180.00   26.89    5.38 -180.00   26.89    5.38\n",
      "    2  16681 399F 399F CA  C    3090 3098 -154.42 -180.00   25.58    5.11 -180.00   25.58    5.11\n",
      "    3  16690 408D 408D CA  C    3161 3166 -154.53 -180.00   25.47    5.09 -180.00   25.47    5.09\n",
      "    4  16691 409K 409K CA  C    3169 3175 -149.83 -180.00   30.17    6.03 -180.00   30.17    6.03\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15696   2A   3E C   N      12   14 -101.73 -117.80   34.77    1.36  -63.60  151.16   18.33\n",
      "    1          3E   3E N   CA     14   15  105.97  136.80                  -40.30\n",
      "    2  15698   4L   5A C   N      29   31  -77.47  -68.20   19.58    1.28  -62.50  157.27   26.53\n",
      "    2          5A   5A N   CA     31   32  162.55  145.30                  -40.90\n",
      "    3  15699   5A   6G C   N      34   36  -63.40  -62.40    1.85    0.33   82.20  154.36   11.76\n",
      "    3          6G   6G N   CA     36   37  -42.76  -41.20                    8.50\n",
      "    4  15700   6G   7T C   N      38   40  -85.51  -78.10   32.32    1.16  -63.20  138.45   19.24\n",
      "    4          7T   7T N   CA     40   41 -178.74  149.80                  -42.10\n",
      "    5  15701   7T   8A C   N      45   47 -114.42 -134.00   95.55    4.90  -68.20  102.80    9.72\n",
      "    5          8A   8A N   CA     47   48   53.48  147.00                  145.30\n",
      "    6  15710  16R  17M C   N     111  113  -77.34  -73.00   39.08    2.57  -63.40  138.36   21.76\n",
      "    6         17M  17M N   CA    113  114 -178.16  143.00                  -40.50\n",
      "    7  15713  19P  20P C   N     132  134  -81.90  -64.50   17.40    1.77  -58.70  179.50   13.04\n",
      "    7         20P  20P N   CA    134  135  147.49  147.20                  -30.50\n",
      "    8  15714  20P  21R C   N     139  141  140.69  -63.00  157.39   27.16  -63.00  157.39   27.16\n",
      "    8         21R  21R N   CA    141  142  -59.60  -41.10                  -41.10\n",
      "    9  15720  26V  27I C   N     188  190 -121.47 -120.60   90.53    5.21  -63.40  101.61   15.45\n",
      "    9         27I  27I N   CA    190  191   39.78  130.30                  -43.60\n",
      "   10  15724  30A  31D C   N     218  220   71.75   54.50   34.72    1.70  -63.30  144.28   24.51\n",
      "   10         31D  31D N   CA    220  221   10.77   40.90                  -40.00\n",
      "   11  15737  43E  44G C   N     318  320   40.28   82.20   89.65    2.53  -62.40  164.83   29.97\n",
      "   11         44G  44G N   CA    320  321   87.74    8.50                  -41.20\n",
      "   12  15738  44G  45G C   N     322  324  114.19   82.20   43.17    1.43  -62.40  177.81   32.52\n",
      "   12         45G  45G N   CA    324  325  -20.48    8.50                  -41.20\n",
      "   13  15752  58T  59G C   N     438  440 -162.84 -167.20   12.03    0.49   82.20 -148.68   15.28\n",
      "   13         59G  59G N   CA    440  441 -174.19  174.60                    8.50\n",
      "   14  15795 101D 102L C   N     765  767  -67.39  -63.50    8.75    1.07 -108.50  170.89    8.75\n",
      "   14        102L 102L N   CA    767  768  -33.37  -41.20                  132.50\n",
      "   15  15798 104S 105K C   N     787  789  -63.81  -70.20   29.45    2.28   56.60  177.60   10.85\n",
      "   15        105K 105K N   CA    789  790  169.15  140.40                   38.60\n",
      "   16  15799 105K 106E C   N     796  798  -57.01  -69.30   25.70    2.31  -63.60  154.77   20.28\n",
      "   16        106E 106E N   CA    798  799  165.07  142.50                  -40.30\n",
      "   17  15800 106E 107G C   N     805  807  123.77   82.20   44.33    2.28  -62.40  177.18   30.07\n",
      "   17        107G 107G N   CA    807  808   -6.90    8.50                  -41.20\n",
      "   18  15813 119I 120E C   N     904  906  -58.76  -69.30   36.38    3.11  -63.60  142.46   18.73\n",
      "   18        120E 120E N   CA    906  907  177.32  142.50                  -40.30\n",
      "   19  15825 131A 132A C   N     996  998  -64.42  -62.50   26.34    4.23 -134.00  175.97    7.84\n",
      "   19        132A 132A N   CA    998  999  -14.63  -40.90                  147.00\n",
      "   20  15852 158G 159Q C   N    1216 1218  -93.22  -73.00   48.08    3.55  -63.80  140.49   19.23\n",
      "   20        159Q 159Q N   CA   1218 1219   97.07  140.70                  -40.30\n",
      "   21  15853 159Q 160E C   N    1225 1227  178.67 -117.80   72.33    2.11  -63.60 -170.64   31.51\n",
      "   21        160E 160E N   CA   1227 1228  171.39  136.80                  -40.30\n",
      "   22  15855 161P 162E C   N    1241 1243   48.58  -69.30  125.01    7.98  -63.60 -179.66   30.01\n",
      "   22        162E 162E N   CA   1243 1244  100.90  142.50                  -40.30\n",
      "   23  15856 162E 163S C   N    1250 1252 -121.74 -136.60   25.62    1.45  -64.10  163.43   15.91\n",
      "   23        163S 163S N   CA   1252 1253  172.07  151.20                  -35.00\n",
      "   24  15857 163S 164L C   N    1256 1258  -45.33  -70.70   36.56    2.33   60.20  135.93    8.41\n",
      "   24        164L 164L N   CA   1258 1259  115.28  141.60                   29.60\n",
      "   25  15862 168E 169D C   N    1300 1302  -61.27  -63.30    5.71    0.61  -70.90  164.65    8.59\n",
      "   25        169D 169D N   CA   1302 1303  -45.33  -40.00                  150.30\n",
      "   26  15863 169D 170S C   N    1308 1310 -117.79 -136.60   26.35    0.85  -64.10  176.13   10.85\n",
      "   26        170S 170S N   CA   1310 1311  132.75  151.20                  -35.00\n",
      "   27  15864 170S 171Q C   N    1314 1316 -112.96 -121.10   22.74    0.92  -63.80  166.21   22.18\n",
      "   27        171Q 171Q N   CA   1316 1317  118.47  139.70                  -40.30\n",
      "   28  15865 171Q 172H C   N    1323 1325 -155.05 -125.60   29.46    1.01  -63.20 -159.66   31.01\n",
      "   28        172H 172H N   CA   1325 1326  139.66  138.80                  -42.30\n",
      "   29  15866 172H 173R C   N    1333 1335 -100.74 -125.20   72.87    2.91  -63.00  119.18   13.99\n",
      "   29        173R 173R N   CA   1335 1336   71.95  140.60                  -41.10\n",
      "   30  15868 174V 175Y C   N    1351 1353 -128.69 -124.30    4.49    0.30  -63.50 -168.82   27.37\n",
      "   30        175Y 175Y N   CA   1353 1354  136.32  135.40                  -43.40\n",
      "   31  15870 176V 177K C   N    1370 1372  -66.69  -70.20   16.51    1.28  -62.90  162.71   21.34\n",
      "   31        177K 177K N   CA   1372 1373  156.53  140.40                  -40.80\n",
      "   32  15871 177K 178G C   N    1379 1381  -66.23  -80.20   32.36    0.75   82.20 -158.41    6.62\n",
      "   32        178G 178G N   CA   1381 1382  144.92  174.10                    8.50\n",
      "   33  15872 178G 179A C   N    1383 1385  -86.81  -68.20   28.25    1.78  -62.50  154.47   26.57\n",
      "   33        179A 179A N   CA   1385 1386  166.56  145.30                  -40.90\n",
      "   34  15873 179A 180G C   N    1388 1390  -63.43  -62.40    5.95    0.84   82.20  152.09   11.46\n",
      "   34        180G 180G N   CA   1390 1391  -35.34  -41.20                    8.50\n",
      "   35  15874 180G 181V C   N    1392 1394  -58.56  -62.40    5.21    0.88 -125.40 -170.03   10.68\n",
      "   35        181V 181V N   CA   1394 1395  -38.88  -42.40                  143.30\n",
      "   36  15875 181V 182H C   N    1399 1401   56.03   56.30    7.36    0.63  -63.20  149.66   25.55\n",
      "   36        182H 182H N   CA   1401 1402   48.16   40.80                  -42.30\n",
      "   37  15876 182H 183S C   N    1409 1411 -102.87  -72.40   51.51    3.98  -64.10  150.94    9.37\n",
      "   37        183S 183S N   CA   1411 1412  110.87  152.40                  -35.00\n",
      "   38  15879 185L 186S C   N    1430 1432  -94.43  -72.40   41.61    3.16  -64.10  155.10    9.85\n",
      "   38        186S 186S N   CA   1432 1433  117.10  152.40                  -35.00\n",
      "   39  15881 187L 188E C   N    1444 1446  178.45 -117.80   76.75    2.27  -63.60 -176.82   30.64\n",
      "   39        188E 188E N   CA   1446 1447  179.55  136.80                  -40.30\n",
      "   40  15890 196K 197G C   N    1523 1525   70.30   78.70   14.48    0.73   82.20  174.02    7.85\n",
      "   40        197G 197G N   CA   1525 1526 -177.89 -166.10                    8.50\n",
      "   41  15891 197G 198N C   N    1527 1529 -116.58 -119.90    4.18    0.22  -63.20 -172.87   26.44\n",
      "   41        198N 198N N   CA   1529 1530  139.55  137.00                  -41.10\n",
      "   42  15892 198N 199A C   N    1535 1537  -73.63  -68.20   10.36    0.67  -62.50  165.35   27.66\n",
      "   42        199A 199A N   CA   1537 1538  154.13  145.30                  -40.90\n",
      "   43  15893 199A 200N C   N    1540 1542  -82.93  -71.20   42.97    3.08  -63.20  143.91   16.98\n",
      "   43        200N 200N N   CA   1542 1543  101.46  142.80                  -41.10\n",
      "   44  15894 200N 201R C   N    1548 1550 -137.52 -125.20   16.60    0.50  -63.00 -176.97   29.82\n",
      "   44        201R 201R N   CA   1550 1551  151.73  140.60                  -41.10\n",
      "   45  15895 201R 202Q C   N    1559 1561  -89.91  -73.00   23.65    1.84  -63.80  166.54   23.19\n",
      "   45        202Q 202Q N   CA   1561 1562  124.17  140.70                  -40.30\n",
      "   46  15896 202Q 203T C   N    1568 1570 -136.41 -124.80   21.22    0.66  -63.20  172.91   26.95\n",
      "   46        203T 203T N   CA   1570 1571  161.26  143.50                  -42.10\n",
      "   47  15897 203T 204G C   N    1575 1577 -119.76  -80.20   66.47    4.45   82.20 -166.20   15.13\n",
      "   47        204G 204G N   CA   1577 1578  120.67  174.10                    8.50\n",
      "   48  15898 204G 205E C   N    1579 1581 -157.92 -117.80   43.39    1.29  -63.60 -168.75   30.72\n",
      "   48        205E 205E N   CA   1581 1582  153.33  136.80                  -40.30\n",
      "   49  15899 205E 206T C   N    1588 1590  -86.75  -78.10   13.84    0.87  -63.20 -179.55   24.71\n",
      "   49        206T 206T N   CA   1590 1591  138.99  149.80                  -42.10\n",
      "   50  15900 206T 207K C   N    1595 1597 -121.17 -118.00    9.23    0.48  -62.90 -179.13   21.08\n",
      "   50        207K 207K N   CA   1597 1598  130.43  139.10                  -40.80\n",
      "   51  15902 208M 209N C   N    1612 1614  -58.54  -71.20   18.14    1.46  -63.20  163.17   20.15\n",
      "   51        209N 209N N   CA   1614 1615  155.80  142.80                  -41.10\n",
      "   52  15903 209N 210E C   N    1620 1622  -67.19  -69.30   12.35    1.01  -63.60  165.07   22.21\n",
      "   52        210E 210E N   CA   1622 1623  154.67  142.50                  -40.30\n",
      "   53  15906 212S 213S C   N    1642 1644  143.82 -136.60   81.28    2.82  -64.10 -141.23   26.75\n",
      "   53        213S 213S N   CA   1644 1645  167.74  151.20                  -35.00\n",
      "   54  15907 213S 214R C   N    1648 1650 -126.79 -125.20    7.45    0.32  -63.00 -177.47   29.05\n",
      "   54        214R 214R N   CA   1650 1651  147.87  140.60                  -41.10\n",
      "   55  15908 214R 215S C   N    1659 1661 -103.48  -72.40   45.45    3.60  -64.10  159.18    9.91\n",
      "   55        215S 215S N   CA   1661 1662  119.24  152.40                  -35.00\n",
      "   56  15909 215S 216H C   N    1665 1667 -164.14 -125.60   77.19    2.14  -63.20  150.79   25.06\n",
      "   56        216H 216H N   CA   1667 1668 -154.32  138.80                  -42.30\n",
      "   57  15910 216H 217A C   N    1675 1677 -138.49 -134.00    4.70    0.11  -62.50 -173.11   34.70\n",
      "   57        217A 217A N   CA   1677 1678  148.37  147.00                  -40.90\n",
      "   58  15912 218I 219V C   N    1688 1690  -47.21  -73.50  130.99   11.46 -125.40  146.78    8.44\n",
      "   58        219V 219V N   CA   1690 1691  -92.48  139.20                  143.30\n",
      "   59  15913 219V 220Q C   N    1695 1697  -68.91  -63.80    8.15    1.05  -73.00  174.70   11.34\n",
      "   59        220Q 220Q N   CA   1697 1698  -33.95  -40.30                  140.70\n",
      "   60  15917 223I 224S C   N    1727 1729   66.79  -72.40  145.62    9.00  -64.10 -164.95   23.54\n",
      "   60        224S 224S N   CA   1729 1730  109.61  152.40                  -35.00\n",
      "   61  15918 224S 225Q C   N    1733 1735 -137.88 -121.10   23.60    0.77  -63.80  179.40   30.26\n",
      "   61        225Q 225Q N   CA   1735 1736  156.30  139.70                  -40.30\n",
      "   62  15922 228G 229S C   N    1767 1769  -61.71  -72.40   44.52    2.27  -64.10  144.21   10.59\n",
      "   62        229S 229S N   CA   1769 1770  109.19  152.40                  -35.00\n",
      "   63  15923 229S 230L C   N    1773 1775  -67.15  -70.70   11.30    0.74  -63.50  172.12   23.69\n",
      "   63        230L 230L N   CA   1775 1776  130.88  141.60                  -41.20\n",
      "   64  15924 230L 231D C   N    1781 1783 -141.58  -70.90   93.98    4.06  -63.30  133.20   21.37\n",
      "   64        231D 231D N   CA   1783 1784 -147.77  150.30                  -40.00\n",
      "   65  15926 232M 233R C   N    1797 1799 -151.58 -125.20   35.92    1.96  -63.00 -179.45   20.44\n",
      "   65        233R 233R N   CA   1799 1800  116.22  140.60                  -41.10\n",
      "   66  15927 233R 234D C   N    1808 1810  -69.18  -63.30   89.34   11.39  -63.30   89.34   11.39\n",
      "   66        234D 234D N   CA   1810 1811 -129.15  -40.00                  -40.00\n",
      "   67  15930 236E 237S C   N    1832 1834  125.36 -136.60  113.52    3.46  -64.10 -153.43   26.84\n",
      "   67        237S 237S N   CA   1834 1835 -151.57  151.20                  -35.00\n",
      "   68  15933 239V 240L C   N    1852 1854   76.87  -70.70  153.72   12.39  -63.50 -161.91   35.70\n",
      "   68        240L 240L N   CA   1854 1855   98.57  141.60                  -41.20\n",
      "   69  15934 240L 241L C   N    1860 1862   58.75  -70.70  130.63   11.27  -63.50 -154.43   35.94\n",
      "   69        241L 241L N   CA   1862 1863  124.07  141.60                  -41.20\n",
      "   70  15947 253T 254E C   N    1957 1959   43.41  -69.30  118.60   10.15  -63.60  176.46   20.51\n",
      "   70        254E 254E N   CA   1959 1960  179.39  142.50                  -40.30\n",
      "   71  15948 254E 255S C   N    1966 1968  -86.05  -72.40   19.69    0.93  -64.10  159.91   13.02\n",
      "   71        255S 255S N   CA   1968 1969  166.60  152.40                  -35.00\n",
      "   72  15949 255S 256T C   N    1972 1974  -87.63  -78.10   41.55    1.50  -63.20  129.98   18.30\n",
      "   72        256T 256T N   CA   1974 1975 -169.76  149.80                  -42.10\n",
      "   73  15950 256T 257G C   N    1979 1981 -102.35  -80.20   43.77    1.03   82.20 -124.79    7.85\n",
      "   73        257G 257G N   CA   1981 1982 -148.15  174.10                    8.50\n",
      "   74  15952 258V 259A C   N    1990 1992 -146.00 -134.00   62.76    3.25  -62.50  138.50   27.08\n",
      "   74        259A 259A N   CA   1992 1993 -151.40  147.00                  -40.90\n",
      "   75  15953 259A 260F C   N    1995 1997 -107.52 -124.20   35.30    1.18  -63.20  162.65   19.85\n",
      "   75        260F 260F N   CA   1997 1998  112.20  143.30                  -44.30\n",
      "   76  15954 260F 261E C   N    2006 2008  -60.40  -69.30   23.98    1.51  -63.60  160.56   21.58\n",
      "   76        261E 261E N   CA   2008 2009  120.23  142.50                  -40.30\n",
      "   77  15955 261E 262E C   N    2015 2017 -150.50 -117.80   37.92    1.11  -63.60 -174.67   29.54\n",
      "   77        262E 262E N   CA   2017 2018  156.01  136.80                  -40.30\n",
      "   78  15956 262E 263A C   N    2024 2026  -67.55  -68.20    8.64    0.67  -62.50  177.66   28.87\n",
      "   78        263A 263A N   CA   2026 2027  136.69  145.30                  -40.90\n",
      "   79  15957 263A 264K C   N    2029 2031 -136.73 -118.00   18.81    0.68  -62.90 -167.00   29.48\n",
      "   79        264K 264K N   CA   2031 2032  140.88  139.10                  -40.80\n",
      "   80  15958 264K 265K C   N    2038 2040 -105.14 -118.00   46.75    2.06  -62.90  141.41   16.56\n",
      "   80        265K 265K N   CA   2040 2041   94.16  139.10                  -40.80\n",
      "   81  15960 266I 267N C   N    2055 2057 -131.77 -119.90   17.24    0.94  -63.20  179.23   19.53\n",
      "   81        267N 267N N   CA   2057 2058  124.49  137.00                  -41.10\n",
      "   82  15961 267N 268Q C   N    2063 2065 -156.84 -121.10   38.01    1.21  -63.80 -168.78   32.91\n",
      "   82        268Q 268Q N   CA   2065 2066  152.65  139.70                  -40.30\n",
      "   83  15963 269S 270L C   N    2078 2080  -78.79  -70.70   11.59    0.74  -63.50  169.59   24.45\n",
      "   83        270L 270L N   CA   2080 2081  149.90  141.60                  -41.20\n",
      "   84  15964 270L 271L C   N    2086 2088  -75.46  -70.70    5.05    0.40  -63.50  175.92   25.13\n",
      "   84        271L 271L N   CA   2088 2089  143.28  141.60                  -41.20\n",
      "   85  15965 271L 272M C   N    2094 2096 -116.19 -125.60   13.85    0.43  -63.40  178.80   23.17\n",
      "   85        272M 272M N   CA   2096 2097  130.33  140.50                  -40.50\n",
      "   86  15966 272M 273L C   N    2102 2104 -130.17 -108.50   39.98    1.88  -63.50  166.62   27.26\n",
      "   86        273L 273L N   CA   2104 2105  166.10  132.50                  -41.20\n",
      "   87  15967 273L 274G C   N    2110 2112   96.36   78.70   25.25    0.56   82.20  167.94    8.86\n",
      "   87        274G 274G N   CA   2112 2113  175.84 -166.10                    8.50\n",
      "   88  15968 274G 275R C   N    2114 2116 -128.46 -125.20   67.96    3.21  -63.00  128.37   21.73\n",
      "   88        275R 275R N   CA   2116 2117 -151.52  140.60                  -41.10\n",
      "   89  15969 275R 276A C   N    2125 2127  -69.07  -68.20   49.09    3.99  -62.50  137.28   22.18\n",
      "   89        276A 276A N   CA   2127 2128   96.22  145.30                  -40.90\n",
      "   90  15970 276A 277M C   N    2130 2132  148.09 -125.60   86.38    3.33  -73.00  138.91   10.51\n",
      "   90        277M 277M N   CA   2132 2133  143.98  140.50                  143.00\n",
      "   91  15972 278N 279S C   N    2146 2148  -78.60  -72.40   82.43    5.06  -64.10  106.20    6.96\n",
      "   91        279S 279S N   CA   2148 2149   70.21  152.40                  -35.00\n",
      "   92  15992 298T 299R C   N    2297 2299 -152.23 -125.20   33.56    1.79  -63.00 -175.21   20.93\n",
      "   92        299R 299R N   CA   2299 2300  120.72  140.60                  -41.10\n",
      "   93  15993 299R 300L C   N    2308 2310 -146.39 -108.50   37.98    1.78  -63.50 -165.07   23.92\n",
      "   93        300L 300L N   CA   2310 2311  135.23  132.50                  -41.20\n",
      "   94  15994 300L 301L C   N    2316 2318 -102.88 -108.50   11.28    0.54  -63.50  168.59   21.49\n",
      "   94        301L 301L N   CA   2318 2319  122.72  132.50                  -41.20\n",
      "   95  15995 301L 302S C   N    2324 2326 -139.74 -136.60    9.97    0.43  -64.10 -179.09   18.51\n",
      "   95        302S 302S N   CA   2326 2327  160.66  151.20                  -35.00\n",
      "   96  15996 302S 303E C   N    2330 2332  -69.36  -69.30    2.26    0.17  -63.60  179.55   24.26\n",
      "   96        303E 303E N   CA   2332 2333  140.24  142.50                  -40.30\n",
      "   97  15997 303E 304S C   N    2339 2341 -133.59 -136.60    7.28    0.41  -64.10 -178.96   18.05\n",
      "   97        304S 304S N   CA   2341 2342  157.83  151.20                  -35.00\n",
      "   98  15998 304S 305F C   N    2345 2347  -98.64  -71.40   31.88    2.67  -63.20  172.14   21.64\n",
      "   98        305F 305F N   CA   2347 2348  124.15  140.70                  -44.30\n",
      "   99  15999 305F 306G C   N    2356 2358   65.35   78.70   21.77    0.39   82.20  158.31    8.53\n",
      "   99        306G 306G N   CA   2358 2359 -148.91 -166.10                    8.50\n",
      "  100  16000 306G 307G C   N    2360 2362   90.75   78.70   35.71    0.58   82.20  152.02    7.81\n",
      "  100        307G 307G N   CA   2362 2363  160.29 -166.10                    8.50\n",
      "  101  16001 307G 308N C   N    2364 2366 -116.71 -119.90    4.25    0.12  -63.20 -176.73   20.44\n",
      "  101        308N 308N N   CA   2366 2367  134.18  137.00                  -41.10\n",
      "  102  16002 308N 309S C   N    2372 2374  -75.61  -72.40    5.73    0.44  -64.10  177.72   13.59\n",
      "  102        309S 309S N   CA   2374 2375  147.65  152.40                  -35.00\n",
      "  103  16003 309S 310K C   N    2378 2380 -133.11 -118.00   24.06    0.89  -62.90  175.98   27.06\n",
      "  103        310K 310K N   CA   2380 2381  157.83  139.10                  -40.80\n",
      "  104  16005 311T 312W C   N    2394 2396  -79.22  -71.30   14.87    0.91  -63.00  165.02   21.65\n",
      "  104        312W 312W N   CA   2396 2397  151.59  139.00                  -44.20\n",
      "  105  16006 312W 313M C   N    2408 2410  -67.78  -73.00    5.84    0.47  -63.40  173.94   26.31\n",
      "  105        313M 313M N   CA   2410 2411  145.61  143.00                  -40.50\n",
      "  106  16007 313M 314L C   N    2416 2418  -72.25  -70.70    5.34    0.35  -63.50  172.31   24.44\n",
      "  106        314L 314L N   CA   2418 2419  146.71  141.60                  -41.20\n",
      "  107  16008 314L 315A C   N    2424 2426 -112.79 -134.00   24.00    0.55  -62.50 -176.33   28.10\n",
      "  107        315A 315A N   CA   2426 2427  135.76  147.00                  -40.90\n",
      "  108  16009 315A 316T C   N    2429 2431  -97.31  -78.10   31.86    1.08  -63.20  146.71   21.09\n",
      "  108        316T 316T N   CA   2431 2432  175.21  149.80                  -42.10\n",
      "  109  16010 316T 317V C   N    2436 2438  -64.41  -62.40    2.82    0.47 -125.40 -177.21   10.24\n",
      "  109        317V 317V N   CA   2438 2439  -44.38  -42.40                  143.30\n",
      "  110  16012 318S 319T C   N    2449 2451  -78.07  -78.10    1.92    0.08  -63.20  170.67   22.87\n",
      "  110        319T 319T N   CA   2451 2452  147.88  149.80                  -42.10\n",
      "  111  16015 321A 322N C   N    2466 2468  -66.78  -71.20   23.65    1.64   55.90  176.23   10.34\n",
      "  111        322N 322N N   CA   2468 2469  166.03  142.80                   39.50\n",
      "  112  16033 339A 340I C   N    2603 2605  -66.47  -63.40    3.28    0.61 -120.60 -176.77    8.49\n",
      "  112        340I 340I N   CA   2605 2606  -44.75  -43.60                  130.30\n",
      "  113  16035 341T 342N C   N    2618 2620 -108.31 -119.90   16.63    0.90  -63.20  175.85   24.55\n",
      "  113        342N 342N N   CA   2620 2621  148.93  137.00                  -41.10\n",
      "  114  16036 342N 343K C   N    2626 2628  -58.15  -70.20   35.09    2.26  -62.90  148.31   19.52\n",
      "  114        343K 343K N   CA   2628 2629  107.44  140.40                  -40.80\n",
      "  115  16037 343K 344A C   N    2635 2637   72.36  -68.20  140.63   11.77  -62.50 -136.54   33.76\n",
      "  115        344A 344A N   CA   2637 2638  140.93  145.30                  -40.90\n",
      "  116  16038 344A 345K C   N    2640 2642  -84.79  -70.20   80.68    6.18  -62.90  104.17   12.50\n",
      "  116        345K 345K N   CA   2642 2643   61.05  140.40                  -40.80\n",
      "  117  16040 346V 347N C   N    2656 2658  -76.34 -119.90   78.15    2.47   55.90  136.20   13.46\n",
      "  117        347N 347N N   CA   2658 2659   72.12  137.00                   39.50\n",
      "  118  16043 349T 350K C   N    2679 2681   86.48   56.60  123.27   11.13  -62.90 -140.37   26.38\n",
      "  118        350K 350K N   CA   2681 2682  158.19   38.60                  -40.80\n",
      "  119  16044 350K 351K C   N    2688 2690  -90.45  -70.20   26.23    1.58  -62.90  164.45   22.95\n",
      "  119        351K 351K N   CA   2690 2691  157.07  140.40                  -40.80\n",
      "  120  16045 351K 352N C   N    2697 2699 -130.26 -119.90   51.64    2.56  -63.20  144.06   15.51\n",
      "  120        352N 352N N   CA   2699 2700   86.41  137.00                  -41.10\n",
      "  121  16046 352N 353I C   N    2705 2707  -82.98  -63.40   21.79    3.54 -120.60  168.60    8.22\n",
      "  121        353I 353I N   CA   2707 2708  -34.04  -43.60                  130.30\n",
      "  122  16047 353I 354E C   N    2713 2715  175.43 -117.80   84.14    4.33  -63.60  174.60   20.72\n",
      "  122        354E 354E N   CA   2715 2716   85.60  136.80                  -40.30\n",
      "  123  16048 354E 355L C   N    2722 2724  -73.86  -63.50   54.10    8.14  -63.50   54.10    8.14\n",
      "  123        355L 355L N   CA   2724 2725  -94.30  -41.20                  -41.20\n",
      "  124  16049 355L 356K C   N    2730 2732  -96.63  -62.90   46.83    7.99  -62.90   46.83    7.99\n",
      "  124        356K 356K N   CA   2732 2733  -73.29  -40.80                  -40.80\n",
      "  125  16056 362V 363A C   N    2793 2795  169.98 -134.00   57.40    1.48  -68.20  122.64    9.90\n",
      "  125        363A 363A N   CA   2795 2796  159.51  147.00                  145.30\n",
      "  126  16057 363A 364S C   N    2798 2800 -159.27 -136.60   54.28    2.18  -64.10  156.69   18.22\n",
      "  126        364S 364S N   CA   2800 2801 -159.48  151.20                  -35.00\n",
      "  127  16058 364S 365L C   N    2804 2806  -80.34  -70.70   40.17    2.70  -63.50  139.23   20.34\n",
      "  127        365L 365L N   CA   2806 2807 -179.41  141.60                  -41.20\n",
      "  128  16059 365L 366E C   N    2812 2814  -73.87  -69.30   49.04    3.58  -63.60  128.78   17.74\n",
      "  128        366E 366E N   CA   2814 2815 -168.67  142.50                  -40.30\n",
      "  129  16061 367G 368R C   N    2825 2827   67.92   57.30   26.01    1.26  -63.00  142.15   25.75\n",
      "  129        368R 368R N   CA   2827 2828   14.26   38.00                  -41.10\n",
      "  130  16062 368R 369L C   N    2836 2838  -97.65 -108.50   37.84    1.91  -63.50  141.63   18.01\n",
      "  130        369L 369L N   CA   2838 2839   96.26  132.50                  -41.20\n",
      "  131  16063 369L 370D C   N    2844 2846 -112.75  -96.50   21.00    0.88  -63.30  174.64   19.12\n",
      "  131        370D 370D N   CA   2846 2847  127.50  114.20                  -40.00\n",
      "  132  16067 373A 374L C   N    2872 2874 -174.00 -108.50   76.54    3.38  -70.70  107.71    8.67\n",
      "  132        374L 374L N   CA   2874 2875  172.11  132.50                  141.60\n",
      "  133  16068 374L 375E C   N    2880 2882  130.35 -117.80  113.93    3.69  -63.60 -128.56   39.30\n",
      "  133        375E 375E N   CA   2882 2883  158.48  136.80                  -40.30\n",
      "  134  16069 375E 376K C   N    2889 2891 -101.84 -118.00   37.05    1.52  -62.90  151.64   17.95\n",
      "  134        376K 376K N   CA   2891 2892  105.76  139.10                  -40.80\n",
      "  135  16070 376K 377Q C   N    2898 2900 -154.25 -121.10   47.81    1.57  -63.80  171.36   29.79\n",
      "  135        377Q 377Q N   CA   2900 2901  174.15  139.70                  -40.30\n",
      "  136  16071 377Q 378R C   N    2907 2909 -108.43 -125.20   30.12    1.02  -63.00  163.13   19.41\n",
      "  136        378R 378R N   CA   2909 2910  115.58  140.60                  -41.10\n",
      "  137  16072 378R 379K C   N    2918 2920 -140.99 -118.00   24.06    0.81  -62.90 -170.20   29.32\n",
      "  137        379K 379K N   CA   2920 2921  146.21  139.10                  -40.80\n",
      "  138  16073 379K 380Q C   N    2927 2929 -176.00 -121.10   65.97    2.04  -63.80 -177.90   32.28\n",
      "  138        380Q 380Q N   CA   2929 2930  176.27  139.70                  -40.30\n",
      "  139  16074 380Q 381E C   N    2936 2938 -136.36 -117.80   20.40    0.60  -63.60 -170.99   29.30\n",
      "  139        381E 381E N   CA   2938 2939  145.25  136.80                  -40.30\n",
      "  140  16075 381E 382E C   N    2945 2947 -125.40 -117.80   18.89    0.77  -63.60  176.76   27.06\n",
      "  140        382E 382E N   CA   2947 2948  154.10  136.80                  -40.30\n",
      "  141  16077 383I 384G C   N    2962 2964 -146.08 -167.20   30.37    0.69   82.20 -164.64   14.84\n",
      "  141        384G 384G N   CA   2964 2965  152.78  174.60                    8.50\n",
      "  142  16078 384G 385R C   N    2966 2968 -127.92 -125.20   11.04    0.47  -63.00  179.73   28.75\n",
      "  142        385R 385R N   CA   2968 2969  151.30  140.60                  -41.10\n",
      "  143  16079 385R 386L C   N    2977 2979 -106.41 -108.50    7.41    0.38  -63.50  172.03   21.83\n",
      "  143        386L 386L N   CA   2979 2980  125.39  132.50                  -41.20\n",
      "  144  16080 386L 387T C   N    2985 2987 -137.90 -124.80   21.07    0.66  -63.20  174.68   27.27\n",
      "  144        387T 387T N   CA   2987 2988  160.00  143.50                  -42.10\n",
      "  145  16081 387T 388Q C   N    2992 2994  -97.35 -121.10   44.11    1.60  -63.80  146.72   19.97\n",
      "  145        388Q 388Q N   CA   2994 2995  102.53  139.70                  -40.30\n",
      "  146  16082 388Q 389E C   N    3001 3003 -169.96 -117.80   58.13    1.70  -63.60 -170.18   31.09\n",
      "  146        389E 389E N   CA   3003 3004  162.47  136.80                  -40.30\n",
      "  147  16083 389E 390R C   N    3010 3012  -88.39  -72.10   54.73    4.53  -63.00  133.19   16.43\n",
      "  147        390R 390R N   CA   3012 3013   89.65  141.90                  -41.10\n",
      "  148  16084 390R 391D C   N    3021 3023 -130.45  -96.50   33.96    1.39  -63.30  167.68   17.88\n",
      "  148        391D 391D N   CA   3023 3024  113.65  114.20                  -40.00\n",
      "  149  16085 391D 392K C   N    3029 3031  -94.34 -118.00   45.99    1.81  -62.90  143.94   17.23\n",
      "  149        392K 392K N   CA   3031 3032   99.66  139.10                  -40.80\n",
      "  150  16086 392K 393L C   N    3038 3040 -122.72 -108.50   17.11    0.76  -63.50 -173.56   29.55\n",
      "  150        393L 393L N   CA   3040 3041  142.01  132.50                  -41.20\n",
      "  151  16087 393L 394R C   N    3046 3048 -125.59 -125.20    5.40    0.25  -63.00 -176.10   29.16\n",
      "  151        394R 394R N   CA   3048 3049  145.98  140.60                  -41.10\n",
      "  152  16088 394R 395Q C   N    3057 3059 -144.62 -121.10   23.85    0.98  -63.80 -166.32   25.15\n",
      "  152        395Q 395Q N   CA   3059 3060  135.71  139.70                  -40.30\n",
      "  153  16089 395Q 396E C   N    3066 3068  175.25 -117.80   68.57    2.19  -63.60 -152.81   34.16\n",
      "  153        396E 396E N   CA   3068 3069  151.62  136.80                  -40.30\n",
      "  154  16091 397V 398A C   N    3082 3084  168.15 -134.00   90.25    5.40  -68.20  140.91   13.69\n",
      "  154        398A 398A N   CA   3084 3085   77.73  147.00                  145.30\n",
      "  155  16092 398A 399F C   N    3087 3089  -99.97  -63.20   44.37    8.01  -63.20   44.37    8.01\n",
      "  155        399F 399F N   CA   3089 3090  -69.14  -44.30                  -44.30\n",
      "  156  16094 400A 401D C   N    3103 3105 -132.50  -63.30   77.67   13.26  -63.30   77.67   13.26\n",
      "  156        401D 401D N   CA   3105 3106  -75.27  -40.00                  -40.00\n",
      "  157  16097 403V 404H C   N    3124 3126   62.41  -63.20  140.67   18.35  -63.20  140.67   18.35\n",
      "  157        404H 404H N   CA   3126 3127 -105.64  -42.30                  -42.30\n",
      "  158  16098 404H 405D C   N    3134 3136  115.79   54.50  101.66   12.08  -63.30 -118.51   40.49\n",
      "  158        405D 405D N   CA   3136 3137  122.01   40.90                  -40.00\n",
      "  159  16099 405D 406A C   N    3142 3144  152.85 -134.00   77.53    1.81  -62.50 -154.19   41.08\n",
      "  159        406A 406A N   CA   3144 3145  172.69  147.00                  -40.90\n",
      "  160  16102 408D 409K C   N    3166 3168 -111.26  -62.90   78.54   13.03  -62.90   78.54   13.03\n",
      "  160        409K 409K N   CA   3168 3169 -102.68  -40.80                  -40.80\n",
      "  161  16104 410L 411E C   N    3183 3185  -87.66  -63.60   47.83    7.71  -63.60   47.83    7.71\n",
      "  161        411E 411E N   CA   3185 3186  -81.64  -40.30                  -40.30\n",
      "  162  16108 414L 415N C   N    3213 3215  139.38 -119.90  100.96    4.36  -63.20 -127.46   25.67\n",
      "  162        415N 415N N   CA   3215 3216  130.06  137.00                  -41.10\n",
      "  163  16109 415N 416N C   N    3221 3223 -154.90 -119.90   39.50    1.21  -63.20 -172.44   28.57\n",
      "  163        416N 416N N   CA   3223 3224  155.29  137.00                  -41.10\n",
      "  164  16111 417I 418R C   N    3237 3239 -130.63 -125.20   10.57    0.37  -63.00 -177.75   29.27\n",
      "  164        418R 418R N   CA   3239 3240  149.67  140.60                  -41.10\n",
      "  165  16112 418R 419L C   N    3248 3250 -108.42 -108.50   13.46    0.73  -63.50  178.58   27.55\n",
      "  165        419L 419L N   CA   3250 3251  145.96  132.50                  -41.20\n",
      "  166  16114 420S 421N C   N    3262 3264 -157.06 -119.90   47.51    1.38  -63.20  178.89   27.56\n",
      "  166        421N 421N N   CA   3264 3265  166.61  137.00                  -41.10\n",
      "  167  16116 422I 423A C   N    3278 3280 -121.58 -134.00   19.35    0.65  -62.50 -177.14   27.74\n",
      "  167        423A 423A N   CA   3280 3281  132.15  147.00                  -40.90\n",
      "  168  16117 423A 424L C   N    3283 3285 -123.89 -108.50   15.73    0.72  -63.50 -173.02   23.29\n",
      "  168        424L 424L N   CA   3285 3286  135.76  132.50                  -41.20\n",
      "  169  16118 424L 425R C   N    3291 3293 -137.60 -125.20   25.30    1.40  -63.00  176.21   20.11\n",
      "  169        425R 425R N   CA   3293 3294  118.54  140.60                  -41.10\n",
      "  170  16119 425R 426R C   N    3302 3304 -156.65 -125.20   31.64    1.18  -63.00 -161.66   33.13\n",
      "  170        426R 426R N   CA   3304 3305  144.06  140.60                  -41.10\n",
      "  171  16122 428V 429E C   N    3331 3333 -136.12 -117.80   19.27    0.86  -63.60 -174.15   21.75\n",
      "  171        429E 429E N   CA   3333 3334  130.82  136.80                  -40.30\n",
      "  172  16123 429E 430A C   N    3340 3342  177.43 -134.00   49.59    1.30  -62.50 -158.26   39.37\n",
      "  172        430A 430A N   CA   3342 3343  156.99  147.00                  -40.90\n",
      "  173  16124 430A 431A C   N    3345 3347  105.32 -134.00  122.63    3.30  -62.50 -134.69   35.27\n",
      "  173        431A 431A N   CA   3347 3348  168.77  147.00                  -40.90\n",
      "  174  16125 431A 432S C   N    3350 3352  -67.98  -72.40    5.67    0.28  -64.10  176.19   13.00\n",
      "  174        432S 432S N   CA   3352 3353  148.85  152.40                  -35.00\n",
      "  175  16126 432S 433E C   N    3356 3358 -112.96 -117.80    4.87    0.17  -63.60 -176.62   22.12\n",
      "  175        433E 433E N   CA   3358 3359  136.31  136.80                  -40.30\n",
      "  176  16127 433E 434G C   N    3365 3367 -172.63 -167.20   23.14    0.95   82.20  177.99   13.13\n",
      "  176        434G 434G N   CA   3367 3368  152.10  174.60                    8.50\n",
      "  177  16130 436I 437H C   N    3388 3390 -107.05 -125.60   64.48    2.07  -63.20  127.15   13.78\n",
      "  177        437H 437H N   CA   3390 3391   77.05  138.80                  -42.30\n",
      "  178  16131 437H 438S C   N    3398 3400 -115.15 -136.60   25.76    0.79  -64.10  179.34   11.08\n",
      "  178        438S 438S N   CA   3400 3401  136.92  151.20                  -35.00\n",
      "  179  16132 438S 439L C   N    3404 3406 -105.65 -108.50   24.80    1.31  -63.50  154.91   19.53\n",
      "  179        439L 439L N   CA   3406 3407  107.87  132.50                  -41.20\n",
      "  180  16134 440D 441N C   N    3420 3422 -129.35 -119.90   18.22    0.59  -63.20  179.00   26.13\n",
      "  180        441N 441N N   CA   3422 3423  152.58  137.00                  -41.10\n",
      "  181  16135 441N 442K C   N    3428 3430 -114.09 -118.00   78.35    3.75  -62.90  113.81   13.07\n",
      "  181        442K 442K N   CA   3430 3431   60.85  139.10                  -40.80\n",
      "  182  16138 444C 445F C   N    3449 3451 -156.88 -124.20   40.65    2.18  -63.20 -171.62   21.39\n",
      "  182        445F 445F N   CA   3451 3452  119.14  143.30                  -44.30\n",
      "  183  16139 445F 446L C   N    3460 3462 -147.00 -108.50   48.64    2.16  -63.50  177.44   29.79\n",
      "  183        446L 446L N   CA   3462 3463  162.23  132.50                  -41.20\n",
      "  184  16140 446L 447F C   N    3468 3470 -112.72 -124.20   45.25    1.82  -63.20  152.12   18.16\n",
      "  184        447F 447F N   CA   3470 3471   99.53  143.30                  -44.30\n",
      "  185  16141 447F 448F C   N    3479 3481 -118.10 -124.20   89.44    4.06  -63.20  112.66   12.82\n",
      "  185        448F 448F N   CA   3481 3482   54.07  143.30                  -44.30\n",
      "  186  16142 448F 449K C   N    3490 3492   49.37   56.60    7.40    0.78  -62.90  138.42   24.04\n",
      "  186        449K 449K N   CA   3492 3493   40.16   38.60                  -40.80\n",
      "  187  16144 450G 451R C   N    3503 3505  -61.20  -72.10   14.00    0.91  -63.00  174.23   23.66\n",
      "  187        451R 451R N   CA   3505 3506  133.12  141.90                  -41.10\n",
      "  188  16145 451R 452C C   N    3514 3516  -64.68  -63.00    9.88    1.33 -117.90  176.29    7.60\n",
      "  188        452C 452C N   CA   3516 3517  -50.83  -41.10                  141.10\n",
      "  189  16146 452C 453S C   N    3520 3522  170.56 -136.60   62.21    1.89  -64.10 -171.37   22.68\n",
      "  189        453S 453S N   CA   3522 3523 -175.97  151.20                  -35.00\n",
      "  190  16148 454I 455T C   N    3534 3536  -67.10  -78.10   13.18    0.87  -63.20  160.88   20.89\n",
      "  190        455T 455T N   CA   3536 3537  157.06  149.80                  -42.10\n",
      "  191  16149 455T 456L C   N    3541 3543 -147.21 -108.50   38.75    1.82  -63.50 -165.50   23.86\n",
      "  191        456L 456L N   CA   3543 3544  134.37  132.50                  -41.20\n",
      "  192  16153 459V 460L C   N    3571 3573  113.39 -108.50  162.47    7.17  -63.50 -156.44   29.04\n",
      "  192        460L 460L N   CA   3573 3574 -141.94  132.50                  -41.20\n",
      "  193  16155 461R 462G C   N    3590 3592  -56.79 -167.20  114.71    1.68   78.70  144.56    8.06\n",
      "  193        462G 462G N   CA   3592 3593  143.50  174.60                 -166.10\n",
      "  194  16177 483L 484H C   N    3759 3761   56.27   56.30    4.23    0.37  -63.20  147.98   25.30\n",
      "  194        484H 484H N   CA   3761 3762   45.03   40.80                  -42.30\n",
      "  195  16179 485I 486Q C   N    3777 3779 -101.06 -121.10   20.22    0.71  -63.80 -178.81   24.83\n",
      "  195        486Q 486Q N   CA   3779 3780  137.01  139.70                  -40.30\n",
      "  196  16180 486Q 487L C   N    3786 3788 -105.89 -108.50   13.78    0.71  -63.50  165.69   20.98\n",
      "  196        487L 487L N   CA   3788 3789  118.97  132.50                  -41.20\n",
      "  197  16181 487L 488F C   N    3794 3796 -146.67 -124.20   24.53    0.69  -63.20 -177.27   30.72\n",
      "  197        488F 488F N   CA   3796 3797  153.15  143.30                  -44.30\n",
      "  198  16184 490C 491E C   N    3818 3820 -105.26 -117.80   17.86    0.58  -63.60  169.59   20.61\n",
      "  198        491E 491E N   CA   3820 3821  124.09  136.80                  -40.30\n",
      "  199  16185 491E 492H C   N    3827 3829 -143.53 -125.60   22.72    0.56  -63.20 -176.53   28.16\n",
      "  199        492H 492H N   CA   3829 3830  152.75  138.80                  -42.30\n",
      "  200  16186 492H 493H C   N    3837 3839 -111.80 -125.60   15.67    0.41  -63.20 -179.65   20.00\n",
      "  200        493H 493H N   CA   3839 3840  131.38  138.80                  -42.30\n",
      "  201  16187 493H 494A C   N    3847 3849 -102.67 -134.00   34.61    0.79  -62.50  177.77   27.45\n",
      "  201        494A 494A N   CA   3849 3850  132.28  147.00                  -40.90\n",
      "  202  16189 495N 496E C   N    3860 3862 -136.67 -117.80   30.51    1.05  -63.60  174.92   27.42\n",
      "  202        496E 496E N   CA   3862 3863  160.77  136.80                  -40.30\n",
      "  203  16191 497R 498N C   N    3880 3882  -61.50  -71.20   30.61    2.23  -63.20  147.08   18.30\n",
      "  203        498N 498N N   CA   3882 3883  171.83  142.80                  -41.10\n",
      "  204  16192 498N 499D C   N    3888 3890   53.10   54.50   11.43    0.73  -63.30  148.51   25.11\n",
      "  204        499D 499D N   CA   3890 3891   52.24   40.90                  -40.00\n",
      "  205  16194 500P 501M C   N    3903 3905  -88.11  -73.00   28.85    2.22  -63.40  160.83   22.20\n",
      "  205        501M 501M N   CA   3905 3906  118.42  143.00                  -40.50\n",
      "  206  16195 501M 502Q C   N    3911 3913 -163.38 -121.10   42.30    1.62  -63.80 -155.42   26.31\n",
      "  206        502Q 502Q N   CA   3913 3914  138.40  139.70                  -40.30\n",
      "  207  16196 502Q 503F C   N    3920 3922 -141.59 -124.20   36.12    1.99  -63.20  174.54   20.02\n",
      "  207        503F 503F N   CA   3922 3923  111.64  143.30                  -44.30\n",
      "  208  16205 511C 512L C   N    3994 3996 -113.87 -108.50   49.83    2.63  -63.50  145.74   23.35\n",
      "  208        512L 512L N   CA   3996 3997 -177.96  132.50                  -41.20\n",
      "  209  16206 512L 513H C   N    4002 4004 -123.75 -125.60   33.39    1.30  -63.20  157.65   23.67\n",
      "  209        513H 513H N   CA   4004 4005  172.14  138.80                  -42.30\n",
      "  210  16209 515V 516G C   N    4026 4028  -73.12  -80.20   20.99    0.52   82.20 -146.94    6.91\n",
      "  210        516G 516G N   CA   4028 4029  154.35  174.10                    8.50\n",
      "  211  16210 516G 517A C   N    4030 4032  -68.99  -68.20    8.12    0.68  -62.50  178.24   28.90\n",
      "  211        517A 517A N   CA   4032 4033  137.22  145.30                  -40.90\n",
      "  212  16211 517A 518S C   N    4035 4037 -157.75 -136.60   23.01    0.72  -64.10 -170.50   20.47\n",
      "  212        518S 518S N   CA   4037 4038  160.26  151.20                  -35.00\n",
      "  213  16212 518S 519G C   N    4041 4043 -166.36 -167.20   33.05    1.29   82.20  173.56   13.05\n",
      "  213        519G 519G N   CA   4043 4044  141.56  174.60                    8.50\n",
      "  214  16215 521P 522K C   N    4060 4062 -136.83 -118.00   20.30    0.68  -62.90 -172.30   28.80\n",
      "  214        522K 522K N   CA   4062 4063  146.67  139.10                  -40.80\n",
      "  215  16216 522K 523A C   N    4069 4071 -129.03 -134.00    5.61    0.13  -62.50 -173.05   34.20\n",
      "  215        523A 523A N   CA   4071 4072  144.39  147.00                  -40.90\n",
      "  216  16217 523A 524F C   N    4074 4076 -126.30 -124.20    6.58    0.25  -63.20  177.74   28.84\n",
      "  216        524F 524F N   CA   4076 4077  149.54  143.30                  -44.30\n",
      "  217  16219 525V 526A C   N    4092 4094  171.60 -134.00   58.51    1.34  -62.50 -163.72   38.71\n",
      "  217        526A 526A N   CA   4094 4095  168.53  147.00                  -40.90\n",
      "  218  16220 526A 527H C   N    4097 4099  -87.48  -67.60   49.11    4.27  -63.20  139.53   16.07\n",
      "  218        527H 527H N   CA   4099 4100   95.10  140.00                  -42.30\n",
      "  219  16221 527H 528T C   N    4107 4109 -126.34 -124.80   14.99    0.60  -63.20  171.54   26.16\n",
      "  219        528T 528T N   CA   4109 4110  158.41  143.50                  -42.10\n",
      "  220  16222 528T 529F C   N    4114 4116  -77.05  -71.40   13.13    0.88  -63.20  163.73   23.67\n",
      "  220        529F 529F N   CA   4116 4117  152.55  140.70                  -44.30\n",
      "  221  16224 530C 531K C   N    4131 4133   60.61   56.60    5.73    0.36  -62.90  144.65   25.24\n",
      "  221        531K 531K N   CA   4133 4134   34.51   38.60                  -40.80\n",
      "  222  16225 531K 532F C   N    4140 4142  -65.68  -71.40    5.96    0.46  -63.20  173.36   24.24\n",
      "  222        532F 532F N   CA   4142 4143  142.36  140.70                  -44.30\n",
      "  223  16226 532F 533T C   N    4151 4153  -88.18  -78.10   53.32    2.62  -63.20  141.76   16.65\n",
      "  223        533T 533T N   CA   4153 4154   97.44  149.80                  -42.10\n",
      "  224  16228 534L 535L C   N    4166 4168  -62.60  -70.70   13.40    0.83  -63.50  172.12   23.95\n",
      "  224        535L 535L N   CA   4168 4169  130.92  141.60                  -41.20\n",
      "  225  16229 535L 536H C   N    4174 4176 -147.56 -125.60   22.69    0.68  -63.20 -167.36   29.57\n",
      "  225        536H 536H N   CA   4176 4177  144.51  138.80                  -42.30\n",
      "  226  16232 538R 539E C   N    4203 4205  149.05 -117.80  119.53    3.64  -63.60 -177.31   31.38\n",
      "  226        539E 539E N   CA   4205 4206 -148.29  136.80                  -40.30\n",
      "  227  16233 539E 540E C   N    4212 4214  -63.48  -69.30   34.43    2.82  -63.60  143.26   19.10\n",
      "  227        540E 540E N   CA   4214 4215  176.44  142.50                  -40.30\n",
      "  228  16234 540E 541R C   N    4221 4223 -131.01 -125.20   49.92    2.28  -63.00  145.59   24.27\n",
      "  228        541R 541R N   CA   4223 4224 -169.83  140.60                  -41.10\n",
      "  229  16235 541R 542Y C   N    4232 4234  -80.16  -98.40   23.59    1.66  -63.50  174.04   28.47\n",
      "  229        542Y 542Y N   CA   4234 4235  143.35  128.40                  -43.40\n",
      "  230  16236 542Y 543F C   N    4244 4246 -153.92 -124.20   34.68    1.81  -63.20 -167.54   21.96\n",
      "  230        543F 543F N   CA   4246 4247  125.43  143.30                  -44.30\n",
      "  231  16237 543F 544T C   N    4255 4257 -154.05 -124.80   31.13    1.34  -63.20 -172.74   29.86\n",
      "  231        544T 544T N   CA   4257 4258  154.16  143.50                  -42.10\n",
      "  232  16238 544T 545T C   N    4262 4264 -139.64 -124.80   26.38    0.82  -63.20  170.68   26.85\n",
      "  232        545T 545T N   CA   4264 4265  165.30  143.50                  -42.10\n",
      "  233  16239 545T 546S C   N    4269 4271 -141.87 -136.60   11.06    0.63  -64.10 -167.15   12.05\n",
      "  233        546S 546S N   CA   4271 4272  141.47  151.20                  -35.00\n",
      "  234  16240 546S 547T C   N    4275 4277 -129.12 -124.80   41.50    1.91  -63.20  158.67   17.01\n",
      "  234        547T 547T N   CA   4277 4278  102.23  143.50                  -42.10\n",
      "  235  16242 548A 549E C   N    4287 4289  -53.98  -69.30   15.48    1.25  -63.60  175.26   22.86\n",
      "  235        549E 549E N   CA   4289 4290  144.70  142.50                  -40.30\n",
      "  236  16243 549E 550N C   N    4296 4298 -116.29 -119.90   16.46    0.82  -63.20  174.13   24.79\n",
      "  236        550N 550N N   CA   4298 4299  153.06  137.00                  -41.10\n",
      "  237  16244 550N 551T C   N    4304 4306  -89.26  -78.10   29.91    1.00  -63.20  142.75   20.04\n",
      "  237        551T 551T N   CA   4306 4307  177.55  149.80                  -42.10\n",
      "  238  16245 551T 552Q C   N    4311 4313  -72.47  -73.00   61.58    4.04  -63.80  117.74   17.77\n",
      "  238        552Q 552Q N   CA   4313 4314 -157.72  140.70                  -40.30\n",
      "  239  16246 552Q 553N C   N    4320 4322  -51.70  -71.20   19.83    1.52  -63.20  172.90   21.00\n",
      "  239        553N 553N N   CA   4322 4323  146.38  142.80                  -41.10\n",
      "  240  16248 554P 555R C   N    4335 4337 -126.67 -125.20   42.97    2.04  -63.00  149.58   24.55\n",
      "  240        555R 555R N   CA   4337 4338 -176.45  140.60                  -41.10\n",
      "  241  16249 555R 556W C   N    4346 4348  -61.05  -71.30   10.28    0.71  -63.00  177.62   22.09\n",
      "  241        556W 556W N   CA   4348 4349  138.19  139.00                  -44.20\n",
      "  242  16250 556W 557G C   N    4360 4362  145.12 -167.20   48.80    1.20   82.20  167.90   11.09\n",
      "  242        557G 557G N   CA   4362 4363  164.16  174.60                    8.50\n",
      "  243  16251 557G 558Y C   N    4364 4366 -113.74  -98.40   18.84    1.24  -63.50  168.51   24.40\n",
      "  243        558Y 558Y N   CA   4366 4367  117.45  128.40                  -43.40\n",
      "  244  16252 558Y 559V C   N    4376 4378  -65.36  -62.40    8.10    1.21 -125.40  177.23    9.94\n",
      "  244        559V 559V N   CA   4378 4379  -49.94  -42.40                  143.30\n",
      "  245  16253 559V 560K C   N    4383 4385 -150.61 -118.00   45.49    1.60  -62.90  172.36   27.61\n",
      "  245        560K 560K N   CA   4385 4386  170.82  139.10                  -40.80\n",
      "  246  16255 561V 562F C   N    4399 4401  -80.76  -71.40   13.51    1.19  -63.20  176.15   23.30\n",
      "  246        562F 562F N   CA   4401 4402  130.97  140.70                  -44.30\n",
      "  247  16256 562F 563E C   N    4410 4412  -89.25  -69.30   31.47    1.77  -63.60  154.99   22.11\n",
      "  247        563E 563E N   CA   4412 4413  166.84  142.50                  -40.30\n",
      "  248  16259 565P 566E C   N    4434 4436  149.62 -117.80  103.85    5.00  -63.60 -163.90   23.80\n",
      "  248        566E 566E N   CA   4436 4437   89.74  136.80                  -40.30\n",
      "  249  16260 566E 567L C   N    4443 4445 -158.55 -108.50   59.75    2.64  -63.50 -179.31   30.92\n",
      "  249        567L 567L N   CA   4445 4446  165.14  132.50                  -41.20\n",
      "  250  16261 567L 568T C   N    4451 4453 -103.58  -78.10   43.60    1.46  -63.20  138.73   20.48\n",
      "  250        568T 568T N   CA   4453 4454 -174.82  149.80                  -42.10\n",
      "  251  16269 575F 576C C   N    4515 4517   63.47   57.40   17.25    0.55 -117.90 -144.10    9.25\n",
      "  251        576C 576C N   CA   4517 4518   19.85   36.00                  141.10\n",
      "  252  16270 576C 577E C   N    4521 4523  -66.44  -69.30   11.90    0.81  -63.60  171.27   22.68\n",
      "  252        577E 577E N   CA   4523 4524  130.94  142.50                  -40.30\n",
      "  253  16271 577E 578H C   N    4530 4532 -165.37 -125.60   48.77    1.21  -63.20 -177.96   29.26\n",
      "  253        578H 578H N   CA   4532 4533  167.03  138.80                  -42.30\n",
      "  254  16272 578H 579T C   N    4540 4542 -116.06 -124.80    9.08    0.53  -63.20  179.87   26.58\n",
      "  254        579T 579T N   CA   4542 4543  145.97  143.50                  -42.10\n",
      "  255  16273 579T 580V C   N    4547 4549  -65.46  -62.40    4.08    0.69 -125.40 -178.23   10.17\n",
      "  255        580V 580V N   CA   4549 4550  -45.09  -42.40                  143.30\n",
      "  256  16274 580V 581F C   N    4554 4556  -63.48  -71.40   19.65    1.33  -63.20  167.02   23.17\n",
      "  256        581F 581F N   CA   4556 4557  122.72  140.70                  -44.30\n",
      "  257  16275 581F 582A C   N    4565 4567  167.04 -134.00   72.40    1.81  -62.50 -175.77   36.81\n",
      "  257        582A 582A N   CA   4567 4568 -170.98  147.00                  -40.90\n",
      "  258  16276 582A 583F C   N    4570 4572  -71.29  -71.40   28.39    2.29  -63.20  146.83   20.93\n",
      "  258        583F 583F N   CA   4572 4573  169.09  140.70                  -44.30\n",
      "  259  16279 585V 586F C   N    4597 4599 -131.95 -124.20   22.11    0.82  -63.20  166.54   27.59\n",
      "  259        586F 586F N   CA   4599 4600  164.01  143.30                  -44.30\n",
      "  260  16280 586F 587A C   N    4608 4610 -106.68 -134.00   33.15    0.82  -62.50  174.80   26.84\n",
      "  260        587A 587A N   CA   4610 4611  128.22  147.00                  -40.90\n",
      "  261  16281 587A 588F C   N    4613 4615 -156.60 -124.20   40.85    1.06  -63.20  174.59   30.06\n",
      "  261        588F 588F N   CA   4615 4616  168.19  143.30                  -44.30\n",
      "  262  16282 588F 589N C   N    4624 4626 -158.24 -119.90   79.89    2.66  -63.20  146.75   23.40\n",
      "  262        589N 589N N   CA   4626 4627 -152.91  137.00                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33487  51F 537D CD2 CB    370 4188   13.56    4.19    9.37    5.07    4.19    9.37    5.07\n",
      "    2  33488  51F 537D CD2 CG    370 4189   12.80    4.31    8.49    4.56    4.31    8.49    4.56\n",
      "    3  33489  51F 537D CD2 OD2   370 4191   11.79    3.71    8.09    4.52    3.71    8.09    4.52\n",
      "    4  33497  51F 537D CE2 CB    372 4188   14.60    4.68    9.92    5.19    4.68    9.92    5.19\n",
      "    5  33498  51F 537D CE2 CG    372 4189   13.77    4.91    8.86    4.57    4.91    8.86    4.57\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   90  161  568  490  774  566  628  771  879  933\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00200.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      590     4639     4639\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00200.B99990001.pdb        7219.51172   -41066.57422        0.21725\n",
      "EAO00200.B99990002.pdb        6887.04492   -41338.19922        0.36672\n",
      "EAO00200.B99990003.pdb        6516.79297   -43080.46094        0.51103\n",
      "EAO00200.B99990004.pdb        6572.15137   -41755.18750        0.32041\n",
      "EAO00200.B99990005.pdb        6858.95117   -40617.16406        0.36892\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69668734   68035.875    66.441\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69669262   68036.391    66.442\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69717862   68083.852    66.488\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69731162   68096.836    66.501\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70000428   68359.797    66.758\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     73561551   71837.453    70.154\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73563793   71839.641    70.156\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73565243   71841.055    70.157\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73568143   71843.891    70.160\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73573943   71849.555    70.166\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     87012342   84972.992    82.981\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87023942   84984.320    82.993\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87047142   85006.977    83.015\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87093542   85052.289    83.059\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87186342   85142.914    83.147\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     93905542   91704.633    89.555\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00201.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70002748   68362.062    66.760\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70004198   68363.477    66.761\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70007098   68366.305    66.764\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70012898   68371.969    66.770\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70014378   68373.414    66.771\n",
      "\n",
      "Read the alignment from file       : ali/EAO00201.ali\n",
      "\n",
      "Total number of alignment positions:   371\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00201     371      1    EAO00201\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     70016195   68375.188    66.773\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70341370   68692.742    67.083\n",
      "profile_iteration_> processing sequence:       1    371      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70342448   68693.797    67.084\n",
      "profile_iteration_> processing sequence:    6720    371      1     3.5937500     0.0005348     0.0000021  1870\n",
      "profile_iteration_> processing sequence:   13439    371      1     7.0625000     0.0005255     0.0000021  1903\n",
      "profile_iteration_> processing sequence:   20158    371      1    10.5156250     0.0005217     0.0000020  1917\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70342823   68694.164    67.084\n",
      "profile_iteration_> processing sequence:   26877    371      1    14.0312500     0.0005221     0.0000021  1916\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70343059   68694.391    67.084\n",
      "profile_iteration_> processing sequence:   33596    371      1    17.5312500     0.0005218     0.0000020  1916\n",
      "profile_iteration_> processing sequence:   40315    371      1    21.0000000     0.0005209     0.0000021  1920\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70346009   68697.273    67.087\n",
      "profile_iteration_> processing sequence:   47034    371      1    24.5312500     0.0005216     0.0000021  1917\n",
      "profile_iteration_> processing sequence:   53753    371      1    28.1250000     0.0005232     0.0000021  1911\n",
      "profile_iteration_> processing sequence:   60472    371      1    31.6093750     0.0005227     0.0000020  1913\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70349519   68700.703    67.091\n",
      "profile_iteration_> processing sequence:   67191    371      1    35.2500000     0.0005246     0.0000020  1906\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    37.61182\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    230.4766\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0214,      121,        0.0076,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00045   0.00000\n",
      "Z:         1  -2.55000   0.00030   0.00001\n",
      "Z:         1  -2.45000   0.00015   0.00004\n",
      "Z:         1  -2.35000   0.00134   0.00016\n",
      "Z:         1  -2.25000   0.00298   0.00055\n",
      "Z:         1  -2.15000   0.00298   0.00163\n",
      "Z:         1  -2.05000   0.00685   0.00416\n",
      "Z:         1  -1.95000   0.01235   0.00933\n",
      "Z:         1  -1.85000   0.02783   0.01872\n",
      "Z:         1  -1.75000   0.03825   0.03400\n",
      "Z:         1  -1.65000   0.06519   0.05658\n",
      "Z:         1  -1.55000   0.09272   0.08721\n",
      "Z:         1  -1.45000   0.12516   0.12565\n",
      "Z:         1  -1.35000   0.17234   0.17059\n",
      "Z:         1  -1.25000   0.22607   0.21981\n",
      "Z:         1  -1.15000   0.28009   0.27052\n",
      "Z:         1  -1.05000   0.29498   0.31973\n",
      "Z:         1  -0.95000   0.35049   0.36469\n",
      "Z:         1  -0.85000   0.37653   0.40316\n",
      "Z:         1  -0.75000   0.42207   0.43360\n",
      "Z:         1  -0.65000   0.45288   0.45519\n",
      "Z:         1  -0.55000   0.45452   0.46779\n",
      "Z:         1  -0.45000   0.51450   0.47182\n",
      "Z:         1  -0.35000   0.45243   0.46811\n",
      "Z:         1  -0.25000   0.42624   0.45775\n",
      "Z:         1  -0.15000   0.43517   0.44196\n",
      "Z:         1  -0.05000   0.40109   0.42194\n",
      "Z:         1   0.05000   0.41493   0.39889\n",
      "Z:         1   0.15000   0.36984   0.37383\n",
      "Z:         1   0.25000   0.34260   0.34769\n",
      "Z:         1   0.35000   0.31373   0.32121\n",
      "Z:         1   0.45000   0.30911   0.29500\n",
      "Z:         1   0.55000   0.26432   0.26953\n",
      "Z:         1   0.65000   0.24571   0.24513\n",
      "Z:         1   0.75000   0.23887   0.22205\n",
      "Z:         1   0.85000   0.21104   0.20043\n",
      "Z:         1   0.95000   0.18157   0.18036\n",
      "Z:         1   1.05000   0.15925   0.16185\n",
      "Z:         1   1.15000   0.15091   0.14489\n",
      "Z:         1   1.25000   0.12903   0.12944\n",
      "Z:         1   1.35000   0.12100   0.11542\n",
      "Z:         1   1.45000   0.10165   0.10275\n",
      "Z:         1   1.55000   0.08974   0.09133\n",
      "Z:         1   1.65000   0.08081   0.08109\n",
      "Z:         1   1.75000   0.07724   0.07191\n",
      "Z:         1   1.85000   0.06385   0.06371\n",
      "Z:         1   1.95000   0.06340   0.05639\n",
      "Z:         1   2.05000   0.05179   0.04988\n",
      "Z:         1   2.15000   0.04078   0.04409\n",
      "Z:         1   2.25000   0.04242   0.03895\n",
      "Z:         1   2.35000   0.03215   0.03439\n",
      "Z:         1   2.45000   0.03185   0.03035\n",
      "Z:         1   2.55000   0.02724   0.02678\n",
      "Z:         1   2.65000   0.02575   0.02361\n",
      "Z:         1   2.75000   0.02143   0.02082\n",
      "Z:         1   2.85000   0.01890   0.01835\n",
      "Z:         1   2.95000   0.01875   0.01617\n",
      "Z:         1   3.05000   0.01369   0.01424\n",
      "Z:         1   3.15000   0.01220   0.01255\n",
      "Z:         1   3.25000   0.00982   0.01105\n",
      "Z:         1   3.35000   0.01116   0.00973\n",
      "Z:         1   3.45000   0.00923   0.00857\n",
      "Z:         1   3.55000   0.00714   0.00754\n",
      "Z:         1   3.65000   0.00789   0.00664\n",
      "Z:         1   3.75000   0.00699   0.00584\n",
      "Z:         1   3.85000   0.00417   0.00514\n",
      "Z:         1   3.95000   0.00357   0.00453\n",
      "Z:         1   4.05000   0.00432   0.00398\n",
      "Z:         1   4.15000   0.00298   0.00350\n",
      "Z:         1   4.25000   0.00446   0.00308\n",
      "Z:         1   4.35000   0.00342   0.00271\n",
      "Z:         1   4.45000   0.00208   0.00239\n",
      "Z:         1   4.55000   0.00223   0.00210\n",
      "Z:         1   4.65000   0.00149   0.00185\n",
      "Z:         1   4.75000   0.00119   0.00163\n",
      "Z:         1   4.85000   0.00253   0.00143\n",
      "Z:         1   4.95000   0.00149   0.00126\n",
      "Z:         1   5.05000   0.00208   0.00111\n",
      "Z:         1   5.15000   0.00089   0.00097\n",
      "Z:         1   5.25000   0.00119   0.00086\n",
      "Z:         1   5.35000   0.00119   0.00075\n",
      "Z:         1   5.45000   0.00074   0.00066\n",
      "Z:         1   5.55000   0.00089   0.00058\n",
      "Z:         1   5.65000   0.00045   0.00051\n",
      "Z:         1   5.75000   0.00089   0.00045\n",
      "Z:         1   5.85000   0.00030   0.00040\n",
      "Z:         1   5.95000   0.00089   0.00035\n",
      "Z:         1   6.05000   0.00015   0.00031\n",
      "Z:         1   6.15000   0.00000   0.00027\n",
      "Z:         1   6.25000   0.00030   0.00024\n",
      "Z:         1   6.35000   0.00000   0.00021\n",
      "Z:         1   6.45000   0.00015   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00000   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00030   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00015   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00000   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00000   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     70663319   69007.148    67.390\n",
      "> 3qleA                      1    2018    5750     182     371   27.34    0.28E-05       2   116   156   283     8   135\n",
      "> 2ghtA                      1    5124    7550     180     371   31.13    0.12E-09       3   131   153   293     2   152\n",
      "> 2hhlA                      1   19478    8000     180     371   31.08    0.75E-11       4   128   153   281    12   159\n",
      "> 2hhlD                      1   19479    8000     172     371   31.08    0.75E-11       5   128   153   281     3   150\n",
      "> 2q5eA                      1   21212    7650     181     371   30.14    0.75E-10       6   126   155   281     2   147\n",
      "> 4qqfA                      1   45603    5850     198     371   25.56    0.18E-05       7   121   156   283    23   155\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.015625\n",
      "profile_iteration__> Score Collection        [sec]:    35.234375\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000524\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1908.694214\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.015625\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000526\n",
      "profile_iteration__> Overall performance [seq/sec]:  1901.099854\n",
      "openf___224_> Open           buildprofile/EAO00201build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70365148   68715.969    67.105\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70366628   68717.406    67.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70367088   68717.859    67.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70367608   68718.367    67.108\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70368116   68718.859    67.108\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70368624   68719.359    67.109\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70369124   68719.844    67.109\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70369604   68720.312    67.110\n",
      "openf___224_> Open           buildprofile/EAO00201build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69668734   68035.875    66.441\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69669262   68036.391    66.442\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69717862   68083.852    66.488\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69731162   68096.836    66.501\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70000428   68359.797    66.758\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     73561551   71837.453    70.154\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73563793   71839.641    70.156\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73565243   71841.055    70.157\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73568143   71843.891    70.160\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     73573943   71849.555    70.166\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     87012342   84972.992    82.981\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87023942   84984.320    82.993\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87047142   85006.977    83.015\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87093542   85052.289    83.059\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     87186342   85142.914    83.147\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     93905542   91704.633    89.555\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00202.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70002748   68362.062    66.760\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70004198   68363.477    66.761\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70007098   68366.305    66.764\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70012898   68371.969    66.770\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70024498   68383.297    66.781\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70026434   68385.188    66.782\n",
      "\n",
      "Read the alignment from file       : ali/EAO00202.ali\n",
      "\n",
      "Total number of alignment positions:   485\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00202     485      1    EAO00202\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     70028707   68387.406    66.785\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70353882   68704.961    67.095\n",
      "profile_iteration_> processing sequence:       1    485      1     0.0156250     0.0156250     0.0156250     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70354960   68706.016    67.096\n",
      "profile_iteration_> processing sequence:    6720    485      1     4.7187500     0.0007022     0.0000027  1424\n",
      "profile_iteration_> processing sequence:   13439    485      1     9.2812500     0.0006906     0.0000027  1448\n",
      "profile_iteration_> processing sequence:   20158    485      1    13.8281250     0.0006860     0.0000027  1458\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70355335   68706.383    67.096\n",
      "profile_iteration_> processing sequence:   26877    485      1    18.4375000     0.0006860     0.0000027  1458\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70355571   68706.609    67.096\n",
      "profile_iteration_> processing sequence:   33596    485      1    23.0468750     0.0006860     0.0000027  1458\n",
      "profile_iteration_> processing sequence:   40315    485      1    27.5937500     0.0006845     0.0000027  1461\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70358521   68709.492    67.099\n",
      "profile_iteration_> processing sequence:   47034    485      1    32.2343750     0.0006853     0.0000027  1459\n",
      "profile_iteration_> processing sequence:   53753    485      1    36.9687500     0.0006878     0.0000027  1454\n",
      "profile_iteration_> processing sequence:   60472    485      1    41.5468750     0.0006870     0.0000027  1456\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     70362031   68712.922    67.102\n",
      "profile_iteration_> processing sequence:   67191    485      1    46.3281250     0.0006895     0.0000027  1450\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    34.09155\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    250.2159\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0113,      121,        0.0048,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00015   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00015   0.00016\n",
      "Z:         1  -2.25000   0.00149   0.00055\n",
      "Z:         1  -2.15000   0.00149   0.00163\n",
      "Z:         1  -2.05000   0.00685   0.00416\n",
      "Z:         1  -1.95000   0.00997   0.00933\n",
      "Z:         1  -1.85000   0.02024   0.01872\n",
      "Z:         1  -1.75000   0.03944   0.03400\n",
      "Z:         1  -1.65000   0.05432   0.05658\n",
      "Z:         1  -1.55000   0.08647   0.08721\n",
      "Z:         1  -1.45000   0.12516   0.12565\n",
      "Z:         1  -1.35000   0.18574   0.17059\n",
      "Z:         1  -1.25000   0.20419   0.21981\n",
      "Z:         1  -1.15000   0.27429   0.27052\n",
      "Z:         1  -1.05000   0.30748   0.31973\n",
      "Z:         1  -0.95000   0.35838   0.36469\n",
      "Z:         1  -0.85000   0.40407   0.40316\n",
      "Z:         1  -0.75000   0.44068   0.43360\n",
      "Z:         1  -0.65000   0.43606   0.45519\n",
      "Z:         1  -0.55000   0.46315   0.46779\n",
      "Z:         1  -0.45000   0.47416   0.47182\n",
      "Z:         1  -0.35000   0.47000   0.46811\n",
      "Z:         1  -0.25000   0.44604   0.45775\n",
      "Z:         1  -0.15000   0.44410   0.44196\n",
      "Z:         1  -0.05000   0.41687   0.42194\n",
      "Z:         1   0.05000   0.40407   0.39889\n",
      "Z:         1   0.15000   0.37147   0.37383\n",
      "Z:         1   0.25000   0.35451   0.34769\n",
      "Z:         1   0.35000   0.31551   0.32121\n",
      "Z:         1   0.45000   0.29974   0.29500\n",
      "Z:         1   0.55000   0.27235   0.26953\n",
      "Z:         1   0.65000   0.23693   0.24513\n",
      "Z:         1   0.75000   0.21863   0.22205\n",
      "Z:         1   0.85000   0.20196   0.20043\n",
      "Z:         1   0.95000   0.17264   0.18036\n",
      "Z:         1   1.05000   0.16133   0.16185\n",
      "Z:         1   1.15000   0.14540   0.14489\n",
      "Z:         1   1.25000   0.12338   0.12944\n",
      "Z:         1   1.35000   0.11579   0.11542\n",
      "Z:         1   1.45000   0.10850   0.10275\n",
      "Z:         1   1.55000   0.09570   0.09133\n",
      "Z:         1   1.65000   0.08275   0.08109\n",
      "Z:         1   1.75000   0.07501   0.07191\n",
      "Z:         1   1.85000   0.06281   0.06371\n",
      "Z:         1   1.95000   0.05864   0.05639\n",
      "Z:         1   2.05000   0.04807   0.04988\n",
      "Z:         1   2.15000   0.04539   0.04409\n",
      "Z:         1   2.25000   0.04152   0.03895\n",
      "Z:         1   2.35000   0.03378   0.03439\n",
      "Z:         1   2.45000   0.03349   0.03035\n",
      "Z:         1   2.55000   0.03021   0.02678\n",
      "Z:         1   2.65000   0.02337   0.02361\n",
      "Z:         1   2.75000   0.02262   0.02082\n",
      "Z:         1   2.85000   0.01890   0.01835\n",
      "Z:         1   2.95000   0.02128   0.01617\n",
      "Z:         1   3.05000   0.01176   0.01424\n",
      "Z:         1   3.15000   0.01429   0.01255\n",
      "Z:         1   3.25000   0.01310   0.01105\n",
      "Z:         1   3.35000   0.01176   0.00973\n",
      "Z:         1   3.45000   0.01027   0.00857\n",
      "Z:         1   3.55000   0.00804   0.00754\n",
      "Z:         1   3.65000   0.00833   0.00664\n",
      "Z:         1   3.75000   0.00670   0.00584\n",
      "Z:         1   3.85000   0.00536   0.00514\n",
      "Z:         1   3.95000   0.00387   0.00453\n",
      "Z:         1   4.05000   0.00461   0.00398\n",
      "Z:         1   4.15000   0.00253   0.00350\n",
      "Z:         1   4.25000   0.00491   0.00308\n",
      "Z:         1   4.35000   0.00298   0.00271\n",
      "Z:         1   4.45000   0.00104   0.00239\n",
      "Z:         1   4.55000   0.00149   0.00210\n",
      "Z:         1   4.65000   0.00283   0.00185\n",
      "Z:         1   4.75000   0.00223   0.00163\n",
      "Z:         1   4.85000   0.00179   0.00143\n",
      "Z:         1   4.95000   0.00119   0.00126\n",
      "Z:         1   5.05000   0.00149   0.00111\n",
      "Z:         1   5.15000   0.00074   0.00097\n",
      "Z:         1   5.25000   0.00089   0.00086\n",
      "Z:         1   5.35000   0.00074   0.00075\n",
      "Z:         1   5.45000   0.00030   0.00066\n",
      "Z:         1   5.55000   0.00074   0.00058\n",
      "Z:         1   5.65000   0.00015   0.00051\n",
      "Z:         1   5.75000   0.00074   0.00045\n",
      "Z:         1   5.85000   0.00045   0.00040\n",
      "Z:         1   5.95000   0.00030   0.00035\n",
      "Z:         1   6.05000   0.00060   0.00031\n",
      "Z:         1   6.15000   0.00074   0.00027\n",
      "Z:         1   6.25000   0.00000   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00030   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00015   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00000   0.00011\n",
      "Z:         1   6.95000   0.00015   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00015   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00015   0.00006\n",
      "Z:         1   7.45000   0.00030   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00000   0.00004\n",
      "Z:         1   7.75000   0.00000   0.00003\n",
      "Z:         1   7.85000   0.00015   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     70767031   69108.430    67.489\n",
      "> 4xyjA                      1    2404   11650     768     485   27.53     0.0           2   323    90   438     7   362\n",
      "> 4xyjH                      1    2405   11500     739     485   27.87     0.0           3   315    98   438     7   354\n",
      "> 1pfkA                      1    5070   12850     320     485   32.88     0.0           4   213    98   323     5   223\n",
      "> 2pfkA                      1    5112   12850     301     485   32.88     0.0           5   213    98   323     5   223\n",
      "> 2f48A                      1    8746   11700     551     485   30.96     0.0           6   222    98   324    72   310\n",
      "> 2higA                      1   18382   91350     440     485   79.91     0.0           7   438     8   452     2   439\n",
      "> 2higB                      1   18383   86300     422     485   80.48     0.0           8   420     8   452     2   421\n",
      "> 6qu3A                      1   23836   93650     464     485   78.02     0.0           9   464    17   484     1   464\n",
      "> 6qu3B                      1   23837   89150     446     485   78.25     0.0          10   446    19   483     1   446\n",
      "> 6qu4B                      1   23838   88000     439     485   78.77     0.0          11   438    22   484     2   439\n",
      "> 3hnoA                      1   27694    6750     395     485   25.60    0.41E-06      12   161   174   335    84   251\n",
      "> 3o8lA                      1   30771   10700     748     485   28.32     0.0          13   311    98   436    10   355\n",
      "> 3o8oA                      1   30773    9350     750     485   27.71     0.0          14   221    98   324     4   252\n",
      "> 3o8oB                      1   30774   11000     763     485   27.56     0.0          15   226    94   324     1   254\n",
      "> 3k2qA                      1   36261    8850     392     485   26.64    0.75E-11      16   214   104   324    10   238\n",
      "> 4xz2A                      1   36633    7900     708     485   26.69    0.23E-08      17   298    96   438     1   326\n",
      "> 4xz2B                      1   36634   11050     728     485   28.03     0.0          18   313    96   438     1   346\n",
      "> 4xz2D                      1   36635   11000     725     485   28.20     0.0          19   311    98   438     2   345\n",
      "> 5xz7A                      1   36673   12600     322     485   33.33     0.0          20   216    98   324     4   225\n",
      "> 4a3sA                      1   38332   13100     319     485   35.09     0.0          21   222    98   332     4   231\n",
      "> 3opyA                      1   45836   10750     936     485   29.88     0.0          22   217    98   324   176   416\n",
      "> 3opyB                      1   45837   10500     884     485   28.63     0.0          23   218    98   324   140   380\n",
      "> 4i7eA                      1   48753   13750     319     485   34.41     0.0          24   240    98   350     4   250\n",
      "> 1kzhB                      1   55999   11700     530     485   30.96     0.0          25   222    98   324    73   311\n",
      "> 1zxxA                      1   59081   13300     319     485   32.00     0.0          26   269    98   407     4   278\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.015625\n",
      "profile_iteration__> Score Collection        [sec]:    46.312500\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000689\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1453.782227\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.078125\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000692\n",
      "profile_iteration__> Overall performance [seq/sec]:  1444.989258\n",
      "openf___224_> Open           buildprofile/EAO00202build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70572049   68918.016    67.303\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70573985   68919.906    67.305\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70575273   68921.164    67.306\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70576529   68922.391    67.307\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70577377   68923.219    67.308\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70578225   68924.047    67.309\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70579109   68924.914    67.309\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70580857   68926.617    67.311\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70582533   68928.258    67.313\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70584385   68930.062    67.315\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70586165   68931.805    67.316\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70587913   68933.508    67.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70588553   68934.133    67.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70589793   68935.344    67.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70590673   68936.203    67.321\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70591573   68937.086    67.321\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70592425   68937.914    67.322\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70593613   68939.078    67.323\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70594861   68940.297    67.325\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70596101   68941.508    67.326\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70596961   68942.344    67.327\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70597845   68943.211    67.327\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70598709   68944.055    67.328\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70599577   68944.898    67.329\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70600533   68945.836    67.330\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70601417   68946.695    67.331\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70602489   68947.742    67.332\n",
      "openf___224_> Open           buildprofile/EAO00202build_profile.ali\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69668734   68035.875    66.441\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69669262   68036.391    66.442\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69717862   68083.852    66.488\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69731162   68096.836    66.501\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           2hig.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69743372   68108.766    66.512\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69743448   68108.836    66.513\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69743619   68109.000    66.513\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69744469   68109.836    66.514\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69745727   68111.062    66.515\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69747631   68112.922    66.517\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69747671   68112.961    66.517\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69747761   68113.047    66.517\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69750617   68115.836    66.519\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69750677   68115.898    66.519\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69750812   68116.031    66.520\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69755096   68120.211    66.524\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69755184   68120.297    66.524\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69755382   68120.492    66.524\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69761808   68126.766    66.530\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69761940   68126.891    66.530\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69762237   68127.188    66.530\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69771859   68136.578    66.540\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69772059   68136.773    66.540\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69772509   68137.219    66.540\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69786959   68151.328    66.554\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69787259   68151.617    66.554\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69787934   68152.281    66.555\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69809592   68173.430    66.576\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69810040   68173.867    66.576\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69811048   68174.852    66.577\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69843552   68206.594    66.608\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     69844224   68207.250    66.609\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69845736   68208.727    66.610\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     69894492   68256.344    66.657\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     70242332   68596.031    66.988\n",
      "read_mo_297_> Segments, residues, atoms:        1      440     3374\n",
      "read_mo_298_> Segment:        1  9:A  455:A     3374\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70255063   68608.461    67.000\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     70507511   68854.992    67.241\n",
      "openf___224_> Open           ali/EAO00202.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70647879   68992.070    67.375\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70783399   69124.414    67.504\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70785335   69126.305    67.506\n",
      "\n",
      "Read the alignment from file       : ali/EAO00202.ali\n",
      "\n",
      "Total number of alignment positions:   485\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      2higB     440      1    2hig.pdb MOL_ID  1; MOLECULE  6-PHOSPHO-1-FRUCTOKINASE; CHAIN  A, B; SYNONYM  ATP-DEPENDE\n",
      "  2   EAO00202     485      1    EAO00202\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          485\n",
      "  Score                    :  356520.3438\n",
      "openf___224_> Open           ali/EAO00202-2higB.ali\n",
      "openf___224_> Open           ali/EAO00202-2higB.pap\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69668734   68035.875    66.441\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69669262   68036.391    66.442\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69717862   68083.852    66.488\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     69731162   68096.836    66.501\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           ali/EAO00202-2higB.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     69743127   68108.523    66.512\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     69744577   68109.938    66.514\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     69747477   68112.773    66.516\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     69753277   68118.438    66.522\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     69764877   68129.766    66.533\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     70005685   68364.930    66.763\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70007441   68366.641    66.764\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70009377   68368.531    66.766\n",
      "\n",
      "Read the alignment from file       : ali/EAO00202-2higB.ali\n",
      "\n",
      "Total number of alignment positions:   485\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      2higB     440      1    2hig.pdb MOL_ID  1; MOLECULE  6-PHOSPHO-1-FRUCTOKINASE; CHAIN  A, B; SYNONYM  ATP-DEPENDE\n",
      "  2   EAO00202     485      1    EAO00202\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           2hig.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70143102   68499.125    66.894\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70143102   68499.125    66.894\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70143273   68499.289    66.894\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70144123   68500.117    66.895\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70145381   68501.352    66.896\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70147285   68503.211    66.898\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70147285   68503.211    66.898\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70147375   68503.297    66.898\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70150231   68506.086    66.900\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70150231   68506.086    66.900\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70150366   68506.219    66.901\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70154650   68510.398    66.905\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70154650   68510.398    66.905\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70154848   68510.594    66.905\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70161274   68516.867    66.911\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70161274   68516.867    66.911\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70161571   68517.156    66.911\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70171193   68526.555    66.920\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70171193   68526.555    66.920\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70171643   68526.992    66.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70186093   68541.109    66.935\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70186093   68541.109    66.935\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70186768   68541.766    66.935\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70208426   68562.914    66.956\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70208426   68562.914    66.956\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70209434   68563.898    66.957\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70241938   68595.641    66.988\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     70241938   68595.641    66.988\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70243450   68597.117    66.989\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     70292206   68644.734    67.036\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     70291621   68644.164    67.035\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     65     1   67    73      E     G    9.027\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70430873   68780.148    67.168\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70432473   68781.711    67.170\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70436453   68785.602    67.173\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70438453   68787.555    67.175\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70441453   68790.484    67.178\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70445953   68794.875    67.182\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70451525   68800.320    67.188\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70454325   68803.055    67.190\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70458525   68807.156    67.194\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70464825   68813.305    67.200\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70467597   68816.016    67.203\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     70468997   68817.383    67.204\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     70544985   68891.586    67.277\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     70915425   69253.344    67.630\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     70970505   69307.133    67.683\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      485   3761   3825      0         0      1530\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     71129493   69462.398    67.834\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      485   3761   3825   5170      6061      1530\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     71288549   69617.727    67.986\n",
      "patch_______> segment topology patched using RTF:  485 ;  LEU  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      485   3762   3826   5172      6063      1531\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      485   3762   3826   5172      6063      1531\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     3202      560\n",
      "The following 2 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 152:A (type TYR)>\n",
      "   <Residue 417:A (type TYR)>\n",
      "openf___224_> Open           EAO00202.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      485     3762     3762\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71395998   69722.656    68.089\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71404190   69730.656    68.096\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71420574   69746.656    68.112\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71453342   69778.656    68.143\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71486110   69810.656    68.174\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71551646   69874.656    68.237\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71584414   69906.656    68.268\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71617182   69938.656    68.299\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71682718   70002.656    68.362\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71813790   70130.656    68.487\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71879326   70194.656    68.549\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     71944862   70258.656    68.612\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     72075934   70386.656    68.737\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     72338078   70642.656    68.987\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     72469150   70770.656    69.112\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     3826    5172    5732    1512\n",
      "              Total number of restraints before, now         :        0    16242\n",
      "make_re_422_> Number of previous, current restraints         :        0    16242\n",
      "make_re_423_> Number of previous, current selected restraints:        0    16242\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     72745958   71040.977    69.376\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  2higB\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     72877046   71168.992    69.501\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     73139190   71424.992    69.751\n",
      "make_re_422_> Number of previous, current restraints         :    16242    16725\n",
      "make_re_423_> Number of previous, current selected restraints:    16242    16725\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     73139190   71424.992    69.751\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  2higB\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      503\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    16725    17209\n",
      "make_re_423_> Number of previous, current selected restraints:    16725    16706\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     73139190   71424.992    69.751\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  2higB\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    17209    17612\n",
      "make_re_423_> Number of previous, current selected restraints:    16706    17109\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     73139190   71424.992    69.751\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  2higB\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     73401334   71680.992    70.001\n",
      "make_re_422_> Number of previous, current restraints         :    17612    17907\n",
      "make_re_423_> Number of previous, current selected restraints:    17109    17404\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     73401334   71680.992    70.001\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  2higB\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    17907    18047\n",
      "make_re_423_> Number of previous, current selected restraints:    17404    17544\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     73401334   71680.992    70.001\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  2higB\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    18047    18111\n",
      "make_re_423_> Number of previous, current selected restraints:    17544    17608\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     74484754   72739.016    71.034\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     75009042   73251.016    71.534\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     75271186   73507.016    71.784\n",
      "make_re_422_> Number of previous, current restraints         :    18111    28324\n",
      "make_re_423_> Number of previous, current selected restraints:    17608    27821\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     75795474   74019.016    72.284\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     76844050   75043.016    73.284\n",
      "make_re_422_> Number of previous, current restraints         :    28324    38593\n",
      "make_re_423_> Number of previous, current selected restraints:    27821    38090\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     77368338   75555.016    73.784\n",
      "make_re_422_> Number of previous, current restraints         :    38593    44662\n",
      "make_re_423_> Number of previous, current selected restraints:    38090    44159\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    44662    47957\n",
      "make_re_423_> Number of previous, current selected restraints:    44159    47454\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    47957    47957\n",
      "make_re_423_> Number of previous, current selected restraints:    47454    47454\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    47957    47957\n",
      "make_re_423_> Number of previous, current selected restraints:    47454    47454\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    47957    47957\n",
      "make_re_423_> Number of previous, current selected restraints:    47454    47454\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    47957    47957\n",
      "make_re_423_> Number of previous, current selected restraints:    47454    47454\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    47957    47957\n",
      "make_re_423_> Number of previous, current selected restraints:    47454    47454\n",
      "rmdupl__427_>     2912 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    47957    44542\n",
      "openf___224_> Open           EAO00202.rsr\n",
      "openf___224_> Open           EAO00202.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     77588213   75769.742    73.994\n",
      "rdcsr2__307_> Number of restraints read    :    44542\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    44542    44542\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           EAO00202.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     77588245   75769.773    73.994\n",
      "read_mo_297_> Segments, residues, atoms:        1      485     3762\n",
      "read_mo_298_> Segment:        1  1:A  485:A     3762\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3762     3762        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    22222\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    24895\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    26327\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    27906\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    29602\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    31568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    34587\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    36710\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    38160\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     77971105   76143.656    74.359\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    41001\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    42797\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    44542\n",
      "iupac_m_486_> OE1/2 will be swapped:      103.8285        2  2\n",
      "iupac_m_487_> NH1/2 swapped:      177.0284        6  6\n",
      "iupac_m_487_> NH1/2 swapped:      178.6216       13  13\n",
      "iupac_m_486_> OE1/2 will be swapped:      147.6254       25  25\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1767       47  47\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.6523       49  49\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.4145       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -162.7111       55  55\n",
      "iupac_m_486_> OE1/2 will be swapped:      100.5575       65  65\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.2484       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.0651       81  81\n",
      "iupac_m_487_> NH1/2 swapped:      178.2535       88  88\n",
      "iupac_m_487_> NH1/2 swapped:      177.3270      115  115\n",
      "iupac_m_487_> NH1/2 swapped:     -177.6106      129  129\n",
      "iupac_m_486_> OE1/2 will be swapped:     -159.8440      150  150\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.0545      184  184\n",
      "iupac_m_487_> NH1/2 swapped:     -175.6417      185  185\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.3506      209  209\n",
      "iupac_m_487_> NH1/2 swapped:      175.1566      213  213\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.9604      248  248\n",
      "iupac_m_487_> NH1/2 swapped:     -178.6590      251  251\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.8888      254  254\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.3430      256  256\n",
      "iupac_m_487_> NH1/2 swapped:      177.0246      272  272\n",
      "iupac_m_485_> OD1/2 will be swapped:      143.1162      301  301\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7771      309  309\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0997      310  310\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -106.6953      311  311\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.9975      323  323\n",
      "iupac_m_486_> OE1/2 will be swapped:     -168.1174      330  330\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4032      360  360\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -106.6297      373  373\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.2117      381  381\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -99.1079      392  392\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -174.6287      416  416\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -98.1093      420  420\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7776      433  433\n",
      "iupac_m_485_> OD1/2 will be swapped:      175.2224      436  436\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.9105      447  447\n",
      "iupac_m_485_> OD1/2 will be swapped:      149.4379      455  455\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.8139      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -135.8865      461  461\n",
      "iupac_m_487_> NH1/2 swapped:     -172.0733      465  465\n",
      "iupac_m_487_> NH1/2 swapped:     -175.3071      466  466\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.2886      469  469\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.6394      476  476\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8703      477  477\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     78354165   76517.742    74.724\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     78534741   76694.086    74.897\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     78895893   77046.773    75.241\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     79618197   77752.148    75.930\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     81062805   79162.898    77.308\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     83952021   81984.398    80.063\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     89730453   87627.398    85.574\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   745376\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -54215.8242\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -54215.824219\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    79.773003\n",
      "Sequence length          :          485\n",
      "Compactness              :     0.144531\n",
      "Native energy (pair)     :  -541.541658\n",
      "Native energy (surface)  :   -14.639782\n",
      "Native energy (combined) :   -11.359458\n",
      "Z score (pair)           :   -12.360723\n",
      "Z score (surface)        :    -7.310700\n",
      "Z score (combined)       :   -14.420664\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7719\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2546.7625\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3826       0      0   0.006   0.006      33.865       1.000\n",
      " 2 Bond angle potential               :    5172       1     10   1.876   1.876      362.15       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2317       0     64  48.472  48.472      875.21       1.000\n",
      " 4 Stereochemical improper torsion pot:    1512       0      0   1.157   1.157      49.212       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7719       0      0   0.002   0.002      2.5187       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10213       0      4   0.157   0.157      151.65       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10269       1      6   0.204   0.204      235.14       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     484       0      3   3.826   3.826      82.925       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     403       0      3  63.486  63.486      51.521       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     295       0      0  70.185  70.185      86.531       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     140       0      0  77.990  77.990      86.139       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0  95.885  95.885      44.893       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    6069       0      0   0.352   0.352      144.06       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     483      41     35  23.662  52.891      66.372       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    3295       0      5   0.738   0.738      274.57       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00202.V99990001\n",
      "# Heavy relative violation of each residue is written to: EAO00202.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   45889.4688\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6234 225T 225T N   CA   1749 1750  122.72  107.00   15.72    4.52  107.00   15.72    4.52\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  32476 332G 298I N   O    2554 2292   12.17    9.70    2.48    4.63    9.70    2.48    4.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12828   1M   2E C   N       7    9 -112.65 -117.80   19.53    0.87  -63.60  165.69   19.83\n",
      "    1          2E   2E N   CA      9   10  117.96  136.80                  -40.30\n",
      "    2  12829   2E   3N C   N      16   18   57.65   55.90   13.74    0.84  -63.20  138.16   22.71\n",
      "    2          3N   3N N   CA     18   19   25.87   39.50                  -41.10\n",
      "    3  12830   3N   4R C   N      24   26 -131.06 -125.20   10.54    0.36  -63.00 -177.31   29.36\n",
      "    3          4R   4R N   CA     26   27  149.36  140.60                  -41.10\n",
      "    4  12831   4R   5L C   N      35   37  -85.21  -70.70   19.74    1.28  -63.50  165.25   24.25\n",
      "    4          5L   5L N   CA     37   38  154.98  141.60                  -41.20\n",
      "    5  12832   5L   6R C   N      43   45   66.34   57.30   29.71    1.47  -63.00  138.96   25.11\n",
      "    5          6R   6R N   CA     45   46    9.69   38.00                  -41.10\n",
      "    6  12833   6R   7D C   N      54   56 -134.60  -96.50   38.36    1.56  -63.30  165.83   17.62\n",
      "    6          7D   7D N   CA     56   57  109.73  114.20                  -40.00\n",
      "    7  12842  15H  16A C   N     127  129 -146.49 -134.00   88.42    5.26   55.40  159.53   18.17\n",
      "    7         16A  16A N   CA    129  130   59.47  147.00                   38.20\n",
      "    8  12891  64Q  65E C   N     537  539 -178.74 -117.80   74.84    2.23  -63.60 -179.16   30.20\n",
      "    8         65E  65E N   CA    539  540 -179.75  136.80                  -40.30\n",
      "    9  12894  67V  68H C   N     557  559  -67.04  -67.60   51.78    3.97  -63.20  130.58   16.09\n",
      "    9         68H  68H N   CA    559  560   88.23  140.00                  -42.30\n",
      "   10  12895  68H  69S C   N     567  569 -165.42 -136.60   29.30    1.30  -64.10 -154.28   22.20\n",
      "   10         69S  69S N   CA    569  570  145.96  151.20                  -35.00\n",
      "   11  12897  70E  71L C   N     582  584  -82.03  -70.70   29.75    1.89  -63.50  150.84   22.05\n",
      "   11         71L  71L N   CA    584  585  169.10  141.60                  -41.20\n",
      "   12  13058 231S 232F C   N    1800 1802 -107.67  -63.20   44.74    7.49  -63.20   44.74    7.49\n",
      "   12        232F 232F N   CA   1802 1803  -49.24  -44.30                  -44.30\n",
      "   13  13156 329W 330E C   N    2539 2541   30.34  -69.30  115.64   10.41  -63.60  151.22   17.62\n",
      "   13        330E 330E N   CA   2541 2542 -158.81  142.50                  -40.30\n",
      "   14  13157 330E 331G C   N    2548 2550  -65.06  -62.40    7.93    1.09   82.20  153.19   11.50\n",
      "   14        331G 331G N   CA   2550 2551  -33.73  -41.20                    8.50\n",
      "   15  13188 361K 362N C   N    2778 2780 -111.60 -119.90   82.13    3.54  -63.20  107.86   11.63\n",
      "   15        362N 362N N   CA   2780 2781   55.29  137.00                  -41.10\n",
      "   16  13279 452L 453Q C   N    3489 3491 -124.95 -121.10   26.20    1.18  -63.80  165.78   27.61\n",
      "   16        453Q 453Q N   CA   3491 3492  165.61  139.70                  -40.30\n",
      "   17  13280 453Q 454D C   N    3498 3500  -59.52  -70.90   22.70    0.76  -63.30  170.70   21.30\n",
      "   17        454D 454D N   CA   3500 3501  130.66  150.30                  -40.00\n",
      "   18  13281 454D 455D C   N    3506 3508  -89.73  -96.50   21.40    0.90  -63.30  136.48   15.43\n",
      "   18        455D 455D N   CA   3508 3509   93.90  114.20                  -40.00\n",
      "   19  13284 457R 458A C   N    3532 3534  160.20 -134.00   66.30    1.90  -62.50 -146.11   42.18\n",
      "   19        458A 458A N   CA   3534 3535  155.10  147.00                  -40.90\n",
      "   20  13285 458A 459F C   N    3537 3539 -126.04 -124.20   35.39    1.71  -63.20  164.72   19.27\n",
      "   20        459F 459F N   CA   3539 3540  107.96  143.30                  -44.30\n",
      "   21  13286 459F 460K C   N    3548 3550 -127.83 -118.00   29.51    1.52  -62.90  165.35   19.06\n",
      "   21        460K 460K N   CA   3550 3551  111.27  139.10                  -40.80\n",
      "   22  13287 460K 461E C   N    3557 3559 -135.75 -117.80   27.66    0.93  -63.60  177.20   27.68\n",
      "   22        461E 461E N   CA   3559 3560  157.85  136.80                  -40.30\n",
      "   23  13288 461E 462A C   N    3566 3568 -174.15 -134.00   41.05    1.07  -62.50 -161.99   38.35\n",
      "   23        462A 462A N   CA   3568 3569  155.57  147.00                  -40.90\n",
      "   24  13289 462A 463E C   N    3571 3573  -71.16  -69.30    3.38    0.20  -63.60  174.54   23.69\n",
      "   24        463E 463E N   CA   3573 3574  145.33  142.50                  -40.30\n",
      "   25  13291 464V 465R C   N    3587 3589 -170.27 -125.20   86.89    3.03  -63.00  149.42   26.81\n",
      "   25        465R 465R N   CA   3589 3590 -145.12  140.60                  -41.10\n",
      "   26  13292 465R 466R C   N    3598 3600  -56.70  -72.10   20.74    1.34  -63.00  169.22   23.28\n",
      "   26        466R 466R N   CA   3600 3601  128.00  141.90                  -41.10\n",
      "   27  13293 466R 467E C   N    3609 3611  -85.76  -69.30   21.99    2.02  -63.60  169.67   21.48\n",
      "   27        467E 467E N   CA   3611 3612  127.92  142.50                  -40.30\n",
      "   28  13294 467E 468L C   N    3618 3620 -140.91 -108.50   37.44    2.02  -63.50  173.23   21.23\n",
      "   28        468L 468L N   CA   3620 3621  113.77  132.50                  -41.20\n",
      "   29  13295 468L 469E C   N    3626 3628  -85.54 -117.80   72.97    2.90  -63.60  113.78   14.08\n",
      "   29        469E 469E N   CA   3628 3629   71.34  136.80                  -40.30\n",
      "   30  13296 469E 470A C   N    3635 3637 -147.73 -134.00   14.16    0.35  -62.50 -171.05   35.53\n",
      "   30        470A 470A N   CA   3637 3638  150.47  147.00                  -40.90\n",
      "   31  13298 471I 472S C   N    3648 3650  -59.25  -72.40   48.36    2.43  -64.10  140.95   10.51\n",
      "   31        472S 472S N   CA   3650 3651  105.86  152.40                  -35.00\n",
      "   32  13299 472S 473L C   N    3654 3656 -141.82 -108.50   33.32    1.58  -63.50 -169.20   23.45\n",
      "   32        473L 473L N   CA   3656 3657  132.79  132.50                  -41.20\n",
      "   33  13300 473L 474V C   N    3662 3664  -63.83  -62.40    2.80    0.31 -125.40 -172.88   10.47\n",
      "   33        474V 474V N   CA   3664 3665  -39.99  -42.40                  143.30\n",
      "   34  13301 474V 475R C   N    3669 3671 -117.11 -125.20   12.60    0.70  -63.00  177.11   27.66\n",
      "   34        475R 475R N   CA   3671 3672  150.26  140.60                  -41.10\n",
      "   35  13302 475R 476E C   N    3680 3682  -99.98 -117.80   21.24    0.63  -63.60  169.50   20.81\n",
      "   35        476E 476E N   CA   3682 3683  125.24  136.80                  -40.30\n",
      "   36  13303 476E 477R C   N    3689 3691 -146.93 -125.20   30.10    0.91  -63.00  178.44   29.78\n",
      "   36        477R 477R N   CA   3691 3692  161.43  140.60                  -41.10\n",
      "   37  13304 477R 478L C   N    3700 3702  -90.14  -70.70   19.50    1.81  -63.50 -179.44   26.67\n",
      "   37        478L 478L N   CA   3702 3703  140.22  141.60                  -41.20\n",
      "   38  13306 479I 480G C   N    3716 3718   59.78   78.70   45.65    0.68  -80.20  152.83   11.27\n",
      "   38        480G 480G N   CA   3718 3719 -124.56 -166.10                  174.10\n",
      "   39  13307 480G 481Q C   N    3720 3722  -85.35  -73.00   16.23    0.95  -63.80  169.84   26.12\n",
      "   39        481Q 481Q N   CA   3722 3723  151.23  140.70                  -40.30\n",
      "   40  13309 482L 483S C   N    3737 3739  -75.10  -72.40   23.23    1.46  -64.10  164.70   11.32\n",
      "   40        483S 483S N   CA   3739 3740  129.33  152.40                  -35.00\n",
      "   41  13310 483S 484K C   N    3743 3745 -156.71 -118.00   41.16    1.81  -62.90 -169.39   21.91\n",
      "   41        484K 484K N   CA   3745 3746  125.13  139.10                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   20   44  151  229  340  345  391  504  615  618\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00202.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      485     3762     3762\n",
      "openf___224_> Open           EAO00202.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     89437713   87341.516    85.294\n",
      "read_mo_297_> Segments, residues, atoms:        1      485     3762\n",
      "read_mo_298_> Segment:        1  1:A  485:A     3762\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3762     3762        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    22222\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    24895\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    26327\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    27906\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    29602\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    31568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    34587\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    36710\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    38160\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    41001\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    42797\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    44542\n",
      "iupac_m_486_> OE1/2 will be swapped:     -158.1038        2  2\n",
      "iupac_m_487_> NH1/2 swapped:     -176.8518        4  4\n",
      "iupac_m_487_> NH1/2 swapped:      179.3957        6  6\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.7632        7  7\n",
      "iupac_m_487_> NH1/2 swapped:      177.3528       13  13\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.6473       25  25\n",
      "iupac_m_486_> OE1/2 will be swapped:     -109.7616       30  30\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.5432       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      109.6075       55  55\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3648       62  62\n",
      "iupac_m_486_> OE1/2 will be swapped:      116.0255       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.9585       81  81\n",
      "iupac_m_487_> NH1/2 swapped:      167.4449       86  86\n",
      "iupac_m_487_> NH1/2 swapped:      178.6333       88  88\n",
      "iupac_m_487_> NH1/2 swapped:     -174.6365      115  115\n",
      "iupac_m_487_> NH1/2 swapped:      177.9848      126  126\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1987      161  161\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2923      171  171\n",
      "iupac_m_487_> NH1/2 swapped:      179.7317      201  201\n",
      "iupac_m_486_> OE1/2 will be swapped:     -128.5087      209  209\n",
      "iupac_m_487_> NH1/2 swapped:     -177.9058      212  212\n",
      "iupac_m_487_> NH1/2 swapped:     -171.3377      213  213\n",
      "iupac_m_487_> NH1/2 swapped:      173.2351      235  235\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.8371      248  248\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4297      251  251\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.9251      254  254\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      103.6000      280  280\n",
      "iupac_m_487_> NH1/2 swapped:     -178.3036      310  310\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.9594      323  323\n",
      "iupac_m_486_> OE1/2 will be swapped:     -144.1838      330  330\n",
      "iupac_m_487_> NH1/2 swapped:      178.6656      360  360\n",
      "iupac_m_486_> OE1/2 will be swapped:     -157.5455      364  364\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -106.7939      373  373\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.7706      392  392\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.3154      403  403\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.6680      420  420\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -176.6851      432  432\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2832      433  433\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.0041      436  436\n",
      "iupac_m_487_> NH1/2 swapped:      179.9184      438  438\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.5954      447  447\n",
      "iupac_m_485_> OD1/2 will be swapped:     -164.8197      454  454\n",
      "iupac_m_487_> NH1/2 swapped:     -177.8421      457  457\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.4105      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.7165      463  463\n",
      "iupac_m_487_> NH1/2 swapped:     -173.5892      465  465\n",
      "iupac_m_487_> NH1/2 swapped:      178.1162      475  475\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.8080      476  476\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   749014\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -53857.0703\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -53857.070312\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    79.773003\n",
      "Sequence length          :          485\n",
      "Compactness              :     0.156191\n",
      "Native energy (pair)     :  -540.974829\n",
      "Native energy (surface)  :   -17.080092\n",
      "Native energy (combined) :   -12.373671\n",
      "Z score (pair)           :   -13.033229\n",
      "Z score (surface)        :    -7.578434\n",
      "Z score (combined)       :   -13.814387\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7709\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2607.1523\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3826       0      0   0.005   0.005      31.620       1.000\n",
      " 2 Bond angle potential               :    5172       0     11   1.874   1.874      363.00       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2317       0     73  48.814  48.814      877.82       1.000\n",
      " 4 Stereochemical improper torsion pot:    1512       0      0   1.222   1.222      52.903       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7709       0      0   0.002   0.002      2.7118       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10213       0     10   0.151   0.151      150.43       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10269       0      5   0.197   0.197      238.22       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     484       0      3   3.744   3.744      79.452       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     403       0      2  62.059  62.059      52.827       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     295       0      1  70.619  70.619      84.389       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     140       0      0  77.222  77.222      91.560       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0  95.891  95.891      43.377       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    6069       0      0   0.352   0.352      142.10       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     483      40     49  26.574  53.608      115.52       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    3295       0      8   0.750   0.750      281.23       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00202.V99990002\n",
      "# Heavy relative violation of each residue is written to: EAO00202.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   46289.9414\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12828   1M   2E C   N       7    9 -154.47  -63.60  108.23   18.62  -63.60  108.23   18.62\n",
      "    1          2E   2E N   CA      9   10  -99.10  -40.30                  -40.30\n",
      "    2  12830   3N   4R C   N      24   26  -84.15  -72.10   17.62    1.14  -63.00  165.50   23.81\n",
      "    2          4R   4R N   CA     26   27  154.75  141.90                  -41.10\n",
      "    3  12831   4R   5L C   N      35   37  -86.89 -108.50   36.55    1.70  -63.50  146.11   19.07\n",
      "    3          5L   5L N   CA     37   38  103.03  132.50                  -41.20\n",
      "    4  12832   5L   6R C   N      43   45 -157.54 -125.20   68.49    2.48  -63.00  151.14   26.53\n",
      "    4          6R   6R N   CA     45   46 -159.02  140.60                  -41.10\n",
      "    5  12833   6R   7D C   N      54   56  -68.60  -70.90    2.33    0.16  -63.30  170.17   21.33\n",
      "    5          7D   7D N   CA     56   57  149.91  150.30                  -40.00\n",
      "    6  12842  15H  16A C   N     127  129 -145.25 -134.00   88.67    5.25   55.40  160.71   18.28\n",
      "    6         16A  16A N   CA    129  130   59.04  147.00                   38.20\n",
      "    7  12891  64Q  65E C   N     537  539  173.57 -117.80   71.93    2.20  -63.60 -157.23   33.61\n",
      "    7         65E  65E N   CA    539  540  158.36  136.80                  -40.30\n",
      "    8  12894  67V  68H C   N     557  559  -58.92  -67.60   38.32    2.68  -63.20  145.05   18.40\n",
      "    8         68H  68H N   CA    559  560  102.68  140.00                  -42.30\n",
      "    9  12895  68H  69S C   N     567  569  171.90 -136.60   51.51    2.02  -64.10 -147.36   24.36\n",
      "    9         69S  69S N   CA    569  570  152.25  151.20                  -35.00\n",
      "   10  12897  70E  71L C   N     582  584 -127.81 -108.50   31.69    1.46  -63.50  173.53   28.08\n",
      "   10         71L  71L N   CA    584  585  157.63  132.50                  -41.20\n",
      "   11  13058 231S 232F C   N    1800 1802 -108.74  -63.20   45.81    7.66  -63.20   45.81    7.66\n",
      "   11        232F 232F N   CA   1802 1803  -49.24  -44.30                  -44.30\n",
      "   12  13156 329W 330E C   N    2539 2541   76.98  -69.30  148.70   10.30  -63.60 -149.95   35.34\n",
      "   12        330E 330E N   CA   2541 2542  115.77  142.50                  -40.30\n",
      "   13  13188 361K 362N C   N    2778 2780 -107.88 -119.90   82.72    3.47  -63.20  106.12   11.49\n",
      "   13        362N 362N N   CA   2780 2781   55.16  137.00                  -41.10\n",
      "   14  13279 452L 453Q C   N    3489 3491  163.57 -121.10   77.18    2.58  -63.80 -149.71   37.38\n",
      "   14        453Q 453Q N   CA   3491 3492  156.51  139.70                  -40.30\n",
      "   15  13280 453Q 454D C   N    3498 3500  134.49   54.50  108.81   13.69  -63.30 -135.88   25.56\n",
      "   15        454D 454D N   CA   3500 3501  114.66   40.90                  -40.00\n",
      "   16  13281 454D 455D C   N    3506 3508  -70.51  -96.50   26.13    1.06  -63.30  157.07   18.96\n",
      "   16        455D 455D N   CA   3508 3509  116.90  114.20                  -40.00\n",
      "   17  13283 456V 457R C   N    3521 3523  -59.87  -72.10   18.76    1.22  -63.00  168.80   23.01\n",
      "   17        457R 457R N   CA   3523 3524  127.68  141.90                  -41.10\n",
      "   18  13284 457R 458A C   N    3532 3534  170.96 -134.00   64.00    1.50  -62.50 -171.69   37.37\n",
      "   18        458A 458A N   CA   3534 3535  179.65  147.00                  -40.90\n",
      "   19  13285 458A 459F C   N    3537 3539  -66.62  -71.40    7.64    0.46  -63.20  179.08   24.64\n",
      "   19        459F 459F N   CA   3539 3540  134.74  140.70                  -44.30\n",
      "   20  13286 459F 460K C   N    3548 3550  129.73 -118.00  112.28    4.22  -62.90 -115.08   41.42\n",
      "   20        460K 460K N   CA   3550 3551  140.39  139.10                  -40.80\n",
      "   21  13287 460K 461E C   N    3557 3559 -137.68 -117.80   30.96    1.67  -63.60  170.32   19.79\n",
      "   21        461E 461E N   CA   3559 3560  113.07  136.80                  -40.30\n",
      "   22  13288 461E 462A C   N    3566 3568 -151.05 -134.00   29.87    1.82  -62.50 -174.17   27.84\n",
      "   22        462A 462A N   CA   3568 3569  122.47  147.00                  -40.90\n",
      "   23  13289 462A 463E C   N    3571 3573 -131.01 -117.80   41.11    2.23  -63.60  153.74   17.85\n",
      "   23        463E 463E N   CA   3573 3574   97.87  136.80                  -40.30\n",
      "   24  13291 464V 465R C   N    3587 3589 -134.68 -125.20   35.27    1.87  -63.00  164.20   18.70\n",
      "   24        465R 465R N   CA   3589 3590  106.63  140.60                  -41.10\n",
      "   25  13292 465R 466R C   N    3598 3600 -174.05 -125.20   58.56    1.73  -63.00 -176.56   32.01\n",
      "   25        466R 466R N   CA   3600 3601  172.89  140.60                  -41.10\n",
      "   26  13293 466R 467E C   N    3609 3611  -88.59  -69.30   39.07    3.52  -63.60  150.91   18.86\n",
      "   26        467E 467E N   CA   3611 3612  108.53  142.50                  -40.30\n",
      "   27  13294 467E 468L C   N    3618 3620  -96.33  -70.70   53.31    3.32  -63.50  134.52   20.69\n",
      "   27        468L 468L N   CA   3620 3621 -171.65  141.60                  -41.20\n",
      "   28  13295 468L 469E C   N    3626 3628  -86.74  -69.30   50.66    4.40  -63.60  137.21   17.13\n",
      "   28        469E 469E N   CA   3628 3629   94.94  142.50                  -40.30\n",
      "   29  13296 469E 470A C   N    3635 3637  138.39 -134.00   96.66    2.20  -62.50 -153.73   41.62\n",
      "   29        470A 470A N   CA   3637 3638 -172.17  147.00                  -40.90\n",
      "   30  13298 471I 472S C   N    3648 3650  -91.19  -72.40   49.55    2.36  -64.10  129.61   11.22\n",
      "   30        472S 472S N   CA   3650 3651 -161.75  152.40                  -35.00\n",
      "   31  13299 472S 473L C   N    3654 3656  -72.92  -70.70   53.09    4.11  -63.50  130.10   17.55\n",
      "   31        473L 473L N   CA   3656 3657   88.56  141.60                  -41.20\n",
      "   32  13300 473L 474V C   N    3662 3664  -64.67  -62.40    8.14    1.17 -125.40  177.21    9.94\n",
      "   32        474V 474V N   CA   3664 3665  -50.22  -42.40                  143.30\n",
      "   33  13301 474V 475R C   N    3669 3671  179.04 -125.20   92.74    3.04  -63.00  157.39   28.41\n",
      "   33        475R 475R N   CA   3671 3672 -145.30  140.60                  -41.10\n",
      "   34  13302 475R 476E C   N    3680 3682  -53.98  -69.30   19.85    1.11  -63.60  170.45   23.26\n",
      "   34        476E 476E N   CA   3682 3683  129.87  142.50                  -40.30\n",
      "   35  13303 476E 477R C   N    3689 3691 -122.14 -125.20   22.12    1.13  -63.00  167.20   26.66\n",
      "   35        477R 477R N   CA   3691 3692  162.51  140.60                  -41.10\n",
      "   36  13306 479I 480G C   N    3716 3718  131.36   78.70   68.74    3.76  -80.20  161.64    8.16\n",
      "   36        480G 480G N   CA   3718 3719 -121.92 -166.10                  174.10\n",
      "   37  13307 480G 481Q C   N    3720 3722  -95.24  -73.00   23.90    1.53  -63.80  173.10   27.13\n",
      "   37        481Q 481Q N   CA   3722 3723  149.47  140.70                  -40.30\n",
      "   38  13308 481Q 482L C   N    3729 3731  -91.90 -108.50   17.29    0.89  -63.50 -179.22   23.62\n",
      "   38        482L 482L N   CA   3731 3732  137.33  132.50                  -41.20\n",
      "   39  13309 482L 483S C   N    3737 3739 -140.51 -136.60   18.90    0.87  -64.10  173.08   18.01\n",
      "   39        483S 483S N   CA   3739 3740  169.69  151.20                  -35.00\n",
      "   40  13310 483S 484K C   N    3743 3745  -73.73  -70.20   12.09    0.79  -62.90  167.59   22.37\n",
      "   40        484K 484K N   CA   3745 3746  151.96  140.40                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   12   44  165  230  376  316  419  484  590  619\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00202.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      485     3762     3762\n",
      "openf___224_> Open           EAO00202.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     89437713   87341.516    85.294\n",
      "read_mo_297_> Segments, residues, atoms:        1      485     3762\n",
      "read_mo_298_> Segment:        1  1:A  485:A     3762\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3762     3762        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    22222\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    24895\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    26327\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    27906\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    29602\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    31568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    34587\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    36710\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    38160\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    41001\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    42797\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    44542\n",
      "iupac_m_487_> NH1/2 swapped:      178.8548        4  4\n",
      "iupac_m_485_> OD1/2 will be swapped:     -138.8714        7  7\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4968       10  10\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.7988       21  21\n",
      "iupac_m_487_> NH1/2 swapped:     -172.2226       31  31\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.8575       48  48\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.0721       49  49\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -93.1904       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      113.2372       52  52\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.1558       55  55\n",
      "iupac_m_487_> NH1/2 swapped:     -176.0170       62  62\n",
      "iupac_m_487_> NH1/2 swapped:      173.3360       88  88\n",
      "iupac_m_487_> NH1/2 swapped:      179.7529      129  129\n",
      "iupac_m_487_> NH1/2 swapped:      177.4576      134  134\n",
      "iupac_m_487_> NH1/2 swapped:      173.3212      153  153\n",
      "iupac_m_487_> NH1/2 swapped:      177.2458      161  161\n",
      "iupac_m_487_> NH1/2 swapped:     -176.8772      171  171\n",
      "iupac_m_486_> OE1/2 will be swapped:      126.6481      178  178\n",
      "iupac_m_485_> OD1/2 will be swapped:      158.2482      197  197\n",
      "iupac_m_486_> OE1/2 will be swapped:     -178.9343      209  209\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.3524      232  232\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      148.1441      240  240\n",
      "iupac_m_485_> OD1/2 will be swapped:      167.7435      244  244\n",
      "iupac_m_487_> NH1/2 swapped:      178.2498      251  251\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.9674      254  254\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.9191      280  280\n",
      "iupac_m_486_> OE1/2 will be swapped:      102.0992      295  295\n",
      "iupac_m_487_> NH1/2 swapped:     -176.2183      309  309\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.3181      311  311\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -128.5340      312  312\n",
      "iupac_m_487_> NH1/2 swapped:      174.3752      315  315\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.1021      323  323\n",
      "iupac_m_486_> OE1/2 will be swapped:     -160.2786      330  330\n",
      "iupac_m_487_> NH1/2 swapped:      178.5553      354  354\n",
      "iupac_m_487_> NH1/2 swapped:     -176.9376      365  365\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -105.3139      373  373\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -94.4040      392  392\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2224      432  432\n",
      "iupac_m_487_> NH1/2 swapped:      177.6528      443  443\n",
      "iupac_m_485_> OD1/2 will be swapped:      107.0902      455  455\n",
      "iupac_m_487_> NH1/2 swapped:      177.9031      457  457\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.3838      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.4647      461  461\n",
      "iupac_m_486_> OE1/2 will be swapped:     -160.5821      463  463\n",
      "iupac_m_487_> NH1/2 swapped:     -173.8611      465  465\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1457      466  466\n",
      "iupac_m_486_> OE1/2 will be swapped:      175.5839      469  469\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.1143      476  476\n",
      "iupac_m_487_> NH1/2 swapped:      179.0192      477  477\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   747002\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -53809.7461\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -53809.746094\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    79.773003\n",
      "Sequence length          :          485\n",
      "Compactness              :     0.180571\n",
      "Native energy (pair)     :  -562.553129\n",
      "Native energy (surface)  :   -16.275414\n",
      "Native energy (combined) :   -12.651168\n",
      "Z score (pair)           :   -13.346086\n",
      "Z score (surface)        :    -7.559342\n",
      "Z score (combined)       :   -14.266255\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7756\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2768.7705\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3826       0      0   0.006   0.006      35.099       1.000\n",
      " 2 Bond angle potential               :    5172       0      9   1.937   1.937      385.65       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2317       0     56  47.875  47.875      854.70       1.000\n",
      " 4 Stereochemical improper torsion pot:    1512       0      0   1.307   1.307      58.823       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7756       0      0   0.002   0.002      3.1005       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10213       1      4   0.164   0.164      159.21       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10269       5     12   0.220   0.220      301.30       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     484       1      4   9.072   9.072      90.395       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     403       0      4  63.150  63.150      62.148       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     295       0      1  68.182  68.182      97.131       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     140       0      0  77.159  77.159      91.392       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0 113.508 113.508      45.951       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    6069       0      0   0.395   0.395      174.63       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     483      43     40  26.373  54.723      116.61       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    3295       0      3   0.762   0.762      292.63       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00202.V99990003\n",
      "# Heavy relative violation of each residue is written to: EAO00202.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   47561.1211\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16034  58M  66G CA  CA    480  549   17.75   13.81    3.94    4.83   13.81    3.94    4.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33096 369G 261N N   O    2844 2032    6.55    3.92    2.63    5.66    3.92    2.63    5.66\n",
      "    2  33120 370T 261N N   O    2848 2032    6.90    4.16    2.74    5.53    4.16    2.74    5.53\n",
      "    3  33145 371V 261N N   O    2855 2032    9.76    7.34    2.42    5.27    7.34    2.42    5.27\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12828   1M   2E C   N       7    9 -114.30 -117.80   77.39    3.95  -63.60  117.13   18.45\n",
      "    1          2E   2E N   CA      9   10 -145.89  136.80                  -40.30\n",
      "    2  12831   4R   5L C   N      35   37 -124.83 -108.50   31.22    1.48  -63.50  171.06   27.55\n",
      "    2          5L   5L N   CA     37   38  159.11  132.50                  -41.20\n",
      "    3  12832   5L   6R C   N      43   45  -95.30  -72.10   58.47    4.94  -63.00  133.30   16.08\n",
      "    3          6R   6R N   CA     45   46   88.23  141.90                  -41.10\n",
      "    4  12833   6R   7D C   N      54   56 -122.95  -70.90   53.26    3.60  -63.30  169.28   24.75\n",
      "    4          7D   7D N   CA     56   57  161.58  150.30                  -40.00\n",
      "    5  12842  15H  16A C   N     127  129 -143.67 -134.00   90.67    5.34   55.40  162.00   18.35\n",
      "    5         16A  16A N   CA    129  130   56.85  147.00                   38.20\n",
      "    6  12891  64Q  65E C   N     537  539  -97.46 -117.80   22.83    1.10  -63.60  175.84   25.35\n",
      "    6         65E  65E N   CA    539  540  147.15  136.80                  -40.30\n",
      "    7  12892  65E  66G C   N     546  548  116.60   78.70   43.16    2.42  -80.20  168.14    9.73\n",
      "    7         66G  66G N   CA    548  549 -145.45 -166.10                  174.10\n",
      "    8  12894  67V  68H C   N     557  559  -58.84  -67.60   12.32    0.73  -63.20  173.69   21.99\n",
      "    8         68H  68H N   CA    559  560  131.34  140.00                  -42.30\n",
      "    9  12895  68H  69S C   N     567  569 -136.92 -136.60   32.66    1.65  -64.10  158.82   16.73\n",
      "    9         69S  69S N   CA    569  570 -176.14  151.20                  -35.00\n",
      "   10  12897  70E  71L C   N     582  584 -136.04 -108.50   30.27    1.34  -63.50 -171.74   30.63\n",
      "   10         71L  71L N   CA    584  585  145.07  132.50                  -41.20\n",
      "   11  13058 231S 232F C   N    1800 1802 -107.80  -63.20   44.89    7.51  -63.20   44.89    7.51\n",
      "   11        232F 232F N   CA   1802 1803  -49.31  -44.30                  -44.30\n",
      "   12  13087 260L 261N C   N    2023 2025  -53.91  -63.20   15.95    1.72  -71.20  164.05   11.00\n",
      "   12        261N 261N N   CA   2025 2026  -54.07  -41.10                  142.80\n",
      "   13  13088 261N 262Y C   N    2031 2033 -134.18 -124.30   55.72    3.38   55.90  174.81   21.11\n",
      "   13        262Y 262Y N   CA   2033 2034   80.56  135.40                   39.50\n",
      "   14  13156 329W 330E C   N    2539 2541   10.98  -69.30  109.84   10.10  -63.60  126.53   14.67\n",
      "   14        330E 330E N   CA   2541 2542 -142.52  142.50                  -40.30\n",
      "   15  13157 330E 331G C   N    2548 2550  -69.77  -62.40   12.08    1.66   82.20  157.18   11.73\n",
      "   15        331G 331G N   CA   2550 2551  -31.63  -41.20                    8.50\n",
      "   16  13188 361K 362N C   N    2778 2780 -109.45 -119.90   80.74    3.42  -63.20  108.40   11.72\n",
      "   16        362N 362N N   CA   2780 2781   56.94  137.00                  -41.10\n",
      "   17  13279 452L 453Q C   N    3489 3491 -168.31 -121.10   57.08    1.76  -63.80 -178.89   31.85\n",
      "   17        453Q 453Q N   CA   3491 3492  171.78  139.70                  -40.30\n",
      "   18  13280 453Q 454D C   N    3498 3500  -84.68  -70.90   36.68    1.26  -63.30  137.38   18.31\n",
      "   18        454D 454D N   CA   3500 3501 -175.71  150.30                  -40.00\n",
      "   19  13281 454D 455D C   N    3506 3508  -90.10  -70.90   35.87    1.22  -63.30  141.96   19.23\n",
      "   19        455D 455D N   CA   3508 3509 -179.40  150.30                  -40.00\n",
      "   20  13283 456V 457R C   N    3521 3523 -119.43 -125.20   25.37    1.33  -63.00  163.63   25.99\n",
      "   20        457R 457R N   CA   3523 3524  165.30  140.60                  -41.10\n",
      "   21  13284 457R 458A C   N    3532 3534 -138.85 -134.00   26.68    1.39  -62.50  164.64   31.07\n",
      "   21        458A 458A N   CA   3534 3535  173.24  147.00                  -40.90\n",
      "   22  13285 458A 459F C   N    3537 3539  -76.49  -71.40   12.51    0.85  -63.20  164.11   23.68\n",
      "   22        459F 459F N   CA   3539 3540  152.13  140.70                  -44.30\n",
      "   23  13287 460K 461E C   N    3557 3559 -110.80 -117.80    7.17    0.23  -63.60 -178.21   22.00\n",
      "   23        461E 461E N   CA   3559 3560  135.25  136.80                  -40.30\n",
      "   24  13288 461E 462A C   N    3566 3568 -132.59 -134.00    1.59    0.08  -62.50 -174.85   34.10\n",
      "   24        462A 462A N   CA   3568 3569  147.74  147.00                  -40.90\n",
      "   25  13289 462A 463E C   N    3571 3573  -71.90  -69.30    9.63    0.64  -63.60  168.13   22.87\n",
      "   25        463E 463E N   CA   3573 3574  151.77  142.50                  -40.30\n",
      "   26  13291 464V 465R C   N    3587 3589  -76.27  -72.10    4.26    0.34  -63.00  178.36   25.00\n",
      "   26        465R 465R N   CA   3589 3590  141.03  141.90                  -41.10\n",
      "   27  13292 465R 466R C   N    3598 3600   82.50   57.30  125.19   10.14  -63.00 -145.01   25.08\n",
      "   27        466R 466R N   CA   3600 3601  160.62   38.00                  -41.10\n",
      "   28  13293 466R 467E C   N    3609 3611  -73.52  -69.30    7.52    0.43  -63.60  171.25   23.38\n",
      "   28        467E 467E N   CA   3611 3612  148.73  142.50                  -40.30\n",
      "   29  13294 467E 468L C   N    3618 3620 -112.85 -108.50   31.99    1.68  -63.50  162.29   25.58\n",
      "   29        468L 468L N   CA   3620 3621  164.20  132.50                  -41.20\n",
      "   30  13295 468L 469E C   N    3626 3628 -132.94 -117.80   40.61    1.69  -63.60  160.92   25.34\n",
      "   30        469E 469E N   CA   3628 3629  174.48  136.80                  -40.30\n",
      "   31  13296 469E 470A C   N    3635 3637  104.90 -134.00  121.17    4.17  -62.50 -116.93   37.38\n",
      "   31        470A 470A N   CA   3637 3638  142.85  147.00                  -40.90\n",
      "   32  13297 470A 471I C   N    3640 3642  -66.84  -63.40    4.41    0.83 -120.60 -175.33    8.58\n",
      "   32        471I 471I N   CA   3642 3643  -46.37  -43.60                  130.30\n",
      "   33  13298 471I 472S C   N    3648 3650 -178.50 -136.60   71.67    2.54  -64.10  162.67   19.97\n",
      "   33        472S 472S N   CA   3650 3651 -150.65  151.20                  -35.00\n",
      "   34  13299 472S 473L C   N    3654 3656  -72.97  -70.70   44.19    3.44  -63.50  138.99   18.78\n",
      "   34        473L 473L N   CA   3656 3657   97.47  141.60                  -41.20\n",
      "   35  13301 474V 475R C   N    3669 3671  -62.81  -72.10   14.22    1.23  -63.00  166.24   22.44\n",
      "   35        475R 475R N   CA   3671 3672  152.66  141.90                  -41.10\n",
      "   36  13302 475R 476E C   N    3680 3682  -61.66  -69.30   18.45    1.14  -63.60  166.02   22.24\n",
      "   36        476E 476E N   CA   3682 3683  125.70  142.50                  -40.30\n",
      "   37  13303 476E 477R C   N    3689 3691 -125.88 -125.20   34.64    1.69  -63.00  159.95   18.37\n",
      "   37        477R 477R N   CA   3691 3692  105.97  140.60                  -41.10\n",
      "   38  13304 477R 478L C   N    3700 3702 -100.41 -108.50   37.24    2.08  -63.50  154.42   23.70\n",
      "   38        478L 478L N   CA   3702 3703  168.85  132.50                  -41.20\n",
      "   39  13305 478L 479I C   N    3708 3710   58.65  -97.30  157.47    6.61 -120.60 -179.77   13.08\n",
      "   39        479I 479I N   CA   3710 3711  149.03  127.20                  130.30\n",
      "   40  13307 480G 481Q C   N    3720 3722 -133.60 -121.10   12.75    0.43  -63.80 -169.32   31.73\n",
      "   40        481Q 481Q N   CA   3722 3723  142.25  139.70                  -40.30\n",
      "   41  13308 481Q 482L C   N    3729 3731  -96.41 -108.50   31.78    1.57  -63.50  148.01   18.93\n",
      "   41        482L 482L N   CA   3731 3732  103.11  132.50                  -41.20\n",
      "   42  13309 482L 483S C   N    3737 3739 -144.65 -136.60    8.39    0.28  -64.10 -170.56   19.48\n",
      "   42        483S 483S N   CA   3739 3740  153.54  151.20                  -35.00\n",
      "   43  13310 483S 484K C   N    3743 3745 -143.36 -118.00   44.00    1.67  -62.90  165.07   26.24\n",
      "   43        484K 484K N   CA   3745 3746  175.06  139.10                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   26   41  145  238  364  344  402  518  597  591\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00202.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      485     3762     3762\n",
      "openf___224_> Open           EAO00202.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     89437713   87341.516    85.294\n",
      "read_mo_297_> Segments, residues, atoms:        1      485     3762\n",
      "read_mo_298_> Segment:        1  1:A  485:A     3762\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3762     3762        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    22222\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    24895\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    26327\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    27906\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    29602\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    31568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    34587\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    36710\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    38160\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    41001\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    42797\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    44542\n",
      "iupac_m_487_> NH1/2 swapped:      176.1503        6  6\n",
      "iupac_m_485_> OD1/2 will be swapped:     -140.1014        7  7\n",
      "iupac_m_487_> NH1/2 swapped:     -169.1562       10  10\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.8465       25  25\n",
      "iupac_m_486_> OE1/2 will be swapped:     -128.9233       30  30\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.1477       37  37\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.5764       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:      107.8520       81  81\n",
      "iupac_m_487_> NH1/2 swapped:      171.4951       86  86\n",
      "iupac_m_487_> NH1/2 swapped:      179.5555      126  126\n",
      "iupac_m_487_> NH1/2 swapped:     -177.8155      129  129\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1770      171  171\n",
      "iupac_m_487_> NH1/2 swapped:     -179.6379      185  185\n",
      "iupac_m_487_> NH1/2 swapped:      176.7341      201  201\n",
      "iupac_m_486_> OE1/2 will be swapped:      122.7297      208  208\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.9461      209  209\n",
      "iupac_m_487_> NH1/2 swapped:      173.8866      213  213\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.7226      232  232\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.4212\n",
      "iupac_m_487_> NH1/2 swapped:      173.1862      235  235\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -101.1601      254  254\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -174.0901      272  272\n",
      "iupac_m_486_> OE1/2 will be swapped:      165.3361      280  280\n",
      "iupac_m_487_> NH1/2 swapped:     -176.0177      310  310\n",
      "iupac_m_486_> OE1/2 will be swapped:      147.5921      330  330\n",
      "iupac_m_487_> NH1/2 swapped:     -177.3527      354  354\n",
      "iupac_m_487_> NH1/2 swapped:      178.5804      360  360\n",
      "iupac_m_486_> OE1/2 will be swapped:     -134.2679      364  364\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -105.9274      373  373\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9802      381  381\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -101.0702      392  392\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.9257      403  403\n",
      "iupac_m_487_> NH1/2 swapped:      170.3775      416  416\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.3065      420  420\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -169.6276      432  432\n",
      "iupac_m_485_> OD1/2 will be swapped:      166.3339      436  436\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4681      443  443\n",
      "iupac_m_487_> NH1/2 swapped:      179.9657      446  446\n",
      "iupac_m_487_> NH1/2 swapped:      177.2080      457  457\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -100.3808      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      177.3466      466  466\n",
      "iupac_m_486_> OE1/2 will be swapped:      134.2121      467  467\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.7838      469  469\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.0964      476  476\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   742658\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -53850.4336\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -53850.433594\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    79.773003\n",
      "Sequence length          :          485\n",
      "Compactness              :     0.177460\n",
      "Native energy (pair)     :  -576.065540\n",
      "Native energy (surface)  :   -15.455272\n",
      "Native energy (combined) :   -12.841144\n",
      "Z score (pair)           :   -13.348824\n",
      "Z score (surface)        :    -7.333493\n",
      "Z score (combined)       :   -14.028459\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7732\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2696.5942\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3826       0      0   0.006   0.006      36.984       1.000\n",
      " 2 Bond angle potential               :    5172       1     14   1.963   1.963      397.28       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2317       0     70  48.574  48.574      871.37       1.000\n",
      " 4 Stereochemical improper torsion pot:    1512       0      0   1.231   1.231      54.338       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7732       0      2   0.004   0.004      17.425       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10213       0      9   0.157   0.157      165.66       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10269       0     10   0.215   0.215      280.07       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     484       0      1   3.862   3.862      84.546       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     403       0      4  59.151  59.151      52.943       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     295       0      1  69.981  69.981      93.680       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     140       0      0  76.770  76.770      97.798       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0  94.187  94.187      44.954       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    6069       0      0   0.313   0.313      117.67       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     483      40     45  26.515  51.791      134.42       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    3295       0      3   0.706   0.706      247.44       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00202.V99990004\n",
      "# Heavy relative violation of each residue is written to: EAO00202.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   45711.2734\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4209  35R  35R CG  CD    280  281  127.02  107.50   19.52    5.16  107.50   19.52    5.16\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12828   1M   2E C   N       7    9  167.76 -117.80  125.98    6.87  -63.60  149.15   19.42\n",
      "    1          2E   2E N   CA      9   10   35.16  136.80                  -40.30\n",
      "    2  12829   2E   3N C   N      16   18  177.28 -119.90  105.91    3.27  -63.20  153.70   25.13\n",
      "    2          3N   3N N   CA     18   19 -137.73  137.00                  -41.10\n",
      "    3  12831   4R   5L C   N      35   37  -72.34  -70.70    5.33    0.35  -63.50  172.35   24.45\n",
      "    3          5L   5L N   CA     37   38  146.67  141.60                  -41.20\n",
      "    4  12832   5L   6R C   N      43   45 -109.39 -125.20   46.46    1.85  -63.00  145.60   17.08\n",
      "    4          6R   6R N   CA     45   46   96.92  140.60                  -41.10\n",
      "    5  12833   6R   7D C   N      54   56 -168.79  -96.50   86.70    3.60  -63.30 -170.06   30.14\n",
      "    5          7D   7D N   CA     56   57  162.05  114.20                  -40.00\n",
      "    6  12842  15H  16A C   N     127  129 -146.16 -134.00   88.40    5.25   55.40  159.86   18.20\n",
      "    6         16A  16A N   CA    129  130   59.44  147.00                   38.20\n",
      "    7  12891  64Q  65E C   N     537  539  137.88 -117.80  105.80    3.47  -63.60 -130.99   38.70\n",
      "    7         65E  65E N   CA    539  540  154.42  136.80                  -40.30\n",
      "    8  12895  68H  69S C   N     567  569 -101.06  -72.40   39.88    3.18  -64.10  163.89   10.28\n",
      "    8         69S  69S N   CA    569  570  124.67  152.40                  -35.00\n",
      "    9  12897  70E  71L C   N     582  584  -59.83  -70.70   10.89    0.96  -63.50  178.02   24.51\n",
      "    9         71L  71L N   CA    584  585  140.82  141.60                  -41.20\n",
      "   10  12984 157T 158S C   N    1250 1252 -128.90  -64.10   98.10    7.52  -64.10   98.10    7.52\n",
      "   10        158S 158S N   CA   1252 1253   38.65  -35.00                  -35.00\n",
      "   11  13058 231S 232F C   N    1800 1802 -105.69  -63.20   43.10    7.33  -63.20   43.10    7.33\n",
      "   11        232F 232F N   CA   1802 1803  -51.55  -44.30                  -44.30\n",
      "   12  13156 329W 330E C   N    2539 2541   74.32  -69.30  146.70   10.03  -63.60 -154.07   34.65\n",
      "   12        330E 330E N   CA   2541 2542  112.62  142.50                  -40.30\n",
      "   13  13188 361K 362N C   N    2778 2780 -111.28 -119.90   81.20    3.49  -63.20  108.59   11.72\n",
      "   13        362N 362N N   CA   2780 2781   56.26  137.00                  -41.10\n",
      "   14  13279 452L 453Q C   N    3489 3491  -76.65  -73.00    7.43    0.43  -63.80  173.01   26.12\n",
      "   14        453Q 453Q N   CA   3491 3492  147.17  140.70                  -40.30\n",
      "   15  13280 453Q 454D C   N    3498 3500  -93.26  -70.90   58.52    2.01  -63.30  119.44   16.67\n",
      "   15        454D 454D N   CA   3500 3501 -155.62  150.30                  -40.00\n",
      "   16  13281 454D 455D C   N    3506 3508  -80.45  -96.50   40.93    1.72  -63.30  117.80   13.59\n",
      "   16        455D 455D N   CA   3508 3509   76.54  114.20                  -40.00\n",
      "   17  13283 456V 457R C   N    3521 3523  -78.10  -72.10   16.78    1.41  -63.00  168.01   21.72\n",
      "   17        457R 457R N   CA   3523 3524  126.23  141.90                  -41.10\n",
      "   18  13284 457R 458A C   N    3532 3534 -162.38 -134.00   29.03    0.75  -62.50 -166.26   37.07\n",
      "   18        458A 458A N   CA   3534 3535  153.09  147.00                  -40.90\n",
      "   19  13285 458A 459F C   N    3537 3539 -162.59 -124.20   44.08    1.17  -63.20 -179.44   31.22\n",
      "   19        459F 459F N   CA   3539 3540  164.96  143.30                  -44.30\n",
      "   20  13286 459F 460K C   N    3548 3550 -131.31 -118.00   21.13    0.78  -62.90  177.41   27.13\n",
      "   20        460K 460K N   CA   3550 3551  155.51  139.10                  -40.80\n",
      "   21  13287 460K 461E C   N    3557 3559 -122.07 -117.80    4.69    0.14  -63.60 -171.66   22.45\n",
      "   21        461E 461E N   CA   3559 3560  138.73  136.80                  -40.30\n",
      "   22  13288 461E 462A C   N    3566 3568  -81.88  -68.20   44.54    4.04  -62.50  145.11   22.91\n",
      "   22        462A 462A N   CA   3568 3569  102.92  145.30                  -40.90\n",
      "   23  13289 462A 463E C   N    3571 3573 -146.69 -117.80   30.67    1.39  -63.60 -173.64   21.62\n",
      "   23        463E 463E N   CA   3573 3574  126.51  136.80                  -40.30\n",
      "   24  13291 464V 465R C   N    3587 3589 -124.08 -125.20   37.73    1.80  -63.00  156.40   17.97\n",
      "   24        465R 465R N   CA   3589 3590  102.89  140.60                  -41.10\n",
      "   25  13292 465R 466R C   N    3598 3600 -119.24 -125.20    7.51    0.22  -63.00 -174.16   21.92\n",
      "   25        466R 466R N   CA   3600 3601  136.02  140.60                  -41.10\n",
      "   26  13293 466R 467E C   N    3609 3611 -139.51  -63.60   83.62   14.34  -63.60   83.62   14.34\n",
      "   26        467E 467E N   CA   3611 3612  -75.38  -40.30                  -40.30\n",
      "   27  13294 467E 468L C   N    3618 3620  -69.77  -70.70   18.02    1.40  -63.50  159.33   22.49\n",
      "   27        468L 468L N   CA   3620 3621  159.59  141.60                  -41.20\n",
      "   28  13295 468L 469E C   N    3626 3628  -87.67  -69.30   22.99    1.30  -63.60  165.15   23.37\n",
      "   28        469E 469E N   CA   3628 3629  156.32  142.50                  -40.30\n",
      "   29  13296 469E 470A C   N    3635 3637 -138.59 -134.00    6.50    0.20  -62.50 -176.03   34.23\n",
      "   29        470A 470A N   CA   3637 3638  151.60  147.00                  -40.90\n",
      "   30  13297 470A 471I C   N    3640 3642  -63.63  -63.40    7.81    1.29 -120.60 -172.83   12.83\n",
      "   30        471I 471I N   CA   3642 3643  -51.41  -43.60                  130.30\n",
      "   31  13298 471I 472S C   N    3648 3650 -164.69 -136.60   30.72    0.96  -64.10 -169.84   21.03\n",
      "   31        472S 472S N   CA   3650 3651  163.62  151.20                  -35.00\n",
      "   32  13299 472S 473L C   N    3654 3656 -116.56 -108.50   30.96    1.58  -63.50  165.15   26.21\n",
      "   32        473L 473L N   CA   3656 3657  162.40  132.50                  -41.20\n",
      "   33  13301 474V 475R C   N    3669 3671 -166.56 -125.20   89.35    3.25  -63.00  143.34   25.74\n",
      "   33        475R 475R N   CA   3671 3672 -140.20  140.60                  -41.10\n",
      "   34  13302 475R 476E C   N    3680 3682  -20.09  -69.30   49.67    3.52  -63.60 -178.66   26.64\n",
      "   34        476E 476E N   CA   3682 3683  135.74  142.50                  -40.30\n",
      "   35  13303 476E 477R C   N    3689 3691 -103.35 -125.20   32.55    1.80  -63.00  159.37   24.32\n",
      "   35        477R 477R N   CA   3691 3692  164.73  140.60                  -41.10\n",
      "   36  13304 477R 478L C   N    3700 3702  -97.12 -108.50   49.91    2.56  -63.50  129.54   16.39\n",
      "   36        478L 478L N   CA   3702 3703   83.90  132.50                  -41.20\n",
      "   37  13306 479I 480G C   N    3716 3718  -55.43  -62.40   12.12    1.66  -80.20  137.04    6.64\n",
      "   37        480G 480G N   CA   3718 3719  -51.12  -41.20                  174.10\n",
      "   38  13307 480G 481Q C   N    3720 3722 -125.77 -121.10   29.81    1.49  -63.80  162.81   21.29\n",
      "   38        481Q 481Q N   CA   3722 3723  110.26  139.70                  -40.30\n",
      "   39  13309 482L 483S C   N    3737 3739  -59.32  -72.40   27.70    1.28  -64.10  163.06   12.10\n",
      "   39        483S 483S N   CA   3739 3740  127.99  152.40                  -35.00\n",
      "   40  13310 483S 484K C   N    3743 3745  -89.21  -70.20   51.96    4.17  -62.90  135.42   16.33\n",
      "   40        484K 484K N   CA   3745 3746   92.04  140.40                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   13   48  138  238  331  332  397  502  613  611\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00202.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      485     3762     3762\n",
      "openf___224_> Open           EAO00202.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     89437713   87341.516    85.294\n",
      "read_mo_297_> Segments, residues, atoms:        1      485     3762\n",
      "read_mo_298_> Segment:        1  1:A  485:A     3762\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     3762     3762        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    22222\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    24895\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    26327\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    27906\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    29602\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    31568\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    34587\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    36710\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    38160\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    41001\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    42797\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     3762    44542\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.7608        2  2\n",
      "iupac_m_487_> NH1/2 swapped:      170.9859        6  6\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.1054        7  7\n",
      "iupac_m_487_> NH1/2 swapped:      176.0583       13  13\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.6041       37  37\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.6736       48  48\n",
      "iupac_m_486_> OE1/2 will be swapped:     -167.2389       49  49\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.1698       50  50\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.6242       70  70\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.7048       81  81\n",
      "iupac_m_487_> NH1/2 swapped:     -177.9121       86  86\n",
      "iupac_m_486_> OE1/2 will be swapped:      176.4399       95  95\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5122      115  115\n",
      "iupac_m_487_> NH1/2 swapped:      175.4562      129  129\n",
      "iupac_m_487_> NH1/2 swapped:      179.6523      153  153\n",
      "iupac_m_487_> NH1/2 swapped:      178.8916      161  161\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.4821      162  162\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -144.5667      178  178\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.3094      197  197\n",
      "iupac_m_487_> NH1/2 swapped:      171.5741      201  201\n",
      "iupac_m_486_> OE1/2 will be swapped:     -104.9623      209  209\n",
      "iupac_m_487_> NH1/2 swapped:     -176.2944      213  213\n",
      "iupac_m_487_> NH1/2 swapped:     -176.6757      235  235\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.7759      240  240\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.8319      244  244\n",
      "iupac_m_487_> NH1/2 swapped:     -177.6768      251  251\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.4898      254  254\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      173.3321      309  309\n",
      "iupac_m_487_> NH1/2 swapped:     -177.2816      310  310\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      108.0854      311  311\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      179.0947      315  315\n",
      "iupac_m_486_> OE1/2 will be swapped:     -162.0194      323  323\n",
      "iupac_m_486_> OE1/2 will be swapped:      149.3683      330  330\n",
      "iupac_m_487_> NH1/2 swapped:      179.8916      354  354\n",
      "iupac_m_487_> NH1/2 swapped:     -172.4895      360  360\n",
      "iupac_m_487_> NH1/2 swapped:     -175.8343      365  365\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -103.5662      373  373\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.0468      392  392\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      167.5343      403  403\n",
      "iupac_m_487_> NH1/2 swapped:     -176.5095      432  432\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.6341      436  436\n",
      "iupac_m_487_> NH1/2 swapped:     -175.8032      438  438\n",
      "iupac_m_487_> NH1/2 swapped:      178.7477      446  446\n",
      "iupac_m_485_> OD1/2 will be swapped:      168.8052      455  455\n",
      "iupac_m_487_> NH1/2 swapped:      178.7008      457  457\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.0783      459  459\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -154.4121      461  461\n",
      "iupac_m_487_> NH1/2 swapped:     -165.0756      465  465\n",
      "iupac_m_487_> NH1/2 swapped:      170.8816      466  466\n",
      "iupac_m_486_> OE1/2 will be swapped:     -130.3551      467  467\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.3798      469  469\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.6055      476  476\n",
      "iupac_m_487_> NH1/2 swapped:     -131.4085      477  477\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   752892\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -53401.2500\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -53401.250000\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : C:\\Program Files\\Modeller10.0/modlib/surf5.de\n",
      "Pair library             : C:\\Program Files\\Modeller10.0/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    79.773003\n",
      "Sequence length          :          485\n",
      "Compactness              :     0.156235\n",
      "Native energy (pair)     :  -529.796312\n",
      "Native energy (surface)  :   -17.392444\n",
      "Native energy (combined) :   -12.392559\n",
      "Z score (pair)           :   -13.193046\n",
      "Z score (surface)        :    -7.524638\n",
      "Z score (combined)       :   -13.784421\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      485\n",
      "Number of all, selected real atoms                :     3762    3762\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    44542   44542\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     7968\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3123.1716\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    3826       0      0   0.006   0.006      38.090       1.000\n",
      " 2 Bond angle potential               :    5172       0     17   2.092   2.092      447.11       1.000\n",
      " 3 Stereochemical cosine torsion poten:    2317       0     66  48.193  48.193      865.10       1.000\n",
      " 4 Stereochemical improper torsion pot:    1512       0      1   1.446   1.446      69.987       1.000\n",
      " 5 Soft-sphere overlap restraints     :    7968       0      0   0.002   0.002      3.3631       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :   10213       1      7   0.168   0.168      182.53       1.000\n",
      "10 Distance restraints 2 (N-O)        :   10269       6     14   0.209   0.209      311.10       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     484       0      5   4.207   4.207      100.31       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     403       0      8  63.723  63.723      93.537       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     295       0      4  75.912  75.912      135.41       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:     140       0      1  80.704  80.704      103.02       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      64       0      0  97.306  97.306      46.539       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    6069       0      0   0.397   0.397      196.49       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     483      38     58  28.364  50.281      187.34       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    3295       0      3   0.824   0.824      343.25       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           EAO00202.V99990005\n",
      "# Heavy relative violation of each residue is written to: EAO00202.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   48501.0312\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  16034  58M  66G CA  CA    480  549   17.64   13.81    3.83    4.70   13.81    3.83    4.70\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33096 369G 261N N   O    2844 2032    6.53    3.92    2.61    5.61    3.92    2.61    5.61\n",
      "    2  33120 370T 261N N   O    2848 2032    6.94    4.16    2.78    5.62    4.16    2.78    5.62\n",
      "    3  33145 371V 261N N   O    2855 2032    9.79    7.34    2.45    5.34    7.34    2.45    5.34\n",
      "    4  35169 480G 302I N   O    3718 2323   12.04    9.37    2.67    4.57    9.37    2.67    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12828   1M   2E C   N       7    9 -106.96  -63.60   88.80   14.24  -63.60   88.80   14.24\n",
      "    1          2E   2E N   CA      9   10 -117.80  -40.30                  -40.30\n",
      "    2  12830   3N   4R C   N      24   26  -73.52  -72.10   10.03    0.74  -63.00  167.40   23.33\n",
      "    2          4R   4R N   CA     26   27  151.83  141.90                  -41.10\n",
      "    3  12831   4R   5L C   N      35   37  -93.15 -108.50   24.21    1.11  -63.50  157.79   20.40\n",
      "    3          5L   5L N   CA     37   38  113.78  132.50                  -41.20\n",
      "    4  12832   5L   6R C   N      43   45 -159.29  -63.00  108.37   19.73  -63.00  108.37   19.73\n",
      "    4          6R   6R N   CA     45   46  -90.83  -41.10                  -41.10\n",
      "    5  12833   6R   7D C   N      54   56  -68.77  -70.90   11.29    0.66  -63.30  158.71   19.92\n",
      "    5          7D   7D N   CA     56   57  161.38  150.30                  -40.00\n",
      "    6  12842  15H  16A C   N     127  129  -86.99  -62.50   30.87    4.93   55.40  154.63   18.37\n",
      "    6         16A  16A N   CA    129  130  -22.10  -40.90                   38.20\n",
      "    7  12891  64Q  65E C   N     537  539  179.95 -117.80   66.59    1.99  -63.60 -162.72   32.57\n",
      "    7         65E  65E N   CA    539  540  160.45  136.80                  -40.30\n",
      "    8  12895  68H  69S C   N     567  569  -78.53  -72.40   12.27    0.92  -64.10  177.36   12.06\n",
      "    8         69S  69S N   CA    569  570  141.77  152.40                  -35.00\n",
      "    9  12897  70E  71L C   N     582  584 -146.25 -108.50   70.77    4.03  -63.50  140.74   17.44\n",
      "    9         71L  71L N   CA    584  585   72.64  132.50                  -41.20\n",
      "   10  12898  71L  72P C   N     590  592  -57.58  -58.70   16.88    1.31  -64.50  165.61   12.73\n",
      "   10         72P  72P N   CA    592  593  -47.34  -30.50                  147.20\n",
      "   11  13058 231S 232F C   N    1800 1802 -107.11  -63.20   44.23    7.42  -63.20   44.23    7.42\n",
      "   11        232F 232F N   CA   1802 1803  -49.62  -44.30                  -44.30\n",
      "   12  13087 260L 261N C   N    2023 2025  -62.13  -63.20    1.88    0.29  -71.20  177.88   11.65\n",
      "   12        261N 261N N   CA   2025 2026  -39.55  -41.10                  142.80\n",
      "   13  13088 261N 262Y C   N    2031 2033 -137.75 -124.30   61.54    3.84   55.90  170.17   20.44\n",
      "   13        262Y 262Y N   CA   2033 2034   75.35  135.40                   39.50\n",
      "   14  13156 329W 330E C   N    2539 2541   90.56  -63.60  155.91   23.58  -63.60  155.91   23.58\n",
      "   14        330E 330E N   CA   2541 2542  -63.58  -40.30                  -40.30\n",
      "   15  13157 330E 331G C   N    2548 2550  -60.29  -62.40   10.18    1.63   82.20  147.93   11.09\n",
      "   15        331G 331G N   CA   2550 2551  -31.24  -41.20                    8.50\n",
      "   16  13188 361K 362N C   N    2778 2780 -111.78 -119.90   79.74    3.43  -63.20  110.07   11.88\n",
      "   16        362N 362N N   CA   2780 2781   57.67  137.00                  -41.10\n",
      "   17  13279 452L 453Q C   N    3489 3491  137.77 -121.10  111.20    3.46  -63.80 -152.65   37.55\n",
      "   17        453Q 453Q N   CA   3491 3492 -174.06  139.70                  -40.30\n",
      "   18  13280 453Q 454D C   N    3498 3500  -79.03  -70.90   31.46    1.19  -63.30  140.20   18.29\n",
      "   18        454D 454D N   CA   3500 3501 -179.32  150.30                  -40.00\n",
      "   19  13281 454D 455D C   N    3506 3508  -82.58  -96.50   23.76    1.00  -63.30  136.31   15.75\n",
      "   19        455D 455D N   CA   3508 3509   94.94  114.20                  -40.00\n",
      "   20  13283 456V 457R C   N    3521 3523  -93.38  -72.10   59.09    4.96  -63.00  131.43   15.93\n",
      "   20        457R 457R N   CA   3523 3524   86.77  141.90                  -41.10\n",
      "   21  13284 457R 458A C   N    3532 3534 -124.32 -134.00    9.76    0.37  -62.50 -178.30   33.08\n",
      "   21        458A 458A N   CA   3534 3535  148.24  147.00                  -40.90\n",
      "   22  13285 458A 459F C   N    3537 3539 -127.75 -124.20   37.67    1.69  -63.20  149.54   24.93\n",
      "   22        459F 459F N   CA   3539 3540 -179.19  143.30                  -44.30\n",
      "   23  13286 459F 460K C   N    3548 3550   43.64   56.60   56.13    3.15  -62.90  171.20   28.46\n",
      "   23        460K 460K N   CA   3550 3551   93.22   38.60                  -40.80\n",
      "   24  13288 461E 462A C   N    3566 3568   84.76 -134.00  142.27    4.09  -62.50 -146.20   32.88\n",
      "   24        462A 462A N   CA   3568 3569  164.10  147.00                  -40.90\n",
      "   25  13289 462A 463E C   N    3571 3573  -93.04 -117.80   28.93    0.85  -63.60  164.79   20.49\n",
      "   25        463E 463E N   CA   3573 3574  121.84  136.80                  -40.30\n",
      "   26  13291 464V 465R C   N    3587 3589  -72.59  -72.10   24.09    1.84  -63.00  153.22   21.35\n",
      "   26        465R 465R N   CA   3589 3590  165.99  141.90                  -41.10\n",
      "   27  13292 465R 466R C   N    3598 3600 -113.68 -125.20   23.22    0.82  -63.00  169.30   19.99\n",
      "   27        466R 466R N   CA   3600 3601  120.44  140.60                  -41.10\n",
      "   28  13293 466R 467E C   N    3609 3611 -157.31 -117.80   82.85    4.55  -63.60  140.20   16.51\n",
      "   28        467E 467E N   CA   3611 3612   63.98  136.80                  -40.30\n",
      "   29  13294 467E 468L C   N    3618 3620 -126.35 -108.50   43.13    2.11  -63.50  159.90   26.09\n",
      "   29        468L 468L N   CA   3620 3621  171.77  132.50                  -41.20\n",
      "   30  13296 469E 470A C   N    3635 3637   63.29  -68.20  141.62   13.49  -62.50  174.68   27.11\n",
      "   30        470A 470A N   CA   3637 3638 -162.10  145.30                  -40.90\n",
      "   31  13298 471I 472S C   N    3648 3650 -127.18 -136.60   94.93    5.00  -64.10  101.36   11.89\n",
      "   31        472S 472S N   CA   3650 3651 -114.34  151.20                  -35.00\n",
      "   32  13299 472S 473L C   N    3654 3656  -96.45  -63.50   67.44   11.39  -63.50   67.44   11.39\n",
      "   32        473L 473L N   CA   3656 3657 -100.04  -41.20                  -41.20\n",
      "   33  13301 474V 475R C   N    3669 3671 -121.75 -125.20   45.95    2.29  -63.00  144.93   23.60\n",
      "   33        475R 475R N   CA   3671 3672 -173.58  140.60                  -41.10\n",
      "   34  13303 476E 477R C   N    3689 3691  169.63 -125.20   89.44    2.70  -63.00  172.97   31.15\n",
      "   34        477R 477R N   CA   3691 3692 -158.13  140.60                  -41.10\n",
      "   35  13304 477R 478L C   N    3700 3702  -78.01  -70.70    9.36    0.94  -63.50  177.56   23.86\n",
      "   35        478L 478L N   CA   3702 3703  135.77  141.60                  -41.20\n",
      "   36  13307 480G 481Q C   N    3720 3722 -101.91 -121.10   19.97    0.65  -63.80  178.60   24.42\n",
      "   36        481Q 481Q N   CA   3722 3723  134.19  139.70                  -40.30\n",
      "   37  13308 481Q 482L C   N    3729 3731 -136.79 -108.50   41.13    2.32  -63.50  161.45   19.78\n",
      "   37        482L 482L N   CA   3731 3732  102.65  132.50                  -41.20\n",
      "   38  13310 483S 484K C   N    3743 3745 -179.97 -118.00   62.64    2.22  -62.90 -152.79   33.83\n",
      "   38        484K 484K N   CA   3745 3746  148.23  139.10                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   18   50  169  239  351  378  433  504  615  637\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           EAO00202.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      485     3762     3762\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "EAO00202.B99990001.pdb        2546.76245   -54215.82422        1.00000\n",
      "EAO00202.B99990002.pdb        2607.15234   -53857.07031        1.00000\n",
      "EAO00202.B99990003.pdb        2768.77051   -53809.74609        1.00000\n",
      "EAO00202.B99990004.pdb        2696.59424   -53850.43359        1.00000\n",
      "EAO00202.B99990005.pdb        3123.17163   -53401.25000        1.00000\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     89629055   87528.375    85.477\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     89629583   87528.891    85.477\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     89678183   87576.352    85.524\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     89691483   87589.336    85.536\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v0}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     89960749   87852.297    85.793\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     93521872   91329.953    89.189\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     93524114   91332.141    89.192\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     93525564   91333.562    89.193\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     93528464   91336.391    89.196\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     93534264   91342.055    89.201\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:    106972663  104465.492   102.017\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:    106984263  104476.820   102.028\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:    107007463  104499.477   102.050\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:    107053863  104544.789   102.095\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:    107146663  104635.414   102.183\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:    113865863  111197.133   108.591\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           ali/EAO00203.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     89963069   87854.562    85.795\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     89964519   87855.977    85.797\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     89967419   87858.805    85.800\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     89973219   87864.469    85.805\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     89974759   87865.977    85.807\n",
      "\n",
      "Read the alignment from file       : ali/EAO00203.ali\n",
      "\n",
      "Total number of alignment positions:   386\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1   EAO00203     386      1    EAO00203\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     89976636   87867.812    85.808\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     90301811   88185.359    86.119\n",
      "profile_iteration_> processing sequence:       1    386      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     90302889   88186.414    86.120\n",
      "profile_iteration_> processing sequence:    6720    386      1     3.8125000     0.0005673     0.0000022  1763\n",
      "profile_iteration_> processing sequence:   13439    386      1     7.5312500     0.0005604     0.0000022  1784\n",
      "profile_iteration_> processing sequence:   20158    386      1    11.2187500     0.0005565     0.0000022  1797\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     90303264   88186.781    86.120\n",
      "profile_iteration_> processing sequence:   26877    386      1    14.9843750     0.0005575     0.0000022  1794\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     90303500   88187.016    86.120\n",
      "profile_iteration_> processing sequence:   33596    386      1    18.7031250     0.0005567     0.0000022  1796\n",
      "profile_iteration_> processing sequence:   40315    386      1    22.3906250     0.0005554     0.0000022  1801\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     90306450   88189.891    86.123\n",
      "profile_iteration_> processing sequence:   47034    386      1    26.1250000     0.0005554     0.0000022  1800\n",
      "profile_iteration_> processing sequence:   53753    386      1    29.9375000     0.0005569     0.0000022  1796\n",
      "profile_iteration_> processing sequence:   60472    386      1    33.6562500     0.0005566     0.0000022  1797\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     90309960   88193.320    86.126\n",
      "profile_iteration_> processing sequence:   67191    386      1    37.5312500     0.0005586     0.0000022  1790\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    21.09059\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    235.5424\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0200,      121,        0.0063,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00015   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00015   0.00000\n",
      "Z:         1  -2.85000   0.00015   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00045   0.00000\n",
      "Z:         1  -2.55000   0.00030   0.00001\n",
      "Z:         1  -2.45000   0.00030   0.00004\n",
      "Z:         1  -2.35000   0.00060   0.00016\n",
      "Z:         1  -2.25000   0.00104   0.00055\n",
      "Z:         1  -2.15000   0.00327   0.00163\n",
      "Z:         1  -2.05000   0.00357   0.00416\n",
      "Z:         1  -1.95000   0.00938   0.00933\n",
      "Z:         1  -1.85000   0.01875   0.01872\n",
      "Z:         1  -1.75000   0.03289   0.03400\n",
      "Z:         1  -1.65000   0.06147   0.05658\n",
      "Z:         1  -1.55000   0.08572   0.08721\n",
      "Z:         1  -1.45000   0.12740   0.12565\n",
      "Z:         1  -1.35000   0.16133   0.17059\n",
      "Z:         1  -1.25000   0.21431   0.21981\n",
      "Z:         1  -1.15000   0.25449   0.27052\n",
      "Z:         1  -1.05000   0.31968   0.31973\n",
      "Z:         1  -0.95000   0.35079   0.36469\n",
      "Z:         1  -0.85000   0.39722   0.40316\n",
      "Z:         1  -0.75000   0.42073   0.43360\n",
      "Z:         1  -0.65000   0.51003   0.45519\n",
      "Z:         1  -0.55000   0.46092   0.46779\n",
      "Z:         1  -0.45000   0.45452   0.47182\n",
      "Z:         1  -0.35000   0.48607   0.46811\n",
      "Z:         1  -0.25000   0.48473   0.45775\n",
      "Z:         1  -0.15000   0.45273   0.44196\n",
      "Z:         1  -0.05000   0.40526   0.42194\n",
      "Z:         1   0.05000   0.37743   0.39889\n",
      "Z:         1   0.15000   0.35213   0.37383\n",
      "Z:         1   0.25000   0.35733   0.34769\n",
      "Z:         1   0.35000   0.30658   0.32121\n",
      "Z:         1   0.45000   0.29304   0.29500\n",
      "Z:         1   0.55000   0.25345   0.26953\n",
      "Z:         1   0.65000   0.27221   0.24513\n",
      "Z:         1   0.75000   0.21892   0.22205\n",
      "Z:         1   0.85000   0.20315   0.20043\n",
      "Z:         1   0.95000   0.17383   0.18036\n",
      "Z:         1   1.05000   0.15865   0.16185\n",
      "Z:         1   1.15000   0.14630   0.14489\n",
      "Z:         1   1.25000   0.13261   0.12944\n",
      "Z:         1   1.35000   0.11415   0.11542\n",
      "Z:         1   1.45000   0.10031   0.10275\n",
      "Z:         1   1.55000   0.09376   0.09133\n",
      "Z:         1   1.65000   0.08290   0.08109\n",
      "Z:         1   1.75000   0.07069   0.07191\n",
      "Z:         1   1.85000   0.06310   0.06371\n",
      "Z:         1   1.95000   0.05715   0.05639\n",
      "Z:         1   2.05000   0.05075   0.04988\n",
      "Z:         1   2.15000   0.04584   0.04409\n",
      "Z:         1   2.25000   0.03855   0.03895\n",
      "Z:         1   2.35000   0.03944   0.03439\n",
      "Z:         1   2.45000   0.03334   0.03035\n",
      "Z:         1   2.55000   0.02560   0.02678\n",
      "Z:         1   2.65000   0.02366   0.02361\n",
      "Z:         1   2.75000   0.02173   0.02082\n",
      "Z:         1   2.85000   0.01890   0.01835\n",
      "Z:         1   2.95000   0.01979   0.01617\n",
      "Z:         1   3.05000   0.01310   0.01424\n",
      "Z:         1   3.15000   0.01146   0.01255\n",
      "Z:         1   3.25000   0.01116   0.01105\n",
      "Z:         1   3.35000   0.00923   0.00973\n",
      "Z:         1   3.45000   0.00804   0.00857\n",
      "Z:         1   3.55000   0.00908   0.00754\n",
      "Z:         1   3.65000   0.00595   0.00664\n",
      "Z:         1   3.75000   0.00521   0.00584\n",
      "Z:         1   3.85000   0.00685   0.00514\n",
      "Z:         1   3.95000   0.00566   0.00453\n",
      "Z:         1   4.05000   0.00595   0.00398\n",
      "Z:         1   4.15000   0.00551   0.00350\n",
      "Z:         1   4.25000   0.00402   0.00308\n",
      "Z:         1   4.35000   0.00372   0.00271\n",
      "Z:         1   4.45000   0.00268   0.00239\n",
      "Z:         1   4.55000   0.00402   0.00210\n",
      "Z:         1   4.65000   0.00283   0.00185\n",
      "Z:         1   4.75000   0.00238   0.00163\n",
      "Z:         1   4.85000   0.00134   0.00143\n",
      "Z:         1   4.95000   0.00208   0.00126\n",
      "Z:         1   5.05000   0.00238   0.00111\n",
      "Z:         1   5.15000   0.00045   0.00097\n",
      "Z:         1   5.25000   0.00119   0.00086\n",
      "Z:         1   5.35000   0.00104   0.00075\n",
      "Z:         1   5.45000   0.00060   0.00066\n",
      "Z:         1   5.55000   0.00104   0.00058\n",
      "Z:         1   5.65000   0.00089   0.00051\n",
      "Z:         1   5.75000   0.00060   0.00045\n",
      "Z:         1   5.85000   0.00030   0.00040\n",
      "Z:         1   5.95000   0.00045   0.00035\n",
      "Z:         1   6.05000   0.00030   0.00031\n",
      "Z:         1   6.15000   0.00060   0.00027\n",
      "Z:         1   6.25000   0.00074   0.00024\n",
      "Z:         1   6.35000   0.00015   0.00021\n",
      "Z:         1   6.45000   0.00060   0.00018\n",
      "Z:         1   6.55000   0.00045   0.00016\n",
      "Z:         1   6.65000   0.00015   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00045   0.00011\n",
      "Z:         1   6.95000   0.00030   0.00010\n",
      "Z:         1   7.05000   0.00015   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00015   0.00005\n",
      "Z:         1   7.55000   0.00030   0.00004\n",
      "Z:         1   7.65000   0.00015   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00030   0.00003\n",
      "Z:         1   7.95000   0.00015   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:     90635760   88511.484    86.437\n",
      "> 5czyA                      1   32625    5350     449     386   29.41    0.12E-03       2   115   211   333    44   162\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    37.531250\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000559\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1791.040405\n",
      "profile_iteration__> Statistics              [sec]:     0.046875\n",
      "profile_iteration__> Add to profile          [sec]:     0.015625\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000560\n",
      "profile_iteration__> Overall performance [seq/sec]:  1785.833862\n",
      "openf___224_> Open           buildprofile/EAO00203build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     90318624   88201.781    86.135\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     90320164   88203.281    86.136\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     90320620   88203.734    86.136\n",
      "openf___224_> Open           buildprofile/EAO00203build_profile.ali\n"
     ]
    },
    {
     "ename": "HTTPError",
     "evalue": "HTTP Error 404: Not Found",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mHTTPError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-30-972361fa1b80>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[1;32mif\u001b[0m \u001b[0m__name__\u001b[0m \u001b[1;33m==\u001b[0m \u001b[1;34m\"__main__\"\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 2\u001b[1;33m     \u001b[0mmain\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[1;32m<ipython-input-29-0612a4c08d03>\u001b[0m in \u001b[0;36mmain\u001b[1;34m()\u001b[0m\n\u001b[0;32m     24\u001b[0m             \u001b[0mbuild_profile\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mID\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     25\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 26\u001b[1;33m             \u001b[0mlength\u001b[0m\u001b[1;33m,\u001b[0m\u001b[0midentity\u001b[0m\u001b[1;33m,\u001b[0m\u001b[0mx\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mprf\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mids\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     27\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     28\u001b[0m             \u001b[1;32mif\u001b[0m \u001b[0mx\u001b[0m \u001b[1;33m==\u001b[0m \u001b[1;34m'cannot'\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m<ipython-input-26-d88ee67be28e>\u001b[0m in \u001b[0;36mprf\u001b[1;34m(ids)\u001b[0m\n\u001b[0;32m     21\u001b[0m             \u001b[1;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mdf\u001b[0m\u001b[1;33m)\u001b[0m \u001b[1;33m>\u001b[0m \u001b[1;36m1\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     22\u001b[0m                 \u001b[0mx\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mpdbids\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 23\u001b[1;33m                 \u001b[0mpdbDn\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mx\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;36m4\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     24\u001b[0m             \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     25\u001b[0m                 \u001b[0mx\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;34m'cannot'\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m<ipython-input-23-05b99af58649>\u001b[0m in \u001b[0;36mpdbDn\u001b[1;34m(pdbid)\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[1;32mdef\u001b[0m \u001b[0mpdbDn\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mpdbid\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 2\u001b[1;33m     \u001b[0murllib\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0murlretrieve\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'http://files.rcsb.org/download/'\u001b[0m\u001b[1;33m+\u001b[0m\u001b[0mpdbid\u001b[0m\u001b[1;33m+\u001b[0m\u001b[1;34m'.pdb'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mpdbid\u001b[0m\u001b[1;33m+\u001b[0m\u001b[1;34m'.pdb'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36murlretrieve\u001b[1;34m(url, filename, reporthook, data)\u001b[0m\n\u001b[0;32m    237\u001b[0m     \u001b[0murl_type\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mpath\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0m_splittype\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0murl\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    238\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 239\u001b[1;33m     \u001b[1;32mwith\u001b[0m \u001b[0mcontextlib\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mclosing\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0murlopen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0murl\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m \u001b[1;32mas\u001b[0m \u001b[0mfp\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    240\u001b[0m         \u001b[0mheaders\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mfp\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0minfo\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    241\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36murlopen\u001b[1;34m(url, data, timeout, cafile, capath, cadefault, context)\u001b[0m\n\u001b[0;32m    212\u001b[0m     \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    213\u001b[0m         \u001b[0mopener\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0m_opener\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 214\u001b[1;33m     \u001b[1;32mreturn\u001b[0m \u001b[0mopener\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mopen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0murl\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    215\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    216\u001b[0m \u001b[1;32mdef\u001b[0m \u001b[0minstall_opener\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mopener\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36mopen\u001b[1;34m(self, fullurl, data, timeout)\u001b[0m\n\u001b[0;32m    521\u001b[0m         \u001b[1;32mfor\u001b[0m \u001b[0mprocessor\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mprocess_response\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mprotocol\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;33m[\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    522\u001b[0m             \u001b[0mmeth\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mgetattr\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mprocessor\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmeth_name\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 523\u001b[1;33m             \u001b[0mresponse\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmeth\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mreq\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mresponse\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    524\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    525\u001b[0m         \u001b[1;32mreturn\u001b[0m \u001b[0mresponse\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36mhttp_response\u001b[1;34m(self, request, response)\u001b[0m\n\u001b[0;32m    630\u001b[0m         \u001b[1;31m# request was successfully received, understood, and accepted.\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    631\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[1;32mnot\u001b[0m \u001b[1;33m(\u001b[0m\u001b[1;36m200\u001b[0m \u001b[1;33m<=\u001b[0m \u001b[0mcode\u001b[0m \u001b[1;33m<\u001b[0m \u001b[1;36m300\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 632\u001b[1;33m             response = self.parent.error(\n\u001b[0m\u001b[0;32m    633\u001b[0m                 'http', request, response, code, msg, hdrs)\n\u001b[0;32m    634\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36merror\u001b[1;34m(self, proto, *args)\u001b[0m\n\u001b[0;32m    559\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mhttp_err\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    560\u001b[0m             \u001b[0margs\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;33m(\u001b[0m\u001b[0mdict\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'default'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'http_error_default'\u001b[0m\u001b[1;33m)\u001b[0m \u001b[1;33m+\u001b[0m \u001b[0morig_args\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 561\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_call_chain\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m*\u001b[0m\u001b[0margs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    562\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    563\u001b[0m \u001b[1;31m# XXX probably also want an abstract factory that knows when it makes\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36m_call_chain\u001b[1;34m(self, chain, kind, meth_name, *args)\u001b[0m\n\u001b[0;32m    492\u001b[0m         \u001b[1;32mfor\u001b[0m \u001b[0mhandler\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mhandlers\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    493\u001b[0m             \u001b[0mfunc\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mgetattr\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mhandler\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmeth_name\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 494\u001b[1;33m             \u001b[0mresult\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mfunc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m*\u001b[0m\u001b[0margs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    495\u001b[0m             \u001b[1;32mif\u001b[0m \u001b[0mresult\u001b[0m \u001b[1;32mis\u001b[0m \u001b[1;32mnot\u001b[0m \u001b[1;32mNone\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    496\u001b[0m                 \u001b[1;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mc:\\users\\paul\\appdata\\local\\programs\\python\\python39\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36mhttp_error_default\u001b[1;34m(self, req, fp, code, msg, hdrs)\u001b[0m\n\u001b[0;32m    639\u001b[0m \u001b[1;32mclass\u001b[0m \u001b[0mHTTPDefaultErrorHandler\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mBaseHandler\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    640\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mhttp_error_default\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mreq\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mfp\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mcode\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmsg\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mhdrs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 641\u001b[1;33m         \u001b[1;32mraise\u001b[0m \u001b[0mHTTPError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mreq\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mfull_url\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mcode\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmsg\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mhdrs\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mfp\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    642\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    643\u001b[0m \u001b[1;32mclass\u001b[0m \u001b[0mHTTPRedirectHandler\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mBaseHandler\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mHTTPError\u001b[0m: HTTP Error 404: Not Found"
     ]
    }
   ],
   "source": [
    "if __name__ == \"__main__\":\n",
    "    main()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
